
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>6666666.540621 - 21 region(s) - antiSMASH results</title>
  <style>img {width: 20px; height: 20px;}</style>
  <link rel="stylesheet" type="text/css" href="css/bacteria.css">
  <meta property="og:title" content="6666666.540621 - 20 record(s) - 21 region(s)" />
  <meta property="og:description" content="">
  <meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
  <meta property="og:image:alt" content="antiSMASH logo" />
  <meta property="og:image:width" content="600" />
  <meta property="og:image:height" content="600" />
  <meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-bb8a1f4b-8d31-4a74-bbf3-16aec2b8ccd5/index.html" />
  <meta property="og:type" content="website" />
  <meta name="twitter:card" content="summary" />
  <meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
  <div id="header">
      <nav>
    <div class="main-link">
     <div>
        <a href="https://antismash.secondarymetabolites.org/">
          <img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
        </a>
       </div>
       <div>
        <a href="https://antismash.secondarymetabolites.org/">
          antiSMASH version 6.0.1
        </a>
      </div>
    </div>
    <div class="custom-description"></div>
    <div class="ancillary-links">
      <div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
        <ul class="dropdown-options">
        
        <li><a href="6666666.540621.zip">Download all results</a></li>
        
          <li><a href="6666666.540621.gbk">Download GenBank summary file</a></li>
        
          <li><a href="bacteria-bb8a1f4b-8d31-4a74-bbf3-16aec2b8ccd5.log">Download log file</a></li>
        
        </ul>
      </div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
      <div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
    </div>
  </nav>
    <div id="buttons">
      <span id="region-type">Select genomic region:</span>
      <div id="regionbuttons">
        <div class="arrow-left" id="prev-region"></div>
        <div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
        <div style="display: flex; flex-wrap: wrap">
        
          
            
            
            <div class="regbutton terpene r7c1">
              <a href="#r7c1">7.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton lassopeptide,phosphonate,ranthipeptide hybrid r17c1">
              <a href="#r17c1">17.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r23c1">
              <a href="#r23c1">23.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r24c1">
              <a href="#r24c1">24.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r31c1">
              <a href="#r31c1">31.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton lanthipeptide-class-i r37c1">
              <a href="#r37c1">37.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton siderophore r45c1">
              <a href="#r45c1">45.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS-like r93c1">
              <a href="#r93c1">93.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r95c1">
              <a href="#r95c1">95.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton LAP,thiopeptide hybrid r98c1">
              <a href="#r98c1">98.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton butyrolactone r101c1">
              <a href="#r101c1">101.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton lanthipeptide-class-v r103c1">
              <a href="#r103c1">103.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r110c1">
              <a href="#r110c1">110.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton RRE-containing r132c1">
              <a href="#r132c1">132.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r155c1">
              <a href="#r155c1">155.1</a>
            </div>
          
            
            
            <div class="regbutton butyrolactone r155c2">
              <a href="#r155c2">155.2</a>
            </div>
          
        
          
            
            
            <div class="regbutton ectoine r157c1">
              <a href="#r157c1">157.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r158c1">
              <a href="#r158c1">158.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r167c1">
              <a href="#r167c1">167.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r173c1">
              <a href="#r173c1">173.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton RiPP-like r175c1">
              <a href="#r175c1">175.1</a>
            </div>
          
        
        </div>
        <div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
      </div>
    </div>
  </div>

  <!-- overview page -->
  
  <div class="page" id="overview">
   
   
     <h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>
   
   <div class="overview-layout">
    <div id="record-tables">
     <div id="single-record-tables">
     <div class="record-overview-header">
        <strong>CC223A_contig_37</strong> (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-7" data-filename="CC223A_contig_37_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-7" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r7c1">
     
     
     <td class="regbutton terpene r7c1">
      <a href="#r7c1">Region&nbsp7.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">4,693</td>
     <td class="digits table-split-left">25,736</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001675/1" target="_blank">murayaquinone</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 10%, #ffffff00 10%)">10%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC223A_contig_40</strong> (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-17" data-filename="CC223A_contig_40_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-17" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r17c1">
     
     
         
     
     <td class="regbutton lassopeptide,phosphonate,ranthipeptide hybrid r17c1">
      <a href="#r17c1">Region&nbsp17.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#phosphonate" target="_blank">phosphonate</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lassopeptide" target="_blank">lassopeptide</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ranthipeptide" target="_blank">ranthipeptide</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">48,794</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000926/1" target="_blank">rhizocticin A</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 16%, #ffffff00 16%)">16%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC223A_contig_32</strong> (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-23" data-filename="CC223A_contig_32_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-23" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r23c1">
     
     
     <td class="regbutton T1PKS r23c1">
      <a href="#r23c1">Region&nbsp23.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">8,162</td>
     <td class="digits table-split-left">51,071</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00024_CC223A_..</strong> (original name was: CC223A_contig_149) (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-24" data-filename="c00024_CC223A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-24" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r24c1">
     
     
     <td class="regbutton NRPS r24c1">
      <a href="#r24c1">Region&nbsp24.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,435</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC223A_contig_25</strong> (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-31" data-filename="CC223A_contig_25_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-31" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r31c1">
     
     
     <td class="regbutton terpene r31c1">
      <a href="#r31c1">Region&nbsp31.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">63,676</td>
     <td class="digits table-split-left">76,965</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000657/1" target="_blank">2-methylisoborneol</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC223A_contig_6</strong> (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-37" data-filename="CC223A_contig_6_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-37" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r37c1">
     
     
     <td class="regbutton lanthipeptide-class-i r37c1">
      <a href="#r37c1">Region&nbsp37.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-i" target="_blank">lanthipeptide-class-i</a>
     </td>
     <td class="digits">1,850</td>
     <td class="digits table-split-left">25,759</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC223A_contig_3</strong> (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-45" data-filename="CC223A_contig_3_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-45" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r45c1">
     
     
     <td class="regbutton siderophore r45c1">
      <a href="#r45c1">Region&nbsp45.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">124,109</td>
     <td class="digits table-split-left">137,716</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001593/1" target="_blank">ficellomycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 5%, #ffffff00 5%)">5%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC223A_contig_44</strong> (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-93" data-filename="CC223A_contig_44_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-93" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r93c1">
     
     
     <td class="regbutton NRPS-like r93c1">
      <a href="#r93c1">Region&nbsp93.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">2,674</td>
     <td class="digits table-split-left">45,256</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC223A_contig_5</strong> (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-95" data-filename="CC223A_contig_5_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-95" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r95c1">
     
     
     <td class="regbutton terpene r95c1">
      <a href="#r95c1">Region&nbsp95.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">104,656</td>
     <td class="digits table-split-left">130,302</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000664/1" target="_blank">isorenieratene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00098_CC223A_..</strong> (original name was: CC223A_contig_108) (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-98" data-filename="c00098_CC223A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-98" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r98c1">
     
     
         
     
     <td class="regbutton LAP,thiopeptide hybrid r98c1">
      <a href="#r98c1">Region&nbsp98.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lap" target="_blank">LAP</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thiopeptide" target="_blank">thiopeptide</a>
     </td>
     <td class="digits">3,597</td>
     <td class="digits table-split-left">19,058</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC223A_contig_22</strong> (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-101" data-filename="CC223A_contig_22_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-101" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r101c1">
     
     
     <td class="regbutton butyrolactone r101c1">
      <a href="#r101c1">Region&nbsp101.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
     </td>
     <td class="digits">14,569</td>
     <td class="digits table-split-left">25,624</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC223A_contig_66</strong> (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-103" data-filename="CC223A_contig_66_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-103" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r103c1">
     
     
     <td class="regbutton lanthipeptide-class-v r103c1">
      <a href="#r103c1">Region&nbsp103.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-v" target="_blank">lanthipeptide-class-v</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">30,337</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001856/1" target="_blank">caniferolide A / caniferolide B / caniferolide C / caniferolide D</a></td>
      <td>Polyketide:Modular type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 8%, #ffffff00 8%)">8%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC223A_contig_46</strong> (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-110" data-filename="CC223A_contig_46_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-110" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r110c1">
     
     
     <td class="regbutton NRPS r110c1">
      <a href="#r110c1">Region&nbsp110.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">45,272</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000359/1" target="_blank">fuscachelin A / fuscachelin B / fuscachelin C</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00132_CC223A_..</strong> (original name was: CC223A_contig_163) (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-132" data-filename="c00132_CC223A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-132" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r132c1">
     
     
     <td class="regbutton RRE-containing r132c1">
      <a href="#r132c1">Region&nbsp132.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#rre-containing" target="_blank">RRE-containing</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">3,419</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC223A_contig_10</strong> (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-155" data-filename="CC223A_contig_10_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-155" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r155c1">
     
     
     <td class="regbutton T1PKS r155c1">
      <a href="#r155c1">Region&nbsp155.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">6,406</td>
     <td class="digits table-split-left">64,045</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000035/1" target="_blank">chalcomycin A</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r155c2">
     
     
     <td class="regbutton butyrolactone r155c2">
      <a href="#r155c2">Region&nbsp155.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
     </td>
     <td class="digits">81,271</td>
     <td class="digits table-split-left">92,047</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC223A_contig_84</strong> (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-157" data-filename="CC223A_contig_84_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-157" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r157c1">
     
     
     <td class="regbutton ectoine r157c1">
      <a href="#r157c1">Region&nbsp157.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">9,962</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000853/1" target="_blank">ectoine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC223A_contig_55</strong> (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-158" data-filename="CC223A_contig_55_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-158" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r158c1">
     
     
     <td class="regbutton T1PKS r158c1">
      <a href="#r158c1">Region&nbsp158.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">35,211</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC223A_contig_48</strong> (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-167" data-filename="CC223A_contig_48_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-167" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r167c1">
     
     
     <td class="regbutton T1PKS r167c1">
      <a href="#r167c1">Region&nbsp167.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">27,556</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001377/1" target="_blank">uncialamycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC223A_contig_56</strong> (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-173" data-filename="CC223A_contig_56_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-173" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r173c1">
     
     
     <td class="regbutton NRPS r173c1">
      <a href="#r173c1">Region&nbsp173.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">24,060</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001503/1" target="_blank">amycolamycin A / amycolamycin B</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC223A_contig_33</strong> (Nocardiopsis sp. CC223A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-175" data-filename="CC223A_contig_33_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-175" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r175c1">
     
     
     <td class="regbutton RiPP-like r175c1">
      <a href="#r175c1">Region&nbsp175.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">8,111</td>
     <td class="digits table-split-left">18,416</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div>
     </div>
     
     <div id="compact-record-table" style="display: none;">
      <div class="record-table">
       <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 
 
  <tr class="linked-row odd" data-anchor="#r7c1">
     
     
     <td class="regbutton terpene r7c1">
      <a href="#r7c1">Region&nbsp7.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">4,693</td>
     <td class="digits table-split-left">25,736</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001675/1" target="_blank">murayaquinone</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 10%, #ffffff00 10%)">10%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r17c1">
     
     
         
     
     <td class="regbutton lassopeptide,phosphonate,ranthipeptide hybrid r17c1">
      <a href="#r17c1">Region&nbsp17.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#phosphonate" target="_blank">phosphonate</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lassopeptide" target="_blank">lassopeptide</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ranthipeptide" target="_blank">ranthipeptide</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">48,794</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000926/1" target="_blank">rhizocticin A</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 16%, #ffffff00 16%)">16%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r23c1">
     
     
     <td class="regbutton T1PKS r23c1">
      <a href="#r23c1">Region&nbsp23.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">8,162</td>
     <td class="digits table-split-left">51,071</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r24c1">
     
     
     <td class="regbutton NRPS r24c1">
      <a href="#r24c1">Region&nbsp24.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,435</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r31c1">
     
     
     <td class="regbutton terpene r31c1">
      <a href="#r31c1">Region&nbsp31.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">63,676</td>
     <td class="digits table-split-left">76,965</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000657/1" target="_blank">2-methylisoborneol</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r37c1">
     
     
     <td class="regbutton lanthipeptide-class-i r37c1">
      <a href="#r37c1">Region&nbsp37.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-i" target="_blank">lanthipeptide-class-i</a>
     </td>
     <td class="digits">1,850</td>
     <td class="digits table-split-left">25,759</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r45c1">
     
     
     <td class="regbutton siderophore r45c1">
      <a href="#r45c1">Region&nbsp45.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">124,109</td>
     <td class="digits table-split-left">137,716</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001593/1" target="_blank">ficellomycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 5%, #ffffff00 5%)">5%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r93c1">
     
     
     <td class="regbutton NRPS-like r93c1">
      <a href="#r93c1">Region&nbsp93.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">2,674</td>
     <td class="digits table-split-left">45,256</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r95c1">
     
     
     <td class="regbutton terpene r95c1">
      <a href="#r95c1">Region&nbsp95.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">104,656</td>
     <td class="digits table-split-left">130,302</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000664/1" target="_blank">isorenieratene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r98c1">
     
     
         
     
     <td class="regbutton LAP,thiopeptide hybrid r98c1">
      <a href="#r98c1">Region&nbsp98.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lap" target="_blank">LAP</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thiopeptide" target="_blank">thiopeptide</a>
     </td>
     <td class="digits">3,597</td>
     <td class="digits table-split-left">19,058</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r101c1">
     
     
     <td class="regbutton butyrolactone r101c1">
      <a href="#r101c1">Region&nbsp101.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
     </td>
     <td class="digits">14,569</td>
     <td class="digits table-split-left">25,624</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r103c1">
     
     
     <td class="regbutton lanthipeptide-class-v r103c1">
      <a href="#r103c1">Region&nbsp103.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-v" target="_blank">lanthipeptide-class-v</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">30,337</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001856/1" target="_blank">caniferolide A / caniferolide B / caniferolide C / caniferolide D</a></td>
      <td>Polyketide:Modular type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 8%, #ffffff00 8%)">8%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r110c1">
     
     
     <td class="regbutton NRPS r110c1">
      <a href="#r110c1">Region&nbsp110.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">45,272</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000359/1" target="_blank">fuscachelin A / fuscachelin B / fuscachelin C</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r132c1">
     
     
     <td class="regbutton RRE-containing r132c1">
      <a href="#r132c1">Region&nbsp132.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#rre-containing" target="_blank">RRE-containing</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">3,419</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r155c1">
     
     
     <td class="regbutton T1PKS r155c1">
      <a href="#r155c1">Region&nbsp155.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">6,406</td>
     <td class="digits table-split-left">64,045</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000035/1" target="_blank">chalcomycin A</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r155c2">
     
     
     <td class="regbutton butyrolactone r155c2">
      <a href="#r155c2">Region&nbsp155.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
     </td>
     <td class="digits">81,271</td>
     <td class="digits table-split-left">92,047</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r157c1">
     
     
     <td class="regbutton ectoine r157c1">
      <a href="#r157c1">Region&nbsp157.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">9,962</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000853/1" target="_blank">ectoine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r158c1">
     
     
     <td class="regbutton T1PKS r158c1">
      <a href="#r158c1">Region&nbsp158.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">35,211</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r167c1">
     
     
     <td class="regbutton T1PKS r167c1">
      <a href="#r167c1">Region&nbsp167.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">27,556</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001377/1" target="_blank">uncialamycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r173c1">
     
     
     <td class="regbutton NRPS r173c1">
      <a href="#r173c1">Region&nbsp173.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">24,060</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001503/1" target="_blank">amycolamycin A / amycolamycin B</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r175c1">
     
     
     <td class="regbutton RiPP-like r175c1">
      <a href="#r175c1">Region&nbsp175.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">8,111</td>
     <td class="digits table-split-left">18,416</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
 </tbody>
</table>
      </div>
     </div>
     
     
     <dl id="regionless-records">
      <dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>
       
         <dd><strong>CC223A_contig_94</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00002_CC223A_..</strong> (original name was: CC223A_contig_135) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00003_CC223A_..</strong> (original name was: CC223A_contig_146) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_53</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00005_CC223A_..</strong> (original name was: CC223A_contig_145) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00006_CC223A_..</strong> (original name was: CC223A_contig_167) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00008_CC223A_..</strong> (original name was: CC223A_contig_156) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00009_CC223A_..</strong> (original name was: CC223A_contig_165) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00010_CC223A_..</strong> (original name was: CC223A_contig_120) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_82</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00012_CC223A_..</strong> (original name was: CC223A_contig_159) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00013_CC223A_..</strong> (original name was: CC223A_contig_110) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_20</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_50</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00016_CC223A_..</strong> (original name was: CC223A_contig_125) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00018_CC223A_..</strong> (original name was: CC223A_contig_115) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00019_CC223A_..</strong> (original name was: CC223A_contig_109) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_90</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_23</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_75</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00025_CC223A_..</strong> (original name was: CC223A_contig_172) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_1</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_63</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00028_CC223A_..</strong> (original name was: CC223A_contig_182) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00029_CC223A_..</strong> (original name was: CC223A_contig_151) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00030_CC223A_..</strong> (original name was: CC223A_contig_116) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00032_CC223A_..</strong> (original name was: CC223A_contig_129) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_19</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00034_CC223A_..</strong> (original name was: CC223A_contig_178) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_96</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_47</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_52</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00039_CC223A_..</strong> (original name was: CC223A_contig_138) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00040_CC223A_..</strong> (original name was: CC223A_contig_181) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00041_CC223A_..</strong> (original name was: CC223A_contig_140) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_39</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00043_CC223A_..</strong> (original name was: CC223A_contig_155) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_26</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00046_CC223A_..</strong> (original name was: CC223A_contig_157) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00047_CC223A_..</strong> (original name was: CC223A_contig_122) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_28</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_62</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_42</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00051_CC223A_..</strong> (original name was: CC223A_contig_173) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_9</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_69</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_14</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_64</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00056_CC223A_..</strong> (original name was: CC223A_contig_179) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00057_CC223A_..</strong> (original name was: CC223A_contig_162) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_11</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00059_CC223A_..</strong> (original name was: CC223A_contig_139) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_67</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00061_CC223A_..</strong> (original name was: CC223A_contig_119) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_57</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_38</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00064_CC223A_..</strong> (original name was: CC223A_contig_113) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_29</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00066_CC223A_..</strong> (original name was: CC223A_contig_102) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_98</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00068_CC223A_..</strong> (original name was: CC223A_contig_128) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00069_CC223A_..</strong> (original name was: CC223A_contig_101) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_60</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00071_CC223A_..</strong> (original name was: CC223A_contig_112) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_34</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00073_CC223A_..</strong> (original name was: CC223A_contig_133) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00074_CC223A_..</strong> (original name was: CC223A_contig_104) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00075_CC223A_..</strong> (original name was: CC223A_contig_148) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_73</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00077_CC223A_..</strong> (original name was: CC223A_contig_160) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00078_CC223A_..</strong> (original name was: CC223A_contig_142) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_92</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_77</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00081_CC223A_..</strong> (original name was: CC223A_contig_150) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00082_CC223A_..</strong> (original name was: CC223A_contig_176) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00083_CC223A_..</strong> (original name was: CC223A_contig_169) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_16</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_65</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00086_CC223A_..</strong> (original name was: CC223A_contig_153) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_88</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_30</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_72</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_70</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_85</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_49</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_83</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00096_CC223A_..</strong> (original name was: CC223A_contig_127) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00097_CC223A_..</strong> (original name was: CC223A_contig_183) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00099_CC223A_..</strong> (original name was: CC223A_contig_144) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00100_CC223A_..</strong> (original name was: CC223A_contig_164) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_35</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_76</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_87</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00106_CC223A_..</strong> (original name was: CC223A_contig_114) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_99</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00108_CC223A_..</strong> (original name was: CC223A_contig_166) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_95</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00111_CC223A_..</strong> (original name was: CC223A_contig_111) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_86</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_93</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_7</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_61</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00116_CC223A_..</strong> (original name was: CC223A_contig_117) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_91</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00118_CC223A_..</strong> (original name was: CC223A_contig_106) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00119_CC223A_..</strong> (original name was: CC223A_contig_124) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00120_CC223A_..</strong> (original name was: CC223A_contig_132) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00121_CC223A_..</strong> (original name was: CC223A_contig_147) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00122_CC223A_..</strong> (original name was: CC223A_contig_136) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00123_CC223A_..</strong> (original name was: CC223A_contig_131) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_36</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_18</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_27</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_54</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_51</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00129_CC223A_..</strong> (original name was: CC223A_contig_170) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00130_CC223A_..</strong> (original name was: CC223A_contig_180) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_58</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00133_CC223A_..</strong> (original name was: CC223A_contig_141) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00134_CC223A_..</strong> (original name was: CC223A_contig_152) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_2</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_89</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_41</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00138_CC223A_..</strong> (original name was: CC223A_contig_175) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00139_CC223A_..</strong> (original name was: CC223A_contig_118) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_31</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_81</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00142_CC223A_..</strong> (original name was: CC223A_contig_137) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00143_CC223A_..</strong> (original name was: CC223A_contig_105) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00144_CC223A_..</strong> (original name was: CC223A_contig_143) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_80</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_8</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_13</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_12</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_17</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_15</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00151_CC223A_..</strong> (original name was: CC223A_contig_177) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_43</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00153_CC223A_..</strong> (original name was: CC223A_contig_154) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_97</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00156_CC223A_..</strong> (original name was: CC223A_contig_161) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00159_CC223A_..</strong> (original name was: CC223A_contig_107) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00160_CC223A_..</strong> (original name was: CC223A_contig_123) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_24</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_45</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_78</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_59</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00165_CC223A_..</strong> (original name was: CC223A_contig_171) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_74</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00168_CC223A_..</strong> (original name was: CC223A_contig_134) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00169_CC223A_..</strong> (original name was: CC223A_contig_121) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_79</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00171_CC223A_..</strong> (original name was: CC223A_contig_174) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_71</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_21</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_4</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00177_CC223A_..</strong> (original name was: CC223A_contig_103) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00178_CC223A_..</strong> (original name was: CC223A_contig_100) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00179_CC223A_..</strong> (original name was: CC223A_contig_126) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00180_CC223A_..</strong> (original name was: CC223A_contig_168) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00181_CC223A_..</strong> (original name was: CC223A_contig_130) (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>CC223A_contig_68</strong> (Nocardiopsis sp. CC223A)</dd>
       
         <dd><strong>c00183_CC223A_..</strong> (original name was: CC223A_contig_158) (Nocardiopsis sp. CC223A)</dd>
       
     </dl>
     
    </div>
    
    <div class="overview-switches">
     <div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    
   </div>
  </div>
  






<div class="page" id='r7c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_37 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-133"></div> <span class="help-tooltip" id="region_svg-help-133">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_37.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r7c1-svg" data-filename="CC223A_contig_37_r7c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 4,693 - 25,736 nt. (total: 21,044 nt)
        
        <a class="cluster-rules-header" id="r7c1-rules-header" href="#r7c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r7c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r7c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-134"></div> <span class="help-tooltip" id="focus-panel-help-134">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r7c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r7c1-clusterblast" data-name="r7c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r7c1-knownclusterblast" data-name="r7c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r7c1-subclusterblast" data-name="r7c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r7c1-MIBiG-cluster-compare" data-name="r7c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r7c1-pfam-details" data-name="r7c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r7c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-1"></div> <span class="help-tooltip" id="cb-general-help-1">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r7c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r7c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r7c1_1.svg">
        NC_021177 (5813000-5841454): Streptomyces fulvissimus DSM 40593, complete seq... (56% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r7c1_2.svg">
        NZ_AUKW01000010 (38714-59752): Streptomyces sp. TAA204 H343DRAFT scaffold0000... (81% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r7c1_3.svg">
        NZ_CP054926 (5654306-5682829): Streptomyces fulvissimus strain NA06532 chromo... (52% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r7c1_4.svg">
        NZ_CP032402 (818709-837146): Actinomadura amylolytica strain YIM 77502 chromo... (64% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r7c1_5.svg">
        NZ_FWXV01000012 (238528-300213): Kibdelosporangium aridum strain DSM 43828, w... (18% of genes show similarity), T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r7c1_6.svg">
        NZ_CP030930 (5495576-5518411): Streptomyces cavourensis strain TJ430 chromoso... (52% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r7c1_7.svg">
        NZ_CP024957 (5464001-5486836): Streptomyces cavourensis strain 1AS2a chromoso... (52% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r7c1_8.svg">
        NZ_JAAATY010000001 (467231-488293): Kibdelosporangium sp. 4NS15 sequence 01, ... (42% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r7c1_9.svg">
        NZ_CP012752 (4956243-4976570): Kibdelosporangium phytohabitans strain KLBMP11... (30% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r7c1_10.svg">
        NZ_CDME01000010 (393133-414174): Kibdelosporangium sp. MJ126-NF4, whole genom... (37% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r7c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r7c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r7c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-2"></div> <span class="help-tooltip" id="cb-known-help-2">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r7c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r7c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r7c1_1.svg">
        murayaquinone
      </option>
      
      <option value = "svg/knownclusterblast_r7c1_2.svg">
        paulomycin
      </option>
      
      <option value = "svg/knownclusterblast_r7c1_3.svg">
        paulomycin
      </option>
      
      <option value = "svg/knownclusterblast_r7c1_4.svg">
        A23187
      </option>
      
      <option value = "svg/knownclusterblast_r7c1_5.svg">
        diazaquinomycin A / diazaquinomycin E / diazaquinomycin F / diazaquinomycin G
      </option>
      
      <option value = "svg/knownclusterblast_r7c1_6.svg">
        streptonigrin
      </option>
      
      <option value = "svg/knownclusterblast_r7c1_7.svg">
        caboxamycin
      </option>
      
      <option value = "svg/knownclusterblast_r7c1_8.svg">
        caboxamycin
      </option>
      
      <option value = "svg/knownclusterblast_r7c1_9.svg">
        limazepine C / limazepine D / limazepine E / limazepine F / limazepine A
      </option>
      
      <option value = "svg/knownclusterblast_r7c1_10.svg">
        phenalinolactone A
      </option>
      
    </select>
    <button id="knownclusterblast-r7c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r7c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r7c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-3"></div> <span class="help-tooltip" id="cb-sub-help-3">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r7c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r7c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r7c1_1.svg">
        DQ230532_c2: phenalinolactone terpenoid (33% of genes show similarity), terpenoid
      </option>
      
      <option value = "svg/subclusterblast_r7c1_2.svg">
        AB103463_c2: medermycin aminodeoxysugar (angolosamine) (28% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r7c1_3.svg">
        DQ230532_c3: phenalinolactone deoxyhexose L-amicetose (25% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r7c1_4.svg">
        DQ116941_c2: chlorothricin D-olivose (28% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r7c1_5.svg">
        FN433113_c2: streptolydigin L-rhodinose (25% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r7c1_6.svg">
        AF322256_c1: simocyclinone dTDP-D-olivose (28% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r7c1_7.svg">
        AF080235_c2: landomycin D-olivose / L-rhodinose (16% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r7c1_8.svg">
        AJ871581_c3: rubradirin D-rubranitrose (40% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r7c1_9.svg">
        AB032523_c1: avermectin oleandrose (25% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r7c1_10.svg">
        AB088119_c2: staurosporine deoxysugar (20% of genes show similarity), sugar
      </option>
      
    </select>
    <button id="subclusterblast-r7c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r7c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r7c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-4"></div> <span class="help-tooltip" id="MIBiG-help-4">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r7c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r7c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r7c1"></div>
    <div id="comparison-MIBiG-r7c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000193.1: 0-14807">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000193/index.html#r1c1" target="_new">BGC0000193.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.56, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>aclacinomycin</td>
        
        
        <td>Streptomyces galilaeus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000899.1: 0-17561">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000899/index.html#r1c1" target="_new">BGC0000899.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.55, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Other</td>
        
        <td>desosamine</td>
        
        
        <td>Streptomyces narbonensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000813.1: 0-20300">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000813/index.html#r1c1" target="_new">BGC0000813.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.53, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Alkaloid</td>
        
        <td>K-252a</td>
        
        
        <td>Nonomuraea longicatena</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000898.1: 118-12196">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000898/index.html#r1c1" target="_new">BGC0000898.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.55, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Saccharide</td>
        
        <td>desosamine</td>
        
        
        <td>Streptomyces venezuelae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.52, order:0.51, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Other</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000278.1: 25-5408">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000278/index.html#r1c1" target="_new">BGC0000278.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>urdamycin</td>
        
        
        <td>Streptomyces fradiae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.52, order:0.51, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>NRP, Polyketide</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001817.1: 4399-89071">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001817/index.html#r1c1" target="_new">BGC0001817.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.19</td>
        <td>Other</td>
        
        <td>vancosamine</td>
        
        
        <td>Streptomyces sp. WAC2288</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Terpene</td>
        
        <td>pentalenolactone</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000794.1: 492-19862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000794/index.html#r1c1" target="_new">BGC0000794.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.19</td>
        <td>Saccharide</td>
        
        <td>S-layer glycan</td>
        
        
        <td>Geobacillus stearothermophilus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r7c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000193.1: 0-14807">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000193/index.html#r1c1" target="_new">BGC0000193.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.56, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>aclacinomycin</td>
      <td>Streptomyces galilaeus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000899.1: 0-17561">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000899/index.html#r1c1" target="_new">BGC0000899.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.55, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Other</td>
      <td>desosamine</td>
      <td>Streptomyces narbonensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000813.1: 0-20300">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000813/index.html#r1c1" target="_new">BGC0000813.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.53, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Alkaloid</td>
      <td>K-252a</td>
      <td>Nonomuraea longicatena</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000898.1: 118-12196">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000898/index.html#r1c1" target="_new">BGC0000898.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.55, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Saccharide</td>
      <td>desosamine</td>
      <td>Streptomyces venezuelae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.52, order:0.51, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Other</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000278.1: 25-5408">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000278/index.html#r1c1" target="_new">BGC0000278.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>urdamycin</td>
      <td>Streptomyces fradiae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.52, order:0.51, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>NRP, Polyketide</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001817.1: 4399-89071">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001817/index.html#r1c1" target="_new">BGC0001817.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Other</td>
      <td>vancosamine</td>
      <td>Streptomyces sp. WAC2288</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000794.1: 492-19862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000794/index.html#r1c1" target="_new">BGC0000794.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Saccharide</td>
      <td>S-layer glycan</td>
      <td>Geobacillus stearothermophilus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r7c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-5"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-5">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r7c1-pfam-details-svg" data-filename="CC223A_contig_37_r7c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r7c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r17c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_40 - Region 1 - lassopeptide,phosphonate,ranthipeptide <div class="help-container"> <div class="help-icon" data-id="region_svg-help-135"></div> <span class="help-tooltip" id="region_svg-help-135">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_40.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r17c1-svg" data-filename="CC223A_contig_40_r17c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 48,794 nt. (total: 48,794 nt)
        
        <a class="cluster-rules-header" id="r17c1-rules-header" href="#r17c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r17c1-rules">phosphonate: phosphonates<br>lassopeptide: (PF13471 and Asn_synthase or micJ25 or mcjC)<br>ranthipeptide: (TIGR03973 or SCIFF or (cds(PF04055 and (TIGR04085 or SPASM)) and minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE]) and not (TIGR01716 or subtilosin or TIGR02111 or TIGR03966 or TIGR04103 or YcaO or TIGR03882)))</div>
      
      <div class="region-svg-container">
       <div id='r17c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-136"></div> <span class="help-tooltip" id="focus-panel-help-136">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r17c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r17c1-clusterblast" data-name="r17c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r17c1-knownclusterblast" data-name="r17c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r17c1-subclusterblast" data-name="r17c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r17c1-lassopeptides" data-name="r17c1-lassopeptides">
        Lasso peptides
      </div>
     
    
     
    
     
      <div class="body-details-header r17c1-MIBiG-cluster-compare" data-name="r17c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r17c1-pfam-details" data-name="r17c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r17c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-6"></div> <span class="help-tooltip" id="cb-general-help-6">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r17c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r17c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r17c1_1.svg">
        NZ_CP016279 (7290813-7394787): Streptomyces griseochromogenes strain ATCC 145... (14% of genes show similarity), NRPS,NRPS-like,T3PKS,betalactone,ladderane,phosphonate
      </option>
      
      <option value = "svg/clusterblast_r17c1_2.svg">
        NZ_ATYW01000010 (4191-45100): Glycomyces tenuis DSM 44171 YWWDRAFT scaffold 8... (29% of genes show similarity), phosphonate
      </option>
      
      <option value = "svg/clusterblast_r17c1_3.svg">
        NZ_CP026121 (4785697-4826633): Streptomyces sp. Go-475 chromosome, complete g... (31% of genes show similarity), phosphonate
      </option>
      
      <option value = "svg/clusterblast_r17c1_4.svg">
        NC_020504 (4565918-4606854): Streptomyces davaonensis JCM 4913, complete genome (40% of genes show similarity), phosphonate
      </option>
      
      <option value = "svg/clusterblast_r17c1_5.svg">
        NZ_CP040916 (8006403-8047321): Streptomyces spectabilis strain NRRL 2792 chro... (26% of genes show similarity), phosphonate
      </option>
      
      <option value = "svg/clusterblast_r17c1_6.svg">
        NZ_CP010848 (485568-540281): Rathayibacter toxicus strain 70137, complete genome (21% of genes show similarity), LAP,phosphonate,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r17c1_7.svg">
        NZ_KN049967 (1827892-1868783): Bacillus intestinalis strain BST scaffold1, wh... (25% of genes show similarity), phosphonate
      </option>
      
      <option value = "svg/clusterblast_r17c1_8.svg">
        NZ_VWVU01000006 (125843-148544): Nocardiopsis quinghaiensis strain YIM 28A4 c... (26% of genes show similarity), lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r17c1_9.svg">
        NZ_KQ950181 (560207-622278): Thermobifida cellulosilytica TB100 scaffold2, wh... (13% of genes show similarity), T1PKS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r17c1_10.svg">
        NZ_FNRR01000002 (667639-708542): Paenibacillus sp. 276b, whole genome shotgun... (19% of genes show similarity), phosphonate
      </option>
      
    </select>
    <button id= "clusterblast-r17c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r17c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r17c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-7"></div> <span class="help-tooltip" id="cb-known-help-7">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r17c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r17c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r17c1_1.svg">
        rhizocticin A
      </option>
      
      <option value = "svg/knownclusterblast_r17c1_2.svg">
        phosphonoacetic Acid
      </option>
      
    </select>
    <button id="knownclusterblast-r17c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r17c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r17c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-8"></div> <span class="help-tooltip" id="cb-sub-help-8">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r17c1-lassopeptides">
      <div class="details">
    <div class="heading">
      <span>Lasso peptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lasso-body-help-9"></div> <span class="help-tooltip" id="lasso-body-help-9">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lassopeptide(s) for cds26471_26591 - Putative Class II</h4>
    <hr>
      <dl class="details-text">
        
          <dt><span class="serif">cds26471_26591</span> leader / core peptide / cleaved tail</dt>
          <dd>
            MEQQTAYEPPTVTDLGDARELIL
            -
            <span class ="macrolactam">G</span><span class ="macrolactam">T</span><span class ="macrolactam">G</span><span class ="macrolactam">A</span><span class ="macrolactam">H</span><span class ="macrolactam">D</span><span class ="macrolactam">T</span><span class ="macrolactam">A</span><span class ="macrolactam">D</span>MNTARYN
          </dd>
        
      </dl>
    
  <div class="legend">
    <h4> Legend: </h4>
    <div>
      <span class="macrolactam">Red</span>: Putative macrolactam<br>
      <span class="cut">Purple</span>: Putative cleaved off residues<br>
    </div>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r17c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-12"></div> <span class="help-tooltip" id="MIBiG-help-12">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r17c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r17c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r17c1"></div>
    <div id="comparison-MIBiG-r17c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">phosphonate</span></th>
         
         <th><span class="cc-heat-product">lassopeptide</span></th>
         
         <th><span class="cc-heat-product">ranthipeptide</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001673.1: 77019-79729">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001673/index.html#r1c1" target="_new">BGC0001673.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>1.07</td>
        <td>RiPP</td>
        
        <td>moomysin</td>
        
        
        <td>Streptomyces cattleya NRRL 8057 = DSM 46488</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001539.1: 77018-79728">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001539/index.html#r1c1" target="_new">BGC0001539.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>1.07</td>
        <td>RiPP</td>
        
        <td>cattlecin</td>
        
        
        <td>Streptomyces cattleya NRRL 8057 = DSM 46488</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001548.1: 108185-111640">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001548/index.html#r1c1" target="_new">BGC0001548.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>1.07</td>
        <td>RiPP</td>
        
        <td>citrulassin A</td>
        
        
        <td>Streptomyces albulus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001550.1: 2949-7545">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001550/index.html#r1c1" target="_new">BGC0001550.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>1.02</td>
        <td>RiPP</td>
        
        <td>citrulassin D</td>
        
        
        <td>Streptomyces katrae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001549.1: 98590-103146">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001549/index.html#r1c1" target="_new">BGC0001549.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.99</td>
        <td>RiPP</td>
        
        <td>citrulassin B</td>
        
        
        <td>Streptomyces aurantiacus JA 4570</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001507.1: 43037-46158">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001507/index.html#r1c1" target="_new">BGC0001507.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.98</td>
        <td>RiPP</td>
        
        <td>anantin C</td>
        
        
        <td>Streptomyces olindensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001552.1: 6877532-6881873">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001552/index.html#r1c1" target="_new">BGC0001552.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.97</td>
        <td>RiPP</td>
        
        <td>citrulassin F</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002053.1: 242-2917">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002053/index.html#r1c1" target="_new">BGC0002053.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.97</td>
        <td>RiPP</td>
        
        <td>ulleungdin</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002006.1: 2833927-2837273">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002006/index.html#r1c1" target="_new">BGC0002006.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.96</td>
        <td>RiPP</td>
        
        <td>albusnodin</td>
        
        
        <td>Streptomyces albus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002074.1: 0-2630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002074/index.html#r1c1" target="_new">BGC0002074.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.94</td>
        <td>RiPP</td>
        
        <td>huascopeptin</td>
        
        
        <td>Streptomyces huasconensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r17c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001673.1: 77019-79729">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001673/index.html#r1c1" target="_new">BGC0001673.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>RiPP</td>
      <td>moomysin</td>
      <td>Streptomyces cattleya NRRL 8057 = DSM 46488</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001539.1: 77018-79728">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001539/index.html#r1c1" target="_new">BGC0001539.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>RiPP</td>
      <td>cattlecin</td>
      <td>Streptomyces cattleya NRRL 8057 = DSM 46488</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001859.1: 6609-19144">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001859/index.html#r1c1" target="_new">BGC0001859.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>Other,  , (, P, h, o, s, p, h, o, n, a, t, e, )</td>
      <td>fosfomycin</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001550.1: 2949-7545">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001550/index.html#r1c1" target="_new">BGC0001550.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>RiPP</td>
      <td>citrulassin D</td>
      <td>Streptomyces katrae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001548.1: 108185-111640">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001548/index.html#r1c1" target="_new">BGC0001548.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>RiPP</td>
      <td>citrulassin A</td>
      <td>Streptomyces albulus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001549.1: 98590-103146">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001549/index.html#r1c1" target="_new">BGC0001549.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>RiPP</td>
      <td>citrulassin B</td>
      <td>Streptomyces aurantiacus JA 4570</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001552.1: 6877532-6881873">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001552/index.html#r1c1" target="_new">BGC0001552.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>RiPP</td>
      <td>citrulassin F</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001634.1: 9372-14335">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001634/index.html#r1c1" target="_new">BGC0001634.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>RiPP</td>
      <td>keywimysin</td>
      <td>Streptomyces sp. NRRL F-5702</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001507.1: 43037-46158">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001507/index.html#r1c1" target="_new">BGC0001507.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>RiPP</td>
      <td>anantin C</td>
      <td>Streptomyces olindensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002053.1: 242-2917">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002053/index.html#r1c1" target="_new">BGC0002053.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>RiPP</td>
      <td>ulleungdin</td>
      <td>Streptomyces sp.</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r17c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-13"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-13">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r17c1-pfam-details-svg" data-filename="CC223A_contig_40_r17c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r17c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r17c1-lassopeptides" data-name="r17c1-lassopeptides">
        Lasso peptides
      </div>
     
    
     
      <div class="sidepanel-details-header r17c1-RREfinder" data-name="r17c1-RREfinder">
        RREFinder
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r17c1-lassopeptides">
      <div class="more-details">
    <div class="heading">
      <span>Core peptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lasso-side-help-10"></div> <span class="help-tooltip" id="lasso-side-help-10">Lists the possible core peptides in the region. Each core peptide lists the number of disulfide bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
    </div>

  <dl class ="prediction-text">
    <dt><span class="serif">cds26471_26591</span> - Class II</dt>
      <dd>Cleavage pHMM score: -2.30</dd>
      <dd>RODEO score: 16</dd>
      <dd>Monoisotopic mass: 1675.7 Da</dd>
      <dd>Molecular weight: 1676.7 Da</dd>
      <dd>Number of bridges: 0</dd>
    
  </dl>

</div>

    </div>
     
    
     
    <div class="sidepanel-details-section r17c1-RREfinder">
      <div class="more-details">
  <div class="heading">
    <span>RRE predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="rre-side-help-11"></div> <span class="help-tooltip" id="rre-side-help-11">RREfinder results sidepanel.</span></div>
  </div>
  <dl class = 'prediction-text'>
    
      <span><strong>lassopeptide protocluster</strong> (13984...36463)</span>
      <dt></dt>
      
      
      
        
        <strong>cds24400_24679</strong> <div class="collapser collapser-target-cds24400_24679 collapser-level-cds"> </div><div class="collapser-content">
        
          <dd>RRE type: Stand_Alone_Lasso_RRE</dd>
          <dd>Location: 6 - 86</dd>
          <dd>Score: 69.3</dd>
        
        </div><br>
      <br>
    
      <span><strong>ranthipeptide protocluster</strong> (14400...48794)</span>
      <dt></dt>
      
      
      
        
        <strong>cds24400_24679</strong> <div class="collapser collapser-target-cds24400_24679 collapser-level-cds"> </div><div class="collapser-content">
        
          <dd>RRE type: Stand_Alone_Lasso_RRE</dd>
          <dd>Location: 6 - 86</dd>
          <dd>Score: 69.3</dd>
        
        </div><br>
      <br>
    
    </dl>
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r23c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_32 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-137"></div> <span class="help-tooltip" id="region_svg-help-137">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_32.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r23c1-svg" data-filename="CC223A_contig_32_r23c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 8,162 - 51,071 nt. (total: 42,910 nt)
        
        <a class="cluster-rules-header" id="r23c1-rules-header" href="#r23c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r23c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r23c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-138"></div> <span class="help-tooltip" id="focus-panel-help-138">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r23c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r23c1-nrps_pks" data-name="r23c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r23c1-clusterblast" data-name="r23c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r23c1-knownclusterblast" data-name="r23c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r23c1-subclusterblast" data-name="r23c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r23c1-MIBiG-cluster-compare" data-name="r23c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r23c1-pfam-details" data-name="r23c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r23c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-14"></div> <span class="help-tooltip" id="nrps-domain-help-14">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r23c1-details-svg" data-filename="CC223A_contig_32_r23c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r23c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r23c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-15"></div> <span class="help-tooltip" id="cb-general-help-15">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r23c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r23c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r23c1_1.svg">
        NZ_VWVW01000031 (10028-83588): Nocardiopsis sp. FR6 contig031, whole genome s... (12% of genes show similarity), NRPS,bacteriocin,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r23c1_2.svg">
        NZ_VWVV01000020 (1859-50260): Nocardiopsis sp. FR4 contig020, whole genome sh... (17% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r23c1_3.svg">
        NZ_CP030772 (328562-418666): Streptomyces sp. YIM 121038 plasmid pSSP121038, ... (12% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r23c1_4.svg">
        NZ_FUWS01000004 (331733-374685): Marinactinospora thermotolerans DSM 45154, w... (8% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r23c1_5.svg">
        NC_018524 (2732112-2774442): Nocardiopsis alba ATCC BAA-2165, complete sequence (7% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r23c1_6.svg">
        NZ_BAZE01000003 (172594-215492): Nocardiopsis alba strain TP-A0876, whole gen... (8% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r23c1_7.svg">
        NZ_LJSN01000002 (116173-177542): Streptomyces noursei strain JCM 4701 scaffol... (16% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r23c1_8.svg">
        NZ_KB913036 (1804528-1951486): Salinispora arenicola DSM 45545 strain CNS-991... (6% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r23c1_9.svg">
        NZ_SNYN01000002 (236393-279327): Actinorugispora endophytica strain DSM 46770... (10% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r23c1_10.svg">
        NZ_KB906860 (104638-156127): Nocardiopsis sp. CNS-639 strain CNS639 G011DRAFT... (8% of genes show similarity), T1PKS,lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r23c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r23c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r23c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-16"></div> <span class="help-tooltip" id="cb-known-help-16">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r23c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-17"></div> <span class="help-tooltip" id="cb-sub-help-17">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r23c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-18"></div> <span class="help-tooltip" id="MIBiG-help-18">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r23c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r23c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r23c1"></div>
    <div id="comparison-MIBiG-r23c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>esperamicin</td>
        
        
        <td>Actinomadura verrucosospora</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000882.1: 6124-25938">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000882/index.html#r1c1" target="_new">BGC0000882.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Other</td>
        
        <td>mildiomycin</td>
        
        
        <td>Streptomyces rimofaciens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>6-methylsalicyclic acid</td>
        
        
        <td>Aspergillus terreus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>asperlactone</td>
        
        
        <td>Aspergillus ochraceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400.1: 371433-386839">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>citreoviridin</td>
        
        
        <td>Aspergillus terreus NIH2624</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000780.1: 48-63265">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000780/index.html#r1c1" target="_new">BGC0000780.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>O&amp;K-antigen</td>
        
        
        <td>Vibrio parahaemolyticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001138.1: 0-39199">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001138/index.html#r1c1" target="_new">BGC0001138.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other,  , (, N, u, c, l, e, o, s, i, d, e, )</td>
        
        <td>a201a</td>
        
        
        <td>Marinactinospora thermotolerans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
        
        
        <td>Parastagonospora nodorum SN15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124.1: 125832-132922">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>pyranonigrin E</td>
        
        
        <td>Aspergillus niger ATCC 1015</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254.1: 82-7456">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Polyketide</td>
        
        <td>ACT-Toxin II</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r23c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>esperamicin</td>
      <td>Actinomadura verrucosospora</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000882.1: 6124-25938">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000882/index.html#r1c1" target="_new">BGC0000882.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Other</td>
      <td>mildiomycin</td>
      <td>Streptomyces rimofaciens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Aspergillus terreus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Polyketide</td>
      <td>asperlactone</td>
      <td>Aspergillus ochraceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400.1: 371433-386839">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Polyketide</td>
      <td>citreoviridin</td>
      <td>Aspergillus terreus NIH2624</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000780.1: 48-63265">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000780/index.html#r1c1" target="_new">BGC0000780.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Saccharide</td>
      <td>O&amp;K-antigen</td>
      <td>Vibrio parahaemolyticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001138.1: 0-39199">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001138/index.html#r1c1" target="_new">BGC0001138.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other,  , (, N, u, c, l, e, o, s, i, d, e, )</td>
      <td>a201a</td>
      <td>Marinactinospora thermotolerans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Polyketide</td>
      <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
      <td>Parastagonospora nodorum SN15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124.1: 125832-132922">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Polyketide</td>
      <td>pyranonigrin E</td>
      <td>Aspergillus niger ATCC 1015</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254.1: 82-7456">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Polyketide</td>
      <td>ACT-Toxin II</td>
      <td>Alternaria alternata</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r23c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-19"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-19">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r23c1-pfam-details-svg" data-filename="CC223A_contig_32_r23c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r23c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r24c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00024_CC223A_.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-139"></div> <span class="help-tooltip" id="region_svg-help-139">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00024_CC223A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r24c1-svg" data-filename="c00024_CC223A_.._r24c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 6,435 nt. (total: 6,435 nt)
        
        <a class="cluster-rules-header" id="r24c1-rules-header" href="#r24c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r24c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r24c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-140"></div> <span class="help-tooltip" id="focus-panel-help-140">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r24c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r24c1-nrps_pks" data-name="r24c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r24c1-clusterblast" data-name="r24c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r24c1-knownclusterblast" data-name="r24c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r24c1-subclusterblast" data-name="r24c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r24c1-MIBiG-cluster-compare" data-name="r24c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r24c1-pfam-details" data-name="r24c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r24c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-20"></div> <span class="help-tooltip" id="nrps-domain-help-20">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r24c1-details-svg" data-filename="c00024_CC223A_.._r24c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r24c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r24c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-23"></div> <span class="help-tooltip" id="cb-general-help-23">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r24c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r24c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r24c1_1.svg">
        NZ_FQZK01000011 (77970-131650): Nocardiopsis flavescens strain CGMCC 4.5723, ... (9% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r24c1_2.svg">
        NZ_JACHJO010000001 (353703-413717): Nocardiopsis algeriensis strain CECT 8712... (8% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r24c1_3.svg">
        NZ_VWVV01000010 (14266-79323): Nocardiopsis sp. FR4 contig010, whole genome s... (4% of genes show similarity), LAP,NRPS,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r24c1_4.svg">
        NC_015376 (2469016-2536299): Burkholderia gladioli BSR3 chromosome 2, complet... (3% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r24c1_5.svg">
        NZ_CP023278 (5183323-5244731): Nostoc sp. CENA543 chromosome, complete genome (6% of genes show similarity), NRPS,betalactone,microviridin
      </option>
      
      <option value = "svg/clusterblast_r24c1_6.svg">
        NZ_CP048815 (2024858-2138435): Caulobacter rhizosphaerae strain KCTC 52515 ch... (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r24c1_7.svg">
        NZ_CP058977 (4940936-5145692): Streptomyces sp. NEAU-sy36 chromosome, complet... (4% of genes show similarity), NRPS,T1PKS,lanthipeptide,terpene
      </option>
      
      <option value = "svg/clusterblast_r24c1_8.svg">
        NZ_QQAU01000001 (563296-631526): Lentzea flaviverrucosa strain DSM 44664 Ga02... (8% of genes show similarity), NRPS,arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r24c1_9.svg">
        NZ_CP034550 (7985027-8048499): Saccharothrix syringae strain NRRL B-16468 chr... (8% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r24c1_10.svg">
        NZ_CP029194 (785458-901611): Streptomyces venezuelae strain ATCC 15068 chromo... (5% of genes show similarity), NRPS,T1PKS,lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r24c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r24c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r24c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-24"></div> <span class="help-tooltip" id="cb-known-help-24">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r24c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-25"></div> <span class="help-tooltip" id="cb-sub-help-25">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r24c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-26"></div> <span class="help-tooltip" id="MIBiG-help-26">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r24c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r24c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r24c1"></div>
    <div id="comparison-MIBiG-r24c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342.1: 1020-10416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>enniatin</td>
        
        
        <td>Fusarium equiseti</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313.1: 0-33475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>beauvericin</td>
        
        
        <td>Beauveria bassiana</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>AM-toxin</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r24c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342.1: 1020-10416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>enniatin</td>
      <td>Fusarium equiseti</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313.1: 0-33475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>beauvericin</td>
      <td>Beauveria bassiana</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>AM-toxin</td>
      <td>Alternaria alternata</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r24c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-27"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-27">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r24c1-pfam-details-svg" data-filename="c00024_CC223A_.._r24c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r24c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r24c1-nrps_pks_products" data-name="r24c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r24c1-nrps_pks" data-name="r24c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r24c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-21"></div> <span class="help-tooltip" id="nrps-structures-help-21">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 6435:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r24c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r24c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r24c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-22"></div> <span class="help-tooltip" id="nrps-monomers-help-22">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds509_6434</span></strong>: X - X
      <div class="collapser collapser-target-cds509_6434 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (462..857)</strong>: X
          <div class="collapser collapser-target-cds509_6434 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds509_6434-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DgLWmGGvFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1516..1911)</strong>: X
          <div class="collapser collapser-target-cds509_6434 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds509_6434-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DgLWmGGiFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r31c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_25 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-141"></div> <span class="help-tooltip" id="region_svg-help-141">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_25.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r31c1-svg" data-filename="CC223A_contig_25_r31c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 63,676 - 76,965 nt. (total: 13,290 nt)
        
        <a class="cluster-rules-header" id="r31c1-rules-header" href="#r31c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r31c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r31c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-142"></div> <span class="help-tooltip" id="focus-panel-help-142">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r31c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r31c1-clusterblast" data-name="r31c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r31c1-knownclusterblast" data-name="r31c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r31c1-subclusterblast" data-name="r31c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r31c1-MIBiG-cluster-compare" data-name="r31c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r31c1-pfam-details" data-name="r31c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r31c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-28"></div> <span class="help-tooltip" id="cb-general-help-28">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r31c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r31c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r31c1_1.svg">
        NZ_BAZE01000001 (132274-153558): Nocardiopsis alba strain TP-A0876, whole gen... (33% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r31c1_2.svg">
        NZ_FQZK01000004 (15271-36477): Nocardiopsis flavescens strain CGMCC 4.5723, w... (40% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r31c1_3.svg">
        NZ_VWVS01000004 (47217-70975): Nocardiopsis sp. H13 contig004, whole genome s... (21% of genes show similarity), bacteriocin,terpene
      </option>
      
      <option value = "svg/clusterblast_r31c1_4.svg">
        NZ_VWVW01000027 (35542-56883): Nocardiopsis sp. FR6 contig027, whole genome s... (23% of genes show similarity), bacteriocin,terpene
      </option>
      
      <option value = "svg/clusterblast_r31c1_5.svg">
        NZ_VWVV01000016 (78824-100165): Nocardiopsis sp. FR4 contig016, whole genome ... (23% of genes show similarity), bacteriocin,terpene
      </option>
      
      <option value = "svg/clusterblast_r31c1_6.svg">
        NZ_LR134501 (1477796-1500417): Nocardiopsis dassonvillei strain NCTC10488 chr... (25% of genes show similarity), bacteriocin,terpene
      </option>
      
      <option value = "svg/clusterblast_r31c1_7.svg">
        NZ_CP022434 (3985595-4007887): Nocardiopsis dassonvillei strain HZNU N 1 chro... (25% of genes show similarity), bacteriocin,terpene
      </option>
      
      <option value = "svg/clusterblast_r31c1_8.svg">
        NZ_KB906858 (362675-386810): Nocardiopsis sp. CNS-639 strain CNS639 G011DRAFT... (25% of genes show similarity), bacteriocin,terpene
      </option>
      
      <option value = "svg/clusterblast_r31c1_9.svg">
        NZ_CP017965 (3922709-3943105): Nocardiopsis dassonvillei strain NOCA502F, com... (25% of genes show similarity), bacteriocin,terpene
      </option>
      
      <option value = "svg/clusterblast_r31c1_10.svg">
        NZ_MCOK01000001 (2568613-2588578): Nocardiopsis sinuspersici strain UTMC102 N... (20% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r31c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r31c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r31c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-29"></div> <span class="help-tooltip" id="cb-known-help-29">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r31c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r31c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r31c1_1.svg">
        2-methylisoborneol
      </option>
      
    </select>
    <button id="knownclusterblast-r31c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r31c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r31c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-30"></div> <span class="help-tooltip" id="cb-sub-help-30">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r31c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-31"></div> <span class="help-tooltip" id="MIBiG-help-31">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r31c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r31c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r31c1"></div>
    <div id="comparison-MIBiG-r31c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000658.1: 1490670-1492896">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000658/index.html#r1c1" target="_new">BGC0000658.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.79 (id:0.49, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
         
        
        <td>0.51</td>
        <td>Terpene</td>
        
        <td>2-methylisoborneol</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000659.1: 4075162-4077099">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000659/index.html#r1c1" target="_new">BGC0000659.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.75 (id:0.56, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
         
        
        <td>0.45</td>
        <td>Terpene</td>
        
        <td>2-methylisoborneol</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.52, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.41</td>
        <td>Terpene</td>
        
        <td>geosmin</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000657.1: 643-5774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000657/index.html#r1c1" target="_new">BGC0000657.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.52, order:0.66, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>Terpene</td>
        
        <td>2-methylisoborneol</td>
        
        
        <td>Pseudanabaena sp. dqh15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001361.1: 2257657-2262237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001361/index.html#r1c1" target="_new">BGC0001361.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Terpene</td>
        
        <td>sodorifen</td>
        
        
        <td>Serratia plymuthica 4Rx13</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>auriculamide</td>
        
        
        <td>Herpetosiphon aurantiacus DSM 785</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.45, order:N/A, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
        <td>0.12</td>
        <td>Other</td>
        
        <td>5-isoprenylindole-3-carboxylate -D-glycosyl ester</td>
        
        
        <td>Streptomyces sp. RM-5-8</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843.1: 3437627-3461293">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
        <td>0.11</td>
        <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
        
        <td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000606.1: 0-14244">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000606/index.html#r1c1" target="_new">BGC0000606.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.34 (id:0.43, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
         
        
        <td>0.09</td>
        <td>RiPP</td>
        
        <td>lactazole</td>
        
        
        <td>Streptomyces lactacystinaeus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r31c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000658.1: 1490670-1492896">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000658/index.html#r1c1" target="_new">BGC0000658.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.79 (id:0.49, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
      <td>0.79</td>
      <td>Terpene</td>
      <td>2-methylisoborneol</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000659.1: 4075162-4077099">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000659/index.html#r1c1" target="_new">BGC0000659.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.75 (id:0.56, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
      <td>0.75</td>
      <td>Terpene</td>
      <td>2-methylisoborneol</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.52, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Terpene</td>
      <td>geosmin</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000657.1: 643-5774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000657/index.html#r1c1" target="_new">BGC0000657.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.52, order:0.66, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Terpene</td>
      <td>2-methylisoborneol</td>
      <td>Pseudanabaena sp. dqh15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001361.1: 2257657-2262237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001361/index.html#r1c1" target="_new">BGC0001361.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Terpene</td>
      <td>sodorifen</td>
      <td>Serratia plymuthica 4Rx13</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>auriculamide</td>
      <td>Herpetosiphon aurantiacus DSM 785</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.40 (id:0.45, order:N/A, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
      <td>0.40</td>
      <td>Other</td>
      <td>5-isoprenylindole-3-carboxylate -D-glycosyl ester</td>
      <td>Streptomyces sp. RM-5-8</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843.1: 3437627-3461293">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
      <td>0.38</td>
      <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
      <td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000606.1: 0-14244">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000606/index.html#r1c1" target="_new">BGC0000606.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.34 (id:0.43, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
      <td>0.34</td>
      <td>RiPP</td>
      <td>lactazole</td>
      <td>Streptomyces lactacystinaeus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r31c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-32"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-32">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r31c1-pfam-details-svg" data-filename="CC223A_contig_25_r31c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r31c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r37c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_6 - Region 1 - lanthipeptide-class-i <div class="help-container"> <div class="help-icon" data-id="region_svg-help-143"></div> <span class="help-tooltip" id="region_svg-help-143">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_6.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r37c1-svg" data-filename="CC223A_contig_6_r37c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1,850 - 25,759 nt. (total: 23,910 nt)
        
        <a class="cluster-rules-header" id="r37c1-rules-header" href="#r37c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r37c1-rules">lanthipeptide-class-i: ((LANC_like and (Lant_dehydr_N or Lant_dehydr_C)) and not (YcaO or TIGR03882))</div>
      
      <div class="region-svg-container">
       <div id='r37c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-144"></div> <span class="help-tooltip" id="focus-panel-help-144">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r37c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r37c1-lanthipeptides" data-name="r37c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r37c1-clusterblast" data-name="r37c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r37c1-knownclusterblast" data-name="r37c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r37c1-subclusterblast" data-name="r37c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r37c1-MIBiG-cluster-compare" data-name="r37c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r37c1-pfam-details" data-name="r37c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r37c1-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-33"></div> <span class="help-tooltip" id="lanthi-body-help-33">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lanthipeptide(s) for cds11849_14633 - Putative Class I</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds11552_11762</span> leader / core peptide
          </dt>
          <dd>
            MSTLLADRTDTGLVPRGRDRALRTTMTDPFDLDLRVEVPVVDQSLWSM
            -
            <span class="dhb">Dhb</span>NDG<span class="cys">C</span>GQ<span class="dhb">Dhb</span><span class="cys">C</span>Q<span class="dha">Dha</span>A<span class="cys">C</span>AP<span class="dha">Dha</span><span class="cys">C</span><span class="dhb">Dhb</span>DNG
        </dd>
      </dl>
    
    <div class="legend">
      <h4> Legend: </h4>
      <div>
        <span class="dha">Dha</span>: Didehydroalanine<br>
        <span class="dhb">Dhb</span>: Didehydrobutyrine<br>
      </div>
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r37c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-35"></div> <span class="help-tooltip" id="cb-general-help-35">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r37c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r37c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r37c1_1.svg">
        NZ_JACHJO010000003 (3322-27079): Nocardiopsis algeriensis strain CECT 8712 Ga... (60% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r37c1_2.svg">
        NZ_VWVS01000013 (167446-212940): Nocardiopsis sp. H13 contig013, whole genome... (11% of genes show similarity), T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r37c1_3.svg">
        NZ_KQ950184 (206081-230373): Thermobifida cellulosilytica TB100 scaffold5, wh... (22% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r37c1_4.svg">
        NZ_VWVW01000031 (10028-83588): Nocardiopsis sp. FR6 contig031, whole genome s... (9% of genes show similarity), NRPS,bacteriocin,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r37c1_5.svg">
        NZ_KB896615 (5693-29032): Salinispora arenicola CNY280 C580DRAFT scaffold0000... (26% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r37c1_6.svg">
        NC_018524 (4593850-4617646): Nocardiopsis alba ATCC BAA-2165, complete sequence (17% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r37c1_7.svg">
        NZ_KB906860 (104638-156127): Nocardiopsis sp. CNS-639 strain CNS639 G011DRAFT... (11% of genes show similarity), T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r37c1_8.svg">
        NZ_KB900240 (10021-35422): Salinispora arenicola CNH962 F582DRAFT scaffold000... (23% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r37c1_9.svg">
        NC_014666 (1096690-1123776): Frankia inefficax, complete sequence (19% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r37c1_10.svg">
        NZ_CP048875 (127948-151892): Streptomyces albus strain CAS922 chromosome, com... (23% of genes show similarity), lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r37c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r37c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r37c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-36"></div> <span class="help-tooltip" id="cb-known-help-36">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r37c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-37"></div> <span class="help-tooltip" id="cb-sub-help-37">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r37c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-39"></div> <span class="help-tooltip" id="MIBiG-help-39">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r37c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r37c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r37c1"></div>
    <div id="comparison-MIBiG-r37c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-i</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000540.1: 117-11692">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000540/index.html#r1c1" target="_new">BGC0000540.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>RiPP</td>
        
        <td>paenibacillin</td>
        
        
        <td>Paenibacillus polymyxa OSY-DF</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000560.1: 1037-9106">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000560/index.html#r1c1" target="_new">BGC0000560.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.41, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>RiPP</td>
        
        <td>subtilomycin</td>
        
        
        <td>Bacillus subtilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000506.1: 405-13975">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000506/index.html#r1c1" target="_new">BGC0000506.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.51)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>entianin</td>
        
        
        <td>Bacillus subtilis subsp. spizizenii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000559.1: 494-13831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000559/index.html#r1c1" target="_new">BGC0000559.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.44, order:N/A, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>subtilin</td>
        
        
        <td>Bacillus subtilis subsp. spizizenii ATCC 6633</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000535.1: 827-14152">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000535/index.html#r1c1" target="_new">BGC0000535.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>RiPP</td>
        
        <td>nisin A</td>
        
        
        <td>Lactococcus lactis subsp. lactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000544.1: 14969-30259">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000544/index.html#r1c1" target="_new">BGC0000544.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.43, order:0.38, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>RiPP</td>
        
        <td>planosporicin</td>
        
        
        <td>Planomonospora alba</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000698.1: 2269-33779">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000698/index.html#r1c1" target="_new">BGC0000698.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>hygromycin A</td>
        
        
        <td>Streptomyces hygroscopicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000511.1: 0-14033">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000511/index.html#r1c1" target="_new">BGC0000511.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>RiPP</td>
        
        <td>ericin S</td>
        
        
        <td>Bacillus subtilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000529.1: 1655-23062">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000529/index.html#r1c1" target="_new">BGC0000529.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
        <td>0.12</td>
        <td>RiPP</td>
        
        <td>microbisporicin A2</td>
        
        
        <td>Microbispora corallina</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001727.1: 1552553-1565755">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001727/index.html#r1c1" target="_new">BGC0001727.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.39 (id:0.41, order:0.30, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
         
        
        <td>0.12</td>
        <td>RiPP</td>
        
        <td>paenilan</td>
        
        
        <td>Paenibacillus polymyxa E681</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r37c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000540.1: 117-11692">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000540/index.html#r1c1" target="_new">BGC0000540.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>RiPP</td>
      <td>paenibacillin</td>
      <td>Paenibacillus polymyxa OSY-DF</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000560.1: 1037-9106">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000560/index.html#r1c1" target="_new">BGC0000560.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.41, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>RiPP</td>
      <td>subtilomycin</td>
      <td>Bacillus subtilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000506.1: 405-13975">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000506/index.html#r1c1" target="_new">BGC0000506.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.51)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>entianin</td>
      <td>Bacillus subtilis subsp. spizizenii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000559.1: 494-13831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000559/index.html#r1c1" target="_new">BGC0000559.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.44, order:N/A, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>RiPP</td>
      <td>subtilin</td>
      <td>Bacillus subtilis subsp. spizizenii ATCC 6633</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000535.1: 827-14152">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000535/index.html#r1c1" target="_new">BGC0000535.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>RiPP</td>
      <td>nisin A</td>
      <td>Lactococcus lactis subsp. lactis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000544.1: 14969-30259">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000544/index.html#r1c1" target="_new">BGC0000544.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.43, order:0.38, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>RiPP</td>
      <td>planosporicin</td>
      <td>Planomonospora alba</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000698.1: 2269-33779">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000698/index.html#r1c1" target="_new">BGC0000698.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>hygromycin A</td>
      <td>Streptomyces hygroscopicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000511.1: 0-14033">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000511/index.html#r1c1" target="_new">BGC0000511.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>RiPP</td>
      <td>ericin S</td>
      <td>Bacillus subtilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000529.1: 1655-23062">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000529/index.html#r1c1" target="_new">BGC0000529.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
      <td>0.40</td>
      <td>RiPP</td>
      <td>microbisporicin A2</td>
      <td>Microbispora corallina</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001727.1: 1552553-1565755">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001727/index.html#r1c1" target="_new">BGC0001727.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.39 (id:0.41, order:0.30, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
      <td>0.39</td>
      <td>RiPP</td>
      <td>paenilan</td>
      <td>Paenibacillus polymyxa E681</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r37c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-40"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-40">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r37c1-pfam-details-svg" data-filename="CC223A_contig_6_r37c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r37c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r37c1-lanthipeptides" data-name="r37c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
      <div class="sidepanel-details-header r37c1-RREfinder" data-name="r37c1-RREfinder">
        RREFinder
      </div>
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r37c1-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-34"></div> <span class="help-tooltip" id="lanthi-side-help-34">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

    <dl class ="prediction-text">
    <dt><span class="serif">cds11552_11762</span> - Class I</dt>
        <dd>Cleavage pHMM score: -3.20</dd>
        <dd>RODEO score: 23</dd>
        <dd>Monoisotopic mass: 1941.7 Da</dd>
        <dd>Molecular weight: 1943.0 Da</dd>
        <dd>Number of bridges: 4</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 1961.0 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
            </dl>
          </dd>
        
  </dl>

</div>

    </div>
     
    
     
    
     
    <div class="sidepanel-details-section r37c1-RREfinder">
      <div class="more-details">
  <div class="heading">
    <span>RRE predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="rre-side-help-38"></div> <span class="help-tooltip" id="rre-side-help-38">RREfinder results sidepanel.</span></div>
  </div>
  <dl class = 'prediction-text'>
    
      <span><strong>lanthipeptide-class-i protocluster</strong> (1849...25759)</span>
      <dt></dt>
      
      
      
        
        <strong>cds11849_14633</strong> <div class="collapser collapser-target-cds11849_14633 collapser-level-cds"> </div><div class="collapser-content">
        
          <dd>RRE type: Lanthipeptide_LanB_RRE</dd>
          <dd>Location: 147 - 237</dd>
          <dd>Score: 46.1</dd>
        
        </div><br>
      <br>
    
    </dl>
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r45c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_3 - Region 1 - siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-145"></div> <span class="help-tooltip" id="region_svg-help-145">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_3.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r45c1-svg" data-filename="CC223A_contig_3_r45c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 124,109 - 137,716 nt. (total: 13,608 nt)
        
        <a class="cluster-rules-header" id="r45c1-rules-header" href="#r45c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r45c1-rules">siderophore: IucA_IucC</div>
      
      <div class="region-svg-container">
       <div id='r45c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-146"></div> <span class="help-tooltip" id="focus-panel-help-146">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r45c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r45c1-clusterblast" data-name="r45c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r45c1-knownclusterblast" data-name="r45c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r45c1-subclusterblast" data-name="r45c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r45c1-MIBiG-cluster-compare" data-name="r45c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r45c1-pfam-details" data-name="r45c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r45c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-41"></div> <span class="help-tooltip" id="cb-general-help-41">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r45c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r45c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r45c1_1.svg">
        NZ_JACJIA010000010 (18894-35272): Actinomadura namibiensis strain DSM 44197 G... (60% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r45c1_2.svg">
        NZ_FOLM01000003 (454258-470880): Streptomyces aidingensis strain CGMCC 4.5739... (50% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r45c1_3.svg">
        NZ_AUKR01000016 (57044-73497): Streptomyces sp. TAA040 H300DRAFT scaffold0001... (46% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r45c1_4.svg">
        NZ_FQZK01000023 (36841-50470): Nocardiopsis flavescens strain CGMCC 4.5723, w... (63% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r45c1_5.svg">
        NZ_VWVV01000028 (7862-21704): Nocardiopsis sp. FR4 contig028, whole genome sh... (77% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r45c1_6.svg">
        NZ_VWVW01000042 (39689-53531): Nocardiopsis sp. FR6 contig042, whole genome s... (77% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r45c1_7.svg">
        NZ_ML703342 (42217-56095): Nocardiopsis quinghaiensis strain YIM 28A4 Scaffol... (77% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r45c1_8.svg">
        NZ_LWLB01000023 (128542-142393): Nocardiopsis sp. TSRI0078 scaffold3, whole g... (77% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r45c1_9.svg">
        NZ_QGTF01000003 (542064-555858): Nocardiopsis sp. L17-MgMaSL7 Ga0060225 103, ... (77% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r45c1_10.svg">
        NZ_KZ559467 (558295-574574): Nonomuraea indica strain DRQ-2 Scaffold2, whole ... (50% of genes show similarity), siderophore
      </option>
      
    </select>
    <button id= "clusterblast-r45c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r45c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r45c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-42"></div> <span class="help-tooltip" id="cb-known-help-42">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r45c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r45c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r45c1_1.svg">
        ficellomycin
      </option>
      
      <option value = "svg/knownclusterblast_r45c1_2.svg">
        macrotetrolide
      </option>
      
    </select>
    <button id="knownclusterblast-r45c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r45c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r45c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-43"></div> <span class="help-tooltip" id="cb-sub-help-43">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r45c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-44"></div> <span class="help-tooltip" id="MIBiG-help-44">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r45c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r45c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r45c1"></div>
    <div id="comparison-MIBiG-r45c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">siderophore</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001478.1: 100-5365">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001478/index.html#r1c1" target="_new">BGC0001478.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.47, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Streptomyces sp. ID38640</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.46, order:0.66, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Pantoea agglomerans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000941.1: 5578821-5584948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000941/index.html#r1c1" target="_new">BGC0000941.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.47, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>Other</td>
        
        <td>desferrioxamin B</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000942.1: 1864728-1871818">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000942/index.html#r1c1" target="_new">BGC0000942.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Other</td>
        
        <td>petrobactin</td>
        
        
        <td>Bacillus anthracis str. Ames</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453.1: 0-7234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.47, order:0.48, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>Other</td>
        
        <td>desferrioxamine</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940.1: 3033354-3040682">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.47, order:0.48, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>Other</td>
        
        <td>desferrioxamin B, desferrioxamine E</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001531.1: 1112-9953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001531/index.html#r1c1" target="_new">BGC0001531.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.44, order:0.38, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Other</td>
        
        <td>bisucaberin B</td>
        
        
        <td>Tenacibaculum mesophilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499.1: 2267-11578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.44, order:0.42, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Other</td>
        
        <td>aerobactin</td>
        
        
        <td>Pantoea ananatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.45, order:0.42, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Other</td>
        
        <td>ochrobactin</td>
        
        
        <td>Ochrobactrum anthropi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000295.1: 569-12865">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000295/index.html#r1c1" target="_new">BGC0000295.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.46, order:0.37, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>NRP</td>
        
        <td>acinetoferrin</td>
        
        
        <td>Acinetobacter haemolyticus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r45c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001478.1: 100-5365">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001478/index.html#r1c1" target="_new">BGC0001478.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.47, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Streptomyces sp. ID38640</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.46, order:0.66, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Pantoea agglomerans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000941.1: 5578821-5584948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000941/index.html#r1c1" target="_new">BGC0000941.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.47, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>desferrioxamin B</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000942.1: 1864728-1871818">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000942/index.html#r1c1" target="_new">BGC0000942.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other</td>
      <td>petrobactin</td>
      <td>Bacillus anthracis str. Ames</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453.1: 0-7234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.47, order:0.48, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Other</td>
      <td>desferrioxamine</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940.1: 3033354-3040682">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.47, order:0.48, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Other</td>
      <td>desferrioxamin B, desferrioxamine E</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001531.1: 1112-9953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001531/index.html#r1c1" target="_new">BGC0001531.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.44, order:0.38, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Other</td>
      <td>bisucaberin B</td>
      <td>Tenacibaculum mesophilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499.1: 2267-11578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.44, order:0.42, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Other</td>
      <td>aerobactin</td>
      <td>Pantoea ananatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.45, order:0.42, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Other</td>
      <td>ochrobactin</td>
      <td>Ochrobactrum anthropi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000295.1: 569-12865">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000295/index.html#r1c1" target="_new">BGC0000295.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.46, order:0.37, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>NRP</td>
      <td>acinetoferrin</td>
      <td>Acinetobacter haemolyticus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r45c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-45"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-45">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r45c1-pfam-details-svg" data-filename="CC223A_contig_3_r45c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r45c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r93c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_44 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-147"></div> <span class="help-tooltip" id="region_svg-help-147">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_44.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r93c1-svg" data-filename="CC223A_contig_44_r93c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 2,674 - 45,256 nt. (total: 42,583 nt)
        
        <a class="cluster-rules-header" id="r93c1-rules-header" href="#r93c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r93c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r93c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-148"></div> <span class="help-tooltip" id="focus-panel-help-148">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r93c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r93c1-nrps_pks" data-name="r93c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r93c1-clusterblast" data-name="r93c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r93c1-knownclusterblast" data-name="r93c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r93c1-subclusterblast" data-name="r93c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r93c1-MIBiG-cluster-compare" data-name="r93c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r93c1-pfam-details" data-name="r93c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r93c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-46"></div> <span class="help-tooltip" id="nrps-domain-help-46">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r93c1-details-svg" data-filename="CC223A_contig_44_r93c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r93c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r93c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-47"></div> <span class="help-tooltip" id="cb-general-help-47">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r93c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r93c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r93c1_1.svg">
        NZ_LR594584 (4863171-4905808): Gemmataceae bacterium isolate CJuql4, whole ge... (5% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r93c1_2.svg">
        NZ_KN708638 (7541130-7653004): Streptomyces sp. CT34 Scaffold1, whole genome ... (2% of genes show similarity), NRPS-like,T1PKS,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r93c1_3.svg">
        NZ_CP041695 (5984175-6106448): Nocardia otitidiscaviarum strain NEB252 chromo... (1% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r93c1_4.svg">
        NZ_FAOZ01000001 (39973-112570): Frankia irregularis strain DSM 45899, whole g... (3% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r93c1_5.svg">
        NZ_CP023695 (161547-183370): Streptomyces alboniger strain ATCC 12461 chromos... (16% of genes show similarity), lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r93c1_6.svg">
        NZ_LIZN01000011 (30101-89796): Thermobifida halotolerans strain YIM 90462 sca... (3% of genes show similarity), LAP,T3PKS,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r93c1_7.svg">
        NZ_FMZK01000013 (87822-140445): Streptomyces prasinopilosus strain CGMCC 4.35... (7% of genes show similarity), NRPS-like,T1PKS,betalactone
      </option>
      
      <option value = "svg/clusterblast_r93c1_8.svg">
        NZ_CP010407 (398539-466549): Streptomyces vietnamensis strain GIMV4.0001 chro... (4% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r93c1_9.svg">
        NZ_VISR01000007 (183433-224807): Streptomyces sp. BK340 Ga0307703 107, whole ... (9% of genes show similarity), other
      </option>
      
      <option value = "svg/clusterblast_r93c1_10.svg">
        NZ_LIVQ01000005 (226975-299470): Streptomyces sp. CB00455 scaffold5, whole ge... (3% of genes show similarity), T2PKS
      </option>
      
    </select>
    <button id= "clusterblast-r93c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r93c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r93c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-48"></div> <span class="help-tooltip" id="cb-known-help-48">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r93c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-49"></div> <span class="help-tooltip" id="cb-sub-help-49">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r93c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-50"></div> <span class="help-tooltip" id="MIBiG-help-50">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r93c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r93c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r93c1"></div>
    <div id="comparison-MIBiG-r93c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>auriculamide</td>
        
        
        <td>Herpetosiphon aurantiacus DSM 785</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514.1: 0-37469">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Other</td>
        
        <td>aristeromycin</td>
        
        
        <td>Streptomyces citricolor</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001972.1: 267-32147">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001972/index.html#r1c1" target="_new">BGC0001972.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>toxoflavin, fervenulin</td>
        
        
        <td>Streptomyces hiroshimensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001973.1: 267-32147">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001973/index.html#r1c1" target="_new">BGC0001973.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>fervenulin</td>
        
        
        <td>Streptomyces hiroshimensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.45, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>5-isoprenylindole-3-carboxylate -D-glycosyl ester</td>
        
        
        <td>Streptomyces sp. RM-5-8</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Terpene</td>
        
        <td>platencin</td>
        
        
        <td>Streptomyces platensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001810.1: 122-16114">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001810/index.html#r1c1" target="_new">BGC0001810.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Polyketide</td>
        
        <td>trehangelin</td>
        
        
        <td>Polymorphospora rubra</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.47, order:0.30, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
        <td>0.11</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000807.1: 6008597-6044686">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000807/index.html#r1c1" target="_new">BGC0000807.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.36 (id:0.48, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
         
        
        <td>0.10</td>
        <td>Saccharide</td>
        
        <td>phosphonoglycans</td>
        
        
        <td>Stackebrandtia nassauensis DSM 44728</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000383.1: 2221616-2239952">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000383/index.html#r1c1" target="_new">BGC0000383.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.35 (id:0.44, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
         
        
        <td>0.10</td>
        <td>NRP, Polyketide</td>
        
        <td>luminmycin A</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r93c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>auriculamide</td>
      <td>Herpetosiphon aurantiacus DSM 785</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514.1: 0-37469">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Other</td>
      <td>aristeromycin</td>
      <td>Streptomyces citricolor</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001972.1: 267-32147">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001972/index.html#r1c1" target="_new">BGC0001972.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Other</td>
      <td>toxoflavin, fervenulin</td>
      <td>Streptomyces hiroshimensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001973.1: 267-32147">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001973/index.html#r1c1" target="_new">BGC0001973.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Other</td>
      <td>fervenulin</td>
      <td>Streptomyces hiroshimensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.45, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other</td>
      <td>5-isoprenylindole-3-carboxylate -D-glycosyl ester</td>
      <td>Streptomyces sp. RM-5-8</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Terpene</td>
      <td>platencin</td>
      <td>Streptomyces platensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001810.1: 122-16114">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001810/index.html#r1c1" target="_new">BGC0001810.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Polyketide</td>
      <td>trehangelin</td>
      <td>Polymorphospora rubra</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.38 (id:0.47, order:0.30, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
      <td>0.38</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000807.1: 6008597-6044686">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000807/index.html#r1c1" target="_new">BGC0000807.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.36 (id:0.48, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
      <td>0.36</td>
      <td>Saccharide</td>
      <td>phosphonoglycans</td>
      <td>Stackebrandtia nassauensis DSM 44728</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000383.1: 2221616-2239952">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000383/index.html#r1c1" target="_new">BGC0000383.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.35 (id:0.44, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
      <td>0.35</td>
      <td>NRP, Polyketide</td>
      <td>luminmycin A</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r93c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-51"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-51">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r93c1-pfam-details-svg" data-filename="CC223A_contig_44_r93c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r93c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r95c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_5 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-149"></div> <span class="help-tooltip" id="region_svg-help-149">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_5.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r95c1-svg" data-filename="CC223A_contig_5_r95c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 104,656 - 130,302 nt. (total: 25,647 nt)
        
        <a class="cluster-rules-header" id="r95c1-rules-header" href="#r95c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r95c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r95c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-150"></div> <span class="help-tooltip" id="focus-panel-help-150">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r95c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r95c1-clusterblast" data-name="r95c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r95c1-knownclusterblast" data-name="r95c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r95c1-subclusterblast" data-name="r95c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r95c1-MIBiG-cluster-compare" data-name="r95c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r95c1-pfam-details" data-name="r95c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r95c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-52"></div> <span class="help-tooltip" id="cb-general-help-52">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r95c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r95c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r95c1_1.svg">
        NZ_FQZK01000005 (156992-182554): Nocardiopsis flavescens strain CGMCC 4.5723,... (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r95c1_2.svg">
        NZ_VWVW01000041 (10683-36398): Nocardiopsis sp. FR6 contig041, whole genome s... (86% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r95c1_3.svg">
        NZ_VWVV01000037 (10678-36393): Nocardiopsis sp. FR4 contig037, whole genome s... (86% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r95c1_4.svg">
        NZ_QGTF01000008 (298473-324087): Nocardiopsis sp. L17-MgMaSL7 Ga0060225 108, ... (90% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r95c1_5.svg">
        NZ_MCOK01000001 (3774048-3799908): Nocardiopsis sinuspersici strain UTMC102 N... (85% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r95c1_6.svg">
        NZ_BAZE01000013 (58110-83658): Nocardiopsis alba strain TP-A0876, whole genom... (90% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r95c1_7.svg">
        NZ_LR134501 (2632434-2656006): Nocardiopsis dassonvillei strain NCTC10488 chr... (80% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r95c1_8.svg">
        NZ_CP017965 (5046462-5072222): Nocardiopsis dassonvillei strain NOCA502F, com... (80% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r95c1_9.svg">
        NZ_CP022434 (5142886-5168645): Nocardiopsis dassonvillei strain HZNU N 1 chro... (77% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r95c1_10.svg">
        NZ_KB906857 (22875-48634): Nocardiopsis sp. CNS-639 strain CNS639 G011DRAFT s... (77% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r95c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r95c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r95c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-53"></div> <span class="help-tooltip" id="cb-known-help-53">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r95c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r95c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r95c1_1.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r95c1_2.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r95c1_3.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r95c1_4.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r95c1_5.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r95c1_6.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r95c1_7.svg">
        nocamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r95c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r95c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r95c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-54"></div> <span class="help-tooltip" id="cb-sub-help-54">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r95c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-55"></div> <span class="help-tooltip" id="MIBiG-help-55">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r95c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r95c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r95c1"></div>
    <div id="comparison-MIBiG-r95c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.76 (id:0.52, order:0.84, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
         
        
        <td>0.46</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.75 (id:0.43, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
         
        
        <td>0.45</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Rhodobacter sphaeroides</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.52, order:0.73, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.53, order:0.60, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656.1: 169-3770">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.45, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>Terpene</td>
        
        <td>zeaxanthin</td>
        
        
        <td>Xanthobacter autotrophicus Py2</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.53, order:0.52, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.52, order:0.52, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces collinus Tu 365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000639.1: 324-6564">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000639/index.html#r1c1" target="_new">BGC0000639.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Enterobacteriaceae bacterium DC260</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000641.1: 1208-7454">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000641/index.html#r1c1" target="_new">BGC0000641.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Enterobacteriaceae bacterium DC416</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000642.1: 1771-9054">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000642/index.html#r1c1" target="_new">BGC0000642.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Enterobacteriaceae bacterium DC413</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r95c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.76 (id:0.52, order:0.84, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
      <td>0.76</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.75 (id:0.43, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
      <td>0.75</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Rhodobacter sphaeroides</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.52, order:0.73, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.53, order:0.60, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656.1: 169-3770">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.45, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Terpene</td>
      <td>zeaxanthin</td>
      <td>Xanthobacter autotrophicus Py2</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.53, order:0.52, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.52, order:0.52, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces collinus Tu 365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000639.1: 324-6564">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000639/index.html#r1c1" target="_new">BGC0000639.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Enterobacteriaceae bacterium DC260</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000641.1: 1208-7454">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000641/index.html#r1c1" target="_new">BGC0000641.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Enterobacteriaceae bacterium DC416</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000642.1: 1771-9054">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000642/index.html#r1c1" target="_new">BGC0000642.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Enterobacteriaceae bacterium DC413</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r95c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-56"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-56">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r95c1-pfam-details-svg" data-filename="CC223A_contig_5_r95c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r95c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r98c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00098_CC223A_.. - Region 1 - LAP,thiopeptide <div class="help-container"> <div class="help-icon" data-id="region_svg-help-151"></div> <span class="help-tooltip" id="region_svg-help-151">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00098_CC223A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r98c1-svg" data-filename="c00098_CC223A_.._r98c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 3,597 - 19,058 nt. (total: 15,462 nt)
        
        <a class="cluster-rules-header" id="r98c1-rules-header" href="#r98c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r98c1-rules">LAP: ((((goadsporin_like or PF00881 or TIGR03605) and (YcaO or TIGR03882 or TIGR03604)) or TIGR03603 or TIGR03602) and not cyanobactin_synth)<br>thiopeptide: ((YcaO or TIGR03882) and ((thio_amide and (PF06968 or PF04055 or PF07366)) or Lant_dehydr_C or Lant_dehydr_N or PF07366 or PF06968 or PF04055) or thiostrepton)</div>
      
      <div class="region-svg-container">
       <div id='r98c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-152"></div> <span class="help-tooltip" id="focus-panel-help-152">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r98c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r98c1-thiopeptides" data-name="r98c1-thiopeptides">
        Thiopeptides
      </div>
     
    
     
      <div class="body-details-header r98c1-clusterblast" data-name="r98c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r98c1-knownclusterblast" data-name="r98c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r98c1-subclusterblast" data-name="r98c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r98c1-MIBiG-cluster-compare" data-name="r98c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r98c1-pfam-details" data-name="r98c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r98c1-thiopeptides">
      <div class="details">
    <div class="heading">
      <span>Thiopeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="thio-body-help-57"></div> <span class="help-tooltip" id="thio-body-help-57">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash. Predicted tail sequences are also shown.</span></div>
    </div>
  
    <div class="details-text">
      No core peptides found.
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r98c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-59"></div> <span class="help-tooltip" id="cb-general-help-59">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r98c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r98c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r98c1_1.svg">
        NZ_FQZK01000011 (132938-163488): Nocardiopsis flavescens strain CGMCC 4.5723,... (39% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r98c1_2.svg">
        NZ_VWVV01000010 (14266-79323): Nocardiopsis sp. FR4 contig010, whole genome s... (16% of genes show similarity), LAP,NRPS,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r98c1_3.svg">
        NZ_VWVW01000032 (2698-39366): Nocardiopsis sp. FR6 contig032, whole genome sh... (29% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r98c1_4.svg">
        NC_018524 (1661697-1743639): Nocardiopsis alba ATCC BAA-2165, complete sequence (12% of genes show similarity), LAP,NRPS,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r98c1_5.svg">
        NZ_BAZE01000010 (68643-96285): Nocardiopsis alba strain TP-A0876, whole genom... (33% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r98c1_6.svg">
        NZ_LWLB01000035 (211356-242066): Nocardiopsis sp. TSRI0078 scaffold7, whole g... (32% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r98c1_7.svg">
        NZ_VWVU01000004 (108615-139163): Nocardiopsis quinghaiensis strain YIM 28A4 c... (15% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r98c1_8.svg">
        NZ_MCOK01000001 (4757426-4788248): Nocardiopsis sinuspersici strain UTMC102 N... (11% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r98c1_9.svg">
        NZ_SMGH01000004 (289201-319032): Curtobacterium sp. PhB136 Ga0304861 101, who... (11% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r98c1_10.svg">
        NZ_RKII01000006 (866479-896310): Curtobacterium sp. PhB130 Ga0304858 101, who... (10% of genes show similarity), LAP,thiopeptide
      </option>
      
    </select>
    <button id= "clusterblast-r98c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r98c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r98c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-60"></div> <span class="help-tooltip" id="cb-known-help-60">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r98c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-61"></div> <span class="help-tooltip" id="cb-sub-help-61">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r98c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-62"></div> <span class="help-tooltip" id="MIBiG-help-62">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r98c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r98c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r98c1"></div>
    <div id="comparison-MIBiG-r98c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">LAP</span></th>
         
         <th><span class="cc-heat-product">thiopeptide</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000569.1: 0-9892">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000569/index.html#r1c1" target="_new">BGC0000569.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.52</td>
        <td>RiPP</td>
        
        <td>plantazolicin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001173.1: 223360-233305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001173/index.html#r1c1" target="_new">BGC0001173.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.51</td>
        <td>RiPP</td>
        
        <td>plantazolicin</td>
        
        
        <td>Bacillus pumilus ATCC 7061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001174.1: 1876430-1882234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001174/index.html#r1c1" target="_new">BGC0001174.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:0.42, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:0.42, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.46</td>
        <td>RiPP</td>
        
        <td>corynazolicin</td>
        
        
        <td>Corynebacterium urealyticum DSM 7111</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001146.1: 0-12498">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001146/index.html#r1c1" target="_new">BGC0001146.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.44</td>
        <td>RiPP</td>
        
        <td>cyclothiazomycin C</td>
        
        
        <td>Streptomyces sp. NRRL WC-3908</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001849.1: 155003-161953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001849/index.html#r1c1" target="_new">BGC0001849.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.40</td>
        <td>RiPP</td>
        
        <td>thiovarsolin A, thiovarsolin B, thiovarsolin C, thiovarsolin D</td>
        
        
        <td>Streptomyces varsoviensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001145.1: 35018-50744">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001145/index.html#r1c1" target="_new">BGC0001145.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.36</td>
        <td>RiPP</td>
        
        <td>cyclothiazomycin b1</td>
        
        
        <td>Streptomyces mobaraensis NBRC 13819 = DSM 40847</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.34</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000603.1: 383-24939">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000603/index.html#r1c1" target="_new">BGC0000603.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.33</td>
        <td>RiPP</td>
        
        <td>cyclothiazomycin</td>
        
        
        <td>Streptomyces hygroscopicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000606.1: 0-14244">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000606/index.html#r1c1" target="_new">BGC0000606.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:0.39, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:0.39, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.32</td>
        <td>RiPP</td>
        
        <td>lactazole</td>
        
        
        <td>Streptomyces lactacystinaeus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002013.1: 235087-246792">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002013/index.html#r1c1" target="_new">BGC0002013.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:0.31, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:0.31, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.32</td>
        <td>RiPP</td>
        
        <td>curacozole</td>
        
        
        <td>Streptomyces curacoi</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r98c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000569.1: 0-9892">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000569/index.html#r1c1" target="_new">BGC0000569.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>RiPP</td>
      <td>plantazolicin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001173.1: 223360-233305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001173/index.html#r1c1" target="_new">BGC0001173.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>RiPP</td>
      <td>plantazolicin</td>
      <td>Bacillus pumilus ATCC 7061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001174.1: 1876430-1882234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001174/index.html#r1c1" target="_new">BGC0001174.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.42, order:0.42, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>RiPP</td>
      <td>corynazolicin</td>
      <td>Corynebacterium urealyticum DSM 7111</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001146.1: 0-12498">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001146/index.html#r1c1" target="_new">BGC0001146.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>RiPP</td>
      <td>cyclothiazomycin C</td>
      <td>Streptomyces sp. NRRL WC-3908</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001849.1: 155003-161953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001849/index.html#r1c1" target="_new">BGC0001849.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>thiovarsolin A, thiovarsolin B, thiovarsolin C, thiovarsolin D</td>
      <td>Streptomyces varsoviensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001145.1: 35018-50744">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001145/index.html#r1c1" target="_new">BGC0001145.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>RiPP</td>
      <td>cyclothiazomycin b1</td>
      <td>Streptomyces mobaraensis NBRC 13819 = DSM 40847</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000603.1: 383-24939">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000603/index.html#r1c1" target="_new">BGC0000603.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>cyclothiazomycin</td>
      <td>Streptomyces hygroscopicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000606.1: 0-14244">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000606/index.html#r1c1" target="_new">BGC0000606.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:0.39, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>lactazole</td>
      <td>Streptomyces lactacystinaeus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002013.1: 235087-246792">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002013/index.html#r1c1" target="_new">BGC0002013.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:0.31, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>curacozole</td>
      <td>Streptomyces curacoi</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r98c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-63"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-63">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r98c1-pfam-details-svg" data-filename="c00098_CC223A_.._r98c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r98c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r98c1-thiopeptides" data-name="r98c1-thiopeptides">
        Thiopeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r98c1-thiopeptides">
      <div class="more-details">
    <div class="heading">
      <span>Core peptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="thio-side-help-58"></div> <span class="help-tooltip" id="thio-side-help-58">Lists the possible core peptides in the region. Each core peptide lists its possible molecular weights and the scores for cleavage site prediction and RODEO. If relevant, other features, such as macrocycle and amidation, will also be listed.</span></div>
    </div>
  
  <div class="prediction-text">No core peptides found.</div>

</div>
    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r101c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_22 - Region 1 - butyrolactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-153"></div> <span class="help-tooltip" id="region_svg-help-153">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_22.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r101c1-svg" data-filename="CC223A_contig_22_r101c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 14,569 - 25,624 nt. (total: 11,056 nt)
        
        <a class="cluster-rules-header" id="r101c1-rules-header" href="#r101c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r101c1-rules">butyrolactone: AfsA</div>
      
      <div class="region-svg-container">
       <div id='r101c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-154"></div> <span class="help-tooltip" id="focus-panel-help-154">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r101c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r101c1-clusterblast" data-name="r101c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r101c1-knownclusterblast" data-name="r101c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r101c1-subclusterblast" data-name="r101c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r101c1-MIBiG-cluster-compare" data-name="r101c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r101c1-pfam-details" data-name="r101c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r101c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-64"></div> <span class="help-tooltip" id="cb-general-help-64">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r101c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r101c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r101c1_1.svg">
        NZ_VWVS01000004 (446614-457667): Nocardiopsis sp. H13 contig004, whole genome... (100% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r101c1_2.svg">
        NZ_VWVW01000021 (69515-80568): Nocardiopsis sp. FR6 contig021, whole genome s... (100% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r101c1_3.svg">
        NZ_VWVV01000004 (262797-273850): Nocardiopsis sp. FR4 contig004, whole genome... (100% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r101c1_4.svg">
        NC_016109 (2543489-2553384): Kitasatospora setae KM-6054, complete genome (83% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r101c1_5.svg">
        NZ_CP009922 (1925838-1936218): Streptomyces xiamenensis strain MCCC 1A01550 c... (83% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r101c1_6.svg">
        NZ_KQ950180 (530779-541841): Thermobifida cellulosilytica TB100 scaffold1, wh... (100% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r101c1_7.svg">
        NZ_FQZK01000013 (105095-116139): Nocardiopsis flavescens strain CGMCC 4.5723,... (87% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r101c1_8.svg">
        NZ_CP043638 (4550091-4635172): Streptomyces sp. INR7 chromosome, complete genome (11% of genes show similarity), T1PKS,butyrolactone,ladderane,phosphonate
      </option>
      
      <option value = "svg/clusterblast_r101c1_9.svg">
        NZ_NBCN01000005 (221566-232145): Streptomyces sp. CB03238 scaffold5, whole ge... (44% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r101c1_10.svg">
        NZ_KQ948939 (8201-19221): Streptomyces caeruleatus strain NRRL B-24802 PRJNA2... (57% of genes show similarity), butyrolactone
      </option>
      
    </select>
    <button id= "clusterblast-r101c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r101c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r101c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-65"></div> <span class="help-tooltip" id="cb-known-help-65">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r101c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-66"></div> <span class="help-tooltip" id="cb-sub-help-66">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r101c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-67"></div> <span class="help-tooltip" id="MIBiG-help-67">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r101c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r101c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r101c1"></div>
    <div id="comparison-MIBiG-r101c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">butyrolactone</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>-butyrolactone</td>
        
        
        <td>Streptomyces acidiscabies</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001458.1: 4714477-4725572">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001458/index.html#r1c1" target="_new">BGC0001458.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.19</td>
        <td>Other,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
        
        <td>salinipostin G</td>
        
        
        <td>Salinispora tropica CNB-440</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000841.1: 1152118-1163141">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000841/index.html#r1c1" target="_new">BGC0000841.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
        
        <td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate, clavulanic acid</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001484.1: 0-36432">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001484/index.html#r1c1" target="_new">BGC0001484.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>2&#39;-chloropentostatin, 2&#39;-amino-2&#39;-deoxyadenosine</td>
        
        
        <td>Actinomadura sp. ATCC 39365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001223.1: 24900-34582">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001223/index.html#r1c1" target="_new">BGC0001223.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>hydroxysporine</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000665.1: 0-21259">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000665/index.html#r1c1" target="_new">BGC0000665.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Terpene, Alkaloid</td>
        
        <td>xiamycin</td>
        
        
        <td>Streptomyces sp. HKI0576</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000666.1: 0-29371">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000666/index.html#r1c1" target="_new">BGC0000666.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Terpene</td>
        
        <td>xiamycin A</td>
        
        
        <td>Streptomyces sp. SCSIO 02999</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000265.1: 0-12969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000265/index.html#r1c1" target="_new">BGC0000265.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>rhodomycin</td>
        
        
        <td>Streptomyces purpurascens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001387.1: 150-29437">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001387/index.html#r1c1" target="_new">BGC0001387.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>nucleocidin</td>
        
        
        <td>Streptomyces calvus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000698.1: 2269-33779">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000698/index.html#r1c1" target="_new">BGC0000698.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>hygromycin A</td>
        
        
        <td>Streptomyces hygroscopicus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r101c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>-butyrolactone</td>
      <td>Streptomyces acidiscabies</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001458.1: 4714477-4725572">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001458/index.html#r1c1" target="_new">BGC0001458.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Other,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
      <td>salinipostin G</td>
      <td>Salinispora tropica CNB-440</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000841.1: 1152118-1163141">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000841/index.html#r1c1" target="_new">BGC0000841.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
      <td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate, clavulanic acid</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001484.1: 0-36432">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001484/index.html#r1c1" target="_new">BGC0001484.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>2&#39;-chloropentostatin, 2&#39;-amino-2&#39;-deoxyadenosine</td>
      <td>Actinomadura sp. ATCC 39365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001223.1: 24900-34582">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001223/index.html#r1c1" target="_new">BGC0001223.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>hydroxysporine</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000665.1: 0-21259">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000665/index.html#r1c1" target="_new">BGC0000665.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Terpene, Alkaloid</td>
      <td>xiamycin</td>
      <td>Streptomyces sp. HKI0576</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000666.1: 0-29371">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000666/index.html#r1c1" target="_new">BGC0000666.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Terpene</td>
      <td>xiamycin A</td>
      <td>Streptomyces sp. SCSIO 02999</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000265.1: 0-12969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000265/index.html#r1c1" target="_new">BGC0000265.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Polyketide</td>
      <td>rhodomycin</td>
      <td>Streptomyces purpurascens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001387.1: 150-29437">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001387/index.html#r1c1" target="_new">BGC0001387.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other</td>
      <td>nucleocidin</td>
      <td>Streptomyces calvus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000698.1: 2269-33779">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000698/index.html#r1c1" target="_new">BGC0000698.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Saccharide</td>
      <td>hygromycin A</td>
      <td>Streptomyces hygroscopicus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r101c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-68"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-68">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r101c1-pfam-details-svg" data-filename="CC223A_contig_22_r101c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r101c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r103c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_66 - Region 1 - lanthipeptide-class-v <div class="help-container"> <div class="help-icon" data-id="region_svg-help-155"></div> <span class="help-tooltip" id="region_svg-help-155">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_66.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r103c1-svg" data-filename="CC223A_contig_66_r103c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 30,337 nt. (total: 30,337 nt)
        
        <a class="cluster-rules-header" id="r103c1-rules-header" href="#r103c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r103c1-rules">lanthipeptide-class-v: (APH and HopA1)</div>
      
      <div class="region-svg-container">
       <div id='r103c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-156"></div> <span class="help-tooltip" id="focus-panel-help-156">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r103c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r103c1-lanthipeptides" data-name="r103c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r103c1-clusterblast" data-name="r103c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r103c1-knownclusterblast" data-name="r103c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r103c1-subclusterblast" data-name="r103c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r103c1-MIBiG-cluster-compare" data-name="r103c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r103c1-pfam-details" data-name="r103c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r103c1-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-69"></div> <span class="help-tooltip" id="lanthi-body-help-69">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    <div class="details-text">
      Lanthipeptides - No core peptides found.
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r103c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-71"></div> <span class="help-tooltip" id="cb-general-help-71">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r103c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r103c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r103c1_1.svg">
        NZ_CP029241 (4610093-4728602): Streptomyces sp. NHF165 chromosome, complete g... (14% of genes show similarity), NRPS,T1PKS,lanthidin
      </option>
      
      <option value = "svg/clusterblast_r103c1_2.svg">
        NZ_QGGR01000009 (313795-355918): Actinoplanes xinjiangensis strain DSM 45184 ... (21% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r103c1_3.svg">
        NZ_CP029711 (10423180-10473520): Streptosporangium sp. &#39;caverna&#39; chromosome (21% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r103c1_4.svg">
        NZ_CM000950 (3952882-4003996): Streptomyces pristinaespiralis ATCC 25486 chro... (18% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r103c1_5.svg">
        NZ_JACHJG010000007 (354125-396212): Streptomyces netropsis strain CECT 3265 G... (20% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r103c1_6.svg">
        NZ_FUWS01000003 (183244-290796): Marinactinospora thermotolerans DSM 45154, w... (9% of genes show similarity), T2PKS,lanthidin
      </option>
      
      <option value = "svg/clusterblast_r103c1_7.svg">
        NZ_CP016438 (6900062-6941100): Streptomyces lincolnensis strain NRRL 2936 chr... (17% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r103c1_8.svg">
        NZ_CP026094 (7451582-7532434): Streptomyces albulus strain CK-15 chromosome, ... (15% of genes show similarity), T2PKS,lanthidin,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r103c1_9.svg">
        NZ_CP007574 (7475062-7559549): Streptomyces albulus strain NK660 chromosome, ... (14% of genes show similarity), T2PKS,lanthidin,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r103c1_10.svg">
        NZ_NDXL01000002 (1312054-1354198): Streptomyces kasugaensis strain BCRC 12349... (17% of genes show similarity), lanthidin
      </option>
      
    </select>
    <button id= "clusterblast-r103c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r103c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r103c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-72"></div> <span class="help-tooltip" id="cb-known-help-72">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r103c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r103c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r103c1_1.svg">
        caniferolide A / caniferolide B / caniferolide C / caniferolide D
      </option>
      
      <option value = "svg/knownclusterblast_r103c1_2.svg">
        avermectin
      </option>
      
      <option value = "svg/knownclusterblast_r103c1_3.svg">
        staurosporine
      </option>
      
      <option value = "svg/knownclusterblast_r103c1_4.svg">
        mithramycin
      </option>
      
      <option value = "svg/knownclusterblast_r103c1_5.svg">
        mithramycin
      </option>
      
      <option value = "svg/knownclusterblast_r103c1_6.svg">
        cyphomycin
      </option>
      
      <option value = "svg/knownclusterblast_r103c1_7.svg">
        sipanmycin
      </option>
      
      <option value = "svg/knownclusterblast_r103c1_8.svg">
        auroramycin
      </option>
      
      <option value = "svg/knownclusterblast_r103c1_9.svg">
        cinerubin B
      </option>
      
      <option value = "svg/knownclusterblast_r103c1_10.svg">
        saquayamycin Z
      </option>
      
    </select>
    <button id="knownclusterblast-r103c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r103c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r103c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-73"></div> <span class="help-tooltip" id="cb-sub-help-73">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r103c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r103c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r103c1_1.svg">
        FJ483966_c3: polyketomycin beta-D-amicetose / alpha-L-axenose (45% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r103c1_2.svg">
        AB032523_c1: avermectin oleandrose (50% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r103c1_3.svg">
        AF333038_c2: avilamycin D-olivose, 2-deoxy-D-evalose, 4-O-methyl-D-fucose (23% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r103c1_4.svg">
        AF080235_c2: landomycin D-olivose / L-rhodinose (25% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r103c1_5.svg">
        AB103463_c2: medermycin aminodeoxysugar (angolosamine) (42% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r103c1_6.svg">
        AB088119_c2: staurosporine deoxysugar (30% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r103c1_7.svg">
        DQ116941_c2: chlorothricin D-olivose (42% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r103c1_8.svg">
        GU233672_c2: granaticin deoxysugar (30% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r103c1_9.svg">
        DQ149987_c2: concanamycin 4&#39;-O-carbamoyl-2&#39;-deoxyrhamnose (42% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r103c1_10.svg">
        GY241466_c2: everninomicin L- and D-deoxyglucose, D-deoxymannose (22% of genes show similarity), sugar
      </option>
      
    </select>
    <button id="subclusterblast-r103c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r103c1-svg"></div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r103c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-74"></div> <span class="help-tooltip" id="MIBiG-help-74">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r103c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r103c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r103c1"></div>
    <div id="comparison-MIBiG-r103c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-v</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000716.1: 435-27161">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000716/index.html#r1c1" target="_new">BGC0000716.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.51, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>Saccharide</td>
        
        <td>spectinomycin</td>
        
        
        <td>Streptomyces netropsis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000813.1: 0-20300">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000813/index.html#r1c1" target="_new">BGC0000813.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.54, order:0.38, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Alkaloid</td>
        
        <td>K-252a</td>
        
        
        <td>Nonomuraea longicatena</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000632.1: 2720626-2736748">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000632/index.html#r1c1" target="_new">BGC0000632.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>Terpene, Saccharide</td>
        
        <td>brasilicardin A</td>
        
        
        <td>Nocardia terpenica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000192.1: 0-15150">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000192/index.html#r1c1" target="_new">BGC0000192.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.51, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>aclacinomycin</td>
        
        
        <td>Streptomyces galilaeus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000723.1: 1114-36970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000723/index.html#r1c1" target="_new">BGC0000723.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Saccharide</td>
        
        <td>-D-galactosylvalidoxylamine-A</td>
        
        
        <td>Streptomyces hygroscopicus subsp. limoneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000690.1: 13-25423">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000690/index.html#r1c1" target="_new">BGC0000690.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.53, order:0.36, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Saccharide</td>
        
        <td>hydroxystreptomycin</td>
        
        
        <td>Streptomyces glaucescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001618.1: 70-8595">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001618/index.html#r1c1" target="_new">BGC0001618.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>RiPP</td>
        
        <td>hyicin 3682</td>
        
        
        <td>Staphylococcus hyicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000715.1: 0-17984">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000715/index.html#r1c1" target="_new">BGC0000715.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.47, order:0.33, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Saccharide</td>
        
        <td>actinospectacin</td>
        
        
        <td>Streptomyces spectabilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000702.1: 2-25370">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000702/index.html#r1c1" target="_new">BGC0000702.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000796.1: 0-27180">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000796/index.html#r1c1" target="_new">BGC0000796.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.47, order:0.35, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Saccharide</td>
        
        <td>S-layer glycan</td>
        
        
        <td>Paenibacillus alvei</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r103c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000716.1: 435-27161">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000716/index.html#r1c1" target="_new">BGC0000716.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.51, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Saccharide</td>
      <td>spectinomycin</td>
      <td>Streptomyces netropsis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000813.1: 0-20300">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000813/index.html#r1c1" target="_new">BGC0000813.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.54, order:0.38, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Alkaloid</td>
      <td>K-252a</td>
      <td>Nonomuraea longicatena</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000632.1: 2720626-2736748">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000632/index.html#r1c1" target="_new">BGC0000632.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Terpene, Saccharide</td>
      <td>brasilicardin A</td>
      <td>Nocardia terpenica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000192.1: 0-15150">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000192/index.html#r1c1" target="_new">BGC0000192.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.51, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Polyketide</td>
      <td>aclacinomycin</td>
      <td>Streptomyces galilaeus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000723.1: 1114-36970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000723/index.html#r1c1" target="_new">BGC0000723.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Saccharide</td>
      <td>-D-galactosylvalidoxylamine-A</td>
      <td>Streptomyces hygroscopicus subsp. limoneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000690.1: 13-25423">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000690/index.html#r1c1" target="_new">BGC0000690.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.53, order:0.36, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Saccharide</td>
      <td>hydroxystreptomycin</td>
      <td>Streptomyces glaucescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001618.1: 70-8595">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001618/index.html#r1c1" target="_new">BGC0001618.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>RiPP</td>
      <td>hyicin 3682</td>
      <td>Staphylococcus hyicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000715.1: 0-17984">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000715/index.html#r1c1" target="_new">BGC0000715.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.47, order:0.33, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Saccharide</td>
      <td>actinospectacin</td>
      <td>Streptomyces spectabilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000702.1: 2-25370">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000702/index.html#r1c1" target="_new">BGC0000702.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000796.1: 0-27180">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000796/index.html#r1c1" target="_new">BGC0000796.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.47, order:0.35, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Saccharide</td>
      <td>S-layer glycan</td>
      <td>Paenibacillus alvei</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r103c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-75"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-75">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r103c1-pfam-details-svg" data-filename="CC223A_contig_66_r103c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r103c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r103c1-lanthipeptides" data-name="r103c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r103c1-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-70"></div> <span class="help-tooltip" id="lanthi-side-help-70">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

  <div class="prediction-text">No core peptides found.</div>

</div>

    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r110c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_46 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-157"></div> <span class="help-tooltip" id="region_svg-help-157">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_46.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r110c1-svg" data-filename="CC223A_contig_46_r110c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 45,272 nt. (total: 45,272 nt)
        
        <a class="cluster-rules-header" id="r110c1-rules-header" href="#r110c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r110c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r110c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-158"></div> <span class="help-tooltip" id="focus-panel-help-158">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r110c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r110c1-nrps_pks" data-name="r110c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r110c1-clusterblast" data-name="r110c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r110c1-knownclusterblast" data-name="r110c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r110c1-subclusterblast" data-name="r110c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r110c1-MIBiG-cluster-compare" data-name="r110c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r110c1-pfam-details" data-name="r110c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r110c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-76"></div> <span class="help-tooltip" id="nrps-domain-help-76">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r110c1-details-svg" data-filename="CC223A_contig_46_r110c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r110c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r110c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-79"></div> <span class="help-tooltip" id="cb-general-help-79">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r110c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r110c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r110c1_1.svg">
        NC_018524 (4467870-4546632): Nocardiopsis alba ATCC BAA-2165, complete sequence (20% of genes show similarity), NRPS,ectoine
      </option>
      
      <option value = "svg/clusterblast_r110c1_2.svg">
        NZ_BAZE01000014 (41461-120203): Nocardiopsis alba strain TP-A0876, whole geno... (21% of genes show similarity), NRPS,ectoine
      </option>
      
      <option value = "svg/clusterblast_r110c1_3.svg">
        NZ_QGTF01000011 (131802-195193): Nocardiopsis sp. L17-MgMaSL7 Ga0060225 111, ... (28% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r110c1_4.svg">
        NC_007333 (2152723-2214822): Thermobifida fusca YX, complete genome (31% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r110c1_5.svg">
        NZ_KQ950181 (204224-266537): Thermobifida cellulosilytica TB100 scaffold2, wh... (31% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r110c1_6.svg">
        NZ_PYGA01000006 (192314-251463): Murinocardiopsis flavida strain DSM 45312 Ga... (27% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r110c1_7.svg">
        NZ_VTZW01000019 (142180-206393): Streptomonospora sp. PA3 scaffold6.1, whole ... (30% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r110c1_8.svg">
        NZ_CP036455 (3239083-3306974): Streptomonospora sp. M2 chromosome, complete g... (30% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r110c1_9.svg">
        NZ_WPNZ01000006 (70926-138819): Streptomyces sp. p1417 Scaffold6, whole genom... (17% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r110c1_10.svg">
        NZ_QEKW01000001 (503288-585948): Actinomycetospora cinnamomea strain DSM 4577... (17% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r110c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r110c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r110c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-80"></div> <span class="help-tooltip" id="cb-known-help-80">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r110c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r110c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r110c1_1.svg">
        fuscachelin A / fuscachelin B / fuscachelin C
      </option>
      
      <option value = "svg/knownclusterblast_r110c1_2.svg">
        qinichelins
      </option>
      
      <option value = "svg/knownclusterblast_r110c1_3.svg">
        erythrochelin
      </option>
      
      <option value = "svg/knownclusterblast_r110c1_4.svg">
        gobichelin A / gobichelin B
      </option>
      
      <option value = "svg/knownclusterblast_r110c1_5.svg">
        salinichelins
      </option>
      
      <option value = "svg/knownclusterblast_r110c1_6.svg">
        bacillibactin
      </option>
      
      <option value = "svg/knownclusterblast_r110c1_7.svg">
        scabichelin
      </option>
      
      <option value = "svg/knownclusterblast_r110c1_8.svg">
        albachelin
      </option>
      
      <option value = "svg/knownclusterblast_r110c1_9.svg">
        atratumycin
      </option>
      
      <option value = "svg/knownclusterblast_r110c1_10.svg">
        myxochelin A / myxochelin B
      </option>
      
    </select>
    <button id="knownclusterblast-r110c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r110c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r110c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-81"></div> <span class="help-tooltip" id="cb-sub-help-81">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r110c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-82"></div> <span class="help-tooltip" id="MIBiG-help-82">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r110c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r110c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r110c1"></div>
    <div id="comparison-MIBiG-r110c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000359.1: 2172723-2196882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000359/index.html#r1c1" target="_new">BGC0000359.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.80 (id:0.55, order:1.00, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
         
        
        <td>0.53</td>
        <td>NRP</td>
        
        <td>fuscachelin A, fuscachelin B, fuscachelin C</td>
        
        
        <td>Thermobifida fusca YX</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.77 (id:0.50, order:0.90, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
         
        
        <td>0.47</td>
        <td>NRP</td>
        
        <td>bacillibactin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.44, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
        <td>0.42</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.46, order:1.00, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
        <td>0.41</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401.1: 0-12882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.51, order:0.76, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>NRP</td>
        
        <td>paenibactin</td>
        
        
        <td>Paenibacillus elgii B69</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.45, order:1.00, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050.1: 0-19730">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.45, order:1.00, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>NRP, Polyketide</td>
        
        <td>thalassospiramide A</td>
        
        
        <td>Tistrella bauzanensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>hexose-palythine-serine, hexose-shinorine</td>
        
        
        <td>Heteroscytonema crispum UCFS10</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r110c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000359.1: 2172723-2196882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000359/index.html#r1c1" target="_new">BGC0000359.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.80 (id:0.55, order:1.00, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
      <td>0.80</td>
      <td>NRP</td>
      <td>fuscachelin A, fuscachelin B, fuscachelin C</td>
      <td>Thermobifida fusca YX</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.77 (id:0.50, order:0.90, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
      <td>0.77</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.44, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.46, order:1.00, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401.1: 0-12882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.51, order:0.76, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>NRP</td>
      <td>paenibactin</td>
      <td>Paenibacillus elgii B69</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.45, order:1.00, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050.1: 0-19730">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.45, order:1.00, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>NRP, Polyketide</td>
      <td>thalassospiramide A</td>
      <td>Tistrella bauzanensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>hexose-palythine-serine, hexose-shinorine</td>
      <td>Heteroscytonema crispum UCFS10</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r110c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-83"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-83">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r110c1-pfam-details-svg" data-filename="CC223A_contig_46_r110c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r110c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r110c1-nrps_pks_products" data-name="r110c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r110c1-nrps_pks" data-name="r110c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r110c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-77"></div> <span class="help-tooltip" id="nrps-structures-help-77">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 45272:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r110c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NCC(=O)NCC(=O)NC(CO)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-X - gly - gly) + (ser) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x,gly,gly&amp;nrps2=ser&amp;nrps3=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x*,gly*,gly*&amp;nrps2=ser*&amp;nrps3=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r110c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NCC(=O)NCC(=O)NC(CO)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r110c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-78"></div> <span class="help-tooltip" id="nrps-monomers-help-78">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds2812_4504</span></strong>: dhb
      <div class="collapser collapser-target-cds2812_4504 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=diOH-Bz" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=diOH-Bz*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (50..462)</strong>: dhb
          <div class="collapser collapser-target-cds2812_4504 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  dhb
              <div class="collapser collapser-target-cds2812_4504-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>dhb, sal</dd>
   <dt>Small clusters prediction:</dt>
   <dd>dhb, sal</dd>
   <dt>Single AA prediction:</dt>
   <dd>dhb</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">PMPAQGVVNK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>dhb</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds6797_17780</span></strong>: X - gly - gly
      <div class="collapser collapser-target-cds6797_17780 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,gly,gly" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,gly*,gly*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (491..912)</strong>: X
          <div class="collapser collapser-target-cds6797_17780 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds6797_17780-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAwDtGfVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2019..2403)</strong>: gly
          <div class="collapser collapser-target-cds6797_17780 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-cds6797_17780-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILQLGVIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3062..3453)</strong>: gly
          <div class="collapser collapser-target-cds6797_17780 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-cds6797_17780-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILQLGVIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds17776_21160</span></strong>: ser
      <div class="collapser collapser-target-cds17776_21160 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (459..917)</strong>: ser
          <div class="collapser collapser-target-cds17776_21160 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds17776_21160-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWHISLIDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds21192_25272</span></strong>: X
      <div class="collapser collapser-target-cds21192_25272 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (468..874)</strong>: X
          <div class="collapser collapser-target-cds21192_25272 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds21192_25272-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DMwNLGLINK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r132c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00132_CC223A_.. - Region 1 - RRE-containing <div class="help-container"> <div class="help-icon" data-id="region_svg-help-159"></div> <span class="help-tooltip" id="region_svg-help-159">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00132_CC223A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r132c1-svg" data-filename="c00132_CC223A_.._r132c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 3,419 nt. (total: 3,419 nt)
        
        <a class="cluster-rules-header" id="r132c1-rules-header" href="#r132c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r132c1-rules">RRE-containing: minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE])</div>
      
      <div class="region-svg-container">
       <div id='r132c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-160"></div> <span class="help-tooltip" id="focus-panel-help-160">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r132c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r132c1-clusterblast" data-name="r132c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r132c1-knownclusterblast" data-name="r132c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r132c1-subclusterblast" data-name="r132c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r132c1-MIBiG-cluster-compare" data-name="r132c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r132c1-pfam-details" data-name="r132c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r132c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-84"></div> <span class="help-tooltip" id="cb-general-help-84">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r132c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r132c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r132c1_1.svg">
        NZ_FQZK01000011 (132938-163488): Nocardiopsis flavescens strain CGMCC 4.5723,... (13% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r132c1_2.svg">
        NZ_LWLB01000035 (211356-242066): Nocardiopsis sp. TSRI0078 scaffold7, whole g... (12% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r132c1_3.svg">
        NZ_VWVW01000032 (2698-39366): Nocardiopsis sp. FR6 contig032, whole genome sh... (11% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r132c1_4.svg">
        NZ_MCOK01000001 (4757426-4788248): Nocardiopsis sinuspersici strain UTMC102 N... (11% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r132c1_5.svg">
        NZ_VWVU01000004 (108615-139163): Nocardiopsis quinghaiensis strain YIM 28A4 c... (11% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r132c1_6.svg">
        NZ_VWVV01000010 (14266-79323): Nocardiopsis sp. FR4 contig010, whole genome s... (6% of genes show similarity), LAP,NRPS,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r132c1_7.svg">
        NZ_BAZE01000010 (68643-96285): Nocardiopsis alba strain TP-A0876, whole genom... (14% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r132c1_8.svg">
        NC_018524 (1661697-1743639): Nocardiopsis alba ATCC BAA-2165, complete sequence (3% of genes show similarity), LAP,NRPS,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r132c1_9.svg">
        NZ_CP036403 (1970011-2000634): Ornithinimicrobium sp. HY006 chromosome, compl... (8% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r132c1_10.svg">
        NZ_SMGH01000004 (289201-319032): Curtobacterium sp. PhB136 Ga0304861 101, who... (7% of genes show similarity), LAP,thiopeptide
      </option>
      
    </select>
    <button id= "clusterblast-r132c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r132c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r132c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-85"></div> <span class="help-tooltip" id="cb-known-help-85">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r132c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-86"></div> <span class="help-tooltip" id="cb-sub-help-86">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r132c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-88"></div> <span class="help-tooltip" id="MIBiG-help-88">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r132c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r132c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r132c1"></div>
    <div id="comparison-MIBiG-r132c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">RRE-containing</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000606.1: 0-14244">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000606/index.html#r1c1" target="_new">BGC0000606.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.43, order:0.69, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>RiPP</td>
        
        <td>lactazole</td>
        
        
        <td>Streptomyces lactacystinaeus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001860.1: 604805-621192">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001860/index.html#r1c1" target="_new">BGC0001860.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.33 (id:0.42, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
         
        
        <td>0.09</td>
        <td>RiPP</td>
        
        <td>thiomuracin</td>
        
        
        <td>Thermobispora bispora DSM 43833</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r132c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000606.1: 0-14244">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000606/index.html#r1c1" target="_new">BGC0000606.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.43, order:0.69, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>RiPP</td>
      <td>lactazole</td>
      <td>Streptomyces lactacystinaeus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001860.1: 604805-621192">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001860/index.html#r1c1" target="_new">BGC0001860.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.33 (id:0.42, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
      <td>0.33</td>
      <td>RiPP</td>
      <td>thiomuracin</td>
      <td>Thermobispora bispora DSM 43833</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r132c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-89"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-89">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r132c1-pfam-details-svg" data-filename="c00132_CC223A_.._r132c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r132c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r132c1-RREfinder" data-name="r132c1-RREfinder">
        RREFinder
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r132c1-RREfinder">
      <div class="more-details">
  <div class="heading">
    <span>RRE predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="rre-side-help-87"></div> <span class="help-tooltip" id="rre-side-help-87">RREfinder results sidepanel.</span></div>
  </div>
  <dl class = 'prediction-text'>
    
      <span><strong>RRE-containing protocluster</strong> (0...3419)</span>
      <dt></dt>
      
      
      
        
        <strong>cds1_520</strong> <div class="collapser collapser-target-cds1_520 collapser-level-cds"> </div><div class="collapser-content">
        
          <dd>RRE type: Thiopeptide_F_RRE</dd>
          <dd>Location: 12 - 81</dd>
          <dd>Score: 29.7</dd>
        
        </div><br>
      <br>
    
    </dl>
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r155c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_10 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-161"></div> <span class="help-tooltip" id="region_svg-help-161">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_10.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r155c1-svg" data-filename="CC223A_contig_10_r155c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 6,406 - 64,045 nt. (total: 57,640 nt)
        
        <a class="cluster-rules-header" id="r155c1-rules-header" href="#r155c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r155c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r155c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-162"></div> <span class="help-tooltip" id="focus-panel-help-162">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r155c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r155c1-nrps_pks" data-name="r155c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r155c1-clusterblast" data-name="r155c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r155c1-knownclusterblast" data-name="r155c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r155c1-subclusterblast" data-name="r155c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r155c1-MIBiG-cluster-compare" data-name="r155c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r155c1-pfam-details" data-name="r155c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r155c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-90"></div> <span class="help-tooltip" id="nrps-domain-help-90">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r155c1-details-svg" data-filename="CC223A_contig_10_r155c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r155c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r155c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-93"></div> <span class="help-tooltip" id="cb-general-help-93">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r155c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r155c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r155c1_1.svg">
        NZ_FQZK01000002 (27104-84713): Nocardiopsis flavescens strain CGMCC 4.5723, w... (100% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r155c1_2.svg">
        NZ_QGTF01000001 (493592-553681): Nocardiopsis sp. L17-MgMaSL7 Ga0060225 101, ... (94% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r155c1_3.svg">
        NZ_CP017965 (1348462-1406121): Nocardiopsis dassonvillei strain NOCA502F, com... (84% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r155c1_4.svg">
        NZ_CP022434 (1371384-1430115): Nocardiopsis dassonvillei strain HZNU N 1 chro... (89% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r155c1_5.svg">
        NZ_KB906854 (58742-118971): Nocardiopsis sp. CNS-639 strain CNS639 G011DRAFT ... (89% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r155c1_6.svg">
        NZ_MCOK01000001 (542010-600312): Nocardiopsis sinuspersici strain UTMC102 NOS... (86% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r155c1_7.svg">
        NZ_LWLB01000012 (477811-535551): Nocardiopsis sp. TSRI0078 scaffold2, whole g... (75% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r155c1_8.svg">
        NZ_BAZE01000009 (87533-143372): Nocardiopsis alba strain TP-A0876, whole geno... (77% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r155c1_9.svg">
        NC_018524 (3481637-3537422): Nocardiopsis alba ATCC BAA-2165, complete sequence (75% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r155c1_10.svg">
        NZ_VWVU01000017 (5049-62773): Nocardiopsis quinghaiensis strain YIM 28A4 cont... (74% of genes show similarity), T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r155c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r155c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r155c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-94"></div> <span class="help-tooltip" id="cb-known-help-94">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r155c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r155c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r155c1_1.svg">
        chalcomycin A
      </option>
      
      <option value = "svg/knownclusterblast_r155c1_2.svg">
        10,11-dihydro-8-deoxy-12,13-deepoxy-12,13-dihydrochalcomycin
      </option>
      
      <option value = "svg/knownclusterblast_r155c1_3.svg">
        tylactone
      </option>
      
      <option value = "svg/knownclusterblast_r155c1_4.svg">
        aldgamycin J / aldgamycin K / aldgamycin P / aldgamycin E
      </option>
      
      <option value = "svg/knownclusterblast_r155c1_5.svg">
        aureothin
      </option>
      
      <option value = "svg/knownclusterblast_r155c1_6.svg">
        pentamycin
      </option>
      
      <option value = "svg/knownclusterblast_r155c1_7.svg">
        filipin
      </option>
      
      <option value = "svg/knownclusterblast_r155c1_8.svg">
        laidlomycin
      </option>
      
      <option value = "svg/knownclusterblast_r155c1_9.svg">
        nanchangmycin
      </option>
      
      <option value = "svg/knownclusterblast_r155c1_10.svg">
        nigericin
      </option>
      
    </select>
    <button id="knownclusterblast-r155c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r155c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r155c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-95"></div> <span class="help-tooltip" id="cb-sub-help-95">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r155c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-96"></div> <span class="help-tooltip" id="MIBiG-help-96">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r155c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r155c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r155c1"></div>
    <div id="comparison-MIBiG-r155c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.74 (id:0.48, order:1.00, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
         
        
        <td>0.43</td>
        <td>Polyketide</td>
        
        <td>oronofacic acid</td>
        
        
        <td>Pseudomonas syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Polyketide</td>
        
        <td>strobilurin</td>
        
        
        <td>Strobilurus tenacellus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001647.1: 413229-431538">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001647/index.html#r1c1" target="_new">BGC0001647.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Polyketide</td>
        
        <td>lagunapyrone A, lagunapyrone B, lagunapyrone C</td>
        
        
        <td>Streptomyces sp. MP131-18</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Polyketide</td>
        
        <td>depudecin</td>
        
        
        <td>Alternaria brassicicola</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000067.1: 14949-58348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000067/index.html#r1c1" target="_new">BGC0000067.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.49, order:0.66, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Polyketide</td>
        
        <td>geldanamycin</td>
        
        
        <td>Streptomyces hygroscopicus subsp. duamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001544.1: 576137-603864">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001544/index.html#r1c1" target="_new">BGC0001544.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP, Polyketide</td>
        
        <td>chlorflavonin</td>
        
        
        <td>Aspergillus campestris IBT 28561</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>6-methylsalicyclic acid</td>
        
        
        <td>Aspergillus terreus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>asperlactone</td>
        
        
        <td>Aspergillus ochraceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>(-)-Mellein</td>
        
        
        <td>Parastagonospora nodorum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998.1: 1175677-1189808">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Polyketide</td>
        
        <td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r155c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.74 (id:0.48, order:1.00, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
      <td>0.74</td>
      <td>Polyketide</td>
      <td>oronofacic acid</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Polyketide</td>
      <td>strobilurin</td>
      <td>Strobilurus tenacellus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001647.1: 413229-431538">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001647/index.html#r1c1" target="_new">BGC0001647.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>lagunapyrone A, lagunapyrone B, lagunapyrone C</td>
      <td>Streptomyces sp. MP131-18</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>depudecin</td>
      <td>Alternaria brassicicola</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000067.1: 14949-58348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000067/index.html#r1c1" target="_new">BGC0000067.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.49, order:0.66, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>geldanamycin</td>
      <td>Streptomyces hygroscopicus subsp. duamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001544.1: 576137-603864">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001544/index.html#r1c1" target="_new">BGC0001544.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP, Polyketide</td>
      <td>chlorflavonin</td>
      <td>Aspergillus campestris IBT 28561</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Aspergillus terreus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>asperlactone</td>
      <td>Aspergillus ochraceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>(-)-Mellein</td>
      <td>Parastagonospora nodorum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998.1: 1175677-1189808">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r155c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-97"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-97">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r155c1-pfam-details-svg" data-filename="CC223A_contig_10_r155c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r155c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r155c1-nrps_pks_products" data-name="r155c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r155c1-nrps_pks" data-name="r155c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r155c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-91"></div> <span class="help-tooltip" id="nrps-structures-help-91">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 6405 - 64045:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r155c1-1" class="smiles-canvas" data-smiles="C(C)C(=O)C(C)C(=O)C(C)=CC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(Me-mal) + (Me-mal) + (Me-ccmal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-mal&amp;nrps2=me-mal&amp;nrps3=me-ccmal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-mal*&amp;nrps2=me-mal*&amp;nrps3=me-ccmal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r155c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C(C)C(=O)C(C)C(=O)C(C)=CC(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r155c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-92"></div> <span class="help-tooltip" id="nrps-monomers-help-92">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds26405_32648</span></strong>: mmal
      <div class="collapser collapser-target-cds26405_32648 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (579..874)</strong>: mmal
          <div class="collapser collapser-target-cds26405_32648 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds26405_32648-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>Methoxymalonyl-CoA: 62.5%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds26405_32648-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 149.1</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 135.4</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 134.9</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 105.2</dt>

  <dd></dd><dt>Propionyl-CoA: 94.0</dt>

  <dd></dd><dt>CHC-CoA: 92.1</dt>

  <dd></dd><dt>trans-1,2-CPDA: 88.4</dt>

  <dd></dd><dt>Malonyl-CoA: 61.7</dt>

  <dd></dd><dt>Isobutyryl-CoA: 41.9</dt>

  <dd></dd><dt>fatty_acid: 39.6</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 30.8</dt>

  <dd></dd><dt>Benzoyl-CoA: 28.2</dt>

  <dd></dd><dt>Acetyl-CoA: 26.3</dt>

  <dd></dd><dt>inactive: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds32680_35995</span></strong>: mmal
      <div class="collapser collapser-target-cds32680_35995 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (563..861)</strong>: mmal
          <div class="collapser collapser-target-cds32680_35995 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds32680_35995-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 95.8%</dd>
<dd>Malonyl-CoA: 87.5%</dd>
<dd>Methoxymalonyl-CoA: 87.5%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds32680_35995-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 194.1</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 187.9</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 169.9</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 134.1</dt>

  <dd></dd><dt>trans-1,2-CPDA: 121.7</dt>

  <dd></dd><dt>Propionyl-CoA: 116.7</dt>

  <dd></dd><dt>Malonyl-CoA: 103.3</dt>

  <dd></dd><dt>CHC-CoA: 90.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 87.5</dt>

  <dd></dd><dt>Benzoyl-CoA: 60.5</dt>

  <dd></dd><dt>Acetyl-CoA: 54.8</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 42.1</dt>

  <dd></dd><dt>inactive: 33.8</dt>

  <dd></dd><dt>fatty_acid: 28.6</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds35981_44045</span></strong>: mmal
      <div class="collapser collapser-target-cds35981_44045 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (524..828)</strong>: mmal
          <div class="collapser collapser-target-cds35981_44045 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds35981_44045-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 91.7%</dd>
<dd>Methoxymalonyl-CoA: 83.3%</dd>
<dd>Ethylmalonyl-CoA: 79.2%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds35981_44045-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 189.1</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 173.1</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 149.5</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 128.2</dt>

  <dd></dd><dt>trans-1,2-CPDA: 107.7</dt>

  <dd></dd><dt>Propionyl-CoA: 100.9</dt>

  <dd></dd><dt>CHC-CoA: 96.4</dt>

  <dd></dd><dt>Malonyl-CoA: 83.1</dt>

  <dd></dd><dt>Isobutyryl-CoA: 76.5</dt>

  <dd></dd><dt>Benzoyl-CoA: 38.2</dt>

  <dd></dd><dt>inactive: 31.4</dt>

  <dd></dd><dt>Acetyl-CoA: 30.6</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 22.2</dt>

  <dd></dd><dt>fatty_acid: 18.8</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r155c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_10 - Region 2 - butyrolactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-163"></div> <span class="help-tooltip" id="region_svg-help-163">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_10.region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r155c2-svg" data-filename="CC223A_contig_10_r155c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 81,271 - 92,047 nt. (total: 10,777 nt)
        
        <a class="cluster-rules-header" id="r155c2-rules-header" href="#r155c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r155c2-rules">butyrolactone: AfsA</div>
      
      <div class="region-svg-container">
       <div id='r155c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-164"></div> <span class="help-tooltip" id="focus-panel-help-164">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r155c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r155c2-clusterblast" data-name="r155c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r155c2-knownclusterblast" data-name="r155c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r155c2-subclusterblast" data-name="r155c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r155c2-MIBiG-cluster-compare" data-name="r155c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r155c2-pfam-details" data-name="r155c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r155c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-98"></div> <span class="help-tooltip" id="cb-general-help-98">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r155c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r155c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r155c2_1.svg">
        NZ_VWVS01000035 (14699-25521): Nocardiopsis sp. H13 contig035, whole genome s... (60% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r155c2_2.svg">
        NZ_MCOK01000001 (2402670-2413402): Nocardiopsis sinuspersici strain UTMC102 N... (66% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r155c2_3.svg">
        NZ_QJRW01000003 (568743-579550): Pseudomonas chlororaphis strain LBs-160603 N... (57% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r155c2_4.svg">
        NZ_CP027710 (4099505-4110312): Pseudomonas chlororaphis subsp. piscium strain... (57% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r155c2_5.svg">
        NZ_CP027736 (4029800-4040607): Pseudomonas chlororaphis subsp. piscium strain... (57% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r155c2_6.svg">
        NZ_CP027737 (4064454-4075261): Pseudomonas chlororaphis subsp. piscium strain... (57% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r155c2_7.svg">
        NZ_CP027739 (4109486-4120293): Pseudomonas chlororaphis subsp. piscium strain... (57% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r155c2_8.svg">
        NZ_CP027740 (4113695-4124502): Pseudomonas chlororaphis subsp. piscium strain... (57% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r155c2_9.svg">
        NZ_CP027711 (4084250-4095057): Pseudomonas chlororaphis subsp. piscium strain... (57% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r155c2_10.svg">
        NZ_MOBW01000060 (384855-395662): Pseudomonas chlororaphis strain 48G9 48G9 NO... (57% of genes show similarity), butyrolactone
      </option>
      
    </select>
    <button id= "clusterblast-r155c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r155c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r155c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-99"></div> <span class="help-tooltip" id="cb-known-help-99">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r155c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-100"></div> <span class="help-tooltip" id="cb-sub-help-100">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r155c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-101"></div> <span class="help-tooltip" id="MIBiG-help-101">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r155c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r155c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r155c2"></div>
    <div id="comparison-MIBiG-r155c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">butyrolactone</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001935.1: 0-9112">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001935/index.html#r1c1" target="_new">BGC0001935.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>Other</td>
        
        <td>3,7-dihydroxytropolone</td>
        
        
        <td>Streptomyces cyaneofuscatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001487.1: 0-9112">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001487/index.html#r1c1" target="_new">BGC0001487.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>Other</td>
        
        <td>3,7-dihydroxytropolone</td>
        
        
        <td>Streptomyces cyaneofuscatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514.1: 0-37469">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Other</td>
        
        <td>aristeromycin</td>
        
        
        <td>Streptomyces citricolor</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000495.1: 41-38168">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000495/index.html#r1c1" target="_new">BGC0000495.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.50, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>RiPP</td>
        
        <td>actagardine</td>
        
        
        <td>Actinoplanes garbadinensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001440.1: 0-13771">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001440/index.html#r1c1" target="_new">BGC0001440.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.39 (id:0.42, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
         
        
        <td>0.12</td>
        <td>Other</td>
        
        <td>cystargolide A, cystargolide B</td>
        
        
        <td>Kitasatospora cystarginea</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001771.1: 13254-31732">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001771/index.html#r1c1" target="_new">BGC0001771.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.33 (id:0.50, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
         
        
        <td>0.09</td>
        <td>NRP</td>
        
        <td>scleric acid</td>
        
        
        <td>Streptomyces sclerotialus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001378.1: 3458-48711">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001378/index.html#r1c1" target="_new">BGC0001378.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.30 (id:0.42, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
         
        
        <td>0.08</td>
        <td>Polyketide</td>
        
        <td>tiancimycin</td>
        
        
        <td>Streptomyces sp. CB03234</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001804.1: 3458-48711">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001804/index.html#r1c1" target="_new">BGC0001804.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.30 (id:0.42, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
         
        
        <td>0.08</td>
        <td>Polyketide</td>
        
        <td>tiancimycin</td>
        
        
        <td>Streptomyces sp. CB03234</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001300.1: 649-59711">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001300/index.html#r1c1" target="_new">BGC0001300.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.29 (id:0.42, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
         
        
        <td>0.07</td>
        <td>Polyketide</td>
        
        <td>anthracimycin</td>
        
        
        <td>Streptomyces sp. CNH365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001301.1: 139-53189">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001301/index.html#r1c1" target="_new">BGC0001301.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.25 (id:0.42, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
         
        
        <td>0.06</td>
        <td>Polyketide</td>
        
        <td>anthracimycin</td>
        
        
        <td>Streptomyces sp. T676</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r155c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001935.1: 0-9112">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001935/index.html#r1c1" target="_new">BGC0001935.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>3,7-dihydroxytropolone</td>
      <td>Streptomyces cyaneofuscatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001487.1: 0-9112">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001487/index.html#r1c1" target="_new">BGC0001487.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>3,7-dihydroxytropolone</td>
      <td>Streptomyces cyaneofuscatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514.1: 0-37469">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Other</td>
      <td>aristeromycin</td>
      <td>Streptomyces citricolor</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000495.1: 41-38168">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000495/index.html#r1c1" target="_new">BGC0000495.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.50, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>RiPP</td>
      <td>actagardine</td>
      <td>Actinoplanes garbadinensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001440.1: 0-13771">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001440/index.html#r1c1" target="_new">BGC0001440.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.39 (id:0.42, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
      <td>0.39</td>
      <td>Other</td>
      <td>cystargolide A, cystargolide B</td>
      <td>Kitasatospora cystarginea</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001771.1: 13254-31732">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001771/index.html#r1c1" target="_new">BGC0001771.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.33 (id:0.50, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
      <td>0.33</td>
      <td>NRP</td>
      <td>scleric acid</td>
      <td>Streptomyces sclerotialus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001378.1: 3458-48711">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001378/index.html#r1c1" target="_new">BGC0001378.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.30 (id:0.42, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
      <td>0.30</td>
      <td>Polyketide</td>
      <td>tiancimycin</td>
      <td>Streptomyces sp. CB03234</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001804.1: 3458-48711">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001804/index.html#r1c1" target="_new">BGC0001804.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.30 (id:0.42, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
      <td>0.30</td>
      <td>Polyketide</td>
      <td>tiancimycin</td>
      <td>Streptomyces sp. CB03234</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001300.1: 649-59711">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001300/index.html#r1c1" target="_new">BGC0001300.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.29 (id:0.42, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
      <td>0.29</td>
      <td>Polyketide</td>
      <td>anthracimycin</td>
      <td>Streptomyces sp. CNH365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001301.1: 139-53189">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001301/index.html#r1c1" target="_new">BGC0001301.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.25 (id:0.42, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
      <td>0.25</td>
      <td>Polyketide</td>
      <td>anthracimycin</td>
      <td>Streptomyces sp. T676</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r155c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-102"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-102">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r155c2-pfam-details-svg" data-filename="CC223A_contig_10_r155c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r155c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r157c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_84 - Region 1 - ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-165"></div> <span class="help-tooltip" id="region_svg-help-165">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_84.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r157c1-svg" data-filename="CC223A_contig_84_r157c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 9,962 nt. (total: 9,962 nt)
        
        <a class="cluster-rules-header" id="r157c1-rules-header" href="#r157c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r157c1-rules">ectoine: ectoine_synt</div>
      
      <div class="region-svg-container">
       <div id='r157c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-166"></div> <span class="help-tooltip" id="focus-panel-help-166">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r157c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r157c1-clusterblast" data-name="r157c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r157c1-knownclusterblast" data-name="r157c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r157c1-subclusterblast" data-name="r157c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r157c1-MIBiG-cluster-compare" data-name="r157c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r157c1-pfam-details" data-name="r157c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r157c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-103"></div> <span class="help-tooltip" id="cb-general-help-103">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r157c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r157c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r157c1_1.svg">
        NZ_FQZK01000022 (19382-29775): Nocardiopsis flavescens strain CGMCC 4.5723, w... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r157c1_2.svg">
        NZ_BAZE01000003 (19209-29611): Nocardiopsis alba strain TP-A0876, whole genom... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r157c1_3.svg">
        NC_018524 (2927647-2937223): Nocardiopsis alba ATCC BAA-2165, complete sequence (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r157c1_4.svg">
        NZ_MCOK01000001 (5971971-5982361): Nocardiopsis sinuspersici strain UTMC102 N... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r157c1_5.svg">
        NZ_ML776455 (277383-287797): Nocardiopsis sp. FR4 Scaffold5, whole genome sho... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r157c1_6.svg">
        NZ_VWVW01000003 (232833-243247): Nocardiopsis sp. FR6 contig003, whole genome... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r157c1_7.svg">
        NZ_VWVU01000001 (419442-429850): Nocardiopsis quinghaiensis strain YIM 28A4 c... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r157c1_8.svg">
        NZ_AZXF01000030 (34590-45007): Nocardiopsis sp. CNT312 B042DRAFT scaffold 24.... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r157c1_9.svg">
        NZ_QGTF01000019 (54821-65223): Nocardiopsis sp. L17-MgMaSL7 Ga0060225 119, wh... (90% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r157c1_10.svg">
        NZ_JACHJO010000016 (17066-27465): Nocardiopsis algeriensis strain CECT 8712 G... (100% of genes show similarity), ectoine
      </option>
      
    </select>
    <button id= "clusterblast-r157c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r157c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r157c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-104"></div> <span class="help-tooltip" id="cb-known-help-104">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r157c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r157c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r157c1_1.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r157c1_2.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r157c1_3.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r157c1_4.svg">
        ectoine
      </option>
      
    </select>
    <button id="knownclusterblast-r157c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r157c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r157c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-105"></div> <span class="help-tooltip" id="cb-sub-help-105">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r157c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-106"></div> <span class="help-tooltip" id="MIBiG-help-106">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r157c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r157c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r157c1"></div>
    <div id="comparison-MIBiG-r157c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">ectoine</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.75 (id:0.55, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
         
        
        <td>0.44</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.74 (id:0.54, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
         
        
        <td>0.43</td>
        <td>Other,  , (, E, c, t, o, i, n, e, )</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.49, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylobacter marinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylophaga thalassica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852.1: 415-2854">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Sporosarcina pasteurii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855.1: 245-3455">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium kenyense</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857.1: 300-4871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylophaga alcalica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860.1: 710-5134">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.49, order:0.51, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylarcula marina</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.50, order:0.48, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum 20Z</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r157c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.75 (id:0.55, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
      <td>0.75</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.74 (id:0.54, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
      <td>0.74</td>
      <td>Other,  , (, E, c, t, o, i, n, e, )</td>
      <td>ectoine</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.49, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylobacter marinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylophaga thalassica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852.1: 415-2854">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Sporosarcina pasteurii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855.1: 245-3455">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium kenyense</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857.1: 300-4871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylophaga alcalica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860.1: 710-5134">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.49, order:0.51, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylarcula marina</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.50, order:0.48, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum 20Z</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r157c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-107"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-107">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r157c1-pfam-details-svg" data-filename="CC223A_contig_84_r157c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r157c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r158c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_55 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-167"></div> <span class="help-tooltip" id="region_svg-help-167">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_55.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r158c1-svg" data-filename="CC223A_contig_55_r158c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 35,211 nt. (total: 35,211 nt)
        
        <a class="cluster-rules-header" id="r158c1-rules-header" href="#r158c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r158c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r158c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-168"></div> <span class="help-tooltip" id="focus-panel-help-168">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r158c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r158c1-nrps_pks" data-name="r158c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r158c1-clusterblast" data-name="r158c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r158c1-knownclusterblast" data-name="r158c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r158c1-subclusterblast" data-name="r158c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r158c1-MIBiG-cluster-compare" data-name="r158c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r158c1-pfam-details" data-name="r158c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r158c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-108"></div> <span class="help-tooltip" id="nrps-domain-help-108">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r158c1-details-svg" data-filename="CC223A_contig_55_r158c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r158c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r158c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-109"></div> <span class="help-tooltip" id="cb-general-help-109">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r158c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r158c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r158c1_1.svg">
        NZ_ML703337 (102841-145742): Nocardiopsis quinghaiensis strain YIM 28A4 Scaff... (37% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r158c1_2.svg">
        NZ_VWVS01000013 (167446-212940): Nocardiopsis sp. H13 contig013, whole genome... (18% of genes show similarity), T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r158c1_3.svg">
        NZ_VWVW01000023 (53849-96744): Nocardiopsis sp. FR6 contig023, whole genome s... (23% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r158c1_4.svg">
        NZ_PYGA01000024 (3152-46056): Murinocardiopsis flavida strain DSM 45312 Ga018... (15% of genes show similarity), T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r158c1_5.svg">
        NZ_KB906860 (104638-156127): Nocardiopsis sp. CNS-639 strain CNS639 G011DRAFT... (6% of genes show similarity), T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r158c1_6.svg">
        NZ_KB913032 (7649635-7752386): Amycolatopsis alba DSM 44262 scaffold1, whole ... (3% of genes show similarity), NRPS,T1PKS,T3PKS,betalactone,terpene
      </option>
      
      <option value = "svg/clusterblast_r158c1_7.svg">
        NZ_CP029788 (2973465-3014586): Streptomyces actuosus strain ATCC 25421 chromo... (8% of genes show similarity), T1PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r158c1_8.svg">
        NZ_SNYN01000002 (236393-279327): Actinorugispora endophytica strain DSM 46770... (7% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r158c1_9.svg">
        NZ_JAALLH010000001 (4445863-4628722): Streptomyces malaysiensis strain DSM 14... (23% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r158c1_10.svg">
        NZ_KB894405 (270527-383383): Actinokineospora enzanensis DSM 44649 C503DRAFT ... (12% of genes show similarity), T1PKS,butyrolactone
      </option>
      
    </select>
    <button id= "clusterblast-r158c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r158c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r158c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-110"></div> <span class="help-tooltip" id="cb-known-help-110">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r158c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-111"></div> <span class="help-tooltip" id="cb-sub-help-111">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r158c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-112"></div> <span class="help-tooltip" id="MIBiG-help-112">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r158c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r158c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r158c1"></div>
    <div id="comparison-MIBiG-r158c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Polyketide</td>
        
        <td>strobilurin</td>
        
        
        <td>Strobilurus tenacellus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>esperamicin</td>
        
        
        <td>Actinomadura verrucosospora</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>6-methylsalicyclic acid</td>
        
        
        <td>Aspergillus terreus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>asperlactone</td>
        
        
        <td>Aspergillus ochraceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>(-)-Mellein</td>
        
        
        <td>Parastagonospora nodorum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275.1: 1063-6614">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>6-methylsalicyclic acid</td>
        
        
        <td>Glarea lozoyensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998.1: 1175677-1189808">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400.1: 371433-386839">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>citreoviridin</td>
        
        
        <td>Aspergillus terreus NIH2624</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219.1: 1404646-1428146">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>shanorellin</td>
        
        
        <td>Chaetomium globosum CBS 148.51</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000698.1: 2269-33779">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000698/index.html#r1c1" target="_new">BGC0000698.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>hygromycin A</td>
        
        
        <td>Streptomyces hygroscopicus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r158c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>strobilurin</td>
      <td>Strobilurus tenacellus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>esperamicin</td>
      <td>Actinomadura verrucosospora</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Aspergillus terreus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Polyketide</td>
      <td>asperlactone</td>
      <td>Aspergillus ochraceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Polyketide</td>
      <td>(-)-Mellein</td>
      <td>Parastagonospora nodorum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275.1: 1063-6614">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Glarea lozoyensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998.1: 1175677-1189808">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Polyketide</td>
      <td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400.1: 371433-386839">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Polyketide</td>
      <td>citreoviridin</td>
      <td>Aspergillus terreus NIH2624</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219.1: 1404646-1428146">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Polyketide</td>
      <td>shanorellin</td>
      <td>Chaetomium globosum CBS 148.51</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000698.1: 2269-33779">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000698/index.html#r1c1" target="_new">BGC0000698.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>hygromycin A</td>
      <td>Streptomyces hygroscopicus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r158c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-113"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-113">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r158c1-pfam-details-svg" data-filename="CC223A_contig_55_r158c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r158c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r167c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_48 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-169"></div> <span class="help-tooltip" id="region_svg-help-169">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_48.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r167c1-svg" data-filename="CC223A_contig_48_r167c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 27,556 nt. (total: 27,556 nt)
        
        <a class="cluster-rules-header" id="r167c1-rules-header" href="#r167c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r167c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r167c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-170"></div> <span class="help-tooltip" id="focus-panel-help-170">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r167c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r167c1-nrps_pks" data-name="r167c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r167c1-clusterblast" data-name="r167c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r167c1-knownclusterblast" data-name="r167c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r167c1-subclusterblast" data-name="r167c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r167c1-MIBiG-cluster-compare" data-name="r167c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r167c1-pfam-details" data-name="r167c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r167c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-114"></div> <span class="help-tooltip" id="nrps-domain-help-114">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r167c1-details-svg" data-filename="CC223A_contig_48_r167c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r167c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r167c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-115"></div> <span class="help-tooltip" id="cb-general-help-115">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r167c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r167c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r167c1_1.svg">
        NZ_FQZK01000013 (164633-210378): Nocardiopsis flavescens strain CGMCC 4.5723,... (55% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r167c1_2.svg">
        NZ_MQUQ01000004 (336949-412620): Amycolatopsis coloradensis strain DSM 44225 ... (16% of genes show similarity), NRPS-like,T1PKS,amglyccycl,other
      </option>
      
      <option value = "svg/clusterblast_r167c1_3.svg">
        NZ_VWVW01000006 (2217-67259): Nocardiopsis sp. FR6 contig006, whole genome sh... (13% of genes show similarity), T1PKS,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r167c1_4.svg">
        NZ_VWVV01000004 (157978-223020): Nocardiopsis sp. FR4 contig004, whole genome... (13% of genes show similarity), T1PKS,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r167c1_5.svg">
        NZ_KZ559466 (688432-733955): Nonomuraea indica strain DRQ-2 Scaffold1, whole ... (25% of genes show similarity), T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r167c1_6.svg">
        NZ_VLKB01000010 (168892-252219): Calothrix desertica PCC 7102 Cal7102DRAFT CD... (13% of genes show similarity), NRPS,T1PKS,phosphonate
      </option>
      
      <option value = "svg/clusterblast_r167c1_7.svg">
        NZ_FNCC01000008 (352592-465366): Lechevalieria fradiae strain CGMCC 4.3506, w... (10% of genes show similarity), NRPS,T1PKS,ladderane,terpene
      </option>
      
      <option value = "svg/clusterblast_r167c1_8.svg">
        NZ_FOYL01000001 (114410-159356): Lentzea waywayandensis strain DSM 44232, who... (18% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r167c1_9.svg">
        NZ_FWYC01000003 (252564-297964): Lentzea albidocapillata strain DSM 44073, wh... (18% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r167c1_10.svg">
        NZ_QQAU01000001 (281123-325511): Lentzea flaviverrucosa strain DSM 44664 Ga02... (18% of genes show similarity), T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r167c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r167c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r167c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-116"></div> <span class="help-tooltip" id="cb-known-help-116">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r167c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r167c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_1.svg">
        uncialamycin
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_2.svg">
        uncialamycin
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_3.svg">
        amycolamycin A / amycolamycin B
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_4.svg">
        maduropeptin
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_5.svg">
        kedarcidin
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_6.svg">
        sporolide A / sporolide B
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_7.svg">
        neocarzinostatin
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_8.svg">
        calicheamicin
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_9.svg">
        C-1027
      </option>
      
      <option value = "svg/knownclusterblast_r167c1_10.svg">
        C-1027
      </option>
      
    </select>
    <button id="knownclusterblast-r167c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r167c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r167c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-117"></div> <span class="help-tooltip" id="cb-sub-help-117">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r167c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r167c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r167c1_1.svg">
        AY117439_c3: neocarzinostatin enediyne (25% of genes show similarity), enediyne
      </option>
      
      <option value = "svg/subclusterblast_r167c1_2.svg">
        AY271660_c2: maduropeptin enediyne (25% of genes show similarity), enediyne
      </option>
      
      <option value = "svg/subclusterblast_r167c1_3.svg">
        AF497482_c2: calicheamicin enediyne (23% of genes show similarity), enediyne
      </option>
      
      <option value = "svg/subclusterblast_r167c1_4.svg">
        AY048670_c3: C-1027 enediyne (25% of genes show similarity), enediyne
      </option>
      
    </select>
    <button id="subclusterblast-r167c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r167c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r167c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-118"></div> <span class="help-tooltip" id="MIBiG-help-118">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r167c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r167c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r167c1"></div>
    <div id="comparison-MIBiG-r167c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.37</td>
        <td>Polyketide</td>
        
        <td>esperamicin</td>
        
        
        <td>Actinomadura verrucosospora</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Terpene, Polyketide</td>
        
        <td>pyripyropene A</td>
        
        
        <td>unidentified unclassified sequences.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>RiPP</td>
        
        <td>venezuelin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001579.1: 85465-94303">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001579/index.html#r1c1" target="_new">BGC0001579.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>RiPP</td>
        
        <td>duramycin</td>
        
        
        <td>Streptomyces cinnamoneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000503.1: 0-17083">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000503/index.html#r1c1" target="_new">BGC0000503.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>RiPP</td>
        
        <td>cinnamycin</td>
        
        
        <td>Streptomyces cinnamoneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001779.1: 7655772-7662064">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001779/index.html#r1c1" target="_new">BGC0001779.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>siamycin</td>
        
        
        <td>Streptomyces nodosus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001891.1: 3812149-3823545">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001891/index.html#r1c1" target="_new">BGC0001891.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>closthioamide</td>
        
        
        <td>Ruminiclostridium cellulolyticum H10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000616.1: 3044505-3048678">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000616/index.html#r1c1" target="_new">BGC0000616.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>amylocyclicin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001210.1: 4366120-4375355">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001210/index.html#r1c1" target="_new">BGC0001210.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>pseudomycoicidin</td>
        
        
        <td>Bacillus pseudomycoides DSM 12442</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000557.1: 243-11649">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000557/index.html#r1c1" target="_new">BGC0000557.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>streptococcin A-FF22</td>
        
        
        <td>Streptococcus pyogenes</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r167c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>Polyketide</td>
      <td>esperamicin</td>
      <td>Actinomadura verrucosospora</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Terpene, Polyketide</td>
      <td>pyripyropene A</td>
      <td>unidentified unclassified sequences.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>RiPP</td>
      <td>venezuelin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001579.1: 85465-94303">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001579/index.html#r1c1" target="_new">BGC0001579.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>RiPP</td>
      <td>duramycin</td>
      <td>Streptomyces cinnamoneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000503.1: 0-17083">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000503/index.html#r1c1" target="_new">BGC0000503.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>RiPP</td>
      <td>cinnamycin</td>
      <td>Streptomyces cinnamoneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001779.1: 7655772-7662064">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001779/index.html#r1c1" target="_new">BGC0001779.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>siamycin</td>
      <td>Streptomyces nodosus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001891.1: 3812149-3823545">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001891/index.html#r1c1" target="_new">BGC0001891.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>closthioamide</td>
      <td>Ruminiclostridium cellulolyticum H10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000616.1: 3044505-3048678">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000616/index.html#r1c1" target="_new">BGC0000616.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>amylocyclicin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001210.1: 4366120-4375355">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001210/index.html#r1c1" target="_new">BGC0001210.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>pseudomycoicidin</td>
      <td>Bacillus pseudomycoides DSM 12442</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000557.1: 243-11649">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000557/index.html#r1c1" target="_new">BGC0000557.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>RiPP</td>
      <td>streptococcin A-FF22</td>
      <td>Streptococcus pyogenes</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r167c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-119"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-119">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r167c1-pfam-details-svg" data-filename="CC223A_contig_48_r167c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r167c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r173c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_56 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-171"></div> <span class="help-tooltip" id="region_svg-help-171">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_56.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r173c1-svg" data-filename="CC223A_contig_56_r173c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 24,060 nt. (total: 24,060 nt)
        
        <a class="cluster-rules-header" id="r173c1-rules-header" href="#r173c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r173c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r173c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-172"></div> <span class="help-tooltip" id="focus-panel-help-172">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r173c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r173c1-nrps_pks" data-name="r173c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r173c1-clusterblast" data-name="r173c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r173c1-knownclusterblast" data-name="r173c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r173c1-subclusterblast" data-name="r173c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r173c1-MIBiG-cluster-compare" data-name="r173c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r173c1-pfam-details" data-name="r173c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r173c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-120"></div> <span class="help-tooltip" id="nrps-domain-help-120">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r173c1-details-svg" data-filename="CC223A_contig_56_r173c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r173c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r173c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-123"></div> <span class="help-tooltip" id="cb-general-help-123">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r173c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r173c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r173c1_1.svg">
        NZ_FQZK01000011 (77970-131650): Nocardiopsis flavescens strain CGMCC 4.5723, ... (48% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r173c1_2.svg">
        NZ_VWVV01000010 (14266-79323): Nocardiopsis sp. FR4 contig010, whole genome s... (30% of genes show similarity), LAP,NRPS,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r173c1_3.svg">
        NZ_JACHJO010000001 (353703-413717): Nocardiopsis algeriensis strain CECT 8712... (40% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r173c1_4.svg">
        NC_018524 (1661697-1743639): Nocardiopsis alba ATCC BAA-2165, complete sequence (21% of genes show similarity), LAP,NRPS,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r173c1_5.svg">
        NC_015564 (4125762-4178994): Hoyosella subflava DQS3-9A1, complete sequence (20% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r173c1_6.svg">
        NZ_VSSO01000005 (71184-133111): Mycobacterium sp. LNNU 331112 NODE 5 length 2... (18% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r173c1_7.svg">
        NZ_CP015219 (3208694-3279611): Rhodococcus sp. PBTS 1 chromosome, complete ge... (12% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r173c1_8.svg">
        NZ_LMRN01000025 (563001-619111): Rhodococcus sp. Leaf225 contig 8, whole geno... (17% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r173c1_9.svg">
        NZ_LMRP01000001 (1380010-1436027): Rhodococcus sp. Leaf247 contig 1, whole ge... (17% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r173c1_10.svg">
        NZ_RCDD01000002 (352277-441120): Actinokineospora cianjurensis strain DSM 456... (8% of genes show similarity), NRPS,T3PKS
      </option>
      
    </select>
    <button id= "clusterblast-r173c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r173c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r173c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-124"></div> <span class="help-tooltip" id="cb-known-help-124">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r173c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r173c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r173c1_1.svg">
        amycolamycin A / amycolamycin B
      </option>
      
      <option value = "svg/knownclusterblast_r173c1_2.svg">
        ecumicin
      </option>
      
      <option value = "svg/knownclusterblast_r173c1_3.svg">
        griseoviridin / fijimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r173c1_4.svg">
        teleocidin B1
      </option>
      
      <option value = "svg/knownclusterblast_r173c1_5.svg">
        scabichelin
      </option>
      
      <option value = "svg/knownclusterblast_r173c1_6.svg">
        WAP-8294A2
      </option>
      
      <option value = "svg/knownclusterblast_r173c1_7.svg">
        qinichelins
      </option>
      
      <option value = "svg/knownclusterblast_r173c1_8.svg">
        jawsamycin
      </option>
      
      <option value = "svg/knownclusterblast_r173c1_9.svg">
        sporolide A / sporolide B
      </option>
      
      <option value = "svg/knownclusterblast_r173c1_10.svg">
        friulimicin A / friulimicin B / friulimicin C / friulimicin D
      </option>
      
    </select>
    <button id="knownclusterblast-r173c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r173c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r173c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-125"></div> <span class="help-tooltip" id="cb-sub-help-125">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r173c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-126"></div> <span class="help-tooltip" id="MIBiG-help-126">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r173c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r173c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r173c1"></div>
    <div id="comparison-MIBiG-r173c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>hexose-palythine-serine, hexose-shinorine</td>
        
        
        <td>Heteroscytonema crispum UCFS10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>NRP, Alkaloid</td>
        
        <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.59, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>fellutamide B</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131.1: 144016-173487">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>ambactin</td>
        
        
        <td>Xenorhabdus miraniensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r173c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>hexose-palythine-serine, hexose-shinorine</td>
      <td>Heteroscytonema crispum UCFS10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP, Alkaloid</td>
      <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.59, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>fellutamide B</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131.1: 144016-173487">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>ambactin</td>
      <td>Xenorhabdus miraniensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r173c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-127"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-127">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r173c1-pfam-details-svg" data-filename="CC223A_contig_56_r173c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r173c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r173c1-nrps_pks_products" data-name="r173c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r173c1-nrps_pks" data-name="r173c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r173c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-121"></div> <span class="help-tooltip" id="nrps-structures-help-121">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 24060:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r173c1-1" class="smiles-canvas" data-smiles="NC(C(C)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(val)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r173c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(C(C)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r173c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-122"></div> <span class="help-tooltip" id="nrps-monomers-help-122">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds1_4060</span></strong>: val
      <div class="collapser collapser-target-cds1_4060 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (496..903)</strong>: val
          <div class="collapser collapser-target-cds1_4060 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds1_4060-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>val</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAYyWsGvwK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r175c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC223A_contig_33 - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-173"></div> <span class="help-tooltip" id="region_svg-help-173">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC223A_contig_33.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r175c1-svg" data-filename="CC223A_contig_33_r175c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 8,111 - 18,416 nt. (total: 10,306 nt)
        
        <a class="cluster-rules-header" id="r175c1-rules-header" href="#r175c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r175c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193)</div>
      
      <div class="region-svg-container">
       <div id='r175c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-174"></div> <span class="help-tooltip" id="focus-panel-help-174">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r175c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r175c1-clusterblast" data-name="r175c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r175c1-knownclusterblast" data-name="r175c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r175c1-subclusterblast" data-name="r175c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r175c1-MIBiG-cluster-compare" data-name="r175c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r175c1-pfam-details" data-name="r175c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r175c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-128"></div> <span class="help-tooltip" id="cb-general-help-128">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r175c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r175c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r175c1_1.svg">
        NZ_FQZK01000003 (302649-312955): Nocardiopsis flavescens strain CGMCC 4.5723,... (83% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r175c1_2.svg">
        NZ_VWVW01000031 (10028-83588): Nocardiopsis sp. FR6 contig031, whole genome s... (3% of genes show similarity), NRPS,bacteriocin,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r175c1_3.svg">
        NZ_QGTF01000002 (435061-445361): Nocardiopsis sp. L17-MgMaSL7 Ga0060225 102, ... (22% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r175c1_4.svg">
        NZ_VWVU01000003 (56778-67075): Nocardiopsis quinghaiensis strain YIM 28A4 con... (20% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r175c1_5.svg">
        NZ_VWVS01000004 (47217-70975): Nocardiopsis sp. H13 contig004, whole genome s... (10% of genes show similarity), bacteriocin,terpene
      </option>
      
      <option value = "svg/clusterblast_r175c1_6.svg">
        NZ_LR134501 (1477796-1500417): Nocardiopsis dassonvillei strain NCTC10488 chr... (12% of genes show similarity), bacteriocin,terpene
      </option>
      
      <option value = "svg/clusterblast_r175c1_7.svg">
        NZ_CP017965 (3922709-3943105): Nocardiopsis dassonvillei strain NOCA502F, com... (12% of genes show similarity), bacteriocin,terpene
      </option>
      
      <option value = "svg/clusterblast_r175c1_8.svg">
        NZ_CP022434 (3985595-4007887): Nocardiopsis dassonvillei strain HZNU N 1 chro... (12% of genes show similarity), bacteriocin,terpene
      </option>
      
      <option value = "svg/clusterblast_r175c1_9.svg">
        NZ_KB906858 (362675-386810): Nocardiopsis sp. CNS-639 strain CNS639 G011DRAFT... (12% of genes show similarity), bacteriocin,terpene
      </option>
      
      <option value = "svg/clusterblast_r175c1_10.svg">
        NZ_VWVW01000027 (35542-56883): Nocardiopsis sp. FR6 contig027, whole genome s... (11% of genes show similarity), bacteriocin,terpene
      </option>
      
    </select>
    <button id= "clusterblast-r175c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r175c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r175c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-129"></div> <span class="help-tooltip" id="cb-known-help-129">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r175c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-130"></div> <span class="help-tooltip" id="cb-sub-help-130">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r175c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-131"></div> <span class="help-tooltip" id="MIBiG-help-131">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r175c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r175c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r175c1"></div>
    <div id="comparison-MIBiG-r175c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">RiPP-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000504.1: 606-7471">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000504/index.html#r1c1" target="_new">BGC0000504.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>cytolysin ClyLl, cytolysin ClyLs</td>
        
        
        <td>Plasmid pAD1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500.1: 0-8863">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>RiPP</td>
        
        <td>carnolysin A1, carnolysin A2</td>
        
        
        <td>Carnobacterium maltaromaticum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001120.1: 2543311-2579913">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001120/index.html#r1c1" target="_new">BGC0001120.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>NRP, Polyketide</td>
        
        <td>burkholderic acid</td>
        
        
        <td>Burkholderia thailandensis E264</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000768.1: 0-30166">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000768/index.html#r1c1" target="_new">BGC0000768.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>gellan polysaccharide</td>
        
        
        <td>Sphingomonas elodea ATCC 31461</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000548.1: 588-10610">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000548/index.html#r1c1" target="_new">BGC0000548.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>RiPP</td>
        
        <td>salivaricin A</td>
        
        
        <td>Streptococcus salivarius</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000797.1: 324-28703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000797/index.html#r1c1" target="_new">BGC0000797.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>sphingan polysaccharide</td>
        
        
        <td>Sphingomonas sp. S88</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001783.1: 55-63934">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001783/index.html#r1c1" target="_new">BGC0001783.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>streptonigrin</td>
        
        
        <td>Streptomyces flocculus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001138.1: 0-39199">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001138/index.html#r1c1" target="_new">BGC0001138.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other,  , (, N, u, c, l, e, o, s, i, d, e, )</td>
        
        <td>a201a</td>
        
        
        <td>Marinactinospora thermotolerans</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r175c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000504.1: 606-7471">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000504/index.html#r1c1" target="_new">BGC0000504.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>RiPP</td>
      <td>cytolysin ClyLl, cytolysin ClyLs</td>
      <td>Plasmid pAD1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500.1: 0-8863">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>RiPP</td>
      <td>carnolysin A1, carnolysin A2</td>
      <td>Carnobacterium maltaromaticum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001120.1: 2543311-2579913">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001120/index.html#r1c1" target="_new">BGC0001120.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>NRP, Polyketide</td>
      <td>burkholderic acid</td>
      <td>Burkholderia thailandensis E264</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000768.1: 0-30166">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000768/index.html#r1c1" target="_new">BGC0000768.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>gellan polysaccharide</td>
      <td>Sphingomonas elodea ATCC 31461</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000548.1: 588-10610">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000548/index.html#r1c1" target="_new">BGC0000548.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>RiPP</td>
      <td>salivaricin A</td>
      <td>Streptococcus salivarius</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000797.1: 324-28703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000797/index.html#r1c1" target="_new">BGC0000797.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>sphingan polysaccharide</td>
      <td>Sphingomonas sp. S88</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001783.1: 55-63934">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001783/index.html#r1c1" target="_new">BGC0001783.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>streptonigrin</td>
      <td>Streptomyces flocculus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001138.1: 0-39199">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001138/index.html#r1c1" target="_new">BGC0001138.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other,  , (, N, u, c, l, e, o, s, i, d, e, )</td>
      <td>a201a</td>
      <td>Marinactinospora thermotolerans</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r175c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-132"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-132">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r175c1-pfam-details-svg" data-filename="CC223A_contig_33_r175c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r175c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>


  <footer class="footer">
    <div class="container">
      <div>
        <img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
      </div>
      <div class="cite-me">
        If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
      </div>
      <div>
        <img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
      </div>
    </div>
  </footer>

  <script src="js/jquery.js"></script>
  <script src="js/antismash.js"></script>
  <script src="regions.js"></script>
  <script>
    $(document).ready(function() {
        viewer["start"](all_regions, details_data, resultsData, recordData);
    })
  </script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink">
 <defs>
  <filter id="inset-shadow">
   <feOffset dx="-2" dy="-2"></feOffset>
   <feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
   <feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
   <feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
   <feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
   <feComponentTransfer in="shadow" result="shadow">
    <feFuncA type="linear" slope=".95"></feFuncA>
   </feComponentTransfer>
   <feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
  </filter>
 </defs>
</svg>
</body>
</html>