
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>56099.17 - 32 region(s) - antiSMASH results</title>
  <style>img {width: 20px; height: 20px;}</style>
  <link rel="stylesheet" type="text/css" href="css/bacteria.css">
  <meta property="og:title" content="56099.17 - 19 record(s) - 32 region(s)" />
  <meta property="og:description" content="">
  <meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
  <meta property="og:image:alt" content="antiSMASH logo" />
  <meta property="og:image:width" content="600" />
  <meta property="og:image:height" content="600" />
  <meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-aaceac8b-9ba6-4717-b4aa-0f79bc335d92/index.html" />
  <meta property="og:type" content="website" />
  <meta name="twitter:card" content="summary" />
  <meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
  <div id="header">
      <nav>
    <div class="main-link">
     <div>
        <a href="https://antismash.secondarymetabolites.org/">
          <img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
        </a>
       </div>
       <div>
        <a href="https://antismash.secondarymetabolites.org/">
          antiSMASH version 6.0.1
        </a>
      </div>
    </div>
    <div class="custom-description"></div>
    <div class="ancillary-links">
      <div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
        <ul class="dropdown-options">
        
        <li><a href="56099.17.zip">Download all results</a></li>
        
          <li><a href="56099.17.gbk">Download GenBank summary file</a></li>
        
          <li><a href="bacteria-aaceac8b-9ba6-4717-b4aa-0f79bc335d92.log">Download log file</a></li>
        
        </ul>
      </div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
      <div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
    </div>
  </nav>
    <div id="buttons">
      <span id="region-type">Select genomic region:</span>
      <div id="regionbuttons">
        <div class="arrow-left" id="prev-region"></div>
        <div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
        <div style="display: flex; flex-wrap: wrap">
        
          
            
            
            <div class="regbutton siderophore r1c1">
              <a href="#r1c1">1.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NAPAA r3c1">
              <a href="#r3c1">3.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton RiPP-like r4c1">
              <a href="#r4c1">4.1</a>
            </div>
          
            
            
                
            
            <div class="regbutton NRPS,PKS-like,T1PKS,indole hybrid r4c2">
              <a href="#r4c2">4.2</a>
            </div>
          
            
            
            <div class="regbutton lanthipeptide-class-ii r4c3">
              <a href="#r4c3">4.3</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton PKS-like,RRE-containing,T2PKS hybrid r5c1">
              <a href="#r5c1">5.1</a>
            </div>
          
            
            
            <div class="regbutton lanthipeptide-class-iii r5c2">
              <a href="#r5c2">5.2</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r6c1">
              <a href="#r6c1">6.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,T1PKS hybrid r10c1">
              <a href="#r10c1">10.1</a>
            </div>
          
            
            
            <div class="regbutton terpene r10c2">
              <a href="#r10c2">10.2</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r16c1">
              <a href="#r16c1">16.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T3PKS r21c1">
              <a href="#r21c1">21.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,NRPS-like hybrid r22c1">
              <a href="#r22c1">22.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r25c1">
              <a href="#r25c1">25.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r26c1">
              <a href="#r26c1">26.1</a>
            </div>
          
            
            
            <div class="regbutton NRPS-like r26c2">
              <a href="#r26c2">26.2</a>
            </div>
          
            
            
                
            
            <div class="regbutton CDPS,NRPS,T1PKS hybrid r26c3">
              <a href="#r26c3">26.3</a>
            </div>
          
            
            
                
            
            <div class="regbutton NRPS-like,PKS-like,T1PKS,arylpolyene hybrid r26c4">
              <a href="#r26c4">26.4</a>
            </div>
          
            
            
            <div class="regbutton terpene r26c5">
              <a href="#r26c5">26.5</a>
            </div>
          
            
            
                
            
            <div class="regbutton NRPS,T1PKS,amglyccycl,hglE-KS hybrid r26c6">
              <a href="#r26c6">26.6</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton LAP,thiopeptide hybrid r30c1">
              <a href="#r30c1">30.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r31c1">
              <a href="#r31c1">31.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton T1PKS,ladderane hybrid r32c1">
              <a href="#r32c1">32.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r34c1">
              <a href="#r34c1">34.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS-like,PKS-like,T2PKS hybrid r36c1">
              <a href="#r36c1">36.1</a>
            </div>
          
            
            
            <div class="regbutton T1PKS r36c2">
              <a href="#r36c2">36.2</a>
            </div>
          
        
          
            
            
            <div class="regbutton redox-cofactor r38c1">
              <a href="#r38c1">38.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton betalactone r48c1">
              <a href="#r48c1">48.1</a>
            </div>
          
            
            
            <div class="regbutton terpene r48c2">
              <a href="#r48c2">48.2</a>
            </div>
          
            
            
            <div class="regbutton NRPS r48c3">
              <a href="#r48c3">48.3</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS-like r54c1">
              <a href="#r54c1">54.1</a>
            </div>
          
            
            
            <div class="regbutton terpene r54c2">
              <a href="#r54c2">54.2</a>
            </div>
          
        
        </div>
        <div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
      </div>
    </div>
  </div>

  <!-- overview page -->
  
  <div class="page" id="overview">
   
   
     <h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>
   
   <div class="overview-layout">
    <div id="record-tables">
     <div id="single-record-tables">
     <div class="record-overview-header">
        <strong>c00001_CC55NOD..</strong> (original name was: CC55NODE_8_length_285679_cov_96.543015) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-1" data-filename="c00001_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-1" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r1c1">
     
     
     <td class="regbutton siderophore r1c1">
      <a href="#r1c1">Region&nbsp1.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">70,618</td>
     <td class="digits table-split-left">83,429</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00003_CC55NOD..</strong> (original name was: CC55NODE_36_length_75259_cov_109.933853) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-3" data-filename="c00003_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-3" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r3c1">
     
     
     <td class="regbutton NAPAA r3c1">
      <a href="#r3c1">Region&nbsp3.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
     </td>
     <td class="digits">2,611</td>
     <td class="digits table-split-left">36,348</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00004_CC55NOD..</strong> (original name was: CC55NODE_19_length_679679_cov_92.099632) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-4" data-filename="c00004_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-4" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r4c1">
     
     
     <td class="regbutton RiPP-like r4c1">
      <a href="#r4c1">Region&nbsp4.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">27,080</td>
     <td class="digits table-split-left">37,898</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000341/1" target="_blank">enduracidin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r4c2">
     
     
         
     
     <td class="regbutton NRPS,PKS-like,T1PKS,indole hybrid r4c2">
      <a href="#r4c2">Region&nbsp4.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#pks-like" target="_blank">PKS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">146,422</td>
     <td class="digits table-split-left">250,749</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000695/1" target="_blank">fortimicin</a></td>
      <td>Saccharide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r4c3">
     
     
     <td class="regbutton lanthipeptide-class-ii r4c3">
      <a href="#r4c3">Region&nbsp4.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-ii" target="_blank">lanthipeptide-class-ii</a>
     </td>
     <td class="digits">351,062</td>
     <td class="digits table-split-left">373,707</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00005_CC55NOD..</strong> (original name was: CC55NODE_53_length_414845_cov_108.017349...) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-5" data-filename="c00005_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-5" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r5c1">
     
     
         
     
     <td class="regbutton PKS-like,RRE-containing,T2PKS hybrid r5c1">
      <a href="#r5c1">Region&nbsp5.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#pks-like" target="_blank">PKS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#rre-containing" target="_blank">RRE-containing</a>
     </td>
     <td class="digits">198,168</td>
     <td class="digits table-split-left">270,647</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001979/1" target="_blank">LL-D49194Î±1 (LLD)</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 32%, #ffffff00 32%)">32%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r5c2">
     
     
     <td class="regbutton lanthipeptide-class-iii r5c2">
      <a href="#r5c2">Region&nbsp5.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
     </td>
     <td class="digits">284,572</td>
     <td class="digits table-split-left">307,136</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000513/1" target="_blank">Ery-9 / Ery-6 / Ery-8 / Ery-7 / Ery-5 / Ery-4 / Ery-3</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00006_CC55NOD..</strong> (original name was: CC55NODE_30_length_110141_cov_74.601418) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-6" data-filename="c00006_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-6" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r6c1">
     
     
     <td class="regbutton T1PKS r6c1">
      <a href="#r6c1">Region&nbsp6.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">45,512</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001008/1" target="_blank">maduropeptin</a></td>
      <td>Polyketide:Iterative type I + Polyketide:Enediyne type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00010_CC55NOD..</strong> (original name was: CC55NODE_13_length_315557_cov_78.710854) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-10" data-filename="c00010_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-10" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r10c1">
     
     
         
     
     <td class="regbutton NRPS,T1PKS hybrid r10c1">
      <a href="#r10c1">Region&nbsp10.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">128,757</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001537/1" target="_blank">butyrolactol A</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 66%, #ffffff00 66%)">66%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r10c2">
     
     
     <td class="regbutton terpene r10c2">
      <a href="#r10c2">Region&nbsp10.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">255,867</td>
     <td class="digits table-split-left">276,793</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001456/1" target="_blank">isorenieratene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00016_CC55NOD..</strong> (original name was: CC55NODE_23_length_190473_cov_91.729958) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-16" data-filename="c00016_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-16" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r16c1">
     
     
     <td class="regbutton terpene r16c1">
      <a href="#r16c1">Region&nbsp16.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,493</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000269/1" target="_blank">SF2575</a></td>
      <td>Polyketide:Type II + Saccharide:Hybrid/tailoring</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00021_CC55NOD..</strong> (original name was: CC55NODE_18_length_134405_cov_79.384209) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-21" data-filename="c00021_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-21" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r21c1">
     
     
     <td class="regbutton T3PKS r21c1">
      <a href="#r21c1">Region&nbsp21.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">69,039</td>
     <td class="digits table-split-left">110,076</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001077/1" target="_blank">alkyl-O-dihydrogeranyl-methoxyhydroquinones</a></td>
      <td>Terpene + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 57%, #ffffff00 57%)">57%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00022_CC55NOD..</strong> (original name was: CC55NODE_24_length_111335_cov_77.089287) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-22" data-filename="c00022_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-22" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r22c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like hybrid r22c1">
      <a href="#r22c1">Region&nbsp22.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">79,213</td>
     <td class="digits table-split-left">111,423</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001046/1" target="_blank">streptolydigin</a></td>
      <td>NRP + Polyketide:Modular type I + Saccharide:Hybrid/tailoring</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 10%, #ffffff00 10%)">10%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00025_CC55NOD..</strong> (original name was: CC55NODE_52_length_181317_cov_82.414185) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-25" data-filename="c00025_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-25" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r25c1">
     
     
     <td class="regbutton NRPS r25c1">
      <a href="#r25c1">Region&nbsp25.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">103,568</td>
     <td class="digits table-split-left">154,430</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000325/1" target="_blank">coelichelin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 72%, #ffffff00 72%)">72%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00026_CC55NOD..</strong> (original name was: CC55NODE_68_length_802849_cov_83.287560) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-26" data-filename="c00026_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-26" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r26c1">
     
     
     <td class="regbutton T1PKS r26c1">
      <a href="#r26c1">Region&nbsp26.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">84</td>
     <td class="digits table-split-left">50,680</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000125/1" target="_blank">pimaricin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 29%, #ffffff00 29%)">29%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r26c2">
     
     
     <td class="regbutton NRPS-like r26c2">
      <a href="#r26c2">Region&nbsp26.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">52,748</td>
     <td class="digits table-split-left">109,991</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000950/1" target="_blank">napsamycin A / napsamycin B / napsamycin C / napsamycin D / mureidomycin A / mureidomycin B</a></td>
      <td>NRP:Uridylpeptide + Other:Nucleoside</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r26c3">
     
     
         
     
     <td class="regbutton CDPS,NRPS,T1PKS hybrid r26c3">
      <a href="#r26c3">Region&nbsp26.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#cdps" target="_blank">CDPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">120,986</td>
     <td class="digits table-split-left">222,233</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000324/1" target="_blank">coelibactin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 36%, #ffffff00 36%)">36%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r26c4">
     
     
         
     
     <td class="regbutton NRPS-like,PKS-like,T1PKS,arylpolyene hybrid r26c4">
      <a href="#r26c4">Region&nbsp26.4</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#pks-like" target="_blank">PKS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#arylpolyene" target="_blank">arylpolyene</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">493,491</td>
     <td class="digits table-split-left">592,270</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001116/1" target="_blank">virginiamycin S1</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 22%, #ffffff00 22%)">22%</td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r26c5">
     
     
     <td class="regbutton terpene r26c5">
      <a href="#r26c5">Region&nbsp26.5</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">678,213</td>
     <td class="digits table-split-left">700,280</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000661/1" target="_blank">geosmin</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r26c6">
     
     
         
     
     <td class="regbutton NRPS,T1PKS,amglyccycl,hglE-KS hybrid r26c6">
      <a href="#r26c6">Region&nbsp26.6</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hgle-ks" target="_blank">hglE-KS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#amglyccycl" target="_blank">amglyccycl</a>
     </td>
     <td class="digits">702,309</td>
     <td class="digits table-split-left">770,886</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000153/1" target="_blank">stigmatellin</a></td>
      <td>NRP + Polyketide:Modular type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00030_CC55NOD..</strong> (original name was: CC55NODE_17_length_92810_cov_110.266121) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-30" data-filename="c00030_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-30" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r30c1">
     
     
         
     
     <td class="regbutton LAP,thiopeptide hybrid r30c1">
      <a href="#r30c1">Region&nbsp30.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lap" target="_blank">LAP</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thiopeptide" target="_blank">thiopeptide</a>
     </td>
     <td class="digits">29,780</td>
     <td class="digits table-split-left">57,586</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00031_CC55NOD..</strong> (original name was: CC55NODE_26_length_65334_cov_79.637802) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-31" data-filename="c00031_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-31" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r31c1">
     
     
     <td class="regbutton NRPS r31c1">
      <a href="#r31c1">Region&nbsp31.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">34,009</td>
     <td class="digits table-split-left">65,422</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00032_CC55NOD..</strong> (original name was: CC55NODE_55_length_75369_cov_73.364777) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-32" data-filename="c00032_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-32" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r32c1">
     
     
         
     
     <td class="regbutton T1PKS,ladderane hybrid r32c1">
      <a href="#r32c1">Region&nbsp32.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ladderane" target="_blank">ladderane</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">61,982</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001817/1" target="_blank">vancosamine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00034_CC55NOD..</strong> (original name was: CC55NODE_39_length_155803_cov_80.421997) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-34" data-filename="c00034_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-34" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r34c1">
     
     
     <td class="regbutton NRPS r34c1">
      <a href="#r34c1">Region&nbsp34.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">66,342</td>
     <td class="digits table-split-left">111,786</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001569/1" target="_blank">dechlorocuracomycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00036_CC55NOD..</strong> (original name was: CC55NODE_64_length_453949_cov_75.650940) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-36" data-filename="c00036_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-36" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r36c1">
     
     
         
     
     <td class="regbutton NRPS-like,PKS-like,T2PKS hybrid r36c1">
      <a href="#r36c1">Region&nbsp36.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#pks-like" target="_blank">PKS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">126,166</td>
     <td class="digits table-split-left">238,151</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000279/1" target="_blank">xantholipin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r36c2">
     
     
     <td class="regbutton T1PKS r36c2">
      <a href="#r36c2">Region&nbsp36.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">283,753</td>
     <td class="digits table-split-left">328,261</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001298/1" target="_blank">4-Z-annimycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 33%, #ffffff00 33%)">33%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00038_CC55NOD..</strong> (original name was: CC55NODE_22_length_967145_cov_115.144608...) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-38" data-filename="c00038_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-38" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r38c1">
     
     
     <td class="regbutton redox-cofactor r38c1">
      <a href="#r38c1">Region&nbsp38.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#redox-cofactor" target="_blank">redox-cofactor</a>
     </td>
     <td class="digits">140,124</td>
     <td class="digits table-split-left">162,166</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001100/1" target="_blank">lankacidin C</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 26%, #ffffff00 26%)">26%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00048_CC55NOD..</strong> (original name was: CC55NODE_62_length_199831_cov_82.632797) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-48" data-filename="c00048_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-48" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r48c1">
     
     
     <td class="regbutton betalactone r48c1">
      <a href="#r48c1">Region&nbsp48.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>
     </td>
     <td class="digits">38,923</td>
     <td class="digits table-split-left">69,839</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001156/1" target="_blank">platencin</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r48c2">
     
     
     <td class="regbutton terpene r48c2">
      <a href="#r48c2">Region&nbsp48.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">92,689</td>
     <td class="digits table-split-left">113,762</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000657/1" target="_blank">2-methylisoborneol</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r48c3">
     
     
     <td class="regbutton NRPS r48c3">
      <a href="#r48c3">Region&nbsp48.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">139,992</td>
     <td class="digits table-split-left">199,919</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001379/1" target="_blank">BD-12</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 7%, #ffffff00 7%)">7%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00054_CC55NOD..</strong> (original name was: CC55NODE_27_length_232082_cov_109.862045...) (Lentzea sp. cc55)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-54" data-filename="c00054_CC55NOD.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-54" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r54c1">
     
     
     <td class="regbutton NRPS-like r54c1">
      <a href="#r54c1">Region&nbsp54.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">35,700</td>
     <td class="digits table-split-left">79,563</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000727/1" target="_blank">indigoidine</a></td>
      <td>Saccharide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r54c2">
     
     
     <td class="regbutton terpene r54c2">
      <a href="#r54c2">Region&nbsp54.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">96,938</td>
     <td class="digits table-split-left">119,082</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001181/1" target="_blank">geosmin</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div>
     </div>
     
     <div id="compact-record-table" style="display: none;">
      <div class="record-table">
       <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 
 
  <tr class="linked-row odd" data-anchor="#r1c1">
     
     
     <td class="regbutton siderophore r1c1">
      <a href="#r1c1">Region&nbsp1.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">70,618</td>
     <td class="digits table-split-left">83,429</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r3c1">
     
     
     <td class="regbutton NAPAA r3c1">
      <a href="#r3c1">Region&nbsp3.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
     </td>
     <td class="digits">2,611</td>
     <td class="digits table-split-left">36,348</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r4c1">
     
     
     <td class="regbutton RiPP-like r4c1">
      <a href="#r4c1">Region&nbsp4.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">27,080</td>
     <td class="digits table-split-left">37,898</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000341/1" target="_blank">enduracidin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r4c2">
     
     
         
     
     <td class="regbutton NRPS,PKS-like,T1PKS,indole hybrid r4c2">
      <a href="#r4c2">Region&nbsp4.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#pks-like" target="_blank">PKS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">146,422</td>
     <td class="digits table-split-left">250,749</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000695/1" target="_blank">fortimicin</a></td>
      <td>Saccharide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r4c3">
     
     
     <td class="regbutton lanthipeptide-class-ii r4c3">
      <a href="#r4c3">Region&nbsp4.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-ii" target="_blank">lanthipeptide-class-ii</a>
     </td>
     <td class="digits">351,062</td>
     <td class="digits table-split-left">373,707</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r5c1">
     
     
         
     
     <td class="regbutton PKS-like,RRE-containing,T2PKS hybrid r5c1">
      <a href="#r5c1">Region&nbsp5.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#pks-like" target="_blank">PKS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#rre-containing" target="_blank">RRE-containing</a>
     </td>
     <td class="digits">198,168</td>
     <td class="digits table-split-left">270,647</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001979/1" target="_blank">LL-D49194Î±1 (LLD)</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 32%, #ffffff00 32%)">32%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r5c2">
     
     
     <td class="regbutton lanthipeptide-class-iii r5c2">
      <a href="#r5c2">Region&nbsp5.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
     </td>
     <td class="digits">284,572</td>
     <td class="digits table-split-left">307,136</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000513/1" target="_blank">Ery-9 / Ery-6 / Ery-8 / Ery-7 / Ery-5 / Ery-4 / Ery-3</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r6c1">
     
     
     <td class="regbutton T1PKS r6c1">
      <a href="#r6c1">Region&nbsp6.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">45,512</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001008/1" target="_blank">maduropeptin</a></td>
      <td>Polyketide:Iterative type I + Polyketide:Enediyne type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r10c1">
     
     
         
     
     <td class="regbutton NRPS,T1PKS hybrid r10c1">
      <a href="#r10c1">Region&nbsp10.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">128,757</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001537/1" target="_blank">butyrolactol A</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 66%, #ffffff00 66%)">66%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r10c2">
     
     
     <td class="regbutton terpene r10c2">
      <a href="#r10c2">Region&nbsp10.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">255,867</td>
     <td class="digits table-split-left">276,793</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001456/1" target="_blank">isorenieratene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r16c1">
     
     
     <td class="regbutton terpene r16c1">
      <a href="#r16c1">Region&nbsp16.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,493</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000269/1" target="_blank">SF2575</a></td>
      <td>Polyketide:Type II + Saccharide:Hybrid/tailoring</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r21c1">
     
     
     <td class="regbutton T3PKS r21c1">
      <a href="#r21c1">Region&nbsp21.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">69,039</td>
     <td class="digits table-split-left">110,076</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001077/1" target="_blank">alkyl-O-dihydrogeranyl-methoxyhydroquinones</a></td>
      <td>Terpene + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 57%, #ffffff00 57%)">57%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r22c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like hybrid r22c1">
      <a href="#r22c1">Region&nbsp22.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">79,213</td>
     <td class="digits table-split-left">111,423</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001046/1" target="_blank">streptolydigin</a></td>
      <td>NRP + Polyketide:Modular type I + Saccharide:Hybrid/tailoring</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 10%, #ffffff00 10%)">10%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r25c1">
     
     
     <td class="regbutton NRPS r25c1">
      <a href="#r25c1">Region&nbsp25.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">103,568</td>
     <td class="digits table-split-left">154,430</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000325/1" target="_blank">coelichelin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 72%, #ffffff00 72%)">72%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r26c1">
     
     
     <td class="regbutton T1PKS r26c1">
      <a href="#r26c1">Region&nbsp26.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">84</td>
     <td class="digits table-split-left">50,680</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000125/1" target="_blank">pimaricin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 29%, #ffffff00 29%)">29%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r26c2">
     
     
     <td class="regbutton NRPS-like r26c2">
      <a href="#r26c2">Region&nbsp26.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">52,748</td>
     <td class="digits table-split-left">109,991</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000950/1" target="_blank">napsamycin A / napsamycin B / napsamycin C / napsamycin D / mureidomycin A / mureidomycin B</a></td>
      <td>NRP:Uridylpeptide + Other:Nucleoside</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r26c3">
     
     
         
     
     <td class="regbutton CDPS,NRPS,T1PKS hybrid r26c3">
      <a href="#r26c3">Region&nbsp26.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#cdps" target="_blank">CDPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">120,986</td>
     <td class="digits table-split-left">222,233</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000324/1" target="_blank">coelibactin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 36%, #ffffff00 36%)">36%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r26c4">
     
     
         
     
     <td class="regbutton NRPS-like,PKS-like,T1PKS,arylpolyene hybrid r26c4">
      <a href="#r26c4">Region&nbsp26.4</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#pks-like" target="_blank">PKS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#arylpolyene" target="_blank">arylpolyene</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">493,491</td>
     <td class="digits table-split-left">592,270</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001116/1" target="_blank">virginiamycin S1</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 22%, #ffffff00 22%)">22%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r26c5">
     
     
     <td class="regbutton terpene r26c5">
      <a href="#r26c5">Region&nbsp26.5</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">678,213</td>
     <td class="digits table-split-left">700,280</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000661/1" target="_blank">geosmin</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r26c6">
     
     
         
     
     <td class="regbutton NRPS,T1PKS,amglyccycl,hglE-KS hybrid r26c6">
      <a href="#r26c6">Region&nbsp26.6</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hgle-ks" target="_blank">hglE-KS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#amglyccycl" target="_blank">amglyccycl</a>
     </td>
     <td class="digits">702,309</td>
     <td class="digits table-split-left">770,886</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000153/1" target="_blank">stigmatellin</a></td>
      <td>NRP + Polyketide:Modular type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r30c1">
     
     
         
     
     <td class="regbutton LAP,thiopeptide hybrid r30c1">
      <a href="#r30c1">Region&nbsp30.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lap" target="_blank">LAP</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thiopeptide" target="_blank">thiopeptide</a>
     </td>
     <td class="digits">29,780</td>
     <td class="digits table-split-left">57,586</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r31c1">
     
     
     <td class="regbutton NRPS r31c1">
      <a href="#r31c1">Region&nbsp31.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">34,009</td>
     <td class="digits table-split-left">65,422</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r32c1">
     
     
         
     
     <td class="regbutton T1PKS,ladderane hybrid r32c1">
      <a href="#r32c1">Region&nbsp32.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ladderane" target="_blank">ladderane</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">61,982</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001817/1" target="_blank">vancosamine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r34c1">
     
     
     <td class="regbutton NRPS r34c1">
      <a href="#r34c1">Region&nbsp34.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">66,342</td>
     <td class="digits table-split-left">111,786</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001569/1" target="_blank">dechlorocuracomycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r36c1">
     
     
         
     
     <td class="regbutton NRPS-like,PKS-like,T2PKS hybrid r36c1">
      <a href="#r36c1">Region&nbsp36.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#pks-like" target="_blank">PKS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">126,166</td>
     <td class="digits table-split-left">238,151</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000279/1" target="_blank">xantholipin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r36c2">
     
     
     <td class="regbutton T1PKS r36c2">
      <a href="#r36c2">Region&nbsp36.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">283,753</td>
     <td class="digits table-split-left">328,261</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001298/1" target="_blank">4-Z-annimycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 33%, #ffffff00 33%)">33%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r38c1">
     
     
     <td class="regbutton redox-cofactor r38c1">
      <a href="#r38c1">Region&nbsp38.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#redox-cofactor" target="_blank">redox-cofactor</a>
     </td>
     <td class="digits">140,124</td>
     <td class="digits table-split-left">162,166</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001100/1" target="_blank">lankacidin C</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 26%, #ffffff00 26%)">26%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r48c1">
     
     
     <td class="regbutton betalactone r48c1">
      <a href="#r48c1">Region&nbsp48.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>
     </td>
     <td class="digits">38,923</td>
     <td class="digits table-split-left">69,839</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001156/1" target="_blank">platencin</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r48c2">
     
     
     <td class="regbutton terpene r48c2">
      <a href="#r48c2">Region&nbsp48.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">92,689</td>
     <td class="digits table-split-left">113,762</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000657/1" target="_blank">2-methylisoborneol</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 50%, #ffffff00 50%)">50%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r48c3">
     
     
     <td class="regbutton NRPS r48c3">
      <a href="#r48c3">Region&nbsp48.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">139,992</td>
     <td class="digits table-split-left">199,919</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001379/1" target="_blank">BD-12</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 7%, #ffffff00 7%)">7%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r54c1">
     
     
     <td class="regbutton NRPS-like r54c1">
      <a href="#r54c1">Region&nbsp54.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">35,700</td>
     <td class="digits table-split-left">79,563</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000727/1" target="_blank">indigoidine</a></td>
      <td>Saccharide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r54c2">
     
     
     <td class="regbutton terpene r54c2">
      <a href="#r54c2">Region&nbsp54.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">96,938</td>
     <td class="digits table-split-left">119,082</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001181/1" target="_blank">geosmin</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
 </tbody>
</table>
      </div>
     </div>
     
     
     <dl id="regionless-records">
      <dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>
       
         <dd><strong>c00002_CC55NOD..</strong> (original name was: CC55NODE_56_length_5991_cov_89.962280) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00007_CC55NOD..</strong> (original name was: CC55NODE_11_length_249411_cov_102.446289...) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00008_CC55NOD..</strong> (original name was: CC55NODE_45_length_110333_cov_85.993401) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00009_CC55NOD..</strong> (original name was: CC55NODE_15_length_69621_cov_96.277603) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00011_CC55NOD..</strong> (original name was: CC55NODE_33_length_48947_cov_101.870758) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00012_CC55NOD..</strong> (original name was: CC55NODE_1_length_122011_cov_117.538292) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00013_CC55NOD..</strong> (original name was: CC55NODE_14_length_90655_cov_93.571266) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00014_CC55NOD..</strong> (original name was: CC55NODE_65_length_34033_cov_72.075630) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00015_CC55NOD..</strong> (original name was: CC55NODE_3_length_60700_cov_110.828224) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00017_CC55NOD..</strong> (original name was: CC55NODE_42_length_12109_cov_94.573540) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00018_CC55NOD..</strong> (original name was: CC55NODE_59_length_1112_cov_454.714020) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00019_CC55NOD..</strong> (original name was: CC55NODE_31_length_76005_cov_114.315765) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00020_CC55NOD..</strong> (original name was: CC55NODE_2_length_266448_cov_118.251289) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00023_CC55NOD..</strong> (original name was: CC55NODE_20_length_17464_cov_94.759331) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00024_CC55NOD..</strong> (original name was: CC55NODE_21_length_143841_cov_111.841316...) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00027_CC55NOD..</strong> (original name was: CC55NODE_50_length_206893_cov_110.066849...) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00028_CC55NOD..</strong> (original name was: CC55NODE_9_length_117591_cov_118.331856) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00029_CC55NOD..</strong> (original name was: CC55NODE_6_length_107020_cov_116.770821) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00033_CC55NOD..</strong> (original name was: CC55NODE_7_length_90783_cov_117.514847) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00035_CC55NOD..</strong> (original name was: CC55NODE_4_length_252179_cov_110.464157) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00037_CC55NOD..</strong> (original name was: CC55NODE_12_length_4915_cov_78.712105) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00039_CC55NOD..</strong> (original name was: CC55NODE_46_length_1085_cov_372.353912) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00040_CC55NOD..</strong> (original name was: CC55NODE_40_length_5064_cov_90.334320) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00041_CC55NOD..</strong> (original name was: CC55NODE_54_length_994_cov_312.082489) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00042_CC55NOD..</strong> (original name was: CC55NODE_25_length_42024_cov_112.745956) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00043_CC55NOD..</strong> (original name was: CC55NODE_10_length_14942_cov_111.171196) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00044_CC55NOD..</strong> (original name was: CC55NODE_16_length_63508_cov_75.735321) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00045_CC55NOD..</strong> (original name was: CC55NODE_32_length_48761_cov_74.667053) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00046_CC55NOD..</strong> (original name was: CC55NODE_43_length_8747_cov_97.232536) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00047_CC55NOD..</strong> (original name was: CC55NODE_47_length_1085_cov_70.260826) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00049_CC55NOD..</strong> (original name was: CC55NODE_5_length_99458_cov_119.252106) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00050_CC55NOD..</strong> (original name was: CC55NODE_60_length_1024_cov_309.119141) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00051_CC55NOD..</strong> (original name was: CC55NODE_48_length_1250_cov_454.226410) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00052_CC55NOD..</strong> (original name was: CC55NODE_34_length_35736_cov_114.410255) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00053_CC55NOD..</strong> (original name was: CC55NODE_58_length_6936_cov_65.133507) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00055_CC55NOD..</strong> (original name was: CC55NODE_44_length_5388_cov_579.222900) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00056_CC55NOD..</strong> (original name was: CC55NODE_90_length_254216_cov_110.476784...) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00057_CC55NOD..</strong> (original name was: CC55NODE_51_length_14851_cov_76.226784) (Lentzea sp. cc55)</dd>
       
         <dd><strong>c00058_CC55NOD..</strong> (original name was: CC55NODE_41_length_33373_cov_107.603691) (Lentzea sp. cc55)</dd>
       
     </dl>
     
    </div>
    
    <div class="overview-switches">
     <div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    
   </div>
  </div>
  






<div class="page" id='r1c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00001_CC55NOD.. - Region 1 - siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-220"></div> <span class="help-tooltip" id="region_svg-help-220">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00001_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1c1-svg" data-filename="c00001_CC55NOD.._r1c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 70,618 - 83,429 nt. (total: 12,812 nt)
        
        <a class="cluster-rules-header" id="r1c1-rules-header" href="#r1c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r1c1-rules">siderophore: IucA_IucC</div>
      
      <div class="region-svg-container">
       <div id='r1c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-221"></div> <span class="help-tooltip" id="focus-panel-help-221">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1c1-clusterblast" data-name="r1c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1c1-knownclusterblast" data-name="r1c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1c1-subclusterblast" data-name="r1c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1c1-MIBiG-cluster-compare" data-name="r1c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1c1-pfam-details" data-name="r1c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-1"></div> <span class="help-tooltip" id="cb-general-help-1">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1c1_1.svg">
        NZ_JAAMPJ010000011 (339414-352236): Lentzea sp. NEAU-D13 Scaffold11, whole ge... (87% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c1_2.svg">
        NZ_FNIX01000027 (24929-37738): Lentzea jiangxiensis strain CGMCC 4.6609, whol... (66% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c1_3.svg">
        NZ_FNET01000007 (220926-233732): Lentzea violacea strain DSM 44796, whole gen... (38% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c1_4.svg">
        NZ_FOYL01000008 (35459-48265): Lentzea waywayandensis strain DSM 44232, whole... (50% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c1_5.svg">
        NZ_QQAU01000019 (183336-196142): Lentzea flaviverrucosa strain DSM 44664 Ga02... (40% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c1_6.svg">
        NZ_QGHB01000005 (302402-363940): Lechevalieria deserti strain DSM 45480 Ga021... (9% of genes show similarity), NRPS,siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c1_7.svg">
        NZ_QLTT01000004 (312046-371091): Lechevalieria atacamensis strain DSM 45479 G... (10% of genes show similarity), NRPS,siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c1_8.svg">
        NZ_FOFV01000003 (315336-328142): Lentzea albida strain DSM 44437, whole genom... (36% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c1_9.svg">
        NZ_BBOJ01000009 (76285-131678): Lentzea aerocolonigenes strain NBRC 13195, wh... (9% of genes show similarity), NRPS,PKS-like,siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c1_10.svg">
        NZ_CP016793 (5290430-5351462): Lentzea guizhouensis strain DHS C013 chromosom... (6% of genes show similarity), NRPS,siderophore
      </option>
      
    </select>
    <button id= "clusterblast-r1c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-2"></div> <span class="help-tooltip" id="cb-known-help-2">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r1c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-3"></div> <span class="help-tooltip" id="cb-sub-help-3">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-4"></div> <span class="help-tooltip" id="MIBiG-help-4">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1c1"></div>
    <div id="comparison-MIBiG-r1c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">siderophore</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000946.1: 282-7531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000946/index.html#r1c1" target="_new">BGC0000946.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Other</td>
        
        <td>vibrioferrin</td>
        
        
        <td>Vibrio parahaemolyticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001408.1: 1377589-1387887">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001408/index.html#r1c1" target="_new">BGC0001408.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Other</td>
        
        <td>xanthoferrin</td>
        
        
        <td>Xanthomonas oryzae pv. oryzae KACC 10331</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000947.1: 153-15744">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000947/index.html#r1c1" target="_new">BGC0000947.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Other</td>
        
        <td>vibrioferrin</td>
        
        
        <td>Vibrio alginolyticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000943.1: 76520-93234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000943/index.html#r1c1" target="_new">BGC0000943.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.41, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
        <td>0.12</td>
        <td>Other</td>
        
        <td>staphylobactin</td>
        
        
        <td>Staphylococcus aureus subsp. aureus NCTC 8325</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001154.1: 3108358-3124919">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001154/index.html#r1c1" target="_new">BGC0001154.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.24 (id:0.43, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
         
        
        <td>0.06</td>
        <td>Other</td>
        
        <td>cacibiocin B</td>
        
        
        <td>Catenulispora acidiphila DSM 44928</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001194.1: 0-72530">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001194/index.html#r1c1" target="_new">BGC0001194.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.21 (id:0.45, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
         
        
        <td>0.05</td>
        <td>Polyketide</td>
        
        <td>hitachimycin</td>
        
        
        <td>Streptomyces scabrisporus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001593.1: 93-65807">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001593/index.html#r1c1" target="_new">BGC0001593.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.21 (id:0.44, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
         
        
        <td>0.05</td>
        <td>NRP</td>
        
        <td>ficellomycin</td>
        
        
        <td>Streptomyces ficellus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001447.1: 70635-183316">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001447/index.html#r1c1" target="_new">BGC0001447.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.19 (id:0.48, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
         
        
        <td>0.04</td>
        <td>Polyketide</td>
        
        <td>primycin</td>
        
        
        <td>Saccharomonospora azurea SZMC 14600</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002097.1: 5660055-5708619">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002097/index.html#r1c1" target="_new">BGC0002097.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.19 (id:0.45, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
         
        
        <td>0.04</td>
        <td>NRP, Polyketide, Saccharide</td>
        
        <td>kosinostatin</td>
        
        
        <td>Micromonospora sp. 28ISP2-46</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001492.1: 0-70308">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001492/index.html#r1c1" target="_new">BGC0001492.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.18 (id:0.44, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
         
        
        <td>0.04</td>
        <td>Polyketide</td>
        
        <td>abyssomicin M, abyssomicin N, abyssomicin O, abyssomicin P, abyssomicin Q, abyssomicin R, abyssomicin S, abyssomicin T, abyssomicin U, abyssomicin V, abyssomicin W, abyssomicin X</td>
        
        
        <td>Streptomyces sp. LC-6-2</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000946.1: 282-7531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000946/index.html#r1c1" target="_new">BGC0000946.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Other</td>
      <td>vibrioferrin</td>
      <td>Vibrio parahaemolyticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001408.1: 1377589-1387887">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001408/index.html#r1c1" target="_new">BGC0001408.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>xanthoferrin</td>
      <td>Xanthomonas oryzae pv. oryzae KACC 10331</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000947.1: 153-15744">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000947/index.html#r1c1" target="_new">BGC0000947.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Other</td>
      <td>vibrioferrin</td>
      <td>Vibrio alginolyticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000943.1: 76520-93234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000943/index.html#r1c1" target="_new">BGC0000943.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.40 (id:0.41, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
      <td>0.40</td>
      <td>Other</td>
      <td>staphylobactin</td>
      <td>Staphylococcus aureus subsp. aureus NCTC 8325</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001154.1: 3108358-3124919">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001154/index.html#r1c1" target="_new">BGC0001154.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.24 (id:0.43, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
      <td>0.24</td>
      <td>Other</td>
      <td>cacibiocin B</td>
      <td>Catenulispora acidiphila DSM 44928</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001194.1: 0-72530">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001194/index.html#r1c1" target="_new">BGC0001194.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.21 (id:0.45, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
      <td>0.21</td>
      <td>Polyketide</td>
      <td>hitachimycin</td>
      <td>Streptomyces scabrisporus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001593.1: 93-65807">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001593/index.html#r1c1" target="_new">BGC0001593.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.21 (id:0.44, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
      <td>0.21</td>
      <td>NRP</td>
      <td>ficellomycin</td>
      <td>Streptomyces ficellus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001447.1: 70635-183316">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001447/index.html#r1c1" target="_new">BGC0001447.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.19 (id:0.48, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
      <td>0.19</td>
      <td>Polyketide</td>
      <td>primycin</td>
      <td>Saccharomonospora azurea SZMC 14600</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002097.1: 5660055-5708619">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002097/index.html#r1c1" target="_new">BGC0002097.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.19 (id:0.45, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
      <td>0.19</td>
      <td>NRP, Polyketide, Saccharide</td>
      <td>kosinostatin</td>
      <td>Micromonospora sp. 28ISP2-46</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001492.1: 0-70308">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001492/index.html#r1c1" target="_new">BGC0001492.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.18 (id:0.44, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
      <td>0.18</td>
      <td>Polyketide</td>
      <td>abyssomicin M, abyssomicin N, abyssomicin O, abyssomicin P, abyssomicin Q, abyssomicin R, abyssomicin S, abyssomicin T, abyssomicin U, abyssomicin V, abyssomicin W, abyssomicin X</td>
      <td>Streptomyces sp. LC-6-2</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-5"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-5">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1c1-pfam-details-svg" data-filename="c00001_CC55NOD.._r1c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r3c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00003_CC55NOD.. - Region 1 - NAPAA <div class="help-container"> <div class="help-icon" data-id="region_svg-help-222"></div> <span class="help-tooltip" id="region_svg-help-222">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00003_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r3c1-svg" data-filename="c00003_CC55NOD.._r3c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 2,611 - 36,348 nt. (total: 33,738 nt)
        
        <a class="cluster-rules-header" id="r3c1-rules-header" href="#r3c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r3c1-rules">NAPAA: (AMP-binding and PP-binding and TIGR02353)</div>
      
      <div class="region-svg-container">
       <div id='r3c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-223"></div> <span class="help-tooltip" id="focus-panel-help-223">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r3c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r3c1-nrps_pks" data-name="r3c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
      <div class="body-details-header r3c1-clusterblast" data-name="r3c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r3c1-knownclusterblast" data-name="r3c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r3c1-subclusterblast" data-name="r3c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r3c1-MIBiG-cluster-compare" data-name="r3c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r3c1-pfam-details" data-name="r3c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r3c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-6"></div> <span class="help-tooltip" id="nrps-domain-help-6">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r3c1-details-svg" data-filename="c00003_CC55NOD.._r3c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r3c1-details-svg"></div>
</div>
    </div>
     
    
     
    <div class="body-details-section r3c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-7"></div> <span class="help-tooltip" id="cb-general-help-7">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r3c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r3c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r3c1_1.svg">
        NZ_FNIX01000001 (655470-699208): Lentzea jiangxiensis strain CGMCC 4.6609, wh... (68% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r3c1_2.svg">
        NZ_FOFR01000009 (146441-190179): Lechevalieria xinjiangensis strain CGMCC 4.3... (69% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r3c1_3.svg">
        NZ_FOFV01000013 (214217-257955): Lentzea albida strain DSM 44437, whole genom... (71% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r3c1_4.svg">
        NZ_QQAU01000002 (452097-495835): Lentzea flaviverrucosa strain DSM 44664 Ga02... (70% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r3c1_5.svg">
        NZ_FWYC01000004 (559540-603278): Lentzea albidocapillata strain DSM 44073, wh... (66% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r3c1_6.svg">
        NZ_FNCC01000004 (315930-359668): Lechevalieria fradiae strain CGMCC 4.3506, w... (60% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r3c1_7.svg">
        NZ_FOYL01000003 (496824-540562): Lentzea waywayandensis strain DSM 44232, who... (65% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r3c1_8.svg">
        NZ_FNET01000003 (400628-444366): Lentzea violacea strain DSM 44796, whole gen... (63% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r3c1_9.svg">
        NZ_BBOJ01000005 (23592-67330): Lentzea aerocolonigenes strain NBRC 13195, who... (64% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r3c1_10.svg">
        NZ_JAAMPJ010000001 (945172-988910): Lentzea sp. NEAU-D13 Scaffold1, whole gen... (50% of genes show similarity), NRPS-like
      </option>
      
    </select>
    <button id= "clusterblast-r3c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r3c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r3c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-8"></div> <span class="help-tooltip" id="cb-known-help-8">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r3c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-9"></div> <span class="help-tooltip" id="cb-sub-help-9">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r3c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-10"></div> <span class="help-tooltip" id="MIBiG-help-10">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r3c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r3c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r3c1"></div>
    <div id="comparison-MIBiG-r3c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NAPAA</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>ergovaline</td>
        
        
        <td>Epichloe festucae var. lolii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Other</td>
        
        <td>ferrichrome</td>
        
        
        <td>Aspergillus oryzae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>dimethylcoprogen</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876.1: 71-9235">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Other</td>
        
        <td>neopolyoxin C</td>
        
        
        <td>Streptomyces tendae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 10E9</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 22D11</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561.1: 1236-18814">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>curacomycin</td>
        
        
        <td>Streptomyces curacoi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001985.1: 0-26853">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001985/index.html#r1c1" target="_new">BGC0001985.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.57, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Other</td>
        
        <td>pyrazomycin</td>
        
        
        <td>Streptomyces candidus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r3c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>ergovaline</td>
      <td>Epichloe festucae var. lolii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>ferrichrome</td>
      <td>Aspergillus oryzae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>dimethylcoprogen</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876.1: 71-9235">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Other</td>
      <td>neopolyoxin C</td>
      <td>Streptomyces tendae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 10E9</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 22D11</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561.1: 1236-18814">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>curacomycin</td>
      <td>Streptomyces curacoi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001985.1: 0-26853">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001985/index.html#r1c1" target="_new">BGC0001985.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.57, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Other</td>
      <td>pyrazomycin</td>
      <td>Streptomyces candidus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r3c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-11"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-11">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r3c1-pfam-details-svg" data-filename="c00003_CC55NOD.._r3c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r3c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r4c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00004_CC55NOD.. - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-224"></div> <span class="help-tooltip" id="region_svg-help-224">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00004_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r4c1-svg" data-filename="c00004_CC55NOD.._r4c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 27,080 - 37,898 nt. (total: 10,819 nt)
        
        <a class="cluster-rules-header" id="r4c1-rules-header" href="#r4c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r4c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193)</div>
      
      <div class="region-svg-container">
       <div id='r4c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-225"></div> <span class="help-tooltip" id="focus-panel-help-225">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r4c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r4c1-clusterblast" data-name="r4c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r4c1-knownclusterblast" data-name="r4c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r4c1-subclusterblast" data-name="r4c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r4c1-MIBiG-cluster-compare" data-name="r4c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r4c1-pfam-details" data-name="r4c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r4c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-12"></div> <span class="help-tooltip" id="cb-general-help-12">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r4c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r4c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r4c1_1.svg">
        NZ_FNIX01000030 (5535-16354): Lentzea jiangxiensis strain CGMCC 4.6609, whole... (100% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r4c1_2.svg">
        NZ_FWYC01000023 (31922-42741): Lentzea albidocapillata strain DSM 44073, whol... (90% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r4c1_3.svg">
        NZ_FNET01000025 (32528-43347): Lentzea violacea strain DSM 44796, whole genom... (90% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r4c1_4.svg">
        NZ_FOFR01000010 (32499-43318): Lechevalieria xinjiangensis strain CGMCC 4.352... (90% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r4c1_5.svg">
        NZ_JAAMPJ010000023 (34129-44939): Lentzea sp. NEAU-D13 Scaffold23, whole geno... (90% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r4c1_6.svg">
        NZ_QGHB01000025 (34661-45480): Lechevalieria deserti strain DSM 45480 Ga02156... (90% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r4c1_7.svg">
        NZ_QLTT01000019 (34603-45413): Lechevalieria atacamensis strain DSM 45479 Ga0... (90% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r4c1_8.svg">
        NZ_FNCC01000012 (217343-228162): Lechevalieria fradiae strain CGMCC 4.3506, w... (90% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r4c1_9.svg">
        NZ_CP016793 (209864-220683): Lentzea guizhouensis strain DHS C013 chromosome,... (90% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r4c1_10.svg">
        NZ_LT629701 (4156233-4226865): Allokutzneria albata strain DSM 44149 chromoso... (18% of genes show similarity), NRPS,bacteriocin
      </option>
      
    </select>
    <button id= "clusterblast-r4c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r4c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-13"></div> <span class="help-tooltip" id="cb-known-help-13">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r4c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r4c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r4c1_1.svg">
        enduracidin
      </option>
      
    </select>
    <button id="knownclusterblast-r4c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r4c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-14"></div> <span class="help-tooltip" id="cb-sub-help-14">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r4c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-15"></div> <span class="help-tooltip" id="MIBiG-help-15">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r4c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r4c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r4c1"></div>
    <div id="comparison-MIBiG-r4c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
    <div><p>No matches found.</p></div>
  
</div>
   
    <div id="comparison-MIBiG-r4c1-RegionToRegion_RiQ" class="comparison-container ">
  
    <div><p>No matches found.</p></div>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r4c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-16"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-16">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c1-pfam-details-svg" data-filename="c00004_CC55NOD.._r4c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r4c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r4c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00004_CC55NOD.. - Region 2 - NRPS,PKS-like,T1PKS,indole <div class="help-container"> <div class="help-icon" data-id="region_svg-help-226"></div> <span class="help-tooltip" id="region_svg-help-226">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00004_CC55NOD...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r4c2-svg" data-filename="c00004_CC55NOD.._r4c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 146,422 - 250,749 nt. (total: 104,328 nt)
        
        <a class="cluster-rules-header" id="r4c2-rules-header" href="#r4c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r4c2-rules">PKS-like: (t2pks2 or ksIII)<br>indole: (indsynth or dmat or indole_PTase)<br>NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r4c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-227"></div> <span class="help-tooltip" id="focus-panel-help-227">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r4c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r4c2-nrps_pks" data-name="r4c2-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r4c2-clusterblast" data-name="r4c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r4c2-knownclusterblast" data-name="r4c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r4c2-subclusterblast" data-name="r4c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r4c2-MIBiG-cluster-compare" data-name="r4c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r4c2-pfam-details" data-name="r4c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r4c2-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-17"></div> <span class="help-tooltip" id="nrps-domain-help-17">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c2-details-svg" data-filename="c00004_CC55NOD.._r4c2_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r4c2-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r4c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-20"></div> <span class="help-tooltip" id="cb-general-help-20">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r4c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r4c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r4c2_1.svg">
        NZ_FNCC01000005 (140482-218575): Lechevalieria fradiae strain CGMCC 4.3506, w... (79% of genes show similarity), NRPS,PKS-like,T1PKS,indole
      </option>
      
      <option value = "svg/clusterblast_r4c2_2.svg">
        NZ_FNIX01000008 (14464-95770): Lentzea jiangxiensis strain CGMCC 4.6609, whol... (70% of genes show similarity), NRPS,PKS-like,T1PKS,indole,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r4c2_3.svg">
        NZ_QLTT01000004 (532095-593645): Lechevalieria atacamensis strain DSM 45479 G... (70% of genes show similarity), NRPS,T1PKS,indole
      </option>
      
      <option value = "svg/clusterblast_r4c2_4.svg">
        NZ_QGHB01000007 (176720-238264): Lechevalieria deserti strain DSM 45480 Ga021... (63% of genes show similarity), NRPS,T1PKS,indole
      </option>
      
      <option value = "svg/clusterblast_r4c2_5.svg">
        NZ_FOFR01000005 (220547-279807): Lechevalieria xinjiangensis strain CGMCC 4.3... (61% of genes show similarity), NRPS,PKS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r4c2_6.svg">
        NZ_FOYL01000001 (985826-1040913): Lentzea waywayandensis strain DSM 44232, wh... (60% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r4c2_7.svg">
        NZ_FOFV01000027 (14505-131137): Lentzea albida strain DSM 44437, whole genome... (26% of genes show similarity), NRPS,PKS-like,T1PKS,arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r4c2_8.svg">
        NZ_CP016793 (9363786-9440248): Lentzea guizhouensis strain DHS C013 chromosom... (51% of genes show similarity), T1PKS,indole
      </option>
      
      <option value = "svg/clusterblast_r4c2_9.svg">
        NZ_BBOJ01000021 (37506-58526): Lentzea aerocolonigenes strain NBRC 13195, who... (82% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r4c2_10.svg">
        NZ_FOYL01000001 (1049763-1070870): Lentzea waywayandensis strain DSM 44232, w... (73% of genes show similarity), indole
      </option>
      
    </select>
    <button id= "clusterblast-r4c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r4c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-21"></div> <span class="help-tooltip" id="cb-known-help-21">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r4c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r4c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_1.svg">
        fortimicin
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_2.svg">
        phthoxazolin
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_3.svg">
        kedarcidin
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_4.svg">
        tallysomycin A
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_5.svg">
        curacomycin
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_6.svg">
        zorbamycin
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_7.svg">
        naphthyridinomycin
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_8.svg">
        vazabitide A
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_9.svg">
        (2S,6R)-diamino-(5R,7)-dihydroxy-heptanoic acid
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_10.svg">
        s56-p1
      </option>
      
    </select>
    <button id="knownclusterblast-r4c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r4c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-22"></div> <span class="help-tooltip" id="cb-sub-help-22">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r4c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-23"></div> <span class="help-tooltip" id="MIBiG-help-23">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r4c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r4c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r4c2"></div>
    <div id="comparison-MIBiG-r4c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">PKS-like</span></th>
         
         <th><span class="cc-heat-product">indole</span></th>
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
         <th><span class="cc-heat-product">PKS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050.1: 0-19730">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.45, order:1.00, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.45, order:1.00, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.45, order:1.00, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>1.16</td>
        <td>NRP, Polyketide</td>
        
        <td>thalassospiramide A</td>
        
        
        <td>Tistrella bauzanensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.46, order:1.00, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.46, order:1.00, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.46, order:1.00, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>1.15</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>1.08</td>
        <td>Polyketide</td>
        
        <td>strobilurin</td>
        
        
        <td>Strobilurus tenacellus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001023.1: 0-29111">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001023/index.html#r1c1" target="_new">BGC0001023.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.45, order:1.00, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.45, order:1.00, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.45, order:1.00, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>1.08</td>
        <td>NRP, Polyketide</td>
        
        <td>myxochromide S1</td>
        
        
        <td>Stigmatella aurantiaca</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.46, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.46, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.46, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>1.08</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294.1: 26867-35886">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.74 (id:0.55, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.55, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>1.06</td>
        <td>Other</td>
        
        <td>7-prenylisatin</td>
        
        
        <td>Streptomyces sp. MBT28</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001876.1: 0-25465">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001876/index.html#r1c1" target="_new">BGC0001876.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.33 (id:0.42, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.44, order:0.83, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.44, order:0.83, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.44, order:0.83, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>1.03</td>
        <td>NRP</td>
        
        <td>thalassospiramide A</td>
        
        
        <td>Thalassospira sp. CNJ-328</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>1.01</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>1.00</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000065.1: 315-41590">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000065/index.html#r1c1" target="_new">BGC0000065.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.99</td>
        <td>Polyketide</td>
        
        <td>rustmicin</td>
        
        
        <td>Streptomyces galbus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r4c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050.1: 0-19730">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.45, order:1.00, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>NRP, Polyketide</td>
      <td>thalassospiramide A</td>
      <td>Tistrella bauzanensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.46, order:1.00, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001023.1: 0-29111">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001023/index.html#r1c1" target="_new">BGC0001023.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.45, order:1.00, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP, Polyketide</td>
      <td>myxochromide S1</td>
      <td>Stigmatella aurantiaca</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.46, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001876.1: 0-25465">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001876/index.html#r1c1" target="_new">BGC0001876.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:0.83, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>thalassospiramide A</td>
      <td>Thalassospira sp. CNJ-328</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>vicibactin</td>
      <td>Rhizobium etli CFN 42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002058.1: 29690-42292">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002058/index.html#r1c1" target="_new">BGC0002058.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>Reutericyclin A, Reutericyclin B, Reutericyclin C, Reutericyclin D</td>
      <td>Streptococcus mutans B04Sm5</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001686.1: 28257-44195">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001686/index.html#r1c1" target="_new">BGC0001686.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>N-octanoyl-Met-Phe-H</td>
      <td>Clostridium sp. CAG:567</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r4c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-24"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-24">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c2-pfam-details-svg" data-filename="c00004_CC55NOD.._r4c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r4c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r4c2-nrps_pks_products" data-name="r4c2-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r4c2-nrps_pks" data-name="r4c2-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r4c2-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-18"></div> <span class="help-tooltip" id="nrps-structures-help-18">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 146421 - 250749:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r4c2-2" class="smiles-canvas" data-smiles="NC(C(C)C)C(=O)NC([*])C(=O)CC(=O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(val) + (X) + (mal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val&amp;nrps2=x&amp;nrps3=mal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*&amp;nrps2=x&amp;nrps3=mal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r4c2-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(C(C)C)C(=O)NC([*])C(=O)CC(=O)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 5, location 191589 - 240920:
          <div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r4c2-5" class="smiles-canvas" data-smiles="NC(C(C)C)C(=O)NC([*])C(=O)CC(=O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(val) + (X) + (mal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val&amp;nrps2=x&amp;nrps3=mal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*&amp;nrps2=x&amp;nrps3=mal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r4c2-5-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(C(C)C)C(=O)NC([*])C(=O)CC(=O)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 6, location 209732 - 250749:
          <div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r4c2-6" class="smiles-canvas" data-smiles="NC(C(C)C)C(=O)NC([*])C(=O)CC(=O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(val) + (X) + (mal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val&amp;nrps2=x&amp;nrps3=mal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*&amp;nrps2=x&amp;nrps3=mal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r4c2-6-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(C(C)C)C(=O)NC([*])C(=O)CC(=O)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r4c2-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-19"></div> <span class="help-tooltip" id="nrps-monomers-help-19">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds162050_163673</span></strong>: X
      <div class="collapser collapser-target-cds162050_163673 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (20..440)</strong>: X
          <div class="collapser collapser-target-cds162050_163673 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds162050_163673-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DfvTIAGVgK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds211589_214748</span></strong>: val
      <div class="collapser collapser-target-cds211589_214748 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (460..850)</strong>: val
          <div class="collapser collapser-target-cds211589_214748 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds211589_214748-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAFfvGGTFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds214762_217972</span></strong>: X
      <div class="collapser collapser-target-cds214762_217972 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (487..871)</strong>: X
          <div class="collapser collapser-target-cds214762_217972 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala, val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds214762_217972-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVAcmGvIwK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>arg</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds217968_220920</span></strong>: mal
      <div class="collapser collapser-target-cds217968_220920 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (514..781)</strong>: mal
          <div class="collapser collapser-target-cds217968_220920 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA, inactive
              <div class="collapser collapser-target-cds217968_220920-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 87.5%</dd>
<dd>inactive: 87.5%</dd>
<dd>2-Rhyd-Malonyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds217968_220920-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 173.5</dt>

  <dd></dd><dt>inactive: 104.9</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 81.2</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 69.1</dt>

  <dd></dd><dt>Propionyl-CoA: 67.1</dt>

  <dd></dd><dt>CHC-CoA: 45.1</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 44.1</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 43.3</dt>

  <dd></dd><dt>Isobutyryl-CoA: 38.5</dt>

  <dd></dd><dt>trans-1,2-CPDA: 21.8</dt>

  <dd></dd><dt>Acetyl-CoA: 21.2</dt>

  <dd></dd><dt>Benzoyl-CoA: 16.5</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 15.1</dt>

  <dd></dd><dt>fatty_acid: 8.1</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r4c3' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00004_CC55NOD.. - Region 3 - lanthipeptide-class-ii <div class="help-container"> <div class="help-icon" data-id="region_svg-help-228"></div> <span class="help-tooltip" id="region_svg-help-228">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00004_CC55NOD...region003.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r4c3-svg" data-filename="c00004_CC55NOD.._r4c3.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 351,062 - 373,707 nt. (total: 22,646 nt)
        
        <a class="cluster-rules-header" id="r4c3-rules-header" href="#r4c3">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r4c3-rules">lanthipeptide-class-ii: cds(LANC_like and DUF4135)</div>
      
      <div class="region-svg-container">
       <div id='r4c3-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-229"></div> <span class="help-tooltip" id="focus-panel-help-229">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r4c3">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r4c3-lanthipeptides" data-name="r4c3-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r4c3-clusterblast" data-name="r4c3-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r4c3-knownclusterblast" data-name="r4c3-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r4c3-subclusterblast" data-name="r4c3-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r4c3-MIBiG-cluster-compare" data-name="r4c3-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r4c3-pfam-details" data-name="r4c3-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r4c3-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-25"></div> <span class="help-tooltip" id="lanthi-body-help-25">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    <div class="details-text">
      Lanthipeptides - No core peptides found.
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r4c3-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-27"></div> <span class="help-tooltip" id="cb-general-help-27">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r4c3-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r4c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r4c3_1.svg">
        NZ_FNIX01000005 (97962-120665): Lentzea jiangxiensis strain CGMCC 4.6609, who... (95% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r4c3_2.svg">
        NZ_QLTT01000004 (737128-759825): Lechevalieria atacamensis strain DSM 45479 G... (95% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r4c3_3.svg">
        NZ_QGHB01000007 (394044-416723): Lechevalieria deserti strain DSM 45480 Ga021... (95% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r4c3_4.svg">
        NZ_QFWW01000059 (171227-193900): Lentzea terrae strain NEAU-LZS 42 C1219, who... (90% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r4c3_5.svg">
        NZ_BBOJ01000034 (42474-65069): Lentzea aerocolonigenes strain NBRC 13195, who... (90% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r4c3_6.svg">
        NZ_CP016793 (9812295-9834995): Lentzea guizhouensis strain DHS C013 chromosom... (90% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r4c3_7.svg">
        NZ_FOFV01000025 (9934-32778): Lentzea albida strain DSM 44437, whole genome s... (84% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r4c3_8.svg">
        NZ_FNCC01000005 (327311-349957): Lechevalieria fradiae strain CGMCC 4.3506, w... (90% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r4c3_9.svg">
        NZ_FOYL01000001 (674460-697067): Lentzea waywayandensis strain DSM 44232, who... (58% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r4c3_10.svg">
        NZ_QQAU01000012 (188441-211054): Lentzea flaviverrucosa strain DSM 44664 Ga02... (58% of genes show similarity), lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r4c3-download">Download graphic</button>
  </div>
  <div id="clusterblast-r4c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c3-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-28"></div> <span class="help-tooltip" id="cb-known-help-28">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r4c3-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-29"></div> <span class="help-tooltip" id="cb-sub-help-29">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r4c3-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-30"></div> <span class="help-tooltip" id="MIBiG-help-30">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r4c3-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r4c3-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r4c3"></div>
    <div id="comparison-MIBiG-r4c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-ii</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000525.1: 36-3837">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000525/index.html#r1c1" target="_new">BGC0000525.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>RiPP</td>
        
        <td>lichenicidin VK21 A1, lichenicidin VK21 A2</td>
        
        
        <td>Bacillus licheniformis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001407.1: 59853-70655">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001407/index.html#r1c1" target="_new">BGC0001407.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>RiPP</td>
        
        <td>bicereucin</td>
        
        
        <td>Bacillus cereus SJ1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500.1: 0-8863">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.41, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>RiPP</td>
        
        <td>carnolysin A1, carnolysin A2</td>
        
        
        <td>Carnobacterium maltaromaticum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000516.1: 8706-19626">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000516/index.html#r1c1" target="_new">BGC0000516.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>geobacillin II</td>
        
        
        <td>Geobacillus thermodenitrificans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000769.1: 2-26368">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000769/index.html#r1c1" target="_new">BGC0000769.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Saccharide</td>
        
        <td>glycopeptidolipid</td>
        
        
        <td>Mycobacterium avium subsp. hominissuis A5</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000502.1: 0-15725">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000502/index.html#r1c1" target="_new">BGC0000502.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>RiPP</td>
        
        <td>cerecidin, cerecidin A1, cerecidin A2, cerecidin A3, cerecidin A4, cerecidin A5, cerecidin A6, cerecidin A7</td>
        
        
        <td>Bacillus cereus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294.1: 26867-35886">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>7-prenylisatin</td>
        
        
        <td>Streptomyces sp. MBT28</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001210.1: 4366120-4375355">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001210/index.html#r1c1" target="_new">BGC0001210.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>pseudomycoicidin</td>
        
        
        <td>Bacillus pseudomycoides DSM 12442</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000527.1: 0-12324">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000527/index.html#r1c1" target="_new">BGC0000527.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>mersacidin</td>
        
        
        <td>Bacillus sp. HIL-Y85/54728</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r4c3-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000525.1: 36-3837">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000525/index.html#r1c1" target="_new">BGC0000525.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>RiPP</td>
      <td>lichenicidin VK21 A1, lichenicidin VK21 A2</td>
      <td>Bacillus licheniformis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001407.1: 59853-70655">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001407/index.html#r1c1" target="_new">BGC0001407.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>RiPP</td>
      <td>bicereucin</td>
      <td>Bacillus cereus SJ1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500.1: 0-8863">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.41, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>RiPP</td>
      <td>carnolysin A1, carnolysin A2</td>
      <td>Carnobacterium maltaromaticum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000516.1: 8706-19626">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000516/index.html#r1c1" target="_new">BGC0000516.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>RiPP</td>
      <td>geobacillin II</td>
      <td>Geobacillus thermodenitrificans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000769.1: 2-26368">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000769/index.html#r1c1" target="_new">BGC0000769.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Saccharide</td>
      <td>glycopeptidolipid</td>
      <td>Mycobacterium avium subsp. hominissuis A5</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000502.1: 0-15725">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000502/index.html#r1c1" target="_new">BGC0000502.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>RiPP</td>
      <td>cerecidin, cerecidin A1, cerecidin A2, cerecidin A3, cerecidin A4, cerecidin A5, cerecidin A6, cerecidin A7</td>
      <td>Bacillus cereus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294.1: 26867-35886">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>7-prenylisatin</td>
      <td>Streptomyces sp. MBT28</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001210.1: 4366120-4375355">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001210/index.html#r1c1" target="_new">BGC0001210.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>RiPP</td>
      <td>pseudomycoicidin</td>
      <td>Bacillus pseudomycoides DSM 12442</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000527.1: 0-12324">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000527/index.html#r1c1" target="_new">BGC0000527.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>RiPP</td>
      <td>mersacidin</td>
      <td>Bacillus sp. HIL-Y85/54728</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r4c3-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-31"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-31">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c3-pfam-details-svg" data-filename="c00004_CC55NOD.._r4c3_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r4c3-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r4c3-lanthipeptides" data-name="r4c3-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r4c3-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-26"></div> <span class="help-tooltip" id="lanthi-side-help-26">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

  <div class="prediction-text">No core peptides found.</div>

</div>

    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r5c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00005_CC55NOD.. - Region 1 - PKS-like,RRE-containing,T2PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-230"></div> <span class="help-tooltip" id="region_svg-help-230">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00005_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r5c1-svg" data-filename="c00005_CC55NOD.._r5c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 198,168 - 270,647 nt. (total: 72,480 nt)
        
        <a class="cluster-rules-header" id="r5c1-rules-header" href="#r5c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r5c1-rules">T2PKS: (t2ks and t2clf)<br>PKS-like: (t2pks2 or ksIII)<br>RRE-containing: minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE])</div>
      
      <div class="region-svg-container">
       <div id='r5c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-231"></div> <span class="help-tooltip" id="focus-panel-help-231">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r5c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r5c1-nrps_pks" data-name="r5c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r5c1-clusterblast" data-name="r5c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r5c1-knownclusterblast" data-name="r5c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r5c1-subclusterblast" data-name="r5c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
    
     
      <div class="body-details-header r5c1-MIBiG-cluster-compare" data-name="r5c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r5c1-pfam-details" data-name="r5c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r5c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-32"></div> <span class="help-tooltip" id="nrps-domain-help-32">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r5c1-details-svg" data-filename="c00005_CC55NOD.._r5c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r5c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r5c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-33"></div> <span class="help-tooltip" id="cb-general-help-33">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r5c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r5c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r5c1_1.svg">
        NZ_FNET01000005 (185091-271621): Lentzea violacea strain DSM 44796, whole gen... (68% of genes show similarity), PKS-like,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r5c1_2.svg">
        NZ_FOYL01000007 (226175-310922): Lentzea waywayandensis strain DSM 44232, who... (67% of genes show similarity), PKS-like,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r5c1_3.svg">
        NZ_FNIX01000011 (151052-223532): Lentzea jiangxiensis strain CGMCC 4.6609, wh... (75% of genes show similarity), PKS-like,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r5c1_4.svg">
        NZ_FOFR01000004 (148068-233969): Lechevalieria xinjiangensis strain CGMCC 4.3... (67% of genes show similarity), PKS-like,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r5c1_5.svg">
        NZ_CP016793 (475346-559219): Lentzea guizhouensis strain DHS C013 chromosome,... (65% of genes show similarity), PKS-like,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r5c1_6.svg">
        NZ_QGHB01000013 (204036-276516): Lechevalieria deserti strain DSM 45480 Ga021... (71% of genes show similarity), PKS-like,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r5c1_7.svg">
        NZ_QLTT01000007 (167213-239693): Lechevalieria atacamensis strain DSM 45479 G... (71% of genes show similarity), PKS-like,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r5c1_8.svg">
        NZ_FOFV01000016 (177238-249718): Lentzea albida strain DSM 44437, whole genom... (58% of genes show similarity), LAP,PKS-like,T2PKS,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r5c1_9.svg">
        NZ_BBOJ01000004 (229340-301820): Lentzea aerocolonigenes strain NBRC 13195, w... (51% of genes show similarity), LAP,PKS-like,T2PKS,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r5c1_10.svg">
        NZ_KB897778 (287279-386421): Streptomyces sp. CNT372 C593DRAFT scaffold00003.... (30% of genes show similarity), NRPS-like,PKS-like,T2PKS,betalactone
      </option>
      
    </select>
    <button id= "clusterblast-r5c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r5c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r5c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-34"></div> <span class="help-tooltip" id="cb-known-help-34">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r5c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r5c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r5c1_1.svg">
        LL-D49194Î±1 (LLD)
      </option>
      
      <option value = "svg/knownclusterblast_r5c1_2.svg">
        hedamycin
      </option>
      
      <option value = "svg/knownclusterblast_r5c1_3.svg">
        granaticin
      </option>
      
      <option value = "svg/knownclusterblast_r5c1_4.svg">
        erdacin
      </option>
      
      <option value = "svg/knownclusterblast_r5c1_5.svg">
        cinerubin B
      </option>
      
      <option value = "svg/knownclusterblast_r5c1_6.svg">
        steffimycin D
      </option>
      
      <option value = "svg/knownclusterblast_r5c1_7.svg">
        aranciamycin
      </option>
      
      <option value = "svg/knownclusterblast_r5c1_8.svg">
        cosmomycin D
      </option>
      
      <option value = "svg/knownclusterblast_r5c1_9.svg">
        cytorhodin
      </option>
      
      <option value = "svg/knownclusterblast_r5c1_10.svg">
        cosmomycin C
      </option>
      
    </select>
    <button id="knownclusterblast-r5c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r5c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r5c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-35"></div> <span class="help-tooltip" id="cb-sub-help-35">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r5c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r5c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r5c1_1.svg">
        GU233672_c1: granaticin t2pks (71% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r5c1_2.svg">
        AF324838_c1: simocyclinone t2pks (66% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r5c1_3.svg">
        AF080235_c1: landomycin t2pks (50% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r5c1_4.svg">
        FJ483966_c1: polyketomycin t2pks (37% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r5c1_5.svg">
        AB103463_c1: medermycin t2pks (75% of genes show similarity), t2pks
      </option>
      
    </select>
    <button id="subclusterblast-r5c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r5c1-svg"></div>
  
</div>
    </div>
     
    
     
    
     
    
     
    <div class="body-details-section r5c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-38"></div> <span class="help-tooltip" id="MIBiG-help-38">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r5c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r5c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r5c1"></div>
    <div id="comparison-MIBiG-r5c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T2PKS</span></th>
         
         <th><span class="cc-heat-product">PKS-like</span></th>
         
         <th><span class="cc-heat-product">RRE-containing</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.77 (id:0.52, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.77 (id:0.52, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.77 (id:0.52, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
         
        
        <td>1.41</td>
        <td>Polyketide</td>
        
        <td>naphtocyclinone</td>
        
        
        <td>Streptomyces arenae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.52, order:0.65, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.52, order:0.63, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.52, order:0.56, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>1.07</td>
        <td>Polyketide</td>
        
        <td>griseusin</td>
        
        
        <td>Streptomyces griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.84</td>
        <td>Polyketide</td>
        
        <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
        
        
        <td>Parastagonospora nodorum SN15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.50, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.50, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.50, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.82</td>
        <td>Polyketide</td>
        
        <td>spore pigment</td>
        
        
        <td>Streptomyces collinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.56, order:0.45, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.56, order:0.44, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.56, order:0.36, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.78</td>
        <td>Polyketide</td>
        
        <td>jadomycin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723.1: 128-10733">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.56, order:0.41, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.56, order:0.38, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.56, order:0.31, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.77</td>
        <td>Polyketide</td>
        
        <td>oviedomycin</td>
        
        
        <td>Streptomyces antibioticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.52, order:0.43, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.52, order:0.43, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.52, order:0.42, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.75</td>
        <td>Polyketide</td>
        
        <td>granaticin</td>
        
        
        <td>Streptomyces violaceoruber</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.50, order:0.45, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.50, order:0.45, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.50, order:0.45, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.75</td>
        <td>Polyketide</td>
        
        <td>curamycin</td>
        
        
        <td>Streptomyces cyaneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007.1: 5922480-5942549">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.74</td>
        <td>Terpene</td>
        
        <td>atolypene A, atolypene B</td>
        
        
        <td>Amycolatopsis tolypomycina</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.73</td>
        <td>Polyketide</td>
        
        <td>esperamicin</td>
        
        
        <td>Actinomadura verrucosospora</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r5c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.77 (id:0.52, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
      <td>0.77</td>
      <td>Polyketide</td>
      <td>naphtocyclinone</td>
      <td>Streptomyces arenae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.52, order:0.65, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Polyketide</td>
      <td>griseusin</td>
      <td>Streptomyces griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
      <td>Parastagonospora nodorum SN15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723.1: 128-10733">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.56, order:0.41, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>oviedomycin</td>
      <td>Streptomyces antibioticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.56, order:0.45, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>jadomycin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.50, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>spore pigment</td>
      <td>Streptomyces collinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001546.1: 0-10917">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001546/index.html#r1c1" target="_new">BGC0001546.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Alkaloid</td>
      <td>chuangxinmycin</td>
      <td>Actinoplanes tsinanensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001485.1: 0-18179">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001485/index.html#r1c1" target="_new">BGC0001485.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>3-demethylchuangxinmycin</td>
      <td>Actinoplanes tsinanensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002034.1: 86568-105957">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002034/index.html#r1c1" target="_new">BGC0002034.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>perquinoline A, perquinoline B, perquinoline C</td>
      <td>Streptomyces sp. IB2014 016-6</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007.1: 5922480-5942549">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>atolypene A, atolypene B</td>
      <td>Amycolatopsis tolypomycina</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r5c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-39"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-39">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r5c1-pfam-details-svg" data-filename="c00005_CC55NOD.._r5c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r5c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
      <div class="sidepanel-details-header r5c1-t2pks" data-name="r5c1-t2pks">
        Type II PKS
      </div>
     
    
     
      <div class="sidepanel-details-header r5c1-RREfinder" data-name="r5c1-RREfinder">
        RREFinder
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    <div class="sidepanel-details-section r5c1-t2pks">
      <div class="more-details">
    <div class="heading">
      <span>Type II PKS gene cluster prediction details</span>
      <div class="help-container"> <div class="help-icon" data-id="t2pks-help-36"></div> <span class="help-tooltip" id="t2pks-help-36">Predictions of starter units, elongations, product classes, and potential molecular weights for type II PKS clusters.<br>More detailed information is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/t2pks' target='_blank'>here</a>.</span></div>
    </div>
	
	
	<dl class="prediction-text">
        <dt><strong>Cluster section:</strong></dt>
        <dd>198167 to 270647</dd>
		
		<dt><strong>Starter unit(s):</strong></dt>
		<dd>
				2-methylbutyryl-CoA (Score: 427.0; E-value: 3.11e-154)<br>
			
				propionyl-CoA (Score: 276.0; E-value: 4.63e-95)<br>
			
	    <dd>
		</tr>
		
		
		<dt><strong>Malonyl elongations:</strong></dt>
        <dd>
			
			8|9 (Score: 623.4; E-value: 3.3e-190)<br>
			
    	</dd>
		
		
		<dt><strong>Product class(es):</strong></td>
        <dd>
				
				anthracycline<br>
				
				aureolic acid<br>
				
				tetracycline<br>
				
		</dd>
		
		
		<dt><strong>Product molecular weight(s):</strong></dt>
		<dd>
			
			2-methylbutyryl-CoA_8: 381.48 Da<br>
			
			2-methylbutyryl-CoA_9: 423.52 Da<br>
			
			propionyl-CoA_8: 353.43 Da<br>
			
			propionyl-CoA_9: 395.47 Da<br>
			
		</dd>
		
	</dl>
	<dl class="prediction-text">
		
		<dt><strong>cds211853_212609</strong></dt>
			
			<dd>
				KR
				
				(Score: 44.2; E-value: 4.3e-14)<br>
			</dd>
			
		
		<dt><strong>cds231248_232043</strong></dt>
			
			<dd>
				CYC
				
					C5-C14/C3-C16
				
				(Score: 407.8; E-value: 4.1e-125)<br>
			</dd>
			
		
		<dt><strong>cds232182_233100</strong></dt>
			
			<dd>
				AT
				
				(Score: 365.8; E-value: 4.7e-112)<br>
			</dd>
			
			<dd>
				AT
				
					BAB72049.1_Aklavinone_propionyl-CoA
				
				(Score: 276.0; E-value: 4.63e-95)<br>
			</dd>
			
		
		<dt><strong>cds233167_234382</strong></dt>
			
			<dd>
				CLF
				
					8|9
				
				(Score: 623.4; E-value: 3.3e-190)<br>
			</dd>
			
		
		<dt><strong>cds234378_235647</strong></dt>
			
			<dd>
				KS
				
				(Score: 737.0; E-value: 1.3e-224)<br>
			</dd>
			
		
		<dt><strong>cds236573_237503</strong></dt>
			
			<dd>
				CYC
				
					C7-C12
				
				(Score: 420.6; E-value: 6.9e-129)<br>
			</dd>
			
		
		<dt><strong>cds237531_237783</strong></dt>
			
			<dd>
				ACP
				
				(Score: 94.4; E-value: 7.5e-30)<br>
			</dd>
			
		
		<dt><strong>cds244629_245637</strong></dt>
			
			<dd>
				KSIII
				
				(Score: 435.8; E-value: 2.4e-133)<br>
			</dd>
			
			<dd>
				KSIII
				
					AKT74268.1_Trioxacarcin_2-methylbutyryl-CoA
				
				(Score: 427.0; E-value: 3.11e-154)<br>
			</dd>
			
		
		<dt><strong>cds248269_249049</strong></dt>
			
			<dd>
				KR
				
					C9
				
				(Score: 455.8; E-value: 9.4e-140)<br>
			</dd>
			
		
		<dt><strong>cds263885_264566</strong></dt>
			
			<dd>
				KR
				
				(Score: 42.5; E-value: 1.4e-13)<br>
			</dd>
			
		
		<dt><strong>cds265617_266508</strong></dt>
			
			<dd>
				KR
				
				(Score: 80.1; E-value: 4.9e-25)<br>
			</dd>
			
		
	</dl>
    
</div>
    </div>
     
    
     
    <div class="sidepanel-details-section r5c1-RREfinder">
      <div class="more-details">
  <div class="heading">
    <span>RRE predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="rre-side-help-37"></div> <span class="help-tooltip" id="rre-side-help-37">RREfinder results sidepanel.</span></div>
  </div>
  <dl class = 'prediction-text'>
    
      <span><strong>RRE-containing protocluster</strong> (229194...251447)</span>
      <dt></dt>
      
      
      
        
        <strong>cds239194_241447</strong> <div class="collapser collapser-target-cds239194_241447 collapser-level-cds"> </div><div class="collapser-content">
        
          <dd>RRE type: Goadsporin_CD_RRE</dd>
          <dd>Location: 7 - 108</dd>
          <dd>Score: 74.9</dd>
        
          <dd>RRE type: Cyanobactin_D_RRE</dd>
          <dd>Location: 9 - 84</dd>
          <dd>Score: 51.3</dd>
        
          <dd>RRE type: NHLP_CD_RRE</dd>
          <dd>Location: 7 - 91</dd>
          <dd>Score: 47.3</dd>
        
        </div><br>
      <br>
    
    </dl>
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r5c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00005_CC55NOD.. - Region 2 - lanthipeptide-class-iii <div class="help-container"> <div class="help-icon" data-id="region_svg-help-232"></div> <span class="help-tooltip" id="region_svg-help-232">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00005_CC55NOD...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r5c2-svg" data-filename="c00005_CC55NOD.._r5c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 284,572 - 307,136 nt. (total: 22,565 nt)
        
        <a class="cluster-rules-header" id="r5c2-rules-header" href="#r5c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r5c2-rules">lanthipeptide-class-iii: (micKC and not PKS_KS)</div>
      
      <div class="region-svg-container">
       <div id='r5c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-233"></div> <span class="help-tooltip" id="focus-panel-help-233">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r5c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r5c2-lanthipeptides" data-name="r5c2-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r5c2-clusterblast" data-name="r5c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r5c2-knownclusterblast" data-name="r5c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r5c2-subclusterblast" data-name="r5c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r5c2-MIBiG-cluster-compare" data-name="r5c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r5c2-pfam-details" data-name="r5c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r5c2-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-40"></div> <span class="help-tooltip" id="lanthi-body-help-40">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lanthipeptide(s) for cds294571_297136 - Putative Class III</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds297172_297334</span> leader / core peptide
          </dt>
          <dd>
            MGFILDMQDLETPEAPSNAM
            -
            A<span class="dha">Dha</span>GHGGGGGG<span class="dha">Dha</span><span class="dhb">Dhb</span><span class="dhb">Dhb</span>A<span class="dha">Dha</span>NA<span class="dha">Dha</span>LLLP<span class="cys">C</span><span class="dha">Dha</span>H<span class="dha">Dha</span><span class="dhb">Dhb</span>V<span class="dha">Dha</span>LLA<span class="cys">C</span>
        </dd>
      </dl>
    
    <div class="legend">
      <h4> Legend: </h4>
      <div>
        <span class="dha">Dha</span>: Didehydroalanine<br>
        <span class="dhb">Dhb</span>: Didehydrobutyrine<br>
      </div>
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r5c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-42"></div> <span class="help-tooltip" id="cb-general-help-42">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r5c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r5c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r5c2_1.svg">
        NZ_QQAU01000015 (116096-138661): Lentzea flaviverrucosa strain DSM 44664 Ga02... (100% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r5c2_2.svg">
        NZ_CP016793 (571019-592203): Lentzea guizhouensis strain DHS C013 chromosome,... (100% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r5c2_3.svg">
        NZ_FNIX01000011 (117131-139696): Lentzea jiangxiensis strain CGMCC 4.6609, wh... (90% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r5c2_4.svg">
        NZ_BBOJ01000004 (355159-377727): Lentzea aerocolonigenes strain NBRC 13195, w... (100% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r5c2_5.svg">
        NZ_FOFV01000016 (117192-139757): Lentzea albida strain DSM 44437, whole genom... (94% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r5c2_6.svg">
        NZ_FOFR01000004 (110864-133429): Lechevalieria xinjiangensis strain CGMCC 4.3... (85% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r5c2_7.svg">
        NZ_FOYL01000007 (326427-348989): Lentzea waywayandensis strain DSM 44232, who... (94% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r5c2_8.svg">
        NZ_FNET01000005 (148292-170857): Lentzea violacea strain DSM 44796, whole gen... (100% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r5c2_9.svg">
        NZ_QLTT01000007 (133405-155973): Lechevalieria atacamensis strain DSM 45479 G... (94% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r5c2_10.svg">
        NZ_QGHB01000016 (156867-179435): Lechevalieria deserti strain DSM 45480 Ga021... (94% of genes show similarity), lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r5c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r5c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r5c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-43"></div> <span class="help-tooltip" id="cb-known-help-43">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r5c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r5c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r5c2_1.svg">
        Ery-9 / Ery-6 / Ery-8 / Ery-7 / Ery-5 / Ery-4 / Ery-3
      </option>
      
      <option value = "svg/knownclusterblast_r5c2_2.svg">
        chlorizidine A
      </option>
      
    </select>
    <button id="knownclusterblast-r5c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r5c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r5c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-44"></div> <span class="help-tooltip" id="cb-sub-help-44">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r5c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-45"></div> <span class="help-tooltip" id="MIBiG-help-45">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r5c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r5c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r5c2"></div>
    <div id="comparison-MIBiG-r5c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-iii</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.46, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>RiPP</td>
        
        <td>SapB</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496.1: 2822112-2829457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>RiPP</td>
        
        <td>AmfS</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.48, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>RiPP</td>
        
        <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
        
        
        <td>Actinomadura namibiensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.48, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>RiPP</td>
        
        <td>catenulipeptin</td>
        
        
        <td>Catenulispora acidiphila DSM 44928</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001903.1: 8904-19774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001903/index.html#r1c1" target="_new">BGC0001903.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Other</td>
        
        <td>neocarazostatin A</td>
        
        
        <td>Streptomyces sp. MA37</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>viguiepinol</td>
        
        
        <td>Streptomyces sp. KO-3988</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001407.1: 59853-70655">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001407/index.html#r1c1" target="_new">BGC0001407.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>RiPP</td>
        
        <td>bicereucin</td>
        
        
        <td>Bacillus cereus SJ1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000513.1: 4704393-4710596">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000513/index.html#r1c1" target="_new">BGC0000513.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.52, order:0.57, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.22</td>
        <td>RiPP</td>
        
        <td>Ery-9, Ery-6, Ery-8, Ery-7, Ery-5, Ery-4, Ery-3</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000502.1: 0-15725">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000502/index.html#r1c1" target="_new">BGC0000502.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>RiPP</td>
        
        <td>cerecidin, cerecidin A1, cerecidin A2, cerecidin A3, cerecidin A4, cerecidin A5, cerecidin A6, cerecidin A7</td>
        
        
        <td>Bacillus cereus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r5c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.46, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>RiPP</td>
      <td>SapB</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496.1: 2822112-2829457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>RiPP</td>
      <td>AmfS</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.48, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>RiPP</td>
      <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
      <td>Actinomadura namibiensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.48, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>RiPP</td>
      <td>catenulipeptin</td>
      <td>Catenulispora acidiphila DSM 44928</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001903.1: 8904-19774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001903/index.html#r1c1" target="_new">BGC0001903.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>neocarazostatin A</td>
      <td>Streptomyces sp. MA37</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>viguiepinol</td>
      <td>Streptomyces sp. KO-3988</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001407.1: 59853-70655">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001407/index.html#r1c1" target="_new">BGC0001407.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>RiPP</td>
      <td>bicereucin</td>
      <td>Bacillus cereus SJ1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000513.1: 4704393-4710596">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000513/index.html#r1c1" target="_new">BGC0000513.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.52, order:0.57, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>RiPP</td>
      <td>Ery-9, Ery-6, Ery-8, Ery-7, Ery-5, Ery-4, Ery-3</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000502.1: 0-15725">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000502/index.html#r1c1" target="_new">BGC0000502.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>cerecidin, cerecidin A1, cerecidin A2, cerecidin A3, cerecidin A4, cerecidin A5, cerecidin A6, cerecidin A7</td>
      <td>Bacillus cereus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r5c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-46"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-46">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r5c2-pfam-details-svg" data-filename="c00005_CC55NOD.._r5c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r5c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r5c2-lanthipeptides" data-name="r5c2-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r5c2-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-41"></div> <span class="help-tooltip" id="lanthi-side-help-41">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

    <dl class ="prediction-text">
    <dt><span class="serif">cds297172_297334</span> - Class III</dt>
        <dd>Cleavage pHMM score: 9.00</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 2789.4 Da</dd>
        <dd>Molecular weight: 2791.0 Da</dd>
        <dd>Number of bridges: 2</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 2809.1 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2827.1 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2845.1 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2863.1 Da</span>
                    <span class="alt-weight-n"> N = 4</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2881.1 Da</span>
                    <span class="alt-weight-n"> N = 5</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2899.2 Da</span>
                    <span class="alt-weight-n"> N = 6</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2917.2 Da</span>
                    <span class="alt-weight-n"> N = 7</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2935.2 Da</span>
                    <span class="alt-weight-n"> N = 8</span>
                  </dd>
                
            </dl>
          </dd>
        
  </dl>

</div>

    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r6c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00006_CC55NOD.. - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-234"></div> <span class="help-tooltip" id="region_svg-help-234">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00006_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r6c1-svg" data-filename="c00006_CC55NOD.._r6c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 45,512 nt. (total: 45,512 nt)
        
        <a class="cluster-rules-header" id="r6c1-rules-header" href="#r6c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r6c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r6c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-235"></div> <span class="help-tooltip" id="focus-panel-help-235">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r6c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r6c1-nrps_pks" data-name="r6c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r6c1-clusterblast" data-name="r6c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r6c1-knownclusterblast" data-name="r6c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r6c1-subclusterblast" data-name="r6c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r6c1-MIBiG-cluster-compare" data-name="r6c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r6c1-pfam-details" data-name="r6c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r6c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-47"></div> <span class="help-tooltip" id="nrps-domain-help-47">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c1-details-svg" data-filename="c00006_CC55NOD.._r6c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r6c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r6c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-48"></div> <span class="help-tooltip" id="cb-general-help-48">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r6c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r6c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r6c1_1.svg">
        NZ_FNIX01000003 (61825-104853): Lentzea jiangxiensis strain CGMCC 4.6609, who... (91% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c1_2.svg">
        NZ_JACHJP010000002 (507725-567004): Streptosporangium saharense strain CECT 8... (39% of genes show similarity), NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c1_3.svg">
        NZ_FNDJ01000004 (479356-594289): Nonomuraea jiangxiensis strain CGMCC 4.6533,... (22% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c1_4.svg">
        NZ_LIVZ01000008 (11420-78313): Streptomyces sp. TSRI0107 scaffold8, whole gen... (32% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c1_5.svg">
        NZ_JABBWU010000001 (840603-939139): Microbispora sp. H10836 NODE 1, whole gen... (25% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r6c1_6.svg">
        NZ_JABBXA010000001 (24711-123279): Microbispora sp. H13382 NODE 1, whole geno... (23% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r6c1_7.svg">
        NZ_JABBWV010000001 (120354-208245): Microbispora sp. H11081 NODE 1, whole gen... (26% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r6c1_8.svg">
        NZ_AP022870 (462640-542700): Phytohabitans flavus strain NBRC 107702 (31% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c1_9.svg">
        NZ_PVMZ01000010 (126366-190391): Actinoplanes italicus strain DSM 43146 Ga018... (38% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c1_10.svg">
        NZ_KB905359 (62180-160075): Salinispora pacifica DSM 45544 Salpac3DRAFT Scaff... (26% of genes show similarity), NRPS,T1PKS,other
      </option>
      
    </select>
    <button id= "clusterblast-r6c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r6c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-49"></div> <span class="help-tooltip" id="cb-known-help-49">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r6c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r6c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_1.svg">
        maduropeptin
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_2.svg">
        sporolide A / sporolide B
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_3.svg">
        kedarcidin
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_4.svg">
        neocarzinostatin
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_5.svg">
        amycolamycin A / amycolamycin B
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_6.svg">
        lidamycin
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_7.svg">
        C-1027
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_8.svg">
        C-1027
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_9.svg">
        calicheamicin
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_10.svg">
        uncialamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r6c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r6c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-50"></div> <span class="help-tooltip" id="cb-sub-help-50">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r6c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r6c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r6c1_1.svg">
        AY117439_c3: neocarzinostatin enediyne (100% of genes show similarity), enediyne
      </option>
      
      <option value = "svg/subclusterblast_r6c1_2.svg">
        AY271660_c2: maduropeptin enediyne (91% of genes show similarity), enediyne
      </option>
      
      <option value = "svg/subclusterblast_r6c1_3.svg">
        AY048670_c3: C-1027 enediyne (100% of genes show similarity), enediyne
      </option>
      
      <option value = "svg/subclusterblast_r6c1_4.svg">
        AF497482_c2: calicheamicin enediyne (38% of genes show similarity), enediyne
      </option>
      
    </select>
    <button id="subclusterblast-r6c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r6c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r6c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-51"></div> <span class="help-tooltip" id="MIBiG-help-51">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r6c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r6c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r6c1"></div>
    <div id="comparison-MIBiG-r6c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.49, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>Polyketide</td>
        
        <td>esperamicin</td>
        
        
        <td>Actinomadura verrucosospora</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000674.1: 74961-79952">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000674/index.html#r1c1" target="_new">BGC0000674.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Terpene</td>
        
        <td>(-)-Î´-cadinene</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000823.1: 1-16912">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000823/index.html#r1c1" target="_new">BGC0000823.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Alkaloid</td>
        
        <td>rebeccamycin</td>
        
        
        <td>Lechevalieria aerocolonigenes</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide, Saccharide</td>
        
        <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Other</td>
        
        <td>flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Terpene, Polyketide</td>
        
        <td>pyripyropene A</td>
        
        
        <td>unidentified unclassified sequences.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001817.1: 4399-89071">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001817/index.html#r1c1" target="_new">BGC0001817.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.56, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Other</td>
        
        <td>vancosamine</td>
        
        
        <td>Streptomyces sp. WAC2288</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000821.1: 421-26011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000821/index.html#r1c1" target="_new">BGC0000821.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Other,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
        
        <td>rebeccamycin</td>
        
        
        <td>Lechevalieria aerocolonigenes</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000822.1: 1-25681">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000822/index.html#r1c1" target="_new">BGC0000822.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Alkaloid</td>
        
        <td>rebeccamycin</td>
        
        
        <td>Lechevalieria aerocolonigenes</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Polyketide</td>
        
        <td>viguiepinol</td>
        
        
        <td>Streptomyces sp. KO-3988</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r6c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Polyketide</td>
      <td>esperamicin</td>
      <td>Actinomadura verrucosospora</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000674.1: 74961-79952">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000674/index.html#r1c1" target="_new">BGC0000674.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>(-)-Î´-cadinene</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000823.1: 1-16912">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000823/index.html#r1c1" target="_new">BGC0000823.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Alkaloid</td>
      <td>rebeccamycin</td>
      <td>Lechevalieria aerocolonigenes</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide, Saccharide</td>
      <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
      <td>Saccharopolyspora erythraea</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Other</td>
      <td>flaviolin</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Terpene, Polyketide</td>
      <td>pyripyropene A</td>
      <td>unidentified unclassified sequences.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001817.1: 4399-89071">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001817/index.html#r1c1" target="_new">BGC0001817.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.56, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Other</td>
      <td>vancosamine</td>
      <td>Streptomyces sp. WAC2288</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000821.1: 421-26011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000821/index.html#r1c1" target="_new">BGC0000821.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Other,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
      <td>rebeccamycin</td>
      <td>Lechevalieria aerocolonigenes</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000822.1: 1-25681">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000822/index.html#r1c1" target="_new">BGC0000822.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Alkaloid</td>
      <td>rebeccamycin</td>
      <td>Lechevalieria aerocolonigenes</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>viguiepinol</td>
      <td>Streptomyces sp. KO-3988</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r6c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-52"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-52">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c1-pfam-details-svg" data-filename="c00006_CC55NOD.._r6c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r6c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r10c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00010_CC55NOD.. - Region 1 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-236"></div> <span class="help-tooltip" id="region_svg-help-236">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00010_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r10c1-svg" data-filename="c00010_CC55NOD.._r10c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 128,757 nt. (total: 128,757 nt)
        
        <a class="cluster-rules-header" id="r10c1-rules-header" href="#r10c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r10c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r10c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-237"></div> <span class="help-tooltip" id="focus-panel-help-237">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r10c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r10c1-nrps_pks" data-name="r10c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r10c1-clusterblast" data-name="r10c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r10c1-knownclusterblast" data-name="r10c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r10c1-subclusterblast" data-name="r10c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r10c1-MIBiG-cluster-compare" data-name="r10c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r10c1-pfam-details" data-name="r10c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r10c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-53"></div> <span class="help-tooltip" id="nrps-domain-help-53">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r10c1-details-svg" data-filename="c00010_CC55NOD.._r10c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r10c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r10c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-56"></div> <span class="help-tooltip" id="cb-general-help-56">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r10c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r10c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r10c1_1.svg">
        NZ_FNIX01000003 (116265-269458): Lentzea jiangxiensis strain CGMCC 4.6609, wh... (59% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r10c1_2.svg">
        NZ_QQAU01000003 (37577-138068): Lentzea flaviverrucosa strain DSM 44664 Ga024... (57% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r10c1_3.svg">
        NZ_FOFV01000017 (45749-135727): Lentzea albida strain DSM 44437, whole genome... (58% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r10c1_4.svg">
        NZ_BBOM01000012 (192010-305833): Streptomyces sp. NBRC 109706, whole genome s... (16% of genes show similarity), NRPS,T1PKS,lanthipeptide,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r10c1_5.svg">
        NZ_KB913032 (8767740-8887913): Amycolatopsis alba DSM 44262 scaffold1, whole ... (32% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r10c1_6.svg">
        NZ_CP017780 (1529059-1634061): Amycolatopsis sp. BJA-103 chromosome, complete... (45% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r10c1_7.svg">
        NZ_LT629701 (596188-693090): Allokutzneria albata strain DSM 44149 chromosome I (42% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r10c1_8.svg">
        NZ_SNXZ01000002 (316964-487364): Labedaea rhizosphaerae strain DSM 45361 Ga03... (12% of genes show similarity), NRPS,T1PKS,betalactone,transAT-PKS-like
      </option>
      
      <option value = "svg/clusterblast_r10c1_9.svg">
        NZ_AQRJ01000070 (440148-528806): Streptomyces sp. AmelKG-E11A YW7DRAFT scaffo... (20% of genes show similarity), NRPS,T1PKS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r10c1_10.svg">
        NZ_SLVT01000005 (183837-315370): Micromonospora sp. CNZ303 Ga0193651 105, who... (14% of genes show similarity), NRPS,T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r10c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r10c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r10c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-57"></div> <span class="help-tooltip" id="cb-known-help-57">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r10c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r10c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r10c1_1.svg">
        butyrolactol A
      </option>
      
      <option value = "svg/knownclusterblast_r10c1_2.svg">
        funisamine
      </option>
      
      <option value = "svg/knownclusterblast_r10c1_3.svg">
        tetronasin
      </option>
      
      <option value = "svg/knownclusterblast_r10c1_4.svg">
        mediomycin A
      </option>
      
      <option value = "svg/knownclusterblast_r10c1_5.svg">
        mediomiycin A
      </option>
      
      <option value = "svg/knownclusterblast_r10c1_6.svg">
        A83543A
      </option>
      
      <option value = "svg/knownclusterblast_r10c1_7.svg">
        caniferolide A / caniferolide B / caniferolide C / caniferolide D
      </option>
      
      <option value = "svg/knownclusterblast_r10c1_8.svg">
        67-121C
      </option>
      
      <option value = "svg/knownclusterblast_r10c1_9.svg">
        pentamycin
      </option>
      
      <option value = "svg/knownclusterblast_r10c1_10.svg">
        apoptolidin
      </option>
      
    </select>
    <button id="knownclusterblast-r10c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r10c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r10c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-58"></div> <span class="help-tooltip" id="cb-sub-help-58">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r10c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r10c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r10c1_1.svg">
        DQ149987_c1: concanamycin methoxymalonate (40% of genes show similarity), methoxymalonate
      </option>
      
    </select>
    <button id="subclusterblast-r10c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r10c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r10c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-59"></div> <span class="help-tooltip" id="MIBiG-help-59">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r10c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r10c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r10c1"></div>
    <div id="comparison-MIBiG-r10c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.47, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.51</td>
        <td>Other</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000353.1: 60-28528">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000353/index.html#r1c1" target="_new">BGC0000353.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.51, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.50</td>
        <td>NRP</td>
        
        <td>tacrolimus</td>
        
        
        <td>Streptomyces sp. MA6548</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001202.1: 0-27730">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001202/index.html#r1c1" target="_new">BGC0001202.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.45</td>
        <td>NRP, Polyketide</td>
        
        <td>landepoxcin</td>
        
        
        <td>uncultured bacterium AR_412</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001235.1: 53-50000">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001235/index.html#r1c1" target="_new">BGC0001235.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.49, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.44</td>
        <td>NRP, Polyketide</td>
        
        <td>antalid</td>
        
        
        <td>Polyangium spumosum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002000.1: 2056-14558">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002000/index.html#r1c1" target="_new">BGC0002000.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.43</td>
        <td>Polyketide</td>
        
        <td>setomimycin</td>
        
        
        <td>Streptomyces aurantiacus JA 4570</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001366.1: 10-15268">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001366/index.html#r1c1" target="_new">BGC0001366.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.47, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.47, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.43</td>
        <td>Polyketide</td>
        
        <td>BE-24566B, zunyimycin A</td>
        
        
        <td>Streptomyces sp. FJS31-2</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001829.1: 0-25136">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001829/index.html#r1c1" target="_new">BGC0001829.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.42</td>
        <td>NRP, Polyketide</td>
        
        <td>Î±,Î²-epoxyketone</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000345.1: 27-25783">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000345/index.html#r1c1" target="_new">BGC0000345.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.42</td>
        <td>NRP, Polyketide</td>
        
        <td>eponemycin</td>
        
        
        <td>Streptomyces hygroscopicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002026.1: 219-57197">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002026/index.html#r1c1" target="_new">BGC0002026.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.47, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.48, order:0.37, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.42</td>
        <td>NRP</td>
        
        <td>thaxteramide A1, thaxteramide A2, thaxteramide B1, thaxteramide B2</td>
        
        
        <td>Jahnella sp. MSr9139</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001878.1: 37-15759">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001878/index.html#r1c1" target="_new">BGC0001878.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.41, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.42</td>
        <td>NRP, Polyketide</td>
        
        <td>detoxin N2, detoxin N3</td>
        
        
        <td>Streptomyces sp. NRRL B-1347</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r10c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000018.1: 243-42343">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000018/index.html#r1c1" target="_new">BGC0000018.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:0.90, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Polyketide</td>
      <td>angolamycin</td>
      <td>Streptomyces eurythermus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.44, order:1.00, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.50, order:0.75, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Polyketide</td>
      <td>oronofacic acid</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000298.1: 105-25090">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000298/index.html#r1c1" target="_new">BGC0000298.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>aeruginosin 98-A, aeruginosin 98-B, aeruginosin 98-C</td>
      <td>Microcystis aeruginosa NIES-98</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000067.1: 14949-58348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000067/index.html#r1c1" target="_new">BGC0000067.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.50, order:0.66, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>geldanamycin</td>
      <td>Streptomyces hygroscopicus subsp. duamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001502.1: 2741865-2757479">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001502/index.html#r1c1" target="_new">BGC0001502.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.44, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>amonabactin P 750</td>
      <td>Aeromonas hydrophila subsp. hydrophila ATCC 7966</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.44, order:0.69, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>exochelin</td>
      <td>Mycolicibacterium smegmatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001023.1: 0-29111">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001023/index.html#r1c1" target="_new">BGC0001023.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.45, order:0.77, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP, Polyketide</td>
      <td>myxochromide S1</td>
      <td>Stigmatella aurantiaca</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r10c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-60"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-60">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r10c1-pfam-details-svg" data-filename="c00010_CC55NOD.._r10c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r10c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r10c1-nrps_pks_products" data-name="r10c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r10c1-nrps_pks" data-name="r10c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r10c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-54"></div> <span class="help-tooltip" id="nrps-structures-help-54">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 128757:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r10c1-1" class="smiles-canvas" data-smiles="C=CCCC=CCC(O)CCC=CC=CCCCCCC(O)CC(=O)CC(=O)NC(C(C)C)C(=O)NC(C(C)C)C(=O)NC(C(O)C)C(=O)NC(C(C)C)C(=O)NC(C(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ccmal - redmal) + (ccmal) + (ohmal) + (redmal - ccmal - ccmal - redmal) + (redmal) + (ohmal) + (mal) + (mal) + (val - D-val - D-thr - val) + (D-val - X - D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal,redmal&amp;nrps2=ccmal&amp;nrps3=ohmal&amp;nrps4=redmal,ccmal,ccmal,redmal&amp;nrps5=redmal&amp;nrps6=ohmal&amp;nrps7=mal&amp;nrps8=mal&amp;nrps9=val,d-val,d-thr,val&amp;nrps10=d-val,x,d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal*,redmal*&amp;nrps2=ccmal*&amp;nrps3=ohmal*&amp;nrps4=redmal*,ccmal*,ccmal*,redmal*&amp;nrps5=redmal*&amp;nrps6=ohmal*&amp;nrps7=mal*&amp;nrps8=mal*&amp;nrps9=val*,d-val*,d-thr*,val*&amp;nrps10=d-val*,x,d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r10c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C=CCCC=CCC(O)CCC=CC=CCCCCCC(O)CC(=O)CC(=O)NC(C(C)C)C(=O)NC(C(C)C)C(=O)NC(C(O)C)C(=O)NC(C(C)C)C(=O)NC(C(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 0 - 47584:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r10c1-2" class="smiles-canvas" data-smiles="CC(=O)CC(=O)NC(C(C)C)C(=O)NC(C(C)C)C(=O)NC(C(O)C)C(=O)NC(C(C)C)C(=O)NC(C(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(mal) + (mal) + (val - D-val - D-thr - val) + (D-val - X - D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal&amp;nrps2=mal&amp;nrps3=val,d-val,d-thr,val&amp;nrps4=d-val,x,d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal*&amp;nrps2=mal*&amp;nrps3=val*,d-val*,d-thr*,val*&amp;nrps4=d-val*,x,d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r10c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(=O)CC(=O)NC(C(C)C)C(=O)NC(C(C)C)C(=O)NC(C(O)C)C(=O)NC(C(C)C)C(=O)NC(C(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 26966 - 128757:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r10c1-3" class="smiles-canvas" data-smiles="C=CCCC=CCC(O)CCC=CC=CCCCCCC(O)CC(=O)CC(=O)NC(C(C)C)C(=O)NC(C(C)C)C(=O)NC(C(O)C)C(=O)NC(C(C)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ccmal - redmal) + (ccmal) + (ohmal) + (redmal - ccmal - ccmal - redmal) + (redmal) + (ohmal) + (mal) + (mal) + (val - D-val - D-thr - val)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal,redmal&amp;nrps2=ccmal&amp;nrps3=ohmal&amp;nrps4=redmal,ccmal,ccmal,redmal&amp;nrps5=redmal&amp;nrps6=ohmal&amp;nrps7=mal&amp;nrps8=mal&amp;nrps9=val,d-val,d-thr,val" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal*,redmal*&amp;nrps2=ccmal*&amp;nrps3=ohmal*&amp;nrps4=redmal*,ccmal*,ccmal*,redmal*&amp;nrps5=redmal*&amp;nrps6=ohmal*&amp;nrps7=mal*&amp;nrps8=mal*&amp;nrps9=val*,d-val*,d-thr*,val*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r10c1-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C=CCCC=CCC(O)CCC=CC=CCCCCCC(O)CC(=O)CC(=O)NC(C(C)C)C(=O)NC(C(C)C)C(=O)NC(C(O)C)C(=O)NC(C(C)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r10c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-55"></div> <span class="help-tooltip" id="nrps-monomers-help-55">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds2_13985</span></strong>: val - X - X
      <div class="collapser collapser-target-cds2_13985 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val,X,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*,X*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (496..883)</strong>: val
          <div class="collapser collapser-target-cds2_13985 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds2_13985-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>val</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAYWWGGTFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1986..2359)</strong>: X
          <div class="collapser collapser-target-cds2_13985 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds2_13985-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DtlDmGfVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3000..3393)</strong>: X
          <div class="collapser collapser-target-cds2_13985 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophilic
              <div class="collapser collapser-target-cds2_13985-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAWsaGLvDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds13997_27584</span></strong>: val - val - thr - val
      <div class="collapser collapser-target-cds13997_27584 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val,val,thr,val" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*,val*,thr*,val*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (28..419)</strong>: val
          <div class="collapser collapser-target-cds13997_27584 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds13997_27584-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>val</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAYFWGVTFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1072..1468)</strong>: val
          <div class="collapser collapser-target-cds13997_27584 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds13997_27584-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>val</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAYWWGGTFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2544..2911)</strong>: thr
          <div class="collapser collapser-target-cds13997_27584 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds13997_27584-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNVGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3946..4332)</strong>: val
          <div class="collapser collapser-target-cds13997_27584 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds13997_27584-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>val</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAYWWGGTFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds40603_42016</span></strong>: X
      <div class="collapser collapser-target-cds40603_42016 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (2..371)</strong>: X
          <div class="collapser collapser-target-cds40603_42016 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds40603_42016-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">qYsdCGsnvK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pip</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>40% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds46966_49777</span></strong>: mal
      <div class="collapser collapser-target-cds46966_49777 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (544..848)</strong>: mal
          <div class="collapser collapser-target-cds46966_49777 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds46966_49777-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
<dd>inactive: 66.7%</dd>
<dd>2-Rhyd-Malonyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds46966_49777-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 85.1</dt>

  <dd></dd><dt>inactive: 44.5</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 43.2</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 28.7</dt>

  <dd></dd><dt>Propionyl-CoA: 15.6</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 10.6</dt>

  <dd></dd><dt>fatty_acid: 7.9</dt>

  <dd></dd><dt>CHC-CoA: 7.0</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds49784_58556</span></strong>: mal
      <div class="collapser collapser-target-cds49784_58556 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (558..830)</strong>: mal
          <div class="collapser collapser-target-cds49784_58556 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds49784_58556-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 91.7%</dd>
<dd>inactive: 70.8%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds49784_58556-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 203.4</dt>

  <dd></dd><dt>inactive: 104.2</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 64.1</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 60.7</dt>

  <dd></dd><dt>Propionyl-CoA: 27.5</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 23.7</dt>

  <dd></dd><dt>Isobutyryl-CoA: 22.2</dt>

  <dd></dd><dt>Acetyl-CoA: 21.7</dt>

  <dd></dd><dt>CHC-CoA: 17.9</dt>

  <dd></dd><dt>Benzoyl-CoA: 16.8</dt>

  <dd></dd><dt>trans-1,2-CPDA: 13.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 13.7</dt>

  <dd></dd><dt>fatty_acid: 11.2</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds58585_64930</span></strong>: mal
      <div class="collapser collapser-target-cds58585_64930 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (557..847)</strong>: mal
          <div class="collapser collapser-target-cds58585_64930 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds58585_64930-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>inactive: 66.7%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds58585_64930-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 187.8</dt>

  <dd></dd><dt>inactive: 98.1</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 71.9</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 55.1</dt>

  <dd></dd><dt>Propionyl-CoA: 30.2</dt>

  <dd></dd><dt>Benzoyl-CoA: 25.3</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 22.6</dt>

  <dd></dd><dt>Acetyl-CoA: 20.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 19.1</dt>

  <dd></dd><dt>CHC-CoA: 10.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 9.8</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 3.6</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds64994_87098</span></strong>: mal - mal - mal - mal
      <div class="collapser collapser-target-cds64994_87098 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (562..842)</strong>: mal
          <div class="collapser collapser-target-cds64994_87098 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds64994_87098-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 91.7%</dd>
<dd>inactive: 83.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds64994_87098-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 217.3</dt>

  <dd></dd><dt>inactive: 124.1</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 64.0</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 56.5</dt>

  <dd></dd><dt>Propionyl-CoA: 33.1</dt>

  <dd></dd><dt>Isobutyryl-CoA: 29.8</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 23.1</dt>

  <dd></dd><dt>Acetyl-CoA: 19.3</dt>

  <dd></dd><dt>Benzoyl-CoA: 19.0</dt>

  <dd></dd><dt>fatty_acid: 12.2</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 11.6</dt>

  <dd></dd><dt>CHC-CoA: 9.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 8.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (2552..2832)</strong>: mal
          <div class="collapser collapser-target-cds64994_87098 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds64994_87098-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 91.7%</dd>
<dd>inactive: 79.2%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds64994_87098-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 207.5</dt>

  <dd></dd><dt>inactive: 108.8</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 56.5</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 54.5</dt>

  <dd></dd><dt>Isobutyryl-CoA: 31.5</dt>

  <dd></dd><dt>Benzoyl-CoA: 25.6</dt>

  <dd></dd><dt>Acetyl-CoA: 23.2</dt>

  <dd></dd><dt>Propionyl-CoA: 16.8</dt>

  <dd></dd><dt>fatty_acid: 12.6</dt>

  <dd></dd><dt>CHC-CoA: 11.2</dt>

  <dd></dd><dt>trans-1,2-CPDA: 10.6</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 10.2</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: -3.8</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (4193..4463)</strong>: mal
          <div class="collapser collapser-target-cds64994_87098 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds64994_87098-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 91.7%</dd>
<dd>inactive: 79.2%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds64994_87098-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 213.3</dt>

  <dd></dd><dt>inactive: 106.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 64.0</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 62.5</dt>

  <dd></dd><dt>Isobutyryl-CoA: 34.8</dt>

  <dd></dd><dt>Acetyl-CoA: 28.6</dt>

  <dd></dd><dt>Benzoyl-CoA: 28.0</dt>

  <dd></dd><dt>Propionyl-CoA: 26.1</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 24.3</dt>

  <dd></dd><dt>fatty_acid: 17.6</dt>

  <dd></dd><dt>CHC-CoA: 15.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 13.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 13.2</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (5835..6115)</strong>: mal
          <div class="collapser collapser-target-cds64994_87098 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds64994_87098-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 75.0%</dd>
<dd>2-Rhyd-Malonyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds64994_87098-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 218.5</dt>

  <dd></dd><dt>inactive: 110.1</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 50.8</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 45.3</dt>

  <dd></dd><dt>Propionyl-CoA: 29.7</dt>

  <dd></dd><dt>Acetyl-CoA: 20.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 15.6</dt>

  <dd></dd><dt>Benzoyl-CoA: 10.1</dt>

  <dd></dd><dt>CHC-CoA: 8.8</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 5.9</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds87121_91582</span></strong>: mal
      <div class="collapser collapser-target-cds87121_91582 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (556..828)</strong>: mal
          <div class="collapser collapser-target-cds87121_91582 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds87121_91582-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 100.0%</dd>
<dd>inactive: 87.5%</dd>
<dd>Propionyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds87121_91582-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 188.3</dt>

  <dd></dd><dt>inactive: 89.9</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 59.1</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 48.2</dt>

  <dd></dd><dt>Propionyl-CoA: 36.6</dt>

  <dd></dd><dt>Acetyl-CoA: 23.3</dt>

  <dd></dd><dt>Isobutyryl-CoA: 20.6</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 20.3</dt>

  <dd></dd><dt>CHC-CoA: 18.5</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 18.1</dt>

  <dd></dd><dt>Benzoyl-CoA: 17.1</dt>

  <dd></dd><dt>trans-1,2-CPDA: 14.4</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds91581_97008</span></strong>: mal
      <div class="collapser collapser-target-cds91581_97008 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (565..858)</strong>: mal
          <div class="collapser collapser-target-cds91581_97008 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds91581_97008-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 66.7%</dd>
<dd>inactive: 58.3%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds91581_97008-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 156.1</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 76.0</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 73.8</dt>

  <dd></dd><dt>inactive: 68.5</dt>

  <dd></dd><dt>Propionyl-CoA: 41.1</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 40.3</dt>

  <dd></dd><dt>Isobutyryl-CoA: 38.2</dt>

  <dd></dd><dt>Acetyl-CoA: 32.5</dt>

  <dd></dd><dt>Benzoyl-CoA: 25.5</dt>

  <dd></dd><dt>fatty_acid: 23.3</dt>

  <dd></dd><dt>CHC-CoA: 22.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 21.6</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 10.9</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds97054_108757</span></strong>: mal - mal
      <div class="collapser collapser-target-cds97054_108757 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (653..952)</strong>: mal
          <div class="collapser collapser-target-cds97054_108757 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds97054_108757-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 91.7%</dd>
<dd>inactive: 79.2%</dd>
<dd>Propionyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds97054_108757-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 202.4</dt>

  <dd></dd><dt>inactive: 85.1</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 68.7</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 60.3</dt>

  <dd></dd><dt>Propionyl-CoA: 47.9</dt>

  <dd></dd><dt>Isobutyryl-CoA: 33.3</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 31.6</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 31.4</dt>

  <dd></dd><dt>Acetyl-CoA: 26.4</dt>

  <dd></dd><dt>trans-1,2-CPDA: 26.1</dt>

  <dd></dd><dt>Benzoyl-CoA: 23.7</dt>

  <dd></dd><dt>CHC-CoA: 12.5</dt>

  <dd></dd><dt>fatty_acid: 12.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (2351..2635)</strong>: mal
          <div class="collapser collapser-target-cds97054_108757 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds97054_108757-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 91.7%</dd>
<dd>inactive: 75.0%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds97054_108757-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 199.5</dt>

  <dd></dd><dt>inactive: 98.0</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 70.8</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 61.7</dt>

  <dd></dd><dt>Isobutyryl-CoA: 43.1</dt>

  <dd></dd><dt>Propionyl-CoA: 42.2</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 30.8</dt>

  <dd></dd><dt>Benzoyl-CoA: 29.0</dt>

  <dd></dd><dt>Acetyl-CoA: 24.4</dt>

  <dd></dd><dt>CHC-CoA: 23.4</dt>

  <dd></dd><dt>fatty_acid: 14.4</dt>

  <dd></dd><dt>trans-1,2-CPDA: 13.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 9.8</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds109137_110064</span></strong>: mxmal
      <div class="collapser collapser-target-cds109137_110064 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (6..278)</strong>: mxmal
          <div class="collapser collapser-target-cds109137_110064 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  (unknown)
              <div class="collapser collapser-target-cds109137_110064-ATSignature collapser-level-none"> </div><div class="collapser-content">
                No matches above 50%
              </div><br>
          
              minowa:  Methoxymalonyl-CoA
              <div class="collapser collapser-target-cds109137_110064-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methoxymalonyl-CoA: 16.2</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 11.4</dt>

  <dd></dd><dt>Propionyl-CoA: 10.9</dt>

  <dd></dd><dt>CHC-CoA: 8.4</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 6.4</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>inactive: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>Malonyl-CoA: 0.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r10c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00010_CC55NOD.. - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-238"></div> <span class="help-tooltip" id="region_svg-help-238">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00010_CC55NOD...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r10c2-svg" data-filename="c00010_CC55NOD.._r10c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 255,867 - 276,793 nt. (total: 20,927 nt)
        
        <a class="cluster-rules-header" id="r10c2-rules-header" href="#r10c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r10c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r10c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-239"></div> <span class="help-tooltip" id="focus-panel-help-239">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r10c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r10c2-clusterblast" data-name="r10c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r10c2-knownclusterblast" data-name="r10c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r10c2-subclusterblast" data-name="r10c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r10c2-MIBiG-cluster-compare" data-name="r10c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r10c2-pfam-details" data-name="r10c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r10c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-61"></div> <span class="help-tooltip" id="cb-general-help-61">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r10c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r10c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r10c2_1.svg">
        NZ_FNIX01000003 (374074-395001): Lentzea jiangxiensis strain CGMCC 4.6609, wh... (86% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r10c2_2.svg">
        NZ_FOYL01000002 (551545-571403): Lentzea waywayandensis strain DSM 44232, who... (90% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r10c2_3.svg">
        NZ_QQAU01000003 (707969-728896): Lentzea flaviverrucosa strain DSM 44664 Ga02... (85% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r10c2_4.svg">
        NZ_FWYC01000005 (351516-372443): Lentzea albidocapillata strain DSM 44073, wh... (80% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r10c2_5.svg">
        NZ_FNET01000001 (720110-741037): Lentzea violacea strain DSM 44796, whole gen... (76% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r10c2_6.svg">
        NZ_BBOJ01000021 (68541-89468): Lentzea aerocolonigenes strain NBRC 13195, who... (60% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r10c2_7.svg">
        NZ_CP016793 (6849137-6869648): Lentzea guizhouensis strain DHS C013 chromosom... (52% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r10c2_8.svg">
        NZ_FOFR01000006 (189838-210765): Lechevalieria xinjiangensis strain CGMCC 4.3... (55% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r10c2_9.svg">
        NZ_CP015866 (133416-190400): Streptomyces parvulus strain 2297 chromosome, co... (27% of genes show similarity), NRPS-like,terpene
      </option>
      
      <option value = "svg/clusterblast_r10c2_10.svg">
        NZ_JAAMPJ010000012 (45324-66251): Lentzea sp. NEAU-D13 Scaffold12, whole geno... (52% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r10c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r10c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r10c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-62"></div> <span class="help-tooltip" id="cb-known-help-62">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r10c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r10c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r10c2_1.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r10c2_2.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r10c2_3.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r10c2_4.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r10c2_5.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r10c2_6.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r10c2_7.svg">
        nocamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r10c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r10c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r10c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-63"></div> <span class="help-tooltip" id="cb-sub-help-63">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r10c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-64"></div> <span class="help-tooltip" id="MIBiG-help-64">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r10c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r10c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r10c2"></div>
    <div id="comparison-MIBiG-r10c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.52, order:0.65, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.52, order:0.58, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:0.38, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Myxococcus xanthus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644.1: 97-6276">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.22</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Dietzia sp. CQ4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.53, order:0.44, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Terpene</td>
        
        <td>Î²-carotein</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.52, order:0.42, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces collinus Tu 365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.52, order:0.42, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640.1: 2399-8466">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Enterobacteriaceae bacterium DC404</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645.1: 0-7475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.43, order:0.39, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Halobacillus halophilus DSM 2266</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r10c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.52, order:0.65, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.52, order:0.58, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:0.38, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Myxococcus xanthus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644.1: 97-6276">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Dietzia sp. CQ4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.53, order:0.44, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Terpene</td>
      <td>Î²-carotein</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.52, order:0.42, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces collinus Tu 365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.52, order:0.42, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640.1: 2399-8466">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Enterobacteriaceae bacterium DC404</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645.1: 0-7475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.43, order:0.39, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Halobacillus halophilus DSM 2266</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r10c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-65"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-65">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r10c2-pfam-details-svg" data-filename="c00010_CC55NOD.._r10c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r10c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r16c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00016_CC55NOD.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-240"></div> <span class="help-tooltip" id="region_svg-help-240">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00016_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r16c1-svg" data-filename="c00016_CC55NOD.._r16c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 11,493 nt. (total: 11,493 nt)
        
        <a class="cluster-rules-header" id="r16c1-rules-header" href="#r16c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r16c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r16c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-241"></div> <span class="help-tooltip" id="focus-panel-help-241">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r16c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r16c1-clusterblast" data-name="r16c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r16c1-knownclusterblast" data-name="r16c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r16c1-subclusterblast" data-name="r16c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r16c1-MIBiG-cluster-compare" data-name="r16c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r16c1-pfam-details" data-name="r16c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r16c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-66"></div> <span class="help-tooltip" id="cb-general-help-66">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r16c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r16c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r16c1_1.svg">
        NZ_CP016793 (5635267-5652907): Lentzea guizhouensis strain DHS C013 chromosom... (83% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r16c1_2.svg">
        NC_019673 (6944767-6965850): Saccharothrix espanaensis DSM 44229, complete ge... (75% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r16c1_3.svg">
        NZ_CP034550 (8574431-8594945): Saccharothrix syringae strain NRRL B-16468 chr... (69% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r16c1_4.svg">
        NZ_CP023445 (6035927-6057007): Actinosynnema pretiosum strain X47 chromosome,... (69% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r16c1_5.svg">
        NC_013093 (6122957-6144037): Actinosynnema mirum DSM 43827, complete sequence (69% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r16c1_6.svg">
        NZ_CP029607 (6278682-6299762): Actinosynnema pretiosum subsp. pretiosum strai... (69% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r16c1_7.svg">
        NZ_FMZZ01000001 (271529-292642): Alloactinosynnema iranicum strain IBRC-M 104... (42% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r16c1_8.svg">
        NZ_KK037166 (1493902-1513992): Kutzneria sp. 744 supercont1.1, whole genome s... (42% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r16c1_9.svg">
        NZ_CP014859 (2557032-2577329): Actinoalloteichus hymeniacidonis strain HPA177... (40% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r16c1_10.svg">
        NZ_CP022521 (2687462-2707538): Actinoalloteichus hoggarensis strain DSM 45943... (44% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r16c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r16c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r16c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-67"></div> <span class="help-tooltip" id="cb-known-help-67">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r16c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r16c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r16c1_1.svg">
        SF2575
      </option>
      
    </select>
    <button id="knownclusterblast-r16c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r16c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r16c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-68"></div> <span class="help-tooltip" id="cb-sub-help-68">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r16c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-69"></div> <span class="help-tooltip" id="MIBiG-help-69">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r16c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r16c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r16c1"></div>
    <div id="comparison-MIBiG-r16c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001484.1: 0-36432">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001484/index.html#r1c1" target="_new">BGC0001484.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>2&#39;-chloropentostatin, 2&#39;-amino-2&#39;-deoxyadenosine</td>
        
        
        <td>Actinomadura sp. ATCC 39365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500.1: 0-8863">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>RiPP</td>
        
        <td>carnolysin A1, carnolysin A2</td>
        
        
        <td>Carnobacterium maltaromaticum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001120.1: 2543311-2579913">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001120/index.html#r1c1" target="_new">BGC0001120.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>NRP, Polyketide</td>
        
        <td>burkholderic acid</td>
        
        
        <td>Burkholderia thailandensis E264</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000797.1: 324-28703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000797/index.html#r1c1" target="_new">BGC0000797.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Saccharide</td>
        
        <td>sphingan polysaccharide</td>
        
        
        <td>Sphingomonas sp. S88</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000768.1: 0-30166">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000768/index.html#r1c1" target="_new">BGC0000768.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Saccharide</td>
        
        <td>gellan polysaccharide</td>
        
        
        <td>Sphingomonas elodea ATCC 31461</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000207.1: 821-16213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000207/index.html#r1c1" target="_new">BGC0000207.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>Polyketide</td>
        
        <td>chartreusin</td>
        
        
        <td>Streptomyces chartreusis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001639.1: 180937-185949">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001639/index.html#r1c1" target="_new">BGC0001639.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>RiPP</td>
        
        <td>klebsidin</td>
        
        
        <td>Klebsiella pneumoniae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001407.1: 59853-70655">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001407/index.html#r1c1" target="_new">BGC0001407.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.12</td>
        <td>RiPP</td>
        
        <td>bicereucin</td>
        
        
        <td>Bacillus cereus SJ1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r16c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001484.1: 0-36432">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001484/index.html#r1c1" target="_new">BGC0001484.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>2&#39;-chloropentostatin, 2&#39;-amino-2&#39;-deoxyadenosine</td>
      <td>Actinomadura sp. ATCC 39365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500.1: 0-8863">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>RiPP</td>
      <td>carnolysin A1, carnolysin A2</td>
      <td>Carnobacterium maltaromaticum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001120.1: 2543311-2579913">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001120/index.html#r1c1" target="_new">BGC0001120.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>NRP, Polyketide</td>
      <td>burkholderic acid</td>
      <td>Burkholderia thailandensis E264</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000797.1: 324-28703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000797/index.html#r1c1" target="_new">BGC0000797.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Saccharide</td>
      <td>sphingan polysaccharide</td>
      <td>Sphingomonas sp. S88</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000768.1: 0-30166">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000768/index.html#r1c1" target="_new">BGC0000768.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Saccharide</td>
      <td>gellan polysaccharide</td>
      <td>Sphingomonas elodea ATCC 31461</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000207.1: 821-16213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000207/index.html#r1c1" target="_new">BGC0000207.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>Polyketide</td>
      <td>chartreusin</td>
      <td>Streptomyces chartreusis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001639.1: 180937-185949">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001639/index.html#r1c1" target="_new">BGC0001639.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>RiPP</td>
      <td>klebsidin</td>
      <td>Klebsiella pneumoniae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001407.1: 59853-70655">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001407/index.html#r1c1" target="_new">BGC0001407.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>RiPP</td>
      <td>bicereucin</td>
      <td>Bacillus cereus SJ1</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r16c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-70"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-70">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r16c1-pfam-details-svg" data-filename="c00016_CC55NOD.._r16c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r16c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r21c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00021_CC55NOD.. - Region 1 - T3PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-242"></div> <span class="help-tooltip" id="region_svg-help-242">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00021_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r21c1-svg" data-filename="c00021_CC55NOD.._r21c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 69,039 - 110,076 nt. (total: 41,038 nt)
        
        <a class="cluster-rules-header" id="r21c1-rules-header" href="#r21c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r21c1-rules">T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)</div>
      
      <div class="region-svg-container">
       <div id='r21c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-243"></div> <span class="help-tooltip" id="focus-panel-help-243">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r21c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
    
     
      <div class="body-details-header r21c1-clusterblast" data-name="r21c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r21c1-knownclusterblast" data-name="r21c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r21c1-subclusterblast" data-name="r21c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r21c1-MIBiG-cluster-compare" data-name="r21c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r21c1-pfam-details" data-name="r21c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    
     
    <div class="body-details-section r21c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-71"></div> <span class="help-tooltip" id="cb-general-help-71">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r21c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r21c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r21c1_1.svg">
        NZ_FNIX01000003 (502313-543354): Lentzea jiangxiensis strain CGMCC 4.6609, wh... (85% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r21c1_2.svg">
        NZ_CP016793 (8892480-8933521): Lentzea guizhouensis strain DHS C013 chromosom... (37% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r21c1_3.svg">
        NZ_QGHB01000010 (128544-169594): Lechevalieria deserti strain DSM 45480 Ga021... (22% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r21c1_4.svg">
        NZ_QLTT01000001 (612416-653466): Lechevalieria atacamensis strain DSM 45479 G... (22% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r21c1_5.svg">
        NZ_FOFV01000022 (34452-75502): Lentzea albida strain DSM 44437, whole genome ... (23% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r21c1_6.svg">
        NZ_GL877878 (952941-1006520): Saccharopolyspora spinosa NRRL 18395 Scaffold00... (11% of genes show similarity), T3PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r21c1_7.svg">
        NZ_LWLB01000023 (335377-376475): Nocardiopsis sp. TSRI0078 scaffold3, whole g... (17% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r21c1_8.svg">
        NC_013947 (127864-226020): Stackebrandtia nassauensis DSM 44728, complete seq... (5% of genes show similarity), NRPS,T1PKS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r21c1_9.svg">
        NZ_FOUP01000001 (230963-&gt;280812): Saccharopolyspora antimicrobica strain CPCC... (10% of genes show similarity), CDPS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r21c1_10.svg">
        NC_008278 (7010052-7051072): Frankia alni ACN14a, complete genome (16% of genes show similarity), T3PKS
      </option>
      
    </select>
    <button id= "clusterblast-r21c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r21c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r21c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-72"></div> <span class="help-tooltip" id="cb-known-help-72">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r21c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r21c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r21c1_1.svg">
        alkyl-O-dihydrogeranyl-methoxyhydroquinones
      </option>
      
      <option value = "svg/knownclusterblast_r21c1_2.svg">
        A40926
      </option>
      
    </select>
    <button id="knownclusterblast-r21c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r21c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r21c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-73"></div> <span class="help-tooltip" id="cb-sub-help-73">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r21c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-74"></div> <span class="help-tooltip" id="MIBiG-help-74">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r21c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r21c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r21c1"></div>
    <div id="comparison-MIBiG-r21c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T3PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001077.1: 8379669-8386219">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001077/index.html#r1c1" target="_new">BGC0001077.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.52, order:0.62, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.33</td>
        <td>Terpene, Polyketide</td>
        
        <td>alkyl-O-dihydrogeranyl-methoxyhydroquinones</td>
        
        
        <td>Actinoplanes missouriensis 431</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.22</td>
        <td>Other</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>NRP, Polyketide</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000204.1: 4242-15189">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000204/index.html#r1c1" target="_new">BGC0000204.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>benastatin A, benastatin B, BE-43767, TAN1532B</td>
        
        
        <td>Streptomyces sp. A2991200</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>frankiamicin</td>
        
        
        <td>Frankia sp. EAN1pec</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001177.1: 10040-36741">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001177/index.html#r1c1" target="_new">BGC0001177.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>mensacaricin</td>
        
        
        <td>Streptomyces bottropensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001983.1: 622-39496">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001983/index.html#r1c1" target="_new">BGC0001983.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.19</td>
        <td>Other</td>
        
        <td>triacsins</td>
        
        
        <td>Kitasatospora aureofaciens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723.1: 128-10733">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>oviedomycin</td>
        
        
        <td>Streptomyces antibioticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000219.1: 2-24201">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000219/index.html#r1c1" target="_new">BGC0000219.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Polyketide, Saccharide</td>
        
        <td>elloramycin</td>
        
        
        <td>Streptomyces olivaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Polyketide</td>
        
        <td>griseusin</td>
        
        
        <td>Streptomyces griseus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r21c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001077.1: 8379669-8386219">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001077/index.html#r1c1" target="_new">BGC0001077.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.52, order:0.62, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Terpene, Polyketide</td>
      <td>alkyl-O-dihydrogeranyl-methoxyhydroquinones</td>
      <td>Actinoplanes missouriensis 431</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Other</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP, Polyketide</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000204.1: 4242-15189">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000204/index.html#r1c1" target="_new">BGC0000204.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>benastatin A, benastatin B, BE-43767, TAN1532B</td>
      <td>Streptomyces sp. A2991200</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>frankiamicin</td>
      <td>Frankia sp. EAN1pec</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001177.1: 10040-36741">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001177/index.html#r1c1" target="_new">BGC0001177.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>mensacaricin</td>
      <td>Streptomyces bottropensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001983.1: 622-39496">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001983/index.html#r1c1" target="_new">BGC0001983.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Other</td>
      <td>triacsins</td>
      <td>Kitasatospora aureofaciens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723.1: 128-10733">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Polyketide</td>
      <td>oviedomycin</td>
      <td>Streptomyces antibioticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000219.1: 2-24201">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000219/index.html#r1c1" target="_new">BGC0000219.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Polyketide, Saccharide</td>
      <td>elloramycin</td>
      <td>Streptomyces olivaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Polyketide</td>
      <td>griseusin</td>
      <td>Streptomyces griseus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r21c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-75"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-75">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r21c1-pfam-details-svg" data-filename="c00021_CC55NOD.._r21c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r21c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r22c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00022_CC55NOD.. - Region 1 - NRPS,NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-244"></div> <span class="help-tooltip" id="region_svg-help-244">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00022_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r22c1-svg" data-filename="c00022_CC55NOD.._r22c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 79,213 - 111,423 nt. (total: 32,211 nt)
        
        <a class="cluster-rules-header" id="r22c1-rules-header" href="#r22c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r22c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r22c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-245"></div> <span class="help-tooltip" id="focus-panel-help-245">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r22c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r22c1-nrps_pks" data-name="r22c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r22c1-clusterblast" data-name="r22c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r22c1-knownclusterblast" data-name="r22c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r22c1-subclusterblast" data-name="r22c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r22c1-MIBiG-cluster-compare" data-name="r22c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r22c1-pfam-details" data-name="r22c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r22c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-76"></div> <span class="help-tooltip" id="nrps-domain-help-76">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r22c1-details-svg" data-filename="c00022_CC55NOD.._r22c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r22c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r22c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-79"></div> <span class="help-tooltip" id="cb-general-help-79">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r22c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r22c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r22c1_1.svg">
        NZ_FNIX01000002 (625660-689832): Lentzea jiangxiensis strain CGMCC 4.6609, wh... (53% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r22c1_2.svg">
        NZ_FWYC01000003 (47754-110994): Lentzea albidocapillata strain DSM 44073, who... (42% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r22c1_3.svg">
        NZ_FNET01000001 (1057357-1121788): Lentzea violacea strain DSM 44796, whole g... (41% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r22c1_4.svg">
        NZ_KB913037 (10240901-10328441): Amycolatopsis balhimycina FH 1894 strain DSM... (27% of genes show similarity), NRPS,bacteriocin,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r22c1_5.svg">
        NZ_CP048397 (7441299-7554123): Streptomyces sp. S4.7 chromosome, complete genome (17% of genes show similarity), NRPS,linaridin,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r22c1_6.svg">
        NZ_CP018047 (239407-353926): Streptomyces niveus strain SCSIO 3406 chromosome... (18% of genes show similarity), NRPS,linaridin,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r22c1_7.svg">
        NC_018524 (1661697-1743639): Nocardiopsis alba ATCC BAA-2165, complete sequence (25% of genes show similarity), LAP,NRPS,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r22c1_8.svg">
        NZ_CP024985 (2144919-2206084): Streptomyces lavendulae subsp. lavendulae stra... (34% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r22c1_9.svg">
        NZ_CP029194 (785458-901611): Streptomyces venezuelae strain ATCC 15068 chromo... (18% of genes show similarity), NRPS,T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r22c1_10.svg">
        NZ_CP023701 (414377-475382): Streptomyces subrutilus strain ATCC 27467 chromo... (29% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r22c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r22c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r22c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-80"></div> <span class="help-tooltip" id="cb-known-help-80">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r22c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r22c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_1.svg">
        streptolydigin
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_2.svg">
        friulimicin A / friulimicin B / friulimicin C / friulimicin D
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_3.svg">
        vicenistatin
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_4.svg">
        sceliphrolactam
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_5.svg">
        sceliphrolactam
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_6.svg">
        malacidin A / malacidin B
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_7.svg">
        RP-1776
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_8.svg">
        neopolyoxin C
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_9.svg">
        auriculamide
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_10.svg">
        macrotermycins
      </option>
      
    </select>
    <button id="knownclusterblast-r22c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r22c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r22c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-81"></div> <span class="help-tooltip" id="cb-sub-help-81">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r22c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r22c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r22c1_1.svg">
        FN433113_c1: streptolydigin methylasparagine (100% of genes show similarity), methylasparagine
      </option>
      
    </select>
    <button id="subclusterblast-r22c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r22c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r22c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-82"></div> <span class="help-tooltip" id="MIBiG-help-82">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r22c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r22c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r22c1"></div>
    <div id="comparison-MIBiG-r22c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.68</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.64</td>
        <td>NRP</td>
        
        <td>AM-toxin</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.64</td>
        <td>Other</td>
        
        <td>ferrichrome</td>
        
        
        <td>Aspergillus oryzae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.63</td>
        <td>NRP</td>
        
        <td>ergovaline</td>
        
        
        <td>Epichloe festucae var. lolii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.63</td>
        <td>NRP</td>
        
        <td>dimethylcoprogen</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.62</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.61</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.61</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.60</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.58</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r22c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>AM-toxin</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Other</td>
      <td>ferrichrome</td>
      <td>Aspergillus oryzae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>ergovaline</td>
      <td>Epichloe festucae var. lolii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>dimethylcoprogen</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r22c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-83"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-83">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r22c1-pfam-details-svg" data-filename="c00022_CC55NOD.._r22c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r22c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r22c1-nrps_pks_products" data-name="r22c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r22c1-nrps_pks" data-name="r22c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r22c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-77"></div> <span class="help-tooltip" id="nrps-structures-help-77">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 79212 - 111423:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r22c1-1" class="smiles-canvas" data-smiles="NC(CC(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)NC(C(O)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(leu) + (X - X) + (thr)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu&amp;nrps2=x,x&amp;nrps3=thr" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*&amp;nrps2=x,x&amp;nrps3=thr*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r22c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CC(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)NC(C(O)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 81484 - 111423:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r22c1-2" class="smiles-canvas" data-smiles="NC(CC(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)NC(C(O)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(leu) + (X - X) + (thr)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu&amp;nrps2=x,x&amp;nrps3=thr" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*&amp;nrps2=x,x&amp;nrps3=thr*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r22c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CC(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)NC(C(O)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r22c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-78"></div> <span class="help-tooltip" id="nrps-monomers-help-78">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds98585_100256</span></strong>: leu
      <div class="collapser collapser-target-cds98585_100256 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (9..385)</strong>: leu
          <div class="collapser collapser-target-cds98585_100256 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  leu
              <div class="collapser collapser-target-cds98585_100256-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAvdFGLVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds101484_106119</span></strong>: X - X
      <div class="collapser collapser-target-cds101484_106119 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (8..373)</strong>: X
          <div class="collapser collapser-target-cds101484_106119 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asp, asn, glu, gln, aad
              <div class="collapser collapser-target-cds101484_106119-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVenVGAItK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>arg</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (986..1363)</strong>: X
          <div class="collapser collapser-target-cds101484_106119 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds101484_106119-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DALqfGGgFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds106115_110519</span></strong>: thr
      <div class="collapser collapser-target-cds106115_110519 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (448..837)</strong>: thr
          <div class="collapser collapser-target-cds106115_110519 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds106115_110519-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DwWcmGlVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r25c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00025_CC55NOD.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-246"></div> <span class="help-tooltip" id="region_svg-help-246">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00025_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r25c1-svg" data-filename="c00025_CC55NOD.._r25c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 103,568 - 154,430 nt. (total: 50,863 nt)
        
        <a class="cluster-rules-header" id="r25c1-rules-header" href="#r25c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r25c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r25c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-247"></div> <span class="help-tooltip" id="focus-panel-help-247">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r25c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r25c1-nrps_pks" data-name="r25c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r25c1-clusterblast" data-name="r25c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r25c1-knownclusterblast" data-name="r25c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r25c1-subclusterblast" data-name="r25c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r25c1-MIBiG-cluster-compare" data-name="r25c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r25c1-pfam-details" data-name="r25c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r25c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-84"></div> <span class="help-tooltip" id="nrps-domain-help-84">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r25c1-details-svg" data-filename="c00025_CC55NOD.._r25c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r25c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r25c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-87"></div> <span class="help-tooltip" id="cb-general-help-87">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r25c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r25c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r25c1_1.svg">
        NZ_FNIX01000002 (116142-166823): Lentzea jiangxiensis strain CGMCC 4.6609, wh... (81% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r25c1_2.svg">
        NZ_QQAU01000007 (100290-166753): Lentzea flaviverrucosa strain DSM 44664 Ga02... (48% of genes show similarity), LAP,NRPS,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r25c1_3.svg">
        NZ_FOFV01000010 (150413-201210): Lentzea albida strain DSM 44437, whole genom... (57% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r25c1_4.svg">
        NZ_FNET01000001 (38226-110060): Lentzea violacea strain DSM 44796, whole geno... (34% of genes show similarity), LAP,NRPS,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r25c1_5.svg">
        NZ_FOFR01000008 (122870-183136): Lechevalieria xinjiangensis strain CGMCC 4.3... (54% of genes show similarity), LAP,NRPS,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r25c1_6.svg">
        NZ_FOYL01000002 (1104912-1182491): Lentzea waywayandensis strain DSM 44232, w... (29% of genes show similarity), LAP,NRPS,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r25c1_7.svg">
        NZ_JAAMPJ010000003 (554244-605080): Lentzea sp. NEAU-D13 Scaffold3, whole gen... (45% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r25c1_8.svg">
        NZ_FNCC01000003 (322314-382259): Lechevalieria fradiae strain CGMCC 4.3506, w... (29% of genes show similarity), LAP,NRPS,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r25c1_9.svg">
        NZ_CP016793 (6700984-6749222): Lentzea guizhouensis strain DHS C013 chromosom... (36% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r25c1_10.svg">
        NZ_KQ948777 (222024-272956): Streptomyces griseoruber strain DSM 40281 PRJNA2... (32% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r25c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r25c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r25c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-88"></div> <span class="help-tooltip" id="cb-known-help-88">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r25c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r25c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r25c1_1.svg">
        coelichelin
      </option>
      
      <option value = "svg/knownclusterblast_r25c1_2.svg">
        salinichelins
      </option>
      
    </select>
    <button id="knownclusterblast-r25c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r25c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r25c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-89"></div> <span class="help-tooltip" id="cb-sub-help-89">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r25c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-90"></div> <span class="help-tooltip" id="MIBiG-help-90">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r25c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r25c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r25c1"></div>
    <div id="comparison-MIBiG-r25c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000325.1: 510822-533448">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000325/index.html#r1c1" target="_new">BGC0000325.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.58, order:0.63, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.36</td>
        <td>NRP</td>
        
        <td>coelichelin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.49, order:0.69, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>exochelin</td>
        
        
        <td>Mycolicibacterium smegmatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001699.1: 1017269-1047696">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001699/index.html#r1c1" target="_new">BGC0001699.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>nidulanin A</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000424.1: 2645250-2687402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000424/index.html#r1c1" target="_new">BGC0000424.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>serobactin C, serobactin B, serobactin A</td>
        
        
        <td>Herbaspirillum seropedicae SmR1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001824.1: 39732-57269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001824/index.html#r1c1" target="_new">BGC0001824.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>xefoampeptides A-G</td>
        
        
        <td>Xenorhabdus beddingii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002016.1: 2881710-2910119">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002016/index.html#r1c1" target="_new">BGC0002016.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Polyketide</td>
        
        <td>lugdunomycin</td>
        
        
        <td>Streptomyces sp. QL37</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001791.1: 5526778-5556971">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001791/index.html#r1c1" target="_new">BGC0001791.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>sunshinamide</td>
        
        
        <td>Gynuella sunshinyii YC6258</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001796.1: 4751364-4791946">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001796/index.html#r1c1" target="_new">BGC0001796.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>NRP</td>
        
        <td>tauramamide</td>
        
        
        <td>Brevibacillus laterosporus DSM 25</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561.1: 1236-18814">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>NRP</td>
        
        <td>curacomycin</td>
        
        
        <td>Streptomyces curacoi</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r25c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000325.1: 510822-533448">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000325/index.html#r1c1" target="_new">BGC0000325.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.58, order:0.63, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP</td>
      <td>coelichelin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.49, order:0.69, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>exochelin</td>
      <td>Mycolicibacterium smegmatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001699.1: 1017269-1047696">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001699/index.html#r1c1" target="_new">BGC0001699.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>nidulanin A</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000424.1: 2645250-2687402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000424/index.html#r1c1" target="_new">BGC0000424.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>serobactin C, serobactin B, serobactin A</td>
      <td>Herbaspirillum seropedicae SmR1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001824.1: 39732-57269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001824/index.html#r1c1" target="_new">BGC0001824.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>xefoampeptides A-G</td>
      <td>Xenorhabdus beddingii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002016.1: 2881710-2910119">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002016/index.html#r1c1" target="_new">BGC0002016.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>lugdunomycin</td>
      <td>Streptomyces sp. QL37</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001791.1: 5526778-5556971">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001791/index.html#r1c1" target="_new">BGC0001791.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP</td>
      <td>sunshinamide</td>
      <td>Gynuella sunshinyii YC6258</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001796.1: 4751364-4791946">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001796/index.html#r1c1" target="_new">BGC0001796.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP</td>
      <td>tauramamide</td>
      <td>Brevibacillus laterosporus DSM 25</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561.1: 1236-18814">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP</td>
      <td>curacomycin</td>
      <td>Streptomyces curacoi</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r25c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-91"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-91">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r25c1-pfam-details-svg" data-filename="c00025_CC55NOD.._r25c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r25c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r25c1-nrps_pks_products" data-name="r25c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r25c1-nrps_pks" data-name="r25c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r25c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-85"></div> <span class="help-tooltip" id="nrps-structures-help-85">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 103567 - 154430:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r25c1-1" class="smiles-canvas" data-smiles="NC(CCCN)C(=O)NC(C)C(=O)NC(CCCN)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-orn - D-ala - orn)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-orn,d-ala,orn" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-orn*,d-ala*,orn*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r25c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CCCN)C(=O)NC(C)C(=O)NC(CCCN)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r25c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-86"></div> <span class="help-tooltip" id="nrps-monomers-help-86">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds123567_134430</span></strong>: orn - ala - orn
      <div class="collapser collapser-target-cds123567_134430 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn,ala,orn" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn*,ala*,orn*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (143..537)</strong>: orn
          <div class="collapser collapser-target-cds123567_134430 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn
              <div class="collapser collapser-target-cds123567_134430-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>orn, horn</dd>
   <dt>Single AA prediction:</dt>
   <dd>orn</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DINYWGGIGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1601..2013)</strong>: ala
          <div class="collapser collapser-target-cds123567_134430 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ala
              <div class="collapser collapser-target-cds123567_134430-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>ala</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWCIAmIYK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ala</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3041..3439)</strong>: orn
          <div class="collapser collapser-target-cds123567_134430 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn
              <div class="collapser collapser-target-cds123567_134430-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DMwNLGLINK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r26c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00026_CC55NOD.. - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-248"></div> <span class="help-tooltip" id="region_svg-help-248">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00026_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r26c1-svg" data-filename="c00026_CC55NOD.._r26c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 84 - 50,680 nt. (total: 50,597 nt)
        
        <a class="cluster-rules-header" id="r26c1-rules-header" href="#r26c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r26c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r26c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-249"></div> <span class="help-tooltip" id="focus-panel-help-249">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r26c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r26c1-nrps_pks" data-name="r26c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r26c1-clusterblast" data-name="r26c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r26c1-knownclusterblast" data-name="r26c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r26c1-subclusterblast" data-name="r26c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r26c1-MIBiG-cluster-compare" data-name="r26c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r26c1-pfam-details" data-name="r26c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r26c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-92"></div> <span class="help-tooltip" id="nrps-domain-help-92">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r26c1-details-svg" data-filename="c00026_CC55NOD.._r26c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r26c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r26c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-95"></div> <span class="help-tooltip" id="cb-general-help-95">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r26c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r26c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r26c1_1.svg">
        NZ_CP022685 (7732018-7885851): Streptomyces formicae strain KY5 chromosome, c... (6% of genes show similarity), NRPS,T1PKS,T2PKS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r26c1_2.svg">
        NZ_QHCP01000004 (432438-594976): Actinokineospora mzabensis strain CECT 8578 ... (10% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r26c1_3.svg">
        NZ_CP012382 (5967338-&gt;6143326): Streptomyces ambofaciens ATCC 23877 chromosom... (8% of genes show similarity), NRPS,T1PKS,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r26c1_4.svg">
        NZ_CP012949 (5818422-5994901): Streptomyces ambofaciens strain DSM 40697 chro... (8% of genes show similarity), NRPS,T1PKS,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r26c1_5.svg">
        NZ_CP026730 (&lt;567526-660791): Streptomyces sp. CB09001 chromosome, complete g... (7% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r26c1_6.svg">
        NZ_CP020700 (7480407-7604528): Streptomyces tsukubensis strain NRRL 18488 chr... (21% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r26c1_7.svg">
        NZ_CP031194 (7766856-7973496): Streptomyces paludis strain GSSD-12 chromosome... (8% of genes show similarity), NRPS-like,T1PKS,T3PKS,butyrolactone,other,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r26c1_8.svg">
        NZ_CP042266 (12581-225174): Streptomyces qinzhouensis strain SSL-25 chromosom... (7% of genes show similarity), NRPS,T1PKS,T3PKS,ectoine,melanin,other,phenazine
      </option>
      
      <option value = "svg/clusterblast_r26c1_9.svg">
        NZ_PVTF01000007 (201336-364251): Umezawaea tangerina strain DSM 44720 Ga01809... (9% of genes show similarity), NRPS,T1PKS,phosphonate
      </option>
      
      <option value = "svg/clusterblast_r26c1_10.svg">
        NZ_CP010519 (1434220-1508463): Streptomyces albus strain DSM 41398 chromosome... (27% of genes show similarity), T1PKS,terpene
      </option>
      
    </select>
    <button id= "clusterblast-r26c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r26c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r26c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-96"></div> <span class="help-tooltip" id="cb-known-help-96">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r26c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r26c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_1.svg">
        pimaricin
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_2.svg">
        nystatin A1
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_3.svg">
        sceliphrolactam
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_4.svg">
        sceliphrolactam
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_5.svg">
        fostriecin
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_6.svg">
        lobosamide A / lobosamide B / lobosamide C
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_7.svg">
        primycin
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_8.svg">
        cyclizidine
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_9.svg">
        spirangien O
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_10.svg">
        lobophorin B
      </option>
      
    </select>
    <button id="knownclusterblast-r26c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r26c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r26c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-97"></div> <span class="help-tooltip" id="cb-sub-help-97">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r26c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-98"></div> <span class="help-tooltip" id="MIBiG-help-98">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r26c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r26c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r26c1"></div>
    <div id="comparison-MIBiG-r26c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>Polyketide</td>
        
        <td>6-methylsalicyclic acid</td>
        
        
        <td>Aspergillus terreus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Polyketide</td>
        
        <td>(-)-Mellein</td>
        
        
        <td>Parastagonospora nodorum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275.1: 1063-6614">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Polyketide</td>
        
        <td>6-methylsalicyclic acid</td>
        
        
        <td>Glarea lozoyensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001988.1: 106884-139050">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001988/index.html#r1c1" target="_new">BGC0001988.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>neosartorin</td>
        
        
        <td>Aspergillus novofumigatus IBT 16806</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001854.1: 27635-56388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001854/index.html#r1c1" target="_new">BGC0001854.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>tricholignan A</td>
        
        
        <td>Trichoderma harzianum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001901.1: 22313-56388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001901/index.html#r1c1" target="_new">BGC0001901.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>tricholignan A</td>
        
        
        <td>Trichoderma harzianum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000122.1: 1816-33971">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000122/index.html#r1c1" target="_new">BGC0000122.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.51, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>phenylnannolone A, phenylnannolone B, phenylnannolone C</td>
        
        
        <td>Nannocystis pusilla</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000068.1: 1-27296">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000068/index.html#r1c1" target="_new">BGC0000068.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>geldanamycin</td>
        
        
        <td>Streptomyces hygroscopicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257.1: 399-7066">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>1,3,6,8-tetrahydroxynaphthalene</td>
        
        
        <td>Nodulisporium sp. ATCC74245</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>asperlactone</td>
        
        
        <td>Aspergillus ochraceus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r26c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Aspergillus terreus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>(-)-Mellein</td>
      <td>Parastagonospora nodorum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275.1: 1063-6614">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Glarea lozoyensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001988.1: 106884-139050">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001988/index.html#r1c1" target="_new">BGC0001988.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>neosartorin</td>
      <td>Aspergillus novofumigatus IBT 16806</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001854.1: 27635-56388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001854/index.html#r1c1" target="_new">BGC0001854.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>tricholignan A</td>
      <td>Trichoderma harzianum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001901.1: 22313-56388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001901/index.html#r1c1" target="_new">BGC0001901.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>tricholignan A</td>
      <td>Trichoderma harzianum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000122.1: 1816-33971">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000122/index.html#r1c1" target="_new">BGC0000122.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.51, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>phenylnannolone A, phenylnannolone B, phenylnannolone C</td>
      <td>Nannocystis pusilla</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000068.1: 1-27296">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000068/index.html#r1c1" target="_new">BGC0000068.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>geldanamycin</td>
      <td>Streptomyces hygroscopicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257.1: 399-7066">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>1,3,6,8-tetrahydroxynaphthalene</td>
      <td>Nodulisporium sp. ATCC74245</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>asperlactone</td>
      <td>Aspergillus ochraceus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r26c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-99"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-99">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r26c1-pfam-details-svg" data-filename="c00026_CC55NOD.._r26c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r26c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r26c1-nrps_pks_products" data-name="r26c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r26c1-nrps_pks" data-name="r26c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r26c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-93"></div> <span class="help-tooltip" id="nrps-structures-help-93">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 83 - 50680:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r26c1-1" class="smiles-canvas" data-smiles="C(C)C(=O)C=CC=CC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(Me-mal - ccmal) + (ccmal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-mal,ccmal&amp;nrps2=ccmal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-mal*,ccmal*&amp;nrps2=ccmal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r26c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C(C)C(=O)C=CC=CC(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r26c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-94"></div> <span class="help-tooltip" id="nrps-monomers-help-94">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds13537_21064</span></strong>: mmal - mal
      <div class="collapser collapser-target-cds13537_21064 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (529..821)</strong>: mmal
          <div class="collapser collapser-target-cds13537_21064 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds13537_21064-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds13537_21064-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 199.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 185.7</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 163.5</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 133.6</dt>

  <dd></dd><dt>trans-1,2-CPDA: 130.7</dt>

  <dd></dd><dt>Propionyl-CoA: 116.5</dt>

  <dd></dd><dt>Malonyl-CoA: 97.0</dt>

  <dd></dd><dt>CHC-CoA: 84.8</dt>

  <dd></dd><dt>Isobutyryl-CoA: 77.2</dt>

  <dd></dd><dt>Benzoyl-CoA: 67.1</dt>

  <dd></dd><dt>Acetyl-CoA: 39.4</dt>

  <dd></dd><dt>inactive: 32.9</dt>

  <dd></dd><dt>fatty_acid: 29.8</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 23.6</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (1518..1801)</strong>: mal
          <div class="collapser collapser-target-cds13537_21064 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds13537_21064-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 91.7%</dd>
<dd>inactive: 66.7%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds13537_21064-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 204.7</dt>

  <dd></dd><dt>inactive: 89.1</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 78.4</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 67.9</dt>

  <dd></dd><dt>Propionyl-CoA: 39.6</dt>

  <dd></dd><dt>Isobutyryl-CoA: 32.3</dt>

  <dd></dd><dt>CHC-CoA: 30.1</dt>

  <dd></dd><dt>Benzoyl-CoA: 27.4</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 22.6</dt>

  <dd></dd><dt>Acetyl-CoA: 21.9</dt>

  <dd></dd><dt>fatty_acid: 21.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 18.5</dt>

  <dd></dd><dt>trans-1,2-CPDA: 12.6</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 8.3</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds21520_26692</span></strong>: mal
      <div class="collapser collapser-target-cds21520_26692 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (546..823)</strong>: mal
          <div class="collapser collapser-target-cds21520_26692 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds21520_26692-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 100.0%</dd>
<dd>inactive: 83.3%</dd>
<dd>2-Rhyd-Malonyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds21520_26692-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 217.2</dt>

  <dd></dd><dt>inactive: 110.4</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 84.0</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 66.1</dt>

  <dd></dd><dt>Propionyl-CoA: 45.8</dt>

  <dd></dd><dt>Isobutyryl-CoA: 33.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 28.7</dt>

  <dd></dd><dt>fatty_acid: 23.8</dt>

  <dd></dd><dt>Benzoyl-CoA: 21.7</dt>

  <dd></dd><dt>Acetyl-CoA: 21.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 20.0</dt>

  <dd></dd><dt>trans-1,2-CPDA: 13.7</dt>

  <dd></dd><dt>CHC-CoA: 12.4</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds31037_31967</span></strong>: mal
      <div class="collapser collapser-target-cds31037_31967 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (10..299)</strong>: mal
          <div class="collapser collapser-target-cds31037_31967 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds31037_31967-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds31037_31967-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 38.5</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 31.6</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 20.1</dt>

  <dd></dd><dt>Benzoyl-CoA: 14.7</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 14.6</dt>

  <dd></dd><dt>fatty_acid: 12.5</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 9.9</dt>

  <dd></dd><dt>CHC-CoA: 9.8</dt>

  <dd></dd><dt>trans-1,2-CPDA: 9.3</dt>

  <dd></dd><dt>Acetyl-CoA: 9.2</dt>

  <dd></dd><dt>inactive: 6.0</dt>

  <dd></dd><dt>Propionyl-CoA: 0.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds38468_40184</span></strong>: X
      <div class="collapser collapser-target-cds38468_40184 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (47..442)</strong>: X
          <div class="collapser collapser-target-cds38468_40184 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds38468_40184-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">akWceGGaqK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>glu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>40% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds48110_49790</span></strong>: X
      <div class="collapser collapser-target-cds48110_49790 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (45..438)</strong>: X
          <div class="collapser collapser-target-cds48110_49790 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds48110_49790-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">lgYmwSr-WK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>cys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>40% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r26c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00026_CC55NOD.. - Region 2 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-250"></div> <span class="help-tooltip" id="region_svg-help-250">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00026_CC55NOD...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r26c2-svg" data-filename="c00026_CC55NOD.._r26c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 52,748 - 109,991 nt. (total: 57,244 nt)
        
        <a class="cluster-rules-header" id="r26c2-rules-header" href="#r26c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r26c2-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r26c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-251"></div> <span class="help-tooltip" id="focus-panel-help-251">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r26c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r26c2-nrps_pks" data-name="r26c2-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r26c2-clusterblast" data-name="r26c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r26c2-knownclusterblast" data-name="r26c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r26c2-subclusterblast" data-name="r26c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r26c2-MIBiG-cluster-compare" data-name="r26c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r26c2-pfam-details" data-name="r26c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r26c2-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-100"></div> <span class="help-tooltip" id="nrps-domain-help-100">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r26c2-details-svg" data-filename="c00026_CC55NOD.._r26c2_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r26c2-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r26c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-103"></div> <span class="help-tooltip" id="cb-general-help-103">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r26c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r26c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r26c2_1.svg">
        NZ_FNCC01000008 (305243-348155): Lechevalieria fradiae strain CGMCC 4.3506, w... (48% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r26c2_2.svg">
        NZ_KI632509 (3781294-3823804): Amycolatopsis halophila YIM 93223 AmyhaDRAFT S... (52% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r26c2_3.svg">
        NZ_CP040605 (7487303-7528543): Saccharopolyspora sp. ASAGF58 chromosome, comp... (48% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r26c2_4.svg">
        NZ_CP029254 (4362936-4430766): Streptomyces spongiicola strain HNM0071 chromo... (32% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r26c2_5.svg">
        NZ_CP050692 (5732585-5775372): Streptomyces antibioticus strain DSM 41481 chr... (47% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r26c2_6.svg">
        NZ_FWZW01000009 (219300-263227): Streptomyces sp. Amel2xC10, whole genome sho... (47% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r26c2_7.svg">
        NZ_BBNN01000027 (23746-67583): Streptomyces sp. NBRC 110035 strain TP-A0873, ... (45% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r26c2_8.svg">
        NZ_CP021080 (6094558-6137680): Streptomyces pluripotens strain MUSC 135 chrom... (47% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r26c2_9.svg">
        NC_020504 (8869884-8983549): Streptomyces davaonensis JCM 4913, complete genome (9% of genes show similarity), NRPS,NRPS-like,T3PKS,cyanobactin,other,terpene
      </option>
      
      <option value = "svg/clusterblast_r26c2_10.svg">
        NZ_CP049838 (8807862-8850765): Streptomyces cacaoi subsp. asoensis strain H2S... (18% of genes show similarity), NRPS-like
      </option>
      
    </select>
    <button id= "clusterblast-r26c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r26c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r26c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-104"></div> <span class="help-tooltip" id="cb-known-help-104">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r26c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r26c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r26c2_1.svg">
        napsamycin A / napsamycin B / napsamycin C / napsamycin D / mureidomycin A / mureidomycin B
      </option>
      
      <option value = "svg/knownclusterblast_r26c2_2.svg">
        totopotensamide A / totopotensamide B
      </option>
      
      <option value = "svg/knownclusterblast_r26c2_3.svg">
        malacidin A / malacidin B
      </option>
      
      <option value = "svg/knownclusterblast_r26c2_4.svg">
        lobophorin A
      </option>
      
      <option value = "svg/knownclusterblast_r26c2_5.svg">
        friulimicin A / friulimicin B / friulimicin C / friulimicin D
      </option>
      
      <option value = "svg/knownclusterblast_r26c2_6.svg">
        kirromycin
      </option>
      
      <option value = "svg/knownclusterblast_r26c2_7.svg">
        pacidamycin 1 / pacidamycin 2 / pacidamycin 3 / pacidamycin 4 / pacidamycin 5 / pacidamycin 6 / pacidamycin 7 / pacidamycin D
      </option>
      
      <option value = "svg/knownclusterblast_r26c2_8.svg">
        perquinoline A / perquinoline B / perquinoline C
      </option>
      
      <option value = "svg/knownclusterblast_r26c2_9.svg">
        chloramphenicol
      </option>
      
      <option value = "svg/knownclusterblast_r26c2_10.svg">
        67-121C
      </option>
      
    </select>
    <button id="knownclusterblast-r26c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r26c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r26c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-105"></div> <span class="help-tooltip" id="cb-sub-help-105">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r26c2-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r26c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r26c2_1.svg">
        HQ287563_c1: napsamycin diaminobutyric acid (75% of genes show similarity), DABA
      </option>
      
      <option value = "svg/subclusterblast_r26c2_2.svg">
        AB116234_c1: chloramphenicol p-amino-phenylpyruvate (100% of genes show similarity), PAPA
      </option>
      
    </select>
    <button id="subclusterblast-r26c2-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r26c2-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r26c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-106"></div> <span class="help-tooltip" id="MIBiG-help-106">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r26c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r26c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r26c2"></div>
    <div id="comparison-MIBiG-r26c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.52, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.52, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.80</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.72</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000727.1: 21-29286">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000727/index.html#r1c1" target="_new">BGC0000727.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.52, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.52, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.63</td>
        <td>Saccharide</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces aureofaciens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561.1: 1236-18814">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.59</td>
        <td>NRP</td>
        
        <td>curacomycin</td>
        
        
        <td>Streptomyces curacoi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000957.1: 361-40389">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000957/index.html#r1c1" target="_new">BGC0000957.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.56</td>
        <td>NRP, Polyketide</td>
        
        <td>mycotrienin I</td>
        
        
        <td>Streptomyces sp. XZQH13</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.55</td>
        <td>NRP</td>
        
        <td>ergovaline</td>
        
        
        <td>Epichloe festucae var. lolii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.55</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 10E9</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.55</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 22D11</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002034.1: 86568-105957">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002034/index.html#r1c1" target="_new">BGC0002034.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.54, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.51, order:0.27, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.54</td>
        <td>Other</td>
        
        <td>perquinoline A, perquinoline B, perquinoline C</td>
        
        
        <td>Streptomyces sp. IB2014 016-6</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000145.1: 1-32637">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000145/index.html#r1c1" target="_new">BGC0000145.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.52</td>
        <td>Polyketide</td>
        
        <td>salinosporamide A</td>
        
        
        <td>Salinispora tropica</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r26c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.52, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000727.1: 21-29286">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000727/index.html#r1c1" target="_new">BGC0000727.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.52, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Saccharide</td>
      <td>indigoidine</td>
      <td>Streptomyces aureofaciens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561.1: 1236-18814">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>curacomycin</td>
      <td>Streptomyces curacoi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000957.1: 361-40389">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000957/index.html#r1c1" target="_new">BGC0000957.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP, Polyketide</td>
      <td>mycotrienin I</td>
      <td>Streptomyces sp. XZQH13</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 10E9</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 22D11</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>ergovaline</td>
      <td>Epichloe festucae var. lolii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000145.1: 1-32637">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000145/index.html#r1c1" target="_new">BGC0000145.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>salinosporamide A</td>
      <td>Salinispora tropica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001041.1: 1140546-1182556">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001041/index.html#r1c1" target="_new">BGC0001041.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP, Polyketide</td>
      <td>salinosporamide A</td>
      <td>Salinispora tropica CNB-440</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r26c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-107"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-107">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r26c2-pfam-details-svg" data-filename="c00026_CC55NOD.._r26c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r26c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r26c2-nrps_pks_products" data-name="r26c2-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r26c2-nrps_pks" data-name="r26c2-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r26c2-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-101"></div> <span class="help-tooltip" id="nrps-structures-help-101">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 52747 - 109991:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r26c2-2" class="smiles-canvas" data-smiles="NC(CCC(=O)N)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(gln) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln&amp;nrps2=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln*&amp;nrps2=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r26c2-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CCC(=O)N)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 52747 - 95553:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r26c2-3" class="smiles-canvas" data-smiles="NC(CCC(=O)N)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(gln) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln&amp;nrps2=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln*&amp;nrps2=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r26c2-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CCC(=O)N)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 4, location 67750 - 109991:
          <div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r26c2-4" class="smiles-canvas" data-smiles="NC(CCC(=O)N)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(gln) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln&amp;nrps2=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln*&amp;nrps2=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r26c2-4-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CCC(=O)N)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r26c2-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-102"></div> <span class="help-tooltip" id="nrps-monomers-help-102">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds72725_76562</span></strong>: gln
      <div class="collapser collapser-target-cds72725_76562 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (31..431)</strong>: gln
          <div class="collapser collapser-target-cds72725_76562 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gln
              <div class="collapser collapser-target-cds72725_76562-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAWQFGLInK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds87565_90775</span></strong>: X
      <div class="collapser collapser-target-cds87565_90775 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (231..625)</strong>: X
          <div class="collapser collapser-target-cds87565_90775 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds87565_90775-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DiWQstadDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds93864_95553</span></strong>: X
      <div class="collapser collapser-target-cds93864_95553 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (59..441)</strong>: X
          <div class="collapser collapser-target-cds93864_95553 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds93864_95553-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">hLmmVmrwGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asp</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>40% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds95755_97105</span></strong>: X
      <div class="collapser collapser-target-cds95755_97105 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (70..298)</strong>: X
          <div class="collapser collapser-target-cds95755_97105 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds95755_97105-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">---aFiersK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>20% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds108767_109991</span></strong>: mxmal
      <div class="collapser collapser-target-cds108767_109991 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (72..293)</strong>: mxmal
          <div class="collapser collapser-target-cds108767_109991 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  (unknown)
              <div class="collapser collapser-target-cds108767_109991-ATSignature collapser-level-none"> </div><div class="collapser-content">
                No matches above 50%
              </div><br>
          
              minowa:  Methoxymalonyl-CoA
              <div class="collapser collapser-target-cds108767_109991-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methoxymalonyl-CoA: 32.4</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 24.4</dt>

  <dd></dd><dt>Malonyl-CoA: 23.1</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 11.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 10.2</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>inactive: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>Propionyl-CoA: 0.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r26c3' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00026_CC55NOD.. - Region 3 - CDPS,NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-252"></div> <span class="help-tooltip" id="region_svg-help-252">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00026_CC55NOD...region003.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r26c3-svg" data-filename="c00026_CC55NOD.._r26c3.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 120,986 - 222,233 nt. (total: 101,248 nt)
        
        <a class="cluster-rules-header" id="r26c3-rules-header" href="#r26c3">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r26c3-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>CDPS: (CDPS or cycdipepsynth)<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r26c3-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-253"></div> <span class="help-tooltip" id="focus-panel-help-253">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r26c3">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r26c3-nrps_pks" data-name="r26c3-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r26c3-clusterblast" data-name="r26c3-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r26c3-knownclusterblast" data-name="r26c3-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r26c3-subclusterblast" data-name="r26c3-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r26c3-MIBiG-cluster-compare" data-name="r26c3-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r26c3-pfam-details" data-name="r26c3-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r26c3-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-108"></div> <span class="help-tooltip" id="nrps-domain-help-108">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r26c3-details-svg" data-filename="c00026_CC55NOD.._r26c3_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r26c3-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r26c3-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-111"></div> <span class="help-tooltip" id="cb-general-help-111">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r26c3-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r26c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r26c3_1.svg">
        NZ_FNCC01000001 (591149-683224): Lechevalieria fradiae strain CGMCC 4.3506, w... (29% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r26c3_2.svg">
        NZ_CP015362 (290184-501485): Streptomyces sp. S8 chromosome, complete genome (11% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r26c3_3.svg">
        NZ_CP027306 (376515-471919): Streptomyces atratus strain SCSIO ZH16 chromosom... (18% of genes show similarity), NRPS,PKS-like
      </option>
      
      <option value = "svg/clusterblast_r26c3_4.svg">
        NZ_LT629701 (1164706-1204842): Allokutzneria albata strain DSM 44149 chromoso... (21% of genes show similarity), CDPS,TfuA-related
      </option>
      
      <option value = "svg/clusterblast_r26c3_5.svg">
        NZ_JAAATY010000017 (18387-57585): Kibdelosporangium sp. 4NS15 sequence 17, wh... (34% of genes show similarity), CDPS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r26c3_6.svg">
        NZ_CP033581 (898663-1139331): Streptomyces sp. ADI95-16 chromosome, complete ... (26% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r26c3_7.svg">
        NZ_CP021748 (7086813-7200542): Streptomyces alboflavus strain MDJK44 chromoso... (15% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r26c3_8.svg">
        NZ_CP048641 (2265713-2350147): Streptomyces aureoverticillatus strain HN6 chr... (26% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r26c3_9.svg">
        NC_021252 (5896779-6053955): Amycolatopsis keratiniphila, complete sequence (10% of genes show similarity), NRPS,NRPS-like,T1PKS,T3PKS,lanthidin
      </option>
      
      <option value = "svg/clusterblast_r26c3_10.svg">
        NZ_CP017780 (7027475-7048162): Amycolatopsis sp. BJA-103 chromosome, complete... (42% of genes show similarity), CDPS
      </option>
      
    </select>
    <button id= "clusterblast-r26c3-download">Download graphic</button>
  </div>
  <div id="clusterblast-r26c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r26c3-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-112"></div> <span class="help-tooltip" id="cb-known-help-112">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r26c3-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r26c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r26c3_1.svg">
        coelibactin
      </option>
      
      <option value = "svg/knownclusterblast_r26c3_2.svg">
        thiazostatin / watasemycin A / watasemycin B / 2-hydroxyphenylthiazoline enantiopyochelin / isopyochelin
      </option>
      
      <option value = "svg/knownclusterblast_r26c3_3.svg">
        amychelin
      </option>
      
      <option value = "svg/knownclusterblast_r26c3_4.svg">
        gobichelin A / gobichelin B
      </option>
      
      <option value = "svg/knownclusterblast_r26c3_5.svg">
        mycobactin
      </option>
      
      <option value = "svg/knownclusterblast_r26c3_6.svg">
        celesticetin
      </option>
      
      <option value = "svg/knownclusterblast_r26c3_7.svg">
        tetronasin
      </option>
      
      <option value = "svg/knownclusterblast_r26c3_8.svg">
        paramagnetoquinone 1 / paramagnetoquinone 2
      </option>
      
      <option value = "svg/knownclusterblast_r26c3_9.svg">
        scleric acid
      </option>
      
      <option value = "svg/knownclusterblast_r26c3_10.svg">
        SW-163C / UK-63598 / SW-163E / SW-163F / SW-163G
      </option>
      
    </select>
    <button id="knownclusterblast-r26c3-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r26c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r26c3-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-113"></div> <span class="help-tooltip" id="cb-sub-help-113">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r26c3-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-114"></div> <span class="help-tooltip" id="MIBiG-help-114">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r26c3-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r26c3-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r26c3"></div>
    <div id="comparison-MIBiG-r26c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">CDPS</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001405.1: 2447653-2507157">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001405/index.html#r1c1" target="_new">BGC0001405.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.53</td>
        <td>Polyketide</td>
        
        <td>chaetoviridin E, 11-epichaetomugilin A</td>
        
        
        <td>Chaetomium globosum CBS 148.51</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.52, order:0.33, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.44</td>
        <td>Other</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001292.1: 5468-37257">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001292/index.html#r1c1" target="_new">BGC0001292.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.50, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.43</td>
        <td>Other</td>
        
        <td>A33853</td>
        
        
        <td>Streptomyces sp. NRRL 12068</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001838.1: 118-30958">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001838/index.html#r1c1" target="_new">BGC0001838.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.35 (id:0.45, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.45, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.42</td>
        <td>NRP</td>
        
        <td>WLIP</td>
        
        
        <td>Pseudomonas putida</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000463.1: 4623-47830">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000463/index.html#r1c1" target="_new">BGC0000463.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:0.57, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.33 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.39 (id:0.43, order:0.39, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
         
        
        <td>0.41</td>
        <td>NRP</td>
        
        <td>xantholysin A, xantholysin B, xantholysin C</td>
        
        
        <td>Pseudomonas putida</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001030.1: 0-22759">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001030/index.html#r1c1" target="_new">BGC0001030.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.36 (id:0.44, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.41</td>
        <td>NRP, Polyketide</td>
        
        <td>ochratoxin A</td>
        
        
        <td>Penicillium nordicum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.31 (id:0.44, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
        <td>0.41</td>
        <td>NRP</td>
        
        <td>exochelin</td>
        
        
        <td>Mycolicibacterium smegmatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000134.1: 1391-30251">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000134/index.html#r1c1" target="_new">BGC0000134.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.45, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.37 (id:0.45, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.40</td>
        <td>Polyketide</td>
        
        <td>radicicol</td>
        
        
        <td>Pochonia chlamydosporia</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001647.1: 413229-431538">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001647/index.html#r1c1" target="_new">BGC0001647.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.45, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.36 (id:0.45, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.40</td>
        <td>Polyketide</td>
        
        <td>lagunapyrone A, lagunapyrone B, lagunapyrone C</td>
        
        
        <td>Streptomyces sp. MP131-18</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.43, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.33 (id:0.42, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
         
        
        <td>0.40</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r26c3-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>auriculamide</td>
      <td>Herpetosiphon aurantiacus DSM 785</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000373.1: 6204479-6222797">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000373/index.html#r1c1" target="_new">BGC0000373.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.49, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>holomycin</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002091.1: 3593-21565">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002091/index.html#r1c1" target="_new">BGC0002091.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>holomycin</td>
      <td>Yersinia ruckeri ATCC 29473</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000294.1: 181-31792">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000294/index.html#r1c1" target="_new">BGC0000294.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>acinetobactin</td>
      <td>Acinetobacter baumannii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.43, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001292.1: 5468-37257">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001292/index.html#r1c1" target="_new">BGC0001292.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.50, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>A33853</td>
      <td>Streptomyces sp. NRRL 12068</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343.1: 0-34054">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>enterobactin</td>
      <td>Pseudomonas sp. J465</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000935.1: 151-29847">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000935/index.html#r1c1" target="_new">BGC0000935.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.51, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Other, Polyketide,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
      <td>esmeraldin</td>
      <td>Streptomyces antibioticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester</td>
      <td>Streptomyces sp. RM-5-8</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r26c3-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-115"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-115">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r26c3-pfam-details-svg" data-filename="c00026_CC55NOD.._r26c3_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r26c3-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r26c3-nrps_pks_products" data-name="r26c3-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r26c3-nrps_pks" data-name="r26c3-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r26c3-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-109"></div> <span class="help-tooltip" id="nrps-structures-help-109">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 5, location 120985 - 222233:
          <div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r26c3-5" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(CS)C(=O)NC(CS)C(=O)C([*])C(-O)"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X - cys) + (cys) + (pk)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,cys&amp;nrps2=cys&amp;nrps3=pk" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,cys*&amp;nrps2=cys*&amp;nrps3=pk*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r26c3-5-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(CS)C(=O)NC(CS)C(=O)C([*])C(-O)"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 6, location 120985 - 173243:
          <div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r26c3-6" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(CS)C(=O)NC(CS)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X - cys) + (cys)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,cys&amp;nrps2=cys" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,cys*&amp;nrps2=cys*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r26c3-6-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(CS)C(=O)NC(CS)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 8, location 177033 - 222233:
          <div class="collapser collapser-target-CC8 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r26c3-8" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pk" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pk*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r26c3-8-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r26c3-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-110"></div> <span class="help-tooltip" id="nrps-monomers-help-110">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds140494_148786</span></strong>: X - cys
      <div class="collapser collapser-target-cds140494_148786 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,cys" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,cys*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (575..966)</strong>: X
          <div class="collapser collapser-target-cds140494_148786 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds140494_148786-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLYsMglIhK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>cys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2175..2574)</strong>: cys
          <div class="collapser collapser-target-cds140494_148786 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  cys
              <div class="collapser collapser-target-cds140494_148786-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Small clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Single AA prediction:</dt>
   <dd>cys</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLFNLSLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>cys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds148782_154233</span></strong>: cys
      <div class="collapser collapser-target-cds148782_154233 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (525..921)</strong>: cys
          <div class="collapser collapser-target-cds148782_154233 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  cys
              <div class="collapser collapser-target-cds148782_154233-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Small clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Single AA prediction:</dt>
   <dd>cys</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLwNLSLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>cys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds157847_159464</span></strong>: dhb
      <div class="collapser collapser-target-cds157847_159464 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=diOH-Bz" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=diOH-Bz*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (32..437)</strong>: dhb
          <div class="collapser collapser-target-cds157847_159464 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  dhb
              <div class="collapser collapser-target-cds157847_159464-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>dhb, sal</dd>
   <dt>Small clusters prediction:</dt>
   <dd>dhb, sal</dd>
   <dt>Single AA prediction:</dt>
   <dd>dhb</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">tmPAQGVLCK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>dhb</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds183878_185510</span></strong>: X
      <div class="collapser collapser-target-cds183878_185510 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (38..448)</strong>: X
          <div class="collapser collapser-target-cds183878_185510 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds183878_185510-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">gSgLagvnCK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>his</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>40% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds196867_204166</span></strong>: pk
      <div class="collapser collapser-target-cds196867_204166 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (527..818)</strong>: pk
          <div class="collapser collapser-target-cds196867_204166 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds196867_204166-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
<dd>Propionyl-CoA: 66.7%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds196867_204166-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 100.1</dt>

  <dd></dd><dt>Malonyl-CoA: 92.4</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 89.4</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 60.7</dt>

  <dd></dd><dt>Propionyl-CoA: 50.0</dt>

  <dd></dd><dt>trans-1,2-CPDA: 45.4</dt>

  <dd></dd><dt>Acetyl-CoA: 37.1</dt>

  <dd></dd><dt>Benzoyl-CoA: 35.6</dt>

  <dd></dd><dt>Isobutyryl-CoA: 21.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 21.4</dt>

  <dd></dd><dt>fatty_acid: 21.1</dt>

  <dd></dd><dt>inactive: 19.3</dt>

  <dd></dd><dt>CHC-CoA: 18.2</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 15.5</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r26c4' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00026_CC55NOD.. - Region 4 - NRPS-like,PKS-like,T1PKS,arylpolyene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-254"></div> <span class="help-tooltip" id="region_svg-help-254">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00026_CC55NOD...region004.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r26c4-svg" data-filename="c00026_CC55NOD.._r26c4.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 493,491 - 592,270 nt. (total: 98,780 nt)
        
        <a class="cluster-rules-header" id="r26c4-rules-header" href="#r26c4">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r26c4-rules">PKS-like: (t2pks2 or ksIII)<br>arylpolyene: (APE_KS1 or APE_KS2)<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r26c4-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-255"></div> <span class="help-tooltip" id="focus-panel-help-255">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r26c4">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r26c4-nrps_pks" data-name="r26c4-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r26c4-clusterblast" data-name="r26c4-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r26c4-knownclusterblast" data-name="r26c4-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r26c4-subclusterblast" data-name="r26c4-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r26c4-MIBiG-cluster-compare" data-name="r26c4-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r26c4-pfam-details" data-name="r26c4-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r26c4-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-116"></div> <span class="help-tooltip" id="nrps-domain-help-116">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r26c4-details-svg" data-filename="c00026_CC55NOD.._r26c4_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r26c4-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r26c4-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-119"></div> <span class="help-tooltip" id="cb-general-help-119">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r26c4-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r26c4_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r26c4_1.svg">
        NZ_CP054925 (2269688-2365233): Amycolatopsis sp. Hca4 chromosome, complete ge... (27% of genes show similarity), PKS-like,T1PKS,amglyccycl,betalactone
      </option>
      
      <option value = "svg/clusterblast_r26c4_2.svg">
        NZ_FNCC01000023 (11440-65126): Lechevalieria fradiae strain CGMCC 4.3506, who... (40% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r26c4_3.svg">
        NZ_FOFR01000033 (1331-76006): Lechevalieria xinjiangensis strain CGMCC 4.3525... (38% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r26c4_4.svg">
        NZ_FWXV01000001 (119666-310509): Kibdelosporangium aridum strain DSM 43828, w... (20% of genes show similarity), PKS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r26c4_5.svg">
        NZ_KQ948552 (290861-419345): Streptomyces longwoodensis strain DSM 41677 PRJN... (26% of genes show similarity), PKS-like,T1PKS,T3PKS,ectoine
      </option>
      
      <option value = "svg/clusterblast_r26c4_6.svg">
        NZ_CP015726 (1522234-1730838): Streptomyces sp. RTd22 chromosome, complete ge... (16% of genes show similarity), NRPS,PKS-like,T1PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r26c4_7.svg">
        NZ_FOZX01000007 (91981-279762): Saccharopolyspora flava strain DSM 44771, who... (12% of genes show similarity), NRPS,T1PKS,T3PKS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r26c4_8.svg">
        NZ_MQUH01000016 (702507-806193): Streptomyces sp. M1013 M1013 C16, whole geno... (15% of genes show similarity), PKS-like,T1PKS,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r26c4_9.svg">
        NZ_CP034352 (1037-312401): Streptomyces sp. W1SF4 plasmid p2, complete sequence (16% of genes show similarity), NRPS,NRPS-like,T1PKS,T2PKS,butyrolactone,melanin
      </option>
      
      <option value = "svg/clusterblast_r26c4_10.svg">
        NZ_CP048397 (156576-320082): Streptomyces sp. S4.7 chromosome, complete genome (13% of genes show similarity), NRPS-like,T1PKS,T3PKS,arylpolyene,butyrolactone,phenazine
      </option>
      
    </select>
    <button id= "clusterblast-r26c4-download">Download graphic</button>
  </div>
  <div id="clusterblast-r26c4-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r26c4-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-120"></div> <span class="help-tooltip" id="cb-known-help-120">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r26c4-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r26c4_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r26c4_1.svg">
        virginiamycin S1
      </option>
      
      <option value = "svg/knownclusterblast_r26c4_2.svg">
        naphthomycin A
      </option>
      
      <option value = "svg/knownclusterblast_r26c4_3.svg">
        natamycin
      </option>
      
      <option value = "svg/knownclusterblast_r26c4_4.svg">
        kijanimicin
      </option>
      
      <option value = "svg/knownclusterblast_r26c4_5.svg">
        branched-chain fatty acids
      </option>
      
      <option value = "svg/knownclusterblast_r26c4_6.svg">
        ECO-02301
      </option>
      
      <option value = "svg/knownclusterblast_r26c4_7.svg">
        X-14547
      </option>
      
      <option value = "svg/knownclusterblast_r26c4_8.svg">
        selvamicin
      </option>
      
      <option value = "svg/knownclusterblast_r26c4_9.svg">
        coelimycin P1
      </option>
      
      <option value = "svg/knownclusterblast_r26c4_10.svg">
        chaxamycin A / chaxamycin B / chaxamycin C / chaxamycin D
      </option>
      
    </select>
    <button id="knownclusterblast-r26c4-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r26c4-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r26c4-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-121"></div> <span class="help-tooltip" id="cb-sub-help-121">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r26c4-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r26c4_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r26c4_1.svg">
        AL939125_c1: prodigiosin 2-undecylpyrrole (25% of genes show similarity), 2-undecylpyrrole
      </option>
      
      <option value = "svg/subclusterblast_r26c4_2.svg">
        FN563137_c1: pristinamycin L-phenylglycine (40% of genes show similarity), PG
      </option>
      
    </select>
    <button id="subclusterblast-r26c4-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r26c4-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r26c4-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-122"></div> <span class="help-tooltip" id="MIBiG-help-122">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r26c4-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r26c4-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r26c4"></div>
    <div id="comparison-MIBiG-r26c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">PKS-like</span></th>
         
         <th><span class="cc-heat-product">arylpolyene</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>1.11</td>
        <td>Terpene, Polyketide</td>
        
        <td>pyripyropene A</td>
        
        
        <td>unidentified unclassified sequences.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>1.06</td>
        <td>Polyketide</td>
        
        <td>asperlactone</td>
        
        
        <td>Aspergillus ochraceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>1.04</td>
        <td>Polyketide</td>
        
        <td>(-)-Mellein</td>
        
        
        <td>Parastagonospora nodorum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.45, order:0.34, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>1.03</td>
        <td>Other</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000065.1: 315-41590">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000065/index.html#r1c1" target="_new">BGC0000065.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>1.00</td>
        <td>Polyketide</td>
        
        <td>rustmicin</td>
        
        
        <td>Streptomyces galbus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.45, order:0.34, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.99</td>
        <td>NRP, Polyketide</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000170.1: 25032-49030">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000170/index.html#r1c1" target="_new">BGC0000170.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.98</td>
        <td>Polyketide</td>
        
        <td>yanuthone D</td>
        
        
        <td>Aspergillus niger ATCC 1015</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.95</td>
        <td>Polyketide</td>
        
        <td>esperamicin</td>
        
        
        <td>Actinomadura verrucosospora</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000003.1: 12401-36790">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000003/index.html#r1c1" target="_new">BGC0000003.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.92</td>
        <td>Polyketide</td>
        
        <td>AF-toxin</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001281.1: 8011-344234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001281/index.html#r1c1" target="_new">BGC0001281.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.43, order:0.25, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.90</td>
        <td>Polyketide</td>
        
        <td>ustilagic acid</td>
        
        
        <td>Ustilago maydis 521</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r26c4-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001534.1: 799-5264">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001534/index.html#r1c1" target="_new">BGC0001534.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.53, order:1.00, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>Other</td>
      <td>branched-chain fatty acids</td>
      <td>Streptomyces filamentosus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001535.1: 334-3938">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001535/index.html#r1c1" target="_new">BGC0001535.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.48, order:1.00, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>Other</td>
      <td>branched-chain fatty acids</td>
      <td>Streptomyces filamentosus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001075.1: 2658-14492">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001075/index.html#r1c1" target="_new">BGC0001075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Terpene, Polyketide</td>
      <td>aurachin RE</td>
      <td>Rhodococcus erythropolis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Aspergillus terreus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275.1: 1063-6614">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Glarea lozoyensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Polyketide</td>
      <td>asperlactone</td>
      <td>Aspergillus ochraceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Polyketide</td>
      <td>(-)-Mellein</td>
      <td>Parastagonospora nodorum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000170.1: 25032-49030">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000170/index.html#r1c1" target="_new">BGC0000170.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>yanuthone D</td>
      <td>Aspergillus niger ATCC 1015</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000160.1: 89033-116126">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000160/index.html#r1c1" target="_new">BGC0000160.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Polyketide</td>
      <td>terreic acid</td>
      <td>Aspergillus terreus NIH2624</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000068.1: 1-27296">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000068/index.html#r1c1" target="_new">BGC0000068.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Polyketide</td>
      <td>geldanamycin</td>
      <td>Streptomyces hygroscopicus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r26c4-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-123"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-123">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r26c4-pfam-details-svg" data-filename="c00026_CC55NOD.._r26c4_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r26c4-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r26c4-nrps_pks_products" data-name="r26c4-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r26c4-nrps_pks" data-name="r26c4-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r26c4-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-117"></div> <span class="help-tooltip" id="nrps-structures-help-117">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 9, location 493490 - 592270:
          <div class="collapser collapser-target-CC9 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r26c4-9" class="smiles-canvas" data-smiles="C(C)C(=O)C=CC=CCC(=O)C([*])C(-O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(Me-mal - ccmal - ccmal) + (mal) + (pk) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-mal,ccmal,ccmal&amp;nrps2=mal&amp;nrps3=pk&amp;nrps4=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-mal*,ccmal*,ccmal*&amp;nrps2=mal*&amp;nrps3=pk*&amp;nrps4=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r26c4-9-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C(C)C(=O)C=CC=CCC(=O)C([*])C(-O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 12, location 515853 - 585696:
          <div class="collapser collapser-target-CC12 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r26c4-12" class="smiles-canvas" data-smiles="C(C)C(=O)C=CC=CCC(=O)C([*])C(-O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(Me-mal - ccmal - ccmal) + (mal) + (pk) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-mal,ccmal,ccmal&amp;nrps2=mal&amp;nrps3=pk&amp;nrps4=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-mal*,ccmal*,ccmal*&amp;nrps2=mal*&amp;nrps3=pk*&amp;nrps4=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r26c4-12-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C(C)C(=O)C=CC=CCC(=O)C([*])C(-O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 13, location 552903 - 592270:
          <div class="collapser collapser-target-CC13 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r26c4-13" class="smiles-canvas" data-smiles="CC(=O)C([*])C(-O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(mal) + (pk) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal&amp;nrps2=pk&amp;nrps3=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal*&amp;nrps2=pk*&amp;nrps3=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r26c4-13-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(=O)C([*])C(-O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r26c4-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-118"></div> <span class="help-tooltip" id="nrps-monomers-help-118">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds510352_511987</span></strong>: pk
      <div class="collapser collapser-target-cds510352_511987 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>CAL_domain (34..463)</strong>: pk
          <div class="collapser collapser-target-cds510352_511987 collapser-level-candidate"> </div><div class="collapser-content">
          
              minowa:  AHBA
              <div class="collapser collapser-target-cds510352_511987-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>AHBA: 44.7</dt>

  <dd></dd><dt>NH2: 19.6</dt>

  <dd></dd><dt>shikimic_acid: 12.9</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds517667_518546</span></strong>: mal
      <div class="collapser collapser-target-cds517667_518546 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (5..282)</strong>: mal
          <div class="collapser collapser-target-cds517667_518546 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds517667_518546-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>inactive: 70.8%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds517667_518546-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 109.9</dt>

  <dd></dd><dt>inactive: 44.7</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 30.6</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 16.7</dt>

  <dd></dd><dt>Propionyl-CoA: 14.1</dt>

  <dd></dd><dt>Isobutyryl-CoA: 8.6</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds534646_548185</span></strong>: mmal - mal - mal
      <div class="collapser collapser-target-cds534646_548185 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (538..831)</strong>: mmal
          <div class="collapser collapser-target-cds534646_548185 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds534646_548185-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds534646_548185-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 192.8</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 174.2</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 155.3</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 115.9</dt>

  <dd></dd><dt>trans-1,2-CPDA: 101.5</dt>

  <dd></dd><dt>Propionyl-CoA: 96.2</dt>

  <dd></dd><dt>CHC-CoA: 87.3</dt>

  <dd></dd><dt>Malonyl-CoA: 85.1</dt>

  <dd></dd><dt>Isobutyryl-CoA: 61.8</dt>

  <dd></dd><dt>Benzoyl-CoA: 54.4</dt>

  <dd></dd><dt>Acetyl-CoA: 32.1</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 31.1</dt>

  <dd></dd><dt>inactive: 23.7</dt>

  <dd></dd><dt>fatty_acid: 15.7</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (1507..1724)</strong>: mal
          <div class="collapser collapser-target-cds534646_548185 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds534646_548185-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 66.7%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds534646_548185-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 70.2</dt>

  <dd></dd><dt>inactive: 23.8</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 23.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 20.7</dt>

  <dd></dd><dt>Benzoyl-CoA: 16.4</dt>

  <dd></dd><dt>CHC-CoA: 13.8</dt>

  <dd></dd><dt>Isobutyryl-CoA: 12.8</dt>

  <dd></dd><dt>Acetyl-CoA: 9.9</dt>

  <dd></dd><dt>Propionyl-CoA: 7.0</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (3103..3394)</strong>: mal
          <div class="collapser collapser-target-cds534646_548185 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds534646_548185-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 91.7%</dd>
<dd>inactive: 66.7%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds534646_548185-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 190.6</dt>

  <dd></dd><dt>inactive: 75.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 61.7</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 55.9</dt>

  <dd></dd><dt>Propionyl-CoA: 37.6</dt>

  <dd></dd><dt>Acetyl-CoA: 23.8</dt>

  <dd></dd><dt>Isobutyryl-CoA: 19.9</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 19.3</dt>

  <dd></dd><dt>CHC-CoA: 15.2</dt>

  <dd></dd><dt>fatty_acid: 14.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 12.8</dt>

  <dd></dd><dt>Benzoyl-CoA: 9.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 3.6</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds553448_554948</span></strong>: mal
      <div class="collapser collapser-target-cds553448_554948 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (11..304)</strong>: mal
          <div class="collapser collapser-target-cds553448_554948 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds553448_554948-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 58.3%</dd>
<dd>Propionyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds553448_554948-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 50.8</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 37.7</dt>

  <dd></dd><dt>Benzoyl-CoA: 30.4</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 28.6</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 25.7</dt>

  <dd></dd><dt>fatty_acid: 20.8</dt>

  <dd></dd><dt>trans-1,2-CPDA: 14.4</dt>

  <dd></dd><dt>Propionyl-CoA: 12.1</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 9.6</dt>

  <dd></dd><dt>Isobutyryl-CoA: 8.8</dt>

  <dd></dd><dt>Acetyl-CoA: 8.5</dt>

  <dd></dd><dt>CHC-CoA: 4.4</dt>

  <dd></dd><dt>inactive: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds560786_565988</span></strong>: pk
      <div class="collapser collapser-target-cds560786_565988 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (541..833)</strong>: pk
          <div class="collapser collapser-target-cds560786_565988 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds560786_565988-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 87.5%</dd>
<dd>Methylmalonyl-CoA: 75.0%</dd>
<dd>Ethylmalonyl-CoA: 66.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds560786_565988-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 110.5</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 91.5</dt>

  <dd></dd><dt>Malonyl-CoA: 81.9</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 81.3</dt>

  <dd></dd><dt>trans-1,2-CPDA: 69.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 55.4</dt>

  <dd></dd><dt>Isobutyryl-CoA: 50.7</dt>

  <dd></dd><dt>Propionyl-CoA: 45.7</dt>

  <dd></dd><dt>CHC-CoA: 37.6</dt>

  <dd></dd><dt>Acetyl-CoA: 31.1</dt>

  <dd></dd><dt>inactive: 26.8</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 26.6</dt>

  <dd></dd><dt>Benzoyl-CoA: 19.7</dt>

  <dd></dd><dt>fatty_acid: 16.9</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds570408_572907</span></strong>: X
      <div class="collapser collapser-target-cds570408_572907 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (25..386)</strong>: X
          <div class="collapser collapser-target-cds570408_572907 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds570408_572907-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">g-hcrstaGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>glu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>20% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds578998_580297</span></strong>: X
      <div class="collapser collapser-target-cds578998_580297 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (77..338)</strong>: X
          <div class="collapser collapser-target-cds578998_580297 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds578998_580297-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">tGGMiGigpK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r26c5' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00026_CC55NOD.. - Region 5 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-256"></div> <span class="help-tooltip" id="region_svg-help-256">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00026_CC55NOD...region005.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r26c5-svg" data-filename="c00026_CC55NOD.._r26c5.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 678,213 - 700,280 nt. (total: 22,068 nt)
        
        <a class="cluster-rules-header" id="r26c5-rules-header" href="#r26c5">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r26c5-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r26c5-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-257"></div> <span class="help-tooltip" id="focus-panel-help-257">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r26c5">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r26c5-clusterblast" data-name="r26c5-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r26c5-knownclusterblast" data-name="r26c5-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r26c5-subclusterblast" data-name="r26c5-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r26c5-MIBiG-cluster-compare" data-name="r26c5-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r26c5-pfam-details" data-name="r26c5-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r26c5-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-124"></div> <span class="help-tooltip" id="cb-general-help-124">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r26c5-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r26c5_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r26c5_1.svg">
        NZ_JAAMPJ010000003 (158881-180979): Lentzea sp. NEAU-D13 Scaffold3, whole gen... (77% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r26c5_2.svg">
        NZ_QQAU01000020 (22083-44166): Lentzea flaviverrucosa strain DSM 44664 Ga0244... (77% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r26c5_3.svg">
        NZ_FOFR01000013 (110121-132234): Lechevalieria xinjiangensis strain CGMCC 4.3... (82% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r26c5_4.svg">
        NZ_FNCC01000015 (71788-93868): Lechevalieria fradiae strain CGMCC 4.3506, who... (77% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r26c5_5.svg">
        NZ_CP016793 (5812946-5833518): Lentzea guizhouensis strain DHS C013 chromosom... (81% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r26c5_6.svg">
        NZ_FOFV01000005 (294485-316550): Lentzea albida strain DSM 44437, whole genom... (81% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r26c5_7.svg">
        NZ_FNET01000015 (39089-61214): Lentzea violacea strain DSM 44796, whole genom... (78% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r26c5_8.svg">
        NZ_FOYL01000013 (100546-122671): Lentzea waywayandensis strain DSM 44232, who... (83% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r26c5_9.svg">
        NZ_QGHB01000003 (198283-219148): Lechevalieria deserti strain DSM 45480 Ga021... (77% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r26c5_10.svg">
        NZ_QLTT01000001 (1625458-1647538): Lechevalieria atacamensis strain DSM 45479... (77% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r26c5-download">Download graphic</button>
  </div>
  <div id="clusterblast-r26c5-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r26c5-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-125"></div> <span class="help-tooltip" id="cb-known-help-125">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r26c5-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r26c5_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r26c5_1.svg">
        geosmin
      </option>
      
      <option value = "svg/knownclusterblast_r26c5_2.svg">
        geosmin
      </option>
      
      <option value = "svg/knownclusterblast_r26c5_3.svg">
        xiamycin
      </option>
      
      <option value = "svg/knownclusterblast_r26c5_4.svg">
        xiamycin A
      </option>
      
      <option value = "svg/knownclusterblast_r26c5_5.svg">
        platensimycin / platencin
      </option>
      
      <option value = "svg/knownclusterblast_r26c5_6.svg">
        thioplatensimycin / thioplatencin
      </option>
      
      <option value = "svg/knownclusterblast_r26c5_7.svg">
        prejadomycin / rabelomycin / gaudimycin C / gaudimycin D / UWM6 / gaudimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r26c5_8.svg">
        lavendiol
      </option>
      
      <option value = "svg/knownclusterblast_r26c5_9.svg">
        lavendiol
      </option>
      
      <option value = "svg/knownclusterblast_r26c5_10.svg">
        2-methylisoborneol
      </option>
      
    </select>
    <button id="knownclusterblast-r26c5-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r26c5-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r26c5-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-126"></div> <span class="help-tooltip" id="cb-sub-help-126">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r26c5-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-127"></div> <span class="help-tooltip" id="MIBiG-help-127">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r26c5-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r26c5-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r26c5"></div>
    <div id="comparison-MIBiG-r26c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.76 (id:0.53, order:0.83, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
         
        
        <td>0.46</td>
        <td>Terpene</td>
        
        <td>geosmin</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001181.1: 6666218-6668399">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001181/index.html#r1c1" target="_new">BGC0001181.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.74 (id:0.54, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
         
        
        <td>0.43</td>
        <td>Terpene</td>
        
        <td>geosmin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001361.1: 2257657-2262237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001361/index.html#r1c1" target="_new">BGC0001361.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>Terpene</td>
        
        <td>sodorifen</td>
        
        
        <td>Serratia plymuthica 4Rx13</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000657.1: 643-5774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000657/index.html#r1c1" target="_new">BGC0000657.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.52, order:0.36, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>2-methylisoborneol</td>
        
        
        <td>Pseudanabaena sp. dqh15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>raimonol</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514.1: 0-37469">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.19</td>
        <td>Other</td>
        
        <td>aristeromycin</td>
        
        
        <td>Streptomyces citricolor</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.50, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Terpene</td>
        
        <td>pentalenolactone</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000386.1: 2748378-2778190">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000386/index.html#r1c1" target="_new">BGC0000386.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>NRP</td>
        
        <td>malleobactin A, malleobactin B, malleobactin C, malleobactin D</td>
        
        
        <td>Burkholderia thailandensis E264</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000774.1: 133-37753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000774/index.html#r1c1" target="_new">BGC0000774.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Saccharide</td>
        
        <td>lipopolysaccharide</td>
        
        
        <td>Xanthomonas campestris pv. campestris</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000772.1: 960-13404">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000772/index.html#r1c1" target="_new">BGC0000772.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Saccharide</td>
        
        <td>lipopolysaccharide</td>
        
        
        <td>Brucella melitensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r26c5-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.76 (id:0.53, order:0.83, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
      <td>0.76</td>
      <td>Terpene</td>
      <td>geosmin</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001181.1: 6666218-6668399">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001181/index.html#r1c1" target="_new">BGC0001181.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.74 (id:0.54, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
      <td>0.74</td>
      <td>Terpene</td>
      <td>geosmin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001361.1: 2257657-2262237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001361/index.html#r1c1" target="_new">BGC0001361.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Terpene</td>
      <td>sodorifen</td>
      <td>Serratia plymuthica 4Rx13</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000657.1: 643-5774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000657/index.html#r1c1" target="_new">BGC0000657.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.52, order:0.36, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>2-methylisoborneol</td>
      <td>Pseudanabaena sp. dqh15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>raimonol</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514.1: 0-37469">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Other</td>
      <td>aristeromycin</td>
      <td>Streptomyces citricolor</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.50, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000386.1: 2748378-2778190">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000386/index.html#r1c1" target="_new">BGC0000386.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>NRP</td>
      <td>malleobactin A, malleobactin B, malleobactin C, malleobactin D</td>
      <td>Burkholderia thailandensis E264</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000774.1: 133-37753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000774/index.html#r1c1" target="_new">BGC0000774.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Saccharide</td>
      <td>lipopolysaccharide</td>
      <td>Xanthomonas campestris pv. campestris</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000772.1: 960-13404">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000772/index.html#r1c1" target="_new">BGC0000772.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Saccharide</td>
      <td>lipopolysaccharide</td>
      <td>Brucella melitensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r26c5-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-128"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-128">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r26c5-pfam-details-svg" data-filename="c00026_CC55NOD.._r26c5_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r26c5-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r26c6' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00026_CC55NOD.. - Region 6 - NRPS,T1PKS,amglyccycl,hglE-KS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-258"></div> <span class="help-tooltip" id="region_svg-help-258">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00026_CC55NOD...region006.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r26c6-svg" data-filename="c00026_CC55NOD.._r26c6.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 702,309 - 770,886 nt. (total: 68,578 nt)
        
        <a class="cluster-rules-header" id="r26c6-rules-header" href="#r26c6">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r26c6-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))<br>hglE-KS: (hglE or hglD)<br>amglyccycl: (strH or strK1 or strK2 or NeoL or DOIS or ValA or SpcFG or SpcDK_glyc or salQ)</div>
      
      <div class="region-svg-container">
       <div id='r26c6-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-259"></div> <span class="help-tooltip" id="focus-panel-help-259">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r26c6">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r26c6-nrps_pks" data-name="r26c6-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r26c6-clusterblast" data-name="r26c6-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r26c6-knownclusterblast" data-name="r26c6-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r26c6-subclusterblast" data-name="r26c6-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r26c6-MIBiG-cluster-compare" data-name="r26c6-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r26c6-pfam-details" data-name="r26c6-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r26c6-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-129"></div> <span class="help-tooltip" id="nrps-domain-help-129">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r26c6-details-svg" data-filename="c00026_CC55NOD.._r26c6_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r26c6-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r26c6-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-132"></div> <span class="help-tooltip" id="cb-general-help-132">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r26c6-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r26c6_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r26c6_1.svg">
        NZ_CP054925 (2269688-2365233): Amycolatopsis sp. Hca4 chromosome, complete ge... (33% of genes show similarity), PKS-like,T1PKS,amglyccycl,betalactone
      </option>
      
      <option value = "svg/clusterblast_r26c6_2.svg">
        NZ_CP016174 (2217344-2387536): Amycolatopsis orientalis strain B-37 chromosom... (15% of genes show similarity), NRPS,NRPS-like,T1PKS,amglyccycl,indole
      </option>
      
      <option value = "svg/clusterblast_r26c6_3.svg">
        NZ_FOFV01000002 (97834-147982): Lentzea albida strain DSM 44437, whole genome... (34% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r26c6_4.svg">
        NZ_CP016793 (6470590-6546084): Lentzea guizhouensis strain DHS C013 chromosom... (34% of genes show similarity), T1PKS,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r26c6_5.svg">
        NZ_QQAU01000001 (802885-853036): Lentzea flaviverrucosa strain DSM 44664 Ga02... (37% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r26c6_6.svg">
        NZ_JAAMPJ010000019 (100615-150739): Lentzea sp. NEAU-D13 Scaffold19, whole ge... (35% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r26c6_7.svg">
        NZ_QGHB01000006 (35735-85892): Lechevalieria deserti strain DSM 45480 Ga02156... (35% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r26c6_8.svg">
        NZ_FOFR01000024 (34473-110156): Lechevalieria xinjiangensis strain CGMCC 4.35... (34% of genes show similarity), T1PKS,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r26c6_9.svg">
        NZ_FOFR01000033 (1331-76006): Lechevalieria xinjiangensis strain CGMCC 4.3525... (50% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r26c6_10.svg">
        NZ_QLTT01000003 (160374-210531): Lechevalieria atacamensis strain DSM 45479 G... (28% of genes show similarity), hglE-KS
      </option>
      
    </select>
    <button id= "clusterblast-r26c6-download">Download graphic</button>
  </div>
  <div id="clusterblast-r26c6-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r26c6-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-133"></div> <span class="help-tooltip" id="cb-known-help-133">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r26c6-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r26c6_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r26c6_1.svg">
        stigmatellin
      </option>
      
      <option value = "svg/knownclusterblast_r26c6_2.svg">
        herboxidiene
      </option>
      
      <option value = "svg/knownclusterblast_r26c6_3.svg">
        pellasoren
      </option>
      
      <option value = "svg/knownclusterblast_r26c6_4.svg">
        phenalamide
      </option>
      
      <option value = "svg/knownclusterblast_r26c6_5.svg">
        myxochromide D
      </option>
      
      <option value = "svg/knownclusterblast_r26c6_6.svg">
        phenalamide A2
      </option>
      
      <option value = "svg/knownclusterblast_r26c6_7.svg">
        myxalamid
      </option>
      
      <option value = "svg/knownclusterblast_r26c6_8.svg">
        nannocystin a
      </option>
      
      <option value = "svg/knownclusterblast_r26c6_9.svg">
        aurafuron A
      </option>
      
      <option value = "svg/knownclusterblast_r26c6_10.svg">
        ambruticin
      </option>
      
    </select>
    <button id="knownclusterblast-r26c6-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r26c6-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r26c6-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-134"></div> <span class="help-tooltip" id="cb-sub-help-134">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r26c6-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-135"></div> <span class="help-tooltip" id="MIBiG-help-135">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r26c6-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r26c6-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r26c6"></div>
    <div id="comparison-MIBiG-r26c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">hglE-KS</span></th>
         
         <th><span class="cc-heat-product">amglyccycl</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000701.1: 2122-5539">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000701/index.html#r1c1" target="_new">BGC0000701.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.51, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.51, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.51, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.51, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>1.13</td>
        <td>Saccharide</td>
        
        <td>jinggangmycin</td>
        
        
        <td>Streptomyces hygroscopicus subsp. yingchengensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.51, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.51, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.51, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.51, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>1.12</td>
        <td>Other</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.48, order:1.00, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.48, order:1.00, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.48, order:1.00, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.44, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>1.09</td>
        <td>Polyketide</td>
        
        <td>oronofacic acid</td>
        
        
        <td>Pseudomonas syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.51, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.51, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.51, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.51, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>1.04</td>
        <td>NRP, Polyketide</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001201.1: 11915724-11918015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001201/index.html#r1c1" target="_new">BGC0001201.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.95</td>
        <td>Terpene</td>
        
        <td>eremophilene</td>
        
        
        <td>Sorangium cellulosum So ce56</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.88</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000723.1: 1114-36970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000723/index.html#r1c1" target="_new">BGC0000723.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.51, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.51, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.51, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.51, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.87</td>
        <td>Saccharide</td>
        
        <td>Î²-D-galactosylvalidoxylamine-A</td>
        
        
        <td>Streptomyces hygroscopicus subsp. limoneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001292.1: 5468-37257">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001292/index.html#r1c1" target="_new">BGC0001292.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.47, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.85</td>
        <td>Other</td>
        
        <td>A33853</td>
        
        
        <td>Streptomyces sp. NRRL 12068</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000861.1: 0-20498">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000861/index.html#r1c1" target="_new">BGC0000861.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.84</td>
        <td>Other</td>
        
        <td>eicosapentaenoic acid</td>
        
        
        <td>Shewanella sp. BR-2</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000863.1: 1224-16560">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000863/index.html#r1c1" target="_new">BGC0000863.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:0.57, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.83</td>
        <td>Other</td>
        
        <td>eicosapentaenoic acid</td>
        
        
        <td>Pseudoalteromonas sp. DS-12</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r26c6-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000861.1: 0-20498">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000861/index.html#r1c1" target="_new">BGC0000861.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Other</td>
      <td>eicosapentaenoic acid</td>
      <td>Shewanella sp. BR-2</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.48, order:1.00, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Polyketide</td>
      <td>oronofacic acid</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000048.1: 0-52827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000048/index.html#r1c1" target="_new">BGC0000048.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.48, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>dothistromin</td>
      <td>Dothistroma septosporum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000152.1: 1909-53697">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000152/index.html#r1c1" target="_new">BGC0000152.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.48, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>sterigmatocystin</td>
      <td>Aspergillus nidulans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000862.1: 6120-34330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000862/index.html#r1c1" target="_new">BGC0000862.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>eicosapentaenoic acid</td>
      <td>Shewanella pneumatophori</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000284.1: 3041748-3053697">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000284/index.html#r1c1" target="_new">BGC0000284.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>phenolic lipids</td>
      <td>Azotobacter vinelandii DJ</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.51, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000935.1: 151-29847">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000935/index.html#r1c1" target="_new">BGC0000935.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Other, Polyketide,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
      <td>esmeraldin</td>
      <td>Streptomyces antibioticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000869.1: 0-17945">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000869/index.html#r1c1" target="_new">BGC0000869.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Other</td>
      <td>heterocyst glycolipids</td>
      <td>Nostoc sp. &#39;Peltigera membranacea cyanobiont&#39;</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001104.1: 4797713-4809194">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001104/index.html#r1c1" target="_new">BGC0001104.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP, Polyketide</td>
      <td>myxovirescin A1</td>
      <td>Myxococcus xanthus DK 1622</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r26c6-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-136"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-136">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r26c6-pfam-details-svg" data-filename="c00026_CC55NOD.._r26c6_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r26c6-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r26c6-nrps_pks_products" data-name="r26c6-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r26c6-nrps_pks" data-name="r26c6-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r26c6-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-130"></div> <span class="help-tooltip" id="nrps-structures-help-130">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 15, location 702308 - 770886:
          <div class="collapser collapser-target-CC15 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r26c6-15" class="smiles-canvas" data-smiles="NCC(=O)C([*])C(-O)C([*])C(-O)"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(gly) + (pk) + (pk)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly&amp;nrps2=pk&amp;nrps3=pk" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*&amp;nrps2=pk*&amp;nrps3=pk*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r26c6-15-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NCC(=O)C([*])C(-O)C([*])C(-O)"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 16, location 702308 - 745857:
          <div class="collapser collapser-target-CC16 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r26c6-16" class="smiles-canvas" data-smiles="NCC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(gly)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r26c6-16-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NCC(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 17, location 705862 - 749168:
          <div class="collapser collapser-target-CC17 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r26c6-17" class="smiles-canvas" data-smiles="NCC(=O)C([*])C(-O)"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(gly) + (pk)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly&amp;nrps2=pk" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*&amp;nrps2=pk*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r26c6-17-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NCC(=O)C([*])C(-O)"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r26c6-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-131"></div> <span class="help-tooltip" id="nrps-monomers-help-131">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds706572_708834</span></strong>: X
      <div class="collapser collapser-target-cds706572_708834 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (24..393)</strong>: X
          <div class="collapser collapser-target-cds706572_708834 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds706572_708834-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">gmmYvGGcVK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds715870_716794</span></strong>: mal
      <div class="collapser collapser-target-cds715870_716794 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (10..300)</strong>: mal
          <div class="collapser collapser-target-cds715870_716794 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds715870_716794-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds715870_716794-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 44.5</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 28.5</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 22.4</dt>

  <dd></dd><dt>inactive: 21.6</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 21.4</dt>

  <dd></dd><dt>Propionyl-CoA: 18.7</dt>

  <dd></dd><dt>Benzoyl-CoA: 15.8</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 14.5</dt>

  <dd></dd><dt>CHC-CoA: 11.7</dt>

  <dd></dd><dt>fatty_acid: 11.0</dt>

  <dd></dd><dt>Acetyl-CoA: 9.0</dt>

  <dd></dd><dt>trans-1,2-CPDA: 8.4</dt>

  <dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds722308_725857</span></strong>: mmal - mmal
      <div class="collapser collapser-target-cds722308_725857 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (280..575)</strong>: mmal
          <div class="collapser collapser-target-cds722308_725857 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA, Methylmalonyl-CoA
              <div class="collapser collapser-target-cds722308_725857-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
<dd>Methylmalonyl-CoA: 70.8%</dd>
<dd>Propionyl-CoA: 66.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds722308_725857-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 123.7</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 106.5</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 102.7</dt>

  <dd></dd><dt>Malonyl-CoA: 68.4</dt>

  <dd></dd><dt>Isobutyryl-CoA: 66.6</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 62.1</dt>

  <dd></dd><dt>Benzoyl-CoA: 59.2</dt>

  <dd></dd><dt>Propionyl-CoA: 56.3</dt>

  <dd></dd><dt>trans-1,2-CPDA: 49.3</dt>

  <dd></dd><dt>Acetyl-CoA: 42.2</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 37.7</dt>

  <dd></dd><dt>fatty_acid: 34.5</dt>

  <dd></dd><dt>CHC-CoA: 25.3</dt>

  <dd></dd><dt>inactive: 16.8</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
          <dd><strong>PKS_AT (704..993)</strong>: mmal
          <div class="collapser collapser-target-cds722308_725857 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds722308_725857-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 70.8%</dd>
<dd>Ethylmalonyl-CoA: 66.7%</dd>
<dd>Malonyl-CoA: 62.5%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds722308_725857-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 126.1</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 113.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 109.8</dt>

  <dd></dd><dt>Isobutyryl-CoA: 81.9</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 66.6</dt>

  <dd></dd><dt>Malonyl-CoA: 66.2</dt>

  <dd></dd><dt>Propionyl-CoA: 61.9</dt>

  <dd></dd><dt>trans-1,2-CPDA: 60.3</dt>

  <dd></dd><dt>Benzoyl-CoA: 49.9</dt>

  <dd></dd><dt>Acetyl-CoA: 43.8</dt>

  <dd></dd><dt>CHC-CoA: 41.2</dt>

  <dd></dd><dt>fatty_acid: 13.3</dt>

  <dd></dd><dt>inactive: 12.7</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 12.2</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds725862_729168</span></strong>: gly
      <div class="collapser collapser-target-cds725862_729168 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (509..892)</strong>: gly
          <div class="collapser collapser-target-cds725862_729168 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-cds725862_729168-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILQLGLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds740936_747233</span></strong>: mal
      <div class="collapser collapser-target-cds740936_747233 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (1132..1413)</strong>: mal
          <div class="collapser collapser-target-cds740936_747233 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds740936_747233-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>inactive: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds740936_747233-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 62.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 25.2</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 23.6</dt>

  <dd></dd><dt>inactive: 21.5</dt>

  <dd></dd><dt>Propionyl-CoA: 20.3</dt>

  <dd></dd><dt>trans-1,2-CPDA: 9.8</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds747220_750886</span></strong>: pk
      <div class="collapser collapser-target-cds747220_750886 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (564..764)</strong>: pk
          <div class="collapser collapser-target-cds747220_750886 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  (unknown)
              <div class="collapser collapser-target-cds747220_750886-ATSignature collapser-level-none"> </div><div class="collapser-content">
                No matches above 50%
              </div><br>
          
              minowa:  trans-1,2-CPDA
              <div class="collapser collapser-target-cds747220_750886-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>inactive: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>Propionyl-CoA: 0.0</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 0.0</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 0.0</dt>

  <dd></dd><dt>Malonyl-CoA: 0.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r30c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00030_CC55NOD.. - Region 1 - LAP,thiopeptide <div class="help-container"> <div class="help-icon" data-id="region_svg-help-260"></div> <span class="help-tooltip" id="region_svg-help-260">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00030_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r30c1-svg" data-filename="c00030_CC55NOD.._r30c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 29,780 - 57,586 nt. (total: 27,807 nt)
        
        <a class="cluster-rules-header" id="r30c1-rules-header" href="#r30c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r30c1-rules">LAP: ((((goadsporin_like or PF00881 or TIGR03605) and (YcaO or TIGR03882 or TIGR03604)) or TIGR03603 or TIGR03602) and not cyanobactin_synth)<br>thiopeptide: ((YcaO or TIGR03882) and ((thio_amide and (PF06968 or PF04055 or PF07366)) or Lant_dehydr_C or Lant_dehydr_N or PF07366 or PF06968 or PF04055) or thiostrepton)</div>
      
      <div class="region-svg-container">
       <div id='r30c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-261"></div> <span class="help-tooltip" id="focus-panel-help-261">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r30c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r30c1-thiopeptides" data-name="r30c1-thiopeptides">
        Thiopeptides
      </div>
     
    
     
      <div class="body-details-header r30c1-clusterblast" data-name="r30c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r30c1-knownclusterblast" data-name="r30c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r30c1-subclusterblast" data-name="r30c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r30c1-MIBiG-cluster-compare" data-name="r30c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r30c1-pfam-details" data-name="r30c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r30c1-thiopeptides">
      <div class="details">
    <div class="heading">
      <span>Thiopeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="thio-body-help-137"></div> <span class="help-tooltip" id="thio-body-help-137">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash. Predicted tail sequences are also shown.</span></div>
    </div>
  
    <div class="details-text">
      No core peptides found.
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r30c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-139"></div> <span class="help-tooltip" id="cb-general-help-139">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r30c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r30c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r30c1_1.svg">
        NZ_FNIX01000001 (1196786-1224651): Lentzea jiangxiensis strain CGMCC 4.6609, ... (96% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r30c1_2.svg">
        NZ_FOFV01000006 (206872-234697): Lentzea albida strain DSM 44437, whole genom... (96% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r30c1_3.svg">
        NZ_FNCC01000007 (110185-138016): Lechevalieria fradiae strain CGMCC 4.3506, w... (85% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r30c1_4.svg">
        NZ_QQAU01000006 (395379-421751): Lentzea flaviverrucosa strain DSM 44664 Ga02... (88% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r30c1_5.svg">
        NZ_FOFR01000003 (369103-395120): Lechevalieria xinjiangensis strain CGMCC 4.3... (88% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r30c1_6.svg">
        NZ_BBOJ01000020 (50094-78015): Lentzea aerocolonigenes strain NBRC 13195, who... (82% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r30c1_7.svg">
        NZ_JAAMPJ010000001 (210130-237935): Lentzea sp. NEAU-D13 Scaffold1, whole gen... (82% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r30c1_8.svg">
        NZ_FNET01000013 (47905-75820): Lentzea violacea strain DSM 44796, whole genom... (82% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r30c1_9.svg">
        NZ_FOYL01000005 (477931-505747): Lentzea waywayandensis strain DSM 44232, who... (78% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r30c1_10.svg">
        NZ_QLTT01000009 (107146-133918): Lechevalieria atacamensis strain DSM 45479 G... (75% of genes show similarity), LAP,thiopeptide
      </option>
      
    </select>
    <button id= "clusterblast-r30c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r30c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r30c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-140"></div> <span class="help-tooltip" id="cb-known-help-140">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r30c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-141"></div> <span class="help-tooltip" id="cb-sub-help-141">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r30c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-142"></div> <span class="help-tooltip" id="MIBiG-help-142">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r30c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r30c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r30c1"></div>
    <div id="comparison-MIBiG-r30c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">LAP</span></th>
         
         <th><span class="cc-heat-product">thiopeptide</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001174.1: 1876430-1882234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001174/index.html#r1c1" target="_new">BGC0001174.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.65</td>
        <td>RiPP</td>
        
        <td>corynazolicin</td>
        
        
        <td>Corynebacterium urealyticum DSM 7111</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001173.1: 223360-233305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001173/index.html#r1c1" target="_new">BGC0001173.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.41, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.41, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.42</td>
        <td>RiPP</td>
        
        <td>plantazolicin</td>
        
        
        <td>Bacillus pumilus ATCC 7061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294.1: 26867-35886">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.32</td>
        <td>Other</td>
        
        <td>7-prenylisatin</td>
        
        
        <td>Streptomyces sp. MBT28</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001803.1: 3663-28345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001803/index.html#r1c1" target="_new">BGC0001803.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.44, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.44, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.31</td>
        <td>RiPP</td>
        
        <td>thioviridamide</td>
        
        
        <td>Streptomyces olivoviridis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000625.1: 3663-28345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000625/index.html#r1c1" target="_new">BGC0000625.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.44, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.44, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.31</td>
        <td>RiPP</td>
        
        <td>thioviridamide</td>
        
        
        <td>Streptomyces olivoviridis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.29</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000249.1: 0-12635">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000249/index.html#r1c1" target="_new">BGC0000249.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>nogalamycin</td>
        
        
        <td>Streptomyces nogalater</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001387.1: 150-29437">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001387/index.html#r1c1" target="_new">BGC0001387.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.29</td>
        <td>Other</td>
        
        <td>nucleocidin</td>
        
        
        <td>Streptomyces calvus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>Alkaloid</td>
        
        <td>kitasetaline</td>
        
        
        <td>Kitasatospora setae KM-6054</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000605.1: 7233-24412">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000605/index.html#r1c1" target="_new">BGC0000605.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.44, order:0.28, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
        <td>0.26</td>
        <td>RiPP</td>
        
        <td>GE37468</td>
        
        
        <td>Streptomyces sp. ATCC 55365</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r30c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001174.1: 1876430-1882234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001174/index.html#r1c1" target="_new">BGC0001174.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>RiPP</td>
      <td>corynazolicin</td>
      <td>Corynebacterium urealyticum DSM 7111</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Alkaloid</td>
      <td>kitasetaline</td>
      <td>Kitasatospora setae KM-6054</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001546.1: 0-10917">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001546/index.html#r1c1" target="_new">BGC0001546.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Alkaloid</td>
      <td>chuangxinmycin</td>
      <td>Actinoplanes tsinanensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001485.1: 0-18179">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001485/index.html#r1c1" target="_new">BGC0001485.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>3-demethylchuangxinmycin</td>
      <td>Actinoplanes tsinanensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001201.1: 11915724-11918015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001201/index.html#r1c1" target="_new">BGC0001201.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>eremophilene</td>
      <td>Sorangium cellulosum So ce56</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000607.1: 112-24009">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000607/index.html#r1c1" target="_new">BGC0000607.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>RiPP</td>
      <td>micrococcin P1</td>
      <td>Macrococcus caseolyticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310.1: 6460417-6463122">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Terpene</td>
      <td>naringenin</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Other</td>
      <td>flaviolin</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001224.1: 1518-11352">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001224/index.html#r1c1" target="_new">BGC0001224.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Other</td>
      <td>reductasporine</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001173.1: 223360-233305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001173/index.html#r1c1" target="_new">BGC0001173.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.41, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>RiPP</td>
      <td>plantazolicin</td>
      <td>Bacillus pumilus ATCC 7061</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r30c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-143"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-143">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r30c1-pfam-details-svg" data-filename="c00030_CC55NOD.._r30c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r30c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r30c1-thiopeptides" data-name="r30c1-thiopeptides">
        Thiopeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r30c1-thiopeptides">
      <div class="more-details">
    <div class="heading">
      <span>Core peptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="thio-side-help-138"></div> <span class="help-tooltip" id="thio-side-help-138">Lists the possible core peptides in the region. Each core peptide lists its possible molecular weights and the scores for cleavage site prediction and RODEO. If relevant, other features, such as macrocycle and amidation, will also be listed.</span></div>
    </div>
  
  <div class="prediction-text">No core peptides found.</div>

</div>
    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r31c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00031_CC55NOD.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-262"></div> <span class="help-tooltip" id="region_svg-help-262">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00031_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r31c1-svg" data-filename="c00031_CC55NOD.._r31c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 34,009 - 65,422 nt. (total: 31,414 nt)
        
        <a class="cluster-rules-header" id="r31c1-rules-header" href="#r31c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r31c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r31c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-263"></div> <span class="help-tooltip" id="focus-panel-help-263">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r31c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r31c1-nrps_pks" data-name="r31c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r31c1-clusterblast" data-name="r31c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r31c1-knownclusterblast" data-name="r31c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r31c1-subclusterblast" data-name="r31c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r31c1-MIBiG-cluster-compare" data-name="r31c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r31c1-pfam-details" data-name="r31c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r31c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-144"></div> <span class="help-tooltip" id="nrps-domain-help-144">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r31c1-details-svg" data-filename="c00031_CC55NOD.._r31c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r31c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r31c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-147"></div> <span class="help-tooltip" id="cb-general-help-147">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r31c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r31c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r31c1_1.svg">
        NZ_FNIX01000002 (625660-689832): Lentzea jiangxiensis strain CGMCC 4.6609, wh... (48% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r31c1_2.svg">
        NZ_FNET01000001 (1057357-1121788): Lentzea violacea strain DSM 44796, whole g... (16% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r31c1_3.svg">
        NZ_FWYC01000003 (47754-110994): Lentzea albidocapillata strain DSM 44073, who... (14% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r31c1_4.svg">
        NZ_KB913037 (10240901-10328441): Amycolatopsis balhimycina FH 1894 strain DSM... (9% of genes show similarity), NRPS,bacteriocin,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r31c1_5.svg">
        NZ_CP015098 (7375426-7510438): Streptomyces qaidamensis strain S10 chromosome... (16% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r31c1_6.svg">
        NZ_CP029188 (6285171-6416007): Streptomyces tirandamycinicus strain HNM0039 c... (5% of genes show similarity), NRPS,PKS-like,T2PKS,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r31c1_7.svg">
        NZ_CP029607 (5317453-5650379): Actinosynnema pretiosum subsp. pretiosum strai... (3% of genes show similarity), CDPS,NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r31c1_8.svg">
        NZ_JAIY01000010 (1241882-1310055): Kitasatospora sp. MBT66 scaffold10, whole ... (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r31c1_9.svg">
        NZ_CP016174 (3685051-3929323): Amycolatopsis orientalis strain B-37 chromosom... (7% of genes show similarity), NRPS,T1PKS,TfuA-related,other
      </option>
      
      <option value = "svg/clusterblast_r31c1_10.svg">
        NZ_LISY01000001 (579264-724552): Streptomyces sp. TSRI0261 scaffold1, whole g... (7% of genes show similarity), NRPS,T1PKS,arylpolyene,butyrolactone,ectoine,ladderane
      </option>
      
    </select>
    <button id= "clusterblast-r31c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r31c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r31c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-148"></div> <span class="help-tooltip" id="cb-known-help-148">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r31c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-149"></div> <span class="help-tooltip" id="cb-sub-help-149">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r31c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-150"></div> <span class="help-tooltip" id="MIBiG-help-150">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r31c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r31c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r31c1"></div>
    <div id="comparison-MIBiG-r31c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>NRP</td>
        
        <td>hexose-palythine-serine, hexose-shinorine</td>
        
        
        <td>Heteroscytonema crispum UCFS10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>auriculamide</td>
        
        
        <td>Herpetosiphon aurantiacus DSM 785</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP, Alkaloid</td>
        
        <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001149.1: 196767-209553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001149/index.html#r1c1" target="_new">BGC0001149.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>NRP, Saccharide</td>
        
        <td>brabantamide A</td>
        
        
        <td>Pseudomonas sp. SHC52</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>rhodochelin</td>
        
        
        <td>Rhodococcus jostii RHA1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>vicibactin</td>
        
        
        <td>Rhizobium etli CFN 42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890.1: 1392460-1420815">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
        
        
        <td>Vibrio campbellii ATCC BAA-1116</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r31c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>hexose-palythine-serine, hexose-shinorine</td>
      <td>Heteroscytonema crispum UCFS10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>auriculamide</td>
      <td>Herpetosiphon aurantiacus DSM 785</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP, Alkaloid</td>
      <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001149.1: 196767-209553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001149/index.html#r1c1" target="_new">BGC0001149.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP, Saccharide</td>
      <td>brabantamide A</td>
      <td>Pseudomonas sp. SHC52</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>rhodochelin</td>
      <td>Rhodococcus jostii RHA1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>vicibactin</td>
      <td>Rhizobium etli CFN 42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890.1: 1392460-1420815">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
      <td>Vibrio campbellii ATCC BAA-1116</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r31c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-151"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-151">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r31c1-pfam-details-svg" data-filename="c00031_CC55NOD.._r31c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r31c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r31c1-nrps_pks_products" data-name="r31c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r31c1-nrps_pks" data-name="r31c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r31c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-145"></div> <span class="help-tooltip" id="nrps-structures-help-145">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 34008 - 65422:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r31c1-1" class="smiles-canvas" data-smiles="NC(CC(C)C)C(=O)NC(CO)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(leu - ser)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu,ser" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*,ser*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r31c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CC(C)C)C(=O)NC(CO)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r31c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-146"></div> <span class="help-tooltip" id="nrps-monomers-help-146">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds54008_62753</span></strong>: leu - ser
      <div class="collapser collapser-target-cds54008_62753 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu,ser" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*,ser*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (994..1374)</strong>: leu
          <div class="collapser collapser-target-cds54008_62753 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  leu
              <div class="collapser collapser-target-cds54008_62753-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>leu</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAlyMGGiiK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2123..2502)</strong>: ser
          <div class="collapser collapser-target-cds54008_62753 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds54008_62753-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWHVSLVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r32c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00032_CC55NOD.. - Region 1 - T1PKS,ladderane <div class="help-container"> <div class="help-icon" data-id="region_svg-help-264"></div> <span class="help-tooltip" id="region_svg-help-264">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00032_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r32c1-svg" data-filename="c00032_CC55NOD.._r32c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 61,982 nt. (total: 61,982 nt)
        
        <a class="cluster-rules-header" id="r32c1-rules-header" href="#r32c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r32c1-rules">ladderane: ladderane<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r32c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-265"></div> <span class="help-tooltip" id="focus-panel-help-265">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r32c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r32c1-nrps_pks" data-name="r32c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r32c1-clusterblast" data-name="r32c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r32c1-knownclusterblast" data-name="r32c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r32c1-subclusterblast" data-name="r32c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r32c1-MIBiG-cluster-compare" data-name="r32c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r32c1-pfam-details" data-name="r32c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r32c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-152"></div> <span class="help-tooltip" id="nrps-domain-help-152">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r32c1-details-svg" data-filename="c00032_CC55NOD.._r32c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r32c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r32c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-155"></div> <span class="help-tooltip" id="cb-general-help-155">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r32c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r32c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r32c1_1.svg">
        NZ_FOFV01000027 (14505-131137): Lentzea albida strain DSM 44437, whole genome... (41% of genes show similarity), NRPS,PKS-like,T1PKS,arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r32c1_2.svg">
        NZ_FNCC01000001 (706198-769358): Lechevalieria fradiae strain CGMCC 4.3506, w... (49% of genes show similarity), NRPS-like,arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r32c1_3.svg">
        NZ_CP016793 (9154521-9196355): Lentzea guizhouensis strain DHS C013 chromosom... (59% of genes show similarity), arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r32c1_4.svg">
        NZ_JH636049 (3418739-3461013): Saccharomonospora xinjiangensis XJ-54 Sacxisca... (61% of genes show similarity), arylpolyene,ladderane
      </option>
      
      <option value = "svg/clusterblast_r32c1_5.svg">
        NZ_CP038101 (3531689-3573963): Saccharomonospora sp. 31sw chromosome, complet... (61% of genes show similarity), arylpolyene,ladderane
      </option>
      
      <option value = "svg/clusterblast_r32c1_6.svg">
        NZ_CP016076 (1247976-1290304): Actinoalloteichus fjordicus strain ADI127-7 ch... (69% of genes show similarity), ladderane
      </option>
      
      <option value = "svg/clusterblast_r32c1_7.svg">
        NC_013159 (3194889-3237154): Saccharomonospora viridis DSM 43017, complete se... (63% of genes show similarity), ladderane
      </option>
      
      <option value = "svg/clusterblast_r32c1_8.svg">
        NZ_JACHWU010000004 (169861-212051): Prauserella isguenensis strain CECT 8577 ... (61% of genes show similarity), ladderane
      </option>
      
      <option value = "svg/clusterblast_r32c1_9.svg">
        NZ_CM001440 (4125367-4165048): Saccharomonospora cyanea NA-134 chromosome, wh... (60% of genes show similarity), arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r32c1_10.svg">
        NZ_CP015453 (2487321-2529448): Dietzia psychralcaliphila strain ILA-1 chromos... (64% of genes show similarity), ladderane
      </option>
      
    </select>
    <button id= "clusterblast-r32c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r32c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r32c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-156"></div> <span class="help-tooltip" id="cb-known-help-156">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r32c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r32c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_1.svg">
        vancosamine
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_2.svg">
        phthoxazolin
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_3.svg">
        arimetamycin B / arimetamycin C / arimetamycin A
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_4.svg">
        decaplanin
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_5.svg">
        vancomycin
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_6.svg">
        decaplanin
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_7.svg">
        aculeximycin
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_8.svg">
        komodoquinone B
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_9.svg">
        komodoquinone B
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_10.svg">
        UK-68,597
      </option>
      
    </select>
    <button id="knownclusterblast-r32c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r32c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r32c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-157"></div> <span class="help-tooltip" id="cb-sub-help-157">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r32c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r32c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r32c1_1.svg">
        AB088119_c2: staurosporine deoxysugar (40% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_2.svg">
        HE589771_c4: vancomycin glucose / vancosamine (71% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_3.svg">
        AB032523_c1: avermectin oleandrose (37% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_4.svg">
        Y16952_c4: balhimycin glucose / vancosamine (50% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_5.svg">
        AF497482_c3: calicheamicin 4 nucleotide sugars (16% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_6.svg">
        AB103463_c2: medermycin aminodeoxysugar (angolosamine) (28% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_7.svg">
        AJ871581_c3: rubradirin D-rubranitrose (40% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_8.svg">
        AJ871581_c2: rubradirin nitrosugar (66% of genes show similarity), nitrosugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_9.svg">
        AF187532_c1: nogalamycin nogalamine (25% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_10.svg">
        FJ483966_c3: polyketomycin beta-D-amicetose / alpha-L-axenose (18% of genes show similarity), sugar
      </option>
      
    </select>
    <button id="subclusterblast-r32c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r32c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r32c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-158"></div> <span class="help-tooltip" id="MIBiG-help-158">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r32c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r32c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r32c1"></div>
    <div id="comparison-MIBiG-r32c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">ladderane</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.54</td>
        <td>Other</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001184.1: 3593876-3601174">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001184/index.html#r1c1" target="_new">BGC0001184.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.54</td>
        <td>Other</td>
        
        <td>bacilysin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888.1: 0-4342">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.54</td>
        <td>Other</td>
        
        <td>bacilysin</td>
        
        
        <td>Bacillus sp. CS93</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001343.1: 61-19132">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001343/index.html#r1c1" target="_new">BGC0001343.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.53</td>
        <td>Polyketide</td>
        
        <td>aurachin A, aurachin D</td>
        
        
        <td>Stigmatella aurantiaca Sg a15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000193.1: 0-14807">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000193/index.html#r1c1" target="_new">BGC0000193.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.59, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.59, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.52</td>
        <td>Polyketide</td>
        
        <td>aclacinomycin</td>
        
        
        <td>Streptomyces galilaeus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000873.1: 469-33484">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000873/index.html#r1c1" target="_new">BGC0000873.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.49</td>
        <td>Other</td>
        
        <td>A-201A</td>
        
        
        <td>Saccharothrix mutabilis subsp. capreolus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001060.1: 2115-74911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001060/index.html#r1c1" target="_new">BGC0001060.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.47</td>
        <td>Polyketide</td>
        
        <td>dynemicin A</td>
        
        
        <td>Micromonospora chersina</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002012.1: 8348-34051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002012/index.html#r1c1" target="_new">BGC0002012.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.46</td>
        <td>Polyketide</td>
        
        <td>julichrome Q3-3, julichrome Q3-5</td>
        
        
        <td>Streptomyces afghaniensis 772</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000273.1: 50-42876">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000273/index.html#r1c1" target="_new">BGC0000273.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.46</td>
        <td>Polyketide, Saccharide</td>
        
        <td>steffimycin D</td>
        
        
        <td>Streptomyces steffisburgensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000224.1: 32-32515">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000224/index.html#r1c1" target="_new">BGC0000224.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.48, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.48, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.46</td>
        <td>Polyketide</td>
        
        <td>fredericamycin A</td>
        
        
        <td>Streptomyces griseus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r32c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001343.1: 61-19132">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001343/index.html#r1c1" target="_new">BGC0001343.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>aurachin A, aurachin D</td>
      <td>Stigmatella aurantiaca Sg a15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001184.1: 3593876-3601174">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001184/index.html#r1c1" target="_new">BGC0001184.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other</td>
      <td>bacilysin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888.1: 0-4342">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other</td>
      <td>bacilysin</td>
      <td>Bacillus sp. CS93</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>depudecin</td>
      <td>Alternaria brassicicola</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002012.1: 8348-34051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002012/index.html#r1c1" target="_new">BGC0002012.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>julichrome Q3-3, julichrome Q3-5</td>
      <td>Streptomyces afghaniensis 772</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000193.1: 0-14807">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000193/index.html#r1c1" target="_new">BGC0000193.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.59, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>aclacinomycin</td>
      <td>Streptomyces galilaeus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002000.1: 2056-14558">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002000/index.html#r1c1" target="_new">BGC0002000.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>setomimycin</td>
      <td>Streptomyces aurantiacus JA 4570</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001895.1: 2411904-2429820">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001895/index.html#r1c1" target="_new">BGC0001895.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>huanglongmycin A, huanglongmycin B, huanglongmycin C</td>
      <td>Streptomyces sp. CB09001</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001927.1: 0-27705">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001927/index.html#r1c1" target="_new">BGC0001927.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Other</td>
      <td>A-94964</td>
      <td>Streptomyces sp.</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r32c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-159"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-159">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r32c1-pfam-details-svg" data-filename="c00032_CC55NOD.._r32c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r32c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r32c1-nrps_pks_products" data-name="r32c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r32c1-nrps_pks" data-name="r32c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r32c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-153"></div> <span class="help-tooltip" id="nrps-structures-help-153">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 61982:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r32c1-1" class="smiles-canvas" data-smiles="C(C)C(=O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(Me-mal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-mal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-mal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r32c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C(C)C(=O)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 18298 - 61982:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r32c1-3" class="smiles-canvas" data-smiles="C(C)C(=O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(Me-mal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-mal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-mal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r32c1-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C(C)C(=O)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r32c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-154"></div> <span class="help-tooltip" id="nrps-monomers-help-154">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds14571_15486</span></strong>: mal
      <div class="collapser collapser-target-cds14571_15486 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (4..281)</strong>: mal
          <div class="collapser collapser-target-cds14571_15486 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds14571_15486-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>inactive: 62.5%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds14571_15486-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 65.2</dt>

  <dd></dd><dt>inactive: 47.9</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 28.8</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 23.1</dt>

  <dd></dd><dt>Benzoyl-CoA: 22.4</dt>

  <dd></dd><dt>fatty_acid: 19.1</dt>

  <dd></dd><dt>Isobutyryl-CoA: 17.4</dt>

  <dd></dd><dt>Propionyl-CoA: 9.9</dt>

  <dd></dd><dt>Acetyl-CoA: 7.9</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 3.1</dt>

  <dd></dd><dt>CHC-CoA: 0.5</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: -0.8</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds38298_41982</span></strong>: mmal
      <div class="collapser collapser-target-cds38298_41982 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (529..812)</strong>: mmal
          <div class="collapser collapser-target-cds38298_41982 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds38298_41982-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 83.3%</dd>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Ethylmalonyl-CoA: 66.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds38298_41982-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 136.8</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 123.8</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 118.6</dt>

  <dd></dd><dt>Malonyl-CoA: 97.9</dt>

  <dd></dd><dt>Propionyl-CoA: 75.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 70.9</dt>

  <dd></dd><dt>Acetyl-CoA: 68.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 56.0</dt>

  <dd></dd><dt>CHC-CoA: 43.9</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 41.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 38.9</dt>

  <dd></dd><dt>Benzoyl-CoA: 29.2</dt>

  <dd></dd><dt>inactive: 25.7</dt>

  <dd></dd><dt>fatty_acid: 25.5</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r34c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00034_CC55NOD.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-266"></div> <span class="help-tooltip" id="region_svg-help-266">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00034_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r34c1-svg" data-filename="c00034_CC55NOD.._r34c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 66,342 - 111,786 nt. (total: 45,445 nt)
        
        <a class="cluster-rules-header" id="r34c1-rules-header" href="#r34c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r34c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r34c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-267"></div> <span class="help-tooltip" id="focus-panel-help-267">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r34c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r34c1-nrps_pks" data-name="r34c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r34c1-clusterblast" data-name="r34c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r34c1-knownclusterblast" data-name="r34c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r34c1-subclusterblast" data-name="r34c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r34c1-MIBiG-cluster-compare" data-name="r34c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r34c1-pfam-details" data-name="r34c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r34c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-160"></div> <span class="help-tooltip" id="nrps-domain-help-160">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r34c1-details-svg" data-filename="c00034_CC55NOD.._r34c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r34c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r34c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-163"></div> <span class="help-tooltip" id="cb-general-help-163">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r34c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r34c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r34c1_1.svg">
        NZ_FNIX01000003 (116265-269458): Lentzea jiangxiensis strain CGMCC 4.6609, wh... (16% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r34c1_2.svg">
        NZ_CP016793 (7250155-7446409): Lentzea guizhouensis strain DHS C013 chromosom... (7% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r34c1_3.svg">
        NZ_FNCC01000009 (135072-176155): Lechevalieria fradiae strain CGMCC 4.3506, w... (20% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r34c1_4.svg">
        NZ_WWIP01000001 (&lt;313358-404415): Streptomyces sp. SID5466 SID5466.c1, whole ... (17% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r34c1_5.svg">
        NZ_CP015362 (290184-501485): Streptomyces sp. S8 chromosome, complete genome (6% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r34c1_6.svg">
        NZ_CP007155 (4289286-4476829): Kutzneria albida DSM 43870 chromosome, complet... (6% of genes show similarity), NRPS,T1PKS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r34c1_7.svg">
        NZ_FNCC01000010 (375836-416895): Lechevalieria fradiae strain CGMCC 4.3506, w... (22% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r34c1_8.svg">
        NZ_WPNZ01000002 (300259-347340): Streptomyces sp. p1417 Scaffold2, whole geno... (13% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r34c1_9.svg">
        NZ_CP040916 (8699135-8800056): Streptomyces spectabilis strain NRRL 2792 chro... (9% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r34c1_10.svg">
        NZ_FOFR01000023 (23693-101789): Lechevalieria xinjiangensis strain CGMCC 4.35... (11% of genes show similarity), NRPS,terpene
      </option>
      
    </select>
    <button id= "clusterblast-r34c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r34c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r34c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-164"></div> <span class="help-tooltip" id="cb-known-help-164">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r34c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r34c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r34c1_1.svg">
        dechlorocuracomycin
      </option>
      
      <option value = "svg/knownclusterblast_r34c1_2.svg">
        kanamycin
      </option>
      
      <option value = "svg/knownclusterblast_r34c1_3.svg">
        daptomycin
      </option>
      
      <option value = "svg/knownclusterblast_r34c1_4.svg">
        hygromycin A
      </option>
      
    </select>
    <button id="knownclusterblast-r34c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r34c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r34c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-165"></div> <span class="help-tooltip" id="cb-sub-help-165">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r34c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-166"></div> <span class="help-tooltip" id="MIBiG-help-166">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r34c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r34c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r34c1"></div>
    <div id="comparison-MIBiG-r34c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>pseudomonine</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
        
        
        <td>Penicillium rubens Wisconsin 54-1255</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000392.1: 3113323-3141040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000392/index.html#r1c1" target="_new">BGC0000392.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>mirubactin</td>
        
        
        <td>Actinosynnema mirum DSM 43827</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001824.1: 39732-57269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001824/index.html#r1c1" target="_new">BGC0001824.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>xefoampeptides A-G</td>
        
        
        <td>Xenorhabdus beddingii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r34c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>pseudomonine</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
      <td>Penicillium rubens Wisconsin 54-1255</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000392.1: 3113323-3141040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000392/index.html#r1c1" target="_new">BGC0000392.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>mirubactin</td>
      <td>Actinosynnema mirum DSM 43827</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001824.1: 39732-57269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001824/index.html#r1c1" target="_new">BGC0001824.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>xefoampeptides A-G</td>
      <td>Xenorhabdus beddingii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r34c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-167"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-167">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r34c1-pfam-details-svg" data-filename="c00034_CC55NOD.._r34c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r34c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r34c1-nrps_pks_products" data-name="r34c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r34c1-nrps_pks" data-name="r34c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r34c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-161"></div> <span class="help-tooltip" id="nrps-structures-help-161">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 66341 - 111786:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r34c1-1" class="smiles-canvas" data-smiles="NC(CC(=O)N)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-asn)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-asn" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-asn*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r34c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CC(=O)N)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r34c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-162"></div> <span class="help-tooltip" id="nrps-monomers-help-162">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds86341_91786</span></strong>: asn
      <div class="collapser collapser-target-cds86341_91786 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asn" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asn*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (812..1190)</strong>: asn
          <div class="collapser collapser-target-cds86341_91786 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asn
              <div class="collapser collapser-target-cds86341_91786-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>asp, asn</dd>
   <dt>Single AA prediction:</dt>
   <dd>asn</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLTKVGEVGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r36c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00036_CC55NOD.. - Region 1 - NRPS-like,PKS-like,T2PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-268"></div> <span class="help-tooltip" id="region_svg-help-268">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00036_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r36c1-svg" data-filename="c00036_CC55NOD.._r36c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 126,166 - 238,151 nt. (total: 111,986 nt)
        
        <a class="cluster-rules-header" id="r36c1-rules-header" href="#r36c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r36c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>PKS-like: (t2pks2 or ksIII)<br>T2PKS: (t2ks and t2clf)</div>
      
      <div class="region-svg-container">
       <div id='r36c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-269"></div> <span class="help-tooltip" id="focus-panel-help-269">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r36c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r36c1-nrps_pks" data-name="r36c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r36c1-clusterblast" data-name="r36c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r36c1-knownclusterblast" data-name="r36c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r36c1-subclusterblast" data-name="r36c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r36c1-MIBiG-cluster-compare" data-name="r36c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r36c1-pfam-details" data-name="r36c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r36c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-168"></div> <span class="help-tooltip" id="nrps-domain-help-168">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r36c1-details-svg" data-filename="c00036_CC55NOD.._r36c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r36c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r36c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-171"></div> <span class="help-tooltip" id="cb-general-help-171">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r36c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r36c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r36c1_1.svg">
        NZ_FOFR01000017 (31827-126369): Lechevalieria xinjiangensis strain CGMCC 4.35... (24% of genes show similarity), NRPS,T2PKS,ectoine
      </option>
      
      <option value = "svg/clusterblast_r36c1_2.svg">
        NZ_CP029541 (1534579-1768027): Streptomyces sp. NEAU-S7GS2 chromosome, comple... (14% of genes show similarity), NRPS,T1PKS,T2PKS,melanin
      </option>
      
      <option value = "svg/clusterblast_r36c1_3.svg">
        NZ_KL585965 (58363-130864): Streptomyces sp. NRRL S-474 P279 Doro1 scaffold20... (19% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r36c1_4.svg">
        NZ_KB911581 (189645-303579): Streptomyces bottropensis ATCC 25435 StrboDRAFT ... (14% of genes show similarity), NRPS,T1PKS,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r36c1_5.svg">
        NC_003155 (3449231-3645978): Streptomyces avermitilis MA-4680 = NBRC 14893, c... (12% of genes show similarity), T1PKS,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r36c1_6.svg">
        NZ_RSFG01000001 (1258527-1346100): Streptomyces sp. CB01881 Scaffold1, whole ... (18% of genes show similarity), T2PKS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r36c1_7.svg">
        NZ_CP010849 (497743-653174): Streptomyces cyaneogriseus subsp. noncyanogenus ... (16% of genes show similarity), T1PKS,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r36c1_8.svg">
        NZ_KQ948926 (116667-201811): Streptomyces caeruleatus strain NRRL B-24802 PRJ... (17% of genes show similarity), T1PKS,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r36c1_9.svg">
        NZ_CP019724 (2841941-3025130): Streptomyces pactum strain ACT12 chromosome, c... (12% of genes show similarity), T1PKS,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r36c1_10.svg">
        NZ_CP020700 (6593191-6680602): Streptomyces tsukubensis strain NRRL 18488 chr... (18% of genes show similarity), T2PKS,terpene
      </option>
      
    </select>
    <button id= "clusterblast-r36c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r36c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r36c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-172"></div> <span class="help-tooltip" id="cb-known-help-172">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r36c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r36c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r36c1_1.svg">
        xantholipin
      </option>
      
      <option value = "svg/knownclusterblast_r36c1_2.svg">
        formicamycins A-M
      </option>
      
      <option value = "svg/knownclusterblast_r36c1_3.svg">
        griseorhodin A
      </option>
      
      <option value = "svg/knownclusterblast_r36c1_4.svg">
        allocyclinone
      </option>
      
      <option value = "svg/knownclusterblast_r36c1_5.svg">
        spore pigment
      </option>
      
      <option value = "svg/knownclusterblast_r36c1_6.svg">
        arixanthomycin A / arixanthomycin B / arixanthomycin C
      </option>
      
      <option value = "svg/knownclusterblast_r36c1_7.svg">
        WS79089A / hexaricin B / hexaricin C
      </option>
      
      <option value = "svg/knownclusterblast_r36c1_8.svg">
        lactonamycin
      </option>
      
      <option value = "svg/knownclusterblast_r36c1_9.svg">
        benastatin A / benastatin B / BE-43767 / TAN1532B
      </option>
      
      <option value = "svg/knownclusterblast_r36c1_10.svg">
        TLN-05220
      </option>
      
    </select>
    <button id="knownclusterblast-r36c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r36c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r36c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-173"></div> <span class="help-tooltip" id="cb-sub-help-173">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r36c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r36c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r36c1_1.svg">
        AF324838_c1: simocyclinone t2pks (50% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r36c1_2.svg">
        AF080235_c1: landomycin t2pks (50% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r36c1_3.svg">
        FJ483966_c1: polyketomycin t2pks (25% of genes show similarity), t2pks
      </option>
      
    </select>
    <button id="subclusterblast-r36c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r36c1-svg"></div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r36c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-175"></div> <span class="help-tooltip" id="MIBiG-help-175">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r36c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r36c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r36c1"></div>
    <div id="comparison-MIBiG-r36c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
         <th><span class="cc-heat-product">PKS-like</span></th>
         
         <th><span class="cc-heat-product">T2PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000903.1: 126-11373">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000903/index.html#r1c1" target="_new">BGC0000903.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.61, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.61, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.71</td>
        <td>Other</td>
        
        <td>fluorometabolite</td>
        
        
        <td>Streptomyces cattleya</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.41, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.63</td>
        <td>Other</td>
        
        <td>5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester</td>
        
        
        <td>Streptomyces sp. RM-5-8</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.47, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.52, order:0.73, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.59</td>
        <td>Polyketide</td>
        
        <td>curamycin</td>
        
        
        <td>Streptomyces cyaneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000218.1: 27-8720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000218/index.html#r1c1" target="_new">BGC0000218.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.49, order:0.36, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.47</td>
        <td>Polyketide</td>
        
        <td>doxorubicin</td>
        
        
        <td>Streptomyces peucetius</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001592.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001592/index.html#r1c1" target="_new">BGC0001592.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.49, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.47</td>
        <td>Other</td>
        
        <td>ferroverdin</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002029.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002029/index.html#r1c1" target="_new">BGC0002029.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.49, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.47</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B, bagremycin C, bagremycin E, bagremycin F, bagremycin G, ferroverdin A, ferroverdin B, ferroverdin C</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001343.1: 61-19132">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001343/index.html#r1c1" target="_new">BGC0001343.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:0.37, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.46</td>
        <td>Polyketide</td>
        
        <td>aurachin A, aurachin D</td>
        
        
        <td>Stigmatella aurantiaca Sg a15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001940.1: 108-33418">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001940/index.html#r1c1" target="_new">BGC0001940.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.47, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.48, order:0.35, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.45</td>
        <td>Polyketide</td>
        
        <td>pyxidicycline A, pyxidicycline B</td>
        
        
        <td>Pyxidicoccus fallax</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001177.1: 10040-36741">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001177/index.html#r1c1" target="_new">BGC0001177.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.58, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.54, order:0.22, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.44</td>
        <td>Polyketide</td>
        
        <td>mensacaricin</td>
        
        
        <td>Streptomyces bottropensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002081.1: 0-30441">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002081/index.html#r1c1" target="_new">BGC0002081.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.45, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.49, order:0.25, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.44</td>
        <td>Polyketide</td>
        
        <td>streptoketide A, streptoketide B, streptoketide C, S2502, S2507, UWM5</td>
        
        
        <td>Streptomyces sp. Tue6314</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r36c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.74 (id:0.51, order:0.79, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
      <td>0.74</td>
      <td>Polyketide</td>
      <td>spore pigment</td>
      <td>Streptomyces collinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.52, order:0.73, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Polyketide</td>
      <td>curamycin</td>
      <td>Streptomyces cyaneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.54, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Polyketide</td>
      <td>granaticin</td>
      <td>Streptomyces violaceoruber</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.51, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>naphtocyclinone</td>
      <td>Streptomyces arenae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Alkaloid</td>
      <td>kitasetaline</td>
      <td>Kitasatospora setae KM-6054</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000271.1: 0-11118">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000271/index.html#r1c1" target="_new">BGC0000271.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.51, order:0.52, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>spore pigment</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001334.1: 9570-30347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001334/index.html#r1c1" target="_new">BGC0001334.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Alkaloid</td>
      <td>caldoniamide A, cladoniamide B, cladoniamide C7, cladoniamide D, cladoniamide E, cladoniamide F, cladoniamide G</td>
      <td>Streptomyces uncialis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.50, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>griseusin</td>
      <td>Streptomyces griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP, Polyketide</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Other</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r36c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-176"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-176">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r36c1-pfam-details-svg" data-filename="c00036_CC55NOD.._r36c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r36c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r36c1-nrps_pks_products" data-name="r36c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r36c1-nrps_pks" data-name="r36c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
      <div class="sidepanel-details-header r36c1-t2pks" data-name="r36c1-t2pks">
        Type II PKS
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r36c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-169"></div> <span class="help-tooltip" id="nrps-structures-help-169">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 126165 - 238151:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r36c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r36c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 126165 - 168601:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r36c1-2" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r36c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
    
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r36c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-170"></div> <span class="help-tooltip" id="nrps-monomers-help-170">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds126250_127642</span></strong>: X
      <div class="collapser collapser-target-cds126250_127642 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (2..368)</strong>: X
          <div class="collapser collapser-target-cds126250_127642 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds126250_127642-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">hvTfLGgpGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds146165_148601</span></strong>: X
      <div class="collapser collapser-target-cds146165_148601 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (8..354)</strong>: X
          <div class="collapser collapser-target-cds146165_148601 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds146165_148601-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">cyshTAmarK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>30% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds178000_179431</span></strong>: X
      <div class="collapser collapser-target-cds178000_179431 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (32..380)</strong>: X
          <div class="collapser collapser-target-cds178000_179431 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds178000_179431-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">gLclICiaiK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ala</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>40% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    <div class="sidepanel-details-section r36c1-t2pks">
      <div class="more-details">
    <div class="heading">
      <span>Type II PKS gene cluster prediction details</span>
      <div class="help-container"> <div class="help-icon" data-id="t2pks-help-174"></div> <span class="help-tooltip" id="t2pks-help-174">Predictions of starter units, elongations, product classes, and potential molecular weights for type II PKS clusters.<br>More detailed information is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/t2pks' target='_blank'>here</a>.</span></div>
    </div>
	
	
	<dl class="prediction-text">
        <dt><strong>Cluster section:</strong></dt>
        <dd>165641 to 238151</dd>
		
		<dt><strong>Starter unit(s):</strong></dt>
		<dd>
				acetyl-CoA (Score: 0.0; E-value: 0)<br>
			
	    <dd>
		</tr>
		
		
		
		<dt><strong>Product class(es):</strong></td>
        <dd>
				
				pentangular polyphenol<br>
				
				tetracenomycin<br>
				
		</dd>
		
		
	</dl>
	<dl class="prediction-text">
		
		<dt><strong>cds200255_200645</strong></dt>
			
			<dd>
				CYC
				
					C5-C18
				
				(Score: 200.5; E-value: 1.8e-62)<br>
			</dd>
			
		
		<dt><strong>cds200641_201898</strong></dt>
			
			<dd>
				KS
				
				(Score: 690.0; E-value: 2.4e-210)<br>
			</dd>
			
		
		<dt><strong>cds201894_203151</strong></dt>
			
			<dd>
				CLF
				
				(Score: 515.0; E-value: 3e-157)<br>
			</dd>
			
		
		<dt><strong>cds203152_203401</strong></dt>
			
			<dd>
				ACP
				
				(Score: 43.1; E-value: 7.5e-14)<br>
			</dd>
			
		
		<dt><strong>cds203397_203856</strong></dt>
			
			<dd>
				CYC
				
					C9-C14
				
				(Score: 246.7; E-value: 1.5e-76)<br>
			</dd>
			
		
		<dt><strong>cds203840_204170</strong></dt>
			
			<dd>
				CYC
				
					C4-C21/C2-C23|C2-C19
				
				(Score: 139.8; E-value: 6.8e-44)<br>
			</dd>
			
		
		<dt><strong>cds204162_205710</strong></dt>
			
			<dd>
				OXY
				
				(Score: 263.1; E-value: 1.3e-80)<br>
			</dd>
			
		
		<dt><strong>cds215016_216186</strong></dt>
			
			<dd>
				OXY
				
				(Score: 49.5; E-value: 7.6e-16)<br>
			</dd>
			
		
	</dl>
    
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r36c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00036_CC55NOD.. - Region 2 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-270"></div> <span class="help-tooltip" id="region_svg-help-270">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00036_CC55NOD...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r36c2-svg" data-filename="c00036_CC55NOD.._r36c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 283,753 - 328,261 nt. (total: 44,509 nt)
        
        <a class="cluster-rules-header" id="r36c2-rules-header" href="#r36c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r36c2-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r36c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-271"></div> <span class="help-tooltip" id="focus-panel-help-271">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r36c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r36c2-nrps_pks" data-name="r36c2-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r36c2-clusterblast" data-name="r36c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r36c2-knownclusterblast" data-name="r36c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r36c2-subclusterblast" data-name="r36c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r36c2-MIBiG-cluster-compare" data-name="r36c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r36c2-pfam-details" data-name="r36c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r36c2-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-177"></div> <span class="help-tooltip" id="nrps-domain-help-177">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r36c2-details-svg" data-filename="c00036_CC55NOD.._r36c2_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r36c2-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r36c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-178"></div> <span class="help-tooltip" id="cb-general-help-178">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r36c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r36c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r36c2_1.svg">
        NZ_CP022088 (2427001-2578997): Nocardia brasiliensis strain FDAARGOS 352 chro... (5% of genes show similarity), NRPS,NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r36c2_2.svg">
        NZ_FNVT01000023 (84955-145914): Nonomuraea solani strain CGMCC 4.7037, whole ... (13% of genes show similarity), LAP,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r36c2_3.svg">
        NC_016582 (1820779-1955837): Streptomyces bingchenggensis BCW-1, complete seq... (21% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r36c2_4.svg">
        NC_018681 (4143514-4296483): Nocardia brasiliensis ATCC 700358, complete sequ... (6% of genes show similarity), NRPS,NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r36c2_5.svg">
        NZ_CP029607 (3093395-3248136): Actinosynnema pretiosum subsp. pretiosum strai... (13% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r36c2_6.svg">
        NZ_LT607733 (4985451-5129834): Micromonospora echinofusca strain DSM 43913 ch... (14% of genes show similarity), LAP,NRPS,NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r36c2_7.svg">
        NZ_CP029607 (5317453-5650379): Actinosynnema pretiosum subsp. pretiosum strai... (5% of genes show similarity), CDPS,NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r36c2_8.svg">
        NC_013093 (3002519-3157721): Actinosynnema mirum DSM 43827, complete sequence (12% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r36c2_9.svg">
        NZ_JACHJQ010000006 (133973-295806): Actinophytocola algeriensis strain CECT 8... (8% of genes show similarity), LAP,NRPS,NRPS-like,T1PKS,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r36c2_10.svg">
        NZ_CP036455 (3545303-3647129): Streptomonospora sp. M2 chromosome, complete g... (16% of genes show similarity), T1PKS,T3PKS
      </option>
      
    </select>
    <button id= "clusterblast-r36c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r36c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r36c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-179"></div> <span class="help-tooltip" id="cb-known-help-179">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r36c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r36c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r36c2_1.svg">
        4-Z-annimycin
      </option>
      
      <option value = "svg/knownclusterblast_r36c2_2.svg">
        caniferolide A / caniferolide B / caniferolide C / caniferolide D
      </option>
      
      <option value = "svg/knownclusterblast_r36c2_3.svg">
        macrotermycins
      </option>
      
    </select>
    <button id="knownclusterblast-r36c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r36c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r36c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-180"></div> <span class="help-tooltip" id="cb-sub-help-180">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r36c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-181"></div> <span class="help-tooltip" id="MIBiG-help-181">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r36c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r36c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r36c2"></div>
    <div id="comparison-MIBiG-r36c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000122.1: 1816-33971">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000122/index.html#r1c1" target="_new">BGC0000122.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>phenylnannolone A, phenylnannolone B, phenylnannolone C</td>
        
        
        <td>Nannocystis pusilla</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340.1: 1840-51432">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>byssochlamic acid</td>
        
        
        <td>Byssochlamys fulva</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001794.1: 3660-23178">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001794/index.html#r1c1" target="_new">BGC0001794.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.47, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>swainsonine</td>
        
        
        <td>Alternaria oxytropis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>6-methylsalicyclic acid</td>
        
        
        <td>Aspergillus terreus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000043.1: 0-13097">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000043/index.html#r1c1" target="_new">BGC0000043.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.48, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>curacin A</td>
        
        
        <td>Moorea producens 19L</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275.1: 1063-6614">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>6-methylsalicyclic acid</td>
        
        
        <td>Glarea lozoyensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>asperlactone</td>
        
        
        <td>Aspergillus ochraceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998.1: 1175677-1189808">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>(-)-Mellein</td>
        
        
        <td>Parastagonospora nodorum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254.1: 82-7456">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Polyketide</td>
        
        <td>ACT-Toxin II</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r36c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000122.1: 1816-33971">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000122/index.html#r1c1" target="_new">BGC0000122.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>phenylnannolone A, phenylnannolone B, phenylnannolone C</td>
      <td>Nannocystis pusilla</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340.1: 1840-51432">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>byssochlamic acid</td>
      <td>Byssochlamys fulva</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001794.1: 3660-23178">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001794/index.html#r1c1" target="_new">BGC0001794.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.47, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>swainsonine</td>
      <td>Alternaria oxytropis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Aspergillus terreus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000043.1: 0-13097">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000043/index.html#r1c1" target="_new">BGC0000043.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.48, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>curacin A</td>
      <td>Moorea producens 19L</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275.1: 1063-6614">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Glarea lozoyensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>asperlactone</td>
      <td>Aspergillus ochraceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998.1: 1175677-1189808">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Polyketide</td>
      <td>(-)-Mellein</td>
      <td>Parastagonospora nodorum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254.1: 82-7456">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Polyketide</td>
      <td>ACT-Toxin II</td>
      <td>Alternaria alternata</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r36c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-182"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-182">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r36c2-pfam-details-svg" data-filename="c00036_CC55NOD.._r36c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r36c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r38c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00038_CC55NOD.. - Region 1 - redox-cofactor <div class="help-container"> <div class="help-icon" data-id="region_svg-help-272"></div> <span class="help-tooltip" id="region_svg-help-272">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00038_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r38c1-svg" data-filename="c00038_CC55NOD.._r38c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 140,124 - 162,166 nt. (total: 22,043 nt)
        
        <a class="cluster-rules-header" id="r38c1-rules-header" href="#r38c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r38c1-rules">redox-cofactor: (cds(PF04055 and (TIGR04085 or SPASM)) and (TIGR02111 or TIGR03966) and minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE]))</div>
      
      <div class="region-svg-container">
       <div id='r38c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-273"></div> <span class="help-tooltip" id="focus-panel-help-273">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r38c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r38c1-clusterblast" data-name="r38c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r38c1-knownclusterblast" data-name="r38c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r38c1-subclusterblast" data-name="r38c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r38c1-MIBiG-cluster-compare" data-name="r38c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r38c1-pfam-details" data-name="r38c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r38c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-183"></div> <span class="help-tooltip" id="cb-general-help-183">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r38c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r38c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r38c1_1.svg">
        NZ_CM001440 (2417778-2490285): Saccharomonospora cyanea NA-134 chromosome, wh... (5% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r38c1_2.svg">
        NZ_CM001484 (2127088-2199595): Saccharomonospora glauca K62 chromosome, whole... (5% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r38c1_3.svg">
        NZ_KB912942 (1559551-1582098): Amycolatopsis benzoatilytica AK 16/65 AmybeDRA... (18% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r38c1_4.svg">
        NZ_QJKF01000015 (47984-155231): Nocardia tenerifensis strain DSM 44704 Ga0244... (4% of genes show similarity), NRPS,phosphonate,terpene
      </option>
      
      <option value = "svg/clusterblast_r38c1_5.svg">
        NZ_CP034587 (467666-619676): Streptomyces luteoverticillatus strain CGMCC 150... (9% of genes show similarity), NRPS-like,T1PKS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r38c1_6.svg">
        NZ_CP007155 (3951723-4011898): Kutzneria albida DSM 43870 chromosome, complet... (8% of genes show similarity), LAP,T1PKS,lassopeptide,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r38c1_7.svg">
        NZ_VIVN01000022 (51990-81104): Neobacillus bataviensis strain 2482 Ga0310713 ... (12% of genes show similarity), LAP,bacteriocin,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r38c1_8.svg">
        NZ_LT985361 (5008717-5063929): Nocardia suismassiliense strain S-137, whole g... (8% of genes show similarity), CDPS,NRPS
      </option>
      
      <option value = "svg/clusterblast_r38c1_9.svg">
        NZ_FOFV01000001 (744814-808396): Lentzea albida strain DSM 44437, whole genom... (11% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r38c1_10.svg">
        NZ_CP046172 (4110814-4162368): Nocardia arthritidis strain AUSMDU00012717 chr... (7% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r38c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r38c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r38c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-184"></div> <span class="help-tooltip" id="cb-known-help-184">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r38c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r38c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r38c1_1.svg">
        lankacidin C
      </option>
      
    </select>
    <button id="knownclusterblast-r38c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r38c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r38c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-185"></div> <span class="help-tooltip" id="cb-sub-help-185">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r38c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-187"></div> <span class="help-tooltip" id="MIBiG-help-187">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r38c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r38c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r38c1"></div>
    <div id="comparison-MIBiG-r38c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">redox-cofactor</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>frankiamicin</td>
        
        
        <td>Frankia sp. EAN1pec</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000218.1: 27-8720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000218/index.html#r1c1" target="_new">BGC0000218.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.51)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>doxorubicin</td>
        
        
        <td>Streptomyces peucetius</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000217.1: 2-8089">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000217/index.html#r1c1" target="_new">BGC0000217.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>daunorubicin</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>griseusin</td>
        
        
        <td>Streptomyces griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888.1: 0-4342">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>bacilysin</td>
        
        
        <td>Bacillus sp. CS93</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000878.1: 0-9074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000878/index.html#r1c1" target="_new">BGC0000878.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>puromycin</td>
        
        
        <td>Streptomyces alboniger</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000607.1: 112-24009">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000607/index.html#r1c1" target="_new">BGC0000607.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>micrococcin P1</td>
        
        
        <td>Macrococcus caseolyticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000867.1: 21-5822">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000867/index.html#r1c1" target="_new">BGC0000867.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>polyhydroxyalkanoic acids</td>
        
        
        <td>Ectothiorhodospira shaposhnikovii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000509.1: 4195-15261">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000509/index.html#r1c1" target="_new">BGC0000509.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>RiPP</td>
        
        <td>epilancin 15x</td>
        
        
        <td>Staphylococcus epidermidis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000593.1: 241-2678">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000593/index.html#r1c1" target="_new">BGC0000593.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>RiPP</td>
        
        <td>microviridin J</td>
        
        
        <td>Microcystis aeruginosa MRC</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r38c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>frankiamicin</td>
      <td>Frankia sp. EAN1pec</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000218.1: 27-8720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000218/index.html#r1c1" target="_new">BGC0000218.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.51)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Polyketide</td>
      <td>doxorubicin</td>
      <td>Streptomyces peucetius</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000217.1: 2-8089">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000217/index.html#r1c1" target="_new">BGC0000217.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Polyketide</td>
      <td>daunorubicin</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Polyketide</td>
      <td>griseusin</td>
      <td>Streptomyces griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888.1: 0-4342">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>bacilysin</td>
      <td>Bacillus sp. CS93</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000878.1: 0-9074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000878/index.html#r1c1" target="_new">BGC0000878.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>puromycin</td>
      <td>Streptomyces alboniger</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000607.1: 112-24009">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000607/index.html#r1c1" target="_new">BGC0000607.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>micrococcin P1</td>
      <td>Macrococcus caseolyticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000867.1: 21-5822">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000867/index.html#r1c1" target="_new">BGC0000867.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>polyhydroxyalkanoic acids</td>
      <td>Ectothiorhodospira shaposhnikovii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000509.1: 4195-15261">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000509/index.html#r1c1" target="_new">BGC0000509.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>RiPP</td>
      <td>epilancin 15x</td>
      <td>Staphylococcus epidermidis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000593.1: 241-2678">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000593/index.html#r1c1" target="_new">BGC0000593.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>RiPP</td>
      <td>microviridin J</td>
      <td>Microcystis aeruginosa MRC</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r38c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-188"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-188">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r38c1-pfam-details-svg" data-filename="c00038_CC55NOD.._r38c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r38c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r38c1-RREfinder" data-name="r38c1-RREfinder">
        RREFinder
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r38c1-RREfinder">
      <div class="more-details">
  <div class="heading">
    <span>RRE predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="rre-side-help-186"></div> <span class="help-tooltip" id="rre-side-help-186">RREfinder results sidepanel.</span></div>
  </div>
  <dl class = 'prediction-text'>
    
      <span><strong>redox-cofactor protocluster</strong> (140123...162166)</span>
      <dt></dt>
      
      
      
        
        <strong>cds150815_151094</strong> <div class="collapser collapser-target-cds150815_151094 collapser-level-cds"> </div><div class="collapser-content">
        
          <dd>RRE type: PqqD_RRE</dd>
          <dd>Location: 3 - 89</dd>
          <dd>Score: 66.4</dd>
        
        </div><br>
      <br>
    
    </dl>
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r48c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00048_CC55NOD.. - Region 1 - betalactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-274"></div> <span class="help-tooltip" id="region_svg-help-274">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00048_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r48c1-svg" data-filename="c00048_CC55NOD.._r48c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 38,923 - 69,839 nt. (total: 30,917 nt)
        
        <a class="cluster-rules-header" id="r48c1-rules-header" href="#r48c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r48c1-rules">betalactone: (HMGL-like and AMP-binding)</div>
      
      <div class="region-svg-container">
       <div id='r48c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-275"></div> <span class="help-tooltip" id="focus-panel-help-275">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r48c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r48c1-clusterblast" data-name="r48c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r48c1-knownclusterblast" data-name="r48c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r48c1-subclusterblast" data-name="r48c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r48c1-MIBiG-cluster-compare" data-name="r48c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r48c1-pfam-details" data-name="r48c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r48c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-189"></div> <span class="help-tooltip" id="cb-general-help-189">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r48c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r48c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r48c1_1.svg">
        NZ_FNIX01000002 (528106-559052): Lentzea jiangxiensis strain CGMCC 4.6609, wh... (100% of genes show similarity), betalactone
      </option>
      
      <option value = "svg/clusterblast_r48c1_2.svg">
        NZ_FOFV01000012 (88555-119985): Lentzea albida strain DSM 44437, whole genome... (97% of genes show similarity), betalactone
      </option>
      
      <option value = "svg/clusterblast_r48c1_3.svg">
        NZ_QQAU01000009 (192249-223226): Lentzea flaviverrucosa strain DSM 44664 Ga02... (100% of genes show similarity), betalactone
      </option>
      
      <option value = "svg/clusterblast_r48c1_4.svg">
        NZ_FNET01000001 (633076-664053): Lentzea violacea strain DSM 44796, whole gen... (100% of genes show similarity), betalactone
      </option>
      
      <option value = "svg/clusterblast_r48c1_5.svg">
        NZ_FOYL01000002 (706883-738534): Lentzea waywayandensis strain DSM 44232, who... (97% of genes show similarity), betalactone
      </option>
      
      <option value = "svg/clusterblast_r48c1_6.svg">
        NZ_CP016793 (8101894-8132765): Lentzea guizhouensis strain DHS C013 chromosom... (96% of genes show similarity), betalactone
      </option>
      
      <option value = "svg/clusterblast_r48c1_7.svg">
        NZ_QGHB01000004 (228426-259808): Lechevalieria deserti strain DSM 45480 Ga021... (90% of genes show similarity), betalactone
      </option>
      
      <option value = "svg/clusterblast_r48c1_8.svg">
        NZ_QLTT01000001 (471273-502655): Lechevalieria atacamensis strain DSM 45479 G... (90% of genes show similarity), betalactone
      </option>
      
      <option value = "svg/clusterblast_r48c1_9.svg">
        NZ_AP022560 (1192791-1247455): Mycolicibacterium moriokaense strain JCM 6375 (52% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r48c1_10.svg">
        NZ_CM002285 (223342-251706): Streptomyces roseochromogenus subsp. oscitans DS... (79% of genes show similarity), betalactone
      </option>
      
    </select>
    <button id= "clusterblast-r48c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r48c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r48c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-190"></div> <span class="help-tooltip" id="cb-known-help-190">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r48c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r48c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r48c1_1.svg">
        platencin
      </option>
      
      <option value = "svg/knownclusterblast_r48c1_2.svg">
        platensimycin / platencin
      </option>
      
      <option value = "svg/knownclusterblast_r48c1_3.svg">
        thioplatensimycin / thioplatencin
      </option>
      
    </select>
    <button id="knownclusterblast-r48c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r48c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r48c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-191"></div> <span class="help-tooltip" id="cb-sub-help-191">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r48c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-192"></div> <span class="help-tooltip" id="MIBiG-help-192">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r48c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r48c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r48c1"></div>
    <div id="comparison-MIBiG-r48c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">betalactone</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Other</td>
        
        <td>showdomycin</td>
        
        
        <td>Streptomyces showdoensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001302.1: 0-25686">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001302/index.html#r1c1" target="_new">BGC0001302.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Other</td>
        
        <td>lomofungin</td>
        
        
        <td>Streptomyces lomondensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076.1: 161645-164819">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>Other</td>
        
        <td>4,5-Dehydroacaterin, Acaterin</td>
        
        
        <td>Pseudomonas jessenii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001177.1: 10040-36741">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001177/index.html#r1c1" target="_new">BGC0001177.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>mensacaricin</td>
        
        
        <td>Streptomyces bottropensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>frankiamicin</td>
        
        
        <td>Frankia sp. EAN1pec</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002034.1: 86568-105957">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002034/index.html#r1c1" target="_new">BGC0002034.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Other</td>
        
        <td>perquinoline A, perquinoline B, perquinoline C</td>
        
        
        <td>Streptomyces sp. IB2014 016-6</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000236.1: 35-34224">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000236/index.html#r1c1" target="_new">BGC0000236.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>kinamycin</td>
        
        
        <td>Streptomyces murayamaensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888.1: 0-4342">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Other</td>
        
        <td>bacilysin</td>
        
        
        <td>Bacillus sp. CS93</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001596.1: 5974061-6020749">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001596/index.html#r1c1" target="_new">BGC0001596.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>fluostatins M-Q</td>
        
        
        <td>Streptomyces albus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002000.1: 2056-14558">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002000/index.html#r1c1" target="_new">BGC0002000.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>setomimycin</td>
        
        
        <td>Streptomyces aurantiacus JA 4570</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r48c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other</td>
      <td>showdomycin</td>
      <td>Streptomyces showdoensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001302.1: 0-25686">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001302/index.html#r1c1" target="_new">BGC0001302.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>lomofungin</td>
      <td>Streptomyces lomondensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076.1: 161645-164819">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>4,5-Dehydroacaterin, Acaterin</td>
      <td>Pseudomonas jessenii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001177.1: 10040-36741">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001177/index.html#r1c1" target="_new">BGC0001177.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>mensacaricin</td>
      <td>Streptomyces bottropensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>frankiamicin</td>
      <td>Frankia sp. EAN1pec</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002034.1: 86568-105957">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002034/index.html#r1c1" target="_new">BGC0002034.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Other</td>
      <td>perquinoline A, perquinoline B, perquinoline C</td>
      <td>Streptomyces sp. IB2014 016-6</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000236.1: 35-34224">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000236/index.html#r1c1" target="_new">BGC0000236.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>kinamycin</td>
      <td>Streptomyces murayamaensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888.1: 0-4342">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Other</td>
      <td>bacilysin</td>
      <td>Bacillus sp. CS93</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001596.1: 5974061-6020749">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001596/index.html#r1c1" target="_new">BGC0001596.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>fluostatins M-Q</td>
      <td>Streptomyces albus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002000.1: 2056-14558">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002000/index.html#r1c1" target="_new">BGC0002000.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>setomimycin</td>
      <td>Streptomyces aurantiacus JA 4570</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r48c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-193"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-193">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r48c1-pfam-details-svg" data-filename="c00048_CC55NOD.._r48c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r48c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r48c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00048_CC55NOD.. - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-276"></div> <span class="help-tooltip" id="region_svg-help-276">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00048_CC55NOD...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r48c2-svg" data-filename="c00048_CC55NOD.._r48c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 92,689 - 113,762 nt. (total: 21,074 nt)
        
        <a class="cluster-rules-header" id="r48c2-rules-header" href="#r48c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r48c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r48c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-277"></div> <span class="help-tooltip" id="focus-panel-help-277">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r48c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r48c2-clusterblast" data-name="r48c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r48c2-knownclusterblast" data-name="r48c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r48c2-subclusterblast" data-name="r48c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r48c2-MIBiG-cluster-compare" data-name="r48c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r48c2-pfam-details" data-name="r48c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r48c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-194"></div> <span class="help-tooltip" id="cb-general-help-194">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r48c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r48c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r48c2_1.svg">
        NZ_FNET01000002 (279352-408642): Lentzea violacea strain DSM 44796, whole gen... (23% of genes show similarity), NRPS,arylpolyene,terpene
      </option>
      
      <option value = "svg/clusterblast_r48c2_2.svg">
        NZ_QQAU01000001 (647416-747806): Lentzea flaviverrucosa strain DSM 44664 Ga02... (30% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r48c2_3.svg">
        NZ_FWYC01000003 (421261-550656): Lentzea albidocapillata strain DSM 44073, wh... (21% of genes show similarity), NRPS,arylpolyene,terpene
      </option>
      
      <option value = "svg/clusterblast_r48c2_4.svg">
        NZ_FNIX01000002 (490912-511986): Lentzea jiangxiensis strain CGMCC 4.6609, wh... (58% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r48c2_5.svg">
        NZ_FOFR01000027 (10941-32024): Lechevalieria xinjiangensis strain CGMCC 4.352... (73% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r48c2_6.svg">
        NZ_FOYL01000002 (764316-785381): Lentzea waywayandensis strain DSM 44232, who... (68% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r48c2_7.svg">
        NZ_FOFV01000012 (151400-172465): Lentzea albida strain DSM 44437, whole genom... (70% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r48c2_8.svg">
        NZ_JAAMPJ010000010 (63759-84824): Lentzea sp. NEAU-D13 Scaffold10, whole geno... (64% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r48c2_9.svg">
        NZ_BBOJ01000012 (189208-210273): Lentzea aerocolonigenes strain NBRC 13195, w... (50% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r48c2_10.svg">
        NZ_CP016793 (6601561-6620559): Lentzea guizhouensis strain DHS C013 chromosom... (50% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r48c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r48c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r48c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-195"></div> <span class="help-tooltip" id="cb-known-help-195">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r48c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r48c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r48c2_1.svg">
        2-methylisoborneol
      </option>
      
    </select>
    <button id="knownclusterblast-r48c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r48c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r48c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-196"></div> <span class="help-tooltip" id="cb-sub-help-196">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r48c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-197"></div> <span class="help-tooltip" id="MIBiG-help-197">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r48c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r48c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r48c2"></div>
    <div id="comparison-MIBiG-r48c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000659.1: 4075162-4077099">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000659/index.html#r1c1" target="_new">BGC0000659.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.53, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
        <td>0.42</td>
        <td>Terpene</td>
        
        <td>2-methylisoborneol</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000658.1: 1490670-1492896">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000658/index.html#r1c1" target="_new">BGC0000658.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.51, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>Terpene</td>
        
        <td>2-methylisoborneol</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000657.1: 643-5774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000657/index.html#r1c1" target="_new">BGC0000657.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.55, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>Terpene</td>
        
        <td>2-methylisoborneol</td>
        
        
        <td>Pseudanabaena sp. dqh15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>auriculamide</td>
        
        
        <td>Herpetosiphon aurantiacus DSM 785</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843.1: 3437627-3461293">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
        
        <td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001783.1: 55-63934">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001783/index.html#r1c1" target="_new">BGC0001783.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>streptonigrin</td>
        
        
        <td>Streptomyces flocculus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001735.1: 0-39550">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001735/index.html#r1c1" target="_new">BGC0001735.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Other</td>
        
        <td>pentostatine, vidarabine</td>
        
        
        <td>Streptomyces antibioticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001940.1: 108-33418">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001940/index.html#r1c1" target="_new">BGC0001940.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
        <td>0.11</td>
        <td>Polyketide</td>
        
        <td>pyxidicycline A, pyxidicycline B</td>
        
        
        <td>Pyxidicoccus fallax</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001810.1: 122-16114">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001810/index.html#r1c1" target="_new">BGC0001810.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.36 (id:0.49, order:0.27, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
         
        
        <td>0.10</td>
        <td>Polyketide</td>
        
        <td>trehangelin</td>
        
        
        <td>Polymorphospora rubra</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r48c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000659.1: 4075162-4077099">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000659/index.html#r1c1" target="_new">BGC0000659.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.53, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>Terpene</td>
      <td>2-methylisoborneol</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000658.1: 1490670-1492896">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000658/index.html#r1c1" target="_new">BGC0000658.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.51, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>Terpene</td>
      <td>2-methylisoborneol</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000657.1: 643-5774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000657/index.html#r1c1" target="_new">BGC0000657.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.55, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Terpene</td>
      <td>2-methylisoborneol</td>
      <td>Pseudanabaena sp. dqh15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>auriculamide</td>
      <td>Herpetosiphon aurantiacus DSM 785</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843.1: 3437627-3461293">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
      <td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001783.1: 55-63934">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001783/index.html#r1c1" target="_new">BGC0001783.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Other</td>
      <td>streptonigrin</td>
      <td>Streptomyces flocculus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001735.1: 0-39550">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001735/index.html#r1c1" target="_new">BGC0001735.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Other</td>
      <td>pentostatine, vidarabine</td>
      <td>Streptomyces antibioticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001940.1: 108-33418">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001940/index.html#r1c1" target="_new">BGC0001940.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
      <td>0.38</td>
      <td>Polyketide</td>
      <td>pyxidicycline A, pyxidicycline B</td>
      <td>Pyxidicoccus fallax</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001810.1: 122-16114">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001810/index.html#r1c1" target="_new">BGC0001810.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.36 (id:0.49, order:0.27, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
      <td>0.36</td>
      <td>Polyketide</td>
      <td>trehangelin</td>
      <td>Polymorphospora rubra</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r48c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-198"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-198">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r48c2-pfam-details-svg" data-filename="c00048_CC55NOD.._r48c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r48c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r48c3' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00048_CC55NOD.. - Region 3 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-278"></div> <span class="help-tooltip" id="region_svg-help-278">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00048_CC55NOD...region003.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r48c3-svg" data-filename="c00048_CC55NOD.._r48c3.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 139,992 - 199,919 nt. (total: 59,928 nt)
        
        <a class="cluster-rules-header" id="r48c3-rules-header" href="#r48c3">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r48c3-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r48c3-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-279"></div> <span class="help-tooltip" id="focus-panel-help-279">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r48c3">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r48c3-nrps_pks" data-name="r48c3-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r48c3-clusterblast" data-name="r48c3-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r48c3-knownclusterblast" data-name="r48c3-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r48c3-subclusterblast" data-name="r48c3-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r48c3-MIBiG-cluster-compare" data-name="r48c3-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r48c3-pfam-details" data-name="r48c3-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r48c3-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-199"></div> <span class="help-tooltip" id="nrps-domain-help-199">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r48c3-details-svg" data-filename="c00048_CC55NOD.._r48c3_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r48c3-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r48c3-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-202"></div> <span class="help-tooltip" id="cb-general-help-202">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r48c3-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r48c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r48c3_1.svg">
        NZ_FNIX01000002 (392025-464690): Lentzea jiangxiensis strain CGMCC 4.6609, wh... (46% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r48c3_2.svg">
        NZ_FNET01000002 (279352-408642): Lentzea violacea strain DSM 44796, whole gen... (16% of genes show similarity), NRPS,arylpolyene,terpene
      </option>
      
      <option value = "svg/clusterblast_r48c3_3.svg">
        NZ_FWYC01000003 (421261-550656): Lentzea albidocapillata strain DSM 44073, wh... (17% of genes show similarity), NRPS,arylpolyene,terpene
      </option>
      
      <option value = "svg/clusterblast_r48c3_4.svg">
        NZ_QLTT01000003 (533738-612412): Lechevalieria atacamensis strain DSM 45479 G... (23% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r48c3_5.svg">
        NZ_FOYL01000002 (50583-128329): Lentzea waywayandensis strain DSM 44232, whol... (23% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r48c3_6.svg">
        NZ_QQAU01000001 (647416-747806): Lentzea flaviverrucosa strain DSM 44664 Ga02... (13% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r48c3_7.svg">
        NZ_BBOJ01000006 (302903-440011): Lentzea aerocolonigenes strain NBRC 13195, w... (13% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r48c3_8.svg">
        NZ_CP047144 (5928875-6009287): Streptomyces sp. HF10 chromosome, complete genome (11% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r48c3_9.svg">
        NZ_LT629701 (2429365-2651872): Allokutzneria albata strain DSM 44149 chromoso... (6% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r48c3_10.svg">
        NZ_JADG01000003 (334520-483052): Actinomadura oligospora ATCC 43269 P696DRAFT... (12% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r48c3-download">Download graphic</button>
  </div>
  <div id="clusterblast-r48c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r48c3-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-203"></div> <span class="help-tooltip" id="cb-known-help-203">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r48c3-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r48c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r48c3_1.svg">
        BD-12
      </option>
      
      <option value = "svg/knownclusterblast_r48c3_2.svg">
        streptothricin
      </option>
      
      <option value = "svg/knownclusterblast_r48c3_3.svg">
        mannopeptimycin
      </option>
      
    </select>
    <button id="knownclusterblast-r48c3-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r48c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r48c3-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-204"></div> <span class="help-tooltip" id="cb-sub-help-204">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r48c3-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-205"></div> <span class="help-tooltip" id="MIBiG-help-205">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r48c3-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r48c3-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r48c3"></div>
    <div id="comparison-MIBiG-r48c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.44, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
        <td>0.42</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001843.1: 2526145-2583676">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001843/index.html#r1c1" target="_new">BGC0001843.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.45, order:0.88, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>NRP</td>
        
        <td>tridecaptin M</td>
        
        
        <td>Paenibacillus sp. M-152</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:0.69, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>exochelin</td>
        
        
        <td>Mycolicibacterium smegmatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000438.1: 137-73725">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000438/index.html#r1c1" target="_new">BGC0000438.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.43, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>syringopeptin 25A</td>
        
        
        <td>Pseudomonas syringae pv. syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001502.1: 2741865-2757479">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001502/index.html#r1c1" target="_new">BGC0001502.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>amonabactin P 750</td>
        
        
        <td>Aeromonas hydrophila subsp. hydrophila ATCC 7966</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001629.1: 209-63961">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001629/index.html#r1c1" target="_new">BGC0001629.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.42, order:0.83, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>Jessenipeptin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
        
        
        <td>Penicillium rubens Wisconsin 54-1255</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>rhodochelin</td>
        
        
        <td>Rhodococcus jostii RHA1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r48c3-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.44, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001843.1: 2526145-2583676">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001843/index.html#r1c1" target="_new">BGC0001843.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.45, order:0.88, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>NRP</td>
      <td>tridecaptin M</td>
      <td>Paenibacillus sp. M-152</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.45, order:0.69, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>exochelin</td>
      <td>Mycolicibacterium smegmatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000438.1: 137-73725">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000438/index.html#r1c1" target="_new">BGC0000438.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.43, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>syringopeptin 25A</td>
      <td>Pseudomonas syringae pv. syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001502.1: 2741865-2757479">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001502/index.html#r1c1" target="_new">BGC0001502.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>amonabactin P 750</td>
      <td>Aeromonas hydrophila subsp. hydrophila ATCC 7966</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001629.1: 209-63961">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001629/index.html#r1c1" target="_new">BGC0001629.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.42, order:0.83, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>Jessenipeptin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
      <td>Penicillium rubens Wisconsin 54-1255</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>rhodochelin</td>
      <td>Rhodococcus jostii RHA1</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r48c3-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-206"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-206">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r48c3-pfam-details-svg" data-filename="c00048_CC55NOD.._r48c3_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r48c3-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r48c3-nrps_pks_products" data-name="r48c3-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r48c3-nrps_pks" data-name="r48c3-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r48c3-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-200"></div> <span class="help-tooltip" id="nrps-structures-help-200">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 139991 - 199919:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r48c3-3" class="smiles-canvas" data-smiles="NC(CCCN)C(=O)NC(CCCN)C(=O)NC(CCCN)C(=O)NC(CCCN)C(=O)NC(CCCN)C(=O)NC(CCCN)C(=O)NC(CC(C)C)C(=O)NC(CC(C)C)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-orn - D-orn - D-orn - D-orn) + (D-orn - D-orn) + (D-leu - leu - D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-orn,d-orn,d-orn,d-orn&amp;nrps2=d-orn,d-orn&amp;nrps3=d-leu,leu,d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-orn*,d-orn*,d-orn*,d-orn*&amp;nrps2=d-orn*,d-orn*&amp;nrps3=d-leu*,leu*,d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r48c3-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CCCN)C(=O)NC(CCCN)C(=O)NC(CCCN)C(=O)NC(CCCN)C(=O)NC(CCCN)C(=O)NC(CCCN)C(=O)NC(CC(C)C)C(=O)NC(CC(C)C)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r48c3-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-201"></div> <span class="help-tooltip" id="nrps-monomers-help-201">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds159991_171784</span></strong>: leu - leu - X
      <div class="collapser collapser-target-cds159991_171784 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu,leu,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*,leu*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (460..847)</strong>: leu
          <div class="collapser collapser-target-cds159991_171784 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  leu
              <div class="collapser collapser-target-cds159991_171784-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAWFLGmVyK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1922..2308)</strong>: leu
          <div class="collapser collapser-target-cds159991_171784 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  leu
              <div class="collapser collapser-target-cds159991_171784-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAWFLGmVyK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2949..3334)</strong>: X
          <div class="collapser collapser-target-cds159991_171784 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala, val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds159991_171784-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DtFLvGamtK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phg</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds171999_180141</span></strong>: orn - orn
      <div class="collapser collapser-target-cds171999_180141 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn,orn" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn*,orn*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (222..606)</strong>: orn
          <div class="collapser collapser-target-cds171999_180141 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn
              <div class="collapser collapser-target-cds171999_180141-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>orn, horn</dd>
   <dt>Single AA prediction:</dt>
   <dd>orn</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVGEIGvIDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1687..2075)</strong>: orn
          <div class="collapser collapser-target-cds171999_180141 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn
              <div class="collapser collapser-target-cds171999_180141-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>orn, horn</dd>
   <dt>Single AA prediction:</dt>
   <dd>orn</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVGEIGvIDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds180137_198380</span></strong>: orn - orn - orn - orn
      <div class="collapser collapser-target-cds180137_198380 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn,orn,orn,orn" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn*,orn*,orn*,orn*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (439..839)</strong>: orn
          <div class="collapser collapser-target-cds180137_198380 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn
              <div class="collapser collapser-target-cds180137_198380-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>orn, horn</dd>
   <dt>Single AA prediction:</dt>
   <dd>orn</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVGEIGvIDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1925..2313)</strong>: orn
          <div class="collapser collapser-target-cds180137_198380 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn
              <div class="collapser collapser-target-cds180137_198380-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVGEIGvIDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3396..3781)</strong>: orn
          <div class="collapser collapser-target-cds180137_198380 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn
              <div class="collapser collapser-target-cds180137_198380-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>orn, horn</dd>
   <dt>Single AA prediction:</dt>
   <dd>orn</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVGEIGvIDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (4855..5245)</strong>: orn
          <div class="collapser collapser-target-cds180137_198380 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn
              <div class="collapser collapser-target-cds180137_198380-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>orn, horn</dd>
   <dt>Single AA prediction:</dt>
   <dd>orn</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVGEIGvIDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r54c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00054_CC55NOD.. - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-280"></div> <span class="help-tooltip" id="region_svg-help-280">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00054_CC55NOD...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r54c1-svg" data-filename="c00054_CC55NOD.._r54c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 35,700 - 79,563 nt. (total: 43,864 nt)
        
        <a class="cluster-rules-header" id="r54c1-rules-header" href="#r54c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r54c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r54c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-281"></div> <span class="help-tooltip" id="focus-panel-help-281">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r54c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r54c1-nrps_pks" data-name="r54c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r54c1-clusterblast" data-name="r54c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r54c1-knownclusterblast" data-name="r54c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r54c1-subclusterblast" data-name="r54c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r54c1-MIBiG-cluster-compare" data-name="r54c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r54c1-pfam-details" data-name="r54c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r54c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-207"></div> <span class="help-tooltip" id="nrps-domain-help-207">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r54c1-details-svg" data-filename="c00054_CC55NOD.._r54c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r54c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r54c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-210"></div> <span class="help-tooltip" id="cb-general-help-210">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r54c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r54c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r54c1_1.svg">
        NZ_FNIX01000006 (177513-221377): Lentzea jiangxiensis strain CGMCC 4.6609, wh... (69% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r54c1_2.svg">
        NZ_QQAU01000010 (211728-255592): Lentzea flaviverrucosa strain DSM 44664 Ga02... (56% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r54c1_3.svg">
        NZ_FNCC01000006 (406206-450070): Lechevalieria fradiae strain CGMCC 4.3506, w... (53% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r54c1_4.svg">
        NZ_FOFV01000014 (65906-109770): Lentzea albida strain DSM 44437, whole genome... (59% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r54c1_5.svg">
        NZ_FOYL01000006 (103079-146943): Lentzea waywayandensis strain DSM 44232, who... (52% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r54c1_6.svg">
        NZ_FWYC01000013 (291000-335099): Lentzea albidocapillata strain DSM 44073, wh... (46% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r54c1_7.svg">
        NZ_FNET01000010 (19791-62685): Lentzea violacea strain DSM 44796, whole genom... (47% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r54c1_8.svg">
        NZ_CP016793 (1314195-1370140): Lentzea guizhouensis strain DHS C013 chromosom... (28% of genes show similarity), NRPS-like,betalactone,other
      </option>
      
      <option value = "svg/clusterblast_r54c1_9.svg">
        NZ_AP022871 (561656-628024): Phytohabitans suffuscus strain NBRC 105367 (8% of genes show similarity), NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r54c1_10.svg">
        NZ_CP034539 (226201-313417): Streptomyces cyaneochromogenes strain MK-45 chro... (10% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r54c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r54c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r54c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-211"></div> <span class="help-tooltip" id="cb-known-help-211">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r54c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r54c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r54c1_1.svg">
        indigoidine
      </option>
      
      <option value = "svg/knownclusterblast_r54c1_2.svg">
        glycinocin A
      </option>
      
      <option value = "svg/knownclusterblast_r54c1_3.svg">
        friulimicin A / friulimicin B / friulimicin C / friulimicin D
      </option>
      
    </select>
    <button id="knownclusterblast-r54c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r54c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r54c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-212"></div> <span class="help-tooltip" id="cb-sub-help-212">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r54c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-213"></div> <span class="help-tooltip" id="MIBiG-help-213">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r54c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r54c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r54c1"></div>
    <div id="comparison-MIBiG-r54c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.74 (id:0.55, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
         
        
        <td>0.43</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Other</td>
        
        <td>ferrichrome</td>
        
        
        <td>Aspergillus oryzae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876.1: 71-9235">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Other</td>
        
        <td>neopolyoxin C</td>
        
        
        <td>Streptomyces tendae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 10E9</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 22D11</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001717.1: 2898-22560">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001717/index.html#r1c1" target="_new">BGC0001717.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.22</td>
        <td>Alkaloid</td>
        
        <td>okaramine B</td>
        
        
        <td>Penicillium simplicissimum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718.1: 582791-598007">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>okaramine D</td>
        
        
        <td>Aspergillus aculeatus ATCC 16872</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000145.1: 1-32637">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000145/index.html#r1c1" target="_new">BGC0000145.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Polyketide</td>
        
        <td>salinosporamide A</td>
        
        
        <td>Salinispora tropica</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r54c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.74 (id:0.55, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
      <td>0.74</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>ferrichrome</td>
      <td>Aspergillus oryzae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876.1: 71-9235">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Other</td>
      <td>neopolyoxin C</td>
      <td>Streptomyces tendae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 10E9</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 22D11</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001717.1: 2898-22560">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001717/index.html#r1c1" target="_new">BGC0001717.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Alkaloid</td>
      <td>okaramine B</td>
      <td>Penicillium simplicissimum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718.1: 582791-598007">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>okaramine D</td>
      <td>Aspergillus aculeatus ATCC 16872</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000145.1: 1-32637">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000145/index.html#r1c1" target="_new">BGC0000145.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>salinosporamide A</td>
      <td>Salinispora tropica</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r54c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-214"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-214">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r54c1-pfam-details-svg" data-filename="c00054_CC55NOD.._r54c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r54c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r54c1-nrps_pks_products" data-name="r54c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r54c1-nrps_pks" data-name="r54c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r54c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-208"></div> <span class="help-tooltip" id="nrps-structures-help-208">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 35699 - 79563:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r54c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r54c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r54c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-209"></div> <span class="help-tooltip" id="nrps-monomers-help-209">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds55699_59563</span></strong>: X
      <div class="collapser collapser-target-cds55699_59563 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (16..406)</strong>: X
          <div class="collapser collapser-target-cds55699_59563 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds55699_59563-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAWQFGLInK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r54c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00054_CC55NOD.. - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-282"></div> <span class="help-tooltip" id="region_svg-help-282">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00054_CC55NOD...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r54c2-svg" data-filename="c00054_CC55NOD.._r54c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 96,938 - 119,082 nt. (total: 22,145 nt)
        
        <a class="cluster-rules-header" id="r54c2-rules-header" href="#r54c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r54c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r54c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-283"></div> <span class="help-tooltip" id="focus-panel-help-283">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r54c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r54c2-clusterblast" data-name="r54c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r54c2-knownclusterblast" data-name="r54c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r54c2-subclusterblast" data-name="r54c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r54c2-MIBiG-cluster-compare" data-name="r54c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r54c2-pfam-details" data-name="r54c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r54c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-215"></div> <span class="help-tooltip" id="cb-general-help-215">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r54c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r54c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r54c2_1.svg">
        NZ_FOFV01000014 (138399-160514): Lentzea albida strain DSM 44437, whole genom... (85% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r54c2_2.svg">
        NZ_FOFR01000022 (23924-46063): Lechevalieria xinjiangensis strain CGMCC 4.352... (78% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r54c2_3.svg">
        NZ_FNET01000010 (89251-111384): Lentzea violacea strain DSM 44796, whole geno... (70% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r54c2_4.svg">
        NZ_QLTT01000005 (497310-519488): Lechevalieria atacamensis strain DSM 45479 G... (70% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r54c2_5.svg">
        NZ_FWYC01000013 (242786-264943): Lentzea albidocapillata strain DSM 44073, wh... (68% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r54c2_6.svg">
        NZ_FNCC01000006 (369058-391206): Lechevalieria fradiae strain CGMCC 4.3506, w... (70% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r54c2_7.svg">
        NZ_FOYL01000006 (175034-197188): Lentzea waywayandensis strain DSM 44232, who... (77% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r54c2_8.svg">
        NZ_QGHB01000011 (231780-253958): Lechevalieria deserti strain DSM 45480 Ga021... (73% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r54c2_9.svg">
        NZ_QQAU01000010 (164569-186714): Lentzea flaviverrucosa strain DSM 44664 Ga02... (52% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r54c2_10.svg">
        NZ_CP016793 (1265738-1285054): Lentzea guizhouensis strain DHS C013 chromosom... (50% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r54c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r54c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r54c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-216"></div> <span class="help-tooltip" id="cb-known-help-216">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r54c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r54c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r54c2_1.svg">
        geosmin
      </option>
      
    </select>
    <button id="knownclusterblast-r54c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r54c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r54c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-217"></div> <span class="help-tooltip" id="cb-sub-help-217">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r54c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-218"></div> <span class="help-tooltip" id="MIBiG-help-218">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r54c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r54c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r54c2"></div>
    <div id="comparison-MIBiG-r54c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001181.1: 6666218-6668399">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001181/index.html#r1c1" target="_new">BGC0001181.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.52, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.41</td>
        <td>Terpene</td>
        
        <td>geosmin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.51, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>Terpene</td>
        
        <td>geosmin</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000918.1: 0-4994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000918/index.html#r1c1" target="_new">BGC0000918.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Other</td>
        
        <td>ochronotic pigment</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294.1: 26867-35886">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>7-prenylisatin</td>
        
        
        <td>Streptomyces sp. MBT28</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001803.1: 3663-28345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001803/index.html#r1c1" target="_new">BGC0001803.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>RiPP</td>
        
        <td>thioviridamide</td>
        
        
        <td>Streptomyces olivoviridis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000625.1: 3663-28345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000625/index.html#r1c1" target="_new">BGC0000625.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>RiPP</td>
        
        <td>thioviridamide</td>
        
        
        <td>Streptomyces olivoviridis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001387.1: 150-29437">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001387/index.html#r1c1" target="_new">BGC0001387.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>nucleocidin</td>
        
        
        <td>Streptomyces calvus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
        <td>0.12</td>
        <td>Other</td>
        
        <td>5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester</td>
        
        
        <td>Streptomyces sp. RM-5-8</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001366.1: 10-15268">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001366/index.html#r1c1" target="_new">BGC0001366.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
        <td>0.11</td>
        <td>Polyketide</td>
        
        <td>BE-24566B, zunyimycin A</td>
        
        
        <td>Streptomyces sp. FJS31-2</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.46, order:0.36, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
        <td>0.11</td>
        <td>Polyketide</td>
        
        <td>frankiamicin</td>
        
        
        <td>Frankia sp. EAN1pec</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r54c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001181.1: 6666218-6668399">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001181/index.html#r1c1" target="_new">BGC0001181.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.52, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Terpene</td>
      <td>geosmin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.51, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>Terpene</td>
      <td>geosmin</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000918.1: 0-4994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000918/index.html#r1c1" target="_new">BGC0000918.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Other</td>
      <td>ochronotic pigment</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294.1: 26867-35886">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>7-prenylisatin</td>
      <td>Streptomyces sp. MBT28</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001803.1: 3663-28345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001803/index.html#r1c1" target="_new">BGC0001803.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>RiPP</td>
      <td>thioviridamide</td>
      <td>Streptomyces olivoviridis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000625.1: 3663-28345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000625/index.html#r1c1" target="_new">BGC0000625.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>RiPP</td>
      <td>thioviridamide</td>
      <td>Streptomyces olivoviridis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001387.1: 150-29437">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001387/index.html#r1c1" target="_new">BGC0001387.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>nucleocidin</td>
      <td>Streptomyces calvus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
      <td>0.40</td>
      <td>Other</td>
      <td>5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester</td>
      <td>Streptomyces sp. RM-5-8</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001366.1: 10-15268">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001366/index.html#r1c1" target="_new">BGC0001366.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
      <td>0.38</td>
      <td>Polyketide</td>
      <td>BE-24566B, zunyimycin A</td>
      <td>Streptomyces sp. FJS31-2</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.38 (id:0.46, order:0.36, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
      <td>0.38</td>
      <td>Polyketide</td>
      <td>frankiamicin</td>
      <td>Frankia sp. EAN1pec</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r54c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-219"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-219">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r54c2-pfam-details-svg" data-filename="c00054_CC55NOD.._r54c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r54c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>


  <footer class="footer">
    <div class="container">
      <div>
        <img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
      </div>
      <div class="cite-me">
        If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
      </div>
      <div>
        <img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
      </div>
    </div>
  </footer>

  <script src="js/jquery.js"></script>
  <script src="js/antismash.js"></script>
  <script src="regions.js"></script>
  <script>
    $(document).ready(function() {
        viewer["start"](all_regions, details_data, resultsData, recordData);
    })
  </script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink">
 <defs>
  <filter id="inset-shadow">
   <feOffset dx="-2" dy="-2"></feOffset>
   <feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
   <feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
   <feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
   <feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
   <feComponentTransfer in="shadow" result="shadow">
    <feFuncA type="linear" slope=".95"></feFuncA>
   </feComponentTransfer>
   <feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
  </filter>
 </defs>
</svg>
</body>
</html>