
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>6666666.540227 - 33 region(s) - antiSMASH results</title>
  <style>img {width: 20px; height: 20px;}</style>
  <link rel="stylesheet" type="text/css" href="css/bacteria.css">
  <meta property="og:title" content="6666666.540227 - 21 record(s) - 33 region(s)" />
  <meta property="og:description" content="">
  <meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
  <meta property="og:image:alt" content="antiSMASH logo" />
  <meta property="og:image:width" content="600" />
  <meta property="og:image:height" content="600" />
  <meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-fa6b285f-3542-4fef-8d20-bb2cc7c749bd/index.html" />
  <meta property="og:type" content="website" />
  <meta name="twitter:card" content="summary" />
  <meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
  <div id="header">
      <nav>
    <div class="main-link">
     <div>
        <a href="https://antismash.secondarymetabolites.org/">
          <img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
        </a>
       </div>
       <div>
        <a href="https://antismash.secondarymetabolites.org/">
          antiSMASH version 6.0.1
        </a>
      </div>
    </div>
    <div class="custom-description"></div>
    <div class="ancillary-links">
      <div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
        <ul class="dropdown-options">
        
        <li><a href="6666666.540227.zip">Download all results</a></li>
        
          <li><a href="6666666.540227.gbk">Download GenBank summary file</a></li>
        
          <li><a href="bacteria-fa6b285f-3542-4fef-8d20-bb2cc7c749bd.log">Download log file</a></li>
        
        </ul>
      </div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
      <div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
    </div>
  </nav>
    <div id="buttons">
      <span id="region-type">Select genomic region:</span>
      <div id="regionbuttons">
        <div class="arrow-left" id="prev-region"></div>
        <div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
        <div style="display: flex; flex-wrap: wrap">
        
          
            
            
                
            
            <div class="regbutton NRPS,T1PKS hybrid r1c1">
              <a href="#r1c1">1.1</a>
            </div>
          
            
            
            <div class="regbutton siderophore r1c2">
              <a href="#r1c2">1.2</a>
            </div>
          
            
            
            <div class="regbutton T1PKS r1c3">
              <a href="#r1c3">1.3</a>
            </div>
          
        
          
            
            
            <div class="regbutton hglE-KS r3c1">
              <a href="#r3c1">3.1</a>
            </div>
          
            
            
            <div class="regbutton terpene r3c2">
              <a href="#r3c2">3.2</a>
            </div>
          
            
            
            <div class="regbutton terpene r3c3">
              <a href="#r3c3">3.3</a>
            </div>
          
            
            
                
            
            <div class="regbutton T3PKS,arylpolyene hybrid r3c4">
              <a href="#r3c4">3.4</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r4c1">
              <a href="#r4c1">4.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton RiPP-like r6c1">
              <a href="#r6c1">6.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r8c1">
              <a href="#r8c1">8.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r13c1">
              <a href="#r13c1">13.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r16c1">
              <a href="#r16c1">16.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton ectoine r17c1">
              <a href="#r17c1">17.1</a>
            </div>
          
            
            
                
            
            <div class="regbutton NRPS,NRPS-like,terpene hybrid r17c2">
              <a href="#r17c2">17.2</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r18c1">
              <a href="#r18c1">18.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS-like r21c1">
              <a href="#r21c1">21.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r23c1">
              <a href="#r23c1">23.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r24c1">
              <a href="#r24c1">24.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS-like r26c1">
              <a href="#r26c1">26.1</a>
            </div>
          
            
            
            <div class="regbutton terpene r26c2">
              <a href="#r26c2">26.2</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r33c1">
              <a href="#r33c1">33.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r37c1">
              <a href="#r37c1">37.1</a>
            </div>
          
            
            
            <div class="regbutton NRPS r37c2">
              <a href="#r37c2">37.2</a>
            </div>
          
            
            
                
            
            <div class="regbutton NRPS,NRPS-like hybrid r37c3">
              <a href="#r37c3">37.3</a>
            </div>
          
            
            
            <div class="regbutton terpene r37c4">
              <a href="#r37c4">37.4</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,terpene hybrid r39c1">
              <a href="#r39c1">39.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,T2PKS hybrid r40c1">
              <a href="#r40c1">40.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r42c1">
              <a href="#r42c1">42.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r43c1">
              <a href="#r43c1">43.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r44c1">
              <a href="#r44c1">44.1</a>
            </div>
          
            
            
            <div class="regbutton T2PKS r44c2">
              <a href="#r44c2">44.2</a>
            </div>
          
            
            
            <div class="regbutton NAPAA r44c3">
              <a href="#r44c3">44.3</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r47c1">
              <a href="#r47c1">47.1</a>
            </div>
          
        
        </div>
        <div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
      </div>
    </div>
  </div>

  <!-- overview page -->
  
  <div class="page" id="overview">
   
   
     <h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>
   
   <div class="overview-layout">
    <div id="record-tables">
     <div id="single-record-tables">
     <div class="record-overview-header">
        <strong>CC227C_contig_5</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-1" data-filename="CC227C_contig_5_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-1" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r1c1">
     
     
         
     
     <td class="regbutton NRPS,T1PKS hybrid r1c1">
      <a href="#r1c1">Region&nbsp1.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">205,026</td>
     <td class="digits table-split-left">281,512</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001046/1" target="_blank">streptolydigin</a></td>
      <td>NRP + Polyketide:Modular type I + Saccharide:Hybrid/tailoring</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 7%, #ffffff00 7%)">7%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r1c2">
     
     
     <td class="regbutton siderophore r1c2">
      <a href="#r1c2">Region&nbsp1.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">379,292</td>
     <td class="digits table-split-left">391,100</td>
     
      <td colspan="3"></td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r1c3">
     
     
     <td class="regbutton T1PKS r1c3">
      <a href="#r1c3">Region&nbsp1.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">425,463</td>
     <td class="digits table-split-left">471,804</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000100/1" target="_blank">monensin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 5%, #ffffff00 5%)">5%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_1</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-3" data-filename="CC227C_contig_1_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-3" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r3c1">
     
     
     <td class="regbutton hglE-KS r3c1">
      <a href="#r3c1">Region&nbsp3.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hgle-ks" target="_blank">hglE-KS</a>
     </td>
     <td class="digits">62,765</td>
     <td class="digits table-split-left">109,052</td>
     
      <td colspan="3"></td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r3c2">
     
     
     <td class="regbutton terpene r3c2">
      <a href="#r3c2">Region&nbsp3.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">353,564</td>
     <td class="digits table-split-left">373,100</td>
     
      <td colspan="3"></td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r3c3">
     
     
     <td class="regbutton terpene r3c3">
      <a href="#r3c3">Region&nbsp3.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">420,374</td>
     <td class="digits table-split-left">441,501</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000633/1" target="_blank">carotenoid</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 27%, #ffffff00 27%)">27%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r3c4">
     
     
         
     
     <td class="regbutton T3PKS,arylpolyene hybrid r3c4">
      <a href="#r3c4">Region&nbsp3.4</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#arylpolyene" target="_blank">arylpolyene</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">630,687</td>
     <td class="digits table-split-left">686,697</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_23</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-4" data-filename="CC227C_contig_23_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-4" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r4c1">
     
     
     <td class="regbutton T1PKS r4c1">
      <a href="#r4c1">Region&nbsp4.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">44,137</td>
     <td class="digits table-split-left">90,481</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000093/1" target="_blank">meilingmycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 2%, #ffffff00 2%)">2%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_4</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-6" data-filename="CC227C_contig_4_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-6" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r6c1">
     
     
     <td class="regbutton RiPP-like r6c1">
      <a href="#r6c1">Region&nbsp6.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">549,784</td>
     <td class="digits table-split-left">555,719</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_8</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-8" data-filename="CC227C_contig_8_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-8" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r8c1">
     
     
     <td class="regbutton NRPS r8c1">
      <a href="#r8c1">Region&nbsp8.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">35,696</td>
     <td class="digits table-split-left">89,420</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_15</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-13" data-filename="CC227C_contig_15_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-13" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r13c1">
     
     
     <td class="regbutton NRPS r13c1">
      <a href="#r13c1">Region&nbsp13.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">15,666</td>
     <td class="digits table-split-left">69,320</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000324/1" target="_blank">coelibactin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 27%, #ffffff00 27%)">27%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_18</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-16" data-filename="CC227C_contig_18_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-16" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r16c1">
     
     
     <td class="regbutton NRPS r16c1">
      <a href="#r16c1">Region&nbsp16.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">40,481</td>
     <td class="digits table-split-left">84,770</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000366/1" target="_blank">gobichelin A / gobichelin B</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 11%, #ffffff00 11%)">11%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_9</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-17" data-filename="CC227C_contig_9_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-17" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r17c1">
     
     
     <td class="regbutton ectoine r17c1">
      <a href="#r17c1">Region&nbsp17.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">114,521</td>
     <td class="digits table-split-left">124,916</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000853/1" target="_blank">ectoine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r17c2">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like,terpene hybrid r17c2">
      <a href="#r17c2">Region&nbsp17.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">167,098</td>
     <td class="digits table-split-left">253,921</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_35</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-18" data-filename="CC227C_contig_35_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-18" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r18c1">
     
     
     <td class="regbutton NRPS r18c1">
      <a href="#r18c1">Region&nbsp18.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">10,719</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_20</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-21" data-filename="CC227C_contig_20_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-21" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r21c1">
     
     
     <td class="regbutton NRPS-like r21c1">
      <a href="#r21c1">Region&nbsp21.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">19,012</td>
     <td class="digits table-split-left">61,864</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_33</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-23" data-filename="CC227C_contig_33_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-23" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r23c1">
     
     
     <td class="regbutton NRPS r23c1">
      <a href="#r23c1">Region&nbsp23.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">23,361</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001227/1" target="_blank">isorenieratene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_11</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-24" data-filename="CC227C_contig_11_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-24" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r24c1">
     
     
     <td class="regbutton T1PKS r24c1">
      <a href="#r24c1">Region&nbsp24.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">53,665</td>
     <td class="digits table-split-left">98,929</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_14</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-26" data-filename="CC227C_contig_14_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-26" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r26c1">
     
     
     <td class="regbutton NRPS-like r26c1">
      <a href="#r26c1">Region&nbsp26.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">27,896</td>
     
      <td colspan="3"></td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r26c2">
     
     
     <td class="regbutton terpene r26c2">
      <a href="#r26c2">Region&nbsp26.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">114,144</td>
     <td class="digits table-split-left">135,208</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_6</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-33" data-filename="CC227C_contig_6_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-33" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r33c1">
     
     
     <td class="regbutton NRPS r33c1">
      <a href="#r33c1">Region&nbsp33.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">330,481</td>
     <td class="digits table-split-left">384,106</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000033/1" target="_blank">calicheamicin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_2</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-37" data-filename="CC227C_contig_2_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-37" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r37c1">
     
     
     <td class="regbutton NRPS r37c1">
      <a href="#r37c1">Region&nbsp37.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">213</td>
     <td class="digits table-split-left">45,810</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001058/1" target="_blank">zorbamycin</a></td>
      <td>NRP:Glycopeptide + Polyketide:Modular type I + Saccharide:Hybrid/tailoring</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r37c2">
     
     
     <td class="regbutton NRPS r37c2">
      <a href="#r37c2">Region&nbsp37.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">191,810</td>
     <td class="digits table-split-left">239,251</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000371/1" target="_blank">heterobactin A / heterobactin S2</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 63%, #ffffff00 63%)">63%</td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r37c3">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like hybrid r37c3">
      <a href="#r37c3">Region&nbsp37.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">307,896</td>
     <td class="digits table-split-left">439,197</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001537/1" target="_blank">butyrolactol A</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 20%, #ffffff00 20%)">20%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r37c4">
     
     
     <td class="regbutton terpene r37c4">
      <a href="#r37c4">Region&nbsp37.4</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">518,305</td>
     <td class="digits table-split-left">539,420</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_3</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-39" data-filename="CC227C_contig_3_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-39" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r39c1">
     
     
         
     
     <td class="regbutton NRPS,terpene hybrid r39c1">
      <a href="#r39c1">Region&nbsp39.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">86,207</td>
     <td class="digits table-split-left">182,976</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001975/1" target="_blank">atratumycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 5%, #ffffff00 5%)">5%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_22</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-40" data-filename="CC227C_contig_22_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-40" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r40c1">
     
     
         
     
     <td class="regbutton NRPS,T2PKS hybrid r40c1">
      <a href="#r40c1">Region&nbsp40.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">99,304</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000227/1" target="_blank">granaticin</a></td>
      <td>Polyketide:Type II</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 24%, #ffffff00 24%)">24%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_17</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-42" data-filename="CC227C_contig_17_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-42" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r42c1">
     
     
     <td class="regbutton T1PKS r42c1">
      <a href="#r42c1">Region&nbsp42.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">40,335</td>
     <td class="digits table-split-left">83,400</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_12</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-43" data-filename="CC227C_contig_12_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-43" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r43c1">
     
     
     <td class="regbutton NRPS r43c1">
      <a href="#r43c1">Region&nbsp43.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">93,964</td>
     <td class="digits table-split-left">138,142</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_7</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-44" data-filename="CC227C_contig_7_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-44" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r44c1">
     
     
     <td class="regbutton terpene r44c1">
      <a href="#r44c1">Region&nbsp44.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">213,702</td>
     <td class="digits table-split-left">234,718</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000269/1" target="_blank">SF2575</a></td>
      <td>Polyketide:Type II + Saccharide:Hybrid/tailoring</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r44c2">
     
     
     <td class="regbutton T2PKS r44c2">
      <a href="#r44c2">Region&nbsp44.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">236,267</td>
     <td class="digits table-split-left">308,671</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000220/1" target="_blank">enterocin</a></td>
      <td>Polyketide:Type II</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r44c3">
     
     
     <td class="regbutton NAPAA r44c3">
      <a href="#r44c3">Region&nbsp44.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
     </td>
     <td class="digits">343,733</td>
     <td class="digits table-split-left">377,761</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC227C_contig_30</strong> (Nocardia sp. CC227C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-47" data-filename="CC227C_contig_30_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-47" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r47c1">
     
     
     <td class="regbutton NRPS r47c1">
      <a href="#r47c1">Region&nbsp47.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">39,227</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div>
     </div>
     
     <div id="compact-record-table" style="display: none;">
      <div class="record-table">
       <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 
 
  <tr class="linked-row odd" data-anchor="#r1c1">
     
     
         
     
     <td class="regbutton NRPS,T1PKS hybrid r1c1">
      <a href="#r1c1">Region&nbsp1.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">205,026</td>
     <td class="digits table-split-left">281,512</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001046/1" target="_blank">streptolydigin</a></td>
      <td>NRP + Polyketide:Modular type I + Saccharide:Hybrid/tailoring</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 7%, #ffffff00 7%)">7%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r1c2">
     
     
     <td class="regbutton siderophore r1c2">
      <a href="#r1c2">Region&nbsp1.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">379,292</td>
     <td class="digits table-split-left">391,100</td>
     
      <td colspan="3"></td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r1c3">
     
     
     <td class="regbutton T1PKS r1c3">
      <a href="#r1c3">Region&nbsp1.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">425,463</td>
     <td class="digits table-split-left">471,804</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000100/1" target="_blank">monensin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 5%, #ffffff00 5%)">5%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r3c1">
     
     
     <td class="regbutton hglE-KS r3c1">
      <a href="#r3c1">Region&nbsp3.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hgle-ks" target="_blank">hglE-KS</a>
     </td>
     <td class="digits">62,765</td>
     <td class="digits table-split-left">109,052</td>
     
      <td colspan="3"></td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r3c2">
     
     
     <td class="regbutton terpene r3c2">
      <a href="#r3c2">Region&nbsp3.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">353,564</td>
     <td class="digits table-split-left">373,100</td>
     
      <td colspan="3"></td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r3c3">
     
     
     <td class="regbutton terpene r3c3">
      <a href="#r3c3">Region&nbsp3.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">420,374</td>
     <td class="digits table-split-left">441,501</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000633/1" target="_blank">carotenoid</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 27%, #ffffff00 27%)">27%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r3c4">
     
     
         
     
     <td class="regbutton T3PKS,arylpolyene hybrid r3c4">
      <a href="#r3c4">Region&nbsp3.4</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#arylpolyene" target="_blank">arylpolyene</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">630,687</td>
     <td class="digits table-split-left">686,697</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r4c1">
     
     
     <td class="regbutton T1PKS r4c1">
      <a href="#r4c1">Region&nbsp4.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">44,137</td>
     <td class="digits table-split-left">90,481</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000093/1" target="_blank">meilingmycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 2%, #ffffff00 2%)">2%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r6c1">
     
     
     <td class="regbutton RiPP-like r6c1">
      <a href="#r6c1">Region&nbsp6.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">549,784</td>
     <td class="digits table-split-left">555,719</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r8c1">
     
     
     <td class="regbutton NRPS r8c1">
      <a href="#r8c1">Region&nbsp8.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">35,696</td>
     <td class="digits table-split-left">89,420</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r13c1">
     
     
     <td class="regbutton NRPS r13c1">
      <a href="#r13c1">Region&nbsp13.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">15,666</td>
     <td class="digits table-split-left">69,320</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000324/1" target="_blank">coelibactin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 27%, #ffffff00 27%)">27%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r16c1">
     
     
     <td class="regbutton NRPS r16c1">
      <a href="#r16c1">Region&nbsp16.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">40,481</td>
     <td class="digits table-split-left">84,770</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000366/1" target="_blank">gobichelin A / gobichelin B</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 11%, #ffffff00 11%)">11%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r17c1">
     
     
     <td class="regbutton ectoine r17c1">
      <a href="#r17c1">Region&nbsp17.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">114,521</td>
     <td class="digits table-split-left">124,916</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000853/1" target="_blank">ectoine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r17c2">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like,terpene hybrid r17c2">
      <a href="#r17c2">Region&nbsp17.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">167,098</td>
     <td class="digits table-split-left">253,921</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r18c1">
     
     
     <td class="regbutton NRPS r18c1">
      <a href="#r18c1">Region&nbsp18.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">10,719</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r21c1">
     
     
     <td class="regbutton NRPS-like r21c1">
      <a href="#r21c1">Region&nbsp21.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">19,012</td>
     <td class="digits table-split-left">61,864</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r23c1">
     
     
     <td class="regbutton NRPS r23c1">
      <a href="#r23c1">Region&nbsp23.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">23,361</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001227/1" target="_blank">isorenieratene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r24c1">
     
     
     <td class="regbutton T1PKS r24c1">
      <a href="#r24c1">Region&nbsp24.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">53,665</td>
     <td class="digits table-split-left">98,929</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r26c1">
     
     
     <td class="regbutton NRPS-like r26c1">
      <a href="#r26c1">Region&nbsp26.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">27,896</td>
     
      <td colspan="3"></td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r26c2">
     
     
     <td class="regbutton terpene r26c2">
      <a href="#r26c2">Region&nbsp26.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">114,144</td>
     <td class="digits table-split-left">135,208</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r33c1">
     
     
     <td class="regbutton NRPS r33c1">
      <a href="#r33c1">Region&nbsp33.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">330,481</td>
     <td class="digits table-split-left">384,106</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000033/1" target="_blank">calicheamicin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r37c1">
     
     
     <td class="regbutton NRPS r37c1">
      <a href="#r37c1">Region&nbsp37.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">213</td>
     <td class="digits table-split-left">45,810</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001058/1" target="_blank">zorbamycin</a></td>
      <td>NRP:Glycopeptide + Polyketide:Modular type I + Saccharide:Hybrid/tailoring</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r37c2">
     
     
     <td class="regbutton NRPS r37c2">
      <a href="#r37c2">Region&nbsp37.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">191,810</td>
     <td class="digits table-split-left">239,251</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000371/1" target="_blank">heterobactin A / heterobactin S2</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 63%, #ffffff00 63%)">63%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r37c3">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like hybrid r37c3">
      <a href="#r37c3">Region&nbsp37.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">307,896</td>
     <td class="digits table-split-left">439,197</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001537/1" target="_blank">butyrolactol A</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 20%, #ffffff00 20%)">20%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r37c4">
     
     
     <td class="regbutton terpene r37c4">
      <a href="#r37c4">Region&nbsp37.4</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">518,305</td>
     <td class="digits table-split-left">539,420</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r39c1">
     
     
         
     
     <td class="regbutton NRPS,terpene hybrid r39c1">
      <a href="#r39c1">Region&nbsp39.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">86,207</td>
     <td class="digits table-split-left">182,976</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001975/1" target="_blank">atratumycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 5%, #ffffff00 5%)">5%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r40c1">
     
     
         
     
     <td class="regbutton NRPS,T2PKS hybrid r40c1">
      <a href="#r40c1">Region&nbsp40.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">99,304</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000227/1" target="_blank">granaticin</a></td>
      <td>Polyketide:Type II</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 24%, #ffffff00 24%)">24%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r42c1">
     
     
     <td class="regbutton T1PKS r42c1">
      <a href="#r42c1">Region&nbsp42.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">40,335</td>
     <td class="digits table-split-left">83,400</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r43c1">
     
     
     <td class="regbutton NRPS r43c1">
      <a href="#r43c1">Region&nbsp43.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">93,964</td>
     <td class="digits table-split-left">138,142</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r44c1">
     
     
     <td class="regbutton terpene r44c1">
      <a href="#r44c1">Region&nbsp44.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">213,702</td>
     <td class="digits table-split-left">234,718</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000269/1" target="_blank">SF2575</a></td>
      <td>Polyketide:Type II + Saccharide:Hybrid/tailoring</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r44c2">
     
     
     <td class="regbutton T2PKS r44c2">
      <a href="#r44c2">Region&nbsp44.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">236,267</td>
     <td class="digits table-split-left">308,671</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000220/1" target="_blank">enterocin</a></td>
      <td>Polyketide:Type II</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r44c3">
     
     
     <td class="regbutton NAPAA r44c3">
      <a href="#r44c3">Region&nbsp44.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
     </td>
     <td class="digits">343,733</td>
     <td class="digits table-split-left">377,761</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r47c1">
     
     
     <td class="regbutton NRPS r47c1">
      <a href="#r47c1">Region&nbsp47.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">39,227</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
 </tbody>
</table>
      </div>
     </div>
     
     
     <dl id="regionless-records">
      <dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>
       
         <dd><strong>CC227C_contig_39</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_45</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_37</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_47</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_40</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_38</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_29</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_10</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_13</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_32</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_28</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_42</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_19</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_31</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_26</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_36</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_43</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_27</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_44</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_34</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_46</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_25</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_41</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_16</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_21</strong> (Nocardia sp. CC227C)</dd>
       
         <dd><strong>CC227C_contig_24</strong> (Nocardia sp. CC227C)</dd>
       
     </dl>
     
    </div>
    
    <div class="overview-switches">
     <div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    
   </div>
  </div>
  






<div class="page" id='r1c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_5 - Region 1 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-231"></div> <span class="help-tooltip" id="region_svg-help-231">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_5.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1c1-svg" data-filename="CC227C_contig_5_r1c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 205,026 - 281,512 nt. (total: 76,487 nt)
        
        <a class="cluster-rules-header" id="r1c1-rules-header" href="#r1c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r1c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r1c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-232"></div> <span class="help-tooltip" id="focus-panel-help-232">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1c1-nrps_pks" data-name="r1c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r1c1-clusterblast" data-name="r1c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1c1-knownclusterblast" data-name="r1c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1c1-subclusterblast" data-name="r1c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1c1-MIBiG-cluster-compare" data-name="r1c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1c1-pfam-details" data-name="r1c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-1"></div> <span class="help-tooltip" id="nrps-domain-help-1">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1c1-details-svg" data-filename="CC227C_contig_5_r1c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r1c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r1c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-4"></div> <span class="help-tooltip" id="cb-general-help-4">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1c1_1.svg">
        NZ_CP041695 (5214817-5262074): Nocardia otitidiscaviarum strain NEB252 chromo... (80% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c1_2.svg">
        NZ_CP022088 (2427001-2578997): Nocardia brasiliensis strain FDAARGOS 352 chro... (18% of genes show similarity), NRPS,NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c1_3.svg">
        NC_018681 (4143514-4296483): Nocardia brasiliensis ATCC 700358, complete sequ... (17% of genes show similarity), NRPS,NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c1_4.svg">
        NZ_QJKF01000001 (601453-773852): Nocardia tenerifensis strain DSM 44704 Ga024... (15% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r1c1_5.svg">
        NZ_CP046171 (3657253-3753862): Nocardia brasiliensis strain AUSMDU00024985 ch... (21% of genes show similarity), NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c1_6.svg">
        NZ_CP029710 (609462-693459): Nocardia sp. CS682 chromosome, complete genome (26% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c1_7.svg">
        NZ_LT985361 (5223174-5306577): Nocardia suismassiliense strain S-137, whole g... (25% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c1_8.svg">
        NZ_CP029710 (770702-906407): Nocardia sp. CS682 chromosome, complete genome (20% of genes show similarity), PKS-like,T1PKS,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r1c1_9.svg">
        NZ_CP018082 (3265443-3314905): Nocardia mangyaensis strain Y48 chromosome, co... (25% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c1_10.svg">
        NZ_LT985361 (5085654-5171615): Nocardia suismassiliense strain S-137, whole g... (18% of genes show similarity), NRPS,T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r1c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-5"></div> <span class="help-tooltip" id="cb-known-help-5">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r1c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r1c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r1c1_1.svg">
        streptolydigin
      </option>
      
      <option value = "svg/knownclusterblast_r1c1_2.svg">
        foxicins A-D
      </option>
      
      <option value = "svg/knownclusterblast_r1c1_3.svg">
        friulimicin A / friulimicin B / friulimicin C / friulimicin D
      </option>
      
    </select>
    <button id="knownclusterblast-r1c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r1c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-6"></div> <span class="help-tooltip" id="cb-sub-help-6">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r1c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r1c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r1c1_1.svg">
        FN433113_c1: streptolydigin methylasparagine (75% of genes show similarity), methylasparagine
      </option>
      
    </select>
    <button id="subclusterblast-r1c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r1c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-7"></div> <span class="help-tooltip" id="MIBiG-help-7">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1c1"></div>
    <div id="comparison-MIBiG-r1c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007.1: 5922480-5942549">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.53</td>
        <td>Terpene</td>
        
        <td>atolypene A, atolypene B</td>
        
        
        <td>Amycolatopsis tolypomycina</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000907.1: 0-38075">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000907/index.html#r1c1" target="_new">BGC0000907.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.49</td>
        <td>Other</td>
        
        <td>lincomycin</td>
        
        
        <td>Streptomyces lincolnensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000632.1: 2720626-2736748">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000632/index.html#r1c1" target="_new">BGC0000632.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.49</td>
        <td>Terpene, Saccharide</td>
        
        <td>brasilicardin A</td>
        
        
        <td>Nocardia terpenica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001336.1: 2595-15215">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001336/index.html#r1c1" target="_new">BGC0001336.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.49</td>
        <td>Alkaloid</td>
        
        <td>erdasporine A, erdasporine B, erdasporine C</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001441.1: 0-27983">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001441/index.html#r1c1" target="_new">BGC0001441.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.48</td>
        <td>Other</td>
        
        <td>belactosin A, belactosin C</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000825.1: 52-22475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000825/index.html#r1c1" target="_new">BGC0000825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.46</td>
        <td>Alkaloid</td>
        
        <td>staurosporine</td>
        
        
        <td>Streptomyces sp. TP-A0274</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002042.1: 1539-27652">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002042/index.html#r1c1" target="_new">BGC0002042.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.45</td>
        <td>NRP</td>
        
        <td>anthelvencin A, anthelvencin B, anthelvencin C</td>
        
        
        <td>Streptomyces venezuelae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000827.1: 2672472-2696772">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000827/index.html#r1c1" target="_new">BGC0000827.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.44</td>
        <td>Alkaloid</td>
        
        <td>staurosporine</td>
        
        
        <td>Salinispora arenicola CNS-205</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.44</td>
        <td>NRP</td>
        
        <td>pseudomonine</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001546.1: 0-10917">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001546/index.html#r1c1" target="_new">BGC0001546.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.44</td>
        <td>Alkaloid</td>
        
        <td>chuangxinmycin</td>
        
        
        <td>Actinoplanes tsinanensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001337.1: 3106-25639">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001337/index.html#r1c1" target="_new">BGC0001337.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Alkaloid</td>
      <td>lazarimide A, lazarimide B, lazarimide C</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000653.1: 0-21693">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000653/index.html#r1c1" target="_new">BGC0000653.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces arenae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
      <td>Parastagonospora nodorum SN15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000924.1: 136-5698">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000924/index.html#r1c1" target="_new">BGC0000924.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>pyrrolnitrin</td>
      <td>Pseudomonas chlororaphis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001405.1: 2447653-2507157">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001405/index.html#r1c1" target="_new">BGC0001405.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>chaetoviridin E, 11-epichaetomugilin A</td>
      <td>Chaetomium globosum CBS 148.51</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002000.1: 2056-14558">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002000/index.html#r1c1" target="_new">BGC0002000.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>setomimycin</td>
      <td>Streptomyces aurantiacus JA 4570</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001904.1: 3058-37159">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001904/index.html#r1c1" target="_new">BGC0001904.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>fluostatin</td>
      <td>Micromonospora rosaria</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>betaenone C, betaenone A</td>
      <td>Phoma betae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>betaenone A, betaenone B, betaenone C</td>
      <td>Phoma betae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Terpene, Polyketide</td>
      <td>pyripyropene A</td>
      <td>unidentified unclassified sequences.</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-8"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-8">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1c1-pfam-details-svg" data-filename="CC227C_contig_5_r1c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r1c1-nrps_pks_products" data-name="r1c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r1c1-nrps_pks" data-name="r1c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r1c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-2"></div> <span class="help-tooltip" id="nrps-structures-help-2">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 205025 - 281512:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r1c1-1" class="smiles-canvas" data-smiles="C(C)=CC([*])C(-O)"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(Me-ccmal) + (pk)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-ccmal&amp;nrps2=pk" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-ccmal*&amp;nrps2=pk*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r1c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C(C)=CC([*])C(-O)"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r1c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-3"></div> <span class="help-tooltip" id="nrps-monomers-help-3">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds224496_232944</span></strong>: pk
      <div class="collapser collapser-target-cds224496_232944 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (553..850)</strong>: pk
          <div class="collapser collapser-target-cds224496_232944 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds224496_232944-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>CE-Malonyl-CoA: 62.5%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds224496_232944-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 129.7</dt>

  <dd></dd><dt>Malonyl-CoA: 119.5</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 105.3</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 104.5</dt>

  <dd></dd><dt>Isobutyryl-CoA: 90.0</dt>

  <dd></dd><dt>trans-1,2-CPDA: 72.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 69.9</dt>

  <dd></dd><dt>Propionyl-CoA: 67.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 57.1</dt>

  <dd></dd><dt>Acetyl-CoA: 47.2</dt>

  <dd></dd><dt>CHC-CoA: 42.0</dt>

  <dd></dd><dt>inactive: 36.0</dt>

  <dd></dd><dt>fatty_acid: 34.9</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 33.5</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds250211_261512</span></strong>: pk - mmal
      <div class="collapser collapser-target-cds250211_261512 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>CAL_domain (38..474)</strong>: pk
          <div class="collapser collapser-target-cds250211_261512 collapser-level-candidate"> </div><div class="collapser-content">
          
              minowa:  fatty_acid
              <div class="collapser collapser-target-cds250211_261512-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>fatty_acid: 17.5</dt>

  <dd></dd><dt>NH2: 10.5</dt>

  <dd></dd><dt>AHBA: 0.0</dt>

  <dd></dd><dt>shikimic_acid: -1.4</dt>

  <dd></dd><dt>Acetyl-CoA: -2.9</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (1348..1644)</strong>: mmal
          <div class="collapser collapser-target-cds250211_261512 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methoxymalonyl-CoA, Methylmalonyl-CoA
              <div class="collapser collapser-target-cds250211_261512-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methoxymalonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 75.0%</dd>
<dd>Ethylmalonyl-CoA: 70.8%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds250211_261512-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 138.2</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 135.8</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 132.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 83.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 71.5</dt>

  <dd></dd><dt>Propionyl-CoA: 68.7</dt>

  <dd></dd><dt>Malonyl-CoA: 61.0</dt>

  <dd></dd><dt>Acetyl-CoA: 49.9</dt>

  <dd></dd><dt>CHC-CoA: 47.2</dt>

  <dd></dd><dt>Isobutyryl-CoA: 36.8</dt>

  <dd></dd><dt>Benzoyl-CoA: 34.4</dt>

  <dd></dd><dt>fatty_acid: 24.3</dt>

  <dd></dd><dt>inactive: 19.5</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 13.2</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r1c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_5 - Region 2 - siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-233"></div> <span class="help-tooltip" id="region_svg-help-233">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_5.region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1c2-svg" data-filename="CC227C_contig_5_r1c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 379,292 - 391,100 nt. (total: 11,809 nt)
        
        <a class="cluster-rules-header" id="r1c2-rules-header" href="#r1c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r1c2-rules">siderophore: IucA_IucC</div>
      
      <div class="region-svg-container">
       <div id='r1c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-234"></div> <span class="help-tooltip" id="focus-panel-help-234">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1c2-clusterblast" data-name="r1c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1c2-knownclusterblast" data-name="r1c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1c2-subclusterblast" data-name="r1c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1c2-MIBiG-cluster-compare" data-name="r1c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1c2-pfam-details" data-name="r1c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-9"></div> <span class="help-tooltip" id="cb-general-help-9">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1c2_1.svg">
        NZ_CP041695 (5341365-5389155): Nocardia otitidiscaviarum strain NEB252 chromo... (16% of genes show similarity), NRPS,siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c2_2.svg">
        NZ_CP018082 (3763505-&gt;3862386): Nocardia mangyaensis strain Y48 chromosome, c... (5% of genes show similarity), NRPS,siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c2_3.svg">
        NZ_CP049836 (4889822-4952908): Gordonia terrae strain RL-JC02 chromosome, com... (7% of genes show similarity), NRPS,siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c2_4.svg">
        NZ_CP047235 (642792-706509): Gordonia sp. JH63 chromosome, complete genome (7% of genes show similarity), NRPS,siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c2_5.svg">
        NZ_WXYI01000001 (331170-394073): Gordonia sp. YY1 BPODLACK 1, whole genome sh... (7% of genes show similarity), NRPS,siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c2_6.svg">
        NZ_MVKV01000001 (237350-300894): Gordonia sp. IITR100 Scaffold 1, whole genom... (7% of genes show similarity), NRPS,siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c2_7.svg">
        NZ_CP023405 (493385-562212): Gordonia sp. 1D chromosome (6% of genes show similarity), NRPS,siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c2_8.svg">
        NZ_MKKG01000033 (260304-325147): Gordonia sp. CNJ-863 NODE 3 length 369250 co... (7% of genes show similarity), NRPS,siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c2_9.svg">
        NZ_CP046257 (583564-645371): Gordonia sp. 135 chromosome, complete genome (7% of genes show similarity), NRPS,siderophore
      </option>
      
      <option value = "svg/clusterblast_r1c2_10.svg">
        NZ_CP022580 (611517-673907): Gordonia rubripertincta strain CWB2 chromosome, ... (7% of genes show similarity), NRPS,siderophore
      </option>
      
    </select>
    <button id= "clusterblast-r1c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-10"></div> <span class="help-tooltip" id="cb-known-help-10">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r1c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-11"></div> <span class="help-tooltip" id="cb-sub-help-11">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-12"></div> <span class="help-tooltip" id="MIBiG-help-12">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1c2"></div>
    <div id="comparison-MIBiG-r1c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">siderophore</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000941.1: 5578821-5584948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000941/index.html#r1c1" target="_new">BGC0000941.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>Other</td>
        
        <td>desferrioxamin B</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453.1: 0-7234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.49, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Other</td>
        
        <td>desferrioxamine</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940.1: 3033354-3040682">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.49, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Other</td>
        
        <td>desferrioxamin B, desferrioxamine E</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001478.1: 100-5365">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001478/index.html#r1c1" target="_new">BGC0001478.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.48, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Streptomyces sp. ID38640</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.48, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Pantoea agglomerans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001531.1: 1112-9953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001531/index.html#r1c1" target="_new">BGC0001531.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.47, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Other</td>
        
        <td>bisucaberin B</td>
        
        
        <td>Tenacibaculum mesophilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499.1: 2267-11578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>aerobactin</td>
        
        
        <td>Pantoea ananatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>ochrobactin</td>
        
        
        <td>Ochrobactrum anthropi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000942.1: 1864728-1871818">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000942/index.html#r1c1" target="_new">BGC0000942.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>petrobactin</td>
        
        
        <td>Bacillus anthracis str. Ames</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000295.1: 569-12865">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000295/index.html#r1c1" target="_new">BGC0000295.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.43, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>NRP</td>
        
        <td>acinetoferrin</td>
        
        
        <td>Acinetobacter haemolyticus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000941.1: 5578821-5584948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000941/index.html#r1c1" target="_new">BGC0000941.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>desferrioxamin B</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453.1: 0-7234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.49, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Other</td>
      <td>desferrioxamine</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940.1: 3033354-3040682">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.49, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Other</td>
      <td>desferrioxamin B, desferrioxamine E</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001478.1: 100-5365">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001478/index.html#r1c1" target="_new">BGC0001478.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.48, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Streptomyces sp. ID38640</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.48, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Pantoea agglomerans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001531.1: 1112-9953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001531/index.html#r1c1" target="_new">BGC0001531.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.47, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Other</td>
      <td>bisucaberin B</td>
      <td>Tenacibaculum mesophilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499.1: 2267-11578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Other</td>
      <td>aerobactin</td>
      <td>Pantoea ananatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>ochrobactin</td>
      <td>Ochrobactrum anthropi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000942.1: 1864728-1871818">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000942/index.html#r1c1" target="_new">BGC0000942.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Other</td>
      <td>petrobactin</td>
      <td>Bacillus anthracis str. Ames</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000295.1: 569-12865">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000295/index.html#r1c1" target="_new">BGC0000295.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.43, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>NRP</td>
      <td>acinetoferrin</td>
      <td>Acinetobacter haemolyticus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-13"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-13">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1c2-pfam-details-svg" data-filename="CC227C_contig_5_r1c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r1c3' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_5 - Region 3 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-235"></div> <span class="help-tooltip" id="region_svg-help-235">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_5.region003.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1c3-svg" data-filename="CC227C_contig_5_r1c3.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 425,463 - 471,804 nt. (total: 46,342 nt)
        
        <a class="cluster-rules-header" id="r1c3-rules-header" href="#r1c3">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r1c3-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r1c3-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-236"></div> <span class="help-tooltip" id="focus-panel-help-236">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1c3">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1c3-nrps_pks" data-name="r1c3-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r1c3-clusterblast" data-name="r1c3-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1c3-knownclusterblast" data-name="r1c3-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1c3-subclusterblast" data-name="r1c3-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1c3-MIBiG-cluster-compare" data-name="r1c3-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1c3-pfam-details" data-name="r1c3-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1c3-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-14"></div> <span class="help-tooltip" id="nrps-domain-help-14">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1c3-details-svg" data-filename="CC227C_contig_5_r1c3_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r1c3-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r1c3-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-17"></div> <span class="help-tooltip" id="cb-general-help-17">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1c3-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1c3_1.svg">
        NZ_CP041695 (5420101-5464995): Nocardia otitidiscaviarum strain NEB252 chromo... (80% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c3_2.svg">
        NZ_CP032568 (2268306-2324037): Nocardia sp. CFHS0054 chromosome (24% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c3_3.svg">
        NZ_CP017839 (2413578-2459929): Nocardia seriolae strain EM150506 chromosome, ... (27% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c3_4.svg">
        NZ_CP059399 (7914166-7960523): Nocardia sp. WCH-YHL-001 chromosome, complete ... (27% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c3_5.svg">
        NZ_SNXK01000002 (352920-503468): Nocardia ignorata strain DSM 44496 Ga0244500... (4% of genes show similarity), NRPS,T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r1c3_6.svg">
        NZ_AP022569 (1446632-1581504): Mycobacterium cookii strain JCM 12404 (6% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c3_7.svg">
        NC_016887 (3477801-3523919): Nocardia cyriacigeorgica GUH-2, complete genome (10% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c3_8.svg">
        NZ_CP006850 (5145309-5194926): Nocardia nova SH22a chromosome, complete genome (19% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c3_9.svg">
        NZ_LR215973 (1853625-1898576): Nocardia cyriacigeorgica strain 3012STDY675650... (8% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c3_10.svg">
        NZ_CP026746 (4428350-4474156): Nocardia cyriacigeorgica strain MDA3349 isolat... (7% of genes show similarity), T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r1c3-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c3-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-18"></div> <span class="help-tooltip" id="cb-known-help-18">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r1c3-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r1c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r1c3_1.svg">
        monensin
      </option>
      
    </select>
    <button id="knownclusterblast-r1c3-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r1c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c3-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-19"></div> <span class="help-tooltip" id="cb-sub-help-19">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1c3-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-20"></div> <span class="help-tooltip" id="MIBiG-help-20">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1c3-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1c3-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1c3"></div>
    <div id="comparison-MIBiG-r1c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000003.1: 12401-36790">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000003/index.html#r1c1" target="_new">BGC0000003.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Polyketide</td>
        
        <td>AF-toxin</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
        
        
        <td>Parastagonospora nodorum SN15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>betaenone C, betaenone A</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>betaenone A, betaenone B, betaenone C</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Polyketide</td>
        
        <td>asperlactone</td>
        
        
        <td>Aspergillus ochraceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Polyketide</td>
        
        <td>(-)-Mellein</td>
        
        
        <td>Parastagonospora nodorum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998.1: 1175677-1189808">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Polyketide</td>
        
        <td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400.1: 371433-386839">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>Polyketide</td>
        
        <td>citreoviridin</td>
        
        
        <td>Aspergillus terreus NIH2624</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001281.1: 8011-344234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001281/index.html#r1c1" target="_new">BGC0001281.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>ustilagic acid</td>
        
        
        <td>Ustilago maydis 521</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001141.1: 262-37660">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001141/index.html#r1c1" target="_new">BGC0001141.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>4-epi-15-epi-brefeldin A</td>
        
        
        <td>Penicillium brefeldianum</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1c3-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000003.1: 12401-36790">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000003/index.html#r1c1" target="_new">BGC0000003.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>AF-toxin</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
      <td>Parastagonospora nodorum SN15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>betaenone C, betaenone A</td>
      <td>Phoma betae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>betaenone A, betaenone B, betaenone C</td>
      <td>Phoma betae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>asperlactone</td>
      <td>Aspergillus ochraceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>(-)-Mellein</td>
      <td>Parastagonospora nodorum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998.1: 1175677-1189808">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400.1: 371433-386839">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Polyketide</td>
      <td>citreoviridin</td>
      <td>Aspergillus terreus NIH2624</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001281.1: 8011-344234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001281/index.html#r1c1" target="_new">BGC0001281.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>ustilagic acid</td>
      <td>Ustilago maydis 521</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001141.1: 262-37660">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001141/index.html#r1c1" target="_new">BGC0001141.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>4-epi-15-epi-brefeldin A</td>
      <td>Penicillium brefeldianum</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1c3-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-21"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-21">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1c3-pfam-details-svg" data-filename="CC227C_contig_5_r1c3_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1c3-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r1c3-nrps_pks_products" data-name="r1c3-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r1c3-nrps_pks" data-name="r1c3-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r1c3-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-15"></div> <span class="help-tooltip" id="nrps-structures-help-15">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 425462 - 471804:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r1c3-3" class="smiles-canvas" data-smiles="CCC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(redmal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=redmal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=redmal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r1c3-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CCC(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r1c3-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-16"></div> <span class="help-tooltip" id="nrps-monomers-help-16">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds445462_451804</span></strong>: mal
      <div class="collapser collapser-target-cds445462_451804 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (528..824)</strong>: mal
          <div class="collapser collapser-target-cds445462_451804 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds445462_451804-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>CE-Malonyl-CoA: 62.5%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds445462_451804-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 108.2</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 99.4</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 86.9</dt>

  <dd></dd><dt>Isobutyryl-CoA: 70.6</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 70.5</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 42.1</dt>

  <dd></dd><dt>Propionyl-CoA: 41.0</dt>

  <dd></dd><dt>trans-1,2-CPDA: 40.4</dt>

  <dd></dd><dt>Acetyl-CoA: 40.2</dt>

  <dd></dd><dt>fatty_acid: 27.6</dt>

  <dd></dd><dt>inactive: 25.1</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 20.5</dt>

  <dd></dd><dt>CHC-CoA: 10.9</dt>

  <dd></dd><dt>Benzoyl-CoA: 10.2</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r3c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_1 - Region 1 - hglE-KS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-237"></div> <span class="help-tooltip" id="region_svg-help-237">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_1.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r3c1-svg" data-filename="CC227C_contig_1_r3c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 62,765 - 109,052 nt. (total: 46,288 nt)
        
        <a class="cluster-rules-header" id="r3c1-rules-header" href="#r3c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r3c1-rules">hglE-KS: (hglE or hglD)</div>
      
      <div class="region-svg-container">
       <div id='r3c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-238"></div> <span class="help-tooltip" id="focus-panel-help-238">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r3c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r3c1-clusterblast" data-name="r3c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r3c1-knownclusterblast" data-name="r3c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r3c1-subclusterblast" data-name="r3c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r3c1-MIBiG-cluster-compare" data-name="r3c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r3c1-pfam-details" data-name="r3c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r3c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-22"></div> <span class="help-tooltip" id="cb-general-help-22">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r3c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r3c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r3c1_1.svg">
        NZ_CP041695 (1572274-1618619): Nocardia otitidiscaviarum strain NEB252 chromo... (67% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r3c1_2.svg">
        NZ_CP032568 (1699278-1745398): Nocardia sp. CFHS0054 chromosome (48% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r3c1_3.svg">
        NZ_CP017839 (5565279-5610899): Nocardia seriolae strain EM150506 chromosome, ... (42% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r3c1_4.svg">
        NZ_CP006850 (3459813-3506341): Nocardia nova SH22a chromosome, complete genome (47% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r3c1_5.svg">
        NZ_WEGK01000001 (152111-282271): Nocardia sp. RB20 NRB20 1, whole genome shot... (18% of genes show similarity), NRPS,T1PKS,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r3c1_6.svg">
        NC_018681 (3677724-3724255): Nocardia brasiliensis ATCC 700358, complete sequ... (53% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r3c1_7.svg">
        NZ_QGTL01000001 (60693-107173): Nocardia neocaledoniensis strain DSM 44717 Ga... (42% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r3c1_8.svg">
        NZ_LT985361 (4062162-4108576): Nocardia suismassiliense strain S-137, whole g... (44% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r3c1_9.svg">
        NZ_CP029710 (8344791-8391196): Nocardia sp. CS682 chromosome, complete genome (47% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r3c1_10.svg">
        NC_008278 (3444291-3491737): Frankia alni ACN14a, complete genome (47% of genes show similarity), hglE-KS
      </option>
      
    </select>
    <button id= "clusterblast-r3c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r3c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r3c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-23"></div> <span class="help-tooltip" id="cb-known-help-23">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r3c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-24"></div> <span class="help-tooltip" id="cb-sub-help-24">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r3c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-25"></div> <span class="help-tooltip" id="MIBiG-help-25">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r3c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r3c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r3c1"></div>
    <div id="comparison-MIBiG-r3c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">hglE-KS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001343.1: 61-19132">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001343/index.html#r1c1" target="_new">BGC0001343.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Polyketide</td>
        
        <td>aurachin A, aurachin D</td>
        
        
        <td>Stigmatella aurantiaca Sg a15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B</td>
        
        
        <td>Streptomyces sp. WK-5344</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001592.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001592/index.html#r1c1" target="_new">BGC0001592.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>Other</td>
        
        <td>ferroverdin</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002029.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002029/index.html#r1c1" target="_new">BGC0002029.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B, bagremycin C, bagremycin E, bagremycin F, bagremycin G, ferroverdin A, ferroverdin B, ferroverdin C</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000257.1: 2579-43239">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000257/index.html#r1c1" target="_new">BGC0000257.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Polyketide</td>
        
        <td>prodigiosin</td>
        
        
        <td>Streptomyces griseoviridis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001063.1: 6432565-6464206">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001063/index.html#r1c1" target="_new">BGC0001063.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>NRP, Polyketide</td>
        
        <td>undecylprodigiosin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001895.1: 2411904-2429820">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001895/index.html#r1c1" target="_new">BGC0001895.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>huanglongmycin A, huanglongmycin B, huanglongmycin C</td>
        
        
        <td>Streptomyces sp. CB09001</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000861.1: 0-20498">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000861/index.html#r1c1" target="_new">BGC0000861.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:0.48, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Other</td>
        
        <td>eicosapentaenoic acid</td>
        
        
        <td>Shewanella sp. BR-2</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001281.1: 8011-344234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001281/index.html#r1c1" target="_new">BGC0001281.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>ustilagic acid</td>
        
        
        <td>Ustilago maydis 521</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001983.1: 622-39496">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001983/index.html#r1c1" target="_new">BGC0001983.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.47, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Other</td>
        
        <td>triacsins</td>
        
        
        <td>Kitasatospora aureofaciens</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r3c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001343.1: 61-19132">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001343/index.html#r1c1" target="_new">BGC0001343.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>aurachin A, aurachin D</td>
      <td>Stigmatella aurantiaca Sg a15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B</td>
      <td>Streptomyces sp. WK-5344</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001592.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001592/index.html#r1c1" target="_new">BGC0001592.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Other</td>
      <td>ferroverdin</td>
      <td>Streptomyces lunaelactis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002029.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002029/index.html#r1c1" target="_new">BGC0002029.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B, bagremycin C, bagremycin E, bagremycin F, bagremycin G, ferroverdin A, ferroverdin B, ferroverdin C</td>
      <td>Streptomyces lunaelactis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000257.1: 2579-43239">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000257/index.html#r1c1" target="_new">BGC0000257.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>prodigiosin</td>
      <td>Streptomyces griseoviridis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001063.1: 6432565-6464206">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001063/index.html#r1c1" target="_new">BGC0001063.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP, Polyketide</td>
      <td>undecylprodigiosin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001895.1: 2411904-2429820">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001895/index.html#r1c1" target="_new">BGC0001895.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>huanglongmycin A, huanglongmycin B, huanglongmycin C</td>
      <td>Streptomyces sp. CB09001</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000861.1: 0-20498">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000861/index.html#r1c1" target="_new">BGC0000861.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.47, order:0.48, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Other</td>
      <td>eicosapentaenoic acid</td>
      <td>Shewanella sp. BR-2</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001281.1: 8011-344234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001281/index.html#r1c1" target="_new">BGC0001281.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>ustilagic acid</td>
      <td>Ustilago maydis 521</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001983.1: 622-39496">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001983/index.html#r1c1" target="_new">BGC0001983.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.47, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Other</td>
      <td>triacsins</td>
      <td>Kitasatospora aureofaciens</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r3c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-26"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-26">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r3c1-pfam-details-svg" data-filename="CC227C_contig_1_r3c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r3c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r3c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_1 - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-239"></div> <span class="help-tooltip" id="region_svg-help-239">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_1.region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r3c2-svg" data-filename="CC227C_contig_1_r3c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 353,564 - 373,100 nt. (total: 19,537 nt)
        
        <a class="cluster-rules-header" id="r3c2-rules-header" href="#r3c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r3c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r3c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-240"></div> <span class="help-tooltip" id="focus-panel-help-240">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r3c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r3c2-clusterblast" data-name="r3c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r3c2-knownclusterblast" data-name="r3c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r3c2-subclusterblast" data-name="r3c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r3c2-MIBiG-cluster-compare" data-name="r3c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r3c2-pfam-details" data-name="r3c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r3c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-27"></div> <span class="help-tooltip" id="cb-general-help-27">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r3c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r3c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r3c2_1.svg">
        NZ_CP041695 (1869586-1889517): Nocardia otitidiscaviarum strain NEB252 chromo... (89% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r3c2_2.svg">
        NZ_CP017839 (4845704-4865664): Nocardia seriolae strain EM150506 chromosome, ... (9% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r3c2_3.svg">
        NZ_CP046173 (10781-51521): Nocardia terpenica strain AUSMDU00012715 chromosom... (8% of genes show similarity), PKS-like,terpene
      </option>
      
      <option value = "svg/clusterblast_r3c2_4.svg">
        NZ_CP023778 (5994862-6107669): Nocardia terpenica strain NC YFY NT001 chromos... (3% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r3c2_5.svg">
        NZ_LWLE01000035 (317262-338240): Streptomyces sp. TSRI0281 scaffold6, whole g... (14% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r3c2_6.svg">
        NZ_CP011664 (555243-614655): Streptomyces sp. Mg1 chromosome 1, complete sequ... (3% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r3c2_7.svg">
        NZ_ML123037 (532719-592974): Streptomyces sp. ADI91-18 scaffold002, whole gen... (3% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r3c2_8.svg">
        NZ_CP007699 (1447272-1495029): Streptomyces lydicus strain A02 chromosome, co... (5% of genes show similarity), NRPS-like,terpene
      </option>
      
      <option value = "svg/clusterblast_r3c2_9.svg">
        NZ_CP047146 (6239644-&gt;6260334): Streptomyces sp. GS7 chromosome, complete genome (11% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r3c2_10.svg">
        NZ_LIWA01000010 (36912-57908): Streptomyces sp. CB01883 scaffold10, whole gen... (10% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r3c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r3c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r3c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-28"></div> <span class="help-tooltip" id="cb-known-help-28">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r3c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-29"></div> <span class="help-tooltip" id="cb-sub-help-29">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r3c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-30"></div> <span class="help-tooltip" id="MIBiG-help-30">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r3c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r3c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r3c2"></div>
    <div id="comparison-MIBiG-r3c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001201.1: 11915724-11918015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001201/index.html#r1c1" target="_new">BGC0001201.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Terpene</td>
        
        <td>eremophilene</td>
        
        
        <td>Sorangium cellulosum So ce56</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001224.1: 1518-11352">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001224/index.html#r1c1" target="_new">BGC0001224.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>Other</td>
        
        <td>reductasporine</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>viguiepinol</td>
        
        
        <td>Streptomyces sp. KO-3988</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001336.1: 2595-15215">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001336/index.html#r1c1" target="_new">BGC0001336.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Alkaloid</td>
        
        <td>erdasporine A, erdasporine B, erdasporine C</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.48, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Other</td>
        
        <td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>
        
        
        <td>Streptomyces sp. RM-5-8</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.45, order:0.39, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Terpene</td>
        
        <td>pentalenolactone</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Other</td>
        
        <td>flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.44, order:0.33, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Terpene</td>
        
        <td>raimonol</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001075.1: 2658-14492">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001075/index.html#r1c1" target="_new">BGC0001075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.19</td>
        <td>Terpene, Polyketide</td>
        
        <td>aurachin RE</td>
        
        
        <td>Rhodococcus erythropolis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000886.1: 2774-14963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000886/index.html#r1c1" target="_new">BGC0000886.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Other</td>
        
        <td>allylmalonyl-CoA</td>
        
        
        <td>Streptomyces tsukubensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r3c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001201.1: 11915724-11918015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001201/index.html#r1c1" target="_new">BGC0001201.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>eremophilene</td>
      <td>Sorangium cellulosum So ce56</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001224.1: 1518-11352">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001224/index.html#r1c1" target="_new">BGC0001224.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>reductasporine</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>viguiepinol</td>
      <td>Streptomyces sp. KO-3988</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001336.1: 2595-15215">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001336/index.html#r1c1" target="_new">BGC0001336.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Alkaloid</td>
      <td>erdasporine A, erdasporine B, erdasporine C</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.48, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Other</td>
      <td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>
      <td>Streptomyces sp. RM-5-8</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.45, order:0.39, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Other</td>
      <td>flaviolin</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.44, order:0.33, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Terpene</td>
      <td>raimonol</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001075.1: 2658-14492">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001075/index.html#r1c1" target="_new">BGC0001075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Terpene, Polyketide</td>
      <td>aurachin RE</td>
      <td>Rhodococcus erythropolis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000886.1: 2774-14963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000886/index.html#r1c1" target="_new">BGC0000886.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Other</td>
      <td>allylmalonyl-CoA</td>
      <td>Streptomyces tsukubensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r3c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-31"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-31">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r3c2-pfam-details-svg" data-filename="CC227C_contig_1_r3c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r3c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r3c3' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_1 - Region 3 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-241"></div> <span class="help-tooltip" id="region_svg-help-241">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_1.region003.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r3c3-svg" data-filename="CC227C_contig_1_r3c3.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 420,374 - 441,501 nt. (total: 21,128 nt)
        
        <a class="cluster-rules-header" id="r3c3-rules-header" href="#r3c3">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r3c3-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r3c3-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-242"></div> <span class="help-tooltip" id="focus-panel-help-242">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r3c3">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r3c3-clusterblast" data-name="r3c3-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r3c3-knownclusterblast" data-name="r3c3-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r3c3-subclusterblast" data-name="r3c3-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r3c3-MIBiG-cluster-compare" data-name="r3c3-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r3c3-pfam-details" data-name="r3c3-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r3c3-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-32"></div> <span class="help-tooltip" id="cb-general-help-32">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r3c3-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r3c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r3c3_1.svg">
        NZ_CP023778 (2834064-2858705): Nocardia terpenica strain NC YFY NT001 chromos... (68% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r3c3_2.svg">
        NZ_KV411304 (3066859-3087837): Nocardia terpenica strain IFM 0406 scaffold2, ... (89% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r3c3_3.svg">
        NZ_CP046173 (2849974-2874667): Nocardia terpenica strain AUSMDU00012715 chrom... (68% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r3c3_4.svg">
        NZ_CP017839 (3550853-3571363): Nocardia seriolae strain EM150506 chromosome, ... (90% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r3c3_5.svg">
        NZ_CP032568 (4019050-4039165): Nocardia sp. CFHS0054 chromosome (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r3c3_6.svg">
        NZ_QQAZ01000006 (327184-347121): Nocardia mexicana strain DSM 44952 Ga0244508... (89% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r3c3_7.svg">
        NZ_CP006850 (2548167-2568493): Nocardia nova SH22a chromosome, complete genome (85% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r3c3_8.svg">
        NZ_QQBC01000006 (162937-183870): Nocardia pseudobrasiliensis strain DSM 44290... (85% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r3c3_9.svg">
        NZ_QRCM01000001 (3302884-3323841): Rhodococcus rhodnii strain ATCC 35071 cont... (89% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r3c3_10.svg">
        NC_015564 (1342134-1362195): Hoyosella subflava DQS3-9A1, complete sequence (89% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r3c3-download">Download graphic</button>
  </div>
  <div id="clusterblast-r3c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r3c3-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-33"></div> <span class="help-tooltip" id="cb-known-help-33">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r3c3-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r3c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r3c3_1.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r3c3_2.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r3c3_3.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r3c3_4.svg">
        microansamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r3c3-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r3c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r3c3-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-34"></div> <span class="help-tooltip" id="cb-sub-help-34">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r3c3-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-35"></div> <span class="help-tooltip" id="MIBiG-help-35">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r3c3-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r3c3-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r3c3"></div>
    <div id="comparison-MIBiG-r3c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>auriculamide</td>
        
        
        <td>Herpetosiphon aurantiacus DSM 785</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645.1: 0-7475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Halobacillus halophilus DSM 2266</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644.1: 97-6276">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Dietzia sp. CQ4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:0.42, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Myxococcus xanthus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.49, order:0.49, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.49, order:0.47, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000698.1: 2269-33779">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000698/index.html#r1c1" target="_new">BGC0000698.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.54, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Saccharide</td>
        
        <td>hygromycin A</td>
        
        
        <td>Streptomyces hygroscopicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640.1: 2399-8466">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.19</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Enterobacteriaceae bacterium DC404</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.49, order:0.42, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces collinus Tu 365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000636.1: 2-14167">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000636/index.html#r1c1" target="_new">BGC0000636.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Brevibacterium linens</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r3c3-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>auriculamide</td>
      <td>Herpetosiphon aurantiacus DSM 785</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645.1: 0-7475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Halobacillus halophilus DSM 2266</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644.1: 97-6276">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Dietzia sp. CQ4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:0.42, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Myxococcus xanthus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.49, order:0.49, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.49, order:0.47, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000698.1: 2269-33779">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000698/index.html#r1c1" target="_new">BGC0000698.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.54, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Saccharide</td>
      <td>hygromycin A</td>
      <td>Streptomyces hygroscopicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640.1: 2399-8466">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Enterobacteriaceae bacterium DC404</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.49, order:0.42, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces collinus Tu 365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000636.1: 2-14167">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000636/index.html#r1c1" target="_new">BGC0000636.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Brevibacterium linens</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r3c3-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-36"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-36">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r3c3-pfam-details-svg" data-filename="CC227C_contig_1_r3c3_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r3c3-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r3c4' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_1 - Region 4 - T3PKS,arylpolyene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-243"></div> <span class="help-tooltip" id="region_svg-help-243">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_1.region004.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r3c4-svg" data-filename="CC227C_contig_1_r3c4.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 630,687 - 686,697 nt. (total: 56,011 nt)
        
        <a class="cluster-rules-header" id="r3c4-rules-header" href="#r3c4">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r3c4-rules">arylpolyene: (APE_KS1 or APE_KS2)<br>T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)</div>
      
      <div class="region-svg-container">
       <div id='r3c4-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-244"></div> <span class="help-tooltip" id="focus-panel-help-244">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r3c4">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
    
     
      <div class="body-details-header r3c4-clusterblast" data-name="r3c4-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r3c4-knownclusterblast" data-name="r3c4-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r3c4-subclusterblast" data-name="r3c4-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r3c4-MIBiG-cluster-compare" data-name="r3c4-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r3c4-pfam-details" data-name="r3c4-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    
     
    <div class="body-details-section r3c4-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-37"></div> <span class="help-tooltip" id="cb-general-help-37">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r3c4-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r3c4_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r3c4_1.svg">
        NZ_CP041695 (2142289-2196309): Nocardia otitidiscaviarum strain NEB252 chromo... (85% of genes show similarity), T3PKS,arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r3c4_2.svg">
        NZ_CP059399 (3896151-3951306): Nocardia sp. WCH-YHL-001 chromosome, complete ... (64% of genes show similarity), T3PKS,arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r3c4_3.svg">
        NZ_CP032568 (4735402-4786581): Nocardia sp. CFHS0054 chromosome (72% of genes show similarity), T3PKS,arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r3c4_4.svg">
        NC_016887 (2331187-2372333): Nocardia cyriacigeorgica GUH-2, complete genome (72% of genes show similarity), arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r3c4_5.svg">
        NZ_CP017839 (3815652-3866764): Nocardia seriolae strain EM150506 chromosome, ... (67% of genes show similarity), T3PKS,arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r3c4_6.svg">
        NZ_CP026746 (5428321-5469467): Nocardia cyriacigeorgica strain MDA3349 isolat... (75% of genes show similarity), arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r3c4_7.svg">
        NZ_LR215973 (779979-821125): Nocardia cyriacigeorgica strain 3012STDY6756504 ... (74% of genes show similarity), arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r3c4_8.svg">
        NZ_CP018082 (2172158-2213304): Nocardia mangyaensis strain Y48 chromosome, co... (70% of genes show similarity), arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r3c4_9.svg">
        NZ_QGTL01000012 (101749-142895): Nocardia neocaledoniensis strain DSM 44717 G... (71% of genes show similarity), arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r3c4_10.svg">
        NZ_CP046173 (3065139-3106291): Nocardia terpenica strain AUSMDU00012715 chrom... (68% of genes show similarity), arylpolyene
      </option>
      
    </select>
    <button id= "clusterblast-r3c4-download">Download graphic</button>
  </div>
  <div id="clusterblast-r3c4-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r3c4-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-38"></div> <span class="help-tooltip" id="cb-known-help-38">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r3c4-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-39"></div> <span class="help-tooltip" id="cb-sub-help-39">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r3c4-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-40"></div> <span class="help-tooltip" id="MIBiG-help-40">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r3c4-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r3c4-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r3c4"></div>
    <div id="comparison-MIBiG-r3c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">arylpolyene</span></th>
         
         <th><span class="cc-heat-product">T3PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.47</td>
        <td>Polyketide</td>
        
        <td>frankiamicin</td>
        
        
        <td>Frankia sp. EAN1pec</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.41</td>
        <td>Polyketide</td>
        
        <td>griseusin</td>
        
        
        <td>Streptomyces griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000204.1: 4242-15189">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000204/index.html#r1c1" target="_new">BGC0000204.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.40</td>
        <td>Polyketide</td>
        
        <td>benastatin A, benastatin B, BE-43767, TAN1532B</td>
        
        
        <td>Streptomyces sp. A2991200</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.40</td>
        <td>Other</td>
        
        <td>flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.40</td>
        <td>Polyketide</td>
        
        <td>granaticin</td>
        
        
        <td>Streptomyces violaceoruber</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.37</td>
        <td>Polyketide, Saccharide</td>
        
        <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.37</td>
        <td>Polyketide</td>
        
        <td>jadomycin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001077.1: 8379669-8386219">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001077/index.html#r1c1" target="_new">BGC0001077.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.37</td>
        <td>Terpene, Polyketide</td>
        
        <td>alkyl-O-dihydrogeranyl-methoxyhydroquinones</td>
        
        
        <td>Actinoplanes missouriensis 431</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001441.1: 0-27983">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001441/index.html#r1c1" target="_new">BGC0001441.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.45, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.36</td>
        <td>Other</td>
        
        <td>belactosin A, belactosin C</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000218.1: 27-8720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000218/index.html#r1c1" target="_new">BGC0000218.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.36</td>
        <td>Polyketide</td>
        
        <td>doxorubicin</td>
        
        
        <td>Streptomyces peucetius</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r3c4-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007.1: 5922480-5942549">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.49, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Terpene</td>
      <td>atolypene A, atolypene B</td>
      <td>Amycolatopsis tolypomycina</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>frankiamicin</td>
      <td>Frankia sp. EAN1pec</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001441.1: 0-27983">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001441/index.html#r1c1" target="_new">BGC0001441.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Other</td>
      <td>belactosin A, belactosin C</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002000.1: 2056-14558">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002000/index.html#r1c1" target="_new">BGC0002000.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>setomimycin</td>
      <td>Streptomyces aurantiacus JA 4570</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000204.1: 4242-15189">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000204/index.html#r1c1" target="_new">BGC0000204.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>benastatin A, benastatin B, BE-43767, TAN1532B</td>
      <td>Streptomyces sp. A2991200</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000679.1: 255997-303913">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000679/index.html#r1c1" target="_new">BGC0000679.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Terpene</td>
      <td>diazepinomicin</td>
      <td>Micromonospora sp. M42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001343.1: 61-19132">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001343/index.html#r1c1" target="_new">BGC0001343.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Polyketide</td>
      <td>aurachin A, aurachin D</td>
      <td>Stigmatella aurantiaca Sg a15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Polyketide</td>
      <td>griseusin</td>
      <td>Streptomyces griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001173.1: 223360-233305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001173/index.html#r1c1" target="_new">BGC0001173.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>plantazolicin</td>
      <td>Bacillus pumilus ATCC 7061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Polyketide, Saccharide</td>
      <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
      <td>Saccharopolyspora erythraea</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r3c4-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-41"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-41">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r3c4-pfam-details-svg" data-filename="CC227C_contig_1_r3c4_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r3c4-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r4c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_23 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-245"></div> <span class="help-tooltip" id="region_svg-help-245">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_23.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r4c1-svg" data-filename="CC227C_contig_23_r4c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 44,137 - 90,481 nt. (total: 46,345 nt)
        
        <a class="cluster-rules-header" id="r4c1-rules-header" href="#r4c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r4c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r4c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-246"></div> <span class="help-tooltip" id="focus-panel-help-246">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r4c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r4c1-nrps_pks" data-name="r4c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r4c1-clusterblast" data-name="r4c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r4c1-knownclusterblast" data-name="r4c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r4c1-subclusterblast" data-name="r4c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r4c1-MIBiG-cluster-compare" data-name="r4c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r4c1-pfam-details" data-name="r4c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r4c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-42"></div> <span class="help-tooltip" id="nrps-domain-help-42">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c1-details-svg" data-filename="CC227C_contig_23_r4c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r4c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r4c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-45"></div> <span class="help-tooltip" id="cb-general-help-45">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r4c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r4c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r4c1_1.svg">
        NZ_CP016795 (7081622-7222651): Streptomyces olivaceus strain KLBMP 5084 chrom... (18% of genes show similarity), NRPS,NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r4c1_2.svg">
        NZ_KI912270 (6698043-6777541): Mycolicibacterium tusciae JS617 MyctuDRAFT MTC... (29% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r4c1_3.svg">
        NZ_FNET01000012 (124294-196840): Lentzea violacea strain DSM 44796, whole gen... (14% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r4c1_4.svg">
        NZ_FNHV01000020 (18054-65836): Streptomyces sp. cf386, whole genome shotgun s... (23% of genes show similarity), NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r4c1_5.svg">
        NZ_CP017780 (5487396-5530454): Amycolatopsis sp. BJA-103 chromosome, complete... (22% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r4c1_6.svg">
        NZ_JPLW01000003 (1121403-1165186): Amycolatopsis sp. MJM2582 contig00003, who... (22% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r4c1_7.svg">
        NZ_LQMT02000010 (240527-284310): Amycolatopsis keratiniphila subsp. keratinip... (22% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r4c1_8.svg">
        NZ_LQCI01000034 (14812-58610): Amycolatopsis regifaucium strain GY080 contig ... (21% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r4c1_9.svg">
        NZ_CP016174 (6783374-6825525): Amycolatopsis orientalis strain B-37 chromosom... (22% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r4c1_10.svg">
        NC_021252 (6752084-6795861): Amycolatopsis keratiniphila, complete sequence (22% of genes show similarity), NRPS-like
      </option>
      
    </select>
    <button id= "clusterblast-r4c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r4c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-46"></div> <span class="help-tooltip" id="cb-known-help-46">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r4c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r4c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r4c1_1.svg">
        meilingmycin
      </option>
      
    </select>
    <button id="knownclusterblast-r4c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r4c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-47"></div> <span class="help-tooltip" id="cb-sub-help-47">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r4c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-48"></div> <span class="help-tooltip" id="MIBiG-help-48">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r4c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r4c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r4c1"></div>
    <div id="comparison-MIBiG-r4c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001903.1: 8904-19774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001903/index.html#r1c1" target="_new">BGC0001903.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Other</td>
        
        <td>neocarazostatin A</td>
        
        
        <td>Streptomyces sp. MA37</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Polyketide</td>
        
        <td>depudecin</td>
        
        
        <td>Alternaria brassicicola</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000204.1: 4242-15189">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000204/index.html#r1c1" target="_new">BGC0000204.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.51, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Polyketide</td>
        
        <td>benastatin A, benastatin B, BE-43767, TAN1532B</td>
        
        
        <td>Streptomyces sp. A2991200</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Polyketide</td>
        
        <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
        
        
        <td>Parastagonospora nodorum SN15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>esperamicin</td>
        
        
        <td>Actinomadura verrucosospora</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>betaenone C, betaenone A</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>betaenone A, betaenone B, betaenone C</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP, Polyketide</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>viguiepinol</td>
        
        
        <td>Streptomyces sp. KO-3988</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000194.1: 5513808-5535091">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000194/index.html#r1c1" target="_new">BGC0000194.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Polyketide</td>
        
        <td>actinorhodin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r4c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001903.1: 8904-19774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001903/index.html#r1c1" target="_new">BGC0001903.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other</td>
      <td>neocarazostatin A</td>
      <td>Streptomyces sp. MA37</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>depudecin</td>
      <td>Alternaria brassicicola</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000204.1: 4242-15189">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000204/index.html#r1c1" target="_new">BGC0000204.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.51, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>benastatin A, benastatin B, BE-43767, TAN1532B</td>
      <td>Streptomyces sp. A2991200</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
      <td>Parastagonospora nodorum SN15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>esperamicin</td>
      <td>Actinomadura verrucosospora</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>betaenone C, betaenone A</td>
      <td>Phoma betae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>betaenone A, betaenone B, betaenone C</td>
      <td>Phoma betae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP, Polyketide</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>viguiepinol</td>
      <td>Streptomyces sp. KO-3988</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000194.1: 5513808-5535091">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000194/index.html#r1c1" target="_new">BGC0000194.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>actinorhodin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r4c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-49"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-49">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c1-pfam-details-svg" data-filename="CC227C_contig_23_r4c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r4c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r4c1-nrps_pks_products" data-name="r4c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r4c1-nrps_pks" data-name="r4c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r4c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-43"></div> <span class="help-tooltip" id="nrps-structures-help-43">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 44136 - 90481:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r4c1-1" class="smiles-canvas" data-smiles="CCC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(redmal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=redmal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=redmal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r4c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CCC(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r4c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-44"></div> <span class="help-tooltip" id="nrps-monomers-help-44">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds64136_70481</span></strong>: mal
      <div class="collapser collapser-target-cds64136_70481 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (527..823)</strong>: mal
          <div class="collapser collapser-target-cds64136_70481 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds64136_70481-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>CE-Malonyl-CoA: 66.7%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds64136_70481-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 104.5</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 102.4</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 94.6</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 79.2</dt>

  <dd></dd><dt>Isobutyryl-CoA: 64.3</dt>

  <dd></dd><dt>Propionyl-CoA: 56.2</dt>

  <dd></dd><dt>trans-1,2-CPDA: 46.3</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 43.2</dt>

  <dd></dd><dt>Acetyl-CoA: 41.8</dt>

  <dd></dd><dt>fatty_acid: 41.5</dt>

  <dd></dd><dt>Benzoyl-CoA: 26.6</dt>

  <dd></dd><dt>CHC-CoA: 24.6</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 20.7</dt>

  <dd></dd><dt>inactive: 20.3</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r6c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_4 - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-247"></div> <span class="help-tooltip" id="region_svg-help-247">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_4.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r6c1-svg" data-filename="CC227C_contig_4_r6c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 549,784 - 555,719 nt. (total: 5,936 nt)
        
        <a class="cluster-rules-header" id="r6c1-rules-header" href="#r6c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r6c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193)</div>
      
      <div class="region-svg-container">
       <div id='r6c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-248"></div> <span class="help-tooltip" id="focus-panel-help-248">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r6c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r6c1-clusterblast" data-name="r6c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r6c1-knownclusterblast" data-name="r6c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r6c1-subclusterblast" data-name="r6c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r6c1-MIBiG-cluster-compare" data-name="r6c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r6c1-pfam-details" data-name="r6c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r6c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-50"></div> <span class="help-tooltip" id="cb-general-help-50">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r6c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r6c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r6c1_1.svg">
        NZ_CP041695 (107738-270958): Nocardia otitidiscaviarum strain NEB252 chromoso... (7% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r6c1_2.svg">
        NZ_CP059399 (2142701-2153499): Nocardia sp. WCH-YHL-001 chromosome, complete ... (60% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r6c1_3.svg">
        NZ_CP017839 (1479309-1588618): Nocardia seriolae strain EM150506 chromosome, ... (9% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r6c1_4.svg">
        NZ_CP032568 (691161-701496): Nocardia sp. CFHS0054 chromosome (46% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r6c1_5.svg">
        NZ_CP046172 (8935832-9019422): Nocardia arthritidis strain AUSMDU00012717 chr... (12% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r6c1_6.svg">
        NZ_CP029710 (3661313-3767589): Nocardia sp. CS682 chromosome, complete genome (9% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r6c1_7.svg">
        NZ_OBEG01000002 (807336-887847): Nocardia amikacinitolerans strain DSM 45537,... (14% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r6c1_8.svg">
        NZ_LT985361 (8547642-8654705): Nocardia suismassiliense strain S-137, whole g... (9% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r6c1_9.svg">
        NZ_QJKF01000002 (71512-175150): Nocardia tenerifensis strain DSM 44704 Ga0244... (9% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r6c1_10.svg">
        NZ_CP046171 (6972095-7074285): Nocardia brasiliensis strain AUSMDU00024985 ch... (10% of genes show similarity), NRPS,bacteriocin
      </option>
      
    </select>
    <button id= "clusterblast-r6c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r6c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-51"></div> <span class="help-tooltip" id="cb-known-help-51">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r6c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-52"></div> <span class="help-tooltip" id="cb-sub-help-52">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r6c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-53"></div> <span class="help-tooltip" id="MIBiG-help-53">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r6c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r6c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r6c1"></div>
    <div id="comparison-MIBiG-r6c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">RiPP-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001882.1: 2571644-2615001">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001882/index.html#r1c1" target="_new">BGC0001882.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Polyketide</td>
        
        <td>chrysoxanthone A, chrysoxanthone B, chrysoxanthone C</td>
        
        
        <td>Penicillium rubens Wisconsin 54-1255</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000882.1: 6124-25938">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000882/index.html#r1c1" target="_new">BGC0000882.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Other</td>
        
        <td>mildiomycin</td>
        
        
        <td>Streptomyces rimofaciens</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r6c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001882.1: 2571644-2615001">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001882/index.html#r1c1" target="_new">BGC0001882.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Polyketide</td>
      <td>chrysoxanthone A, chrysoxanthone B, chrysoxanthone C</td>
      <td>Penicillium rubens Wisconsin 54-1255</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000882.1: 6124-25938">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000882/index.html#r1c1" target="_new">BGC0000882.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Other</td>
      <td>mildiomycin</td>
      <td>Streptomyces rimofaciens</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r6c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-54"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-54">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c1-pfam-details-svg" data-filename="CC227C_contig_4_r6c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r6c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r8c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_8 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-249"></div> <span class="help-tooltip" id="region_svg-help-249">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_8.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r8c1-svg" data-filename="CC227C_contig_8_r8c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 35,696 - 89,420 nt. (total: 53,725 nt)
        
        <a class="cluster-rules-header" id="r8c1-rules-header" href="#r8c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r8c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r8c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-250"></div> <span class="help-tooltip" id="focus-panel-help-250">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r8c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r8c1-nrps_pks" data-name="r8c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r8c1-clusterblast" data-name="r8c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r8c1-knownclusterblast" data-name="r8c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r8c1-subclusterblast" data-name="r8c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r8c1-MIBiG-cluster-compare" data-name="r8c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r8c1-pfam-details" data-name="r8c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r8c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-55"></div> <span class="help-tooltip" id="nrps-domain-help-55">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r8c1-details-svg" data-filename="CC227C_contig_8_r8c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r8c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r8c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-58"></div> <span class="help-tooltip" id="cb-general-help-58">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r8c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r8c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r8c1_1.svg">
        NZ_CP041695 (4669456-4722422): Nocardia otitidiscaviarum strain NEB252 chromo... (93% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r8c1_2.svg">
        NZ_CP032568 (6737592-6793837): Nocardia sp. CFHS0054 chromosome (63% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r8c1_3.svg">
        NZ_CP059399 (7019521-7073775): Nocardia sp. WCH-YHL-001 chromosome, complete ... (55% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r8c1_4.svg">
        NZ_CP017839 (6663629-6715818): Nocardia seriolae strain EM150506 chromosome, ... (65% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r8c1_5.svg">
        NZ_RFFH01000001 (1128971-1176417): Nocardia stercoris strain NEAU-LL90 Scaffo... (44% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r8c1_6.svg">
        NZ_QQBC01000007 (401878-454112): Nocardia pseudobrasiliensis strain DSM 44290... (34% of genes show similarity), NRPS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c1_7.svg">
        NZ_CP046172 (7402906-7450865): Nocardia arthritidis strain AUSMDU00012717 chr... (28% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r8c1_8.svg">
        NZ_PYHL01000002 (640158-703095): Rhodococcus sp. AD45-ID NODE 2 length 171571... (23% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r8c1_9.svg">
        NZ_NSDY01000001 (394512-452801): Rhodococcus sp. ACPA4 Contig 1, whole genome... (20% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r8c1_10.svg">
        NZ_CP023778 (6239080-6281579): Nocardia terpenica strain NC YFY NT001 chromos... (31% of genes show similarity), NRPS-like
      </option>
      
    </select>
    <button id= "clusterblast-r8c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r8c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r8c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-59"></div> <span class="help-tooltip" id="cb-known-help-59">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r8c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-60"></div> <span class="help-tooltip" id="cb-sub-help-60">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r8c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-61"></div> <span class="help-tooltip" id="MIBiG-help-61">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r8c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r8c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r8c1"></div>
    <div id="comparison-MIBiG-r8c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000392.1: 3113323-3141040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000392/index.html#r1c1" target="_new">BGC0000392.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.45, order:N/A, components:0.97)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>mirubactin</td>
        
        
        <td>Actinosynnema mirum DSM 43827</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000325.1: 510822-533448">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000325/index.html#r1c1" target="_new">BGC0000325.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>coelichelin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000666.1: 0-29371">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000666/index.html#r1c1" target="_new">BGC0000666.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.63, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Terpene</td>
        
        <td>xiamycin A</td>
        
        
        <td>Streptomyces sp. SCSIO 02999</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000665.1: 0-21259">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000665/index.html#r1c1" target="_new">BGC0000665.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.63, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Terpene, Alkaloid</td>
        
        <td>xiamycin</td>
        
        
        <td>Streptomyces sp. HKI0576</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001767.1: 18413-48220">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001767/index.html#r1c1" target="_new">BGC0001767.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>salinichelins</td>
        
        
        <td>Salinispora pacifica CNY331</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.61, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>Terpene</td>
        
        <td>platencin</td>
        
        
        <td>Streptomyces platensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001671.1: 5106-76272">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001671/index.html#r1c1" target="_new">BGC0001671.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>monobactam</td>
        
        
        <td>Agrobacterium tumefaciens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>pseudomonine</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001343.1: 61-19132">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001343/index.html#r1c1" target="_new">BGC0001343.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Polyketide</td>
        
        <td>aurachin A, aurachin D</td>
        
        
        <td>Stigmatella aurantiaca Sg a15</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r8c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000392.1: 3113323-3141040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000392/index.html#r1c1" target="_new">BGC0000392.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.45, order:N/A, components:0.97)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>mirubactin</td>
      <td>Actinosynnema mirum DSM 43827</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000325.1: 510822-533448">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000325/index.html#r1c1" target="_new">BGC0000325.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>coelichelin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000666.1: 0-29371">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000666/index.html#r1c1" target="_new">BGC0000666.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.63, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Terpene</td>
      <td>xiamycin A</td>
      <td>Streptomyces sp. SCSIO 02999</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000665.1: 0-21259">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000665/index.html#r1c1" target="_new">BGC0000665.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.63, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Terpene, Alkaloid</td>
      <td>xiamycin</td>
      <td>Streptomyces sp. HKI0576</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001767.1: 18413-48220">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001767/index.html#r1c1" target="_new">BGC0001767.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>salinichelins</td>
      <td>Salinispora pacifica CNY331</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.61, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Terpene</td>
      <td>platencin</td>
      <td>Streptomyces platensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001671.1: 5106-76272">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001671/index.html#r1c1" target="_new">BGC0001671.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>monobactam</td>
      <td>Agrobacterium tumefaciens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>pseudomonine</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001343.1: 61-19132">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001343/index.html#r1c1" target="_new">BGC0001343.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>aurachin A, aurachin D</td>
      <td>Stigmatella aurantiaca Sg a15</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r8c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-62"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-62">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r8c1-pfam-details-svg" data-filename="CC227C_contig_8_r8c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r8c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r8c1-nrps_pks_products" data-name="r8c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r8c1-nrps_pks" data-name="r8c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r8c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-56"></div> <span class="help-tooltip" id="nrps-structures-help-56">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 35695 - 89420:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r8c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-X - X - D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x,x,d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x*,x,d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r8c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r8c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-57"></div> <span class="help-tooltip" id="nrps-monomers-help-57">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds55695_69420</span></strong>: X - X - X
      <div class="collapser collapser-target-cds55695_69420 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,X,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,X*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (473..868)</strong>: X
          <div class="collapser collapser-target-cds55695_69420 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds55695_69420-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DieNVGgVtK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2108..2507)</strong>: X
          <div class="collapser collapser-target-cds55695_69420 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala, val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds55695_69420-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAhvsAlVwK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3172..3558)</strong>: X
          <div class="collapser collapser-target-cds55695_69420 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds55695_69420-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DaFVAaLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r13c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_15 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-251"></div> <span class="help-tooltip" id="region_svg-help-251">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_15.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r13c1-svg" data-filename="CC227C_contig_15_r13c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 15,666 - 69,320 nt. (total: 53,655 nt)
        
        <a class="cluster-rules-header" id="r13c1-rules-header" href="#r13c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r13c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r13c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-252"></div> <span class="help-tooltip" id="focus-panel-help-252">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r13c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r13c1-nrps_pks" data-name="r13c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r13c1-clusterblast" data-name="r13c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r13c1-knownclusterblast" data-name="r13c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r13c1-subclusterblast" data-name="r13c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r13c1-MIBiG-cluster-compare" data-name="r13c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r13c1-pfam-details" data-name="r13c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r13c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-63"></div> <span class="help-tooltip" id="nrps-domain-help-63">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r13c1-details-svg" data-filename="CC227C_contig_15_r13c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r13c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r13c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-66"></div> <span class="help-tooltip" id="cb-general-help-66">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r13c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r13c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r13c1_1.svg">
        NZ_CP041695 (3379050-3443782): Nocardia otitidiscaviarum strain NEB252 chromo... (47% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r13c1_2.svg">
        NZ_CP031418 (46977-117153): Nocardia farcinica strain W6977 chromosome, compl... (22% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r13c1_3.svg">
        NZ_FWXV01000001 (2032182-2089632): Kibdelosporangium aridum strain DSM 43828,... (23% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r13c1_4.svg">
        NZ_CP016438 (1586890-1638995): Streptomyces lincolnensis strain NRRL 2936 chr... (31% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r13c1_5.svg">
        NZ_QNRE01000001 (71972-144257): Nocardia puris strain DSM 44599 Ga0244502 101... (17% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r13c1_6.svg">
        NC_016887 (2842969-2896069): Nocardia cyriacigeorgica GUH-2, complete genome (29% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r13c1_7.svg">
        NZ_SOAV01000001 (705630-797415): Streptomyces sp. BK208 Ga0307686 101, whole ... (21% of genes show similarity), NRPS,T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r13c1_8.svg">
        NZ_GG657756 (109378-172818): Streptomyces lividans TK24 supercont1.1, whole g... (16% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r13c1_9.svg">
        NZ_FMCV01000003 (147139-293651): Micromonospora marina strain DSM 45555, whol... (9% of genes show similarity), NRPS,PKS-like,oligosaccharide,siderophore,terpene,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r13c1_10.svg">
        NZ_KB895825 (127282-277441): Micromonospora sp. CNB394 C569DRAFT scaffold0000... (8% of genes show similarity), NRPS,PKS-like,oligosaccharide,siderophore,terpene,transAT-PKS
      </option>
      
    </select>
    <button id= "clusterblast-r13c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r13c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r13c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-67"></div> <span class="help-tooltip" id="cb-known-help-67">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r13c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r13c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r13c1_1.svg">
        coelibactin
      </option>
      
      <option value = "svg/knownclusterblast_r13c1_2.svg">
        nataxazole
      </option>
      
      <option value = "svg/knownclusterblast_r13c1_3.svg">
        A33853
      </option>
      
      <option value = "svg/knownclusterblast_r13c1_4.svg">
        thiazostatin / watasemycin A / watasemycin B / 2-hydroxyphenylthiazoline enantiopyochelin / isopyochelin
      </option>
      
    </select>
    <button id="knownclusterblast-r13c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r13c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r13c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-68"></div> <span class="help-tooltip" id="cb-sub-help-68">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r13c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-69"></div> <span class="help-tooltip" id="MIBiG-help-69">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r13c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r13c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r13c1"></div>
    <div id="comparison-MIBiG-r13c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.42, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000373.1: 6204479-6222797">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000373/index.html#r1c1" target="_new">BGC0000373.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>holomycin</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000412.1: 804-15023">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000412/index.html#r1c1" target="_new">BGC0000412.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:0.58, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>pyochelin</td>
        
        
        <td>Pseudomonas aeruginosa PAO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002091.1: 3593-21565">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002091/index.html#r1c1" target="_new">BGC0002091.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>holomycin</td>
        
        
        <td>Yersinia ruckeri ATCC 29473</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001336.1: 2595-15215">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001336/index.html#r1c1" target="_new">BGC0001336.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Alkaloid</td>
        
        <td>erdasporine A, erdasporine B, erdasporine C</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.50, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>Other</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.50, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP, Polyketide</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000352.1: 2699482-2725713">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000352/index.html#r1c1" target="_new">BGC0000352.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>fimsbactin A</td>
        
        
        <td>Acinetobacter sp. ADP1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000294.1: 181-31792">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000294/index.html#r1c1" target="_new">BGC0000294.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>acinetobactin</td>
        
        
        <td>Acinetobacter baumannii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000632.1: 2720626-2736748">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000632/index.html#r1c1" target="_new">BGC0000632.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.48, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene, Saccharide</td>
        
        <td>brasilicardin A</td>
        
        
        <td>Nocardia terpenica</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r13c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.42, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000373.1: 6204479-6222797">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000373/index.html#r1c1" target="_new">BGC0000373.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>holomycin</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000412.1: 804-15023">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000412/index.html#r1c1" target="_new">BGC0000412.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.46, order:0.58, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>pyochelin</td>
      <td>Pseudomonas aeruginosa PAO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002091.1: 3593-21565">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002091/index.html#r1c1" target="_new">BGC0002091.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>holomycin</td>
      <td>Yersinia ruckeri ATCC 29473</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001336.1: 2595-15215">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001336/index.html#r1c1" target="_new">BGC0001336.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Alkaloid</td>
      <td>erdasporine A, erdasporine B, erdasporine C</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.50, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Other</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.50, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP, Polyketide</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000352.1: 2699482-2725713">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000352/index.html#r1c1" target="_new">BGC0000352.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>fimsbactin A</td>
      <td>Acinetobacter sp. ADP1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000294.1: 181-31792">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000294/index.html#r1c1" target="_new">BGC0000294.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>acinetobactin</td>
      <td>Acinetobacter baumannii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000632.1: 2720626-2736748">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000632/index.html#r1c1" target="_new">BGC0000632.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.48, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene, Saccharide</td>
      <td>brasilicardin A</td>
      <td>Nocardia terpenica</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r13c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-70"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-70">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r13c1-pfam-details-svg" data-filename="CC227C_contig_15_r13c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r13c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r13c1-nrps_pks_products" data-name="r13c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r13c1-nrps_pks" data-name="r13c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r13c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-64"></div> <span class="help-tooltip" id="nrps-structures-help-64">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 15665 - 69320:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r13c1-1" class="smiles-canvas" data-smiles="NC(CS)C(=O)NC(CS)C(=O)NC(CS)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(cys - cys) + (cys)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys,cys&amp;nrps2=cys" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*,cys*&amp;nrps2=cys*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r13c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CS)C(=O)NC(CS)C(=O)NC(CS)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r13c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-65"></div> <span class="help-tooltip" id="nrps-monomers-help-65">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds35665_43852</span></strong>: cys - cys
      <div class="collapser collapser-target-cds35665_43852 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys,cys" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*,cys*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (956..1219)</strong>: cys
          <div class="collapser collapser-target-cds35665_43852 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  cys
              <div class="collapser collapser-target-cds35665_43852-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Small clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Single AA prediction:</dt>
   <dd>cys</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLFNLgLIhK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>cys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2179..2548)</strong>: cys
          <div class="collapser collapser-target-cds35665_43852 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  cys
              <div class="collapser collapser-target-cds35665_43852-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Small clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Single AA prediction:</dt>
   <dd>cys</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLYNLSLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>cys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds43848_49320</span></strong>: cys
      <div class="collapser collapser-target-cds43848_49320 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (552..919)</strong>: cys
          <div class="collapser collapser-target-cds43848_49320 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  cys
              <div class="collapser collapser-target-cds43848_49320-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Small clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Single AA prediction:</dt>
   <dd>cys</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLYNLSLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>cys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r16c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_18 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-253"></div> <span class="help-tooltip" id="region_svg-help-253">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_18.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r16c1-svg" data-filename="CC227C_contig_18_r16c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 40,481 - 84,770 nt. (total: 44,290 nt)
        
        <a class="cluster-rules-header" id="r16c1-rules-header" href="#r16c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r16c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r16c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-254"></div> <span class="help-tooltip" id="focus-panel-help-254">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r16c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r16c1-nrps_pks" data-name="r16c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r16c1-clusterblast" data-name="r16c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r16c1-knownclusterblast" data-name="r16c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r16c1-subclusterblast" data-name="r16c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r16c1-MIBiG-cluster-compare" data-name="r16c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r16c1-pfam-details" data-name="r16c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r16c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-71"></div> <span class="help-tooltip" id="nrps-domain-help-71">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r16c1-details-svg" data-filename="CC227C_contig_18_r16c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r16c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r16c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-74"></div> <span class="help-tooltip" id="cb-general-help-74">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r16c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r16c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r16c1_1.svg">
        NZ_CP041695 (6214598-6259658): Nocardia otitidiscaviarum strain NEB252 chromo... (72% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r16c1_2.svg">
        NZ_CP059399 (991225-1036428): Nocardia sp. WCH-YHL-001 chromosome, complete g... (65% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r16c1_3.svg">
        NZ_CP017839 (8189446-8282410): Nocardia seriolae strain EM150506 chromosome, ... (49% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r16c1_4.svg">
        NZ_OBEG01000005 (238-33536): Nocardia amikacinitolerans strain DSM 45537, who... (70% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r16c1_5.svg">
        NZ_CP046173 (726450-771240): Nocardia terpenica strain AUSMDU00012715 chromos... (57% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r16c1_6.svg">
        NZ_CP029710 (5359237-5403811): Nocardia sp. CS682 chromosome, complete genome (55% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r16c1_7.svg">
        NZ_QQBC01000014 (172937-218145): Nocardia pseudobrasiliensis strain DSM 44290... (52% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r16c1_8.svg">
        NZ_KV411303 (3031496-3074888): Nocardia terpenica strain IFM 0406 scaffold1, ... (57% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r16c1_9.svg">
        NZ_CP023778 (824172-868547): Nocardia terpenica strain NC YFY NT001 chromosom... (57% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r16c1_10.svg">
        NZ_KB907309 (401148-446671): Nocardia sp. BMG111209 G361DRAFT scaffold00003.3... (48% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r16c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r16c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r16c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-75"></div> <span class="help-tooltip" id="cb-known-help-75">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r16c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r16c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r16c1_1.svg">
        gobichelin A / gobichelin B
      </option>
      
      <option value = "svg/knownclusterblast_r16c1_2.svg">
        amychelin
      </option>
      
      <option value = "svg/knownclusterblast_r16c1_3.svg">
        vulnibactin
      </option>
      
      <option value = "svg/knownclusterblast_r16c1_4.svg">
        mycobactin
      </option>
      
      <option value = "svg/knownclusterblast_r16c1_5.svg">
        SF2575
      </option>
      
      <option value = "svg/knownclusterblast_r16c1_6.svg">
        mirubactin
      </option>
      
    </select>
    <button id="knownclusterblast-r16c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r16c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r16c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-76"></div> <span class="help-tooltip" id="cb-sub-help-76">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r16c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-77"></div> <span class="help-tooltip" id="MIBiG-help-77">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r16c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r16c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r16c1"></div>
    <div id="comparison-MIBiG-r16c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002091.1: 3593-21565">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002091/index.html#r1c1" target="_new">BGC0002091.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>holomycin</td>
        
        
        <td>Yersinia ruckeri ATCC 29473</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000632.1: 2720626-2736748">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000632/index.html#r1c1" target="_new">BGC0000632.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene, Saccharide</td>
        
        <td>brasilicardin A</td>
        
        
        <td>Nocardia terpenica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B</td>
        
        
        <td>Streptomyces sp. WK-5344</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>grixazone</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000373.1: 6204479-6222797">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000373/index.html#r1c1" target="_new">BGC0000373.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>holomycin</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001292.1: 5468-37257">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001292/index.html#r1c1" target="_new">BGC0001292.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.51, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Other</td>
        
        <td>A33853</td>
        
        
        <td>Streptomyces sp. NRRL 12068</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000412.1: 804-15023">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000412/index.html#r1c1" target="_new">BGC0000412.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.50, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>pyochelin</td>
        
        
        <td>Pseudomonas aeruginosa PAO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001592.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001592/index.html#r1c1" target="_new">BGC0001592.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Other</td>
        
        <td>ferroverdin</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002029.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002029/index.html#r1c1" target="_new">BGC0002029.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B, bagremycin C, bagremycin E, bagremycin F, bagremycin G, ferroverdin A, ferroverdin B, ferroverdin C</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r16c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002091.1: 3593-21565">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002091/index.html#r1c1" target="_new">BGC0002091.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>holomycin</td>
      <td>Yersinia ruckeri ATCC 29473</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000632.1: 2720626-2736748">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000632/index.html#r1c1" target="_new">BGC0000632.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Terpene, Saccharide</td>
      <td>brasilicardin A</td>
      <td>Nocardia terpenica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B</td>
      <td>Streptomyces sp. WK-5344</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene</td>
      <td>grixazone</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000373.1: 6204479-6222797">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000373/index.html#r1c1" target="_new">BGC0000373.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>holomycin</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001292.1: 5468-37257">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001292/index.html#r1c1" target="_new">BGC0001292.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.51, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Other</td>
      <td>A33853</td>
      <td>Streptomyces sp. NRRL 12068</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000412.1: 804-15023">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000412/index.html#r1c1" target="_new">BGC0000412.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.50, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>pyochelin</td>
      <td>Pseudomonas aeruginosa PAO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001592.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001592/index.html#r1c1" target="_new">BGC0001592.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Other</td>
      <td>ferroverdin</td>
      <td>Streptomyces lunaelactis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002029.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002029/index.html#r1c1" target="_new">BGC0002029.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B, bagremycin C, bagremycin E, bagremycin F, bagremycin G, ferroverdin A, ferroverdin B, ferroverdin C</td>
      <td>Streptomyces lunaelactis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r16c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-78"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-78">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r16c1-pfam-details-svg" data-filename="CC227C_contig_18_r16c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r16c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r16c1-nrps_pks_products" data-name="r16c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r16c1-nrps_pks" data-name="r16c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r16c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-72"></div> <span class="help-tooltip" id="nrps-structures-help-72">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 40480 - 84770:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r16c1-1" class="smiles-canvas" data-smiles="NC(CS)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(cys)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r16c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CS)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r16c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-73"></div> <span class="help-tooltip" id="nrps-monomers-help-73">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds60480_64770</span></strong>: cys
      <div class="collapser collapser-target-cds60480_64770 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (813..1220)</strong>: cys
          <div class="collapser collapser-target-cds60480_64770 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  cys
              <div class="collapser collapser-target-cds60480_64770-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Small clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Single AA prediction:</dt>
   <dd>cys</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLYNigLIhK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>cys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds64918_66538</span></strong>: sal
      <div class="collapser collapser-target-cds64918_66538 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=diOH-Bz" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=diOH-Bz*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (41..445)</strong>: sal
          <div class="collapser collapser-target-cds64918_66538 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  sal
              <div class="collapser collapser-target-cds64918_66538-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>dhb, sal</dd>
   <dt>Small clusters prediction:</dt>
   <dd>dhb, sal</dd>
   <dt>Single AA prediction:</dt>
   <dd>dhb</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">PLPAQGVLNK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>sal</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r17c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_9 - Region 1 - ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-255"></div> <span class="help-tooltip" id="region_svg-help-255">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_9.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r17c1-svg" data-filename="CC227C_contig_9_r17c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 114,521 - 124,916 nt. (total: 10,396 nt)
        
        <a class="cluster-rules-header" id="r17c1-rules-header" href="#r17c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r17c1-rules">ectoine: ectoine_synt</div>
      
      <div class="region-svg-container">
       <div id='r17c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-256"></div> <span class="help-tooltip" id="focus-panel-help-256">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r17c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r17c1-clusterblast" data-name="r17c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r17c1-knownclusterblast" data-name="r17c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r17c1-subclusterblast" data-name="r17c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r17c1-MIBiG-cluster-compare" data-name="r17c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r17c1-pfam-details" data-name="r17c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r17c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-79"></div> <span class="help-tooltip" id="cb-general-help-79">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r17c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r17c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r17c1_1.svg">
        NZ_CP041695 (2459961-2470357): Nocardia otitidiscaviarum strain NEB252 chromo... (91% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r17c1_2.svg">
        NZ_QQAZ01000007 (176529-186922): Nocardia mexicana strain DSM 44952 Ga0244508... (60% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r17c1_3.svg">
        NZ_QQBC01000001 (1002432-1012825): Nocardia pseudobrasiliensis strain DSM 442... (66% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r17c1_4.svg">
        NZ_CP023778 (6348209-6358602): Nocardia terpenica strain NC YFY NT001 chromos... (66% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r17c1_5.svg">
        NZ_CP046173 (6324871-6335264): Nocardia terpenica strain AUSMDU00012715 chrom... (66% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r17c1_6.svg">
        NZ_CP029710 (1212676-1223075): Nocardia sp. CS682 chromosome, complete genome (66% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r17c1_7.svg">
        NZ_AXVD01000007 (169410-179809): Nocardia sp. CNY236 K940DRAFT scaffold00006.... (60% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r17c1_8.svg">
        NZ_LT985361 (6111559-6121958): Nocardia suismassiliense strain S-137, whole g... (66% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r17c1_9.svg">
        NZ_KV411303 (1477051-1487444): Nocardia terpenica strain IFM 0406 scaffold1, ... (66% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r17c1_10.svg">
        NZ_CP006850 (6131706-6142099): Nocardia nova SH22a chromosome, complete genome (75% of genes show similarity), ectoine
      </option>
      
    </select>
    <button id= "clusterblast-r17c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r17c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r17c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-80"></div> <span class="help-tooltip" id="cb-known-help-80">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r17c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r17c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r17c1_1.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r17c1_2.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r17c1_3.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r17c1_4.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r17c1_5.svg">
        ectoine
      </option>
      
    </select>
    <button id="knownclusterblast-r17c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r17c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r17c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-81"></div> <span class="help-tooltip" id="cb-sub-help-81">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r17c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-82"></div> <span class="help-tooltip" id="MIBiG-help-82">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r17c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r17c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r17c1"></div>
    <div id="comparison-MIBiG-r17c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">ectoine</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
         
        
        <td>0.54</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.81 (id:0.52, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
         
        
        <td>0.54</td>
        <td>Other,  , (, E, c, t, o, i, n, e, )</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857.1: 300-4871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.52, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylophaga alcalica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852.1: 415-2854">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Sporosarcina pasteurii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860.1: 710-5134">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.49, order:0.64, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylarcula marina</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.51, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylobacter marinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855.1: 245-3455">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.50, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium kenyense</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.50, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylophaga thalassica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.50, order:0.48, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum 20Z</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r17c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
      <td>0.81</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.81 (id:0.52, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
      <td>0.81</td>
      <td>Other,  , (, E, c, t, o, i, n, e, )</td>
      <td>ectoine</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857.1: 300-4871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.52, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylophaga alcalica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852.1: 415-2854">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Sporosarcina pasteurii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860.1: 710-5134">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.49, order:0.64, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylarcula marina</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.51, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylobacter marinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855.1: 245-3455">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.50, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium kenyense</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.50, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylophaga thalassica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.50, order:0.48, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum 20Z</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r17c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-83"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-83">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r17c1-pfam-details-svg" data-filename="CC227C_contig_9_r17c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r17c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r17c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_9 - Region 2 - NRPS,NRPS-like,terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-257"></div> <span class="help-tooltip" id="region_svg-help-257">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_9.region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r17c2-svg" data-filename="CC227C_contig_9_r17c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 167,098 - 253,921 nt. (total: 86,824 nt)
        
        <a class="cluster-rules-header" id="r17c2-rules-header" href="#r17c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r17c2-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))<br>terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r17c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-258"></div> <span class="help-tooltip" id="focus-panel-help-258">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r17c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r17c2-nrps_pks" data-name="r17c2-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r17c2-clusterblast" data-name="r17c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r17c2-knownclusterblast" data-name="r17c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r17c2-subclusterblast" data-name="r17c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r17c2-MIBiG-cluster-compare" data-name="r17c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r17c2-pfam-details" data-name="r17c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r17c2-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-84"></div> <span class="help-tooltip" id="nrps-domain-help-84">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r17c2-details-svg" data-filename="CC227C_contig_9_r17c2_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r17c2-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r17c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-87"></div> <span class="help-tooltip" id="cb-general-help-87">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r17c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r17c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r17c2_1.svg">
        NZ_CP041695 (2305189-2396969): Nocardia otitidiscaviarum strain NEB252 chromo... (39% of genes show similarity), NRPS,butyrolactone,terpene
      </option>
      
      <option value = "svg/clusterblast_r17c2_2.svg">
        NZ_CP026746 (2452609-2630012): Nocardia cyriacigeorgica strain MDA3349 isolat... (14% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r17c2_3.svg">
        NC_016887 (5275203-5451304): Nocardia cyriacigeorgica GUH-2, complete genome (14% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r17c2_4.svg">
        NZ_LR215973 (3860935-4031992): Nocardia cyriacigeorgica strain 3012STDY675650... (13% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r17c2_5.svg">
        NC_006361 (5224713-5381654): Nocardia farcinica IFM 10152, complete genome (15% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r17c2_6.svg">
        NZ_CP031418 (4331502-4496115): Nocardia farcinica strain W6977 chromosome, co... (14% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r17c2_7.svg">
        NZ_QQAZ01000004 (291543-403900): Nocardia mexicana strain DSM 44952 Ga0244508... (16% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r17c2_8.svg">
        NZ_QJKF01000002 (71512-175150): Nocardia tenerifensis strain DSM 44704 Ga0244... (15% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r17c2_9.svg">
        NZ_OBEG01000002 (894236-978079): Nocardia amikacinitolerans strain DSM 45537,... (23% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r17c2_10.svg">
        NZ_CP029710 (3661313-3767589): Nocardia sp. CS682 chromosome, complete genome (13% of genes show similarity), NRPS,bacteriocin
      </option>
      
    </select>
    <button id= "clusterblast-r17c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r17c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r17c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-88"></div> <span class="help-tooltip" id="cb-known-help-88">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r17c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-89"></div> <span class="help-tooltip" id="cb-sub-help-89">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r17c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-90"></div> <span class="help-tooltip" id="MIBiG-help-90">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r17c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r17c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r17c2"></div>
    <div id="comparison-MIBiG-r17c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000657.1: 643-5774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000657/index.html#r1c1" target="_new">BGC0000657.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.53, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.53, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
        <td>0.83</td>
        <td>Terpene</td>
        
        <td>2-methylisoborneol</td>
        
        
        <td>Pseudanabaena sp. dqh15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.48</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.46</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001824.1: 39732-57269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001824/index.html#r1c1" target="_new">BGC0001824.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.46</td>
        <td>NRP</td>
        
        <td>xefoampeptides A-G</td>
        
        
        <td>Xenorhabdus beddingii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.46</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.46</td>
        <td>NRP</td>
        
        <td>rhodochelin</td>
        
        
        <td>Rhodococcus jostii RHA1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.45</td>
        <td>NRP</td>
        
        <td>bacillibactin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.44</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.44</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001149.1: 196767-209553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001149/index.html#r1c1" target="_new">BGC0001149.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.45, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.43</td>
        <td>NRP, Saccharide</td>
        
        <td>brabantamide A</td>
        
        
        <td>Pseudomonas sp. SHC52</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r17c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000657.1: 643-5774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000657/index.html#r1c1" target="_new">BGC0000657.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.53, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>Terpene</td>
      <td>2-methylisoborneol</td>
      <td>Pseudanabaena sp. dqh15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>rhodochelin</td>
      <td>Rhodococcus jostii RHA1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001599.1: 222370-231743">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001599/index.html#r1c1" target="_new">BGC0001599.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>fragin</td>
      <td>Burkholderia cenocepacia H111</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001900.1: 225531-231743">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001900/index.html#r1c1" target="_new">BGC0001900.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>fragin</td>
      <td>Burkholderia cenocepacia H111</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>vicibactin</td>
      <td>Rhizobium etli CFN 42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401.1: 0-12882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>paenibactin</td>
      <td>Paenibacillus elgii B69</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000454.1: 24-19884">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000454/index.html#r1c1" target="_new">BGC0000454.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>vanchrobactin</td>
      <td>Vibrio anguillarum RV22</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r17c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-91"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-91">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r17c2-pfam-details-svg" data-filename="CC227C_contig_9_r17c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r17c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r17c2-nrps_pks_products" data-name="r17c2-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r17c2-nrps_pks" data-name="r17c2-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r17c2-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-85"></div> <span class="help-tooltip" id="nrps-structures-help-85">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 167097 - 253921:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r17c2-2" class="smiles-canvas" data-smiles="N1C(CCC1)C(=O)NC(CC(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-pro - D-leu - X - X) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-pro,d-leu,x,x&amp;nrps2=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-pro*,d-leu*,x,x&amp;nrps2=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r17c2-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="N1C(CCC1)C(=O)NC(CC(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 167097 - 208389:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r17c2-3" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r17c2-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 4, location 197598 - 253921:
          <div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r17c2-4" class="smiles-canvas" data-smiles="N1C(CCC1)C(=O)NC(CC(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-pro - D-leu - X - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-pro,d-leu,x,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-pro*,d-leu*,x,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r17c2-4-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="N1C(CCC1)C(=O)NC(CC(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r17c2-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-86"></div> <span class="help-tooltip" id="nrps-monomers-help-86">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds177275_179078</span></strong>: X
      <div class="collapser collapser-target-cds177275_179078 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (31..469)</strong>: X
          <div class="collapser collapser-target-cds177275_179078 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds177275_179078-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">---LIGgaaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>40% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds186085_189088</span></strong>: X
      <div class="collapser collapser-target-cds186085_189088 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (11..398)</strong>: X
          <div class="collapser collapser-target-cds186085_189088 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds186085_189088-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWyFayIfK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds208416_210531</span></strong>: X
      <div class="collapser collapser-target-cds208416_210531 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (200..590)</strong>: X
          <div class="collapser collapser-target-cds208416_210531 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds208416_210531-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">htslaCiwAK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ala</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>30% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds217598_233921</span></strong>: pro - leu - X - X
      <div class="collapser collapser-target-cds217598_233921 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pro,leu,X,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pro*,leu*,X*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (31..426)</strong>: pro
          <div class="collapser collapser-target-cds217598_233921 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  pro
              <div class="collapser collapser-target-cds217598_233921-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>pro</dd>
   <dt>Single AA prediction:</dt>
   <dd>pro</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVLLVAGVLK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pro</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1778..2179)</strong>: leu
          <div class="collapser collapser-target-cds217598_233921 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  leu
              <div class="collapser collapser-target-cds217598_233921-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>leu</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DPliSGgiiK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>glu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3547..3938)</strong>: X
          <div class="collapser collapser-target-cds217598_233921 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  phe, trp, phg, tyr, bht
              <div class="collapser collapser-target-cds217598_233921-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DqeisGGIGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (4600..4995)</strong>: X
          <div class="collapser collapser-target-cds217598_233921 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds217598_233921-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLWHLiLvsK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r18c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_35 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-259"></div> <span class="help-tooltip" id="region_svg-help-259">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_35.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r18c1-svg" data-filename="CC227C_contig_35_r18c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 10,719 nt. (total: 10,719 nt)
        
        <a class="cluster-rules-header" id="r18c1-rules-header" href="#r18c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r18c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r18c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-260"></div> <span class="help-tooltip" id="focus-panel-help-260">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r18c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r18c1-nrps_pks" data-name="r18c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r18c1-clusterblast" data-name="r18c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r18c1-knownclusterblast" data-name="r18c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r18c1-subclusterblast" data-name="r18c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r18c1-MIBiG-cluster-compare" data-name="r18c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r18c1-pfam-details" data-name="r18c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r18c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-92"></div> <span class="help-tooltip" id="nrps-domain-help-92">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r18c1-details-svg" data-filename="CC227C_contig_35_r18c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r18c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r18c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-95"></div> <span class="help-tooltip" id="cb-general-help-95">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <p>No significant ClusterBlast hits found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r18c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-96"></div> <span class="help-tooltip" id="cb-known-help-96">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r18c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-97"></div> <span class="help-tooltip" id="cb-sub-help-97">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r18c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-98"></div> <span class="help-tooltip" id="MIBiG-help-98">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r18c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r18c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r18c1"></div>
    <div id="comparison-MIBiG-r18c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000307.1: 505-35485">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000307/index.html#r1c1" target="_new">BGC0000307.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>AbT1</td>
        
        
        <td>Aureobasidium pullulans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>vicibactin</td>
        
        
        <td>Rhizobium etli CFN 42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r18c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000307.1: 505-35485">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000307/index.html#r1c1" target="_new">BGC0000307.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>AbT1</td>
      <td>Aureobasidium pullulans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>vicibactin</td>
      <td>Rhizobium etli CFN 42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r18c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-99"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-99">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r18c1-pfam-details-svg" data-filename="CC227C_contig_35_r18c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r18c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r18c1-nrps_pks_products" data-name="r18c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r18c1-nrps_pks" data-name="r18c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r18c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-93"></div> <span class="help-tooltip" id="nrps-structures-help-93">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 10719:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r18c1-1" class="smiles-canvas" data-smiles="NC(C(O)C)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(thr - D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr,d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*,d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r18c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(C(O)C)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r18c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-94"></div> <span class="help-tooltip" id="nrps-monomers-help-94">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds0_10719</span></strong>: thr - X
      <div class="collapser collapser-target-cds0_10719 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (663..1085)</strong>: thr
          <div class="collapser collapser-target-cds0_10719 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds0_10719-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNVGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1776..2176)</strong>: X
          <div class="collapser collapser-target-cds0_10719 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala, val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds0_10719-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DaLfCgGIMK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>aeo</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r21c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_20 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-261"></div> <span class="help-tooltip" id="region_svg-help-261">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_20.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r21c1-svg" data-filename="CC227C_contig_20_r21c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 19,012 - 61,864 nt. (total: 42,853 nt)
        
        <a class="cluster-rules-header" id="r21c1-rules-header" href="#r21c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r21c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r21c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-262"></div> <span class="help-tooltip" id="focus-panel-help-262">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r21c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r21c1-nrps_pks" data-name="r21c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r21c1-clusterblast" data-name="r21c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r21c1-knownclusterblast" data-name="r21c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r21c1-subclusterblast" data-name="r21c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r21c1-MIBiG-cluster-compare" data-name="r21c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r21c1-pfam-details" data-name="r21c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r21c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-100"></div> <span class="help-tooltip" id="nrps-domain-help-100">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r21c1-details-svg" data-filename="CC227C_contig_20_r21c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r21c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r21c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-103"></div> <span class="help-tooltip" id="cb-general-help-103">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r21c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r21c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r21c1_1.svg">
        NZ_LR134352 (4722000-4795266): Nocardia asteroides strain NCTC11293 chromosome 1 (40% of genes show similarity), NRPS-like,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r21c1_2.svg">
        NZ_CP023778 (3809658-3851139): Nocardia terpenica strain NC YFY NT001 chromos... (60% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r21c1_3.svg">
        NZ_KV411304 (2058818-2146514): Nocardia terpenica strain IFM 0406 scaffold2, ... (26% of genes show similarity), NRPS,NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r21c1_4.svg">
        NZ_CP046173 (3798006-3884863): Nocardia terpenica strain AUSMDU00012715 chrom... (26% of genes show similarity), NRPS,NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r21c1_5.svg">
        NZ_CP046172 (5993894-6035474): Nocardia arthritidis strain AUSMDU00012717 chr... (59% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r21c1_6.svg">
        NZ_CP029710 (1394230-1433865): Nocardia sp. CS682 chromosome, complete genome (58% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r21c1_7.svg">
        NZ_LT985361 (6298032-6339142): Nocardia suismassiliense strain S-137, whole g... (54% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r21c1_8.svg">
        NZ_QJKF01000012 (135614-178302): Nocardia tenerifensis strain DSM 44704 Ga024... (50% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r21c1_9.svg">
        NZ_QGTL01000017 (12682-53855): Nocardia neocaledoniensis strain DSM 44717 Ga0... (20% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r21c1_10.svg">
        NZ_VMSD01000003 (110426-151599): Nocardia caishijiensis strain DSM 44831 Ga02... (22% of genes show similarity), T3PKS
      </option>
      
    </select>
    <button id= "clusterblast-r21c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r21c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r21c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-104"></div> <span class="help-tooltip" id="cb-known-help-104">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r21c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-105"></div> <span class="help-tooltip" id="cb-sub-help-105">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r21c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-106"></div> <span class="help-tooltip" id="MIBiG-help-106">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r21c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r21c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r21c1"></div>
    <div id="comparison-MIBiG-r21c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>auriculamide</td>
        
        
        <td>Herpetosiphon aurantiacus DSM 785</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843.1: 3437627-3461293">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
        
        <td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Other</td>
        
        <td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>
        
        
        <td>Streptomyces sp. RM-5-8</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.42, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>showdomycin</td>
        
        
        <td>Streptomyces showdoensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001510.1: 425-14535">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001510/index.html#r1c1" target="_new">BGC0001510.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>anisomycin</td>
        
        
        <td>Streptomyces hygrospinosus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001783.1: 55-63934">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001783/index.html#r1c1" target="_new">BGC0001783.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>streptonigrin</td>
        
        
        <td>Streptomyces flocculus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>RiPP</td>
        
        <td>informatipeptin</td>
        
        
        <td>Streptomyces viridochromogenes DSM 40736</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000897.1: 0-20143">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000897/index.html#r1c1" target="_new">BGC0000897.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>dehydrophos</td>
        
        
        <td>Streptomyces luridus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r21c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>auriculamide</td>
      <td>Herpetosiphon aurantiacus DSM 785</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843.1: 3437627-3461293">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
      <td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Other</td>
      <td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>
      <td>Streptomyces sp. RM-5-8</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.42, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>showdomycin</td>
      <td>Streptomyces showdoensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001510.1: 425-14535">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001510/index.html#r1c1" target="_new">BGC0001510.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Other</td>
      <td>anisomycin</td>
      <td>Streptomyces hygrospinosus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001783.1: 55-63934">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001783/index.html#r1c1" target="_new">BGC0001783.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>streptonigrin</td>
      <td>Streptomyces flocculus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>RiPP</td>
      <td>informatipeptin</td>
      <td>Streptomyces viridochromogenes DSM 40736</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000897.1: 0-20143">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000897/index.html#r1c1" target="_new">BGC0000897.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>dehydrophos</td>
      <td>Streptomyces luridus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r21c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-107"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-107">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r21c1-pfam-details-svg" data-filename="CC227C_contig_20_r21c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r21c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r21c1-nrps_pks_products" data-name="r21c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r21c1-nrps_pks" data-name="r21c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r21c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-101"></div> <span class="help-tooltip" id="nrps-structures-help-101">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 19011 - 61864:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r21c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r21c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r21c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-102"></div> <span class="help-tooltip" id="nrps-monomers-help-102">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds39011_41864</span></strong>: X
      <div class="collapser collapser-target-cds39011_41864 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (12..363)</strong>: X
          <div class="collapser collapser-target-cds39011_41864 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds39011_41864-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">cysaTAmarK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>30% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r23c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_33 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-263"></div> <span class="help-tooltip" id="region_svg-help-263">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_33.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r23c1-svg" data-filename="CC227C_contig_33_r23c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 23,361 nt. (total: 23,361 nt)
        
        <a class="cluster-rules-header" id="r23c1-rules-header" href="#r23c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r23c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r23c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-264"></div> <span class="help-tooltip" id="focus-panel-help-264">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r23c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r23c1-nrps_pks" data-name="r23c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r23c1-clusterblast" data-name="r23c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r23c1-knownclusterblast" data-name="r23c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r23c1-subclusterblast" data-name="r23c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r23c1-MIBiG-cluster-compare" data-name="r23c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r23c1-pfam-details" data-name="r23c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r23c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-108"></div> <span class="help-tooltip" id="nrps-domain-help-108">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r23c1-details-svg" data-filename="CC227C_contig_33_r23c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r23c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r23c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-111"></div> <span class="help-tooltip" id="cb-general-help-111">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r23c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r23c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r23c1_1.svg">
        NZ_CP041695 (107738-270958): Nocardia otitidiscaviarum strain NEB252 chromoso... (24% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r23c1_2.svg">
        NZ_CP017839 (1479309-1588618): Nocardia seriolae strain EM150506 chromosome, ... (15% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r23c1_3.svg">
        NZ_CP029710 (3661313-3767589): Nocardia sp. CS682 chromosome, complete genome (15% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r23c1_4.svg">
        NZ_LT985361 (8547642-8654705): Nocardia suismassiliense strain S-137, whole g... (15% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r23c1_5.svg">
        NZ_CP022088 (7416645-7521220): Nocardia brasiliensis strain FDAARGOS 352 chro... (16% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r23c1_6.svg">
        NC_018681 (8472101-8575358): Nocardia brasiliensis ATCC 700358, complete sequ... (16% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r23c1_7.svg">
        NZ_CP059399 (1970888-2141609): Nocardia sp. WCH-YHL-001 chromosome, complete ... (18% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r23c1_8.svg">
        NZ_QQBC01000004 (306033-391403): Nocardia pseudobrasiliensis strain DSM 44290... (19% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r23c1_9.svg">
        NZ_CP046171 (6972095-7074285): Nocardia brasiliensis strain AUSMDU00024985 ch... (15% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r23c1_10.svg">
        NZ_CP032568 (1453702-1606647): Nocardia sp. CFHS0054 chromosome (14% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r23c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r23c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r23c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-112"></div> <span class="help-tooltip" id="cb-known-help-112">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r23c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r23c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r23c1_1.svg">
        isorenieratene
      </option>
      
    </select>
    <button id="knownclusterblast-r23c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r23c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r23c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-113"></div> <span class="help-tooltip" id="cb-sub-help-113">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r23c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-114"></div> <span class="help-tooltip" id="MIBiG-help-114">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r23c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r23c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r23c1"></div>
    <div id="comparison-MIBiG-r23c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131.1: 144016-173487">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>ambactin</td>
        
        
        <td>Xenorhabdus miraniensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001686.1: 28257-44195">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001686/index.html#r1c1" target="_new">BGC0001686.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>N-octanoyl-Met-Phe-H</td>
        
        
        <td>Clostridium sp. CAG:567</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342.1: 1020-10416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>enniatin</td>
        
        
        <td>Fusarium equiseti</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r23c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131.1: 144016-173487">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>ambactin</td>
      <td>Xenorhabdus miraniensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001686.1: 28257-44195">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001686/index.html#r1c1" target="_new">BGC0001686.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>N-octanoyl-Met-Phe-H</td>
      <td>Clostridium sp. CAG:567</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342.1: 1020-10416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>enniatin</td>
      <td>Fusarium equiseti</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r23c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-115"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-115">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r23c1-pfam-details-svg" data-filename="CC227C_contig_33_r23c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r23c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r23c1-nrps_pks_products" data-name="r23c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r23c1-nrps_pks" data-name="r23c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r23c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-109"></div> <span class="help-tooltip" id="nrps-structures-help-109">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 23361:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r23c1-1" class="smiles-canvas" data-smiles="NC(CO)C(=O)NC([*])C(=O)NC(CO)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ser - X - ser)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,x,ser" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,x,ser*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r23c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CO)C(=O)NC([*])C(=O)NC(CO)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r23c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-110"></div> <span class="help-tooltip" id="nrps-monomers-help-110">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds0_12216</span></strong>: ser - X - ser - X
      <div class="collapser collapser-target-cds0_12216 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,X,ser,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,X*,ser*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (482..902)</strong>: ser
          <div class="collapser collapser-target-cds0_12216 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds0_12216-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWdFiLVsK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1603..2015)</strong>: X
          <div class="collapser collapser-target-cds0_12216 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds0_12216-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWslAlVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2683..3120)</strong>: ser
          <div class="collapser collapser-target-cds0_12216 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds0_12216-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWHFSLVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3831..4070)</strong>: X
          <div class="collapser collapser-target-cds0_12216 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aromatic
              <div class="collapser collapser-target-cds0_12216-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">Dg-----tGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>glu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>30% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r24c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_11 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-265"></div> <span class="help-tooltip" id="region_svg-help-265">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_11.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r24c1-svg" data-filename="CC227C_contig_11_r24c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 53,665 - 98,929 nt. (total: 45,265 nt)
        
        <a class="cluster-rules-header" id="r24c1-rules-header" href="#r24c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r24c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r24c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-266"></div> <span class="help-tooltip" id="focus-panel-help-266">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r24c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r24c1-nrps_pks" data-name="r24c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r24c1-clusterblast" data-name="r24c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r24c1-knownclusterblast" data-name="r24c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r24c1-subclusterblast" data-name="r24c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r24c1-MIBiG-cluster-compare" data-name="r24c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r24c1-pfam-details" data-name="r24c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r24c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-116"></div> <span class="help-tooltip" id="nrps-domain-help-116">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r24c1-details-svg" data-filename="CC227C_contig_11_r24c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r24c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r24c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-119"></div> <span class="help-tooltip" id="cb-general-help-119">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r24c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r24c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r24c1_1.svg">
        NZ_CP041695 (6823441-6868679): Nocardia otitidiscaviarum strain NEB252 chromo... (100% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r24c1_2.svg">
        NZ_CP023778 (156653-200113): Nocardia terpenica strain NC YFY NT001 chromosom... (76% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r24c1_3.svg">
        NZ_KV411303 (2233545-2277315): Nocardia terpenica strain IFM 0406 scaffold1, ... (76% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r24c1_4.svg">
        NZ_CP046173 (166522-210404): Nocardia terpenica strain AUSMDU00012715 chromos... (76% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r24c1_5.svg">
        NZ_QQBC01000013 (193550-238662): Nocardia pseudobrasiliensis strain DSM 44290... (71% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r24c1_6.svg">
        NZ_CP006850 (150734-195543): Nocardia nova SH22a chromosome, complete genome (70% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r24c1_7.svg">
        NZ_OBEG01000004 (147256-190572): Nocardia amikacinitolerans strain DSM 45537,... (67% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r24c1_8.svg">
        NZ_CP046171 (200316-245058): Nocardia brasiliensis strain AUSMDU00024985 chro... (70% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r24c1_9.svg">
        NZ_CP029710 (4787934-4831388): Nocardia sp. CS682 chromosome, complete genome (67% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r24c1_10.svg">
        NZ_LT985361 (555246-598774): Nocardia suismassiliense strain S-137, whole gen... (70% of genes show similarity), T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r24c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r24c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r24c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-120"></div> <span class="help-tooltip" id="cb-known-help-120">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r24c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-121"></div> <span class="help-tooltip" id="cb-sub-help-121">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r24c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-122"></div> <span class="help-tooltip" id="MIBiG-help-122">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r24c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r24c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r24c1"></div>
    <div id="comparison-MIBiG-r24c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Polyketide</td>
        
        <td>strobilurin</td>
        
        
        <td>Strobilurus tenacellus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831.1: 8162481-8168632">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>alkylpyrone-407, alkylpyrone-393</td>
        
        
        <td>Myxococcus xanthus DK 1622</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Terpene, Polyketide</td>
        
        <td>pyripyropene A</td>
        
        
        <td>unidentified unclassified sequences.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001161.1: 0-9161">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001161/index.html#r1c1" target="_new">BGC0001161.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:1.00, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>1-heptadecene</td>
        
        
        <td>Moorea bouillonii PNG</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001163.1: 308352-317651">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001163/index.html#r1c1" target="_new">BGC0001163.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:1.00, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>1-heptadecene</td>
        
        
        <td>Moorea producens 3L</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001162.1: 0-9348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001162/index.html#r1c1" target="_new">BGC0001162.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:1.00, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>1-heptadecene</td>
        
        
        <td>Moorea producens JHB</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007.1: 5922480-5942549">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Terpene</td>
        
        <td>atolypene A, atolypene B</td>
        
        
        <td>Amycolatopsis tolypomycina</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000236.1: 35-34224">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000236/index.html#r1c1" target="_new">BGC0000236.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>kinamycin</td>
        
        
        <td>Streptomyces murayamaensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002081.1: 0-30441">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002081/index.html#r1c1" target="_new">BGC0002081.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>streptoketide A, streptoketide B, streptoketide C, S2502, S2507, UWM5</td>
        
        
        <td>Streptomyces sp. Tue6314</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000044.1: 1916098-1937355">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000044/index.html#r1c1" target="_new">BGC0000044.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>dawenol</td>
        
        
        <td>Stigmatella aurantiaca DW4/3-1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r24c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Polyketide</td>
      <td>strobilurin</td>
      <td>Strobilurus tenacellus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831.1: 8162481-8168632">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>alkylpyrone-407, alkylpyrone-393</td>
      <td>Myxococcus xanthus DK 1622</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene, Polyketide</td>
      <td>pyripyropene A</td>
      <td>unidentified unclassified sequences.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001161.1: 0-9161">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001161/index.html#r1c1" target="_new">BGC0001161.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.44, order:1.00, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>1-heptadecene</td>
      <td>Moorea bouillonii PNG</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001163.1: 308352-317651">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001163/index.html#r1c1" target="_new">BGC0001163.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.44, order:1.00, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>1-heptadecene</td>
      <td>Moorea producens 3L</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001162.1: 0-9348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001162/index.html#r1c1" target="_new">BGC0001162.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.44, order:1.00, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>1-heptadecene</td>
      <td>Moorea producens JHB</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007.1: 5922480-5942549">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Terpene</td>
      <td>atolypene A, atolypene B</td>
      <td>Amycolatopsis tolypomycina</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000236.1: 35-34224">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000236/index.html#r1c1" target="_new">BGC0000236.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>kinamycin</td>
      <td>Streptomyces murayamaensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002081.1: 0-30441">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002081/index.html#r1c1" target="_new">BGC0002081.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.48, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>streptoketide A, streptoketide B, streptoketide C, S2502, S2507, UWM5</td>
      <td>Streptomyces sp. Tue6314</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000044.1: 1916098-1937355">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000044/index.html#r1c1" target="_new">BGC0000044.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>dawenol</td>
      <td>Stigmatella aurantiaca DW4/3-1</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r24c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-123"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-123">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r24c1-pfam-details-svg" data-filename="CC227C_contig_11_r24c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r24c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r24c1-nrps_pks_products" data-name="r24c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r24c1-nrps_pks" data-name="r24c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r24c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-117"></div> <span class="help-tooltip" id="nrps-structures-help-117">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 53664 - 98929:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r24c1-1" class="smiles-canvas" data-smiles="C(CC)C(=O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(emal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=emal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=emal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r24c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C(CC)C(=O)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r24c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-118"></div> <span class="help-tooltip" id="nrps-monomers-help-118">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds73664_78929</span></strong>: emal
      <div class="collapser collapser-target-cds73664_78929 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (709..1013)</strong>: emal
          <div class="collapser collapser-target-cds73664_78929 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Ethylmalonyl-CoA
              <div class="collapser collapser-target-cds73664_78929-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
<dd>Methoxymalonyl-CoA: 54.2%</dd>
<dd>Methylmalonyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  fatty_acid
              <div class="collapser collapser-target-cds73664_78929-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>fatty_acid: 138.3</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 87.6</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 78.7</dt>

  <dd></dd><dt>Malonyl-CoA: 74.6</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 62.3</dt>

  <dd></dd><dt>trans-1,2-CPDA: 47.6</dt>

  <dd></dd><dt>Propionyl-CoA: 47.4</dt>

  <dd></dd><dt>Acetyl-CoA: 40.3</dt>

  <dd></dd><dt>Isobutyryl-CoA: 33.3</dt>

  <dd></dd><dt>Benzoyl-CoA: 17.8</dt>

  <dd></dd><dt>CHC-CoA: 17.6</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 16.6</dt>

  <dd></dd><dt>inactive: 12.1</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: -1.3</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds79030_80953</span></strong>: pk
      <div class="collapser collapser-target-cds79030_80953 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>CAL_domain (57..521)</strong>: pk
          <div class="collapser collapser-target-cds79030_80953 collapser-level-candidate"> </div><div class="collapser-content">
          
              minowa:  Acetyl-CoA
              <div class="collapser collapser-target-cds79030_80953-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Acetyl-CoA: 39.8</dt>

  <dd></dd><dt>AHBA: 15.0</dt>

  <dd></dd><dt>shikimic_acid: 5.3</dt>

  <dd></dd><dt>NH2: 4.9</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r26c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_14 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-267"></div> <span class="help-tooltip" id="region_svg-help-267">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_14.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r26c1-svg" data-filename="CC227C_contig_14_r26c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 27,896 nt. (total: 27,896 nt)
        
        <a class="cluster-rules-header" id="r26c1-rules-header" href="#r26c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r26c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r26c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-268"></div> <span class="help-tooltip" id="focus-panel-help-268">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r26c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r26c1-nrps_pks" data-name="r26c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r26c1-clusterblast" data-name="r26c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r26c1-knownclusterblast" data-name="r26c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r26c1-subclusterblast" data-name="r26c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r26c1-MIBiG-cluster-compare" data-name="r26c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r26c1-pfam-details" data-name="r26c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r26c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-124"></div> <span class="help-tooltip" id="nrps-domain-help-124">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r26c1-details-svg" data-filename="CC227C_contig_14_r26c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r26c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r26c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-127"></div> <span class="help-tooltip" id="cb-general-help-127">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r26c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r26c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r26c1_1.svg">
        NZ_CP041695 (4596294-4639507): Nocardia otitidiscaviarum strain NEB252 chromo... (42% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r26c1_2.svg">
        NZ_CP059399 (6912997-6954140): Nocardia sp. WCH-YHL-001 chromosome, complete ... (33% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r26c1_3.svg">
        NZ_QHCP01000005 (265233-318765): Actinokineospora mzabensis strain CECT 8578 ... (12% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r26c1_4.svg">
        NZ_MKQR01000007 (12026-65701): Actinokineospora bangkokensis strain 44EHW Sca... (13% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r26c1_5.svg">
        NZ_AP022583 (3415795-3456803): Mycobacterium noviomagense strain JCM 16367 (14% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r26c1_6.svg">
        NZ_LMFE01000008 (338405-381864): Nocardia sp. Root136 contig 3, whole genome ... (8% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r26c1_7.svg">
        NZ_QGTL01000001 (574039-636770): Nocardia neocaledoniensis strain DSM 44717 G... (15% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r26c1_8.svg">
        NZ_CP018082 (4653919-4716026): Nocardia mangyaensis strain Y48 chromosome, co... (16% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r26c1_9.svg">
        NZ_VMSD01000003 (664462-727013): Nocardia caishijiensis strain DSM 44831 Ga02... (15% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r26c1_10.svg">
        NZ_SNXK01000001 (751230-814154): Nocardia ignorata strain DSM 44496 Ga0244500... (16% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r26c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r26c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r26c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-128"></div> <span class="help-tooltip" id="cb-known-help-128">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r26c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-129"></div> <span class="help-tooltip" id="cb-sub-help-129">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r26c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-130"></div> <span class="help-tooltip" id="MIBiG-help-130">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r26c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r26c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r26c1"></div>
    <div id="comparison-MIBiG-r26c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001972.1: 267-32147">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001972/index.html#r1c1" target="_new">BGC0001972.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>toxoflavin, fervenulin</td>
        
        
        <td>Streptomyces hiroshimensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001973.1: 267-32147">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001973/index.html#r1c1" target="_new">BGC0001973.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>fervenulin</td>
        
        
        <td>Streptomyces hiroshimensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001794.1: 3660-23178">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001794/index.html#r1c1" target="_new">BGC0001794.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>swainsonine</td>
        
        
        <td>Alternaria oxytropis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000393.1: 4498819-4546737">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000393/index.html#r1c1" target="_new">BGC0000393.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>NRP, Polyketide</td>
        
        <td>myxoprincomide-c506</td>
        
        
        <td>Myxococcus xanthus DK 1622</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596.1: 5668766-5690544">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>RiPP</td>
        
        <td>SLI-2138</td>
        
        
        <td>Streptomyces lividans TK24</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595.1: 2616325-2638096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>RiPP</td>
        
        <td>SCO-2138</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000805.1: 897-5623">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000805/index.html#r1c1" target="_new">BGC0000805.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>Saccharide</td>
        
        <td>moenomycin</td>
        
        
        <td>Streptomyces viridosporus ATCC 14672</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597.1: 1489806-1500652">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>RiPP</td>
        
        <td>SWA-2138</td>
        
        
        <td>Streptomyces sp. e14</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001627.1: 106733-117539">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001627/index.html#r1c1" target="_new">BGC0001627.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
         
        
        <td>0.12</td>
        <td>NRP</td>
        
        <td>isonitrile lipopeptides</td>
        
        
        <td>Mycobacterium tuberculosis H37Rv</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r26c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001972.1: 267-32147">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001972/index.html#r1c1" target="_new">BGC0001972.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>toxoflavin, fervenulin</td>
      <td>Streptomyces hiroshimensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001973.1: 267-32147">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001973/index.html#r1c1" target="_new">BGC0001973.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>fervenulin</td>
      <td>Streptomyces hiroshimensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001794.1: 3660-23178">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001794/index.html#r1c1" target="_new">BGC0001794.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Polyketide</td>
      <td>swainsonine</td>
      <td>Alternaria oxytropis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000393.1: 4498819-4546737">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000393/index.html#r1c1" target="_new">BGC0000393.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>NRP, Polyketide</td>
      <td>myxoprincomide-c506</td>
      <td>Myxococcus xanthus DK 1622</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596.1: 5668766-5690544">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>RiPP</td>
      <td>SLI-2138</td>
      <td>Streptomyces lividans TK24</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595.1: 2616325-2638096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>RiPP</td>
      <td>SCO-2138</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000805.1: 897-5623">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000805/index.html#r1c1" target="_new">BGC0000805.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>Saccharide</td>
      <td>moenomycin</td>
      <td>Streptomyces viridosporus ATCC 14672</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597.1: 1489806-1500652">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>RiPP</td>
      <td>SWA-2138</td>
      <td>Streptomyces sp. e14</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001627.1: 106733-117539">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001627/index.html#r1c1" target="_new">BGC0001627.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
      <td>0.39</td>
      <td>NRP</td>
      <td>isonitrile lipopeptides</td>
      <td>Mycobacterium tuberculosis H37Rv</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r26c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-131"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-131">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r26c1-pfam-details-svg" data-filename="CC227C_contig_14_r26c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r26c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r26c1-nrps_pks_products" data-name="r26c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r26c1-nrps_pks" data-name="r26c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r26c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-125"></div> <span class="help-tooltip" id="nrps-structures-help-125">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 27896:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r26c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r26c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r26c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-126"></div> <span class="help-tooltip" id="nrps-monomers-help-126">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds4428_7896</span></strong>: X
      <div class="collapser collapser-target-cds4428_7896 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (76..503)</strong>: X
          <div class="collapser collapser-target-cds4428_7896 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds4428_7896-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">vamvVGgGGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>tcl</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r26c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_14 - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-269"></div> <span class="help-tooltip" id="region_svg-help-269">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_14.region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r26c2-svg" data-filename="CC227C_contig_14_r26c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 114,144 - 135,208 nt. (total: 21,065 nt)
        
        <a class="cluster-rules-header" id="r26c2-rules-header" href="#r26c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r26c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r26c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-270"></div> <span class="help-tooltip" id="focus-panel-help-270">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r26c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r26c2-clusterblast" data-name="r26c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r26c2-knownclusterblast" data-name="r26c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r26c2-subclusterblast" data-name="r26c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r26c2-MIBiG-cluster-compare" data-name="r26c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r26c2-pfam-details" data-name="r26c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r26c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-132"></div> <span class="help-tooltip" id="cb-general-help-132">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r26c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r26c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r26c2_1.svg">
        NZ_CP041695 (4490686-4509720): Nocardia otitidiscaviarum strain NEB252 chromo... (66% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r26c2_2.svg">
        NZ_CP059399 (1154071-1235666): Nocardia sp. WCH-YHL-001 chromosome, complete ... (13% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r26c2_3.svg">
        NZ_CP046172 (6997766-7143233): Nocardia arthritidis strain AUSMDU00012717 chr... (6% of genes show similarity), NRPS,T1PKS,T3PKS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r26c2_4.svg">
        NZ_CP027541 (2790335-2866056): Mycolicibacterium smegmatis MKD8 chromosome, c... (13% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r26c2_5.svg">
        NZ_CP059399 (6292362-6372674): Nocardia sp. WCH-YHL-001 chromosome, complete ... (8% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r26c2_6.svg">
        NZ_QQBC01000004 (682703-704911): Nocardia pseudobrasiliensis strain DSM 44290... (17% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r26c2_7.svg">
        NZ_CP007155 (8097644-8146334): Kutzneria albida DSM 43870 chromosome, complet... (12% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r26c2_8.svg">
        NZ_LN850107 (3679951-3708317): Alloactinosynnema sp. L-07 isolate Alloactinos... (17% of genes show similarity), LAP,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r26c2_9.svg">
        NC_014814 (3436057-3476538): Mycolicibacterium gilvum Spyr1, complete sequence (10% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r26c2_10.svg">
        NC_009338 (4226373-4266854): Mycolicibacterium gilvum PYR-GCK, complete sequence (10% of genes show similarity), T3PKS
      </option>
      
    </select>
    <button id= "clusterblast-r26c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r26c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r26c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-133"></div> <span class="help-tooltip" id="cb-known-help-133">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r26c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-134"></div> <span class="help-tooltip" id="cb-sub-help-134">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r26c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-135"></div> <span class="help-tooltip" id="MIBiG-help-135">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r26c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r26c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r26c2"></div>
    <div id="comparison-MIBiG-r26c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Terpene</td>
        
        <td>albaflavenone</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>Terpene</td>
        
        <td>raimonol</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene</td>
        
        <td>(+)-T-muurolol</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>Terpene</td>
        
        <td>pentalenolactone</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002013.1: 235087-246792">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002013/index.html#r1c1" target="_new">BGC0002013.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>RiPP</td>
        
        <td>curacozole</td>
        
        
        <td>Streptomyces curacoi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677.1: 120-4940">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Terpene</td>
        
        <td>cyclooctatin</td>
        
        
        <td>Streptomyces melanosporofaciens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>viguiepinol</td>
        
        
        <td>Streptomyces sp. KO-3988</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000686.1: 3892065-3908342">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000686/index.html#r1c1" target="_new">BGC0000686.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Terpene</td>
        
        <td>helvolic acid</td>
        
        
        <td>Aspergillus fumigatus Af293</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317.1: 0-277585">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Terpene</td>
        
        <td>lupeol</td>
        
        
        <td>Lotus japonicus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r26c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>albaflavenone</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene</td>
      <td>raimonol</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>(+)-T-muurolol</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002013.1: 235087-246792">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002013/index.html#r1c1" target="_new">BGC0002013.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>RiPP</td>
      <td>curacozole</td>
      <td>Streptomyces curacoi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677.1: 120-4940">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Terpene</td>
      <td>cyclooctatin</td>
      <td>Streptomyces melanosporofaciens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Polyketide</td>
      <td>viguiepinol</td>
      <td>Streptomyces sp. KO-3988</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000686.1: 3892065-3908342">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000686/index.html#r1c1" target="_new">BGC0000686.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Terpene</td>
      <td>helvolic acid</td>
      <td>Aspergillus fumigatus Af293</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317.1: 0-277585">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Terpene</td>
      <td>lupeol</td>
      <td>Lotus japonicus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r26c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-136"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-136">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r26c2-pfam-details-svg" data-filename="CC227C_contig_14_r26c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r26c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r33c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_6 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-271"></div> <span class="help-tooltip" id="region_svg-help-271">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_6.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r33c1-svg" data-filename="CC227C_contig_6_r33c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 330,481 - 384,106 nt. (total: 53,626 nt)
        
        <a class="cluster-rules-header" id="r33c1-rules-header" href="#r33c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r33c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r33c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-272"></div> <span class="help-tooltip" id="focus-panel-help-272">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r33c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r33c1-nrps_pks" data-name="r33c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r33c1-clusterblast" data-name="r33c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r33c1-knownclusterblast" data-name="r33c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r33c1-subclusterblast" data-name="r33c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r33c1-MIBiG-cluster-compare" data-name="r33c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r33c1-pfam-details" data-name="r33c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r33c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-137"></div> <span class="help-tooltip" id="nrps-domain-help-137">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r33c1-details-svg" data-filename="CC227C_contig_6_r33c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r33c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r33c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-140"></div> <span class="help-tooltip" id="cb-general-help-140">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r33c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r33c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r33c1_1.svg">
        NZ_CP041695 (2863252-2915077): Nocardia otitidiscaviarum strain NEB252 chromo... (90% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r33c1_2.svg">
        NZ_CP032568 (4192977-4246366): Nocardia sp. CFHS0054 chromosome (84% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r33c1_3.svg">
        NC_016887 (3616734-3669828): Nocardia cyriacigeorgica GUH-2, complete genome (80% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r33c1_4.svg">
        NZ_CP026746 (4258711-4311645): Nocardia cyriacigeorgica strain MDA3349 isolat... (80% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r33c1_5.svg">
        NZ_LR215973 (2090480-2141961): Nocardia cyriacigeorgica strain 3012STDY675650... (79% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r33c1_6.svg">
        NZ_CP046172 (5825229-5895006): Nocardia arthritidis strain AUSMDU00012717 chr... (54% of genes show similarity), NRPS,TfuA-related
      </option>
      
      <option value = "svg/clusterblast_r33c1_7.svg">
        NZ_CP059399 (4647677-4734064): Nocardia sp. WCH-YHL-001 chromosome, complete ... (46% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r33c1_8.svg">
        NC_018681 (6019488-6071567): Nocardia brasiliensis ATCC 700358, complete sequ... (69% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r33c1_9.svg">
        NZ_CP046171 (4721548-4773733): Nocardia brasiliensis strain AUSMDU00024985 ch... (61% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r33c1_10.svg">
        NZ_CP022088 (1004952-1057671): Nocardia brasiliensis strain FDAARGOS 352 chro... (65% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r33c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r33c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r33c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-141"></div> <span class="help-tooltip" id="cb-known-help-141">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r33c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r33c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r33c1_1.svg">
        calicheamicin
      </option>
      
      <option value = "svg/knownclusterblast_r33c1_2.svg">
        pepticinnamin E
      </option>
      
      <option value = "svg/knownclusterblast_r33c1_3.svg">
        A54145
      </option>
      
      <option value = "svg/knownclusterblast_r33c1_4.svg">
        arixanthomycin A / arixanthomycin B / arixanthomycin C
      </option>
      
      <option value = "svg/knownclusterblast_r33c1_5.svg">
        ibomycin
      </option>
      
      <option value = "svg/knownclusterblast_r33c1_6.svg">
        primycin
      </option>
      
      <option value = "svg/knownclusterblast_r33c1_7.svg">
        cadaside A / cadaside B
      </option>
      
    </select>
    <button id="knownclusterblast-r33c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r33c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r33c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-142"></div> <span class="help-tooltip" id="cb-sub-help-142">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r33c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-143"></div> <span class="help-tooltip" id="MIBiG-help-143">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r33c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r33c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r33c1"></div>
    <div id="comparison-MIBiG-r33c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>rhodochelin</td>
        
        
        <td>Rhodococcus jostii RHA1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131.1: 144016-173487">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>ambactin</td>
        
        
        <td>Xenorhabdus miraniensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001149.1: 196767-209553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001149/index.html#r1c1" target="_new">BGC0001149.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>NRP, Saccharide</td>
        
        <td>brabantamide A</td>
        
        
        <td>Pseudomonas sp. SHC52</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r33c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>rhodochelin</td>
      <td>Rhodococcus jostii RHA1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131.1: 144016-173487">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>ambactin</td>
      <td>Xenorhabdus miraniensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001149.1: 196767-209553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001149/index.html#r1c1" target="_new">BGC0001149.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP, Saccharide</td>
      <td>brabantamide A</td>
      <td>Pseudomonas sp. SHC52</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r33c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-144"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-144">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r33c1-pfam-details-svg" data-filename="CC227C_contig_6_r33c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r33c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r33c1-nrps_pks_products" data-name="r33c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r33c1-nrps_pks" data-name="r33c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r33c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-138"></div> <span class="help-tooltip" id="nrps-structures-help-138">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 330480 - 384106:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r33c1-1" class="smiles-canvas" data-smiles="NC(CO)C(=O)NC([*])C(=O)NC([*])C(=O)NC(CC(C)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ser - X - X - leu)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,x,x,leu" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,x,x,leu*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r33c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CO)C(=O)NC([*])C(=O)NC([*])C(=O)NC(CC(C)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r33c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-139"></div> <span class="help-tooltip" id="nrps-monomers-help-139">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds350480_364106</span></strong>: ser - X - X - leu
      <div class="collapser collapser-target-cds350480_364106 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,X,X,leu" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,X*,X*,leu*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (478..882)</strong>: ser
          <div class="collapser collapser-target-cds350480_364106 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds350480_364106-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWHFSLVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1549..1961)</strong>: X
          <div class="collapser collapser-target-cds350480_364106 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds350480_364106-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DALFiGmVaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2614..3017)</strong>: X
          <div class="collapser collapser-target-cds350480_364106 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds350480_364106-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAFfMGgVvK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3682..4083)</strong>: leu
          <div class="collapser collapser-target-cds350480_364106 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  leu
              <div class="collapser collapser-target-cds350480_364106-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>leu</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAlFLGviVK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r37c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_2 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-273"></div> <span class="help-tooltip" id="region_svg-help-273">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_2.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r37c1-svg" data-filename="CC227C_contig_2_r37c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 213 - 45,810 nt. (total: 45,598 nt)
        
        <a class="cluster-rules-header" id="r37c1-rules-header" href="#r37c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r37c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r37c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-274"></div> <span class="help-tooltip" id="focus-panel-help-274">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r37c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r37c1-nrps_pks" data-name="r37c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r37c1-clusterblast" data-name="r37c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r37c1-knownclusterblast" data-name="r37c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r37c1-subclusterblast" data-name="r37c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r37c1-MIBiG-cluster-compare" data-name="r37c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r37c1-pfam-details" data-name="r37c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r37c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-145"></div> <span class="help-tooltip" id="nrps-domain-help-145">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r37c1-details-svg" data-filename="CC227C_contig_2_r37c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r37c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r37c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-148"></div> <span class="help-tooltip" id="cb-general-help-148">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r37c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r37c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r37c1_1.svg">
        NZ_CP041695 (1436688-1490065): Nocardia otitidiscaviarum strain NEB252 chromo... (80% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c1_2.svg">
        NZ_CP059399 (3200704-3253761): Nocardia sp. WCH-YHL-001 chromosome, complete ... (68% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c1_3.svg">
        NZ_CP017839 (3063797-3118270): Nocardia seriolae strain EM150506 chromosome, ... (71% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c1_4.svg">
        NZ_CP032568 (5010563-5062987): Nocardia sp. CFHS0054 chromosome (56% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c1_5.svg">
        NZ_QQAZ01000002 (279921-330574): Nocardia mexicana strain DSM 44952 Ga0244508... (42% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c1_6.svg">
        NZ_RFFH01000004 (518916-591446): Nocardia stercoris strain NEAU-LL90 Scaffold... (36% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c1_7.svg">
        NZ_LR026975 (1379812-1520278): Mycolicibacterium hassiacum DSM 44199 isolate ... (21% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r37c1_8.svg">
        NZ_CP046572 (2587564-2636808): Rhodococcus sp. WAY2 chromosome, complete genome (42% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r37c1_9.svg">
        NZ_AP022565 (55047-114896): Mycolicibacterium alvei strain JCM 12272 (33% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c1_10.svg">
        NZ_CP043474 (4780432-4845783): Mycobacterium grossiae strain DSM 104744 chrom... (31% of genes show similarity), NRPS,T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r37c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r37c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r37c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-149"></div> <span class="help-tooltip" id="cb-known-help-149">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r37c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r37c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r37c1_1.svg">
        zorbamycin
      </option>
      
      <option value = "svg/knownclusterblast_r37c1_2.svg">
        hygrocin A / hygrocin B
      </option>
      
    </select>
    <button id="knownclusterblast-r37c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r37c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r37c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-150"></div> <span class="help-tooltip" id="cb-sub-help-150">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r37c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-151"></div> <span class="help-tooltip" id="MIBiG-help-151">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r37c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r37c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r37c1"></div>
    <div id="comparison-MIBiG-r37c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>AM-toxin</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001686.1: 28257-44195">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001686/index.html#r1c1" target="_new">BGC0001686.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>N-octanoyl-Met-Phe-H</td>
        
        
        <td>Clostridium sp. CAG:567</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313.1: 0-33475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>beauvericin</td>
        
        
        <td>Beauveria bassiana</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342.1: 1020-10416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>enniatin</td>
        
        
        <td>Fusarium equiseti</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001824.1: 39732-57269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001824/index.html#r1c1" target="_new">BGC0001824.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>xefoampeptides A-G</td>
        
        
        <td>Xenorhabdus beddingii</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r37c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>AM-toxin</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001686.1: 28257-44195">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001686/index.html#r1c1" target="_new">BGC0001686.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>N-octanoyl-Met-Phe-H</td>
      <td>Clostridium sp. CAG:567</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313.1: 0-33475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>beauvericin</td>
      <td>Beauveria bassiana</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342.1: 1020-10416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>enniatin</td>
      <td>Fusarium equiseti</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001824.1: 39732-57269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001824/index.html#r1c1" target="_new">BGC0001824.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>xefoampeptides A-G</td>
      <td>Xenorhabdus beddingii</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r37c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-152"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-152">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r37c1-pfam-details-svg" data-filename="CC227C_contig_2_r37c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r37c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r37c1-nrps_pks_products" data-name="r37c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r37c1-nrps_pks" data-name="r37c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r37c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-146"></div> <span class="help-tooltip" id="nrps-structures-help-146">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 212 - 45810:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r37c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-X - X - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x,x,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x*,x,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r37c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r37c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-147"></div> <span class="help-tooltip" id="nrps-monomers-help-147">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds12921_26478</span></strong>: X - X - X
      <div class="collapser collapser-target-cds12921_26478 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,X,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,X*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (475..865)</strong>: X
          <div class="collapser collapser-target-cds12921_26478 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds12921_26478-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DALcsGsVaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2061..2440)</strong>: X
          <div class="collapser collapser-target-cds12921_26478 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds12921_26478-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DALiVGsVaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>tyr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3136..3525)</strong>: X
          <div class="collapser collapser-target-cds12921_26478 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds12921_26478-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAlAIgdVwK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r37c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_2 - Region 2 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-275"></div> <span class="help-tooltip" id="region_svg-help-275">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_2.region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r37c2-svg" data-filename="CC227C_contig_2_r37c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 191,810 - 239,251 nt. (total: 47,442 nt)
        
        <a class="cluster-rules-header" id="r37c2-rules-header" href="#r37c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r37c2-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r37c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-276"></div> <span class="help-tooltip" id="focus-panel-help-276">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r37c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r37c2-nrps_pks" data-name="r37c2-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r37c2-clusterblast" data-name="r37c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r37c2-knownclusterblast" data-name="r37c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r37c2-subclusterblast" data-name="r37c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r37c2-MIBiG-cluster-compare" data-name="r37c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r37c2-pfam-details" data-name="r37c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r37c2-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-153"></div> <span class="help-tooltip" id="nrps-domain-help-153">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r37c2-details-svg" data-filename="CC227C_contig_2_r37c2_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r37c2-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r37c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-156"></div> <span class="help-tooltip" id="cb-general-help-156">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r37c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r37c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r37c2_1.svg">
        NZ_CP041695 (1290260-1341904): Nocardia otitidiscaviarum strain NEB252 chromo... (64% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c2_2.svg">
        NZ_SNXK01000002 (759010-812684): Nocardia ignorata strain DSM 44496 Ga0244500... (31% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c2_3.svg">
        NZ_CP059399 (3103032-3154746): Nocardia sp. WCH-YHL-001 chromosome, complete ... (27% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c2_4.svg">
        NZ_QGTL01000003 (27202-103789): Nocardia neocaledoniensis strain DSM 44717 Ga... (27% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c2_5.svg">
        NZ_VMSD01000002 (751255-804515): Nocardia caishijiensis strain DSM 44831 Ga02... (31% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c2_6.svg">
        NZ_LMFE01000007 (49955-101235): Nocardia sp. Root136 contig 2, whole genome s... (27% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c2_7.svg">
        NZ_QQBC01000012 (69683-164300): Nocardia pseudobrasiliensis strain DSM 44290 ... (21% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c2_8.svg">
        NZ_CP026746 (3397399-3517271): Nocardia cyriacigeorgica strain MDA3349 isolat... (15% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c2_9.svg">
        NZ_LR215973 (2920716-3040911): Nocardia cyriacigeorgica strain 3012STDY675650... (15% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c2_10.svg">
        NZ_LT985361 (7588440-7642319): Nocardia suismassiliense strain S-137, whole g... (25% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r37c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r37c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r37c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-157"></div> <span class="help-tooltip" id="cb-known-help-157">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r37c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r37c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r37c2_1.svg">
        heterobactin A / heterobactin S2
      </option>
      
      <option value = "svg/knownclusterblast_r37c2_2.svg">
        mirubactin
      </option>
      
      <option value = "svg/knownclusterblast_r37c2_3.svg">
        myxochelin A / myxochelin B
      </option>
      
      <option value = "svg/knownclusterblast_r37c2_4.svg">
        salinichelins
      </option>
      
      <option value = "svg/knownclusterblast_r37c2_5.svg">
        streptobactin
      </option>
      
      <option value = "svg/knownclusterblast_r37c2_6.svg">
        amychelin
      </option>
      
      <option value = "svg/knownclusterblast_r37c2_7.svg">
        bacillibactin
      </option>
      
      <option value = "svg/knownclusterblast_r37c2_8.svg">
        amphi-enterobactin 1 / amphi-enterobactin 2 / amphi-enterobactin 3 / amphi-enterobactin 4
      </option>
      
      <option value = "svg/knownclusterblast_r37c2_9.svg">
        amonabactin P 750
      </option>
      
      <option value = "svg/knownclusterblast_r37c2_10.svg">
        gobichelin A / gobichelin B
      </option>
      
    </select>
    <button id="knownclusterblast-r37c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r37c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r37c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-158"></div> <span class="help-tooltip" id="cb-sub-help-158">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r37c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-159"></div> <span class="help-tooltip" id="MIBiG-help-159">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r37c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r37c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r37c2"></div>
    <div id="comparison-MIBiG-r37c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>pseudomonine</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>fellutamide B</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001302.1: 0-25686">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001302/index.html#r1c1" target="_new">BGC0001302.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.50, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Other</td>
        
        <td>lomofungin</td>
        
        
        <td>Streptomyces lomondensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.48, order:0.52, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>bacillibactin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000935.1: 151-29847">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000935/index.html#r1c1" target="_new">BGC0000935.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.50, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Other, Polyketide,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
        
        <td>esmeraldin</td>
        
        
        <td>Streptomyces antibioticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001149.1: 196767-209553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001149/index.html#r1c1" target="_new">BGC0001149.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP, Saccharide</td>
        
        <td>brabantamide A</td>
        
        
        <td>Pseudomonas sp. SHC52</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349.1: 3324847-3348992">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>erythrochelin</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>AM-toxin</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000220.1: 512-21855">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000220/index.html#r1c1" target="_new">BGC0000220.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>enterocin</td>
        
        
        <td>Streptomyces maritimus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r37c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>pseudomonine</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>fellutamide B</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001302.1: 0-25686">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001302/index.html#r1c1" target="_new">BGC0001302.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.50, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>lomofungin</td>
      <td>Streptomyces lomondensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.48, order:0.52, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000935.1: 151-29847">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000935/index.html#r1c1" target="_new">BGC0000935.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.50, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Other, Polyketide,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
      <td>esmeraldin</td>
      <td>Streptomyces antibioticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001149.1: 196767-209553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001149/index.html#r1c1" target="_new">BGC0001149.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP, Saccharide</td>
      <td>brabantamide A</td>
      <td>Pseudomonas sp. SHC52</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349.1: 3324847-3348992">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>erythrochelin</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>AM-toxin</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000220.1: 512-21855">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000220/index.html#r1c1" target="_new">BGC0000220.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>enterocin</td>
      <td>Streptomyces maritimus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r37c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-160"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-160">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r37c2-pfam-details-svg" data-filename="CC227C_contig_2_r37c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r37c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r37c2-nrps_pks_products" data-name="r37c2-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r37c2-nrps_pks" data-name="r37c2-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r37c2-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-154"></div> <span class="help-tooltip" id="nrps-structures-help-154">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 191809 - 239251:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r37c2-2" class="smiles-canvas" data-smiles="NC(CC(C)C)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(leu - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r37c2-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CC(C)C)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r37c2-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-155"></div> <span class="help-tooltip" id="nrps-monomers-help-155">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds206445_208101</span></strong>: dhb
      <div class="collapser collapser-target-cds206445_208101 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=diOH-Bz" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=diOH-Bz*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (36..447)</strong>: dhb
          <div class="collapser collapser-target-cds206445_208101 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  dhb
              <div class="collapser collapser-target-cds206445_208101-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>dhb, sal</dd>
   <dt>Small clusters prediction:</dt>
   <dd>dhb, sal</dd>
   <dt>Single AA prediction:</dt>
   <dd>dhb</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">PMPgQGVVcK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>dhb</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds211519_219325</span></strong>: leu - X
      <div class="collapser collapser-target-cds211519_219325 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (481..989)</strong>: leu
          <div class="collapser collapser-target-cds211519_219325 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  leu
              <div class="collapser collapser-target-cds211519_219325-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DALdaGLVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1633..2010)</strong>: X
          <div class="collapser collapser-target-cds211519_219325 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  phe, trp, phg, tyr, bht
              <div class="collapser collapser-target-cds211519_219325-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAlqIGlVCK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>lys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r37c3' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_2 - Region 3 - NRPS,NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-277"></div> <span class="help-tooltip" id="region_svg-help-277">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_2.region003.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r37c3-svg" data-filename="CC227C_contig_2_r37c3.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 307,896 - 439,197 nt. (total: 131,302 nt)
        
        <a class="cluster-rules-header" id="r37c3-rules-header" href="#r37c3">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r37c3-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r37c3-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-278"></div> <span class="help-tooltip" id="focus-panel-help-278">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r37c3">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r37c3-nrps_pks" data-name="r37c3-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r37c3-clusterblast" data-name="r37c3-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r37c3-knownclusterblast" data-name="r37c3-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r37c3-subclusterblast" data-name="r37c3-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r37c3-MIBiG-cluster-compare" data-name="r37c3-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r37c3-pfam-details" data-name="r37c3-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r37c3-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-161"></div> <span class="help-tooltip" id="nrps-domain-help-161">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r37c3-details-svg" data-filename="CC227C_contig_2_r37c3_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r37c3-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r37c3-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-164"></div> <span class="help-tooltip" id="cb-general-help-164">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r37c3-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r37c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r37c3_1.svg">
        NZ_CP041695 (1073700-1220450): Nocardia otitidiscaviarum strain NEB252 chromo... (66% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c3_2.svg">
        NZ_QQAZ01000003 (765480-830667): Nocardia mexicana strain DSM 44952 Ga0244508... (25% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c3_3.svg">
        NZ_CP059399 (4977517-5050230): Nocardia sp. WCH-YHL-001 chromosome, complete ... (18% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r37c3_4.svg">
        NZ_LAND01000003 (75654-141826): Streptomyces sp. WMMB 714 H181DRAFT scaffold0... (30% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c3_5.svg">
        NZ_FOSG01000001 (301588-365419): Streptomyces pini strain PL19, whole genome ... (22% of genes show similarity), NRPS,linaridin
      </option>
      
      <option value = "svg/clusterblast_r37c3_6.svg">
        NZ_CP046572 (1545061-1612190): Rhodococcus sp. WAY2 chromosome, complete genome (22% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c3_7.svg">
        NZ_CP051855 (6208832-6274881): Rhodococcus imtechensis strain XM24D chromosom... (20% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c3_8.svg">
        NC_008268 (2297420-2364003): Rhodococcus jostii RHA1, complete sequence (21% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c3_9.svg">
        NC_012522 (2016164-2082620): Rhodococcus opacus B4, complete genome (21% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c3_10.svg">
        NZ_CP009111 (6152659-6218679): Rhodococcus opacus strain 1CP, complete genome (20% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r37c3-download">Download graphic</button>
  </div>
  <div id="clusterblast-r37c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r37c3-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-165"></div> <span class="help-tooltip" id="cb-known-help-165">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r37c3-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r37c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r37c3_1.svg">
        butyrolactol A
      </option>
      
      <option value = "svg/knownclusterblast_r37c3_2.svg">
        kedarcidin
      </option>
      
      <option value = "svg/knownclusterblast_r37c3_3.svg">
        chondrochloren A
      </option>
      
      <option value = "svg/knownclusterblast_r37c3_4.svg">
        bengamide
      </option>
      
      <option value = "svg/knownclusterblast_r37c3_5.svg">
        miharamycin A / miharamycin B
      </option>
      
      <option value = "svg/knownclusterblast_r37c3_6.svg">
        rustmicin
      </option>
      
      <option value = "svg/knownclusterblast_r37c3_7.svg">
        concanamycin A
      </option>
      
      <option value = "svg/knownclusterblast_r37c3_8.svg">
        macbecin / macbecin II
      </option>
      
      <option value = "svg/knownclusterblast_r37c3_9.svg">
        oxalomycin B
      </option>
      
      <option value = "svg/knownclusterblast_r37c3_10.svg">
        totopotensamide A / totopotensamide B
      </option>
      
    </select>
    <button id="knownclusterblast-r37c3-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r37c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r37c3-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-166"></div> <span class="help-tooltip" id="cb-sub-help-166">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r37c3-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r37c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r37c3_1.svg">
        DQ149987_c1: concanamycin methoxymalonate (40% of genes show similarity), methoxymalonate
      </option>
      
    </select>
    <button id="subclusterblast-r37c3-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r37c3-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r37c3-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-167"></div> <span class="help-tooltip" id="MIBiG-help-167">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r37c3-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r37c3-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r37c3"></div>
    <div id="comparison-MIBiG-r37c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.44, order:0.75, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.44, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.67</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.42, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.64</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.63</td>
        <td>Other</td>
        
        <td>ferrichrome</td>
        
        
        <td>Aspergillus oryzae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.63</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.62</td>
        <td>NRP</td>
        
        <td>ergovaline</td>
        
        
        <td>Epichloe festucae var. lolii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.44, order:0.75, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.43, order:0.75, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.60</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:0.69, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.58</td>
        <td>NRP</td>
        
        <td>exochelin</td>
        
        
        <td>Mycolicibacterium smegmatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000392.1: 3113323-3141040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000392/index.html#r1c1" target="_new">BGC0000392.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.58</td>
        <td>NRP</td>
        
        <td>mirubactin</td>
        
        
        <td>Actinosynnema mirum DSM 43827</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.57</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.57</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r37c3-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.42, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001629.1: 209-63961">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001629/index.html#r1c1" target="_new">BGC0001629.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.44, order:0.83, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP</td>
      <td>Jessenipeptin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.44, order:0.75, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.44, order:0.75, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131.1: 144016-173487">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>ambactin</td>
      <td>Xenorhabdus miraniensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r37c3-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-168"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-168">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r37c3-pfam-details-svg" data-filename="CC227C_contig_2_r37c3_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r37c3-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r37c3-nrps_pks_products" data-name="r37c3-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r37c3-nrps_pks" data-name="r37c3-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r37c3-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-162"></div> <span class="help-tooltip" id="nrps-structures-help-162">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 307895 - 439197:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r37c3-3" class="smiles-canvas" data-smiles="NC([*])C(=O)CC(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC(CC(C)C)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC(CC(C)C)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X) + (mal) + (D-X - D-thr) + (thr - X) + (thr - X - X - X - X - X - leu) + (thr - D-X - X - D-X - D-X - leu - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=mal&amp;nrps3=d-x,d-thr&amp;nrps4=thr,x&amp;nrps5=thr,x,x,x,x,x,leu&amp;nrps6=thr,d-x,x,d-x,d-x,leu,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=mal*&amp;nrps3=d-x*,d-thr*&amp;nrps4=thr*,x&amp;nrps5=thr*,x,x,x,x,x,leu*&amp;nrps6=thr*,d-x*,x,d-x*,d-x*,leu*,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r37c3-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)CC(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC(CC(C)C)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC(CC(C)C)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 4, location 307895 - 431458:
          <div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r37c3-4" class="smiles-canvas" data-smiles="NC([*])C(=O)CC(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC(CC(C)C)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC(CC(C)C)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X) + (mal) + (D-X - D-thr) + (thr - X) + (thr - X - X - X - X - X - leu) + (thr - D-X - X - D-X - D-X - leu - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=mal&amp;nrps3=d-x,d-thr&amp;nrps4=thr,x&amp;nrps5=thr,x,x,x,x,x,leu&amp;nrps6=thr,d-x,x,d-x,d-x,leu,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=mal*&amp;nrps3=d-x*,d-thr*&amp;nrps4=thr*,x&amp;nrps5=thr*,x,x,x,x,x,leu*&amp;nrps6=thr*,d-x*,x,d-x*,d-x*,leu*,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r37c3-4-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)CC(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC(CC(C)C)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC(CC(C)C)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 5, location 397328 - 439197:
          <div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r37c3-5" class="smiles-canvas" data-smiles="NC([*])C(=O)CC(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X) + (mal) + (D-X - D-thr) + (thr - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=mal&amp;nrps3=d-x,d-thr&amp;nrps4=thr,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=mal*&amp;nrps3=d-x*,d-thr*&amp;nrps4=thr*,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r37c3-5-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)CC(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r37c3-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-163"></div> <span class="help-tooltip" id="nrps-monomers-help-163">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds309312_310971</span></strong>: X
      <div class="collapser collapser-target-cds309312_310971 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (57..454)</strong>: X
          <div class="collapser collapser-target-cds309312_310971 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds309312_310971-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">hgSTfAlViK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>bht</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds327895_356392</span></strong>: thr - X - X - X - X - leu - X
      <div class="collapser collapser-target-cds327895_356392 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr,X,X,X,X,leu,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*,X*,X*,X*,X*,leu*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (468..873)</strong>: thr
          <div class="collapser collapser-target-cds327895_356392 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds327895_356392-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNIGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1842..2246)</strong>: X
          <div class="collapser collapser-target-cds327895_356392 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds327895_356392-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DaLfmGLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3381..3781)</strong>: X
          <div class="collapser collapser-target-cds327895_356392 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala
              <div class="collapser collapser-target-cds327895_356392-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAFvsAlVwK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (4451..4859)</strong>: X
          <div class="collapser collapser-target-cds327895_356392 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  pro, pip
              <div class="collapser collapser-target-cds327895_356392-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>pro, pip</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAlfsGVViK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>glu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (6004..6409)</strong>: X
          <div class="collapser collapser-target-cds327895_356392 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds327895_356392-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DaLfLGLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (7545..7949)</strong>: leu
          <div class="collapser collapser-target-cds327895_356392 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  leu
              <div class="collapser collapser-target-cds327895_356392-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DALFmGLVwK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (8617..9036)</strong>: X
          <div class="collapser collapser-target-cds327895_356392 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds327895_356392-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DaLfLGLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds357675_359355</span></strong>: pk
      <div class="collapser collapser-target-cds357675_359355 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>CAL_domain (40..462)</strong>: pk
          <div class="collapser collapser-target-cds357675_359355 collapser-level-candidate"> </div><div class="collapser-content">
          
              minowa:  fatty_acid
              <div class="collapser collapser-target-cds357675_359355-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>fatty_acid: 18.7</dt>

  <dd></dd><dt>NH2: 1.5</dt>

  <dd></dd><dt>shikimic_acid: 0.0</dt>

  <dd></dd><dt>AHBA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: -3.9</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds365083_388153</span></strong>: thr - X - X - X - X - X - leu
      <div class="collapser collapser-target-cds365083_388153 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr,X,X,X,X,X,leu" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*,X*,X*,X*,X*,X*,leu*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (480..886)</strong>: thr
          <div class="collapser collapser-target-cds365083_388153 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds365083_388153-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNIGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1548..1956)</strong>: X
          <div class="collapser collapser-target-cds365083_388153 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds365083_388153-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DlWHVaLVsK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2615..3012)</strong>: X
          <div class="collapser collapser-target-cds365083_388153 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds365083_388153-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAlfIGAIaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3669..4061)</strong>: X
          <div class="collapser collapser-target-cds365083_388153 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds365083_388153-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DALfvGAVaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (4722..5113)</strong>: X
          <div class="collapser collapser-target-cds365083_388153 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds365083_388153-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DALfvGgiMK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (5782..6167)</strong>: X
          <div class="collapser collapser-target-cds365083_388153 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds365083_388153-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVlvVGgIaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>arg</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (6826..7229)</strong>: leu
          <div class="collapser collapser-target-cds365083_388153 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  leu
              <div class="collapser collapser-target-cds365083_388153-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>leu</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DALfMGAVaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds393402_400935</span></strong>: thr - X
      <div class="collapser collapser-target-cds393402_400935 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (452..851)</strong>: thr
          <div class="collapser collapser-target-cds393402_400935 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds393402_400935-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNIGMVfK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1505..1882)</strong>: X
          <div class="collapser collapser-target-cds393402_400935 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala, val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds393402_400935-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DmwmaAlsYK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>40% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds400931_411458</span></strong>: X - thr
      <div class="collapser collapser-target-cds400931_411458 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,thr" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,thr*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (428..827)</strong>: X
          <div class="collapser collapser-target-cds400931_411458 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds400931_411458-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWqLaLIaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1967..2403)</strong>: thr
          <div class="collapser collapser-target-cds400931_411458 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds400931_411458-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWSVGlVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds417328_419197</span></strong>: X
      <div class="collapser collapser-target-cds417328_419197 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (19..421)</strong>: X
          <div class="collapser collapser-target-cds417328_419197 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala, val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds417328_419197-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DGmvTAEVtK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>tyr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r37c4' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_2 - Region 4 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-279"></div> <span class="help-tooltip" id="region_svg-help-279">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_2.region004.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r37c4-svg" data-filename="CC227C_contig_2_r37c4.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 518,305 - 539,420 nt. (total: 21,116 nt)
        
        <a class="cluster-rules-header" id="r37c4-rules-header" href="#r37c4">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r37c4-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r37c4-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-280"></div> <span class="help-tooltip" id="focus-panel-help-280">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r37c4">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r37c4-clusterblast" data-name="r37c4-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r37c4-knownclusterblast" data-name="r37c4-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r37c4-subclusterblast" data-name="r37c4-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r37c4-MIBiG-cluster-compare" data-name="r37c4-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r37c4-pfam-details" data-name="r37c4-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r37c4-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-169"></div> <span class="help-tooltip" id="cb-general-help-169">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r37c4-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r37c4_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r37c4_1.svg">
        NZ_CP018082 (5953144-5974080): Nocardia mangyaensis strain Y48 chromosome, co... (53% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r37c4_2.svg">
        NZ_QLTT01000003 (910062-966479): Lechevalieria atacamensis strain DSM 45479 G... (7% of genes show similarity), T3PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r37c4_3.svg">
        NZ_FOAZ01000004 (25636-103565): Streptacidiphilus jiangxiensis strain CGMCC 4... (4% of genes show similarity), T2PKS,butyrolactone,terpene
      </option>
      
      <option value = "svg/clusterblast_r37c4_4.svg">
        NZ_CP054938 (35742-117199): Streptomyces harbinensis strain NA02264 chromosom... (3% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c4_5.svg">
        NZ_FPAB01000008 (33416-118394): Streptomyces harbinensis strain CGMCC 4.7047,... (3% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c4_6.svg">
        NZ_CP023778 (5336818-5483506): Nocardia terpenica strain NC YFY NT001 chromos... (3% of genes show similarity), NRPS,NRPS-like,lassopeptide,terpene
      </option>
      
      <option value = "svg/clusterblast_r37c4_7.svg">
        NZ_CP046173 (5416618-5569969): Nocardia terpenica strain AUSMDU00012715 chrom... (2% of genes show similarity), NRPS,NRPS-like,terpene
      </option>
      
      <option value = "svg/clusterblast_r37c4_8.svg">
        NZ_CP041061 (3630537-3713356): Micromonospora sp. HM134 chromosome, complete ... (4% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r37c4_9.svg">
        NZ_CDME01000002 (697157-783840): Kibdelosporangium sp. MJ126-NF4, whole genom... (4% of genes show similarity), T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r37c4_10.svg">
        NC_016111 (5403542-5463532): Streptomyces cattleya NRRL 8057 = DSM 46488, com... (4% of genes show similarity), T3PKS,terpene
      </option>
      
    </select>
    <button id= "clusterblast-r37c4-download">Download graphic</button>
  </div>
  <div id="clusterblast-r37c4-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r37c4-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-170"></div> <span class="help-tooltip" id="cb-known-help-170">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r37c4-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-171"></div> <span class="help-tooltip" id="cb-sub-help-171">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r37c4-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-172"></div> <span class="help-tooltip" id="MIBiG-help-172">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r37c4-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r37c4-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r37c4"></div>
    <div id="comparison-MIBiG-r37c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969.1: 0-7810">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>Terpene</td>
        
        <td>asperterpenoid A</td>
        
        
        <td>Talaromyces wortmannii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.48, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Terpene</td>
        
        <td>pentalenolactone</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Terpene</td>
        
        <td>platencin</td>
        
        
        <td>Streptomyces platensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Myxococcus xanthus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007.1: 5922480-5942549">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>Terpene</td>
        
        <td>atolypene A, atolypene B</td>
        
        
        <td>Amycolatopsis tolypomycina</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595.1: 2616325-2638096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>RiPP</td>
        
        <td>SCO-2138</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001529.1: 219-12060">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001529/index.html#r1c1" target="_new">BGC0001529.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Alkaloid</td>
        
        <td>bezastatin derivates</td>
        
        
        <td>Streptomyces sp. RI18</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001914.1: 219-12060">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001914/index.html#r1c1" target="_new">BGC0001914.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>benzastatin J, 7-hydroxyl benzastatin B, 7-hydroxyl O-demethylbenzastatin A, JBIR-67, 7-hydroxyl benzastatin F, 7-hydroxyl O-demethylbenzastatin D, O-demethylvirantmycin, 7-hydroxyl benzastatin D, virantmycin, benzastatin K, methylbenzastatin K</td>
        
        
        <td>Streptomyces sp. RI18</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596.1: 5668766-5690544">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>RiPP</td>
        
        <td>SLI-2138</td>
        
        
        <td>Streptomyces lividans TK24</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000933.1: 0-5189">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000933/index.html#r1c1" target="_new">BGC0000933.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>xiamenmycin A</td>
        
        
        <td>Streptomyces xiamenensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r37c4-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969.1: 0-7810">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Terpene</td>
      <td>asperterpenoid A</td>
      <td>Talaromyces wortmannii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.48, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Terpene</td>
      <td>platencin</td>
      <td>Streptomyces platensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Myxococcus xanthus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007.1: 5922480-5942549">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Terpene</td>
      <td>atolypene A, atolypene B</td>
      <td>Amycolatopsis tolypomycina</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595.1: 2616325-2638096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>RiPP</td>
      <td>SCO-2138</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001529.1: 219-12060">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001529/index.html#r1c1" target="_new">BGC0001529.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Alkaloid</td>
      <td>bezastatin derivates</td>
      <td>Streptomyces sp. RI18</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001914.1: 219-12060">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001914/index.html#r1c1" target="_new">BGC0001914.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Other</td>
      <td>benzastatin J, 7-hydroxyl benzastatin B, 7-hydroxyl O-demethylbenzastatin A, JBIR-67, 7-hydroxyl benzastatin F, 7-hydroxyl O-demethylbenzastatin D, O-demethylvirantmycin, 7-hydroxyl benzastatin D, virantmycin, benzastatin K, methylbenzastatin K</td>
      <td>Streptomyces sp. RI18</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596.1: 5668766-5690544">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>RiPP</td>
      <td>SLI-2138</td>
      <td>Streptomyces lividans TK24</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000933.1: 0-5189">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000933/index.html#r1c1" target="_new">BGC0000933.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>xiamenmycin A</td>
      <td>Streptomyces xiamenensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r37c4-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-173"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-173">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r37c4-pfam-details-svg" data-filename="CC227C_contig_2_r37c4_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r37c4-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r39c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_3 - Region 1 - NRPS,terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-281"></div> <span class="help-tooltip" id="region_svg-help-281">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_3.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r39c1-svg" data-filename="CC227C_contig_3_r39c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 86,207 - 182,976 nt. (total: 96,770 nt)
        
        <a class="cluster-rules-header" id="r39c1-rules-header" href="#r39c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r39c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r39c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-282"></div> <span class="help-tooltip" id="focus-panel-help-282">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r39c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r39c1-nrps_pks" data-name="r39c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r39c1-clusterblast" data-name="r39c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r39c1-knownclusterblast" data-name="r39c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r39c1-subclusterblast" data-name="r39c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r39c1-MIBiG-cluster-compare" data-name="r39c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r39c1-pfam-details" data-name="r39c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r39c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-174"></div> <span class="help-tooltip" id="nrps-domain-help-174">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r39c1-details-svg" data-filename="CC227C_contig_3_r39c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r39c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r39c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-177"></div> <span class="help-tooltip" id="cb-general-help-177">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r39c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r39c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r39c1_1.svg">
        NZ_CP041695 (5984175-6106448): Nocardia otitidiscaviarum strain NEB252 chromo... (68% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r39c1_2.svg">
        NZ_KV411303 (3216646-3390056): Nocardia terpenica strain IFM 0406 scaffold1, ... (36% of genes show similarity), NRPS,PKS-like,butyrolactone,lipolanthine,terpene
      </option>
      
      <option value = "svg/clusterblast_r39c1_3.svg">
        NZ_CP023778 (1008246-1174316): Nocardia terpenica strain NC YFY NT001 chromos... (38% of genes show similarity), NRPS,PKS-like,butyrolactone,lipolanthine,terpene
      </option>
      
      <option value = "svg/clusterblast_r39c1_4.svg">
        NZ_CP046173 (936393-1113895): Nocardia terpenica strain AUSMDU00012715 chromo... (35% of genes show similarity), NRPS,PKS-like,butyrolactone,lipolanthine,terpene
      </option>
      
      <option value = "svg/clusterblast_r39c1_5.svg">
        NZ_LR215973 (5517006-5643566): Nocardia cyriacigeorgica strain 3012STDY675650... (45% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r39c1_6.svg">
        NZ_CP026746 (652632-778599): Nocardia cyriacigeorgica strain MDA3349 isolate ... (47% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r39c1_7.svg">
        NZ_KB907309 (58628-202751): Nocardia sp. BMG111209 G361DRAFT scaffold00003.3,... (46% of genes show similarity), NRPS,PKS-like,terpene
      </option>
      
      <option value = "svg/clusterblast_r39c1_8.svg">
        NC_016887 (776755-917098): Nocardia cyriacigeorgica GUH-2, complete genome (47% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r39c1_9.svg">
        NZ_CP046171 (863997-1006849): Nocardia brasiliensis strain AUSMDU00024985 chr... (51% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r39c1_10.svg">
        NZ_CP032568 (6004378-6130235): Nocardia sp. CFHS0054 chromosome (46% of genes show similarity), NRPS,terpene
      </option>
      
    </select>
    <button id= "clusterblast-r39c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r39c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r39c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-178"></div> <span class="help-tooltip" id="cb-known-help-178">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r39c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r39c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r39c1_1.svg">
        atratumycin
      </option>
      
      <option value = "svg/knownclusterblast_r39c1_2.svg">
        nocobactin NA
      </option>
      
    </select>
    <button id="knownclusterblast-r39c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r39c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r39c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-179"></div> <span class="help-tooltip" id="cb-sub-help-179">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r39c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-180"></div> <span class="help-tooltip" id="MIBiG-help-180">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r39c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r39c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r39c1"></div>
    <div id="comparison-MIBiG-r39c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">terpene</span></th>
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000321.1: 1657-43703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000321/index.html#r1c1" target="_new">BGC0000321.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.31 (id:0.41, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.58</td>
        <td>NRP</td>
        
        <td>chaetocin</td>
        
        
        <td>Collariella virescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.44, order:1.00, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.58</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001900.1: 225531-231743">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001900/index.html#r1c1" target="_new">BGC0001900.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.58</td>
        <td>NRP</td>
        
        <td>fragin</td>
        
        
        <td>Burkholderia cenocepacia H111</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.57</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.44, order:1.00, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.57</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.57</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.57</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.56</td>
        <td>NRP</td>
        
        <td>bacillibactin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.55</td>
        <td>NRP</td>
        
        <td>AM-toxin</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.44, order:0.44, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.55</td>
        <td>NRP</td>
        
        <td>rhodochelin</td>
        
        
        <td>Rhodococcus jostii RHA1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r39c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:0.73, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>pseudomonine</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r39c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-181"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-181">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r39c1-pfam-details-svg" data-filename="CC227C_contig_3_r39c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r39c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r39c1-nrps_pks_products" data-name="r39c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r39c1-nrps_pks" data-name="r39c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r39c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-175"></div> <span class="help-tooltip" id="nrps-structures-help-175">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 86206 - 182976:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r39c1-1" class="smiles-canvas" data-smiles="N1C(CCC1)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CO)C(=O)NC([*])C(=O)NC(CC(C)C)C(=O)NC(CC(C)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-pro - thr) + (X - ser - X - leu) + (leu)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-pro,thr&amp;nrps2=x,ser,x,leu&amp;nrps3=leu" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-pro*,thr*&amp;nrps2=x,ser*,x,leu*&amp;nrps3=leu*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r39c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="N1C(CCC1)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CO)C(=O)NC([*])C(=O)NC(CC(C)C)C(=O)NC(CC(C)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 86206 - 147593:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r39c1-2" class="smiles-canvas" data-smiles="N1C(CCC1)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CO)C(=O)NC([*])C(=O)NC(CC(C)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-pro - thr) + (X - ser - X - leu)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-pro,thr&amp;nrps2=x,ser,x,leu" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-pro*,thr*&amp;nrps2=x,ser*,x,leu*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r39c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="N1C(CCC1)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CO)C(=O)NC([*])C(=O)NC(CC(C)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 4, location 139673 - 182976:
          <div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r39c1-4" class="smiles-canvas" data-smiles="NC(CC(C)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(leu)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r39c1-4-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CC(C)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r39c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-176"></div> <span class="help-tooltip" id="nrps-monomers-help-176">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds91294_92914</span></strong>: X
      <div class="collapser collapser-target-cds91294_92914 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (56..446)</strong>: X
          <div class="collapser collapser-target-cds91294_92914 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds91294_92914-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">ac-iFGVyaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ile</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>40% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds104636_114035</span></strong>: pro - thr
      <div class="collapser collapser-target-cds104636_114035 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pro,thr" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pro*,thr*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (28..432)</strong>: pro
          <div class="collapser collapser-target-cds104636_114035 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  pro
              <div class="collapser collapser-target-cds104636_114035-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>pro</dd>
   <dt>Single AA prediction:</dt>
   <dd>pro</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVLLVAGiLK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pro</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1796..2205)</strong>: thr
          <div class="collapser collapser-target-cds104636_114035 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds104636_114035-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNVGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds114248_128195</span></strong>: X - ser - X - leu
      <div class="collapser collapser-target-cds114248_128195 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,ser,X,leu" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,ser*,X*,leu*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (27..430)</strong>: X
          <div class="collapser collapser-target-cds114248_128195 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds114248_128195-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DALfvGgiMK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1092..1495)</strong>: ser
          <div class="collapser collapser-target-cds114248_128195 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds114248_128195-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWdFiLVsK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2172..2572)</strong>: X
          <div class="collapser collapser-target-cds114248_128195 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds114248_128195-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLWHLiLvsK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3292..3694)</strong>: leu
          <div class="collapser collapser-target-cds114248_128195 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  leu
              <div class="collapser collapser-target-cds114248_128195-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>leu</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DPliSGaiiK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>glu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds159673_162976</span></strong>: leu
      <div class="collapser collapser-target-cds159673_162976 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (528..917)</strong>: leu
          <div class="collapser collapser-target-cds159673_162976 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  leu
              <div class="collapser collapser-target-cds159673_162976-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>leu</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DALHpGhvCK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>hpg</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds167529_168639</span></strong>: X
      <div class="collapser collapser-target-cds167529_168639 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (147..341)</strong>: X
          <div class="collapser collapser-target-cds167529_168639 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds167529_168639-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">--vvdTAgVK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>trp, tyr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>40% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r40c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_22 - Region 1 - NRPS,T2PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-283"></div> <span class="help-tooltip" id="region_svg-help-283">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_22.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r40c1-svg" data-filename="CC227C_contig_22_r40c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 99,304 nt. (total: 99,304 nt)
        
        <a class="cluster-rules-header" id="r40c1-rules-header" href="#r40c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r40c1-rules">T2PKS: (t2ks and t2clf)<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r40c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-284"></div> <span class="help-tooltip" id="focus-panel-help-284">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r40c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r40c1-nrps_pks" data-name="r40c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r40c1-clusterblast" data-name="r40c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r40c1-knownclusterblast" data-name="r40c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r40c1-subclusterblast" data-name="r40c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r40c1-MIBiG-cluster-compare" data-name="r40c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r40c1-pfam-details" data-name="r40c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r40c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-182"></div> <span class="help-tooltip" id="nrps-domain-help-182">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r40c1-details-svg" data-filename="CC227C_contig_22_r40c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r40c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r40c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-185"></div> <span class="help-tooltip" id="cb-general-help-185">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r40c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r40c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r40c1_1.svg">
        NZ_CP041695 (107738-270958): Nocardia otitidiscaviarum strain NEB252 chromoso... (32% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r40c1_2.svg">
        NZ_CP059399 (1970888-2141609): Nocardia sp. WCH-YHL-001 chromosome, complete ... (31% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r40c1_3.svg">
        NZ_CP006850 (7372077-7523891): Nocardia nova SH22a chromosome, complete genome (34% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r40c1_4.svg">
        NZ_CP032568 (1453702-1606647): Nocardia sp. CFHS0054 chromosome (28% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r40c1_5.svg">
        NC_006361 (5224713-5381654): Nocardia farcinica IFM 10152, complete genome (32% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r40c1_6.svg">
        NZ_CP031418 (4331502-4496115): Nocardia farcinica strain W6977 chromosome, co... (29% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r40c1_7.svg">
        NZ_KB907310 (247635-408422): Nocardia sp. BMG111209 G361DRAFT scaffold00004.4... (29% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r40c1_8.svg">
        NZ_LR215973 (3860935-4031992): Nocardia cyriacigeorgica strain 3012STDY675650... (25% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r40c1_9.svg">
        NZ_SMFR01000001 (1136371-1262509): Nocardia alba strain DSM 44684 Ga0244504 1... (32% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r40c1_10.svg">
        NZ_LMFE01000001 (2587146-2712008): Nocardia sp. Root136 contig 1, whole genom... (32% of genes show similarity), NRPS,bacteriocin
      </option>
      
    </select>
    <button id= "clusterblast-r40c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r40c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r40c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-186"></div> <span class="help-tooltip" id="cb-known-help-186">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r40c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r40c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r40c1_1.svg">
        granaticin
      </option>
      
      <option value = "svg/knownclusterblast_r40c1_2.svg">
        erdacin
      </option>
      
      <option value = "svg/knownclusterblast_r40c1_3.svg">
        JBIR-76 / JBIR-77
      </option>
      
      <option value = "svg/knownclusterblast_r40c1_4.svg">
        isofuranonaphthoquinone
      </option>
      
      <option value = "svg/knownclusterblast_r40c1_5.svg">
        SF2575
      </option>
      
      <option value = "svg/knownclusterblast_r40c1_6.svg">
        granaticin
      </option>
      
      <option value = "svg/knownclusterblast_r40c1_7.svg">
        fogacin A / fogacin B / fogacin C
      </option>
      
      <option value = "svg/knownclusterblast_r40c1_8.svg">
        alnumycin A / alnumycin B / alnumycin C / alnumycin P / prealnumycin / thalnumycin A / thalnumycin B / K1115A / 1,6-dihydro-8-propylanthraquinone
      </option>
      
      <option value = "svg/knownclusterblast_r40c1_9.svg">
        hiroshidine
      </option>
      
      <option value = "svg/knownclusterblast_r40c1_10.svg">
        rabelomycin / dehydrorabelomycin / fluostatin F / fluostatin G / fluostatin H
      </option>
      
    </select>
    <button id="knownclusterblast-r40c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r40c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r40c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-187"></div> <span class="help-tooltip" id="cb-sub-help-187">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r40c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r40c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r40c1_1.svg">
        GU233672_c1: granaticin t2pks (85% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r40c1_2.svg">
        AB103463_c1: medermycin t2pks (100% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r40c1_3.svg">
        AF324838_c1: simocyclinone t2pks (50% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r40c1_4.svg">
        AF080235_c1: landomycin t2pks (50% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r40c1_5.svg">
        FJ483966_c1: polyketomycin t2pks (25% of genes show similarity), t2pks
      </option>
      
    </select>
    <button id="subclusterblast-r40c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r40c1-svg"></div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r40c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-189"></div> <span class="help-tooltip" id="MIBiG-help-189">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r40c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r40c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r40c1"></div>
    <div id="comparison-MIBiG-r40c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T2PKS</span></th>
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
         
        
        <td>1.09</td>
        <td>Polyketide</td>
        
        <td>naphtocyclinone</td>
        
        
        <td>Streptomyces arenae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.75 (id:0.45, order:1.00, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
         
        
        <td>0.72</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.75 (id:0.45, order:1.00, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
         
        
        <td>0.70</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.54, order:0.64, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.53, order:0.69, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.67</td>
        <td>Polyketide</td>
        
        <td>granaticin</td>
        
        
        <td>Streptomyces violaceoruber</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.67</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.66</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.65</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.51, order:0.55, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.51, order:0.64, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.65</td>
        <td>Polyketide</td>
        
        <td>griseusin</td>
        
        
        <td>Streptomyces griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.64</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.62</td>
        <td>NRP</td>
        
        <td>bacillibactin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r40c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
      <td>0.81</td>
      <td>Polyketide</td>
      <td>naphtocyclinone</td>
      <td>Streptomyces arenae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.75 (id:0.45, order:1.00, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
      <td>0.75</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.75 (id:0.45, order:1.00, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
      <td>0.75</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.54, order:0.69, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>Polyketide</td>
      <td>granaticin</td>
      <td>Streptomyces violaceoruber</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.51, order:0.60, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Polyketide</td>
      <td>griseusin</td>
      <td>Streptomyces griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401.1: 0-12882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.47, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>paenibactin</td>
      <td>Paenibacillus elgii B69</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r40c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-190"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-190">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r40c1-pfam-details-svg" data-filename="CC227C_contig_22_r40c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r40c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r40c1-nrps_pks_products" data-name="r40c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r40c1-nrps_pks" data-name="r40c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
      <div class="sidepanel-details-header r40c1-t2pks" data-name="r40c1-t2pks">
        Type II PKS
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r40c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-183"></div> <span class="help-tooltip" id="nrps-structures-help-183">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 99304:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r40c1-1" class="smiles-canvas" data-smiles="NC(CO)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CO)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC(CO)C(=O)NC(CC(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ser - X - thr - thr - X) + (ser - X - X - X - ser - leu - X - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,x,thr,thr,x&amp;nrps2=ser,x,x,x,ser,leu,x,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,x,thr*,thr*,x&amp;nrps2=ser*,x,x,x,ser*,leu*,x,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r40c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CO)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CO)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC(CO)C(=O)NC(CC(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 0 - 62982:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r40c1-2" class="smiles-canvas" data-smiles="NC(CO)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CO)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC(CO)C(=O)NC(CC(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ser - X - thr - thr - X) + (ser - X - X - X - ser - leu - X - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,x,thr,thr,x&amp;nrps2=ser,x,x,x,ser,leu,x,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,x,thr*,thr*,x&amp;nrps2=ser*,x,x,x,ser*,leu*,x,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r40c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CO)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CO)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC(CO)C(=O)NC(CC(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 23291 - 99304:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r40c1-3" class="smiles-canvas" data-smiles="NC(CO)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CO)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC(CO)C(=O)NC(CC(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ser - X - thr - thr - X) + (ser - X - X - X - ser - leu - X - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,x,thr,thr,x&amp;nrps2=ser,x,x,x,ser,leu,x,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,x,thr*,thr*,x&amp;nrps2=ser*,x,x,x,ser*,leu*,x,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r40c1-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CO)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CO)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)NC(CO)C(=O)NC(CC(C)C)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r40c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-184"></div> <span class="help-tooltip" id="nrps-monomers-help-184">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds1007_2435</span></strong>: X
      <div class="collapser collapser-target-cds1007_2435 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (0..371)</strong>: X
          <div class="collapser collapser-target-cds1007_2435 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds1007_2435-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">hTaKisfItK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>bht</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>40% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds43291_61693</span></strong>: ser - X - thr - thr - X
      <div class="collapser collapser-target-cds43291_61693 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,X,thr,thr,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,X*,thr*,thr*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (480..893)</strong>: ser
          <div class="collapser collapser-target-cds43291_61693 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds43291_61693-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVpHwSLVeK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1712..2112)</strong>: X
          <div class="collapser collapser-target-cds43291_61693 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds43291_61693-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWslAlVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2768..3171)</strong>: thr
          <div class="collapser collapser-target-cds43291_61693 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds43291_61693-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNIGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3833..4236)</strong>: thr
          <div class="collapser collapser-target-cds43291_61693 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds43291_61693-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNIGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (4900..5296)</strong>: X
          <div class="collapser collapser-target-cds43291_61693 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds43291_61693-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLWyLiLvsK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds61692_87198</span></strong>: ser - X - X - X - ser - leu - X - X
      <div class="collapser collapser-target-cds61692_87198 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,X,X,X,ser,leu,X,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,X*,X*,X*,ser*,leu*,X*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (128..542)</strong>: ser
          <div class="collapser collapser-target-cds61692_87198 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds61692_87198-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWHFSLVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1205..1611)</strong>: X
          <div class="collapser collapser-target-cds61692_87198 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds61692_87198-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWdFaLVtK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2277..2674)</strong>: X
          <div class="collapser collapser-target-cds61692_87198 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds61692_87198-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWslAlVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3353..3752)</strong>: X
          <div class="collapser collapser-target-cds61692_87198 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn, lys, arg
              <div class="collapser collapser-target-cds61692_87198-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVwEhGaIsK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (4415..4873)</strong>: ser
          <div class="collapser collapser-target-cds61692_87198 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds61692_87198-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWHFSLVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (5539..5929)</strong>: leu
          <div class="collapser collapser-target-cds61692_87198 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  leu
              <div class="collapser collapser-target-cds61692_87198-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>leu</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DplLhGAIvK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (6582..6982)</strong>: X
          <div class="collapser collapser-target-cds61692_87198 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  pro, pip
              <div class="collapser collapser-target-cds61692_87198-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>pro, pip</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAmLsGsItK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (7658..8053)</strong>: X
          <div class="collapser collapser-target-cds61692_87198 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds61692_87198-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLWHLiLvsK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    <div class="sidepanel-details-section r40c1-t2pks">
      <div class="more-details">
    <div class="heading">
      <span>Type II PKS gene cluster prediction details</span>
      <div class="help-container"> <div class="help-icon" data-id="t2pks-help-188"></div> <span class="help-tooltip" id="t2pks-help-188">Predictions of starter units, elongations, product classes, and potential molecular weights for type II PKS clusters.<br>More detailed information is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/t2pks' target='_blank'>here</a>.</span></div>
    </div>
	
	
	<dl class="prediction-text">
        <dt><strong>Cluster section:</strong></dt>
        <dd>0 to 62982</dd>
		
		<dt><strong>Starter unit(s):</strong></dt>
		<dd>
				acetyl-CoA (Score: 0.0; E-value: 0)<br>
			
	    <dd>
		</tr>
		
		
		<dt><strong>Malonyl elongations:</strong></dt>
        <dd>
			
			7 (Score: 675.2; E-value: 6.5e-206)<br>
			
    	</dd>
		
		
		<dt><strong>Product class(es):</strong></td>
        <dd>
				
				benzoisochromanequinone<br>
				
		</dd>
		
		
		<dt><strong>Product molecular weight(s):</strong></dt>
		<dd>
			
			acetyl-CoA_7: 502.51 Da<br>
			
		</dd>
		
	</dl>
	<dl class="prediction-text">
		
		<dt><strong>cds18755_19919</strong></dt>
			
			<dd>
				GT
				
				(Score: 370.7; E-value: 2.6e-113)<br>
			</dd>
			
		
		<dt><strong>cds24615_25353</strong></dt>
			
			<dd>
				MET
				
				(Score: 31.4; E-value: 2.4e-10)<br>
			</dd>
			
		
		<dt><strong>cds25481_26750</strong></dt>
			
			<dd>
				KS
				
				(Score: 752.8; E-value: 2.1e-229)<br>
			</dd>
			
		
		<dt><strong>cds26746_27982</strong></dt>
			
			<dd>
				CLF
				
					7
				
				(Score: 675.2; E-value: 6.5e-206)<br>
			</dd>
			
		
		<dt><strong>cds27989_28235</strong></dt>
			
			<dd>
				ACP
				
				(Score: 94.8; E-value: 5.5e-30)<br>
			</dd>
			
		
		<dt><strong>cds28240_29164</strong></dt>
			
			<dd>
				CYC
				
					C7-C12
				
				(Score: 381.1; E-value: 7.2e-117)<br>
			</dd>
			
		
		<dt><strong>cds29160_30078</strong></dt>
			
			<dd>
				CYC
				
					C5-C14
				
				(Score: 445.0; E-value: 2.7e-136)<br>
			</dd>
			
		
		<dt><strong>cds7383_8169</strong></dt>
			
			<dd>
				KR
				
					C9
				
				(Score: 440.4; E-value: 4.6e-135)<br>
			</dd>
			
		
	</dl>
    
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r42c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_17 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-285"></div> <span class="help-tooltip" id="region_svg-help-285">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_17.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r42c1-svg" data-filename="CC227C_contig_17_r42c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 40,335 - 83,400 nt. (total: 43,066 nt)
        
        <a class="cluster-rules-header" id="r42c1-rules-header" href="#r42c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r42c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r42c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-286"></div> <span class="help-tooltip" id="focus-panel-help-286">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r42c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r42c1-nrps_pks" data-name="r42c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r42c1-clusterblast" data-name="r42c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r42c1-knownclusterblast" data-name="r42c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r42c1-subclusterblast" data-name="r42c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r42c1-MIBiG-cluster-compare" data-name="r42c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r42c1-pfam-details" data-name="r42c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r42c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-191"></div> <span class="help-tooltip" id="nrps-domain-help-191">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r42c1-details-svg" data-filename="CC227C_contig_17_r42c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r42c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r42c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-192"></div> <span class="help-tooltip" id="cb-general-help-192">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r42c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r42c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r42c1_1.svg">
        NZ_KB907309 (1170359-1218012): Nocardia sp. BMG111209 G361DRAFT scaffold00003... (55% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r42c1_2.svg">
        NZ_OBEG01000004 (423642-469825): Nocardia amikacinitolerans strain DSM 45537,... (57% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r42c1_3.svg">
        NZ_QZFU01000016 (780478-837671): Nocardia panacis strain YIM PH 21724 NODE 1,... (43% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r42c1_4.svg">
        NZ_CP046173 (9149291-9195486): Nocardia terpenica strain AUSMDU00012715 chrom... (61% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r42c1_5.svg">
        NZ_CP046172 (9874683-9919975): Nocardia arthritidis strain AUSMDU00012717 chr... (52% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r42c1_6.svg">
        NC_016887 (6079902-6125401): Nocardia cyriacigeorgica GUH-2, complete genome (56% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r42c1_7.svg">
        NZ_WEGK01000025 (44392-92060): Nocardia sp. RB20 NRB20 25, whole genome shotg... (48% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r42c1_8.svg">
        NZ_CP023778 (8695397-8741961): Nocardia terpenica strain NC YFY NT001 chromos... (59% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r42c1_9.svg">
        NZ_CP029710 (4481430-4547720): Nocardia sp. CS682 chromosome, complete genome (33% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r42c1_10.svg">
        NZ_LR215973 (4623500-4665343): Nocardia cyriacigeorgica strain 3012STDY675650... (56% of genes show similarity), T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r42c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r42c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r42c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-193"></div> <span class="help-tooltip" id="cb-known-help-193">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r42c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-194"></div> <span class="help-tooltip" id="cb-sub-help-194">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r42c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-195"></div> <span class="help-tooltip" id="MIBiG-help-195">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r42c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r42c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r42c1"></div>
    <div id="comparison-MIBiG-r42c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Other</td>
        
        <td>showdomycin</td>
        
        
        <td>Streptomyces showdoensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001895.1: 2411904-2429820">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001895/index.html#r1c1" target="_new">BGC0001895.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>huanglongmycin A, huanglongmycin B, huanglongmycin C</td>
        
        
        <td>Streptomyces sp. CB09001</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000257.1: 2579-43239">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000257/index.html#r1c1" target="_new">BGC0000257.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>prodigiosin</td>
        
        
        <td>Streptomyces griseoviridis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000896.1: 74-20352">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000896/index.html#r1c1" target="_new">BGC0000896.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Other</td>
        
        <td>cycloserine</td>
        
        
        <td>Streptomyces lavendulae subsp. lavendulae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001983.1: 622-39496">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001983/index.html#r1c1" target="_new">BGC0001983.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>triacsins</td>
        
        
        <td>Kitasatospora aureofaciens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001440.1: 0-13771">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001440/index.html#r1c1" target="_new">BGC0001440.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>cystargolide A, cystargolide B</td>
        
        
        <td>Kitasatospora cystarginea</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000346.1: 176-29878">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000346/index.html#r1c1" target="_new">BGC0000346.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.44, order:N/A, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>NRP, Polyketide</td>
        
        <td>epoxomicin</td>
        
        
        <td>Goodfellowiella coeruleoviolacea</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996.1: 4892741-4909316">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.43, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>NRP, Polyketide</td>
        
        <td>frontalamide B</td>
        
        
        <td>Streptomyces sp. SPB78</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000805.1: 897-5623">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000805/index.html#r1c1" target="_new">BGC0000805.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>moenomycin</td>
        
        
        <td>Streptomyces viridosporus ATCC 14672</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r42c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>showdomycin</td>
      <td>Streptomyces showdoensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001895.1: 2411904-2429820">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001895/index.html#r1c1" target="_new">BGC0001895.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>huanglongmycin A, huanglongmycin B, huanglongmycin C</td>
      <td>Streptomyces sp. CB09001</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000257.1: 2579-43239">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000257/index.html#r1c1" target="_new">BGC0000257.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>prodigiosin</td>
      <td>Streptomyces griseoviridis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000896.1: 74-20352">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000896/index.html#r1c1" target="_new">BGC0000896.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Other</td>
      <td>cycloserine</td>
      <td>Streptomyces lavendulae subsp. lavendulae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001983.1: 622-39496">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001983/index.html#r1c1" target="_new">BGC0001983.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Other</td>
      <td>triacsins</td>
      <td>Kitasatospora aureofaciens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001440.1: 0-13771">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001440/index.html#r1c1" target="_new">BGC0001440.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>cystargolide A, cystargolide B</td>
      <td>Kitasatospora cystarginea</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000346.1: 176-29878">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000346/index.html#r1c1" target="_new">BGC0000346.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.44, order:N/A, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>NRP, Polyketide</td>
      <td>epoxomicin</td>
      <td>Goodfellowiella coeruleoviolacea</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996.1: 4892741-4909316">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.43, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>NRP, Polyketide</td>
      <td>frontalamide B</td>
      <td>Streptomyces sp. SPB78</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000805.1: 897-5623">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000805/index.html#r1c1" target="_new">BGC0000805.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Saccharide</td>
      <td>moenomycin</td>
      <td>Streptomyces viridosporus ATCC 14672</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r42c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-196"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-196">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r42c1-pfam-details-svg" data-filename="CC227C_contig_17_r42c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r42c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r43c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_12 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-287"></div> <span class="help-tooltip" id="region_svg-help-287">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_12.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r43c1-svg" data-filename="CC227C_contig_12_r43c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 93,964 - 138,142 nt. (total: 44,179 nt)
        
        <a class="cluster-rules-header" id="r43c1-rules-header" href="#r43c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r43c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r43c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-288"></div> <span class="help-tooltip" id="focus-panel-help-288">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r43c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r43c1-nrps_pks" data-name="r43c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r43c1-clusterblast" data-name="r43c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r43c1-knownclusterblast" data-name="r43c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r43c1-subclusterblast" data-name="r43c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r43c1-MIBiG-cluster-compare" data-name="r43c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r43c1-pfam-details" data-name="r43c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r43c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-197"></div> <span class="help-tooltip" id="nrps-domain-help-197">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r43c1-details-svg" data-filename="CC227C_contig_12_r43c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r43c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r43c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-200"></div> <span class="help-tooltip" id="cb-general-help-200">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r43c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r43c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r43c1_1.svg">
        NZ_CP041695 (4070940-4112789): Nocardia otitidiscaviarum strain NEB252 chromo... (71% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r43c1_2.svg">
        NZ_LMRN01000009 (754442-889509): Rhodococcus sp. Leaf225 contig 17, whole gen... (14% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r43c1_3.svg">
        NZ_CP038030 (3584773-3726507): Rhodococcus ruber strain R1 chromosome, comple... (18% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r43c1_4.svg">
        NZ_CP024315 (1678363-1825611): Rhodococcus ruber strain P14 chromosome, compl... (17% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r43c1_5.svg">
        NZ_LN482607 (218061-344048): Rhodococcus ruber, whole genome shotgun sequence (17% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r43c1_6.svg">
        NZ_CP015529 (1355299-1502363): Rhodococcus sp. WB1 chromosome, complete genome (15% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r43c1_7.svg">
        NZ_CP029146 (3563809-3711111): Rhodococcus ruber strain SD3 chromosome, compl... (16% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r43c1_8.svg">
        NZ_CP050952 (4046630-4193798): Rhodococcus sp. DMU1 chromosome, complete genome (15% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r43c1_9.svg">
        NZ_CP024890 (3113189-3260452): Rhodococcus ruber strain YYL chromosome, compl... (16% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r43c1_10.svg">
        NZ_CP023714 (3916960-4062833): Rhodococcus ruber strain YC-YT1 chromosome, co... (16% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r43c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r43c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r43c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-201"></div> <span class="help-tooltip" id="cb-known-help-201">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r43c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-202"></div> <span class="help-tooltip" id="cb-sub-help-202">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r43c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-203"></div> <span class="help-tooltip" id="MIBiG-help-203">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r43c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r43c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r43c1"></div>
    <div id="comparison-MIBiG-r43c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP, Alkaloid</td>
        
        <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890.1: 1392460-1420815">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
        
        
        <td>Vibrio campbellii ATCC BAA-1116</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene</td>
        
        <td>raimonol</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001710.1: 451-19435">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001710/index.html#r1c1" target="_new">BGC0001710.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>obafluorin</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001437.1: 451-19435">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001437/index.html#r1c1" target="_new">BGC0001437.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>obafluorin</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001149.1: 196767-209553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001149/index.html#r1c1" target="_new">BGC0001149.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP, Saccharide</td>
        
        <td>brabantamide A</td>
        
        
        <td>Pseudomonas sp. SHC52</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r43c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP, Alkaloid</td>
      <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890.1: 1392460-1420815">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
      <td>Vibrio campbellii ATCC BAA-1116</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Terpene</td>
      <td>raimonol</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001710.1: 451-19435">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001710/index.html#r1c1" target="_new">BGC0001710.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>obafluorin</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001437.1: 451-19435">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001437/index.html#r1c1" target="_new">BGC0001437.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>obafluorin</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001149.1: 196767-209553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001149/index.html#r1c1" target="_new">BGC0001149.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP, Saccharide</td>
      <td>brabantamide A</td>
      <td>Pseudomonas sp. SHC52</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r43c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-204"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-204">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r43c1-pfam-details-svg" data-filename="CC227C_contig_12_r43c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r43c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r43c1-nrps_pks_products" data-name="r43c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r43c1-nrps_pks" data-name="r43c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r43c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-198"></div> <span class="help-tooltip" id="nrps-structures-help-198">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 93963 - 138142:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r43c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r43c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r43c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-199"></div> <span class="help-tooltip" id="nrps-monomers-help-199">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds113963_118142</span></strong>: X
      <div class="collapser collapser-target-cds113963_118142 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (563..948)</strong>: X
          <div class="collapser collapser-target-cds113963_118142 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala, val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds113963_118142-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DALlvGgVMK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r44c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_7 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-289"></div> <span class="help-tooltip" id="region_svg-help-289">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_7.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r44c1-svg" data-filename="CC227C_contig_7_r44c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 213,702 - 234,718 nt. (total: 21,017 nt)
        
        <a class="cluster-rules-header" id="r44c1-rules-header" href="#r44c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r44c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r44c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-290"></div> <span class="help-tooltip" id="focus-panel-help-290">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r44c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r44c1-clusterblast" data-name="r44c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r44c1-knownclusterblast" data-name="r44c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r44c1-subclusterblast" data-name="r44c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r44c1-MIBiG-cluster-compare" data-name="r44c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r44c1-pfam-details" data-name="r44c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r44c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-205"></div> <span class="help-tooltip" id="cb-general-help-205">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r44c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r44c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r44c1_1.svg">
        NZ_CP041695 (3652496-3673155): Nocardia otitidiscaviarum strain NEB252 chromo... (78% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r44c1_2.svg">
        NZ_QQAZ01000001 (569981-632913): Nocardia mexicana strain DSM 44952 Ga0244508... (25% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r44c1_3.svg">
        NZ_AXVD01000004 (210384-231497): Nocardia sp. CNY236 K940DRAFT scaffold00004.... (42% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r44c1_4.svg">
        NZ_SNXK01000001 (366753-387803): Nocardia ignorata strain DSM 44496 Ga0244500... (32% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r44c1_5.svg">
        NZ_CP021354 (3304653-3362384): Rhodococcus sp. S2-17 chromosome, complete genome (15% of genes show similarity), T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r44c1_6.svg">
        NC_023150 (551627-622211): Rhodococcus pyridinivorans SB3094, complete sequence (14% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r44c1_7.svg">
        NZ_CP018063 (1546307-1616906): Rhodococcus sp. 2G chromosome, complete genome (14% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r44c1_8.svg">
        NZ_CP040719 (2152330-2220009): Rhodococcus pyridinivorans strain YF3 chromoso... (12% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r44c1_9.svg">
        NZ_CP046572 (3399705-3457443): Rhodococcus sp. WAY2 chromosome, complete genome (16% of genes show similarity), T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r44c1_10.svg">
        NZ_CP038030 (2558483-2578981): Rhodococcus ruber strain R1 chromosome, comple... (38% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r44c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r44c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r44c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-206"></div> <span class="help-tooltip" id="cb-known-help-206">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r44c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r44c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r44c1_1.svg">
        SF2575
      </option>
      
    </select>
    <button id="knownclusterblast-r44c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r44c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r44c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-207"></div> <span class="help-tooltip" id="cb-sub-help-207">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r44c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-208"></div> <span class="help-tooltip" id="MIBiG-help-208">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r44c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r44c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r44c1"></div>
    <div id="comparison-MIBiG-r44c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Polyketide, Saccharide</td>
        
        <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000569.1: 0-9892">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000569/index.html#r1c1" target="_new">BGC0000569.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>RiPP</td>
        
        <td>plantazolicin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001173.1: 223360-233305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001173/index.html#r1c1" target="_new">BGC0001173.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.41, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>plantazolicin</td>
        
        
        <td>Bacillus pumilus ATCC 7061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Terpene</td>
        
        <td>platencin</td>
        
        
        <td>Streptomyces platensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000616.1: 3044505-3048678">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000616/index.html#r1c1" target="_new">BGC0000616.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>amylocyclicin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000010.1: 4801-68604">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000010/index.html#r1c1" target="_new">BGC0000010.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>aflatoxin</td>
        
        
        <td>Aspergillus flavus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000507.1: 0-7191">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000507/index.html#r1c1" target="_new">BGC0000507.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>RiPP</td>
        
        <td>epicidin</td>
        
        
        <td>Staphylococcus epidermidis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001407.1: 59853-70655">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001407/index.html#r1c1" target="_new">BGC0001407.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>RiPP</td>
        
        <td>bicereucin</td>
        
        
        <td>Bacillus cereus SJ1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000207.1: 821-16213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000207/index.html#r1c1" target="_new">BGC0000207.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>chartreusin</td>
        
        
        <td>Streptomyces chartreusis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r44c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Polyketide, Saccharide</td>
      <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
      <td>Saccharopolyspora erythraea</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Other</td>
      <td>flaviolin</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000569.1: 0-9892">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000569/index.html#r1c1" target="_new">BGC0000569.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>RiPP</td>
      <td>plantazolicin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001173.1: 223360-233305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001173/index.html#r1c1" target="_new">BGC0001173.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.41, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>plantazolicin</td>
      <td>Bacillus pumilus ATCC 7061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Terpene</td>
      <td>platencin</td>
      <td>Streptomyces platensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000616.1: 3044505-3048678">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000616/index.html#r1c1" target="_new">BGC0000616.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>RiPP</td>
      <td>amylocyclicin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000010.1: 4801-68604">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000010/index.html#r1c1" target="_new">BGC0000010.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Polyketide</td>
      <td>aflatoxin</td>
      <td>Aspergillus flavus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000507.1: 0-7191">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000507/index.html#r1c1" target="_new">BGC0000507.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>RiPP</td>
      <td>epicidin</td>
      <td>Staphylococcus epidermidis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001407.1: 59853-70655">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001407/index.html#r1c1" target="_new">BGC0001407.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>RiPP</td>
      <td>bicereucin</td>
      <td>Bacillus cereus SJ1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000207.1: 821-16213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000207/index.html#r1c1" target="_new">BGC0000207.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Polyketide</td>
      <td>chartreusin</td>
      <td>Streptomyces chartreusis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r44c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-209"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-209">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r44c1-pfam-details-svg" data-filename="CC227C_contig_7_r44c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r44c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r44c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_7 - Region 2 - T2PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-291"></div> <span class="help-tooltip" id="region_svg-help-291">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_7.region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r44c2-svg" data-filename="CC227C_contig_7_r44c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 236,267 - 308,671 nt. (total: 72,405 nt)
        
        <a class="cluster-rules-header" id="r44c2-rules-header" href="#r44c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r44c2-rules">T2PKS: (t2ks and t2clf)</div>
      
      <div class="region-svg-container">
       <div id='r44c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-292"></div> <span class="help-tooltip" id="focus-panel-help-292">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r44c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r44c2-nrps_pks" data-name="r44c2-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r44c2-clusterblast" data-name="r44c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r44c2-knownclusterblast" data-name="r44c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r44c2-subclusterblast" data-name="r44c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r44c2-MIBiG-cluster-compare" data-name="r44c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r44c2-pfam-details" data-name="r44c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r44c2-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-210"></div> <span class="help-tooltip" id="nrps-domain-help-210">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r44c2-details-svg" data-filename="CC227C_contig_7_r44c2_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r44c2-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r44c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-211"></div> <span class="help-tooltip" id="cb-general-help-211">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r44c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r44c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r44c2_1.svg">
        NZ_KB900614 (4487856-4560259): Salinispora pacifica DSM 45549 = CNT-150 strai... (21% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r44c2_2.svg">
        NZ_CP036534 (6603759-6707561): Streptomyces sp. VN1 chromosome, complete genome (15% of genes show similarity), NRPS,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r44c2_3.svg">
        NZ_CM002273 (6244611-6317103): Streptomyces sp. PVA 94-07 strain PVA 94-07 ch... (19% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r44c2_4.svg">
        NZ_CP026490 (675214-747706): Streptomyces sp. 604F chromosome, complete genome (18% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r44c2_5.svg">
        NZ_CM002271 (6236572-6309064): Streptomyces sp. GBA 94-10 chromosome, whole g... (18% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r44c2_6.svg">
        NZ_ML123108 (521900-594392): Streptomyces sp. ADI96-15 scaffold002, whole gen... (20% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r44c2_7.svg">
        NZ_FOGO01000003 (222978-295485): Streptomyces qinglanensis strain CGMCC 4.682... (20% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r44c2_8.svg">
        NZ_KE384240 (267801-340308): Streptomyces sp. CNT318 H296DRAFT scaffold00004.... (19% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r44c2_9.svg">
        NZ_AUAX01000007 (171670-212291): Hamadaea tsunoensis DSM 44101 G401DRAFT scaf... (36% of genes show similarity), other
      </option>
      
      <option value = "svg/clusterblast_r44c2_10.svg">
        NZ_CP010849 (7560897-7633389): Streptomyces cyaneogriseus subsp. noncyanogenu... (16% of genes show similarity), T2PKS
      </option>
      
    </select>
    <button id= "clusterblast-r44c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r44c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r44c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-212"></div> <span class="help-tooltip" id="cb-known-help-212">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r44c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r44c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r44c2_1.svg">
        enterocin
      </option>
      
      <option value = "svg/knownclusterblast_r44c2_2.svg">
        frenolicin B
      </option>
      
      <option value = "svg/knownclusterblast_r44c2_3.svg">
        actinorhodin
      </option>
      
      <option value = "svg/knownclusterblast_r44c2_4.svg">
        granaticin
      </option>
      
      <option value = "svg/knownclusterblast_r44c2_5.svg">
        granaticin
      </option>
      
      <option value = "svg/knownclusterblast_r44c2_6.svg">
        naphtocyclinone
      </option>
      
      <option value = "svg/knownclusterblast_r44c2_7.svg">
        cinerubin B
      </option>
      
      <option value = "svg/knownclusterblast_r44c2_8.svg">
        rishirilide B / rishirilide A
      </option>
      
      <option value = "svg/knownclusterblast_r44c2_9.svg">
        olimycin A / olimycin B
      </option>
      
      <option value = "svg/knownclusterblast_r44c2_10.svg">
        hedamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r44c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r44c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r44c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-213"></div> <span class="help-tooltip" id="cb-sub-help-213">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r44c2-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r44c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r44c2_1.svg">
        AF324838_c1: simocyclinone t2pks (50% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r44c2_2.svg">
        AB103463_c1: medermycin t2pks (75% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r44c2_3.svg">
        GU233672_c1: granaticin t2pks (42% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r44c2_4.svg">
        AF080235_c1: landomycin t2pks (50% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r44c2_5.svg">
        FJ483966_c1: polyketomycin t2pks (25% of genes show similarity), t2pks
      </option>
      
    </select>
    <button id="subclusterblast-r44c2-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r44c2-svg"></div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r44c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-215"></div> <span class="help-tooltip" id="MIBiG-help-215">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r44c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r44c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r44c2"></div>
    <div id="comparison-MIBiG-r44c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T2PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.52, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
        <td>0.42</td>
        <td>Polyketide</td>
        
        <td>naphtocyclinone</td>
        
        
        <td>Streptomyces arenae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.51, order:0.65, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.36</td>
        <td>Polyketide</td>
        
        <td>griseusin</td>
        
        
        <td>Streptomyces griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.51, order:0.69, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>Polyketide</td>
        
        <td>granaticin</td>
        
        
        <td>Streptomyces violaceoruber</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001075.1: 2658-14492">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001075/index.html#r1c1" target="_new">BGC0001075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>Terpene, Polyketide</td>
        
        <td>aurachin RE</td>
        
        
        <td>Rhodococcus erythropolis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.50, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Polyketide</td>
        
        <td>spore pigment</td>
        
        
        <td>Streptomyces collinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene</td>
        
        <td>pentalenolactone</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001104.1: 4797713-4809194">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001104/index.html#r1c1" target="_new">BGC0001104.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP, Polyketide</td>
        
        <td>myxovirescin A1</td>
        
        
        <td>Myxococcus xanthus DK 1622</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Other</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>raimonol</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001292.1: 5468-37257">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001292/index.html#r1c1" target="_new">BGC0001292.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Other</td>
        
        <td>A33853</td>
        
        
        <td>Streptomyces sp. NRRL 12068</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r44c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.52, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>Polyketide</td>
      <td>naphtocyclinone</td>
      <td>Streptomyces arenae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.51, order:0.65, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>Polyketide</td>
      <td>griseusin</td>
      <td>Streptomyces griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.51, order:0.69, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Polyketide</td>
      <td>granaticin</td>
      <td>Streptomyces violaceoruber</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001075.1: 2658-14492">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001075/index.html#r1c1" target="_new">BGC0001075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Terpene, Polyketide</td>
      <td>aurachin RE</td>
      <td>Rhodococcus erythropolis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.50, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>spore pigment</td>
      <td>Streptomyces collinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001104.1: 4797713-4809194">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001104/index.html#r1c1" target="_new">BGC0001104.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP, Polyketide</td>
      <td>myxovirescin A1</td>
      <td>Myxococcus xanthus DK 1622</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene</td>
      <td>raimonol</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001292.1: 5468-37257">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001292/index.html#r1c1" target="_new">BGC0001292.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>A33853</td>
      <td>Streptomyces sp. NRRL 12068</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r44c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-216"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-216">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r44c2-pfam-details-svg" data-filename="CC227C_contig_7_r44c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r44c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
      <div class="sidepanel-details-header r44c2-t2pks" data-name="r44c2-t2pks">
        Type II PKS
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    <div class="sidepanel-details-section r44c2-t2pks">
      <div class="more-details">
    <div class="heading">
      <span>Type II PKS gene cluster prediction details</span>
      <div class="help-container"> <div class="help-icon" data-id="t2pks-help-214"></div> <span class="help-tooltip" id="t2pks-help-214">Predictions of starter units, elongations, product classes, and potential molecular weights for type II PKS clusters.<br>More detailed information is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/t2pks' target='_blank'>here</a>.</span></div>
    </div>
	
	
	<dl class="prediction-text">
        <dt><strong>Cluster section:</strong></dt>
        <dd>236266 to 308671</dd>
		
		<dt><strong>Starter unit(s):</strong></dt>
		<dd>
				benzoyl-CoA (Score: 287.0; E-value: 4.63e-99)<br>
			
	    <dd>
		</tr>
		
		
		
		
	</dl>
	<dl class="prediction-text">
		
		<dt><strong>cds269970_270984</strong></dt>
			
			<dd>
				AT
				
				(Score: 346.9; E-value: 2.7e-106)<br>
			</dd>
			
			<dd>
				AT
				
					AAF81731.1_Enterocin_benzoyl-CoA
				
				(Score: 287.0; E-value: 4.63e-99)<br>
			</dd>
			
		
		<dt><strong>cds270980_271223</strong></dt>
			
			<dd>
				ACP
				
				(Score: 89.7; E-value: 2.1e-28)<br>
			</dd>
			
		
		<dt><strong>cds271266_272469</strong></dt>
			
			<dd>
				CLF
				
				(Score: 551.6; E-value: 2.2e-168)<br>
			</dd>
			
		
		<dt><strong>cds272465_273671</strong></dt>
			
			<dd>
				KS
				
				(Score: 660.6; E-value: 2e-201)<br>
			</dd>
			
		
		<dt><strong>cds273859_274621</strong></dt>
			
			<dd>
				KR
				
					C9
				
				(Score: 369.9; E-value: 1.5e-113)<br>
			</dd>
			
		
		<dt><strong>cds274644_275313</strong></dt>
			
			<dd>
				MET
				
				(Score: 59.9; E-value: 5e-19)<br>
			</dd>
			
		
		<dt><strong>cds285957_287070</strong></dt>
			
			<dd>
				MET
				
				(Score: 75.9; E-value: 7.3e-24)<br>
			</dd>
			
		
		<dt><strong>cds293953_294736</strong></dt>
			
			<dd>
				KR
				
				(Score: 103.4; E-value: 3.7e-32)<br>
			</dd>
			
		
		<dt><strong>cds306596_307739</strong></dt>
			
			<dd>
				OXY
				
				(Score: 174.9; E-value: 7.3e-54)<br>
			</dd>
			
		
	</dl>
    
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r44c3' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_7 - Region 3 - NAPAA <div class="help-container"> <div class="help-icon" data-id="region_svg-help-293"></div> <span class="help-tooltip" id="region_svg-help-293">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_7.region003.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r44c3-svg" data-filename="CC227C_contig_7_r44c3.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 343,733 - 377,761 nt. (total: 34,029 nt)
        
        <a class="cluster-rules-header" id="r44c3-rules-header" href="#r44c3">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r44c3-rules">NAPAA: (AMP-binding and PP-binding and TIGR02353)</div>
      
      <div class="region-svg-container">
       <div id='r44c3-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-294"></div> <span class="help-tooltip" id="focus-panel-help-294">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r44c3">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r44c3-nrps_pks" data-name="r44c3-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
      <div class="body-details-header r44c3-clusterblast" data-name="r44c3-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r44c3-knownclusterblast" data-name="r44c3-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r44c3-subclusterblast" data-name="r44c3-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r44c3-MIBiG-cluster-compare" data-name="r44c3-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r44c3-pfam-details" data-name="r44c3-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r44c3-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-217"></div> <span class="help-tooltip" id="nrps-domain-help-217">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r44c3-details-svg" data-filename="CC227C_contig_7_r44c3_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r44c3-details-svg"></div>
</div>
    </div>
     
    
     
    <div class="body-details-section r44c3-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-218"></div> <span class="help-tooltip" id="cb-general-help-218">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r44c3-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r44c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r44c3_1.svg">
        NZ_CP041695 (3480827-3524833): Nocardia otitidiscaviarum strain NEB252 chromo... (67% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r44c3_2.svg">
        NZ_CP059399 (5694118-5736382): Nocardia sp. WCH-YHL-001 chromosome, complete ... (17% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r44c3_3.svg">
        NZ_RBWZ01000006 (677249-758328): Microbacterium sp. AG790 Ga0222384 101, whol... (7% of genes show similarity), NRPS-like,other
      </option>
      
      <option value = "svg/clusterblast_r44c3_4.svg">
        NZ_QTTW01000002 (184584-228529): Rhodococcus sp. OK269 Ga0189761 102, whole g... (10% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r44c3_5.svg">
        NC_020504 (699925-769986): Streptomyces davaonensis JCM 4913, complete genome (10% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r44c3_6.svg">
        NZ_QQAZ01000005 (184542-255060): Nocardia mexicana strain DSM 44952 Ga0244508... (6% of genes show similarity), NRPS-like,PKS-like,terpene
      </option>
      
      <option value = "svg/clusterblast_r44c3_7.svg">
        NZ_CP043638 (286867-339427): Streptomyces sp. INR7 chromosome, complete genome (9% of genes show similarity), CDPS,NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r44c3_8.svg">
        NZ_LBMK01000008 (126658-176178): Streptomyces yangpuensis strain fd2-tb scaff... (9% of genes show similarity), CDPS,NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r44c3_9.svg">
        NZ_CP018870 (7221370-7265318): Streptomyces sp. TN58 chromosome, complete genome (11% of genes show similarity), CDPS,NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r44c3_10.svg">
        NZ_LQYB01000008 (9857-59498): Streptomyces sp. IMTB 1903 contig 20, whole gen... (9% of genes show similarity), CDPS,NRPS-like
      </option>
      
    </select>
    <button id= "clusterblast-r44c3-download">Download graphic</button>
  </div>
  <div id="clusterblast-r44c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r44c3-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-219"></div> <span class="help-tooltip" id="cb-known-help-219">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r44c3-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-220"></div> <span class="help-tooltip" id="cb-sub-help-220">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r44c3-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-221"></div> <span class="help-tooltip" id="MIBiG-help-221">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r44c3-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r44c3-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r44c3"></div>
    <div id="comparison-MIBiG-r44c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NAPAA</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>ergovaline</td>
        
        
        <td>Epichloe festucae var. lolii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Other</td>
        
        <td>ferrichrome</td>
        
        
        <td>Aspergillus oryzae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561.1: 1236-18814">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>curacomycin</td>
        
        
        <td>Streptomyces curacoi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.19</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000308.1: 176050-183901">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000308/index.html#r1c1" target="_new">BGC0000308.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>NRP</td>
        
        <td>aureusimine</td>
        
        
        <td>Staphylococcus aureus subsp. aureus str. JKD6008</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514.1: 0-37469">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Other</td>
        
        <td>aristeromycin</td>
        
        
        <td>Streptomyces citricolor</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r44c3-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>ergovaline</td>
      <td>Epichloe festucae var. lolii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>ferrichrome</td>
      <td>Aspergillus oryzae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561.1: 1236-18814">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>curacomycin</td>
      <td>Streptomyces curacoi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000308.1: 176050-183901">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000308/index.html#r1c1" target="_new">BGC0000308.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>NRP</td>
      <td>aureusimine</td>
      <td>Staphylococcus aureus subsp. aureus str. JKD6008</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514.1: 0-37469">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Other</td>
      <td>aristeromycin</td>
      <td>Streptomyces citricolor</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r44c3-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-222"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-222">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r44c3-pfam-details-svg" data-filename="CC227C_contig_7_r44c3_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r44c3-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r47c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC227C_contig_30 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-295"></div> <span class="help-tooltip" id="region_svg-help-295">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC227C_contig_30.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r47c1-svg" data-filename="CC227C_contig_30_r47c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 39,227 nt. (total: 39,227 nt)
        
        <a class="cluster-rules-header" id="r47c1-rules-header" href="#r47c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r47c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r47c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-296"></div> <span class="help-tooltip" id="focus-panel-help-296">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r47c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r47c1-nrps_pks" data-name="r47c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r47c1-clusterblast" data-name="r47c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r47c1-knownclusterblast" data-name="r47c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r47c1-subclusterblast" data-name="r47c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r47c1-MIBiG-cluster-compare" data-name="r47c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r47c1-pfam-details" data-name="r47c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r47c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-223"></div> <span class="help-tooltip" id="nrps-domain-help-223">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r47c1-details-svg" data-filename="CC227C_contig_30_r47c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r47c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r47c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-226"></div> <span class="help-tooltip" id="cb-general-help-226">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r47c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r47c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r47c1_1.svg">
        NZ_CP041695 (107738-270958): Nocardia otitidiscaviarum strain NEB252 chromoso... (25% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r47c1_2.svg">
        NZ_CP017839 (1479309-1588618): Nocardia seriolae strain EM150506 chromosome, ... (23% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r47c1_3.svg">
        NZ_QQAZ01000004 (291543-403900): Nocardia mexicana strain DSM 44952 Ga0244508... (21% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r47c1_4.svg">
        NZ_QQBC01000004 (306033-391403): Nocardia pseudobrasiliensis strain DSM 44290... (26% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r47c1_5.svg">
        NZ_CP018082 (6419767-6551173): Nocardia mangyaensis strain Y48 chromosome, co... (15% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r47c1_6.svg">
        NZ_LT985361 (8547642-8654705): Nocardia suismassiliense strain S-137, whole g... (15% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r47c1_7.svg">
        NZ_CP046173 (7951972-8035752): Nocardia terpenica strain AUSMDU00012715 chrom... (19% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r47c1_8.svg">
        NZ_CP026746 (2452609-2630012): Nocardia cyriacigeorgica strain MDA3349 isolat... (13% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r47c1_9.svg">
        NZ_LR215973 (3860935-4031992): Nocardia cyriacigeorgica strain 3012STDY675650... (12% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r47c1_10.svg">
        NZ_KB907310 (247635-408422): Nocardia sp. BMG111209 G361DRAFT scaffold00004.4... (17% of genes show similarity), NRPS,bacteriocin
      </option>
      
    </select>
    <button id= "clusterblast-r47c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r47c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r47c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-227"></div> <span class="help-tooltip" id="cb-known-help-227">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r47c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-228"></div> <span class="help-tooltip" id="cb-sub-help-228">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r47c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-229"></div> <span class="help-tooltip" id="MIBiG-help-229">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r47c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r47c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r47c1"></div>
    <div id="comparison-MIBiG-r47c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131.1: 144016-173487">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>ambactin</td>
        
        
        <td>Xenorhabdus miraniensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001240.1: 89510-118026">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001240/index.html#r1c1" target="_new">BGC0001240.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>serinocyclin A, serinocyclin B</td>
        
        
        <td>Metarhizium robertsii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001220.1: 56221-78301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001220/index.html#r1c1" target="_new">BGC0001220.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>aculeacin A</td>
        
        
        <td>Aspergillus japonicus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r47c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131.1: 144016-173487">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>ambactin</td>
      <td>Xenorhabdus miraniensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001240.1: 89510-118026">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001240/index.html#r1c1" target="_new">BGC0001240.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>serinocyclin A, serinocyclin B</td>
      <td>Metarhizium robertsii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001220.1: 56221-78301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001220/index.html#r1c1" target="_new">BGC0001220.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>aculeacin A</td>
      <td>Aspergillus japonicus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r47c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-230"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-230">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r47c1-pfam-details-svg" data-filename="CC227C_contig_30_r47c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r47c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r47c1-nrps_pks_products" data-name="r47c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r47c1-nrps_pks" data-name="r47c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r47c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-224"></div> <span class="help-tooltip" id="nrps-structures-help-224">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 39227:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r47c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(CO)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X - ser - X - X - D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,ser,x,x,d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,ser*,x,x,d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r47c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(CO)C(=O)NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r47c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-225"></div> <span class="help-tooltip" id="nrps-monomers-help-225">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds0_20667</span></strong>: X - ser - X - X - X
      <div class="collapser collapser-target-cds0_20667 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,ser,X,X,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,ser*,X*,X*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (646..1064)</strong>: X
          <div class="collapser collapser-target-cds0_20667 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds0_20667-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWslAlVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1730..2163)</strong>: ser
          <div class="collapser collapser-target-cds0_20667 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds0_20667-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWHFSLVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2832..3231)</strong>: X
          <div class="collapser collapser-target-cds0_20667 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds0_20667-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLWHLiLvsK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3898..4295)</strong>: X
          <div class="collapser collapser-target-cds0_20667 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds0_20667-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLWHLiLvsK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (5070..5474)</strong>: X
          <div class="collapser collapser-target-cds0_20667 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds0_20667-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAlLIGAIaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>


  <footer class="footer">
    <div class="container">
      <div>
        <img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
      </div>
      <div class="cite-me">
        If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
      </div>
      <div>
        <img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
      </div>
    </div>
  </footer>

  <script src="js/jquery.js"></script>
  <script src="js/antismash.js"></script>
  <script src="regions.js"></script>
  <script>
    $(document).ready(function() {
        viewer["start"](all_regions, details_data, resultsData, recordData);
    })
  </script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink">
 <defs>
  <filter id="inset-shadow">
   <feOffset dx="-2" dy="-2"></feOffset>
   <feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
   <feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
   <feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
   <feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
   <feComponentTransfer in="shadow" result="shadow">
    <feFuncA type="linear" slope=".95"></feFuncA>
   </feComponentTransfer>
   <feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
  </filter>
 </defs>
</svg>
</body>
</html>