
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>6666666.590938 - 36 region(s) - antiSMASH results</title>
  <style>img {width: 20px; height: 20px;}</style>
  <link rel="stylesheet" type="text/css" href="css/bacteria.css">
  <meta property="og:title" content="6666666.590938 - 33 record(s) - 36 region(s)" />
  <meta property="og:description" content="">
  <meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
  <meta property="og:image:alt" content="antiSMASH logo" />
  <meta property="og:image:width" content="600" />
  <meta property="og:image:height" content="600" />
  <meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-83721dc9-3412-4f36-b032-a71db769e588/index.html" />
  <meta property="og:type" content="website" />
  <meta name="twitter:card" content="summary" />
  <meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
  <div id="header">
      <nav>
    <div class="main-link">
     <div>
        <a href="https://antismash.secondarymetabolites.org/">
          <img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
        </a>
       </div>
       <div>
        <a href="https://antismash.secondarymetabolites.org/">
          antiSMASH version 6.0.1
        </a>
      </div>
    </div>
    <div class="custom-description"></div>
    <div class="ancillary-links">
      <div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
        <ul class="dropdown-options">
        
        <li><a href="6666666.590938.zip">Download all results</a></li>
        
          <li><a href="6666666.590938.gbk">Download GenBank summary file</a></li>
        
          <li><a href="bacteria-83721dc9-3412-4f36-b032-a71db769e588.log">Download log file</a></li>
        
        </ul>
      </div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
      <div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
    </div>
  </nav>
    <div id="buttons">
      <span id="region-type">Select genomic region:</span>
      <div id="regionbuttons">
        <div class="arrow-left" id="prev-region"></div>
        <div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
        <div style="display: flex; flex-wrap: wrap">
        
          
            
            
            <div class="regbutton hglE-KS r17c1">
              <a href="#r17c1">17.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T2PKS r26c1">
              <a href="#r26c1">26.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r27c1">
              <a href="#r27c1">27.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r32c1">
              <a href="#r32c1">32.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,NRPS-like,T1PKS hybrid r33c1">
              <a href="#r33c1">33.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton ectoine r34c1">
              <a href="#r34c1">34.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton siderophore r70c1">
              <a href="#r70c1">70.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton indole r78c1">
              <a href="#r78c1">78.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton butyrolactone r100c1">
              <a href="#r100c1">100.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r104c1">
              <a href="#r104c1">104.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton melanin r106c1">
              <a href="#r106c1">106.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton RiPP-like r110c1">
              <a href="#r110c1">110.1</a>
            </div>
          
            
            
            <div class="regbutton terpene r110c2">
              <a href="#r110c2">110.2</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r121c1">
              <a href="#r121c1">121.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T3PKS r135c1">
              <a href="#r135c1">135.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton T1PKS,hglE-KS hybrid r137c1">
              <a href="#r137c1">137.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton siderophore r147c1">
              <a href="#r147c1">147.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r153c1">
              <a href="#r153c1">153.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r160c1">
              <a href="#r160c1">160.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton indole r161c1">
              <a href="#r161c1">161.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T2PKS r168c1">
              <a href="#r168c1">168.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T2PKS r169c1">
              <a href="#r169c1">169.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r175c1">
              <a href="#r175c1">175.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS-like r176c1">
              <a href="#r176c1">176.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS-like r190c1">
              <a href="#r190c1">190.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r215c1">
              <a href="#r215c1">215.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton RiPP-like r216c1">
              <a href="#r216c1">216.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r222c1">
              <a href="#r222c1">222.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton lanthipeptide-class-iii r237c1">
              <a href="#r237c1">237.1</a>
            </div>
          
            
            
            <div class="regbutton lanthipeptide-class-i r237c2">
              <a href="#r237c2">237.2</a>
            </div>
          
            
            
            <div class="regbutton NRPS r237c3">
              <a href="#r237c3">237.3</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r244c1">
              <a href="#r244c1">244.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton siderophore r247c1">
              <a href="#r247c1">247.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton lanthipeptide-class-iii r250c1">
              <a href="#r250c1">250.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton lanthipeptide-class-v r251c1">
              <a href="#r251c1">251.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r259c1">
              <a href="#r259c1">259.1</a>
            </div>
          
        
        </div>
        <div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
      </div>
    </div>
  </div>

  <!-- overview page -->
  
  <div class="page" id="overview">
   
   
     <h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>
   
   <div class="overview-layout">
    <div id="record-tables">
     <div id="single-record-tables">
     <div class="record-overview-header">
        <strong>c00017_NODE_11..</strong> (original name was: NODE_116_length_11990_cov_126.721352) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-17" data-filename="c00017_NODE_11.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-17" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r17c1">
     
     
     <td class="regbutton hglE-KS r17c1">
      <a href="#r17c1">Region&nbsp17.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hgle-ks" target="_blank">hglE-KS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">12,038</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001484/1" target="_blank">2&#39;-chloropentostatin / 2&#39;-amino-2&#39;-deoxyadenosine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00026_NODE_35..</strong> (original name was: NODE_35_length_11251_cov_244.322281) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-26" data-filename="c00026_NODE_35.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-26" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r26c1">
     
     
     <td class="regbutton T2PKS r26c1">
      <a href="#r26c1">Region&nbsp26.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,299</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000236/1" target="_blank">kinamycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 31%, #ffffff00 31%)">31%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00027_NODE_72..</strong> (original name was: NODE_72_length_34283_cov_197.560074) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-27" data-filename="c00027_NODE_72.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-27" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r27c1">
     
     
     <td class="regbutton terpene r27c1">
      <a href="#r27c1">Region&nbsp27.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">19,619</td>
     <td class="digits table-split-left">34,331</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000660/1" target="_blank">albaflavenone</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00032_NODE_36..</strong> (original name was: NODE_362_length_165478_cov_143.106430) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-32" data-filename="c00032_NODE_36.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-32" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r32c1">
     
     
     <td class="regbutton T1PKS r32c1">
      <a href="#r32c1">Region&nbsp32.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">37,583</td>
     <td class="digits table-split-left">110,872</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000053/1" target="_blank">elaiophylin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 37%, #ffffff00 37%)">37%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00033_NODE_50..</strong> (original name was: NODE_50_length_48310_cov_119.834732) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-33" data-filename="c00033_NODE_50.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-33" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r33c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like,T1PKS hybrid r33c1">
      <a href="#r33c1">Region&nbsp33.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">48,358</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000034/1" target="_blank">candicidin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 90%, #ffffff00 90%)">90%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00034_NODE_78..</strong> (original name was: NODE_78_length_82232_cov_185.025970) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-34" data-filename="c00034_NODE_78.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-34" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r34c1">
     
     
     <td class="regbutton ectoine r34c1">
      <a href="#r34c1">Region&nbsp34.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">44,363</td>
     <td class="digits table-split-left">54,761</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000853/1" target="_blank">ectoine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00070_NODE_28..</strong> (original name was: NODE_288_length_80883_cov_179.989243) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-70" data-filename="c00070_NODE_28.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-70" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r70c1">
     
     
     <td class="regbutton siderophore r70c1">
      <a href="#r70c1">Region&nbsp70.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">10,012</td>
     <td class="digits table-split-left">22,072</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00078_NODE_61..</strong> (original name was: NODE_61_length_70330_cov_129.495972) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-78" data-filename="c00078_NODE_61.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-78" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r78c1">
     
     
     <td class="regbutton indole r78c1">
      <a href="#r78c1">Region&nbsp78.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
     </td>
     <td class="digits">42,578</td>
     <td class="digits table-split-left">63,756</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001294/1" target="_blank">7-prenylisatin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00100_NODE_16..</strong> (original name was: NODE_161_length_2408_cov_180.036545) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-100" data-filename="c00100_NODE_16.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-100" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r100c1">
     
     
     <td class="regbutton butyrolactone r100c1">
      <a href="#r100c1">Region&nbsp100.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">2,456</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000238/1" target="_blank">lactonamycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 3%, #ffffff00 3%)">3%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00104_NODE_43..</strong> (original name was: NODE_431_length_63126_cov_130.693649) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-104" data-filename="c00104_NODE_43.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-104" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r104c1">
     
     
     <td class="regbutton terpene r104c1">
      <a href="#r104c1">Region&nbsp104.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">17,236</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000664/1" target="_blank">isorenieratene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 71%, #ffffff00 71%)">71%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00106_NODE_14..</strong> (original name was: NODE_144_length_17782_cov_176.222137) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-106" data-filename="c00106_NODE_14.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-106" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r106c1">
     
     
     <td class="regbutton melanin r106c1">
      <a href="#r106c1">Region&nbsp106.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#melanin" target="_blank">melanin</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,859</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000909/1" target="_blank">melanin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00110_NODE_21..</strong> (original name was: NODE_21_length_34499_cov_143.338150) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-110" data-filename="c00110_NODE_21.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-110" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r110c1">
     
     
     <td class="regbutton RiPP-like r110c1">
      <a href="#r110c1">Region&nbsp110.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">7,665</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000518/1" target="_blank">informatipeptin</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r110c2">
     
     
     <td class="regbutton terpene r110c2">
      <a href="#r110c2">Region&nbsp110.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">20,623</td>
     <td class="digits table-split-left">34,547</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001204/1" target="_blank">versipelostatin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 5%, #ffffff00 5%)">5%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00121_NODE_36..</strong> (original name was: NODE_368_length_31545_cov_161.806305) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-121" data-filename="c00121_NODE_36.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-121" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r121c1">
     
     
     <td class="regbutton NRPS r121c1">
      <a href="#r121c1">Region&nbsp121.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">31,593</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001370/1" target="_blank">lipopeptide 8D1-1 / lipopeptide 8D1-2</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 29%, #ffffff00 29%)">29%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00135_NODE_31..</strong> (original name was: NODE_318_length_90431_cov_156.345795) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-135" data-filename="c00135_NODE_31.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-135" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r135c1">
     
     
     <td class="regbutton T3PKS r135c1">
      <a href="#r135c1">Region&nbsp135.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">27,867</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000902/1" target="_blank">flaviolin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00137_NODE_16..</strong> (original name was: NODE_165_length_96273_cov_115.652596) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-137" data-filename="c00137_NODE_16.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-137" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r137c1">
     
     
         
     
     <td class="regbutton T1PKS,hglE-KS hybrid r137c1">
      <a href="#r137c1">Region&nbsp137.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hgle-ks" target="_blank">hglE-KS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">96,321</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000034/1" target="_blank">candicidin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 33%, #ffffff00 33%)">33%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00147_NODE_39..</strong> (original name was: NODE_39_length_59920_cov_165.983368) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-147" data-filename="c00147_NODE_39.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-147" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r147c1">
     
     
     <td class="regbutton siderophore r147c1">
      <a href="#r147c1">Region&nbsp147.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">29,075</td>
     <td class="digits table-split-left">42,248</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001732/1" target="_blank">paulomycin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00153_NODE_46..</strong> (original name was: NODE_46_length_69365_cov_140.049072) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-153" data-filename="c00153_NODE_46.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-153" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r153c1">
     
     
     <td class="regbutton terpene r153c1">
      <a href="#r153c1">Region&nbsp153.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">28,082</td>
     <td class="digits table-split-left">49,866</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000242/1" target="_blank">lysolipin I</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00160_NODE_14..</strong> (original name was: NODE_140_length_19703_cov_132.320007) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-160" data-filename="c00160_NODE_14.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-160" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r160c1">
     
     
     <td class="regbutton terpene r160c1">
      <a href="#r160c1">Region&nbsp160.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">4,770</td>
     <td class="digits table-split-left">19,751</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000633/1" target="_blank">carotenoid</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 45%, #ffffff00 45%)">45%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00161_NODE_7_..</strong> (original name was: NODE_7_length_133672_cov_140.510620) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-161" data-filename="c00161_NODE_7_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-161" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r161c1">
     
     
     <td class="regbutton indole r161c1">
      <a href="#r161c1">Region&nbsp161.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
     </td>
     <td class="digits">81,865</td>
     <td class="digits table-split-left">102,986</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001483/1" target="_blank">5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00168_NODE_99..</strong> (original name was: NODE_99_length_39965_cov_186.470657) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-168" data-filename="c00168_NODE_99.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-168" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r168c1">
     
     
     <td class="regbutton T2PKS r168c1">
      <a href="#r168c1">Region&nbsp168.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">40,013</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000271/1" target="_blank">spore pigment</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 66%, #ffffff00 66%)">66%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00169_NODE_16..</strong> (original name was: NODE_166_length_10390_cov_228.446289) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-169" data-filename="c00169_NODE_16.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-169" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r169c1">
     
     
     <td class="regbutton T2PKS r169c1">
      <a href="#r169c1">Region&nbsp169.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">10,438</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000230/1" target="_blank">griseorhodin A</a></td>
      <td>Polyketide:Type II</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00175_NODE_49..</strong> (original name was: NODE_49_length_120568_cov_137.885986) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-175" data-filename="c00175_NODE_49.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-175" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r175c1">
     
     
     <td class="regbutton NRPS r175c1">
      <a href="#r175c1">Region&nbsp175.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">49,804</td>
     <td class="digits table-split-left">100,801</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000325/1" target="_blank">coelichelin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00176_NODE_21..</strong> (original name was: NODE_210_length_35221_cov_200.389709) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-176" data-filename="c00176_NODE_21.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-176" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r176c1">
     
     
     <td class="regbutton NRPS-like r176c1">
      <a href="#r176c1">Region&nbsp176.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">10,509</td>
     <td class="digits table-split-left">35,269</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000843/1" target="_blank">alanylclavam / 2-hydroxymethylclavam / 2-formyloxymethylclavam / clavam-2-carboxylate</a></td>
      <td>Other:Non-NRP beta-lactam</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00190_NODE_14..</strong> (original name was: NODE_145_length_34603_cov_119.263680) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-190" data-filename="c00190_NODE_14.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-190" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r190c1">
     
     
     <td class="regbutton NRPS-like r190c1">
      <a href="#r190c1">Region&nbsp190.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">34,651</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000432/1" target="_blank">streptothricin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00215_NODE_27..</strong> (original name was: NODE_278_length_105898_cov_154.234192) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-215" data-filename="c00215_NODE_27.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-215" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r215c1">
     
     
     <td class="regbutton terpene r215c1">
      <a href="#r215c1">Region&nbsp215.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">5,200</td>
     <td class="digits table-split-left">31,919</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000663/1" target="_blank">hopene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00216_NODE_14..</strong> (original name was: NODE_147_length_11367_cov_167.855103) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-216" data-filename="c00216_NODE_14.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-216" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r216c1">
     
     
     <td class="regbutton RiPP-like r216c1">
      <a href="#r216c1">Region&nbsp216.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">5,514</td>
     <td class="digits table-split-left">11,415</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00222_NODE_47..</strong> (original name was: NODE_47_length_64561_cov_165.305359) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-222" data-filename="c00222_NODE_47.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-222" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r222c1">
     
     
     <td class="regbutton terpene r222c1">
      <a href="#r222c1">Region&nbsp222.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">16,889</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001181/1" target="_blank">geosmin</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00237_NODE_6_..</strong> (original name was: NODE_6_length_273093_cov_134.389847) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-237" data-filename="c00237_NODE_6_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-237" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r237c1">
     
     
     <td class="regbutton lanthipeptide-class-iii r237c1">
      <a href="#r237c1">Region&nbsp237.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
     </td>
     <td class="digits">149,353</td>
     <td class="digits table-split-left">171,935</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000551/1" target="_blank">SapB</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r237c2">
     
     
     <td class="regbutton lanthipeptide-class-i r237c2">
      <a href="#r237c2">Region&nbsp237.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-i" target="_blank">lanthipeptide-class-i</a>
     </td>
     <td class="digits">184,507</td>
     <td class="digits table-split-left">209,100</td>
     
      <td colspan="3"></td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r237c3">
     
     
     <td class="regbutton NRPS r237c3">
      <a href="#r237c3">Region&nbsp237.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">212,872</td>
     <td class="digits table-split-left">265,197</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000324/1" target="_blank">coelibactin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00244_NODE_43..</strong> (original name was: NODE_430_length_27561_cov_151.331635) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-244" data-filename="c00244_NODE_43.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-244" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r244c1">
     
     
     <td class="regbutton NRPS r244c1">
      <a href="#r244c1">Region&nbsp244.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">27,609</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001370/1" target="_blank">lipopeptide 8D1-1 / lipopeptide 8D1-2</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00247_NODE_34..</strong> (original name was: NODE_340_length_77411_cov_188.628815) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-247" data-filename="c00247_NODE_34.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-247" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r247c1">
     
     
     <td class="regbutton siderophore r247c1">
      <a href="#r247c1">Region&nbsp247.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">11,174</td>
     <td class="digits table-split-left">22,946</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000940/1" target="_blank">desferrioxamin B / desferrioxamine E</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 83%, #ffffff00 83%)">83%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00250_NODE_18..</strong> (original name was: NODE_18_length_61109_cov_192.928314) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-250" data-filename="c00250_NODE_18.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-250" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r250c1">
     
     
     <td class="regbutton lanthipeptide-class-iii r250c1">
      <a href="#r250c1">Region&nbsp250.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
     </td>
     <td class="digits">33,832</td>
     <td class="digits table-split-left">56,435</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000501/1" target="_blank">catenulipeptin</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00251_NODE_13..</strong> (original name was: NODE_137_length_6721_cov_140.441757) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-251" data-filename="c00251_NODE_13.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-251" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r251c1">
     
     
     <td class="regbutton lanthipeptide-class-v r251c1">
      <a href="#r251c1">Region&nbsp251.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-v" target="_blank">lanthipeptide-class-v</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,769</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00259_NODE_48..</strong> (original name was: NODE_481_length_10956_cov_119.354507) (Streptomyces rochei CC71)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-259" data-filename="c00259_NODE_48.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-259" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r259c1">
     
     
     <td class="regbutton T1PKS r259c1">
      <a href="#r259c1">Region&nbsp259.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,004</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000110/1" target="_blank">spectinabilin / orinocin / SNF4435C / SNF4435D</a></td>
      <td>Polyketide:Modular type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 36%, #ffffff00 36%)">36%</td>
     
   </tr>
 
 </tbody>
</table>
      </div>
     </div>
     
     <div id="compact-record-table" style="display: none;">
      <div class="record-table">
       <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 
 
  <tr class="linked-row odd" data-anchor="#r17c1">
     
     
     <td class="regbutton hglE-KS r17c1">
      <a href="#r17c1">Region&nbsp17.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hgle-ks" target="_blank">hglE-KS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">12,038</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001484/1" target="_blank">2&#39;-chloropentostatin / 2&#39;-amino-2&#39;-deoxyadenosine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r26c1">
     
     
     <td class="regbutton T2PKS r26c1">
      <a href="#r26c1">Region&nbsp26.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,299</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000236/1" target="_blank">kinamycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 31%, #ffffff00 31%)">31%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r27c1">
     
     
     <td class="regbutton terpene r27c1">
      <a href="#r27c1">Region&nbsp27.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">19,619</td>
     <td class="digits table-split-left">34,331</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000660/1" target="_blank">albaflavenone</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r32c1">
     
     
     <td class="regbutton T1PKS r32c1">
      <a href="#r32c1">Region&nbsp32.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">37,583</td>
     <td class="digits table-split-left">110,872</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000053/1" target="_blank">elaiophylin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 37%, #ffffff00 37%)">37%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r33c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like,T1PKS hybrid r33c1">
      <a href="#r33c1">Region&nbsp33.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">48,358</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000034/1" target="_blank">candicidin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 90%, #ffffff00 90%)">90%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r34c1">
     
     
     <td class="regbutton ectoine r34c1">
      <a href="#r34c1">Region&nbsp34.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">44,363</td>
     <td class="digits table-split-left">54,761</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000853/1" target="_blank">ectoine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r70c1">
     
     
     <td class="regbutton siderophore r70c1">
      <a href="#r70c1">Region&nbsp70.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">10,012</td>
     <td class="digits table-split-left">22,072</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r78c1">
     
     
     <td class="regbutton indole r78c1">
      <a href="#r78c1">Region&nbsp78.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
     </td>
     <td class="digits">42,578</td>
     <td class="digits table-split-left">63,756</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001294/1" target="_blank">7-prenylisatin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r100c1">
     
     
     <td class="regbutton butyrolactone r100c1">
      <a href="#r100c1">Region&nbsp100.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">2,456</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000238/1" target="_blank">lactonamycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 3%, #ffffff00 3%)">3%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r104c1">
     
     
     <td class="regbutton terpene r104c1">
      <a href="#r104c1">Region&nbsp104.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">17,236</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000664/1" target="_blank">isorenieratene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 71%, #ffffff00 71%)">71%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r106c1">
     
     
     <td class="regbutton melanin r106c1">
      <a href="#r106c1">Region&nbsp106.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#melanin" target="_blank">melanin</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,859</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000909/1" target="_blank">melanin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r110c1">
     
     
     <td class="regbutton RiPP-like r110c1">
      <a href="#r110c1">Region&nbsp110.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">7,665</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000518/1" target="_blank">informatipeptin</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r110c2">
     
     
     <td class="regbutton terpene r110c2">
      <a href="#r110c2">Region&nbsp110.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">20,623</td>
     <td class="digits table-split-left">34,547</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001204/1" target="_blank">versipelostatin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 5%, #ffffff00 5%)">5%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r121c1">
     
     
     <td class="regbutton NRPS r121c1">
      <a href="#r121c1">Region&nbsp121.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">31,593</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001370/1" target="_blank">lipopeptide 8D1-1 / lipopeptide 8D1-2</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 29%, #ffffff00 29%)">29%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r135c1">
     
     
     <td class="regbutton T3PKS r135c1">
      <a href="#r135c1">Region&nbsp135.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">27,867</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000902/1" target="_blank">flaviolin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r137c1">
     
     
         
     
     <td class="regbutton T1PKS,hglE-KS hybrid r137c1">
      <a href="#r137c1">Region&nbsp137.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hgle-ks" target="_blank">hglE-KS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">96,321</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000034/1" target="_blank">candicidin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 33%, #ffffff00 33%)">33%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r147c1">
     
     
     <td class="regbutton siderophore r147c1">
      <a href="#r147c1">Region&nbsp147.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">29,075</td>
     <td class="digits table-split-left">42,248</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001732/1" target="_blank">paulomycin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r153c1">
     
     
     <td class="regbutton terpene r153c1">
      <a href="#r153c1">Region&nbsp153.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">28,082</td>
     <td class="digits table-split-left">49,866</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000242/1" target="_blank">lysolipin I</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r160c1">
     
     
     <td class="regbutton terpene r160c1">
      <a href="#r160c1">Region&nbsp160.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">4,770</td>
     <td class="digits table-split-left">19,751</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000633/1" target="_blank">carotenoid</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 45%, #ffffff00 45%)">45%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r161c1">
     
     
     <td class="regbutton indole r161c1">
      <a href="#r161c1">Region&nbsp161.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
     </td>
     <td class="digits">81,865</td>
     <td class="digits table-split-left">102,986</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001483/1" target="_blank">5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r168c1">
     
     
     <td class="regbutton T2PKS r168c1">
      <a href="#r168c1">Region&nbsp168.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">40,013</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000271/1" target="_blank">spore pigment</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 66%, #ffffff00 66%)">66%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r169c1">
     
     
     <td class="regbutton T2PKS r169c1">
      <a href="#r169c1">Region&nbsp169.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">10,438</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000230/1" target="_blank">griseorhodin A</a></td>
      <td>Polyketide:Type II</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r175c1">
     
     
     <td class="regbutton NRPS r175c1">
      <a href="#r175c1">Region&nbsp175.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">49,804</td>
     <td class="digits table-split-left">100,801</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000325/1" target="_blank">coelichelin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r176c1">
     
     
     <td class="regbutton NRPS-like r176c1">
      <a href="#r176c1">Region&nbsp176.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">10,509</td>
     <td class="digits table-split-left">35,269</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000843/1" target="_blank">alanylclavam / 2-hydroxymethylclavam / 2-formyloxymethylclavam / clavam-2-carboxylate</a></td>
      <td>Other:Non-NRP beta-lactam</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r190c1">
     
     
     <td class="regbutton NRPS-like r190c1">
      <a href="#r190c1">Region&nbsp190.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">34,651</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000432/1" target="_blank">streptothricin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r215c1">
     
     
     <td class="regbutton terpene r215c1">
      <a href="#r215c1">Region&nbsp215.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">5,200</td>
     <td class="digits table-split-left">31,919</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000663/1" target="_blank">hopene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r216c1">
     
     
     <td class="regbutton RiPP-like r216c1">
      <a href="#r216c1">Region&nbsp216.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">5,514</td>
     <td class="digits table-split-left">11,415</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r222c1">
     
     
     <td class="regbutton terpene r222c1">
      <a href="#r222c1">Region&nbsp222.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">16,889</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001181/1" target="_blank">geosmin</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r237c1">
     
     
     <td class="regbutton lanthipeptide-class-iii r237c1">
      <a href="#r237c1">Region&nbsp237.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
     </td>
     <td class="digits">149,353</td>
     <td class="digits table-split-left">171,935</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000551/1" target="_blank">SapB</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r237c2">
     
     
     <td class="regbutton lanthipeptide-class-i r237c2">
      <a href="#r237c2">Region&nbsp237.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-i" target="_blank">lanthipeptide-class-i</a>
     </td>
     <td class="digits">184,507</td>
     <td class="digits table-split-left">209,100</td>
     
      <td colspan="3"></td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r237c3">
     
     
     <td class="regbutton NRPS r237c3">
      <a href="#r237c3">Region&nbsp237.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">212,872</td>
     <td class="digits table-split-left">265,197</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000324/1" target="_blank">coelibactin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r244c1">
     
     
     <td class="regbutton NRPS r244c1">
      <a href="#r244c1">Region&nbsp244.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">27,609</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001370/1" target="_blank">lipopeptide 8D1-1 / lipopeptide 8D1-2</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r247c1">
     
     
     <td class="regbutton siderophore r247c1">
      <a href="#r247c1">Region&nbsp247.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">11,174</td>
     <td class="digits table-split-left">22,946</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000940/1" target="_blank">desferrioxamin B / desferrioxamine E</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 83%, #ffffff00 83%)">83%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r250c1">
     
     
     <td class="regbutton lanthipeptide-class-iii r250c1">
      <a href="#r250c1">Region&nbsp250.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
     </td>
     <td class="digits">33,832</td>
     <td class="digits table-split-left">56,435</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000501/1" target="_blank">catenulipeptin</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r251c1">
     
     
     <td class="regbutton lanthipeptide-class-v r251c1">
      <a href="#r251c1">Region&nbsp251.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-v" target="_blank">lanthipeptide-class-v</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,769</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r259c1">
     
     
     <td class="regbutton T1PKS r259c1">
      <a href="#r259c1">Region&nbsp259.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,004</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000110/1" target="_blank">spectinabilin / orinocin / SNF4435C / SNF4435D</a></td>
      <td>Polyketide:Modular type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 36%, #ffffff00 36%)">36%</td>
     
   </tr>
  
 
 </tbody>
</table>
      </div>
     </div>
     
     
     <dl id="regionless-records">
      <dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>
       
         <dd><strong>c00001_NODE_20..</strong> (original name was: NODE_209_length_10346_cov_151.307068) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00002_NODE_56..</strong> (original name was: NODE_56_length_96297_cov_178.661728) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00003_NODE_51..</strong> (original name was: NODE_51_length_8044_cov_197.163727) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00004_NODE_22..</strong> (original name was: NODE_220_length_1345_cov_357.331604) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00005_NODE_89..</strong> (original name was: NODE_89_length_9666_cov_137.999588) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00006_NODE_12..</strong> (original name was: NODE_121_length_19618_cov_196.407272) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00007_NODE_17..</strong> (original name was: NODE_176_length_106293_cov_197.631683) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00008_NODE_26..</strong> (original name was: NODE_26_length_73059_cov_175.433258) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00009_NODE_20..</strong> (original name was: NODE_202_length_17586_cov_191.794388) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00010_NODE_70..</strong> (original name was: NODE_70_length_50128_cov_204.657257) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00011_NODE_24..</strong> (original name was: NODE_245_length_7454_cov_183.545609) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00012_NODE_18..</strong> (original name was: NODE_187_length_2563_cov_246.483414) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00013_NODE_21..</strong> (original name was: NODE_214_length_2301_cov_174.895264) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00014_NODE_29..</strong> (original name was: NODE_291_length_1227_cov_218.104324) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00015_NODE_16..</strong> (original name was: NODE_167_length_15880_cov_180.648743) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00016_NODE_11..</strong> (original name was: NODE_11_length_14701_cov_180.983948) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00018_NODE_10..</strong> (original name was: NODE_104_length_3080_cov_177.748047) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00019_NODE_36..</strong> (original name was: NODE_369_length_18134_cov_179.950150) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00020_NODE_24..</strong> (original name was: NODE_240_length_4564_cov_172.204651) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00021_NODE_17..</strong> (original name was: NODE_171_length_3755_cov_154.800262) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00022_NODE_27..</strong> (original name was: NODE_273_length_56675_cov_131.924408) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00023_NODE_40..</strong> (original name was: NODE_40_length_79958_cov_175.192307) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00024_NODE_50..</strong> (original name was: NODE_508_length_34275_cov_181.180634) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00025_NODE_13..</strong> (original name was: NODE_138_length_63602_cov_192.728516) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00028_NODE_65..</strong> (original name was: NODE_65_length_38326_cov_192.375259) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00029_NODE_33..</strong> (original name was: NODE_33_length_38720_cov_204.147491) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00030_NODE_28..</strong> (original name was: NODE_283_length_14127_cov_178.889709) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00031_NODE_23..</strong> (original name was: NODE_235_length_7855_cov_191.668243) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00035_NODE_17..</strong> (original name was: NODE_177_length_30413_cov_153.888306) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00036_NODE_90..</strong> (original name was: NODE_90_length_18337_cov_152.939636) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00037_NODE_23..</strong> (original name was: NODE_23_length_158052_cov_140.173767) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00038_NODE_28..</strong> (original name was: NODE_28_length_35546_cov_197.734344) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00039_NODE_83..</strong> (original name was: NODE_83_length_8770_cov_165.869675) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00040_NODE_17..</strong> (original name was: NODE_175_length_18935_cov_135.794083) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00041_NODE_16..</strong> (original name was: NODE_163_length_9879_cov_199.096970) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00042_NODE_21..</strong> (original name was: NODE_212_length_15797_cov_181.155975) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00043_NODE_14..</strong> (original name was: NODE_14_length_39063_cov_195.894272) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00044_NODE_18..</strong> (original name was: NODE_183_length_20005_cov_189.887482) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00045_NODE_15..</strong> (original name was: NODE_150_length_5924_cov_159.052841) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00046_NODE_31..</strong> (original name was: NODE_311_length_1905_cov_145.436218) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00047_NODE_19..</strong> (original name was: NODE_19_length_22900_cov_192.827988) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00048_NODE_12..</strong> (original name was: NODE_122_length_3364_cov_173.453323) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00049_NODE_4_..</strong> (original name was: NODE_4_length_38772_cov_187.740891) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00050_NODE_20..</strong> (original name was: NODE_208_length_10821_cov_177.495056) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00051_NODE_35..</strong> (original name was: NODE_350_length_94051_cov_171.940826) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00052_NODE_28..</strong> (original name was: NODE_281_length_1692_cov_84.157211) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00053_NODE_93..</strong> (original name was: NODE_93_length_9005_cov_172.482849) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00054_NODE_16..</strong> (original name was: NODE_160_length_4386_cov_297.100098) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00055_NODE_11..</strong> (original name was: NODE_115_length_12586_cov_185.839981) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00056_NODE_12..</strong> (original name was: NODE_128_length_16561_cov_208.532516) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00057_NODE_57..</strong> (original name was: NODE_57_length_28044_cov_213.695480) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00058_NODE_24..</strong> (original name was: NODE_242_length_1256_cov_107.253983) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00059_NODE_84..</strong> (original name was: NODE_84_length_1522_cov_155.319321) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00060_NODE_38..</strong> (original name was: NODE_38_length_37517_cov_191.088287) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00061_NODE_20..</strong> (original name was: NODE_200_length_12106_cov_219.710968) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00062_NODE_27..</strong> (original name was: NODE_270_length_3313_cov_169.323578) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00063_NODE_14..</strong> (original name was: NODE_148_length_25643_cov_189.940567) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00064_NODE_31..</strong> (original name was: NODE_316_length_58573_cov_140.370422) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00065_NODE_2_..</strong> (original name was: NODE_2_length_18292_cov_240.483276) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00066_NODE_85..</strong> (original name was: NODE_85_length_18663_cov_176.075226) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00067_NODE_28..</strong> (original name was: NODE_282_length_3945_cov_148.646637) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00068_NODE_41..</strong> (original name was: NODE_41_length_5750_cov_131.665558) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00069_NODE_20..</strong> (original name was: NODE_206_length_1689_cov_219.580811) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00071_NODE_10..</strong> (original name was: NODE_106_length_11885_cov_186.266556) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00072_NODE_31..</strong> (original name was: NODE_317_length_2191_cov_126.881790) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00073_NODE_9_..</strong> (original name was: NODE_9_length_53854_cov_170.543564) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00074_NODE_15..</strong> (original name was: NODE_151_length_50127_cov_184.846695) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00075_NODE_32..</strong> (original name was: NODE_32_length_74496_cov_197.723206) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00076_NODE_33..</strong> (original name was: NODE_339_length_17304_cov_194.319931) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00077_NODE_14..</strong> (original name was: NODE_142_length_21105_cov_200.062164) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00079_NODE_63..</strong> (original name was: NODE_63_length_21099_cov_178.574295) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00080_NODE_48..</strong> (original name was: NODE_48_length_60080_cov_190.110397) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00081_NODE_13..</strong> (original name was: NODE_131_length_5335_cov_705.117310) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00082_NODE_32..</strong> (original name was: NODE_322_length_11041_cov_130.986328) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00083_NODE_25..</strong> (original name was: NODE_25_length_36557_cov_167.749985) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00084_NODE_20..</strong> (original name was: NODE_204_length_8108_cov_204.970398) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00085_NODE_75..</strong> (original name was: NODE_75_length_55985_cov_189.137543) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00086_NODE_16..</strong> (original name was: NODE_168_length_54454_cov_138.765411) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00087_NODE_74..</strong> (original name was: NODE_74_length_38226_cov_194.640671) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00088_NODE_19..</strong> (original name was: NODE_193_length_12092_cov_175.693100) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00089_NODE_23..</strong> (original name was: NODE_230_length_1724_cov_144.174011) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00090_NODE_10..</strong> (original name was: NODE_105_length_22773_cov_132.104599) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00091_NODE_62..</strong> (original name was: NODE_62_length_29696_cov_172.249420) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00092_NODE_15..</strong> (original name was: NODE_158_length_10165_cov_144.378159) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00093_NODE_23..</strong> (original name was: NODE_233_length_43288_cov_160.194122) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00094_NODE_22..</strong> (original name was: NODE_222_length_3211_cov_174.169724) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00095_NODE_87..</strong> (original name was: NODE_87_length_11583_cov_165.422684) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00096_NODE_37..</strong> (original name was: NODE_372_length_1299_cov_145.191681) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00097_NODE_29..</strong> (original name was: NODE_29_length_11574_cov_255.580185) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00098_NODE_27..</strong> (original name was: NODE_27_length_7514_cov_166.064285) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00099_NODE_79..</strong> (original name was: NODE_79_length_1004_cov_1086.673340) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00101_NODE_17..</strong> (original name was: NODE_174_length_5406_cov_161.229553) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00102_NODE_10..</strong> (original name was: NODE_102_length_53212_cov_197.492859) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00103_NODE_27..</strong> (original name was: NODE_276_length_1437_cov_114.169106) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00105_NODE_36..</strong> (original name was: NODE_36_length_73280_cov_163.148758) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00107_NODE_24..</strong> (original name was: NODE_249_length_7863_cov_135.677979) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00108_NODE_31..</strong> (original name was: NODE_31_length_57994_cov_212.035446) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00109_NODE_10..</strong> (original name was: NODE_109_length_7587_cov_174.555298) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00111_NODE_92..</strong> (original name was: NODE_92_length_26193_cov_197.784210) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00112_NODE_15..</strong> (original name was: NODE_153_length_21306_cov_200.556656) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00113_NODE_68..</strong> (original name was: NODE_68_length_8398_cov_160.372345) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00114_NODE_23..</strong> (original name was: NODE_238_length_123611_cov_188.532959) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00115_NODE_16..</strong> (original name was: NODE_162_length_25094_cov_162.416595) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00116_NODE_24..</strong> (original name was: NODE_241_length_1221_cov_153.294846) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00117_NODE_10..</strong> (original name was: NODE_107_length_13278_cov_160.781525) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00118_NODE_17..</strong> (original name was: NODE_172_length_19657_cov_176.754135) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00119_NODE_19..</strong> (original name was: NODE_198_length_23061_cov_162.045059) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00120_NODE_12..</strong> (original name was: NODE_129_length_40626_cov_145.339661) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00122_NODE_95..</strong> (original name was: NODE_95_length_23328_cov_160.695251) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00123_NODE_67..</strong> (original name was: NODE_67_length_65308_cov_175.915924) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00124_NODE_22..</strong> (original name was: NODE_22_length_9634_cov_145.667435) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00125_NODE_23..</strong> (original name was: NODE_237_length_10784_cov_163.221344) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00126_NODE_13..</strong> (original name was: NODE_130_length_10139_cov_135.238190) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00127_NODE_35..</strong> (original name was: NODE_356_length_997_cov_101.018051) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00128_NODE_22..</strong> (original name was: NODE_227_length_2865_cov_136.117981) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00129_NODE_24..</strong> (original name was: NODE_246_length_12834_cov_192.122406) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00130_NODE_10..</strong> (original name was: NODE_10_length_46763_cov_165.152176) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00131_NODE_24..</strong> (original name was: NODE_248_length_8043_cov_155.156784) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00132_NODE_22..</strong> (original name was: NODE_226_length_3854_cov_154.385834) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00133_NODE_15..</strong> (original name was: NODE_154_length_18930_cov_139.854996) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00134_NODE_23..</strong> (original name was: NODE_232_length_79662_cov_147.648270) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00136_NODE_37..</strong> (original name was: NODE_376_length_2612_cov_114.934151) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00138_NODE_14..</strong> (original name was: NODE_141_length_16618_cov_186.445297) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00139_NODE_59..</strong> (original name was: NODE_59_length_16717_cov_158.144760) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00140_NODE_31..</strong> (original name was: NODE_312_length_52362_cov_148.792160) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00141_NODE_42..</strong> (original name was: NODE_427_length_144827_cov_178.664261) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00142_NODE_69..</strong> (original name was: NODE_69_length_51108_cov_150.529160) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00143_NODE_88..</strong> (original name was: NODE_88_length_2796_cov_189.976395) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00144_NODE_30..</strong> (original name was: NODE_302_length_18436_cov_189.158936) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00145_NODE_11..</strong> (original name was: NODE_111_length_17049_cov_215.684555) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00146_NODE_23..</strong> (original name was: NODE_239_length_6389_cov_193.314285) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00148_NODE_24..</strong> (original name was: NODE_247_length_4374_cov_235.518524) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00149_NODE_30..</strong> (original name was: NODE_300_length_1133_cov_276.363647) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00150_NODE_17..</strong> (original name was: NODE_17_length_38748_cov_176.676987) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00151_NODE_34..</strong> (original name was: NODE_34_length_5516_cov_122.964828) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00152_NODE_27..</strong> (original name was: NODE_271_length_4002_cov_137.457764) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00154_NODE_94..</strong> (original name was: NODE_94_length_5975_cov_175.754303) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00155_NODE_12..</strong> (original name was: NODE_120_length_21644_cov_206.503326) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00156_NODE_13..</strong> (original name was: NODE_139_length_28590_cov_187.969330) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00157_NODE_15..</strong> (original name was: NODE_15_length_4820_cov_186.110992) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00158_NODE_80..</strong> (original name was: NODE_80_length_15960_cov_241.178009) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00159_NODE_17..</strong> (original name was: NODE_179_length_17709_cov_201.051605) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00162_NODE_34..</strong> (original name was: NODE_348_length_9390_cov_129.708939) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00163_NODE_3_..</strong> (original name was: NODE_3_length_5440_cov_240.622055) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00164_NODE_20..</strong> (original name was: NODE_203_length_1791_cov_127.723061) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00165_NODE_10..</strong> (original name was: NODE_100_length_11726_cov_142.680450) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00166_NODE_28..</strong> (original name was: NODE_286_length_19936_cov_170.215744) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00167_NODE_11..</strong> (original name was: NODE_114_length_72767_cov_196.782990) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00170_NODE_12..</strong> (original name was: NODE_125_length_22285_cov_163.708282) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00171_NODE_21..</strong> (original name was: NODE_217_length_10073_cov_145.691650) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00172_NODE_97..</strong> (original name was: NODE_97_length_21204_cov_160.076218) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00173_NODE_49..</strong> (original name was: NODE_498_length_13140_cov_148.216736) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00174_NODE_42..</strong> (original name was: NODE_42_length_21157_cov_181.779510) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00177_NODE_30..</strong> (original name was: NODE_305_length_1054_cov_96.184059) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00178_NODE_82..</strong> (original name was: NODE_82_length_119780_cov_194.412689) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00179_NODE_96..</strong> (original name was: NODE_96_length_20914_cov_192.589981) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00180_NODE_15..</strong> (original name was: NODE_152_length_15538_cov_181.648987) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00181_NODE_36..</strong> (original name was: NODE_364_length_64466_cov_136.869202) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00182_NODE_34..</strong> (original name was: NODE_344_length_1088_cov_107.263786) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00183_NODE_11..</strong> (original name was: NODE_112_length_24534_cov_161.316910) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00184_NODE_16..</strong> (original name was: NODE_16_length_58765_cov_195.864517) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00185_NODE_17..</strong> (original name was: NODE_178_length_24929_cov_129.240799) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00186_NODE_13..</strong> (original name was: NODE_13_length_20534_cov_153.967804) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00187_NODE_1_..</strong> (original name was: NODE_1_length_23716_cov_174.450027) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00188_NODE_60..</strong> (original name was: NODE_60_length_1638_cov_169.601349) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00189_NODE_73..</strong> (original name was: NODE_73_length_39641_cov_204.044876) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00191_NODE_12..</strong> (original name was: NODE_12_length_29864_cov_185.106949) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00192_NODE_71..</strong> (original name was: NODE_71_length_7870_cov_197.208511) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00193_NODE_26..</strong> (original name was: NODE_264_length_20820_cov_129.177094) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00194_NODE_15..</strong> (original name was: NODE_155_length_11085_cov_149.632477) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00195_NODE_18..</strong> (original name was: NODE_186_length_3887_cov_161.215073) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00196_NODE_15..</strong> (original name was: NODE_159_length_35439_cov_192.640228) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00197_NODE_20..</strong> (original name was: NODE_207_length_3160_cov_211.009491) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00198_NODE_86..</strong> (original name was: NODE_86_length_20138_cov_171.713974) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00199_NODE_98..</strong> (original name was: NODE_98_length_106983_cov_165.864029) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00200_NODE_21..</strong> (original name was: NODE_216_length_61729_cov_184.597748) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00201_NODE_30..</strong> (original name was: NODE_309_length_1143_cov_216.303589) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00202_NODE_39..</strong> (original name was: NODE_399_length_1575_cov_96.405083) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00203_NODE_15..</strong> (original name was: NODE_157_length_13752_cov_150.681213) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00204_NODE_40..</strong> (original name was: NODE_408_length_12493_cov_164.820938) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00205_NODE_12..</strong> (original name was: NODE_127_length_14772_cov_151.542786) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00206_NODE_64..</strong> (original name was: NODE_64_length_52481_cov_198.566147) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00207_NODE_23..</strong> (original name was: NODE_231_length_2731_cov_104.028564) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00208_NODE_26..</strong> (original name was: NODE_266_length_49965_cov_182.898270) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00209_NODE_31..</strong> (original name was: NODE_310_length_71928_cov_157.313843) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00210_NODE_11..</strong> (original name was: NODE_117_length_16291_cov_166.084213) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00211_NODE_45..</strong> (original name was: NODE_45_length_52482_cov_137.238937) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00212_NODE_26..</strong> (original name was: NODE_267_length_42822_cov_190.779907) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00213_NODE_29..</strong> (original name was: NODE_294_length_1216_cov_126.629112) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00214_NODE_36..</strong> (original name was: NODE_366_length_1510_cov_86.387421) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00217_NODE_91..</strong> (original name was: NODE_91_length_22619_cov_146.596268) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00218_NODE_11..</strong> (original name was: NODE_118_length_65153_cov_150.191605) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00219_NODE_19..</strong> (original name was: NODE_191_length_19095_cov_137.782135) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00220_NODE_13..</strong> (original name was: NODE_132_length_120291_cov_201.849319) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00221_NODE_30..</strong> (original name was: NODE_30_length_26864_cov_146.302444) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00223_NODE_18..</strong> (original name was: NODE_188_length_16308_cov_169.977982) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00224_NODE_15..</strong> (original name was: NODE_156_length_1604_cov_221.970078) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00225_NODE_8_..</strong> (original name was: NODE_8_length_19828_cov_160.358276) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00226_NODE_21..</strong> (original name was: NODE_211_length_6001_cov_157.385773) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00227_NODE_13..</strong> (original name was: NODE_136_length_1532_cov_1139.179443) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00228_NODE_16..</strong> (original name was: NODE_169_length_61076_cov_163.165695) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00229_NODE_76..</strong> (original name was: NODE_76_length_35450_cov_229.873428) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00230_NODE_22..</strong> (original name was: NODE_228_length_2314_cov_175.668106) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00231_NODE_22..</strong> (original name was: NODE_221_length_2404_cov_123.160568) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00232_NODE_18..</strong> (original name was: NODE_184_length_3008_cov_129.975739) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00233_NODE_40..</strong> (original name was: NODE_409_length_54278_cov_141.597198) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00234_NODE_46..</strong> (original name was: NODE_462_length_15491_cov_159.648056) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00235_NODE_43..</strong> (original name was: NODE_43_length_75861_cov_199.864365) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00236_NODE_16..</strong> (original name was: NODE_164_length_10446_cov_187.899490) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00238_NODE_28..</strong> (original name was: NODE_284_length_1940_cov_111.847939) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00239_NODE_37..</strong> (original name was: NODE_37_length_90736_cov_194.300827) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00240_NODE_28..</strong> (original name was: NODE_289_length_28605_cov_192.728516) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00241_NODE_5_..</strong> (original name was: NODE_5_length_38578_cov_138.216293) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00242_NODE_53..</strong> (original name was: NODE_53_length_17934_cov_186.430084) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00243_NODE_10..</strong> (original name was: NODE_108_length_13150_cov_183.457031) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00245_NODE_49..</strong> (original name was: NODE_496_length_4360_cov_107.771561) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00246_NODE_54..</strong> (original name was: NODE_54_length_9872_cov_171.030792) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00248_NODE_24..</strong> (original name was: NODE_244_length_47645_cov_179.298096) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00249_NODE_58..</strong> (original name was: NODE_58_length_26118_cov_183.939728) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00252_NODE_66..</strong> (original name was: NODE_66_length_21604_cov_200.828873) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00253_NODE_11..</strong> (original name was: NODE_110_length_13145_cov_254.147964) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00254_NODE_81..</strong> (original name was: NODE_81_length_14207_cov_133.921661) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00255_NODE_20..</strong> (original name was: NODE_201_length_3291_cov_140.891220) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00256_NODE_12..</strong> (original name was: NODE_126_length_7452_cov_190.375610) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00257_NODE_20..</strong> (original name was: NODE_205_length_31564_cov_162.860062) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00258_NODE_11..</strong> (original name was: NODE_113_length_23691_cov_168.801529) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00260_NODE_77..</strong> (original name was: NODE_77_length_36901_cov_178.868835) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00261_NODE_21..</strong> (original name was: NODE_219_length_19525_cov_151.493271) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00262_NODE_24..</strong> (original name was: NODE_24_length_40887_cov_136.087784) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00263_NODE_10..</strong> (original name was: NODE_103_length_5380_cov_284.215057) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00264_NODE_25..</strong> (original name was: NODE_259_length_2070_cov_152.408218) (Streptomyces rochei CC71)</dd>
       
         <dd><strong>c00265_NODE_20..</strong> (original name was: NODE_20_length_8241_cov_207.865067) (Streptomyces rochei CC71)</dd>
       
     </dl>
     
    </div>
    
    <div class="overview-switches">
     <div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    
   </div>
  </div>
  






<div class="page" id='r17c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00017_NODE_11.. - Region 1 - hglE-KS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-221"></div> <span class="help-tooltip" id="region_svg-help-221">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00017_NODE_11...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r17c1-svg" data-filename="c00017_NODE_11.._r17c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 12,038 nt. (total: 12,038 nt)
        
        <a class="cluster-rules-header" id="r17c1-rules-header" href="#r17c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r17c1-rules">hglE-KS: (hglE or hglD)</div>
      
      <div class="region-svg-container">
       <div id='r17c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-222"></div> <span class="help-tooltip" id="focus-panel-help-222">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r17c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r17c1-clusterblast" data-name="r17c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r17c1-knownclusterblast" data-name="r17c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r17c1-subclusterblast" data-name="r17c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r17c1-MIBiG-cluster-compare" data-name="r17c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r17c1-pfam-details" data-name="r17c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r17c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-1"></div> <span class="help-tooltip" id="cb-general-help-1">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r17c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r17c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r17c1_1.svg">
        NZ_CP039123 (260692-377530): Streptomyces sp. SS52 chromosome, complete genome (23% of genes show similarity), T1PKS,hglE-KS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r17c1_2.svg">
        NZ_CP027297 (282629-492289): Streptomyces sp. SGAir0924 chromosome, complete ... (17% of genes show similarity), NRPS,NRPS-like,T1PKS,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r17c1_3.svg">
        NZ_KB898996 (196360-249325): Streptomyces sp. CNB091 D581DRAFT scaffold00007.... (14% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r17c1_4.svg">
        NZ_MQUH01000030 (148526-201094): Streptomyces sp. M1013 M1013 C30, whole geno... (10% of genes show similarity), hglE-KS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r17c1_5.svg">
        NZ_KZ195574 (8607713-8660285): Streptomyces sp. CS113 scaffold00001, whole ge... (10% of genes show similarity), hglE-KS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r17c1_6.svg">
        NZ_NNBK01000015 (132360-192089): Streptomyces sp. CB01373 scaffold15, whole g... (9% of genes show similarity), NRPS-like,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r17c1_7.svg">
        NZ_CP029342 (2295889-2349406): Streptomyces sp. SM18 chromosome, complete genome (12% of genes show similarity), T1PKS,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r17c1_8.svg">
        NZ_PTJS01000001 (7702304-7755051): Streptomyces sp. QL37 scaffold1 size877290... (13% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r17c1_9.svg">
        NZ_LIPG01000007 (406620-459428): Streptomyces sp. CB02058 scaffold7, whole ge... (10% of genes show similarity), T1PKS,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r17c1_10.svg">
        NZ_FMYR01000006 (292185-345700): Streptomyces sp. AmelKG-A3, whole genome sho... (11% of genes show similarity), T1PKS,hglE-KS
      </option>
      
    </select>
    <button id= "clusterblast-r17c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r17c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r17c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-2"></div> <span class="help-tooltip" id="cb-known-help-2">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r17c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r17c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r17c1_1.svg">
        2&#39;-chloropentostatin / 2&#39;-amino-2&#39;-deoxyadenosine
      </option>
      
      <option value = "svg/knownclusterblast_r17c1_2.svg">
        himastatin
      </option>
      
      <option value = "svg/knownclusterblast_r17c1_3.svg">
        phenalinolactone A
      </option>
      
      <option value = "svg/knownclusterblast_r17c1_4.svg">
        tiancilactone
      </option>
      
    </select>
    <button id="knownclusterblast-r17c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r17c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r17c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-3"></div> <span class="help-tooltip" id="cb-sub-help-3">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r17c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-4"></div> <span class="help-tooltip" id="MIBiG-help-4">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r17c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r17c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r17c1"></div>
    <div id="comparison-MIBiG-r17c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">hglE-KS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>raimonol</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Polyketide</td>
        
        <td>betaenone C, betaenone A</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Polyketide</td>
        
        <td>betaenone A, betaenone B, betaenone C</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Polyketide</td>
        
        <td>esperamicin</td>
        
        
        <td>Actinomadura verrucosospora</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>cyslabdan</td>
        
        
        <td>Streptomyces cyslabdanicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>albaflavenone</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Terpene, Polyketide</td>
        
        <td>pyripyropene A</td>
        
        
        <td>unidentified unclassified sequences.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>viguiepinol</td>
        
        
        <td>Streptomyces sp. KO-3988</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
        
        
        <td>Parastagonospora nodorum SN15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Terpene</td>
        
        <td>(+)-T-muurolol</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r17c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene</td>
      <td>raimonol</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>betaenone C, betaenone A</td>
      <td>Phoma betae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>betaenone A, betaenone B, betaenone C</td>
      <td>Phoma betae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>esperamicin</td>
      <td>Actinomadura verrucosospora</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene</td>
      <td>cyslabdan</td>
      <td>Streptomyces cyslabdanicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>albaflavenone</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene, Polyketide</td>
      <td>pyripyropene A</td>
      <td>unidentified unclassified sequences.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>viguiepinol</td>
      <td>Streptomyces sp. KO-3988</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
      <td>Parastagonospora nodorum SN15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Terpene</td>
      <td>(+)-T-muurolol</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r17c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-5"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-5">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r17c1-pfam-details-svg" data-filename="c00017_NODE_11.._r17c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r17c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r26c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00026_NODE_35.. - Region 1 - T2PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-223"></div> <span class="help-tooltip" id="region_svg-help-223">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00026_NODE_35...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r26c1-svg" data-filename="c00026_NODE_35.._r26c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 11,299 nt. (total: 11,299 nt)
        
        <a class="cluster-rules-header" id="r26c1-rules-header" href="#r26c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r26c1-rules">T2PKS: (t2ks and t2clf)</div>
      
      <div class="region-svg-container">
       <div id='r26c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-224"></div> <span class="help-tooltip" id="focus-panel-help-224">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r26c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r26c1-nrps_pks" data-name="r26c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r26c1-clusterblast" data-name="r26c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r26c1-knownclusterblast" data-name="r26c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r26c1-subclusterblast" data-name="r26c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r26c1-MIBiG-cluster-compare" data-name="r26c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r26c1-pfam-details" data-name="r26c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r26c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-6"></div> <span class="help-tooltip" id="nrps-domain-help-6">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r26c1-details-svg" data-filename="c00026_NODE_35.._r26c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r26c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r26c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-7"></div> <span class="help-tooltip" id="cb-general-help-7">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r26c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r26c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r26c1_1.svg">
        NZ_CP029191 (7559-80048): Streptomyces venezuelae strain ATCC 14585 chromosom... (20% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r26c1_2.svg">
        NZ_CP029193 (2368-74848): Streptomyces venezuelae strain ATCC 14583 chromosom... (20% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r26c1_3.svg">
        NZ_CP029192 (14795-87275): Streptomyces venezuelae strain ATCC 14584 chromoso... (19% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r26c1_4.svg">
        NZ_MTBP01000002 (62814-135282): Actinomadura sp. RB29 Contig 2, whole genome ... (16% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r26c1_5.svg">
        NZ_JAASAG010000006 (278929-351898): Streptomyces sp. HNM0575 scfid6, whole ge... (16% of genes show similarity), NRPS-like,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r26c1_6.svg">
        NZ_LIVP01000011 (73180-164619): Streptomyces sp. MJM1172 scaffold11, whole ge... (13% of genes show similarity), T2PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r26c1_7.svg">
        NC_019673 (6102781-&gt;6188944): Saccharothrix espanaensis DSM 44229, complete g... (13% of genes show similarity), NRPS,T1PKS,T2PKS,lassopeptide,terpene
      </option>
      
      <option value = "svg/clusterblast_r26c1_8.svg">
        NZ_CP046905 (3196271-3267701): Streptomyces sp. QHH-9511 chromosome, complete... (17% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r26c1_9.svg">
        NZ_CP041650 (6216592-6287586): Streptomyces sp. RLB1-8 chromosome, complete g... (18% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r26c1_10.svg">
        NZ_CP041610 (5831568-5902526): Streptomyces sp. RLB3-17 chromosome, complete ... (18% of genes show similarity), T2PKS,butyrolactone
      </option>
      
    </select>
    <button id= "clusterblast-r26c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r26c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r26c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-8"></div> <span class="help-tooltip" id="cb-known-help-8">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r26c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r26c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_1.svg">
        kinamycin
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_2.svg">
        fluostatins M-Q
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_3.svg">
        oviedomycin
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_4.svg">
        oviedomycin
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_5.svg">
        prejadomycin / rabelomycin / gaudimycin C / gaudimycin D / UWM6 / gaudimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_6.svg">
        frigocyclinone
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_7.svg">
        auricin
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_8.svg">
        landomycin A
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_9.svg">
        mayamycin
      </option>
      
      <option value = "svg/knownclusterblast_r26c1_10.svg">
        saquayamycin Z
      </option>
      
    </select>
    <button id="knownclusterblast-r26c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r26c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r26c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-9"></div> <span class="help-tooltip" id="cb-sub-help-9">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r26c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r26c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r26c1_1.svg">
        AF080235_c1: landomycin t2pks (100% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r26c1_2.svg">
        AF324838_c1: simocyclinone t2pks (100% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r26c1_3.svg">
        GU233672_c1: granaticin t2pks (71% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r26c1_4.svg">
        AB103463_c1: medermycin t2pks (75% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r26c1_5.svg">
        FJ483966_c1: polyketomycin t2pks (25% of genes show similarity), t2pks
      </option>
      
    </select>
    <button id="subclusterblast-r26c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r26c1-svg"></div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r26c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-11"></div> <span class="help-tooltip" id="MIBiG-help-11">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r26c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r26c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r26c1"></div>
    <div id="comparison-MIBiG-r26c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T2PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
         
        
        <td>0.58</td>
        <td>Polyketide</td>
        
        <td>griseusin</td>
        
        
        <td>Streptomyces griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.78 (id:0.61, order:0.85, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.78), rgb(0, 0, 0, 0.78) 78%, white 78%)"></td>
         
        
        <td>0.48</td>
        <td>Polyketide</td>
        
        <td>jadomycin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.77 (id:0.52, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
         
        
        <td>0.48</td>
        <td>Polyketide</td>
        
        <td>naphtocyclinone</td>
        
        
        <td>Streptomyces arenae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723.1: 128-10733">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.75 (id:0.57, order:0.77, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
         
        
        <td>0.44</td>
        <td>Polyketide</td>
        
        <td>oviedomycin</td>
        
        
        <td>Streptomyces antibioticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000236.1: 35-34224">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000236/index.html#r1c1" target="_new">BGC0000236.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.59, order:1.00, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.37</td>
        <td>Polyketide</td>
        
        <td>kinamycin</td>
        
        
        <td>Streptomyces murayamaensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001596.1: 5974061-6020749">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001596/index.html#r1c1" target="_new">BGC0001596.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.58, order:0.95, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Polyketide</td>
        
        <td>fluostatins M-Q</td>
        
        
        <td>Streptomyces albus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000235.1: 118281-139606">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000235/index.html#r1c1" target="_new">BGC0000235.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.56, order:0.70, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>kiamycin</td>
        
        
        <td>Streptomyces sp. W007</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000253.1: 1-24720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000253/index.html#r1c1" target="_new">BGC0000253.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.57, order:0.71, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Polyketide</td>
        
        <td>oviedomycin</td>
        
        
        <td>Streptomyces antibioticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.52, order:0.46, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Polyketide</td>
        
        <td>granaticin</td>
        
        
        <td>Streptomyces violaceoruber</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002016.1: 2881710-2910119">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002016/index.html#r1c1" target="_new">BGC0002016.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.56, order:0.70, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Polyketide</td>
        
        <td>lugdunomycin</td>
        
        
        <td>Streptomyces sp. QL37</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r26c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
      <td>0.83</td>
      <td>Polyketide</td>
      <td>griseusin</td>
      <td>Streptomyces griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.78 (id:0.61, order:0.85, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.78), rgb(0, 0, 0, 0.78) 78%, white 78%)"></td>
      <td>0.78</td>
      <td>Polyketide</td>
      <td>jadomycin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.77 (id:0.52, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
      <td>0.77</td>
      <td>Polyketide</td>
      <td>naphtocyclinone</td>
      <td>Streptomyces arenae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723.1: 128-10733">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.75 (id:0.57, order:0.77, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
      <td>0.75</td>
      <td>Polyketide</td>
      <td>oviedomycin</td>
      <td>Streptomyces antibioticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000236.1: 35-34224">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000236/index.html#r1c1" target="_new">BGC0000236.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.59, order:1.00, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>Polyketide</td>
      <td>kinamycin</td>
      <td>Streptomyces murayamaensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001596.1: 5974061-6020749">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001596/index.html#r1c1" target="_new">BGC0001596.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.58, order:0.95, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Polyketide</td>
      <td>fluostatins M-Q</td>
      <td>Streptomyces albus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000235.1: 118281-139606">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000235/index.html#r1c1" target="_new">BGC0000235.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.56, order:0.70, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>kiamycin</td>
      <td>Streptomyces sp. W007</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000253.1: 1-24720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000253/index.html#r1c1" target="_new">BGC0000253.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.57, order:0.71, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>oviedomycin</td>
      <td>Streptomyces antibioticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.52, order:0.46, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>granaticin</td>
      <td>Streptomyces violaceoruber</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002016.1: 2881710-2910119">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002016/index.html#r1c1" target="_new">BGC0002016.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.56, order:0.70, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>lugdunomycin</td>
      <td>Streptomyces sp. QL37</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r26c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-12"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-12">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r26c1-pfam-details-svg" data-filename="c00026_NODE_35.._r26c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r26c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
      <div class="sidepanel-details-header r26c1-t2pks" data-name="r26c1-t2pks">
        Type II PKS
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    <div class="sidepanel-details-section r26c1-t2pks">
      <div class="more-details">
    <div class="heading">
      <span>Type II PKS gene cluster prediction details</span>
      <div class="help-container"> <div class="help-icon" data-id="t2pks-help-10"></div> <span class="help-tooltip" id="t2pks-help-10">Predictions of starter units, elongations, product classes, and potential molecular weights for type II PKS clusters.<br>More detailed information is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/t2pks' target='_blank'>here</a>.</span></div>
    </div>
	
	
	<dl class="prediction-text">
        <dt><strong>Cluster section:</strong></dt>
        <dd>0 to 11299</dd>
		
		<dt><strong>Starter unit(s):</strong></dt>
		<dd>
				acetyl-CoA (Score: 0.0; E-value: 0)<br>
			
	    <dd>
		</tr>
		
		
		<dt><strong>Malonyl elongations:</strong></dt>
        <dd>
			
			8|9 (Score: 704.2; E-value: 9.7e-215)<br>
			
    	</dd>
		
		
		<dt><strong>Product class(es):</strong></td>
        <dd>
				
				angucycline<br>
				
		</dd>
		
		
		<dt><strong>Product molecular weight(s):</strong></dt>
		<dd>
			
			acetyl-CoA_8: 381.35 Da<br>
			
			acetyl-CoA_9: 423.39 Da<br>
			
		</dd>
		
	</dl>
	<dl class="prediction-text">
		
		<dt><strong>cds3222_3552</strong></dt>
			
			<dd>
				CYC
				
					C4-C17/C2-C19
				
				(Score: 199.8; E-value: 9.5e-62)<br>
			</dd>
			
		
		<dt><strong>cds3548_4820</strong></dt>
			
			<dd>
				KS
				
				(Score: 779.7; E-value: 1.4e-237)<br>
			</dd>
			
		
		<dt><strong>cds4816_6028</strong></dt>
			
			<dd>
				CLF
				
					8|9
				
				(Score: 704.2; E-value: 9.7e-215)<br>
			</dd>
			
		
		<dt><strong>cds6094_6364</strong></dt>
			
			<dd>
				ACP
				
				(Score: 126.1; E-value: 9.2e-40)<br>
			</dd>
			
		
		<dt><strong>cds6376_7165</strong></dt>
			
			<dd>
				KR
				
					C9
				
				(Score: 469.3; E-value: 7.3e-144)<br>
			</dd>
			
		
		<dt><strong>cds7205_8150</strong></dt>
			
			<dd>
				CYC
				
					C7-C12
				
				(Score: 472.8; E-value: 8.8e-145)<br>
			</dd>
			
		
		<dt><strong>cds739_2212</strong></dt>
			
			<dd>
				OXY
				
				(Score: 512.5; E-value: 2.7e-156)<br>
			</dd>
			
		
		<dt><strong>cds8153_9629</strong></dt>
			
			<dd>
				OXY
				
				(Score: 583.3; E-value: 9.5e-178)<br>
			</dd>
			
		
		<dt><strong>cds9731_11246</strong></dt>
			
			<dd>
				OXY
				
				(Score: 573.6; E-value: 7.8e-175)<br>
			</dd>
			
		
	</dl>
    
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r27c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00027_NODE_72.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-225"></div> <span class="help-tooltip" id="region_svg-help-225">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00027_NODE_72...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r27c1-svg" data-filename="c00027_NODE_72.._r27c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 19,619 - 34,331 nt. (total: 14,713 nt)
        
        <a class="cluster-rules-header" id="r27c1-rules-header" href="#r27c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r27c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r27c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-226"></div> <span class="help-tooltip" id="focus-panel-help-226">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r27c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r27c1-clusterblast" data-name="r27c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r27c1-knownclusterblast" data-name="r27c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r27c1-subclusterblast" data-name="r27c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r27c1-MIBiG-cluster-compare" data-name="r27c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r27c1-pfam-details" data-name="r27c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r27c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-13"></div> <span class="help-tooltip" id="cb-general-help-13">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r27c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r27c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r27c1_1.svg">
        NZ_KZ195574 (2987642-3008355): Streptomyces sp. CS113 scaffold00001, whole ge... (73% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r27c1_2.svg">
        NZ_MQUH01000016 (913268-934354): Streptomyces sp. M1013 M1013 C16, whole geno... (73% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r27c1_3.svg">
        NZ_LT963352 (3038615-3059139): Streptomyces chartreusis NRRL 3882 isolate NRR... (73% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r27c1_4.svg">
        NZ_CP026121 (5743113-5763801): Streptomyces sp. Go-475 chromosome, complete g... (73% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r27c1_5.svg">
        NZ_KL585964 (41932-63018): Streptomyces sp. NRRL S-474 P279 Doro1 scaffold19,... (87% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r27c1_6.svg">
        NZ_JACHXE010000002 (199442-220528): Streptomyces violarus strain CECT 3237 Ga... (93% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r27c1_7.svg">
        NZ_JACHCX010000014 (4003-25089): Streptomyces sp. AK010 Ga0401617 14, whole g... (73% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r27c1_8.svg">
        NC_021985 (5642201-5662349): Streptomyces collinus Tu 365, complete sequence (77% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r27c1_9.svg">
        NZ_SLZB01000005 (278342-299356): Streptomyces sp. BK308 Ga0307715 105, whole ... (73% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r27c1_10.svg">
        NZ_SOAR01000002 (792404-813418): Streptomyces sp. BK141 Ga0307717 102, whole ... (77% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r27c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r27c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r27c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-14"></div> <span class="help-tooltip" id="cb-known-help-14">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r27c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r27c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r27c1_1.svg">
        albaflavenone
      </option>
      
      <option value = "svg/knownclusterblast_r27c1_2.svg">
        julichrome Q3-3 / julichrome Q3-5
      </option>
      
    </select>
    <button id="knownclusterblast-r27c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r27c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r27c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-15"></div> <span class="help-tooltip" id="cb-sub-help-15">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r27c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-16"></div> <span class="help-tooltip" id="MIBiG-help-16">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r27c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r27c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r27c1"></div>
    <div id="comparison-MIBiG-r27c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.89 (id:0.71, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.89), rgb(0, 0, 0, 0.89) 89%, white 89%)"></td>
         
        
        <td>0.71</td>
        <td>Terpene</td>
        
        <td>albaflavenone</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001201.1: 11915724-11918015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001201/index.html#r1c1" target="_new">BGC0001201.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Terpene</td>
        
        <td>eremophilene</td>
        
        
        <td>Sorangium cellulosum So ce56</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>Terpene</td>
        
        <td>raimonol</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000674.1: 74961-79952">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000674/index.html#r1c1" target="_new">BGC0000674.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>Terpene</td>
        
        <td>(-)-Î´-cadinene</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Terpene</td>
        
        <td>(+)-T-muurolol</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>cyslabdan</td>
        
        
        <td>Streptomyces cyslabdanicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000207.1: 821-16213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000207/index.html#r1c1" target="_new">BGC0000207.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>chartreusin</td>
        
        
        <td>Streptomyces chartreusis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002013.1: 235087-246792">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002013/index.html#r1c1" target="_new">BGC0002013.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>RiPP</td>
        
        <td>curacozole</td>
        
        
        <td>Streptomyces curacoi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650.1: 28-7720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.17</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Algoriphagus sp. KK10202C</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:0.39, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Terpene</td>
        
        <td>pentalenolactone</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r27c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.89 (id:0.71, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.89), rgb(0, 0, 0, 0.89) 89%, white 89%)"></td>
      <td>0.89</td>
      <td>Terpene</td>
      <td>albaflavenone</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001201.1: 11915724-11918015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001201/index.html#r1c1" target="_new">BGC0001201.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>eremophilene</td>
      <td>Sorangium cellulosum So ce56</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Terpene</td>
      <td>raimonol</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000674.1: 74961-79952">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000674/index.html#r1c1" target="_new">BGC0000674.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene</td>
      <td>(-)-Î´-cadinene</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene</td>
      <td>(+)-T-muurolol</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>cyslabdan</td>
      <td>Streptomyces cyslabdanicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000207.1: 821-16213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000207/index.html#r1c1" target="_new">BGC0000207.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>chartreusin</td>
      <td>Streptomyces chartreusis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002013.1: 235087-246792">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002013/index.html#r1c1" target="_new">BGC0002013.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>RiPP</td>
      <td>curacozole</td>
      <td>Streptomyces curacoi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650.1: 28-7720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Algoriphagus sp. KK10202C</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:0.39, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r27c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-17"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-17">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r27c1-pfam-details-svg" data-filename="c00027_NODE_72.._r27c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r27c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r32c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00032_NODE_36.. - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-227"></div> <span class="help-tooltip" id="region_svg-help-227">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00032_NODE_36...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r32c1-svg" data-filename="c00032_NODE_36.._r32c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 37,583 - 110,872 nt. (total: 73,290 nt)
        
        <a class="cluster-rules-header" id="r32c1-rules-header" href="#r32c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r32c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r32c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-228"></div> <span class="help-tooltip" id="focus-panel-help-228">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r32c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r32c1-nrps_pks" data-name="r32c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r32c1-clusterblast" data-name="r32c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r32c1-knownclusterblast" data-name="r32c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r32c1-subclusterblast" data-name="r32c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r32c1-MIBiG-cluster-compare" data-name="r32c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r32c1-pfam-details" data-name="r32c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r32c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-18"></div> <span class="help-tooltip" id="nrps-domain-help-18">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r32c1-details-svg" data-filename="c00032_NODE_36.._r32c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r32c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r32c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-21"></div> <span class="help-tooltip" id="cb-general-help-21">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r32c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r32c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r32c1_1.svg">
        NZ_CP027297 (7044671-7137479): Streptomyces sp. SGAir0924 chromosome, complet... (47% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r32c1_2.svg">
        NZ_CP039123 (7125674-7220676): Streptomyces sp. SS52 chromosome, complete genome (42% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r32c1_3.svg">
        NZ_CP042278 (655292-744828): Streptomyces sp. WAC06273 strain WAC6273 substr.... (68% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r32c1_4.svg">
        NZ_CP053109 (702058-782269): Streptomyces sp. Z423-1 chromosome (25% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r32c1_5.svg">
        NZ_CP015849 (6566127-6664649): Streptomyces sp. SAT1 chromosome, complete genome (26% of genes show similarity), T1PKS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r32c1_6.svg">
        NZ_KZ819182 (5051008-5146512): Streptomyces sp. CS147 scaffold00001, whole ge... (29% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r32c1_7.svg">
        NZ_CP029541 (982476-1223386): Streptomyces sp. NEAU-S7GS2 chromosome, complet... (27% of genes show similarity), NRPS-like,PKS-like,T1PKS,oligosaccharide,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r32c1_8.svg">
        NZ_WWIP01000001 (6090667-6194339): Streptomyces sp. SID5466 SID5466.c1, whole... (30% of genes show similarity), T1PKS,oligosaccharide,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r32c1_9.svg">
        NZ_AP019620 (1767528-1841875): Streptomyces antimycoticus strain NBRC 100767 (29% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r32c1_10.svg">
        NZ_CP021744 (1819585-2023440): Streptomyces albireticuli strain MDJK11 chromo... (21% of genes show similarity), NRPS,T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r32c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r32c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r32c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-22"></div> <span class="help-tooltip" id="cb-known-help-22">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r32c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r32c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_1.svg">
        elaiophylin
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_2.svg">
        incednine
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_3.svg">
        vicenistatin
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_4.svg">
        mediomycin A
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_5.svg">
        mediomiycin A
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_6.svg">
        BE-14106
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_7.svg">
        lobosamide A / lobosamide B / lobosamide C
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_8.svg">
        auroramycin
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_9.svg">
        sipanmycin
      </option>
      
      <option value = "svg/knownclusterblast_r32c1_10.svg">
        JBIR-100
      </option>
      
    </select>
    <button id="knownclusterblast-r32c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r32c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r32c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-23"></div> <span class="help-tooltip" id="cb-sub-help-23">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r32c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r32c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r32c1_1.svg">
        AB032523_c1: avermectin oleandrose (37% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_2.svg">
        AF497482_c3: calicheamicin 4 nucleotide sugars (16% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_3.svg">
        DQ297453_c2: AT2433 aminopentose (25% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_4.svg">
        FJ483966_c3: polyketomycin beta-D-amicetose / alpha-L-axenose (18% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_5.svg">
        AB088119_c2: staurosporine deoxysugar (20% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_6.svg">
        AF333038_c2: avilamycin D-olivose, 2-deoxy-D-evalose, 4-O-methyl-D-fucose (11% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_7.svg">
        AY048670_c1: C-1027 deoxy aminosugar (28% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_8.svg">
        DQ149987_c2: concanamycin 4&#39;-O-carbamoyl-2&#39;-deoxyrhamnose (28% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_9.svg">
        AF080235_c2: landomycin D-olivose / L-rhodinose (16% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r32c1_10.svg">
        AB103463_c2: medermycin aminodeoxysugar (angolosamine) (28% of genes show similarity), sugar
      </option>
      
    </select>
    <button id="subclusterblast-r32c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r32c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r32c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-24"></div> <span class="help-tooltip" id="MIBiG-help-24">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r32c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r32c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r32c1"></div>
    <div id="comparison-MIBiG-r32c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Polyketide</td>
        
        <td>(-)-Mellein</td>
        
        
        <td>Parastagonospora nodorum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>Polyketide</td>
        
        <td>6-methylsalicyclic acid</td>
        
        
        <td>Aspergillus terreus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Polyketide</td>
        
        <td>asperlactone</td>
        
        
        <td>Aspergillus ochraceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Polyketide</td>
        
        <td>strobilurin</td>
        
        
        <td>Strobilurus tenacellus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000068.1: 1-27296">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000068/index.html#r1c1" target="_new">BGC0000068.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.54, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Polyketide</td>
        
        <td>geldanamycin</td>
        
        
        <td>Streptomyces hygroscopicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000263.1: 0-32801">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000263/index.html#r1c1" target="_new">BGC0000263.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.54, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Polyketide</td>
        
        <td>ravidomycin</td>
        
        
        <td>Streptomyces ravidus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.52, order:0.75, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Polyketide</td>
        
        <td>oronofacic acid</td>
        
        
        <td>Pseudomonas syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001235.1: 53-50000">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001235/index.html#r1c1" target="_new">BGC0001235.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.57, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>NRP, Polyketide</td>
        
        <td>antalid</td>
        
        
        <td>Polyangium spumosum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000353.1: 60-28528">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000353/index.html#r1c1" target="_new">BGC0000353.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.59, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>tacrolimus</td>
        
        
        <td>Streptomyces sp. MA6548</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000226.1: 798-33394">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000226/index.html#r1c1" target="_new">BGC0000226.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.54, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Polyketide</td>
        
        <td>gilvocarcin V</td>
        
        
        <td>Streptomyces griseoflavus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r32c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Polyketide</td>
      <td>(-)-Mellein</td>
      <td>Parastagonospora nodorum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Aspergillus terreus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Polyketide</td>
      <td>asperlactone</td>
      <td>Aspergillus ochraceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Polyketide</td>
      <td>strobilurin</td>
      <td>Strobilurus tenacellus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000068.1: 1-27296">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000068/index.html#r1c1" target="_new">BGC0000068.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.54, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Polyketide</td>
      <td>geldanamycin</td>
      <td>Streptomyces hygroscopicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000263.1: 0-32801">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000263/index.html#r1c1" target="_new">BGC0000263.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.54, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>ravidomycin</td>
      <td>Streptomyces ravidus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.52, order:0.75, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>oronofacic acid</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001235.1: 53-50000">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001235/index.html#r1c1" target="_new">BGC0001235.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.57, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP, Polyketide</td>
      <td>antalid</td>
      <td>Polyangium spumosum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000353.1: 60-28528">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000353/index.html#r1c1" target="_new">BGC0000353.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.59, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>tacrolimus</td>
      <td>Streptomyces sp. MA6548</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000226.1: 798-33394">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000226/index.html#r1c1" target="_new">BGC0000226.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.54, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>gilvocarcin V</td>
      <td>Streptomyces griseoflavus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r32c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-25"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-25">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r32c1-pfam-details-svg" data-filename="c00032_NODE_36.._r32c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r32c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r32c1-nrps_pks_products" data-name="r32c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r32c1-nrps_pks" data-name="r32c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r32c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-19"></div> <span class="help-tooltip" id="nrps-structures-help-19">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 37582 - 110872:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r32c1-1" class="smiles-canvas" data-smiles="C=CC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ccmal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r32c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C=CC(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r32c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-20"></div> <span class="help-tooltip" id="nrps-monomers-help-20">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds57582_66801</span></strong>: mal - mal
      <div class="collapser collapser-target-cds57582_66801 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (722..1016)</strong>: mal
          <div class="collapser collapser-target-cds57582_66801 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds57582_66801-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 91.7%</dd>
<dd>inactive: 83.3%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds57582_66801-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 219.4</dt>

  <dd></dd><dt>inactive: 113.4</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 75.7</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 66.4</dt>

  <dd></dd><dt>Isobutyryl-CoA: 41.6</dt>

  <dd></dd><dt>Propionyl-CoA: 38.6</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 29.7</dt>

  <dd></dd><dt>CHC-CoA: 22.2</dt>

  <dd></dd><dt>Acetyl-CoA: 18.5</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 16.9</dt>

  <dd></dd><dt>trans-1,2-CPDA: 14.9</dt>

  <dd></dd><dt>Benzoyl-CoA: 13.8</dt>

  <dd></dd><dt>fatty_acid: 8.5</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (2541..2784)</strong>: mal
          <div class="collapser collapser-target-cds57582_66801 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds57582_66801-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 83.3%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds57582_66801-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 192.6</dt>

  <dd></dd><dt>inactive: 103.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 65.9</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 44.2</dt>

  <dd></dd><dt>Propionyl-CoA: 21.8</dt>

  <dd></dd><dt>Isobutyryl-CoA: 19.8</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 18.5</dt>

  <dd></dd><dt>Benzoyl-CoA: 9.2</dt>

  <dd></dd><dt>CHC-CoA: 8.5</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 7.5</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: -1.7</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds70453_73456</span></strong>: mal
      <div class="collapser collapser-target-cds70453_73456 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (0..263)</strong>: mal
          <div class="collapser collapser-target-cds70453_73456 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds70453_73456-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 91.7%</dd>
<dd>inactive: 79.2%</dd>
<dd>2-Rhyd-Malonyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds70453_73456-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 211.9</dt>

  <dd></dd><dt>inactive: 111.7</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 71.8</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 49.8</dt>

  <dd></dd><dt>Propionyl-CoA: 28.9</dt>

  <dd></dd><dt>Isobutyryl-CoA: 26.6</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 14.3</dt>

  <dd></dd><dt>Benzoyl-CoA: 9.3</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 8.6</dt>

  <dd></dd><dt>CHC-CoA: 8.3</dt>

  <dd></dd><dt>fatty_acid: 5.4</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: -0.9</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds78538_81673</span></strong>: mal
      <div class="collapser collapser-target-cds78538_81673 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (867..1033)</strong>: mal
          <div class="collapser collapser-target-cds78538_81673 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds78538_81673-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 62.5%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds78538_81673-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 115.1</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 44.2</dt>

  <dd></dd><dt>inactive: 40.0</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 33.9</dt>

  <dd></dd><dt>Isobutyryl-CoA: 16.9</dt>

  <dd></dd><dt>Propionyl-CoA: 16.1</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 12.3</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 10.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 9.0</dt>

  <dd></dd><dt>CHC-CoA: 8.5</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: -1.1</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds107430_108987</span></strong>: X
      <div class="collapser collapser-target-cds107430_108987 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (18..418)</strong>: X
          <div class="collapser collapser-target-cds107430_108987 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds107430_108987-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DcFFFalVtK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ile</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds109010_109973</span></strong>: pk
      <div class="collapser collapser-target-cds109010_109973 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (15..290)</strong>: pk
          <div class="collapser collapser-target-cds109010_109973 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  (unknown)
              <div class="collapser collapser-target-cds109010_109973-ATSignature collapser-level-none"> </div><div class="collapser-content">
                No matches above 50%
              </div><br>
          
              minowa:  trans-1,2-CPDA
              <div class="collapser collapser-target-cds109010_109973-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>inactive: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>Propionyl-CoA: 0.0</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 0.0</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 0.0</dt>

  <dd></dd><dt>Malonyl-CoA: 0.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r33c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00033_NODE_50.. - Region 1 - NRPS,NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-229"></div> <span class="help-tooltip" id="region_svg-help-229">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00033_NODE_50...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r33c1-svg" data-filename="c00033_NODE_50.._r33c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 48,358 nt. (total: 48,358 nt)
        
        <a class="cluster-rules-header" id="r33c1-rules-header" href="#r33c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r33c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r33c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-230"></div> <span class="help-tooltip" id="focus-panel-help-230">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r33c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r33c1-nrps_pks" data-name="r33c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r33c1-clusterblast" data-name="r33c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r33c1-knownclusterblast" data-name="r33c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r33c1-subclusterblast" data-name="r33c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r33c1-MIBiG-cluster-compare" data-name="r33c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r33c1-pfam-details" data-name="r33c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r33c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-26"></div> <span class="help-tooltip" id="nrps-domain-help-26">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r33c1-details-svg" data-filename="c00033_NODE_50.._r33c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r33c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r33c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-29"></div> <span class="help-tooltip" id="cb-general-help-29">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r33c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r33c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r33c1_1.svg">
        NZ_CP027297 (282629-492289): Streptomyces sp. SGAir0924 chromosome, complete ... (41% of genes show similarity), NRPS,NRPS-like,T1PKS,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r33c1_2.svg">
        NZ_CP042278 (117268-184951): Streptomyces sp. WAC06273 strain WAC6273 substr.... (50% of genes show similarity), NRPS,NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r33c1_3.svg">
        NC_020990 (6567851-6838639): Streptomyces albidoflavus, complete sequence (30% of genes show similarity), NRPS,NRPS-like,T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r33c1_4.svg">
        NZ_CM002271 (6545259-6848841): Streptomyces sp. GBA 94-10 chromosome, whole g... (23% of genes show similarity), NRPS,NRPS-like,T1PKS,T3PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r33c1_5.svg">
        NZ_CP047147 (63289-333437): Streptomyces sp. GF20 chromosome, complete genome (30% of genes show similarity), NRPS,NRPS-like,T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r33c1_6.svg">
        NZ_CM002273 (6555444-6843909): Streptomyces sp. PVA 94-07 strain PVA 94-07 ch... (26% of genes show similarity), NRPS,NRPS-like,T1PKS,T3PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r33c1_7.svg">
        NZ_CP016824 (54347-384189): Streptomyces sampsonii strain KJ40 chromosome, co... (20% of genes show similarity), NRPS,NRPS-like,T1PKS,T2PKS,butyrolactone,ectoine,lanthipeptide,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r33c1_8.svg">
        NZ_CP031742 (6968727-7175511): Streptomyces koyangensis strain VK-A60T chromo... (33% of genes show similarity), NRPS,NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r33c1_9.svg">
        NZ_ML123129 (239463-453412): Streptomyces sp. ADI98-12 scaffold004, whole gen... (39% of genes show similarity), NRPS,NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r33c1_10.svg">
        NZ_CP026490 (191791-441920): Streptomyces sp. 604F chromosome, complete genome (25% of genes show similarity), NRPS,NRPS-like,T1PKS,T3PKS,lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r33c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r33c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r33c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-30"></div> <span class="help-tooltip" id="cb-known-help-30">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r33c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r33c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r33c1_1.svg">
        candicidin
      </option>
      
      <option value = "svg/knownclusterblast_r33c1_2.svg">
        levorin A3 / C06690 / FR-008-III / candicidin A / UNII-AP5PEF5W7U
      </option>
      
      <option value = "svg/knownclusterblast_r33c1_3.svg">
        67-121C
      </option>
      
      <option value = "svg/knownclusterblast_r33c1_4.svg">
        nystatin A1
      </option>
      
      <option value = "svg/knownclusterblast_r33c1_5.svg">
        antimycin
      </option>
      
      <option value = "svg/knownclusterblast_r33c1_6.svg">
        splenocin
      </option>
      
      <option value = "svg/knownclusterblast_r33c1_7.svg">
        antimycin
      </option>
      
      <option value = "svg/knownclusterblast_r33c1_8.svg">
        nystatin
      </option>
      
      <option value = "svg/knownclusterblast_r33c1_9.svg">
        natamycin
      </option>
      
      <option value = "svg/knownclusterblast_r33c1_10.svg">
        natamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r33c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r33c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r33c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-31"></div> <span class="help-tooltip" id="cb-sub-help-31">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r33c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-32"></div> <span class="help-tooltip" id="MIBiG-help-32">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r33c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r33c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r33c1"></div>
    <div id="comparison-MIBiG-r33c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001544.1: 576137-603864">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001544/index.html#r1c1" target="_new">BGC0001544.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.96</td>
        <td>NRP, Polyketide</td>
        
        <td>chlorflavonin</td>
        
        
        <td>Aspergillus campestris IBT 28561</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.91</td>
        <td>Polyketide</td>
        
        <td>depudecin</td>
        
        
        <td>Alternaria brassicicola</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.88</td>
        <td>Polyketide</td>
        
        <td>betaenone C, betaenone A</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.88</td>
        <td>Polyketide</td>
        
        <td>betaenone A, betaenone B, betaenone C</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001141.1: 262-37660">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001141/index.html#r1c1" target="_new">BGC0001141.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.82</td>
        <td>Polyketide</td>
        
        <td>4-epi-15-epi-brefeldin A</td>
        
        
        <td>Penicillium brefeldianum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000122.1: 1816-33971">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000122/index.html#r1c1" target="_new">BGC0000122.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.50, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.50, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.81</td>
        <td>Polyketide</td>
        
        <td>phenylnannolone A, phenylnannolone B, phenylnannolone C</td>
        
        
        <td>Nannocystis pusilla</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001455.1: 0-26743">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001455/index.html#r1c1" target="_new">BGC0001455.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.64, order:0.46, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.64, order:0.52, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.44, order:0.31, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.44, order:0.31, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
        <td>0.81</td>
        <td>NRP, Polyketide</td>
        
        <td>antimycin</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000958.1: 81952-106578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000958/index.html#r1c1" target="_new">BGC0000958.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.58, order:0.48, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.58, order:0.52, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
        <td>0.81</td>
        <td>NRP, Polyketide</td>
        
        <td>antimycin</td>
        
        
        <td>Streptomyces sp. S4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001216.1: 6355-31635">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001216/index.html#r1c1" target="_new">BGC0001216.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.58, order:0.46, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.58, order:0.52, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
        <td>0.81</td>
        <td>NRP, Polyketide</td>
        
        <td>splenocin</td>
        
        
        <td>Streptomyces sp. CNQ431</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000043.1: 0-13097">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000043/index.html#r1c1" target="_new">BGC0000043.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.47, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.47, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.80</td>
        <td>Polyketide</td>
        
        <td>curacin A</td>
        
        
        <td>Moorea producens 19L</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r33c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001455.1: 0-26743">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001455/index.html#r1c1" target="_new">BGC0001455.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.64, order:0.46, components:0.97)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP, Polyketide</td>
      <td>antimycin</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002058.1: 29690-42292">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002058/index.html#r1c1" target="_new">BGC0002058.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Other</td>
      <td>Reutericyclin A, Reutericyclin B, Reutericyclin C, Reutericyclin D</td>
      <td>Streptococcus mutans B04Sm5</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000958.1: 81952-106578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000958/index.html#r1c1" target="_new">BGC0000958.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.58, order:0.48, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP, Polyketide</td>
      <td>antimycin</td>
      <td>Streptomyces sp. S4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001216.1: 6355-31635">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001216/index.html#r1c1" target="_new">BGC0001216.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.58, order:0.46, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP, Polyketide</td>
      <td>splenocin</td>
      <td>Streptomyces sp. CNQ431</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001544.1: 576137-603864">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001544/index.html#r1c1" target="_new">BGC0001544.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP, Polyketide</td>
      <td>chlorflavonin</td>
      <td>Aspergillus campestris IBT 28561</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000220.1: 512-21855">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000220/index.html#r1c1" target="_new">BGC0000220.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>enterocin</td>
      <td>Streptomyces maritimus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002012.1: 8348-34051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002012/index.html#r1c1" target="_new">BGC0002012.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>julichrome Q3-3, julichrome Q3-5</td>
      <td>Streptomyces afghaniensis 772</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>rhodochelin</td>
      <td>Rhodococcus jostii RHA1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.47, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Alkaloid</td>
      <td>kitasetaline</td>
      <td>Kitasatospora setae KM-6054</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r33c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-33"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-33">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r33c1-pfam-details-svg" data-filename="c00033_NODE_50.._r33c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r33c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r33c1-nrps_pks_products" data-name="r33c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r33c1-nrps_pks" data-name="r33c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r33c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-27"></div> <span class="help-tooltip" id="nrps-structures-help-27">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 48358:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r33c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)CC(=O)NC(C(O)C)C(=O)CC(=O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X - mal) + (thr) + (mal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,mal&amp;nrps2=thr&amp;nrps3=mal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,mal*&amp;nrps2=thr*&amp;nrps3=mal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r33c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)CC(=O)NC(C(O)C)C(=O)CC(=O)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 0 - 34863:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r33c1-2" class="smiles-canvas" data-smiles="NC(C(O)C)C(=O)CC(=O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(thr) + (mal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr&amp;nrps2=mal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*&amp;nrps2=mal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r33c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(C(O)C)C(=O)CC(=O)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 18704 - 48358:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r33c1-3" class="smiles-canvas" data-smiles="NC([*])C(=O)CC(=O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X - mal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,mal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,mal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r33c1-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)CC(=O)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r33c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-28"></div> <span class="help-tooltip" id="nrps-monomers-help-28">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds2241_6237</span></strong>: mal
      <div class="collapser collapser-target-cds2241_6237 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (594..877)</strong>: mal
          <div class="collapser collapser-target-cds2241_6237 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds2241_6237-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 62.5%</dd>
<dd>2-Rhyd-Malonyl-CoA: 54.2%</dd>
<dd>inactive: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds2241_6237-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 100.6</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 61.3</dt>

  <dd></dd><dt>inactive: 50.0</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 41.3</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 25.9</dt>

  <dd></dd><dt>Isobutyryl-CoA: 19.1</dt>

  <dd></dd><dt>Propionyl-CoA: 18.3</dt>

  <dd></dd><dt>fatty_acid: 8.9</dt>

  <dd></dd><dt>Benzoyl-CoA: 2.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds6229_14863</span></strong>: thr - pk
      <div class="collapser collapser-target-cds6229_14863 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (461..874)</strong>: thr
          <div class="collapser collapser-target-cds6229_14863 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds6229_14863-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWSVGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>CAL_domain (1707..2116)</strong>: pk
          <div class="collapser collapser-target-cds6229_14863 collapser-level-candidate"> </div><div class="collapser-content">
          
              minowa:  fatty_acid
              <div class="collapser collapser-target-cds6229_14863-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>fatty_acid: 15.2</dt>

  <dd></dd><dt>AHBA: 0.6</dt>

  <dd></dd><dt>shikimic_acid: 0.0</dt>

  <dd></dd><dt>NH2: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: -2.8</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds38704_44065</span></strong>: X - mal
      <div class="collapser collapser-target-cds38704_44065 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (29..439)</strong>: X
          <div class="collapser collapser-target-cds38704_44065 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds38704_44065-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">cFQvLaVngK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pip</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (1222..1523)</strong>: mal
          <div class="collapser collapser-target-cds38704_44065 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds38704_44065-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 79.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds38704_44065-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 187.4</dt>

  <dd></dd><dt>inactive: 81.7</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 58.7</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 53.9</dt>

  <dd></dd><dt>Propionyl-CoA: 31.5</dt>

  <dd></dd><dt>Acetyl-CoA: 29.9</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 27.4</dt>

  <dd></dd><dt>Isobutyryl-CoA: 26.2</dt>

  <dd></dd><dt>Benzoyl-CoA: 20.7</dt>

  <dd></dd><dt>fatty_acid: 20.3</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 19.2</dt>

  <dd></dd><dt>CHC-CoA: 13.3</dt>

  <dd></dd><dt>trans-1,2-CPDA: 12.2</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 10.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r34c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00034_NODE_78.. - Region 1 - ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-231"></div> <span class="help-tooltip" id="region_svg-help-231">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00034_NODE_78...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r34c1-svg" data-filename="c00034_NODE_78.._r34c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 44,363 - 54,761 nt. (total: 10,399 nt)
        
        <a class="cluster-rules-header" id="r34c1-rules-header" href="#r34c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r34c1-rules">ectoine: ectoine_synt</div>
      
      <div class="region-svg-container">
       <div id='r34c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-232"></div> <span class="help-tooltip" id="focus-panel-help-232">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r34c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r34c1-clusterblast" data-name="r34c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r34c1-knownclusterblast" data-name="r34c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r34c1-subclusterblast" data-name="r34c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r34c1-MIBiG-cluster-compare" data-name="r34c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r34c1-pfam-details" data-name="r34c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r34c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-34"></div> <span class="help-tooltip" id="cb-general-help-34">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r34c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r34c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r34c1_1.svg">
        NZ_CP027297 (2090741-2101140): Streptomyces sp. SGAir0924 chromosome, complet... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r34c1_2.svg">
        NZ_CP042278 (5919713-5930112): Streptomyces sp. WAC06273 strain WAC6273 subst... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r34c1_3.svg">
        NZ_CP039123 (1992789-2003188): Streptomyces sp. SS52 chromosome, complete genome (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r34c1_4.svg">
        NZ_KZ819184 (2424815-2435214): Streptomyces sp. CS207 scaffold00001, whole ge... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r34c1_5.svg">
        NZ_CP019724 (6551596-6561995): Streptomyces pactum strain ACT12 chromosome, c... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r34c1_6.svg">
        NZ_CP012382 (2000971-2011370): Streptomyces ambofaciens ATCC 23877 chromosome... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r34c1_7.svg">
        NZ_CP012949 (1943699-1954098): Streptomyces ambofaciens strain DSM 40697 chro... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r34c1_8.svg">
        NZ_CP015866 (1603589-1613988): Streptomyces parvulus strain 2297 chromosome, ... (90% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r34c1_9.svg">
        NZ_CP029043 (1620690-1631089): Streptomyces nigra strain 452 chromosome, comp... (90% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r34c1_10.svg">
        NZ_AUKV01000020 (91812-102211): Streptomyces sp. 142MFCol3.1 H305DRAFT scaffo... (90% of genes show similarity), ectoine
      </option>
      
    </select>
    <button id= "clusterblast-r34c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r34c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r34c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-35"></div> <span class="help-tooltip" id="cb-known-help-35">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r34c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r34c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r34c1_1.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r34c1_2.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r34c1_3.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r34c1_4.svg">
        ectoine
      </option>
      
    </select>
    <button id="knownclusterblast-r34c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r34c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r34c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-36"></div> <span class="help-tooltip" id="cb-sub-help-36">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r34c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-37"></div> <span class="help-tooltip" id="MIBiG-help-37">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r34c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r34c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r34c1"></div>
    <div id="comparison-MIBiG-r34c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">ectoine</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.86 (id:0.63, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
         
        
        <td>0.63</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.85 (id:0.61, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.85), rgb(0, 0, 0, 0.85) 85%, white 85%)"></td>
         
        
        <td>0.61</td>
        <td>Other,  , (, E, c, t, o, i, n, e, )</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855.1: 245-3455">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.49, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium kenyense</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylophaga thalassica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852.1: 415-2854">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Sporosarcina pasteurii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylobacter marinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860.1: 710-5134">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylarcula marina</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857.1: 300-4871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylophaga alcalica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.49, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum 20Z</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r34c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.86 (id:0.63, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
      <td>0.86</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.85 (id:0.61, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.85), rgb(0, 0, 0, 0.85) 85%, white 85%)"></td>
      <td>0.85</td>
      <td>Other,  , (, E, c, t, o, i, n, e, )</td>
      <td>ectoine</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855.1: 245-3455">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.49, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium kenyense</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylophaga thalassica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852.1: 415-2854">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Sporosarcina pasteurii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylobacter marinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860.1: 710-5134">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylarcula marina</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857.1: 300-4871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylophaga alcalica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.49, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum 20Z</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r34c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-38"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-38">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r34c1-pfam-details-svg" data-filename="c00034_NODE_78.._r34c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r34c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r70c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00070_NODE_28.. - Region 1 - siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-233"></div> <span class="help-tooltip" id="region_svg-help-233">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00070_NODE_28...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r70c1-svg" data-filename="c00070_NODE_28.._r70c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 10,012 - 22,072 nt. (total: 12,061 nt)
        
        <a class="cluster-rules-header" id="r70c1-rules-header" href="#r70c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r70c1-rules">siderophore: IucA_IucC</div>
      
      <div class="region-svg-container">
       <div id='r70c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-234"></div> <span class="help-tooltip" id="focus-panel-help-234">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r70c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r70c1-clusterblast" data-name="r70c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r70c1-knownclusterblast" data-name="r70c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r70c1-subclusterblast" data-name="r70c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r70c1-MIBiG-cluster-compare" data-name="r70c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r70c1-pfam-details" data-name="r70c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r70c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-39"></div> <span class="help-tooltip" id="cb-general-help-39">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r70c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r70c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r70c1_1.svg">
        NZ_CP027297 (5967715-5977876): Streptomyces sp. SGAir0924 chromosome, complet... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r70c1_2.svg">
        NZ_CP039123 (5826629-5836745): Streptomyces sp. SS52 chromosome, complete genome (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r70c1_3.svg">
        NZ_KZ819185 (1054307-1066185): Streptomyces sp. CS207 scaffold00002, whole ge... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r70c1_4.svg">
        NZ_CP042278 (2115455-2127234): Streptomyces sp. WAC06273 strain WAC6273 subst... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r70c1_5.svg">
        NZ_LIPF01000012 (57742-69674): Streptomyces sp. CB02414 scaffold12, whole gen... (87% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r70c1_6.svg">
        NZ_KN039947 (1103154-1115065): Streptomyces luteus strain TRM 45540 Scaffold2... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r70c1_7.svg">
        NZ_SOPG01000001 (61605-72956): Streptomyces sp. H23 NODE 1 length 804249 cov ... (87% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r70c1_8.svg">
        NZ_CP034353 (2126381-2137722): Streptomyces sp. KPB2 chromosome, complete genome (87% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r70c1_9.svg">
        NZ_CP054920 (2094409-2105750): Streptomyces sp. NA03103 chromosome, complete ... (87% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r70c1_10.svg">
        NZ_CP009754 (2384491-2396165): Streptomyces sp. CCM MD2014 chromosome, comple... (87% of genes show similarity), siderophore
      </option>
      
    </select>
    <button id= "clusterblast-r70c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r70c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r70c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-40"></div> <span class="help-tooltip" id="cb-known-help-40">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r70c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-41"></div> <span class="help-tooltip" id="cb-sub-help-41">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r70c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-42"></div> <span class="help-tooltip" id="MIBiG-help-42">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r70c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r70c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r70c1"></div>
    <div id="comparison-MIBiG-r70c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">siderophore</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Pantoea agglomerans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000946.1: 282-7531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000946/index.html#r1c1" target="_new">BGC0000946.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>vibrioferrin</td>
        
        
        <td>Vibrio parahaemolyticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499.1: 2267-11578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.44, order:0.48, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>aerobactin</td>
        
        
        <td>Pantoea ananatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>ochrobactin</td>
        
        
        <td>Ochrobactrum anthropi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000942.1: 1864728-1871818">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000942/index.html#r1c1" target="_new">BGC0000942.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>petrobactin</td>
        
        
        <td>Bacillus anthracis str. Ames</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Other,  , (, E, c, t, o, i, n, e, )</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001408.1: 1377589-1387887">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001408/index.html#r1c1" target="_new">BGC0001408.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>xanthoferrin</td>
        
        
        <td>Xanthomonas oryzae pv. oryzae KACC 10331</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylophaga thalassica</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r70c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Pantoea agglomerans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000946.1: 282-7531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000946/index.html#r1c1" target="_new">BGC0000946.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Other</td>
      <td>vibrioferrin</td>
      <td>Vibrio parahaemolyticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499.1: 2267-11578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.44, order:0.48, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Other</td>
      <td>aerobactin</td>
      <td>Pantoea ananatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>ochrobactin</td>
      <td>Ochrobactrum anthropi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000942.1: 1864728-1871818">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000942/index.html#r1c1" target="_new">BGC0000942.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>petrobactin</td>
      <td>Bacillus anthracis str. Ames</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other,  , (, E, c, t, o, i, n, e, )</td>
      <td>ectoine</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001408.1: 1377589-1387887">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001408/index.html#r1c1" target="_new">BGC0001408.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>xanthoferrin</td>
      <td>Xanthomonas oryzae pv. oryzae KACC 10331</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylophaga thalassica</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r70c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-43"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-43">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r70c1-pfam-details-svg" data-filename="c00070_NODE_28.._r70c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r70c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r78c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00078_NODE_61.. - Region 1 - indole <div class="help-container"> <div class="help-icon" data-id="region_svg-help-235"></div> <span class="help-tooltip" id="region_svg-help-235">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00078_NODE_61...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r78c1-svg" data-filename="c00078_NODE_61.._r78c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 42,578 - 63,756 nt. (total: 21,179 nt)
        
        <a class="cluster-rules-header" id="r78c1-rules-header" href="#r78c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r78c1-rules">indole: (indsynth or dmat or indole_PTase)</div>
      
      <div class="region-svg-container">
       <div id='r78c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-236"></div> <span class="help-tooltip" id="focus-panel-help-236">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r78c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r78c1-clusterblast" data-name="r78c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r78c1-knownclusterblast" data-name="r78c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r78c1-subclusterblast" data-name="r78c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r78c1-MIBiG-cluster-compare" data-name="r78c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r78c1-pfam-details" data-name="r78c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r78c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-44"></div> <span class="help-tooltip" id="cb-general-help-44">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r78c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r78c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r78c1_1.svg">
        NZ_KZ819184 (4061414-4082518): Streptomyces sp. CS207 scaffold00001, whole ge... (92% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r78c1_2.svg">
        NZ_CP027297 (176678-196791): Streptomyces sp. SGAir0924 chromosome, complete ... (92% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r78c1_3.svg">
        NZ_CP039123 (56484-76520): Streptomyces sp. SS52 chromosome, complete genome (92% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r78c1_4.svg">
        NZ_KZ195574 (1936738-1956743): Streptomyces sp. CS113 scaffold00001, whole ge... (23% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r78c1_5.svg">
        NZ_KQ948937 (33031-153700): Streptomyces caeruleatus strain NRRL B-24802 PRJN... (5% of genes show similarity), NRPS,PKS-like,T1PKS,T3PKS,indole
      </option>
      
      <option value = "svg/clusterblast_r78c1_6.svg">
        NZ_CP047318 (4897393-4918563): Streptomyces sp. HM190 chromosome, complete ge... (23% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r78c1_7.svg">
        NZ_FRBI01000005 (103301-124465): Streptomyces paucisporeus strain CGMCC 4.202... (17% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r78c1_8.svg">
        NZ_CP045178 (8168001-8189168): Streptomyces tsukubensis strain AT3 chromosome... (15% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r78c1_9.svg">
        NZ_FNCC01000005 (140482-218575): Lechevalieria fradiae strain CGMCC 4.3506, w... (4% of genes show similarity), NRPS,PKS-like,T1PKS,indole
      </option>
      
      <option value = "svg/clusterblast_r78c1_10.svg">
        NZ_CP016793 (9363786-9440248): Lentzea guizhouensis strain DHS C013 chromosom... (7% of genes show similarity), T1PKS,indole
      </option>
      
    </select>
    <button id= "clusterblast-r78c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r78c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r78c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-45"></div> <span class="help-tooltip" id="cb-known-help-45">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r78c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r78c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r78c1_1.svg">
        7-prenylisatin
      </option>
      
      <option value = "svg/knownclusterblast_r78c1_2.svg">
        5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester
      </option>
      
    </select>
    <button id="knownclusterblast-r78c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r78c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r78c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-46"></div> <span class="help-tooltip" id="cb-sub-help-46">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r78c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-47"></div> <span class="help-tooltip" id="MIBiG-help-47">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r78c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r78c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r78c1"></div>
    <div id="comparison-MIBiG-r78c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">indole</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294.1: 26867-35886">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.87 (id:0.90, order:0.73, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.87), rgb(0, 0, 0, 0.87) 87%, white 87%)"></td>
         
        
        <td>0.65</td>
        <td>Other</td>
        
        <td>7-prenylisatin</td>
        
        
        <td>Streptomyces sp. MBT28</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000772.1: 960-13404">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000772/index.html#r1c1" target="_new">BGC0000772.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>lipopolysaccharide</td>
        
        
        <td>Brucella melitensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001387.1: 150-29437">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001387/index.html#r1c1" target="_new">BGC0001387.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>nucleocidin</td>
        
        
        <td>Streptomyces calvus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.54, order:0.37, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>Other</td>
        
        <td>5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester</td>
        
        
        <td>Streptomyces sp. RM-5-8</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000503.1: 0-17083">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000503/index.html#r1c1" target="_new">BGC0000503.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
         
        
        <td>0.12</td>
        <td>RiPP</td>
        
        <td>cinnamycin</td>
        
        
        <td>Streptomyces cinnamoneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.36 (id:0.46, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
         
        
        <td>0.10</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B</td>
        
        
        <td>Streptomyces sp. WK-5344</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001513.1: 120-11698">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001513/index.html#r1c1" target="_new">BGC0001513.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.34 (id:0.46, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
         
        
        <td>0.09</td>
        <td>Polyketide</td>
        
        <td>anthrabenzoxocinone</td>
        
        
        <td>Actinobacteria bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885.1: 0-16706">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.32 (id:0.45, order:0.26, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
         
        
        <td>0.08</td>
        <td>Other</td>
        
        <td>4-hydroxy-3-nitrosobenzamide</td>
        
        
        <td>Streptomyces murayamaensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.31 (id:0.45, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
         
        
        <td>0.08</td>
        <td>Terpene</td>
        
        <td>grixazone</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000877.1: 436-46016">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000877/index.html#r1c1" target="_new">BGC0000877.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.31 (id:0.45, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
         
        
        <td>0.08</td>
        <td>Other</td>
        
        <td>polyoxin A, polyoxin H</td>
        
        
        <td>Streptomyces cacaoi subsp. asoensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r78c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294.1: 26867-35886">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.87 (id:0.90, order:0.73, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.87), rgb(0, 0, 0, 0.87) 87%, white 87%)"></td>
      <td>0.87</td>
      <td>Other</td>
      <td>7-prenylisatin</td>
      <td>Streptomyces sp. MBT28</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000772.1: 960-13404">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000772/index.html#r1c1" target="_new">BGC0000772.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Saccharide</td>
      <td>lipopolysaccharide</td>
      <td>Brucella melitensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001387.1: 150-29437">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001387/index.html#r1c1" target="_new">BGC0001387.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>nucleocidin</td>
      <td>Streptomyces calvus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.54, order:0.37, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>Other</td>
      <td>5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester</td>
      <td>Streptomyces sp. RM-5-8</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000503.1: 0-17083">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000503/index.html#r1c1" target="_new">BGC0000503.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
      <td>0.39</td>
      <td>RiPP</td>
      <td>cinnamycin</td>
      <td>Streptomyces cinnamoneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.36 (id:0.46, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
      <td>0.36</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B</td>
      <td>Streptomyces sp. WK-5344</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001513.1: 120-11698">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001513/index.html#r1c1" target="_new">BGC0001513.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.34 (id:0.46, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
      <td>0.34</td>
      <td>Polyketide</td>
      <td>anthrabenzoxocinone</td>
      <td>Actinobacteria bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885.1: 0-16706">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.32 (id:0.45, order:0.26, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
      <td>0.32</td>
      <td>Other</td>
      <td>4-hydroxy-3-nitrosobenzamide</td>
      <td>Streptomyces murayamaensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.31 (id:0.45, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
      <td>0.31</td>
      <td>Terpene</td>
      <td>grixazone</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000877.1: 436-46016">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000877/index.html#r1c1" target="_new">BGC0000877.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.31 (id:0.45, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
      <td>0.31</td>
      <td>Other</td>
      <td>polyoxin A, polyoxin H</td>
      <td>Streptomyces cacaoi subsp. asoensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r78c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-48"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-48">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r78c1-pfam-details-svg" data-filename="c00078_NODE_61.._r78c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r78c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r100c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00100_NODE_16.. - Region 1 - butyrolactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-237"></div> <span class="help-tooltip" id="region_svg-help-237">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00100_NODE_16...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r100c1-svg" data-filename="c00100_NODE_16.._r100c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 2,456 nt. (total: 2,456 nt)
        
        <a class="cluster-rules-header" id="r100c1-rules-header" href="#r100c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r100c1-rules">butyrolactone: AfsA</div>
      
      <div class="region-svg-container">
       <div id='r100c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-238"></div> <span class="help-tooltip" id="focus-panel-help-238">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r100c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r100c1-clusterblast" data-name="r100c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r100c1-knownclusterblast" data-name="r100c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r100c1-subclusterblast" data-name="r100c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r100c1-MIBiG-cluster-compare" data-name="r100c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r100c1-pfam-details" data-name="r100c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r100c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-49"></div> <span class="help-tooltip" id="cb-general-help-49">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r100c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r100c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r100c1_1.svg">
        NZ_CP019724 (231807-242133): Streptomyces pactum strain ACT12 chromosome, com... (44% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r100c1_2.svg">
        NZ_CP019724 (8271294-8319558): Streptomyces pactum strain ACT12 chromosome, c... (12% of genes show similarity), T1PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r100c1_3.svg">
        NZ_WXVT01000003 (94101-105088): Streptomyces sp. KO7888 A913 AS4 SC10, whole ... (50% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r100c1_4.svg">
        NZ_KZ819184 (4102654-4188512): Streptomyces sp. CS207 scaffold00001, whole ge... (6% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r100c1_5.svg">
        NZ_CP027297 (19263-103747): Streptomyces sp. SGAir0924 chromosome, complete g... (7% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r100c1_6.svg">
        NZ_CP030864 (11545-320938): Streptomyces globosus strain LZH-48 plasmid unnam... (1% of genes show similarity), NRPS,NRPS-like,T2PKS,butyrolactone,ectoine,hglE-KS,linaridin,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r100c1_7.svg">
        NZ_CP034352 (1037-312401): Streptomyces sp. W1SF4 plasmid p2, complete sequence (1% of genes show similarity), NRPS,NRPS-like,T1PKS,T2PKS,butyrolactone,melanin
      </option>
      
      <option value = "svg/clusterblast_r100c1_8.svg">
        NZ_LMEQ01000019 (2441-13434): Streptomyces sp. Root1310 contig 26, whole geno... (37% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r100c1_9.svg">
        NZ_KQ948433 (288896-399709): Streptomyces antibioticus strain DSM 40234 PRJNA... (3% of genes show similarity), T3PKS,butyrolactone,other,phenazine
      </option>
      
      <option value = "svg/clusterblast_r100c1_10.svg">
        NZ_SLXR01000016 (76322-138372): Streptomyces sp. BK438 Ga0307687 116, whole g... (7% of genes show similarity), NRPS,arylpolyene,butyrolactone
      </option>
      
    </select>
    <button id= "clusterblast-r100c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r100c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r100c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-50"></div> <span class="help-tooltip" id="cb-known-help-50">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r100c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r100c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r100c1_1.svg">
        lactonamycin
      </option>
      
      <option value = "svg/knownclusterblast_r100c1_2.svg">
        prejadomycin / rabelomycin / gaudimycin C / gaudimycin D / UWM6 / gaudimycin A
      </option>
      
    </select>
    <button id="knownclusterblast-r100c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r100c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r100c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-51"></div> <span class="help-tooltip" id="cb-sub-help-51">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r100c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-52"></div> <span class="help-tooltip" id="MIBiG-help-52">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r100c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r100c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r100c1"></div>
    <div id="comparison-MIBiG-r100c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">butyrolactone</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000848.1: 8273443-8274388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000848/index.html#r1c1" target="_new">BGC0000848.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.36</td>
        <td>Other</td>
        
        <td>A-factor</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000849.1: 6890527-6892237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000849/index.html#r1c1" target="_new">BGC0000849.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Other,  , (, B, u, t, y, r, o, l, a, c, t, o, n, e, )</td>
        
        <td>SCB1, SCB2, SCB3</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Other</td>
        
        <td>Î³-butyrolactone</td>
        
        
        <td>Streptomyces acidiscabies</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001083.1: 0-57523">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001083/index.html#r1c1" target="_new">BGC0001083.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.19 (id:0.53, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
         
        
        <td>0.04</td>
        <td>Polyketide, Terpene</td>
        
        <td>merochlorin A, merochlorin B, deschloro-merochlorin A, deschloro-merochlorin B, isochloro-merochlorin B, dichloro-merochlorin B, merochlorin D, merochlorin C</td>
        
        
        <td>Streptomyces sp. CNH189</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001771.1: 13254-31732">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001771/index.html#r1c1" target="_new">BGC0001771.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.17 (id:0.44, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
         
        
        <td>0.03</td>
        <td>NRP</td>
        
        <td>scleric acid</td>
        
        
        <td>Streptomyces sclerotialus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007.1: 5922480-5942549">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.15 (id:0.46, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>
         
        
        <td>0.03</td>
        <td>Terpene</td>
        
        <td>atolypene A, atolypene B</td>
        
        
        <td>Amycolatopsis tolypomycina</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000813.1: 0-20300">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000813/index.html#r1c1" target="_new">BGC0000813.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.15 (id:0.43, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>
         
        
        <td>0.03</td>
        <td>Alkaloid</td>
        
        <td>K-252a</td>
        
        
        <td>Nonomuraea longicatena</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r100c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000848.1: 8273443-8274388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000848/index.html#r1c1" target="_new">BGC0000848.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>Other</td>
      <td>A-factor</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000849.1: 6890527-6892237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000849/index.html#r1c1" target="_new">BGC0000849.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other,  , (, B, u, t, y, r, o, l, a, c, t, o, n, e, )</td>
      <td>SCB1, SCB2, SCB3</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.48, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Other</td>
      <td>Î³-butyrolactone</td>
      <td>Streptomyces acidiscabies</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001083.1: 0-57523">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001083/index.html#r1c1" target="_new">BGC0001083.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.19 (id:0.53, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
      <td>0.19</td>
      <td>Polyketide, Terpene</td>
      <td>merochlorin A, merochlorin B, deschloro-merochlorin A, deschloro-merochlorin B, isochloro-merochlorin B, dichloro-merochlorin B, merochlorin D, merochlorin C</td>
      <td>Streptomyces sp. CNH189</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001771.1: 13254-31732">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001771/index.html#r1c1" target="_new">BGC0001771.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.17 (id:0.44, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
      <td>0.17</td>
      <td>NRP</td>
      <td>scleric acid</td>
      <td>Streptomyces sclerotialus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007.1: 5922480-5942549">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.15 (id:0.46, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>
      <td>0.15</td>
      <td>Terpene</td>
      <td>atolypene A, atolypene B</td>
      <td>Amycolatopsis tolypomycina</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000813.1: 0-20300">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000813/index.html#r1c1" target="_new">BGC0000813.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.15 (id:0.43, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>
      <td>0.15</td>
      <td>Alkaloid</td>
      <td>K-252a</td>
      <td>Nonomuraea longicatena</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r100c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-53"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-53">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r100c1-pfam-details-svg" data-filename="c00100_NODE_16.._r100c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r100c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r104c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00104_NODE_43.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-239"></div> <span class="help-tooltip" id="region_svg-help-239">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00104_NODE_43...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r104c1-svg" data-filename="c00104_NODE_43.._r104c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 17,236 nt. (total: 17,236 nt)
        
        <a class="cluster-rules-header" id="r104c1-rules-header" href="#r104c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r104c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r104c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-240"></div> <span class="help-tooltip" id="focus-panel-help-240">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r104c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r104c1-clusterblast" data-name="r104c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r104c1-knownclusterblast" data-name="r104c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r104c1-subclusterblast" data-name="r104c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r104c1-MIBiG-cluster-compare" data-name="r104c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r104c1-pfam-details" data-name="r104c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r104c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-54"></div> <span class="help-tooltip" id="cb-general-help-54">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r104c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r104c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r104c1_1.svg">
        NZ_CP027297 (200986-224830): Streptomyces sp. SGAir0924 chromosome, complete ... (72% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r104c1_2.svg">
        NZ_CP042278 (266672-291012): Streptomyces sp. WAC06273 strain WAC6273 substr.... (52% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r104c1_3.svg">
        NZ_CP039123 (80719-99478): Streptomyces sp. SS52 chromosome, complete genome (61% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r104c1_4.svg">
        NZ_WHZR01000001 (1953433-2014904): Streptomyces sp. OM5714 A912 AS4 SC1 R2, w... (20% of genes show similarity), NRPS-like,T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r104c1_5.svg">
        NZ_BBNN01000031 (81263-106859): Streptomyces sp. NBRC 110035 strain TP-A0873,... (38% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r104c1_6.svg">
        NZ_CP023696 (6169626-6193098): Streptomyces fradiae ATCC 10745 = DSM 40063 st... (47% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r104c1_7.svg">
        NZ_CP017316 (5980432-6004534): Streptomyces rubrolavendulae strain MJM4426 ch... (47% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r104c1_8.svg">
        NZ_CP015866 (133416-190400): Streptomyces parvulus strain 2297 chromosome, co... (18% of genes show similarity), NRPS-like,terpene
      </option>
      
      <option value = "svg/clusterblast_r104c1_9.svg">
        NZ_KZ195574 (1673262-1697664): Streptomyces sp. CS113 scaffold00001, whole ge... (37% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r104c1_10.svg">
        NZ_RSEG01000001 (59184-84720): Streptomyces tendae strain UTMC 3329 Scaffold1... (40% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r104c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r104c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r104c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-55"></div> <span class="help-tooltip" id="cb-known-help-55">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r104c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r104c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r104c1_1.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r104c1_2.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r104c1_3.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r104c1_4.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r104c1_5.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r104c1_6.svg">
        rustmicin
      </option>
      
    </select>
    <button id="knownclusterblast-r104c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r104c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r104c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-56"></div> <span class="help-tooltip" id="cb-sub-help-56">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r104c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-57"></div> <span class="help-tooltip" id="MIBiG-help-57">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r104c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r104c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r104c1"></div>
    <div id="comparison-MIBiG-r104c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.57, order:0.62, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.38</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.34</td>
        <td>Terpene</td>
        
        <td>Î²-carotein</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Rhodobacter sphaeroides</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656.1: 169-3770">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>Terpene</td>
        
        <td>zeaxanthin</td>
        
        
        <td>Xanthobacter autotrophicus Py2</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.56, order:0.58, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.57, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.56, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces collinus Tu 365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640.1: 2399-8466">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:0.48, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Enterobacteriaceae bacterium DC404</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630.1: 389-6051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.43, order:0.48, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene</td>
        
        <td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
        
        
        <td>Paracoccus haeundaensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.55, order:0.38, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r104c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.57, order:0.62, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Terpene</td>
      <td>Î²-carotein</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Rhodobacter sphaeroides</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656.1: 169-3770">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>zeaxanthin</td>
      <td>Xanthobacter autotrophicus Py2</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.56, order:0.58, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.57, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.56, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces collinus Tu 365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640.1: 2399-8466">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.44, order:0.48, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Enterobacteriaceae bacterium DC404</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630.1: 389-6051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.43, order:0.48, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Terpene</td>
      <td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
      <td>Paracoccus haeundaensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.55, order:0.38, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r104c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-58"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-58">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r104c1-pfam-details-svg" data-filename="c00104_NODE_43.._r104c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r104c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r106c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00106_NODE_14.. - Region 1 - melanin <div class="help-container"> <div class="help-icon" data-id="region_svg-help-241"></div> <span class="help-tooltip" id="region_svg-help-241">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00106_NODE_14...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r106c1-svg" data-filename="c00106_NODE_14.._r106c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 6,859 nt. (total: 6,859 nt)
        
        <a class="cluster-rules-header" id="r106c1-rules-header" href="#r106c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r106c1-rules">melanin: melC</div>
      
      <div class="region-svg-container">
       <div id='r106c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-242"></div> <span class="help-tooltip" id="focus-panel-help-242">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r106c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r106c1-clusterblast" data-name="r106c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r106c1-knownclusterblast" data-name="r106c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r106c1-subclusterblast" data-name="r106c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r106c1-MIBiG-cluster-compare" data-name="r106c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r106c1-pfam-details" data-name="r106c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r106c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-59"></div> <span class="help-tooltip" id="cb-general-help-59">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r106c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r106c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r106c1_1.svg">
        NZ_CP042278 (5037528-5047948): Streptomyces sp. WAC06273 strain WAC6273 subst... (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r106c1_2.svg">
        NZ_CP027297 (2976866-2987475): Streptomyces sp. SGAir0924 chromosome, complet... (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r106c1_3.svg">
        NZ_KZ819184 (1544416-1555025): Streptomyces sp. CS207 scaffold00001, whole ge... (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r106c1_4.svg">
        NZ_CP039123 (2898042-2908534): Streptomyces sp. SS52 chromosome, complete genome (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r106c1_5.svg">
        NZ_LIPF01000001 (964784-975399): Streptomyces sp. CB02414 scaffold1, whole ge... (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r106c1_6.svg">
        NZ_KZ195577 (5438387-5448843): Streptomyces sp. CS159 scaffold00001, whole ge... (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r106c1_7.svg">
        NZ_WXVT01000001 (5625797-5636259): Streptomyces sp. KO7888 A913 AS4 SC1 R3, w... (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r106c1_8.svg">
        NZ_WHZR01000001 (5341496-5352129): Streptomyces sp. OM5714 A912 AS4 SC1 R2, w... (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r106c1_9.svg">
        NZ_CP045547 (5677136-5687784): Streptomyces sp. SYP-A7193 chromosome, complet... (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r106c1_10.svg">
        NZ_CP034353 (5318189-5328651): Streptomyces sp. KPB2 chromosome, complete genome (66% of genes show similarity), melanin
      </option>
      
    </select>
    <button id= "clusterblast-r106c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r106c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r106c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-60"></div> <span class="help-tooltip" id="cb-known-help-60">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r106c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r106c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r106c1_1.svg">
        melanin
      </option>
      
      <option value = "svg/knownclusterblast_r106c1_2.svg">
        istamycin
      </option>
      
      <option value = "svg/knownclusterblast_r106c1_3.svg">
        melanin
      </option>
      
      <option value = "svg/knownclusterblast_r106c1_4.svg">
        melanin
      </option>
      
      <option value = "svg/knownclusterblast_r106c1_5.svg">
        lactonamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r106c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r106c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r106c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-61"></div> <span class="help-tooltip" id="cb-sub-help-61">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r106c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-62"></div> <span class="help-tooltip" id="MIBiG-help-62">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r106c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r106c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r106c1"></div>
    <div id="comparison-MIBiG-r106c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">melanin</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000910.1: 2943455-2944875">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000910/index.html#r1c1" target="_new">BGC0000910.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.87 (id:0.67, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.87), rgb(0, 0, 0, 0.87) 87%, white 87%)"></td>
         
        
        <td>0.67</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000911.1: 607505-608793">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000911/index.html#r1c1" target="_new">BGC0000911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.80 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
         
        
        <td>0.52</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000912.1: 2884831-2886096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000912/index.html#r1c1" target="_new">BGC0000912.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000909.1: 0-6625">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000909/index.html#r1c1" target="_new">BGC0000909.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.55, order:0.56, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000908.1: 0-6497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000908/index.html#r1c1" target="_new">BGC0000908.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.50, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.49, order:0.43, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Terpene</td>
        
        <td>grixazone</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.47, order:0.43, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B</td>
        
        
        <td>Streptomyces sp. WK-5344</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001592.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001592/index.html#r1c1" target="_new">BGC0001592.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.47, order:0.43, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>ferroverdin</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002029.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002029/index.html#r1c1" target="_new">BGC0002029.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.47, order:0.43, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B, bagremycin C, bagremycin E, bagremycin F, bagremycin G, ferroverdin A, ferroverdin B, ferroverdin C</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885.1: 0-16706">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.47, order:0.43, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
        <td>0.12</td>
        <td>Other</td>
        
        <td>4-hydroxy-3-nitrosobenzamide</td>
        
        
        <td>Streptomyces murayamaensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r106c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000910.1: 2943455-2944875">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000910/index.html#r1c1" target="_new">BGC0000910.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.87 (id:0.67, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.87), rgb(0, 0, 0, 0.87) 87%, white 87%)"></td>
      <td>0.87</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000911.1: 607505-608793">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000911/index.html#r1c1" target="_new">BGC0000911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.80 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
      <td>0.80</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000912.1: 2884831-2886096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000912/index.html#r1c1" target="_new">BGC0000912.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000909.1: 0-6625">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000909/index.html#r1c1" target="_new">BGC0000909.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.55, order:0.56, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000908.1: 0-6497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000908/index.html#r1c1" target="_new">BGC0000908.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.50, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.49, order:0.43, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Terpene</td>
      <td>grixazone</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.47, order:0.43, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B</td>
      <td>Streptomyces sp. WK-5344</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001592.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001592/index.html#r1c1" target="_new">BGC0001592.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.47, order:0.43, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other</td>
      <td>ferroverdin</td>
      <td>Streptomyces lunaelactis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002029.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002029/index.html#r1c1" target="_new">BGC0002029.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.47, order:0.43, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B, bagremycin C, bagremycin E, bagremycin F, bagremycin G, ferroverdin A, ferroverdin B, ferroverdin C</td>
      <td>Streptomyces lunaelactis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885.1: 0-16706">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.40 (id:0.47, order:0.43, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
      <td>0.40</td>
      <td>Other</td>
      <td>4-hydroxy-3-nitrosobenzamide</td>
      <td>Streptomyces murayamaensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r106c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-63"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-63">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r106c1-pfam-details-svg" data-filename="c00106_NODE_14.._r106c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r106c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r110c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00110_NODE_21.. - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-243"></div> <span class="help-tooltip" id="region_svg-help-243">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00110_NODE_21...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r110c1-svg" data-filename="c00110_NODE_21.._r110c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 7,665 nt. (total: 7,665 nt)
        
        <a class="cluster-rules-header" id="r110c1-rules-header" href="#r110c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r110c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193)</div>
      
      <div class="region-svg-container">
       <div id='r110c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-244"></div> <span class="help-tooltip" id="focus-panel-help-244">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r110c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r110c1-clusterblast" data-name="r110c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r110c1-knownclusterblast" data-name="r110c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r110c1-subclusterblast" data-name="r110c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r110c1-MIBiG-cluster-compare" data-name="r110c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r110c1-pfam-details" data-name="r110c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r110c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-64"></div> <span class="help-tooltip" id="cb-general-help-64">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r110c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r110c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r110c1_1.svg">
        NZ_CP027297 (7299543-7309759): Streptomyces sp. SGAir0924 chromosome, complet... (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r110c1_2.svg">
        NZ_CP039123 (7391811-7402027): Streptomyces sp. SS52 chromosome, complete genome (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r110c1_3.svg">
        NZ_CP042278 (467057-475278): Streptomyces sp. WAC06273 strain WAC6273 substr.... (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r110c1_4.svg">
        NZ_LIPF01000005 (435694-445910): Streptomyces sp. CB02414 scaffold5, whole ge... (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r110c1_5.svg">
        NZ_JACJIE010000009 (222688-285948): Streptomyces calvus strain CECT 3271 Ga04... (11% of genes show similarity), T1PKS,bacteriocin,terpene
      </option>
      
      <option value = "svg/clusterblast_r110c1_6.svg">
        NZ_CP019724 (719804-727563): Streptomyces pactum strain ACT12 chromosome, com... (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r110c1_7.svg">
        NZ_CP012949 (7116522-7124326): Streptomyces ambofaciens strain DSM 40697 chro... (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r110c1_8.svg">
        NZ_KZ195574 (719561-727704): Streptomyces sp. CS113 scaffold00001, whole geno... (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r110c1_9.svg">
        NZ_CP012382 (7314193-7324409): Streptomyces ambofaciens ATCC 23877 chromosome... (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r110c1_10.svg">
        NZ_GG657756 (7543514-7553730): Streptomyces lividans TK24 supercont1.1, whole... (66% of genes show similarity), bacteriocin
      </option>
      
    </select>
    <button id= "clusterblast-r110c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r110c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r110c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-65"></div> <span class="help-tooltip" id="cb-known-help-65">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r110c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r110c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r110c1_1.svg">
        informatipeptin
      </option>
      
    </select>
    <button id="knownclusterblast-r110c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r110c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r110c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-66"></div> <span class="help-tooltip" id="cb-sub-help-66">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r110c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-67"></div> <span class="help-tooltip" id="MIBiG-help-67">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r110c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r110c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r110c1"></div>
    <div id="comparison-MIBiG-r110c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">RiPP-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.61, order:0.40, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>RiPP</td>
        
        <td>informatipeptin</td>
        
        
        <td>Streptomyces viridochromogenes DSM 40736</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928.1: 108-8402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>
        
        
        <td>Streptomyces sp. MA37</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000484.1: 184-6641">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000484/index.html#r1c1" target="_new">BGC0000484.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>RiPP</td>
        
        <td>glycocin F</td>
        
        
        <td>Lactobacillus plantarum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000759.1: 1-29377">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000759/index.html#r1c1" target="_new">BGC0000759.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>diutan polysaccharide</td>
        
        
        <td>Sphingomonas sp. ATCC 53159</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000797.1: 324-28703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000797/index.html#r1c1" target="_new">BGC0000797.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>sphingan polysaccharide</td>
        
        
        <td>Sphingomonas sp. S88</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000768.1: 0-30166">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000768/index.html#r1c1" target="_new">BGC0000768.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>gellan polysaccharide</td>
        
        
        <td>Sphingomonas elodea ATCC 31461</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000019.1: 339-17245">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000019/index.html#r1c1" target="_new">BGC0000019.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>angolamycin</td>
        
        
        <td>Streptomyces eurythermus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000584.1: 8922-12604">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000584/index.html#r1c1" target="_new">BGC0000584.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>RiPP</td>
        
        <td>colicin V</td>
        
        
        <td>Escherichia coli</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r110c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.61, order:0.40, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>RiPP</td>
      <td>informatipeptin</td>
      <td>Streptomyces viridochromogenes DSM 40736</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928.1: 108-8402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>
      <td>Streptomyces sp. MA37</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000484.1: 184-6641">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000484/index.html#r1c1" target="_new">BGC0000484.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>RiPP</td>
      <td>glycocin F</td>
      <td>Lactobacillus plantarum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000759.1: 1-29377">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000759/index.html#r1c1" target="_new">BGC0000759.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>diutan polysaccharide</td>
      <td>Sphingomonas sp. ATCC 53159</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000797.1: 324-28703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000797/index.html#r1c1" target="_new">BGC0000797.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>sphingan polysaccharide</td>
      <td>Sphingomonas sp. S88</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000768.1: 0-30166">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000768/index.html#r1c1" target="_new">BGC0000768.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>gellan polysaccharide</td>
      <td>Sphingomonas elodea ATCC 31461</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000019.1: 339-17245">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000019/index.html#r1c1" target="_new">BGC0000019.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Polyketide</td>
      <td>angolamycin</td>
      <td>Streptomyces eurythermus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000584.1: 8922-12604">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000584/index.html#r1c1" target="_new">BGC0000584.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>RiPP</td>
      <td>colicin V</td>
      <td>Escherichia coli</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r110c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-68"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-68">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r110c1-pfam-details-svg" data-filename="c00110_NODE_21.._r110c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r110c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r110c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00110_NODE_21.. - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-245"></div> <span class="help-tooltip" id="region_svg-help-245">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00110_NODE_21...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r110c2-svg" data-filename="c00110_NODE_21.._r110c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 20,623 - 34,547 nt. (total: 13,925 nt)
        
        <a class="cluster-rules-header" id="r110c2-rules-header" href="#r110c2">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r110c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r110c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-246"></div> <span class="help-tooltip" id="focus-panel-help-246">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r110c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r110c2-clusterblast" data-name="r110c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r110c2-knownclusterblast" data-name="r110c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r110c2-subclusterblast" data-name="r110c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r110c2-MIBiG-cluster-compare" data-name="r110c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r110c2-pfam-details" data-name="r110c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r110c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-69"></div> <span class="help-tooltip" id="cb-general-help-69">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r110c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r110c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r110c2_1.svg">
        NZ_LIPF01000005 (398682-419714): Streptomyces sp. CB02414 scaffold5, whole ge... (80% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r110c2_2.svg">
        NZ_CP039123 (7357810-7378842): Streptomyces sp. SS52 chromosome, complete genome (80% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r110c2_3.svg">
        NZ_CP042278 (491440-510671): Streptomyces sp. WAC06273 strain WAC6273 substr.... (76% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r110c2_4.svg">
        NZ_KN039948 (312041-333073): Streptomyces luteus strain TRM 45540 Scaffold3, ... (72% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r110c2_5.svg">
        NZ_CP019724 (747229-767899): Streptomyces pactum strain ACT12 chromosome, com... (69% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r110c2_6.svg">
        NZ_CP027297 (7267113-7288145): Streptomyces sp. SGAir0924 chromosome, complet... (70% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r110c2_7.svg">
        NZ_GG657757 (8121096-8142128): Streptomyces viridochromogenes DSM 40736 super... (62% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r110c2_8.svg">
        NZ_KZ671983 (72944-93976): Streptomyces sp. 46 scaffold20, whole genome shotg... (62% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r110c2_9.svg">
        NZ_CP036534 (7656961-7677990): Streptomyces sp. VN1 chromosome, complete genome (68% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r110c2_10.svg">
        NZ_CP016795 (7522334-7543363): Streptomyces olivaceus strain KLBMP 5084 chrom... (68% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r110c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r110c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r110c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-70"></div> <span class="help-tooltip" id="cb-known-help-70">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r110c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r110c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r110c2_1.svg">
        versipelostatin
      </option>
      
      <option value = "svg/knownclusterblast_r110c2_2.svg">
        chaxamycin A / chaxamycin B / chaxamycin C / chaxamycin D
      </option>
      
      <option value = "svg/knownclusterblast_r110c2_3.svg">
        pentamycin
      </option>
      
      <option value = "svg/knownclusterblast_r110c2_4.svg">
        filipin
      </option>
      
      <option value = "svg/knownclusterblast_r110c2_5.svg">
        griseorhodin A
      </option>
      
      <option value = "svg/knownclusterblast_r110c2_6.svg">
        meoabyssomicin / abyssomicin
      </option>
      
      <option value = "svg/knownclusterblast_r110c2_7.svg">
        nystatin
      </option>
      
    </select>
    <button id="knownclusterblast-r110c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r110c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r110c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-71"></div> <span class="help-tooltip" id="cb-sub-help-71">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r110c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-72"></div> <span class="help-tooltip" id="MIBiG-help-72">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r110c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r110c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r110c2"></div>
    <div id="comparison-MIBiG-r110c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000683.1: 86072-87080">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000683/index.html#r1c1" target="_new">BGC0000683.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Terpene</td>
        
        <td>avermitilol</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001746.1: 7136894-7138013">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001746/index.html#r1c1" target="_new">BGC0001746.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Terpene</td>
        
        <td>pristinol</td>
        
        
        <td>Streptomyces pristinaespiralis ATCC 25486</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Alkaloid</td>
        
        <td>kitasetaline</td>
        
        
        <td>Kitasatospora setae KM-6054</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631.1: 2217593-2245275">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>botrydial</td>
        
        
        <td>Botrytis cinerea B05.10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:0.46, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene</td>
        
        <td>raimonol</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001223.1: 24900-34582">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001223/index.html#r1c1" target="_new">BGC0001223.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Other</td>
        
        <td>hydroxysporine</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.47, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>cyslabdan</td>
        
        
        <td>Streptomyces cyslabdanicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310.1: 6460417-6463122">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Terpene</td>
        
        <td>naringenin</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001224.1: 1518-11352">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001224/index.html#r1c1" target="_new">BGC0001224.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.46, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.22</td>
        <td>Other</td>
        
        <td>reductasporine</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001986.1: 0-3657">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001986/index.html#r1c1" target="_new">BGC0001986.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Other</td>
        
        <td>purincyclamide</td>
        
        
        <td>Streptomyces chrestomyceticus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r110c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000683.1: 86072-87080">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000683/index.html#r1c1" target="_new">BGC0000683.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>avermitilol</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001746.1: 7136894-7138013">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001746/index.html#r1c1" target="_new">BGC0001746.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>pristinol</td>
      <td>Streptomyces pristinaespiralis ATCC 25486</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Alkaloid</td>
      <td>kitasetaline</td>
      <td>Kitasatospora setae KM-6054</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631.1: 2217593-2245275">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>botrydial</td>
      <td>Botrytis cinerea B05.10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.46, order:0.46, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>raimonol</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001223.1: 24900-34582">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001223/index.html#r1c1" target="_new">BGC0001223.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>hydroxysporine</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.47, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>cyslabdan</td>
      <td>Streptomyces cyslabdanicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310.1: 6460417-6463122">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Terpene</td>
      <td>naringenin</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001224.1: 1518-11352">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001224/index.html#r1c1" target="_new">BGC0001224.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.46, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Other</td>
      <td>reductasporine</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001986.1: 0-3657">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001986/index.html#r1c1" target="_new">BGC0001986.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Other</td>
      <td>purincyclamide</td>
      <td>Streptomyces chrestomyceticus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r110c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-73"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-73">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r110c2-pfam-details-svg" data-filename="c00110_NODE_21.._r110c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r110c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r121c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00121_NODE_36.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-247"></div> <span class="help-tooltip" id="region_svg-help-247">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00121_NODE_36...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r121c1-svg" data-filename="c00121_NODE_36.._r121c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 31,593 nt. (total: 31,593 nt)
        
        <a class="cluster-rules-header" id="r121c1-rules-header" href="#r121c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r121c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r121c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-248"></div> <span class="help-tooltip" id="focus-panel-help-248">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r121c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r121c1-nrps_pks" data-name="r121c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r121c1-clusterblast" data-name="r121c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r121c1-knownclusterblast" data-name="r121c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r121c1-subclusterblast" data-name="r121c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r121c1-MIBiG-cluster-compare" data-name="r121c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r121c1-pfam-details" data-name="r121c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r121c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-74"></div> <span class="help-tooltip" id="nrps-domain-help-74">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r121c1-details-svg" data-filename="c00121_NODE_36.._r121c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r121c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r121c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-77"></div> <span class="help-tooltip" id="cb-general-help-77">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r121c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r121c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r121c1_1.svg">
        NZ_KZ819185 (488550-566486): Streptomyces sp. CS207 scaffold00002, whole geno... (54% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r121c1_2.svg">
        NZ_CP039123 (6294253-6369829): Streptomyces sp. SS52 chromosome, complete genome (57% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r121c1_3.svg">
        NZ_CP042278 (1583458-1663462): Streptomyces sp. WAC06273 strain WAC6273 subst... (50% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r121c1_4.svg">
        NZ_CP027297 (6501004-6580734): Streptomyces sp. SGAir0924 chromosome, complet... (32% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r121c1_5.svg">
        NZ_JTIY01000002 (691244-774422): Streptomyces sp. AcH 505 scaffold00002, whol... (39% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r121c1_6.svg">
        NZ_CP042324 (3524876-3603956): Streptomyces coelicolor A3(2) strain CFB NBC 0... (36% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r121c1_7.svg">
        NZ_LT629768 (3748808-3827608): Streptomyces sp. 2114.2 chromosome I (35% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r121c1_8.svg">
        NZ_GG657756 (4710103-4788943): Streptomyces lividans TK24 supercont1.1, whole... (37% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r121c1_9.svg">
        NZ_BEWD01000011 (217053-297488): Streptomyces rubrogriseus strain NBRC 15455,... (35% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r121c1_10.svg">
        NZ_CP026730 (2884074-2986849): Streptomyces sp. CB09001 chromosome, complete ... (23% of genes show similarity), NRPS,T1PKS,butyrolactone
      </option>
      
    </select>
    <button id= "clusterblast-r121c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r121c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r121c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-78"></div> <span class="help-tooltip" id="cb-known-help-78">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r121c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r121c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r121c1_1.svg">
        lipopeptide 8D1-1 / lipopeptide 8D1-2
      </option>
      
      <option value = "svg/knownclusterblast_r121c1_2.svg">
        CDA1b / CDA2a / CDA2b / CDA3a / CDA3b / CDA4a / CDA4b
      </option>
      
      <option value = "svg/knownclusterblast_r121c1_3.svg">
        friulimicin A / friulimicin B / friulimicin C / friulimicin D
      </option>
      
      <option value = "svg/knownclusterblast_r121c1_4.svg">
        RP-1776
      </option>
      
      <option value = "svg/knownclusterblast_r121c1_5.svg">
        cadaside A / cadaside B
      </option>
      
      <option value = "svg/knownclusterblast_r121c1_6.svg">
        A54145
      </option>
      
      <option value = "svg/knownclusterblast_r121c1_7.svg">
        glycinocin A
      </option>
      
      <option value = "svg/knownclusterblast_r121c1_8.svg">
        daptomycin
      </option>
      
      <option value = "svg/knownclusterblast_r121c1_9.svg">
        marformycin A / marformycin B / marformycin C / marformycin D / marformycin E / marformycin F
      </option>
      
      <option value = "svg/knownclusterblast_r121c1_10.svg">
        atratumycin
      </option>
      
    </select>
    <button id="knownclusterblast-r121c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r121c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r121c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-79"></div> <span class="help-tooltip" id="cb-sub-help-79">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r121c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r121c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r121c1_1.svg">
        AL939125_c1: prodigiosin 2-undecylpyrrole (25% of genes show similarity), 2-undecylpyrrole
      </option>
      
    </select>
    <button id="subclusterblast-r121c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r121c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r121c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-80"></div> <span class="help-tooltip" id="MIBiG-help-80">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r121c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r121c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r121c1"></div>
    <div id="comparison-MIBiG-r121c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.74 (id:0.45, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
         
        
        <td>0.42</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.46, order:1.00, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
        <td>0.41</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001629.1: 209-63961">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001629/index.html#r1c1" target="_new">BGC0001629.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.45, order:1.00, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.37</td>
        <td>NRP</td>
        
        <td>Jessenipeptin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050.1: 0-19730">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.44, order:1.00, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.36</td>
        <td>NRP, Polyketide</td>
        
        <td>thalassospiramide A</td>
        
        
        <td>Tistrella bauzanensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000438.1: 137-73725">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000438/index.html#r1c1" target="_new">BGC0000438.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.45, order:1.00, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.36</td>
        <td>NRP</td>
        
        <td>syringopeptin 25A</td>
        
        
        <td>Pseudomonas syringae pv. syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001370.1: 10-79794">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001370/index.html#r1c1" target="_new">BGC0001370.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.76, order:0.59, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.36</td>
        <td>NRP</td>
        
        <td>lipopeptide 8D1-1, lipopeptide 8D1-2</td>
        
        
        <td>Streptomyces rochei</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001903.1: 8904-19774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001903/index.html#r1c1" target="_new">BGC0001903.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.54, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.36</td>
        <td>Other</td>
        
        <td>neocarazostatin A</td>
        
        
        <td>Streptomyces sp. MA37</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.49, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>NRP</td>
        
        <td>bacillibactin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r121c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.74 (id:0.45, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
      <td>0.74</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.46, order:1.00, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001629.1: 209-63961">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001629/index.html#r1c1" target="_new">BGC0001629.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.45, order:1.00, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP</td>
      <td>Jessenipeptin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050.1: 0-19730">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.44, order:1.00, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP, Polyketide</td>
      <td>thalassospiramide A</td>
      <td>Tistrella bauzanensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000438.1: 137-73725">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000438/index.html#r1c1" target="_new">BGC0000438.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.45, order:1.00, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP</td>
      <td>syringopeptin 25A</td>
      <td>Pseudomonas syringae pv. syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001370.1: 10-79794">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001370/index.html#r1c1" target="_new">BGC0001370.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.76, order:0.59, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP</td>
      <td>lipopeptide 8D1-1, lipopeptide 8D1-2</td>
      <td>Streptomyces rochei</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001903.1: 8904-19774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001903/index.html#r1c1" target="_new">BGC0001903.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.54, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>Other</td>
      <td>neocarazostatin A</td>
      <td>Streptomyces sp. MA37</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.49, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r121c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-81"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-81">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r121c1-pfam-details-svg" data-filename="c00121_NODE_36.._r121c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r121c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r121c1-nrps_pks_products" data-name="r121c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r121c1-nrps_pks" data-name="r121c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r121c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-75"></div> <span class="help-tooltip" id="nrps-structures-help-75">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 31593:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r121c1-1" class="smiles-canvas" data-smiles="NCC(=O)NC(CC(=O)N)C(=O)NC([*])C(=O)NC([*])C(=O)NC(CC1=CNc2c1cccc2)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(gly - D-asn) + (X) + (X - trp)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly,d-asn&amp;nrps2=x&amp;nrps3=x,trp" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*,d-asn*&amp;nrps2=x&amp;nrps3=x,trp*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r121c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NCC(=O)NC(CC(=O)N)C(=O)NC([*])C(=O)NC([*])C(=O)NC(CC1=CNc2c1cccc2)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r121c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-76"></div> <span class="help-tooltip" id="nrps-monomers-help-76">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds2_7991</span></strong>: gly - asn
      <div class="collapser collapser-target-cds2_7991 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly,asn" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*,asn*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (528..908)</strong>: gly
          <div class="collapser collapser-target-cds2_7991 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-cds2_7991-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILQLGLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1576..1983)</strong>: asn
          <div class="collapser collapser-target-cds2_7991 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asn
              <div class="collapser collapser-target-cds2_7991-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>asp, asn</dd>
   <dt>Single AA prediction:</dt>
   <dd>asn</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLTKVGEVGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds7975_10804</span></strong>: X
      <div class="collapser collapser-target-cds7975_10804 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (472..879)</strong>: X
          <div class="collapser collapser-target-cds7975_10804 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds7975_10804-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>asp, asn</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DrGKTGVGHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>3-me-glu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds10800_15198</span></strong>: trp
      <div class="collapser collapser-target-cds10800_15198 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=trp" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=trp*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (604..997)</strong>: trp
          <div class="collapser collapser-target-cds10800_15198 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  trp
              <div class="collapser collapser-target-cds10800_15198-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>phe, trp</dd>
   <dt>Single AA prediction:</dt>
   <dd>trp</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DGWAVASVCK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>trp</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r135c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00135_NODE_31.. - Region 1 - T3PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-249"></div> <span class="help-tooltip" id="region_svg-help-249">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00135_NODE_31...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r135c1-svg" data-filename="c00135_NODE_31.._r135c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 27,867 nt. (total: 27,867 nt)
        
        <a class="cluster-rules-header" id="r135c1-rules-header" href="#r135c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r135c1-rules">T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)</div>
      
      <div class="region-svg-container">
       <div id='r135c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-250"></div> <span class="help-tooltip" id="focus-panel-help-250">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r135c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
    
     
      <div class="body-details-header r135c1-clusterblast" data-name="r135c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r135c1-knownclusterblast" data-name="r135c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r135c1-subclusterblast" data-name="r135c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r135c1-MIBiG-cluster-compare" data-name="r135c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r135c1-pfam-details" data-name="r135c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    
     
    <div class="body-details-section r135c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-82"></div> <span class="help-tooltip" id="cb-general-help-82">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r135c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r135c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r135c1_1.svg">
        NZ_CP027297 (1390525-1431632): Streptomyces sp. SGAir0924 chromosome, complet... (73% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r135c1_2.svg">
        NZ_CP039123 (1295156-1336263): Streptomyces sp. SS52 chromosome, complete genome (73% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r135c1_3.svg">
        NZ_CP036534 (992278-1033391): Streptomyces sp. VN1 chromosome, complete genome (70% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r135c1_4.svg">
        NZ_CP016795 (899757-940870): Streptomyces olivaceus strain KLBMP 5084 chromos... (70% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r135c1_5.svg">
        NZ_RSEG01000003 (301668-342793): Streptomyces tendae strain UTMC 3329 Scaffol... (67% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r135c1_6.svg">
        NZ_KZ195574 (7369564-7408348): Streptomyces sp. CS113 scaffold00001, whole ge... (67% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r135c1_7.svg">
        NZ_BEWD01000004 (246376-287501): Streptomyces rubrogriseus strain NBRC 15455,... (67% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r135c1_8.svg">
        NZ_CP042324 (1258218-1297041): Streptomyces coelicolor A3(2) strain CFB NBC 0... (67% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r135c1_9.svg">
        NZ_LT629768 (1312358-1351204): Streptomyces sp. 2114.2 chromosome I (67% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r135c1_10.svg">
        NZ_GG657756 (7014496-7055621): Streptomyces lividans TK24 supercont1.1, whole... (67% of genes show similarity), T3PKS
      </option>
      
    </select>
    <button id= "clusterblast-r135c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r135c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r135c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-83"></div> <span class="help-tooltip" id="cb-known-help-83">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r135c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r135c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r135c1_1.svg">
        flaviolin
      </option>
      
      <option value = "svg/knownclusterblast_r135c1_2.svg">
        flaviolin rhamnoside / 3,3&#39;-diflaviolin / flaviolin
      </option>
      
      <option value = "svg/knownclusterblast_r135c1_3.svg">
        herboxidiene
      </option>
      
      <option value = "svg/knownclusterblast_r135c1_4.svg">
        napyradiomycin
      </option>
      
      <option value = "svg/knownclusterblast_r135c1_5.svg">
        napyradiomycin A80915C
      </option>
      
      <option value = "svg/knownclusterblast_r135c1_6.svg">
        merochlorin A / merochlorin B / deschloro-merochlorin A / deschloro-merochlorin B / isochloro-merochlorin B / dichloro-merochlorin B / merochlorin D / merochlorin C
      </option>
      
      <option value = "svg/knownclusterblast_r135c1_7.svg">
        furaquinocin B
      </option>
      
      <option value = "svg/knownclusterblast_r135c1_8.svg">
        akaeolide
      </option>
      
      <option value = "svg/knownclusterblast_r135c1_9.svg">
        salinomycin
      </option>
      
      <option value = "svg/knownclusterblast_r135c1_10.svg">
        thiolutin
      </option>
      
    </select>
    <button id="knownclusterblast-r135c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r135c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r135c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-84"></div> <span class="help-tooltip" id="cb-sub-help-84">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r135c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-85"></div> <span class="help-tooltip" id="MIBiG-help-85">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r135c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r135c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r135c1"></div>
    <div id="comparison-MIBiG-r135c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T3PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310.1: 6460417-6463122">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
         
        
        <td>0.54</td>
        <td>Terpene</td>
        
        <td>naringenin</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.75 (id:0.55, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
         
        
        <td>0.44</td>
        <td>Other</td>
        
        <td>flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000280.1: 384-6033">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000280/index.html#r1c1" target="_new">BGC0000280.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>Polyketide</td>
        
        <td>2,4-diacetylphloroglucinol</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000281.1: 0-8137">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000281/index.html#r1c1" target="_new">BGC0000281.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>Polyketide</td>
        
        <td>2,4-diacetylphloroglucinol</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282.1: 548717-551330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Polyketide</td>
        
        <td>alkylresorcinol</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001962.1: 0-7027">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001962/index.html#r1c1" target="_new">BGC0001962.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Polyketide</td>
        
        <td>hierridin B, hierridin C</td>
        
        
        <td>Cyanobium sp. LEGE 06113</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001938.1: 90047-91866">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001938/index.html#r1c1" target="_new">BGC0001938.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Polyketide</td>
        
        <td>entotheonella</td>
        
        
        <td>Candidatus Entotheonella serta</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.55, order:0.48, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>Polyketide, Saccharide</td>
        
        <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001077.1: 8379669-8386219">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001077/index.html#r1c1" target="_new">BGC0001077.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Terpene, Polyketide</td>
        
        <td>alkyl-O-dihydrogeranyl-methoxyhydroquinones</td>
        
        
        <td>Actinoplanes missouriensis 431</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831.1: 8162481-8168632">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Polyketide</td>
        
        <td>alkylpyrone-407, alkylpyrone-393</td>
        
        
        <td>Myxococcus xanthus DK 1622</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r135c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310.1: 6460417-6463122">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
      <td>0.81</td>
      <td>Terpene</td>
      <td>naringenin</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.75 (id:0.55, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
      <td>0.75</td>
      <td>Other</td>
      <td>flaviolin</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000280.1: 384-6033">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000280/index.html#r1c1" target="_new">BGC0000280.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Polyketide</td>
      <td>2,4-diacetylphloroglucinol</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000281.1: 0-8137">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000281/index.html#r1c1" target="_new">BGC0000281.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Polyketide</td>
      <td>2,4-diacetylphloroglucinol</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282.1: 548717-551330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>alkylresorcinol</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001962.1: 0-7027">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001962/index.html#r1c1" target="_new">BGC0001962.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>hierridin B, hierridin C</td>
      <td>Cyanobium sp. LEGE 06113</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001938.1: 90047-91866">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001938/index.html#r1c1" target="_new">BGC0001938.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Polyketide</td>
      <td>entotheonella</td>
      <td>Candidatus Entotheonella serta</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.55, order:0.48, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide, Saccharide</td>
      <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
      <td>Saccharopolyspora erythraea</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001077.1: 8379669-8386219">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001077/index.html#r1c1" target="_new">BGC0001077.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Terpene, Polyketide</td>
      <td>alkyl-O-dihydrogeranyl-methoxyhydroquinones</td>
      <td>Actinoplanes missouriensis 431</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831.1: 8162481-8168632">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>alkylpyrone-407, alkylpyrone-393</td>
      <td>Myxococcus xanthus DK 1622</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r135c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-86"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-86">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r135c1-pfam-details-svg" data-filename="c00135_NODE_31.._r135c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r135c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r137c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00137_NODE_16.. - Region 1 - T1PKS,hglE-KS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-251"></div> <span class="help-tooltip" id="region_svg-help-251">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00137_NODE_16...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r137c1-svg" data-filename="c00137_NODE_16.._r137c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 96,321 nt. (total: 96,321 nt)
        
        <a class="cluster-rules-header" id="r137c1-rules-header" href="#r137c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r137c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>hglE-KS: (hglE or hglD)</div>
      
      <div class="region-svg-container">
       <div id='r137c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-252"></div> <span class="help-tooltip" id="focus-panel-help-252">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r137c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r137c1-nrps_pks" data-name="r137c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r137c1-clusterblast" data-name="r137c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r137c1-knownclusterblast" data-name="r137c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r137c1-subclusterblast" data-name="r137c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r137c1-MIBiG-cluster-compare" data-name="r137c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r137c1-pfam-details" data-name="r137c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r137c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-87"></div> <span class="help-tooltip" id="nrps-domain-help-87">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r137c1-details-svg" data-filename="c00137_NODE_16.._r137c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r137c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r137c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-90"></div> <span class="help-tooltip" id="cb-general-help-90">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r137c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r137c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r137c1_1.svg">
        NZ_GG657757 (4852428-4915328): Streptomyces viridochromogenes DSM 40736 super... (19% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r137c1_2.svg">
        NZ_CP054925 (398667-559119): Amycolatopsis sp. Hca4 chromosome, complete genome (35% of genes show similarity), NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r137c1_3.svg">
        NZ_KB912942 (2125778-2296948): Amycolatopsis benzoatilytica AK 16/65 AmybeDRA... (13% of genes show similarity), NRPS-like,T1PKS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r137c1_4.svg">
        NZ_CM002273 (6555444-6843909): Streptomyces sp. PVA 94-07 strain PVA 94-07 ch... (9% of genes show similarity), NRPS,NRPS-like,T1PKS,T3PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r137c1_5.svg">
        NZ_CP026490 (191791-441920): Streptomyces sp. 604F chromosome, complete genome (8% of genes show similarity), NRPS,NRPS-like,T1PKS,T3PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r137c1_6.svg">
        NZ_CP031742 (6968727-7175511): Streptomyces koyangensis strain VK-A60T chromo... (10% of genes show similarity), NRPS,NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r137c1_7.svg">
        NZ_CP034687 (7530582-7715884): Streptomyces griseoviridis strain F1-27 chromo... (11% of genes show similarity), NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r137c1_8.svg">
        NZ_JTIY01000001 (6423575-6583345): Streptomyces sp. AcH 505 scaffold00001, wh... (17% of genes show similarity), T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r137c1_9.svg">
        NZ_CP016824 (54347-384189): Streptomyces sampsonii strain KJ40 chromosome, co... (8% of genes show similarity), NRPS,NRPS-like,T1PKS,T2PKS,butyrolactone,ectoine,lanthipeptide,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r137c1_10.svg">
        NZ_JACHJG010000002 (414742-582166): Streptomyces netropsis strain CECT 3265 G... (11% of genes show similarity), LAP,T1PKS,bacteriocin,terpene,thiopeptide
      </option>
      
    </select>
    <button id= "clusterblast-r137c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r137c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r137c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-91"></div> <span class="help-tooltip" id="cb-known-help-91">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r137c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r137c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r137c1_1.svg">
        candicidin
      </option>
      
      <option value = "svg/knownclusterblast_r137c1_2.svg">
        levorin A3 / C06690 / FR-008-III / candicidin A / UNII-AP5PEF5W7U
      </option>
      
      <option value = "svg/knownclusterblast_r137c1_3.svg">
        nystatin-like Pseudonocardia polyene
      </option>
      
      <option value = "svg/knownclusterblast_r137c1_4.svg">
        nystatin A1
      </option>
      
      <option value = "svg/knownclusterblast_r137c1_5.svg">
        A83543A
      </option>
      
      <option value = "svg/knownclusterblast_r137c1_6.svg">
        nystatin
      </option>
      
      <option value = "svg/knownclusterblast_r137c1_7.svg">
        selvamicin
      </option>
      
      <option value = "svg/knownclusterblast_r137c1_8.svg">
        filipin
      </option>
      
      <option value = "svg/knownclusterblast_r137c1_9.svg">
        JBIR-100
      </option>
      
      <option value = "svg/knownclusterblast_r137c1_10.svg">
        cyphomycin
      </option>
      
    </select>
    <button id="knownclusterblast-r137c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r137c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r137c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-92"></div> <span class="help-tooltip" id="cb-sub-help-92">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r137c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-93"></div> <span class="help-tooltip" id="MIBiG-help-93">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r137c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r137c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r137c1"></div>
    <div id="comparison-MIBiG-r137c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
         <th><span class="cc-heat-product">hglE-KS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.57</td>
        <td>Polyketide</td>
        
        <td>6-methylsalicyclic acid</td>
        
        
        <td>Aspergillus terreus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.53, order:0.75, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.48, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.57</td>
        <td>Polyketide</td>
        
        <td>oronofacic acid</td>
        
        
        <td>Pseudomonas syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000068.1: 1-27296">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000068/index.html#r1c1" target="_new">BGC0000068.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.56, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.49, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.54</td>
        <td>Polyketide</td>
        
        <td>geldanamycin</td>
        
        
        <td>Streptomyces hygroscopicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000160.1: 89033-116126">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000160/index.html#r1c1" target="_new">BGC0000160.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.50</td>
        <td>Polyketide</td>
        
        <td>terreic acid</td>
        
        
        <td>Aspergillus terreus NIH2624</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001164.1: 2029234-2037562">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001164/index.html#r1c1" target="_new">BGC0001164.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.48</td>
        <td>Polyketide</td>
        
        <td>1-heptadecene</td>
        
        
        <td>Cyanothece sp. PCC 7822</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000067.1: 14949-58348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000067/index.html#r1c1" target="_new">BGC0000067.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.56, order:0.54, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.50, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.46</td>
        <td>Polyketide</td>
        
        <td>geldanamycin</td>
        
        
        <td>Streptomyces hygroscopicus subsp. duamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001235.1: 53-50000">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001235/index.html#r1c1" target="_new">BGC0001235.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.55, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.49, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.46</td>
        <td>NRP, Polyketide</td>
        
        <td>antalid</td>
        
        
        <td>Polyangium spumosum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.45</td>
        <td>Polyketide</td>
        
        <td>strobilurin</td>
        
        
        <td>Strobilurus tenacellus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000353.1: 60-28528">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000353/index.html#r1c1" target="_new">BGC0000353.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.56, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.49, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.45</td>
        <td>NRP</td>
        
        <td>tacrolimus</td>
        
        
        <td>Streptomyces sp. MA6548</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001138.1: 0-39199">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001138/index.html#r1c1" target="_new">BGC0001138.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.55, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.55, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.45</td>
        <td>Other,  , (, N, u, c, l, e, o, s, i, d, e, )</td>
        
        <td>a201a</td>
        
        
        <td>Marinactinospora thermotolerans</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r137c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.53, order:0.75, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>Polyketide</td>
      <td>oronofacic acid</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000068.1: 1-27296">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000068/index.html#r1c1" target="_new">BGC0000068.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.56, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>geldanamycin</td>
      <td>Streptomyces hygroscopicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>(-)-Mellein</td>
      <td>Parastagonospora nodorum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>asperlactone</td>
      <td>Aspergillus ochraceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275.1: 1063-6614">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Glarea lozoyensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000067.1: 14949-58348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000067/index.html#r1c1" target="_new">BGC0000067.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.56, order:0.54, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>geldanamycin</td>
      <td>Streptomyces hygroscopicus subsp. duamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001164.1: 2029234-2037562">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001164/index.html#r1c1" target="_new">BGC0001164.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>1-heptadecene</td>
      <td>Cyanothece sp. PCC 7822</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001504.1: 4663086-4676820">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001504/index.html#r1c1" target="_new">BGC0001504.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.49, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>amycomicin</td>
      <td>Amycolatopsis sp. AA4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Aspergillus terreus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000170.1: 25032-49030">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000170/index.html#r1c1" target="_new">BGC0000170.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>yanuthone D</td>
      <td>Aspergillus niger ATCC 1015</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r137c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-94"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-94">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r137c1-pfam-details-svg" data-filename="c00137_NODE_16.._r137c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r137c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r137c1-nrps_pks_products" data-name="r137c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r137c1-nrps_pks" data-name="r137c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r137c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-88"></div> <span class="help-tooltip" id="nrps-structures-help-88">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 96321:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r137c1-1" class="smiles-canvas" data-smiles="C=CCC(O)C([*])C(-O)CC(O)CC(O)CC(O)CC(O)CC(O)CCCCCC(=O)C(C)C(O)C(C)CC(C)=CC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ccmal) + (ohmal - pk - ohmal - ohmal) + (ohmal - ohmal) + (ohmal - redmal) + (redmal) + (mal) + (Me-ohmal - Me-redmal - Me-ccmal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal&amp;nrps2=ohmal,pk,ohmal,ohmal&amp;nrps3=ohmal,ohmal&amp;nrps4=ohmal,redmal&amp;nrps5=redmal&amp;nrps6=mal&amp;nrps7=me-ohmal,me-redmal,me-ccmal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal*&amp;nrps2=ohmal*,pk*,ohmal*,ohmal*&amp;nrps3=ohmal*,ohmal*&amp;nrps4=ohmal*,redmal*&amp;nrps5=redmal*&amp;nrps6=mal*&amp;nrps7=me-ohmal*,me-redmal*,me-ccmal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r137c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C=CCC(O)C([*])C(-O)CC(O)CC(O)CC(O)CC(O)CC(O)CCCCCC(=O)C(C)C(O)C(C)CC(C)=CC(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r137c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-89"></div> <span class="help-tooltip" id="nrps-monomers-help-89">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds2501_9005</span></strong>: mmal
      <div class="collapser collapser-target-cds2501_9005 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (931..1164)</strong>: mmal
          <div class="collapser collapser-target-cds2501_9005 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  (unknown)
              <div class="collapser collapser-target-cds2501_9005-ATSignature collapser-level-none"> </div><div class="collapser-content">
                No matches above 50%
              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds2501_9005-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 28.7</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 24.5</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 20.9</dt>

  <dd></dd><dt>Propionyl-CoA: 19.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 17.4</dt>

  <dd></dd><dt>Malonyl-CoA: 8.4</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>inactive: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds10731_17019</span></strong>: mal
      <div class="collapser collapser-target-cds10731_17019 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (561..862)</strong>: mal
          <div class="collapser collapser-target-cds10731_17019 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds10731_17019-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 83.3%</dd>
<dd>Methylmalonyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds10731_17019-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 210.4</dt>

  <dd></dd><dt>inactive: 100.0</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 47.2</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 39.3</dt>

  <dd></dd><dt>Isobutyryl-CoA: 32.3</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 21.2</dt>

  <dd></dd><dt>Benzoyl-CoA: 18.6</dt>

  <dd></dd><dt>Acetyl-CoA: 15.7</dt>

  <dd></dd><dt>Propionyl-CoA: 13.2</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 9.7</dt>

  <dd></dd><dt>CHC-CoA: 7.1</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds17015_32153</span></strong>: mal - pk - mal - mal
      <div class="collapser collapser-target-cds17015_32153 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (192..499)</strong>: mal
          <div class="collapser collapser-target-cds17015_32153 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds17015_32153-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 100.0%</dd>
<dd>inactive: 75.0%</dd>
<dd>Methylmalonyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds17015_32153-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 227.7</dt>

  <dd></dd><dt>inactive: 101.1</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 63.9</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 53.1</dt>

  <dd></dd><dt>Propionyl-CoA: 47.7</dt>

  <dd></dd><dt>Isobutyryl-CoA: 26.1</dt>

  <dd></dd><dt>Acetyl-CoA: 22.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 19.1</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 17.6</dt>

  <dd></dd><dt>Benzoyl-CoA: 15.5</dt>

  <dd></dd><dt>fatty_acid: 10.2</dt>

  <dd></dd><dt>CHC-CoA: 9.8</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (1816..2113)</strong>: pk
          <div class="collapser collapser-target-cds17015_32153 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds17015_32153-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Ethylmalonyl-CoA
              <div class="collapser collapser-target-cds17015_32153-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Ethylmalonyl-CoA: 226.2</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 222.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 184.5</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 157.0</dt>

  <dd></dd><dt>trans-1,2-CPDA: 129.8</dt>

  <dd></dd><dt>Propionyl-CoA: 118.6</dt>

  <dd></dd><dt>Malonyl-CoA: 114.8</dt>

  <dd></dd><dt>CHC-CoA: 113.9</dt>

  <dd></dd><dt>Isobutyryl-CoA: 75.2</dt>

  <dd></dd><dt>Benzoyl-CoA: 60.3</dt>

  <dd></dd><dt>Acetyl-CoA: 50.8</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 35.5</dt>

  <dd></dd><dt>fatty_acid: 34.5</dt>

  <dd></dd><dt>inactive: 30.2</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (3217..3524)</strong>: mal
          <div class="collapser collapser-target-cds17015_32153 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds17015_32153-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 100.0%</dd>
<dd>inactive: 83.3%</dd>
<dd>Propionyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds17015_32153-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 213.7</dt>

  <dd></dd><dt>inactive: 103.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 68.9</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 58.3</dt>

  <dd></dd><dt>Propionyl-CoA: 27.9</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 21.1</dt>

  <dd></dd><dt>Acetyl-CoA: 19.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 15.0</dt>

  <dd></dd><dt>trans-1,2-CPDA: 11.6</dt>

  <dd></dd><dt>fatty_acid: 11.1</dt>

  <dd></dd><dt>CHC-CoA: 8.4</dt>

  <dd></dd><dt>Benzoyl-CoA: 8.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 5.1</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 4.3</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (4753..4950)</strong>: mal
          <div class="collapser collapser-target-cds17015_32153 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds17015_32153-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 87.5%</dd>
<dd>inactive: 75.0%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds17015_32153-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 139.4</dt>

  <dd></dd><dt>inactive: 60.1</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 45.4</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 41.4</dt>

  <dd></dd><dt>Isobutyryl-CoA: 27.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 24.8</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 20.4</dt>

  <dd></dd><dt>Acetyl-CoA: 12.6</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 9.2</dt>

  <dd></dd><dt>CHC-CoA: 8.3</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>Propionyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds32361_39153</span></strong>: mal
      <div class="collapser collapser-target-cds32361_39153 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (1169..1470)</strong>: mal
          <div class="collapser collapser-target-cds32361_39153 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds32361_39153-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 79.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds32361_39153-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 222.7</dt>

  <dd></dd><dt>inactive: 102.6</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 60.9</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 52.8</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 29.5</dt>

  <dd></dd><dt>Isobutyryl-CoA: 27.4</dt>

  <dd></dd><dt>Benzoyl-CoA: 25.5</dt>

  <dd></dd><dt>Propionyl-CoA: 24.5</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 20.2</dt>

  <dd></dd><dt>Acetyl-CoA: 18.1</dt>

  <dd></dd><dt>CHC-CoA: 13.0</dt>

  <dd></dd><dt>fatty_acid: 12.2</dt>

  <dd></dd><dt>trans-1,2-CPDA: 5.6</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds47594_50621</span></strong>: mal
      <div class="collapser collapser-target-cds47594_50621 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (1..182)</strong>: mal
          <div class="collapser collapser-target-cds47594_50621 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  (unknown)
              <div class="collapser collapser-target-cds47594_50621-ATSignature collapser-level-none"> </div><div class="collapser-content">
                No matches above 50%
              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds47594_50621-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 117.1</dt>

  <dd></dd><dt>inactive: 70.1</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 35.0</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 22.5</dt>

  <dd></dd><dt>Propionyl-CoA: 5.4</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds50578_61729</span></strong>: mal - mal
      <div class="collapser collapser-target-cds50578_61729 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (1052..1211)</strong>: mal
          <div class="collapser collapser-target-cds50578_61729 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds50578_61729-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>inactive: 66.7%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds50578_61729-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 132.7</dt>

  <dd></dd><dt>inactive: 58.0</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 46.9</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 43.3</dt>

  <dd></dd><dt>Benzoyl-CoA: 19.1</dt>

  <dd></dd><dt>Acetyl-CoA: 18.6</dt>

  <dd></dd><dt>Isobutyryl-CoA: 17.9</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 17.9</dt>

  <dd></dd><dt>Propionyl-CoA: 13.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 13.0</dt>

  <dd></dd><dt>CHC-CoA: 12.8</dt>

  <dd></dd><dt>fatty_acid: 9.2</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (2549..2847)</strong>: mal
          <div class="collapser collapser-target-cds50578_61729 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds50578_61729-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 83.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds50578_61729-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 229.7</dt>

  <dd></dd><dt>inactive: 123.7</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 62.2</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 56.1</dt>

  <dd></dd><dt>Isobutyryl-CoA: 23.3</dt>

  <dd></dd><dt>Propionyl-CoA: 20.5</dt>

  <dd></dd><dt>Acetyl-CoA: 20.2</dt>

  <dd></dd><dt>Benzoyl-CoA: 19.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 14.5</dt>

  <dd></dd><dt>CHC-CoA: 12.7</dt>

  <dd></dd><dt>fatty_acid: 7.5</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 5.3</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds65293_68179</span></strong>: mal
      <div class="collapser collapser-target-cds65293_68179 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (1..185)</strong>: mal
          <div class="collapser collapser-target-cds65293_68179 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  (unknown)
              <div class="collapser collapser-target-cds65293_68179-ATSignature collapser-level-none"> </div><div class="collapser-content">
                No matches above 50%
              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds65293_68179-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 122.7</dt>

  <dd></dd><dt>inactive: 69.0</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 37.7</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 24.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>Propionyl-CoA: 0.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds68449_85390</span></strong>: mmal - mmal - mmal
      <div class="collapser collapser-target-cds68449_85390 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (567..864)</strong>: mmal
          <div class="collapser collapser-target-cds68449_85390 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds68449_85390-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds68449_85390-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 213.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 204.9</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 168.8</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 158.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 134.8</dt>

  <dd></dd><dt>Propionyl-CoA: 116.0</dt>

  <dd></dd><dt>Malonyl-CoA: 100.6</dt>

  <dd></dd><dt>CHC-CoA: 94.4</dt>

  <dd></dd><dt>Isobutyryl-CoA: 75.5</dt>

  <dd></dd><dt>Benzoyl-CoA: 71.0</dt>

  <dd></dd><dt>fatty_acid: 53.0</dt>

  <dd></dd><dt>Acetyl-CoA: 44.7</dt>

  <dd></dd><dt>inactive: 33.7</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 26.8</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (2144..2441)</strong>: mmal
          <div class="collapser collapser-target-cds68449_85390 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds68449_85390-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds68449_85390-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 217.5</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 214.4</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 169.8</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 148.5</dt>

  <dd></dd><dt>trans-1,2-CPDA: 132.5</dt>

  <dd></dd><dt>Propionyl-CoA: 120.8</dt>

  <dd></dd><dt>CHC-CoA: 99.7</dt>

  <dd></dd><dt>Malonyl-CoA: 99.1</dt>

  <dd></dd><dt>Benzoyl-CoA: 71.6</dt>

  <dd></dd><dt>Isobutyryl-CoA: 65.7</dt>

  <dd></dd><dt>Acetyl-CoA: 54.6</dt>

  <dd></dd><dt>fatty_acid: 35.2</dt>

  <dd></dd><dt>inactive: 24.9</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 23.2</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (4287..4584)</strong>: mmal
          <div class="collapser collapser-target-cds68449_85390 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds68449_85390-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 95.8%</dd>
<dd>Methoxymalonyl-CoA: 91.7%</dd>
<dd>Ethylmalonyl-CoA: 87.5%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds68449_85390-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 200.9</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 197.4</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 158.2</dt>

  <dd></dd><dt>trans-1,2-CPDA: 120.3</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 118.6</dt>

  <dd></dd><dt>Propionyl-CoA: 103.6</dt>

  <dd></dd><dt>CHC-CoA: 102.0</dt>

  <dd></dd><dt>Malonyl-CoA: 95.5</dt>

  <dd></dd><dt>Isobutyryl-CoA: 76.6</dt>

  <dd></dd><dt>Benzoyl-CoA: 57.0</dt>

  <dd></dd><dt>Acetyl-CoA: 41.5</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 27.8</dt>

  <dd></dd><dt>fatty_acid: 26.1</dt>

  <dd></dd><dt>inactive: 16.7</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds87293_92438</span></strong>: mal
      <div class="collapser collapser-target-cds87293_92438 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (672..964)</strong>: mal
          <div class="collapser collapser-target-cds87293_92438 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds87293_92438-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 100.0%</dd>
<dd>inactive: 83.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds87293_92438-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 221.9</dt>

  <dd></dd><dt>inactive: 105.1</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 50.0</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 41.9</dt>

  <dd></dd><dt>Propionyl-CoA: 26.4</dt>

  <dd></dd><dt>Acetyl-CoA: 19.1</dt>

  <dd></dd><dt>Benzoyl-CoA: 17.5</dt>

  <dd></dd><dt>fatty_acid: 15.3</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 12.8</dt>

  <dd></dd><dt>Isobutyryl-CoA: 12.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 5.8</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds92422_93265</span></strong>: mal
      <div class="collapser collapser-target-cds92422_93265 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (0..180)</strong>: mal
          <div class="collapser collapser-target-cds92422_93265 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds92422_93265-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 58.3%</dd>
<dd>inactive: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds92422_93265-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 120.9</dt>

  <dd></dd><dt>inactive: 60.2</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 39.4</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 27.0</dt>

  <dd></dd><dt>Propionyl-CoA: 17.5</dt>

  <dd></dd><dt>Acetyl-CoA: 16.1</dt>

  <dd></dd><dt>Benzoyl-CoA: 12.6</dt>

  <dd></dd><dt>Isobutyryl-CoA: 9.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 8.9</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 7.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r147c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00147_NODE_39.. - Region 1 - siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-253"></div> <span class="help-tooltip" id="region_svg-help-253">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00147_NODE_39...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r147c1-svg" data-filename="c00147_NODE_39.._r147c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 29,075 - 42,248 nt. (total: 13,174 nt)
        
        <a class="cluster-rules-header" id="r147c1-rules-header" href="#r147c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r147c1-rules">siderophore: IucA_IucC</div>
      
      <div class="region-svg-container">
       <div id='r147c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-254"></div> <span class="help-tooltip" id="focus-panel-help-254">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r147c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r147c1-clusterblast" data-name="r147c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r147c1-knownclusterblast" data-name="r147c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r147c1-subclusterblast" data-name="r147c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r147c1-MIBiG-cluster-compare" data-name="r147c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r147c1-pfam-details" data-name="r147c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r147c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-95"></div> <span class="help-tooltip" id="cb-general-help-95">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r147c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r147c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r147c1_1.svg">
        NZ_CP039123 (6262702-6275876): Streptomyces sp. SS52 chromosome, complete genome (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r147c1_2.svg">
        NZ_CP027297 (6415484-6428658): Streptomyces sp. SGAir0924 chromosome, complet... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r147c1_3.svg">
        NZ_KZ819185 (584229-597141): Streptomyces sp. CS207 scaffold00002, whole geno... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r147c1_4.svg">
        NZ_CP042278 (1679091-1691990): Streptomyces sp. WAC06273 strain WAC6273 subst... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r147c1_5.svg">
        NZ_CP028834 (1750947-1763790): Streptomyces sp. M2 chromosome, complete genome (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r147c1_6.svg">
        NZ_PZZX01000001 (376626-389467): Streptomyces sp. VMFN-G11Ma Ga0213531 101, w... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r147c1_7.svg">
        NZ_SZNQ01000001 (6157782-6171012): Streptomyces lasalocidi strain X-537 Scaff... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r147c1_8.svg">
        NZ_KQ948550 (324475-337639): Streptomyces longwoodensis strain DSM 41677 PRJN... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r147c1_9.svg">
        NZ_SLZS01000002 (278621-291704): Streptomyces sp. BK335 Ga0307713 102, whole ... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r147c1_10.svg">
        NZ_SLZH01000012 (34230-47272): Streptomyces sp. BK042 Ga0307683 112, whole ge... (92% of genes show similarity), siderophore
      </option>
      
    </select>
    <button id= "clusterblast-r147c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r147c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r147c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-96"></div> <span class="help-tooltip" id="cb-known-help-96">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r147c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r147c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r147c1_1.svg">
        paulomycin
      </option>
      
      <option value = "svg/knownclusterblast_r147c1_2.svg">
        paulomycin
      </option>
      
      <option value = "svg/knownclusterblast_r147c1_3.svg">
        grincamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r147c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r147c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r147c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-97"></div> <span class="help-tooltip" id="cb-sub-help-97">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r147c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-98"></div> <span class="help-tooltip" id="MIBiG-help-98">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r147c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r147c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r147c1"></div>
    <div id="comparison-MIBiG-r147c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">siderophore</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000946.1: 282-7531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000946/index.html#r1c1" target="_new">BGC0000946.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Other</td>
        
        <td>vibrioferrin</td>
        
        
        <td>Vibrio parahaemolyticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000939.1: 1834-15938">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000939/index.html#r1c1" target="_new">BGC0000939.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Other</td>
        
        <td>aerobactin</td>
        
        
        <td>Grimontia hollisae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000947.1: 153-15744">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000947/index.html#r1c1" target="_new">BGC0000947.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Other</td>
        
        <td>vibrioferrin</td>
        
        
        <td>Vibrio alginolyticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.48, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide, Saccharide</td>
        
        <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514.1: 0-37469">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>aristeromycin</td>
        
        
        <td>Streptomyces citricolor</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001374.1: 52-17210">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001374/index.html#r1c1" target="_new">BGC0001374.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Other</td>
        
        <td>thienodolin</td>
        
        
        <td>Streptomyces albogriseolus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001314.1: 14191710-14246296">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001314/index.html#r1c1" target="_new">BGC0001314.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>Terpene</td>
        
        <td>tirucalla</td>
        
        
        <td>Arabidopsis thaliana</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001654.1: 4337975-4357778">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001654/index.html#r1c1" target="_new">BGC0001654.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>Saccharide</td>
        
        <td>lipooligosaccharide</td>
        
        
        <td>Bacteroides thetaiotaomicron VPI-5482</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000277.1: 237-11479">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000277/index.html#r1c1" target="_new">BGC0000277.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.48, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
        <td>0.12</td>
        <td>Polyketide</td>
        
        <td>urdamycin</td>
        
        
        <td>Streptomyces fradiae</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r147c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000946.1: 282-7531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000946/index.html#r1c1" target="_new">BGC0000946.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Other</td>
      <td>vibrioferrin</td>
      <td>Vibrio parahaemolyticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000939.1: 1834-15938">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000939/index.html#r1c1" target="_new">BGC0000939.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Other</td>
      <td>aerobactin</td>
      <td>Grimontia hollisae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000947.1: 153-15744">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000947/index.html#r1c1" target="_new">BGC0000947.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Other</td>
      <td>vibrioferrin</td>
      <td>Vibrio alginolyticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Other</td>
      <td>flaviolin</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.48, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Polyketide, Saccharide</td>
      <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
      <td>Saccharopolyspora erythraea</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514.1: 0-37469">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other</td>
      <td>aristeromycin</td>
      <td>Streptomyces citricolor</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001374.1: 52-17210">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001374/index.html#r1c1" target="_new">BGC0001374.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Other</td>
      <td>thienodolin</td>
      <td>Streptomyces albogriseolus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001314.1: 14191710-14246296">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001314/index.html#r1c1" target="_new">BGC0001314.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>Terpene</td>
      <td>tirucalla</td>
      <td>Arabidopsis thaliana</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001654.1: 4337975-4357778">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001654/index.html#r1c1" target="_new">BGC0001654.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>Saccharide</td>
      <td>lipooligosaccharide</td>
      <td>Bacteroides thetaiotaomicron VPI-5482</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000277.1: 237-11479">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000277/index.html#r1c1" target="_new">BGC0000277.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.40 (id:0.48, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
      <td>0.40</td>
      <td>Polyketide</td>
      <td>urdamycin</td>
      <td>Streptomyces fradiae</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r147c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-99"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-99">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r147c1-pfam-details-svg" data-filename="c00147_NODE_39.._r147c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r147c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r153c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00153_NODE_46.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-255"></div> <span class="help-tooltip" id="region_svg-help-255">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00153_NODE_46...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r153c1-svg" data-filename="c00153_NODE_46.._r153c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 28,082 - 49,866 nt. (total: 21,785 nt)
        
        <a class="cluster-rules-header" id="r153c1-rules-header" href="#r153c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r153c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r153c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-256"></div> <span class="help-tooltip" id="focus-panel-help-256">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r153c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r153c1-clusterblast" data-name="r153c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r153c1-knownclusterblast" data-name="r153c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r153c1-subclusterblast" data-name="r153c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r153c1-MIBiG-cluster-compare" data-name="r153c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r153c1-pfam-details" data-name="r153c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r153c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-100"></div> <span class="help-tooltip" id="cb-general-help-100">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r153c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r153c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r153c1_1.svg">
        NZ_KZ819184 (3998342-4020226): Streptomyces sp. CS207 scaffold00001, whole ge... (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r153c1_2.svg">
        NZ_CP039123 (398269-418830): Streptomyces sp. SS52 chromosome, complete genome (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r153c1_3.svg">
        NZ_CP027297 (508491-529070): Streptomyces sp. SGAir0924 chromosome, complete ... (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r153c1_4.svg">
        NZ_CP042278 (7485676-7507561): Streptomyces sp. WAC06273 strain WAC6273 subst... (90% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r153c1_5.svg">
        NZ_SLVZ01000003 (101179-194010): Streptomyces sp. BK387 Ga0307671 103, whole ... (9% of genes show similarity), T2PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r153c1_6.svg">
        NZ_CP009754 (4774729-4794869): Streptomyces sp. CCM MD2014 chromosome, comple... (19% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r153c1_7.svg">
        NZ_KQ948994 (276268-298086): Streptomyces resistomycificus strain DSM 40133 P... (26% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r153c1_8.svg">
        NZ_CP029241 (1812201-1833226): Streptomyces sp. NHF165 chromosome, complete g... (19% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r153c1_9.svg">
        NZ_CM000951 (7531237-7552547): Streptomyces sviceus ATCC 29083 chromosome, wh... (21% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r153c1_10.svg">
        NZ_VSKT01000015 (394029-415881): Streptomyces cacaoi strain OABC16 Scaffold15... (21% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r153c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r153c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r153c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-101"></div> <span class="help-tooltip" id="cb-known-help-101">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r153c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r153c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r153c1_1.svg">
        lysolipin I
      </option>
      
      <option value = "svg/knownclusterblast_r153c1_2.svg">
        arylomycin
      </option>
      
    </select>
    <button id="knownclusterblast-r153c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r153c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r153c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-102"></div> <span class="help-tooltip" id="cb-sub-help-102">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r153c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-103"></div> <span class="help-tooltip" id="MIBiG-help-103">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r153c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r153c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r153c1"></div>
    <div id="comparison-MIBiG-r153c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001184.1: 3593876-3601174">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001184/index.html#r1c1" target="_new">BGC0001184.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>bacilysin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888.1: 0-4342">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>bacilysin</td>
        
        
        <td>Bacillus sp. CS93</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076.1: 161645-164819">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Other</td>
        
        <td>4,5-Dehydroacaterin, Acaterin</td>
        
        
        <td>Pseudomonas jessenii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000636.1: 2-14167">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000636/index.html#r1c1" target="_new">BGC0000636.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.50, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Brevibacterium linens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>frankiamicin</td>
        
        
        <td>Frankia sp. EAN1pec</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001927.1: 0-27705">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001927/index.html#r1c1" target="_new">BGC0001927.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Other</td>
        
        <td>A-94964</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.44, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>granaticin</td>
        
        
        <td>Streptomyces violaceoruber</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000509.1: 4195-15261">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000509/index.html#r1c1" target="_new">BGC0000509.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>RiPP</td>
        
        <td>epilancin 15x</td>
        
        
        <td>Staphylococcus epidermidis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Polyketide</td>
        
        <td>jadomycin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000218.1: 27-8720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000218/index.html#r1c1" target="_new">BGC0000218.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Polyketide</td>
        
        <td>doxorubicin</td>
        
        
        <td>Streptomyces peucetius</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r153c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001184.1: 3593876-3601174">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001184/index.html#r1c1" target="_new">BGC0001184.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>bacilysin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888.1: 0-4342">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>bacilysin</td>
      <td>Bacillus sp. CS93</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076.1: 161645-164819">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Other</td>
      <td>4,5-Dehydroacaterin, Acaterin</td>
      <td>Pseudomonas jessenii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000636.1: 2-14167">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000636/index.html#r1c1" target="_new">BGC0000636.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.50, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Brevibacterium linens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>frankiamicin</td>
      <td>Frankia sp. EAN1pec</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001927.1: 0-27705">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001927/index.html#r1c1" target="_new">BGC0001927.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Other</td>
      <td>A-94964</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.44, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>granaticin</td>
      <td>Streptomyces violaceoruber</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000509.1: 4195-15261">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000509/index.html#r1c1" target="_new">BGC0000509.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>RiPP</td>
      <td>epilancin 15x</td>
      <td>Staphylococcus epidermidis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Polyketide</td>
      <td>jadomycin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000218.1: 27-8720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000218/index.html#r1c1" target="_new">BGC0000218.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Polyketide</td>
      <td>doxorubicin</td>
      <td>Streptomyces peucetius</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r153c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-104"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-104">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r153c1-pfam-details-svg" data-filename="c00153_NODE_46.._r153c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r153c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r160c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00160_NODE_14.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-257"></div> <span class="help-tooltip" id="region_svg-help-257">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00160_NODE_14...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r160c1-svg" data-filename="c00160_NODE_14.._r160c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 4,770 - 19,751 nt. (total: 14,982 nt)
        
        <a class="cluster-rules-header" id="r160c1-rules-header" href="#r160c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r160c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r160c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-258"></div> <span class="help-tooltip" id="focus-panel-help-258">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r160c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r160c1-clusterblast" data-name="r160c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r160c1-knownclusterblast" data-name="r160c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r160c1-subclusterblast" data-name="r160c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r160c1-MIBiG-cluster-compare" data-name="r160c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r160c1-pfam-details" data-name="r160c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r160c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-105"></div> <span class="help-tooltip" id="cb-general-help-105">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r160c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r160c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r160c1_1.svg">
        NZ_KZ819184 (3584225-3618334): Streptomyces sp. CS207 scaffold00001, whole ge... (38% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r160c1_2.svg">
        NZ_CP027297 (879867-913753): Streptomyces sp. SGAir0924 chromosome, complete ... (37% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r160c1_3.svg">
        NZ_CP039123 (803279-837373): Streptomyces sp. SS52 chromosome, complete genome (38% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r160c1_4.svg">
        NZ_LIPF01000007 (539754-574642): Streptomyces sp. CB02414 scaffold7, whole ge... (40% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r160c1_5.svg">
        NZ_CP042278 (7075905-7099325): Streptomyces sp. WAC06273 strain WAC6273 subst... (57% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r160c1_6.svg">
        NZ_KZ195577 (7640490-7663727): Streptomyces sp. CS159 scaffold00001, whole ge... (52% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r160c1_7.svg">
        NZ_WHZR01000001 (7566803-7589989): Streptomyces sp. OM5714 A912 AS4 SC1 R2, w... (52% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r160c1_8.svg">
        NZ_CP045547 (7830327-7853529): Streptomyces sp. SYP-A7193 chromosome, complet... (50% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r160c1_9.svg">
        NZ_KZ195574 (8147839-8171950): Streptomyces sp. CS113 scaffold00001, whole ge... (52% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r160c1_10.svg">
        NZ_CP009754 (7904176-7928198): Streptomyces sp. CCM MD2014 chromosome, comple... (50% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r160c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r160c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r160c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-106"></div> <span class="help-tooltip" id="cb-known-help-106">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r160c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r160c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r160c1_1.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r160c1_2.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r160c1_3.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r160c1_4.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r160c1_5.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r160c1_6.svg">
        nocamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r160c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r160c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r160c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-107"></div> <span class="help-tooltip" id="cb-sub-help-107">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r160c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-108"></div> <span class="help-tooltip" id="MIBiG-help-108">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r160c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r160c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r160c1"></div>
    <div id="comparison-MIBiG-r160c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.56, order:0.62, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.38</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Terpene</td>
        
        <td>Î²-carotein</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Rhodobacter sphaeroides</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.58, order:0.49, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656.1: 169-3770">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Terpene</td>
        
        <td>zeaxanthin</td>
        
        
        <td>Xanthobacter autotrophicus Py2</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.58, order:0.53, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.57, order:0.49, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces collinus Tu 365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.56, order:0.58, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000642.1: 1771-9054">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000642/index.html#r1c1" target="_new">BGC0000642.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Enterobacteriaceae bacterium DC413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000638.1: 224-6452">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000638/index.html#r1c1" target="_new">BGC0000638.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Pantoea ananatis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r160c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.56, order:0.62, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Terpene</td>
      <td>Î²-carotein</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Rhodobacter sphaeroides</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.58, order:0.49, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656.1: 169-3770">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>zeaxanthin</td>
      <td>Xanthobacter autotrophicus Py2</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.58, order:0.53, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.57, order:0.49, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces collinus Tu 365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.56, order:0.58, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000642.1: 1771-9054">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000642/index.html#r1c1" target="_new">BGC0000642.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Enterobacteriaceae bacterium DC413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000638.1: 224-6452">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000638/index.html#r1c1" target="_new">BGC0000638.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Pantoea ananatis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r160c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-109"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-109">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r160c1-pfam-details-svg" data-filename="c00160_NODE_14.._r160c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r160c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r161c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00161_NODE_7_.. - Region 1 - indole <div class="help-container"> <div class="help-icon" data-id="region_svg-help-259"></div> <span class="help-tooltip" id="region_svg-help-259">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00161_NODE_7_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r161c1-svg" data-filename="c00161_NODE_7_.._r161c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 81,865 - 102,986 nt. (total: 21,122 nt)
        
        <a class="cluster-rules-header" id="r161c1-rules-header" href="#r161c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r161c1-rules">indole: (indsynth or dmat or indole_PTase)</div>
      
      <div class="region-svg-container">
       <div id='r161c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-260"></div> <span class="help-tooltip" id="focus-panel-help-260">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r161c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r161c1-clusterblast" data-name="r161c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r161c1-knownclusterblast" data-name="r161c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r161c1-subclusterblast" data-name="r161c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r161c1-MIBiG-cluster-compare" data-name="r161c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r161c1-pfam-details" data-name="r161c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r161c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-110"></div> <span class="help-tooltip" id="cb-general-help-110">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r161c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r161c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r161c1_1.svg">
        NZ_CP039123 (722055-743177): Streptomyces sp. SS52 chromosome, complete genome (100% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r161c1_2.svg">
        NZ_KZ819184 (3677716-3698086): Streptomyces sp. CS207 scaffold00001, whole ge... (100% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r161c1_3.svg">
        NZ_CP027297 (800059-821181): Streptomyces sp. SGAir0924 chromosome, complete ... (100% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r161c1_4.svg">
        NZ_CP042278 (7157989-7179112): Streptomyces sp. WAC06273 strain WAC6273 subst... (100% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r161c1_5.svg">
        NZ_SOAV01000016 (50424-71579): Streptomyces sp. BK208 Ga0307686 116, whole ge... (90% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r161c1_6.svg">
        NZ_JAAIFS010000005 (130509-151637): Streptomyces tendae strain VITAKN NODE 5 ... (90% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r161c1_7.svg">
        NZ_RSEG01000018 (86367-107495): Streptomyces tendae strain UTMC 3329 Scaffold... (90% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r161c1_8.svg">
        NZ_KZ195574 (8224620-8245748): Streptomyces sp. CS113 scaffold00001, whole ge... (95% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r161c1_9.svg">
        NZ_BBOK01000016 (53332-74460): Streptomyces sp. NBRC 110030, whole genome sho... (82% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r161c1_10.svg">
        NZ_CP036534 (467587-488715): Streptomyces sp. VN1 chromosome, complete genome (90% of genes show similarity), indole
      </option>
      
    </select>
    <button id= "clusterblast-r161c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r161c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r161c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-111"></div> <span class="help-tooltip" id="cb-known-help-111">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r161c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r161c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r161c1_1.svg">
        5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester
      </option>
      
      <option value = "svg/knownclusterblast_r161c1_2.svg">
        JBIR-06
      </option>
      
      <option value = "svg/knownclusterblast_r161c1_3.svg">
        antimycin
      </option>
      
      <option value = "svg/knownclusterblast_r161c1_4.svg">
        splenocin
      </option>
      
      <option value = "svg/knownclusterblast_r161c1_5.svg">
        neoantimycin
      </option>
      
      <option value = "svg/knownclusterblast_r161c1_6.svg">
        neoantimycin
      </option>
      
      <option value = "svg/knownclusterblast_r161c1_7.svg">
        ravidomycin
      </option>
      
    </select>
    <button id="knownclusterblast-r161c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r161c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r161c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-112"></div> <span class="help-tooltip" id="cb-sub-help-112">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r161c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-113"></div> <span class="help-tooltip" id="MIBiG-help-113">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r161c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r161c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r161c1"></div>
    <div id="comparison-MIBiG-r161c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">indole</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294.1: 26867-35886">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Other</td>
        
        <td>7-prenylisatin</td>
        
        
        <td>Streptomyces sp. MBT28</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001191.1: 21327-37853">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001191/index.html#r1c1" target="_new">BGC0001191.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>A-102395</td>
        
        
        <td>Amycolatopsis sp. SANK 60206</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>venezuelin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.56, order:0.35, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester</td>
        
        
        <td>Streptomyces sp. RM-5-8</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001525.1: 0-16392">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001525/index.html#r1c1" target="_new">BGC0001525.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>bartoloside 2, bartoloside 3, bartoloside 4</td>
        
        
        <td>Synechocystis salina LEGE 06155</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001526.1: 0-14417">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001526/index.html#r1c1" target="_new">BGC0001526.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>bartolosides E, bartolosides F, bartolosides G, bartolosides H, bartolosides I, bartolosides J, bartolosides K</td>
        
        
        <td>Synechocystis salina LEGE 06099</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
        
        
        <td>Actinomadura namibiensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000706.1: 350-14906">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000706/index.html#r1c1" target="_new">BGC0000706.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001579.1: 85465-94303">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001579/index.html#r1c1" target="_new">BGC0001579.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>duramycin</td>
        
        
        <td>Streptomyces cinnamoneus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r161c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294.1: 26867-35886">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>7-prenylisatin</td>
      <td>Streptomyces sp. MBT28</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001191.1: 21327-37853">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001191/index.html#r1c1" target="_new">BGC0001191.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Other</td>
      <td>A-102395</td>
      <td>Amycolatopsis sp. SANK 60206</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>venezuelin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.56, order:0.35, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester</td>
      <td>Streptomyces sp. RM-5-8</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001525.1: 0-16392">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001525/index.html#r1c1" target="_new">BGC0001525.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>bartoloside 2, bartoloside 3, bartoloside 4</td>
      <td>Synechocystis salina LEGE 06155</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001526.1: 0-14417">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001526/index.html#r1c1" target="_new">BGC0001526.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>bartolosides E, bartolosides F, bartolosides G, bartolosides H, bartolosides I, bartolosides J, bartolosides K</td>
      <td>Synechocystis salina LEGE 06099</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
      <td>Actinomadura namibiensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000706.1: 350-14906">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000706/index.html#r1c1" target="_new">BGC0000706.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001579.1: 85465-94303">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001579/index.html#r1c1" target="_new">BGC0001579.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>duramycin</td>
      <td>Streptomyces cinnamoneus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r161c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-114"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-114">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r161c1-pfam-details-svg" data-filename="c00161_NODE_7_.._r161c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r161c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r168c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00168_NODE_99.. - Region 1 - T2PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-261"></div> <span class="help-tooltip" id="region_svg-help-261">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00168_NODE_99...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r168c1-svg" data-filename="c00168_NODE_99.._r168c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 40,013 nt. (total: 40,013 nt)
        
        <a class="cluster-rules-header" id="r168c1-rules-header" href="#r168c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r168c1-rules">T2PKS: (t2ks and t2clf)</div>
      
      <div class="region-svg-container">
       <div id='r168c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-262"></div> <span class="help-tooltip" id="focus-panel-help-262">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r168c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r168c1-nrps_pks" data-name="r168c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r168c1-clusterblast" data-name="r168c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r168c1-knownclusterblast" data-name="r168c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r168c1-subclusterblast" data-name="r168c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r168c1-MIBiG-cluster-compare" data-name="r168c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r168c1-pfam-details" data-name="r168c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r168c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-115"></div> <span class="help-tooltip" id="nrps-domain-help-115">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r168c1-details-svg" data-filename="c00168_NODE_99.._r168c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r168c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r168c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-116"></div> <span class="help-tooltip" id="cb-general-help-116">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r168c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r168c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r168c1_1.svg">
        NZ_CP042278 (2571712-2644309): Streptomyces sp. WAC06273 strain WAC6273 subst... (59% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r168c1_2.svg">
        NZ_KZ819184 (731930-800153): Streptomyces sp. CS207 scaffold00001, whole geno... (59% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r168c1_3.svg">
        NZ_CP039123 (5308335-5380932): Streptomyces sp. SS52 chromosome, complete genome (69% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r168c1_4.svg">
        NZ_CP027297 (5389991-5462588): Streptomyces sp. SGAir0924 chromosome, complet... (57% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r168c1_5.svg">
        NZ_LIPF01000002 (65767-138313): Streptomyces sp. CB02414 scaffold2, whole gen... (54% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r168c1_6.svg">
        NZ_KZ195574 (2867534-2936491): Streptomyces sp. CS113 scaffold00001, whole ge... (58% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r168c1_7.svg">
        NZ_BEWD01000008 (380990-453533): Streptomyces rubrogriseus strain NBRC 15455,... (56% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r168c1_8.svg">
        NZ_SOPG01000007 (264961-337498): Streptomyces sp. H23 NODE 7 length 422791 co... (60% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r168c1_9.svg">
        NZ_CP054920 (2707128-2779665): Streptomyces sp. NA03103 chromosome, complete ... (54% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r168c1_10.svg">
        NZ_CP034353 (2714569-2787106): Streptomyces sp. KPB2 chromosome, complete genome (59% of genes show similarity), T2PKS
      </option>
      
    </select>
    <button id= "clusterblast-r168c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r168c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r168c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-117"></div> <span class="help-tooltip" id="cb-known-help-117">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r168c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r168c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r168c1_1.svg">
        spore pigment
      </option>
      
      <option value = "svg/knownclusterblast_r168c1_2.svg">
        curamycin
      </option>
      
      <option value = "svg/knownclusterblast_r168c1_3.svg">
        spore pigment
      </option>
      
      <option value = "svg/knownclusterblast_r168c1_4.svg">
        FD-594
      </option>
      
      <option value = "svg/knownclusterblast_r168c1_5.svg">
        anthrabenzoxocinone
      </option>
      
      <option value = "svg/knownclusterblast_r168c1_6.svg">
        xantholipin
      </option>
      
      <option value = "svg/knownclusterblast_r168c1_7.svg">
        WS79089A / hexaricin B / hexaricin C
      </option>
      
      <option value = "svg/knownclusterblast_r168c1_8.svg">
        griseorhodin A
      </option>
      
      <option value = "svg/knownclusterblast_r168c1_9.svg">
        lysolipin I
      </option>
      
      <option value = "svg/knownclusterblast_r168c1_10.svg">
        arixanthomycin A / arixanthomycin B / arixanthomycin C
      </option>
      
    </select>
    <button id="knownclusterblast-r168c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r168c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r168c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-118"></div> <span class="help-tooltip" id="cb-sub-help-118">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r168c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r168c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r168c1_1.svg">
        AF324838_c1: simocyclinone t2pks (50% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r168c1_2.svg">
        AF080235_c1: landomycin t2pks (50% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r168c1_3.svg">
        FJ483966_c1: polyketomycin t2pks (25% of genes show similarity), t2pks
      </option>
      
    </select>
    <button id="subclusterblast-r168c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r168c1-svg"></div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r168c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-120"></div> <span class="help-tooltip" id="MIBiG-help-120">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r168c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r168c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r168c1"></div>
    <div id="comparison-MIBiG-r168c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T2PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.57, order:0.79, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
        <td>0.42</td>
        <td>Polyketide</td>
        
        <td>curamycin</td>
        
        
        <td>Streptomyces cyaneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.57, order:0.66, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.41</td>
        <td>Polyketide</td>
        
        <td>spore pigment</td>
        
        
        <td>Streptomyces collinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000271.1: 0-11118">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000271/index.html#r1c1" target="_new">BGC0000271.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.57, order:0.66, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Polyketide</td>
        
        <td>spore pigment</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.53, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Polyketide</td>
        
        <td>granaticin</td>
        
        
        <td>Streptomyces violaceoruber</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.51, order:0.50, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>Polyketide</td>
        
        <td>naphtocyclinone</td>
        
        
        <td>Streptomyces arenae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.51, order:0.51, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>Polyketide</td>
        
        <td>griseusin</td>
        
        
        <td>Streptomyces griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001513.1: 120-11698">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001513/index.html#r1c1" target="_new">BGC0001513.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.53, order:0.46, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>Polyketide</td>
        
        <td>anthrabenzoxocinone</td>
        
        
        <td>Actinobacteria bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000264.1: 255-15873">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000264/index.html#r1c1" target="_new">BGC0000264.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.48, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>resistomycin, resistoflavine</td>
        
        
        <td>Streptomyces resistomycificus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001366.1: 10-15268">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001366/index.html#r1c1" target="_new">BGC0001366.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.54, order:0.39, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>BE-24566B, zunyimycin A</td>
        
        
        <td>Streptomyces sp. FJS31-2</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.52, order:0.42, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Polyketide</td>
        
        <td>jadomycin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r168c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.57, order:0.79, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>Polyketide</td>
      <td>curamycin</td>
      <td>Streptomyces cyaneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.57, order:0.66, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Polyketide</td>
      <td>spore pigment</td>
      <td>Streptomyces collinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000271.1: 0-11118">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000271/index.html#r1c1" target="_new">BGC0000271.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.57, order:0.66, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>spore pigment</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.53, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Polyketide</td>
      <td>granaticin</td>
      <td>Streptomyces violaceoruber</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.51, order:0.50, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>naphtocyclinone</td>
      <td>Streptomyces arenae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.51, order:0.51, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>griseusin</td>
      <td>Streptomyces griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001513.1: 120-11698">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001513/index.html#r1c1" target="_new">BGC0001513.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.53, order:0.46, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>anthrabenzoxocinone</td>
      <td>Actinobacteria bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000264.1: 255-15873">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000264/index.html#r1c1" target="_new">BGC0000264.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.48, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>resistomycin, resistoflavine</td>
      <td>Streptomyces resistomycificus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001366.1: 10-15268">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001366/index.html#r1c1" target="_new">BGC0001366.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.54, order:0.39, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>BE-24566B, zunyimycin A</td>
      <td>Streptomyces sp. FJS31-2</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.52, order:0.42, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>jadomycin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r168c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-121"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-121">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r168c1-pfam-details-svg" data-filename="c00168_NODE_99.._r168c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r168c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
      <div class="sidepanel-details-header r168c1-t2pks" data-name="r168c1-t2pks">
        Type II PKS
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    <div class="sidepanel-details-section r168c1-t2pks">
      <div class="more-details">
    <div class="heading">
      <span>Type II PKS gene cluster prediction details</span>
      <div class="help-container"> <div class="help-icon" data-id="t2pks-help-119"></div> <span class="help-tooltip" id="t2pks-help-119">Predictions of starter units, elongations, product classes, and potential molecular weights for type II PKS clusters.<br>More detailed information is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/t2pks' target='_blank'>here</a>.</span></div>
    </div>
	
	
	<dl class="prediction-text">
        <dt><strong>Cluster section:</strong></dt>
        <dd>0 to 40013</dd>
		
		<dt><strong>Starter unit(s):</strong></dt>
		<dd>
				acetyl-CoA (Score: 0.0; E-value: 0)<br>
			
	    <dd>
		</tr>
		
		
		
		<dt><strong>Product class(es):</strong></td>
        <dd>
				
				pentangular polyphenol<br>
				
				tetracenomycin<br>
				
		</dd>
		
		
	</dl>
	<dl class="prediction-text">
		
		<dt><strong>cds32088_32424</strong></dt>
			
			<dd>
				CYC
				
					C4-C21/C2-C23|C2-C19
				
				(Score: 153.4; E-value: 4.1e-48)<br>
			</dd>
			
		
		<dt><strong>cds32484_32964</strong></dt>
			
			<dd>
				CYC
				
					C9-C14
				
				(Score: 258.3; E-value: 4.3e-80)<br>
			</dd>
			
		
		<dt><strong>cds32965_33235</strong></dt>
			
			<dd>
				ACP
				
				(Score: 73.9; E-value: 1.8e-23)<br>
			</dd>
			
		
		<dt><strong>cds33899_34655</strong></dt>
			
			<dd>
				CLF
				
				(Score: 302.0; E-value: 1.5e-92)<br>
			</dd>
			
		
		<dt><strong>cds34677_35922</strong></dt>
			
			<dd>
				KS
				
				(Score: 647.6; E-value: 1.8e-197)<br>
			</dd>
			
		
		<dt><strong>cds35918_36386</strong></dt>
			
			<dd>
				CYC
				
					C5-C18
				
				(Score: 210.0; E-value: 2.2e-65)<br>
			</dd>
			
		
		<dt><strong>cds38187_40011</strong></dt>
			
			<dd>
				OXY
				
				(Score: 281.5; E-value: 3.3e-86)<br>
			</dd>
			
		
	</dl>
    
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r169c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00169_NODE_16.. - Region 1 - T2PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-263"></div> <span class="help-tooltip" id="region_svg-help-263">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00169_NODE_16...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r169c1-svg" data-filename="c00169_NODE_16.._r169c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 10,438 nt. (total: 10,438 nt)
        
        <a class="cluster-rules-header" id="r169c1-rules-header" href="#r169c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r169c1-rules">T2PKS: (t2ks and t2clf)</div>
      
      <div class="region-svg-container">
       <div id='r169c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-264"></div> <span class="help-tooltip" id="focus-panel-help-264">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r169c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r169c1-nrps_pks" data-name="r169c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r169c1-clusterblast" data-name="r169c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r169c1-knownclusterblast" data-name="r169c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r169c1-subclusterblast" data-name="r169c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r169c1-MIBiG-cluster-compare" data-name="r169c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r169c1-pfam-details" data-name="r169c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r169c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-122"></div> <span class="help-tooltip" id="nrps-domain-help-122">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r169c1-details-svg" data-filename="c00169_NODE_16.._r169c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r169c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r169c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-123"></div> <span class="help-tooltip" id="cb-general-help-123">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r169c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r169c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r169c1_1.svg">
        NZ_CP027297 (19263-103747): Streptomyces sp. SGAir0924 chromosome, complete g... (16% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r169c1_2.svg">
        NZ_KZ819184 (4102654-4188512): Streptomyces sp. CS207 scaffold00001, whole ge... (16% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r169c1_3.svg">
        NZ_CP012382 (127761-213859): Streptomyces ambofaciens ATCC 23877 chromosome, ... (15% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r169c1_4.svg">
        NZ_CP012382 (8089906-8176435): Streptomyces ambofaciens ATCC 23877 chromosome... (14% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r169c1_5.svg">
        NZ_CP012949 (140374-225289): Streptomyces ambofaciens strain DSM 40697 chromo... (14% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r169c1_6.svg">
        NZ_CP012949 (7911727-7998131): Streptomyces ambofaciens strain DSM 40697 chro... (13% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r169c1_7.svg">
        NZ_CP029241 (7374976-7447633): Streptomyces sp. NHF165 chromosome, complete g... (13% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r169c1_8.svg">
        NZ_CP047140 (110406-298596): Streptomyces sp. Tu 2975 chromosome, complete ge... (8% of genes show similarity), NRPS,PKS-like,T2PKS,ectoine,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r169c1_9.svg">
        NZ_CP030864 (11545-320938): Streptomyces globosus strain LZH-48 plasmid unnam... (4% of genes show similarity), NRPS,NRPS-like,T2PKS,butyrolactone,ectoine,hglE-KS,linaridin,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r169c1_10.svg">
        NZ_CP015867 (893-391629): Streptomyces parvulus strain 2297 plasmid pSPA1, co... (5% of genes show similarity), NRPS,NRPS-like,T1PKS,T2PKS,betalactone,butyrolactone,other
      </option>
      
    </select>
    <button id= "clusterblast-r169c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r169c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r169c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-124"></div> <span class="help-tooltip" id="cb-known-help-124">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r169c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r169c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r169c1_1.svg">
        griseorhodin A
      </option>
      
      <option value = "svg/knownclusterblast_r169c1_2.svg">
        mithramycin
      </option>
      
      <option value = "svg/knownclusterblast_r169c1_3.svg">
        fluostatins M-Q
      </option>
      
      <option value = "svg/knownclusterblast_r169c1_4.svg">
        kinamycin
      </option>
      
      <option value = "svg/knownclusterblast_r169c1_5.svg">
        fluostatin
      </option>
      
      <option value = "svg/knownclusterblast_r169c1_6.svg">
        rabelomycin / dehydrorabelomycin / fluostatin F / fluostatin G / fluostatin H
      </option>
      
      <option value = "svg/knownclusterblast_r169c1_7.svg">
        erdacin
      </option>
      
      <option value = "svg/knownclusterblast_r169c1_8.svg">
        hiroshidine
      </option>
      
      <option value = "svg/knownclusterblast_r169c1_9.svg">
        fogacin A / fogacin B / fogacin C
      </option>
      
      <option value = "svg/knownclusterblast_r169c1_10.svg">
        mithramycin
      </option>
      
    </select>
    <button id="knownclusterblast-r169c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r169c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r169c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-125"></div> <span class="help-tooltip" id="cb-sub-help-125">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r169c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r169c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r169c1_1.svg">
        AF324838_c1: simocyclinone t2pks (33% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r169c1_2.svg">
        AF080235_c1: landomycin t2pks (33% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r169c1_3.svg">
        FJ483966_c1: polyketomycin t2pks (25% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r169c1_4.svg">
        AB103463_c1: medermycin t2pks (50% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r169c1_5.svg">
        GU233672_c1: granaticin t2pks (28% of genes show similarity), t2pks
      </option>
      
    </select>
    <button id="subclusterblast-r169c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r169c1-svg"></div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r169c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-127"></div> <span class="help-tooltip" id="MIBiG-help-127">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r169c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r169c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r169c1"></div>
    <div id="comparison-MIBiG-r169c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T2PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.52, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>naphtocyclinone</td>
        
        
        <td>Streptomyces arenae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.50, order:0.45, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>spore pigment</td>
        
        
        <td>Streptomyces collinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.51, order:0.48, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>granaticin</td>
        
        
        <td>Streptomyces violaceoruber</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.50, order:0.45, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>curamycin</td>
        
        
        <td>Streptomyces cyaneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.50, order:0.45, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>griseusin</td>
        
        
        <td>Streptomyces griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.50, order:0.42, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>jadomycin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.51, order:0.40, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>frankiamicin</td>
        
        
        <td>Frankia sp. EAN1pec</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001366.1: 10-15268">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001366/index.html#r1c1" target="_new">BGC0001366.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.53, order:0.40, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>BE-24566B, zunyimycin A</td>
        
        
        <td>Streptomyces sp. FJS31-2</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000275.1: 58-12211">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000275/index.html#r1c1" target="_new">BGC0000275.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.53, order:0.40, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>tetracenomycin C</td>
        
        
        <td>Streptomyces glaucescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001513.1: 120-11698">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001513/index.html#r1c1" target="_new">BGC0001513.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.53, order:0.40, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>anthrabenzoxocinone</td>
        
        
        <td>Actinobacteria bacterium</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r169c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.52, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>naphtocyclinone</td>
      <td>Streptomyces arenae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.50, order:0.45, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>spore pigment</td>
      <td>Streptomyces collinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.51, order:0.48, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>granaticin</td>
      <td>Streptomyces violaceoruber</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.50, order:0.45, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>curamycin</td>
      <td>Streptomyces cyaneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.50, order:0.45, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>griseusin</td>
      <td>Streptomyces griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.50, order:0.42, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Polyketide</td>
      <td>jadomycin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.51, order:0.40, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>frankiamicin</td>
      <td>Frankia sp. EAN1pec</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001366.1: 10-15268">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001366/index.html#r1c1" target="_new">BGC0001366.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.53, order:0.40, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>BE-24566B, zunyimycin A</td>
      <td>Streptomyces sp. FJS31-2</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000275.1: 58-12211">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000275/index.html#r1c1" target="_new">BGC0000275.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.53, order:0.40, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Polyketide</td>
      <td>tetracenomycin C</td>
      <td>Streptomyces glaucescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001513.1: 120-11698">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001513/index.html#r1c1" target="_new">BGC0001513.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.53, order:0.40, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Polyketide</td>
      <td>anthrabenzoxocinone</td>
      <td>Actinobacteria bacterium</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r169c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-128"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-128">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r169c1-pfam-details-svg" data-filename="c00169_NODE_16.._r169c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r169c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
      <div class="sidepanel-details-header r169c1-t2pks" data-name="r169c1-t2pks">
        Type II PKS
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    <div class="sidepanel-details-section r169c1-t2pks">
      <div class="more-details">
    <div class="heading">
      <span>Type II PKS gene cluster prediction details</span>
      <div class="help-container"> <div class="help-icon" data-id="t2pks-help-126"></div> <span class="help-tooltip" id="t2pks-help-126">Predictions of starter units, elongations, product classes, and potential molecular weights for type II PKS clusters.<br>More detailed information is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/t2pks' target='_blank'>here</a>.</span></div>
    </div>
	
	
	<dl class="prediction-text">
        <dt><strong>Cluster section:</strong></dt>
        <dd>0 to 10438</dd>
		
		<dt><strong>Starter unit(s):</strong></dt>
		<dd>
				acetyl-CoA (Score: 0.0; E-value: 0)<br>
			
	    <dd>
		</tr>
		
		
		<dt><strong>Malonyl elongations:</strong></dt>
        <dd>
			
			8|9 (Score: 582.6; E-value: 8.2e-178)<br>
			
    	</dd>
		
		
		<dt><strong>Product class(es):</strong></td>
        <dd>
				
				angucycline<br>
				
				anthracycline<br>
				
				aureolic acid<br>
				
				tetracycline<br>
				
		</dd>
		
		
		<dt><strong>Product molecular weight(s):</strong></dt>
		<dd>
			
			acetyl-CoA_8: 363.34 Da<br>
			
			acetyl-CoA_9: 405.38 Da<br>
			
		</dd>
		
	</dl>
	<dl class="prediction-text">
		
		<dt><strong>cds2041_3307</strong></dt>
			
			<dd>
				CLF
				
					8|9
				
				(Score: 582.6; E-value: 8.2e-178)<br>
			</dd>
			
		
		<dt><strong>cds4192_5113</strong></dt>
			
			<dd>
				CYC
				
					C7-C12
				
				(Score: 260.5; E-value: 3.7e-80)<br>
			</dd>
			
		
		<dt><strong>cds773_2045</strong></dt>
			
			<dd>
				KS
				
				(Score: 659.8; E-value: 3.5e-201)<br>
			</dd>
			
		
	</dl>
    
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r175c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00175_NODE_49.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-265"></div> <span class="help-tooltip" id="region_svg-help-265">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00175_NODE_49...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r175c1-svg" data-filename="c00175_NODE_49.._r175c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 49,804 - 100,801 nt. (total: 50,998 nt)
        
        <a class="cluster-rules-header" id="r175c1-rules-header" href="#r175c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r175c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r175c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-266"></div> <span class="help-tooltip" id="focus-panel-help-266">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r175c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r175c1-nrps_pks" data-name="r175c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r175c1-clusterblast" data-name="r175c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r175c1-knownclusterblast" data-name="r175c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r175c1-subclusterblast" data-name="r175c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r175c1-MIBiG-cluster-compare" data-name="r175c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r175c1-pfam-details" data-name="r175c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r175c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-129"></div> <span class="help-tooltip" id="nrps-domain-help-129">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r175c1-details-svg" data-filename="c00175_NODE_49.._r175c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r175c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r175c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-132"></div> <span class="help-tooltip" id="cb-general-help-132">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r175c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r175c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r175c1_1.svg">
        NZ_CP039123 (7658917-7709431): Streptomyces sp. SS52 chromosome, complete genome (94% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r175c1_2.svg">
        NZ_LIPF01000004 (95222-146124): Streptomyces sp. CB02414 scaffold4, whole gen... (64% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r175c1_3.svg">
        NZ_CP015866 (6874608-6925528): Streptomyces parvulus strain 2297 chromosome, ... (54% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r175c1_4.svg">
        NZ_CP012949 (7365607-7435627): Streptomyces ambofaciens strain DSM 40697 chro... (35% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r175c1_5.svg">
        NZ_CP019724 (355288-475720): Streptomyces pactum strain ACT12 chromosome, com... (21% of genes show similarity), NRPS,T1PKS,lanthidin
      </option>
      
      <option value = "svg/clusterblast_r175c1_6.svg">
        NZ_CP036534 (7898916-7949836): Streptomyces sp. VN1 chromosome, complete genome (51% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r175c1_7.svg">
        NZ_CP016795 (7767318-7818238): Streptomyces olivaceus strain KLBMP 5084 chrom... (51% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r175c1_8.svg">
        NZ_CP012382 (7565298-7616302): Streptomyces ambofaciens ATCC 23877 chromosome... (48% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r175c1_9.svg">
        NZ_BBOK01000008 (93273-142530): Streptomyces sp. NBRC 110030, whole genome sh... (54% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r175c1_10.svg">
        NZ_MQUH01000001 (362133-413155): Streptomyces sp. M1013 M1013 C01, whole geno... (44% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r175c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r175c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r175c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-133"></div> <span class="help-tooltip" id="cb-known-help-133">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r175c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r175c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r175c1_1.svg">
        coelichelin
      </option>
      
      <option value = "svg/knownclusterblast_r175c1_2.svg">
        borrelidin
      </option>
      
      <option value = "svg/knownclusterblast_r175c1_3.svg">
        borrelidin
      </option>
      
      <option value = "svg/knownclusterblast_r175c1_4.svg">
        gobichelin A / gobichelin B
      </option>
      
      <option value = "svg/knownclusterblast_r175c1_5.svg">
        scabichelin
      </option>
      
      <option value = "svg/knownclusterblast_r175c1_6.svg">
        salinichelins
      </option>
      
      <option value = "svg/knownclusterblast_r175c1_7.svg">
        albachelin
      </option>
      
      <option value = "svg/knownclusterblast_r175c1_8.svg">
        qinichelins
      </option>
      
      <option value = "svg/knownclusterblast_r175c1_9.svg">
        erythrochelin
      </option>
      
      <option value = "svg/knownclusterblast_r175c1_10.svg">
        amychelin
      </option>
      
    </select>
    <button id="knownclusterblast-r175c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r175c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r175c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-134"></div> <span class="help-tooltip" id="cb-sub-help-134">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r175c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-135"></div> <span class="help-tooltip" id="MIBiG-help-135">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r175c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r175c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r175c1"></div>
    <div id="comparison-MIBiG-r175c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000325.1: 510822-533448">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000325/index.html#r1c1" target="_new">BGC0000325.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.86 (id:0.66, order:0.95, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
         
        
        <td>0.63</td>
        <td>NRP</td>
        
        <td>coelichelin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>vicibactin</td>
        
        
        <td>Rhizobium etli CFN 42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>pseudomonine</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001671.1: 5106-76272">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001671/index.html#r1c1" target="_new">BGC0001671.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>monobactam</td>
        
        
        <td>Agrobacterium tumefaciens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
        
        
        <td>Penicillium rubens Wisconsin 54-1255</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000321.1: 1657-43703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000321/index.html#r1c1" target="_new">BGC0000321.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>chaetocin</td>
        
        
        <td>Collariella virescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.49, order:0.69, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>exochelin</td>
        
        
        <td>Mycolicibacterium smegmatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001480.1: 833379-853163">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001480/index.html#r1c1" target="_new">BGC0001480.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Other</td>
        
        <td>tundrenone</td>
        
        
        <td>Methylobacter tundripaludum 21/22</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r175c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000325.1: 510822-533448">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000325/index.html#r1c1" target="_new">BGC0000325.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.86 (id:0.66, order:0.95, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
      <td>0.86</td>
      <td>NRP</td>
      <td>coelichelin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>vicibactin</td>
      <td>Rhizobium etli CFN 42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>pseudomonine</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001671.1: 5106-76272">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001671/index.html#r1c1" target="_new">BGC0001671.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>monobactam</td>
      <td>Agrobacterium tumefaciens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
      <td>Penicillium rubens Wisconsin 54-1255</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000321.1: 1657-43703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000321/index.html#r1c1" target="_new">BGC0000321.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>chaetocin</td>
      <td>Collariella virescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.49, order:0.69, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>exochelin</td>
      <td>Mycolicibacterium smegmatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001480.1: 833379-853163">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001480/index.html#r1c1" target="_new">BGC0001480.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>tundrenone</td>
      <td>Methylobacter tundripaludum 21/22</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r175c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-136"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-136">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r175c1-pfam-details-svg" data-filename="c00175_NODE_49.._r175c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r175c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r175c1-nrps_pks_products" data-name="r175c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r175c1-nrps_pks" data-name="r175c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r175c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-130"></div> <span class="help-tooltip" id="nrps-structures-help-130">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 49803 - 100801:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r175c1-1" class="smiles-canvas" data-smiles="NC(CCCN)C(=O)NC(C(O)C)C(=O)NC(CCCN)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-orn - D-thr - orn)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-orn,d-thr,orn" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-orn*,d-thr*,orn*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r175c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CCCN)C(=O)NC(C(O)C)C(=O)NC(CCCN)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r175c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-131"></div> <span class="help-tooltip" id="nrps-monomers-help-131">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds69803_80801</span></strong>: orn - thr - orn
      <div class="collapser collapser-target-cds69803_80801 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn,thr,orn" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn*,thr*,orn*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (144..542)</strong>: orn
          <div class="collapser collapser-target-cds69803_80801 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn
              <div class="collapser collapser-target-cds69803_80801-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>orn, horn</dd>
   <dt>Single AA prediction:</dt>
   <dd>orn</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DINYWGGIGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1625..2042)</strong>: thr
          <div class="collapser collapser-target-cds69803_80801 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds69803_80801-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNIGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3086..3484)</strong>: orn
          <div class="collapser collapser-target-cds69803_80801 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn
              <div class="collapser collapser-target-cds69803_80801-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DMwNLGLINK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r176c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00176_NODE_21.. - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-267"></div> <span class="help-tooltip" id="region_svg-help-267">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00176_NODE_21...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r176c1-svg" data-filename="c00176_NODE_21.._r176c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 10,509 - 35,269 nt. (total: 24,761 nt)
        
        <a class="cluster-rules-header" id="r176c1-rules-header" href="#r176c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r176c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r176c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-268"></div> <span class="help-tooltip" id="focus-panel-help-268">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r176c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r176c1-nrps_pks" data-name="r176c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r176c1-clusterblast" data-name="r176c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r176c1-knownclusterblast" data-name="r176c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r176c1-subclusterblast" data-name="r176c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r176c1-MIBiG-cluster-compare" data-name="r176c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r176c1-pfam-details" data-name="r176c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r176c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-137"></div> <span class="help-tooltip" id="nrps-domain-help-137">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r176c1-details-svg" data-filename="c00176_NODE_21.._r176c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r176c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r176c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-138"></div> <span class="help-tooltip" id="cb-general-help-138">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r176c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r176c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r176c1_1.svg">
        NZ_CP027297 (4104848-4147846): Streptomyces sp. SGAir0924 chromosome, complet... (55% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r176c1_2.svg">
        NZ_KZ819185 (1873239-1917109): Streptomyces sp. CS207 scaffold00002, whole ge... (57% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r176c1_3.svg">
        NZ_CP039123 (3998239-4041317): Streptomyces sp. SS52 chromosome, complete genome (54% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r176c1_4.svg">
        NZ_KL573540 (212840-285730): Streptomyces baarnensis strain NRRL B-2842 P144 ... (21% of genes show similarity), NRPS,PKS-like
      </option>
      
      <option value = "svg/clusterblast_r176c1_5.svg">
        NZ_KQ948206 (382822-426674): Streptomyces yokosukanensis strain DSM 40224 PRJ... (25% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r176c1_6.svg">
        NZ_CP043638 (3944933-3985968): Streptomyces sp. INR7 chromosome, complete genome (25% of genes show similarity), NRPS-like,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r176c1_7.svg">
        NZ_FNCC01000008 (305243-348155): Lechevalieria fradiae strain CGMCC 4.3506, w... (25% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r176c1_8.svg">
        NZ_VISR01000005 (167631-224502): Streptomyces sp. BK340 Ga0307703 105, whole ... (23% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r176c1_9.svg">
        NZ_CP021080 (6094558-6137680): Streptomyces pluripotens strain MUSC 135 chrom... (23% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r176c1_10.svg">
        NZ_BBNN01000027 (23746-67583): Streptomyces sp. NBRC 110035 strain TP-A0873, ... (22% of genes show similarity), NRPS-like
      </option>
      
    </select>
    <button id= "clusterblast-r176c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r176c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r176c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-139"></div> <span class="help-tooltip" id="cb-known-help-139">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r176c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r176c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r176c1_1.svg">
        alanylclavam / 2-hydroxymethylclavam / 2-formyloxymethylclavam / clavam-2-carboxylate
      </option>
      
      <option value = "svg/knownclusterblast_r176c1_2.svg">
        granaticin
      </option>
      
    </select>
    <button id="knownclusterblast-r176c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r176c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r176c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-140"></div> <span class="help-tooltip" id="cb-sub-help-140">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r176c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-141"></div> <span class="help-tooltip" id="MIBiG-help-141">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r176c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r176c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r176c1"></div>
    <div id="comparison-MIBiG-r176c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>auriculamide</td>
        
        
        <td>Herpetosiphon aurantiacus DSM 785</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.51, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.22</td>
        <td>Other</td>
        
        <td>5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester</td>
        
        
        <td>Streptomyces sp. RM-5-8</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843.1: 3437627-3461293">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.64, order:0.38, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
        
        <td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>fellutamide B</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>Other</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.47, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Other</td>
        
        <td>showdomycin</td>
        
        
        <td>Streptomyces showdoensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001366.1: 10-15268">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001366/index.html#r1c1" target="_new">BGC0001366.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>BE-24566B, zunyimycin A</td>
        
        
        <td>Streptomyces sp. FJS31-2</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r176c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001521.1: 28949-53551">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001521/index.html#r1c1" target="_new">BGC0001521.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>auriculamide</td>
      <td>Herpetosiphon aurantiacus DSM 785</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.51, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Other</td>
      <td>5-isoprenylindole-3-carboxylate Î²-D-glycosyl ester</td>
      <td>Streptomyces sp. RM-5-8</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843.1: 3437627-3461293">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.64, order:0.38, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
      <td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>fellutamide B</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Other</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.47, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Other</td>
      <td>showdomycin</td>
      <td>Streptomyces showdoensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001366.1: 10-15268">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001366/index.html#r1c1" target="_new">BGC0001366.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>BE-24566B, zunyimycin A</td>
      <td>Streptomyces sp. FJS31-2</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r176c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-142"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-142">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r176c1-pfam-details-svg" data-filename="c00176_NODE_21.._r176c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r176c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r190c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00190_NODE_14.. - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-269"></div> <span class="help-tooltip" id="region_svg-help-269">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00190_NODE_14...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r190c1-svg" data-filename="c00190_NODE_14.._r190c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 34,651 nt. (total: 34,651 nt)
        
        <a class="cluster-rules-header" id="r190c1-rules-header" href="#r190c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r190c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r190c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-270"></div> <span class="help-tooltip" id="focus-panel-help-270">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r190c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r190c1-nrps_pks" data-name="r190c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r190c1-clusterblast" data-name="r190c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r190c1-knownclusterblast" data-name="r190c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r190c1-subclusterblast" data-name="r190c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r190c1-MIBiG-cluster-compare" data-name="r190c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r190c1-pfam-details" data-name="r190c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r190c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-143"></div> <span class="help-tooltip" id="nrps-domain-help-143">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r190c1-details-svg" data-filename="c00190_NODE_14.._r190c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r190c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r190c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-146"></div> <span class="help-tooltip" id="cb-general-help-146">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r190c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r190c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r190c1_1.svg">
        NZ_KZ819184 (3357873-3420035): Streptomyces sp. CS207 scaffold00001, whole ge... (50% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r190c1_2.svg">
        NZ_CP027297 (1097774-1156767): Streptomyces sp. SGAir0924 chromosome, complet... (44% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r190c1_3.svg">
        NZ_CP039123 (1002258-1061109): Streptomyces sp. SS52 chromosome, complete genome (50% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r190c1_4.svg">
        NZ_CP047144 (6764485-6843983): Streptomyces sp. HF10 chromosome, complete genome (30% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r190c1_5.svg">
        NZ_JACHJI010000003 (379615-481923): Streptomyces griseostramineus strain CECT... (25% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r190c1_6.svg">
        NZ_LQYB01000003 (322318-382817): Streptomyces sp. IMTB 1903 contig 16, whole ... (37% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r190c1_7.svg">
        NZ_CP019798 (7274155-7334733): Streptomyces sp. fd1-xmd chromosome, complete ... (39% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r190c1_8.svg">
        NZ_LBMK01000002 (23139-83047): Streptomyces yangpuensis strain fd2-tb scaffol... (41% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r190c1_9.svg">
        NZ_JDUZ01000001 (986431-1045212): Streptomyces sp. PCS3-D2 isolate P1 scaffol... (38% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r190c1_10.svg">
        NZ_LIYH01000002 (457133-516769): Streptomyces sp. CB03234 scaffold2, whole ge... (41% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r190c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r190c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r190c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-147"></div> <span class="help-tooltip" id="cb-known-help-147">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r190c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r190c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r190c1_1.svg">
        streptothricin
      </option>
      
      <option value = "svg/knownclusterblast_r190c1_2.svg">
        BD-12
      </option>
      
      <option value = "svg/knownclusterblast_r190c1_3.svg">
        GE81112
      </option>
      
    </select>
    <button id="knownclusterblast-r190c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r190c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r190c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-148"></div> <span class="help-tooltip" id="cb-sub-help-148">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r190c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-149"></div> <span class="help-tooltip" id="MIBiG-help-149">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r190c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r190c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r190c1"></div>
    <div id="comparison-MIBiG-r190c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000432.1: 1563-32878">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000432/index.html#r1c1" target="_new">BGC0000432.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.84 (id:0.78, order:0.82, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.84), rgb(0, 0, 0, 0.84) 84%, white 84%)"></td>
         
        
        <td>0.59</td>
        <td>NRP</td>
        
        <td>streptothricin</td>
        
        
        <td>Streptomyces rochei</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>fellutamide B</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>ergovaline</td>
        
        
        <td>Epichloe festucae var. lolii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>dimethylcoprogen</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561.1: 1236-18814">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>curacomycin</td>
        
        
        <td>Streptomyces curacoi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876.1: 71-9235">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>Other</td>
        
        <td>neopolyoxin C</td>
        
        
        <td>Streptomyces tendae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000921.1: 4334-20040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000921/index.html#r1c1" target="_new">BGC0000921.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Other, Polyketide, NRP,  , (, S, h, i, k, i, m, a, t, e, -, d, e, r, i, v, e, d, )</td>
        
        <td>promysalin</td>
        
        
        <td>Pseudomonas putida</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r190c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000432.1: 1563-32878">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000432/index.html#r1c1" target="_new">BGC0000432.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.84 (id:0.78, order:0.82, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.84), rgb(0, 0, 0, 0.84) 84%, white 84%)"></td>
      <td>0.84</td>
      <td>NRP</td>
      <td>streptothricin</td>
      <td>Streptomyces rochei</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>fellutamide B</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>ergovaline</td>
      <td>Epichloe festucae var. lolii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>dimethylcoprogen</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561.1: 1236-18814">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.46, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>curacomycin</td>
      <td>Streptomyces curacoi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876.1: 71-9235">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>neopolyoxin C</td>
      <td>Streptomyces tendae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000921.1: 4334-20040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000921/index.html#r1c1" target="_new">BGC0000921.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other, Polyketide, NRP,  , (, S, h, i, k, i, m, a, t, e, -, d, e, r, i, v, e, d, )</td>
      <td>promysalin</td>
      <td>Pseudomonas putida</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r190c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-150"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-150">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r190c1-pfam-details-svg" data-filename="c00190_NODE_14.._r190c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r190c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r190c1-nrps_pks_products" data-name="r190c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r190c1-nrps_pks" data-name="r190c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r190c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-144"></div> <span class="help-tooltip" id="nrps-structures-help-144">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 34651:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r190c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r190c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r190c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-145"></div> <span class="help-tooltip" id="nrps-monomers-help-145">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds9472_11152</span></strong>: X
      <div class="collapser collapser-target-cds9472_11152 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (9..419)</strong>: X
          <div class="collapser collapser-target-cds9472_11152 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds9472_11152-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DTEnVGTLVK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>lys-b</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds18807_20817</span></strong>: X
      <div class="collapser collapser-target-cds18807_20817 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (15..418)</strong>: X
          <div class="collapser collapser-target-cds18807_20817 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds18807_20817-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVeeVGVVsK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>trp</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds29949_31467</span></strong>: lys-b
      <div class="collapser collapser-target-cds29949_31467 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys-b" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys-b*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (11..403)</strong>: lys-b
          <div class="collapser collapser-target-cds29949_31467 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  lys-b
              <div class="collapser collapser-target-cds29949_31467-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DTEVVGTLVK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>lys-b</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r215c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00215_NODE_27.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-271"></div> <span class="help-tooltip" id="region_svg-help-271">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00215_NODE_27...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r215c1-svg" data-filename="c00215_NODE_27.._r215c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 5,200 - 31,919 nt. (total: 26,720 nt)
        
        <a class="cluster-rules-header" id="r215c1-rules-header" href="#r215c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r215c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r215c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-272"></div> <span class="help-tooltip" id="focus-panel-help-272">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r215c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r215c1-clusterblast" data-name="r215c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r215c1-knownclusterblast" data-name="r215c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r215c1-subclusterblast" data-name="r215c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r215c1-MIBiG-cluster-compare" data-name="r215c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r215c1-pfam-details" data-name="r215c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r215c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-151"></div> <span class="help-tooltip" id="cb-general-help-151">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r215c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r215c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r215c1_1.svg">
        NZ_CP039123 (6995783-7021739): Streptomyces sp. SS52 chromosome, complete genome (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r215c1_2.svg">
        NZ_KZ819185 (106076-132008): Streptomyces sp. CS207 scaffold00002, whole geno... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r215c1_3.svg">
        NZ_CP009754 (1043336-1069144): Streptomyces sp. CCM MD2014 chromosome, comple... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r215c1_4.svg">
        NZ_CP045547 (1125038-1150932): Streptomyces sp. SYP-A7193 chromosome, complet... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r215c1_5.svg">
        NZ_BBOK01000011 (408408-435138): Streptomyces sp. NBRC 110030, whole genome s... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r215c1_6.svg">
        NZ_CP042278 (848512-874451): Streptomyces sp. WAC06273 strain WAC6273 substr.... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r215c1_7.svg">
        NZ_GG657756 (983546-1009354): Streptomyces lividans TK24 supercont1.1, whole ... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r215c1_8.svg">
        NZ_CP042324 (7506422-7532232): Streptomyces coelicolor A3(2) strain CFB NBC 0... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r215c1_9.svg">
        NZ_CP026730 (6685826-6712568): Streptomyces sp. CB09001 chromosome, complete ... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r215c1_10.svg">
        NZ_CP027297 (6913168-6939092): Streptomyces sp. SGAir0924 chromosome, complet... (100% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r215c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r215c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r215c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-152"></div> <span class="help-tooltip" id="cb-known-help-152">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r215c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r215c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_1.svg">
        hopene
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_2.svg">
        xiamycin
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_3.svg">
        xiamycin A
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_4.svg">
        phosphonoglycans
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_5.svg">
        platencin
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_6.svg">
        platensimycin / platencin
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_7.svg">
        thioplatensimycin / thioplatencin
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_8.svg">
        phenalinolactone A
      </option>
      
      <option value = "svg/knownclusterblast_r215c1_9.svg">
        KS-505a
      </option>
      
    </select>
    <button id="knownclusterblast-r215c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r215c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r215c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-153"></div> <span class="help-tooltip" id="cb-sub-help-153">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r215c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r215c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r215c1_1.svg">
        DQ230532_c2: phenalinolactone terpenoid (33% of genes show similarity), terpenoid
      </option>
      
    </select>
    <button id="subclusterblast-r215c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r215c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r215c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-154"></div> <span class="help-tooltip" id="MIBiG-help-154">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r215c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r215c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r215c1"></div>
    <div id="comparison-MIBiG-r215c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663.1: 7516016-7529773">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.87 (id:0.68, order:0.96, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.87), rgb(0, 0, 0, 0.87) 87%, white 87%)"></td>
         
        
        <td>0.65</td>
        <td>Terpene</td>
        
        <td>hopene</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645.1: 0-7475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Halobacillus halophilus DSM 2266</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Myxococcus xanthus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Rhodobacter sphaeroides</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644.1: 97-6276">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Dietzia sp. CQ4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>Î²-carotein</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630.1: 389-6051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
        
        
        <td>Paracoccus haeundaensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656.1: 169-3770">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Terpene</td>
        
        <td>zeaxanthin</td>
        
        
        <td>Xanthobacter autotrophicus Py2</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000638.1: 224-6452">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000638/index.html#r1c1" target="_new">BGC0000638.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.22</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Pantoea ananatis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r215c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663.1: 7516016-7529773">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.87 (id:0.68, order:0.96, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.87), rgb(0, 0, 0, 0.87) 87%, white 87%)"></td>
      <td>0.87</td>
      <td>Terpene</td>
      <td>hopene</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645.1: 0-7475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Halobacillus halophilus DSM 2266</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Myxococcus xanthus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Rhodobacter sphaeroides</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644.1: 97-6276">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Dietzia sp. CQ4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene</td>
      <td>Î²-carotein</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630.1: 389-6051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
      <td>Paracoccus haeundaensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656.1: 169-3770">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>zeaxanthin</td>
      <td>Xanthobacter autotrophicus Py2</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000638.1: 224-6452">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000638/index.html#r1c1" target="_new">BGC0000638.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Pantoea ananatis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r215c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-155"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-155">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r215c1-pfam-details-svg" data-filename="c00215_NODE_27.._r215c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r215c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r216c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00216_NODE_14.. - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-273"></div> <span class="help-tooltip" id="region_svg-help-273">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00216_NODE_14...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r216c1-svg" data-filename="c00216_NODE_14.._r216c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 5,514 - 11,415 nt. (total: 5,902 nt)
        
        <a class="cluster-rules-header" id="r216c1-rules-header" href="#r216c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r216c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193)</div>
      
      <div class="region-svg-container">
       <div id='r216c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-274"></div> <span class="help-tooltip" id="focus-panel-help-274">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r216c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r216c1-clusterblast" data-name="r216c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r216c1-knownclusterblast" data-name="r216c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r216c1-subclusterblast" data-name="r216c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r216c1-MIBiG-cluster-compare" data-name="r216c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r216c1-pfam-details" data-name="r216c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r216c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-156"></div> <span class="help-tooltip" id="cb-general-help-156">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r216c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r216c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r216c1_1.svg">
        NZ_CP027297 (6235677-6246081): Streptomyces sp. SGAir0924 chromosome, complet... (60% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r216c1_2.svg">
        NZ_CP039123 (6087208-6097618): Streptomyces sp. SS52 chromosome, complete genome (60% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r216c1_3.svg">
        NZ_KZ819185 (768165-779488): Streptomyces sp. CS207 scaffold00002, whole geno... (60% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r216c1_4.svg">
        NZ_CP012949 (6106045-6116468): Streptomyces ambofaciens strain DSM 40697 chro... (60% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r216c1_5.svg">
        NZ_CP012382 (6235410-6245830): Streptomyces ambofaciens ATCC 23877 chromosome... (60% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r216c1_6.svg">
        NZ_LIPF01000013 (9437-20775): Streptomyces sp. CB02414 scaffold13, whole geno... (41% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r216c1_7.svg">
        NZ_CP019724 (2035753-2046113): Streptomyces pactum strain ACT12 chromosome, c... (45% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r216c1_8.svg">
        NZ_CP015866 (5626981-5638028): Streptomyces parvulus strain 2297 chromosome, ... (45% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r216c1_9.svg">
        NZ_CP042278 (1875394-1886717): Streptomyces sp. WAC06273 strain WAC6273 subst... (50% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r216c1_10.svg">
        NZ_KN050743 (103509-153524): Streptomyces mirabilis strain OV308 EW60DRAFT sc... (11% of genes show similarity), NRPS-like,T1PKS,bacteriocin
      </option>
      
    </select>
    <button id= "clusterblast-r216c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r216c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r216c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-157"></div> <span class="help-tooltip" id="cb-known-help-157">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r216c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-158"></div> <span class="help-tooltip" id="cb-sub-help-158">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r216c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-159"></div> <span class="help-tooltip" id="MIBiG-help-159">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r216c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r216c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r216c1"></div>
    <div id="comparison-MIBiG-r216c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">RiPP-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001796.1: 4751364-4791946">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001796/index.html#r1c1" target="_new">BGC0001796.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.25 (id:0.45, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
         
        
        <td>0.06</td>
        <td>NRP</td>
        
        <td>tauramamide</td>
        
        
        <td>Brevibacillus laterosporus DSM 25</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r216c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001796.1: 4751364-4791946">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001796/index.html#r1c1" target="_new">BGC0001796.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.25 (id:0.45, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
      <td>0.25</td>
      <td>NRP</td>
      <td>tauramamide</td>
      <td>Brevibacillus laterosporus DSM 25</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r216c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-160"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-160">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r216c1-pfam-details-svg" data-filename="c00216_NODE_14.._r216c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r216c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r222c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00222_NODE_47.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-275"></div> <span class="help-tooltip" id="region_svg-help-275">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00222_NODE_47...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r222c1-svg" data-filename="c00222_NODE_47.._r222c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 16,889 nt. (total: 16,889 nt)
        
        <a class="cluster-rules-header" id="r222c1-rules-header" href="#r222c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r222c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r222c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-276"></div> <span class="help-tooltip" id="focus-panel-help-276">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r222c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r222c1-clusterblast" data-name="r222c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r222c1-knownclusterblast" data-name="r222c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r222c1-subclusterblast" data-name="r222c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r222c1-MIBiG-cluster-compare" data-name="r222c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r222c1-pfam-details" data-name="r222c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r222c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-161"></div> <span class="help-tooltip" id="cb-general-help-161">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r222c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r222c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r222c1_1.svg">
        NZ_KZ819185 (730569-752750): Streptomyces sp. CS207 scaffold00002, whole geno... (73% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r222c1_2.svg">
        NZ_CP027297 (6262360-6282141): Streptomyces sp. SGAir0924 chromosome, complet... (72% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r222c1_3.svg">
        NZ_CP039123 (6113979-6133747): Streptomyces sp. SS52 chromosome, complete genome (72% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r222c1_4.svg">
        NZ_LIPF01000013 (37533-59714): Streptomyces sp. CB02414 scaffold13, whole gen... (65% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r222c1_5.svg">
        NZ_KN039947 (823163-843789): Streptomyces luteus strain TRM 45540 Scaffold2, ... (65% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r222c1_6.svg">
        NZ_CP019724 (1997020-2017092): Streptomyces pactum strain ACT12 chromosome, c... (72% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r222c1_7.svg">
        NZ_SLZE01000009 (202985-225151): Streptomyces sp. BK674 Ga0307689 109, whole ... (82% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r222c1_8.svg">
        NZ_CP016795 (6331529-6353200): Streptomyces olivaceus strain KLBMP 5084 chrom... (70% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r222c1_9.svg">
        NZ_CP036534 (6392627-6414349): Streptomyces sp. VN1 chromosome, complete genome (66% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r222c1_10.svg">
        NZ_CP034687 (2030906-2049323): Streptomyces griseoviridis strain F1-27 chromo... (82% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r222c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r222c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r222c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-162"></div> <span class="help-tooltip" id="cb-known-help-162">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r222c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r222c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r222c1_1.svg">
        geosmin
      </option>
      
    </select>
    <button id="knownclusterblast-r222c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r222c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r222c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-163"></div> <span class="help-tooltip" id="cb-sub-help-163">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r222c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-164"></div> <span class="help-tooltip" id="MIBiG-help-164">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r222c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r222c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r222c1"></div>
    <div id="comparison-MIBiG-r222c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001181.1: 6666218-6668399">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001181/index.html#r1c1" target="_new">BGC0001181.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.83 (id:0.70, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
         
        
        <td>0.59</td>
        <td>Terpene</td>
        
        <td>geosmin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.52, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Terpene</td>
        
        <td>geosmin</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001746.1: 7136894-7138013">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001746/index.html#r1c1" target="_new">BGC0001746.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Terpene</td>
        
        <td>pristinol</td>
        
        
        <td>Streptomyces pristinaespiralis ATCC 25486</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000849.1: 6890527-6892237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000849/index.html#r1c1" target="_new">BGC0000849.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other,  , (, B, u, t, y, r, o, l, a, c, t, o, n, e, )</td>
        
        <td>SCB1, SCB2, SCB3</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.57, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>Î³-butyrolactone</td>
        
        
        <td>Streptomyces acidiscabies</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001285.1: 8-22479">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001285/index.html#r1c1" target="_new">BGC0001285.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.48, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other,  , (, F, a, t, t, y,  , a, c, i, d, )</td>
        
        <td>pseudopyronine A, pseudopyronine B</td>
        
        
        <td>Pseudomonas putida</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001006.1: 3494093-3519138">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001006/index.html#r1c1" target="_new">BGC0001006.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
         
        
        <td>0.09</td>
        <td>NRP, Polyketide</td>
        
        <td>lymphostin, neolymphostinol B, lymphostinol, neolymphostin B</td>
        
        
        <td>Salinispora tropica CNB-440</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001007.1: 3781644-3805331">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001007/index.html#r1c1" target="_new">BGC0001007.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
         
        
        <td>0.09</td>
        <td>Polyketide, NRP</td>
        
        <td>lymphostin, neolymphostinol B, lymphostinol, neolymphostin b</td>
        
        
        <td>Salinispora arenicola CNS-205</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001669.1: 4732-29763">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001669/index.html#r1c1" target="_new">BGC0001669.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.28 (id:0.43, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
         
        
        <td>0.07</td>
        <td>RiPP</td>
        
        <td>microvionin</td>
        
        
        <td>Microbacterium arborescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001614.1: 0-47882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001614/index.html#r1c1" target="_new">BGC0001614.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.26 (id:0.43, order:N/A, components:0.16)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
         
        
        <td>0.06</td>
        <td>NRP</td>
        
        <td>hassallidin E</td>
        
        
        <td>Planktothrix serta PCC 8927</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r222c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001181.1: 6666218-6668399">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001181/index.html#r1c1" target="_new">BGC0001181.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.83 (id:0.70, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
      <td>0.83</td>
      <td>Terpene</td>
      <td>geosmin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.52, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Terpene</td>
      <td>geosmin</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001746.1: 7136894-7138013">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001746/index.html#r1c1" target="_new">BGC0001746.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>pristinol</td>
      <td>Streptomyces pristinaespiralis ATCC 25486</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000849.1: 6890527-6892237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000849/index.html#r1c1" target="_new">BGC0000849.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other,  , (, B, u, t, y, r, o, l, a, c, t, o, n, e, )</td>
      <td>SCB1, SCB2, SCB3</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.57, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>Î³-butyrolactone</td>
      <td>Streptomyces acidiscabies</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001285.1: 8-22479">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001285/index.html#r1c1" target="_new">BGC0001285.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.48, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other,  , (, F, a, t, t, y,  , a, c, i, d, )</td>
      <td>pseudopyronine A, pseudopyronine B</td>
      <td>Pseudomonas putida</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001006.1: 3494093-3519138">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001006/index.html#r1c1" target="_new">BGC0001006.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
      <td>0.34</td>
      <td>NRP, Polyketide</td>
      <td>lymphostin, neolymphostinol B, lymphostinol, neolymphostin B</td>
      <td>Salinispora tropica CNB-440</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001007.1: 3781644-3805331">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001007/index.html#r1c1" target="_new">BGC0001007.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
      <td>0.34</td>
      <td>Polyketide, NRP</td>
      <td>lymphostin, neolymphostinol B, lymphostinol, neolymphostin b</td>
      <td>Salinispora arenicola CNS-205</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001669.1: 4732-29763">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001669/index.html#r1c1" target="_new">BGC0001669.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.28 (id:0.43, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
      <td>0.28</td>
      <td>RiPP</td>
      <td>microvionin</td>
      <td>Microbacterium arborescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001614.1: 0-47882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001614/index.html#r1c1" target="_new">BGC0001614.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.26 (id:0.43, order:N/A, components:0.16)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
      <td>0.26</td>
      <td>NRP</td>
      <td>hassallidin E</td>
      <td>Planktothrix serta PCC 8927</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r222c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-165"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-165">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r222c1-pfam-details-svg" data-filename="c00222_NODE_47.._r222c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r222c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r237c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00237_NODE_6_.. - Region 1 - lanthipeptide-class-iii <div class="help-container"> <div class="help-icon" data-id="region_svg-help-277"></div> <span class="help-tooltip" id="region_svg-help-277">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00237_NODE_6_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r237c1-svg" data-filename="c00237_NODE_6_.._r237c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 149,353 - 171,935 nt. (total: 22,583 nt)
        
        <a class="cluster-rules-header" id="r237c1-rules-header" href="#r237c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r237c1-rules">lanthipeptide-class-iii: (micKC and not PKS_KS)</div>
      
      <div class="region-svg-container">
       <div id='r237c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-278"></div> <span class="help-tooltip" id="focus-panel-help-278">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r237c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r237c1-lanthipeptides" data-name="r237c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r237c1-clusterblast" data-name="r237c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r237c1-knownclusterblast" data-name="r237c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r237c1-subclusterblast" data-name="r237c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r237c1-MIBiG-cluster-compare" data-name="r237c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r237c1-pfam-details" data-name="r237c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r237c1-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-166"></div> <span class="help-tooltip" id="lanthi-body-help-166">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lanthipeptide(s) for cds159352_161935 - Putative Class III</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds161987_162116</span> leader / core peptide
          </dt>
          <dd>
            MNLFELQSLETPKEEAMGDVE
            -
            <span class="dhb">Dhb</span>G<span class="dha">Dha</span>RA<span class="dha">Dha</span>LLL<span class="cys">C</span>GD<span class="dha">Dha</span><span class="dha">Dha</span>L<span class="dha">Dha</span>V<span class="dhb">Dhb</span><span class="dhb">Dhb</span><span class="cys">C</span>N
        </dd>
      </dl>
    
    <div class="legend">
      <h4> Legend: </h4>
      <div>
        <span class="dha">Dha</span>: Didehydroalanine<br>
        <span class="dhb">Dhb</span>: Didehydrobutyrine<br>
      </div>
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r237c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-168"></div> <span class="help-tooltip" id="cb-general-help-168">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r237c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r237c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r237c1_1.svg">
        NZ_CP039123 (7887552-7910135): Streptomyces sp. SS52 chromosome, complete genome (95% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r237c1_2.svg">
        NZ_CP027299 (182560-205143): Streptomyces sp. SGAir0924 plasmid unnamed 5 (90% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r237c1_3.svg">
        NZ_PDKQ02000044 (15958-58520): Streptomyces barkulensis strain RC 1831 Scaffo... (21% of genes show similarity), lanthipeptide,lipolanthine
      </option>
      
      <option value = "svg/clusterblast_r237c1_4.svg">
        NZ_KN039948 (59878-82461): Streptomyces luteus strain TRM 45540 Scaffold3, wh... (26% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r237c1_5.svg">
        NZ_LIVQ01000005 (144572-167215): Streptomyces sp. CB00455 scaffold5, whole ge... (35% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r237c1_6.svg">
        NZ_CP020042 (7191341-7213981): Streptomyces katrae strain S3 isolate 4 chromo... (28% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r237c1_7.svg">
        NZ_LIPF01000004 (558690-581351): Streptomyces sp. CB02414 scaffold4, whole ge... (26% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r237c1_8.svg">
        NZ_KB897785 (245523-268220): Streptomyces sp. CNT372 C593DRAFT scaffold00010.... (28% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r237c1_9.svg">
        NZ_CP018047 (629761-722249): Streptomyces niveus strain SCSIO 3406 chromosome... (8% of genes show similarity), NRPS,hglE-KS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r237c1_10.svg">
        NZ_LIVX01000004 (489348-511979): Streptomyces sp. CB02261 scaffold4, whole ge... (35% of genes show similarity), lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r237c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r237c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r237c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-169"></div> <span class="help-tooltip" id="cb-known-help-169">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r237c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r237c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r237c1_1.svg">
        SapB
      </option>
      
      <option value = "svg/knownclusterblast_r237c1_2.svg">
        SAL-2242
      </option>
      
      <option value = "svg/knownclusterblast_r237c1_3.svg">
        AmfS
      </option>
      
      <option value = "svg/knownclusterblast_r237c1_4.svg">
        catenulipeptin
      </option>
      
      <option value = "svg/knownclusterblast_r237c1_5.svg">
        informatipeptin
      </option>
      
    </select>
    <button id="knownclusterblast-r237c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r237c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r237c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-170"></div> <span class="help-tooltip" id="cb-sub-help-170">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r237c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-171"></div> <span class="help-tooltip" id="MIBiG-help-171">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r237c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r237c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r237c1"></div>
    <div id="comparison-MIBiG-r237c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-iii</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.84 (id:0.59, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.84), rgb(0, 0, 0, 0.84) 84%, white 84%)"></td>
         
        
        <td>0.60</td>
        <td>RiPP</td>
        
        <td>SapB</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496.1: 2822112-2829457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.50, order:0.58, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>RiPP</td>
        
        <td>AmfS</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.50, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>RiPP</td>
        
        <td>catenulipeptin</td>
        
        
        <td>Catenulispora acidiphila DSM 44928</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.49, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>RiPP</td>
        
        <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
        
        
        <td>Actinomadura namibiensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.51, order:0.48, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>RiPP</td>
        
        <td>informatipeptin</td>
        
        
        <td>Streptomyces viridochromogenes DSM 40736</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000516.1: 8706-19626">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000516/index.html#r1c1" target="_new">BGC0000516.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>RiPP</td>
        
        <td>geobacillin II</td>
        
        
        <td>Geobacillus thermodenitrificans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000546.1: 4112716-4120692">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000546/index.html#r1c1" target="_new">BGC0000546.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.52, order:0.45, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>SAL-2242</td>
        
        
        <td>Streptomyces albidoflavus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000502.1: 0-15725">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000502/index.html#r1c1" target="_new">BGC0000502.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>cerecidin, cerecidin A1, cerecidin A2, cerecidin A3, cerecidin A4, cerecidin A5, cerecidin A6, cerecidin A7</td>
        
        
        <td>Bacillus cereus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000527.1: 0-12324">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000527/index.html#r1c1" target="_new">BGC0000527.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>RiPP</td>
        
        <td>mersacidin</td>
        
        
        <td>Bacillus sp. HIL-Y85/54728</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r237c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.84 (id:0.59, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.84), rgb(0, 0, 0, 0.84) 84%, white 84%)"></td>
      <td>0.84</td>
      <td>RiPP</td>
      <td>SapB</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496.1: 2822112-2829457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.50, order:0.58, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>RiPP</td>
      <td>AmfS</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.50, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>RiPP</td>
      <td>catenulipeptin</td>
      <td>Catenulispora acidiphila DSM 44928</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.49, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>RiPP</td>
      <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
      <td>Actinomadura namibiensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.51, order:0.48, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>RiPP</td>
      <td>informatipeptin</td>
      <td>Streptomyces viridochromogenes DSM 40736</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000516.1: 8706-19626">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000516/index.html#r1c1" target="_new">BGC0000516.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>RiPP</td>
      <td>geobacillin II</td>
      <td>Geobacillus thermodenitrificans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000546.1: 4112716-4120692">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000546/index.html#r1c1" target="_new">BGC0000546.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.52, order:0.45, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>RiPP</td>
      <td>SAL-2242</td>
      <td>Streptomyces albidoflavus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000502.1: 0-15725">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000502/index.html#r1c1" target="_new">BGC0000502.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>RiPP</td>
      <td>cerecidin, cerecidin A1, cerecidin A2, cerecidin A3, cerecidin A4, cerecidin A5, cerecidin A6, cerecidin A7</td>
      <td>Bacillus cereus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000527.1: 0-12324">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000527/index.html#r1c1" target="_new">BGC0000527.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>RiPP</td>
      <td>mersacidin</td>
      <td>Bacillus sp. HIL-Y85/54728</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r237c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-172"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-172">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r237c1-pfam-details-svg" data-filename="c00237_NODE_6_.._r237c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r237c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r237c1-lanthipeptides" data-name="r237c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r237c1-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-167"></div> <span class="help-tooltip" id="lanthi-side-help-167">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

    <dl class ="prediction-text">
    <dt><span class="serif">cds161987_162116</span> - Class III</dt>
        <dd>Cleavage pHMM score: 30.00</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 1940.0 Da</dd>
        <dd>Molecular weight: 1941.2 Da</dd>
        <dd>Number of bridges: 2</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 1959.2 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 1977.2 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 1995.2 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2013.2 Da</span>
                    <span class="alt-weight-n"> N = 4</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2031.3 Da</span>
                    <span class="alt-weight-n"> N = 5</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2049.3 Da</span>
                    <span class="alt-weight-n"> N = 6</span>
                  </dd>
                
            </dl>
          </dd>
        
  </dl>

</div>

    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r237c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00237_NODE_6_.. - Region 2 - lanthipeptide-class-i <div class="help-container"> <div class="help-icon" data-id="region_svg-help-279"></div> <span class="help-tooltip" id="region_svg-help-279">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00237_NODE_6_...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r237c2-svg" data-filename="c00237_NODE_6_.._r237c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 184,507 - 209,100 nt. (total: 24,594 nt)
        
        <a class="cluster-rules-header" id="r237c2-rules-header" href="#r237c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r237c2-rules">lanthipeptide-class-i: ((LANC_like and (Lant_dehydr_N or Lant_dehydr_C)) and not (YcaO or TIGR03882))</div>
      
      <div class="region-svg-container">
       <div id='r237c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-280"></div> <span class="help-tooltip" id="focus-panel-help-280">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r237c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r237c2-lanthipeptides" data-name="r237c2-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r237c2-clusterblast" data-name="r237c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r237c2-knownclusterblast" data-name="r237c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r237c2-subclusterblast" data-name="r237c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r237c2-MIBiG-cluster-compare" data-name="r237c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r237c2-pfam-details" data-name="r237c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r237c2-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-173"></div> <span class="help-tooltip" id="lanthi-body-help-173">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lanthipeptide(s) for cds195921_198504 - Putative Class I</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds199184_199346</span> leader / core peptide
          </dt>
          <dd>
            MRTEIVLSHEAPELDLELDLRVSDLPEQA
            -
            E<span class="dha">Dha</span>FGQG<span class="dhb">Dhb</span>F<span class="dhb">Dhb</span><span class="dha">Dha</span>P<span class="dha">Dha</span><span class="dha">Dha</span>YAIG<span class="dhb">Dhb</span>R<span class="cys">C</span>PI<span class="cys">C</span><span class="cys">C</span>
        </dd>
      </dl>
    
      <h4>Lanthipeptide(s) for cds198473_199100 - Putative Class I</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds199184_199346</span> leader / core peptide
          </dt>
          <dd>
            MRTEIVLSHEAPELDLELDLRVSDLPEQA
            -
            E<span class="dha">Dha</span>FGQG<span class="dhb">Dhb</span>F<span class="dhb">Dhb</span><span class="dha">Dha</span>P<span class="dha">Dha</span><span class="dha">Dha</span>YAIG<span class="dhb">Dhb</span>R<span class="cys">C</span>PI<span class="cys">C</span><span class="cys">C</span>
        </dd>
      </dl>
    
    <div class="legend">
      <h4> Legend: </h4>
      <div>
        <span class="dha">Dha</span>: Didehydroalanine<br>
        <span class="dhb">Dhb</span>: Didehydrobutyrine<br>
      </div>
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r237c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-175"></div> <span class="help-tooltip" id="cb-general-help-175">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r237c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r237c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r237c2_1.svg">
        NZ_CP027299 (147618-170095): Streptomyces sp. SGAir0924 plasmid unnamed 5 (90% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r237c2_2.svg">
        NZ_CP039123 (7801014-7876557): Streptomyces sp. SS52 chromosome, complete genome (30% of genes show similarity), NRPS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r237c2_3.svg">
        NZ_CP054939 (7535875-7560410): Streptomyces sp. NA02536 chromosome, complete ... (16% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r237c2_4.svg">
        NZ_GG657758 (7884995-7909530): Streptomyces griseoflavus Tu4000 supercont1.1,... (13% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r237c2_5.svg">
        NZ_CP012382 (7931837-7968985): Streptomyces ambofaciens ATCC 23877 chromosome... (11% of genes show similarity), lanthipeptide,terpene
      </option>
      
      <option value = "svg/clusterblast_r237c2_6.svg">
        NZ_CP012949 (7717139-7754336): Streptomyces ambofaciens strain DSM 40697 chro... (11% of genes show similarity), lanthipeptide,terpene
      </option>
      
      <option value = "svg/clusterblast_r237c2_7.svg">
        NZ_VIUV01000001 (113609-138120): Streptomyces sp. T12 Ga0311319 101, whole ge... (19% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r237c2_8.svg">
        NZ_CP043959 (4007187-4030459): Streptomyces tendae strain 139 chromosome, com... (20% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r237c2_9.svg">
        NC_016582 (8307758-8331531): Streptomyces bingchenggensis BCW-1, complete seq... (22% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r237c2_10.svg">
        NZ_MQUH01000030 (148526-201094): Streptomyces sp. M1013 M1013 C30, whole geno... (7% of genes show similarity), hglE-KS,lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r237c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r237c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r237c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-176"></div> <span class="help-tooltip" id="cb-known-help-176">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r237c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-177"></div> <span class="help-tooltip" id="cb-sub-help-177">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r237c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-178"></div> <span class="help-tooltip" id="MIBiG-help-178">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r237c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r237c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r237c2"></div>
    <div id="comparison-MIBiG-r237c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-i</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000542.1: 0-9130">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000542/index.html#r1c1" target="_new">BGC0000542.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>RiPP</td>
        
        <td>paenicidin B</td>
        
        
        <td>Paenibacillus terrae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000541.1: 0-9113">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000541/index.html#r1c1" target="_new">BGC0000541.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>paenicidin A</td>
        
        
        <td>Paenibacillus polymyxa</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000559.1: 494-13831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000559/index.html#r1c1" target="_new">BGC0000559.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>subtilin</td>
        
        
        <td>Bacillus subtilis subsp. spizizenii ATCC 6633</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000506.1: 405-13975">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000506/index.html#r1c1" target="_new">BGC0000506.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>entianin</td>
        
        
        <td>Bacillus subtilis subsp. spizizenii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001289.1: 16-10643">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001289/index.html#r1c1" target="_new">BGC0001289.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:0.37, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>penisin</td>
        
        
        <td>Paenibacillus ehimensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001364.1: 317877-335461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001364/index.html#r1c1" target="_new">BGC0001364.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>RiPP</td>
        
        <td>cinnamycin B</td>
        
        
        <td>Actinomadura atramentaria DSM 43919</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000511.1: 0-14033">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000511/index.html#r1c1" target="_new">BGC0000511.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>RiPP</td>
        
        <td>ericin S</td>
        
        
        <td>Bacillus subtilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000508.1: 0-8379">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000508/index.html#r1c1" target="_new">BGC0000508.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>RiPP</td>
        
        <td>epidermin</td>
        
        
        <td>Staphylococcus epidermidis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001868.1: 193423-204464">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001868/index.html#r1c1" target="_new">BGC0001868.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other,  , (, P, h, e, n, a, z, i, n, e, )</td>
        
        <td>iodinin</td>
        
        
        <td>Xenorhabdus szentirmaii DSM 16338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000544.1: 14969-30259">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000544/index.html#r1c1" target="_new">BGC0000544.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:0.36, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>planosporicin</td>
        
        
        <td>Planomonospora alba</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r237c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000542.1: 0-9130">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000542/index.html#r1c1" target="_new">BGC0000542.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>paenicidin B</td>
      <td>Paenibacillus terrae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000541.1: 0-9113">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000541/index.html#r1c1" target="_new">BGC0000541.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>paenicidin A</td>
      <td>Paenibacillus polymyxa</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000559.1: 494-13831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000559/index.html#r1c1" target="_new">BGC0000559.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>subtilin</td>
      <td>Bacillus subtilis subsp. spizizenii ATCC 6633</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000506.1: 405-13975">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000506/index.html#r1c1" target="_new">BGC0000506.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>RiPP</td>
      <td>entianin</td>
      <td>Bacillus subtilis subsp. spizizenii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001289.1: 16-10643">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001289/index.html#r1c1" target="_new">BGC0001289.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:0.37, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>RiPP</td>
      <td>penisin</td>
      <td>Paenibacillus ehimensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001364.1: 317877-335461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001364/index.html#r1c1" target="_new">BGC0001364.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>RiPP</td>
      <td>cinnamycin B</td>
      <td>Actinomadura atramentaria DSM 43919</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000511.1: 0-14033">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000511/index.html#r1c1" target="_new">BGC0000511.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>RiPP</td>
      <td>ericin S</td>
      <td>Bacillus subtilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000508.1: 0-8379">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000508/index.html#r1c1" target="_new">BGC0000508.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>RiPP</td>
      <td>epidermin</td>
      <td>Staphylococcus epidermidis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001868.1: 193423-204464">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001868/index.html#r1c1" target="_new">BGC0001868.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other,  , (, P, h, e, n, a, z, i, n, e, )</td>
      <td>iodinin</td>
      <td>Xenorhabdus szentirmaii DSM 16338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000544.1: 14969-30259">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000544/index.html#r1c1" target="_new">BGC0000544.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:0.36, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>planosporicin</td>
      <td>Planomonospora alba</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r237c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-179"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-179">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r237c2-pfam-details-svg" data-filename="c00237_NODE_6_.._r237c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r237c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r237c2-lanthipeptides" data-name="r237c2-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r237c2-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-174"></div> <span class="help-tooltip" id="lanthi-side-help-174">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

    <dl class ="prediction-text">
    <dt><span class="serif">cds199184_199346</span> - Class I</dt>
        <dd>Cleavage pHMM score: -10.80</dd>
        <dd>RODEO score: 18</dd>
        <dd>Monoisotopic mass: 2385.1 Da</dd>
        <dd>Molecular weight: 2386.7 Da</dd>
        <dd>Number of bridges: 3</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 2404.7 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2422.7 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2440.7 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2458.7 Da</span>
                    <span class="alt-weight-n"> N = 4</span>
                  </dd>
                
            </dl>
          </dd>
        <dt><span class="serif">cds199184_199346</span> - Class I</dt>
        <dd>Cleavage pHMM score: -10.80</dd>
        <dd>RODEO score: 18</dd>
        <dd>Monoisotopic mass: 2385.1 Da</dd>
        <dd>Molecular weight: 2386.7 Da</dd>
        <dd>Number of bridges: 3</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 2404.7 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2422.7 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2440.7 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2458.7 Da</span>
                    <span class="alt-weight-n"> N = 4</span>
                  </dd>
                
            </dl>
          </dd>
        
  </dl>

</div>

    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r237c3' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00237_NODE_6_.. - Region 3 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-281"></div> <span class="help-tooltip" id="region_svg-help-281">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00237_NODE_6_...region003.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r237c3-svg" data-filename="c00237_NODE_6_.._r237c3.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 212,872 - 265,197 nt. (total: 52,326 nt)
        
        <a class="cluster-rules-header" id="r237c3-rules-header" href="#r237c3">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r237c3-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r237c3-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-282"></div> <span class="help-tooltip" id="focus-panel-help-282">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r237c3">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r237c3-nrps_pks" data-name="r237c3-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r237c3-clusterblast" data-name="r237c3-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r237c3-knownclusterblast" data-name="r237c3-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r237c3-subclusterblast" data-name="r237c3-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r237c3-MIBiG-cluster-compare" data-name="r237c3-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r237c3-pfam-details" data-name="r237c3-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r237c3-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-180"></div> <span class="help-tooltip" id="nrps-domain-help-180">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r237c3-details-svg" data-filename="c00237_NODE_6_.._r237c3_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r237c3-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r237c3-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-183"></div> <span class="help-tooltip" id="cb-general-help-183">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r237c3-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r237c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r237c3_1.svg">
        NZ_CP039123 (7801014-7876557): Streptomyces sp. SS52 chromosome, complete genome (58% of genes show similarity), NRPS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r237c3_2.svg">
        NZ_CP029601 (1927434-1979808): Streptomyces sp. WAC 01438 chromosome, complet... (48% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r237c3_3.svg">
        NZ_LIPF01000012 (136427-190557): Streptomyces sp. CB02414 scaffold12, whole g... (51% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r237c3_4.svg">
        NZ_VIWQ01000014 (104505-158528): Streptomyces argenteolus strain 3259 Ga03654... (54% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r237c3_5.svg">
        NZ_BBOK01000008 (269688-323846): Streptomyces sp. NBRC 110030, whole genome s... (42% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r237c3_6.svg">
        NZ_RWGV01000003 (5094-59135): Streptomyces sp. GZWMJZ-114 NODE 12, whole geno... (51% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r237c3_7.svg">
        NZ_CM001165 (304333-357760): Streptomyces sp. Tu6071 chromosome, whole genome... (50% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r237c3_8.svg">
        NZ_KZ195577 (2009280-2100187): Streptomyces sp. CS159 scaffold00001, whole ge... (28% of genes show similarity), NRPS,NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r237c3_9.svg">
        NZ_WWJS01000030 (55457-109572): Streptomyces sp. SID4912 SID4912.c30, whole g... (51% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r237c3_10.svg">
        NZ_CP029342 (6308731-6362130): Streptomyces sp. SM18 chromosome, complete genome (43% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r237c3-download">Download graphic</button>
  </div>
  <div id="clusterblast-r237c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r237c3-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-184"></div> <span class="help-tooltip" id="cb-known-help-184">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r237c3-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r237c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r237c3_1.svg">
        coelibactin
      </option>
      
      <option value = "svg/knownclusterblast_r237c3_2.svg">
        thiazostatin / watasemycin A / watasemycin B / 2-hydroxyphenylthiazoline enantiopyochelin / isopyochelin
      </option>
      
      <option value = "svg/knownclusterblast_r237c3_3.svg">
        gobichelin A / gobichelin B
      </option>
      
      <option value = "svg/knownclusterblast_r237c3_4.svg">
        mycobactin
      </option>
      
      <option value = "svg/knownclusterblast_r237c3_5.svg">
        amychelin
      </option>
      
      <option value = "svg/knownclusterblast_r237c3_6.svg">
        nataxazole
      </option>
      
      <option value = "svg/knownclusterblast_r237c3_7.svg">
        A33853
      </option>
      
      <option value = "svg/knownclusterblast_r237c3_8.svg">
        caboxamycin
      </option>
      
      <option value = "svg/knownclusterblast_r237c3_9.svg">
        caboxamycin
      </option>
      
      <option value = "svg/knownclusterblast_r237c3_10.svg">
        SF2575
      </option>
      
    </select>
    <button id="knownclusterblast-r237c3-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r237c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r237c3-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-185"></div> <span class="help-tooltip" id="cb-sub-help-185">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r237c3-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-186"></div> <span class="help-tooltip" id="MIBiG-help-186">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r237c3-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r237c3-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r237c3"></div>
    <div id="comparison-MIBiG-r237c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000324.1: 8504460-8529581">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000324/index.html#r1c1" target="_new">BGC0000324.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.86 (id:0.67, order:1.00, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
         
        
        <td>0.64</td>
        <td>NRP</td>
        
        <td>coelibactin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.43, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000373.1: 6204479-6222797">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000373/index.html#r1c1" target="_new">BGC0000373.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.48, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>holomycin</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000412.1: 804-15023">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000412/index.html#r1c1" target="_new">BGC0000412.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.46, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>pyochelin</td>
        
        
        <td>Pseudomonas aeruginosa PAO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002091.1: 3593-21565">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002091/index.html#r1c1" target="_new">BGC0002091.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>holomycin</td>
        
        
        <td>Yersinia ruckeri ATCC 29473</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001469.1: 252908-282960">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001469/index.html#r1c1" target="_new">BGC0001469.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>marthiapeptide A</td>
        
        
        <td>Brevibacillus sp. Leaf182</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001343.1: 61-19132">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001343/index.html#r1c1" target="_new">BGC0001343.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>Polyketide</td>
        
        <td>aurachin A, aurachin D</td>
        
        
        <td>Stigmatella aurantiaca Sg a15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001843.1: 2526145-2583676">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001843/index.html#r1c1" target="_new">BGC0001843.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:0.66, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>tridecaptin M</td>
        
        
        <td>Paenibacillus sp. M-152</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890.1: 1392460-1420815">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.49, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
        
        
        <td>Vibrio campbellii ATCC BAA-1116</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r237c3-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000324.1: 8504460-8529581">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000324/index.html#r1c1" target="_new">BGC0000324.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.86 (id:0.67, order:1.00, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
      <td>0.86</td>
      <td>NRP</td>
      <td>coelibactin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.43, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000373.1: 6204479-6222797">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000373/index.html#r1c1" target="_new">BGC0000373.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.48, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>holomycin</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000412.1: 804-15023">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000412/index.html#r1c1" target="_new">BGC0000412.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.46, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>pyochelin</td>
      <td>Pseudomonas aeruginosa PAO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002091.1: 3593-21565">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002091/index.html#r1c1" target="_new">BGC0002091.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>holomycin</td>
      <td>Yersinia ruckeri ATCC 29473</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001469.1: 252908-282960">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001469/index.html#r1c1" target="_new">BGC0001469.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>marthiapeptide A</td>
      <td>Brevibacillus sp. Leaf182</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001343.1: 61-19132">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001343/index.html#r1c1" target="_new">BGC0001343.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>aurachin A, aurachin D</td>
      <td>Stigmatella aurantiaca Sg a15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001843.1: 2526145-2583676">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001843/index.html#r1c1" target="_new">BGC0001843.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:0.66, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>tridecaptin M</td>
      <td>Paenibacillus sp. M-152</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890.1: 1392460-1420815">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.49, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
      <td>Vibrio campbellii ATCC BAA-1116</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r237c3-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-187"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-187">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r237c3-pfam-details-svg" data-filename="c00237_NODE_6_.._r237c3_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r237c3-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r237c3-nrps_pks_products" data-name="r237c3-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r237c3-nrps_pks" data-name="r237c3-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r237c3-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-181"></div> <span class="help-tooltip" id="nrps-structures-help-181">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 212871 - 265197:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r237c3-3" class="smiles-canvas" data-smiles="NC(CS)C(=O)NC(CS)C(=O)NC(CS)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(cys - cys) + (cys)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys,cys&amp;nrps2=cys" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*,cys*&amp;nrps2=cys*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r237c3-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CS)C(=O)NC(CS)C(=O)NC(CS)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r237c3-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-182"></div> <span class="help-tooltip" id="nrps-monomers-help-182">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds232871_238400</span></strong>: cys
      <div class="collapser collapser-target-cds232871_238400 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (543..942)</strong>: cys
          <div class="collapser collapser-target-cds232871_238400 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  cys
              <div class="collapser collapser-target-cds232871_238400-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Small clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Single AA prediction:</dt>
   <dd>cys</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLYNLSLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>cys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds238396_245197</span></strong>: cys - cys
      <div class="collapser collapser-target-cds238396_245197 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys,cys" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*,cys*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (581..976)</strong>: cys
          <div class="collapser collapser-target-cds238396_245197 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  cys
              <div class="collapser collapser-target-cds238396_245197-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DlwQLGLVhK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1642..2068)</strong>: cys
          <div class="collapser collapser-target-cds238396_245197 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  cys
              <div class="collapser collapser-target-cds238396_245197-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Small clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Single AA prediction:</dt>
   <dd>cys</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLYNLSmIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>cys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds245378_247016</span></strong>: X
      <div class="collapser collapser-target-cds245378_247016 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (32..448)</strong>: X
          <div class="collapser collapser-target-cds245378_247016 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds245378_247016-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>dhb, sal</dd>
   <dt>Small clusters prediction:</dt>
   <dd>dhb, sal</dd>
   <dt>Single AA prediction:</dt>
   <dd>dhb</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">afPSQGViNK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>dhb</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r244c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00244_NODE_43.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-283"></div> <span class="help-tooltip" id="region_svg-help-283">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00244_NODE_43...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r244c1-svg" data-filename="c00244_NODE_43.._r244c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 27,609 nt. (total: 27,609 nt)
        
        <a class="cluster-rules-header" id="r244c1-rules-header" href="#r244c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r244c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r244c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-284"></div> <span class="help-tooltip" id="focus-panel-help-284">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r244c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r244c1-nrps_pks" data-name="r244c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r244c1-clusterblast" data-name="r244c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r244c1-knownclusterblast" data-name="r244c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r244c1-subclusterblast" data-name="r244c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r244c1-MIBiG-cluster-compare" data-name="r244c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r244c1-pfam-details" data-name="r244c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r244c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-188"></div> <span class="help-tooltip" id="nrps-domain-help-188">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r244c1-details-svg" data-filename="c00244_NODE_43.._r244c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r244c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r244c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-191"></div> <span class="help-tooltip" id="cb-general-help-191">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r244c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r244c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r244c1_1.svg">
        NZ_KL585949 (177387-330617): Streptomyces sp. NRRL S-474 P279 Doro1 scaffold4... (14% of genes show similarity), NRPS,T1PKS,T3PKS,butyrolactone,other
      </option>
      
      <option value = "svg/clusterblast_r244c1_2.svg">
        NZ_CP027297 (6501004-6580734): Streptomyces sp. SGAir0924 chromosome, complet... (20% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r244c1_3.svg">
        NZ_KZ819185 (488550-566486): Streptomyces sp. CS207 scaffold00002, whole geno... (22% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r244c1_4.svg">
        NZ_LT629768 (3748808-3827608): Streptomyces sp. 2114.2 chromosome I (21% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r244c1_5.svg">
        NZ_CP042324 (3524876-3603956): Streptomyces coelicolor A3(2) strain CFB NBC 0... (22% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r244c1_6.svg">
        NZ_CP026730 (2884074-2986849): Streptomyces sp. CB09001 chromosome, complete ... (16% of genes show similarity), NRPS,T1PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r244c1_7.svg">
        NZ_CP039123 (6294253-6369829): Streptomyces sp. SS52 chromosome, complete genome (25% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r244c1_8.svg">
        NZ_BEWD01000011 (217053-297488): Streptomyces rubrogriseus strain NBRC 15455,... (25% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r244c1_9.svg">
        NZ_JTIY01000002 (691244-774422): Streptomyces sp. AcH 505 scaffold00002, whol... (34% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r244c1_10.svg">
        NZ_CP042278 (1583458-1663462): Streptomyces sp. WAC06273 strain WAC6273 subst... (28% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r244c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r244c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r244c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-192"></div> <span class="help-tooltip" id="cb-known-help-192">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r244c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r244c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r244c1_1.svg">
        lipopeptide 8D1-1 / lipopeptide 8D1-2
      </option>
      
      <option value = "svg/knownclusterblast_r244c1_2.svg">
        CDA1b / CDA2a / CDA2b / CDA3a / CDA3b / CDA4a / CDA4b
      </option>
      
      <option value = "svg/knownclusterblast_r244c1_3.svg">
        cadaside A / cadaside B
      </option>
      
      <option value = "svg/knownclusterblast_r244c1_4.svg">
        telomycin
      </option>
      
      <option value = "svg/knownclusterblast_r244c1_5.svg">
        friulimicin A / friulimicin B / friulimicin C / friulimicin D
      </option>
      
      <option value = "svg/knownclusterblast_r244c1_6.svg">
        glycinocin A
      </option>
      
      <option value = "svg/knownclusterblast_r244c1_7.svg">
        atratumycin
      </option>
      
      <option value = "svg/knownclusterblast_r244c1_8.svg">
        malacidin A / malacidin B
      </option>
      
      <option value = "svg/knownclusterblast_r244c1_9.svg">
        balhimycin
      </option>
      
      <option value = "svg/knownclusterblast_r244c1_10.svg">
        decaplanin
      </option>
      
    </select>
    <button id="knownclusterblast-r244c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r244c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r244c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-193"></div> <span class="help-tooltip" id="cb-sub-help-193">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r244c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r244c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r244c1_1.svg">
        Y16952_c2: balhimycin hydroxyphenylglycine (100% of genes show similarity), HPG
      </option>
      
      <option value = "svg/subclusterblast_r244c1_2.svg">
        AF386507_c1: complestatin hydroxyphenylglycine (75% of genes show similarity), HPG
      </option>
      
      <option value = "svg/subclusterblast_r244c1_3.svg">
        HE589771_c2: vancomycin hydroxyphenylglycine (66% of genes show similarity), HPG
      </option>
      
      <option value = "svg/subclusterblast_r244c1_4.svg">
        AJ632270_c2: teicoplanin hydroxyphenylglycine (66% of genes show similarity), HPG
      </option>
      
    </select>
    <button id="subclusterblast-r244c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r244c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r244c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-194"></div> <span class="help-tooltip" id="MIBiG-help-194">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r244c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r244c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r244c1"></div>
    <div id="comparison-MIBiG-r244c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.44, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001434.1: 0-14218">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001434/index.html#r1c1" target="_new">BGC0001434.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.44, order:0.81, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>nematophin</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001698.1: 0-14218">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001698/index.html#r1c1" target="_new">BGC0001698.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.44, order:0.81, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>nevaltophin A, nevaltophin B, nevaltophin C, nevaltophin D</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000901.1: 1761-23020">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000901/index.html#r1c1" target="_new">BGC0000901.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.43, order:0.69, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>Other</td>
        
        <td>ferrichrome</td>
        
        
        <td>Aspergillus niger</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>vicibactin</td>
        
        
        <td>Rhizobium etli CFN 42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>AM-toxin</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.46, order:0.75, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r244c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.44, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001434.1: 0-14218">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001434/index.html#r1c1" target="_new">BGC0001434.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.44, order:0.81, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>nematophin</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001698.1: 0-14218">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001698/index.html#r1c1" target="_new">BGC0001698.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.44, order:0.81, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>nevaltophin A, nevaltophin B, nevaltophin C, nevaltophin D</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000901.1: 1761-23020">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000901/index.html#r1c1" target="_new">BGC0000901.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.43, order:0.69, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>ferrichrome</td>
      <td>Aspergillus niger</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>vicibactin</td>
      <td>Rhizobium etli CFN 42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>AM-toxin</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.46, order:0.75, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r244c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-195"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-195">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r244c1-pfam-details-svg" data-filename="c00244_NODE_43.._r244c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r244c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r244c1-nrps_pks_products" data-name="r244c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r244c1-nrps_pks" data-name="r244c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r244c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-189"></div> <span class="help-tooltip" id="nrps-structures-help-189">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 27609:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r244c1-1" class="smiles-canvas" data-smiles="NC(CO)C(=O)NC(C(O)C)C(=O)NC(c1ccc(O)cc1)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ser - thr) + (D-hpg)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,thr&amp;nrps2=d-hpg" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,thr*&amp;nrps2=d-hpg*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r244c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CO)C(=O)NC(C(O)C)C(=O)NC(c1ccc(O)cc1)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r244c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-190"></div> <span class="help-tooltip" id="nrps-monomers-help-190">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds4964_15539</span></strong>: ser - thr
      <div class="collapser collapser-target-cds4964_15539 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,thr" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,thr*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (477..873)</strong>: ser
          <div class="collapser collapser-target-cds4964_15539 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds4964_15539-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWHFSLVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1551..1970)</strong>: thr
          <div class="collapser collapser-target-cds4964_15539 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds4964_15539-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNVGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds21334_24697</span></strong>: hpg
      <div class="collapser collapser-target-cds21334_24697 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=hpg" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=hpg*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (48..446)</strong>: hpg
          <div class="collapser collapser-target-cds21334_24697 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hpg
              <div class="collapser collapser-target-cds21334_24697-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>dhpg, hpg</dd>
   <dt>Single AA prediction:</dt>
   <dd>hpg</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVYHLGLLCK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>hpg</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds24693_26886</span></strong>: X
      <div class="collapser collapser-target-cds24693_26886 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (472..712)</strong>: X
          <div class="collapser collapser-target-cds24693_26886 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asp, asn
              <div class="collapser collapser-target-cds24693_26886-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>asp, asn</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLTK-----K</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r247c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00247_NODE_34.. - Region 1 - siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-285"></div> <span class="help-tooltip" id="region_svg-help-285">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00247_NODE_34...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r247c1-svg" data-filename="c00247_NODE_34.._r247c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 11,174 - 22,946 nt. (total: 11,773 nt)
        
        <a class="cluster-rules-header" id="r247c1-rules-header" href="#r247c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r247c1-rules">siderophore: IucA_IucC</div>
      
      <div class="region-svg-container">
       <div id='r247c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-286"></div> <span class="help-tooltip" id="focus-panel-help-286">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r247c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r247c1-clusterblast" data-name="r247c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r247c1-knownclusterblast" data-name="r247c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r247c1-subclusterblast" data-name="r247c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r247c1-MIBiG-cluster-compare" data-name="r247c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r247c1-pfam-details" data-name="r247c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r247c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-196"></div> <span class="help-tooltip" id="cb-general-help-196">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r247c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r247c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r247c1_1.svg">
        NZ_KZ819184 (1446844-1458617): Streptomyces sp. CS207 scaffold00001, whole ge... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r247c1_2.svg">
        NZ_CP027297 (3074078-3085004): Streptomyces sp. SGAir0924 chromosome, complet... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r247c1_3.svg">
        NZ_CP039123 (2995235-3006194): Streptomyces sp. SS52 chromosome, complete genome (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r247c1_4.svg">
        NZ_CP012382 (2980925-2991871): Streptomyces ambofaciens ATCC 23877 chromosome... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r247c1_5.svg">
        NZ_CP012949 (2931338-2942282): Streptomyces ambofaciens strain DSM 40697 chro... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r247c1_6.svg">
        NZ_KZ195574 (5509882-5521667): Streptomyces sp. CS113 scaffold00001, whole ge... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r247c1_7.svg">
        NZ_BEWD01000007 (266566-278345): Streptomyces rubrogriseus strain NBRC 15455,... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r247c1_8.svg">
        NZ_GG657756 (5262727-5274512): Streptomyces lividans TK24 supercont1.1, whole... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r247c1_9.svg">
        NZ_CP019724 (5490638-5502408): Streptomyces pactum strain ACT12 chromosome, c... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r247c1_10.svg">
        NZ_CP042324 (3034638-3045610): Streptomyces coelicolor A3(2) strain CFB NBC 0... (100% of genes show similarity), siderophore
      </option>
      
    </select>
    <button id= "clusterblast-r247c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r247c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r247c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-197"></div> <span class="help-tooltip" id="cb-known-help-197">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r247c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r247c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r247c1_1.svg">
        desferrioxamin B / desferrioxamine E
      </option>
      
      <option value = "svg/knownclusterblast_r247c1_2.svg">
        desferrioxamine
      </option>
      
      <option value = "svg/knownclusterblast_r247c1_3.svg">
        desferrioxamin B
      </option>
      
      <option value = "svg/knownclusterblast_r247c1_4.svg">
        desferrioxamine E
      </option>
      
      <option value = "svg/knownclusterblast_r247c1_5.svg">
        putrebactin / avaroferrin
      </option>
      
      <option value = "svg/knownclusterblast_r247c1_6.svg">
        desferrioxamine E
      </option>
      
      <option value = "svg/knownclusterblast_r247c1_7.svg">
        bisucaberin B
      </option>
      
    </select>
    <button id="knownclusterblast-r247c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r247c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r247c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-198"></div> <span class="help-tooltip" id="cb-sub-help-198">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r247c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-199"></div> <span class="help-tooltip" id="MIBiG-help-199">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r247c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r247c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r247c1"></div>
    <div id="comparison-MIBiG-r247c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">siderophore</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000941.1: 5578821-5584948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000941/index.html#r1c1" target="_new">BGC0000941.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.83 (id:0.58, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
         
        
        <td>0.59</td>
        <td>Other</td>
        
        <td>desferrioxamin B</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001478.1: 100-5365">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001478/index.html#r1c1" target="_new">BGC0001478.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
         
        
        <td>0.58</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Streptomyces sp. ID38640</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940.1: 3033354-3040682">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.80 (id:0.70, order:0.83, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
         
        
        <td>0.53</td>
        <td>Other</td>
        
        <td>desferrioxamin B, desferrioxamine E</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453.1: 0-7234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.77 (id:0.61, order:0.83, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
         
        
        <td>0.48</td>
        <td>Other</td>
        
        <td>desferrioxamine</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.52, order:0.66, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Pantoea agglomerans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001531.1: 1112-9953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001531/index.html#r1c1" target="_new">BGC0001531.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.49, order:0.57, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Other</td>
        
        <td>bisucaberin B</td>
        
        
        <td>Tenacibaculum mesophilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000244.1: 204-8850">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000244/index.html#r1c1" target="_new">BGC0000244.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.46, order:0.48, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>macrotetrolide</td>
        
        
        <td>Streptomyces griseus subsp. griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>ochrobactin</td>
        
        
        <td>Ochrobactrum anthropi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499.1: 2267-11578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.43, order:0.42, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>aerobactin</td>
        
        
        <td>Pantoea ananatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000466.1: 0-35805">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000466/index.html#r1c1" target="_new">BGC0000466.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>NRP</td>
        
        <td>yatakemycin</td>
        
        
        <td>Streptomyces sp. TP-A2060</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r247c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000941.1: 5578821-5584948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000941/index.html#r1c1" target="_new">BGC0000941.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.83 (id:0.58, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
      <td>0.83</td>
      <td>Other</td>
      <td>desferrioxamin B</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001478.1: 100-5365">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001478/index.html#r1c1" target="_new">BGC0001478.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
      <td>0.83</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Streptomyces sp. ID38640</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940.1: 3033354-3040682">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.80 (id:0.70, order:0.83, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
      <td>0.80</td>
      <td>Other</td>
      <td>desferrioxamin B, desferrioxamine E</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453.1: 0-7234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.77 (id:0.61, order:0.83, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
      <td>0.77</td>
      <td>Other</td>
      <td>desferrioxamine</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.52, order:0.66, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Pantoea agglomerans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001531.1: 1112-9953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001531/index.html#r1c1" target="_new">BGC0001531.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.49, order:0.57, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Other</td>
      <td>bisucaberin B</td>
      <td>Tenacibaculum mesophilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000244.1: 204-8850">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000244/index.html#r1c1" target="_new">BGC0000244.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.46, order:0.48, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Polyketide</td>
      <td>macrotetrolide</td>
      <td>Streptomyces griseus subsp. griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>ochrobactin</td>
      <td>Ochrobactrum anthropi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499.1: 2267-11578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.43, order:0.42, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>aerobactin</td>
      <td>Pantoea ananatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000466.1: 0-35805">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000466/index.html#r1c1" target="_new">BGC0000466.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>NRP</td>
      <td>yatakemycin</td>
      <td>Streptomyces sp. TP-A2060</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r247c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-200"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-200">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r247c1-pfam-details-svg" data-filename="c00247_NODE_34.._r247c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r247c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r250c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00250_NODE_18.. - Region 1 - lanthipeptide-class-iii <div class="help-container"> <div class="help-icon" data-id="region_svg-help-287"></div> <span class="help-tooltip" id="region_svg-help-287">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00250_NODE_18...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r250c1-svg" data-filename="c00250_NODE_18.._r250c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 33,832 - 56,435 nt. (total: 22,604 nt)
        
        <a class="cluster-rules-header" id="r250c1-rules-header" href="#r250c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r250c1-rules">lanthipeptide-class-iii: (micKC and not PKS_KS)</div>
      
      <div class="region-svg-container">
       <div id='r250c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-288"></div> <span class="help-tooltip" id="focus-panel-help-288">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r250c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r250c1-lanthipeptides" data-name="r250c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r250c1-clusterblast" data-name="r250c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r250c1-knownclusterblast" data-name="r250c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r250c1-subclusterblast" data-name="r250c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r250c1-MIBiG-cluster-compare" data-name="r250c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r250c1-pfam-details" data-name="r250c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r250c1-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-201"></div> <span class="help-tooltip" id="lanthi-body-help-201">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lanthipeptide(s) for cds43831_46435 - Putative Class III</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds43408_43522</span> leader / core peptide
          </dt>
          <dd>
            MSLLDLQEMELPTDSAE
            -
            E<span class="dhb">Dhb</span>VLG<span class="dha">Dha</span>GK<span class="dha">Dha</span>QH<span class="cys">C</span><span class="dha">Dha</span>DA<span class="dha">Dha</span>WVF<span class="cys">C</span>
          <dt>
            <span class="serif">cds43607_43721</span> leader / core peptide
          </dt>
          <dd>
            MSLLDLQEMELSTDSAE
            -
            E<span class="dhb">Dhb</span>VLG<span class="dha">Dha</span>GQ<span class="dha">Dha</span>QH<span class="cys">C</span><span class="dha">Dha</span>DA<span class="dha">Dha</span>WVF<span class="cys">C</span>
        </dd>
      </dl>
    
    <div class="legend">
      <h4> Legend: </h4>
      <div>
        <span class="dha">Dha</span>: Didehydroalanine<br>
        <span class="dhb">Dhb</span>: Didehydrobutyrine<br>
      </div>
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r250c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-203"></div> <span class="help-tooltip" id="cb-general-help-203">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r250c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r250c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r250c1_1.svg">
        NZ_CP039123 (4260535-4280291): Streptomyces sp. SS52 chromosome, complete genome (88% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r250c1_2.svg">
        NZ_CP027297 (4366293-4388897): Streptomyces sp. SGAir0924 chromosome, complet... (93% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r250c1_3.svg">
        NZ_CP042278 (3654727-3677331): Streptomyces sp. WAC06273 strain WAC6273 subst... (83% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r250c1_4.svg">
        NZ_KZ819185 (2140395-2162999): Streptomyces sp. CS207 scaffold00002, whole ge... (87% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r250c1_5.svg">
        NZ_CP058693 (6316130-6338068): Streptomyces sp. Rer75 chromosome (43% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r250c1_6.svg">
        NZ_JACHJI010000005 (122532-191473): Streptomyces griseostramineus strain CECT... (9% of genes show similarity), NRPS,T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r250c1_7.svg">
        NZ_CP017717 (632883-655445): Nonomuraea sp. ATCC 55076 chromosome, complete g... (27% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r250c1_8.svg">
        NZ_FNVT01000002 (358561-398858): Nonomuraea solani strain CGMCC 4.7037, whole... (14% of genes show similarity), lanthipeptide,terpene
      </option>
      
      <option value = "svg/clusterblast_r250c1_9.svg">
        NZ_JABWGN010000023 (50341-72957): Nonomuraea sp. SMC257 NODE 23 length 123153... (35% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r250c1_10.svg">
        NZ_CP036455 (4271275-4314656): Streptomonospora sp. M2 chromosome, complete g... (15% of genes show similarity), NRPS-like
      </option>
      
    </select>
    <button id= "clusterblast-r250c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r250c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r250c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-204"></div> <span class="help-tooltip" id="cb-known-help-204">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r250c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r250c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r250c1_1.svg">
        catenulipeptin
      </option>
      
      <option value = "svg/knownclusterblast_r250c1_2.svg">
        SapB
      </option>
      
      <option value = "svg/knownclusterblast_r250c1_3.svg">
        labyrinthopeptin A2 / labyrinthopeptin A1 / labyrinthopeptin A3
      </option>
      
    </select>
    <button id="knownclusterblast-r250c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r250c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r250c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-205"></div> <span class="help-tooltip" id="cb-sub-help-205">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r250c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-206"></div> <span class="help-tooltip" id="MIBiG-help-206">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r250c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r250c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r250c1"></div>
    <div id="comparison-MIBiG-r250c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-iii</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.50, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>RiPP</td>
        
        <td>SapB</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>RiPP</td>
        
        <td>venezuelin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496.1: 2822112-2829457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.49, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>RiPP</td>
        
        <td>AmfS</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.50, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>RiPP</td>
        
        <td>catenulipeptin</td>
        
        
        <td>Catenulispora acidiphila DSM 44928</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.49, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>RiPP</td>
        
        <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
        
        
        <td>Actinomadura namibiensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001579.1: 85465-94303">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001579/index.html#r1c1" target="_new">BGC0001579.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>RiPP</td>
        
        <td>duramycin</td>
        
        
        <td>Streptomyces cinnamoneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.52, order:0.40, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>informatipeptin</td>
        
        
        <td>Streptomyces viridochromogenes DSM 40736</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001633.1: 52623-58838">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001633/index.html#r1c1" target="_new">BGC0001633.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Other</td>
        
        <td>ketomemicin B3, ketomemicin B4</td>
        
        
        <td>Streptomyces mobaraensis NBRC 13819 = DSM 40847</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000513.1: 4704393-4710596">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000513/index.html#r1c1" target="_new">BGC0000513.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.48, order:0.57, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>Ery-9, Ery-6, Ery-8, Ery-7, Ery-5, Ery-4, Ery-3</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r250c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.50, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>RiPP</td>
      <td>SapB</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>RiPP</td>
      <td>venezuelin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496.1: 2822112-2829457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.49, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>RiPP</td>
      <td>AmfS</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.50, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>RiPP</td>
      <td>catenulipeptin</td>
      <td>Catenulispora acidiphila DSM 44928</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.49, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>RiPP</td>
      <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
      <td>Actinomadura namibiensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001579.1: 85465-94303">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001579/index.html#r1c1" target="_new">BGC0001579.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>RiPP</td>
      <td>duramycin</td>
      <td>Streptomyces cinnamoneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.52, order:0.40, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>informatipeptin</td>
      <td>Streptomyces viridochromogenes DSM 40736</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001633.1: 52623-58838">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001633/index.html#r1c1" target="_new">BGC0001633.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Other</td>
      <td>ketomemicin B3, ketomemicin B4</td>
      <td>Streptomyces mobaraensis NBRC 13819 = DSM 40847</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000513.1: 4704393-4710596">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000513/index.html#r1c1" target="_new">BGC0000513.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.48, order:0.57, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>RiPP</td>
      <td>Ery-9, Ery-6, Ery-8, Ery-7, Ery-5, Ery-4, Ery-3</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r250c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-207"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-207">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r250c1-pfam-details-svg" data-filename="c00250_NODE_18.._r250c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r250c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r250c1-lanthipeptides" data-name="r250c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r250c1-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-202"></div> <span class="help-tooltip" id="lanthi-side-help-202">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

    <dl class ="prediction-text">
    <dt><span class="serif">cds43408_43522</span> - Class III</dt>
        <dd>Cleavage pHMM score: 6.60</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 2049.9 Da</dd>
        <dd>Molecular weight: 2051.2 Da</dd>
        <dd>Number of bridges: 2</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 2069.3 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2087.3 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2105.3 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
            </dl>
          </dd>
        <dt><span class="serif">cds43607_43721</span> - Class III</dt>
        <dd>Cleavage pHMM score: 4.90</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 2049.9 Da</dd>
        <dd>Molecular weight: 2051.2 Da</dd>
        <dd>Number of bridges: 2</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 2069.2 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2087.2 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2105.3 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
            </dl>
          </dd>
        
  </dl>

</div>

    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r251c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00251_NODE_13.. - Region 1 - lanthipeptide-class-v <div class="help-container"> <div class="help-icon" data-id="region_svg-help-289"></div> <span class="help-tooltip" id="region_svg-help-289">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00251_NODE_13...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r251c1-svg" data-filename="c00251_NODE_13.._r251c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 6,769 nt. (total: 6,769 nt)
        
        <a class="cluster-rules-header" id="r251c1-rules-header" href="#r251c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r251c1-rules">lanthipeptide-class-v: (APH and HopA1)</div>
      
      <div class="region-svg-container">
       <div id='r251c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-290"></div> <span class="help-tooltip" id="focus-panel-help-290">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r251c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r251c1-lanthipeptides" data-name="r251c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r251c1-clusterblast" data-name="r251c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r251c1-knownclusterblast" data-name="r251c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r251c1-subclusterblast" data-name="r251c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r251c1-MIBiG-cluster-compare" data-name="r251c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r251c1-pfam-details" data-name="r251c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r251c1-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-208"></div> <span class="help-tooltip" id="lanthi-body-help-208">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    <div class="details-text">
      Lanthipeptides - No core peptides found.
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r251c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-210"></div> <span class="help-tooltip" id="cb-general-help-210">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r251c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r251c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r251c1_1.svg">
        NZ_CP027297 (4270944-4313277): Streptomyces sp. SGAir0924 chromosome, complet... (18% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r251c1_2.svg">
        NZ_KZ819185 (2045672-2088014): Streptomyces sp. CS207 scaffold00002, whole ge... (18% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r251c1_3.svg">
        NZ_CP039123 (4164285-4206627): Streptomyces sp. SS52 chromosome, complete genome (17% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r251c1_4.svg">
        NZ_CP034539 (5513855-5556227): Streptomyces cyaneochromogenes strain MK-45 ch... (17% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r251c1_5.svg">
        NZ_CP047318 (4671381-4713992): Streptomyces sp. HM190 chromosome, complete ge... (17% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r251c1_6.svg">
        NZ_CP024985 (4504294-4545817): Streptomyces lavendulae subsp. lavendulae stra... (16% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r251c1_7.svg">
        NZ_KB899009 (123942-166305): Streptomyces sp. CNB091 D581DRAFT scaffold00020.... (17% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r251c1_8.svg">
        NZ_FNHV01000011 (278400-320709): Streptomyces sp. cf386, whole genome shotgun... (18% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r251c1_9.svg">
        NC_020504 (4992059-5038915): Streptomyces davaonensis JCM 4913, complete genome (13% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r251c1_10.svg">
        NZ_CP019724 (355288-475720): Streptomyces pactum strain ACT12 chromosome, com... (5% of genes show similarity), NRPS,T1PKS,lanthidin
      </option>
      
    </select>
    <button id= "clusterblast-r251c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r251c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r251c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-211"></div> <span class="help-tooltip" id="cb-known-help-211">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r251c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-212"></div> <span class="help-tooltip" id="cb-sub-help-212">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r251c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-213"></div> <span class="help-tooltip" id="MIBiG-help-213">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r251c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r251c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r251c1"></div>
    <div id="comparison-MIBiG-r251c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-v</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001150.1: 12-23077">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001150/index.html#r1c1" target="_new">BGC0001150.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>pamamycin-607</td>
        
        
        <td>Streptomyces alboniger</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001610.1: 410-17543">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001610/index.html#r1c1" target="_new">BGC0001610.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Other</td>
        
        <td>gougerotin</td>
        
        
        <td>Streptomyces ahygroscopicus subsp. wuzhouensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001188.1: 4621759-4630230">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001188/index.html#r1c1" target="_new">BGC0001188.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
        <td>0.12</td>
        <td>RiPP</td>
        
        <td>legonaridin</td>
        
        
        <td>Streptomyces sp. CT34</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001735.1: 0-39550">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001735/index.html#r1c1" target="_new">BGC0001735.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
        <td>0.11</td>
        <td>Other</td>
        
        <td>pentostatine, vidarabine</td>
        
        
        <td>Streptomyces antibioticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001225.1: 5267-32621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001225/index.html#r1c1" target="_new">BGC0001225.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.29 (id:0.46, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
         
        
        <td>0.07</td>
        <td>NRP</td>
        
        <td>celesticetin</td>
        
        
        <td>Streptomyces caelestis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000700.1: 1-67903">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000700/index.html#r1c1" target="_new">BGC0000700.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.26 (id:0.44, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
         
        
        <td>0.06</td>
        <td>Saccharide</td>
        
        <td>istamycin</td>
        
        
        <td>Streptomyces tenjimariensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000907.1: 0-38075">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000907/index.html#r1c1" target="_new">BGC0000907.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.25 (id:0.45, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
         
        
        <td>0.06</td>
        <td>Other</td>
        
        <td>lincomycin</td>
        
        
        <td>Streptomyces lincolnensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000199.1: 0-40743">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000199/index.html#r1c1" target="_new">BGC0000199.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.24 (id:0.43, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
         
        
        <td>0.06</td>
        <td>Polyketide, Saccharide</td>
        
        <td>arimetamycin B, arimetamycin C, arimetamycin A</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001956.1: 151-29239">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001956/index.html#r1c1" target="_new">BGC0001956.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.23 (id:0.44, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
         
        
        <td>0.05</td>
        <td>Polyketide</td>
        
        <td>miharamycin A, miharamycin B</td>
        
        
        <td>Streptomyces miharaensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000414.1: 5398-44076">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000414/index.html#r1c1" target="_new">BGC0000414.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.23 (id:0.42, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
         
        
        <td>0.05</td>
        <td>NRP</td>
        
        <td>quinocarcin</td>
        
        
        <td>Streptomyces melanovinaceus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r251c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001150.1: 12-23077">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001150/index.html#r1c1" target="_new">BGC0001150.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Polyketide</td>
      <td>pamamycin-607</td>
      <td>Streptomyces alboniger</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001610.1: 410-17543">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001610/index.html#r1c1" target="_new">BGC0001610.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Other</td>
      <td>gougerotin</td>
      <td>Streptomyces ahygroscopicus subsp. wuzhouensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001188.1: 4621759-4630230">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001188/index.html#r1c1" target="_new">BGC0001188.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
      <td>0.40</td>
      <td>RiPP</td>
      <td>legonaridin</td>
      <td>Streptomyces sp. CT34</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001735.1: 0-39550">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001735/index.html#r1c1" target="_new">BGC0001735.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
      <td>0.38</td>
      <td>Other</td>
      <td>pentostatine, vidarabine</td>
      <td>Streptomyces antibioticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001225.1: 5267-32621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001225/index.html#r1c1" target="_new">BGC0001225.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.29 (id:0.46, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
      <td>0.29</td>
      <td>NRP</td>
      <td>celesticetin</td>
      <td>Streptomyces caelestis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000700.1: 1-67903">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000700/index.html#r1c1" target="_new">BGC0000700.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.26 (id:0.44, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
      <td>0.26</td>
      <td>Saccharide</td>
      <td>istamycin</td>
      <td>Streptomyces tenjimariensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000907.1: 0-38075">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000907/index.html#r1c1" target="_new">BGC0000907.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.25 (id:0.45, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
      <td>0.25</td>
      <td>Other</td>
      <td>lincomycin</td>
      <td>Streptomyces lincolnensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000199.1: 0-40743">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000199/index.html#r1c1" target="_new">BGC0000199.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.24 (id:0.43, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
      <td>0.24</td>
      <td>Polyketide, Saccharide</td>
      <td>arimetamycin B, arimetamycin C, arimetamycin A</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001956.1: 151-29239">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001956/index.html#r1c1" target="_new">BGC0001956.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.23 (id:0.44, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
      <td>0.23</td>
      <td>Polyketide</td>
      <td>miharamycin A, miharamycin B</td>
      <td>Streptomyces miharaensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000414.1: 5398-44076">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000414/index.html#r1c1" target="_new">BGC0000414.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.23 (id:0.42, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
      <td>0.23</td>
      <td>NRP</td>
      <td>quinocarcin</td>
      <td>Streptomyces melanovinaceus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r251c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-214"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-214">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r251c1-pfam-details-svg" data-filename="c00251_NODE_13.._r251c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r251c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r251c1-lanthipeptides" data-name="r251c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r251c1-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-209"></div> <span class="help-tooltip" id="lanthi-side-help-209">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

  <div class="prediction-text">No core peptides found.</div>

</div>

    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r259c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00259_NODE_48.. - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-291"></div> <span class="help-tooltip" id="region_svg-help-291">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00259_NODE_48...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r259c1-svg" data-filename="c00259_NODE_48.._r259c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 11,004 nt. (total: 11,004 nt)
        
        <a class="cluster-rules-header" id="r259c1-rules-header" href="#r259c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r259c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r259c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-292"></div> <span class="help-tooltip" id="focus-panel-help-292">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r259c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r259c1-nrps_pks" data-name="r259c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r259c1-clusterblast" data-name="r259c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r259c1-knownclusterblast" data-name="r259c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r259c1-subclusterblast" data-name="r259c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r259c1-MIBiG-cluster-compare" data-name="r259c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r259c1-pfam-details" data-name="r259c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r259c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-215"></div> <span class="help-tooltip" id="nrps-domain-help-215">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r259c1-details-svg" data-filename="c00259_NODE_48.._r259c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r259c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r259c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-216"></div> <span class="help-tooltip" id="cb-general-help-216">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r259c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r259c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r259c1_1.svg">
        NZ_CP007155 (1541779-1650787): Kutzneria albida DSM 43870 chromosome, complet... (15% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r259c1_2.svg">
        NZ_JACHJG010000006 (192877-267393): Streptomyces netropsis strain CECT 3265 G... (14% of genes show similarity), T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r259c1_3.svg">
        NZ_CP045096 (3779869-3844096): Streptomyces sp. GY16 chromosome, complete genome (8% of genes show similarity), T1PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r259c1_4.svg">
        NZ_CP050692 (1320699-1451170): Streptomyces antibioticus strain DSM 41481 chr... (4% of genes show similarity), NRPS-like,T1PKS,T2PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r259c1_5.svg">
        NZ_CP034550 (4383453-4468362): Saccharothrix syringae strain NRRL B-16468 chr... (12% of genes show similarity), NRPS,T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r259c1_6.svg">
        NZ_CSTD01000001 (1085881-1146702): Mycobacterium bohemicum DSM 44277, whole g... (8% of genes show similarity), T1PKS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r259c1_7.svg">
        NC_008278 (2737802-2835803): Frankia alni ACN14a, complete genome (5% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r259c1_8.svg">
        NZ_CP034587 (467666-619676): Streptomyces luteoverticillatus strain CGMCC 150... (22% of genes show similarity), NRPS-like,T1PKS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r259c1_9.svg">
        NZ_CP058693 (3615134-3824665): Streptomyces sp. Rer75 chromosome (14% of genes show similarity), NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r259c1_10.svg">
        NZ_CP015726 (4048739-4258212): Streptomyces sp. RTd22 chromosome, complete ge... (14% of genes show similarity), NRPS-like,T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r259c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r259c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r259c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-217"></div> <span class="help-tooltip" id="cb-known-help-217">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r259c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r259c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r259c1_1.svg">
        spectinabilin / orinocin / SNF4435C / SNF4435D
      </option>
      
      <option value = "svg/knownclusterblast_r259c1_2.svg">
        midecamycin
      </option>
      
      <option value = "svg/knownclusterblast_r259c1_3.svg">
        niphimycins C-E
      </option>
      
      <option value = "svg/knownclusterblast_r259c1_4.svg">
        aculeximycin
      </option>
      
      <option value = "svg/knownclusterblast_r259c1_5.svg">
        cremimycin
      </option>
      
      <option value = "svg/knownclusterblast_r259c1_6.svg">
        nystatin A1
      </option>
      
      <option value = "svg/knownclusterblast_r259c1_7.svg">
        nystatin-like Pseudonocardia polyene
      </option>
      
      <option value = "svg/knownclusterblast_r259c1_8.svg">
        griseochelin
      </option>
      
      <option value = "svg/knownclusterblast_r259c1_9.svg">
        pimaricin
      </option>
      
      <option value = "svg/knownclusterblast_r259c1_10.svg">
        hitachimycin
      </option>
      
    </select>
    <button id="knownclusterblast-r259c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r259c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r259c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-218"></div> <span class="help-tooltip" id="cb-sub-help-218">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r259c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-219"></div> <span class="help-tooltip" id="MIBiG-help-219">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r259c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r259c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r259c1"></div>
    <div id="comparison-MIBiG-r259c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>esperamicin</td>
        
        
        <td>Actinomadura verrucosospora</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.47, order:1.00, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>oronofacic acid</td>
        
        
        <td>Pseudomonas syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.19</td>
        <td>Terpene, Polyketide</td>
        
        <td>pyripyropene A</td>
        
        
        <td>unidentified unclassified sequences.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257.1: 399-7066">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Polyketide</td>
        
        <td>1,3,6,8-tetrahydroxynaphthalene</td>
        
        
        <td>Nodulisporium sp. ATCC74245</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107.1: 1136198-1142873">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Polyketide</td>
        
        <td>naphthopyrone</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Polyketide</td>
        
        <td>strobilurin</td>
        
        
        <td>Strobilurus tenacellus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001265.1: 667-7238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001265/index.html#r1c1" target="_new">BGC0001265.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Polyketide</td>
        
        <td>melanin</td>
        
        
        <td>Bipolaris oryzae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001906.1: 443984-508225">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001906/index.html#r1c1" target="_new">BGC0001906.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Polyketide</td>
        
        <td>naphthalene</td>
        
        
        <td>Daldinia eschscholzii IFB-TL01</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258.1: 73-7295">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Polyketide</td>
        
        <td>1,3,6,8-tetrahydroxynaphthalene</td>
        
        
        <td>Glarea lozoyensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001794.1: 3660-23178">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001794/index.html#r1c1" target="_new">BGC0001794.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>Polyketide</td>
        
        <td>swainsonine</td>
        
        
        <td>Alternaria oxytropis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r259c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>esperamicin</td>
      <td>Actinomadura verrucosospora</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.47, order:1.00, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>oronofacic acid</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Terpene, Polyketide</td>
      <td>pyripyropene A</td>
      <td>unidentified unclassified sequences.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257.1: 399-7066">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Polyketide</td>
      <td>1,3,6,8-tetrahydroxynaphthalene</td>
      <td>Nodulisporium sp. ATCC74245</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107.1: 1136198-1142873">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Polyketide</td>
      <td>naphthopyrone</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Polyketide</td>
      <td>strobilurin</td>
      <td>Strobilurus tenacellus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001265.1: 667-7238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001265/index.html#r1c1" target="_new">BGC0001265.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Polyketide</td>
      <td>melanin</td>
      <td>Bipolaris oryzae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001906.1: 443984-508225">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001906/index.html#r1c1" target="_new">BGC0001906.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Polyketide</td>
      <td>naphthalene</td>
      <td>Daldinia eschscholzii IFB-TL01</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258.1: 73-7295">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Polyketide</td>
      <td>1,3,6,8-tetrahydroxynaphthalene</td>
      <td>Glarea lozoyensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001794.1: 3660-23178">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001794/index.html#r1c1" target="_new">BGC0001794.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Polyketide</td>
      <td>swainsonine</td>
      <td>Alternaria oxytropis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r259c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-220"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-220">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r259c1-pfam-details-svg" data-filename="c00259_NODE_48.._r259c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r259c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>


  <footer class="footer">
    <div class="container">
      <div>
        <img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
      </div>
      <div class="cite-me">
        If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
      </div>
      <div>
        <img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
      </div>
    </div>
  </footer>

  <script src="js/jquery.js"></script>
  <script src="js/antismash.js"></script>
  <script src="regions.js"></script>
  <script>
    $(document).ready(function() {
        viewer["start"](all_regions, details_data, resultsData, recordData);
    })
  </script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink">
 <defs>
  <filter id="inset-shadow">
   <feOffset dx="-2" dy="-2"></feOffset>
   <feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
   <feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
   <feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
   <feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
   <feComponentTransfer in="shadow" result="shadow">
    <feFuncA type="linear" slope=".95"></feFuncA>
   </feComponentTransfer>
   <feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
  </filter>
 </defs>
</svg>
</body>
</html>