
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>6666666.871045 - 36 region(s) - antiSMASH results</title>
  <style>img {width: 20px; height: 20px;}</style>
  <link rel="stylesheet" type="text/css" href="css/bacteria.css">
  <meta property="og:title" content="6666666.871045 - 10 record(s) - 36 region(s)" />
  <meta property="og:description" content="">
  <meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
  <meta property="og:image:alt" content="antiSMASH logo" />
  <meta property="og:image:width" content="600" />
  <meta property="og:image:height" content="600" />
  <meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-ed02d080-9b5c-4f12-84e0-2e7286c23289/index.html" />
  <meta property="og:type" content="website" />
  <meta name="twitter:card" content="summary" />
  <meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
  <div id="header">
      <nav>
    <div class="main-link">
     <div>
        <a href="https://antismash.secondarymetabolites.org/">
          <img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
        </a>
       </div>
       <div>
        <a href="https://antismash.secondarymetabolites.org/">
          antiSMASH version 6.0.1
        </a>
      </div>
    </div>
    <div class="custom-description"></div>
    <div class="ancillary-links">
      <div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
        <ul class="dropdown-options">
        
        <li><a href="6666666.871045.zip">Download all results</a></li>
        
          <li><a href="6666666.871045.gbk">Download GenBank summary file</a></li>
        
          <li><a href="bacteria-ed02d080-9b5c-4f12-84e0-2e7286c23289.log">Download log file</a></li>
        
        </ul>
      </div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
      <div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
    </div>
  </nav>
    <div id="buttons">
      <span id="region-type">Select genomic region:</span>
      <div id="regionbuttons">
        <div class="arrow-left" id="prev-region"></div>
        <div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
        <div style="display: flex; flex-wrap: wrap">
        
          
            
            
                
            
            <div class="regbutton NRPS,NRPS-like,T1PKS hybrid r1c1">
              <a href="#r1c1">1.1</a>
            </div>
          
            
            
                
            
            <div class="regbutton NRPS,T3PKS hybrid r1c2">
              <a href="#r1c2">1.2</a>
            </div>
          
            
            
            <div class="regbutton melanin r1c3">
              <a href="#r1c3">1.3</a>
            </div>
          
            
            
                
            
            <div class="regbutton NRPS,T1PKS hybrid r1c4">
              <a href="#r1c4">1.4</a>
            </div>
          
            
            
            <div class="regbutton RiPP-like r1c5">
              <a href="#r1c5">1.5</a>
            </div>
          
            
            
                
            
            <div class="regbutton NRPS,T1PKS hybrid r1c6">
              <a href="#r1c6">1.6</a>
            </div>
          
            
            
            <div class="regbutton terpene r1c7">
              <a href="#r1c7">1.7</a>
            </div>
          
        
          
            
            
            <div class="regbutton ectoine r2c1">
              <a href="#r2c1">2.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r3c1">
              <a href="#r3c1">3.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,NRPS-like,other hybrid r4c1">
              <a href="#r4c1">4.1</a>
            </div>
          
            
            
            <div class="regbutton T3PKS r4c2">
              <a href="#r4c2">4.2</a>
            </div>
          
            
            
            <div class="regbutton NRPS r4c3">
              <a href="#r4c3">4.3</a>
            </div>
          
            
            
            <div class="regbutton NRPS r4c4">
              <a href="#r4c4">4.4</a>
            </div>
          
            
            
            <div class="regbutton terpene r4c5">
              <a href="#r4c5">4.5</a>
            </div>
          
            
            
            <div class="regbutton butyrolactone r4c6">
              <a href="#r4c6">4.6</a>
            </div>
          
            
            
            <div class="regbutton NRPS-like r4c7">
              <a href="#r4c7">4.7</a>
            </div>
          
        
          
            
            
            <div class="regbutton ectoine r5c1">
              <a href="#r5c1">5.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r6c1">
              <a href="#r6c1">6.1</a>
            </div>
          
            
            
            <div class="regbutton lassopeptide r6c2">
              <a href="#r6c2">6.2</a>
            </div>
          
            
            
            <div class="regbutton lanthipeptide-class-iv r6c3">
              <a href="#r6c3">6.3</a>
            </div>
          
            
            
            <div class="regbutton NRPS-like r6c4">
              <a href="#r6c4">6.4</a>
            </div>
          
            
            
            <div class="regbutton lanthipeptide-class-iii r6c5">
              <a href="#r6c5">6.5</a>
            </div>
          
            
            
                
            
            <div class="regbutton NRPS,NRPS-like hybrid r6c6">
              <a href="#r6c6">6.6</a>
            </div>
          
        
          
            
            
            <div class="regbutton lanthipeptide-class-i r8c1">
              <a href="#r8c1">8.1</a>
            </div>
          
            
            
                
            
            <div class="regbutton butyrolactone,ectoine hybrid r8c2">
              <a href="#r8c2">8.2</a>
            </div>
          
            
            
            <div class="regbutton lanthipeptide-class-ii r8c3">
              <a href="#r8c3">8.3</a>
            </div>
          
            
            
            <div class="regbutton NRPS r8c4">
              <a href="#r8c4">8.4</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r10c1">
              <a href="#r10c1">10.1</a>
            </div>
          
            
            
            <div class="regbutton siderophore r10c2">
              <a href="#r10c2">10.2</a>
            </div>
          
            
            
                
            
            <div class="regbutton NRPS,ladderane,thioamide-NRP hybrid r10c3">
              <a href="#r10c3">10.3</a>
            </div>
          
            
            
                
            
            <div class="regbutton NRPS,T1PKS hybrid r10c4">
              <a href="#r10c4">10.4</a>
            </div>
          
            
            
            <div class="regbutton RiPP-like r10c5">
              <a href="#r10c5">10.5</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton lanthipeptide-class-ii,lanthipeptide-class-iii hybrid r12c1">
              <a href="#r12c1">12.1</a>
            </div>
          
            
            
            <div class="regbutton siderophore r12c2">
              <a href="#r12c2">12.2</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r14c1">
              <a href="#r14c1">14.1</a>
            </div>
          
            
            
                
            
            <div class="regbutton T2PKS,oligosaccharide hybrid r14c2">
              <a href="#r14c2">14.2</a>
            </div>
          
        
        </div>
        <div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
      </div>
    </div>
  </div>

  <!-- overview page -->
  
  <div class="page" id="overview">
   
   
     <h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>
   
   <div class="overview-layout">
    <div id="record-tables">
     <div id="single-record-tables">
     <div class="record-overview-header">
        <strong>c00001_CC216C_..</strong> (original name was: CC216C_hibrido_all_contig_5) (Streptomyces sp. CC216C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-1" data-filename="c00001_CC216C_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-1" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r1c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like,T1PKS hybrid r1c1">
      <a href="#r1c1">Region&nbsp1.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">48,016</td>
     <td class="digits table-split-left">98,982</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000703/1" target="_blank">kanamycin</a></td>
      <td>Saccharide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 2%, #ffffff00 2%)">2%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r1c2">
     
     
         
     
     <td class="regbutton NRPS,T3PKS hybrid r1c2">
      <a href="#r1c2">Region&nbsp1.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">165,861</td>
     <td class="digits table-split-left">270,856</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001975/1" target="_blank">atratumycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 10%, #ffffff00 10%)">10%</td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r1c3">
     
     
     <td class="regbutton melanin r1c3">
      <a href="#r1c3">Region&nbsp1.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#melanin" target="_blank">melanin</a>
     </td>
     <td class="digits">304,424</td>
     <td class="digits table-split-left">314,906</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000911/1" target="_blank">melanin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r1c4">
     
     
         
     
     <td class="regbutton NRPS,T1PKS hybrid r1c4">
      <a href="#r1c4">Region&nbsp1.4</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">334,886</td>
     <td class="digits table-split-left">383,827</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001846/1" target="_blank">valinomycin / montanastatin</a></td>
      <td>NRP + Saccharide:Hybrid/tailoring</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 30%, #ffffff00 30%)">30%</td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r1c5">
     
     
     <td class="regbutton RiPP-like r1c5">
      <a href="#r1c5">Region&nbsp1.5</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">501,872</td>
     <td class="digits table-split-left">512,669</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000163/1" target="_blank">tetronasin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 3%, #ffffff00 3%)">3%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r1c6">
     
     
         
     
     <td class="regbutton NRPS,T1PKS hybrid r1c6">
      <a href="#r1c6">Region&nbsp1.6</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">528,844</td>
     <td class="digits table-split-left">578,266</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000999/1" target="_blank">heat-stable antifungal factor</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r1c7">
     
     
     <td class="regbutton terpene r1c7">
      <a href="#r1c7">Region&nbsp1.7</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">635,237</td>
     <td class="digits table-split-left">661,810</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000663/1" target="_blank">hopene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 69%, #ffffff00 69%)">69%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00002_CC216C_..</strong> (original name was: CC216C_hibrido_all_contig_6) (Streptomyces sp. CC216C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-2" data-filename="c00002_CC216C_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-2" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r2c1">
     
     
     <td class="regbutton ectoine r2c1">
      <a href="#r2c1">Region&nbsp2.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">239,083</td>
     <td class="digits table-split-left">249,481</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000853/1" target="_blank">ectoine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00003_CC216C_..</strong> (original name was: CC216C_hibrido_all_contig_13) (Streptomyces sp. CC216C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-3" data-filename="c00003_CC216C_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-3" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r3c1">
     
     
     <td class="regbutton NRPS r3c1">
      <a href="#r3c1">Region&nbsp3.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">145,103</td>
     <td class="digits table-split-left">199,209</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001740/1" target="_blank">phthoxazolin</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00004_CC216C_..</strong> (original name was: CC216C_hibrido_all_contig_4) (Streptomyces sp. CC216C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-4" data-filename="c00004_CC216C_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-4" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r4c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like,other hybrid r4c1">
      <a href="#r4c1">Region&nbsp4.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">71,542</td>
     <td class="digits table-split-left">138,708</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000296/1" target="_blank">actinomycin D</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 89%, #ffffff00 89%)">89%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r4c2">
     
     
     <td class="regbutton T3PKS r4c2">
      <a href="#r4c2">Region&nbsp4.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">438,821</td>
     <td class="digits table-split-left">479,939</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001065/1" target="_blank">herboxidiene</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r4c3">
     
     
     <td class="regbutton NRPS r4c3">
      <a href="#r4c3">Region&nbsp4.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">525,737</td>
     <td class="digits table-split-left">576,627</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000325/1" target="_blank">coelichelin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 81%, #ffffff00 81%)">81%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r4c4">
     
     
     <td class="regbutton NRPS r4c4">
      <a href="#r4c4">Region&nbsp4.4</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">580,655</td>
     <td class="digits table-split-left">628,091</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000368/1" target="_blank">streptobactin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 94%, #ffffff00 94%)">94%</td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r4c5">
     
     
     <td class="regbutton terpene r4c5">
      <a href="#r4c5">Region&nbsp4.5</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">658,887</td>
     <td class="digits table-split-left">681,100</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001181/1" target="_blank">geosmin</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r4c6">
     
     
     <td class="regbutton butyrolactone r4c6">
      <a href="#r4c6">Region&nbsp4.6</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
     </td>
     <td class="digits">713,448</td>
     <td class="digits table-split-left">724,392</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000038/1" target="_blank">coelimycin P1</a></td>
      <td>Polyketide:Modular type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 16%, #ffffff00 16%)">16%</td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r4c7">
     
     
     <td class="regbutton NRPS-like r4c7">
      <a href="#r4c7">Region&nbsp4.7</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">739,690</td>
     <td class="digits table-split-left">765,295</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000609/1" target="_blank">nocathiacin</a></td>
      <td>RiPP:Thiopeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00005_CC216C_..</strong> (original name was: CC216C_hibrido_all_contig_12) (Streptomyces sp. CC216C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-5" data-filename="c00005_CC216C_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-5" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r5c1">
     
     
     <td class="regbutton ectoine r5c1">
      <a href="#r5c1">Region&nbsp5.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">204,458</td>
     <td class="digits table-split-left">214,832</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001073/1" target="_blank">kosinostatin</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00006_CC216C_..</strong> (original name was: CC216C_hibrido_all_contig_1) (Streptomyces sp. CC216C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-6" data-filename="c00006_CC216C_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-6" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r6c1">
     
     
     <td class="regbutton terpene r6c1">
      <a href="#r6c1">Region&nbsp6.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">24,916</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001456/1" target="_blank">isorenieratene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 87%, #ffffff00 87%)">87%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r6c2">
     
     
     <td class="regbutton lassopeptide r6c2">
      <a href="#r6c2">Region&nbsp6.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lassopeptide" target="_blank">lassopeptide</a>
     </td>
     <td class="digits">94,351</td>
     <td class="digits table-split-left">117,086</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001634/1" target="_blank">keywimysin</a></td>
      <td>RiPP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r6c3">
     
     
     <td class="regbutton lanthipeptide-class-iv r6c3">
      <a href="#r6c3">Region&nbsp6.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iv" target="_blank">lanthipeptide-class-iv</a>
     </td>
     <td class="digits">221,438</td>
     <td class="digits table-split-left">244,296</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000842/1" target="_blank">carbapenem MM4550</a></td>
      <td>Other:Non-NRP beta-lactam</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r6c4">
     
     
     <td class="regbutton NRPS-like r6c4">
      <a href="#r6c4">Region&nbsp6.4</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">862,006</td>
     <td class="digits table-split-left">902,572</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001101/1" target="_blank">leinamycin</a></td>
      <td>NRP + Polyketide:Modular type I + Polyketide:Trans-AT type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r6c5">
     
     
     <td class="regbutton lanthipeptide-class-iii r6c5">
      <a href="#r6c5">Region&nbsp6.5</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
     </td>
     <td class="digits">994,439</td>
     <td class="digits table-split-left">1,017,297</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000496/1" target="_blank">AmfS</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r6c6">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like hybrid r6c6">
      <a href="#r6c6">Region&nbsp6.6</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1,332,794</td>
     <td class="digits table-split-left">1,385,942</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000354/1" target="_blank">friulimicin A / friulimicin B / friulimicin C / friulimicin D</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00008_CC216C_..</strong> (original name was: CC216C_hibrido_all_contig_3) (Streptomyces sp. CC216C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-8" data-filename="c00008_CC216C_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-8" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r8c1">
     
     
     <td class="regbutton lanthipeptide-class-i r8c1">
      <a href="#r8c1">Region&nbsp8.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-i" target="_blank">lanthipeptide-class-i</a>
     </td>
     <td class="digits">30,746</td>
     <td class="digits table-split-left">55,985</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000291/1" target="_blank">A54145</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 3%, #ffffff00 3%)">3%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r8c2">
     
     
         
     
     <td class="regbutton butyrolactone,ectoine hybrid r8c2">
      <a href="#r8c2">Region&nbsp8.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
     </td>
     <td class="digits">216,955</td>
     <td class="digits table-split-left">232,313</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001778/1" target="_blank">showdomycin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 47%, #ffffff00 47%)">47%</td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r8c3">
     
     
     <td class="regbutton lanthipeptide-class-ii r8c3">
      <a href="#r8c3">Region&nbsp8.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-ii" target="_blank">lanthipeptide-class-ii</a>
     </td>
     <td class="digits">313,541</td>
     <td class="digits table-split-left">339,863</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000035/1" target="_blank">chalcomycin A</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r8c4">
     
     
     <td class="regbutton NRPS r8c4">
      <a href="#r8c4">Region&nbsp8.4</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">790,920</td>
     <td class="digits table-split-left">876,022</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001984/1" target="_blank">sarpeptin A / sarpeptin B</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 83%, #ffffff00 83%)">83%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00010_CC216C_..</strong> (original name was: CC216C_hibrido_all_contig_2) (Streptomyces sp. CC216C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-10" data-filename="c00010_CC216C_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-10" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r10c1">
     
     
     <td class="regbutton terpene r10c1">
      <a href="#r10c1">Region&nbsp10.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">53,749</td>
     <td class="digits table-split-left">74,756</td>
     
      <td colspan="3"></td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r10c2">
     
     
     <td class="regbutton siderophore r10c2">
      <a href="#r10c2">Region&nbsp10.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">474,733</td>
     <td class="digits table-split-left">489,455</td>
     
      <td colspan="3"></td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r10c3">
     
     
         
     
     <td class="regbutton NRPS,ladderane,thioamide-NRP hybrid r10c3">
      <a href="#r10c3">Region&nbsp10.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ladderane" target="_blank">ladderane</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thioamide-nrp" target="_blank">thioamide-NRP</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">564,168</td>
     <td class="digits table-split-left">644,707</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001975/1" target="_blank">atratumycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 26%, #ffffff00 26%)">26%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r10c4">
     
     
         
     
     <td class="regbutton NRPS,T1PKS hybrid r10c4">
      <a href="#r10c4">Region&nbsp10.4</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">752,989</td>
     <td class="digits table-split-left">846,684</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000973/1" target="_blank">collismycin A</a></td>
      <td>NRP + Polyketide:Modular type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 74%, #ffffff00 74%)">74%</td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r10c5">
     
     
     <td class="regbutton RiPP-like r10c5">
      <a href="#r10c5">Region&nbsp10.5</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">936,005</td>
     <td class="digits table-split-left">947,333</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00012_CC216C_..</strong> (original name was: CC216C_hibrido_all_contig_11) (Streptomyces sp. CC216C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-12" data-filename="c00012_CC216C_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-12" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r12c1">
     
     
         
     
     <td class="regbutton lanthipeptide-class-ii,lanthipeptide-class-iii hybrid r12c1">
      <a href="#r12c1">Region&nbsp12.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-ii" target="_blank">lanthipeptide-class-ii</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
     </td>
     <td class="digits">14,730</td>
     <td class="digits table-split-left">44,793</td>
     
      <td colspan="3"></td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r12c2">
     
     
     <td class="regbutton siderophore r12c2">
      <a href="#r12c2">Region&nbsp12.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">121,349</td>
     <td class="digits table-split-left">133,127</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000941/1" target="_blank">desferrioxamin B</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00014_CC216C_..</strong> (original name was: CC216C_hibrido_all_contig_9) (Streptomyces sp. CC216C)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-14" data-filename="c00014_CC216C_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-14" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r14c1">
     
     
     <td class="regbutton terpene r14c1">
      <a href="#r14c1">Region&nbsp14.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">125,915</td>
     <td class="digits table-split-left">146,991</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000273/1" target="_blank">steffimycin D</a></td>
      <td>Polyketide:Type II + Saccharide:Hybrid/tailoring</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 19%, #ffffff00 19%)">19%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r14c2">
     
     
         
     
     <td class="regbutton T2PKS,oligosaccharide hybrid r14c2">
      <a href="#r14c2">Region&nbsp14.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#oligosaccharide" target="_blank">oligosaccharide</a>
     </td>
     <td class="digits">267,921</td>
     <td class="digits table-split-left">340,394</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000210/1" target="_blank">chromomycin A3</a></td>
      <td>Polyketide:Type II + Saccharide:Oligosaccharide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 97%, #ffffff00 97%)">97%</td>
     
   </tr>
 
 </tbody>
</table>
      </div>
     </div>
     
     <div id="compact-record-table" style="display: none;">
      <div class="record-table">
       <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 
 
  <tr class="linked-row odd" data-anchor="#r1c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like,T1PKS hybrid r1c1">
      <a href="#r1c1">Region&nbsp1.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">48,016</td>
     <td class="digits table-split-left">98,982</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000703/1" target="_blank">kanamycin</a></td>
      <td>Saccharide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 2%, #ffffff00 2%)">2%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r1c2">
     
     
         
     
     <td class="regbutton NRPS,T3PKS hybrid r1c2">
      <a href="#r1c2">Region&nbsp1.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">165,861</td>
     <td class="digits table-split-left">270,856</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001975/1" target="_blank">atratumycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 10%, #ffffff00 10%)">10%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r1c3">
     
     
     <td class="regbutton melanin r1c3">
      <a href="#r1c3">Region&nbsp1.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#melanin" target="_blank">melanin</a>
     </td>
     <td class="digits">304,424</td>
     <td class="digits table-split-left">314,906</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000911/1" target="_blank">melanin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r1c4">
     
     
         
     
     <td class="regbutton NRPS,T1PKS hybrid r1c4">
      <a href="#r1c4">Region&nbsp1.4</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">334,886</td>
     <td class="digits table-split-left">383,827</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001846/1" target="_blank">valinomycin / montanastatin</a></td>
      <td>NRP + Saccharide:Hybrid/tailoring</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 30%, #ffffff00 30%)">30%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r1c5">
     
     
     <td class="regbutton RiPP-like r1c5">
      <a href="#r1c5">Region&nbsp1.5</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">501,872</td>
     <td class="digits table-split-left">512,669</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000163/1" target="_blank">tetronasin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 3%, #ffffff00 3%)">3%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r1c6">
     
     
         
     
     <td class="regbutton NRPS,T1PKS hybrid r1c6">
      <a href="#r1c6">Region&nbsp1.6</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">528,844</td>
     <td class="digits table-split-left">578,266</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000999/1" target="_blank">heat-stable antifungal factor</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 75%, #ffffff00 75%)">75%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r1c7">
     
     
     <td class="regbutton terpene r1c7">
      <a href="#r1c7">Region&nbsp1.7</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">635,237</td>
     <td class="digits table-split-left">661,810</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000663/1" target="_blank">hopene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 69%, #ffffff00 69%)">69%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r2c1">
     
     
     <td class="regbutton ectoine r2c1">
      <a href="#r2c1">Region&nbsp2.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">239,083</td>
     <td class="digits table-split-left">249,481</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000853/1" target="_blank">ectoine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r3c1">
     
     
     <td class="regbutton NRPS r3c1">
      <a href="#r3c1">Region&nbsp3.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">145,103</td>
     <td class="digits table-split-left">199,209</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001740/1" target="_blank">phthoxazolin</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r4c1">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like,other hybrid r4c1">
      <a href="#r4c1">Region&nbsp4.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">71,542</td>
     <td class="digits table-split-left">138,708</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000296/1" target="_blank">actinomycin D</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 89%, #ffffff00 89%)">89%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r4c2">
     
     
     <td class="regbutton T3PKS r4c2">
      <a href="#r4c2">Region&nbsp4.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">438,821</td>
     <td class="digits table-split-left">479,939</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001065/1" target="_blank">herboxidiene</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r4c3">
     
     
     <td class="regbutton NRPS r4c3">
      <a href="#r4c3">Region&nbsp4.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">525,737</td>
     <td class="digits table-split-left">576,627</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000325/1" target="_blank">coelichelin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 81%, #ffffff00 81%)">81%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r4c4">
     
     
     <td class="regbutton NRPS r4c4">
      <a href="#r4c4">Region&nbsp4.4</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">580,655</td>
     <td class="digits table-split-left">628,091</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000368/1" target="_blank">streptobactin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 94%, #ffffff00 94%)">94%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r4c5">
     
     
     <td class="regbutton terpene r4c5">
      <a href="#r4c5">Region&nbsp4.5</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">658,887</td>
     <td class="digits table-split-left">681,100</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001181/1" target="_blank">geosmin</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r4c6">
     
     
     <td class="regbutton butyrolactone r4c6">
      <a href="#r4c6">Region&nbsp4.6</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
     </td>
     <td class="digits">713,448</td>
     <td class="digits table-split-left">724,392</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000038/1" target="_blank">coelimycin P1</a></td>
      <td>Polyketide:Modular type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 16%, #ffffff00 16%)">16%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r4c7">
     
     
     <td class="regbutton NRPS-like r4c7">
      <a href="#r4c7">Region&nbsp4.7</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">739,690</td>
     <td class="digits table-split-left">765,295</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000609/1" target="_blank">nocathiacin</a></td>
      <td>RiPP:Thiopeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r5c1">
     
     
     <td class="regbutton ectoine r5c1">
      <a href="#r5c1">Region&nbsp5.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">204,458</td>
     <td class="digits table-split-left">214,832</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001073/1" target="_blank">kosinostatin</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r6c1">
     
     
     <td class="regbutton terpene r6c1">
      <a href="#r6c1">Region&nbsp6.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">24,916</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001456/1" target="_blank">isorenieratene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 87%, #ffffff00 87%)">87%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r6c2">
     
     
     <td class="regbutton lassopeptide r6c2">
      <a href="#r6c2">Region&nbsp6.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lassopeptide" target="_blank">lassopeptide</a>
     </td>
     <td class="digits">94,351</td>
     <td class="digits table-split-left">117,086</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001634/1" target="_blank">keywimysin</a></td>
      <td>RiPP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r6c3">
     
     
     <td class="regbutton lanthipeptide-class-iv r6c3">
      <a href="#r6c3">Region&nbsp6.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iv" target="_blank">lanthipeptide-class-iv</a>
     </td>
     <td class="digits">221,438</td>
     <td class="digits table-split-left">244,296</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000842/1" target="_blank">carbapenem MM4550</a></td>
      <td>Other:Non-NRP beta-lactam</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r6c4">
     
     
     <td class="regbutton NRPS-like r6c4">
      <a href="#r6c4">Region&nbsp6.4</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">862,006</td>
     <td class="digits table-split-left">902,572</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001101/1" target="_blank">leinamycin</a></td>
      <td>NRP + Polyketide:Modular type I + Polyketide:Trans-AT type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r6c5">
     
     
     <td class="regbutton lanthipeptide-class-iii r6c5">
      <a href="#r6c5">Region&nbsp6.5</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
     </td>
     <td class="digits">994,439</td>
     <td class="digits table-split-left">1,017,297</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000496/1" target="_blank">AmfS</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r6c6">
     
     
         
     
     <td class="regbutton NRPS,NRPS-like hybrid r6c6">
      <a href="#r6c6">Region&nbsp6.6</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1,332,794</td>
     <td class="digits table-split-left">1,385,942</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000354/1" target="_blank">friulimicin A / friulimicin B / friulimicin C / friulimicin D</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r8c1">
     
     
     <td class="regbutton lanthipeptide-class-i r8c1">
      <a href="#r8c1">Region&nbsp8.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-i" target="_blank">lanthipeptide-class-i</a>
     </td>
     <td class="digits">30,746</td>
     <td class="digits table-split-left">55,985</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000291/1" target="_blank">A54145</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 3%, #ffffff00 3%)">3%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r8c2">
     
     
         
     
     <td class="regbutton butyrolactone,ectoine hybrid r8c2">
      <a href="#r8c2">Region&nbsp8.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
     </td>
     <td class="digits">216,955</td>
     <td class="digits table-split-left">232,313</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001778/1" target="_blank">showdomycin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 47%, #ffffff00 47%)">47%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r8c3">
     
     
     <td class="regbutton lanthipeptide-class-ii r8c3">
      <a href="#r8c3">Region&nbsp8.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-ii" target="_blank">lanthipeptide-class-ii</a>
     </td>
     <td class="digits">313,541</td>
     <td class="digits table-split-left">339,863</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000035/1" target="_blank">chalcomycin A</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r8c4">
     
     
     <td class="regbutton NRPS r8c4">
      <a href="#r8c4">Region&nbsp8.4</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">790,920</td>
     <td class="digits table-split-left">876,022</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001984/1" target="_blank">sarpeptin A / sarpeptin B</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 83%, #ffffff00 83%)">83%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r10c1">
     
     
     <td class="regbutton terpene r10c1">
      <a href="#r10c1">Region&nbsp10.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">53,749</td>
     <td class="digits table-split-left">74,756</td>
     
      <td colspan="3"></td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r10c2">
     
     
     <td class="regbutton siderophore r10c2">
      <a href="#r10c2">Region&nbsp10.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">474,733</td>
     <td class="digits table-split-left">489,455</td>
     
      <td colspan="3"></td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r10c3">
     
     
         
     
     <td class="regbutton NRPS,ladderane,thioamide-NRP hybrid r10c3">
      <a href="#r10c3">Region&nbsp10.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ladderane" target="_blank">ladderane</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thioamide-nrp" target="_blank">thioamide-NRP</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">564,168</td>
     <td class="digits table-split-left">644,707</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001975/1" target="_blank">atratumycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 26%, #ffffff00 26%)">26%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r10c4">
     
     
         
     
     <td class="regbutton NRPS,T1PKS hybrid r10c4">
      <a href="#r10c4">Region&nbsp10.4</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">752,989</td>
     <td class="digits table-split-left">846,684</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000973/1" target="_blank">collismycin A</a></td>
      <td>NRP + Polyketide:Modular type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 74%, #ffffff00 74%)">74%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r10c5">
     
     
     <td class="regbutton RiPP-like r10c5">
      <a href="#r10c5">Region&nbsp10.5</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">936,005</td>
     <td class="digits table-split-left">947,333</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r12c1">
     
     
         
     
     <td class="regbutton lanthipeptide-class-ii,lanthipeptide-class-iii hybrid r12c1">
      <a href="#r12c1">Region&nbsp12.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-ii" target="_blank">lanthipeptide-class-ii</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
     </td>
     <td class="digits">14,730</td>
     <td class="digits table-split-left">44,793</td>
     
      <td colspan="3"></td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r12c2">
     
     
     <td class="regbutton siderophore r12c2">
      <a href="#r12c2">Region&nbsp12.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">121,349</td>
     <td class="digits table-split-left">133,127</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000941/1" target="_blank">desferrioxamin B</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r14c1">
     
     
     <td class="regbutton terpene r14c1">
      <a href="#r14c1">Region&nbsp14.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">125,915</td>
     <td class="digits table-split-left">146,991</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000273/1" target="_blank">steffimycin D</a></td>
      <td>Polyketide:Type II + Saccharide:Hybrid/tailoring</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 19%, #ffffff00 19%)">19%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r14c2">
     
     
         
     
     <td class="regbutton T2PKS,oligosaccharide hybrid r14c2">
      <a href="#r14c2">Region&nbsp14.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#oligosaccharide" target="_blank">oligosaccharide</a>
     </td>
     <td class="digits">267,921</td>
     <td class="digits table-split-left">340,394</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000210/1" target="_blank">chromomycin A3</a></td>
      <td>Polyketide:Type II + Saccharide:Oligosaccharide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 97%, #ffffff00 97%)">97%</td>
     
   </tr>
  
 
 </tbody>
</table>
      </div>
     </div>
     
     
     <dl id="regionless-records">
      <dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>
       
         <dd><strong>c00007_CC216C_..</strong> (original name was: CC216C_hibrido_all_contig_7) (Streptomyces sp. CC216C)</dd>
       
         <dd><strong>c00009_CC216C_..</strong> (original name was: CC216C_hibrido_all_contig_10) (Streptomyces sp. CC216C)</dd>
       
         <dd><strong>c00011_CC216C_..</strong> (original name was: CC216C_hibrido_all_contig_8) (Streptomyces sp. CC216C)</dd>
       
         <dd><strong>c00013_CC216C_..</strong> (original name was: CC216C_hibrido_all_contig_15) (Streptomyces sp. CC216C)</dd>
       
         <dd><strong>c00015_CC216C_..</strong> (original name was: CC216C_hibrido_all_contig_14) (Streptomyces sp. CC216C)</dd>
       
     </dl>
     
    </div>
    
    <div class="overview-switches">
     <div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    
   </div>
  </div>
  






<div class="page" id='r1c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00001_CC216C_.. - Region 1 - NRPS,NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-239"></div> <span class="help-tooltip" id="region_svg-help-239">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00001_CC216C_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1c1-svg" data-filename="c00001_CC216C_.._r1c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 48,016 - 98,982 nt. (total: 50,967 nt)
        
        <a class="cluster-rules-header" id="r1c1-rules-header" href="#r1c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r1c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r1c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-240"></div> <span class="help-tooltip" id="focus-panel-help-240">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1c1-nrps_pks" data-name="r1c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r1c1-clusterblast" data-name="r1c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1c1-knownclusterblast" data-name="r1c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1c1-subclusterblast" data-name="r1c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1c1-MIBiG-cluster-compare" data-name="r1c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1c1-pfam-details" data-name="r1c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-1"></div> <span class="help-tooltip" id="nrps-domain-help-1">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1c1-details-svg" data-filename="c00001_CC216C_.._r1c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r1c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r1c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-4"></div> <span class="help-tooltip" id="cb-general-help-4">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1c1_1.svg">
        NZ_KZ819178 (7742131-7794284): Streptomyces sp. CS131 scaffold00001, whole ge... (91% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c1_2.svg">
        NZ_JNLT01000009 (22315-71597): Streptomyces sp. CNS654 CD02DRAFT scaffold0000... (92% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c1_3.svg">
        NZ_KZ268499 (7487366-7539642): Streptomyces sp. SS07 Scaffold1, whole genome ... (89% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c1_4.svg">
        NZ_KZ819182 (7747573-7799726): Streptomyces sp. CS147 scaffold00001, whole ge... (89% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c1_5.svg">
        NZ_WWGD01000026 (52634-103524): Streptomyces sp. SID8378 SID8378.c26, whole g... (89% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c1_6.svg">
        NZ_LIPC01000003 (862099-914384): Streptomyces sp. TSRI0445 scaffold3, whole g... (86% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c1_7.svg">
        NZ_CP029361 (67104-116237): Streptomyces globisporus strain TFH56 chromosome,... (86% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c1_8.svg">
        NZ_LACR01000023 (47044-99308): Streptomyces sp. Ncost-T6T-1 YW5DRAFT scaffold... (88% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c1_9.svg">
        NZ_KZ679082 (7654742-7706946): Streptomyces sp. CS149 scaffold00001, whole ge... (88% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c1_10.svg">
        NZ_KZ819155 (313264-365477): Streptomyces sp. CS014 scaffold00002, whole geno... (86% of genes show similarity), NRPS,T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r1c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-5"></div> <span class="help-tooltip" id="cb-known-help-5">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r1c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r1c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r1c1_1.svg">
        kanamycin
      </option>
      
      <option value = "svg/knownclusterblast_r1c1_2.svg">
        daptomycin
      </option>
      
      <option value = "svg/knownclusterblast_r1c1_3.svg">
        paulomycin
      </option>
      
      <option value = "svg/knownclusterblast_r1c1_4.svg">
        paulomycin
      </option>
      
      <option value = "svg/knownclusterblast_r1c1_5.svg">
        foxicins A-D
      </option>
      
      <option value = "svg/knownclusterblast_r1c1_6.svg">
        herbimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r1c1_7.svg">
        collismycin A
      </option>
      
      <option value = "svg/knownclusterblast_r1c1_8.svg">
        diisonitrile antibiotic SF2768
      </option>
      
    </select>
    <button id="knownclusterblast-r1c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r1c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-6"></div> <span class="help-tooltip" id="cb-sub-help-6">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-7"></div> <span class="help-tooltip" id="MIBiG-help-7">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1c1"></div>
    <div id="comparison-MIBiG-r1c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>1.04</td>
        <td>Polyketide</td>
        
        <td>strobilurin</td>
        
        
        <td>Strobilurus tenacellus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001163.1: 308352-317651">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001163/index.html#r1c1" target="_new">BGC0001163.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.43, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.43, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.43, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.93</td>
        <td>Polyketide</td>
        
        <td>1-heptadecene</td>
        
        
        <td>Moorea producens 3L</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001162.1: 0-9348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001162/index.html#r1c1" target="_new">BGC0001162.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.43, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.43, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.43, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.92</td>
        <td>Polyketide</td>
        
        <td>1-heptadecene</td>
        
        
        <td>Moorea producens JHB</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.92</td>
        <td>NRP</td>
        
        <td>vicibactin</td>
        
        
        <td>Rhizobium etli CFN 42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001161.1: 0-9161">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001161/index.html#r1c1" target="_new">BGC0001161.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.42, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.42, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.42, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.92</td>
        <td>Polyketide</td>
        
        <td>1-heptadecene</td>
        
        
        <td>Moorea bouillonii PNG</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.42, order:1.00, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.42, order:1.00, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.42, order:1.00, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.90</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001281.1: 8011-344234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001281/index.html#r1c1" target="_new">BGC0001281.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.77</td>
        <td>Polyketide</td>
        
        <td>ustilagic acid</td>
        
        
        <td>Ustilago maydis 521</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.76</td>
        <td>NRP</td>
        
        <td>rhodochelin</td>
        
        
        <td>Rhodococcus jostii RHA1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.76</td>
        <td>NRP</td>
        
        <td>bacillibactin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313.1: 0-33475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.76</td>
        <td>NRP</td>
        
        <td>beauvericin</td>
        
        
        <td>Beauveria bassiana</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Polyketide</td>
      <td>strobilurin</td>
      <td>Strobilurus tenacellus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001163.1: 308352-317651">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001163/index.html#r1c1" target="_new">BGC0001163.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.43, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>1-heptadecene</td>
      <td>Moorea producens 3L</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001162.1: 0-9348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001162/index.html#r1c1" target="_new">BGC0001162.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.43, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>1-heptadecene</td>
      <td>Moorea producens JHB</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>vicibactin</td>
      <td>Rhizobium etli CFN 42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001161.1: 0-9161">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001161/index.html#r1c1" target="_new">BGC0001161.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.42, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>1-heptadecene</td>
      <td>Moorea bouillonii PNG</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.42, order:1.00, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001281.1: 8011-344234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001281/index.html#r1c1" target="_new">BGC0001281.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>ustilagic acid</td>
      <td>Ustilago maydis 521</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002036.1: 7496-23479">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002036/index.html#r1c1" target="_new">BGC0002036.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other</td>
      <td>dehydrofosmidomycin</td>
      <td>Streptomyces lavendulae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>pseudomonine</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001585.1: 100-44133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001585/index.html#r1c1" target="_new">BGC0001585.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Alkaloid</td>
      <td>melinacidin IV</td>
      <td>Escovopsis weberi</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-8"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-8">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1c1-pfam-details-svg" data-filename="c00001_CC216C_.._r1c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r1c1-nrps_pks_products" data-name="r1c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r1c1-nrps_pks" data-name="r1c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r1c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-2"></div> <span class="help-tooltip" id="nrps-structures-help-2">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 48015 - 98982:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r1c1-1" class="smiles-canvas" data-smiles="CC(O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ohmal - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal*,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r1c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 48015 - 90199:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r1c1-2" class="smiles-canvas" data-smiles="CC(O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ohmal - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal*,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r1c1-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 50552 - 98982:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r1c1-3" class="smiles-canvas" data-smiles="CC(O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ohmal - X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal,x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal*,x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r1c1-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r1c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-3"></div> <span class="help-tooltip" id="nrps-monomers-help-3">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds67519_70213</span></strong>: pk
      <div class="collapser collapser-target-cds67519_70213 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>CAL_domain (141..455)</strong>: pk
          <div class="collapser collapser-target-cds67519_70213 collapser-level-candidate"> </div><div class="collapser-content">
          
              minowa:  NH2
              <div class="collapser collapser-target-cds67519_70213-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>NH2: 49.1</dt>

  <dd></dd><dt>fatty_acid: 35.7</dt>

  <dd></dd><dt>shikimic_acid: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>AHBA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds70248_79704</span></strong>: mal - X
      <div class="collapser collapser-target-cds70248_79704 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>PKS_AT (303..585)</strong>: mal
          <div class="collapser collapser-target-cds70248_79704 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds70248_79704-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>inactive: 62.5%</dd>
<dd>Methylmalonyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds70248_79704-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 101.8</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 60.8</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 60.2</dt>

  <dd></dd><dt>inactive: 40.7</dt>

  <dd></dd><dt>fatty_acid: 24.2</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 21.6</dt>

  <dd></dd><dt>Propionyl-CoA: 21.5</dt>

  <dd></dd><dt>Isobutyryl-CoA: 20.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 14.9</dt>

  <dd></dd><dt>Benzoyl-CoA: 13.5</dt>

  <dd></dd><dt>CHC-CoA: 13.2</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 12.9</dt>

  <dd></dd><dt>Acetyl-CoA: 11.3</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1708..2123)</strong>: X
          <div class="collapser collapser-target-cds70248_79704 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asp, asn, glu, gln, aad
              <div class="collapser collapser-target-cds70248_79704-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DArshGLInK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r1c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00001_CC216C_.. - Region 2 - NRPS,T3PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-241"></div> <span class="help-tooltip" id="region_svg-help-241">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00001_CC216C_...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1c2-svg" data-filename="c00001_CC216C_.._r1c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 165,861 - 270,856 nt. (total: 104,996 nt)
        
        <a class="cluster-rules-header" id="r1c2-rules-header" href="#r1c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r1c2-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)</div>
      
      <div class="region-svg-container">
       <div id='r1c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-242"></div> <span class="help-tooltip" id="focus-panel-help-242">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1c2-nrps_pks" data-name="r1c2-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r1c2-clusterblast" data-name="r1c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1c2-knownclusterblast" data-name="r1c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1c2-subclusterblast" data-name="r1c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1c2-MIBiG-cluster-compare" data-name="r1c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1c2-pfam-details" data-name="r1c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1c2-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-9"></div> <span class="help-tooltip" id="nrps-domain-help-9">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1c2-details-svg" data-filename="c00001_CC216C_.._r1c2_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r1c2-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r1c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-12"></div> <span class="help-tooltip" id="cb-general-help-12">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1c2_1.svg">
        NZ_JNLT01000009 (140434-244710): Streptomyces sp. CNS654 CD02DRAFT scaffold00... (98% of genes show similarity), NRPS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r1c2_2.svg">
        NZ_KZ819155 (125553-230429): Streptomyces sp. CS014 scaffold00002, whole geno... (92% of genes show similarity), NRPS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r1c2_3.svg">
        NZ_KZ819178 (7566871-7672115): Streptomyces sp. CS131 scaffold00001, whole ge... (89% of genes show similarity), NRPS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r1c2_4.svg">
        NZ_CP020570 (7617502-7723848): Streptomyces violaceoruber strain S21 chromoso... (86% of genes show similarity), NRPS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r1c2_5.svg">
        NZ_CP058556 (193247-300169): Streptomyces sp. CB04723 chromosome, complete ge... (84% of genes show similarity), NRPS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r1c2_6.svg">
        NZ_CP010833 (250582-354192): Streptomyces sp. Tue 6075 strain Tue6075 chromos... (85% of genes show similarity), NRPS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r1c2_7.svg">
        NZ_LIPB01000003 (10809-116596): Streptomyces sp. CB00072 scaffold3, whole gen... (87% of genes show similarity), NRPS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r1c2_8.svg">
        NZ_LIVW01000003 (10777-116911): Streptomyces sp. CB02115 scaffold3, whole gen... (87% of genes show similarity), NRPS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r1c2_9.svg">
        NZ_WWGL01000009 (194578-299956): Streptomyces sp. SID8369 SID8369.c9, whole g... (85% of genes show similarity), NRPS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r1c2_10.svg">
        NZ_LMEL01000021 (237363-342533): Streptomyces sp. Root1295 contig 6, whole ge... (87% of genes show similarity), NRPS,T3PKS
      </option>
      
    </select>
    <button id= "clusterblast-r1c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-13"></div> <span class="help-tooltip" id="cb-known-help-13">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r1c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r1c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r1c2_1.svg">
        atratumycin
      </option>
      
      <option value = "svg/knownclusterblast_r1c2_2.svg">
        daptomycin
      </option>
      
      <option value = "svg/knownclusterblast_r1c2_3.svg">
        cysteoamide
      </option>
      
      <option value = "svg/knownclusterblast_r1c2_4.svg">
        gobichelin A / gobichelin B
      </option>
      
      <option value = "svg/knownclusterblast_r1c2_5.svg">
        tetronasin
      </option>
      
      <option value = "svg/knownclusterblast_r1c2_6.svg">
        alkylresorcinol
      </option>
      
      <option value = "svg/knownclusterblast_r1c2_7.svg">
        CDA1b / CDA2a / CDA2b / CDA3a / CDA3b / CDA4a / CDA4b
      </option>
      
      <option value = "svg/knownclusterblast_r1c2_8.svg">
        RP-1776
      </option>
      
      <option value = "svg/knownclusterblast_r1c2_9.svg">
        glycinocin A
      </option>
      
      <option value = "svg/knownclusterblast_r1c2_10.svg">
        actinomycin D
      </option>
      
    </select>
    <button id="knownclusterblast-r1c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r1c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-14"></div> <span class="help-tooltip" id="cb-sub-help-14">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-15"></div> <span class="help-tooltip" id="MIBiG-help-15">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1c2"></div>
    <div id="comparison-MIBiG-r1c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">T3PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282.1: 548717-551330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.89 (id:0.69, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.89), rgb(0, 0, 0, 0.89) 89%, white 89%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.89 (id:0.69, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.89), rgb(0, 0, 0, 0.89) 89%, white 89%)"></td>
         
        
        <td>1.39</td>
        <td>Polyketide</td>
        
        <td>alkylresorcinol</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.76 (id:0.45, order:1.00, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.60</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.75 (id:0.45, order:1.00, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.45, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.59</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.44, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.55</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.51</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831.1: 8162481-8168632">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.51, order:0.43, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.51, order:0.43, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.51</td>
        <td>Polyketide</td>
        
        <td>alkylpyrone-407, alkylpyrone-393</td>
        
        
        <td>Myxococcus xanthus DK 1622</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.51</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.50</td>
        <td>NRP</td>
        
        <td>vicibactin</td>
        
        
        <td>Rhizobium etli CFN 42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131.1: 144016-173487">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.49</td>
        <td>NRP</td>
        
        <td>ambactin</td>
        
        
        <td>Xenorhabdus miraniensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.49</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282.1: 548717-551330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.89 (id:0.69, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.89), rgb(0, 0, 0, 0.89) 89%, white 89%)"></td>
      <td>0.89</td>
      <td>Polyketide</td>
      <td>alkylresorcinol</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.76 (id:0.45, order:1.00, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
      <td>0.76</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.75 (id:0.45, order:1.00, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
      <td>0.75</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.44, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001629.1: 209-63961">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001629/index.html#r1c1" target="_new">BGC0001629.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:0.83, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>Jessenipeptin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000438.1: 137-73725">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000438/index.html#r1c1" target="_new">BGC0000438.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.45, order:0.83, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>syringopeptin 25A</td>
      <td>Pseudomonas syringae pv. syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>vicibactin</td>
      <td>Rhizobium etli CFN 42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131.1: 144016-173487">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>ambactin</td>
      <td>Xenorhabdus miraniensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-16"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-16">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1c2-pfam-details-svg" data-filename="c00001_CC216C_.._r1c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r1c2-nrps_pks_products" data-name="r1c2-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r1c2-nrps_pks" data-name="r1c2-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r1c2-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-10"></div> <span class="help-tooltip" id="nrps-structures-help-10">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 4, location 165860 - 270856:
          <div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r1c2-4" class="smiles-canvas" data-smiles="NCC(=O)NC(C(O)C)C(=O)NC(CO)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(C(C)C)C(=O)NC(CCC(=O)O)C(=O)NC([*])C(=O)NCC(=O)NC(CO)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(gly - thr - ser - D-X) + (thr - D-X - val - D-glu) + (D-X - gly - D-ser) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly,thr,ser,d-x&amp;nrps2=thr,d-x,val,d-glu&amp;nrps3=d-x,gly,d-ser&amp;nrps4=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*,thr*,ser*,d-x*&amp;nrps2=thr*,d-x*,val*,d-glu*&amp;nrps3=d-x*,gly*,d-ser*&amp;nrps4=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r1c2-4-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NCC(=O)NC(C(O)C)C(=O)NC(CO)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(C(C)C)C(=O)NC(CCC(=O)O)C(=O)NC([*])C(=O)NCC(=O)NC(CO)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 5, location 165860 - 251871:
          <div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r1c2-5" class="smiles-canvas" data-smiles="NCC(=O)NC(C(O)C)C(=O)NC(CO)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(C(C)C)C(=O)NC(CCC(=O)O)C(=O)NC([*])C(=O)NCC(=O)NC(CO)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(gly - thr - ser - D-X) + (thr - D-X - val - D-glu) + (D-X - gly - D-ser) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly,thr,ser,d-x&amp;nrps2=thr,d-x,val,d-glu&amp;nrps3=d-x,gly,d-ser&amp;nrps4=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*,thr*,ser*,d-x*&amp;nrps2=thr*,d-x*,val*,d-glu*&amp;nrps3=d-x*,gly*,d-ser*&amp;nrps4=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r1c2-5-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NCC(=O)NC(C(O)C)C(=O)NC(CO)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(C(C)C)C(=O)NC(CCC(=O)O)C(=O)NC([*])C(=O)NCC(=O)NC(CO)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 6, location 229803 - 270856:
          <div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r1c2-6" class="smiles-canvas" data-smiles="NCC(=O)NC(C(O)C)C(=O)NC(CO)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(gly - thr - ser - D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly,thr,ser,d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*,thr*,ser*,d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r1c2-6-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NCC(=O)NC(C(O)C)C(=O)NC(CO)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r1c2-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-11"></div> <span class="help-tooltip" id="nrps-monomers-help-11">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds185449_189256</span></strong>: X
      <div class="collapser collapser-target-cds185449_189256 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (678..1086)</strong>: X
          <div class="collapser collapser-target-cds185449_189256 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds185449_189256-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAWegGLvDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds189252_200049</span></strong>: X - gly - ser
      <div class="collapser collapser-target-cds189252_200049 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,gly,ser" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,gly*,ser*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (208..603)</strong>: X
          <div class="collapser collapser-target-cds189252_200049 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn, lys, arg
              <div class="collapser collapser-target-cds189252_200049-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVcvIGAVnK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>arg</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1726..2106)</strong>: gly
          <div class="collapser collapser-target-cds189252_200049 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-cds189252_200049-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILQmGLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2766..3166)</strong>: ser
          <div class="collapser collapser-target-cds189252_200049 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds189252_200049-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWHISLIDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds200839_216397</span></strong>: thr - X - val - glu
      <div class="collapser collapser-target-cds200839_216397 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr,X,val,glu" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*,X*,val*,glu*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (481..900)</strong>: thr
          <div class="collapser collapser-target-cds200839_216397 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds200839_216397-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNVGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1562..1960)</strong>: X
          <div class="collapser collapser-target-cds200839_216397 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  phe, trp, phg, tyr, bht
              <div class="collapser collapser-target-cds200839_216397-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DtWTIAsVdK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3058..3459)</strong>: val
          <div class="collapser collapser-target-cds200839_216397 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds200839_216397-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>val</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAhWWGGTFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (4126..4522)</strong>: glu
          <div class="collapser collapser-target-cds200839_216397 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  glu
              <div class="collapser collapser-target-cds200839_216397-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DvWHFGqVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>glu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds216393_230334</span></strong>: gly - thr - ser - X
      <div class="collapser collapser-target-cds216393_230334 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly,thr,ser,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*,thr*,ser*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (410..804)</strong>: gly
          <div class="collapser collapser-target-cds216393_230334 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-cds216393_230334-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILQLGLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1456..1868)</strong>: thr
          <div class="collapser collapser-target-cds216393_230334 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds216393_230334-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNVGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2524..2922)</strong>: ser
          <div class="collapser collapser-target-cds216393_230334 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds216393_230334-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWHLSLvDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3571..3954)</strong>: X
          <div class="collapser collapser-target-cds216393_230334 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  phe, trp, phg, tyr, bht
              <div class="collapser collapser-target-cds216393_230334-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAmLIGAacK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds230860_233800</span></strong>: gly
      <div class="collapser collapser-target-cds230860_233800 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (478..882)</strong>: gly
          <div class="collapser collapser-target-cds230860_233800 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-cds230860_233800-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILQLGLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r1c3' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00001_CC216C_.. - Region 3 - melanin <div class="help-container"> <div class="help-icon" data-id="region_svg-help-243"></div> <span class="help-tooltip" id="region_svg-help-243">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00001_CC216C_...region003.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1c3-svg" data-filename="c00001_CC216C_.._r1c3.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 304,424 - 314,906 nt. (total: 10,483 nt)
        
        <a class="cluster-rules-header" id="r1c3-rules-header" href="#r1c3">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r1c3-rules">melanin: melC</div>
      
      <div class="region-svg-container">
       <div id='r1c3-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-244"></div> <span class="help-tooltip" id="focus-panel-help-244">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1c3">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1c3-clusterblast" data-name="r1c3-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1c3-knownclusterblast" data-name="r1c3-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1c3-subclusterblast" data-name="r1c3-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1c3-MIBiG-cluster-compare" data-name="r1c3-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1c3-pfam-details" data-name="r1c3-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1c3-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-17"></div> <span class="help-tooltip" id="cb-general-help-17">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1c3-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1c3_1.svg">
        NZ_WWGD01000026 (277015-287498): Streptomyces sp. SID8378 SID8378.c26, whole ... (100% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r1c3_2.svg">
        NZ_JNLT01000009 (278138-288621): Streptomyces sp. CNS654 CD02DRAFT scaffold00... (100% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r1c3_3.svg">
        NZ_KZ819178 (7517175-7527664): Streptomyces sp. CS131 scaffold00001, whole ge... (100% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r1c3_4.svg">
        NZ_KZ819182 (7564822-7575305): Streptomyces sp. CS147 scaffold00001, whole ge... (100% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r1c3_5.svg">
        NZ_CP010833 (395552-406035): Streptomyces sp. Tue 6075 strain Tue6075 chromos... (100% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r1c3_6.svg">
        NZ_KZ303840 (274670-285153): Streptomyces sp. Ncost-T6T-1 YW5DRAFT scaffold 1... (100% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r1c3_7.svg">
        NZ_LIXK01000003 (264153-274621): Streptomyces sp. CB02130 scaffold3, whole ge... (100% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r1c3_8.svg">
        NZ_CP013738 (7287745-7298213): Streptomyces globisporus C-1027 chromosome, co... (100% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r1c3_9.svg">
        NZ_KZ819155 (74455-84938): Streptomyces sp. CS014 scaffold00002, whole genome... (100% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r1c3_10.svg">
        NZ_KZ679082 (7484449-7494932): Streptomyces sp. CS149 scaffold00001, whole ge... (100% of genes show similarity), melanin
      </option>
      
    </select>
    <button id= "clusterblast-r1c3-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c3-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-18"></div> <span class="help-tooltip" id="cb-known-help-18">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r1c3-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r1c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r1c3_1.svg">
        melanin
      </option>
      
      <option value = "svg/knownclusterblast_r1c3_2.svg">
        melanin
      </option>
      
      <option value = "svg/knownclusterblast_r1c3_3.svg">
        melanin
      </option>
      
      <option value = "svg/knownclusterblast_r1c3_4.svg">
        istamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r1c3-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r1c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c3-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-19"></div> <span class="help-tooltip" id="cb-sub-help-19">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1c3-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-20"></div> <span class="help-tooltip" id="MIBiG-help-20">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1c3-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1c3-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1c3"></div>
    <div id="comparison-MIBiG-r1c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">melanin</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000911.1: 607505-608793">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000911/index.html#r1c1" target="_new">BGC0000911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.86 (id:0.63, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
         
        
        <td>0.63</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000910.1: 2943455-2944875">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000910/index.html#r1c1" target="_new">BGC0000910.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.80 (id:0.52, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
         
        
        <td>0.53</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000912.1: 2884831-2886096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000912/index.html#r1c1" target="_new">BGC0000912.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000909.1: 0-6625">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000909/index.html#r1c1" target="_new">BGC0000909.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.51, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000908.1: 0-6497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000908/index.html#r1c1" target="_new">BGC0000908.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.50, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Alkaloid</td>
        
        <td>kitasetaline</td>
        
        
        <td>Kitasatospora setae KM-6054</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B</td>
        
        
        <td>Streptomyces sp. WK-5344</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Terpene</td>
        
        <td>grixazone</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001592.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001592/index.html#r1c1" target="_new">BGC0001592.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.47, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>ferroverdin</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002029.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002029/index.html#r1c1" target="_new">BGC0002029.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.47, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B, bagremycin C, bagremycin E, bagremycin F, bagremycin G, ferroverdin A, ferroverdin B, ferroverdin C</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1c3-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000911.1: 607505-608793">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000911/index.html#r1c1" target="_new">BGC0000911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.86 (id:0.63, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
      <td>0.86</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000910.1: 2943455-2944875">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000910/index.html#r1c1" target="_new">BGC0000910.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.80 (id:0.52, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
      <td>0.80</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000912.1: 2884831-2886096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000912/index.html#r1c1" target="_new">BGC0000912.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000909.1: 0-6625">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000909/index.html#r1c1" target="_new">BGC0000909.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.51, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000908.1: 0-6497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000908/index.html#r1c1" target="_new">BGC0000908.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.50, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Alkaloid</td>
      <td>kitasetaline</td>
      <td>Kitasatospora setae KM-6054</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B</td>
      <td>Streptomyces sp. WK-5344</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Terpene</td>
      <td>grixazone</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001592.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001592/index.html#r1c1" target="_new">BGC0001592.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.47, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Other</td>
      <td>ferroverdin</td>
      <td>Streptomyces lunaelactis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002029.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002029/index.html#r1c1" target="_new">BGC0002029.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.47, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B, bagremycin C, bagremycin E, bagremycin F, bagremycin G, ferroverdin A, ferroverdin B, ferroverdin C</td>
      <td>Streptomyces lunaelactis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1c3-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-21"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-21">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1c3-pfam-details-svg" data-filename="c00001_CC216C_.._r1c3_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1c3-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r1c4' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00001_CC216C_.. - Region 4 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-245"></div> <span class="help-tooltip" id="region_svg-help-245">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00001_CC216C_...region004.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1c4-svg" data-filename="c00001_CC216C_.._r1c4.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 334,886 - 383,827 nt. (total: 48,942 nt)
        
        <a class="cluster-rules-header" id="r1c4-rules-header" href="#r1c4">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r1c4-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r1c4-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-246"></div> <span class="help-tooltip" id="focus-panel-help-246">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1c4">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1c4-nrps_pks" data-name="r1c4-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r1c4-clusterblast" data-name="r1c4-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1c4-knownclusterblast" data-name="r1c4-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1c4-subclusterblast" data-name="r1c4-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1c4-MIBiG-cluster-compare" data-name="r1c4-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1c4-pfam-details" data-name="r1c4-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1c4-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-22"></div> <span class="help-tooltip" id="nrps-domain-help-22">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1c4-details-svg" data-filename="c00001_CC216C_.._r1c4_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r1c4-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r1c4-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-25"></div> <span class="help-tooltip" id="cb-general-help-25">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1c4-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1c4_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1c4_1.svg">
        NZ_CP021707 (7114634-7165274): Streptomyces sp. S063 chromosome, complete genome (66% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c4_2.svg">
        NZ_CP010833 (439229-488558): Streptomyces sp. Tue 6075 strain Tue6075 chromos... (60% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c4_3.svg">
        NZ_WWIP01000001 (7436777-7485679): Streptomyces sp. SID5466 SID5466.c1, whole... (59% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c4_4.svg">
        NZ_KZ819178 (7423537-7483153): Streptomyces sp. CS131 scaffold00001, whole ge... (44% of genes show similarity), NRPS,T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r1c4_5.svg">
        NZ_KZ679082 (7401852-7450519): Streptomyces sp. CS149 scaffold00001, whole ge... (48% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c4_6.svg">
        NZ_KZ819182 (7469349-7530903): Streptomyces sp. CS147 scaffold00001, whole ge... (38% of genes show similarity), NRPS,T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r1c4_7.svg">
        NZ_WWGD01000026 (321385-384253): Streptomyces sp. SID8378 SID8378.c26, whole ... (37% of genes show similarity), NRPS,T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r1c4_8.svg">
        NZ_CP015362 (290184-501485): Streptomyces sp. S8 chromosome, complete genome (15% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c4_9.svg">
        NZ_WWGS01000009 (359295-464199): Streptomyces sp. SID8359 SID8359.c9, whole g... (28% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c4_10.svg">
        NZ_AQRK01000100 (265917-369291): Streptomyces sp. AmelKG-D3 YUWDRAFT scaffold... (28% of genes show similarity), NRPS,T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r1c4-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1c4-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c4-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-26"></div> <span class="help-tooltip" id="cb-known-help-26">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r1c4-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r1c4_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r1c4_1.svg">
        valinomycin / montanastatin
      </option>
      
      <option value = "svg/knownclusterblast_r1c4_2.svg">
        lividomycin
      </option>
      
      <option value = "svg/knownclusterblast_r1c4_3.svg">
        dutomycin
      </option>
      
      <option value = "svg/knownclusterblast_r1c4_4.svg">
        mycotrienin I
      </option>
      
    </select>
    <button id="knownclusterblast-r1c4-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r1c4-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c4-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-27"></div> <span class="help-tooltip" id="cb-sub-help-27">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1c4-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-28"></div> <span class="help-tooltip" id="MIBiG-help-28">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1c4-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1c4-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1c4"></div>
    <div id="comparison-MIBiG-r1c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000013.1: 1119809-1127658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000013/index.html#r1c1" target="_new">BGC0000013.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.57</td>
        <td>Polyketide</td>
        
        <td>alternariol</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001542.1: 0-22797">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001542/index.html#r1c1" target="_new">BGC0001542.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.55</td>
        <td>Polyketide</td>
        
        <td>cercosporin</td>
        
        
        <td>Cercospora zeina</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001144.1: 57769-72970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001144/index.html#r1c1" target="_new">BGC0001144.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.53</td>
        <td>Polyketide</td>
        
        <td>neosartoricin B</td>
        
        
        <td>Trichophyton tonsurans CBS 112818</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000161.1: 399826-432585">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000161/index.html#r1c1" target="_new">BGC0000161.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.53</td>
        <td>Polyketide</td>
        
        <td>isoterrein</td>
        
        
        <td>Aspergillus terreus NIH2624</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000863.1: 1224-16560">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000863/index.html#r1c1" target="_new">BGC0000863.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.52</td>
        <td>Other</td>
        
        <td>eicosapentaenoic acid</td>
        
        
        <td>Pseudoalteromonas sp. DS-12</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001541.1: 2253439-2296209">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001541/index.html#r1c1" target="_new">BGC0001541.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.52</td>
        <td>Polyketide</td>
        
        <td>cercosporin</td>
        
        
        <td>Cercospora beticola</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250.1: 0-14073">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.51</td>
        <td>NRP</td>
        
        <td>epichloenin A</td>
        
        
        <td>Epichloe festucae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251.1: 1268-15341">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.51</td>
        <td>NRP</td>
        
        <td>epichloenin A</td>
        
        
        <td>Epichloe festucae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000048.1: 0-52827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000048/index.html#r1c1" target="_new">BGC0000048.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.49</td>
        <td>Polyketide</td>
        
        <td>dothistromin</td>
        
        
        <td>Dothistroma septosporum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.48</td>
        <td>Polyketide</td>
        
        <td>esperamicin</td>
        
        
        <td>Actinomadura verrucosospora</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1c4-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000013.1: 1119809-1127658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000013/index.html#r1c1" target="_new">BGC0000013.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>alternariol</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001542.1: 0-22797">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001542/index.html#r1c1" target="_new">BGC0001542.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>cercosporin</td>
      <td>Cercospora zeina</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000161.1: 399826-432585">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000161/index.html#r1c1" target="_new">BGC0000161.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>isoterrein</td>
      <td>Aspergillus terreus NIH2624</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001541.1: 2253439-2296209">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001541/index.html#r1c1" target="_new">BGC0001541.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>cercosporin</td>
      <td>Cercospora beticola</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000027.1: 10301-76156">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000027/index.html#r1c1" target="_new">BGC0000027.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>ankaflavin, monascin, rubropunctatine, monascorubrin</td>
      <td>Monascus pilosus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001144.1: 57769-72970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001144/index.html#r1c1" target="_new">BGC0001144.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>neosartoricin B</td>
      <td>Trichophyton tonsurans CBS 112818</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000863.1: 1224-16560">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000863/index.html#r1c1" target="_new">BGC0000863.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other</td>
      <td>eicosapentaenoic acid</td>
      <td>Pseudoalteromonas sp. DS-12</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000048.1: 0-52827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000048/index.html#r1c1" target="_new">BGC0000048.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>dothistromin</td>
      <td>Dothistroma septosporum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250.1: 0-14073">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>epichloenin A</td>
      <td>Epichloe festucae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251.1: 1268-15341">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>epichloenin A</td>
      <td>Epichloe festucae</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1c4-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-29"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-29">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1c4-pfam-details-svg" data-filename="c00001_CC216C_.._r1c4_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1c4-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r1c4-nrps_pks_products" data-name="r1c4-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r1c4-nrps_pks" data-name="r1c4-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r1c4-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-23"></div> <span class="help-tooltip" id="nrps-structures-help-23">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 8, location 334885 - 383827:
          <div class="collapser collapser-target-CC8 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r1c4-8" class="smiles-canvas" data-smiles="CC(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(mal) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal&amp;nrps2=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal*&amp;nrps2=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r1c4-8-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 9, location 334885 - 377942:
          <div class="collapser collapser-target-CC9 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r1c4-9" class="smiles-canvas" data-smiles="CC(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(mal) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal&amp;nrps2=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal*&amp;nrps2=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r1c4-9-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 10, location 342016 - 383827:
          <div class="collapser collapser-target-CC10 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r1c4-10" class="smiles-canvas" data-smiles="CC(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(mal) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal&amp;nrps2=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal*&amp;nrps2=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r1c4-10-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r1c4-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-24"></div> <span class="help-tooltip" id="nrps-monomers-help-24">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds353634_358047</span></strong>: X
      <div class="collapser collapser-target-cds353634_358047 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (433..833)</strong>: X
          <div class="collapser collapser-target-cds353634_358047 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly, ala, val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds353634_358047-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAlvLGVVlK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds360967_364285</span></strong>: mal
      <div class="collapser collapser-target-cds360967_364285 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (548..841)</strong>: mal
          <div class="collapser collapser-target-cds360967_364285 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds360967_364285-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>inactive: 58.3%</dd>
<dd>Methylmalonyl-CoA: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds360967_364285-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 134.8</dt>

  <dd></dd><dt>inactive: 71.5</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 67.9</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 55.6</dt>

  <dd></dd><dt>Isobutyryl-CoA: 42.8</dt>

  <dd></dd><dt>Propionyl-CoA: 39.3</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 34.5</dt>

  <dd></dd><dt>trans-1,2-CPDA: 20.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 18.9</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 17.9</dt>

  <dd></dd><dt>fatty_acid: 15.7</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 14.8</dt>

  <dd></dd><dt>Acetyl-CoA: 14.4</dt>

  <dd></dd><dt>CHC-CoA: 8.6</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r1c5' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00001_CC216C_.. - Region 5 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-247"></div> <span class="help-tooltip" id="region_svg-help-247">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00001_CC216C_...region005.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1c5-svg" data-filename="c00001_CC216C_.._r1c5.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 501,872 - 512,669 nt. (total: 10,798 nt)
        
        <a class="cluster-rules-header" id="r1c5-rules-header" href="#r1c5">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r1c5-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193)</div>
      
      <div class="region-svg-container">
       <div id='r1c5-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-248"></div> <span class="help-tooltip" id="focus-panel-help-248">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1c5">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1c5-clusterblast" data-name="r1c5-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1c5-knownclusterblast" data-name="r1c5-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1c5-subclusterblast" data-name="r1c5-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r1c5-MIBiG-cluster-compare" data-name="r1c5-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1c5-pfam-details" data-name="r1c5-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1c5-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-30"></div> <span class="help-tooltip" id="cb-general-help-30">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1c5-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1c5_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1c5_1.svg">
        NZ_LIPC01000003 (382329-472802): Streptomyces sp. TSRI0445 scaffold3, whole g... (14% of genes show similarity), NRPS,T1PKS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r1c5_2.svg">
        NZ_KZ819178 (7264795-7275593): Streptomyces sp. CS131 scaffold00001, whole ge... (100% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r1c5_3.svg">
        NZ_KZ303840 (70525-81323): Streptomyces sp. Ncost-T6T-1 YW5DRAFT scaffold 1.2... (90% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r1c5_4.svg">
        NZ_KZ819182 (7257301-7311610): Streptomyces sp. CS147 scaffold00001, whole ge... (20% of genes show similarity), NRPS,T1PKS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r1c5_5.svg">
        NZ_JNLT01000002 (131197-339742): Streptomyces sp. CNS654 CD02DRAFT scaffold00... (8% of genes show similarity), NRPS,PKS-like,T1PKS,bacteriocin,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r1c5_6.svg">
        NZ_WWIP01000001 (7309634-7320432): Streptomyces sp. SID5466 SID5466.c1, whole... (90% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r1c5_7.svg">
        NZ_CP010833 (608592-661794): Streptomyces sp. Tue 6075 strain Tue6075 chromos... (20% of genes show similarity), NRPS,T1PKS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r1c5_8.svg">
        NZ_KZ819154 (140377-150440): Streptomyces sp. CS014 scaffold00001, whole geno... (90% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r1c5_9.svg">
        NZ_LZRD01000001 (214014-224812): Streptomyces sp. PTY087I2 A3Q37 contig000001... (90% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r1c5_10.svg">
        NZ_CP029361 (510761-567737): Streptomyces globisporus strain TFH56 chromosome... (19% of genes show similarity), NRPS,T1PKS,bacteriocin
      </option>
      
    </select>
    <button id= "clusterblast-r1c5-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1c5-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c5-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-31"></div> <span class="help-tooltip" id="cb-known-help-31">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r1c5-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r1c5_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r1c5_1.svg">
        tetronasin
      </option>
      
    </select>
    <button id="knownclusterblast-r1c5-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r1c5-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c5-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-32"></div> <span class="help-tooltip" id="cb-sub-help-32">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r1c5-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-33"></div> <span class="help-tooltip" id="MIBiG-help-33">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1c5-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1c5-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1c5"></div>
    <div id="comparison-MIBiG-r1c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">RiPP-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001882.1: 2571644-2615001">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001882/index.html#r1c1" target="_new">BGC0001882.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>chrysoxanthone A, chrysoxanthone B, chrysoxanthone C</td>
        
        
        <td>Penicillium rubens Wisconsin 54-1255</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000791.1: 0-12875">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000791/index.html#r1c1" target="_new">BGC0000791.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Saccharide</td>
        
        <td>O-antigen</td>
        
        
        <td>Cronobacter sakazakii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000788.1: 204-21355">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000788/index.html#r1c1" target="_new">BGC0000788.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Saccharide</td>
        
        <td>O-antigen</td>
        
        
        <td>Salmonella enterica subsp. salamae serovar Greenside</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000732.1: 151-20553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000732/index.html#r1c1" target="_new">BGC0000732.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Saccharide</td>
        
        <td>capsular polysaccharide</td>
        
        
        <td>Raoultella planticola</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000728.1: 151-18855">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000728/index.html#r1c1" target="_new">BGC0000728.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Saccharide</td>
        
        <td>capsular polysaccharide</td>
        
        
        <td>Klebsiella pneumoniae subsp. ozaenae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000729.1: 151-18871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000729/index.html#r1c1" target="_new">BGC0000729.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Saccharide</td>
        
        <td>capsular polysaccharide</td>
        
        
        <td>Klebsiella pneumoniae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000733.1: 151-20552">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000733/index.html#r1c1" target="_new">BGC0000733.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Saccharide</td>
        
        <td>capsular polysaccharide</td>
        
        
        <td>Klebsiella pneumoniae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000730.1: 151-21786">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000730/index.html#r1c1" target="_new">BGC0000730.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Saccharide</td>
        
        <td>capsular polysaccharide</td>
        
        
        <td>Klebsiella pneumoniae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000731.1: 151-19654">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000731/index.html#r1c1" target="_new">BGC0000731.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Saccharide</td>
        
        <td>capsular polysaccharide</td>
        
        
        <td>Klebsiella pneumoniae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1c5-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001882.1: 2571644-2615001">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001882/index.html#r1c1" target="_new">BGC0001882.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>chrysoxanthone A, chrysoxanthone B, chrysoxanthone C</td>
      <td>Penicillium rubens Wisconsin 54-1255</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000791.1: 0-12875">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000791/index.html#r1c1" target="_new">BGC0000791.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Saccharide</td>
      <td>O-antigen</td>
      <td>Cronobacter sakazakii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000788.1: 204-21355">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000788/index.html#r1c1" target="_new">BGC0000788.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Saccharide</td>
      <td>O-antigen</td>
      <td>Salmonella enterica subsp. salamae serovar Greenside</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000732.1: 151-20553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000732/index.html#r1c1" target="_new">BGC0000732.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Saccharide</td>
      <td>capsular polysaccharide</td>
      <td>Raoultella planticola</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000728.1: 151-18855">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000728/index.html#r1c1" target="_new">BGC0000728.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Saccharide</td>
      <td>capsular polysaccharide</td>
      <td>Klebsiella pneumoniae subsp. ozaenae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000729.1: 151-18871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000729/index.html#r1c1" target="_new">BGC0000729.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Saccharide</td>
      <td>capsular polysaccharide</td>
      <td>Klebsiella pneumoniae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000733.1: 151-20552">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000733/index.html#r1c1" target="_new">BGC0000733.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Saccharide</td>
      <td>capsular polysaccharide</td>
      <td>Klebsiella pneumoniae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000730.1: 151-21786">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000730/index.html#r1c1" target="_new">BGC0000730.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Saccharide</td>
      <td>capsular polysaccharide</td>
      <td>Klebsiella pneumoniae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000731.1: 151-19654">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000731/index.html#r1c1" target="_new">BGC0000731.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.51, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Saccharide</td>
      <td>capsular polysaccharide</td>
      <td>Klebsiella pneumoniae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1c5-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-34"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-34">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1c5-pfam-details-svg" data-filename="c00001_CC216C_.._r1c5_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1c5-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r1c6' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00001_CC216C_.. - Region 6 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-249"></div> <span class="help-tooltip" id="region_svg-help-249">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00001_CC216C_...region006.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1c6-svg" data-filename="c00001_CC216C_.._r1c6.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 528,844 - 578,266 nt. (total: 49,423 nt)
        
        <a class="cluster-rules-header" id="r1c6-rules-header" href="#r1c6">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r1c6-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r1c6-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-250"></div> <span class="help-tooltip" id="focus-panel-help-250">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1c6">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1c6-nrps_pks" data-name="r1c6-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r1c6-clusterblast" data-name="r1c6-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1c6-knownclusterblast" data-name="r1c6-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1c6-subclusterblast" data-name="r1c6-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1c6-MIBiG-cluster-compare" data-name="r1c6-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1c6-pfam-details" data-name="r1c6-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1c6-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-35"></div> <span class="help-tooltip" id="nrps-domain-help-35">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1c6-details-svg" data-filename="c00001_CC216C_.._r1c6_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r1c6-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r1c6-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-38"></div> <span class="help-tooltip" id="cb-general-help-38">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1c6-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1c6_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1c6_1.svg">
        NZ_JNLT01000002 (131197-339742): Streptomyces sp. CNS654 CD02DRAFT scaffold00... (29% of genes show similarity), NRPS,PKS-like,T1PKS,bacteriocin,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r1c6_2.svg">
        NZ_KZ819154 (157790-205683): Streptomyces sp. CS014 scaffold00001, whole geno... (86% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c6_3.svg">
        NZ_KZ819178 (7188774-7237177): Streptomyces sp. CS131 scaffold00001, whole ge... (86% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c6_4.svg">
        NZ_KZ819182 (7257301-7311610): Streptomyces sp. CS147 scaffold00001, whole ge... (70% of genes show similarity), NRPS,T1PKS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r1c6_5.svg">
        NZ_WWGD01000026 (543241-597592): Streptomyces sp. SID8378 SID8378.c26, whole ... (70% of genes show similarity), NRPS,T1PKS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r1c6_6.svg">
        NZ_LIPC01000003 (382329-472802): Streptomyces sp. TSRI0445 scaffold3, whole g... (41% of genes show similarity), NRPS,T1PKS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r1c6_7.svg">
        NZ_KZ679082 (7248215-7301545): Streptomyces sp. CS149 scaffold00001, whole ge... (71% of genes show similarity), NRPS,T1PKS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r1c6_8.svg">
        NZ_CP013738 (7040957-7089202): Streptomyces globisporus C-1027 chromosome, co... (76% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1c6_9.svg">
        NZ_CP029361 (510761-567737): Streptomyces globisporus strain TFH56 chromosome... (63% of genes show similarity), NRPS,T1PKS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r1c6_10.svg">
        NZ_KZ268499 (7042388-7090533): Streptomyces sp. SS07 Scaffold1, whole genome ... (78% of genes show similarity), NRPS,T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r1c6-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1c6-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c6-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-39"></div> <span class="help-tooltip" id="cb-known-help-39">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r1c6-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r1c6_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r1c6_1.svg">
        heat-stable antifungal factor
      </option>
      
      <option value = "svg/knownclusterblast_r1c6_2.svg">
        frontalamide B
      </option>
      
      <option value = "svg/knownclusterblast_r1c6_3.svg">
        SGR PTMs
      </option>
      
      <option value = "svg/knownclusterblast_r1c6_4.svg">
        combamide
      </option>
      
      <option value = "svg/knownclusterblast_r1c6_5.svg">
        pactamides
      </option>
      
      <option value = "svg/knownclusterblast_r1c6_6.svg">
        clifednamide A
      </option>
      
      <option value = "svg/knownclusterblast_r1c6_7.svg">
        ikarugamycin
      </option>
      
      <option value = "svg/knownclusterblast_r1c6_8.svg">
        xiamycin A
      </option>
      
    </select>
    <button id="knownclusterblast-r1c6-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r1c6-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c6-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-40"></div> <span class="help-tooltip" id="cb-sub-help-40">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1c6-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-41"></div> <span class="help-tooltip" id="MIBiG-help-41">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1c6-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1c6-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1c6"></div>
    <div id="comparison-MIBiG-r1c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001043.1: 950357-966540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001043/index.html#r1c1" target="_new">BGC0001043.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.83 (id:0.68, order:1.00, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.83 (id:0.68, order:1.00, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
         
        
        <td>1.15</td>
        <td>NRP, Polyketide</td>
        
        <td>SGR PTMs</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.65</td>
        <td>NRP</td>
        
        <td>hexose-palythine-serine, hexose-shinorine</td>
        
        
        <td>Heteroscytonema crispum UCFS10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001726.1: 12489-31637">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001726/index.html#r1c1" target="_new">BGC0001726.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.56, order:0.61, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.56, order:0.61, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.65</td>
        <td>NRP, Polyketide</td>
        
        <td>pactamides</td>
        
        
        <td>Streptomyces pactum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996.1: 4892741-4909316">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.57, order:0.73, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.57, order:0.73, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.64</td>
        <td>NRP, Polyketide</td>
        
        <td>frontalamide B</td>
        
        
        <td>Streptomyces sp. SPB78</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000999.1: 772-20280">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000999/index.html#r1c1" target="_new">BGC0000999.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.56, order:0.58, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.56, order:0.58, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.62</td>
        <td>NRP, Polyketide</td>
        
        <td>heat-stable antifungal factor</td>
        
        
        <td>Lysobacter enzymogenes</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.62</td>
        <td>NRP, Alkaloid</td>
        
        <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001556.1: 568-21149">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001556/index.html#r1c1" target="_new">BGC0001556.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.54, order:0.58, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.54, order:0.58, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.58</td>
        <td>NRP, Polyketide</td>
        
        <td>combamide</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.57</td>
        <td>Terpene, Polyketide</td>
        
        <td>pyripyropene A</td>
        
        
        <td>unidentified unclassified sequences.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.51</td>
        <td>Polyketide</td>
        
        <td>betaenone C, betaenone A</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.51</td>
        <td>Polyketide</td>
        
        <td>betaenone A, betaenone B, betaenone C</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1c6-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001043.1: 950357-966540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001043/index.html#r1c1" target="_new">BGC0001043.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.83 (id:0.68, order:1.00, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
      <td>0.83</td>
      <td>NRP, Polyketide</td>
      <td>SGR PTMs</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>hexose-palythine-serine, hexose-shinorine</td>
      <td>Heteroscytonema crispum UCFS10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001726.1: 12489-31637">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001726/index.html#r1c1" target="_new">BGC0001726.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.56, order:0.61, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP, Polyketide</td>
      <td>pactamides</td>
      <td>Streptomyces pactum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996.1: 4892741-4909316">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.57, order:0.73, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP, Polyketide</td>
      <td>frontalamide B</td>
      <td>Streptomyces sp. SPB78</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000999.1: 772-20280">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000999/index.html#r1c1" target="_new">BGC0000999.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.56, order:0.58, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP, Polyketide</td>
      <td>heat-stable antifungal factor</td>
      <td>Lysobacter enzymogenes</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP, Alkaloid</td>
      <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001556.1: 568-21149">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001556/index.html#r1c1" target="_new">BGC0001556.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.54, order:0.58, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP, Polyketide</td>
      <td>combamide</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene, Polyketide</td>
      <td>pyripyropene A</td>
      <td>unidentified unclassified sequences.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>betaenone C, betaenone A</td>
      <td>Phoma betae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>betaenone A, betaenone B, betaenone C</td>
      <td>Phoma betae</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1c6-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-42"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-42">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1c6-pfam-details-svg" data-filename="c00001_CC216C_.._r1c6_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1c6-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r1c6-nrps_pks_products" data-name="r1c6-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r1c6-nrps_pks" data-name="r1c6-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r1c6-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-36"></div> <span class="help-tooltip" id="nrps-structures-help-36">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 12, location 528843 - 578266:
          <div class="collapser collapser-target-CC12 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r1c6-12" class="smiles-canvas" data-smiles="C([*])C(-O)NC(CCCN)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(pk - orn)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pk,orn" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pk*,orn*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r1c6-12-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)NC(CCCN)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r1c6-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-37"></div> <span class="help-tooltip" id="nrps-monomers-help-37">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds548843_558266</span></strong>: pk - orn
      <div class="collapser collapser-target-cds548843_558266 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>PKS_AT (549..845)</strong>: pk
          <div class="collapser collapser-target-cds548843_558266 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds548843_558266-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds548843_558266-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 126.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 119.8</dt>

  <dd></dd><dt>Malonyl-CoA: 113.5</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 96.4</dt>

  <dd></dd><dt>Isobutyryl-CoA: 91.0</dt>

  <dd></dd><dt>Propionyl-CoA: 73.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 71.1</dt>

  <dd></dd><dt>Acetyl-CoA: 69.5</dt>

  <dd></dd><dt>trans-1,2-CPDA: 68.6</dt>

  <dd></dd><dt>Benzoyl-CoA: 59.7</dt>

  <dd></dd><dt>CHC-CoA: 52.1</dt>

  <dd></dd><dt>fatty_acid: 49.9</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 41.4</dt>

  <dd></dd><dt>inactive: 33.3</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2301..2727)</strong>: orn
          <div class="collapser collapser-target-cds548843_558266 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn
              <div class="collapser collapser-target-cds548843_558266-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>orn, horn</dd>
   <dt>Single AA prediction:</dt>
   <dd>orn</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVGEVGSIDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r1c7' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00001_CC216C_.. - Region 7 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-251"></div> <span class="help-tooltip" id="region_svg-help-251">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00001_CC216C_...region007.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1c7-svg" data-filename="c00001_CC216C_.._r1c7.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 635,237 - 661,810 nt. (total: 26,574 nt)
        
        <a class="cluster-rules-header" id="r1c7-rules-header" href="#r1c7">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r1c7-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r1c7-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-252"></div> <span class="help-tooltip" id="focus-panel-help-252">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1c7">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1c7-clusterblast" data-name="r1c7-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1c7-knownclusterblast" data-name="r1c7-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1c7-subclusterblast" data-name="r1c7-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1c7-MIBiG-cluster-compare" data-name="r1c7-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1c7-pfam-details" data-name="r1c7-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1c7-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-43"></div> <span class="help-tooltip" id="cb-general-help-43">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1c7-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1c7_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1c7_1.svg">
        NZ_LACR01000008 (280010-306584): Streptomyces sp. Ncost-T6T-1 YW5DRAFT scaffo... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1c7_2.svg">
        NZ_CP013738 (6959814-6986388): Streptomyces globisporus C-1027 chromosome, co... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1c7_3.svg">
        NZ_KZ268499 (6960087-6986661): Streptomyces sp. SS07 Scaffold1, whole genome ... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1c7_4.svg">
        NZ_LZRD01000002 (316632-343206): Streptomyces sp. PTY087I2 A3Q37 contig000002... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1c7_5.svg">
        NZ_KZ195572 (1037505-1063787): Streptomyces sp. CS057 scaffold00001, whole ge... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1c7_6.svg">
        NZ_CP029361 (622360-648489): Streptomyces globisporus strain TFH56 chromosome... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1c7_7.svg">
        NZ_CM003601 (7567957-7594528): Streptomyces anulatus strain ATCC 11523 chromo... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1c7_8.svg">
        NC_010572 (1127347-1153611): Streptomyces griseus subsp. griseus NBRC 13350, ... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1c7_9.svg">
        NZ_GL877172 (1246793-1273053): Streptomyces sp. ACT-1 strain XylebKG-1 scaffo... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1c7_10.svg">
        NZ_FLTQ01000002 (269677-296251): Streptomyces sp. MnatMP-M77, whole genome sh... (100% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r1c7-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1c7-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c7-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-44"></div> <span class="help-tooltip" id="cb-known-help-44">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r1c7-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r1c7_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r1c7_1.svg">
        hopene
      </option>
      
      <option value = "svg/knownclusterblast_r1c7_2.svg">
        calicheamicin
      </option>
      
      <option value = "svg/knownclusterblast_r1c7_3.svg">
        phosphonoglycans
      </option>
      
    </select>
    <button id="knownclusterblast-r1c7-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r1c7-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1c7-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-45"></div> <span class="help-tooltip" id="cb-sub-help-45">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1c7-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-46"></div> <span class="help-tooltip" id="MIBiG-help-46">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1c7-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1c7-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1c7"></div>
    <div id="comparison-MIBiG-r1c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663.1: 7516016-7529773">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.62, order:0.58, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>Terpene</td>
        
        <td>hopene</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Myxococcus xanthus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Rhodobacter sphaeroides</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644.1: 97-6276">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Dietzia sp. CQ4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>β-carotein</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000643.1: 363-10046">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000643/index.html#r1c1" target="_new">BGC0000643.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Brevundimonas vesicularis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000634.1: 1312-11784">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000634/index.html#r1c1" target="_new">BGC0000634.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Brevundimonas sp. SD212</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630.1: 389-6051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
        
        
        <td>Paracoccus haeundaensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1c7-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663.1: 7516016-7529773">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.62, order:0.58, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>Terpene</td>
      <td>hopene</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Myxococcus xanthus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Rhodobacter sphaeroides</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644.1: 97-6276">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Dietzia sp. CQ4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene</td>
      <td>β-carotein</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000643.1: 363-10046">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000643/index.html#r1c1" target="_new">BGC0000643.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Brevundimonas vesicularis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000634.1: 1312-11784">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000634/index.html#r1c1" target="_new">BGC0000634.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Brevundimonas sp. SD212</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630.1: 389-6051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
      <td>Paracoccus haeundaensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1c7-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-47"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-47">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1c7-pfam-details-svg" data-filename="c00001_CC216C_.._r1c7_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1c7-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r2c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00002_CC216C_.. - Region 1 - ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-253"></div> <span class="help-tooltip" id="region_svg-help-253">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00002_CC216C_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r2c1-svg" data-filename="c00002_CC216C_.._r2c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 239,083 - 249,481 nt. (total: 10,399 nt)
        
        <a class="cluster-rules-header" id="r2c1-rules-header" href="#r2c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r2c1-rules">ectoine: ectoine_synt</div>
      
      <div class="region-svg-container">
       <div id='r2c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-254"></div> <span class="help-tooltip" id="focus-panel-help-254">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r2c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r2c1-clusterblast" data-name="r2c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r2c1-knownclusterblast" data-name="r2c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r2c1-subclusterblast" data-name="r2c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r2c1-MIBiG-cluster-compare" data-name="r2c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r2c1-pfam-details" data-name="r2c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r2c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-48"></div> <span class="help-tooltip" id="cb-general-help-48">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r2c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r2c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r2c1_1.svg">
        NZ_JNLT01000001 (67287-77686): Streptomyces sp. CNS654 CD02DRAFT scaffold0000... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r2c1_2.svg">
        NZ_NNBN01000016 (68219-78618): Streptomyces sp. CB02613 scaffold16, whole gen... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r2c1_3.svg">
        NZ_KZ819182 (1594076-1604475): Streptomyces sp. CS147 scaffold00001, whole ge... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r2c1_4.svg">
        NZ_KZ679082 (1488853-1499252): Streptomyces sp. CS149 scaffold00001, whole ge... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r2c1_5.svg">
        NZ_WWGD01000034 (67953-78352): Streptomyces sp. SID8378 SID8378.c34, whole ge... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r2c1_6.svg">
        NZ_KZ819154 (6248505-6258904): Streptomyces sp. CS014 scaffold00001, whole ge... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r2c1_7.svg">
        NZ_WWIP01000001 (1254795-1265194): Streptomyces sp. SID5466 SID5466.c1, whole... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r2c1_8.svg">
        NZ_LACR01000001 (68200-78599): Streptomyces sp. Ncost-T6T-1 YW5DRAFT scaffold... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r2c1_9.svg">
        NZ_KZ268499 (1504603-1514702): Streptomyces sp. SS07 Scaffold1, whole genome ... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r2c1_10.svg">
        NZ_CP010833 (6355824-6366223): Streptomyces sp. Tue 6075 strain Tue6075 chrom... (100% of genes show similarity), ectoine
      </option>
      
    </select>
    <button id= "clusterblast-r2c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r2c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r2c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-49"></div> <span class="help-tooltip" id="cb-known-help-49">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r2c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r2c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r2c1_1.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r2c1_2.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r2c1_3.svg">
        ectoine
      </option>
      
    </select>
    <button id="knownclusterblast-r2c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r2c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r2c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-50"></div> <span class="help-tooltip" id="cb-sub-help-50">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r2c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-51"></div> <span class="help-tooltip" id="MIBiG-help-51">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r2c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r2c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r2c1"></div>
    <div id="comparison-MIBiG-r2c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">ectoine</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.91 (id:0.77, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.91), rgb(0, 0, 0, 0.91) 91%, white 91%)"></td>
         
        
        <td>0.76</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.86 (id:0.63, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
         
        
        <td>0.63</td>
        <td>Other,  , (, E, c, t, o, i, n, e, )</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.49, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855.1: 245-3455">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.49, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium kenyense</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylophaga thalassica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852.1: 415-2854">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Sporosarcina pasteurii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylobacter marinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857.1: 300-4871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylophaga alcalica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860.1: 710-5134">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylarcula marina</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.49, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum 20Z</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r2c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.91 (id:0.77, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.91), rgb(0, 0, 0, 0.91) 91%, white 91%)"></td>
      <td>0.91</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.86 (id:0.63, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
      <td>0.86</td>
      <td>Other,  , (, E, c, t, o, i, n, e, )</td>
      <td>ectoine</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.49, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855.1: 245-3455">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.49, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium kenyense</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylophaga thalassica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852.1: 415-2854">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Sporosarcina pasteurii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylobacter marinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857.1: 300-4871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylophaga alcalica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860.1: 710-5134">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylarcula marina</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.49, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum 20Z</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r2c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-52"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-52">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r2c1-pfam-details-svg" data-filename="c00002_CC216C_.._r2c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r2c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r3c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00003_CC216C_.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-255"></div> <span class="help-tooltip" id="region_svg-help-255">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00003_CC216C_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r3c1-svg" data-filename="c00003_CC216C_.._r3c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 145,103 - 199,209 nt. (total: 54,107 nt)
        
        <a class="cluster-rules-header" id="r3c1-rules-header" href="#r3c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r3c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r3c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-256"></div> <span class="help-tooltip" id="focus-panel-help-256">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r3c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r3c1-nrps_pks" data-name="r3c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r3c1-clusterblast" data-name="r3c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r3c1-knownclusterblast" data-name="r3c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r3c1-subclusterblast" data-name="r3c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r3c1-MIBiG-cluster-compare" data-name="r3c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r3c1-pfam-details" data-name="r3c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r3c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-53"></div> <span class="help-tooltip" id="nrps-domain-help-53">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r3c1-details-svg" data-filename="c00003_CC216C_.._r3c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r3c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r3c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-56"></div> <span class="help-tooltip" id="cb-general-help-56">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r3c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r3c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r3c1_1.svg">
        NZ_KZ819178 (3245972-3312553): Streptomyces sp. CS131 scaffold00001, whole ge... (71% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r3c1_2.svg">
        NZ_JNLT01000013 (74219-141705): Streptomyces sp. CNS654 CD02DRAFT scaffold000... (71% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r3c1_3.svg">
        NZ_CP030930 (3077919-3143543): Streptomyces cavourensis strain TJ430 chromoso... (68% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r3c1_4.svg">
        NZ_KZ819182 (3317342-3380708): Streptomyces sp. CS147 scaffold00001, whole ge... (67% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r3c1_5.svg">
        NZ_NNBN01000005 (395337-462847): Streptomyces sp. CB02613 scaffold5, whole ge... (62% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r3c1_6.svg">
        NZ_LIXK01000007 (55441-120892): Streptomyces sp. CB02130 scaffold7, whole gen... (67% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r3c1_7.svg">
        NZ_CP013738 (3335799-3400008): Streptomyces globisporus C-1027 chromosome, co... (67% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r3c1_8.svg">
        NZ_CP029378 (4572292-4638339): Streptomyces bacillaris strain ATCC 15855 chro... (68% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r3c1_9.svg">
        NZ_WWGD01000049 (80370-145112): Streptomyces sp. SID8378 SID8378.c49, whole g... (66% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r3c1_10.svg">
        NZ_CP011522 (5024143-5089376): Streptomyces sp. CFMR 7 strain CFMR-7 chromoso... (62% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r3c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r3c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r3c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-57"></div> <span class="help-tooltip" id="cb-known-help-57">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r3c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r3c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r3c1_1.svg">
        phthoxazolin
      </option>
      
      <option value = "svg/knownclusterblast_r3c1_2.svg">
        phosphonoglycans
      </option>
      
      <option value = "svg/knownclusterblast_r3c1_3.svg">
        echinomycin
      </option>
      
    </select>
    <button id="knownclusterblast-r3c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r3c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r3c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-58"></div> <span class="help-tooltip" id="cb-sub-help-58">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r3c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-59"></div> <span class="help-tooltip" id="MIBiG-help-59">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r3c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r3c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r3c1"></div>
    <div id="comparison-MIBiG-r3c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.45, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>hexose-palythine-serine, hexose-shinorine</td>
        
        
        <td>Heteroscytonema crispum UCFS10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251.1: 1268-15341">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>epichloenin A</td>
        
        
        <td>Epichloe festucae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250.1: 0-14073">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>epichloenin A</td>
        
        
        <td>Epichloe festucae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP, Alkaloid</td>
        
        <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r3c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.45, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.47, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.47, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>hexose-palythine-serine, hexose-shinorine</td>
      <td>Heteroscytonema crispum UCFS10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251.1: 1268-15341">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>epichloenin A</td>
      <td>Epichloe festucae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001250.1: 0-14073">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001250/index.html#r1c1" target="_new">BGC0001250.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>epichloenin A</td>
      <td>Epichloe festucae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP, Alkaloid</td>
      <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r3c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-60"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-60">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r3c1-pfam-details-svg" data-filename="c00003_CC216C_.._r3c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r3c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r3c1-nrps_pks_products" data-name="r3c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r3c1-nrps_pks" data-name="r3c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r3c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-54"></div> <span class="help-tooltip" id="nrps-structures-help-54">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 145102 - 199209:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r3c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(CO)C(=O)NCC(=O)NCC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X) + (ser) + (gly) + (gly)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ser&amp;nrps3=gly&amp;nrps4=gly" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ser*&amp;nrps3=gly*&amp;nrps4=gly*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r3c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(CO)C(=O)NCC(=O)NCC(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r3c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-55"></div> <span class="help-tooltip" id="nrps-monomers-help-55">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds151601_153524</span></strong>: X
      <div class="collapser collapser-target-cds151601_153524 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (53..509)</strong>: X
          <div class="collapser collapser-target-cds151601_153524 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds151601_153524-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">vFmliGgAaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pip</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>40% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds165102_167967</span></strong>: gly
      <div class="collapser collapser-target-cds165102_167967 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (456..844)</strong>: gly
          <div class="collapser collapser-target-cds165102_167967 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-cds165102_167967-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DIvQLGVvWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds167963_169571</span></strong>: X
      <div class="collapser collapser-target-cds167963_169571 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (13..437)</strong>: X
          <div class="collapser collapser-target-cds167963_169571 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds167963_169571-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">nAtFLGypaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds169726_175456</span></strong>: ser
      <div class="collapser collapser-target-cds169726_175456 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (462..854)</strong>: ser
          <div class="collapser collapser-target-cds169726_175456 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds169726_175456-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWHVSLVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds175787_177134</span></strong>: X
      <div class="collapser collapser-target-cds175787_177134 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (0..221)</strong>: X
          <div class="collapser collapser-target-cds175787_177134 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds175787_177134-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DtEdIGSViK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>lys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r4c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00004_CC216C_.. - Region 1 - NRPS,NRPS-like,other <div class="help-container"> <div class="help-icon" data-id="region_svg-help-257"></div> <span class="help-tooltip" id="region_svg-help-257">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00004_CC216C_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r4c1-svg" data-filename="c00004_CC216C_.._r4c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 71,542 - 138,708 nt. (total: 67,167 nt)
        
        <a class="cluster-rules-header" id="r4c1-rules-header" href="#r4c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r4c1-rules">other: (LmbU or Neocarzinostat or fom1 or bcpB or frbD or mitE or vlmB or prnB or CaiA or bacilysin or orf2_PTase)<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r4c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-258"></div> <span class="help-tooltip" id="focus-panel-help-258">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r4c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r4c1-nrps_pks" data-name="r4c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r4c1-clusterblast" data-name="r4c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r4c1-knownclusterblast" data-name="r4c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r4c1-subclusterblast" data-name="r4c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r4c1-MIBiG-cluster-compare" data-name="r4c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r4c1-pfam-details" data-name="r4c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r4c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-61"></div> <span class="help-tooltip" id="nrps-domain-help-61">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c1-details-svg" data-filename="c00004_CC216C_.._r4c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r4c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r4c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-64"></div> <span class="help-tooltip" id="cb-general-help-64">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r4c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r4c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r4c1_1.svg">
        NZ_ML123117 (3430218-3504733): Streptomyces sp. ADI98-10 scaffold001, whole g... (47% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r4c1_2.svg">
        NZ_FWZW01000001 (3616-72654): Streptomyces sp. Amel2xC10, whole genome shotgu... (39% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r4c1_3.svg">
        NZ_CP026652 (8339216-8408852): Streptomyces dengpaensis strain XZHG99 chromos... (38% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r4c1_4.svg">
        NZ_CP050692 (8157648-8227375): Streptomyces antibioticus strain DSM 41481 chr... (39% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r4c1_5.svg">
        NZ_QQVZ01000001 (920916-989679): Streptomyces sp. S816 DV517 scf1, whole geno... (47% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r4c1_6.svg">
        NZ_CP028719 (998300-1066613): Streptomyces sp. endophyte N2 chromosome, compl... (38% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r4c1_7.svg">
        NZ_MTKC01000001 (997525-1063722): Streptomyces griseofuscus strain NG1-7 NG1-... (48% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r4c1_8.svg">
        NZ_CM003601 (7813534-7889853): Streptomyces anulatus strain ATCC 11523 chromo... (52% of genes show similarity), NRPS,bacteriocin,other
      </option>
      
      <option value = "svg/clusterblast_r4c1_9.svg">
        NZ_CP015866 (557261-623748): Streptomyces parvulus strain 2297 chromosome, co... (33% of genes show similarity), NRPS,other
      </option>
      
      <option value = "svg/clusterblast_r4c1_10.svg">
        NZ_KZ819178 (599339-655817): Streptomyces sp. CS131 scaffold00001, whole geno... (47% of genes show similarity), NRPS,other
      </option>
      
    </select>
    <button id= "clusterblast-r4c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r4c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-65"></div> <span class="help-tooltip" id="cb-known-help-65">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r4c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r4c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r4c1_1.svg">
        actinomycin D
      </option>
      
      <option value = "svg/knownclusterblast_r4c1_2.svg">
        marformycin A / marformycin B / marformycin C / marformycin D / marformycin E / marformycin F
      </option>
      
      <option value = "svg/knownclusterblast_r4c1_3.svg">
        virginiamycin S1
      </option>
      
      <option value = "svg/knownclusterblast_r4c1_4.svg">
        cadaside A / cadaside B
      </option>
      
      <option value = "svg/knownclusterblast_r4c1_5.svg">
        glycinocin A
      </option>
      
      <option value = "svg/knownclusterblast_r4c1_6.svg">
        hormaomycin / hormaomycin A1 / hormaomycin A2 / hormaomycin A3 / hormaomycin A4 / hormaomycin A5 / hormaomycin A6
      </option>
      
      <option value = "svg/knownclusterblast_r4c1_7.svg">
        cysteoamide
      </option>
      
      <option value = "svg/knownclusterblast_r4c1_8.svg">
        atratumycin
      </option>
      
      <option value = "svg/knownclusterblast_r4c1_9.svg">
        RP-1776
      </option>
      
      <option value = "svg/knownclusterblast_r4c1_10.svg">
        friulimicin A / friulimicin B / friulimicin C / friulimicin D
      </option>
      
    </select>
    <button id="knownclusterblast-r4c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r4c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-66"></div> <span class="help-tooltip" id="cb-sub-help-66">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r4c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r4c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r4c1_1.svg">
        HM038106_c1: actinomycin 4-methyl-3-hydroxyanthranilic acid (100% of genes show similarity), 4-MHA
      </option>
      
    </select>
    <button id="subclusterblast-r4c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r4c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r4c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-67"></div> <span class="help-tooltip" id="MIBiG-help-67">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r4c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r4c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r4c1"></div>
    <div id="comparison-MIBiG-r4c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">other</span></th>
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>1.06</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>1.06</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>1.00</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876.1: 71-9235">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.47, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.47, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.98</td>
        <td>Other</td>
        
        <td>neopolyoxin C</td>
        
        
        <td>Streptomyces tendae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:0.75, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:0.75, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:0.75, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.96</td>
        <td>NRP</td>
        
        <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.95</td>
        <td>NRP</td>
        
        <td>ergovaline</td>
        
        
        <td>Epichloe festucae var. lolii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.47, order:0.75, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.47, order:0.75, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.92</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.91</td>
        <td>Other</td>
        
        <td>ferrichrome</td>
        
        
        <td>Aspergillus oryzae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561.1: 1236-18814">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.49, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.49, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.47, order:0.25, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.89</td>
        <td>NRP</td>
        
        <td>curacomycin</td>
        
        
        <td>Streptomyces curacoi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.86</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r4c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.47, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876.1: 71-9235">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.47, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Other</td>
      <td>neopolyoxin C</td>
      <td>Streptomyces tendae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>dimethylcoprogen</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.45, order:0.75, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>ergovaline</td>
      <td>Epichloe festucae var. lolii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>ferrichrome</td>
      <td>Aspergillus oryzae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000460.1: 890622-914333">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000460/index.html#r1c1" target="_new">BGC0000460.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.91)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>vulnibactin</td>
      <td>Vibrio vulnificus CMCP6</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r4c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-68"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-68">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c1-pfam-details-svg" data-filename="c00004_CC216C_.._r4c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r4c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r4c1-nrps_pks_products" data-name="r4c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r4c1-nrps_pks" data-name="r4c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r4c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-62"></div> <span class="help-tooltip" id="nrps-structures-help-62">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 71541 - 138708:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r4c1-1" class="smiles-canvas" data-smiles="N1C(CCC1)C(=O)NCC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(pro) + (gly)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pro&amp;nrps2=gly" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pro*&amp;nrps2=gly*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r4c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="N1C(CCC1)C(=O)NCC(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 80103 - 121427:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r4c1-3" class="smiles-canvas" data-smiles="N1C(CCC1)C(=O)NCC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(pro) + (gly)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pro&amp;nrps2=gly" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pro*&amp;nrps2=gly*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r4c1-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="N1C(CCC1)C(=O)NCC(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 4, location 83751 - 138708:
          <div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r4c1-4" class="smiles-canvas" data-smiles="N1C(CCC1)C(=O)NCC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(pro) + (gly)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pro&amp;nrps2=gly" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pro*&amp;nrps2=gly*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r4c1-4-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="N1C(CCC1)C(=O)NCC(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r4c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-63"></div> <span class="help-tooltip" id="nrps-monomers-help-63">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds102755_107177</span></strong>: gly
      <div class="collapser collapser-target-cds102755_107177 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (121..525)</strong>: gly
          <div class="collapser collapser-target-cds102755_107177 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-cds102755_107177-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILQLGLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds107176_109528</span></strong>: pro
      <div class="collapser collapser-target-cds107176_109528 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pro" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=pro*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (17..248)</strong>: pro
          <div class="collapser collapser-target-cds107176_109528 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  pro
              <div class="collapser collapser-target-cds107176_109528-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>pro, pip</dd>
   <dt>Small clusters prediction:</dt>
   <dd>pro</dd>
   <dt>Single AA prediction:</dt>
   <dd>pro</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVQFAAHVVK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pro</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds119790_121194</span></strong>: X
      <div class="collapser collapser-target-cds119790_121194 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (17..367)</strong>: X
          <div class="collapser collapser-target-cds119790_121194 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds119790_121194-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">nmsYvGvLIK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pip</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r4c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00004_CC216C_.. - Region 2 - T3PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-259"></div> <span class="help-tooltip" id="region_svg-help-259">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00004_CC216C_...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r4c2-svg" data-filename="c00004_CC216C_.._r4c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 438,821 - 479,939 nt. (total: 41,119 nt)
        
        <a class="cluster-rules-header" id="r4c2-rules-header" href="#r4c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r4c2-rules">T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)</div>
      
      <div class="region-svg-container">
       <div id='r4c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-260"></div> <span class="help-tooltip" id="focus-panel-help-260">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r4c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
    
     
      <div class="body-details-header r4c2-clusterblast" data-name="r4c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r4c2-knownclusterblast" data-name="r4c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r4c2-subclusterblast" data-name="r4c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r4c2-MIBiG-cluster-compare" data-name="r4c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r4c2-pfam-details" data-name="r4c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    
     
    <div class="body-details-section r4c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-69"></div> <span class="help-tooltip" id="cb-general-help-69">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r4c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r4c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r4c2_1.svg">
        NZ_NNBN01000003 (310678-351797): Streptomyces sp. CB02613 scaffold3, whole ge... (97% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r4c2_2.svg">
        NZ_KZ679082 (350659-390136): Streptomyces sp. CS149 scaffold00001, whole geno... (96% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r4c2_3.svg">
        NZ_LACR01000013 (90658-131777): Streptomyces sp. Ncost-T6T-1 YW5DRAFT scaffol... (94% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r4c2_4.svg">
        NZ_KZ819154 (7478219-7519338): Streptomyces sp. CS014 scaffold00001, whole ge... (96% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r4c2_5.svg">
        NZ_JNLT01000011 (138000-179119): Streptomyces sp. CNS654 CD02DRAFT scaffold00... (93% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r4c2_6.svg">
        NZ_LZRD01000016 (78925-120044): Streptomyces sp. PTY087I2 A3Q37 contig000016,... (96% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r4c2_7.svg">
        NZ_KZ268499 (397269-437213): Streptomyces sp. SS07 Scaffold1, whole genome sh... (93% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r4c2_8.svg">
        NZ_LIXK01000006 (90923-132042): Streptomyces sp. CB02130 scaffold6, whole gen... (93% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r4c2_9.svg">
        NZ_LIPC01000004 (95534-136653): Streptomyces sp. TSRI0445 scaffold4, whole ge... (93% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r4c2_10.svg">
        NZ_CP029361 (7102493-7143612): Streptomyces globisporus strain TFH56 chromoso... (93% of genes show similarity), T3PKS
      </option>
      
    </select>
    <button id= "clusterblast-r4c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r4c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-70"></div> <span class="help-tooltip" id="cb-known-help-70">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r4c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r4c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_1.svg">
        herboxidiene
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_2.svg">
        lasalocid
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_3.svg">
        naringenin
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_4.svg">
        flaviolin
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_5.svg">
        flaviolin rhamnoside / 3,3&#39;-diflaviolin / flaviolin
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_6.svg">
        marineosin A / marineosin B
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_7.svg">
        undecylprodigiosin
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_8.svg">
        lipopeptide 8D1-1 / lipopeptide 8D1-2
      </option>
      
      <option value = "svg/knownclusterblast_r4c2_9.svg">
        CDA1b / CDA2a / CDA2b / CDA3a / CDA3b / CDA4a / CDA4b
      </option>
      
    </select>
    <button id="knownclusterblast-r4c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r4c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-71"></div> <span class="help-tooltip" id="cb-sub-help-71">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r4c2-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r4c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r4c2_1.svg">
        AL939125_c1: prodigiosin 2-undecylpyrrole (25% of genes show similarity), 2-undecylpyrrole
      </option>
      
    </select>
    <button id="subclusterblast-r4c2-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r4c2-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r4c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-72"></div> <span class="help-tooltip" id="MIBiG-help-72">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r4c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r4c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r4c2"></div>
    <div id="comparison-MIBiG-r4c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T3PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310.1: 6460417-6463122">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
         
        
        <td>0.58</td>
        <td>Terpene</td>
        
        <td>naringenin</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000280.1: 384-6033">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000280/index.html#r1c1" target="_new">BGC0000280.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>Polyketide</td>
        
        <td>2,4-diacetylphloroglucinol</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000281.1: 0-8137">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000281/index.html#r1c1" target="_new">BGC0000281.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>Polyketide</td>
        
        <td>2,4-diacetylphloroglucinol</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.53, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Other</td>
        
        <td>flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001903.1: 8904-19774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001903/index.html#r1c1" target="_new">BGC0001903.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.52, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>Other</td>
        
        <td>neocarazostatin A</td>
        
        
        <td>Streptomyces sp. MA37</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.53, order:0.42, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Polyketide, Saccharide</td>
        
        <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001224.1: 1518-11352">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001224/index.html#r1c1" target="_new">BGC0001224.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Other</td>
        
        <td>reductasporine</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897.1: 37423-54017">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>cepacin A</td>
        
        
        <td>Burkholderia ambifaria IOP40-10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001782.1: 1050-39207">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001782/index.html#r1c1" target="_new">BGC0001782.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Other</td>
        
        <td>spiroindimicin A, spiroindimicin B, spiroindimicin C, spiroindimicin D, indimicin A, indimicin B, indimicin C, indimicin D, indimicin E, lynamicin A, lynamicin D, lynamicin F, lynamicin G</td>
        
        
        <td>Streptomyces sp. SCSIO 03032</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Other</td>
        
        <td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>
        
        
        <td>Streptomyces sp. RM-5-8</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r4c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310.1: 6460417-6463122">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
      <td>0.83</td>
      <td>Terpene</td>
      <td>naringenin</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000280.1: 384-6033">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000280/index.html#r1c1" target="_new">BGC0000280.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Polyketide</td>
      <td>2,4-diacetylphloroglucinol</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000281.1: 0-8137">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000281/index.html#r1c1" target="_new">BGC0000281.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Polyketide</td>
      <td>2,4-diacetylphloroglucinol</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.53, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Other</td>
      <td>flaviolin</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001903.1: 8904-19774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001903/index.html#r1c1" target="_new">BGC0001903.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.52, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>neocarazostatin A</td>
      <td>Streptomyces sp. MA37</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.53, order:0.42, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide, Saccharide</td>
      <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
      <td>Saccharopolyspora erythraea</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001224.1: 1518-11352">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001224/index.html#r1c1" target="_new">BGC0001224.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>reductasporine</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001897.1: 37423-54017">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001897/index.html#r1c1" target="_new">BGC0001897.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>cepacin A</td>
      <td>Burkholderia ambifaria IOP40-10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001782.1: 1050-39207">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001782/index.html#r1c1" target="_new">BGC0001782.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Other</td>
      <td>spiroindimicin A, spiroindimicin B, spiroindimicin C, spiroindimicin D, indimicin A, indimicin B, indimicin C, indimicin D, indimicin E, lynamicin A, lynamicin D, lynamicin F, lynamicin G</td>
      <td>Streptomyces sp. SCSIO 03032</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Other</td>
      <td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>
      <td>Streptomyces sp. RM-5-8</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r4c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-73"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-73">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c2-pfam-details-svg" data-filename="c00004_CC216C_.._r4c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r4c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r4c3' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00004_CC216C_.. - Region 3 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-261"></div> <span class="help-tooltip" id="region_svg-help-261">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00004_CC216C_...region003.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r4c3-svg" data-filename="c00004_CC216C_.._r4c3.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 525,737 - 576,627 nt. (total: 50,891 nt)
        
        <a class="cluster-rules-header" id="r4c3-rules-header" href="#r4c3">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r4c3-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r4c3-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-262"></div> <span class="help-tooltip" id="focus-panel-help-262">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r4c3">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r4c3-nrps_pks" data-name="r4c3-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r4c3-clusterblast" data-name="r4c3-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r4c3-knownclusterblast" data-name="r4c3-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r4c3-subclusterblast" data-name="r4c3-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r4c3-MIBiG-cluster-compare" data-name="r4c3-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r4c3-pfam-details" data-name="r4c3-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r4c3-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-74"></div> <span class="help-tooltip" id="nrps-domain-help-74">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c3-details-svg" data-filename="c00004_CC216C_.._r4c3_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r4c3-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r4c3-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-77"></div> <span class="help-tooltip" id="cb-general-help-77">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r4c3-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r4c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r4c3_1.svg">
        NZ_LACR01000013 (176921-227904): Streptomyces sp. Ncost-T6T-1 YW5DRAFT scaffo... (96% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c3_2.svg">
        NZ_KZ819154 (7558050-7660478): Streptomyces sp. CS014 scaffold00001, whole ge... (50% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c3_3.svg">
        NZ_LIPB01000005 (276535-413044): Streptomyces sp. CB00072 scaffold5, whole ge... (34% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c3_4.svg">
        NZ_NNBN01000003 (177666-278460): Streptomyces sp. CB02613 scaffold3, whole ge... (51% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c3_5.svg">
        NZ_LIVW01000005 (318609-452419): Streptomyces sp. CB02115 scaffold5, whole ge... (32% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c3_6.svg">
        NZ_KZ819178 (167473-256583): Streptomyces sp. CS131 scaffold00001, whole geno... (50% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c3_7.svg">
        NZ_CP010833 (7632675-7737618): Streptomyces sp. Tue 6075 strain Tue6075 chrom... (45% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c3_8.svg">
        NZ_KZ819182 (237576-378380): Streptomyces sp. CS147 scaffold00001, whole geno... (33% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c3_9.svg">
        NZ_CP013738 (286741-383233): Streptomyces globisporus C-1027 chromosome, comp... (47% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c3_10.svg">
        NZ_LIXK01000006 (223125-321661): Streptomyces sp. CB02130 scaffold6, whole ge... (46% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r4c3-download">Download graphic</button>
  </div>
  <div id="clusterblast-r4c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c3-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-78"></div> <span class="help-tooltip" id="cb-known-help-78">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r4c3-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r4c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r4c3_1.svg">
        coelichelin
      </option>
      
      <option value = "svg/knownclusterblast_r4c3_2.svg">
        salinichelins
      </option>
      
      <option value = "svg/knownclusterblast_r4c3_3.svg">
        scabichelin
      </option>
      
      <option value = "svg/knownclusterblast_r4c3_4.svg">
        gobichelin A / gobichelin B
      </option>
      
      <option value = "svg/knownclusterblast_r4c3_5.svg">
        qinichelins
      </option>
      
      <option value = "svg/knownclusterblast_r4c3_6.svg">
        salinomycin
      </option>
      
    </select>
    <button id="knownclusterblast-r4c3-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r4c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c3-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-79"></div> <span class="help-tooltip" id="cb-sub-help-79">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r4c3-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-80"></div> <span class="help-tooltip" id="MIBiG-help-80">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r4c3-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r4c3-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r4c3"></div>
    <div id="comparison-MIBiG-r4c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000325.1: 510822-533448">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000325/index.html#r1c1" target="_new">BGC0000325.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.77 (id:0.62, order:0.82, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
         
        
        <td>0.48</td>
        <td>NRP</td>
        
        <td>coelichelin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
        
        
        <td>Penicillium rubens Wisconsin 54-1255</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001671.1: 5106-76272">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001671/index.html#r1c1" target="_new">BGC0001671.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>monobactam</td>
        
        
        <td>Agrobacterium tumefaciens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.49, order:0.69, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>exochelin</td>
        
        
        <td>Mycolicibacterium smegmatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000321.1: 1657-43703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000321/index.html#r1c1" target="_new">BGC0000321.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>chaetocin</td>
        
        
        <td>Collariella virescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001791.1: 5526778-5556971">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001791/index.html#r1c1" target="_new">BGC0001791.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>sunshinamide</td>
        
        
        <td>Gynuella sunshinyii YC6258</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001824.1: 39732-57269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001824/index.html#r1c1" target="_new">BGC0001824.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>xefoampeptides A-G</td>
        
        
        <td>Xenorhabdus beddingii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>rhodochelin</td>
        
        
        <td>Rhodococcus jostii RHA1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r4c3-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000325.1: 510822-533448">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000325/index.html#r1c1" target="_new">BGC0000325.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.77 (id:0.62, order:0.82, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
      <td>0.77</td>
      <td>NRP</td>
      <td>coelichelin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
      <td>Penicillium rubens Wisconsin 54-1255</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001671.1: 5106-76272">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001671/index.html#r1c1" target="_new">BGC0001671.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>monobactam</td>
      <td>Agrobacterium tumefaciens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.49, order:0.69, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>exochelin</td>
      <td>Mycolicibacterium smegmatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000321.1: 1657-43703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000321/index.html#r1c1" target="_new">BGC0000321.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>chaetocin</td>
      <td>Collariella virescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001791.1: 5526778-5556971">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001791/index.html#r1c1" target="_new">BGC0001791.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>sunshinamide</td>
      <td>Gynuella sunshinyii YC6258</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001824.1: 39732-57269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001824/index.html#r1c1" target="_new">BGC0001824.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>xefoampeptides A-G</td>
      <td>Xenorhabdus beddingii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>rhodochelin</td>
      <td>Rhodococcus jostii RHA1</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r4c3-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-81"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-81">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c3-pfam-details-svg" data-filename="c00004_CC216C_.._r4c3_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r4c3-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r4c3-nrps_pks_products" data-name="r4c3-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r4c3-nrps_pks" data-name="r4c3-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r4c3-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-75"></div> <span class="help-tooltip" id="nrps-structures-help-75">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 6, location 525736 - 576627:
          <div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r4c3-6" class="smiles-canvas" data-smiles="NC(CCCN)C(=O)NC(C(O)C)C(=O)NC(CCCN)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-orn - D-thr - orn)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-orn,d-thr,orn" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-orn*,d-thr*,orn*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r4c3-6-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CCCN)C(=O)NC(C(O)C)C(=O)NC(CCCN)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r4c3-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-76"></div> <span class="help-tooltip" id="nrps-monomers-help-76">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds545680_556693</span></strong>: orn - thr - orn
      <div class="collapser collapser-target-cds545680_556693 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn,thr,orn" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn*,thr*,orn*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (144..542)</strong>: orn
          <div class="collapser collapser-target-cds545680_556693 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn
              <div class="collapser collapser-target-cds545680_556693-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>orn, horn</dd>
   <dt>Single AA prediction:</dt>
   <dd>orn</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DINYWGGIGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1626..2039)</strong>: thr
          <div class="collapser collapser-target-cds545680_556693 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds545680_556693-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNVGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3090..3488)</strong>: orn
          <div class="collapser collapser-target-cds545680_556693 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn
              <div class="collapser collapser-target-cds545680_556693-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DMwNLGLINK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r4c4' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00004_CC216C_.. - Region 4 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-263"></div> <span class="help-tooltip" id="region_svg-help-263">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00004_CC216C_...region004.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r4c4-svg" data-filename="c00004_CC216C_.._r4c4.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 580,655 - 628,091 nt. (total: 47,437 nt)
        
        <a class="cluster-rules-header" id="r4c4-rules-header" href="#r4c4">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r4c4-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r4c4-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-264"></div> <span class="help-tooltip" id="focus-panel-help-264">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r4c4">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r4c4-nrps_pks" data-name="r4c4-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r4c4-clusterblast" data-name="r4c4-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r4c4-knownclusterblast" data-name="r4c4-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r4c4-subclusterblast" data-name="r4c4-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r4c4-MIBiG-cluster-compare" data-name="r4c4-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r4c4-pfam-details" data-name="r4c4-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r4c4-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-82"></div> <span class="help-tooltip" id="nrps-domain-help-82">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c4-details-svg" data-filename="c00004_CC216C_.._r4c4_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r4c4-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r4c4-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-85"></div> <span class="help-tooltip" id="cb-general-help-85">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r4c4-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r4c4_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r4c4_1.svg">
        NZ_WWGD01000035 (5537-62231): Streptomyces sp. SID8378 SID8378.c35, whole gen... (83% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c4_2.svg">
        NZ_LACR01000016 (786-57493): Streptomyces sp. Ncost-T6T-1 YW5DRAFT scaffold 1... (85% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c4_3.svg">
        NZ_NNBN01000003 (177666-278460): Streptomyces sp. CB02613 scaffold3, whole ge... (57% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c4_4.svg">
        NZ_KZ819154 (7558050-7660478): Streptomyces sp. CS014 scaffold00001, whole ge... (56% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c4_5.svg">
        NZ_CP010833 (7632675-7737618): Streptomyces sp. Tue 6075 strain Tue6075 chrom... (53% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c4_6.svg">
        NZ_KZ268499 (211509-308519): Streptomyces sp. SS07 Scaffold1, whole genome sh... (57% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c4_7.svg">
        NZ_CP013738 (286741-383233): Streptomyces globisporus C-1027 chromosome, comp... (58% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c4_8.svg">
        NZ_LZRD01000012 (180206-236959): Streptomyces sp. PTY087I2 A3Q37 contig000012... (83% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r4c4_9.svg">
        NZ_WWIF01000005 (279694-397190): Streptomyces sp. SID5594 SID5594.c5, whole g... (46% of genes show similarity), NRPS,PKS-like,T1PKS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r4c4_10.svg">
        NZ_KZ819178 (167473-256583): Streptomyces sp. CS131 scaffold00001, whole geno... (60% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r4c4-download">Download graphic</button>
  </div>
  <div id="clusterblast-r4c4-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c4-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-86"></div> <span class="help-tooltip" id="cb-known-help-86">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r4c4-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r4c4_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r4c4_1.svg">
        streptobactin
      </option>
      
      <option value = "svg/knownclusterblast_r4c4_2.svg">
        bacillibactin
      </option>
      
      <option value = "svg/knownclusterblast_r4c4_3.svg">
        paenibactin
      </option>
      
      <option value = "svg/knownclusterblast_r4c4_4.svg">
        bacillibactin
      </option>
      
      <option value = "svg/knownclusterblast_r4c4_5.svg">
        myxochelin A / myxochelin B
      </option>
      
      <option value = "svg/knownclusterblast_r4c4_6.svg">
        fuscachelin A / fuscachelin B / fuscachelin C
      </option>
      
      <option value = "svg/knownclusterblast_r4c4_7.svg">
        paulomycin
      </option>
      
      <option value = "svg/knownclusterblast_r4c4_8.svg">
        paulomycin
      </option>
      
      <option value = "svg/knownclusterblast_r4c4_9.svg">
        mirubactin
      </option>
      
      <option value = "svg/knownclusterblast_r4c4_10.svg">
        enduracidin
      </option>
      
    </select>
    <button id="knownclusterblast-r4c4-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r4c4-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c4-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-87"></div> <span class="help-tooltip" id="cb-sub-help-87">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r4c4-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-88"></div> <span class="help-tooltip" id="MIBiG-help-88">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r4c4-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r4c4-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r4c4"></div>
    <div id="comparison-MIBiG-r4c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000368.1: 83-26477">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000368/index.html#r1c1" target="_new">BGC0000368.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.86 (id:0.75, order:0.88, components:0.97)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
         
        
        <td>0.65</td>
        <td>NRP</td>
        
        <td>streptobactin</td>
        
        
        <td>Streptomyces sp. ATCC 700974</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401.1: 0-12882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.52, order:0.76, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>NRP</td>
        
        <td>paenibactin</td>
        
        
        <td>Paenibacillus elgii B69</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.52, order:0.72, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>NRP</td>
        
        <td>bacillibactin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>NRP, Alkaloid</td>
        
        <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.42, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>vicibactin</td>
        
        
        <td>Rhizobium etli CFN 42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001149.1: 196767-209553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001149/index.html#r1c1" target="_new">BGC0001149.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>NRP, Saccharide</td>
        
        <td>brabantamide A</td>
        
        
        <td>Pseudomonas sp. SHC52</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000309.1: 3278324-3297919">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000309/index.html#r1c1" target="_new">BGC0000309.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.52, order:0.47, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>bacillibactin</td>
        
        
        <td>Bacillus subtilis subsp. subtilis str. 168</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.47, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r4c4-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000368.1: 83-26477">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000368/index.html#r1c1" target="_new">BGC0000368.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.86 (id:0.75, order:0.88, components:0.97)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
      <td>0.86</td>
      <td>NRP</td>
      <td>streptobactin</td>
      <td>Streptomyces sp. ATCC 700974</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401.1: 0-12882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.52, order:0.76, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>NRP</td>
      <td>paenibactin</td>
      <td>Paenibacillus elgii B69</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.52, order:0.72, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP, Alkaloid</td>
      <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.42, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>vicibactin</td>
      <td>Rhizobium etli CFN 42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001149.1: 196767-209553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001149/index.html#r1c1" target="_new">BGC0001149.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP, Saccharide</td>
      <td>brabantamide A</td>
      <td>Pseudomonas sp. SHC52</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000309.1: 3278324-3297919">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000309/index.html#r1c1" target="_new">BGC0000309.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.52, order:0.47, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus subtilis subsp. subtilis str. 168</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.47, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r4c4-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-89"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-89">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c4-pfam-details-svg" data-filename="c00004_CC216C_.._r4c4_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r4c4-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r4c4-nrps_pks_products" data-name="r4c4-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r4c4-nrps_pks" data-name="r4c4-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r4c4-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-83"></div> <span class="help-tooltip" id="nrps-structures-help-83">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 7, location 580654 - 628091:
          <div class="collapser collapser-target-CC7 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r4c4-7" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(C(O)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X - thr)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,thr" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,thr*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r4c4-7-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(C(O)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r4c4-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-84"></div> <span class="help-tooltip" id="nrps-monomers-help-84">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds591397_593035</span></strong>: dhb
      <div class="collapser collapser-target-cds591397_593035 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=diOH-Bz" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=diOH-Bz*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (32..439)</strong>: dhb
          <div class="collapser collapser-target-cds591397_593035 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  dhb
              <div class="collapser collapser-target-cds591397_593035-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>dhb, sal</dd>
   <dt>Small clusters prediction:</dt>
   <dd>dhb, sal</dd>
   <dt>Single AA prediction:</dt>
   <dd>dhb</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">PLPglGVVNK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>dhb</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds600654_608091</span></strong>: X - thr
      <div class="collapser collapser-target-cds600654_608091 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,thr" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,thr*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (531..944)</strong>: X
          <div class="collapser collapser-target-cds600654_608091 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds600654_608091-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAwDVGLVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>arg</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1621..2030)</strong>: thr
          <div class="collapser collapser-target-cds600654_608091 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds600654_608091-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNVGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r4c5' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00004_CC216C_.. - Region 5 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-265"></div> <span class="help-tooltip" id="region_svg-help-265">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00004_CC216C_...region005.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r4c5-svg" data-filename="c00004_CC216C_.._r4c5.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 658,887 - 681,100 nt. (total: 22,214 nt)
        
        <a class="cluster-rules-header" id="r4c5-rules-header" href="#r4c5">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r4c5-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r4c5-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-266"></div> <span class="help-tooltip" id="focus-panel-help-266">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r4c5">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r4c5-clusterblast" data-name="r4c5-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r4c5-knownclusterblast" data-name="r4c5-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r4c5-subclusterblast" data-name="r4c5-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r4c5-MIBiG-cluster-compare" data-name="r4c5-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r4c5-pfam-details" data-name="r4c5-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r4c5-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-90"></div> <span class="help-tooltip" id="cb-general-help-90">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r4c5-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r4c5_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r4c5_1.svg">
        NZ_JNLT01000022 (63093-85307): Streptomyces sp. CNS654 CD02DRAFT scaffold0002... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r4c5_2.svg">
        NZ_NNBN01000003 (122310-142885): Streptomyces sp. CB02613 scaffold3, whole ge... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r4c5_3.svg">
        NZ_WWGD01000035 (103218-125432): Streptomyces sp. SID8378 SID8378.c35, whole ... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r4c5_4.svg">
        NZ_LACR01000016 (100120-122334): Streptomyces sp. Ncost-T6T-1 YW5DRAFT scaffo... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r4c5_5.svg">
        NZ_KZ819182 (172823-193429): Streptomyces sp. CS147 scaffold00001, whole geno... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r4c5_6.svg">
        NZ_KZ819154 (7701694-7723908): Streptomyces sp. CS014 scaffold00001, whole ge... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r4c5_7.svg">
        NZ_KZ819178 (109136-129767): Streptomyces sp. CS131 scaffold00001, whole geno... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r4c5_8.svg">
        NZ_KZ679082 (85620-106100): Streptomyces sp. CS149 scaffold00001, whole genom... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r4c5_9.svg">
        NZ_KZ268499 (106326-192770): Streptomyces sp. SS07 Scaffold1, whole genome sh... (27% of genes show similarity), PKS-like,T1PKS,T2PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r4c5_10.svg">
        NZ_LIPB01000005 (172426-194640): Streptomyces sp. CB00072 scaffold5, whole ge... (100% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r4c5-download">Download graphic</button>
  </div>
  <div id="clusterblast-r4c5-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c5-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-91"></div> <span class="help-tooltip" id="cb-known-help-91">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r4c5-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r4c5_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r4c5_1.svg">
        geosmin
      </option>
      
      <option value = "svg/knownclusterblast_r4c5_2.svg">
        vazabitide A
      </option>
      
    </select>
    <button id="knownclusterblast-r4c5-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r4c5-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c5-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-92"></div> <span class="help-tooltip" id="cb-sub-help-92">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r4c5-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-93"></div> <span class="help-tooltip" id="MIBiG-help-93">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r4c5-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r4c5-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r4c5"></div>
    <div id="comparison-MIBiG-r4c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001181.1: 6666218-6668399">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001181/index.html#r1c1" target="_new">BGC0001181.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.75 (id:0.56, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
         
        
        <td>0.44</td>
        <td>Terpene</td>
        
        <td>geosmin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.53, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
        <td>0.41</td>
        <td>Terpene</td>
        
        <td>geosmin</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001746.1: 7136894-7138013">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001746/index.html#r1c1" target="_new">BGC0001746.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Terpene</td>
        
        <td>pristinol</td>
        
        
        <td>Streptomyces pristinaespiralis ATCC 25486</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.69, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001150.1: 12-23077">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001150/index.html#r1c1" target="_new">BGC0001150.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>pamamycin-607</td>
        
        
        <td>Streptomyces alboniger</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000252.1: 248-15270">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000252/index.html#r1c1" target="_new">BGC0000252.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>nonactin</td>
        
        
        <td>Streptomyces griseus subsp. griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000805.1: 897-5623">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000805/index.html#r1c1" target="_new">BGC0000805.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>moenomycin</td>
        
        
        <td>Streptomyces viridosporus ATCC 14672</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r4c5-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001181.1: 6666218-6668399">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001181/index.html#r1c1" target="_new">BGC0001181.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.75 (id:0.56, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
      <td>0.75</td>
      <td>Terpene</td>
      <td>geosmin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.53, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>Terpene</td>
      <td>geosmin</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001746.1: 7136894-7138013">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001746/index.html#r1c1" target="_new">BGC0001746.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>pristinol</td>
      <td>Streptomyces pristinaespiralis ATCC 25486</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.69, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001150.1: 12-23077">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001150/index.html#r1c1" target="_new">BGC0001150.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Polyketide</td>
      <td>pamamycin-607</td>
      <td>Streptomyces alboniger</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000252.1: 248-15270">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000252/index.html#r1c1" target="_new">BGC0000252.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Polyketide</td>
      <td>nonactin</td>
      <td>Streptomyces griseus subsp. griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000805.1: 897-5623">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000805/index.html#r1c1" target="_new">BGC0000805.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Saccharide</td>
      <td>moenomycin</td>
      <td>Streptomyces viridosporus ATCC 14672</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r4c5-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-94"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-94">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c5-pfam-details-svg" data-filename="c00004_CC216C_.._r4c5_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r4c5-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r4c6' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00004_CC216C_.. - Region 6 - butyrolactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-267"></div> <span class="help-tooltip" id="region_svg-help-267">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00004_CC216C_...region006.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r4c6-svg" data-filename="c00004_CC216C_.._r4c6.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 713,448 - 724,392 nt. (total: 10,945 nt)
        
        <a class="cluster-rules-header" id="r4c6-rules-header" href="#r4c6">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r4c6-rules">butyrolactone: AfsA</div>
      
      <div class="region-svg-container">
       <div id='r4c6-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-268"></div> <span class="help-tooltip" id="focus-panel-help-268">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r4c6">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r4c6-clusterblast" data-name="r4c6-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r4c6-knownclusterblast" data-name="r4c6-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r4c6-subclusterblast" data-name="r4c6-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r4c6-MIBiG-cluster-compare" data-name="r4c6-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r4c6-pfam-details" data-name="r4c6-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r4c6-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-95"></div> <span class="help-tooltip" id="cb-general-help-95">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r4c6-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r4c6_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r4c6_1.svg">
        NZ_NNBN01000003 (80445-91390): Streptomyces sp. CB02613 scaffold3, whole geno... (100% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r4c6_2.svg">
        NZ_JNLT01000022 (116094-127039): Streptomyces sp. CNS654 CD02DRAFT scaffold00... (100% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r4c6_3.svg">
        NZ_KZ819178 (72267-83212): Streptomyces sp. CS131 scaffold00001, whole genome... (100% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r4c6_4.svg">
        NZ_KZ819182 (133424-144369): Streptomyces sp. CS147 scaffold00001, whole geno... (100% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r4c6_5.svg">
        NZ_WWGD01000035 (153775-164720): Streptomyces sp. SID8378 SID8378.c35, whole ... (100% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r4c6_6.svg">
        NZ_LACR01000016 (153037-163982): Streptomyces sp. Ncost-T6T-1 YW5DRAFT scaffo... (100% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r4c6_7.svg">
        NZ_KZ819154 (7754546-7765491): Streptomyces sp. CS014 scaffold00001, whole ge... (100% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r4c6_8.svg">
        NZ_KZ679082 (43525-52820): Streptomyces sp. CS149 scaffold00001, whole genome... (100% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r4c6_9.svg">
        NZ_CP010833 (7817446-7828391): Streptomyces sp. Tue 6075 strain Tue6075 chrom... (100% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r4c6_10.svg">
        NZ_KZ268499 (62575-71901): Streptomyces sp. SS07 Scaffold1, whole genome shot... (100% of genes show similarity), butyrolactone
      </option>
      
    </select>
    <button id= "clusterblast-r4c6-download">Download graphic</button>
  </div>
  <div id="clusterblast-r4c6-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c6-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-96"></div> <span class="help-tooltip" id="cb-known-help-96">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r4c6-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r4c6_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r4c6_1.svg">
        coelimycin P1
      </option>
      
      <option value = "svg/knownclusterblast_r4c6_2.svg">
        SCB1 / SCB2 / SCB3
      </option>
      
      <option value = "svg/knownclusterblast_r4c6_3.svg">
        A-factor
      </option>
      
      <option value = "svg/knownclusterblast_r4c6_4.svg">
        cyphomycin
      </option>
      
      <option value = "svg/knownclusterblast_r4c6_5.svg">
        oxalomycin B
      </option>
      
      <option value = "svg/knownclusterblast_r4c6_6.svg">
        merochlorin A / merochlorin B / deschloro-merochlorin A / deschloro-merochlorin B / isochloro-merochlorin B / dichloro-merochlorin B / merochlorin D / merochlorin C
      </option>
      
      <option value = "svg/knownclusterblast_r4c6_7.svg">
        griseoviridin / fijimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r4c6_8.svg">
        lactonamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r4c6-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r4c6-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c6-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-97"></div> <span class="help-tooltip" id="cb-sub-help-97">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r4c6-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-98"></div> <span class="help-tooltip" id="MIBiG-help-98">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r4c6-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r4c6-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r4c6"></div>
    <div id="comparison-MIBiG-r4c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">butyrolactone</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000849.1: 6890527-6892237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000849/index.html#r1c1" target="_new">BGC0000849.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
         
        
        <td>0.58</td>
        <td>Other,  , (, B, u, t, y, r, o, l, a, c, t, o, n, e, )</td>
        
        <td>SCB1, SCB2, SCB3</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000848.1: 8273443-8274388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000848/index.html#r1c1" target="_new">BGC0000848.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.79 (id:0.63, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
         
        
        <td>0.52</td>
        <td>Other</td>
        
        <td>A-factor</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.48, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.37</td>
        <td>Other</td>
        
        <td>γ-butyrolactone</td>
        
        
        <td>Streptomyces acidiscabies</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001540.1: 633-38461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001540/index.html#r1c1" target="_new">BGC0001540.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>CC-1065</td>
        
        
        <td>Streptomyces zelensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
        
        
        <td>Parastagonospora nodorum SN15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000189.1: 3771004-3779127">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000189/index.html#r1c1" target="_new">BGC0000189.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Polyketide</td>
        
        <td>3-(2&#39;-hydroxy-3&#39;-oxo-4&#39;-methylpentyl)-indole, 3-(2&#39;-hydroxy-3&#39;-oxo-4&#39;-methylhexyl)-indole, 3-(2&#39;-acetoxy-3&#39;-oxo-4&#39;-methylpentyl)-indole, 3-(2&#39;-acetoxy-3&#39;-oxo-4&#39;-methylhexyl)-indole</td>
        
        
        <td>Xenorhabdus bovienii SS-2004</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001458.1: 4714477-4725572">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001458/index.html#r1c1" target="_new">BGC0001458.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Other,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
        
        <td>salinipostin G</td>
        
        
        <td>Salinispora tropica CNB-440</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001633.1: 52623-58838">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001633/index.html#r1c1" target="_new">BGC0001633.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.46, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Other</td>
        
        <td>ketomemicin B3, ketomemicin B4</td>
        
        
        <td>Streptomyces mobaraensis NBRC 13819 = DSM 40847</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001364.1: 317877-335461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001364/index.html#r1c1" target="_new">BGC0001364.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
         
        
        <td>0.12</td>
        <td>RiPP</td>
        
        <td>cinnamycin B</td>
        
        
        <td>Actinomadura atramentaria DSM 43919</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r4c6-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000849.1: 6890527-6892237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000849/index.html#r1c1" target="_new">BGC0000849.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
      <td>0.83</td>
      <td>Other,  , (, B, u, t, y, r, o, l, a, c, t, o, n, e, )</td>
      <td>SCB1, SCB2, SCB3</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000848.1: 8273443-8274388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000848/index.html#r1c1" target="_new">BGC0000848.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.79 (id:0.63, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
      <td>0.79</td>
      <td>Other</td>
      <td>A-factor</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.48, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>Other</td>
      <td>γ-butyrolactone</td>
      <td>Streptomyces acidiscabies</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001540.1: 633-38461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001540/index.html#r1c1" target="_new">BGC0001540.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Other</td>
      <td>CC-1065</td>
      <td>Streptomyces zelensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Polyketide</td>
      <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
      <td>Parastagonospora nodorum SN15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000189.1: 3771004-3779127">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000189/index.html#r1c1" target="_new">BGC0000189.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Polyketide</td>
      <td>3-(2&#39;-hydroxy-3&#39;-oxo-4&#39;-methylpentyl)-indole, 3-(2&#39;-hydroxy-3&#39;-oxo-4&#39;-methylhexyl)-indole, 3-(2&#39;-acetoxy-3&#39;-oxo-4&#39;-methylpentyl)-indole, 3-(2&#39;-acetoxy-3&#39;-oxo-4&#39;-methylhexyl)-indole</td>
      <td>Xenorhabdus bovienii SS-2004</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001458.1: 4714477-4725572">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001458/index.html#r1c1" target="_new">BGC0001458.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Other,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
      <td>salinipostin G</td>
      <td>Salinispora tropica CNB-440</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001633.1: 52623-58838">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001633/index.html#r1c1" target="_new">BGC0001633.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.46, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Other</td>
      <td>ketomemicin B3, ketomemicin B4</td>
      <td>Streptomyces mobaraensis NBRC 13819 = DSM 40847</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001364.1: 317877-335461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001364/index.html#r1c1" target="_new">BGC0001364.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
      <td>0.39</td>
      <td>RiPP</td>
      <td>cinnamycin B</td>
      <td>Actinomadura atramentaria DSM 43919</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r4c6-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-99"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-99">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c6-pfam-details-svg" data-filename="c00004_CC216C_.._r4c6_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r4c6-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r4c7' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00004_CC216C_.. - Region 7 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-269"></div> <span class="help-tooltip" id="region_svg-help-269">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00004_CC216C_...region007.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r4c7-svg" data-filename="c00004_CC216C_.._r4c7.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 739,690 - 765,295 nt. (total: 25,606 nt)
        
        <a class="cluster-rules-header" id="r4c7-rules-header" href="#r4c7">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r4c7-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r4c7-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-270"></div> <span class="help-tooltip" id="focus-panel-help-270">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r4c7">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r4c7-nrps_pks" data-name="r4c7-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r4c7-clusterblast" data-name="r4c7-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r4c7-knownclusterblast" data-name="r4c7-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r4c7-subclusterblast" data-name="r4c7-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r4c7-MIBiG-cluster-compare" data-name="r4c7-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r4c7-pfam-details" data-name="r4c7-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r4c7-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-100"></div> <span class="help-tooltip" id="nrps-domain-help-100">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c7-details-svg" data-filename="c00004_CC216C_.._r4c7_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r4c7-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r4c7-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-103"></div> <span class="help-tooltip" id="cb-general-help-103">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r4c7-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r4c7_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r4c7_1.svg">
        NZ_KZ819178 (11374-54116): Streptomyces sp. CS131 scaffold00001, whole genome... (42% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r4c7_2.svg">
        NZ_KZ679082 (92-29605): Streptomyces sp. CS149 scaffold00001, whole genome sh... (60% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r4c7_3.svg">
        NZ_NNBN01000003 (22006-63617): Streptomyces sp. CB02613 scaffold3, whole geno... (48% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r4c7_4.svg">
        NZ_KZ819182 (73984-116665): Streptomyces sp. CS147 scaffold00001, whole genom... (36% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r4c7_5.svg">
        NZ_WWGD01000035 (180662-224295): Streptomyces sp. SID8378 SID8378.c35, whole ... (33% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r4c7_6.svg">
        NZ_CP010833 (7845610-7889219): Streptomyces sp. Tue 6075 strain Tue6075 chrom... (36% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r4c7_7.svg">
        NZ_CP021707 (31852-73953): Streptomyces sp. S063 chromosome, complete genome (29% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r4c7_8.svg">
        NZ_LWKZ01000026 (119896-239451): Streptomyces sp. CB02923 scaffold6, whole ge... (6% of genes show similarity), NRPS,T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r4c7_9.svg">
        NZ_CP028159 (4481822-&gt;4593967): Plantactinospora sp. BB1 chromosome, complete... (6% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r4c7_10.svg">
        NZ_JADG01000003 (334520-483052): Actinomadura oligospora ATCC 43269 P696DRAFT... (7% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r4c7-download">Download graphic</button>
  </div>
  <div id="clusterblast-r4c7-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c7-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-104"></div> <span class="help-tooltip" id="cb-known-help-104">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r4c7-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r4c7_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r4c7_1.svg">
        nocathiacin
      </option>
      
      <option value = "svg/knownclusterblast_r4c7_2.svg">
        glycopeptidolipid
      </option>
      
      <option value = "svg/knownclusterblast_r4c7_3.svg">
        borrelidin
      </option>
      
      <option value = "svg/knownclusterblast_r4c7_4.svg">
        borrelidin
      </option>
      
    </select>
    <button id="knownclusterblast-r4c7-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r4c7-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r4c7-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-105"></div> <span class="help-tooltip" id="cb-sub-help-105">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r4c7-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-106"></div> <span class="help-tooltip" id="MIBiG-help-106">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r4c7-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r4c7-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r4c7"></div>
    <div id="comparison-MIBiG-r4c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001972.1: 267-32147">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001972/index.html#r1c1" target="_new">BGC0001972.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.54, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Other</td>
        
        <td>toxoflavin, fervenulin</td>
        
        
        <td>Streptomyces hiroshimensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001973.1: 267-32147">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001973/index.html#r1c1" target="_new">BGC0001973.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.54, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Other</td>
        
        <td>fervenulin</td>
        
        
        <td>Streptomyces hiroshimensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>fellutamide B</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001959.1: 1219-11580">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001959/index.html#r1c1" target="_new">BGC0001959.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.54, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Other</td>
        
        <td>taylorflavin A, taylorflavin B</td>
        
        
        <td>Streptomyces hiroshimensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r4c7-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001972.1: 267-32147">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001972/index.html#r1c1" target="_new">BGC0001972.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.54, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Other</td>
      <td>toxoflavin, fervenulin</td>
      <td>Streptomyces hiroshimensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001973.1: 267-32147">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001973/index.html#r1c1" target="_new">BGC0001973.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.54, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Other</td>
      <td>fervenulin</td>
      <td>Streptomyces hiroshimensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>fellutamide B</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001959.1: 1219-11580">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001959/index.html#r1c1" target="_new">BGC0001959.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.54, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Other</td>
      <td>taylorflavin A, taylorflavin B</td>
      <td>Streptomyces hiroshimensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r4c7-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-107"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-107">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r4c7-pfam-details-svg" data-filename="c00004_CC216C_.._r4c7_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r4c7-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r4c7-nrps_pks_products" data-name="r4c7-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r4c7-nrps_pks" data-name="r4c7-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r4c7-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-101"></div> <span class="help-tooltip" id="nrps-structures-help-101">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 10, location 739689 - 765295:
          <div class="collapser collapser-target-CC10 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r4c7-10" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r4c7-10-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r4c7-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-102"></div> <span class="help-tooltip" id="nrps-monomers-help-102">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds759689_763304</span></strong>: X
      <div class="collapser collapser-target-cds759689_763304 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (23..420)</strong>: X
          <div class="collapser collapser-target-cds759689_763304 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds759689_763304-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DPmivGVVfK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>glu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r5c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00005_CC216C_.. - Region 1 - ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-271"></div> <span class="help-tooltip" id="region_svg-help-271">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00005_CC216C_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r5c1-svg" data-filename="c00005_CC216C_.._r5c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 204,458 - 214,832 nt. (total: 10,375 nt)
        
        <a class="cluster-rules-header" id="r5c1-rules-header" href="#r5c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r5c1-rules">ectoine: ectoine_synt</div>
      
      <div class="region-svg-container">
       <div id='r5c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-272"></div> <span class="help-tooltip" id="focus-panel-help-272">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r5c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r5c1-clusterblast" data-name="r5c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r5c1-knownclusterblast" data-name="r5c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r5c1-subclusterblast" data-name="r5c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r5c1-MIBiG-cluster-compare" data-name="r5c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r5c1-pfam-details" data-name="r5c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r5c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-108"></div> <span class="help-tooltip" id="cb-general-help-108">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r5c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r5c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r5c1_1.svg">
        NZ_LIXK01000005 (546995-557370): Streptomyces sp. CB02130 scaffold5, whole ge... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r5c1_2.svg">
        NZ_CP013738 (6564224-6574599): Streptomyces globisporus C-1027 chromosome, co... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r5c1_3.svg">
        NZ_WUEJ01000008 (328748-379046): Streptomyces sp. YIM 132580 Scaffold8, whole... (22% of genes show similarity), NRPS-like,ectoine
      </option>
      
      <option value = "svg/clusterblast_r5c1_4.svg">
        NZ_CP032543 (6316602-6368005): Streptomyces griseus subsp. griseus strain ATC... (22% of genes show similarity), NRPS-like,arylpolyene,ectoine
      </option>
      
      <option value = "svg/clusterblast_r5c1_5.svg">
        NZ_KL573553 (226824-237199): Streptomyces baarnensis strain NRRL B-2842 P144 ... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r5c1_6.svg">
        NZ_ML123127 (1364625-1375000): Streptomyces sp. ADI98-12 scaffold002, whole g... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r5c1_7.svg">
        NZ_CP023701 (73996-180562): Streptomyces subrutilus strain ATCC 27467 chromos... (12% of genes show similarity), NRPS,butyrolactone,ectoine,terpene
      </option>
      
      <option value = "svg/clusterblast_r5c1_8.svg">
        NZ_CP020039 (73131-126275): Streptomyces sp. 3211 isolate 3 chromosome, compl... (24% of genes show similarity), NRPS,ectoine
      </option>
      
      <option value = "svg/clusterblast_r5c1_9.svg">
        NZ_CP020555 (87136-140489): Streptomyces sp. Sge12 chromosome, complete genome (24% of genes show similarity), NRPS,ectoine
      </option>
      
      <option value = "svg/clusterblast_r5c1_10.svg">
        NZ_LIVQ01000010 (7862-141901): Streptomyces sp. CB00455 scaffold10, whole gen... (13% of genes show similarity), NRPS,butyrolactone,ectoine
      </option>
      
    </select>
    <button id= "clusterblast-r5c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r5c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r5c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-109"></div> <span class="help-tooltip" id="cb-known-help-109">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r5c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r5c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r5c1_1.svg">
        kosinostatin
      </option>
      
      <option value = "svg/knownclusterblast_r5c1_2.svg">
        showdomycin
      </option>
      
      <option value = "svg/knownclusterblast_r5c1_3.svg">
        indigoidine
      </option>
      
    </select>
    <button id="knownclusterblast-r5c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r5c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r5c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-110"></div> <span class="help-tooltip" id="cb-sub-help-110">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r5c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-111"></div> <span class="help-tooltip" id="MIBiG-help-111">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r5c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r5c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r5c1"></div>
    <div id="comparison-MIBiG-r5c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">ectoine</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.51, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylobacter marinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852.1: 415-2854">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Sporosarcina pasteurii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.51, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum 20Z</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.53, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.49, order:0.36, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Other</td>
        
        <td>showdomycin</td>
        
        
        <td>Streptomyces showdoensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000312.1: 0-41205">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000312/index.html#r1c1" target="_new">BGC0000312.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>bassianolide</td>
        
        
        <td>Beauveria bassiana</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641.1: 3124570-3174107">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>NRP</td>
        
        <td>kolossin</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r5c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.51, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylobacter marinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852.1: 415-2854">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Sporosarcina pasteurii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.51, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum 20Z</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.53, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.49, order:0.36, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Other</td>
      <td>showdomycin</td>
      <td>Streptomyces showdoensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000312.1: 0-41205">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000312/index.html#r1c1" target="_new">BGC0000312.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>NRP</td>
      <td>bassianolide</td>
      <td>Beauveria bassiana</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641.1: 3124570-3174107">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>NRP</td>
      <td>kolossin</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r5c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-112"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-112">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r5c1-pfam-details-svg" data-filename="c00005_CC216C_.._r5c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r5c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r6c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00006_CC216C_.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-273"></div> <span class="help-tooltip" id="region_svg-help-273">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00006_CC216C_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r6c1-svg" data-filename="c00006_CC216C_.._r6c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 24,916 nt. (total: 24,916 nt)
        
        <a class="cluster-rules-header" id="r6c1-rules-header" href="#r6c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r6c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r6c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-274"></div> <span class="help-tooltip" id="focus-panel-help-274">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r6c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r6c1-clusterblast" data-name="r6c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r6c1-knownclusterblast" data-name="r6c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r6c1-subclusterblast" data-name="r6c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r6c1-MIBiG-cluster-compare" data-name="r6c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r6c1-pfam-details" data-name="r6c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r6c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-113"></div> <span class="help-tooltip" id="cb-general-help-113">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r6c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r6c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r6c1_1.svg">
        NZ_NNBJ01000021 (52992-78243): Streptomyces sp. CB01201 scaffold21, whole gen... (54% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r6c1_2.svg">
        NC_017765 (1981958-2054213): Streptomyces hygroscopicus subsp. jinggangensis ... (23% of genes show similarity), NRPS,lanthipeptide,terpene
      </option>
      
      <option value = "svg/clusterblast_r6c1_3.svg">
        NZ_CP015098 (5837564-5861246): Streptomyces qaidamensis strain S10 chromosome... (50% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r6c1_4.svg">
        NZ_JACHGV010000003 (643816-667682): Streptomyces paradoxus strain DSM 43350 G... (47% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r6c1_5.svg">
        NZ_CP030073 (8179897-8205433): Streptomyces sp. ZFG47 chromosome, complete ge... (52% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r6c1_6.svg">
        NZ_KQ948355 (75231-148735): Streptomyces corchorusii strain DSM 40340 PRJNA29... (20% of genes show similarity), NRPS,lanthipeptide,terpene
      </option>
      
      <option value = "svg/clusterblast_r6c1_7.svg">
        NZ_CP013219 (162920-234208): Streptomyces hygroscopicus subsp. limoneus strai... (23% of genes show similarity), NRPS,lanthipeptide,terpene
      </option>
      
      <option value = "svg/clusterblast_r6c1_8.svg">
        NZ_GG657756 (8154653-8179458): Streptomyces lividans TK24 supercont1.1, whole... (43% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r6c1_9.svg">
        NZ_RSEG01000001 (59184-84720): Streptomyces tendae strain UTMC 3329 Scaffold1... (45% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r6c1_10.svg">
        NZ_CP026730 (171255-196371): Streptomyces sp. CB09001 chromosome, complete ge... (45% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r6c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r6c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-114"></div> <span class="help-tooltip" id="cb-known-help-114">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r6c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r6c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_1.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_2.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_3.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_4.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_5.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_6.svg">
        rustmicin
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_7.svg">
        carotenoid
      </option>
      
    </select>
    <button id="knownclusterblast-r6c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r6c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-115"></div> <span class="help-tooltip" id="cb-sub-help-115">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r6c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-116"></div> <span class="help-tooltip" id="MIBiG-help-116">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r6c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r6c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r6c1"></div>
    <div id="comparison-MIBiG-r6c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.74 (id:0.56, order:0.73, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
         
        
        <td>0.43</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Terpene</td>
        
        <td>β-carotein</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640.1: 2399-8466">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Enterobacteriaceae bacterium DC404</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.55, order:0.66, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.55, order:0.53, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650.1: 28-7720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Algoriphagus sp. KK10202C</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.56, order:0.47, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.54, order:0.47, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces collinus Tu 365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000641.1: 1208-7454">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000641/index.html#r1c1" target="_new">BGC0000641.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Enterobacteriaceae bacterium DC416</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000639.1: 324-6564">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000639/index.html#r1c1" target="_new">BGC0000639.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Enterobacteriaceae bacterium DC260</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r6c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.74 (id:0.56, order:0.73, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
      <td>0.74</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Terpene</td>
      <td>β-carotein</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640.1: 2399-8466">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Enterobacteriaceae bacterium DC404</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.55, order:0.66, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.55, order:0.53, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650.1: 28-7720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Algoriphagus sp. KK10202C</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.56, order:0.47, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.54, order:0.47, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces collinus Tu 365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000641.1: 1208-7454">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000641/index.html#r1c1" target="_new">BGC0000641.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Enterobacteriaceae bacterium DC416</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000639.1: 324-6564">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000639/index.html#r1c1" target="_new">BGC0000639.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Enterobacteriaceae bacterium DC260</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r6c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-117"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-117">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c1-pfam-details-svg" data-filename="c00006_CC216C_.._r6c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r6c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r6c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00006_CC216C_.. - Region 2 - lassopeptide <div class="help-container"> <div class="help-icon" data-id="region_svg-help-275"></div> <span class="help-tooltip" id="region_svg-help-275">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00006_CC216C_...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r6c2-svg" data-filename="c00006_CC216C_.._r6c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 94,351 - 117,086 nt. (total: 22,736 nt)
        
        <a class="cluster-rules-header" id="r6c2-rules-header" href="#r6c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r6c2-rules">lassopeptide: (PF13471 and Asn_synthase or micJ25 or mcjC)</div>
      
      <div class="region-svg-container">
       <div id='r6c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-276"></div> <span class="help-tooltip" id="focus-panel-help-276">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r6c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r6c2-clusterblast" data-name="r6c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r6c2-knownclusterblast" data-name="r6c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r6c2-subclusterblast" data-name="r6c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r6c2-lassopeptides" data-name="r6c2-lassopeptides">
        Lasso peptides
      </div>
     
    
     
    
     
      <div class="body-details-header r6c2-MIBiG-cluster-compare" data-name="r6c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r6c2-pfam-details" data-name="r6c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r6c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-118"></div> <span class="help-tooltip" id="cb-general-help-118">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r6c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r6c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r6c2_1.svg">
        NZ_JNLT01000024 (9930-32744): Streptomyces sp. CNS654 CD02DRAFT scaffold00023... (100% of genes show similarity), lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r6c2_2.svg">
        NZ_NNBN01000001 (492200-553135): Streptomyces sp. CB02613 scaffold1, whole ge... (43% of genes show similarity), lassopeptide,phosphonate
      </option>
      
      <option value = "svg/clusterblast_r6c2_3.svg">
        NZ_KZ679082 (4327560-4350296): Streptomyces sp. CS149 scaffold00001, whole ge... (95% of genes show similarity), lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r6c2_4.svg">
        NZ_CP010833 (3447648-3520439): Streptomyces sp. Tue 6075 strain Tue6075 chrom... (32% of genes show similarity), NRPS,lassopeptide,siderophore
      </option>
      
      <option value = "svg/clusterblast_r6c2_5.svg">
        NZ_LZRD01000023 (81612-142442): Streptomyces sp. PTY087I2 A3Q37 contig000023,... (43% of genes show similarity), NRPS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r6c2_6.svg">
        NZ_KZ819178 (4293410-4410454): Streptomyces sp. CS131 scaffold00001, whole ge... (25% of genes show similarity), NRPS,NRPS-like,arylpolyene,ladderane,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r6c2_7.svg">
        NZ_KZ819182 (4400634-4463428): Streptomyces sp. CS147 scaffold00001, whole ge... (37% of genes show similarity), lassopeptide,phosphonate
      </option>
      
      <option value = "svg/clusterblast_r6c2_8.svg">
        NZ_KZ819154 (3388426-3411162): Streptomyces sp. CS014 scaffold00001, whole ge... (90% of genes show similarity), lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r6c2_9.svg">
        NZ_CP021707 (4237270-4260016): Streptomyces sp. S063 chromosome, complete genome (95% of genes show similarity), lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r6c2_10.svg">
        NZ_LACR01000007 (314148-336875): Streptomyces sp. Ncost-T6T-1 YW5DRAFT scaffo... (90% of genes show similarity), lassopeptide
      </option>
      
    </select>
    <button id= "clusterblast-r6c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r6c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-119"></div> <span class="help-tooltip" id="cb-known-help-119">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r6c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r6c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r6c2_1.svg">
        keywimysin
      </option>
      
      <option value = "svg/knownclusterblast_r6c2_2.svg">
        SRO15-2005
      </option>
      
      <option value = "svg/knownclusterblast_r6c2_3.svg">
        lasalocid
      </option>
      
      <option value = "svg/knownclusterblast_r6c2_4.svg">
        caniferolide A / caniferolide B / caniferolide C / caniferolide D
      </option>
      
    </select>
    <button id="knownclusterblast-r6c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r6c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-120"></div> <span class="help-tooltip" id="cb-sub-help-120">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r6c2-lassopeptides">
      <div class="details">
    <div class="heading">
      <span>Lasso peptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lasso-body-help-121"></div> <span class="help-tooltip" id="lasso-body-help-121">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lassopeptide(s) for cds104168_104300 - Putative Class II</h4>
    <hr>
      <dl class="details-text">
        
          <dt><span class="serif">cds104168_104300</span> leader / core peptide / cleaved tail</dt>
          <dd>
            MKQQKQQKKAYVKPSMFQQGDFSKKTA
            -
            <span class ="macrolactam">G</span><span class ="macrolactam">Y</span><span class ="macrolactam">F</span><span class ="macrolactam">V</span><span class ="macrolactam">G</span><span class ="macrolactam">S</span><span class ="macrolactam">Y</span><span class ="macrolactam">K</span><span class ="macrolactam">E</span>YWSRRII
          </dd>
        
      </dl>
    
  <div class="legend">
    <h4> Legend: </h4>
    <div>
      <span class="macrolactam">Red</span>: Putative macrolactam<br>
      <span class="cut">Purple</span>: Putative cleaved off residues<br>
    </div>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r6c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-124"></div> <span class="help-tooltip" id="MIBiG-help-124">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r6c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r6c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r6c2"></div>
    <div id="comparison-MIBiG-r6c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lassopeptide</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001634.1: 9372-14335">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001634/index.html#r1c1" target="_new">BGC0001634.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.86 (id:0.63, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
         
        
        <td>0.64</td>
        <td>RiPP</td>
        
        <td>keywimysin</td>
        
        
        <td>Streptomyces sp. NRRL F-5702</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001507.1: 43037-46158">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001507/index.html#r1c1" target="_new">BGC0001507.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.46, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>RiPP</td>
        
        <td>anantin C</td>
        
        
        <td>Streptomyces olindensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001548.1: 108185-111640">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001548/index.html#r1c1" target="_new">BGC0001548.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>RiPP</td>
        
        <td>citrulassin A</td>
        
        
        <td>Streptomyces albulus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001176.1: 9270-14062">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001176/index.html#r1c1" target="_new">BGC0001176.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.34</td>
        <td>RiPP</td>
        
        <td>streptomonomicin</td>
        
        
        <td>Streptomonospora alba</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001673.1: 77019-79729">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001673/index.html#r1c1" target="_new">BGC0001673.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>RiPP</td>
        
        <td>moomysin</td>
        
        
        <td>Streptomyces cattleya NRRL 8057 = DSM 46488</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001539.1: 77018-79728">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001539/index.html#r1c1" target="_new">BGC0001539.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>RiPP</td>
        
        <td>cattlecin</td>
        
        
        <td>Streptomyces cattleya NRRL 8057 = DSM 46488</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001307.1: 0-4083">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001307/index.html#r1c1" target="_new">BGC0001307.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>RiPP</td>
        
        <td>chaxapeptin</td>
        
        
        <td>Streptomyces leeuwenhoekii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002074.1: 0-2630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002074/index.html#r1c1" target="_new">BGC0002074.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>RiPP</td>
        
        <td>huascopeptin</td>
        
        
        <td>Streptomyces huasconensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000578.1: 4289836-4294787">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000578/index.html#r1c1" target="_new">BGC0000578.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.82, order:0.49, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>RiPP</td>
        
        <td>SRO15-2005</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002053.1: 242-2917">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002053/index.html#r1c1" target="_new">BGC0002053.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>RiPP</td>
        
        <td>ulleungdin</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r6c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001634.1: 9372-14335">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001634/index.html#r1c1" target="_new">BGC0001634.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.86 (id:0.63, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
      <td>0.86</td>
      <td>RiPP</td>
      <td>keywimysin</td>
      <td>Streptomyces sp. NRRL F-5702</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001507.1: 43037-46158">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001507/index.html#r1c1" target="_new">BGC0001507.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.46, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>RiPP</td>
      <td>anantin C</td>
      <td>Streptomyces olindensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001548.1: 108185-111640">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001548/index.html#r1c1" target="_new">BGC0001548.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>RiPP</td>
      <td>citrulassin A</td>
      <td>Streptomyces albulus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001176.1: 9270-14062">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001176/index.html#r1c1" target="_new">BGC0001176.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>RiPP</td>
      <td>streptomonomicin</td>
      <td>Streptomonospora alba</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001673.1: 77019-79729">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001673/index.html#r1c1" target="_new">BGC0001673.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>RiPP</td>
      <td>moomysin</td>
      <td>Streptomyces cattleya NRRL 8057 = DSM 46488</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001539.1: 77018-79728">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001539/index.html#r1c1" target="_new">BGC0001539.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>RiPP</td>
      <td>cattlecin</td>
      <td>Streptomyces cattleya NRRL 8057 = DSM 46488</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001307.1: 0-4083">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001307/index.html#r1c1" target="_new">BGC0001307.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>RiPP</td>
      <td>chaxapeptin</td>
      <td>Streptomyces leeuwenhoekii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002074.1: 0-2630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002074/index.html#r1c1" target="_new">BGC0002074.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>RiPP</td>
      <td>huascopeptin</td>
      <td>Streptomyces huasconensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000578.1: 4289836-4294787">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000578/index.html#r1c1" target="_new">BGC0000578.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.82, order:0.49, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>RiPP</td>
      <td>SRO15-2005</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002053.1: 242-2917">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002053/index.html#r1c1" target="_new">BGC0002053.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>RiPP</td>
      <td>ulleungdin</td>
      <td>Streptomyces sp.</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r6c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-125"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-125">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c2-pfam-details-svg" data-filename="c00006_CC216C_.._r6c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r6c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r6c2-lassopeptides" data-name="r6c2-lassopeptides">
        Lasso peptides
      </div>
     
    
     
      <div class="sidepanel-details-header r6c2-RREfinder" data-name="r6c2-RREfinder">
        RREFinder
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r6c2-lassopeptides">
      <div class="more-details">
    <div class="heading">
      <span>Core peptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lasso-side-help-122"></div> <span class="help-tooltip" id="lasso-side-help-122">Lists the possible core peptides in the region. Each core peptide lists the number of disulfide bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
    </div>

  <dl class ="prediction-text">
    <dt><span class="serif">cds104168_104300</span> - Class II</dt>
      <dd>Cleavage pHMM score: -1.40</dd>
      <dd>RODEO score: 22</dd>
      <dd>Monoisotopic mass: 2005.0 Da</dd>
      <dd>Molecular weight: 2006.3 Da</dd>
      <dd>Number of bridges: 0</dd>
    
  </dl>

</div>

    </div>
     
    
     
    <div class="sidepanel-details-section r6c2-RREfinder">
      <div class="more-details">
  <div class="heading">
    <span>RRE predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="rre-side-help-123"></div> <span class="help-tooltip" id="rre-side-help-123">RREfinder results sidepanel.</span></div>
  </div>
  <dl class = 'prediction-text'>
    
      <span><strong>lassopeptide protocluster</strong> (94350...117086)</span>
      <dt></dt>
      
      
      
        
        <strong>cds106328_106580</strong> <div class="collapser collapser-target-cds106328_106580 collapser-level-cds"> </div><div class="collapser-content">
        
          <dd>RRE type: Stand_Alone_Lasso_RRE</dd>
          <dd>Location: 0 - 82</dd>
          <dd>Score: 87.5</dd>
        
        </div><br>
      <br>
    
    </dl>
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r6c3' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00006_CC216C_.. - Region 3 - lanthipeptide-class-iv <div class="help-container"> <div class="help-icon" data-id="region_svg-help-277"></div> <span class="help-tooltip" id="region_svg-help-277">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00006_CC216C_...region003.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r6c3-svg" data-filename="c00006_CC216C_.._r6c3.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 221,438 - 244,296 nt. (total: 22,859 nt)
        
        <a class="cluster-rules-header" id="r6c3-rules-header" href="#r6c3">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r6c3-rules">lanthipeptide-class-iv: (cds(LANC_like and Pkinase) and not micKC)</div>
      
      <div class="region-svg-container">
       <div id='r6c3-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-278"></div> <span class="help-tooltip" id="focus-panel-help-278">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r6c3">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r6c3-lanthipeptides" data-name="r6c3-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r6c3-clusterblast" data-name="r6c3-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r6c3-knownclusterblast" data-name="r6c3-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r6c3-subclusterblast" data-name="r6c3-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r6c3-MIBiG-cluster-compare" data-name="r6c3-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r6c3-pfam-details" data-name="r6c3-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r6c3-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-126"></div> <span class="help-tooltip" id="lanthi-body-help-126">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lanthipeptide(s) for cds231437_234296 - Putative Class III</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds229261_229393</span> leader / core peptide
          </dt>
          <dd>
            MIDNTMTVDVDELQTLEGEESVALAG
            -
            <span class="cys">C</span><span class="dhb">Dhb</span>R<span class="dhb">Dhb</span><span class="cys">C</span><span class="dhb">Dhb</span>W<span class="dhb">Dhb</span><span class="cys">C</span><span class="dhb">Dhb</span>I<span class="dhb">Dhb</span><span class="dha">Dha</span>IEQQ
        </dd>
      </dl>
    
    <div class="legend">
      <h4> Legend: </h4>
      <div>
        <span class="dha">Dha</span>: Didehydroalanine<br>
        <span class="dhb">Dhb</span>: Didehydrobutyrine<br>
      </div>
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r6c3-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-128"></div> <span class="help-tooltip" id="cb-general-help-128">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r6c3-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r6c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r6c3_1.svg">
        NZ_KZ819178 (4508922-4531278): Streptomyces sp. CS131 scaffold00001, whole ge... (78% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r6c3_2.svg">
        NZ_KZ819182 (4560076-4582101): Streptomyces sp. CS147 scaffold00001, whole ge... (83% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r6c3_3.svg">
        NZ_WWGD01000043 (33635-56494): Streptomyces sp. SID8378 SID8378.c43, whole ge... (83% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r6c3_4.svg">
        NZ_NNBN01000001 (380535-402135): Streptomyces sp. CB02613 scaffold1, whole ge... (77% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r6c3_5.svg">
        NZ_JNLT01000004 (407502-430421): Streptomyces sp. CNS654 CD02DRAFT scaffold00... (66% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r6c3_6.svg">
        NZ_ML123117 (1206509-1228779): Streptomyces sp. ADI98-10 scaffold001, whole g... (42% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r6c3_7.svg">
        NZ_LZRD01000027 (55217-84212): Streptomyces sp. PTY087I2 A3Q37 contig000027, ... (34% of genes show similarity), lanthipeptide,melanin
      </option>
      
      <option value = "svg/clusterblast_r6c3_8.svg">
        NZ_KL573566 (63548-86479): Streptomyces baarnensis strain NRRL B-2842 P144 Do... (44% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r6c3_9.svg">
        NZ_LIXK01000001 (135443-157762): Streptomyces sp. CB02130 scaffold1, whole ge... (40% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r6c3_10.svg">
        NZ_CP013738 (4571689-4594032): Streptomyces globisporus C-1027 chromosome, co... (40% of genes show similarity), lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r6c3-download">Download graphic</button>
  </div>
  <div id="clusterblast-r6c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c3-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-129"></div> <span class="help-tooltip" id="cb-known-help-129">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r6c3-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r6c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r6c3_1.svg">
        carbapenem MM4550
      </option>
      
      <option value = "svg/knownclusterblast_r6c3_2.svg">
        quartromicin A1
      </option>
      
      <option value = "svg/knownclusterblast_r6c3_3.svg">
        kistamicin A
      </option>
      
    </select>
    <button id="knownclusterblast-r6c3-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r6c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c3-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-130"></div> <span class="help-tooltip" id="cb-sub-help-130">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r6c3-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-131"></div> <span class="help-tooltip" id="MIBiG-help-131">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r6c3-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r6c3-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r6c3"></div>
    <div id="comparison-MIBiG-r6c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-iv</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.48, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>RiPP</td>
        
        <td>SapB</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000874.1: 0-36223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000874/index.html#r1c1" target="_new">BGC0000874.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.50, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Other</td>
        
        <td>blasticidin S</td>
        
        
        <td>Streptomyces griseochromogenes</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.48, order:0.51, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>RiPP</td>
        
        <td>catenulipeptin</td>
        
        
        <td>Catenulispora acidiphila DSM 44928</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496.1: 2822112-2829457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.47, order:0.51, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>RiPP</td>
        
        <td>AmfS</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:0.43, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>RiPP</td>
        
        <td>venezuelin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000503.1: 0-17083">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000503/index.html#r1c1" target="_new">BGC0000503.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>RiPP</td>
        
        <td>cinnamycin</td>
        
        
        <td>Streptomyces cinnamoneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.53, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>RiPP</td>
        
        <td>informatipeptin</td>
        
        
        <td>Streptomyces viridochromogenes DSM 40736</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:0.45, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>RiPP</td>
        
        <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
        
        
        <td>Actinomadura namibiensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000516.1: 8706-19626">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000516/index.html#r1c1" target="_new">BGC0000516.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>RiPP</td>
        
        <td>geobacillin II</td>
        
        
        <td>Geobacillus thermodenitrificans</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r6c3-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.48, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>RiPP</td>
      <td>SapB</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000874.1: 0-36223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000874/index.html#r1c1" target="_new">BGC0000874.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.50, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>blasticidin S</td>
      <td>Streptomyces griseochromogenes</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.48, order:0.51, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>RiPP</td>
      <td>catenulipeptin</td>
      <td>Catenulispora acidiphila DSM 44928</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496.1: 2822112-2829457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.47, order:0.51, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>RiPP</td>
      <td>AmfS</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:0.43, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>RiPP</td>
      <td>venezuelin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000503.1: 0-17083">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000503/index.html#r1c1" target="_new">BGC0000503.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>RiPP</td>
      <td>cinnamycin</td>
      <td>Streptomyces cinnamoneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.53, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>RiPP</td>
      <td>informatipeptin</td>
      <td>Streptomyces viridochromogenes DSM 40736</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.47, order:0.45, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>RiPP</td>
      <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
      <td>Actinomadura namibiensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000516.1: 8706-19626">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000516/index.html#r1c1" target="_new">BGC0000516.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>RiPP</td>
      <td>geobacillin II</td>
      <td>Geobacillus thermodenitrificans</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r6c3-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-132"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-132">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c3-pfam-details-svg" data-filename="c00006_CC216C_.._r6c3_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r6c3-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r6c3-lanthipeptides" data-name="r6c3-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r6c3-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-127"></div> <span class="help-tooltip" id="lanthi-side-help-127">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

    <dl class ="prediction-text">
    <dt><span class="serif">cds229261_229393</span> - Class III</dt>
        <dd>Cleavage pHMM score: 2.70</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 1847.9 Da</dd>
        <dd>Molecular weight: 1849.1 Da</dd>
        <dd>Number of bridges: 3</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 1867.1 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 1885.1 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 1903.1 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 1921.2 Da</span>
                    <span class="alt-weight-n"> N = 4</span>
                  </dd>
                
            </dl>
          </dd>
        
  </dl>

</div>

    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r6c4' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00006_CC216C_.. - Region 4 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-279"></div> <span class="help-tooltip" id="region_svg-help-279">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00006_CC216C_...region004.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r6c4-svg" data-filename="c00006_CC216C_.._r6c4.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 862,006 - 902,572 nt. (total: 40,567 nt)
        
        <a class="cluster-rules-header" id="r6c4-rules-header" href="#r6c4">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r6c4-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r6c4-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-280"></div> <span class="help-tooltip" id="focus-panel-help-280">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r6c4">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r6c4-nrps_pks" data-name="r6c4-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r6c4-clusterblast" data-name="r6c4-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r6c4-knownclusterblast" data-name="r6c4-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r6c4-subclusterblast" data-name="r6c4-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r6c4-MIBiG-cluster-compare" data-name="r6c4-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r6c4-pfam-details" data-name="r6c4-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r6c4-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-133"></div> <span class="help-tooltip" id="nrps-domain-help-133">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c4-details-svg" data-filename="c00006_CC216C_.._r6c4_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r6c4-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r6c4-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-136"></div> <span class="help-tooltip" id="cb-general-help-136">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r6c4-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r6c4_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r6c4_1.svg">
        NZ_KZ195572 (3029177-3077782): Streptomyces sp. CS057 scaffold00001, whole ge... (80% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r6c4_2.svg">
        NZ_JNLT01000003 (392156-446849): Streptomyces sp. CNS654 CD02DRAFT scaffold00... (71% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c4_3.svg">
        NZ_LACR01000009 (72816-127399): Streptomyces sp. Ncost-T6T-1 YW5DRAFT scaffol... (75% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c4_4.svg">
        NZ_KZ819178 (5134565-5185799): Streptomyces sp. CS131 scaffold00001, whole ge... (71% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c4_5.svg">
        NZ_WWGD01000029 (828732-883264): Streptomyces sp. SID8378 SID8378.c29, whole ... (65% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c4_6.svg">
        NZ_KZ819182 (5306790-5359558): Streptomyces sp. CS147 scaffold00001, whole ge... (70% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c4_7.svg">
        NZ_KZ679082 (5064968-5116934): Streptomyces sp. CS149 scaffold00001, whole ge... (71% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c4_8.svg">
        NZ_NNBN01000009 (231330-286037): Streptomyces sp. CB02613 scaffold9, whole ge... (70% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c4_9.svg">
        NZ_KZ819154 (2551138-2603973): Streptomyces sp. CS014 scaffold00001, whole ge... (73% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c4_10.svg">
        NZ_CP010833 (2686435-2738805): Streptomyces sp. Tue 6075 strain Tue6075 chrom... (75% of genes show similarity), NRPS,T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r6c4-download">Download graphic</button>
  </div>
  <div id="clusterblast-r6c4-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c4-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-137"></div> <span class="help-tooltip" id="cb-known-help-137">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r6c4-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r6c4_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r6c4_1.svg">
        leinamycin
      </option>
      
      <option value = "svg/knownclusterblast_r6c4_2.svg">
        enduracidin
      </option>
      
      <option value = "svg/knownclusterblast_r6c4_3.svg">
        azinomycin B
      </option>
      
      <option value = "svg/knownclusterblast_r6c4_4.svg">
        pyridomycin
      </option>
      
      <option value = "svg/knownclusterblast_r6c4_5.svg">
        JBIR-06
      </option>
      
      <option value = "svg/knownclusterblast_r6c4_6.svg">
        WS9326
      </option>
      
    </select>
    <button id="knownclusterblast-r6c4-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r6c4-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c4-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-138"></div> <span class="help-tooltip" id="cb-sub-help-138">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r6c4-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-139"></div> <span class="help-tooltip" id="MIBiG-help-139">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r6c4-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r6c4-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r6c4"></div>
    <div id="comparison-MIBiG-r6c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 10E9</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 22D11</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001518.1: 3489599-3513098">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001518/index.html#r1c1" target="_new">BGC0001518.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>astellolide A</td>
        
        
        <td>Aspergillus oryzae RIB40</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>AM-toxin</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r6c4-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 10E9</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 22D11</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001518.1: 3489599-3513098">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001518/index.html#r1c1" target="_new">BGC0001518.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Terpene</td>
      <td>astellolide A</td>
      <td>Aspergillus oryzae RIB40</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>AM-toxin</td>
      <td>Alternaria alternata</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r6c4-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-140"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-140">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c4-pfam-details-svg" data-filename="c00006_CC216C_.._r6c4_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r6c4-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r6c4-nrps_pks_products" data-name="r6c4-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r6c4-nrps_pks" data-name="r6c4-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r6c4-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-134"></div> <span class="help-tooltip" id="nrps-structures-help-134">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 4, location 862005 - 902572:
          <div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r6c4-4" class="smiles-canvas" data-smiles="NC(C(O)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(thr)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r6c4-4-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(C(O)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r6c4-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-135"></div> <span class="help-tooltip" id="nrps-monomers-help-135">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds880392_882882</span></strong>: thr
      <div class="collapser collapser-target-cds880392_882882 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (262..646)</strong>: thr
          <div class="collapser collapser-target-cds880392_882882 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds880392_882882-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWSVGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r6c5' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00006_CC216C_.. - Region 5 - lanthipeptide-class-iii <div class="help-container"> <div class="help-icon" data-id="region_svg-help-281"></div> <span class="help-tooltip" id="region_svg-help-281">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00006_CC216C_...region005.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r6c5-svg" data-filename="c00006_CC216C_.._r6c5.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 994,439 - 1,017,297 nt. (total: 22,859 nt)
        
        <a class="cluster-rules-header" id="r6c5-rules-header" href="#r6c5">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r6c5-rules">lanthipeptide-class-iii: (micKC and not PKS_KS)</div>
      
      <div class="region-svg-container">
       <div id='r6c5-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-282"></div> <span class="help-tooltip" id="focus-panel-help-282">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r6c5">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r6c5-lanthipeptides" data-name="r6c5-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r6c5-clusterblast" data-name="r6c5-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r6c5-knownclusterblast" data-name="r6c5-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r6c5-subclusterblast" data-name="r6c5-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r6c5-MIBiG-cluster-compare" data-name="r6c5-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r6c5-pfam-details" data-name="r6c5-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r6c5-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-141"></div> <span class="help-tooltip" id="lanthi-body-help-141">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lanthipeptide(s) for cds1004438_1007297 - Putative Class III</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds1007342_1007474</span> leader / core peptide
          </dt>
          <dd>
            MALLDLQAMDTPAEDSFGELAT
            -
            G<span class="dha">Dha</span>QV<span class="dha">Dha</span>LLV<span class="cys">C</span>EY<span class="dha">Dha</span><span class="dha">Dha</span>L<span class="dha">Dha</span>VVL<span class="cys">C</span><span class="dhb">Dhb</span>P
        </dd>
      </dl>
    
    <div class="legend">
      <h4> Legend: </h4>
      <div>
        <span class="dha">Dha</span>: Didehydroalanine<br>
        <span class="dhb">Dhb</span>: Didehydrobutyrine<br>
      </div>
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r6c5-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-143"></div> <span class="help-tooltip" id="cb-general-help-143">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r6c5-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r6c5_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r6c5_1.svg">
        NZ_KZ268499 (5193041-5215810): Streptomyces sp. SS07 Scaffold1, whole genome ... (100% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r6c5_2.svg">
        NZ_CP029361 (2333124-2354642): Streptomyces globisporus strain TFH56 chromoso... (100% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r6c5_3.svg">
        NZ_LIPC01000001 (768970-790488): Streptomyces sp. TSRI0445 scaffold1, whole g... (100% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r6c5_4.svg">
        NZ_CP013738 (5299670-5322418): Streptomyces globisporus C-1027 chromosome, co... (90% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r6c5_5.svg">
        NZ_KZ679082 (5180486-5203174): Streptomyces sp. CS149 scaffold00001, whole ge... (94% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r6c5_6.svg">
        NZ_GL877172 (3014253-3036151): Streptomyces sp. ACT-1 strain XylebKG-1 scaffo... (94% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r6c5_7.svg">
        NZ_KZ819178 (5258700-5280504): Streptomyces sp. CS131 scaffold00001, whole ge... (94% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r6c5_8.svg">
        NZ_WWGD01000029 (732349-755223): Streptomyces sp. SID8378 SID8378.c29, whole ... (94% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r6c5_9.svg">
        NZ_LIXK01000001 (846090-868838): Streptomyces sp. CB02130 scaffold1, whole ge... (94% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r6c5_10.svg">
        NZ_KZ819182 (5435864-5458711): Streptomyces sp. CS147 scaffold00001, whole ge... (94% of genes show similarity), lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r6c5-download">Download graphic</button>
  </div>
  <div id="clusterblast-r6c5-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c5-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-144"></div> <span class="help-tooltip" id="cb-known-help-144">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r6c5-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r6c5_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r6c5_1.svg">
        AmfS
      </option>
      
      <option value = "svg/knownclusterblast_r6c5_2.svg">
        SAL-2242
      </option>
      
      <option value = "svg/knownclusterblast_r6c5_3.svg">
        SRO15-2212
      </option>
      
      <option value = "svg/knownclusterblast_r6c5_4.svg">
        SapB
      </option>
      
    </select>
    <button id="knownclusterblast-r6c5-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r6c5-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c5-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-145"></div> <span class="help-tooltip" id="cb-sub-help-145">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r6c5-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-146"></div> <span class="help-tooltip" id="MIBiG-help-146">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r6c5-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r6c5-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r6c5"></div>
    <div id="comparison-MIBiG-r6c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-iii</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496.1: 2822112-2829457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.88 (id:0.68, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.88), rgb(0, 0, 0, 0.88) 88%, white 88%)"></td>
         
        
        <td>0.68</td>
        <td>RiPP</td>
        
        <td>AmfS</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.68, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>Terpene</td>
        
        <td>platencin</td>
        
        
        <td>Streptomyces platensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.51, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>RiPP</td>
        
        <td>SapB</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.48, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>RiPP</td>
        
        <td>catenulipeptin</td>
        
        
        <td>Catenulispora acidiphila DSM 44928</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.48, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>RiPP</td>
        
        <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
        
        
        <td>Actinomadura namibiensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000553.1: 5221415-5228789">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000553/index.html#r1c1" target="_new">BGC0000553.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.74, order:0.42, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>RiPP</td>
        
        <td>SRO15-2212</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.47, order:0.49, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>RiPP</td>
        
        <td>informatipeptin</td>
        
        
        <td>Streptomyces viridochromogenes DSM 40736</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000546.1: 4112716-4120692">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000546/index.html#r1c1" target="_new">BGC0000546.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.52, order:0.48, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>RiPP</td>
        
        <td>SAL-2242</td>
        
        
        <td>Streptomyces albidoflavus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500.1: 0-8863">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>carnolysin A1, carnolysin A2</td>
        
        
        <td>Carnobacterium maltaromaticum</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r6c5-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496.1: 2822112-2829457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.88 (id:0.68, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.88), rgb(0, 0, 0, 0.88) 88%, white 88%)"></td>
      <td>0.88</td>
      <td>RiPP</td>
      <td>AmfS</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.68, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Terpene</td>
      <td>platencin</td>
      <td>Streptomyces platensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.51, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>RiPP</td>
      <td>SapB</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.48, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>RiPP</td>
      <td>catenulipeptin</td>
      <td>Catenulispora acidiphila DSM 44928</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.48, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>RiPP</td>
      <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
      <td>Actinomadura namibiensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000553.1: 5221415-5228789">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000553/index.html#r1c1" target="_new">BGC0000553.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.74, order:0.42, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>RiPP</td>
      <td>SRO15-2212</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.47, order:0.49, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>RiPP</td>
      <td>informatipeptin</td>
      <td>Streptomyces viridochromogenes DSM 40736</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000546.1: 4112716-4120692">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000546/index.html#r1c1" target="_new">BGC0000546.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.52, order:0.48, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>SAL-2242</td>
      <td>Streptomyces albidoflavus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500.1: 0-8863">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>RiPP</td>
      <td>carnolysin A1, carnolysin A2</td>
      <td>Carnobacterium maltaromaticum</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r6c5-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-147"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-147">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c5-pfam-details-svg" data-filename="c00006_CC216C_.._r6c5_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r6c5-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r6c5-lanthipeptides" data-name="r6c5-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r6c5-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-142"></div> <span class="help-tooltip" id="lanthi-side-help-142">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

    <dl class ="prediction-text">
    <dt><span class="serif">cds1007342_1007474</span> - Class III</dt>
        <dd>Cleavage pHMM score: 31.80</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 2075.1 Da</dd>
        <dd>Molecular weight: 2076.4 Da</dd>
        <dd>Number of bridges: 2</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 2094.4 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2112.4 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2130.5 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2148.5 Da</span>
                    <span class="alt-weight-n"> N = 4</span>
                  </dd>
                
            </dl>
          </dd>
        
  </dl>

</div>

    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r6c6' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00006_CC216C_.. - Region 6 - NRPS,NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-283"></div> <span class="help-tooltip" id="region_svg-help-283">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00006_CC216C_...region006.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r6c6-svg" data-filename="c00006_CC216C_.._r6c6.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1,332,794 - 1,385,942 nt. (total: 53,149 nt)
        
        <a class="cluster-rules-header" id="r6c6-rules-header" href="#r6c6">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r6c6-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r6c6-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-284"></div> <span class="help-tooltip" id="focus-panel-help-284">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r6c6">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r6c6-nrps_pks" data-name="r6c6-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r6c6-clusterblast" data-name="r6c6-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r6c6-knownclusterblast" data-name="r6c6-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r6c6-subclusterblast" data-name="r6c6-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r6c6-MIBiG-cluster-compare" data-name="r6c6-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r6c6-pfam-details" data-name="r6c6-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r6c6-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-148"></div> <span class="help-tooltip" id="nrps-domain-help-148">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c6-details-svg" data-filename="c00006_CC216C_.._r6c6_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r6c6-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r6c6-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-151"></div> <span class="help-tooltip" id="cb-general-help-151">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r6c6-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r6c6_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r6c6_1.svg">
        NZ_JNLT01000010 (178236-234792): Streptomyces sp. CNS654 CD02DRAFT scaffold00... (78% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r6c6_2.svg">
        NZ_CP020563 (7532286-7716764): Kitasatospora albolonga strain YIM 101047 chro... (19% of genes show similarity), NRPS,T1PKS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r6c6_3.svg">
        NZ_NNBN01000011 (51599-163650): Streptomyces sp. CB02613 scaffold11, whole ge... (21% of genes show similarity), NRPS,PKS-like,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r6c6_4.svg">
        NZ_CP058693 (1251477-1328858): Streptomyces sp. Rer75 chromosome (26% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r6c6_5.svg">
        NZ_CP015726 (9949648-10027267): Streptomyces sp. RTd22 chromosome, complete g... (25% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r6c6_6.svg">
        NZ_CP054916 (1294116-1371329): Streptomyces sp. NA02950 chromosome, complete ... (26% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r6c6_7.svg">
        NZ_CP023697 (867224-937798): Streptomyces prasinus strain ATCC 13879 chromoso... (20% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r6c6_8.svg">
        NZ_CP045480 (5203067-5314623): Amycolatopsis sp. YIM 10 chromosome, complete ... (12% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c6_9.svg">
        NZ_CP045178 (7670134-7762069): Streptomyces tsukubensis strain AT3 chromosome... (12% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r6c6_10.svg">
        NZ_CP054939 (1334801-1412603): Streptomyces sp. NA02536 chromosome, complete ... (15% of genes show similarity), NRPS,other
      </option>
      
    </select>
    <button id= "clusterblast-r6c6-download">Download graphic</button>
  </div>
  <div id="clusterblast-r6c6-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c6-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-152"></div> <span class="help-tooltip" id="cb-known-help-152">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r6c6-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r6c6_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r6c6_1.svg">
        friulimicin A / friulimicin B / friulimicin C / friulimicin D
      </option>
      
      <option value = "svg/knownclusterblast_r6c6_2.svg">
        hormaomycin / hormaomycin A1 / hormaomycin A2 / hormaomycin A3 / hormaomycin A4 / hormaomycin A5 / hormaomycin A6
      </option>
      
      <option value = "svg/knownclusterblast_r6c6_3.svg">
        CDA1b / CDA2a / CDA2b / CDA3a / CDA3b / CDA4a / CDA4b
      </option>
      
      <option value = "svg/knownclusterblast_r6c6_4.svg">
        pepticinnamin E
      </option>
      
      <option value = "svg/knownclusterblast_r6c6_5.svg">
        glycinocin A
      </option>
      
      <option value = "svg/knownclusterblast_r6c6_6.svg">
        actinomycin D
      </option>
      
      <option value = "svg/knownclusterblast_r6c6_7.svg">
        griselimycin
      </option>
      
      <option value = "svg/knownclusterblast_r6c6_8.svg">
        limazepine C / limazepine D / limazepine E / limazepine F / limazepine A
      </option>
      
      <option value = "svg/knownclusterblast_r6c6_9.svg">
        atratumycin
      </option>
      
      <option value = "svg/knownclusterblast_r6c6_10.svg">
        telomycin
      </option>
      
    </select>
    <button id="knownclusterblast-r6c6-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r6c6-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c6-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-153"></div> <span class="help-tooltip" id="cb-sub-help-153">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r6c6-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-154"></div> <span class="help-tooltip" id="MIBiG-help-154">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r6c6-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r6c6-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r6c6"></div>
    <div id="comparison-MIBiG-r6c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.45, order:1.00, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.45, order:1.00, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.71</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.45, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.45, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.71</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.68</td>
        <td>NRP</td>
        
        <td>bacillibactin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.67</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401.1: 0-12882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.49, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.49, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.67</td>
        <td>NRP</td>
        
        <td>paenibactin</td>
        
        
        <td>Paenibacillus elgii B69</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.66</td>
        <td>NRP</td>
        
        <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.66</td>
        <td>NRP</td>
        
        <td>hexose-palythine-serine, hexose-shinorine</td>
        
        
        <td>Heteroscytonema crispum UCFS10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.65</td>
        <td>NRP, Alkaloid</td>
        
        <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.63</td>
        <td>NRP</td>
        
        <td>vicibactin</td>
        
        
        <td>Rhizobium etli CFN 42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.63</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r6c6-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.45, order:1.00, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.45, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401.1: 0-12882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.49, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>paenibactin</td>
      <td>Paenibacillus elgii B69</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>hexose-palythine-serine, hexose-shinorine</td>
      <td>Heteroscytonema crispum UCFS10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>rhodochelin</td>
      <td>Rhodococcus jostii RHA1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP, Alkaloid</td>
      <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r6c6-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-155"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-155">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c6-pfam-details-svg" data-filename="c00006_CC216C_.._r6c6_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r6c6-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r6c6-nrps_pks_products" data-name="r6c6-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r6c6-nrps_pks" data-name="r6c6-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r6c6-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-149"></div> <span class="help-tooltip" id="nrps-structures-help-149">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 6, location 1332793 - 1385942:
          <div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r6c6-6" class="smiles-canvas" data-smiles="NC([*])C(=O)N1C(CCC1)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X - pro) + (X) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,pro&amp;nrps2=x&amp;nrps3=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,pro*&amp;nrps2=x&amp;nrps3=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r6c6-6-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)N1C(CCC1)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 7, location 1332793 - 1373048:
          <div class="collapser collapser-target-CC7 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r6c6-7" class="smiles-canvas" data-smiles="NC([*])C(=O)N1C(CCC1)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X - pro) + (X) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,pro&amp;nrps2=x&amp;nrps3=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,pro*&amp;nrps2=x&amp;nrps3=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r6c6-7-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)N1C(CCC1)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 8, location 1336332 - 1385942:
          <div class="collapser collapser-target-CC8 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r6c6-8" class="smiles-canvas" data-smiles="NC([*])C(=O)N1C(CCC1)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X - pro) + (X) + (X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,pro&amp;nrps2=x&amp;nrps3=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,pro*&amp;nrps2=x&amp;nrps3=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r6c6-8-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)N1C(CCC1)C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r6c6-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-150"></div> <span class="help-tooltip" id="nrps-monomers-help-150">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds1352612_1354427</span></strong>: X
      <div class="collapser collapser-target-cds1352612_1354427 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (12..416)</strong>: X
          <div class="collapser collapser-target-cds1352612_1354427 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-cds1352612_1354427-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DarHaSLmtK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds1356332_1360244</span></strong>: X
      <div class="collapser collapser-target-cds1356332_1360244 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (472..880)</strong>: X
          <div class="collapser collapser-target-cds1356332_1360244 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asp, asn
              <div class="collapser collapser-target-cds1356332_1360244-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>asp, asn</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLseLGsVNK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds1360308_1365942</span></strong>: X - pro
      <div class="collapser collapser-target-cds1360308_1365942 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,pro" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,pro*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (207..606)</strong>: X
          <div class="collapser collapser-target-cds1360308_1365942 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds1360308_1365942-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DALhtGAVeK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1267..1683)</strong>: pro
          <div class="collapser collapser-target-cds1360308_1365942 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  pro
              <div class="collapser collapser-target-cds1360308_1365942-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>pro, pip</dd>
   <dt>Small clusters prediction:</dt>
   <dd>pro</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWcsgLlvK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r8c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00008_CC216C_.. - Region 1 - lanthipeptide-class-i <div class="help-container"> <div class="help-icon" data-id="region_svg-help-285"></div> <span class="help-tooltip" id="region_svg-help-285">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00008_CC216C_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r8c1-svg" data-filename="c00008_CC216C_.._r8c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 30,746 - 55,985 nt. (total: 25,240 nt)
        
        <a class="cluster-rules-header" id="r8c1-rules-header" href="#r8c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r8c1-rules">lanthipeptide-class-i: ((LANC_like and (Lant_dehydr_N or Lant_dehydr_C)) and not (YcaO or TIGR03882))</div>
      
      <div class="region-svg-container">
       <div id='r8c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-286"></div> <span class="help-tooltip" id="focus-panel-help-286">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r8c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r8c1-lanthipeptides" data-name="r8c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r8c1-clusterblast" data-name="r8c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r8c1-knownclusterblast" data-name="r8c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r8c1-subclusterblast" data-name="r8c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r8c1-MIBiG-cluster-compare" data-name="r8c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r8c1-pfam-details" data-name="r8c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r8c1-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-156"></div> <span class="help-tooltip" id="lanthi-body-help-156">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lanthipeptide(s) for cds40745_41765 - Putative Class I</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds46122_46320</span> leader / core peptide
          </dt>
          <dd>
            MTSATLAPPPVAPAADLEDDFAPLDVKVVIAEHSYGHLMCS
            -
            <span class="dhb">Dhb</span>GDG<span class="cys">C</span>G<span class="dhb">Dhb</span><span class="dhb">Dhb</span><span class="cys">C</span>A<span class="dhb">Dhb</span>GA<span class="dha">Dha</span>A<span class="cys">C</span>G<span class="dha">Dha</span>F<span class="dhb">Dhb</span>EDPA
        </dd>
      </dl>
    
      <h4>Lanthipeptide(s) for cds43102_45985 - Putative Class I</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds46122_46320</span> leader / core peptide
          </dt>
          <dd>
            MTSATLAPPPVAPAADLEDDFAPLDVKVVIAEHSYGHLMCS
            -
            <span class="dhb">Dhb</span>GDG<span class="cys">C</span>G<span class="dhb">Dhb</span><span class="dhb">Dhb</span><span class="cys">C</span>A<span class="dhb">Dhb</span>GA<span class="dha">Dha</span>A<span class="cys">C</span>G<span class="dha">Dha</span>F<span class="dhb">Dhb</span>EDPA
        </dd>
      </dl>
    
    <div class="legend">
      <h4> Legend: </h4>
      <div>
        <span class="dha">Dha</span>: Didehydroalanine<br>
        <span class="dhb">Dhb</span>: Didehydrobutyrine<br>
      </div>
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r8c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-158"></div> <span class="help-tooltip" id="cb-general-help-158">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r8c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r8c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r8c1_1.svg">
        NZ_CP026490 (3849234-3872768): Streptomyces sp. 604F chromosome, complete genome (75% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c1_2.svg">
        NZ_GG770539 (2962654-2987941): Streptomyces sp. SPB074 supercont3.1, whole ge... (61% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c1_3.svg">
        NZ_KN549170 (32305-57592): Streptomyces sp. M10 Scaffold25, whole genome shot... (44% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c1_4.svg">
        NZ_CP020563 (3910392-3935661): Kitasatospora albolonga strain YIM 101047 chro... (45% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c1_5.svg">
        NZ_WWGD01000001 (303046-329375): Streptomyces sp. SID8378 SID8378.c1, whole g... (43% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c1_6.svg">
        NZ_KZ819178 (6298999-6325328): Streptomyces sp. CS131 scaffold00001, whole ge... (43% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c1_7.svg">
        NZ_CP026490 (1067380-1089241): Streptomyces sp. 604F chromosome, complete genome (31% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c1_8.svg">
        NZ_LIYH01000005 (250064-275342): Streptomyces sp. CB03234 scaffold5, whole ge... (30% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c1_9.svg">
        NZ_CP031264 (1914215-1937335): Streptacidiphilus bronchialis strain DSM 10643... (36% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c1_10.svg">
        NZ_CP054916 (6577247-6639030): Streptomyces sp. NA02950 chromosome, complete ... (14% of genes show similarity), lanthipeptide,other
      </option>
      
    </select>
    <button id= "clusterblast-r8c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r8c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r8c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-159"></div> <span class="help-tooltip" id="cb-known-help-159">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r8c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r8c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r8c1_1.svg">
        A54145
      </option>
      
      <option value = "svg/knownclusterblast_r8c1_2.svg">
        kanamycin
      </option>
      
      <option value = "svg/knownclusterblast_r8c1_3.svg">
        arsono-polyketide
      </option>
      
    </select>
    <button id="knownclusterblast-r8c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r8c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r8c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-160"></div> <span class="help-tooltip" id="cb-sub-help-160">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r8c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-162"></div> <span class="help-tooltip" id="MIBiG-help-162">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r8c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r8c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r8c1"></div>
    <div id="comparison-MIBiG-r8c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-i</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001289.1: 16-10643">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001289/index.html#r1c1" target="_new">BGC0001289.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>RiPP</td>
        
        <td>penisin</td>
        
        
        <td>Paenibacillus ehimensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001707.1: 0-27246">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001707/index.html#r1c1" target="_new">BGC0001707.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.69, order:N/A, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>nosiheptide</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000542.1: 0-9130">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000542/index.html#r1c1" target="_new">BGC0000542.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>paenicidin B</td>
        
        
        <td>Paenibacillus terrae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001727.1: 1552553-1565755">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001727/index.html#r1c1" target="_new">BGC0001727.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.41, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>RiPP</td>
        
        <td>paenilan</td>
        
        
        <td>Paenibacillus polymyxa E681</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001972.1: 267-32147">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001972/index.html#r1c1" target="_new">BGC0001972.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>toxoflavin, fervenulin</td>
        
        
        <td>Streptomyces hiroshimensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001973.1: 267-32147">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001973/index.html#r1c1" target="_new">BGC0001973.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>fervenulin</td>
        
        
        <td>Streptomyces hiroshimensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000514.1: 578-12691">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000514/index.html#r1c1" target="_new">BGC0000514.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.41, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>gallidermin</td>
        
        
        <td>Staphylococcus gallinarum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000529.1: 1655-23062">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000529/index.html#r1c1" target="_new">BGC0000529.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>RiPP</td>
        
        <td>microbisporicin A2</td>
        
        
        <td>Microbispora corallina</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.52, order:0.36, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000796.1: 0-27180">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000796/index.html#r1c1" target="_new">BGC0000796.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
        <td>0.11</td>
        <td>Saccharide</td>
        
        <td>S-layer glycan</td>
        
        
        <td>Paenibacillus alvei</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r8c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001289.1: 16-10643">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001289/index.html#r1c1" target="_new">BGC0001289.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>RiPP</td>
      <td>penisin</td>
      <td>Paenibacillus ehimensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001707.1: 0-27246">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001707/index.html#r1c1" target="_new">BGC0001707.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.69, order:N/A, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>nosiheptide</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000542.1: 0-9130">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000542/index.html#r1c1" target="_new">BGC0000542.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>paenicidin B</td>
      <td>Paenibacillus terrae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001727.1: 1552553-1565755">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001727/index.html#r1c1" target="_new">BGC0001727.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.41, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>RiPP</td>
      <td>paenilan</td>
      <td>Paenibacillus polymyxa E681</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001972.1: 267-32147">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001972/index.html#r1c1" target="_new">BGC0001972.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Other</td>
      <td>toxoflavin, fervenulin</td>
      <td>Streptomyces hiroshimensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001973.1: 267-32147">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001973/index.html#r1c1" target="_new">BGC0001973.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Other</td>
      <td>fervenulin</td>
      <td>Streptomyces hiroshimensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000514.1: 578-12691">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000514/index.html#r1c1" target="_new">BGC0000514.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.41, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>gallidermin</td>
      <td>Staphylococcus gallinarum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000529.1: 1655-23062">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000529/index.html#r1c1" target="_new">BGC0000529.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>RiPP</td>
      <td>microbisporicin A2</td>
      <td>Microbispora corallina</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.52, order:0.36, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000796.1: 0-27180">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000796/index.html#r1c1" target="_new">BGC0000796.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
      <td>0.38</td>
      <td>Saccharide</td>
      <td>S-layer glycan</td>
      <td>Paenibacillus alvei</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r8c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-163"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-163">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r8c1-pfam-details-svg" data-filename="c00008_CC216C_.._r8c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r8c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r8c1-lanthipeptides" data-name="r8c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
      <div class="sidepanel-details-header r8c1-RREfinder" data-name="r8c1-RREfinder">
        RREFinder
      </div>
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r8c1-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-157"></div> <span class="help-tooltip" id="lanthi-side-help-157">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

    <dl class ="prediction-text">
    <dt><span class="serif">cds46122_46320</span> - Class I</dt>
        <dd>Cleavage pHMM score: -12.30</dd>
        <dd>RODEO score: 17</dd>
        <dd>Monoisotopic mass: 2052.8 Da</dd>
        <dd>Molecular weight: 2054.1 Da</dd>
        <dd>Number of bridges: 3</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 2072.1 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2090.2 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2108.2 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2126.2 Da</span>
                    <span class="alt-weight-n"> N = 4</span>
                  </dd>
                
            </dl>
          </dd>
        <dt><span class="serif">cds46122_46320</span> - Class I</dt>
        <dd>Cleavage pHMM score: -12.30</dd>
        <dd>RODEO score: 17</dd>
        <dd>Monoisotopic mass: 2052.8 Da</dd>
        <dd>Molecular weight: 2054.1 Da</dd>
        <dd>Number of bridges: 3</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 2072.1 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2090.2 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2108.2 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2126.2 Da</span>
                    <span class="alt-weight-n"> N = 4</span>
                  </dd>
                
            </dl>
          </dd>
        
  </dl>

</div>

    </div>
     
    
     
    
     
    <div class="sidepanel-details-section r8c1-RREfinder">
      <div class="more-details">
  <div class="heading">
    <span>RRE predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="rre-side-help-161"></div> <span class="help-tooltip" id="rre-side-help-161">RREfinder results sidepanel.</span></div>
  </div>
  <dl class = 'prediction-text'>
    
      <span><strong>lanthipeptide-class-i protocluster</strong> (30745...55985)</span>
      <dt></dt>
      
      
      
        
        <strong>cds43102_45985</strong> <div class="collapser collapser-target-cds43102_45985 collapser-level-cds"> </div><div class="collapser-content">
        
          <dd>RRE type: Lanthipeptide_LanB_RRE</dd>
          <dd>Location: 141 - 233</dd>
          <dd>Score: 58.6</dd>
        
        </div><br>
      <br>
    
    </dl>
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r8c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00008_CC216C_.. - Region 2 - butyrolactone,ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-287"></div> <span class="help-tooltip" id="region_svg-help-287">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00008_CC216C_...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r8c2-svg" data-filename="c00008_CC216C_.._r8c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 216,955 - 232,313 nt. (total: 15,359 nt)
        
        <a class="cluster-rules-header" id="r8c2-rules-header" href="#r8c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r8c2-rules">ectoine: ectoine_synt<br>butyrolactone: AfsA</div>
      
      <div class="region-svg-container">
       <div id='r8c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-288"></div> <span class="help-tooltip" id="focus-panel-help-288">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r8c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r8c2-clusterblast" data-name="r8c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r8c2-knownclusterblast" data-name="r8c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r8c2-subclusterblast" data-name="r8c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r8c2-MIBiG-cluster-compare" data-name="r8c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r8c2-pfam-details" data-name="r8c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r8c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-164"></div> <span class="help-tooltip" id="cb-general-help-164">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r8c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r8c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r8c2_1.svg">
        NZ_JNLT01000014 (95271-110611): Streptomyces sp. CNS654 CD02DRAFT scaffold000... (94% of genes show similarity), butyrolactone,ectoine
      </option>
      
      <option value = "svg/clusterblast_r8c2_2.svg">
        NZ_KZ268499 (4152321-4166536): Streptomyces sp. SS07 Scaffold1, whole genome ... (94% of genes show similarity), butyrolactone,ectoine
      </option>
      
      <option value = "svg/clusterblast_r8c2_3.svg">
        NZ_BDHJ01000011 (63069-78417): Streptomyces sp. NBRC 110465, whole genome sho... (94% of genes show similarity), butyrolactone,ectoine
      </option>
      
      <option value = "svg/clusterblast_r8c2_4.svg">
        NZ_CP021707 (4077726-4092939): Streptomyces sp. S063 chromosome, complete genome (94% of genes show similarity), butyrolactone,ectoine
      </option>
      
      <option value = "svg/clusterblast_r8c2_5.svg">
        NZ_WWGS01000004 (507659-522991): Streptomyces sp. SID8359 SID8359.c4, whole g... (94% of genes show similarity), butyrolactone,ectoine
      </option>
      
      <option value = "svg/clusterblast_r8c2_6.svg">
        NZ_CP029378 (4296336-&gt;4310564): Streptomyces bacillaris strain ATCC 15855 chr... (94% of genes show similarity), butyrolactone,ectoine
      </option>
      
      <option value = "svg/clusterblast_r8c2_7.svg">
        NZ_CP011522 (4683522-&gt;4697870): Streptomyces sp. CFMR 7 strain CFMR-7 chromos... (94% of genes show similarity), butyrolactone,ectoine
      </option>
      
      <option value = "svg/clusterblast_r8c2_8.svg">
        NZ_CP024957 (3393852-3409171): Streptomyces cavourensis strain 1AS2a chromoso... (94% of genes show similarity), butyrolactone,ectoine
      </option>
      
      <option value = "svg/clusterblast_r8c2_9.svg">
        NZ_CP030930 (3420270-3435589): Streptomyces cavourensis strain TJ430 chromoso... (94% of genes show similarity), butyrolactone,ectoine
      </option>
      
      <option value = "svg/clusterblast_r8c2_10.svg">
        NZ_VEHK01000002 (707620-722949): Streptomyces sp. CFMR 7 strain DE0061 NODE 2... (94% of genes show similarity), butyrolactone,ectoine
      </option>
      
    </select>
    <button id= "clusterblast-r8c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r8c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r8c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-165"></div> <span class="help-tooltip" id="cb-known-help-165">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r8c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r8c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r8c2_1.svg">
        showdomycin
      </option>
      
      <option value = "svg/knownclusterblast_r8c2_2.svg">
        RP-1776
      </option>
      
      <option value = "svg/knownclusterblast_r8c2_3.svg">
        lactonamycin
      </option>
      
      <option value = "svg/knownclusterblast_r8c2_4.svg">
        kosinostatin
      </option>
      
    </select>
    <button id="knownclusterblast-r8c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r8c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r8c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-166"></div> <span class="help-tooltip" id="cb-sub-help-166">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r8c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-167"></div> <span class="help-tooltip" id="MIBiG-help-167">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r8c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r8c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r8c2"></div>
    <div id="comparison-MIBiG-r8c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">ectoine</span></th>
         
         <th><span class="cc-heat-product">butyrolactone</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000849.1: 6890527-6892237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000849/index.html#r1c1" target="_new">BGC0000849.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:0.49, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.78 (id:0.47, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.78), rgb(0, 0, 0, 0.78) 78%, white 78%)"></td>
         
        
        <td>0.75</td>
        <td>Other,  , (, B, u, t, y, r, o, l, a, c, t, o, n, e, )</td>
        
        <td>SCB1, SCB2, SCB3</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000848.1: 8273443-8274388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000848/index.html#r1c1" target="_new">BGC0000848.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.71</td>
        <td>Other</td>
        
        <td>A-factor</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.54, order:0.63, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.53, order:0.41, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.56</td>
        <td>Other</td>
        
        <td>showdomycin</td>
        
        
        <td>Streptomyces showdoensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.45, order:0.34, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.49</td>
        <td>Other</td>
        
        <td>γ-butyrolactone</td>
        
        
        <td>Streptomyces acidiscabies</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000003.1: 12401-36790">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000003/index.html#r1c1" target="_new">BGC0000003.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.47</td>
        <td>Polyketide</td>
        
        <td>AF-toxin</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000890.1: 0-11090">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000890/index.html#r1c1" target="_new">BGC0000890.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.37</td>
        <td>Other,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
        
        <td>pentabromopseudilin</td>
        
        
        <td>Pseudoalteromonas phenolica O-BC30</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000891.1: 32482-43745">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000891/index.html#r1c1" target="_new">BGC0000891.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.36</td>
        <td>Other,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
        
        <td>pentabromopseudilin</td>
        
        
        <td>Pseudoalteromonas luteoviolacea 2ta16</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001465.1: 4521274-4533494">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001465/index.html#r1c1" target="_new">BGC0001465.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.36</td>
        <td>Other</td>
        
        <td>bromopyrroles/bromophenols</td>
        
        
        <td>Marinomonas mediterranea MMB-1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076.1: 161645-164819">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Other</td>
        
        <td>4,5-Dehydroacaterin, Acaterin</td>
        
        
        <td>Pseudomonas jessenii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000131.1: 0-6280">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000131/index.html#r1c1" target="_new">BGC0000131.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.32</td>
        <td>Polyketide</td>
        
        <td>pyrrolomycin</td>
        
        
        <td>Streptomyces sp. UC 11065</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r8c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000849.1: 6890527-6892237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000849/index.html#r1c1" target="_new">BGC0000849.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.78 (id:0.47, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.78), rgb(0, 0, 0, 0.78) 78%, white 78%)"></td>
      <td>0.78</td>
      <td>Other,  , (, B, u, t, y, r, o, l, a, c, t, o, n, e, )</td>
      <td>SCB1, SCB2, SCB3</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000848.1: 8273443-8274388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000848/index.html#r1c1" target="_new">BGC0000848.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Other</td>
      <td>A-factor</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076.1: 161645-164819">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Other</td>
      <td>4,5-Dehydroacaterin, Acaterin</td>
      <td>Pseudomonas jessenii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.53, order:0.46, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>showdomycin</td>
      <td>Streptomyces showdoensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000003.1: 12401-36790">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000003/index.html#r1c1" target="_new">BGC0000003.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>AF-toxin</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:0.43, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>γ-butyrolactone</td>
      <td>Streptomyces acidiscabies</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>griseusin</td>
      <td>Streptomyces griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>jadomycin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>granaticin</td>
      <td>Streptomyces violaceoruber</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000204.1: 4242-15189">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000204/index.html#r1c1" target="_new">BGC0000204.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>benastatin A, benastatin B, BE-43767, TAN1532B</td>
      <td>Streptomyces sp. A2991200</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r8c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-168"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-168">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r8c2-pfam-details-svg" data-filename="c00008_CC216C_.._r8c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r8c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r8c3' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00008_CC216C_.. - Region 3 - lanthipeptide-class-ii <div class="help-container"> <div class="help-icon" data-id="region_svg-help-289"></div> <span class="help-tooltip" id="region_svg-help-289">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00008_CC216C_...region003.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r8c3-svg" data-filename="c00008_CC216C_.._r8c3.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 313,541 - 339,863 nt. (total: 26,323 nt)
        
        <a class="cluster-rules-header" id="r8c3-rules-header" href="#r8c3">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r8c3-rules">lanthipeptide-class-ii: cds(LANC_like and DUF4135)</div>
      
      <div class="region-svg-container">
       <div id='r8c3-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-290"></div> <span class="help-tooltip" id="focus-panel-help-290">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r8c3">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r8c3-lanthipeptides" data-name="r8c3-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r8c3-clusterblast" data-name="r8c3-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r8c3-knownclusterblast" data-name="r8c3-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r8c3-subclusterblast" data-name="r8c3-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r8c3-MIBiG-cluster-compare" data-name="r8c3-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r8c3-pfam-details" data-name="r8c3-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r8c3-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-169"></div> <span class="help-tooltip" id="lanthi-body-help-169">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lanthipeptide(s) for cds323540_326600 - Putative Class II</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds323282_323468</span> leader / core peptide
          </dt>
          <dd>
            MANPNVGILEEIDDQNLDAVSAA
            -
            GQGWG<span class="dhb">Dhb</span>IV<span class="dha">Dha</span><span class="dha">Dha</span>L<span class="dha">Dha</span><span class="cys">C</span>YGA<span class="dha">Dha</span>YVLGNNGNF<span class="cys">C</span><span class="dhb">Dhb</span>A<span class="dhb">Dhb</span>AE<span class="cys">C</span>QNN<span class="cys">C</span>K
          <dt>
            <span class="serif">cds326699_326900</span> leader / core peptide
          </dt>
          <dd>
            MGDIAMNDATGYVSADELVELADA
            -
            PD<span class="dha">Dha</span>LAAGF<span class="dhb">Dhb</span>P<span class="dhb">Dhb</span>IIPA<span class="dhb">Dhb</span>AV<span class="dhb">Dhb</span><span class="dhb">Dhb</span>VIVGA<span class="dha">Dha</span>AAMQGG<span class="cys">C</span>P<span class="dhb">Dhb</span><span class="dha">Dha</span>G<span class="cys">C</span><span class="dhb">Dhb</span>KR<span class="cys">C</span>
        </dd>
      </dl>
    
      <h4>Lanthipeptide(s) for cds326998_329863 - Putative Class II</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds323282_323468</span> leader / core peptide
          </dt>
          <dd>
            MANPNVGILEEIDDQNLDAVSAA
            -
            GQGWG<span class="dhb">Dhb</span>IV<span class="dha">Dha</span><span class="dha">Dha</span>L<span class="dha">Dha</span><span class="cys">C</span>YGA<span class="dha">Dha</span>YVLGNNGNF<span class="cys">C</span><span class="dhb">Dhb</span>A<span class="dhb">Dhb</span>AE<span class="cys">C</span>QNN<span class="cys">C</span>K
          <dt>
            <span class="serif">cds326699_326900</span> leader / core peptide
          </dt>
          <dd>
            MGDIAMNDATGYVSADELVELADA
            -
            PD<span class="dha">Dha</span>LAAGF<span class="dhb">Dhb</span>P<span class="dhb">Dhb</span>IIPA<span class="dhb">Dhb</span>AV<span class="dhb">Dhb</span><span class="dhb">Dhb</span>VIVGA<span class="dha">Dha</span>AAMQGG<span class="cys">C</span>P<span class="dhb">Dhb</span><span class="dha">Dha</span>G<span class="cys">C</span><span class="dhb">Dhb</span>KR<span class="cys">C</span>
          <dt>
            <span class="serif">allorf_0337031_0337210</span> leader / core peptide
          </dt>
          <dd>
            MVRSSYFQWADGVWGHSGRNGFLGPTFGG
            -
            APPEAAHGPWR<span class="dhb">Dhb</span><span class="dha">Dha</span>PG<span class="cys">C</span>VPPDGV<span class="dha">Dha</span><span class="dha">Dha</span>RFRHG<span class="dhb">Dhb</span>
          <dt>
            <span class="serif">allorf_0337045_0337206</span> leader / core peptide
          </dt>
          <dd>
            VPWRKRLDTPSGG
            -
            <span class="dhb">Dhb</span>QPGLVRHGP<span class="cys">C</span>AA<span class="dha">Dha</span>GGAPPKVGPRNPFLPE<span class="cys">C</span>PH<span class="dhb">Dhb</span>P<span class="dha">Dha</span>AHWK
        </dd>
      </dl>
    
    <div class="legend">
      <h4> Legend: </h4>
      <div>
        <span class="dha">Dha</span>: Didehydroalanine<br>
        <span class="dhb">Dhb</span>: Didehydrobutyrine<br>
      </div>
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r8c3-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-171"></div> <span class="help-tooltip" id="cb-general-help-171">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r8c3-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r8c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r8c3_1.svg">
        NZ_JNZV01000231 (16023-42335): Streptomyces baarnensis strain NRRL B-2842 con... (80% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c3_2.svg">
        NZ_KZ819178 (4058090-4083812): Streptomyces sp. CS131 scaffold00001, whole ge... (79% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c3_3.svg">
        NZ_CP021707 (3944934-3971252): Streptomyces sp. S063 chromosome, complete genome (76% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c3_4.svg">
        NZ_NNBN01000001 (764166-790489): Streptomyces sp. CB02613 scaffold1, whole ge... (64% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c3_5.svg">
        NZ_JNLT01000014 (175260-201589): Streptomyces sp. CNS654 CD02DRAFT scaffold00... (64% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c3_6.svg">
        NZ_LZRD01000017 (79660-105953): Streptomyces sp. PTY087I2 A3Q37 contig000017,... (75% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c3_7.svg">
        NZ_GG657754 (6524338-6581074): Streptomyces himastatinicus ATCC 53653 superco... (9% of genes show similarity), arylpolyene,ladderane,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c3_8.svg">
        NZ_VTZW01000015 (179182-205443): Streptomonospora sp. PA3 scaffold7.1, whole ... (21% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c3_9.svg">
        NZ_QKTG01000020 (10179-36392): Curtobacterium sp. MCLR17 036 GRASS36.NODE20, ... (27% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r8c3_10.svg">
        NZ_FQZK01000006 (181352-216911): Nocardiopsis flavescens strain CGMCC 4.5723,... (25% of genes show similarity), lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r8c3-download">Download graphic</button>
  </div>
  <div id="clusterblast-r8c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r8c3-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-172"></div> <span class="help-tooltip" id="cb-known-help-172">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r8c3-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r8c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r8c3_1.svg">
        chalcomycin A
      </option>
      
      <option value = "svg/knownclusterblast_r8c3_2.svg">
        5-isoprenylindole-3-carboxylate β-D-glycosyl ester
      </option>
      
      <option value = "svg/knownclusterblast_r8c3_3.svg">
        guadinomine
      </option>
      
      <option value = "svg/knownclusterblast_r8c3_4.svg">
        CDA1b / CDA2a / CDA2b / CDA3a / CDA3b / CDA4a / CDA4b
      </option>
      
    </select>
    <button id="knownclusterblast-r8c3-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r8c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r8c3-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-173"></div> <span class="help-tooltip" id="cb-sub-help-173">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r8c3-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-174"></div> <span class="help-tooltip" id="MIBiG-help-174">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r8c3-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r8c3-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r8c3"></div>
    <div id="comparison-MIBiG-r8c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-ii</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001579.1: 85465-94303">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001579/index.html#r1c1" target="_new">BGC0001579.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>RiPP</td>
        
        <td>duramycin</td>
        
        
        <td>Streptomyces cinnamoneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000516.1: 8706-19626">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000516/index.html#r1c1" target="_new">BGC0000516.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.33</td>
        <td>RiPP</td>
        
        <td>geobacillin II</td>
        
        
        <td>Geobacillus thermodenitrificans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000552.1: 1388-11054">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000552/index.html#r1c1" target="_new">BGC0000552.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>RiPP</td>
        
        <td>SmbA, SmbB</td>
        
        
        <td>Streptococcus mutans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000555.1: 24100-35850">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000555/index.html#r1c1" target="_new">BGC0000555.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>RiPP</td>
        
        <td>staphylococcin C55 α, staphylococcin C55 β</td>
        
        
        <td>Staphylococcus aureus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000539.1: 155-11740">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000539/index.html#r1c1" target="_new">BGC0000539.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>RiPP</td>
        
        <td>nukacin ISK-1</td>
        
        
        <td>Staphylococcus warneri</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000497.1: 228-10998">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000497/index.html#r1c1" target="_new">BGC0000497.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>RiPP</td>
        
        <td>BHT-Aα, BHT-Aβ</td>
        
        
        <td>Streptococcus ratti</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001787.1: 2408-11843">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001787/index.html#r1c1" target="_new">BGC0001787.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>RiPP</td>
        
        <td>suicin 3908</td>
        
        
        <td>Streptococcus suis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000626.1: 0-7622">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000626/index.html#r1c1" target="_new">BGC0000626.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>RiPP</td>
        
        <td>thuricin</td>
        
        
        <td>Bacillus thuringiensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001364.1: 317877-335461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001364/index.html#r1c1" target="_new">BGC0001364.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>RiPP</td>
        
        <td>cinnamycin B</td>
        
        
        <td>Actinomadura atramentaria DSM 43919</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000547.1: 618-22455">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000547/index.html#r1c1" target="_new">BGC0000547.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>RiPP</td>
        
        <td>salivaricin 9</td>
        
        
        <td>Streptococcus salivarius</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r8c3-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001579.1: 85465-94303">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001579/index.html#r1c1" target="_new">BGC0001579.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>RiPP</td>
      <td>duramycin</td>
      <td>Streptomyces cinnamoneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000516.1: 8706-19626">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000516/index.html#r1c1" target="_new">BGC0000516.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>RiPP</td>
      <td>geobacillin II</td>
      <td>Geobacillus thermodenitrificans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000552.1: 1388-11054">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000552/index.html#r1c1" target="_new">BGC0000552.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>RiPP</td>
      <td>SmbA, SmbB</td>
      <td>Streptococcus mutans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000555.1: 24100-35850">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000555/index.html#r1c1" target="_new">BGC0000555.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>RiPP</td>
      <td>staphylococcin C55 α, staphylococcin C55 β</td>
      <td>Staphylococcus aureus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000539.1: 155-11740">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000539/index.html#r1c1" target="_new">BGC0000539.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>RiPP</td>
      <td>nukacin ISK-1</td>
      <td>Staphylococcus warneri</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000497.1: 228-10998">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000497/index.html#r1c1" target="_new">BGC0000497.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>RiPP</td>
      <td>BHT-Aα, BHT-Aβ</td>
      <td>Streptococcus ratti</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001787.1: 2408-11843">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001787/index.html#r1c1" target="_new">BGC0001787.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>RiPP</td>
      <td>suicin 3908</td>
      <td>Streptococcus suis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000626.1: 0-7622">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000626/index.html#r1c1" target="_new">BGC0000626.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>RiPP</td>
      <td>thuricin</td>
      <td>Bacillus thuringiensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001364.1: 317877-335461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001364/index.html#r1c1" target="_new">BGC0001364.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>RiPP</td>
      <td>cinnamycin B</td>
      <td>Actinomadura atramentaria DSM 43919</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000547.1: 618-22455">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000547/index.html#r1c1" target="_new">BGC0000547.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.48, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>RiPP</td>
      <td>salivaricin 9</td>
      <td>Streptococcus salivarius</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r8c3-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-175"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-175">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r8c3-pfam-details-svg" data-filename="c00008_CC216C_.._r8c3_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r8c3-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r8c3-lanthipeptides" data-name="r8c3-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r8c3-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-170"></div> <span class="help-tooltip" id="lanthi-side-help-170">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

    <dl class ="prediction-text">
    <dt><span class="serif">cds323282_323468</span> - Class II</dt>
        <dd>Cleavage pHMM score: -2.30</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 3790.7 Da</dd>
        <dd>Molecular weight: 3793.1 Da</dd>
        <dd>Number of bridges: 4</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 3811.2 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3829.2 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3847.2 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
            </dl>
          </dd>
        <dt><span class="serif">cds326699_326900</span> - Class II</dt>
        <dd>Cleavage pHMM score: -3.80</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 3841.0 Da</dd>
        <dd>Molecular weight: 3843.4 Da</dd>
        <dd>Number of bridges: 3</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 3861.5 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3879.5 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3897.5 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3915.5 Da</span>
                    <span class="alt-weight-n"> N = 4</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3933.5 Da</span>
                    <span class="alt-weight-n"> N = 5</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3951.6 Da</span>
                    <span class="alt-weight-n"> N = 6</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3969.6 Da</span>
                    <span class="alt-weight-n"> N = 7</span>
                  </dd>
                
            </dl>
          </dd>
        <dt><span class="serif">cds323282_323468</span> - Class II</dt>
        <dd>Cleavage pHMM score: -2.30</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 3790.7 Da</dd>
        <dd>Molecular weight: 3793.1 Da</dd>
        <dd>Number of bridges: 4</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 3811.2 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3829.2 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3847.2 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
            </dl>
          </dd>
        <dt><span class="serif">cds326699_326900</span> - Class II</dt>
        <dd>Cleavage pHMM score: -3.80</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 3841.0 Da</dd>
        <dd>Molecular weight: 3843.4 Da</dd>
        <dd>Number of bridges: 3</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 3861.5 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3879.5 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3897.5 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3915.5 Da</span>
                    <span class="alt-weight-n"> N = 4</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3933.5 Da</span>
                    <span class="alt-weight-n"> N = 5</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3951.6 Da</span>
                    <span class="alt-weight-n"> N = 6</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3969.6 Da</span>
                    <span class="alt-weight-n"> N = 7</span>
                  </dd>
                
            </dl>
          </dd>
        <dt><span class="serif">allorf_0337031_0337210</span> - Class II</dt>
        <dd>Cleavage pHMM score: -5.60</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 3035.5 Da</dd>
        <dd>Molecular weight: 3037.3 Da</dd>
        <dd>Number of bridges: 1</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 3055.3 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3073.4 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3091.4 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 3109.4 Da</span>
                    <span class="alt-weight-n"> N = 4</span>
                  </dd>
                
            </dl>
          </dd>
        <dt><span class="serif">allorf_0337045_0337206</span> - Class II</dt>
        <dd>Cleavage pHMM score: -4.20</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 4079.1 Da</dd>
        <dd>Molecular weight: 4081.6 Da</dd>
        <dd>Number of bridges: 2</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 4099.6 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 4117.7 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
            </dl>
          </dd>
        
  </dl>

</div>

    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r8c4' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00008_CC216C_.. - Region 4 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-291"></div> <span class="help-tooltip" id="region_svg-help-291">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00008_CC216C_...region004.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r8c4-svg" data-filename="c00008_CC216C_.._r8c4.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 790,920 - 876,022 nt. (total: 85,103 nt)
        
        <a class="cluster-rules-header" id="r8c4-rules-header" href="#r8c4">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r8c4-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r8c4-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-292"></div> <span class="help-tooltip" id="focus-panel-help-292">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r8c4">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r8c4-nrps_pks" data-name="r8c4-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r8c4-clusterblast" data-name="r8c4-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r8c4-knownclusterblast" data-name="r8c4-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r8c4-subclusterblast" data-name="r8c4-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r8c4-MIBiG-cluster-compare" data-name="r8c4-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r8c4-pfam-details" data-name="r8c4-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r8c4-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-176"></div> <span class="help-tooltip" id="nrps-domain-help-176">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r8c4-details-svg" data-filename="c00008_CC216C_.._r8c4_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r8c4-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r8c4-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-179"></div> <span class="help-tooltip" id="cb-general-help-179">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r8c4-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r8c4_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r8c4_1.svg">
        NZ_CP029788 (5308791-5405097): Streptomyces actuosus strain ATCC 25421 chromo... (40% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r8c4_2.svg">
        NZ_KL503830 (5710820-5801101): Streptomyces griseorubens strain JSD-1 scaffol... (48% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r8c4_3.svg">
        NZ_KZ195577 (3740723-3837096): Streptomyces sp. CS159 scaffold00001, whole ge... (45% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r8c4_4.svg">
        NZ_CP045547 (3939231-4035772): Streptomyces sp. SYP-A7193 chromosome, complet... (45% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r8c4_5.svg">
        NZ_WXVT01000001 (3963095-4059673): Streptomyces sp. KO7888 A913 AS4 SC1 R3, w... (46% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r8c4_6.svg">
        NZ_WHZR01000001 (3664979-3761558): Streptomyces sp. OM5714 A912 AS4 SC1 R2, w... (39% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r8c4_7.svg">
        NZ_CP050974 (8172268-8267326): Streptomyces sp. RLB1-33 chromosome, complete ... (36% of genes show similarity), NRPS,siderophore
      </option>
      
      <option value = "svg/clusterblast_r8c4_8.svg">
        NZ_CP023698 (1837342-1933027): Streptomyces viridifaciens strain ATCC 11989 c... (33% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r8c4_9.svg">
        NZ_MPLE01000004 (1684525-1779089): Streptomyces viridifaciens strain DSM 4023... (32% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r8c4_10.svg">
        NZ_CP041168 (4922342-5012016): Streptomyces griseorubiginosus strain BTU6 chr... (23% of genes show similarity), NRPS,arylpolyene,ladderane
      </option>
      
    </select>
    <button id= "clusterblast-r8c4-download">Download graphic</button>
  </div>
  <div id="clusterblast-r8c4-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r8c4-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-180"></div> <span class="help-tooltip" id="cb-known-help-180">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r8c4-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r8c4_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r8c4_1.svg">
        sarpeptin A / sarpeptin B
      </option>
      
      <option value = "svg/knownclusterblast_r8c4_2.svg">
        cadaside A / cadaside B
      </option>
      
      <option value = "svg/knownclusterblast_r8c4_3.svg">
        malacidin A / malacidin B
      </option>
      
      <option value = "svg/knownclusterblast_r8c4_4.svg">
        glycinocin A
      </option>
      
      <option value = "svg/knownclusterblast_r8c4_5.svg">
        daptomycin
      </option>
      
      <option value = "svg/knownclusterblast_r8c4_6.svg">
        A54145
      </option>
      
      <option value = "svg/knownclusterblast_r8c4_7.svg">
        friulimicin A / friulimicin B / friulimicin C / friulimicin D
      </option>
      
      <option value = "svg/knownclusterblast_r8c4_8.svg">
        telomycin
      </option>
      
      <option value = "svg/knownclusterblast_r8c4_9.svg">
        CDA1b / CDA2a / CDA2b / CDA3a / CDA3b / CDA4a / CDA4b
      </option>
      
      <option value = "svg/knownclusterblast_r8c4_10.svg">
        taromycin A
      </option>
      
    </select>
    <button id="knownclusterblast-r8c4-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r8c4-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r8c4-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-181"></div> <span class="help-tooltip" id="cb-sub-help-181">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r8c4-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-182"></div> <span class="help-tooltip" id="MIBiG-help-182">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r8c4-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r8c4-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r8c4"></div>
    <div id="comparison-MIBiG-r8c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001984.1: 9394-83565">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001984/index.html#r1c1" target="_new">BGC0001984.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.63, order:0.68, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
        <td>0.42</td>
        <td>NRP</td>
        
        <td>sarpeptin A, sarpeptin B</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.48, order:0.75, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.51, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.37</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050.1: 0-19730">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.45, order:1.00, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.37</td>
        <td>NRP, Polyketide</td>
        
        <td>thalassospiramide A</td>
        
        
        <td>Tistrella bauzanensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.48, order:0.75, components:0.91)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.36</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.36</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001767.1: 18413-48220">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001767/index.html#r1c1" target="_new">BGC0001767.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.51, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.36</td>
        <td>NRP</td>
        
        <td>salinichelins</td>
        
        
        <td>Salinispora pacifica CNY331</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>NRP</td>
        
        <td>vicibactin</td>
        
        
        <td>Rhizobium etli CFN 42</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r8c4-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001984.1: 9394-83565">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001984/index.html#r1c1" target="_new">BGC0001984.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.63, order:0.68, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>NRP</td>
      <td>sarpeptin A, sarpeptin B</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.48, order:0.75, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.51, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050.1: 0-19730">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.45, order:1.00, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP, Polyketide</td>
      <td>thalassospiramide A</td>
      <td>Tistrella bauzanensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.48, order:0.75, components:0.91)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001767.1: 18413-48220">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001767/index.html#r1c1" target="_new">BGC0001767.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.51, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP</td>
      <td>salinichelins</td>
      <td>Salinispora pacifica CNY331</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>vicibactin</td>
      <td>Rhizobium etli CFN 42</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r8c4-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-183"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-183">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r8c4-pfam-details-svg" data-filename="c00008_CC216C_.._r8c4_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r8c4-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r8c4-nrps_pks_products" data-name="r8c4-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r8c4-nrps_pks" data-name="r8c4-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r8c4-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-177"></div> <span class="help-tooltip" id="nrps-structures-help-177">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 6, location 790919 - 876022:
          <div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r8c4-6" class="smiles-canvas" data-smiles="NCC(=O)NC(CC(=O)O)C(=O)NC(Cc1ccc(O)cc1)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CC(=O)O)C(=O)NCC(=O)N1C(CCC1)C(=O)NCC(=O)NC(C)C(=O)NCC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(gly) + (asp) + (tyr) + (thr - D-X) + (asp - gly - pro) + (gly - ala - gly)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly&amp;nrps2=asp&amp;nrps3=tyr&amp;nrps4=thr,d-x&amp;nrps5=asp,gly,pro&amp;nrps6=gly,ala,gly" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*&amp;nrps2=asp*&amp;nrps3=tyr*&amp;nrps4=thr*,d-x*&amp;nrps5=asp*,gly*,pro*&amp;nrps6=gly*,ala*,gly*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r8c4-6-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NCC(=O)NC(CC(=O)O)C(=O)NC(Cc1ccc(O)cc1)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CC(=O)O)C(=O)NCC(=O)N1C(CCC1)C(=O)NCC(=O)NC(C)C(=O)NCC(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r8c4-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-178"></div> <span class="help-tooltip" id="nrps-monomers-help-178">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds810919_815815</span></strong>: gly
      <div class="collapser collapser-target-cds810919_815815 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (466..852)</strong>: gly
          <div class="collapser collapser-target-cds810919_815815 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-cds810919_815815-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILQLGMvWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds815856_819375</span></strong>: asp
      <div class="collapser collapser-target-cds815856_819375 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asp" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asp*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (522..902)</strong>: asp
          <div class="collapser collapser-target-cds815856_819375 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asp
              <div class="collapser collapser-target-cds815856_819375-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>asp, asn</dd>
   <dt>Single AA prediction:</dt>
   <dd>asp</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLTKvGAVNK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asp</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds819338_825347</span></strong>: tyr
      <div class="collapser collapser-target-cds819338_825347 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=tyr" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=tyr*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (923..1306)</strong>: tyr
          <div class="collapser collapser-target-cds819338_825347 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  tyr
              <div class="collapser collapser-target-cds819338_825347-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>tyr, bht</dd>
   <dt>Single AA prediction:</dt>
   <dd>tyr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DASTaAAVCK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>tyr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds825457_831724</span></strong>: thr - X
      <div class="collapser collapser-target-cds825457_831724 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr,X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*,X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (1..349)</strong>: thr
          <div class="collapser collapser-target-cds825457_831724 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds825457_831724-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNVGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1016..1412)</strong>: X
          <div class="collapser collapser-target-cds825457_831724 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds825457_831724-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DALfaGAVMK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds831725_843422</span></strong>: asp - gly - pro
      <div class="collapser collapser-target-cds831725_843422 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asp,gly,pro" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asp*,gly*,pro*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (469..874)</strong>: asp
          <div class="collapser collapser-target-cds831725_843422 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asp
              <div class="collapser collapser-target-cds831725_843422-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>asp, asn</dd>
   <dt>Single AA prediction:</dt>
   <dd>asp</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLTKvGAVNK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asp</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1538..1929)</strong>: gly
          <div class="collapser collapser-target-cds831725_843422 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-cds831725_843422-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILQLGVIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2590..2989)</strong>: pro
          <div class="collapser collapser-target-cds831725_843422 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  pro
              <div class="collapser collapser-target-cds831725_843422-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>pro, pip</dd>
   <dt>Small clusters prediction:</dt>
   <dd>pro</dd>
   <dt>Single AA prediction:</dt>
   <dd>pro</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVQYvAHVVK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pro</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds844418_856022</span></strong>: gly - ala - gly
      <div class="collapser collapser-target-cds844418_856022 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly,ala,gly" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*,ala*,gly*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (477..858)</strong>: gly
          <div class="collapser collapser-target-cds844418_856022 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-cds844418_856022-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILQLGLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1524..1924)</strong>: ala
          <div class="collapser collapser-target-cds844418_856022 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ala
              <div class="collapser collapser-target-cds844418_856022-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>ala</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DvfSvAiVYK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2591..2994)</strong>: gly
          <div class="collapser collapser-target-cds844418_856022 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-cds844418_856022-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILQLGVIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds865894_867673</span></strong>: pk
      <div class="collapser collapser-target-cds865894_867673 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>CAL_domain (42..485)</strong>: pk
          <div class="collapser collapser-target-cds865894_867673 collapser-level-candidate"> </div><div class="collapser-content">
          
              minowa:  fatty_acid
              <div class="collapser collapser-target-cds865894_867673-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>fatty_acid: 42.3</dt>

  <dd></dd><dt>NH2: 17.0</dt>

  <dd></dd><dt>AHBA: 6.6</dt>

  <dd></dd><dt>shikimic_acid: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: -1.1</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r10c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00010_CC216C_.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-293"></div> <span class="help-tooltip" id="region_svg-help-293">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00010_CC216C_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r10c1-svg" data-filename="c00010_CC216C_.._r10c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 53,749 - 74,756 nt. (total: 21,008 nt)
        
        <a class="cluster-rules-header" id="r10c1-rules-header" href="#r10c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r10c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r10c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-294"></div> <span class="help-tooltip" id="focus-panel-help-294">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r10c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r10c1-clusterblast" data-name="r10c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r10c1-knownclusterblast" data-name="r10c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r10c1-subclusterblast" data-name="r10c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r10c1-MIBiG-cluster-compare" data-name="r10c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r10c1-pfam-details" data-name="r10c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r10c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-184"></div> <span class="help-tooltip" id="cb-general-help-184">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r10c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r10c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r10c1_1.svg">
        NZ_KL370899 (346404-367382): Streptomyces sp. CNS654 CD02DRAFT scaffold00005.... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r10c1_2.svg">
        NZ_LISY01000003 (755357-776335): Streptomyces sp. TSRI0261 scaffold3, whole g... (94% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r10c1_3.svg">
        NZ_CP011522 (2586117-2605595): Streptomyces sp. CFMR 7 strain CFMR-7 chromoso... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r10c1_4.svg">
        NZ_WWGV01000011 (199866-220844): Streptomyces sp. SID8356 SID8356.c11, whole ... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r10c1_5.svg">
        NZ_CM003601 (6132384-6151290): Streptomyces anulatus strain ATCC 11523 chromo... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r10c1_6.svg">
        NZ_LIPB01000001 (339174-360152): Streptomyces sp. CB00072 scaffold1, whole ge... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r10c1_7.svg">
        NZ_KZ195572 (2477563-2496443): Streptomyces sp. CS057 scaffold00001, whole ge... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r10c1_8.svg">
        NZ_CP015362 (2076838-2096342): Streptomyces sp. S8 chromosome, complete genome (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r10c1_9.svg">
        NZ_AQRK01000102 (132635-153613): Streptomyces sp. AmelKG-D3 YUWDRAFT scaffold... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r10c1_10.svg">
        NZ_CP029378 (2187558-2207007): Streptomyces bacillaris strain ATCC 15855 chro... (100% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r10c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r10c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r10c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-185"></div> <span class="help-tooltip" id="cb-known-help-185">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r10c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-186"></div> <span class="help-tooltip" id="cb-sub-help-186">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r10c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-187"></div> <span class="help-tooltip" id="MIBiG-help-187">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r10c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r10c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r10c1"></div>
    <div id="comparison-MIBiG-r10c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000243.1: 338-14804">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000243/index.html#r1c1" target="_new">BGC0000243.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>macrotetrolide</td>
        
        
        <td>Streptomyces griseus subsp. griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000913.1: 1-11028">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000913/index.html#r1c1" target="_new">BGC0000913.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>MK-8</td>
        
        
        <td>Enterobacter cloacae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000252.1: 248-15270">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000252/index.html#r1c1" target="_new">BGC0000252.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>nonactin</td>
        
        
        <td>Streptomyces griseus subsp. griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.27 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
         
        
        <td>0.07</td>
        <td>RiPP</td>
        
        <td>informatipeptin</td>
        
        
        <td>Streptomyces viridochromogenes DSM 40736</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002034.1: 86568-105957">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002034/index.html#r1c1" target="_new">BGC0002034.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.26 (id:0.46, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
         
        
        <td>0.06</td>
        <td>Other</td>
        
        <td>perquinoline A, perquinoline B, perquinoline C</td>
        
        
        <td>Streptomyces sp. IB2014 016-6</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000157.1: 1384-70003">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000157/index.html#r1c1" target="_new">BGC0000157.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.25 (id:0.45, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
         
        
        <td>0.06</td>
        <td>Polyketide</td>
        
        <td>tautomycetin</td>
        
        
        <td>Streptomyces sp. CK4412</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000842.1: 990-43743">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000842/index.html#r1c1" target="_new">BGC0000842.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.22 (id:0.43, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
         
        
        <td>0.05</td>
        <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
        
        <td>carbapenem MM4550</td>
        
        
        <td>Streptomyces argenteolus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000874.1: 0-36223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000874/index.html#r1c1" target="_new">BGC0000874.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.22 (id:0.47, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
         
        
        <td>0.05</td>
        <td>Other</td>
        
        <td>blasticidin S</td>
        
        
        <td>Streptomyces griseochromogenes</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001076.1: 0-29709">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001076/index.html#r1c1" target="_new">BGC0001076.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.22 (id:0.43, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
         
        
        <td>0.05</td>
        <td>Other,  , (, N, u, c, l, e, o, s, i, d, e, )</td>
        
        <td>liposidomycin B, liposidomycin A, liposidomycin C, liposidomycin Z, liposidomycin H, liposidomycin G, liposidomycin L, liposidomycin M, liposidomycin K, liposidomycin N</td>
        
        
        <td>Streptomyces sp. SN-1061M</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000872.1: 0-35385">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000872/index.html#r1c1" target="_new">BGC0000872.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.22 (id:0.43, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
         
        
        <td>0.05</td>
        <td>Other</td>
        
        <td>A-90289 A, A-90289 B</td>
        
        
        <td>Streptomyces sp. SANK 60405</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r10c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000243.1: 338-14804">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000243/index.html#r1c1" target="_new">BGC0000243.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Polyketide</td>
      <td>macrotetrolide</td>
      <td>Streptomyces griseus subsp. griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000913.1: 1-11028">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000913/index.html#r1c1" target="_new">BGC0000913.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>MK-8</td>
      <td>Enterobacter cloacae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000252.1: 248-15270">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000252/index.html#r1c1" target="_new">BGC0000252.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Polyketide</td>
      <td>nonactin</td>
      <td>Streptomyces griseus subsp. griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.27 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
      <td>0.27</td>
      <td>RiPP</td>
      <td>informatipeptin</td>
      <td>Streptomyces viridochromogenes DSM 40736</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002034.1: 86568-105957">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002034/index.html#r1c1" target="_new">BGC0002034.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.26 (id:0.46, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
      <td>0.26</td>
      <td>Other</td>
      <td>perquinoline A, perquinoline B, perquinoline C</td>
      <td>Streptomyces sp. IB2014 016-6</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000157.1: 1384-70003">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000157/index.html#r1c1" target="_new">BGC0000157.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.25 (id:0.45, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
      <td>0.25</td>
      <td>Polyketide</td>
      <td>tautomycetin</td>
      <td>Streptomyces sp. CK4412</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000842.1: 990-43743">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000842/index.html#r1c1" target="_new">BGC0000842.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.22 (id:0.43, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
      <td>0.22</td>
      <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
      <td>carbapenem MM4550</td>
      <td>Streptomyces argenteolus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000874.1: 0-36223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000874/index.html#r1c1" target="_new">BGC0000874.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.22 (id:0.47, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
      <td>0.22</td>
      <td>Other</td>
      <td>blasticidin S</td>
      <td>Streptomyces griseochromogenes</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001076.1: 0-29709">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001076/index.html#r1c1" target="_new">BGC0001076.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.22 (id:0.43, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
      <td>0.22</td>
      <td>Other,  , (, N, u, c, l, e, o, s, i, d, e, )</td>
      <td>liposidomycin B, liposidomycin A, liposidomycin C, liposidomycin Z, liposidomycin H, liposidomycin G, liposidomycin L, liposidomycin M, liposidomycin K, liposidomycin N</td>
      <td>Streptomyces sp. SN-1061M</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000872.1: 0-35385">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000872/index.html#r1c1" target="_new">BGC0000872.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.22 (id:0.43, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
      <td>0.22</td>
      <td>Other</td>
      <td>A-90289 A, A-90289 B</td>
      <td>Streptomyces sp. SANK 60405</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r10c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-188"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-188">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r10c1-pfam-details-svg" data-filename="c00010_CC216C_.._r10c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r10c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r10c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00010_CC216C_.. - Region 2 - siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-295"></div> <span class="help-tooltip" id="region_svg-help-295">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00010_CC216C_...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r10c2-svg" data-filename="c00010_CC216C_.._r10c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 474,733 - 489,455 nt. (total: 14,723 nt)
        
        <a class="cluster-rules-header" id="r10c2-rules-header" href="#r10c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r10c2-rules">siderophore: IucA_IucC</div>
      
      <div class="region-svg-container">
       <div id='r10c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-296"></div> <span class="help-tooltip" id="focus-panel-help-296">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r10c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r10c2-clusterblast" data-name="r10c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r10c2-knownclusterblast" data-name="r10c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r10c2-subclusterblast" data-name="r10c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r10c2-MIBiG-cluster-compare" data-name="r10c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r10c2-pfam-details" data-name="r10c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r10c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-189"></div> <span class="help-tooltip" id="cb-general-help-189">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r10c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r10c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r10c2_1.svg">
        NZ_KZ195572 (2050282-2063298): Streptomyces sp. CS057 scaffold00001, whole ge... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c2_2.svg">
        NZ_BDHJ01000025 (21087-36067): Streptomyces sp. NBRC 110465, whole genome sho... (88% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c2_3.svg">
        NZ_GL877172 (2233000-2246435): Streptomyces sp. ACT-1 strain XylebKG-1 scaffo... (88% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c2_4.svg">
        NC_010572 (2026430-2039870): Streptomyces griseus subsp. griseus NBRC 13350, ... (88% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c2_5.svg">
        NZ_FLTQ01000024 (559541-574336): Streptomyces sp. MnatMP-M77, whole genome sh... (88% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c2_6.svg">
        NZ_LIVT01000003 (520205-535024): Streptomyces sp. CB02366 scaffold3, whole ge... (88% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c2_7.svg">
        NZ_KZ819159 (5819360-5834033): Streptomyces sp. CS090A Scaffold00001, whole g... (72% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c2_8.svg">
        NZ_WWIF01000016 (184712-199408): Streptomyces sp. SID5594 SID5594.c16, whole ... (72% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c2_9.svg">
        NZ_LZQS01000012 (22436-37132): Streptomyces sp. IB2014 011-1 STIB 12, whole g... (72% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c2_10.svg">
        NZ_WUEJ01000010 (22103-36851): Streptomyces sp. YIM 132580 Scaffold10, whole ... (80% of genes show similarity), siderophore
      </option>
      
    </select>
    <button id= "clusterblast-r10c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r10c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r10c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-190"></div> <span class="help-tooltip" id="cb-known-help-190">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r10c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-191"></div> <span class="help-tooltip" id="cb-sub-help-191">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r10c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-192"></div> <span class="help-tooltip" id="MIBiG-help-192">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r10c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r10c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r10c2"></div>
    <div id="comparison-MIBiG-r10c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">siderophore</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001531.1: 1112-9953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001531/index.html#r1c1" target="_new">BGC0001531.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>Other</td>
        
        <td>bisucaberin B</td>
        
        
        <td>Tenacibaculum mesophilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Pantoea agglomerans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Other</td>
        
        <td>ochrobactin</td>
        
        
        <td>Ochrobactrum anthropi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001408.1: 1377589-1387887">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001408/index.html#r1c1" target="_new">BGC0001408.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Other</td>
        
        <td>xanthoferrin</td>
        
        
        <td>Xanthomonas oryzae pv. oryzae KACC 10331</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000939.1: 1834-15938">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000939/index.html#r1c1" target="_new">BGC0000939.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>Other</td>
        
        <td>aerobactin</td>
        
        
        <td>Grimontia hollisae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001870.1: 354426-369084">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001870/index.html#r1c1" target="_new">BGC0001870.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Other</td>
        
        <td>putrebactin, avaroferrin</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000942.1: 1864728-1871818">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000942/index.html#r1c1" target="_new">BGC0000942.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:0.42, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Other</td>
        
        <td>petrobactin</td>
        
        
        <td>Bacillus anthracis str. Ames</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000295.1: 569-12865">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000295/index.html#r1c1" target="_new">BGC0000295.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.44, order:0.37, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>NRP</td>
        
        <td>acinetoferrin</td>
        
        
        <td>Acinetobacter haemolyticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Other,  , (, E, c, t, o, i, n, e, )</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r10c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001531.1: 1112-9953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001531/index.html#r1c1" target="_new">BGC0001531.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>bisucaberin B</td>
      <td>Tenacibaculum mesophilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Pantoea agglomerans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>ochrobactin</td>
      <td>Ochrobactrum anthropi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001408.1: 1377589-1387887">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001408/index.html#r1c1" target="_new">BGC0001408.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other</td>
      <td>xanthoferrin</td>
      <td>Xanthomonas oryzae pv. oryzae KACC 10331</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000939.1: 1834-15938">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000939/index.html#r1c1" target="_new">BGC0000939.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>aerobactin</td>
      <td>Grimontia hollisae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001870.1: 354426-369084">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001870/index.html#r1c1" target="_new">BGC0001870.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Other</td>
      <td>putrebactin, avaroferrin</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000942.1: 1864728-1871818">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000942/index.html#r1c1" target="_new">BGC0000942.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:0.42, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Other</td>
      <td>petrobactin</td>
      <td>Bacillus anthracis str. Ames</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000295.1: 569-12865">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000295/index.html#r1c1" target="_new">BGC0000295.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.44, order:0.37, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>NRP</td>
      <td>acinetoferrin</td>
      <td>Acinetobacter haemolyticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Other,  , (, E, c, t, o, i, n, e, )</td>
      <td>ectoine</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r10c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-193"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-193">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r10c2-pfam-details-svg" data-filename="c00010_CC216C_.._r10c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r10c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r10c3' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00010_CC216C_.. - Region 3 - NRPS,ladderane,thioamide-NRP <div class="help-container"> <div class="help-icon" data-id="region_svg-help-297"></div> <span class="help-tooltip" id="region_svg-help-297">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00010_CC216C_...region003.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r10c3-svg" data-filename="c00010_CC216C_.._r10c3.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 564,168 - 644,707 nt. (total: 80,540 nt)
        
        <a class="cluster-rules-header" id="r10c3-rules-header" href="#r10c3">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r10c3-rules">ladderane: ladderane<br>thioamide-NRP: (ATP-grasp and AMP-binding and PP-binding and (DUF98 or Orn_Arg_deC_N or Orn_DAP_Arg_deC))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r10c3-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-298"></div> <span class="help-tooltip" id="focus-panel-help-298">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r10c3">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r10c3-nrps_pks" data-name="r10c3-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r10c3-clusterblast" data-name="r10c3-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r10c3-knownclusterblast" data-name="r10c3-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r10c3-subclusterblast" data-name="r10c3-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r10c3-MIBiG-cluster-compare" data-name="r10c3-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r10c3-pfam-details" data-name="r10c3-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r10c3-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-194"></div> <span class="help-tooltip" id="nrps-domain-help-194">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r10c3-details-svg" data-filename="c00010_CC216C_.._r10c3_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r10c3-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r10c3-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-197"></div> <span class="help-tooltip" id="cb-general-help-197">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r10c3-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r10c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r10c3_1.svg">
        NZ_LIVX01000008 (196106-349760): Streptomyces sp. CB02261 scaffold8, whole ge... (43% of genes show similarity), NRPS,indole,ladderane,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r10c3_2.svg">
        NZ_CP029196 (6369955-6503494): Streptomyces venezuelae strain ATCC 21113 chro... (50% of genes show similarity), NRPS,ladderane,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r10c3_3.svg">
        NC_018750 (6721250-6855167): Streptomyces venezuelae ATCC 10712, complete genome (51% of genes show similarity), NRPS,ladderane,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r10c3_4.svg">
        NZ_CP029195 (1401410-1576477): Streptomyces venezuelae strain ATCC 10595 chro... (41% of genes show similarity), NRPS,NRPS-like,ladderane,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r10c3_5.svg">
        NZ_CP032698 (3157450-3248693): Streptomyces hundungensis strain BH38 chromoso... (52% of genes show similarity), NRPS,bacteriocin,ladderane,thiopeptide
      </option>
      
      <option value = "svg/clusterblast_r10c3_6.svg">
        NZ_LIPD01000004 (477868-549380): Streptomyces sp. CB02056 scaffold4, whole ge... (62% of genes show similarity), NRPS,ladderane,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r10c3_7.svg">
        NZ_JACHJN010000001 (707088-835164): Saccharothrix tamanrassetensis strain CEC... (45% of genes show similarity), NRPS,ladderane,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r10c3_8.svg">
        NC_019673 (3409970-3541086): Saccharothrix espanaensis DSM 44229, complete ge... (38% of genes show similarity), NRPS,ladderane,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r10c3_9.svg">
        NZ_CP023701 (3767226-3853956): Streptomyces subrutilus strain ATCC 27467 chro... (46% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r10c3_10.svg">
        NZ_KZ819182 (4724034-4779057): Streptomyces sp. CS147 scaffold00001, whole ge... (75% of genes show similarity), NRPS,ladderane
      </option>
      
    </select>
    <button id= "clusterblast-r10c3-download">Download graphic</button>
  </div>
  <div id="clusterblast-r10c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r10c3-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-198"></div> <span class="help-tooltip" id="cb-known-help-198">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r10c3-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r10c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r10c3_1.svg">
        atratumycin
      </option>
      
      <option value = "svg/knownclusterblast_r10c3_2.svg">
        RP-1776
      </option>
      
      <option value = "svg/knownclusterblast_r10c3_3.svg">
        pepticinnamin E
      </option>
      
      <option value = "svg/knownclusterblast_r10c3_4.svg">
        telomycin
      </option>
      
      <option value = "svg/knownclusterblast_r10c3_5.svg">
        acyldepsipeptide 1
      </option>
      
      <option value = "svg/knownclusterblast_r10c3_6.svg">
        ulleungmycin
      </option>
      
      <option value = "svg/knownclusterblast_r10c3_7.svg">
        ishigamide
      </option>
      
      <option value = "svg/knownclusterblast_r10c3_8.svg">
        WS9326
      </option>
      
      <option value = "svg/knownclusterblast_r10c3_9.svg">
        kedarcidin
      </option>
      
      <option value = "svg/knownclusterblast_r10c3_10.svg">
        CDA1b / CDA2a / CDA2b / CDA3a / CDA3b / CDA4a / CDA4b
      </option>
      
    </select>
    <button id="knownclusterblast-r10c3-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r10c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r10c3-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-199"></div> <span class="help-tooltip" id="cb-sub-help-199">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r10c3-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r10c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r10c3_1.svg">
        NC_009380_c1: salinosporamide cyclohexenylalanine (50% of genes show similarity), cyclohexenylalanine
      </option>
      
      <option value = "svg/subclusterblast_r10c3_2.svg">
        AY048670_c2: C-1027 benzoxazolinate (28% of genes show similarity), benzoxazolinate
      </option>
      
    </select>
    <button id="subclusterblast-r10c3-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r10c3-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r10c3-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-200"></div> <span class="help-tooltip" id="MIBiG-help-200">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r10c3-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r10c3-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r10c3"></div>
    <div id="comparison-MIBiG-r10c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">ladderane</span></th>
         
         <th><span class="cc-heat-product">thioamide-NRP</span></th>
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.47, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.84</td>
        <td>NRP, Polyketide</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001710.1: 451-19435">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001710/index.html#r1c1" target="_new">BGC0001710.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.46, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.82</td>
        <td>NRP</td>
        
        <td>obafluorin</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001437.1: 451-19435">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001437/index.html#r1c1" target="_new">BGC0001437.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.46, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.82</td>
        <td>NRP</td>
        
        <td>obafluorin</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001824.1: 39732-57269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001824/index.html#r1c1" target="_new">BGC0001824.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.49, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.76</td>
        <td>NRP</td>
        
        <td>xefoampeptides A-G</td>
        
        
        <td>Xenorhabdus beddingii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001480.1: 833379-853163">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001480/index.html#r1c1" target="_new">BGC0001480.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.74</td>
        <td>Other</td>
        
        <td>tundrenone</td>
        
        
        <td>Methylobacter tundripaludum 21/22</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001903.1: 8904-19774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001903/index.html#r1c1" target="_new">BGC0001903.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.53, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.53, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.53, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.73</td>
        <td>Other</td>
        
        <td>neocarazostatin A</td>
        
        
        <td>Streptomyces sp. MA37</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001302.1: 0-25686">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001302/index.html#r1c1" target="_new">BGC0001302.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.72</td>
        <td>Other</td>
        
        <td>lomofungin</td>
        
        
        <td>Streptomyces lomondensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.47, order:0.26, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.47, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.71</td>
        <td>Other</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.55, order:0.51, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.51, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.51, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.71</td>
        <td>Alkaloid</td>
        
        <td>kitasetaline</td>
        
        
        <td>Kitasatospora setae KM-6054</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.69</td>
        <td>NRP</td>
        
        <td>rhodochelin</td>
        
        
        <td>Rhodococcus jostii RHA1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r10c3-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417.1: 2443455-2455531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>rhodochelin</td>
      <td>Rhodococcus jostii RHA1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001710.1: 451-19435">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001710/index.html#r1c1" target="_new">BGC0001710.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>obafluorin</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001437.1: 451-19435">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001437/index.html#r1c1" target="_new">BGC0001437.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>obafluorin</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401.1: 0-12882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>paenibactin</td>
      <td>Paenibacillus elgii B69</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP, Polyketide</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001983.1: 622-39496">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001983/index.html#r1c1" target="_new">BGC0001983.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.48, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Other</td>
      <td>triacsins</td>
      <td>Kitasatospora aureofaciens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001123.1: 59594-103337">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001123/index.html#r1c1" target="_new">BGC0001123.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>aspirochlorine</td>
      <td>Aspergillus oryzae RIB40</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000454.1: 24-19884">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000454/index.html#r1c1" target="_new">BGC0000454.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>vanchrobactin</td>
      <td>Vibrio anguillarum RV22</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001149.1: 196767-209553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001149/index.html#r1c1" target="_new">BGC0001149.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP, Saccharide</td>
      <td>brabantamide A</td>
      <td>Pseudomonas sp. SHC52</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r10c3-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-201"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-201">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r10c3-pfam-details-svg" data-filename="c00010_CC216C_.._r10c3_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r10c3-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r10c3-nrps_pks_products" data-name="r10c3-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r10c3-nrps_pks" data-name="r10c3-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r10c3-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-195"></div> <span class="help-tooltip" id="nrps-structures-help-195">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 564167 - 644707:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r10c3-3" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(Cc1ccccc1)C(=O)NC([*])C(=O)NCC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-X - phe - X - gly)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x,phe,x,gly" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x*,phe*,x,gly*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r10c3-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(Cc1ccccc1)C(=O)NC([*])C(=O)NCC(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 5, location 581786 - 644707:
          <div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r10c3-5" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(Cc1ccccc1)C(=O)NC([*])C(=O)NCC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-X - phe - X - gly)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x,phe,x,gly" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x*,phe*,x,gly*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r10c3-5-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(Cc1ccccc1)C(=O)NC([*])C(=O)NCC(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r10c3-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-196"></div> <span class="help-tooltip" id="nrps-monomers-help-196">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds609752_624707</span></strong>: X - phe - X - gly
      <div class="collapser collapser-target-cds609752_624707 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,phe,X,gly" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,phe*,X*,gly*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (472..873)</strong>: X
          <div class="collapser collapser-target-cds609752_624707 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aromatic
              <div class="collapser collapser-target-cds609752_624707-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DiWeMtadDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1992..2390)</strong>: phe
          <div class="collapser collapser-target-cds609752_624707 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  phe
              <div class="collapser collapser-target-cds609752_624707-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>phe, trp</dd>
   <dt>Single AA prediction:</dt>
   <dd>phe</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAWgIAGlaK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (3053..3460)</strong>: X
          <div class="collapser collapser-target-cds609752_624707 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds609752_624707-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">IDTtISfavK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ala-b</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (4120..4520)</strong>: gly
          <div class="collapser collapser-target-cds609752_624707 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-cds609752_624707-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILQLGLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r10c4' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00010_CC216C_.. - Region 4 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-299"></div> <span class="help-tooltip" id="region_svg-help-299">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00010_CC216C_...region004.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r10c4-svg" data-filename="c00010_CC216C_.._r10c4.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 752,989 - 846,684 nt. (total: 93,696 nt)
        
        <a class="cluster-rules-header" id="r10c4-rules-header" href="#r10c4">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r10c4-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r10c4-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-300"></div> <span class="help-tooltip" id="focus-panel-help-300">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r10c4">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r10c4-nrps_pks" data-name="r10c4-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r10c4-clusterblast" data-name="r10c4-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r10c4-knownclusterblast" data-name="r10c4-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r10c4-subclusterblast" data-name="r10c4-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r10c4-MIBiG-cluster-compare" data-name="r10c4-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r10c4-pfam-details" data-name="r10c4-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r10c4-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-202"></div> <span class="help-tooltip" id="nrps-domain-help-202">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r10c4-details-svg" data-filename="c00010_CC216C_.._r10c4_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r10c4-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r10c4-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-205"></div> <span class="help-tooltip" id="cb-general-help-205">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r10c4-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r10c4_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r10c4_1.svg">
        NZ_KZ679082 (6384735-6443335): Streptomyces sp. CS149 scaffold00001, whole ge... (65% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r10c4_2.svg">
        NZ_NNBN01000007 (237466-293018): Streptomyces sp. CB02613 scaffold7, whole ge... (74% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r10c4_3.svg">
        NZ_WWJR01000065 (50947-96833): Streptomyces sp. SID4913 SID4913.c65, whole ge... (72% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r10c4_4.svg">
        NZ_LMFT01000040 (50750-132998): Streptomyces sp. Root55 contig 9, whole genom... (42% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r10c4_5.svg">
        NZ_ML123098 (400399-483190): Streptomyces sp. ADI97-07 scaffold001, whole gen... (42% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r10c4_6.svg">
        NZ_LIVO01000003 (538817-583016): Streptomyces sp. CB02488 scaffold3, whole ge... (69% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r10c4_7.svg">
        NZ_LISY01000005 (293027-348659): Streptomyces sp. TSRI0261 scaffold5, whole g... (56% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r10c4_8.svg">
        NZ_LACR01000010 (205116-260725): Streptomyces sp. Ncost-T6T-1 YW5DRAFT scaffo... (54% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r10c4_9.svg">
        NZ_WWIP01000001 (6374820-6428716): Streptomyces sp. SID5466 SID5466.c1, whole... (55% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r10c4_10.svg">
        NZ_CP020563 (6484786-6573335): Kitasatospora albolonga strain YIM 101047 chro... (36% of genes show similarity), NRPS,T1PKS,other
      </option>
      
    </select>
    <button id= "clusterblast-r10c4-download">Download graphic</button>
  </div>
  <div id="clusterblast-r10c4-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r10c4-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-206"></div> <span class="help-tooltip" id="cb-known-help-206">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r10c4-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r10c4_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r10c4_1.svg">
        collismycin A
      </option>
      
      <option value = "svg/knownclusterblast_r10c4_2.svg">
        caerulomycin A
      </option>
      
      <option value = "svg/knownclusterblast_r10c4_3.svg">
        neocarzinostatin
      </option>
      
      <option value = "svg/knownclusterblast_r10c4_4.svg">
        sporolide A / sporolide B
      </option>
      
      <option value = "svg/knownclusterblast_r10c4_5.svg">
        maduropeptin
      </option>
      
      <option value = "svg/knownclusterblast_r10c4_6.svg">
        kedarcidin
      </option>
      
      <option value = "svg/knownclusterblast_r10c4_7.svg">
        C-1027
      </option>
      
      <option value = "svg/knownclusterblast_r10c4_8.svg">
        lidamycin
      </option>
      
      <option value = "svg/knownclusterblast_r10c4_9.svg">
        C-1027
      </option>
      
      <option value = "svg/knownclusterblast_r10c4_10.svg">
        amycolamycin A / amycolamycin B
      </option>
      
    </select>
    <button id="knownclusterblast-r10c4-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r10c4-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r10c4-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-207"></div> <span class="help-tooltip" id="cb-sub-help-207">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r10c4-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r10c4_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r10c4_1.svg">
        AY271660_c2: maduropeptin enediyne (75% of genes show similarity), enediyne
      </option>
      
      <option value = "svg/subclusterblast_r10c4_2.svg">
        AY117439_c3: neocarzinostatin enediyne (66% of genes show similarity), enediyne
      </option>
      
      <option value = "svg/subclusterblast_r10c4_3.svg">
        AY048670_c3: C-1027 enediyne (66% of genes show similarity), enediyne
      </option>
      
      <option value = "svg/subclusterblast_r10c4_4.svg">
        AF497482_c2: calicheamicin enediyne (38% of genes show similarity), enediyne
      </option>
      
      <option value = "svg/subclusterblast_r10c4_5.svg">
        AY117439_c1: neocarzinostatin deoxy aminosugar (42% of genes show similarity), sugar
      </option>
      
    </select>
    <button id="subclusterblast-r10c4-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r10c4-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r10c4-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-208"></div> <span class="help-tooltip" id="MIBiG-help-208">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r10c4-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r10c4-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r10c4"></div>
    <div id="comparison-MIBiG-r10c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000973.1: 548-46105">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000973/index.html#r1c1" target="_new">BGC0000973.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.76 (id:0.79, order:0.73, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.75 (id:0.79, order:0.73, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.90</td>
        <td>NRP, Polyketide</td>
        
        <td>collismycin A</td>
        
        
        <td>Streptomyces sp. CS40</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002091.1: 3593-21565">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002091/index.html#r1c1" target="_new">BGC0002091.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.39 (id:0.44, order:N/A, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
         
        
        <td>0.76</td>
        <td>NRP</td>
        
        <td>holomycin</td>
        
        
        <td>Yersinia ruckeri ATCC 29473</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.71</td>
        <td>Other</td>
        
        <td>showdomycin</td>
        
        
        <td>Streptomyces showdoensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.51, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.51, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.70</td>
        <td>NRP, Polyketide</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.43, order:1.00, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.43, order:1.00, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.69</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001560.1: 0-28792">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001560/index.html#r1c1" target="_new">BGC0001560.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.68</td>
        <td>NRP, Polyketide</td>
        
        <td>cryptomaldamide</td>
        
        
        <td>Moorea producens JHB</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.67</td>
        <td>Polyketide</td>
        
        <td>depudecin</td>
        
        
        <td>Alternaria brassicicola</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.66</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.66</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.61</td>
        <td>NRP</td>
        
        <td>fellutamide B</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r10c4-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000973.1: 548-46105">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000973/index.html#r1c1" target="_new">BGC0000973.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.79 (id:0.79, order:0.73, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
      <td>0.79</td>
      <td>NRP, Polyketide</td>
      <td>collismycin A</td>
      <td>Streptomyces sp. CS40</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Polyketide</td>
      <td>esperamicin</td>
      <td>Actinomadura verrucosospora</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.43, order:1.00, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000211.1: 0-34654">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000211/index.html#r1c1" target="_new">BGC0000211.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.51, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Polyketide</td>
      <td>chrysomycin</td>
      <td>Streptomyces albaduncus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000263.1: 0-32801">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000263/index.html#r1c1" target="_new">BGC0000263.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.49, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>ravidomycin</td>
      <td>Streptomyces ravidus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002042.1: 1539-27652">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002042/index.html#r1c1" target="_new">BGC0002042.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.91)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>anthelvencin A, anthelvencin B, anthelvencin C</td>
      <td>Streptomyces venezuelae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>fellutamide B</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.51, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r10c4-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-209"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-209">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r10c4-pfam-details-svg" data-filename="c00010_CC216C_.._r10c4_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r10c4-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r10c4-nrps_pks_products" data-name="r10c4-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r10c4-nrps_pks" data-name="r10c4-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r10c4-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-203"></div> <span class="help-tooltip" id="nrps-structures-help-203">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 6, location 752988 - 846684:
          <div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r10c4-6" class="smiles-canvas" data-smiles="CC(=O)NC(CS)C(=O)NC(C(O)C)C(=O)CC(=O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(mal - cys) + (thr) + (mal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal,cys&amp;nrps2=thr&amp;nrps3=mal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal*,cys*&amp;nrps2=thr*&amp;nrps3=mal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r10c4-6-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(=O)NC(CS)C(=O)NC(C(O)C)C(=O)CC(=O)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 7, location 752988 - 803751:
          <div class="collapser collapser-target-CC7 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r10c4-7" class="smiles-canvas" data-smiles="CC(=O)NC(CS)C(=O)NC(C(O)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(mal - cys) + (thr)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal,cys&amp;nrps2=thr" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal*,cys*&amp;nrps2=thr*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r10c4-7-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(=O)NC(CS)C(=O)NC(C(O)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
        <hr>
      
      
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 8, location 800798 - 846684:
          <div class="collapser collapser-target-CC8 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r10c4-8" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r10c4-8-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r10c4-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-204"></div> <span class="help-tooltip" id="nrps-monomers-help-204">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds764352_765462</span></strong>: X
      <div class="collapser collapser-target-cds764352_765462 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (99..319)</strong>: X
          <div class="collapser collapser-target-cds764352_765462 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds764352_765462-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">tVdVdii--K</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pro</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>30% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds772988_780479</span></strong>: mal - cys
      <div class="collapser collapser-target-cds772988_780479 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>PKS_AT (541..835)</strong>: mal
          <div class="collapser collapser-target-cds772988_780479 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds772988_780479-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 83.3%</dd>
<dd>inactive: 75.0%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds772988_780479-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 153.1</dt>

  <dd></dd><dt>inactive: 92.9</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 86.3</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 69.2</dt>

  <dd></dd><dt>Propionyl-CoA: 61.2</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 43.2</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 38.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 34.0</dt>

  <dd></dd><dt>Acetyl-CoA: 30.7</dt>

  <dd></dd><dt>fatty_acid: 29.1</dt>

  <dd></dd><dt>CHC-CoA: 28.2</dt>

  <dd></dd><dt>trans-1,2-CPDA: 24.2</dt>

  <dd></dd><dt>Benzoyl-CoA: 20.3</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 10.9</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1494..1889)</strong>: cys
          <div class="collapser collapser-target-cds772988_780479 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  cys
              <div class="collapser collapser-target-cds772988_780479-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Small clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Single AA prediction:</dt>
   <dd>cys</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLFNLSLvWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>cys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds780526_783751</span></strong>: thr
      <div class="collapser collapser-target-cds780526_783751 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (468..884)</strong>: thr
          <div class="collapser collapser-target-cds780526_783751 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds780526_783751-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DywsFGlVVK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds820798_826684</span></strong>: mal
      <div class="collapser collapser-target-cds820798_826684 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (637..866)</strong>: mal
          <div class="collapser collapser-target-cds820798_826684 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds820798_826684-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Propionyl-CoA: 58.3%</dd>
<dd>inactive: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds820798_826684-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 82.5</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 47.8</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 46.5</dt>

  <dd></dd><dt>inactive: 28.8</dt>

  <dd></dd><dt>Propionyl-CoA: 24.5</dt>

  <dd></dd><dt>Isobutyryl-CoA: 23.5</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 20.6</dt>

  <dd></dd><dt>fatty_acid: 13.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 9.2</dt>

  <dd></dd><dt>Benzoyl-CoA: 8.1</dt>

  <dd></dd><dt>Acetyl-CoA: 2.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: -0.9</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r10c5' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00010_CC216C_.. - Region 5 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-301"></div> <span class="help-tooltip" id="region_svg-help-301">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00010_CC216C_...region005.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r10c5-svg" data-filename="c00010_CC216C_.._r10c5.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 936,005 - 947,333 nt. (total: 11,329 nt)
        
        <a class="cluster-rules-header" id="r10c5-rules-header" href="#r10c5">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r10c5-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193)</div>
      
      <div class="region-svg-container">
       <div id='r10c5-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-302"></div> <span class="help-tooltip" id="focus-panel-help-302">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r10c5">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r10c5-clusterblast" data-name="r10c5-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r10c5-knownclusterblast" data-name="r10c5-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r10c5-subclusterblast" data-name="r10c5-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r10c5-MIBiG-cluster-compare" data-name="r10c5-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r10c5-pfam-details" data-name="r10c5-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r10c5-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-210"></div> <span class="help-tooltip" id="cb-general-help-210">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r10c5-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r10c5_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r10c5_1.svg">
        NZ_KZ819178 (6435381-6444672): Streptomyces sp. CS131 scaffold00001, whole ge... (100% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r10c5_2.svg">
        NZ_NNBN01000007 (173961-185290): Streptomyces sp. CB02613 scaffold7, whole ge... (90% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r10c5_3.svg">
        NZ_KZ819154 (1006049-1017378): Streptomyces sp. CS014 scaffold00001, whole ge... (100% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r10c5_4.svg">
        NZ_KZ679082 (6501508-6510863): Streptomyces sp. CS149 scaffold00001, whole ge... (100% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r10c5_5.svg">
        NZ_KZ819182 (6547055-6556386): Streptomyces sp. CS147 scaffold00001, whole ge... (100% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r10c5_6.svg">
        NZ_WWGD01000019 (307371-318709): Streptomyces sp. SID8378 SID8378.c19, whole ... (90% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r10c5_7.svg">
        NZ_LISY01000005 (408888-420331): Streptomyces sp. TSRI0261 scaffold5, whole g... (100% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r10c5_8.svg">
        NZ_LMEL01000002 (340679-352008): Streptomyces sp. Root1295 contig 10, whole g... (100% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r10c5_9.svg">
        NZ_KL573543 (113020-124370): Streptomyces baarnensis strain NRRL B-2842 P144 ... (100% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r10c5_10.svg">
        NZ_CP046905 (5913753-5925498): Streptomyces sp. QHH-9511 chromosome, complete... (100% of genes show similarity), bacteriocin
      </option>
      
    </select>
    <button id= "clusterblast-r10c5-download">Download graphic</button>
  </div>
  <div id="clusterblast-r10c5-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r10c5-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-211"></div> <span class="help-tooltip" id="cb-known-help-211">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r10c5-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-212"></div> <span class="help-tooltip" id="cb-sub-help-212">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r10c5-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-213"></div> <span class="help-tooltip" id="MIBiG-help-213">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r10c5-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r10c5-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r10c5"></div>
    <div id="comparison-MIBiG-r10c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">RiPP-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001607.1: 32186-60425">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001607/index.html#r1c1" target="_new">BGC0001607.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>gilvusmycin</td>
        
        
        <td>Streptomyces sp. NRRL B-1347</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001540.1: 633-38461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001540/index.html#r1c1" target="_new">BGC0001540.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>CC-1065</td>
        
        
        <td>Streptomyces zelensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001120.1: 2543311-2579913">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001120/index.html#r1c1" target="_new">BGC0001120.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>NRP, Polyketide</td>
        
        <td>burkholderic acid</td>
        
        
        <td>Burkholderia thailandensis E264</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001138.1: 0-39199">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001138/index.html#r1c1" target="_new">BGC0001138.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other,  , (, N, u, c, l, e, o, s, i, d, e, )</td>
        
        <td>a201a</td>
        
        
        <td>Marinactinospora thermotolerans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000698.1: 2269-33779">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000698/index.html#r1c1" target="_new">BGC0000698.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>hygromycin A</td>
        
        
        <td>Streptomyces hygroscopicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000019.1: 339-17245">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000019/index.html#r1c1" target="_new">BGC0000019.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>angolamycin</td>
        
        
        <td>Streptomyces eurythermus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000794.1: 492-19862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000794/index.html#r1c1" target="_new">BGC0000794.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>Saccharide</td>
        
        <td>S-layer glycan</td>
        
        
        <td>Geobacillus stearothermophilus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000569.1: 0-9892">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000569/index.html#r1c1" target="_new">BGC0000569.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
        <td>0.12</td>
        <td>RiPP</td>
        
        <td>plantazolicin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001173.1: 223360-233305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001173/index.html#r1c1" target="_new">BGC0001173.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
        <td>0.12</td>
        <td>RiPP</td>
        
        <td>plantazolicin</td>
        
        
        <td>Bacillus pumilus ATCC 7061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001803.1: 3663-28345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001803/index.html#r1c1" target="_new">BGC0001803.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.34 (id:0.43, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
         
        
        <td>0.09</td>
        <td>RiPP</td>
        
        <td>thioviridamide</td>
        
        
        <td>Streptomyces olivoviridis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r10c5-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001607.1: 32186-60425">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001607/index.html#r1c1" target="_new">BGC0001607.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>gilvusmycin</td>
      <td>Streptomyces sp. NRRL B-1347</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001540.1: 633-38461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001540/index.html#r1c1" target="_new">BGC0001540.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other</td>
      <td>CC-1065</td>
      <td>Streptomyces zelensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001120.1: 2543311-2579913">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001120/index.html#r1c1" target="_new">BGC0001120.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>NRP, Polyketide</td>
      <td>burkholderic acid</td>
      <td>Burkholderia thailandensis E264</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001138.1: 0-39199">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001138/index.html#r1c1" target="_new">BGC0001138.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other,  , (, N, u, c, l, e, o, s, i, d, e, )</td>
      <td>a201a</td>
      <td>Marinactinospora thermotolerans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000698.1: 2269-33779">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000698/index.html#r1c1" target="_new">BGC0000698.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Saccharide</td>
      <td>hygromycin A</td>
      <td>Streptomyces hygroscopicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000019.1: 339-17245">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000019/index.html#r1c1" target="_new">BGC0000019.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Polyketide</td>
      <td>angolamycin</td>
      <td>Streptomyces eurythermus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000794.1: 492-19862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000794/index.html#r1c1" target="_new">BGC0000794.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>Saccharide</td>
      <td>S-layer glycan</td>
      <td>Geobacillus stearothermophilus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000569.1: 0-9892">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000569/index.html#r1c1" target="_new">BGC0000569.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
      <td>0.40</td>
      <td>RiPP</td>
      <td>plantazolicin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001173.1: 223360-233305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001173/index.html#r1c1" target="_new">BGC0001173.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
      <td>0.40</td>
      <td>RiPP</td>
      <td>plantazolicin</td>
      <td>Bacillus pumilus ATCC 7061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001803.1: 3663-28345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001803/index.html#r1c1" target="_new">BGC0001803.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.34 (id:0.43, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
      <td>0.34</td>
      <td>RiPP</td>
      <td>thioviridamide</td>
      <td>Streptomyces olivoviridis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r10c5-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-214"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-214">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r10c5-pfam-details-svg" data-filename="c00010_CC216C_.._r10c5_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r10c5-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r12c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00012_CC216C_.. - Region 1 - lanthipeptide-class-ii,lanthipeptide-class-iii <div class="help-container"> <div class="help-icon" data-id="region_svg-help-303"></div> <span class="help-tooltip" id="region_svg-help-303">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00012_CC216C_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r12c1-svg" data-filename="c00012_CC216C_.._r12c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 14,730 - 44,793 nt. (total: 30,064 nt)
        
        <a class="cluster-rules-header" id="r12c1-rules-header" href="#r12c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r12c1-rules">lanthipeptide-class-ii: cds(LANC_like and DUF4135)<br>lanthipeptide-class-iii: (micKC and not PKS_KS)</div>
      
      <div class="region-svg-container">
       <div id='r12c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-304"></div> <span class="help-tooltip" id="focus-panel-help-304">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r12c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r12c1-lanthipeptides" data-name="r12c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r12c1-clusterblast" data-name="r12c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r12c1-knownclusterblast" data-name="r12c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r12c1-subclusterblast" data-name="r12c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r12c1-MIBiG-cluster-compare" data-name="r12c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r12c1-pfam-details" data-name="r12c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r12c1-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-215"></div> <span class="help-tooltip" id="lanthi-body-help-215">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lanthipeptide(s) for cds24729_27036 - Putative Class II</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds36621_36780</span> leader / core peptide
          </dt>
          <dd>
            MSESTKPAGDETNETPEVEAHSVLDLQETSVQGG
            -
            D<span class="dhb">Dhb</span><span class="dhb">Dhb</span>DGN<span class="cys">C</span>I<span class="dha">Dha</span>VL<span class="dha">Dha</span>VVEQQR
          <dt>
            <span class="serif">cds36621_36780</span> leader / core peptide
          </dt>
          <dd>
            MSESTKPAGDETNETPEVEAHSVLDLQETSVQGG
            -
            D<span class="dhb">Dhb</span><span class="dhb">Dhb</span>DGN<span class="cys">C</span>I<span class="dha">Dha</span>VL<span class="dha">Dha</span>VVEQQR
        </dd>
      </dl>
    
      <h4>Lanthipeptide(s) for cds32507_34793 - Putative Class II</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds36621_36780</span> leader / core peptide
          </dt>
          <dd>
            MSESTKPAGDETNETPEVEAHSVLDLQETSVQGG
            -
            D<span class="dhb">Dhb</span><span class="dhb">Dhb</span>DGN<span class="cys">C</span>I<span class="dha">Dha</span>VL<span class="dha">Dha</span>VVEQQR
          <dt>
            <span class="serif">cds36621_36780</span> leader / core peptide
          </dt>
          <dd>
            MSESTKPAGDETNETPEVEAHSVLDLQETSVQGG
            -
            D<span class="dhb">Dhb</span><span class="dhb">Dhb</span>DGN<span class="cys">C</span>I<span class="dha">Dha</span>VL<span class="dha">Dha</span>VVEQQR
        </dd>
      </dl>
    
    <div class="legend">
      <h4> Legend: </h4>
      <div>
        <span class="dha">Dha</span>: Didehydroalanine<br>
        <span class="dhb">Dhb</span>: Didehydrobutyrine<br>
      </div>
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r12c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-217"></div> <span class="help-tooltip" id="cb-general-help-217">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r12c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r12c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r12c1_1.svg">
        NZ_CP030930 (2350735-2391954): Streptomyces cavourensis strain TJ430 chromoso... (51% of genes show similarity), lanthipeptide,lipolanthine
      </option>
      
      <option value = "svg/clusterblast_r12c1_2.svg">
        NZ_CP011522 (5824121-5866769): Streptomyces sp. CFMR 7 strain CFMR-7 chromoso... (45% of genes show similarity), lanthipeptide,lipolanthine
      </option>
      
      <option value = "svg/clusterblast_r12c1_3.svg">
        NZ_LACR01000026 (105134-128164): Streptomyces sp. Ncost-T6T-1 YW5DRAFT scaffo... (87% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r12c1_4.svg">
        NZ_NNBN01000010 (156597-179642): Streptomyces sp. CB02613 scaffold10, whole g... (82% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r12c1_5.svg">
        NZ_JNLT01000008 (18089-41134): Streptomyces sp. CNS654 CD02DRAFT scaffold0000... (77% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r12c1_6.svg">
        NZ_KZ819154 (5241225-5264279): Streptomyces sp. CS014 scaffold00001, whole ge... (87% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r12c1_7.svg">
        NZ_LIXJ01000004 (205148-252968): Streptomyces sp. CB01580 scaffold4, whole ge... (42% of genes show similarity), lanthipeptide,lipolanthine
      </option>
      
      <option value = "svg/clusterblast_r12c1_8.svg">
        NZ_ML123127 (129140-204516): Streptomyces sp. ADI98-12 scaffold002, whole gen... (23% of genes show similarity), NRPS,T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r12c1_9.svg">
        NZ_CP056773 (6686330-6958067): Streptomyces violascens strain YIM 100212 chro... (12% of genes show similarity), NRPS,NRPS-like,T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r12c1_10.svg">
        NZ_CM002273 (6555444-6843909): Streptomyces sp. PVA 94-07 strain PVA 94-07 ch... (11% of genes show similarity), NRPS,NRPS-like,T1PKS,T3PKS,lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r12c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r12c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r12c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-218"></div> <span class="help-tooltip" id="cb-known-help-218">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r12c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-219"></div> <span class="help-tooltip" id="cb-sub-help-219">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r12c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-220"></div> <span class="help-tooltip" id="MIBiG-help-220">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r12c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r12c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r12c1"></div>
    <div id="comparison-MIBiG-r12c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-ii</span></th>
         
         <th><span class="cc-heat-product">lanthipeptide-class-iii</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.61</td>
        <td>RiPP</td>
        
        <td>informatipeptin</td>
        
        
        <td>Streptomyces viridochromogenes DSM 40736</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.60</td>
        <td>RiPP</td>
        
        <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
        
        
        <td>Actinomadura namibiensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.59</td>
        <td>RiPP</td>
        
        <td>SapB</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.58</td>
        <td>RiPP</td>
        
        <td>catenulipeptin</td>
        
        
        <td>Catenulispora acidiphila DSM 44928</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.54</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000525.1: 36-3837">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000525/index.html#r1c1" target="_new">BGC0000525.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.51</td>
        <td>RiPP</td>
        
        <td>lichenicidin VK21 A1, lichenicidin VK21 A2</td>
        
        
        <td>Bacillus licheniformis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000502.1: 0-15725">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000502/index.html#r1c1" target="_new">BGC0000502.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.44</td>
        <td>RiPP</td>
        
        <td>cerecidin, cerecidin A1, cerecidin A2, cerecidin A3, cerecidin A4, cerecidin A5, cerecidin A6, cerecidin A7</td>
        
        
        <td>Bacillus cereus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000546.1: 4112716-4120692">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000546/index.html#r1c1" target="_new">BGC0000546.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.48, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.48, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.42</td>
        <td>RiPP</td>
        
        <td>SAL-2242</td>
        
        
        <td>Streptomyces albidoflavus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000495.1: 41-38168">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000495/index.html#r1c1" target="_new">BGC0000495.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.51)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.44, order:N/A, components:0.49)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.32</td>
        <td>RiPP</td>
        
        <td>actagardine</td>
        
        
        <td>Actinoplanes garbadinensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000513.1: 4704393-4710596">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000513/index.html#r1c1" target="_new">BGC0000513.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.32</td>
        <td>RiPP</td>
        
        <td>Ery-9, Ery-6, Ery-8, Ery-7, Ery-5, Ery-4, Ery-3</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r12c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.47, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>RiPP</td>
      <td>informatipeptin</td>
      <td>Streptomyces viridochromogenes DSM 40736</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>RiPP</td>
      <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
      <td>Actinomadura namibiensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>RiPP</td>
      <td>SapB</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>RiPP</td>
      <td>catenulipeptin</td>
      <td>Catenulispora acidiphila DSM 44928</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000525.1: 36-3837">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000525/index.html#r1c1" target="_new">BGC0000525.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>RiPP</td>
      <td>lichenicidin VK21 A1, lichenicidin VK21 A2</td>
      <td>Bacillus licheniformis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000502.1: 0-15725">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000502/index.html#r1c1" target="_new">BGC0000502.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>RiPP</td>
      <td>cerecidin, cerecidin A1, cerecidin A2, cerecidin A3, cerecidin A4, cerecidin A5, cerecidin A6, cerecidin A7</td>
      <td>Bacillus cereus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000546.1: 4112716-4120692">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000546/index.html#r1c1" target="_new">BGC0000546.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.48, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>RiPP</td>
      <td>SAL-2242</td>
      <td>Streptomyces albidoflavus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843.1: 3437627-3461293">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Other,  , (, N, o, n, -, N, R, P,  , b, e, t, a, -, l, a, c, t, a, m, )</td>
      <td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000495.1: 41-38168">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000495/index.html#r1c1" target="_new">BGC0000495.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>RiPP</td>
      <td>actagardine</td>
      <td>Actinoplanes garbadinensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r12c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-221"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-221">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r12c1-pfam-details-svg" data-filename="c00012_CC216C_.._r12c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r12c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r12c1-lanthipeptides" data-name="r12c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r12c1-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-216"></div> <span class="help-tooltip" id="lanthi-side-help-216">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

    <dl class ="prediction-text">
    <dt><span class="serif">cds36621_36780</span> - Class II</dt>
        <dd>Cleavage pHMM score: -4.20</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 1890.9 Da</dd>
        <dd>Molecular weight: 1892.0 Da</dd>
        <dd>Number of bridges: 1</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 1910.1 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 1928.1 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 1946.1 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
            </dl>
          </dd>
        <dt><span class="serif">cds36621_36780</span> - Class II</dt>
        <dd>Cleavage pHMM score: -4.20</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 1890.9 Da</dd>
        <dd>Molecular weight: 1892.0 Da</dd>
        <dd>Number of bridges: 1</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 1910.1 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 1928.1 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 1946.1 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
            </dl>
          </dd>
        <dt><span class="serif">cds36621_36780</span> - Class II</dt>
        <dd>Cleavage pHMM score: -4.20</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 1890.9 Da</dd>
        <dd>Molecular weight: 1892.0 Da</dd>
        <dd>Number of bridges: 1</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 1910.1 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 1928.1 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 1946.1 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
            </dl>
          </dd>
        <dt><span class="serif">cds36621_36780</span> - Class II</dt>
        <dd>Cleavage pHMM score: -4.20</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 1890.9 Da</dd>
        <dd>Molecular weight: 1892.0 Da</dd>
        <dd>Number of bridges: 1</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 1910.1 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 1928.1 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 1946.1 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
            </dl>
          </dd>
        
  </dl>

</div>

    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r12c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00012_CC216C_.. - Region 2 - siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-305"></div> <span class="help-tooltip" id="region_svg-help-305">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00012_CC216C_...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r12c2-svg" data-filename="c00012_CC216C_.._r12c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 121,349 - 133,127 nt. (total: 11,779 nt)
        
        <a class="cluster-rules-header" id="r12c2-rules-header" href="#r12c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r12c2-rules">siderophore: IucA_IucC</div>
      
      <div class="region-svg-container">
       <div id='r12c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-306"></div> <span class="help-tooltip" id="focus-panel-help-306">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r12c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r12c2-clusterblast" data-name="r12c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r12c2-knownclusterblast" data-name="r12c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r12c2-subclusterblast" data-name="r12c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r12c2-MIBiG-cluster-compare" data-name="r12c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r12c2-pfam-details" data-name="r12c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r12c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-222"></div> <span class="help-tooltip" id="cb-general-help-222">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r12c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r12c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r12c2_1.svg">
        NZ_NNBN01000010 (42908-93378): Streptomyces sp. CB02613 scaffold10, whole gen... (29% of genes show similarity), NRPS,betalactone,siderophore
      </option>
      
      <option value = "svg/clusterblast_r12c2_2.svg">
        NZ_JNLT01000008 (127916-139695): Streptomyces sp. CNS654 CD02DRAFT scaffold00... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r12c2_3.svg">
        NZ_KZ819178 (2579139-2589580): Streptomyces sp. CS131 scaffold00001, whole ge... (90% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r12c2_4.svg">
        NZ_KZ819182 (2661129-2671129): Streptomyces sp. CS147 scaffold00001, whole ge... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r12c2_5.svg">
        NZ_WWGD01000003 (258338-270117): Streptomyces sp. SID8378 SID8378.c3, whole g... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r12c2_6.svg">
        NZ_LACR01000026 (20590-32369): Streptomyces sp. Ncost-T6T-1 YW5DRAFT scaffold... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r12c2_7.svg">
        NZ_KZ819154 (5140716-5152495): Streptomyces sp. CS014 scaffold00001, whole ge... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r12c2_8.svg">
        NZ_WWIP01000001 (2418526-2428540): Streptomyces sp. SID5466 SID5466.c1, whole... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r12c2_9.svg">
        NZ_CP010833 (5283917-5295696): Streptomyces sp. Tue 6075 strain Tue6075 chrom... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r12c2_10.svg">
        NZ_KZ679082 (2578642-2588657): Streptomyces sp. CS149 scaffold00001, whole ge... (100% of genes show similarity), siderophore
      </option>
      
    </select>
    <button id= "clusterblast-r12c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r12c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r12c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-223"></div> <span class="help-tooltip" id="cb-known-help-223">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r12c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r12c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r12c2_1.svg">
        desferrioxamin B
      </option>
      
      <option value = "svg/knownclusterblast_r12c2_2.svg">
        desferrioxamin B / desferrioxamine E
      </option>
      
      <option value = "svg/knownclusterblast_r12c2_3.svg">
        desferrioxamine
      </option>
      
      <option value = "svg/knownclusterblast_r12c2_4.svg">
        desferrioxamine E
      </option>
      
      <option value = "svg/knownclusterblast_r12c2_5.svg">
        putrebactin / avaroferrin
      </option>
      
      <option value = "svg/knownclusterblast_r12c2_6.svg">
        desferrioxamine E
      </option>
      
      <option value = "svg/knownclusterblast_r12c2_7.svg">
        bisucaberin B
      </option>
      
    </select>
    <button id="knownclusterblast-r12c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r12c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r12c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-224"></div> <span class="help-tooltip" id="cb-sub-help-224">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r12c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-225"></div> <span class="help-tooltip" id="MIBiG-help-225">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r12c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r12c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r12c2"></div>
    <div id="comparison-MIBiG-r12c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">siderophore</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000941.1: 5578821-5584948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000941/index.html#r1c1" target="_new">BGC0000941.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.89 (id:0.70, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.89), rgb(0, 0, 0, 0.89) 89%, white 89%)"></td>
         
        
        <td>0.70</td>
        <td>Other</td>
        
        <td>desferrioxamin B</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001478.1: 100-5365">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001478/index.html#r1c1" target="_new">BGC0001478.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.83 (id:0.58, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
         
        
        <td>0.58</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Streptomyces sp. ID38640</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940.1: 3033354-3040682">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.76 (id:0.58, order:0.83, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
         
        
        <td>0.45</td>
        <td>Other</td>
        
        <td>desferrioxamin B, desferrioxamine E</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453.1: 0-7234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.75 (id:0.57, order:0.83, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
         
        
        <td>0.45</td>
        <td>Other</td>
        
        <td>desferrioxamine</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.52, order:0.66, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Pantoea agglomerans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001531.1: 1112-9953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001531/index.html#r1c1" target="_new">BGC0001531.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.49, order:0.48, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>Other</td>
        
        <td>bisucaberin B</td>
        
        
        <td>Tenacibaculum mesophilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>ochrobactin</td>
        
        
        <td>Ochrobactrum anthropi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499.1: 2267-11578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:0.43, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>aerobactin</td>
        
        
        <td>Pantoea ananatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000244.1: 204-8850">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000244/index.html#r1c1" target="_new">BGC0000244.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.46, order:0.48, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>macrotetrolide</td>
        
        
        <td>Streptomyces griseus subsp. griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001607.1: 32186-60425">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001607/index.html#r1c1" target="_new">BGC0001607.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>gilvusmycin</td>
        
        
        <td>Streptomyces sp. NRRL B-1347</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r12c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000941.1: 5578821-5584948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000941/index.html#r1c1" target="_new">BGC0000941.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.89 (id:0.70, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.89), rgb(0, 0, 0, 0.89) 89%, white 89%)"></td>
      <td>0.89</td>
      <td>Other</td>
      <td>desferrioxamin B</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001478.1: 100-5365">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001478/index.html#r1c1" target="_new">BGC0001478.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.83 (id:0.58, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
      <td>0.83</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Streptomyces sp. ID38640</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940.1: 3033354-3040682">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.76 (id:0.58, order:0.83, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
      <td>0.76</td>
      <td>Other</td>
      <td>desferrioxamin B, desferrioxamine E</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453.1: 0-7234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.75 (id:0.57, order:0.83, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
      <td>0.75</td>
      <td>Other</td>
      <td>desferrioxamine</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.52, order:0.66, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Pantoea agglomerans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001531.1: 1112-9953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001531/index.html#r1c1" target="_new">BGC0001531.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.49, order:0.48, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Other</td>
      <td>bisucaberin B</td>
      <td>Tenacibaculum mesophilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>ochrobactin</td>
      <td>Ochrobactrum anthropi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499.1: 2267-11578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:0.43, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>aerobactin</td>
      <td>Pantoea ananatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000244.1: 204-8850">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000244/index.html#r1c1" target="_new">BGC0000244.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.46, order:0.48, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Polyketide</td>
      <td>macrotetrolide</td>
      <td>Streptomyces griseus subsp. griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001607.1: 32186-60425">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001607/index.html#r1c1" target="_new">BGC0001607.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Other</td>
      <td>gilvusmycin</td>
      <td>Streptomyces sp. NRRL B-1347</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r12c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-226"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-226">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r12c2-pfam-details-svg" data-filename="c00012_CC216C_.._r12c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r12c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r14c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00014_CC216C_.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-307"></div> <span class="help-tooltip" id="region_svg-help-307">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00014_CC216C_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r14c1-svg" data-filename="c00014_CC216C_.._r14c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 125,915 - 146,991 nt. (total: 21,077 nt)
        
        <a class="cluster-rules-header" id="r14c1-rules-header" href="#r14c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r14c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r14c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-308"></div> <span class="help-tooltip" id="focus-panel-help-308">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r14c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r14c1-clusterblast" data-name="r14c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r14c1-knownclusterblast" data-name="r14c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r14c1-subclusterblast" data-name="r14c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r14c1-MIBiG-cluster-compare" data-name="r14c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r14c1-pfam-details" data-name="r14c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r14c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-227"></div> <span class="help-tooltip" id="cb-general-help-227">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r14c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r14c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r14c1_1.svg">
        NZ_JNLT01000019 (86140-107217): Streptomyces sp. CNS654 CD02DRAFT scaffold000... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r14c1_2.svg">
        NZ_WWGD01000020 (370214-391291): Streptomyces sp. SID8378 SID8378.c20, whole ... (93% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r14c1_3.svg">
        NZ_KZ819182 (1130715-1150271): Streptomyces sp. CS147 scaffold00001, whole ge... (93% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r14c1_4.svg">
        NZ_KZ679082 (1043567-1062377): Streptomyces sp. CS149 scaffold00001, whole ge... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r14c1_5.svg">
        NZ_LACR01000020 (93356-114433): Streptomyces sp. Ncost-T6T-1 YW5DRAFT scaffol... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r14c1_6.svg">
        NZ_NNBN01000006 (390653-411730): Streptomyces sp. CB02613 scaffold6, whole ge... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r14c1_7.svg">
        NZ_KZ819154 (6722180-6742039): Streptomyces sp. CS014 scaffold00001, whole ge... (88% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r14c1_8.svg">
        NZ_KZ268499 (1035616-1054410): Streptomyces sp. SS07 Scaffold1, whole genome ... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r14c1_9.svg">
        NZ_LZRD01000024 (69342-90419): Streptomyces sp. PTY087I2 A3Q37 contig000024, ... (100% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r14c1_10.svg">
        NZ_LIPC01000007 (360005-381082): Streptomyces sp. TSRI0445 scaffold7, whole g... (93% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r14c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r14c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r14c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-228"></div> <span class="help-tooltip" id="cb-known-help-228">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r14c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r14c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r14c1_1.svg">
        steffimycin D
      </option>
      
      <option value = "svg/knownclusterblast_r14c1_2.svg">
        BE-7585A
      </option>
      
      <option value = "svg/knownclusterblast_r14c1_3.svg">
        perquinoline A / perquinoline B / perquinoline C
      </option>
      
      <option value = "svg/knownclusterblast_r14c1_4.svg">
        mediomycin A
      </option>
      
      <option value = "svg/knownclusterblast_r14c1_5.svg">
        mediomiycin A
      </option>
      
    </select>
    <button id="knownclusterblast-r14c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r14c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r14c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-229"></div> <span class="help-tooltip" id="cb-sub-help-229">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r14c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-230"></div> <span class="help-tooltip" id="MIBiG-help-230">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r14c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r14c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r14c1"></div>
    <div id="comparison-MIBiG-r14c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Terpene</td>
        
        <td>geosmin</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000879.1: 1167-12985">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000879/index.html#r1c1" target="_new">BGC0000879.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Other</td>
        
        <td>toyocamycin, sangivamycin</td>
        
        
        <td>Streptomyces rimosus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514.1: 0-37469">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>aristeromycin</td>
        
        
        <td>Streptomyces citricolor</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631.1: 2217593-2245275">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.42, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Terpene</td>
        
        <td>botrydial</td>
        
        
        <td>Botrytis cinerea B05.10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001120.1: 2543311-2579913">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001120/index.html#r1c1" target="_new">BGC0001120.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>NRP, Polyketide</td>
        
        <td>burkholderic acid</td>
        
        
        <td>Burkholderia thailandensis E264</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000772.1: 960-13404">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000772/index.html#r1c1" target="_new">BGC0000772.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>lipopolysaccharide</td>
        
        
        <td>Brucella melitensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000881.1: 83-10493">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000881/index.html#r1c1" target="_new">BGC0000881.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.47, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>toyocamycin</td>
        
        
        <td>Streptomyces ahygroscopicus subsp. wuzhouensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001808.1: 0-10410">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001808/index.html#r1c1" target="_new">BGC0001808.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.47, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>toyocamycin</td>
        
        
        <td>Streptomyces diastatochromogenes</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001607.1: 32186-60425">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001607/index.html#r1c1" target="_new">BGC0001607.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Other</td>
        
        <td>gilvusmycin</td>
        
        
        <td>Streptomyces sp. NRRL B-1347</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001937.1: 4366-12776">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001937/index.html#r1c1" target="_new">BGC0001937.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.47, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
        <td>0.12</td>
        <td>Other</td>
        
        <td>tubercidin</td>
        
        
        <td>Streptomyces tubercidicus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r14c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>geosmin</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000879.1: 1167-12985">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000879/index.html#r1c1" target="_new">BGC0000879.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Other</td>
      <td>toyocamycin, sangivamycin</td>
      <td>Streptomyces rimosus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514.1: 0-37469">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Other</td>
      <td>aristeromycin</td>
      <td>Streptomyces citricolor</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631.1: 2217593-2245275">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.42, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Terpene</td>
      <td>botrydial</td>
      <td>Botrytis cinerea B05.10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001120.1: 2543311-2579913">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001120/index.html#r1c1" target="_new">BGC0001120.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>NRP, Polyketide</td>
      <td>burkholderic acid</td>
      <td>Burkholderia thailandensis E264</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000772.1: 960-13404">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000772/index.html#r1c1" target="_new">BGC0000772.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>lipopolysaccharide</td>
      <td>Brucella melitensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000881.1: 83-10493">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000881/index.html#r1c1" target="_new">BGC0000881.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.47, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>toyocamycin</td>
      <td>Streptomyces ahygroscopicus subsp. wuzhouensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001808.1: 0-10410">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001808/index.html#r1c1" target="_new">BGC0001808.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.47, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>toyocamycin</td>
      <td>Streptomyces diastatochromogenes</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001607.1: 32186-60425">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001607/index.html#r1c1" target="_new">BGC0001607.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Other</td>
      <td>gilvusmycin</td>
      <td>Streptomyces sp. NRRL B-1347</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001937.1: 4366-12776">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001937/index.html#r1c1" target="_new">BGC0001937.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.40 (id:0.47, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
      <td>0.40</td>
      <td>Other</td>
      <td>tubercidin</td>
      <td>Streptomyces tubercidicus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r14c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-231"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-231">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r14c1-pfam-details-svg" data-filename="c00014_CC216C_.._r14c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r14c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r14c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00014_CC216C_.. - Region 2 - T2PKS,oligosaccharide <div class="help-container"> <div class="help-icon" data-id="region_svg-help-309"></div> <span class="help-tooltip" id="region_svg-help-309">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00014_CC216C_...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r14c2-svg" data-filename="c00014_CC216C_.._r14c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 267,921 - 340,394 nt. (total: 72,474 nt)
        
        <a class="cluster-rules-header" id="r14c2-rules-header" href="#r14c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r14c2-rules">T2PKS: (t2ks and t2clf)<br>oligosaccharide: (minimum(3, [DUF1205, Glyco_transf_28, Glycos_transf_1, Glycos_transf_2, MGT]) and minscore(MGT, 150))</div>
      
      <div class="region-svg-container">
       <div id='r14c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-310"></div> <span class="help-tooltip" id="focus-panel-help-310">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r14c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r14c2-nrps_pks" data-name="r14c2-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r14c2-clusterblast" data-name="r14c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r14c2-knownclusterblast" data-name="r14c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r14c2-subclusterblast" data-name="r14c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r14c2-MIBiG-cluster-compare" data-name="r14c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r14c2-pfam-details" data-name="r14c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r14c2-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-232"></div> <span class="help-tooltip" id="nrps-domain-help-232">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r14c2-details-svg" data-filename="c00014_CC216C_.._r14c2_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r14c2-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r14c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-233"></div> <span class="help-tooltip" id="cb-general-help-233">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r14c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r14c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r14c2_1.svg">
        NZ_WUEJ01000002 (617356-689890): Streptomyces sp. YIM 132580 Scaffold2, whole... (55% of genes show similarity), T2PKS,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r14c2_2.svg">
        NZ_CP032543 (1980890-2053424): Streptomyces griseus subsp. griseus strain ATC... (56% of genes show similarity), T2PKS,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r14c2_3.svg">
        NZ_CP045178 (2334672-2428158): Streptomyces tsukubensis strain AT3 chromosome... (42% of genes show similarity), T2PKS,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r14c2_4.svg">
        NZ_FONG01000004 (328797-401295): Streptomyces alni strain CGMCC 4.3510, whole... (30% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r14c2_5.svg">
        NZ_WWGC01000020 (240311-319146): Streptomyces sp. SID8379 SID8379.c20, whole ... (33% of genes show similarity), LAP,PKS-like,T2PKS,betalactone,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r14c2_6.svg">
        NZ_MLCE01000002 (311510-383999): Streptomyces sp. Tue6028 Scaffold1, whole ge... (35% of genes show similarity), PKS-like,T1PKS,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r14c2_7.svg">
        NC_016582 (8115649-8238974): Streptomyces bingchenggensis BCW-1, complete seq... (25% of genes show similarity), NRPS,PKS-like,T2PKS,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r14c2_8.svg">
        NZ_BEWA01000010 (94716-193686): Streptomyces rishiriensis strain NBRC 13407, ... (33% of genes show similarity), NRPS,PKS-like,T2PKS,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r14c2_9.svg">
        NZ_VISR01000029 (51811-142181): Streptomyces sp. BK340 Ga0307703 129, whole g... (33% of genes show similarity), NRPS,PKS-like,T2PKS,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r14c2_10.svg">
        NZ_CP016279 (10479183-10569348): Streptomyces griseochromogenes strain ATCC 1... (32% of genes show similarity), NRPS,PKS-like,T2PKS,oligosaccharide
      </option>
      
    </select>
    <button id= "clusterblast-r14c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r14c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r14c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-234"></div> <span class="help-tooltip" id="cb-known-help-234">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r14c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r14c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r14c2_1.svg">
        chromomycin A3
      </option>
      
      <option value = "svg/knownclusterblast_r14c2_2.svg">
        mithramycin
      </option>
      
      <option value = "svg/knownclusterblast_r14c2_3.svg">
        polyketomycin
      </option>
      
      <option value = "svg/knownclusterblast_r14c2_4.svg">
        dutomycin
      </option>
      
      <option value = "svg/knownclusterblast_r14c2_5.svg">
        mithramycin
      </option>
      
      <option value = "svg/knownclusterblast_r14c2_6.svg">
        dactylocycline A
      </option>
      
      <option value = "svg/knownclusterblast_r14c2_7.svg">
        antibiotic HKI 10311129 / antibiotic A2121-1 / cervimycin D / cervimycin C / cervimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r14c2_8.svg">
        SF2575
      </option>
      
      <option value = "svg/knownclusterblast_r14c2_9.svg">
        chlortetracycline
      </option>
      
      <option value = "svg/knownclusterblast_r14c2_10.svg">
        grincamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r14c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r14c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r14c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-235"></div> <span class="help-tooltip" id="cb-sub-help-235">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r14c2-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r14c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r14c2_1.svg">
        FJ483966_c1: polyketomycin t2pks (87% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r14c2_2.svg">
        AF324838_c1: simocyclinone t2pks (50% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r14c2_3.svg">
        AF080235_c1: landomycin t2pks (50% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r14c2_4.svg">
        GU233672_c1: granaticin t2pks (42% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r14c2_5.svg">
        DQ116941_c2: chlorothricin D-olivose (71% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r14c2_6.svg">
        AB103463_c1: medermycin t2pks (50% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r14c2_7.svg">
        FJ483966_c3: polyketomycin beta-D-amicetose / alpha-L-axenose (45% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r14c2_8.svg">
        FN433113_c2: streptolydigin L-rhodinose (62% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r14c2_9.svg">
        AF322256_c1: simocyclinone dTDP-D-olivose (57% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r14c2_10.svg">
        AF080235_c2: landomycin D-olivose / L-rhodinose (33% of genes show similarity), sugar
      </option>
      
    </select>
    <button id="subclusterblast-r14c2-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r14c2-svg"></div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r14c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-237"></div> <span class="help-tooltip" id="MIBiG-help-237">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r14c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r14c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r14c2"></div>
    <div id="comparison-MIBiG-r14c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T2PKS</span></th>
         
         <th><span class="cc-heat-product">oligosaccharide</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000210.1: 7-41948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000210/index.html#r1c1" target="_new">BGC0000210.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.80 (id:0.57, order:0.94, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.57, order:0.82, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.92</td>
        <td>Polyketide, Saccharide</td>
        
        <td>chromomycin A3</td>
        
        
        <td>Streptomyces griseus subsp. griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.53, order:0.59, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.52, order:0.43, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.58</td>
        <td>Polyketide</td>
        
        <td>naphtocyclinone</td>
        
        
        <td>Streptomyces arenae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000716.1: 435-27161">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000716/index.html#r1c1" target="_new">BGC0000716.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.54, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.54, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.57</td>
        <td>Saccharide</td>
        
        <td>spectinomycin</td>
        
        
        <td>Streptomyces netropsis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000715.1: 0-17984">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000715/index.html#r1c1" target="_new">BGC0000715.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.53, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.53, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.57</td>
        <td>Saccharide</td>
        
        <td>actinospectacin</td>
        
        
        <td>Streptomyces spectabilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.53, order:0.52, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.53, order:0.48, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.50</td>
        <td>Polyketide</td>
        
        <td>granaticin</td>
        
        
        <td>Streptomyces violaceoruber</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.48, order:0.44, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.48, order:0.41, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.48</td>
        <td>Polyketide</td>
        
        <td>spore pigment</td>
        
        
        <td>Streptomyces collinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.52, order:0.45, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.52, order:0.45, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.48</td>
        <td>Polyketide</td>
        
        <td>curamycin</td>
        
        
        <td>Streptomyces cyaneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001225.1: 5267-32621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001225/index.html#r1c1" target="_new">BGC0001225.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.46</td>
        <td>NRP</td>
        
        <td>celesticetin</td>
        
        
        <td>Streptomyces caelestis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.51, order:0.45, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.51, order:0.36, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.44</td>
        <td>Polyketide</td>
        
        <td>griseusin</td>
        
        
        <td>Streptomyces griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000692.1: 0-39979">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000692/index.html#r1c1" target="_new">BGC0000692.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.44</td>
        <td>Saccharide</td>
        
        <td>apramycin</td>
        
        
        <td>Streptoalloteichus hindustanus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r14c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000210.1: 7-41948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000210/index.html#r1c1" target="_new">BGC0000210.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.80 (id:0.57, order:0.94, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
      <td>0.80</td>
      <td>Polyketide, Saccharide</td>
      <td>chromomycin A3</td>
      <td>Streptomyces griseus subsp. griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.53, order:0.59, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Polyketide</td>
      <td>naphtocyclinone</td>
      <td>Streptomyces arenae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000716.1: 435-27161">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000716/index.html#r1c1" target="_new">BGC0000716.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.54, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Saccharide</td>
      <td>spectinomycin</td>
      <td>Streptomyces netropsis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000715.1: 0-17984">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000715/index.html#r1c1" target="_new">BGC0000715.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.53, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Saccharide</td>
      <td>actinospectacin</td>
      <td>Streptomyces spectabilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.53, order:0.52, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>granaticin</td>
      <td>Streptomyces violaceoruber</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.48, order:0.44, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>spore pigment</td>
      <td>Streptomyces collinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.52, order:0.45, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>curamycin</td>
      <td>Streptomyces cyaneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.51, order:0.45, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>griseusin</td>
      <td>Streptomyces griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000410.1: 162-16011">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000410/index.html#r1c1" target="_new">BGC0000410.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>pseudomonine</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001225.1: 5267-32621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001225/index.html#r1c1" target="_new">BGC0001225.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>celesticetin</td>
      <td>Streptomyces caelestis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r14c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-238"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-238">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r14c2-pfam-details-svg" data-filename="c00014_CC216C_.._r14c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r14c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
      <div class="sidepanel-details-header r14c2-t2pks" data-name="r14c2-t2pks">
        Type II PKS
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    <div class="sidepanel-details-section r14c2-t2pks">
      <div class="more-details">
    <div class="heading">
      <span>Type II PKS gene cluster prediction details</span>
      <div class="help-container"> <div class="help-icon" data-id="t2pks-help-236"></div> <span class="help-tooltip" id="t2pks-help-236">Predictions of starter units, elongations, product classes, and potential molecular weights for type II PKS clusters.<br>More detailed information is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/t2pks' target='_blank'>here</a>.</span></div>
    </div>
	
	
	<dl class="prediction-text">
        <dt><strong>Cluster section:</strong></dt>
        <dd>267920 to 340394</dd>
		
		<dt><strong>Starter unit(s):</strong></dt>
		<dd>
				malonamyl-CoA (Score: 422.0; E-value: 1.32e-144)<br>
			
	    <dd>
		</tr>
		
		
		<dt><strong>Malonyl elongations:</strong></dt>
        <dd>
			
			8|9 (Score: 626.5; E-value: 3.9e-191)<br>
			
    	</dd>
		
		
		<dt><strong>Product class(es):</strong></td>
        <dd>
				
				anthracycline<br>
				
				aureolic acid<br>
				
				tetracycline<br>
				
		</dd>
		
		
		<dt><strong>Product molecular weight(s):</strong></dt>
		<dd>
			
			malonamyl-CoA_8: 1174.11 Da<br>
			
			malonamyl-CoA_9: 1216.14 Da<br>
			
		</dd>
		
	</dl>
	<dl class="prediction-text">
		
		<dt><strong>cds272637_274287</strong></dt>
			
			<dd>
				LIG
				
				(Score: 517.1; E-value: 9.6e-158)<br>
			</dd>
			
			<dd>
				LIG
				
					ELQ83297.1_Oxytetracycline_malonamyl-CoA
				
				(Score: 422.0; E-value: 1.32e-144)<br>
			</dd>
			
		
		<dt><strong>cds274276_275068</strong></dt>
			
			<dd>
				CYC
				
					C7-C12
				
				(Score: 300.6; E-value: 2.4e-92)<br>
			</dd>
			
		
		<dt><strong>cds275049_277731</strong></dt>
			
			<dd>
				OXY
				
				(Score: 401.6; E-value: 1.3e-122)<br>
			</dd>
			
		
		<dt><strong>cds278319_279501</strong></dt>
			
			<dd>
				GT
				
				(Score: 394.2; E-value: 1.9e-120)<br>
			</dd>
			
		
		<dt><strong>cds279647_280871</strong></dt>
			
			<dd>
				GT
				
				(Score: 420.1; E-value: 2.5e-128)<br>
			</dd>
			
		
		<dt><strong>cds282247_283456</strong></dt>
			
			<dd>
				GT
				
				(Score: 349.2; E-value: 9.1e-107)<br>
			</dd>
			
		
		<dt><strong>cds293036_294545</strong></dt>
			
			<dd>
				OXY
				
				(Score: 492.5; E-value: 3.3e-150)<br>
			</dd>
			
		
		<dt><strong>cds294557_295730</strong></dt>
			
			<dd>
				GT
				
				(Score: 391.2; E-value: 1.6e-119)<br>
			</dd>
			
		
		<dt><strong>cds295770_296796</strong></dt>
			
			<dd>
				MET
				
					C2O|C2N
				
				(Score: 501.6; E-value: 2.2e-153)<br>
			</dd>
			
		
		<dt><strong>cds297570_298599</strong></dt>
			
			<dd>
				MET
				
					C10
				
				(Score: 532.8; E-value: 6.4e-163)<br>
			</dd>
			
		
		<dt><strong>cds299944_300703</strong></dt>
			
			<dd>
				MET
				
				(Score: 22.1; E-value: 1.6e-07)<br>
			</dd>
			
		
		<dt><strong>cds302920_304189</strong></dt>
			
			<dd>
				KS
				
				(Score: 750.7; E-value: 8.8e-229)<br>
			</dd>
			
		
		<dt><strong>cds304185_305394</strong></dt>
			
			<dd>
				CLF
				
					8|9
				
				(Score: 626.5; E-value: 3.9e-191)<br>
			</dd>
			
		
		<dt><strong>cds305610_306042</strong></dt>
			
			<dd>
				CYC
				
					C1-C18|C2-C19
				
				(Score: 200.2; E-value: 2.9e-62)<br>
			</dd>
			
		
		<dt><strong>cds306074_307358</strong></dt>
			
			<dd>
				OXY
				
				(Score: 169.7; E-value: 2.7e-52)<br>
			</dd>
			
		
		<dt><strong>cds310669_310930</strong></dt>
			
			<dd>
				ACP
				
				(Score: 113.3; E-value: 9.5e-36)<br>
			</dd>
			
		
		<dt><strong>cds310926_311685</strong></dt>
			
			<dd>
				KR
				
				(Score: 115.2; E-value: 9e-36)<br>
			</dd>
			
		
		<dt><strong>cds311747_312521</strong></dt>
			
			<dd>
				CYC
				
					C5-C14/C3-C16
				
				(Score: 412.0; E-value: 2.2e-126)<br>
			</dd>
			
		
	</dl>
    
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>


  <footer class="footer">
    <div class="container">
      <div>
        <img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
      </div>
      <div class="cite-me">
        If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
      </div>
      <div>
        <img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
      </div>
    </div>
  </footer>

  <script src="js/jquery.js"></script>
  <script src="js/antismash.js"></script>
  <script src="regions.js"></script>
  <script>
    $(document).ready(function() {
        viewer["start"](all_regions, details_data, resultsData, recordData);
    })
  </script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink">
 <defs>
  <filter id="inset-shadow">
   <feOffset dx="-2" dy="-2"></feOffset>
   <feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
   <feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
   <feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
   <feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
   <feComponentTransfer in="shadow" result="shadow">
    <feFuncA type="linear" slope=".95"></feFuncA>
   </feComponentTransfer>
   <feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
  </filter>
 </defs>
</svg>
</body>
</html>