
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>6666666.867956 - 4 region(s) - antiSMASH results</title>
  <style>img {width: 20px; height: 20px;}</style>
  <link rel="stylesheet" type="text/css" href="css/bacteria.css">
  <meta property="og:title" content="6666666.867956 - 3 record(s) - 4 region(s)" />
  <meta property="og:description" content="">
  <meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
  <meta property="og:image:alt" content="antiSMASH logo" />
  <meta property="og:image:width" content="600" />
  <meta property="og:image:height" content="600" />
  <meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-58a96dbc-ba55-40e0-b426-3433a1d61d25/index.html" />
  <meta property="og:type" content="website" />
  <meta name="twitter:card" content="summary" />
  <meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
  <div id="header">
      <nav>
    <div class="main-link">
     <div>
        <a href="https://antismash.secondarymetabolites.org/">
          <img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
        </a>
       </div>
       <div>
        <a href="https://antismash.secondarymetabolites.org/">
          antiSMASH version 6.0.1
        </a>
      </div>
    </div>
    <div class="custom-description"></div>
    <div class="ancillary-links">
      <div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
        <ul class="dropdown-options">
        
        <li><a href="6666666.867956.zip">Download all results</a></li>
        
          <li><a href="6666666.867956.gbk">Download GenBank summary file</a></li>
        
          <li><a href="bacteria-58a96dbc-ba55-40e0-b426-3433a1d61d25.log">Download log file</a></li>
        
        </ul>
      </div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
      <div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
    </div>
  </nav>
    <div id="buttons">
      <span id="region-type">Select genomic region:</span>
      <div id="regionbuttons">
        <div class="arrow-left" id="prev-region"></div>
        <div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
        <div style="display: flex; flex-wrap: wrap">
        
          
            
            
            <div class="regbutton terpene r5c1">
              <a href="#r5c1">5.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NAPAA r6c1">
              <a href="#r6c1">6.1</a>
            </div>
          
            
            
            <div class="regbutton T1PKS r6c2">
              <a href="#r6c2">6.2</a>
            </div>
          
        
          
            
            
            <div class="regbutton NAPAA r10c1">
              <a href="#r10c1">10.1</a>
            </div>
          
        
        </div>
        <div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
      </div>
    </div>
  </div>

  <!-- overview page -->
  
  <div class="page" id="overview">
   
   
     <h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>
   
   <div class="overview-layout">
    <div id="record-tables">
     <div id="single-record-tables">
     <div class="record-overview-header">
        <strong>NZ_PTJO01000004</strong> (original name was: NZ_PTJO01000004.1) (Corynebacterium_alimapuense_VA37-3 sp.)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-5" data-filename="NZ_PTJO01000004_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-5" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r5c1">
     
     
     <td class="regbutton terpene r5c1">
      <a href="#r5c1">Region&nbsp5.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">63,941</td>
     <td class="digits table-split-left">84,864</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000637/1" target="_blank">carotenoid</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 62%, #ffffff00 62%)">62%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>NZ_PTJO01000001</strong> (original name was: NZ_PTJO01000001.1) (Corynebacterium_alimapuense_VA37-3 sp.)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-6" data-filename="NZ_PTJO01000001_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-6" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r6c1">
     
     
     <td class="regbutton NAPAA r6c1">
      <a href="#r6c1">Region&nbsp6.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
     </td>
     <td class="digits">86,159</td>
     <td class="digits table-split-left">120,043</td>
     
      <td colspan="3"></td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r6c2">
     
     
     <td class="regbutton T1PKS r6c2">
      <a href="#r6c2">Region&nbsp6.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">204,317</td>
     <td class="digits table-split-left">249,200</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>NZ_PTJO01000003</strong> (original name was: NZ_PTJO01000003.1) (Corynebacterium_alimapuense_VA37-3 sp.)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-10" data-filename="NZ_PTJO01000003_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-10" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r10c1">
     
     
     <td class="regbutton NAPAA r10c1">
      <a href="#r10c1">Region&nbsp10.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
     </td>
     <td class="digits">574,128</td>
     <td class="digits table-split-left">608,141</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div>
     </div>
     
     <div id="compact-record-table" style="display: none;">
      <div class="record-table">
       <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 
 
  <tr class="linked-row odd" data-anchor="#r5c1">
     
     
     <td class="regbutton terpene r5c1">
      <a href="#r5c1">Region&nbsp5.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">63,941</td>
     <td class="digits table-split-left">84,864</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000637/1" target="_blank">carotenoid</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 62%, #ffffff00 62%)">62%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r6c1">
     
     
     <td class="regbutton NAPAA r6c1">
      <a href="#r6c1">Region&nbsp6.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
     </td>
     <td class="digits">86,159</td>
     <td class="digits table-split-left">120,043</td>
     
      <td colspan="3"></td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r6c2">
     
     
     <td class="regbutton T1PKS r6c2">
      <a href="#r6c2">Region&nbsp6.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">204,317</td>
     <td class="digits table-split-left">249,200</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r10c1">
     
     
     <td class="regbutton NAPAA r10c1">
      <a href="#r10c1">Region&nbsp10.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
     </td>
     <td class="digits">574,128</td>
     <td class="digits table-split-left">608,141</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
 </tbody>
</table>
      </div>
     </div>
     
     
     <dl id="regionless-records">
      <dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>
       
         <dd><strong>NZ_PTJO01000008</strong> (original name was: NZ_PTJO01000008.1) (Corynebacterium_alimapuense_VA37-3 sp.)</dd>
       
         <dd><strong>NZ_PTJO01000009</strong> (original name was: NZ_PTJO01000009.1) (Corynebacterium_alimapuense_VA37-3 sp.)</dd>
       
         <dd><strong>NZ_PTJO01000012</strong> (original name was: NZ_PTJO01000012.1) (Corynebacterium_alimapuense_VA37-3 sp.)</dd>
       
         <dd><strong>NZ_PTJO01000010</strong> (original name was: NZ_PTJO01000010.1) (Corynebacterium_alimapuense_VA37-3 sp.)</dd>
       
         <dd><strong>NZ_PTJO01000002</strong> (original name was: NZ_PTJO01000002.1) (Corynebacterium_alimapuense_VA37-3 sp.)</dd>
       
         <dd><strong>NZ_PTJO01000007</strong> (original name was: NZ_PTJO01000007.1) (Corynebacterium_alimapuense_VA37-3 sp.)</dd>
       
         <dd><strong>NZ_PTJO01000005</strong> (original name was: NZ_PTJO01000005.1) (Corynebacterium_alimapuense_VA37-3 sp.)</dd>
       
         <dd><strong>NZ_PTJO01000006</strong> (original name was: NZ_PTJO01000006.1) (Corynebacterium_alimapuense_VA37-3 sp.)</dd>
       
     </dl>
     
    </div>
    
    <div class="overview-switches">
     <div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    
   </div>
  </div>
  






<div class="page" id='r5c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00005_NZ_PTJO.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-26"></div> <span class="help-tooltip" id="region_svg-help-26">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_PTJO01000004.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r5c1-svg" data-filename="NZ_PTJO01000004_r5c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 63,941 - 84,864 nt. (total: 20,924 nt)
        
        <a class="cluster-rules-header" id="r5c1-rules-header" href="#r5c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r5c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r5c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-27"></div> <span class="help-tooltip" id="focus-panel-help-27">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r5c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r5c1-clusterblast" data-name="r5c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r5c1-knownclusterblast" data-name="r5c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r5c1-subclusterblast" data-name="r5c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r5c1-MIBiG-cluster-compare" data-name="r5c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r5c1-pfam-details" data-name="r5c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r5c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-1"></div> <span class="help-tooltip" id="cb-general-help-1">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r5c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r5c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r5c1_1.svg">
        NZ_FXAR01000004 (156160-177048): Corynebacterium pollutisoli strain VDS11, wh... (75% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r5c1_2.svg">
        NZ_CP011545 (600369-621284): Corynebacterium testudinoris strain DSM 44614 ch... (56% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r5c1_3.svg">
        NC_020302 (734955-755891): Corynebacterium halotolerans YIM 70093 = DSM 44683... (40% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r5c1_4.svg">
        NZ_CP046455 (1556582-1577500): Corynebacterium sp. 2039 chromosome, complete ... (47% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r5c1_5.svg">
        NZ_CP005286 (114309-134371): Corynebacterium humireducens NBRC 106098 = DSM 4... (38% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r5c1_6.svg">
        NC_021915 (1172985-1193882): Corynebacterium maris DSM 45190, complete sequence (47% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r5c1_7.svg">
        NZ_LT906473 (2450960-2471803): Corynebacterium cystitidis strain NCTC11863 ch... (47% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r5c1_8.svg">
        NZ_CP007790 (63305-84217): Corynebacterium marinum DSM 44953 chromosome, comp... (35% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r5c1_9.svg">
        NZ_CP046453 (115115-136027): Corynebacterium sp. 2019 chromosome, complete ge... (35% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r5c1_10.svg">
        NZ_CP039247 (2162515-2219110): Corynebacterium endometrii strain LMM-1653 chr... (13% of genes show similarity), NRPS-like,terpene
      </option>
      
    </select>
    <button id= "clusterblast-r5c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r5c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r5c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-2"></div> <span class="help-tooltip" id="cb-known-help-2">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r5c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r5c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r5c1_1.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r5c1_2.svg">
        carotenoid
      </option>
      
    </select>
    <button id="knownclusterblast-r5c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r5c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r5c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-3"></div> <span class="help-tooltip" id="cb-sub-help-3">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r5c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-4"></div> <span class="help-tooltip" id="MIBiG-help-4">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r5c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r5c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r5c1"></div>
    <div id="comparison-MIBiG-r5c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000637.1: 540-9054">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000637/index.html#r1c1" target="_new">BGC0000637.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.51, order:0.66, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Corynebacterium glutamicum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Rhodobacter sphaeroides</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645.1: 0-7475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Halobacillus halophilus DSM 2266</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644.1: 97-6276">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.48, order:0.47, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Dietzia sp. CQ4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650.1: 28-7720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:0.45, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Algoriphagus sp. KK10202C</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:0.38, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Myxococcus xanthus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces collinus Tu 365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630.1: 389-6051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Terpene</td>
        
        <td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
        
        
        <td>Paracoccus haeundaensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640.1: 2399-8466">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Enterobacteriaceae bacterium DC404</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000632.1: 2720626-2736748">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000632/index.html#r1c1" target="_new">BGC0000632.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Terpene, Saccharide</td>
        
        <td>brasilicardin A</td>
        
        
        <td>Nocardia terpenica</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r5c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000637.1: 540-9054">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000637/index.html#r1c1" target="_new">BGC0000637.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.51, order:0.66, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Corynebacterium glutamicum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Rhodobacter sphaeroides</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645.1: 0-7475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Halobacillus halophilus DSM 2266</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644.1: 97-6276">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.48, order:0.47, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Dietzia sp. CQ4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650.1: 28-7720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:0.45, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Algoriphagus sp. KK10202C</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.43, order:0.38, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Myxococcus xanthus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces collinus Tu 365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630.1: 389-6051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Terpene</td>
      <td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
      <td>Paracoccus haeundaensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640.1: 2399-8466">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Enterobacteriaceae bacterium DC404</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000632.1: 2720626-2736748">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000632/index.html#r1c1" target="_new">BGC0000632.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Terpene, Saccharide</td>
      <td>brasilicardin A</td>
      <td>Nocardia terpenica</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r5c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-5"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-5">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r5c1-pfam-details-svg" data-filename="NZ_PTJO01000004_r5c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r5c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r6c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00006_NZ_PTJO.. - Region 1 - NAPAA <div class="help-container"> <div class="help-icon" data-id="region_svg-help-28"></div> <span class="help-tooltip" id="region_svg-help-28">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_PTJO01000001.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r6c1-svg" data-filename="NZ_PTJO01000001_r6c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 86,159 - 120,043 nt. (total: 33,885 nt)
        
        <a class="cluster-rules-header" id="r6c1-rules-header" href="#r6c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r6c1-rules">NAPAA: (AMP-binding and PP-binding and TIGR02353)</div>
      
      <div class="region-svg-container">
       <div id='r6c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-29"></div> <span class="help-tooltip" id="focus-panel-help-29">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r6c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r6c1-nrps_pks" data-name="r6c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
      <div class="body-details-header r6c1-clusterblast" data-name="r6c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r6c1-knownclusterblast" data-name="r6c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r6c1-subclusterblast" data-name="r6c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r6c1-MIBiG-cluster-compare" data-name="r6c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r6c1-pfam-details" data-name="r6c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r6c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-6"></div> <span class="help-tooltip" id="nrps-domain-help-6">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c1-details-svg" data-filename="NZ_PTJO01000001_r6c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r6c1-details-svg"></div>
</div>
    </div>
     
    
     
    <div class="body-details-section r6c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-7"></div> <span class="help-tooltip" id="cb-general-help-7">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r6c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r6c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r6c1_1.svg">
        NZ_CP007790 (2265320-2309214): Corynebacterium marinum DSM 44953 chromosome, ... (71% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r6c1_2.svg">
        NZ_CP011545 (2391569-2435457): Corynebacterium testudinoris strain DSM 44614 ... (69% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r6c1_3.svg">
        NZ_CP039247 (2162515-2219110): Corynebacterium endometrii strain LMM-1653 chr... (56% of genes show similarity), NRPS-like,terpene
      </option>
      
      <option value = "svg/clusterblast_r6c1_4.svg">
        NC_004369 (2702492-2746374): Corynebacterium efficiens YS-314, complete genome (63% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r6c1_5.svg">
        NZ_CP009220 (2523818-2567703): Corynebacterium deserti GIMN1.010, complete ge... (68% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r6c1_6.svg">
        NZ_CP004350 (2726268-2784669): Corynebacterium casei LMG S-19264 chromosome, ... (56% of genes show similarity), NRPS-like,terpene
      </option>
      
      <option value = "svg/clusterblast_r6c1_7.svg">
        NZ_CP009251 (2424700-2468558): Corynebacterium stationis strain 622=DSM 20302... (77% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r6c1_8.svg">
        NZ_CP014279 (2484413-2528271): Corynebacterium stationis strain ATCC 6872, co... (73% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r6c1_9.svg">
        NZ_CP026947 (2160529-2204441): Corynebacterium sp. 2183 chromosome, complete ... (69% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r6c1_10.svg">
        NZ_CP009244 (2407758-2464335): Corynebacterium ammoniagenes DSM 20306 strain ... (59% of genes show similarity), NRPS-like,terpene
      </option>
      
    </select>
    <button id= "clusterblast-r6c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r6c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-8"></div> <span class="help-tooltip" id="cb-known-help-8">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r6c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-9"></div> <span class="help-tooltip" id="cb-sub-help-9">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r6c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-10"></div> <span class="help-tooltip" id="MIBiG-help-10">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r6c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r6c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r6c1"></div>
    <div id="comparison-MIBiG-r6c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NAPAA</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>ergovaline</td>
        
        
        <td>Epichloe festucae var. lolii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 10E9</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 22D11</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876.1: 71-9235">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Other</td>
        
        <td>neopolyoxin C</td>
        
        
        <td>Streptomyces tendae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561.1: 1236-18814">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>curacomycin</td>
        
        
        <td>Streptomyces curacoi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517.1: 168853-184269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>asperphenamate</td>
        
        
        <td>Aspergillus terreus NIH2624</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>NRP</td>
        
        <td>AM-toxin</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r6c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>ergovaline</td>
      <td>Epichloe festucae var. lolii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 10E9</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 22D11</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876.1: 71-9235">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Other</td>
      <td>neopolyoxin C</td>
      <td>Streptomyces tendae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561.1: 1236-18814">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>curacomycin</td>
      <td>Streptomyces curacoi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517.1: 168853-184269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP</td>
      <td>asperphenamate</td>
      <td>Aspergillus terreus NIH2624</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP</td>
      <td>AM-toxin</td>
      <td>Alternaria alternata</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r6c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-11"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-11">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c1-pfam-details-svg" data-filename="NZ_PTJO01000001_r6c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r6c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r6c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00006_NZ_PTJO.. - Region 2 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-30"></div> <span class="help-tooltip" id="region_svg-help-30">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_PTJO01000001.region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r6c2-svg" data-filename="NZ_PTJO01000001_r6c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 204,317 - 249,200 nt. (total: 44,884 nt)
        
        <a class="cluster-rules-header" id="r6c2-rules-header" href="#r6c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r6c2-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r6c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-31"></div> <span class="help-tooltip" id="focus-panel-help-31">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r6c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r6c2-nrps_pks" data-name="r6c2-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r6c2-clusterblast" data-name="r6c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r6c2-knownclusterblast" data-name="r6c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r6c2-subclusterblast" data-name="r6c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r6c2-MIBiG-cluster-compare" data-name="r6c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r6c2-pfam-details" data-name="r6c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r6c2-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-12"></div> <span class="help-tooltip" id="nrps-domain-help-12">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c2-details-svg" data-filename="NZ_PTJO01000001_r6c2_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r6c2-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r6c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-15"></div> <span class="help-tooltip" id="cb-general-help-15">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r6c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r6c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r6c2_1.svg">
        NZ_FXAR01000003 (44531-89373): Corynebacterium pollutisoli strain VDS11, whol... (83% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c2_2.svg">
        NZ_CP005286 (2487175-2532041): Corynebacterium humireducens NBRC 106098 = DSM... (81% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c2_3.svg">
        NZ_CP011545 (2529895-2574698): Corynebacterium testudinoris strain DSM 44614 ... (78% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c2_4.svg">
        NC_021915 (2587501-2632313): Corynebacterium maris DSM 45190, complete sequence (80% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c2_5.svg">
        NZ_KB902183 (20640-65413): Corynebacterium lubricantis DSM 45231 A37ODRAFT sc... (72% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c2_6.svg">
        NZ_CP026947 (2343219-2388037): Corynebacterium sp. 2183 chromosome, complete ... (75% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c2_7.svg">
        NZ_KB892468 (75439-120212): Corynebacterium pilosum DSM 20521 B097DRAFT scaff... (75% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c2_8.svg">
        NZ_CP059833 (1736224-1780970): Corynebacterium sp. NML93-0612 chromosome (75% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c2_9.svg">
        NC_012590 (2573868-2618650): Corynebacterium aurimucosum ATCC 700975, complet... (71% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r6c2_10.svg">
        NZ_CP026948 (2172467-2217264): Corynebacterium sp. 2184 chromosome, complete ... (66% of genes show similarity), T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r6c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r6c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-16"></div> <span class="help-tooltip" id="cb-known-help-16">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r6c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-17"></div> <span class="help-tooltip" id="cb-sub-help-17">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r6c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-18"></div> <span class="help-tooltip" id="MIBiG-help-18">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r6c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r6c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r6c2"></div>
    <div id="comparison-MIBiG-r6c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Polyketide</td>
        
        <td>strobilurin</td>
        
        
        <td>Strobilurus tenacellus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene, Polyketide</td>
        
        <td>pyripyropene A</td>
        
        
        <td>unidentified unclassified sequences.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831.1: 8162481-8168632">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>alkylpyrone-407, alkylpyrone-393</td>
        
        
        <td>Myxococcus xanthus DK 1622</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
        
        
        <td>Parastagonospora nodorum SN15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001163.1: 308352-317651">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001163/index.html#r1c1" target="_new">BGC0001163.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:1.00, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Polyketide</td>
        
        <td>1-heptadecene</td>
        
        
        <td>Moorea producens 3L</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001162.1: 0-9348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001162/index.html#r1c1" target="_new">BGC0001162.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:1.00, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Polyketide</td>
        
        <td>1-heptadecene</td>
        
        
        <td>Moorea producens JHB</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001161.1: 0-9161">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001161/index.html#r1c1" target="_new">BGC0001161.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:1.00, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Polyketide</td>
        
        <td>1-heptadecene</td>
        
        
        <td>Moorea bouillonii PNG</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000008.1: 5134-68984">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000008/index.html#r1c1" target="_new">BGC0000008.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>aflatoxin</td>
        
        
        <td>Aspergillus flavus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000004.1: 15202-78697">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000004/index.html#r1c1" target="_new">BGC0000004.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>aflatoxin G1</td>
        
        
        <td>Aspergillus oryzae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000006.1: 13469-77383">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000006/index.html#r1c1" target="_new">BGC0000006.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>aflatoxin</td>
        
        
        <td>Aspergillus flavus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r6c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909.1: 2476-67615">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Polyketide</td>
      <td>strobilurin</td>
      <td>Strobilurus tenacellus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene, Polyketide</td>
      <td>pyripyropene A</td>
      <td>unidentified unclassified sequences.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831.1: 8162481-8168632">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>alkylpyrone-407, alkylpyrone-393</td>
      <td>Myxococcus xanthus DK 1622</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
      <td>Parastagonospora nodorum SN15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001163.1: 308352-317651">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001163/index.html#r1c1" target="_new">BGC0001163.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:1.00, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>1-heptadecene</td>
      <td>Moorea producens 3L</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001162.1: 0-9348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001162/index.html#r1c1" target="_new">BGC0001162.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:1.00, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>1-heptadecene</td>
      <td>Moorea producens JHB</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001161.1: 0-9161">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001161/index.html#r1c1" target="_new">BGC0001161.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:1.00, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>1-heptadecene</td>
      <td>Moorea bouillonii PNG</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000008.1: 5134-68984">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000008/index.html#r1c1" target="_new">BGC0000008.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>aflatoxin</td>
      <td>Aspergillus flavus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000004.1: 15202-78697">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000004/index.html#r1c1" target="_new">BGC0000004.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>aflatoxin G1</td>
      <td>Aspergillus oryzae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000006.1: 13469-77383">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000006/index.html#r1c1" target="_new">BGC0000006.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>aflatoxin</td>
      <td>Aspergillus flavus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r6c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-19"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-19">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c2-pfam-details-svg" data-filename="NZ_PTJO01000001_r6c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r6c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r6c2-nrps_pks_products" data-name="r6c2-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r6c2-nrps_pks" data-name="r6c2-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r6c2-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-13"></div> <span class="help-tooltip" id="nrps-structures-help-13">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 2, location 204316 - 249200:
          <div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r6c2-2" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=mal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r6c2-2-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r6c2-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-14"></div> <span class="help-tooltip" id="nrps-monomers-help-14">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds224316_229200</span></strong>: mal
      <div class="collapser collapser-target-cds224316_229200 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (645..932)</strong>: mal
          <div class="collapser collapser-target-cds224316_229200 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds224316_229200-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 62.5%</dd>
</dl>

              </div><br>
          
              minowa:  fatty_acid
              <div class="collapser collapser-target-cds224316_229200-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>fatty_acid: 106.0</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 70.1</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 67.0</dt>

  <dd></dd><dt>Malonyl-CoA: 62.7</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 45.7</dt>

  <dd></dd><dt>Propionyl-CoA: 41.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 38.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 33.4</dt>

  <dd></dd><dt>CHC-CoA: 31.2</dt>

  <dd></dd><dt>Isobutyryl-CoA: 29.9</dt>

  <dd></dd><dt>Acetyl-CoA: 28.1</dt>

  <dd></dd><dt>Benzoyl-CoA: 10.2</dt>

  <dd></dd><dt>inactive: 9.7</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds229330_231181</span></strong>: pk
      <div class="collapser collapser-target-cds229330_231181 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>CAL_domain (200..535)</strong>: pk
          <div class="collapser collapser-target-cds229330_231181 collapser-level-candidate"> </div><div class="collapser-content">
          
              minowa:  NH2
              <div class="collapser collapser-target-cds229330_231181-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>NH2: 55.4</dt>

  <dd></dd><dt>Acetyl-CoA: 17.1</dt>

  <dd></dd><dt>fatty_acid: 2.4</dt>

  <dd></dd><dt>shikimic_acid: 0.0</dt>

  <dd></dd><dt>AHBA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r10c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00010_NZ_PTJO.. - Region 1 - NAPAA <div class="help-container"> <div class="help-icon" data-id="region_svg-help-32"></div> <span class="help-tooltip" id="region_svg-help-32">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = NZ_PTJO01000003.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r10c1-svg" data-filename="NZ_PTJO01000003_r10c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 574,128 - 608,141 nt. (total: 34,014 nt)
        
        <a class="cluster-rules-header" id="r10c1-rules-header" href="#r10c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r10c1-rules">NAPAA: (AMP-binding and PP-binding and TIGR02353)</div>
      
      <div class="region-svg-container">
       <div id='r10c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-33"></div> <span class="help-tooltip" id="focus-panel-help-33">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r10c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r10c1-nrps_pks" data-name="r10c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
      <div class="body-details-header r10c1-clusterblast" data-name="r10c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r10c1-knownclusterblast" data-name="r10c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r10c1-subclusterblast" data-name="r10c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r10c1-MIBiG-cluster-compare" data-name="r10c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r10c1-pfam-details" data-name="r10c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r10c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-20"></div> <span class="help-tooltip" id="nrps-domain-help-20">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r10c1-details-svg" data-filename="NZ_PTJO01000003_r10c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r10c1-details-svg"></div>
</div>
    </div>
     
    
     
    <div class="body-details-section r10c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-21"></div> <span class="help-tooltip" id="cb-general-help-21">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r10c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r10c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r10c1_1.svg">
        NZ_CP005286 (&lt;1848056-1890501): Corynebacterium humireducens NBRC 106098 = DS... (55% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r10c1_2.svg">
        NC_020302 (2134770-2177976): Corynebacterium halotolerans YIM 70093 = DSM 446... (44% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r10c1_3.svg">
        NZ_CP009246 (1868623-1911815): Corynebacterium flavescens strain OJ8, complet... (20% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r10c1_4.svg">
        NZ_CP024190 (3750078-3846930): Mycobacterium marinum strain CCUG20998 chromos... (15% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r10c1_5.svg">
        NC_010612 (3964914-4058403): Mycobacterium marinum M, complete genome (15% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r10c1_6.svg">
        NZ_CP046452 (483428-527454): Corynebacterium sp. 1959 chromosome, complete ge... (10% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r10c1_7.svg">
        NZ_AZWL01000009 (128293-200845): Streptomyces sp. CNH099 B121DRAFT scaffold 7... (15% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r10c1_8.svg">
        NZ_LN850107 (6973415-7022083): Alloactinosynnema sp. L-07 isolate Alloactinos... (11% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r10c1_9.svg">
        NC_010617 (1239822-1283725): Kocuria rhizophila DC2201, complete genome (8% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r10c1_10.svg">
        NZ_CP046455 (281097-324469): Corynebacterium sp. 2039 chromosome, complete ge... (8% of genes show similarity), NRPS-like,betalactone
      </option>
      
    </select>
    <button id= "clusterblast-r10c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r10c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r10c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-22"></div> <span class="help-tooltip" id="cb-known-help-22">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r10c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-23"></div> <span class="help-tooltip" id="cb-sub-help-23">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r10c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-24"></div> <span class="help-tooltip" id="MIBiG-help-24">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r10c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r10c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r10c1"></div>
    <div id="comparison-MIBiG-r10c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NAPAA</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527.1: 0-8170">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>basidioferrin</td>
        
        
        <td>Gelatoporia subvermispora</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000871.1: 1789112-1797919">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000871/index.html#r1c1" target="_new">BGC0000871.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Other,  , (, N, u, c, l, e, o, s, i, d, e, )</td>
        
        <td>VEPE, AEPE, TG-1</td>
        
        
        <td>Myxococcus xanthus DK 1622</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002002.1: 780775-795170">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002002/index.html#r1c1" target="_new">BGC0002002.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>1,6-phenazinedimethanol, fontizine A</td>
        
        
        <td>Serratia fonticola</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>NRP</td>
        
        <td>fellutamide B</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001871.1: 2532668-2545659">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001871/index.html#r1c1" target="_new">BGC0001871.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>L-2-amino-4-methoxy-trans-3-butenoic acid</td>
        
        
        <td>Pseudomonas aeruginosa PAO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r10c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527.1: 0-8170">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>basidioferrin</td>
      <td>Gelatoporia subvermispora</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000871.1: 1789112-1797919">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000871/index.html#r1c1" target="_new">BGC0000871.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Other,  , (, N, u, c, l, e, o, s, i, d, e, )</td>
      <td>VEPE, AEPE, TG-1</td>
      <td>Myxococcus xanthus DK 1622</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002002.1: 780775-795170">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002002/index.html#r1c1" target="_new">BGC0002002.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP</td>
      <td>1,6-phenazinedimethanol, fontizine A</td>
      <td>Serratia fonticola</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP</td>
      <td>fellutamide B</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001871.1: 2532668-2545659">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001871/index.html#r1c1" target="_new">BGC0001871.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>NRP</td>
      <td>L-2-amino-4-methoxy-trans-3-butenoic acid</td>
      <td>Pseudomonas aeruginosa PAO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r10c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-25"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-25">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r10c1-pfam-details-svg" data-filename="NZ_PTJO01000003_r10c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r10c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>


  <footer class="footer">
    <div class="container">
      <div>
        <img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
      </div>
      <div class="cite-me">
        If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
      </div>
      <div>
        <img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
      </div>
    </div>
  </footer>

  <script src="js/jquery.js"></script>
  <script src="js/antismash.js"></script>
  <script src="regions.js"></script>
  <script>
    $(document).ready(function() {
        viewer["start"](all_regions, details_data, resultsData, recordData);
    })
  </script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink">
 <defs>
  <filter id="inset-shadow">
   <feOffset dx="-2" dy="-2"></feOffset>
   <feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
   <feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
   <feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
   <feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
   <feComponentTransfer in="shadow" result="shadow">
    <feFuncA type="linear" slope=".95"></feFuncA>
   </feComponentTransfer>
   <feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
  </filter>
 </defs>
</svg>
</body>
</html>