
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>6666666.590939 - 40 region(s) - antiSMASH results</title>
  <style>img {width: 20px; height: 20px;}</style>
  <link rel="stylesheet" type="text/css" href="css/bacteria.css">
  <meta property="og:title" content="6666666.590939 - 35 record(s) - 40 region(s)" />
  <meta property="og:description" content="">
  <meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
  <meta property="og:image:alt" content="antiSMASH logo" />
  <meta property="og:image:width" content="600" />
  <meta property="og:image:height" content="600" />
  <meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-1f5c55ee-d30a-4e9b-aa30-d4d0dd1c47cb/index.html" />
  <meta property="og:type" content="website" />
  <meta name="twitter:card" content="summary" />
  <meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
  <div id="header">
      <nav>
    <div class="main-link">
     <div>
        <a href="https://antismash.secondarymetabolites.org/">
          <img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
        </a>
       </div>
       <div>
        <a href="https://antismash.secondarymetabolites.org/">
          antiSMASH version 6.0.1
        </a>
      </div>
    </div>
    <div class="custom-description"></div>
    <div class="ancillary-links">
      <div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
        <ul class="dropdown-options">
        
        <li><a href="6666666.590939.zip">Download all results</a></li>
        
          <li><a href="6666666.590939.gbk">Download GenBank summary file</a></li>
        
          <li><a href="bacteria-1f5c55ee-d30a-4e9b-aa30-d4d0dd1c47cb.log">Download log file</a></li>
        
        </ul>
      </div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
      <div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
    </div>
  </nav>
    <div id="buttons">
      <span id="region-type">Select genomic region:</span>
      <div id="regionbuttons">
        <div class="arrow-left" id="prev-region"></div>
        <div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
        <div style="display: flex; flex-wrap: wrap">
        
          
            
            
            <div class="regbutton T2PKS r6c1">
              <a href="#r6c1">6.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r9c1">
              <a href="#r9c1">9.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton siderophore r10c1">
              <a href="#r10c1">10.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS-like r13c1">
              <a href="#r13c1">13.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r17c1">
              <a href="#r17c1">17.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS,T1PKS hybrid r22c1">
              <a href="#r22c1">22.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r35c1">
              <a href="#r35c1">35.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T2PKS r37c1">
              <a href="#r37c1">37.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r43c1">
              <a href="#r43c1">43.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r48c1">
              <a href="#r48c1">48.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton melanin r57c1">
              <a href="#r57c1">57.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r65c1">
              <a href="#r65c1">65.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton lanthipeptide-class-iii r76c1">
              <a href="#r76c1">76.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r91c1">
              <a href="#r91c1">91.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r98c1">
              <a href="#r98c1">98.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton siderophore r113c1">
              <a href="#r113c1">113.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton NRPS-like,T1PKS hybrid r114c1">
              <a href="#r114c1">114.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton hglE-KS r124c1">
              <a href="#r124c1">124.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton indole r126c1">
              <a href="#r126c1">126.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T2PKS r133c1">
              <a href="#r133c1">133.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton hglE-KS r147c1">
              <a href="#r147c1">147.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r152c1">
              <a href="#r152c1">152.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton RiPP-like r162c1">
              <a href="#r162c1">162.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T1PKS r205c1">
              <a href="#r205c1">205.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r224c1">
              <a href="#r224c1">224.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r245c1">
              <a href="#r245c1">245.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton T3PKS r255c1">
              <a href="#r255c1">255.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r280c1">
              <a href="#r280c1">280.1</a>
            </div>
          
            
            
            <div class="regbutton indole r280c2">
              <a href="#r280c2">280.2</a>
            </div>
          
            
            
            <div class="regbutton NRPS-like r280c3">
              <a href="#r280c3">280.3</a>
            </div>
          
        
          
            
            
            <div class="regbutton lanthipeptide-class-v r286c1">
              <a href="#r286c1">286.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton RiPP-like r302c1">
              <a href="#r302c1">302.1</a>
            </div>
          
            
            
            <div class="regbutton terpene r302c2">
              <a href="#r302c2">302.2</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton T1PKS,transAT-PKS hybrid r307c1">
              <a href="#r307c1">307.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton ectoine r330c1">
              <a href="#r330c1">330.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton siderophore r341c1">
              <a href="#r341c1">341.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton lanthipeptide-class-iii r344c1">
              <a href="#r344c1">344.1</a>
            </div>
          
            
            
            <div class="regbutton lanthipeptide-class-i r344c2">
              <a href="#r344c2">344.2</a>
            </div>
          
            
            
            <div class="regbutton NRPS r344c3">
              <a href="#r344c3">344.3</a>
            </div>
          
        
          
            
            
            <div class="regbutton butyrolactone r381c1">
              <a href="#r381c1">381.1</a>
            </div>
          
        
        </div>
        <div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
      </div>
    </div>
  </div>

  <!-- overview page -->
  
  <div class="page" id="overview">
   
   
     <h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>
   
   <div class="overview-layout">
    <div id="record-tables">
     <div id="single-record-tables">
     <div class="record-overview-header">
        <strong>c00006_NODE_25..</strong> (original name was: NODE_250_length_11114_cov_181.846405) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-6" data-filename="c00006_NODE_25.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-6" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r6c1">
     
     
     <td class="regbutton T2PKS r6c1">
      <a href="#r6c1">Region&nbsp6.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,162</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000236/1" target="_blank">kinamycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 31%, #ffffff00 31%)">31%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00009_NODE_26..</strong> (original name was: NODE_261_length_5950_cov_115.627563) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-9" data-filename="c00009_NODE_26.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-9" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r9c1">
     
     
     <td class="regbutton NRPS r9c1">
      <a href="#r9c1">Region&nbsp9.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">5,998</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000325/1" target="_blank">coelichelin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00010_NODE_41..</strong> (original name was: NODE_411_length_49125_cov_155.311447) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-10" data-filename="c00010_NODE_41.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-10" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r10c1">
     
     
     <td class="regbutton siderophore r10c1">
      <a href="#r10c1">Region&nbsp10.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">21,712</td>
     <td class="digits table-split-left">34,885</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001732/1" target="_blank">paulomycin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00013_NODE_19..</strong> (original name was: NODE_194_length_8109_cov_182.765823) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-13" data-filename="c00013_NODE_19.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-13" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r13c1">
     
     
     <td class="regbutton NRPS-like r13c1">
      <a href="#r13c1">Region&nbsp13.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">8,157</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000843/1" target="_blank">alanylclavam / 2-hydroxymethylclavam / 2-formyloxymethylclavam / clavam-2-carboxylate</a></td>
      <td>Other:Non-NRP beta-lactam</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00017_NODE_73..</strong> (original name was: NODE_73_length_11154_cov_200.786621) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-17" data-filename="c00017_NODE_73.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-17" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r17c1">
     
     
     <td class="regbutton terpene r17c1">
      <a href="#r17c1">Region&nbsp17.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,202</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000660/1" target="_blank">albaflavenone</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00022_NODE_75..</strong> (original name was: NODE_75_length_25086_cov_93.428406) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-22" data-filename="c00022_NODE_75.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-22" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r22c1">
     
     
         
     
     <td class="regbutton NRPS,T1PKS hybrid r22c1">
      <a href="#r22c1">Region&nbsp22.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">25,134</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001455/1" target="_blank">antimycin</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 31%, #ffffff00 31%)">31%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00035_NODE_72..</strong> (original name was: NODE_720_length_9940_cov_100.728973) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-35" data-filename="c00035_NODE_72.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-35" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r35c1">
     
     
     <td class="regbutton T1PKS r35c1">
      <a href="#r35c1">Region&nbsp35.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">9,988</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000034/1" target="_blank">candicidin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00037_NODE_29..</strong> (original name was: NODE_297_length_10344_cov_178.334396) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-37" data-filename="c00037_NODE_29.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-37" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r37c1">
     
     
     <td class="regbutton T2PKS r37c1">
      <a href="#r37c1">Region&nbsp37.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">10,392</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000230/1" target="_blank">griseorhodin A</a></td>
      <td>Polyketide:Type II</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00043_NODE_32..</strong> (original name was: NODE_320_length_14248_cov_143.055450) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-43" data-filename="c00043_NODE_32.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-43" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r43c1">
     
     
     <td class="regbutton terpene r43c1">
      <a href="#r43c1">Region&nbsp43.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">14,296</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001181/1" target="_blank">geosmin</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00048_NODE_8_..</strong> (original name was: NODE_8_length_69278_cov_108.070297) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-48" data-filename="c00048_NODE_8_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-48" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r48c1">
     
     
     <td class="regbutton terpene r48c1">
      <a href="#r48c1">Region&nbsp48.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">19,460</td>
     <td class="digits table-split-left">41,244</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000242/1" target="_blank">lysolipin I</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00057_NODE_10..</strong> (original name was: NODE_10_length_17696_cov_175.534470) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-57" data-filename="c00057_NODE_10.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-57" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r57c1">
     
     
     <td class="regbutton melanin r57c1">
      <a href="#r57c1">Region&nbsp57.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#melanin" target="_blank">melanin</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,883</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000909/1" target="_blank">melanin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00065_NODE_25..</strong> (original name was: NODE_258_length_30876_cov_138.759811) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-65" data-filename="c00065_NODE_25.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-65" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r65c1">
     
     
     <td class="regbutton terpene r65c1">
      <a href="#r65c1">Region&nbsp65.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">25,575</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000663/1" target="_blank">hopene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00076_NODE_88..</strong> (original name was: NODE_88_length_59415_cov_204.016815) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-76" data-filename="c00076_NODE_88.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-76" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r76c1">
     
     
     <td class="regbutton lanthipeptide-class-iii r76c1">
      <a href="#r76c1">Region&nbsp76.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
     </td>
     <td class="digits">32,111</td>
     <td class="digits table-split-left">54,714</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000501/1" target="_blank">catenulipeptin</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00091_NODE_56..</strong> (original name was: NODE_561_length_40717_cov_93.091827) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-91" data-filename="c00091_NODE_56.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-91" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r91c1">
     
     
     <td class="regbutton T1PKS r91c1">
      <a href="#r91c1">Region&nbsp91.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">38,398</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001491/1" target="_blank">67-121C</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 29%, #ffffff00 29%)">29%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00098_NODE_34..</strong> (original name was: NODE_340_length_31416_cov_147.770920) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-98" data-filename="c00098_NODE_34.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-98" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r98c1">
     
     
     <td class="regbutton NRPS r98c1">
      <a href="#r98c1">Region&nbsp98.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">31,464</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001370/1" target="_blank">lipopeptide 8D1-1 / lipopeptide 8D1-2</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 29%, #ffffff00 29%)">29%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00113_NODE_22..</strong> (original name was: NODE_225_length_16246_cov_177.209412) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-113" data-filename="c00113_NODE_22.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-113" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r113c1">
     
     
     <td class="regbutton siderophore r113c1">
      <a href="#r113c1">Region&nbsp113.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,418</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00114_NODE_26..</strong> (original name was: NODE_267_length_22857_cov_98.544693) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-114" data-filename="c00114_NODE_26.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-114" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r114c1">
     
     
         
     
     <td class="regbutton NRPS-like,T1PKS hybrid r114c1">
      <a href="#r114c1">Region&nbsp114.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">22,905</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000034/1" target="_blank">candicidin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 76%, #ffffff00 76%)">76%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00124_NODE_41..</strong> (original name was: NODE_414_length_4643_cov_97.798409) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-124" data-filename="c00124_NODE_41.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-124" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r124c1">
     
     
     <td class="regbutton hglE-KS r124c1">
      <a href="#r124c1">Region&nbsp124.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hgle-ks" target="_blank">hglE-KS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">4,691</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00126_NODE_11..</strong> (original name was: NODE_113_length_121026_cov_118.024216) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-126" data-filename="c00126_NODE_11.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-126" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r126c1">
     
     
     <td class="regbutton indole r126c1">
      <a href="#r126c1">Region&nbsp126.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
     </td>
     <td class="digits">69,237</td>
     <td class="digits table-split-left">90,358</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001483/1" target="_blank">5-isoprenylindole-3-carboxylate β-D-glycosyl ester</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00133_NODE_44..</strong> (original name was: NODE_440_length_15921_cov_172.800766) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-133" data-filename="c00133_NODE_44.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-133" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r133c1">
     
     
     <td class="regbutton T2PKS r133c1">
      <a href="#r133c1">Region&nbsp133.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">15,969</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000271/1" target="_blank">spore pigment</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 58%, #ffffff00 58%)">58%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00147_NODE_17..</strong> (original name was: NODE_174_length_11397_cov_104.281738) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-147" data-filename="c00147_NODE_17.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-147" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r147c1">
     
     
     <td class="regbutton hglE-KS r147c1">
      <a href="#r147c1">Region&nbsp147.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hgle-ks" target="_blank">hglE-KS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,445</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001484/1" target="_blank">2&#39;-chloropentostatin / 2&#39;-amino-2&#39;-deoxyadenosine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00152_NODE_11..</strong> (original name was: NODE_11_length_30599_cov_113.123276) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-152" data-filename="c00152_NODE_11.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-152" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r152c1">
     
     
     <td class="regbutton NRPS r152c1">
      <a href="#r152c1">Region&nbsp152.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">26,700</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000325/1" target="_blank">coelichelin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 72%, #ffffff00 72%)">72%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00162_NODE_13..</strong> (original name was: NODE_130_length_11306_cov_163.554306) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-162" data-filename="c00162_NODE_13.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-162" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r162c1">
     
     
     <td class="regbutton RiPP-like r162c1">
      <a href="#r162c1">Region&nbsp162.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">5,565</td>
     <td class="digits table-split-left">11,354</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00205_NODE_74..</strong> (original name was: NODE_747_length_139661_cov_120.499809) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-205" data-filename="c00205_NODE_74.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-205" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r205c1">
     
     
     <td class="regbutton T1PKS r205c1">
      <a href="#r205c1">Region&nbsp205.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">11,984</td>
     <td class="digits table-split-left">94,247</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000167/1" target="_blank">vicenistatin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00224_NODE_30..</strong> (original name was: NODE_305_length_4836_cov_94.412117) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-224" data-filename="c00224_NODE_30.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-224" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r224c1">
     
     
     <td class="regbutton terpene r224c1">
      <a href="#r224c1">Region&nbsp224.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">4,884</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000633/1" target="_blank">carotenoid</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 36%, #ffffff00 36%)">36%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00245_NODE_51..</strong> (original name was: NODE_515_length_27279_cov_142.803955) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-245" data-filename="c00245_NODE_51.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-245" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r245c1">
     
     
     <td class="regbutton NRPS r245c1">
      <a href="#r245c1">Region&nbsp245.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">27,327</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000315/1" target="_blank">CDA1b / CDA2a / CDA2b / CDA3a / CDA3b / CDA4a / CDA4b</a></td>
      <td>NRP:Ca+-dependent lipopeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 20%, #ffffff00 20%)">20%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00255_NODE_21..</strong> (original name was: NODE_219_length_44668_cov_143.423187) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-255" data-filename="c00255_NODE_21.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-255" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r255c1">
     
     
     <td class="regbutton T3PKS r255c1">
      <a href="#r255c1">Region&nbsp255.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">16,913</td>
     <td class="digits table-split-left">44,716</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000285/1" target="_blank">flaviolin rhamnoside / 3,3&#39;-diflaviolin / flaviolin</a></td>
      <td>Polyketide:Type III + Saccharide:Oligosaccharide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 33%, #ffffff00 33%)">33%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00280_NODE_63..</strong> (original name was: NODE_633_length_188983_cov_98.981049) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-280" data-filename="c00280_NODE_63.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-280" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r280c1">
     
     
     <td class="regbutton terpene r280c1">
      <a href="#r280c1">Region&nbsp280.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">83,615</td>
     <td class="digits table-split-left">109,197</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000664/1" target="_blank">isorenieratene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 85%, #ffffff00 85%)">85%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r280c2">
     
     
     <td class="regbutton indole r280c2">
      <a href="#r280c2">Region&nbsp280.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
     </td>
     <td class="digits">109,641</td>
     <td class="digits table-split-left">130,819</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001294/1" target="_blank">7-prenylisatin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r280c3">
     
     
     <td class="regbutton NRPS-like r280c3">
      <a href="#r280c3">Region&nbsp280.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">167,481</td>
     <td class="digits table-split-left">189,031</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000432/1" target="_blank">streptothricin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 58%, #ffffff00 58%)">58%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00286_NODE_14..</strong> (original name was: NODE_145_length_6695_cov_149.796570) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-286" data-filename="c00286_NODE_14.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-286" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r286c1">
     
     
     <td class="regbutton lanthipeptide-class-v r286c1">
      <a href="#r286c1">Region&nbsp286.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-v" target="_blank">lanthipeptide-class-v</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,743</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00302_NODE_43..</strong> (original name was: NODE_43_length_34377_cov_120.602173) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-302" data-filename="c00302_NODE_43.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-302" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r302c1">
     
     
     <td class="regbutton RiPP-like r302c1">
      <a href="#r302c1">Region&nbsp302.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">7,630</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000518/1" target="_blank">informatipeptin</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r302c2">
     
     
     <td class="regbutton terpene r302c2">
      <a href="#r302c2">Region&nbsp302.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">20,588</td>
     <td class="digits table-split-left">34,425</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001204/1" target="_blank">versipelostatin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 5%, #ffffff00 5%)">5%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00307_NODE_46..</strong> (original name was: NODE_461_length_50984_cov_92.471992) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-307" data-filename="c00307_NODE_46.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-307" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r307c1">
     
     
         
     
     <td class="regbutton T1PKS,transAT-PKS hybrid r307c1">
      <a href="#r307c1">Region&nbsp307.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#transat-pks" target="_blank">transAT-PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">51,032</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000034/1" target="_blank">candicidin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00330_NODE_14..</strong> (original name was: NODE_144_length_15551_cov_159.117157) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-330" data-filename="c00330_NODE_14.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-330" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r330c1">
     
     
     <td class="regbutton ectoine r330c1">
      <a href="#r330c1">Region&nbsp330.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">3,670</td>
     <td class="digits table-split-left">14,068</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000853/1" target="_blank">ectoine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00341_NODE_32..</strong> (original name was: NODE_32_length_38644_cov_202.136368) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-341" data-filename="c00341_NODE_32.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-341" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r341c1">
     
     
     <td class="regbutton siderophore r341c1">
      <a href="#r341c1">Region&nbsp341.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">11,240</td>
     <td class="digits table-split-left">23,012</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000940/1" target="_blank">desferrioxamin B / desferrioxamine E</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 83%, #ffffff00 83%)">83%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00344_NODE_51..</strong> (original name was: NODE_51_length_134762_cov_105.640472) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-344" data-filename="c00344_NODE_51.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-344" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r344c1">
     
     
     <td class="regbutton lanthipeptide-class-iii r344c1">
      <a href="#r344c1">Region&nbsp344.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
     </td>
     <td class="digits">11,051</td>
     <td class="digits table-split-left">33,633</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000551/1" target="_blank">SapB</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 <tr class="linked-row even" data-anchor="#r344c2">
     
     
     <td class="regbutton lanthipeptide-class-i r344c2">
      <a href="#r344c2">Region&nbsp344.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-i" target="_blank">lanthipeptide-class-i</a>
     </td>
     <td class="digits">46,206</td>
     <td class="digits table-split-left">70,800</td>
     
      <td colspan="3"></td>
     
   </tr>
 <tr class="linked-row odd" data-anchor="#r344c3">
     
     
     <td class="regbutton NRPS r344c3">
      <a href="#r344c3">Region&nbsp344.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">74,572</td>
     <td class="digits table-split-left">126,897</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000324/1" target="_blank">coelibactin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00381_NODE_47..</strong> (original name was: NODE_473_length_8413_cov_144.850586) (Streptomyces rochei CC48)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-381" data-filename="c00381_NODE_47.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-381" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r381c1">
     
     
     <td class="regbutton butyrolactone r381c1">
      <a href="#r381c1">Region&nbsp381.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
     </td>
     <td class="digits">1,842</td>
     <td class="digits table-split-left">8,461</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000238/1" target="_blank">lactonamycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 7%, #ffffff00 7%)">7%</td>
     
   </tr>
 
 </tbody>
</table>
      </div>
     </div>
     
     <div id="compact-record-table" style="display: none;">
      <div class="record-table">
       <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 
 
  <tr class="linked-row odd" data-anchor="#r6c1">
     
     
     <td class="regbutton T2PKS r6c1">
      <a href="#r6c1">Region&nbsp6.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,162</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000236/1" target="_blank">kinamycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 31%, #ffffff00 31%)">31%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r9c1">
     
     
     <td class="regbutton NRPS r9c1">
      <a href="#r9c1">Region&nbsp9.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">5,998</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000325/1" target="_blank">coelichelin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r10c1">
     
     
     <td class="regbutton siderophore r10c1">
      <a href="#r10c1">Region&nbsp10.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">21,712</td>
     <td class="digits table-split-left">34,885</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001732/1" target="_blank">paulomycin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r13c1">
     
     
     <td class="regbutton NRPS-like r13c1">
      <a href="#r13c1">Region&nbsp13.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">8,157</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000843/1" target="_blank">alanylclavam / 2-hydroxymethylclavam / 2-formyloxymethylclavam / clavam-2-carboxylate</a></td>
      <td>Other:Non-NRP beta-lactam</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r17c1">
     
     
     <td class="regbutton terpene r17c1">
      <a href="#r17c1">Region&nbsp17.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,202</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000660/1" target="_blank">albaflavenone</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r22c1">
     
     
         
     
     <td class="regbutton NRPS,T1PKS hybrid r22c1">
      <a href="#r22c1">Region&nbsp22.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">25,134</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001455/1" target="_blank">antimycin</a></td>
      <td>NRP + Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 31%, #ffffff00 31%)">31%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r35c1">
     
     
     <td class="regbutton T1PKS r35c1">
      <a href="#r35c1">Region&nbsp35.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">9,988</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000034/1" target="_blank">candicidin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r37c1">
     
     
     <td class="regbutton T2PKS r37c1">
      <a href="#r37c1">Region&nbsp37.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">10,392</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000230/1" target="_blank">griseorhodin A</a></td>
      <td>Polyketide:Type II</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r43c1">
     
     
     <td class="regbutton terpene r43c1">
      <a href="#r43c1">Region&nbsp43.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">14,296</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001181/1" target="_blank">geosmin</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r48c1">
     
     
     <td class="regbutton terpene r48c1">
      <a href="#r48c1">Region&nbsp48.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">19,460</td>
     <td class="digits table-split-left">41,244</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000242/1" target="_blank">lysolipin I</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r57c1">
     
     
     <td class="regbutton melanin r57c1">
      <a href="#r57c1">Region&nbsp57.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#melanin" target="_blank">melanin</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,883</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000909/1" target="_blank">melanin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r65c1">
     
     
     <td class="regbutton terpene r65c1">
      <a href="#r65c1">Region&nbsp65.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">25,575</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000663/1" target="_blank">hopene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r76c1">
     
     
     <td class="regbutton lanthipeptide-class-iii r76c1">
      <a href="#r76c1">Region&nbsp76.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
     </td>
     <td class="digits">32,111</td>
     <td class="digits table-split-left">54,714</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000501/1" target="_blank">catenulipeptin</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r91c1">
     
     
     <td class="regbutton T1PKS r91c1">
      <a href="#r91c1">Region&nbsp91.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">38,398</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001491/1" target="_blank">67-121C</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 29%, #ffffff00 29%)">29%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r98c1">
     
     
     <td class="regbutton NRPS r98c1">
      <a href="#r98c1">Region&nbsp98.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">31,464</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001370/1" target="_blank">lipopeptide 8D1-1 / lipopeptide 8D1-2</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 29%, #ffffff00 29%)">29%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r113c1">
     
     
     <td class="regbutton siderophore r113c1">
      <a href="#r113c1">Region&nbsp113.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,418</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r114c1">
     
     
         
     
     <td class="regbutton NRPS-like,T1PKS hybrid r114c1">
      <a href="#r114c1">Region&nbsp114.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">22,905</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000034/1" target="_blank">candicidin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 76%, #ffffff00 76%)">76%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r124c1">
     
     
     <td class="regbutton hglE-KS r124c1">
      <a href="#r124c1">Region&nbsp124.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hgle-ks" target="_blank">hglE-KS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">4,691</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r126c1">
     
     
     <td class="regbutton indole r126c1">
      <a href="#r126c1">Region&nbsp126.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
     </td>
     <td class="digits">69,237</td>
     <td class="digits table-split-left">90,358</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001483/1" target="_blank">5-isoprenylindole-3-carboxylate β-D-glycosyl ester</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r133c1">
     
     
     <td class="regbutton T2PKS r133c1">
      <a href="#r133c1">Region&nbsp133.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t2pks" target="_blank">T2PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">15,969</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000271/1" target="_blank">spore pigment</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 58%, #ffffff00 58%)">58%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r147c1">
     
     
     <td class="regbutton hglE-KS r147c1">
      <a href="#r147c1">Region&nbsp147.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hgle-ks" target="_blank">hglE-KS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,445</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001484/1" target="_blank">2&#39;-chloropentostatin / 2&#39;-amino-2&#39;-deoxyadenosine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r152c1">
     
     
     <td class="regbutton NRPS r152c1">
      <a href="#r152c1">Region&nbsp152.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">26,700</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000325/1" target="_blank">coelichelin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 72%, #ffffff00 72%)">72%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r162c1">
     
     
     <td class="regbutton RiPP-like r162c1">
      <a href="#r162c1">Region&nbsp162.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">5,565</td>
     <td class="digits table-split-left">11,354</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r205c1">
     
     
     <td class="regbutton T1PKS r205c1">
      <a href="#r205c1">Region&nbsp205.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
     </td>
     <td class="digits">11,984</td>
     <td class="digits table-split-left">94,247</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000167/1" target="_blank">vicenistatin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r224c1">
     
     
     <td class="regbutton terpene r224c1">
      <a href="#r224c1">Region&nbsp224.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">4,884</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000633/1" target="_blank">carotenoid</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 36%, #ffffff00 36%)">36%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r245c1">
     
     
     <td class="regbutton NRPS r245c1">
      <a href="#r245c1">Region&nbsp245.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">27,327</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000315/1" target="_blank">CDA1b / CDA2a / CDA2b / CDA3a / CDA3b / CDA4a / CDA4b</a></td>
      <td>NRP:Ca+-dependent lipopeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 20%, #ffffff00 20%)">20%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r255c1">
     
     
     <td class="regbutton T3PKS r255c1">
      <a href="#r255c1">Region&nbsp255.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">16,913</td>
     <td class="digits table-split-left">44,716</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000285/1" target="_blank">flaviolin rhamnoside / 3,3&#39;-diflaviolin / flaviolin</a></td>
      <td>Polyketide:Type III + Saccharide:Oligosaccharide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 33%, #ffffff00 33%)">33%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r280c1">
     
     
     <td class="regbutton terpene r280c1">
      <a href="#r280c1">Region&nbsp280.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">83,615</td>
     <td class="digits table-split-left">109,197</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000664/1" target="_blank">isorenieratene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 85%, #ffffff00 85%)">85%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r280c2">
     
     
     <td class="regbutton indole r280c2">
      <a href="#r280c2">Region&nbsp280.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
     </td>
     <td class="digits">109,641</td>
     <td class="digits table-split-left">130,819</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001294/1" target="_blank">7-prenylisatin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r280c3">
     
     
     <td class="regbutton NRPS-like r280c3">
      <a href="#r280c3">Region&nbsp280.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">167,481</td>
     <td class="digits table-split-left">189,031</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000432/1" target="_blank">streptothricin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 58%, #ffffff00 58%)">58%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r286c1">
     
     
     <td class="regbutton lanthipeptide-class-v r286c1">
      <a href="#r286c1">Region&nbsp286.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-v" target="_blank">lanthipeptide-class-v</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,743</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r302c1">
     
     
     <td class="regbutton RiPP-like r302c1">
      <a href="#r302c1">Region&nbsp302.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">7,630</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000518/1" target="_blank">informatipeptin</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r302c2">
     
     
     <td class="regbutton terpene r302c2">
      <a href="#r302c2">Region&nbsp302.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">20,588</td>
     <td class="digits table-split-left">34,425</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001204/1" target="_blank">versipelostatin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 5%, #ffffff00 5%)">5%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r307c1">
     
     
         
     
     <td class="regbutton T1PKS,transAT-PKS hybrid r307c1">
      <a href="#r307c1">Region&nbsp307.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#transat-pks" target="_blank">transAT-PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">51,032</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000034/1" target="_blank">candicidin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r330c1">
     
     
     <td class="regbutton ectoine r330c1">
      <a href="#r330c1">Region&nbsp330.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">3,670</td>
     <td class="digits table-split-left">14,068</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000853/1" target="_blank">ectoine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r341c1">
     
     
     <td class="regbutton siderophore r341c1">
      <a href="#r341c1">Region&nbsp341.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">11,240</td>
     <td class="digits table-split-left">23,012</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000940/1" target="_blank">desferrioxamin B / desferrioxamine E</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 83%, #ffffff00 83%)">83%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r344c1">
     
     
     <td class="regbutton lanthipeptide-class-iii r344c1">
      <a href="#r344c1">Region&nbsp344.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
     </td>
     <td class="digits">11,051</td>
     <td class="digits table-split-left">33,633</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000551/1" target="_blank">SapB</a></td>
      <td>RiPP:Lanthipeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  <tr class="linked-row even" data-anchor="#r344c2">
     
     
     <td class="regbutton lanthipeptide-class-i r344c2">
      <a href="#r344c2">Region&nbsp344.2</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-i" target="_blank">lanthipeptide-class-i</a>
     </td>
     <td class="digits">46,206</td>
     <td class="digits table-split-left">70,800</td>
     
      <td colspan="3"></td>
     
   </tr>
  <tr class="linked-row odd" data-anchor="#r344c3">
     
     
     <td class="regbutton NRPS r344c3">
      <a href="#r344c3">Region&nbsp344.3</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">74,572</td>
     <td class="digits table-split-left">126,897</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000324/1" target="_blank">coelibactin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r381c1">
     
     
     <td class="regbutton butyrolactone r381c1">
      <a href="#r381c1">Region&nbsp381.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
     </td>
     <td class="digits">1,842</td>
     <td class="digits table-split-left">8,461</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000238/1" target="_blank">lactonamycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 7%, #ffffff00 7%)">7%</td>
     
   </tr>
  
 
 </tbody>
</table>
      </div>
     </div>
     
     
     <dl id="regionless-records">
      <dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>
       
         <dd><strong>c00001_NODE_13..</strong> (original name was: NODE_131_length_24838_cov_202.507645) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00002_NODE_32..</strong> (original name was: NODE_329_length_2798_cov_169.691925) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00003_NODE_20..</strong> (original name was: NODE_208_length_30446_cov_131.288025) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00004_NODE_12..</strong> (original name was: NODE_122_length_21167_cov_212.961166) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00005_NODE_60..</strong> (original name was: NODE_60_length_23379_cov_168.501694) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00007_NODE_12..</strong> (original name was: NODE_123_length_2807_cov_131.180969) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00008_NODE_22..</strong> (original name was: NODE_22_length_31259_cov_158.534531) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00011_NODE_64..</strong> (original name was: NODE_64_length_116557_cov_207.192322) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00012_NODE_14..</strong> (original name was: NODE_142_length_12385_cov_176.180618) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00014_NODE_12..</strong> (original name was: NODE_125_length_60640_cov_167.240311) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00015_NODE_37..</strong> (original name was: NODE_373_length_1981_cov_130.536087) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00016_NODE_2_..</strong> (original name was: NODE_2_length_25258_cov_204.870056) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00018_NODE_35..</strong> (original name was: NODE_351_length_17735_cov_123.813644) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00019_NODE_28..</strong> (original name was: NODE_281_length_3278_cov_128.985046) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00020_NODE_23..</strong> (original name was: NODE_235_length_12070_cov_159.073151) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00021_NODE_30..</strong> (original name was: NODE_309_length_140371_cov_174.697174) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00023_NODE_30..</strong> (original name was: NODE_306_length_3050_cov_117.298363) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00024_NODE_67..</strong> (original name was: NODE_67_length_23983_cov_194.415543) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00025_NODE_42..</strong> (original name was: NODE_423_length_1667_cov_182.788239) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00026_NODE_33..</strong> (original name was: NODE_337_length_3714_cov_152.961502) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00027_NODE_22..</strong> (original name was: NODE_228_length_2512_cov_196.024277) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00028_NODE_98..</strong> (original name was: NODE_98_length_2285_cov_165.873093) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00029_NODE_27..</strong> (original name was: NODE_277_length_9776_cov_160.183914) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00030_NODE_85..</strong> (original name was: NODE_85_length_16249_cov_205.403656) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00031_NODE_38..</strong> (original name was: NODE_384_length_2237_cov_100.197136) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00032_NODE_15..</strong> (original name was: NODE_158_length_2945_cov_212.556534) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00033_NODE_92..</strong> (original name was: NODE_92_length_22851_cov_194.800797) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00034_NODE_16..</strong> (original name was: NODE_169_length_7207_cov_121.193283) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00036_NODE_35..</strong> (original name was: NODE_35_length_26778_cov_128.545975) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00038_NODE_87..</strong> (original name was: NODE_87_length_12692_cov_207.833908) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00039_NODE_24..</strong> (original name was: NODE_245_length_6076_cov_187.040329) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00040_NODE_31..</strong> (original name was: NODE_312_length_4374_cov_254.509598) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00041_NODE_9_..</strong> (original name was: NODE_9_length_8152_cov_190.146347) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00042_NODE_71..</strong> (original name was: NODE_71_length_59991_cov_193.014633) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00044_NODE_36..</strong> (original name was: NODE_361_length_33942_cov_181.502747) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00045_NODE_6_..</strong> (original name was: NODE_6_length_38244_cov_200.947174) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00046_NODE_21..</strong> (original name was: NODE_212_length_9700_cov_173.203705) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00047_NODE_16..</strong> (original name was: NODE_161_length_10275_cov_140.578781) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00049_NODE_76..</strong> (original name was: NODE_76_length_73133_cov_138.543900) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00050_NODE_55..</strong> (original name was: NODE_557_length_1022_cov_51.395302) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00051_NODE_55..</strong> (original name was: NODE_551_length_51066_cov_181.149826) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00052_NODE_34..</strong> (original name was: NODE_348_length_1836_cov_160.925385) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00053_NODE_26..</strong> (original name was: NODE_260_length_3491_cov_106.133774) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00054_NODE_64..</strong> (original name was: NODE_644_length_17105_cov_139.617188) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00055_NODE_25..</strong> (original name was: NODE_257_length_20371_cov_192.792404) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00056_NODE_24..</strong> (original name was: NODE_243_length_20551_cov_144.655533) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00058_NODE_13..</strong> (original name was: NODE_132_length_21641_cov_203.512589) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00059_NODE_32..</strong> (original name was: NODE_323_length_3855_cov_190.498566) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00060_NODE_62..</strong> (original name was: NODE_62_length_26113_cov_136.753036) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00061_NODE_24..</strong> (original name was: NODE_242_length_7399_cov_174.361938) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00062_NODE_77..</strong> (original name was: NODE_77_length_22464_cov_212.079956) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00063_NODE_21..</strong> (original name was: NODE_216_length_21101_cov_144.500549) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00064_NODE_10..</strong> (original name was: NODE_100_length_19709_cov_164.534424) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00066_NODE_25..</strong> (original name was: NODE_251_length_6934_cov_124.398468) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00067_NODE_35..</strong> (original name was: NODE_350_length_1928_cov_123.014526) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00068_NODE_29..</strong> (original name was: NODE_294_length_5127_cov_122.913010) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00069_NODE_79..</strong> (original name was: NODE_79_length_14404_cov_193.345673) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00070_NODE_42..</strong> (original name was: NODE_420_length_1717_cov_112.593475) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00071_NODE_35..</strong> (original name was: NODE_359_length_156909_cov_97.831863) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00072_NODE_17..</strong> (original name was: NODE_172_length_13883_cov_148.592957) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00073_NODE_21..</strong> (original name was: NODE_211_length_5667_cov_209.442383) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00074_NODE_20..</strong> (original name was: NODE_20_length_17783_cov_182.588425) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00075_NODE_20..</strong> (original name was: NODE_200_length_17791_cov_217.805740) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00077_NODE_20..</strong> (original name was: NODE_201_length_9486_cov_206.942017) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00078_NODE_32..</strong> (original name was: NODE_328_length_8841_cov_162.920593) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00079_NODE_39..</strong> (original name was: NODE_39_length_1536_cov_1124.403687) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00080_NODE_23..</strong> (original name was: NODE_231_length_11386_cov_173.788681) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00081_NODE_22..</strong> (original name was: NODE_229_length_9331_cov_162.558884) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00082_NODE_74..</strong> (original name was: NODE_74_length_21266_cov_219.935623) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00083_NODE_16..</strong> (original name was: NODE_167_length_13180_cov_182.498856) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00084_NODE_95..</strong> (original name was: NODE_95_length_5385_cov_725.114380) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00085_NODE_50..</strong> (original name was: NODE_503_length_1989_cov_115.346909) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00086_NODE_17..</strong> (original name was: NODE_171_length_1009_cov_1065.574829) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00087_NODE_22..</strong> (original name was: NODE_227_length_7942_cov_199.946365) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00088_NODE_1_..</strong> (original name was: NODE_1_length_9620_cov_128.595734) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00089_NODE_7_..</strong> (original name was: NODE_7_length_49942_cov_161.032852) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00090_NODE_41..</strong> (original name was: NODE_413_length_1806_cov_79.435768) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00092_NODE_21..</strong> (original name was: NODE_214_length_3177_cov_186.365433) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00093_NODE_32..</strong> (original name was: NODE_326_length_3152_cov_162.161484) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00094_NODE_96..</strong> (original name was: NODE_96_length_7202_cov_185.234940) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00095_NODE_47..</strong> (original name was: NODE_472_length_1182_cov_144.719116) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00096_NODE_41..</strong> (original name was: NODE_415_length_2524_cov_126.490891) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00097_NODE_54..</strong> (original name was: NODE_545_length_11251_cov_182.044266) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00099_NODE_13..</strong> (original name was: NODE_133_length_7780_cov_130.561050) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00100_NODE_39..</strong> (original name was: NODE_398_length_14097_cov_185.101013) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00101_NODE_36..</strong> (original name was: NODE_36_length_14598_cov_174.980621) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00102_NODE_61..</strong> (original name was: NODE_61_length_52259_cov_129.290344) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00103_NODE_42..</strong> (original name was: NODE_424_length_5451_cov_103.730690) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00104_NODE_29..</strong> (original name was: NODE_290_length_24444_cov_201.610001) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00105_NODE_15..</strong> (original name was: NODE_157_length_2210_cov_163.003616) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00106_NODE_16..</strong> (original name was: NODE_164_length_17189_cov_163.418167) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00107_NODE_12..</strong> (original name was: NODE_128_length_5750_cov_152.299484) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00108_NODE_19..</strong> (original name was: NODE_196_length_5774_cov_119.969521) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00109_NODE_29..</strong> (original name was: NODE_293_length_8018_cov_203.527191) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00110_NODE_13..</strong> (original name was: NODE_134_length_9870_cov_129.006790) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00111_NODE_89..</strong> (original name was: NODE_89_length_63637_cov_201.284119) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00112_NODE_10..</strong> (original name was: NODE_103_length_65897_cov_167.685013) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00115_NODE_12..</strong> (original name was: NODE_129_length_18015_cov_203.877441) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00116_NODE_10..</strong> (original name was: NODE_109_length_33417_cov_122.036804) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00117_NODE_18..</strong> (original name was: NODE_181_length_11387_cov_158.920258) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00118_NODE_38..</strong> (original name was: NODE_383_length_1689_cov_215.274124) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00119_NODE_57..</strong> (original name was: NODE_570_length_10980_cov_149.837250) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00120_NODE_15..</strong> (original name was: NODE_152_length_3981_cov_229.308212) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00121_NODE_46..</strong> (original name was: NODE_468_length_1123_cov_138.293854) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00122_NODE_15..</strong> (original name was: NODE_150_length_24713_cov_103.126778) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00123_NODE_42..</strong> (original name was: NODE_422_length_2364_cov_167.431473) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00125_NODE_30..</strong> (original name was: NODE_30_length_39598_cov_217.144455) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00127_NODE_12..</strong> (original name was: NODE_127_length_6336_cov_222.081604) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00128_NODE_17..</strong> (original name was: NODE_17_length_5895_cov_165.970825) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00129_NODE_10..</strong> (original name was: NODE_107_length_30205_cov_184.571136) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00130_NODE_47..</strong> (original name was: NODE_47_length_7963_cov_221.335434) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00131_NODE_97..</strong> (original name was: NODE_97_length_53247_cov_189.960831) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00132_NODE_38..</strong> (original name was: NODE_381_length_23404_cov_108.303368) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00134_NODE_23..</strong> (original name was: NODE_237_length_12310_cov_122.988060) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00135_NODE_58..</strong> (original name was: NODE_58_length_70157_cov_213.326065) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00136_NODE_45..</strong> (original name was: NODE_45_length_13610_cov_142.451218) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00137_NODE_26..</strong> (original name was: NODE_266_length_14716_cov_147.049606) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00138_NODE_13..</strong> (original name was: NODE_135_length_12696_cov_194.133591) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00139_NODE_19..</strong> (original name was: NODE_190_length_85166_cov_203.771545) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00140_NODE_42..</strong> (original name was: NODE_426_length_35921_cov_174.430496) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00141_NODE_17..</strong> (original name was: NODE_178_length_16045_cov_155.590591) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00142_NODE_32..</strong> (original name was: NODE_325_length_5650_cov_148.105484) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00143_NODE_35..</strong> (original name was: NODE_352_length_1044_cov_94.663795) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00144_NODE_22..</strong> (original name was: NODE_222_length_11056_cov_166.437042) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00145_NODE_29..</strong> (original name was: NODE_298_length_26962_cov_175.469223) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00146_NODE_24..</strong> (original name was: NODE_248_length_3283_cov_141.431610) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00148_NODE_22..</strong> (original name was: NODE_226_length_16694_cov_140.307663) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00149_NODE_83..</strong> (original name was: NODE_83_length_8160_cov_121.281738) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00150_NODE_14..</strong> (original name was: NODE_148_length_51255_cov_183.886261) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00151_NODE_11..</strong> (original name was: NODE_118_length_12332_cov_217.425232) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00153_NODE_15..</strong> (original name was: NODE_154_length_4801_cov_186.626114) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00154_NODE_16..</strong> (original name was: NODE_166_length_2792_cov_132.521851) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00155_NODE_65..</strong> (original name was: NODE_65_length_51102_cov_127.595200) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00156_NODE_39..</strong> (original name was: NODE_393_length_1566_cov_166.872284) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00157_NODE_34..</strong> (original name was: NODE_341_length_3388_cov_151.683289) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00158_NODE_86..</strong> (original name was: NODE_86_length_11301_cov_121.016815) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00159_NODE_34..</strong> (original name was: NODE_34_length_27587_cov_137.784973) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00160_NODE_99..</strong> (original name was: NODE_99_length_43256_cov_190.124496) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00161_NODE_27..</strong> (original name was: NODE_274_length_15675_cov_189.008163) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00163_NODE_18..</strong> (original name was: NODE_182_length_22008_cov_155.267670) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00164_NODE_81..</strong> (original name was: NODE_81_length_9546_cov_116.825790) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00165_NODE_33..</strong> (original name was: NODE_330_length_6345_cov_137.989120) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00166_NODE_15..</strong> (original name was: NODE_15_length_11201_cov_183.232117) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00167_NODE_11..</strong> (original name was: NODE_114_length_14700_cov_135.935852) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00168_NODE_39..</strong> (original name was: NODE_391_length_1632_cov_102.653801) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00169_NODE_21..</strong> (original name was: NODE_217_length_6639_cov_176.325043) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00170_NODE_16..</strong> (original name was: NODE_160_length_14156_cov_107.992798) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00171_NODE_23..</strong> (original name was: NODE_238_length_8113_cov_201.537537) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00172_NODE_56..</strong> (original name was: NODE_56_length_12000_cov_165.941589) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00173_NODE_30..</strong> (original name was: NODE_307_length_9489_cov_116.230370) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00174_NODE_18..</strong> (original name was: NODE_18_length_36433_cov_151.200470) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00175_NODE_47..</strong> (original name was: NODE_477_length_7447_cov_152.941177) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00176_NODE_16..</strong> (original name was: NODE_168_length_16555_cov_224.550461) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00177_NODE_33..</strong> (original name was: NODE_335_length_15151_cov_109.526237) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00178_NODE_15..</strong> (original name was: NODE_156_length_12050_cov_158.264984) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00179_NODE_12..</strong> (original name was: NODE_126_length_17538_cov_214.273804) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00180_NODE_93..</strong> (original name was: NODE_93_length_28609_cov_178.946304) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00181_NODE_94..</strong> (original name was: NODE_94_length_58850_cov_207.922607) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00182_NODE_19..</strong> (original name was: NODE_192_length_26212_cov_215.093018) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00183_NODE_32..</strong> (original name was: NODE_322_length_5769_cov_155.349625) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00184_NODE_10..</strong> (original name was: NODE_101_length_12878_cov_190.958527) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00185_NODE_20..</strong> (original name was: NODE_207_length_26603_cov_173.479828) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00186_NODE_91..</strong> (original name was: NODE_91_length_5450_cov_152.246979) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00187_NODE_33..</strong> (original name was: NODE_338_length_1986_cov_163.353973) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00188_NODE_24..</strong> (original name was: NODE_24_length_38412_cov_153.987610) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00189_NODE_21..</strong> (original name was: NODE_21_length_22794_cov_109.866676) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00190_NODE_29..</strong> (original name was: NODE_296_length_3530_cov_144.668839) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00191_NODE_36..</strong> (original name was: NODE_364_length_1109_cov_117.331833) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00192_NODE_14..</strong> (original name was: NODE_143_length_4762_cov_148.942886) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00193_NODE_18..</strong> (original name was: NODE_184_length_16554_cov_209.424362) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00194_NODE_36..</strong> (original name was: NODE_365_length_5024_cov_167.400284) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00195_NODE_15..</strong> (original name was: NODE_153_length_18351_cov_176.727921) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00196_NODE_23..</strong> (original name was: NODE_230_length_19095_cov_190.286835) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00197_NODE_31..</strong> (original name was: NODE_319_length_6288_cov_95.312820) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00198_NODE_13..</strong> (original name was: NODE_136_length_36969_cov_196.744980) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00199_NODE_19..</strong> (original name was: NODE_197_length_3820_cov_153.140320) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00200_NODE_25..</strong> (original name was: NODE_259_length_18169_cov_103.672577) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00201_NODE_33..</strong> (original name was: NODE_33_length_6412_cov_123.036652) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00202_NODE_54..</strong> (original name was: NODE_54_length_31337_cov_196.141785) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00203_NODE_23..</strong> (original name was: NODE_234_length_10791_cov_143.051620) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00204_NODE_36..</strong> (original name was: NODE_363_length_6107_cov_100.739479) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00206_NODE_37..</strong> (original name was: NODE_379_length_1442_cov_136.866852) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00207_NODE_11..</strong> (original name was: NODE_116_length_13571_cov_130.416336) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00208_NODE_66..</strong> (original name was: NODE_66_length_73324_cov_205.738480) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00209_NODE_29..</strong> (original name was: NODE_292_length_2767_cov_81.489342) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00210_NODE_4_..</strong> (original name was: NODE_4_length_38972_cov_215.096527) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00211_NODE_14..</strong> (original name was: NODE_147_length_95982_cov_149.888321) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00212_NODE_15..</strong> (original name was: NODE_159_length_7841_cov_109.053696) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00213_NODE_17..</strong> (original name was: NODE_177_length_22605_cov_119.581467) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00214_NODE_19..</strong> (original name was: NODE_19_length_16318_cov_155.253464) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00215_NODE_37..</strong> (original name was: NODE_378_length_34912_cov_196.583099) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00216_NODE_10..</strong> (original name was: NODE_104_length_12654_cov_117.548599) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00217_NODE_34..</strong> (original name was: NODE_343_length_23073_cov_190.797775) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00218_NODE_32..</strong> (original name was: NODE_327_length_58538_cov_169.615616) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00219_NODE_54..</strong> (original name was: NODE_544_length_12696_cov_155.043320) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00220_NODE_27..</strong> (original name was: NODE_275_length_2765_cov_208.108505) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00221_NODE_34..</strong> (original name was: NODE_345_length_2313_cov_101.583656) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00222_NODE_40..</strong> (original name was: NODE_406_length_1492_cov_151.556976) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00223_NODE_49..</strong> (original name was: NODE_49_length_57755_cov_200.095917) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00225_NODE_11..</strong> (original name was: NODE_117_length_133117_cov_112.988632) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00226_NODE_33..</strong> (original name was: NODE_332_length_16595_cov_123.842003) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00227_NODE_26..</strong> (original name was: NODE_262_length_10505_cov_153.361923) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00228_NODE_46..</strong> (original name was: NODE_463_length_1125_cov_131.208893) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00229_NODE_31..</strong> (original name was: NODE_310_length_48228_cov_112.444427) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00230_NODE_45..</strong> (original name was: NODE_457_length_53952_cov_121.083961) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00231_NODE_12..</strong> (original name was: NODE_124_length_8788_cov_105.283684) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00232_NODE_19..</strong> (original name was: NODE_191_length_3270_cov_201.632416) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00233_NODE_34..</strong> (original name was: NODE_347_length_25557_cov_165.756348) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00234_NODE_11..</strong> (original name was: NODE_111_length_23038_cov_139.919998) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00235_NODE_14..</strong> (original name was: NODE_149_length_23035_cov_147.289993) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00236_NODE_29..</strong> (original name was: NODE_29_length_49841_cov_215.638214) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00237_NODE_30..</strong> (original name was: NODE_300_length_1641_cov_160.780014) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00238_NODE_36..</strong> (original name was: NODE_362_length_56386_cov_182.105652) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00239_NODE_45..</strong> (original name was: NODE_452_length_68591_cov_163.574966) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00240_NODE_34..</strong> (original name was: NODE_346_length_4759_cov_174.596970) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00241_NODE_60..</strong> (original name was: NODE_606_length_8154_cov_127.414520) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00242_NODE_26..</strong> (original name was: NODE_263_length_5540_cov_173.290970) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00243_NODE_10..</strong> (original name was: NODE_108_length_26048_cov_188.310074) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00244_NODE_35..</strong> (original name was: NODE_353_length_3328_cov_176.646637) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00246_NODE_16..</strong> (original name was: NODE_16_length_31646_cov_163.703247) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00247_NODE_11..</strong> (original name was: NODE_115_length_16330_cov_131.439377) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00248_NODE_31..</strong> (original name was: NODE_31_length_11112_cov_170.308228) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00249_NODE_25..</strong> (original name was: NODE_255_length_6829_cov_125.516327) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00250_NODE_33..</strong> (original name was: NODE_331_length_10112_cov_117.900612) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00251_NODE_30..</strong> (original name was: NODE_302_length_1265_cov_224.209488) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00252_NODE_10..</strong> (original name was: NODE_105_length_12648_cov_117.635910) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00253_NODE_18..</strong> (original name was: NODE_185_length_19577_cov_181.427689) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00254_NODE_42..</strong> (original name was: NODE_42_length_14712_cov_194.381866) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00256_NODE_14..</strong> (original name was: NODE_146_length_18173_cov_114.752708) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00257_NODE_28..</strong> (original name was: NODE_288_length_1333_cov_264.023254) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00258_NODE_20..</strong> (original name was: NODE_206_length_37345_cov_167.275192) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00259_NODE_24..</strong> (original name was: NODE_241_length_9642_cov_165.494293) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00260_NODE_29..</strong> (original name was: NODE_295_length_7165_cov_143.534683) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00261_NODE_11..</strong> (original name was: NODE_119_length_8375_cov_215.264236) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00262_NODE_35..</strong> (original name was: NODE_355_length_13629_cov_152.235596) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00263_NODE_39..</strong> (original name was: NODE_396_length_2029_cov_125.926567) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00264_NODE_51..</strong> (original name was: NODE_511_length_42691_cov_139.231262) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00265_NODE_20..</strong> (original name was: NODE_209_length_8336_cov_148.776031) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00266_NODE_30..</strong> (original name was: NODE_308_length_8783_cov_126.208588) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00267_NODE_27..</strong> (original name was: NODE_278_length_10151_cov_175.403702) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00268_NODE_37..</strong> (original name was: NODE_374_length_4320_cov_171.563660) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00269_NODE_31..</strong> (original name was: NODE_317_length_8027_cov_129.613052) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00270_NODE_16..</strong> (original name was: NODE_162_length_33840_cov_192.362350) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00271_NODE_16..</strong> (original name was: NODE_165_length_30755_cov_216.208649) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00272_NODE_18..</strong> (original name was: NODE_180_length_9782_cov_210.970566) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00273_NODE_38..</strong> (original name was: NODE_380_length_2107_cov_199.289505) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00274_NODE_53..</strong> (original name was: NODE_53_length_4714_cov_231.834747) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00275_NODE_47..</strong> (original name was: NODE_478_length_1018_cov_120.266205) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00276_NODE_39..</strong> (original name was: NODE_397_length_2218_cov_108.595131) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00277_NODE_70..</strong> (original name was: NODE_70_length_6829_cov_206.748871) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00278_NODE_19..</strong> (original name was: NODE_193_length_31794_cov_106.616592) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00279_NODE_35..</strong> (original name was: NODE_354_length_7866_cov_205.340454) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00281_NODE_90..</strong> (original name was: NODE_90_length_16227_cov_222.467987) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00282_NODE_52..</strong> (original name was: NODE_52_length_16158_cov_200.110291) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00283_NODE_50..</strong> (original name was: NODE_50_length_55809_cov_187.231430) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00284_NODE_41..</strong> (original name was: NODE_418_length_9323_cov_110.667274) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00285_NODE_21..</strong> (original name was: NODE_215_length_7299_cov_194.203308) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00287_NODE_31..</strong> (original name was: NODE_314_length_11860_cov_185.730347) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00288_NODE_21..</strong> (original name was: NODE_218_length_17220_cov_110.495880) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00289_NODE_32..</strong> (original name was: NODE_324_length_12018_cov_138.421616) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00290_NODE_22..</strong> (original name was: NODE_223_length_8467_cov_133.398483) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00291_NODE_29..</strong> (original name was: NODE_299_length_11675_cov_157.289337) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00292_NODE_23..</strong> (original name was: NODE_23_length_15540_cov_184.543503) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00293_NODE_12..</strong> (original name was: NODE_120_length_38980_cov_203.083939) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00294_NODE_40..</strong> (original name was: NODE_403_length_1215_cov_133.531693) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00295_NODE_13..</strong> (original name was: NODE_138_length_5295_cov_148.750137) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00296_NODE_30..</strong> (original name was: NODE_301_length_6073_cov_130.961639) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00297_NODE_48..</strong> (original name was: NODE_48_length_16197_cov_131.143295) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00298_NODE_19..</strong> (original name was: NODE_198_length_13359_cov_207.060028) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00299_NODE_17..</strong> (original name was: NODE_175_length_31712_cov_203.222382) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00300_NODE_3_..</strong> (original name was: NODE_3_length_42283_cov_172.248840) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00301_NODE_47..</strong> (original name was: NODE_470_length_18971_cov_116.221390) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00303_NODE_68..</strong> (original name was: NODE_68_length_21210_cov_188.230499) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00304_NODE_53..</strong> (original name was: NODE_530_length_1420_cov_108.228172) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00305_NODE_31..</strong> (original name was: NODE_318_length_6994_cov_107.154564) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00306_NODE_31..</strong> (original name was: NODE_311_length_8584_cov_167.100296) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00308_NODE_41..</strong> (original name was: NODE_416_length_10593_cov_121.717453) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00309_NODE_21..</strong> (original name was: NODE_213_length_3946_cov_125.106438) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00310_NODE_10..</strong> (original name was: NODE_102_length_21021_cov_209.698257) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00311_NODE_46..</strong> (original name was: NODE_46_length_8011_cov_133.921860) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00312_NODE_40..</strong> (original name was: NODE_407_length_1965_cov_89.096184) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00313_NODE_38..</strong> (original name was: NODE_389_length_1303_cov_212.624710) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00314_NODE_16..</strong> (original name was: NODE_163_length_11442_cov_196.460327) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00315_NODE_5_..</strong> (original name was: NODE_5_length_31291_cov_170.664856) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00316_NODE_22..</strong> (original name was: NODE_224_length_1419_cov_159.744888) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00317_NODE_19..</strong> (original name was: NODE_195_length_18883_cov_117.189484) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00318_NODE_24..</strong> (original name was: NODE_249_length_5485_cov_211.417313) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00319_NODE_27..</strong> (original name was: NODE_276_length_4655_cov_137.216537) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00320_NODE_28..</strong> (original name was: NODE_284_length_5156_cov_197.070984) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00321_NODE_19..</strong> (original name was: NODE_199_length_10793_cov_137.333649) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00322_NODE_18..</strong> (original name was: NODE_186_length_3086_cov_154.308487) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00323_NODE_18..</strong> (original name was: NODE_183_length_4541_cov_123.060562) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00324_NODE_21..</strong> (original name was: NODE_210_length_10943_cov_136.055832) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00325_NODE_63..</strong> (original name was: NODE_63_length_21330_cov_174.249924) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00326_NODE_53..</strong> (original name was: NODE_537_length_1353_cov_71.577972) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00327_NODE_20..</strong> (original name was: NODE_205_length_3148_cov_168.671539) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00328_NODE_55..</strong> (original name was: NODE_55_length_50625_cov_208.639008) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00329_NODE_20..</strong> (original name was: NODE_202_length_1212_cov_130.923264) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00331_NODE_18..</strong> (original name was: NODE_189_length_23258_cov_145.047165) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00332_NODE_28..</strong> (original name was: NODE_287_length_14182_cov_152.640106) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00333_NODE_34..</strong> (original name was: NODE_349_length_4486_cov_173.548599) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00334_NODE_40..</strong> (original name was: NODE_404_length_7316_cov_169.377670) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00335_NODE_27..</strong> (original name was: NODE_27_length_58015_cov_226.341965) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00336_NODE_24..</strong> (original name was: NODE_240_length_16570_cov_208.401093) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00337_NODE_28..</strong> (original name was: NODE_289_length_5631_cov_151.917068) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00338_NODE_82..</strong> (original name was: NODE_82_length_15588_cov_105.532265) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00339_NODE_41..</strong> (original name was: NODE_41_length_27996_cov_203.954529) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00340_NODE_13..</strong> (original name was: NODE_137_length_22630_cov_183.188019) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00342_NODE_57..</strong> (original name was: NODE_57_length_5656_cov_145.612442) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00343_NODE_12..</strong> (original name was: NODE_121_length_19950_cov_170.673584) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00345_NODE_45..</strong> (original name was: NODE_450_length_1583_cov_74.328491) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00346_NODE_40..</strong> (original name was: NODE_405_length_21472_cov_178.641388) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00347_NODE_11..</strong> (original name was: NODE_112_length_19957_cov_182.474976) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00348_NODE_24..</strong> (original name was: NODE_246_length_27687_cov_197.026474) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00349_NODE_47..</strong> (original name was: NODE_475_length_67654_cov_106.668785) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00350_NODE_38..</strong> (original name was: NODE_385_length_57675_cov_120.018326) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00351_NODE_11..</strong> (original name was: NODE_110_length_35799_cov_135.479202) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00352_NODE_20..</strong> (original name was: NODE_203_length_16499_cov_120.485001) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00353_NODE_26..</strong> (original name was: NODE_26_length_41537_cov_193.958786) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00354_NODE_20..</strong> (original name was: NODE_204_length_7959_cov_135.038696) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00355_NODE_23..</strong> (original name was: NODE_233_length_8156_cov_214.536407) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00356_NODE_59..</strong> (original name was: NODE_59_length_37643_cov_149.598358) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00357_NODE_23..</strong> (original name was: NODE_236_length_18359_cov_114.803802) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00358_NODE_26..</strong> (original name was: NODE_264_length_11080_cov_111.908119) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00359_NODE_44..</strong> (original name was: NODE_442_length_1344_cov_96.682289) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00360_NODE_38..</strong> (original name was: NODE_386_length_1323_cov_109.907784) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00361_NODE_22..</strong> (original name was: NODE_220_length_5193_cov_130.509918) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00362_NODE_25..</strong> (original name was: NODE_256_length_3167_cov_222.650146) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00363_NODE_28..</strong> (original name was: NODE_283_length_23536_cov_170.285599) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00364_NODE_28..</strong> (original name was: NODE_28_length_38452_cov_213.273041) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00365_NODE_52..</strong> (original name was: NODE_523_length_13417_cov_136.393005) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00366_NODE_26..</strong> (original name was: NODE_268_length_1785_cov_149.676758) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00367_NODE_60..</strong> (original name was: NODE_608_length_17497_cov_189.616913) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00368_NODE_41..</strong> (original name was: NODE_417_length_6791_cov_124.510086) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00369_NODE_26..</strong> (original name was: NODE_265_length_8432_cov_100.680145) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00370_NODE_37..</strong> (original name was: NODE_375_length_6509_cov_120.294823) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00371_NODE_33..</strong> (original name was: NODE_336_length_19775_cov_164.371780) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00372_NODE_27..</strong> (original name was: NODE_273_length_1220_cov_244.619675) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00373_NODE_17..</strong> (original name was: NODE_176_length_64822_cov_128.841660) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00374_NODE_24..</strong> (original name was: NODE_247_length_5213_cov_227.113373) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00375_NODE_14..</strong> (original name was: NODE_141_length_5405_cov_173.110275) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00376_NODE_69..</strong> (original name was: NODE_69_length_54736_cov_214.402283) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00377_NODE_35..</strong> (original name was: NODE_356_length_4170_cov_153.066422) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00378_NODE_15..</strong> (original name was: NODE_155_length_11144_cov_156.551682) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00379_NODE_78..</strong> (original name was: NODE_78_length_24311_cov_224.645218) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00380_NODE_10..</strong> (original name was: NODE_106_length_21819_cov_112.091988) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00382_NODE_80..</strong> (original name was: NODE_80_length_47287_cov_195.261429) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00383_NODE_28..</strong> (original name was: NODE_285_length_14705_cov_148.494522) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00384_NODE_35..</strong> (original name was: NODE_357_length_4403_cov_116.253693) (Streptomyces rochei CC48)</dd>
       
         <dd><strong>c00385_NODE_15..</strong> (original name was: NODE_151_length_45031_cov_195.034027) (Streptomyces rochei CC48)</dd>
       
     </dl>
     
    </div>
    
    <div class="overview-switches">
     <div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    
   </div>
  </div>
  






<div class="page" id='r6c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00006_NODE_25.. - Region 1 - T2PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-253"></div> <span class="help-tooltip" id="region_svg-help-253">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00006_NODE_25...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r6c1-svg" data-filename="c00006_NODE_25.._r6c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 11,162 nt. (total: 11,162 nt)
        
        <a class="cluster-rules-header" id="r6c1-rules-header" href="#r6c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r6c1-rules">T2PKS: (t2ks and t2clf)</div>
      
      <div class="region-svg-container">
       <div id='r6c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-254"></div> <span class="help-tooltip" id="focus-panel-help-254">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r6c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r6c1-nrps_pks" data-name="r6c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r6c1-clusterblast" data-name="r6c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r6c1-knownclusterblast" data-name="r6c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r6c1-subclusterblast" data-name="r6c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r6c1-MIBiG-cluster-compare" data-name="r6c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r6c1-pfam-details" data-name="r6c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r6c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-1"></div> <span class="help-tooltip" id="nrps-domain-help-1">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c1-details-svg" data-filename="c00006_NODE_25.._r6c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r6c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r6c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-2"></div> <span class="help-tooltip" id="cb-general-help-2">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r6c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r6c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r6c1_1.svg">
        NZ_CP029191 (7559-80048): Streptomyces venezuelae strain ATCC 14585 chromosom... (20% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r6c1_2.svg">
        NZ_CP029193 (2368-74848): Streptomyces venezuelae strain ATCC 14583 chromosom... (20% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r6c1_3.svg">
        NZ_CP029192 (14795-87275): Streptomyces venezuelae strain ATCC 14584 chromoso... (19% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r6c1_4.svg">
        NZ_CP025407 (967544-1050802): Streptomyces sp. CMB-StM0423 chromosome, comple... (14% of genes show similarity), T2PKS,other
      </option>
      
      <option value = "svg/clusterblast_r6c1_5.svg">
        NZ_MTBP01000002 (62814-135282): Actinomadura sp. RB29 Contig 2, whole genome ... (16% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r6c1_6.svg">
        NZ_CP011492 (7057593-7144434): Streptomyces sp. CNQ-509 chromosome, complete ... (16% of genes show similarity), T2PKS,other
      </option>
      
      <option value = "svg/clusterblast_r6c1_7.svg">
        NZ_CP029618 (7300304-7383550): Streptomyces sp. WAC 06738 chromosome, complet... (15% of genes show similarity), T2PKS,other
      </option>
      
      <option value = "svg/clusterblast_r6c1_8.svg">
        NZ_CP029043 (2833052-2903922): Streptomyces nigra strain 452 chromosome, comp... (19% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r6c1_9.svg">
        NZ_CM000951 (6910151-6982631): Streptomyces sviceus ATCC 29083 chromosome, wh... (15% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r6c1_10.svg">
        NZ_CP053189 (7556758-7629244): Streptomyces sp. jing01 chromosome, complete g... (19% of genes show similarity), T2PKS
      </option>
      
    </select>
    <button id= "clusterblast-r6c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r6c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-3"></div> <span class="help-tooltip" id="cb-known-help-3">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r6c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r6c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_1.svg">
        kinamycin
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_2.svg">
        fluostatins M-Q
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_3.svg">
        saprolmycin E
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_4.svg">
        saquayamycin A
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_5.svg">
        BE-7585A
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_6.svg">
        Sch-47554 / Sch-47555
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_7.svg">
        kiamycin
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_8.svg">
        lugdunomycin
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_9.svg">
        oviedomycin
      </option>
      
      <option value = "svg/knownclusterblast_r6c1_10.svg">
        oviedomycin
      </option>
      
    </select>
    <button id="knownclusterblast-r6c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r6c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r6c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-4"></div> <span class="help-tooltip" id="cb-sub-help-4">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r6c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r6c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r6c1_1.svg">
        AF080235_c1: landomycin t2pks (100% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r6c1_2.svg">
        AF324838_c1: simocyclinone t2pks (100% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r6c1_3.svg">
        GU233672_c1: granaticin t2pks (71% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r6c1_4.svg">
        AB103463_c1: medermycin t2pks (75% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r6c1_5.svg">
        FJ483966_c1: polyketomycin t2pks (25% of genes show similarity), t2pks
      </option>
      
    </select>
    <button id="subclusterblast-r6c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r6c1-svg"></div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r6c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-6"></div> <span class="help-tooltip" id="MIBiG-help-6">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r6c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r6c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r6c1"></div>
    <div id="comparison-MIBiG-r6c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T2PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.80 (id:0.58, order:0.90, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
         
        
        <td>0.53</td>
        <td>Polyketide</td>
        
        <td>griseusin</td>
        
        
        <td>Streptomyces griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.78 (id:0.53, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.78), rgb(0, 0, 0, 0.78) 78%, white 78%)"></td>
         
        
        <td>0.48</td>
        <td>Polyketide</td>
        
        <td>naphtocyclinone</td>
        
        
        <td>Streptomyces arenae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.74 (id:0.61, order:0.73, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
         
        
        <td>0.43</td>
        <td>Polyketide</td>
        
        <td>jadomycin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000236.1: 35-34224">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000236/index.html#r1c1" target="_new">BGC0000236.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.59, order:0.72, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Polyketide</td>
        
        <td>kinamycin</td>
        
        
        <td>Streptomyces murayamaensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001596.1: 5974061-6020749">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001596/index.html#r1c1" target="_new">BGC0001596.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.58, order:0.69, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Polyketide</td>
        
        <td>fluostatins M-Q</td>
        
        
        <td>Streptomyces albus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.53, order:0.46, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Polyketide</td>
        
        <td>granaticin</td>
        
        
        <td>Streptomyces violaceoruber</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.53, order:0.45, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Polyketide</td>
        
        <td>curamycin</td>
        
        
        <td>Streptomyces cyaneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723.1: 128-10733">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.57, order:0.35, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>oviedomycin</td>
        
        
        <td>Streptomyces antibioticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.50, order:0.37, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>spore pigment</td>
        
        
        <td>Streptomyces collinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001895.1: 2411904-2429820">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001895/index.html#r1c1" target="_new">BGC0001895.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.58, order:0.46, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>huanglongmycin A, huanglongmycin B, huanglongmycin C</td>
        
        
        <td>Streptomyces sp. CB09001</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r6c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.80 (id:0.58, order:0.90, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
      <td>0.80</td>
      <td>Polyketide</td>
      <td>griseusin</td>
      <td>Streptomyces griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.78 (id:0.53, order:0.88, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.78), rgb(0, 0, 0, 0.78) 78%, white 78%)"></td>
      <td>0.78</td>
      <td>Polyketide</td>
      <td>naphtocyclinone</td>
      <td>Streptomyces arenae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.74 (id:0.61, order:0.73, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
      <td>0.74</td>
      <td>Polyketide</td>
      <td>jadomycin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000236.1: 35-34224">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000236/index.html#r1c1" target="_new">BGC0000236.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.59, order:0.72, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>kinamycin</td>
      <td>Streptomyces murayamaensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001596.1: 5974061-6020749">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001596/index.html#r1c1" target="_new">BGC0001596.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.58, order:0.69, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>fluostatins M-Q</td>
      <td>Streptomyces albus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.53, order:0.46, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>granaticin</td>
      <td>Streptomyces violaceoruber</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.53, order:0.45, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>curamycin</td>
      <td>Streptomyces cyaneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001723.1: 128-10733">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001723/index.html#r1c1" target="_new">BGC0001723.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.57, order:0.35, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>oviedomycin</td>
      <td>Streptomyces antibioticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.50, order:0.37, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>spore pigment</td>
      <td>Streptomyces collinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001895.1: 2411904-2429820">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001895/index.html#r1c1" target="_new">BGC0001895.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.58, order:0.46, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>huanglongmycin A, huanglongmycin B, huanglongmycin C</td>
      <td>Streptomyces sp. CB09001</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r6c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-7"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-7">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c1-pfam-details-svg" data-filename="c00006_NODE_25.._r6c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r6c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
      <div class="sidepanel-details-header r6c1-t2pks" data-name="r6c1-t2pks">
        Type II PKS
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    <div class="sidepanel-details-section r6c1-t2pks">
      <div class="more-details">
    <div class="heading">
      <span>Type II PKS gene cluster prediction details</span>
      <div class="help-container"> <div class="help-icon" data-id="t2pks-help-5"></div> <span class="help-tooltip" id="t2pks-help-5">Predictions of starter units, elongations, product classes, and potential molecular weights for type II PKS clusters.<br>More detailed information is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/t2pks' target='_blank'>here</a>.</span></div>
    </div>
	
	
	<dl class="prediction-text">
        <dt><strong>Cluster section:</strong></dt>
        <dd>0 to 11162</dd>
		
		<dt><strong>Starter unit(s):</strong></dt>
		<dd>
				acetyl-CoA (Score: 0.0; E-value: 0)<br>
			
	    <dd>
		</tr>
		
		
		<dt><strong>Malonyl elongations:</strong></dt>
        <dd>
			
			8|9 (Score: 704.2; E-value: 9.7e-215)<br>
			
    	</dd>
		
		
		<dt><strong>Product class(es):</strong></td>
        <dd>
				
				angucycline<br>
				
		</dd>
		
		
		<dt><strong>Product molecular weight(s):</strong></dt>
		<dd>
			
			acetyl-CoA_8: 381.35 Da<br>
			
			acetyl-CoA_9: 423.39 Da<br>
			
		</dd>
		
	</dl>
	<dl class="prediction-text">
		
		<dt><strong>cds1591_3067</strong></dt>
			
			<dd>
				OXY
				
				(Score: 583.2; E-value: 1e-177)<br>
			</dd>
			
		
		<dt><strong>cds1_1489</strong></dt>
			
			<dd>
				OXY
				
				(Score: 560.9; E-value: 5.9e-171)<br>
			</dd>
			
		
		<dt><strong>cds3070_3628</strong></dt>
			
			<dd>
				CYC
				
					C7-C12
				
				(Score: 261.5; E-value: 1.9e-80)<br>
			</dd>
			
		
		<dt><strong>cds4056_4845</strong></dt>
			
			<dd>
				KR
				
					C9
				
				(Score: 472.0; E-value: 1.1e-144)<br>
			</dd>
			
		
		<dt><strong>cds4857_5127</strong></dt>
			
			<dd>
				ACP
				
				(Score: 122.9; E-value: 9.4e-39)<br>
			</dd>
			
		
		<dt><strong>cds5193_6405</strong></dt>
			
			<dd>
				CLF
				
					8|9
				
				(Score: 704.2; E-value: 9.7e-215)<br>
			</dd>
			
		
		<dt><strong>cds7668_7998</strong></dt>
			
			<dd>
				CYC
				
					C4-C17/C2-C19
				
				(Score: 195.8; E-value: 1.6e-60)<br>
			</dd>
			
		
		<dt><strong>cds9008_10481</strong></dt>
			
			<dd>
				OXY
				
				(Score: 509.7; E-value: 2e-155)<br>
			</dd>
			
		
	</dl>
    
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r9c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00009_NODE_26.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-255"></div> <span class="help-tooltip" id="region_svg-help-255">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00009_NODE_26...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r9c1-svg" data-filename="c00009_NODE_26.._r9c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 5,998 nt. (total: 5,998 nt)
        
        <a class="cluster-rules-header" id="r9c1-rules-header" href="#r9c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r9c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r9c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-256"></div> <span class="help-tooltip" id="focus-panel-help-256">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r9c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r9c1-nrps_pks" data-name="r9c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r9c1-clusterblast" data-name="r9c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r9c1-knownclusterblast" data-name="r9c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r9c1-subclusterblast" data-name="r9c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r9c1-MIBiG-cluster-compare" data-name="r9c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r9c1-pfam-details" data-name="r9c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r9c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-8"></div> <span class="help-tooltip" id="nrps-domain-help-8">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r9c1-details-svg" data-filename="c00009_NODE_26.._r9c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r9c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r9c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-11"></div> <span class="help-tooltip" id="cb-general-help-11">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r9c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r9c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r9c1_1.svg">
        NZ_LT629775 (671746-755178): Streptomyces sp. TLI 053 chromosome I (11% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r9c1_2.svg">
        NZ_ML123100 (938021-1027917): Streptomyces sp. ADI97-07 scaffold003, whole ge... (8% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r9c1_3.svg">
        NZ_RHGC01000002 (377072-523598): Streptomyces alboflavus strain 313 contig002... (4% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r9c1_4.svg">
        NZ_CP039123 (7658917-7709431): Streptomyces sp. SS52 chromosome, complete genome (5% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r9c1_5.svg">
        NZ_LISW01000009 (422925-493837): Streptomyces sp. CB01249 scaffold9, whole ge... (6% of genes show similarity), NRPS,betalactone
      </option>
      
      <option value = "svg/clusterblast_r9c1_6.svg">
        NZ_KB891791 (39875-110337): Streptomyces sp. LaPpAH-165 B070DRAFT scaffold 4.... (6% of genes show similarity), NRPS,betalactone
      </option>
      
      <option value = "svg/clusterblast_r9c1_7.svg">
        NZ_LIVY01000013 (54813-125702): Streptomyces sp. CB02460 scaffold13, whole ge... (6% of genes show similarity), NRPS,betalactone
      </option>
      
      <option value = "svg/clusterblast_r9c1_8.svg">
        NZ_WWJM01000049 (10232-95344): Streptomyces sp. SID4920 SID4920.c49, whole ge... (5% of genes show similarity), NRPS,T3PKS,betalactone
      </option>
      
      <option value = "svg/clusterblast_r9c1_9.svg">
        NZ_PVZV01000006 (121917-228208): Actinocorallia populi strain A251 contig6, w... (3% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r9c1_10.svg">
        NZ_RDBL01000010 (27602-113085): Streptomyces sp. col6 Scaffold 10, whole geno... (4% of genes show similarity), NRPS,T3PKS,betalactone
      </option>
      
    </select>
    <button id= "clusterblast-r9c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r9c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r9c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-12"></div> <span class="help-tooltip" id="cb-known-help-12">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r9c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r9c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r9c1_1.svg">
        coelichelin
      </option>
      
    </select>
    <button id="knownclusterblast-r9c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r9c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r9c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-13"></div> <span class="help-tooltip" id="cb-sub-help-13">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r9c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-14"></div> <span class="help-tooltip" id="MIBiG-help-14">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r9c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r9c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r9c1"></div>
    <div id="comparison-MIBiG-r9c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873.1: 1647607-1656213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>pyrrolizixenamide A</td>
        
        
        <td>Xenorhabdus szentirmaii DSM 16338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>fellutamide B</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000325.1: 510822-533448">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000325/index.html#r1c1" target="_new">BGC0000325.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.65, order:0.69, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>coelichelin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>AM-toxin</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>vicibactin</td>
        
        
        <td>Rhizobium etli CFN 42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r9c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873.1: 1647607-1656213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>pyrrolizixenamide A</td>
      <td>Xenorhabdus szentirmaii DSM 16338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>fellutamide B</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000325.1: 510822-533448">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000325/index.html#r1c1" target="_new">BGC0000325.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.65, order:0.69, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>coelichelin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>AM-toxin</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>vicibactin</td>
      <td>Rhizobium etli CFN 42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r9c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-15"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-15">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r9c1-pfam-details-svg" data-filename="c00009_NODE_26.._r9c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r9c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r9c1-nrps_pks_products" data-name="r9c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r9c1-nrps_pks" data-name="r9c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r9c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-9"></div> <span class="help-tooltip" id="nrps-structures-help-9">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 5998:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r9c1-1" class="smiles-canvas" data-smiles="NC(CCCN)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(orn)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r9c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CCCN)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r9c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-10"></div> <span class="help-tooltip" id="nrps-monomers-help-10">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds1905_5997</span></strong>: orn
      <div class="collapser collapser-target-cds1905_5997 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (784..1182)</strong>: orn
          <div class="collapser collapser-target-cds1905_5997 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn
              <div class="collapser collapser-target-cds1905_5997-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DMwNLGLINK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r10c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00010_NODE_41.. - Region 1 - siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-257"></div> <span class="help-tooltip" id="region_svg-help-257">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00010_NODE_41...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r10c1-svg" data-filename="c00010_NODE_41.._r10c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 21,712 - 34,885 nt. (total: 13,174 nt)
        
        <a class="cluster-rules-header" id="r10c1-rules-header" href="#r10c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r10c1-rules">siderophore: IucA_IucC</div>
      
      <div class="region-svg-container">
       <div id='r10c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-258"></div> <span class="help-tooltip" id="focus-panel-help-258">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r10c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r10c1-clusterblast" data-name="r10c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r10c1-knownclusterblast" data-name="r10c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r10c1-subclusterblast" data-name="r10c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r10c1-MIBiG-cluster-compare" data-name="r10c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r10c1-pfam-details" data-name="r10c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r10c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-16"></div> <span class="help-tooltip" id="cb-general-help-16">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r10c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r10c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r10c1_1.svg">
        NZ_CP039123 (6262702-6275876): Streptomyces sp. SS52 chromosome, complete genome (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c1_2.svg">
        NZ_CP027297 (6415484-6428658): Streptomyces sp. SGAir0924 chromosome, complet... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c1_3.svg">
        NZ_KZ819185 (584229-597141): Streptomyces sp. CS207 scaffold00002, whole geno... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c1_4.svg">
        NZ_CP042278 (1679091-1691990): Streptomyces sp. WAC06273 strain WAC6273 subst... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c1_5.svg">
        NZ_CP028834 (1750947-1763790): Streptomyces sp. M2 chromosome, complete genome (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c1_6.svg">
        NZ_PZZX01000001 (376626-389467): Streptomyces sp. VMFN-G11Ma Ga0213531 101, w... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c1_7.svg">
        NZ_SZNQ01000001 (6157782-6171012): Streptomyces lasalocidi strain X-537 Scaff... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c1_8.svg">
        NZ_KQ948550 (324475-337639): Streptomyces longwoodensis strain DSM 41677 PRJN... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c1_9.svg">
        NZ_SLZS01000002 (278621-291704): Streptomyces sp. BK335 Ga0307713 102, whole ... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r10c1_10.svg">
        NZ_SLZH01000012 (34230-47272): Streptomyces sp. BK042 Ga0307683 112, whole ge... (92% of genes show similarity), siderophore
      </option>
      
    </select>
    <button id= "clusterblast-r10c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r10c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r10c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-17"></div> <span class="help-tooltip" id="cb-known-help-17">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r10c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r10c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r10c1_1.svg">
        paulomycin
      </option>
      
      <option value = "svg/knownclusterblast_r10c1_2.svg">
        paulomycin
      </option>
      
      <option value = "svg/knownclusterblast_r10c1_3.svg">
        grincamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r10c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r10c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r10c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-18"></div> <span class="help-tooltip" id="cb-sub-help-18">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r10c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-19"></div> <span class="help-tooltip" id="MIBiG-help-19">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r10c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r10c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r10c1"></div>
    <div id="comparison-MIBiG-r10c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">siderophore</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000946.1: 282-7531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000946/index.html#r1c1" target="_new">BGC0000946.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Other</td>
        
        <td>vibrioferrin</td>
        
        
        <td>Vibrio parahaemolyticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000939.1: 1834-15938">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000939/index.html#r1c1" target="_new">BGC0000939.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Other</td>
        
        <td>aerobactin</td>
        
        
        <td>Grimontia hollisae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000947.1: 153-15744">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000947/index.html#r1c1" target="_new">BGC0000947.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Other</td>
        
        <td>vibrioferrin</td>
        
        
        <td>Vibrio alginolyticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.48, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide, Saccharide</td>
        
        <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514.1: 0-37469">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>aristeromycin</td>
        
        
        <td>Streptomyces citricolor</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001374.1: 52-17210">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001374/index.html#r1c1" target="_new">BGC0001374.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Other</td>
        
        <td>thienodolin</td>
        
        
        <td>Streptomyces albogriseolus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001314.1: 14191710-14246296">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001314/index.html#r1c1" target="_new">BGC0001314.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>Terpene</td>
        
        <td>tirucalla</td>
        
        
        <td>Arabidopsis thaliana</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000277.1: 237-11479">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000277/index.html#r1c1" target="_new">BGC0000277.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.48, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
        <td>0.12</td>
        <td>Polyketide</td>
        
        <td>urdamycin</td>
        
        
        <td>Streptomyces fradiae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001501.1: 0-42974">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001501/index.html#r1c1" target="_new">BGC0001501.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.35 (id:0.43, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
         
        
        <td>0.10</td>
        <td>Alkaloid</td>
        
        <td>ambiguine P</td>
        
        
        <td>Fischerella sp. TAU</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r10c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000946.1: 282-7531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000946/index.html#r1c1" target="_new">BGC0000946.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Other</td>
      <td>vibrioferrin</td>
      <td>Vibrio parahaemolyticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000939.1: 1834-15938">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000939/index.html#r1c1" target="_new">BGC0000939.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Other</td>
      <td>aerobactin</td>
      <td>Grimontia hollisae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000947.1: 153-15744">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000947/index.html#r1c1" target="_new">BGC0000947.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Other</td>
      <td>vibrioferrin</td>
      <td>Vibrio alginolyticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Other</td>
      <td>flaviolin</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.48, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Polyketide, Saccharide</td>
      <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
      <td>Saccharopolyspora erythraea</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514.1: 0-37469">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other</td>
      <td>aristeromycin</td>
      <td>Streptomyces citricolor</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001374.1: 52-17210">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001374/index.html#r1c1" target="_new">BGC0001374.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Other</td>
      <td>thienodolin</td>
      <td>Streptomyces albogriseolus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001314.1: 14191710-14246296">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001314/index.html#r1c1" target="_new">BGC0001314.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>Terpene</td>
      <td>tirucalla</td>
      <td>Arabidopsis thaliana</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000277.1: 237-11479">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000277/index.html#r1c1" target="_new">BGC0000277.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.40 (id:0.48, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
      <td>0.40</td>
      <td>Polyketide</td>
      <td>urdamycin</td>
      <td>Streptomyces fradiae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001501.1: 0-42974">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001501/index.html#r1c1" target="_new">BGC0001501.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.35 (id:0.43, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
      <td>0.35</td>
      <td>Alkaloid</td>
      <td>ambiguine P</td>
      <td>Fischerella sp. TAU</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r10c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-20"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-20">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r10c1-pfam-details-svg" data-filename="c00010_NODE_41.._r10c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r10c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r13c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00013_NODE_19.. - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-259"></div> <span class="help-tooltip" id="region_svg-help-259">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00013_NODE_19...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r13c1-svg" data-filename="c00013_NODE_19.._r13c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 8,157 nt. (total: 8,157 nt)
        
        <a class="cluster-rules-header" id="r13c1-rules-header" href="#r13c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r13c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r13c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-260"></div> <span class="help-tooltip" id="focus-panel-help-260">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r13c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r13c1-nrps_pks" data-name="r13c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r13c1-clusterblast" data-name="r13c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r13c1-knownclusterblast" data-name="r13c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r13c1-subclusterblast" data-name="r13c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r13c1-MIBiG-cluster-compare" data-name="r13c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r13c1-pfam-details" data-name="r13c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r13c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-21"></div> <span class="help-tooltip" id="nrps-domain-help-21">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r13c1-details-svg" data-filename="c00013_NODE_19.._r13c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r13c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r13c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-24"></div> <span class="help-tooltip" id="cb-general-help-24">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r13c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r13c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r13c1_1.svg">
        NZ_CP027297 (4104848-4147846): Streptomyces sp. SGAir0924 chromosome, complet... (20% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r13c1_2.svg">
        NZ_KZ819185 (1873239-1917109): Streptomyces sp. CS207 scaffold00002, whole ge... (21% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r13c1_3.svg">
        NZ_CP039123 (3998239-4041317): Streptomyces sp. SS52 chromosome, complete genome (20% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r13c1_4.svg">
        NZ_KQ948206 (382822-426674): Streptomyces yokosukanensis strain DSM 40224 PRJ... (13% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r13c1_5.svg">
        NZ_CP049838 (8807862-8850765): Streptomyces cacaoi subsp. asoensis strain H2S... (10% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r13c1_6.svg">
        NZ_CM002285 (1531131-1574167): Streptomyces roseochromogenus subsp. oscitans ... (6% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r13c1_7.svg">
        NZ_MQUS01000007 (280463-324303): Streptomyces sp. IMTB 2501 IMTB 2501 contig ... (5% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r13c1_8.svg">
        NZ_CP021121 (3237564-3281393): Streptomyces sp. SCSIO 3032 strain SCSIO 03032... (8% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r13c1_9.svg">
        NZ_CM000914 (1673227-1717301): Streptomyces clavuligerus strain ATCC 27064 pl... (7% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r13c1_10.svg">
        NZ_CP020563 (830981-874624): Kitasatospora albolonga strain YIM 101047 chromo... (8% of genes show similarity), NRPS-like
      </option>
      
    </select>
    <button id= "clusterblast-r13c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r13c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r13c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-25"></div> <span class="help-tooltip" id="cb-known-help-25">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r13c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r13c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r13c1_1.svg">
        alanylclavam / 2-hydroxymethylclavam / 2-formyloxymethylclavam / clavam-2-carboxylate
      </option>
      
      <option value = "svg/knownclusterblast_r13c1_2.svg">
        granaticin
      </option>
      
    </select>
    <button id="knownclusterblast-r13c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r13c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r13c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-26"></div> <span class="help-tooltip" id="cb-sub-help-26">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r13c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-27"></div> <span class="help-tooltip" id="MIBiG-help-27">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r13c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r13c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r13c1"></div>
    <div id="comparison-MIBiG-r13c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.51, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>ferrichrome</td>
        
        
        <td>Aspergillus oryzae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>dimethylcoprogen</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>AM-toxin</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517.1: 168853-184269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>asperphenamate</td>
        
        
        <td>Aspergillus terreus NIH2624</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 10E9</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 22D11</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r13c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.51, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>ferrichrome</td>
      <td>Aspergillus oryzae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>dimethylcoprogen</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.46, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261.1: 987-14070">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>AM-toxin</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517.1: 168853-184269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>asperphenamate</td>
      <td>Aspergillus terreus NIH2624</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 10E9</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 22D11</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r13c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-28"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-28">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r13c1-pfam-details-svg" data-filename="c00013_NODE_19.._r13c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r13c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r13c1-nrps_pks_products" data-name="r13c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r13c1-nrps_pks" data-name="r13c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r13c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-22"></div> <span class="help-tooltip" id="nrps-structures-help-22">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 8157:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r13c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r13c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r13c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-23"></div> <span class="help-tooltip" id="nrps-monomers-help-23">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds2110_6001</span></strong>: X
      <div class="collapser collapser-target-cds2110_6001 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (26..437)</strong>: X
          <div class="collapser collapser-target-cds2110_6001 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds2110_6001-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAWQyGLInK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r17c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00017_NODE_73.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-261"></div> <span class="help-tooltip" id="region_svg-help-261">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00017_NODE_73...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r17c1-svg" data-filename="c00017_NODE_73.._r17c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 11,202 nt. (total: 11,202 nt)
        
        <a class="cluster-rules-header" id="r17c1-rules-header" href="#r17c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r17c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r17c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-262"></div> <span class="help-tooltip" id="focus-panel-help-262">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r17c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r17c1-clusterblast" data-name="r17c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r17c1-knownclusterblast" data-name="r17c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r17c1-subclusterblast" data-name="r17c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r17c1-MIBiG-cluster-compare" data-name="r17c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r17c1-pfam-details" data-name="r17c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r17c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-29"></div> <span class="help-tooltip" id="cb-general-help-29">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r17c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r17c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r17c1_1.svg">
        NZ_CP012949 (5128146-5147586): Streptomyces ambofaciens strain DSM 40697 chro... (61% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r17c1_2.svg">
        NZ_CP054920 (2829826-2849885): Streptomyces sp. NA03103 chromosome, complete ... (61% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r17c1_3.svg">
        NZ_KZ195574 (2987642-3008355): Streptomyces sp. CS113 scaffold00001, whole ge... (57% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r17c1_4.svg">
        NZ_MQUH01000016 (913268-934354): Streptomyces sp. M1013 M1013 C16, whole geno... (57% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r17c1_5.svg">
        NZ_LT963352 (3038615-3059139): Streptomyces chartreusis NRRL 3882 isolate NRR... (57% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r17c1_6.svg">
        NZ_CP026121 (5743113-5763801): Streptomyces sp. Go-475 chromosome, complete g... (57% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r17c1_7.svg">
        NZ_JACHXE010000002 (199442-220528): Streptomyces violarus strain CECT 3237 Ga... (73% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r17c1_8.svg">
        NZ_KZ671984 (123308-144394): Streptomyces sp. 46 scaffold21, whole genome sho... (61% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r17c1_9.svg">
        NZ_KL585964 (41932-63018): Streptomyces sp. NRRL S-474 P279 Doro1 scaffold19,... (68% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r17c1_10.svg">
        NZ_JACHCX010000014 (4003-25089): Streptomyces sp. AK010 Ga0401617 14, whole g... (57% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r17c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r17c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r17c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-30"></div> <span class="help-tooltip" id="cb-known-help-30">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r17c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r17c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r17c1_1.svg">
        albaflavenone
      </option>
      
    </select>
    <button id="knownclusterblast-r17c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r17c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r17c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-31"></div> <span class="help-tooltip" id="cb-sub-help-31">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r17c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-32"></div> <span class="help-tooltip" id="MIBiG-help-32">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r17c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r17c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r17c1"></div>
    <div id="comparison-MIBiG-r17c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.90 (id:0.72, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.90), rgb(0, 0, 0, 0.90) 90%, white 90%)"></td>
         
        
        <td>0.72</td>
        <td>Terpene</td>
        
        <td>albaflavenone</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001201.1: 11915724-11918015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001201/index.html#r1c1" target="_new">BGC0001201.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Terpene</td>
        
        <td>eremophilene</td>
        
        
        <td>Sorangium cellulosum So ce56</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>Terpene</td>
        
        <td>raimonol</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000674.1: 74961-79952">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000674/index.html#r1c1" target="_new">BGC0000674.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Terpene</td>
        
        <td>(-)-δ-cadinene</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Terpene</td>
        
        <td>(+)-T-muurolol</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>cyslabdan</td>
        
        
        <td>Streptomyces cyslabdanicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002013.1: 235087-246792">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002013/index.html#r1c1" target="_new">BGC0002013.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>RiPP</td>
        
        <td>curacozole</td>
        
        
        <td>Streptomyces curacoi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650.1: 28-7720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.17</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Algoriphagus sp. KK10202C</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:0.39, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.17</td>
        <td>Terpene</td>
        
        <td>pentalenolactone</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001540.1: 633-38461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001540/index.html#r1c1" target="_new">BGC0001540.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>CC-1065</td>
        
        
        <td>Streptomyces zelensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r17c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.90 (id:0.72, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.90), rgb(0, 0, 0, 0.90) 90%, white 90%)"></td>
      <td>0.90</td>
      <td>Terpene</td>
      <td>albaflavenone</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001201.1: 11915724-11918015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001201/index.html#r1c1" target="_new">BGC0001201.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>eremophilene</td>
      <td>Sorangium cellulosum So ce56</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Terpene</td>
      <td>raimonol</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000674.1: 74961-79952">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000674/index.html#r1c1" target="_new">BGC0000674.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene</td>
      <td>(-)-δ-cadinene</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene</td>
      <td>(+)-T-muurolol</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>cyslabdan</td>
      <td>Streptomyces cyslabdanicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002013.1: 235087-246792">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002013/index.html#r1c1" target="_new">BGC0002013.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>RiPP</td>
      <td>curacozole</td>
      <td>Streptomyces curacoi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650.1: 28-7720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Algoriphagus sp. KK10202C</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:0.39, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001540.1: 633-38461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001540/index.html#r1c1" target="_new">BGC0001540.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>CC-1065</td>
      <td>Streptomyces zelensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r17c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-33"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-33">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r17c1-pfam-details-svg" data-filename="c00017_NODE_73.._r17c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r17c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r22c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00022_NODE_75.. - Region 1 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-263"></div> <span class="help-tooltip" id="region_svg-help-263">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00022_NODE_75...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r22c1-svg" data-filename="c00022_NODE_75.._r22c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 25,134 nt. (total: 25,134 nt)
        
        <a class="cluster-rules-header" id="r22c1-rules-header" href="#r22c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r22c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r22c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-264"></div> <span class="help-tooltip" id="focus-panel-help-264">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r22c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r22c1-nrps_pks" data-name="r22c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r22c1-clusterblast" data-name="r22c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r22c1-knownclusterblast" data-name="r22c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r22c1-subclusterblast" data-name="r22c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r22c1-MIBiG-cluster-compare" data-name="r22c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r22c1-pfam-details" data-name="r22c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r22c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-34"></div> <span class="help-tooltip" id="nrps-domain-help-34">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r22c1-details-svg" data-filename="c00022_NODE_75.._r22c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r22c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r22c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-37"></div> <span class="help-tooltip" id="cb-general-help-37">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r22c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r22c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r22c1_1.svg">
        NZ_CP027297 (282629-492289): Streptomyces sp. SGAir0924 chromosome, complete ... (27% of genes show similarity), NRPS,NRPS-like,T1PKS,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r22c1_2.svg">
        NZ_CP042278 (117268-184951): Streptomyces sp. WAC06273 strain WAC6273 substr.... (27% of genes show similarity), NRPS,NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r22c1_3.svg">
        NZ_CP012949 (362398-421882): Streptomyces ambofaciens strain DSM 40697 chromo... (18% of genes show similarity), NRPS,T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r22c1_4.svg">
        NZ_CP012382 (352106-411657): Streptomyces ambofaciens ATCC 23877 chromosome, ... (16% of genes show similarity), NRPS,T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r22c1_5.svg">
        NZ_VIUV01000001 (16420-75093): Streptomyces sp. T12 Ga0311319 101, whole geno... (17% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r22c1_6.svg">
        NZ_CP023407 (334147-391467): Streptomyces fungicidicus strain TXX3120 chromos... (17% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r22c1_7.svg">
        NZ_CP054939 (7193004-7251184): Streptomyces sp. NA02536 chromosome, complete ... (19% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r22c1_8.svg">
        NZ_BBNN01000019 (109185-166589): Streptomyces sp. NBRC 110035 strain TP-A0873... (17% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r22c1_9.svg">
        NZ_SOAV01000004 (327162-385341): Streptomyces sp. BK208 Ga0307686 104, whole ... (18% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r22c1_10.svg">
        NZ_CP043959 (4073746-4132442): Streptomyces tendae strain 139 chromosome, com... (17% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r22c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r22c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r22c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-38"></div> <span class="help-tooltip" id="cb-known-help-38">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r22c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r22c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_1.svg">
        antimycin
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_2.svg">
        splenocin
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_3.svg">
        antimycin
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_4.svg">
        neoantimycin
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_5.svg">
        neoantimycin
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_6.svg">
        pyridomycin
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_7.svg">
        JBIR-06
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_8.svg">
        candicidin
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_9.svg">
        levorin A3 / C06690 / FR-008-III / candicidin A / UNII-AP5PEF5W7U
      </option>
      
      <option value = "svg/knownclusterblast_r22c1_10.svg">
        nystatin A1
      </option>
      
    </select>
    <button id="knownclusterblast-r22c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r22c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r22c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-39"></div> <span class="help-tooltip" id="cb-sub-help-39">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r22c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-40"></div> <span class="help-tooltip" id="MIBiG-help-40">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r22c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r22c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r22c1"></div>
    <div id="comparison-MIBiG-r22c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.66</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001455.1: 0-26743">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001455/index.html#r1c1" target="_new">BGC0001455.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.64, order:0.50, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.64, order:0.50, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.58</td>
        <td>NRP, Polyketide</td>
        
        <td>antimycin</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001216.1: 6355-31635">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001216/index.html#r1c1" target="_new">BGC0001216.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.58, order:0.50, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.58, order:0.50, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.58</td>
        <td>NRP, Polyketide</td>
        
        <td>splenocin</td>
        
        
        <td>Streptomyces sp. CNQ431</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002058.1: 29690-42292">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002058/index.html#r1c1" target="_new">BGC0002058.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.57</td>
        <td>Other</td>
        
        <td>Reutericyclin A, Reutericyclin B, Reutericyclin C, Reutericyclin D</td>
        
        
        <td>Streptococcus mutans B04Sm5</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000958.1: 81952-106578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000958/index.html#r1c1" target="_new">BGC0000958.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.58, order:0.50, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.58, order:0.50, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.57</td>
        <td>NRP, Polyketide</td>
        
        <td>antimycin</td>
        
        
        <td>Streptomyces sp. S4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.55</td>
        <td>NRP</td>
        
        <td>vicibactin</td>
        
        
        <td>Rhizobium etli CFN 42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873.1: 1647607-1656213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.51</td>
        <td>NRP</td>
        
        <td>pyrrolizixenamide A</td>
        
        
        <td>Xenorhabdus szentirmaii DSM 16338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.51</td>
        <td>NRP</td>
        
        <td>bacillibactin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313.1: 0-33475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.51</td>
        <td>NRP</td>
        
        <td>beauvericin</td>
        
        
        <td>Beauveria bassiana</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342.1: 1020-10416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.50</td>
        <td>NRP</td>
        
        <td>enniatin</td>
        
        
        <td>Fusarium equiseti</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r22c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001455.1: 0-26743">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001455/index.html#r1c1" target="_new">BGC0001455.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.64, order:0.50, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP, Polyketide</td>
      <td>antimycin</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001216.1: 6355-31635">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001216/index.html#r1c1" target="_new">BGC0001216.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.58, order:0.50, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP, Polyketide</td>
      <td>splenocin</td>
      <td>Streptomyces sp. CNQ431</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002058.1: 29690-42292">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002058/index.html#r1c1" target="_new">BGC0002058.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>Reutericyclin A, Reutericyclin B, Reutericyclin C, Reutericyclin D</td>
      <td>Streptococcus mutans B04Sm5</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000958.1: 81952-106578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000958/index.html#r1c1" target="_new">BGC0000958.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.58, order:0.50, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP, Polyketide</td>
      <td>antimycin</td>
      <td>Streptomyces sp. S4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>vicibactin</td>
      <td>Rhizobium etli CFN 42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873.1: 1647607-1656213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>pyrrolizixenamide A</td>
      <td>Xenorhabdus szentirmaii DSM 16338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313.1: 0-33475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>beauvericin</td>
      <td>Beauveria bassiana</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342.1: 1020-10416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>enniatin</td>
      <td>Fusarium equiseti</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r22c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-41"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-41">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r22c1-pfam-details-svg" data-filename="c00022_NODE_75.._r22c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r22c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r22c1-nrps_pks_products" data-name="r22c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r22c1-nrps_pks" data-name="r22c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r22c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-35"></div> <span class="help-tooltip" id="nrps-structures-help-35">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 25134:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r22c1-1" class="smiles-canvas" data-smiles="NC(C(O)C)C(=O)CC(=O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(thr) + (mal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr&amp;nrps2=mal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*&amp;nrps2=mal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r22c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(C(O)C)C(=O)CC(=O)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r22c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-36"></div> <span class="help-tooltip" id="nrps-monomers-help-36">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds2203_6199</span></strong>: mal
      <div class="collapser collapser-target-cds2203_6199 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (594..877)</strong>: mal
          <div class="collapser collapser-target-cds2203_6199 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds2203_6199-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 62.5%</dd>
<dd>2-Rhyd-Malonyl-CoA: 54.2%</dd>
<dd>inactive: 54.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds2203_6199-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 100.6</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 61.3</dt>

  <dd></dd><dt>inactive: 50.0</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 41.3</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 25.9</dt>

  <dd></dd><dt>Isobutyryl-CoA: 19.1</dt>

  <dd></dd><dt>Propionyl-CoA: 18.3</dt>

  <dd></dd><dt>fatty_acid: 8.9</dt>

  <dd></dd><dt>Benzoyl-CoA: 2.7</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds6191_14825</span></strong>: thr - pk
      <div class="collapser collapser-target-cds6191_14825 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (461..874)</strong>: thr
          <div class="collapser collapser-target-cds6191_14825 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds6191_14825-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWSVGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>CAL_domain (1707..2116)</strong>: pk
          <div class="collapser collapser-target-cds6191_14825 collapser-level-candidate"> </div><div class="collapser-content">
          
              minowa:  fatty_acid
              <div class="collapser collapser-target-cds6191_14825-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>fatty_acid: 15.2</dt>

  <dd></dd><dt>AHBA: 0.6</dt>

  <dd></dd><dt>shikimic_acid: 0.0</dt>

  <dd></dd><dt>NH2: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: -2.8</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r35c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00035_NODE_72.. - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-265"></div> <span class="help-tooltip" id="region_svg-help-265">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00035_NODE_72...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r35c1-svg" data-filename="c00035_NODE_72.._r35c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 9,988 nt. (total: 9,988 nt)
        
        <a class="cluster-rules-header" id="r35c1-rules-header" href="#r35c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r35c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r35c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-266"></div> <span class="help-tooltip" id="focus-panel-help-266">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r35c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r35c1-nrps_pks" data-name="r35c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r35c1-clusterblast" data-name="r35c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r35c1-knownclusterblast" data-name="r35c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r35c1-subclusterblast" data-name="r35c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r35c1-MIBiG-cluster-compare" data-name="r35c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r35c1-pfam-details" data-name="r35c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r35c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-42"></div> <span class="help-tooltip" id="nrps-domain-help-42">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r35c1-details-svg" data-filename="c00035_NODE_72.._r35c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r35c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r35c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-43"></div> <span class="help-tooltip" id="cb-general-help-43">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r35c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r35c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r35c1_1.svg">
        NZ_CP023445 (3737265-3831134): Actinosynnema pretiosum strain X47 chromosome,... (9% of genes show similarity), NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r35c1_2.svg">
        NZ_LT963352 (7799601-7891829): Streptomyces chartreusis NRRL 3882 isolate NRR... (11% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r35c1_3.svg">
        NZ_LT607413 (6800632-&gt;6912269): Micromonospora echinospora strain DSM 43816 c... (8% of genes show similarity), NRPS,T1PKS,T3PKS
      </option>
      
      <option value = "svg/clusterblast_r35c1_4.svg">
        NC_014623 (6407323-6493561): Stigmatella aurantiaca DW4/3-1, complete sequence (16% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r35c1_5.svg">
        NC_010628 (4167586-4349893): Nostoc punctiforme PCC 73102, complete sequence (4% of genes show similarity), NRPS,T1PKS,bacteriocin,lanthidin,transAT-PKS-like
      </option>
      
      <option value = "svg/clusterblast_r35c1_6.svg">
        NZ_AQRJ01000070 (440148-528806): Streptomyces sp. AmelKG-E11A YW7DRAFT scaffo... (5% of genes show similarity), NRPS,T1PKS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r35c1_7.svg">
        NZ_LFBV01000003 (428239-508216): Streptomyces uncialis strain DCA2648 scaffol... (5% of genes show similarity), NRPS,T1PKS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r35c1_8.svg">
        NC_019678 (7757037-7838092): Rivularia sp. PCC 7116, complete genome (8% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r35c1_9.svg">
        NZ_KZ819154 (1353326-1393816): Streptomyces sp. CS014 scaffold00001, whole ge... (17% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r35c1_10.svg">
        NZ_GG657754 (8595238-8685706): Streptomyces himastatinicus ATCC 53653 superco... (5% of genes show similarity), NRPS,T1PKS,transAT-PKS
      </option>
      
    </select>
    <button id= "clusterblast-r35c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r35c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r35c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-44"></div> <span class="help-tooltip" id="cb-known-help-44">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r35c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r35c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r35c1_1.svg">
        candicidin
      </option>
      
      <option value = "svg/knownclusterblast_r35c1_2.svg">
        levorin A3 / C06690 / FR-008-III / candicidin A / UNII-AP5PEF5W7U
      </option>
      
      <option value = "svg/knownclusterblast_r35c1_3.svg">
        nanchangmycin
      </option>
      
      <option value = "svg/knownclusterblast_r35c1_4.svg">
        incednine
      </option>
      
      <option value = "svg/knownclusterblast_r35c1_5.svg">
        fostriecin
      </option>
      
      <option value = "svg/knownclusterblast_r35c1_6.svg">
        rifamorpholine A / rifamorpholine B / rifamorpholine C / rifamorpholine D / rifamorpholine E
      </option>
      
      <option value = "svg/knownclusterblast_r35c1_7.svg">
        macrotermycins
      </option>
      
      <option value = "svg/knownclusterblast_r35c1_8.svg">
        meridamycin
      </option>
      
      <option value = "svg/knownclusterblast_r35c1_9.svg">
        meridamycin
      </option>
      
      <option value = "svg/knownclusterblast_r35c1_10.svg">
        chlorothricin / deschlorothricin
      </option>
      
    </select>
    <button id="knownclusterblast-r35c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r35c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r35c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-45"></div> <span class="help-tooltip" id="cb-sub-help-45">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r35c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-46"></div> <span class="help-tooltip" id="MIBiG-help-46">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r35c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r35c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r35c1"></div>
    <div id="comparison-MIBiG-r35c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.50, order:1.00, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Polyketide</td>
        
        <td>oronofacic acid</td>
        
        
        <td>Pseudomonas syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000067.1: 14949-58348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000067/index.html#r1c1" target="_new">BGC0000067.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.50, order:1.00, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>geldanamycin</td>
        
        
        <td>Streptomyces hygroscopicus subsp. duamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>esperamicin</td>
        
        
        <td>Actinomadura verrucosospora</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002057.1: 10296-37416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002057/index.html#r1c1" target="_new">BGC0002057.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.44, order:1.00, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>pateamine</td>
        
        
        <td>Kiritimatiellaeota bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000143.1: 2-25809">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000143/index.html#r1c1" target="_new">BGC0000143.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.52, order:1.00, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>salinomycin</td>
        
        
        <td>Streptomyces albus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000157.1: 1384-70003">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000157/index.html#r1c1" target="_new">BGC0000157.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.50, order:1.00, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>tautomycetin</td>
        
        
        <td>Streptomyces sp. CK4412</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002026.1: 219-57197">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002026/index.html#r1c1" target="_new">BGC0002026.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.49, order:1.00, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>NRP</td>
        
        <td>thaxteramide A1, thaxteramide A2, thaxteramide B1, thaxteramide B2</td>
        
        
        <td>Jahnella sp. MSr9139</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002030.1: 382550-429344">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002030/index.html#r1c1" target="_new">BGC0002030.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.44, order:1.00, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>viriditoxin</td>
        
        
        <td>Byssochlamys spectabilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001866.1: 394920-427553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001866/index.html#r1c1" target="_new">BGC0001866.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.44, order:1.00, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>viriditoxin</td>
        
        
        <td>Byssochlamys spectabilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>6-methylsalicyclic acid</td>
        
        
        <td>Aspergillus terreus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r35c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.50, order:1.00, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>oronofacic acid</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000067.1: 14949-58348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000067/index.html#r1c1" target="_new">BGC0000067.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.50, order:1.00, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>geldanamycin</td>
      <td>Streptomyces hygroscopicus subsp. duamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>esperamicin</td>
      <td>Actinomadura verrucosospora</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002057.1: 10296-37416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002057/index.html#r1c1" target="_new">BGC0002057.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.44, order:1.00, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>pateamine</td>
      <td>Kiritimatiellaeota bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000143.1: 2-25809">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000143/index.html#r1c1" target="_new">BGC0000143.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.52, order:1.00, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>salinomycin</td>
      <td>Streptomyces albus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000157.1: 1384-70003">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000157/index.html#r1c1" target="_new">BGC0000157.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.50, order:1.00, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Polyketide</td>
      <td>tautomycetin</td>
      <td>Streptomyces sp. CK4412</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002026.1: 219-57197">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002026/index.html#r1c1" target="_new">BGC0002026.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.49, order:1.00, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP</td>
      <td>thaxteramide A1, thaxteramide A2, thaxteramide B1, thaxteramide B2</td>
      <td>Jahnella sp. MSr9139</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002030.1: 382550-429344">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002030/index.html#r1c1" target="_new">BGC0002030.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.44, order:1.00, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>viriditoxin</td>
      <td>Byssochlamys spectabilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001866.1: 394920-427553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001866/index.html#r1c1" target="_new">BGC0001866.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.44, order:1.00, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>viriditoxin</td>
      <td>Byssochlamys spectabilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Aspergillus terreus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r35c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-47"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-47">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r35c1-pfam-details-svg" data-filename="c00035_NODE_72.._r35c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r35c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r37c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00037_NODE_29.. - Region 1 - T2PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-267"></div> <span class="help-tooltip" id="region_svg-help-267">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00037_NODE_29...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r37c1-svg" data-filename="c00037_NODE_29.._r37c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 10,392 nt. (total: 10,392 nt)
        
        <a class="cluster-rules-header" id="r37c1-rules-header" href="#r37c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r37c1-rules">T2PKS: (t2ks and t2clf)</div>
      
      <div class="region-svg-container">
       <div id='r37c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-268"></div> <span class="help-tooltip" id="focus-panel-help-268">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r37c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r37c1-nrps_pks" data-name="r37c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r37c1-clusterblast" data-name="r37c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r37c1-knownclusterblast" data-name="r37c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r37c1-subclusterblast" data-name="r37c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r37c1-MIBiG-cluster-compare" data-name="r37c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r37c1-pfam-details" data-name="r37c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r37c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-48"></div> <span class="help-tooltip" id="nrps-domain-help-48">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r37c1-details-svg" data-filename="c00037_NODE_29.._r37c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r37c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r37c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-49"></div> <span class="help-tooltip" id="cb-general-help-49">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r37c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r37c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r37c1_1.svg">
        NZ_CP027297 (19263-103747): Streptomyces sp. SGAir0924 chromosome, complete g... (16% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r37c1_2.svg">
        NZ_KZ819184 (4102654-4188512): Streptomyces sp. CS207 scaffold00001, whole ge... (16% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r37c1_3.svg">
        NZ_CP012382 (127761-213859): Streptomyces ambofaciens ATCC 23877 chromosome, ... (15% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r37c1_4.svg">
        NZ_CP012382 (8089906-8176435): Streptomyces ambofaciens ATCC 23877 chromosome... (14% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r37c1_5.svg">
        NZ_CP012949 (140374-225289): Streptomyces ambofaciens strain DSM 40697 chromo... (14% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r37c1_6.svg">
        NZ_CP012949 (7911727-7998131): Streptomyces ambofaciens strain DSM 40697 chro... (13% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r37c1_7.svg">
        NZ_CP029241 (7374976-7447633): Streptomyces sp. NHF165 chromosome, complete g... (13% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r37c1_8.svg">
        NZ_CP047140 (110406-298596): Streptomyces sp. Tu 2975 chromosome, complete ge... (8% of genes show similarity), NRPS,PKS-like,T2PKS,ectoine,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r37c1_9.svg">
        NZ_CP041610 (11556371-11612694): Streptomyces sp. RLB3-17 chromosome, complet... (17% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r37c1_10.svg">
        NZ_CP041604 (343452-395703): Streptomyces sp. S1A1-7 chromosome (18% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r37c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r37c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r37c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-50"></div> <span class="help-tooltip" id="cb-known-help-50">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r37c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r37c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r37c1_1.svg">
        griseorhodin A
      </option>
      
      <option value = "svg/knownclusterblast_r37c1_2.svg">
        mithramycin
      </option>
      
      <option value = "svg/knownclusterblast_r37c1_3.svg">
        fluostatins M-Q
      </option>
      
      <option value = "svg/knownclusterblast_r37c1_4.svg">
        kinamycin
      </option>
      
      <option value = "svg/knownclusterblast_r37c1_5.svg">
        fluostatin
      </option>
      
      <option value = "svg/knownclusterblast_r37c1_6.svg">
        rabelomycin / dehydrorabelomycin / fluostatin F / fluostatin G / fluostatin H
      </option>
      
      <option value = "svg/knownclusterblast_r37c1_7.svg">
        erdacin
      </option>
      
      <option value = "svg/knownclusterblast_r37c1_8.svg">
        fogacin A / fogacin B / fogacin C
      </option>
      
      <option value = "svg/knownclusterblast_r37c1_9.svg">
        mithramycin
      </option>
      
      <option value = "svg/knownclusterblast_r37c1_10.svg">
        aranciamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r37c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r37c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r37c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-51"></div> <span class="help-tooltip" id="cb-sub-help-51">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r37c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r37c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r37c1_1.svg">
        AF324838_c1: simocyclinone t2pks (33% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r37c1_2.svg">
        AB103463_c1: medermycin t2pks (50% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r37c1_3.svg">
        FJ483966_c1: polyketomycin t2pks (25% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r37c1_4.svg">
        AF080235_c1: landomycin t2pks (33% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r37c1_5.svg">
        GU233672_c1: granaticin t2pks (28% of genes show similarity), t2pks
      </option>
      
    </select>
    <button id="subclusterblast-r37c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r37c1-svg"></div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r37c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-53"></div> <span class="help-tooltip" id="MIBiG-help-53">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r37c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r37c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r37c1"></div>
    <div id="comparison-MIBiG-r37c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T2PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.52, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>naphtocyclinone</td>
        
        
        <td>Streptomyces arenae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.50, order:0.45, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Polyketide</td>
        
        <td>spore pigment</td>
        
        
        <td>Streptomyces collinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.51, order:0.48, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>granaticin</td>
        
        
        <td>Streptomyces violaceoruber</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.50, order:0.45, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>curamycin</td>
        
        
        <td>Streptomyces cyaneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.50, order:0.45, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>griseusin</td>
        
        
        <td>Streptomyces griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.51, order:0.41, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>frankiamicin</td>
        
        
        <td>Frankia sp. EAN1pec</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.50, order:0.42, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>jadomycin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001366.1: 10-15268">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001366/index.html#r1c1" target="_new">BGC0001366.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.53, order:0.41, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>BE-24566B, zunyimycin A</td>
        
        
        <td>Streptomyces sp. FJS31-2</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001513.1: 120-11698">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001513/index.html#r1c1" target="_new">BGC0001513.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.53, order:0.41, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>anthrabenzoxocinone</td>
        
        
        <td>Actinobacteria bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000275.1: 58-12211">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000275/index.html#r1c1" target="_new">BGC0000275.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.53, order:0.41, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>tetracenomycin C</td>
        
        
        <td>Streptomyces glaucescens</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r37c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.52, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>naphtocyclinone</td>
      <td>Streptomyces arenae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.50, order:0.45, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>spore pigment</td>
      <td>Streptomyces collinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.51, order:0.48, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>granaticin</td>
      <td>Streptomyces violaceoruber</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.50, order:0.45, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>curamycin</td>
      <td>Streptomyces cyaneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.50, order:0.45, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>griseusin</td>
      <td>Streptomyces griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.51, order:0.41, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Polyketide</td>
      <td>frankiamicin</td>
      <td>Frankia sp. EAN1pec</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.50, order:0.42, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Polyketide</td>
      <td>jadomycin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001366.1: 10-15268">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001366/index.html#r1c1" target="_new">BGC0001366.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.53, order:0.41, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>BE-24566B, zunyimycin A</td>
      <td>Streptomyces sp. FJS31-2</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001513.1: 120-11698">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001513/index.html#r1c1" target="_new">BGC0001513.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.53, order:0.41, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>anthrabenzoxocinone</td>
      <td>Actinobacteria bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000275.1: 58-12211">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000275/index.html#r1c1" target="_new">BGC0000275.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.53, order:0.41, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>tetracenomycin C</td>
      <td>Streptomyces glaucescens</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r37c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-54"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-54">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r37c1-pfam-details-svg" data-filename="c00037_NODE_29.._r37c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r37c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
      <div class="sidepanel-details-header r37c1-t2pks" data-name="r37c1-t2pks">
        Type II PKS
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    <div class="sidepanel-details-section r37c1-t2pks">
      <div class="more-details">
    <div class="heading">
      <span>Type II PKS gene cluster prediction details</span>
      <div class="help-container"> <div class="help-icon" data-id="t2pks-help-52"></div> <span class="help-tooltip" id="t2pks-help-52">Predictions of starter units, elongations, product classes, and potential molecular weights for type II PKS clusters.<br>More detailed information is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/t2pks' target='_blank'>here</a>.</span></div>
    </div>
	
	
	<dl class="prediction-text">
        <dt><strong>Cluster section:</strong></dt>
        <dd>0 to 10392</dd>
		
		<dt><strong>Starter unit(s):</strong></dt>
		<dd>
				acetyl-CoA (Score: 0.0; E-value: 0)<br>
			
	    <dd>
		</tr>
		
		
		<dt><strong>Malonyl elongations:</strong></dt>
        <dd>
			
			8|9 (Score: 572.0; E-value: 1.4e-174)<br>
			
    	</dd>
		
		
		<dt><strong>Product class(es):</strong></td>
        <dd>
				
				angucycline<br>
				
				anthracycline<br>
				
				aureolic acid<br>
				
				tetracycline<br>
				
		</dd>
		
		
		<dt><strong>Product molecular weight(s):</strong></dt>
		<dd>
			
			acetyl-CoA_8: 363.34 Da<br>
			
			acetyl-CoA_9: 405.38 Da<br>
			
		</dd>
		
	</dl>
	<dl class="prediction-text">
		
		<dt><strong>cds2037_3303</strong></dt>
			
			<dd>
				CLF
				
					8|9
				
				(Score: 572.0; E-value: 1.4e-174)<br>
			</dd>
			
		
		<dt><strong>cds4188_5109</strong></dt>
			
			<dd>
				CYC
				
					C7-C12
				
				(Score: 261.8; E-value: 1.5e-80)<br>
			</dd>
			
		
		<dt><strong>cds769_2041</strong></dt>
			
			<dd>
				KS
				
				(Score: 659.8; E-value: 3.5e-201)<br>
			</dd>
			
		
	</dl>
    
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r43c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00043_NODE_32.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-269"></div> <span class="help-tooltip" id="region_svg-help-269">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00043_NODE_32...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r43c1-svg" data-filename="c00043_NODE_32.._r43c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 14,296 nt. (total: 14,296 nt)
        
        <a class="cluster-rules-header" id="r43c1-rules-header" href="#r43c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r43c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r43c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-270"></div> <span class="help-tooltip" id="focus-panel-help-270">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r43c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r43c1-clusterblast" data-name="r43c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r43c1-knownclusterblast" data-name="r43c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r43c1-subclusterblast" data-name="r43c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r43c1-MIBiG-cluster-compare" data-name="r43c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r43c1-pfam-details" data-name="r43c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r43c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-55"></div> <span class="help-tooltip" id="cb-general-help-55">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r43c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r43c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r43c1_1.svg">
        NZ_CP039123 (6113979-6133747): Streptomyces sp. SS52 chromosome, complete genome (66% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r43c1_2.svg">
        NZ_KZ819185 (730569-752750): Streptomyces sp. CS207 scaffold00002, whole geno... (63% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r43c1_3.svg">
        NZ_CP027297 (6262360-6282141): Streptomyces sp. SGAir0924 chromosome, complet... (66% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r43c1_4.svg">
        NZ_CP016795 (6331529-6353200): Streptomyces olivaceus strain KLBMP 5084 chrom... (70% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r43c1_5.svg">
        NZ_CP036534 (6392627-6414349): Streptomyces sp. VN1 chromosome, complete genome (66% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r43c1_6.svg">
        NZ_LIPF01000013 (37533-59714): Streptomyces sp. CB02414 scaffold13, whole gen... (60% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r43c1_7.svg">
        NZ_KN039947 (823163-843789): Streptomyces luteus strain TRM 45540 Scaffold2, ... (60% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r43c1_8.svg">
        NZ_CP019724 (1997020-2017092): Streptomyces pactum strain ACT12 chromosome, c... (66% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r43c1_9.svg">
        NZ_KL585972 (171414-193580): Streptomyces sp. NRRL S-474 P279 Doro1 scaffold2... (76% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r43c1_10.svg">
        NZ_JACHGV010000001 (992424-1014584): Streptomyces paradoxus strain DSM 43350 ... (72% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r43c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r43c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r43c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-56"></div> <span class="help-tooltip" id="cb-known-help-56">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r43c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r43c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r43c1_1.svg">
        geosmin
      </option>
      
    </select>
    <button id="knownclusterblast-r43c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r43c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r43c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-57"></div> <span class="help-tooltip" id="cb-sub-help-57">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r43c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-58"></div> <span class="help-tooltip" id="MIBiG-help-58">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r43c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r43c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r43c1"></div>
    <div id="comparison-MIBiG-r43c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001181.1: 6666218-6668399">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001181/index.html#r1c1" target="_new">BGC0001181.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.84 (id:0.70, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.84), rgb(0, 0, 0, 0.84) 84%, white 84%)"></td>
         
        
        <td>0.59</td>
        <td>Terpene</td>
        
        <td>geosmin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.52, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Terpene</td>
        
        <td>geosmin</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001746.1: 7136894-7138013">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001746/index.html#r1c1" target="_new">BGC0001746.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Terpene</td>
        
        <td>pristinol</td>
        
        
        <td>Streptomyces pristinaespiralis ATCC 25486</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000849.1: 6890527-6892237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000849/index.html#r1c1" target="_new">BGC0000849.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other,  , (, B, u, t, y, r, o, l, a, c, t, o, n, e, )</td>
        
        <td>SCB1, SCB2, SCB3</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.57, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>γ-butyrolactone</td>
        
        
        <td>Streptomyces acidiscabies</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001285.1: 8-22479">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001285/index.html#r1c1" target="_new">BGC0001285.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.47, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Other,  , (, F, a, t, t, y,  , a, c, i, d, )</td>
        
        <td>pseudopyronine A, pseudopyronine B</td>
        
        
        <td>Pseudomonas putida</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001510.1: 425-14535">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001510/index.html#r1c1" target="_new">BGC0001510.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>Other</td>
        
        <td>anisomycin</td>
        
        
        <td>Streptomyces hygrospinosus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001006.1: 3494093-3519138">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001006/index.html#r1c1" target="_new">BGC0001006.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
         
        
        <td>0.09</td>
        <td>NRP, Polyketide</td>
        
        <td>lymphostin, neolymphostinol B, lymphostinol, neolymphostin B</td>
        
        
        <td>Salinispora tropica CNB-440</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001007.1: 3781644-3805331">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001007/index.html#r1c1" target="_new">BGC0001007.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
         
        
        <td>0.09</td>
        <td>Polyketide, NRP</td>
        
        <td>lymphostin, neolymphostinol B, lymphostinol, neolymphostin b</td>
        
        
        <td>Salinispora arenicola CNS-205</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001669.1: 4732-29763">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001669/index.html#r1c1" target="_new">BGC0001669.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.28 (id:0.43, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
         
        
        <td>0.07</td>
        <td>RiPP</td>
        
        <td>microvionin</td>
        
        
        <td>Microbacterium arborescens</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r43c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001181.1: 6666218-6668399">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001181/index.html#r1c1" target="_new">BGC0001181.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.84 (id:0.70, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.84), rgb(0, 0, 0, 0.84) 84%, white 84%)"></td>
      <td>0.84</td>
      <td>Terpene</td>
      <td>geosmin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.52, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Terpene</td>
      <td>geosmin</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001746.1: 7136894-7138013">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001746/index.html#r1c1" target="_new">BGC0001746.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>pristinol</td>
      <td>Streptomyces pristinaespiralis ATCC 25486</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000849.1: 6890527-6892237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000849/index.html#r1c1" target="_new">BGC0000849.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other,  , (, B, u, t, y, r, o, l, a, c, t, o, n, e, )</td>
      <td>SCB1, SCB2, SCB3</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.57, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>γ-butyrolactone</td>
      <td>Streptomyces acidiscabies</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001285.1: 8-22479">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001285/index.html#r1c1" target="_new">BGC0001285.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.47, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Other,  , (, F, a, t, t, y,  , a, c, i, d, )</td>
      <td>pseudopyronine A, pseudopyronine B</td>
      <td>Pseudomonas putida</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001510.1: 425-14535">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001510/index.html#r1c1" target="_new">BGC0001510.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>Other</td>
      <td>anisomycin</td>
      <td>Streptomyces hygrospinosus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001006.1: 3494093-3519138">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001006/index.html#r1c1" target="_new">BGC0001006.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
      <td>0.34</td>
      <td>NRP, Polyketide</td>
      <td>lymphostin, neolymphostinol B, lymphostinol, neolymphostin B</td>
      <td>Salinispora tropica CNB-440</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001007.1: 3781644-3805331">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001007/index.html#r1c1" target="_new">BGC0001007.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
      <td>0.34</td>
      <td>Polyketide, NRP</td>
      <td>lymphostin, neolymphostinol B, lymphostinol, neolymphostin b</td>
      <td>Salinispora arenicola CNS-205</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001669.1: 4732-29763">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001669/index.html#r1c1" target="_new">BGC0001669.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.28 (id:0.43, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
      <td>0.28</td>
      <td>RiPP</td>
      <td>microvionin</td>
      <td>Microbacterium arborescens</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r43c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-59"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-59">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r43c1-pfam-details-svg" data-filename="c00043_NODE_32.._r43c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r43c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r48c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00048_NODE_8_.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-271"></div> <span class="help-tooltip" id="region_svg-help-271">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00048_NODE_8_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r48c1-svg" data-filename="c00048_NODE_8_.._r48c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 19,460 - 41,244 nt. (total: 21,785 nt)
        
        <a class="cluster-rules-header" id="r48c1-rules-header" href="#r48c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r48c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r48c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-272"></div> <span class="help-tooltip" id="focus-panel-help-272">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r48c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r48c1-clusterblast" data-name="r48c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r48c1-knownclusterblast" data-name="r48c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r48c1-subclusterblast" data-name="r48c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r48c1-MIBiG-cluster-compare" data-name="r48c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r48c1-pfam-details" data-name="r48c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r48c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-60"></div> <span class="help-tooltip" id="cb-general-help-60">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r48c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r48c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r48c1_1.svg">
        NZ_KZ819184 (3998342-4020226): Streptomyces sp. CS207 scaffold00001, whole ge... (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r48c1_2.svg">
        NZ_CP039123 (398269-418830): Streptomyces sp. SS52 chromosome, complete genome (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r48c1_3.svg">
        NZ_CP027297 (508491-529070): Streptomyces sp. SGAir0924 chromosome, complete ... (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r48c1_4.svg">
        NZ_CP042278 (7485676-7507561): Streptomyces sp. WAC06273 strain WAC6273 subst... (90% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r48c1_5.svg">
        NZ_SLVZ01000003 (101179-194010): Streptomyces sp. BK387 Ga0307671 103, whole ... (9% of genes show similarity), T2PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r48c1_6.svg">
        NZ_CP009754 (4774729-4794869): Streptomyces sp. CCM MD2014 chromosome, comple... (19% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r48c1_7.svg">
        NZ_KQ948994 (276268-298086): Streptomyces resistomycificus strain DSM 40133 P... (26% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r48c1_8.svg">
        NZ_CP029241 (1812201-1833226): Streptomyces sp. NHF165 chromosome, complete g... (19% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r48c1_9.svg">
        NZ_CM000951 (7531237-7552547): Streptomyces sviceus ATCC 29083 chromosome, wh... (21% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r48c1_10.svg">
        NZ_VSKT01000015 (394029-415881): Streptomyces cacaoi strain OABC16 Scaffold15... (21% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r48c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r48c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r48c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-61"></div> <span class="help-tooltip" id="cb-known-help-61">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r48c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r48c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r48c1_1.svg">
        lysolipin I
      </option>
      
      <option value = "svg/knownclusterblast_r48c1_2.svg">
        arylomycin
      </option>
      
    </select>
    <button id="knownclusterblast-r48c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r48c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r48c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-62"></div> <span class="help-tooltip" id="cb-sub-help-62">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r48c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-63"></div> <span class="help-tooltip" id="MIBiG-help-63">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r48c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r48c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r48c1"></div>
    <div id="comparison-MIBiG-r48c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001184.1: 3593876-3601174">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001184/index.html#r1c1" target="_new">BGC0001184.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>bacilysin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888.1: 0-4342">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>bacilysin</td>
        
        
        <td>Bacillus sp. CS93</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076.1: 161645-164819">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Other</td>
        
        <td>4,5-Dehydroacaterin, Acaterin</td>
        
        
        <td>Pseudomonas jessenii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000636.1: 2-14167">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000636/index.html#r1c1" target="_new">BGC0000636.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.50, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Brevibacterium linens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>frankiamicin</td>
        
        
        <td>Frankia sp. EAN1pec</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001927.1: 0-27705">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001927/index.html#r1c1" target="_new">BGC0001927.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Other</td>
        
        <td>A-94964</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.44, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>granaticin</td>
        
        
        <td>Streptomyces violaceoruber</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000509.1: 4195-15261">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000509/index.html#r1c1" target="_new">BGC0000509.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>RiPP</td>
        
        <td>epilancin 15x</td>
        
        
        <td>Staphylococcus epidermidis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Polyketide</td>
        
        <td>jadomycin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000218.1: 27-8720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000218/index.html#r1c1" target="_new">BGC0000218.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Polyketide</td>
        
        <td>doxorubicin</td>
        
        
        <td>Streptomyces peucetius</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r48c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001184.1: 3593876-3601174">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001184/index.html#r1c1" target="_new">BGC0001184.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>bacilysin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000888.1: 0-4342">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000888/index.html#r1c1" target="_new">BGC0000888.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>bacilysin</td>
      <td>Bacillus sp. CS93</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076.1: 161645-164819">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Other</td>
      <td>4,5-Dehydroacaterin, Acaterin</td>
      <td>Pseudomonas jessenii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000636.1: 2-14167">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000636/index.html#r1c1" target="_new">BGC0000636.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.50, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Brevibacterium linens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Polyketide</td>
      <td>frankiamicin</td>
      <td>Frankia sp. EAN1pec</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001927.1: 0-27705">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001927/index.html#r1c1" target="_new">BGC0001927.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Other</td>
      <td>A-94964</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.44, order:0.48, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>granaticin</td>
      <td>Streptomyces violaceoruber</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000509.1: 4195-15261">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000509/index.html#r1c1" target="_new">BGC0000509.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>RiPP</td>
      <td>epilancin 15x</td>
      <td>Staphylococcus epidermidis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Polyketide</td>
      <td>jadomycin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000218.1: 27-8720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000218/index.html#r1c1" target="_new">BGC0000218.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Polyketide</td>
      <td>doxorubicin</td>
      <td>Streptomyces peucetius</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r48c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-64"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-64">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r48c1-pfam-details-svg" data-filename="c00048_NODE_8_.._r48c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r48c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r57c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00057_NODE_10.. - Region 1 - melanin <div class="help-container"> <div class="help-icon" data-id="region_svg-help-273"></div> <span class="help-tooltip" id="region_svg-help-273">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00057_NODE_10...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r57c1-svg" data-filename="c00057_NODE_10.._r57c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 6,883 nt. (total: 6,883 nt)
        
        <a class="cluster-rules-header" id="r57c1-rules-header" href="#r57c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r57c1-rules">melanin: melC</div>
      
      <div class="region-svg-container">
       <div id='r57c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-274"></div> <span class="help-tooltip" id="focus-panel-help-274">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r57c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r57c1-clusterblast" data-name="r57c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r57c1-knownclusterblast" data-name="r57c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r57c1-subclusterblast" data-name="r57c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r57c1-MIBiG-cluster-compare" data-name="r57c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r57c1-pfam-details" data-name="r57c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r57c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-65"></div> <span class="help-tooltip" id="cb-general-help-65">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r57c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r57c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r57c1_1.svg">
        NZ_CP042278 (5037528-5047948): Streptomyces sp. WAC06273 strain WAC6273 subst... (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r57c1_2.svg">
        NZ_CP027297 (2976866-2987475): Streptomyces sp. SGAir0924 chromosome, complet... (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r57c1_3.svg">
        NZ_KZ819184 (1544416-1555025): Streptomyces sp. CS207 scaffold00001, whole ge... (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r57c1_4.svg">
        NZ_CP039123 (2898042-2908534): Streptomyces sp. SS52 chromosome, complete genome (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r57c1_5.svg">
        NZ_LIPF01000001 (964784-975399): Streptomyces sp. CB02414 scaffold1, whole ge... (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r57c1_6.svg">
        NZ_KZ195577 (5438387-5448843): Streptomyces sp. CS159 scaffold00001, whole ge... (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r57c1_7.svg">
        NZ_WHZR01000001 (5341496-5352129): Streptomyces sp. OM5714 A912 AS4 SC1 R2, w... (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r57c1_8.svg">
        NZ_WXVT01000001 (5625797-5636259): Streptomyces sp. KO7888 A913 AS4 SC1 R3, w... (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r57c1_9.svg">
        NZ_CP045547 (5677136-5687784): Streptomyces sp. SYP-A7193 chromosome, complet... (80% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r57c1_10.svg">
        NZ_CP034353 (5318189-5328651): Streptomyces sp. KPB2 chromosome, complete genome (66% of genes show similarity), melanin
      </option>
      
    </select>
    <button id= "clusterblast-r57c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r57c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r57c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-66"></div> <span class="help-tooltip" id="cb-known-help-66">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r57c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r57c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r57c1_1.svg">
        melanin
      </option>
      
      <option value = "svg/knownclusterblast_r57c1_2.svg">
        istamycin
      </option>
      
      <option value = "svg/knownclusterblast_r57c1_3.svg">
        melanin
      </option>
      
      <option value = "svg/knownclusterblast_r57c1_4.svg">
        melanin
      </option>
      
      <option value = "svg/knownclusterblast_r57c1_5.svg">
        lactonamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r57c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r57c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r57c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-67"></div> <span class="help-tooltip" id="cb-sub-help-67">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r57c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-68"></div> <span class="help-tooltip" id="MIBiG-help-68">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r57c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r57c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r57c1"></div>
    <div id="comparison-MIBiG-r57c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">melanin</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000910.1: 2943455-2944875">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000910/index.html#r1c1" target="_new">BGC0000910.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.88 (id:0.68, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.88), rgb(0, 0, 0, 0.88) 88%, white 88%)"></td>
         
        
        <td>0.68</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000911.1: 607505-608793">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000911/index.html#r1c1" target="_new">BGC0000911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.80 (id:0.51, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
         
        
        <td>0.52</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000912.1: 2884831-2886096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000912/index.html#r1c1" target="_new">BGC0000912.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000909.1: 0-6625">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000909/index.html#r1c1" target="_new">BGC0000909.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.56, order:0.56, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000908.1: 0-6497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000908/index.html#r1c1" target="_new">BGC0000908.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.50, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.49, order:0.43, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Terpene</td>
        
        <td>grixazone</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.47, order:0.43, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B</td>
        
        
        <td>Streptomyces sp. WK-5344</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001592.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001592/index.html#r1c1" target="_new">BGC0001592.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.48, order:0.43, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>ferroverdin</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002029.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002029/index.html#r1c1" target="_new">BGC0002029.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.48, order:0.43, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B, bagremycin C, bagremycin E, bagremycin F, bagremycin G, ferroverdin A, ferroverdin B, ferroverdin C</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885.1: 0-16706">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.47, order:0.43, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>4-hydroxy-3-nitrosobenzamide</td>
        
        
        <td>Streptomyces murayamaensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r57c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000910.1: 2943455-2944875">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000910/index.html#r1c1" target="_new">BGC0000910.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.88 (id:0.68, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.88), rgb(0, 0, 0, 0.88) 88%, white 88%)"></td>
      <td>0.88</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000911.1: 607505-608793">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000911/index.html#r1c1" target="_new">BGC0000911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.80 (id:0.51, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
      <td>0.80</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000912.1: 2884831-2886096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000912/index.html#r1c1" target="_new">BGC0000912.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000909.1: 0-6625">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000909/index.html#r1c1" target="_new">BGC0000909.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.56, order:0.56, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000908.1: 0-6497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000908/index.html#r1c1" target="_new">BGC0000908.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.50, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.49, order:0.43, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Terpene</td>
      <td>grixazone</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.47, order:0.43, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B</td>
      <td>Streptomyces sp. WK-5344</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001592.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001592/index.html#r1c1" target="_new">BGC0001592.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.48, order:0.43, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Other</td>
      <td>ferroverdin</td>
      <td>Streptomyces lunaelactis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002029.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002029/index.html#r1c1" target="_new">BGC0002029.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.48, order:0.43, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B, bagremycin C, bagremycin E, bagremycin F, bagremycin G, ferroverdin A, ferroverdin B, ferroverdin C</td>
      <td>Streptomyces lunaelactis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885.1: 0-16706">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.47, order:0.43, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other</td>
      <td>4-hydroxy-3-nitrosobenzamide</td>
      <td>Streptomyces murayamaensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r57c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-69"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-69">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r57c1-pfam-details-svg" data-filename="c00057_NODE_10.._r57c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r57c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r65c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00065_NODE_25.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-275"></div> <span class="help-tooltip" id="region_svg-help-275">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00065_NODE_25...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r65c1-svg" data-filename="c00065_NODE_25.._r65c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 25,575 nt. (total: 25,575 nt)
        
        <a class="cluster-rules-header" id="r65c1-rules-header" href="#r65c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r65c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r65c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-276"></div> <span class="help-tooltip" id="focus-panel-help-276">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r65c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r65c1-clusterblast" data-name="r65c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r65c1-knownclusterblast" data-name="r65c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r65c1-subclusterblast" data-name="r65c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r65c1-MIBiG-cluster-compare" data-name="r65c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r65c1-pfam-details" data-name="r65c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r65c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-70"></div> <span class="help-tooltip" id="cb-general-help-70">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r65c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r65c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r65c1_1.svg">
        NZ_CP009754 (1043336-1069144): Streptomyces sp. CCM MD2014 chromosome, comple... (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r65c1_2.svg">
        NZ_CP045547 (1125038-1150932): Streptomyces sp. SYP-A7193 chromosome, complet... (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r65c1_3.svg">
        NZ_BBOK01000011 (408408-435138): Streptomyces sp. NBRC 110030, whole genome s... (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r65c1_4.svg">
        NZ_CP027297 (6913168-6939092): Streptomyces sp. SGAir0924 chromosome, complet... (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r65c1_5.svg">
        NZ_KN039947 (111289-137177): Streptomyces luteus strain TRM 45540 Scaffold2, ... (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r65c1_6.svg">
        NZ_LIPF01000005 (78300-104171): Streptomyces sp. CB02414 scaffold5, whole gen... (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r65c1_7.svg">
        NZ_CP019724 (1266965-1292846): Streptomyces pactum strain ACT12 chromosome, c... (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r65c1_8.svg">
        NZ_CP012949 (6684466-6710353): Streptomyces ambofaciens strain DSM 40697 chro... (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r65c1_9.svg">
        NZ_CP012382 (6901408-6927295): Streptomyces ambofaciens ATCC 23877 chromosome... (95% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r65c1_10.svg">
        NZ_MQUH01000026 (63231-88994): Streptomyces sp. M1013 M1013 C26, whole genome... (95% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r65c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r65c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r65c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-71"></div> <span class="help-tooltip" id="cb-known-help-71">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r65c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r65c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r65c1_1.svg">
        hopene
      </option>
      
      <option value = "svg/knownclusterblast_r65c1_2.svg">
        xiamycin
      </option>
      
      <option value = "svg/knownclusterblast_r65c1_3.svg">
        xiamycin A
      </option>
      
      <option value = "svg/knownclusterblast_r65c1_4.svg">
        platencin
      </option>
      
      <option value = "svg/knownclusterblast_r65c1_5.svg">
        platensimycin / platencin
      </option>
      
      <option value = "svg/knownclusterblast_r65c1_6.svg">
        thioplatensimycin / thioplatencin
      </option>
      
      <option value = "svg/knownclusterblast_r65c1_7.svg">
        phenalinolactone A
      </option>
      
      <option value = "svg/knownclusterblast_r65c1_8.svg">
        KS-505a
      </option>
      
      <option value = "svg/knownclusterblast_r65c1_9.svg">
        phosphonoglycans
      </option>
      
    </select>
    <button id="knownclusterblast-r65c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r65c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r65c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-72"></div> <span class="help-tooltip" id="cb-sub-help-72">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r65c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r65c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r65c1_1.svg">
        DQ230532_c2: phenalinolactone terpenoid (33% of genes show similarity), terpenoid
      </option>
      
    </select>
    <button id="subclusterblast-r65c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r65c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r65c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-73"></div> <span class="help-tooltip" id="MIBiG-help-73">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r65c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r65c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r65c1"></div>
    <div id="comparison-MIBiG-r65c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663.1: 7516016-7529773">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.88 (id:0.68, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.88), rgb(0, 0, 0, 0.88) 88%, white 88%)"></td>
         
        
        <td>0.68</td>
        <td>Terpene</td>
        
        <td>hopene</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645.1: 0-7475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Halobacillus halophilus DSM 2266</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Myxococcus xanthus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Rhodobacter sphaeroides</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644.1: 97-6276">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Dietzia sp. CQ4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>β-carotein</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630.1: 389-6051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
        
        
        <td>Paracoccus haeundaensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656.1: 169-3770">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Terpene</td>
        
        <td>zeaxanthin</td>
        
        
        <td>Xanthobacter autotrophicus Py2</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000638.1: 224-6452">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000638/index.html#r1c1" target="_new">BGC0000638.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.22</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Pantoea ananatis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r65c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663.1: 7516016-7529773">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.88 (id:0.68, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.88), rgb(0, 0, 0, 0.88) 88%, white 88%)"></td>
      <td>0.88</td>
      <td>Terpene</td>
      <td>hopene</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645.1: 0-7475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Halobacillus halophilus DSM 2266</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Myxococcus xanthus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Rhodobacter sphaeroides</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644.1: 97-6276">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Dietzia sp. CQ4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene</td>
      <td>β-carotein</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630.1: 389-6051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
      <td>Paracoccus haeundaensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656.1: 169-3770">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>zeaxanthin</td>
      <td>Xanthobacter autotrophicus Py2</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000638.1: 224-6452">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000638/index.html#r1c1" target="_new">BGC0000638.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Pantoea ananatis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r65c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-74"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-74">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r65c1-pfam-details-svg" data-filename="c00065_NODE_25.._r65c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r65c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r76c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00076_NODE_88.. - Region 1 - lanthipeptide-class-iii <div class="help-container"> <div class="help-icon" data-id="region_svg-help-277"></div> <span class="help-tooltip" id="region_svg-help-277">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00076_NODE_88...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r76c1-svg" data-filename="c00076_NODE_88.._r76c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 32,111 - 54,714 nt. (total: 22,604 nt)
        
        <a class="cluster-rules-header" id="r76c1-rules-header" href="#r76c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r76c1-rules">lanthipeptide-class-iii: (micKC and not PKS_KS)</div>
      
      <div class="region-svg-container">
       <div id='r76c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-278"></div> <span class="help-tooltip" id="focus-panel-help-278">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r76c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r76c1-lanthipeptides" data-name="r76c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r76c1-clusterblast" data-name="r76c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r76c1-knownclusterblast" data-name="r76c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r76c1-subclusterblast" data-name="r76c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r76c1-MIBiG-cluster-compare" data-name="r76c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r76c1-pfam-details" data-name="r76c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r76c1-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-75"></div> <span class="help-tooltip" id="lanthi-body-help-75">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lanthipeptide(s) for cds42110_44714 - Putative Class III</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds41687_41801</span> leader / core peptide
          </dt>
          <dd>
            MSLLDLQEMELPTDSAE
            -
            E<span class="dhb">Dhb</span>VLG<span class="dha">Dha</span>GK<span class="dha">Dha</span>QH<span class="cys">C</span><span class="dha">Dha</span>DA<span class="dha">Dha</span>WVF<span class="cys">C</span>
          <dt>
            <span class="serif">cds41886_42000</span> leader / core peptide
          </dt>
          <dd>
            MSLLDLQEMELSTDSAE
            -
            E<span class="dhb">Dhb</span>VLG<span class="dha">Dha</span>GQ<span class="dha">Dha</span>QH<span class="cys">C</span><span class="dha">Dha</span>DA<span class="dha">Dha</span>WVF<span class="cys">C</span>
        </dd>
      </dl>
    
    <div class="legend">
      <h4> Legend: </h4>
      <div>
        <span class="dha">Dha</span>: Didehydroalanine<br>
        <span class="dhb">Dhb</span>: Didehydrobutyrine<br>
      </div>
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r76c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-77"></div> <span class="help-tooltip" id="cb-general-help-77">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r76c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r76c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r76c1_1.svg">
        NZ_CP039123 (4260535-4280291): Streptomyces sp. SS52 chromosome, complete genome (88% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r76c1_2.svg">
        NZ_CP042278 (3654727-3677331): Streptomyces sp. WAC06273 strain WAC6273 subst... (83% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r76c1_3.svg">
        NZ_CP027297 (4366293-4388897): Streptomyces sp. SGAir0924 chromosome, complet... (93% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r76c1_4.svg">
        NZ_KZ819185 (2140395-2162999): Streptomyces sp. CS207 scaffold00002, whole ge... (87% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r76c1_5.svg">
        NZ_CP058693 (6316130-6338068): Streptomyces sp. Rer75 chromosome (43% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r76c1_6.svg">
        NZ_JACHJI010000005 (122532-191473): Streptomyces griseostramineus strain CECT... (9% of genes show similarity), NRPS,T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r76c1_7.svg">
        NZ_CP031194 (203696-226284): Streptomyces paludis strain GSSD-12 chromosome, ... (29% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r76c1_8.svg">
        NZ_BBXF01000001 (1012803-1035389): Herbidospora daliensis strain NBRC 106372,... (26% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r76c1_9.svg">
        NZ_KQ950181 (304332-367165): Thermobifida cellulosilytica TB100 scaffold2, wh... (6% of genes show similarity), T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r76c1_10.svg">
        NZ_FTNI01000016 (162047-203115): Microbispora rosea strain ATCC 12950, whole ... (13% of genes show similarity), T3PKS,lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r76c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r76c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r76c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-78"></div> <span class="help-tooltip" id="cb-known-help-78">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r76c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r76c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r76c1_1.svg">
        catenulipeptin
      </option>
      
      <option value = "svg/knownclusterblast_r76c1_2.svg">
        SapB
      </option>
      
      <option value = "svg/knownclusterblast_r76c1_3.svg">
        labyrinthopeptin A2 / labyrinthopeptin A1 / labyrinthopeptin A3
      </option>
      
    </select>
    <button id="knownclusterblast-r76c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r76c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r76c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-79"></div> <span class="help-tooltip" id="cb-sub-help-79">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r76c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-80"></div> <span class="help-tooltip" id="MIBiG-help-80">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r76c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r76c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r76c1"></div>
    <div id="comparison-MIBiG-r76c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-iii</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.50, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>RiPP</td>
        
        <td>SapB</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>RiPP</td>
        
        <td>venezuelin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496.1: 2822112-2829457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.49, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>RiPP</td>
        
        <td>AmfS</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.50, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>RiPP</td>
        
        <td>catenulipeptin</td>
        
        
        <td>Catenulispora acidiphila DSM 44928</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.49, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>RiPP</td>
        
        <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
        
        
        <td>Actinomadura namibiensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001579.1: 85465-94303">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001579/index.html#r1c1" target="_new">BGC0001579.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>RiPP</td>
        
        <td>duramycin</td>
        
        
        <td>Streptomyces cinnamoneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000503.1: 0-17083">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000503/index.html#r1c1" target="_new">BGC0000503.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>RiPP</td>
        
        <td>cinnamycin</td>
        
        
        <td>Streptomyces cinnamoneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000516.1: 8706-19626">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000516/index.html#r1c1" target="_new">BGC0000516.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>geobacillin II</td>
        
        
        <td>Geobacillus thermodenitrificans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.52, order:0.40, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>informatipeptin</td>
        
        
        <td>Streptomyces viridochromogenes DSM 40736</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r76c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.50, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>RiPP</td>
      <td>SapB</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>RiPP</td>
      <td>venezuelin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496.1: 2822112-2829457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.49, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>RiPP</td>
      <td>AmfS</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.50, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>RiPP</td>
      <td>catenulipeptin</td>
      <td>Catenulispora acidiphila DSM 44928</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.49, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>RiPP</td>
      <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
      <td>Actinomadura namibiensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001579.1: 85465-94303">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001579/index.html#r1c1" target="_new">BGC0001579.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>RiPP</td>
      <td>duramycin</td>
      <td>Streptomyces cinnamoneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000503.1: 0-17083">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000503/index.html#r1c1" target="_new">BGC0000503.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>RiPP</td>
      <td>cinnamycin</td>
      <td>Streptomyces cinnamoneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000516.1: 8706-19626">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000516/index.html#r1c1" target="_new">BGC0000516.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>geobacillin II</td>
      <td>Geobacillus thermodenitrificans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.52, order:0.40, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>informatipeptin</td>
      <td>Streptomyces viridochromogenes DSM 40736</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r76c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-81"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-81">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r76c1-pfam-details-svg" data-filename="c00076_NODE_88.._r76c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r76c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r76c1-lanthipeptides" data-name="r76c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r76c1-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-76"></div> <span class="help-tooltip" id="lanthi-side-help-76">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

    <dl class ="prediction-text">
    <dt><span class="serif">cds41687_41801</span> - Class III</dt>
        <dd>Cleavage pHMM score: 6.60</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 2049.9 Da</dd>
        <dd>Molecular weight: 2051.2 Da</dd>
        <dd>Number of bridges: 2</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 2069.3 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2087.3 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2105.3 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
            </dl>
          </dd>
        <dt><span class="serif">cds41886_42000</span> - Class III</dt>
        <dd>Cleavage pHMM score: 4.90</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 2049.9 Da</dd>
        <dd>Molecular weight: 2051.2 Da</dd>
        <dd>Number of bridges: 2</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 2069.2 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2087.2 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2105.3 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
            </dl>
          </dd>
        
  </dl>

</div>

    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r91c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00091_NODE_56.. - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-279"></div> <span class="help-tooltip" id="region_svg-help-279">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00091_NODE_56...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r91c1-svg" data-filename="c00091_NODE_56.._r91c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 38,398 nt. (total: 38,398 nt)
        
        <a class="cluster-rules-header" id="r91c1-rules-header" href="#r91c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r91c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r91c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-280"></div> <span class="help-tooltip" id="focus-panel-help-280">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r91c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r91c1-nrps_pks" data-name="r91c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r91c1-clusterblast" data-name="r91c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r91c1-knownclusterblast" data-name="r91c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r91c1-subclusterblast" data-name="r91c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r91c1-MIBiG-cluster-compare" data-name="r91c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r91c1-pfam-details" data-name="r91c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r91c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-82"></div> <span class="help-tooltip" id="nrps-domain-help-82">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r91c1-details-svg" data-filename="c00091_NODE_56.._r91c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r91c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r91c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-85"></div> <span class="help-tooltip" id="cb-general-help-85">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r91c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r91c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r91c1_1.svg">
        NZ_CP039123 (260692-377530): Streptomyces sp. SS52 chromosome, complete genome (23% of genes show similarity), T1PKS,hglE-KS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r91c1_2.svg">
        NZ_GG657757 (926849-989730): Streptomyces viridochromogenes DSM 40736 superco... (15% of genes show similarity), T1PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r91c1_3.svg">
        NZ_JTHL01000001 (&lt;753097-913714): Streptomyces sp. 150FB scaffold00001, whole... (29% of genes show similarity), T1PKS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r91c1_4.svg">
        NZ_CP050975 (85585-252422): Streptomyces sp. RPA4-2 chromosome, complete genome (16% of genes show similarity), NRPS,PKS-like,T1PKS,ectoine
      </option>
      
      <option value = "svg/clusterblast_r91c1_5.svg">
        NZ_CP020700 (424700-546610): Streptomyces tsukubensis strain NRRL 18488 chrom... (11% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r91c1_6.svg">
        NC_014623 (6407323-6493561): Stigmatella aurantiaca DW4/3-1, complete sequence (13% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r91c1_7.svg">
        NZ_NNBO01000004 (72395-216506): Kitasatospora sp. CB02891 scaffold4, whole ge... (4% of genes show similarity), NRPS,PKS-like,T1PKS,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r91c1_8.svg">
        NC_020126 (7721561-7824172): Myxococcus stipitatus DSM 14675, complete sequence (11% of genes show similarity), NRPS,T1PKS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r91c1_9.svg">
        NZ_CP045081 (2467048-2531061): Mycobacterium kubicae strain NJH MKUB1 chromos... (8% of genes show similarity), T1PKS,T3PKS,TfuA-related
      </option>
      
      <option value = "svg/clusterblast_r91c1_10.svg">
        NZ_CP045075 (2589326-2653355): Mycobacterium kubicae strain NJH MKUB2 chromos... (7% of genes show similarity), T1PKS,T3PKS,TfuA-related
      </option>
      
    </select>
    <button id= "clusterblast-r91c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r91c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r91c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-86"></div> <span class="help-tooltip" id="cb-known-help-86">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r91c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r91c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r91c1_1.svg">
        67-121C
      </option>
      
      <option value = "svg/knownclusterblast_r91c1_2.svg">
        stambomycin A / stambomycin B / stambomycin C / stambomycin D
      </option>
      
      <option value = "svg/knownclusterblast_r91c1_3.svg">
        ibomycin
      </option>
      
      <option value = "svg/knownclusterblast_r91c1_4.svg">
        meridamycin
      </option>
      
      <option value = "svg/knownclusterblast_r91c1_5.svg">
        meridamycin
      </option>
      
      <option value = "svg/knownclusterblast_r91c1_6.svg">
        JBIR-100
      </option>
      
      <option value = "svg/knownclusterblast_r91c1_7.svg">
        argimycin PI / argimycin PII / nigrifactin / argimycin PIV / argimycin PV / argimycin PVI / argimycin PIX
      </option>
      
      <option value = "svg/knownclusterblast_r91c1_8.svg">
        niphimycins C-E
      </option>
      
      <option value = "svg/knownclusterblast_r91c1_9.svg">
        nystatin A1
      </option>
      
      <option value = "svg/knownclusterblast_r91c1_10.svg">
        nystatin
      </option>
      
    </select>
    <button id="knownclusterblast-r91c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r91c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r91c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-87"></div> <span class="help-tooltip" id="cb-sub-help-87">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r91c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-88"></div> <span class="help-tooltip" id="MIBiG-help-88">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r91c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r91c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r91c1"></div>
    <div id="comparison-MIBiG-r91c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001164.1: 2029234-2037562">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001164/index.html#r1c1" target="_new">BGC0001164.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>1-heptadecene</td>
        
        
        <td>Cyanothece sp. PCC 7822</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>esperamicin</td>
        
        
        <td>Actinomadura verrucosospora</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001138.1: 0-39199">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001138/index.html#r1c1" target="_new">BGC0001138.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.55, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Other,  , (, N, u, c, l, e, o, s, i, d, e, )</td>
        
        <td>a201a</td>
        
        
        <td>Marinactinospora thermotolerans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001544.1: 576137-603864">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001544/index.html#r1c1" target="_new">BGC0001544.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.51, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>NRP, Polyketide</td>
        
        <td>chlorflavonin</td>
        
        
        <td>Aspergillus campestris IBT 28561</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.53, order:0.75, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Polyketide</td>
        
        <td>oronofacic acid</td>
        
        
        <td>Pseudomonas syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000795.1: 2265-27605">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000795/index.html#r1c1" target="_new">BGC0000795.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.53, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Saccharide</td>
        
        <td>S-layer glycan</td>
        
        
        <td>Aneurinibacillus thermoaerophilus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000769.1: 2-26368">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000769/index.html#r1c1" target="_new">BGC0000769.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.19</td>
        <td>Saccharide</td>
        
        <td>glycopeptidolipid</td>
        
        
        <td>Mycobacterium avium subsp. hominissuis A5</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000143.1: 2-25809">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000143/index.html#r1c1" target="_new">BGC0000143.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.53, order:0.43, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Polyketide</td>
        
        <td>salinomycin</td>
        
        
        <td>Streptomyces albus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001118.1: 54333-75724">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001118/index.html#r1c1" target="_new">BGC0001118.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.49, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Polyketide</td>
        
        <td>endocrocin</td>
        
        
        <td>Aspergillus fumigatus Af293</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001793.1: 538-21374">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001793/index.html#r1c1" target="_new">BGC0001793.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Polyketide</td>
        
        <td>swainsonine</td>
        
        
        <td>Chaetothyriaceae sp.</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r91c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001164.1: 2029234-2037562">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001164/index.html#r1c1" target="_new">BGC0001164.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>1-heptadecene</td>
      <td>Cyanothece sp. PCC 7822</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>esperamicin</td>
      <td>Actinomadura verrucosospora</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001138.1: 0-39199">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001138/index.html#r1c1" target="_new">BGC0001138.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.55, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Other,  , (, N, u, c, l, e, o, s, i, d, e, )</td>
      <td>a201a</td>
      <td>Marinactinospora thermotolerans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001544.1: 576137-603864">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001544/index.html#r1c1" target="_new">BGC0001544.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.51, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP, Polyketide</td>
      <td>chlorflavonin</td>
      <td>Aspergillus campestris IBT 28561</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.53, order:0.75, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Polyketide</td>
      <td>oronofacic acid</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000795.1: 2265-27605">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000795/index.html#r1c1" target="_new">BGC0000795.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.53, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Saccharide</td>
      <td>S-layer glycan</td>
      <td>Aneurinibacillus thermoaerophilus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000769.1: 2-26368">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000769/index.html#r1c1" target="_new">BGC0000769.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Saccharide</td>
      <td>glycopeptidolipid</td>
      <td>Mycobacterium avium subsp. hominissuis A5</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000143.1: 2-25809">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000143/index.html#r1c1" target="_new">BGC0000143.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.53, order:0.43, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Polyketide</td>
      <td>salinomycin</td>
      <td>Streptomyces albus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001118.1: 54333-75724">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001118/index.html#r1c1" target="_new">BGC0001118.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.49, order:N/A, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Polyketide</td>
      <td>endocrocin</td>
      <td>Aspergillus fumigatus Af293</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001793.1: 538-21374">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001793/index.html#r1c1" target="_new">BGC0001793.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Polyketide</td>
      <td>swainsonine</td>
      <td>Chaetothyriaceae sp.</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r91c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-89"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-89">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r91c1-pfam-details-svg" data-filename="c00091_NODE_56.._r91c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r91c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r91c1-nrps_pks_products" data-name="r91c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r91c1-nrps_pks" data-name="r91c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r91c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-83"></div> <span class="help-tooltip" id="nrps-structures-help-83">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 38398:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r91c1-1" class="smiles-canvas" data-smiles="CC(O)CC(O)CC(O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ohmal - ohmal - ohmal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal,ohmal,ohmal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ohmal*,ohmal*,ohmal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r91c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="CC(O)CC(O)CC(O)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r91c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-84"></div> <span class="help-tooltip" id="nrps-monomers-help-84">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds5279_18398</span></strong>: mal - mal - mal
      <div class="collapser collapser-target-cds5279_18398 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (197..504)</strong>: mal
          <div class="collapser collapser-target-cds5279_18398 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds5279_18398-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 100.0%</dd>
<dd>inactive: 83.3%</dd>
<dd>Propionyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds5279_18398-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 213.7</dt>

  <dd></dd><dt>inactive: 103.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 68.9</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 58.3</dt>

  <dd></dd><dt>Propionyl-CoA: 27.9</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 21.1</dt>

  <dd></dd><dt>Acetyl-CoA: 19.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 15.0</dt>

  <dd></dd><dt>trans-1,2-CPDA: 11.6</dt>

  <dd></dd><dt>fatty_acid: 11.1</dt>

  <dd></dd><dt>CHC-CoA: 8.4</dt>

  <dd></dd><dt>Benzoyl-CoA: 8.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 5.1</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 4.3</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (1733..2037)</strong>: mal
          <div class="collapser collapser-target-cds5279_18398 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds5279_18398-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 100.0%</dd>
<dd>inactive: 87.5%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds5279_18398-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 221.5</dt>

  <dd></dd><dt>inactive: 110.9</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 63.4</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 56.6</dt>

  <dd></dd><dt>Isobutyryl-CoA: 36.6</dt>

  <dd></dd><dt>Benzoyl-CoA: 25.7</dt>

  <dd></dd><dt>Propionyl-CoA: 22.4</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 19.8</dt>

  <dd></dd><dt>Acetyl-CoA: 12.7</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 11.3</dt>

  <dd></dd><dt>fatty_acid: 10.6</dt>

  <dd></dd><dt>CHC-CoA: 6.8</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (3278..3579)</strong>: mal
          <div class="collapser collapser-target-cds5279_18398 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds5279_18398-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 79.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds5279_18398-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 222.7</dt>

  <dd></dd><dt>inactive: 102.6</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 60.9</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 52.8</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 29.5</dt>

  <dd></dd><dt>Isobutyryl-CoA: 27.4</dt>

  <dd></dd><dt>Benzoyl-CoA: 25.5</dt>

  <dd></dd><dt>Propionyl-CoA: 24.5</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 20.2</dt>

  <dd></dd><dt>Acetyl-CoA: 18.1</dt>

  <dd></dd><dt>CHC-CoA: 13.0</dt>

  <dd></dd><dt>fatty_acid: 12.2</dt>

  <dd></dd><dt>trans-1,2-CPDA: 5.6</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r98c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00098_NODE_34.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-281"></div> <span class="help-tooltip" id="region_svg-help-281">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00098_NODE_34...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r98c1-svg" data-filename="c00098_NODE_34.._r98c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 31,464 nt. (total: 31,464 nt)
        
        <a class="cluster-rules-header" id="r98c1-rules-header" href="#r98c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r98c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r98c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-282"></div> <span class="help-tooltip" id="focus-panel-help-282">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r98c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r98c1-nrps_pks" data-name="r98c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r98c1-clusterblast" data-name="r98c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r98c1-knownclusterblast" data-name="r98c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r98c1-subclusterblast" data-name="r98c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r98c1-MIBiG-cluster-compare" data-name="r98c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r98c1-pfam-details" data-name="r98c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r98c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-90"></div> <span class="help-tooltip" id="nrps-domain-help-90">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r98c1-details-svg" data-filename="c00098_NODE_34.._r98c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r98c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r98c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-93"></div> <span class="help-tooltip" id="cb-general-help-93">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r98c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r98c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r98c1_1.svg">
        NZ_KZ819185 (488550-566486): Streptomyces sp. CS207 scaffold00002, whole geno... (54% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r98c1_2.svg">
        NZ_CP039123 (6294253-6369829): Streptomyces sp. SS52 chromosome, complete genome (54% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r98c1_3.svg">
        NZ_CP042278 (1583458-1663462): Streptomyces sp. WAC06273 strain WAC6273 subst... (38% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r98c1_4.svg">
        NZ_CP027297 (6501004-6580734): Streptomyces sp. SGAir0924 chromosome, complet... (32% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r98c1_5.svg">
        NZ_CP042324 (3524876-3603956): Streptomyces coelicolor A3(2) strain CFB NBC 0... (36% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r98c1_6.svg">
        NZ_LT629768 (3748808-3827608): Streptomyces sp. 2114.2 chromosome I (35% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r98c1_7.svg">
        NZ_JTIY01000002 (691244-774422): Streptomyces sp. AcH 505 scaffold00002, whol... (32% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r98c1_8.svg">
        NZ_GG657756 (4710103-4788943): Streptomyces lividans TK24 supercont1.1, whole... (35% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r98c1_9.svg">
        NZ_BEWD01000011 (217053-297488): Streptomyces rubrogriseus strain NBRC 15455,... (33% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r98c1_10.svg">
        NZ_CP026730 (2884074-2986849): Streptomyces sp. CB09001 chromosome, complete ... (21% of genes show similarity), NRPS,T1PKS,butyrolactone
      </option>
      
    </select>
    <button id= "clusterblast-r98c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r98c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r98c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-94"></div> <span class="help-tooltip" id="cb-known-help-94">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r98c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r98c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r98c1_1.svg">
        lipopeptide 8D1-1 / lipopeptide 8D1-2
      </option>
      
      <option value = "svg/knownclusterblast_r98c1_2.svg">
        CDA1b / CDA2a / CDA2b / CDA3a / CDA3b / CDA4a / CDA4b
      </option>
      
      <option value = "svg/knownclusterblast_r98c1_3.svg">
        daptomycin
      </option>
      
      <option value = "svg/knownclusterblast_r98c1_4.svg">
        feglymycin
      </option>
      
      <option value = "svg/knownclusterblast_r98c1_5.svg">
        RP-1776
      </option>
      
      <option value = "svg/knownclusterblast_r98c1_6.svg">
        WS9326
      </option>
      
      <option value = "svg/knownclusterblast_r98c1_7.svg">
        aurantimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r98c1_8.svg">
        cadaside A / cadaside B
      </option>
      
      <option value = "svg/knownclusterblast_r98c1_9.svg">
        telomycin
      </option>
      
      <option value = "svg/knownclusterblast_r98c1_10.svg">
        A54145
      </option>
      
    </select>
    <button id="knownclusterblast-r98c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r98c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r98c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-95"></div> <span class="help-tooltip" id="cb-sub-help-95">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r98c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r98c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r98c1_1.svg">
        AL939125_c1: prodigiosin 2-undecylpyrrole (25% of genes show similarity), 2-undecylpyrrole
      </option>
      
    </select>
    <button id="subclusterblast-r98c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r98c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r98c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-96"></div> <span class="help-tooltip" id="MIBiG-help-96">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r98c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r98c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r98c1"></div>
    <div id="comparison-MIBiG-r98c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.43, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
        <td>0.41</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001370.1: 10-79794">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001370/index.html#r1c1" target="_new">BGC0001370.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.93, order:0.66, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
        <td>0.41</td>
        <td>NRP</td>
        
        <td>lipopeptide 8D1-1, lipopeptide 8D1-2</td>
        
        
        <td>Streptomyces rochei</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.45, order:1.00, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.45, order:1.00, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>bacillibactin</td>
        
        
        <td>Bacillus velezensis FZB42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890.1: 1392460-1420815">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
        
        
        <td>Vibrio campbellii ATCC BAA-1116</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>hexose-palythine-serine, hexose-shinorine</td>
        
        
        <td>Heteroscytonema crispum UCFS10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401.1: 0-12882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.48, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>paenibactin</td>
        
        
        <td>Paenibacillus elgii B69</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001824.1: 39732-57269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001824/index.html#r1c1" target="_new">BGC0001824.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>xefoampeptides A-G</td>
        
        
        <td>Xenorhabdus beddingii</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r98c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.43, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001370.1: 10-79794">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001370/index.html#r1c1" target="_new">BGC0001370.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.93, order:0.66, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>NRP</td>
      <td>lipopeptide 8D1-1, lipopeptide 8D1-2</td>
      <td>Streptomyces rochei</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.45, order:1.00, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.45, order:1.00, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185.1: 3021249-3032970">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>bacillibactin</td>
      <td>Bacillus velezensis FZB42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890.1: 1392460-1420815">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
      <td>Vibrio campbellii ATCC BAA-1116</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001615.1: 35-10914">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001615/index.html#r1c1" target="_new">BGC0001615.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>hexose-palythine-serine, hexose-shinorine</td>
      <td>Heteroscytonema crispum UCFS10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401.1: 0-12882">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.48, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>paenibactin</td>
      <td>Paenibacillus elgii B69</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001824.1: 39732-57269">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001824/index.html#r1c1" target="_new">BGC0001824.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>xefoampeptides A-G</td>
      <td>Xenorhabdus beddingii</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r98c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-97"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-97">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r98c1-pfam-details-svg" data-filename="c00098_NODE_34.._r98c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r98c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r98c1-nrps_pks_products" data-name="r98c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r98c1-nrps_pks" data-name="r98c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r98c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-91"></div> <span class="help-tooltip" id="nrps-structures-help-91">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 31464:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r98c1-1" class="smiles-canvas" data-smiles="NCC(=O)NC(CC(=O)N)C(=O)NC(C(C)CC(=O))C(=O)NC(CC1=CNc2c1cccc2)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(gly - D-asn) + (3-me-glu - trp)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly,d-asn&amp;nrps2=3-me-glu,trp" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*,d-asn*&amp;nrps2=3-me-glu*,trp*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r98c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NCC(=O)NC(CC(=O)N)C(=O)NC(C(C)CC(=O))C(=O)NC(CC1=CNc2c1cccc2)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r98c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-92"></div> <span class="help-tooltip" id="nrps-monomers-help-92">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds16265_23489</span></strong>: 3-me-glu - trp
      <div class="collapser collapser-target-cds16265_23489 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=3-me-glu,trp" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=3-me-glu*,trp*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (472..879)</strong>: 3-me-glu
          <div class="collapser collapser-target-cds16265_23489 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  3-me-glu
              <div class="collapser collapser-target-cds16265_23489-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>asp, asn</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DQGKTGVGHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>3-me-glu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1546..1939)</strong>: trp
          <div class="collapser collapser-target-cds16265_23489 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  trp
              <div class="collapser collapser-target-cds16265_23489-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>phe, trp</dd>
   <dt>Single AA prediction:</dt>
   <dd>trp</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DGWAVASVCK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>trp</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds23473_31462</span></strong>: gly - asn
      <div class="collapser collapser-target-cds23473_31462 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly,asn" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*,asn*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (528..908)</strong>: gly
          <div class="collapser collapser-target-cds23473_31462 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-cds23473_31462-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILQLGLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1576..1983)</strong>: asn
          <div class="collapser collapser-target-cds23473_31462 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asn
              <div class="collapser collapser-target-cds23473_31462-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>asp, asn</dd>
   <dt>Single AA prediction:</dt>
   <dd>asn</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLTKVGEVGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r113c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00113_NODE_22.. - Region 1 - siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-283"></div> <span class="help-tooltip" id="region_svg-help-283">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00113_NODE_22...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r113c1-svg" data-filename="c00113_NODE_22.._r113c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 6,418 nt. (total: 6,418 nt)
        
        <a class="cluster-rules-header" id="r113c1-rules-header" href="#r113c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r113c1-rules">siderophore: IucA_IucC</div>
      
      <div class="region-svg-container">
       <div id='r113c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-284"></div> <span class="help-tooltip" id="focus-panel-help-284">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r113c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r113c1-clusterblast" data-name="r113c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r113c1-knownclusterblast" data-name="r113c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r113c1-subclusterblast" data-name="r113c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r113c1-MIBiG-cluster-compare" data-name="r113c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r113c1-pfam-details" data-name="r113c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r113c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-98"></div> <span class="help-tooltip" id="cb-general-help-98">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r113c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r113c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r113c1_1.svg">
        NZ_CP027297 (5967715-5977876): Streptomyces sp. SGAir0924 chromosome, complet... (57% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r113c1_2.svg">
        NZ_CP039123 (5826629-5836745): Streptomyces sp. SS52 chromosome, complete genome (57% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r113c1_3.svg">
        NZ_KZ819185 (1054307-1066185): Streptomyces sp. CS207 scaffold00002, whole ge... (57% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r113c1_4.svg">
        NZ_LIPF01000012 (57742-69674): Streptomyces sp. CB02414 scaffold12, whole gen... (50% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r113c1_5.svg">
        NZ_KN039947 (1103154-1115065): Streptomyces luteus strain TRM 45540 Scaffold2... (57% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r113c1_6.svg">
        NZ_CP019724 (2406338-2417063): Streptomyces pactum strain ACT12 chromosome, c... (50% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r113c1_7.svg">
        NZ_CP042278 (2115455-2127234): Streptomyces sp. WAC06273 strain WAC6273 subst... (57% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r113c1_8.svg">
        NZ_CP009754 (2384491-2396165): Streptomyces sp. CCM MD2014 chromosome, comple... (50% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r113c1_9.svg">
        NZ_CP012949 (5699286-5709715): Streptomyces ambofaciens strain DSM 40697 chro... (57% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r113c1_10.svg">
        NZ_KZ195574 (2315925-2389866): Streptomyces sp. CS113 scaffold00001, whole ge... (6% of genes show similarity), T2PKS,siderophore
      </option>
      
    </select>
    <button id= "clusterblast-r113c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r113c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r113c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-99"></div> <span class="help-tooltip" id="cb-known-help-99">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r113c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-100"></div> <span class="help-tooltip" id="cb-sub-help-100">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r113c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-101"></div> <span class="help-tooltip" id="MIBiG-help-101">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r113c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r113c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r113c1"></div>
    <div id="comparison-MIBiG-r113c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">siderophore</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Pantoea agglomerans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000946.1: 282-7531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000946/index.html#r1c1" target="_new">BGC0000946.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>Other</td>
        
        <td>vibrioferrin</td>
        
        
        <td>Vibrio parahaemolyticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499.1: 2267-11578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.44, order:0.57, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>aerobactin</td>
        
        
        <td>Pantoea ananatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000942.1: 1864728-1871818">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000942/index.html#r1c1" target="_new">BGC0000942.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>petrobactin</td>
        
        
        <td>Bacillus anthracis str. Ames</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001408.1: 1377589-1387887">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001408/index.html#r1c1" target="_new">BGC0001408.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>xanthoferrin</td>
        
        
        <td>Xanthomonas oryzae pv. oryzae KACC 10331</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001498.1: 17496-29587">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001498/index.html#r1c1" target="_new">BGC0001498.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:0.57, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>aerobactin</td>
        
        
        <td>Xenorhabdus szentirmaii DSM 16338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.45, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>ochrobactin</td>
        
        
        <td>Ochrobactrum anthropi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001555.1: 43-103273">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001555/index.html#r1c1" target="_new">BGC0001555.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.45, order:0.57, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>RiPP</td>
        
        <td>colicin V</td>
        
        
        <td>Escherichia coli chi7122</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000295.1: 569-12865">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000295/index.html#r1c1" target="_new">BGC0000295.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.46, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>NRP</td>
        
        <td>acinetoferrin</td>
        
        
        <td>Acinetobacter haemolyticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000947.1: 153-15744">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000947/index.html#r1c1" target="_new">BGC0000947.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.39 (id:0.42, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
         
        
        <td>0.12</td>
        <td>Other</td>
        
        <td>vibrioferrin</td>
        
        
        <td>Vibrio alginolyticus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r113c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Pantoea agglomerans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000946.1: 282-7531">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000946/index.html#r1c1" target="_new">BGC0000946.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Other</td>
      <td>vibrioferrin</td>
      <td>Vibrio parahaemolyticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499.1: 2267-11578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.44, order:0.57, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Other</td>
      <td>aerobactin</td>
      <td>Pantoea ananatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000942.1: 1864728-1871818">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000942/index.html#r1c1" target="_new">BGC0000942.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Other</td>
      <td>petrobactin</td>
      <td>Bacillus anthracis str. Ames</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001408.1: 1377589-1387887">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001408/index.html#r1c1" target="_new">BGC0001408.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>xanthoferrin</td>
      <td>Xanthomonas oryzae pv. oryzae KACC 10331</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001498.1: 17496-29587">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001498/index.html#r1c1" target="_new">BGC0001498.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:0.57, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>aerobactin</td>
      <td>Xenorhabdus szentirmaii DSM 16338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.45, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other</td>
      <td>ochrobactin</td>
      <td>Ochrobactrum anthropi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001555.1: 43-103273">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001555/index.html#r1c1" target="_new">BGC0001555.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.45, order:0.57, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>RiPP</td>
      <td>colicin V</td>
      <td>Escherichia coli chi7122</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000295.1: 569-12865">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000295/index.html#r1c1" target="_new">BGC0000295.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.46, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>NRP</td>
      <td>acinetoferrin</td>
      <td>Acinetobacter haemolyticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000947.1: 153-15744">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000947/index.html#r1c1" target="_new">BGC0000947.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.39 (id:0.42, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
      <td>0.39</td>
      <td>Other</td>
      <td>vibrioferrin</td>
      <td>Vibrio alginolyticus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r113c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-102"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-102">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r113c1-pfam-details-svg" data-filename="c00113_NODE_22.._r113c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r113c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r114c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00114_NODE_26.. - Region 1 - NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-285"></div> <span class="help-tooltip" id="region_svg-help-285">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00114_NODE_26...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r114c1-svg" data-filename="c00114_NODE_26.._r114c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 22,905 nt. (total: 22,905 nt)
        
        <a class="cluster-rules-header" id="r114c1-rules-header" href="#r114c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r114c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r114c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-286"></div> <span class="help-tooltip" id="focus-panel-help-286">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r114c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r114c1-nrps_pks" data-name="r114c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r114c1-clusterblast" data-name="r114c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r114c1-knownclusterblast" data-name="r114c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r114c1-subclusterblast" data-name="r114c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r114c1-MIBiG-cluster-compare" data-name="r114c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r114c1-pfam-details" data-name="r114c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r114c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-103"></div> <span class="help-tooltip" id="nrps-domain-help-103">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r114c1-details-svg" data-filename="c00114_NODE_26.._r114c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r114c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r114c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-106"></div> <span class="help-tooltip" id="cb-general-help-106">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r114c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r114c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r114c1_1.svg">
        NZ_CP027297 (282629-492289): Streptomyces sp. SGAir0924 chromosome, complete ... (25% of genes show similarity), NRPS,NRPS-like,T1PKS,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r114c1_2.svg">
        NZ_CP019724 (7901927-8054647): Streptomyces pactum strain ACT12 chromosome, c... (46% of genes show similarity), NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r114c1_3.svg">
        NZ_CP034587 (467666-619676): Streptomyces luteoverticillatus strain CGMCC 150... (45% of genes show similarity), NRPS-like,T1PKS,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r114c1_4.svg">
        NZ_CP030862 (2807577-2955125): Streptomyces globosus strain LZH-48 chromosome... (41% of genes show similarity), NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r114c1_5.svg">
        NZ_CP022310 (154410-242095): Streptomyces asterosporus strain DSM 41452 chrom... (39% of genes show similarity), NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r114c1_6.svg">
        NZ_ML123129 (239463-453412): Streptomyces sp. ADI98-12 scaffold004, whole gen... (31% of genes show similarity), NRPS,NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r114c1_7.svg">
        NC_020990 (6567851-6838639): Streptomyces albidoflavus, complete sequence (21% of genes show similarity), NRPS,NRPS-like,T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r114c1_8.svg">
        NZ_CM002271 (6545259-6848841): Streptomyces sp. GBA 94-10 chromosome, whole g... (15% of genes show similarity), NRPS,NRPS-like,T1PKS,T3PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r114c1_9.svg">
        NZ_CM002273 (6555444-6843909): Streptomyces sp. PVA 94-07 strain PVA 94-07 ch... (17% of genes show similarity), NRPS,NRPS-like,T1PKS,T3PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r114c1_10.svg">
        NZ_CP047147 (63289-333437): Streptomyces sp. GF20 chromosome, complete genome (19% of genes show similarity), NRPS,NRPS-like,T1PKS,lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r114c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r114c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r114c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-107"></div> <span class="help-tooltip" id="cb-known-help-107">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r114c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r114c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r114c1_1.svg">
        candicidin
      </option>
      
      <option value = "svg/knownclusterblast_r114c1_2.svg">
        levorin A3 / C06690 / FR-008-III / candicidin A / UNII-AP5PEF5W7U
      </option>
      
      <option value = "svg/knownclusterblast_r114c1_3.svg">
        67-121C
      </option>
      
      <option value = "svg/knownclusterblast_r114c1_4.svg">
        nystatin
      </option>
      
      <option value = "svg/knownclusterblast_r114c1_5.svg">
        nystatin A1
      </option>
      
      <option value = "svg/knownclusterblast_r114c1_6.svg">
        natamycin
      </option>
      
      <option value = "svg/knownclusterblast_r114c1_7.svg">
        natamycin
      </option>
      
      <option value = "svg/knownclusterblast_r114c1_8.svg">
        pimaricin
      </option>
      
      <option value = "svg/knownclusterblast_r114c1_9.svg">
        rimocidin
      </option>
      
      <option value = "svg/knownclusterblast_r114c1_10.svg">
        nystatin-like Pseudonocardia polyene
      </option>
      
    </select>
    <button id="knownclusterblast-r114c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r114c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r114c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-108"></div> <span class="help-tooltip" id="cb-sub-help-108">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r114c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-109"></div> <span class="help-tooltip" id="MIBiG-help-109">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r114c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r114c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r114c1"></div>
    <div id="comparison-MIBiG-r114c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.66</td>
        <td>Alkaloid</td>
        
        <td>kitasetaline</td>
        
        
        <td>Kitasatospora setae KM-6054</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001544.1: 576137-603864">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001544/index.html#r1c1" target="_new">BGC0001544.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.60</td>
        <td>NRP, Polyketide</td>
        
        <td>chlorflavonin</td>
        
        
        <td>Aspergillus campestris IBT 28561</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.56</td>
        <td>Polyketide</td>
        
        <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
        
        
        <td>Parastagonospora nodorum SN15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.56</td>
        <td>Terpene, Polyketide</td>
        
        <td>pyripyropene A</td>
        
        
        <td>unidentified unclassified sequences.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.55</td>
        <td>Polyketide</td>
        
        <td>depudecin</td>
        
        
        <td>Alternaria brassicicola</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000061.1: 573-137766">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000061/index.html#r1c1" target="_new">BGC0000061.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.62, order:1.00, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.62, order:1.00, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.52</td>
        <td>Polyketide</td>
        
        <td>levorin A3, C06690, FR-008-III, candicidin A, UNII-AP5PEF5W7U</td>
        
        
        <td>Streptomyces sp. FR-008</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000034.1: 573-137766">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000034/index.html#r1c1" target="_new">BGC0000034.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.62, order:1.00, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.62, order:1.00, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.52</td>
        <td>Polyketide</td>
        
        <td>candicidin</td>
        
        
        <td>Streptomyces sp. FR-008</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.51</td>
        <td>Other</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.51</td>
        <td>NRP, Polyketide</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.51</td>
        <td>Polyketide</td>
        
        <td>betaenone C, betaenone A</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r114c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Alkaloid</td>
      <td>kitasetaline</td>
      <td>Kitasatospora setae KM-6054</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001544.1: 576137-603864">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001544/index.html#r1c1" target="_new">BGC0001544.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP, Polyketide</td>
      <td>chlorflavonin</td>
      <td>Aspergillus campestris IBT 28561</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
      <td>Parastagonospora nodorum SN15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene, Polyketide</td>
      <td>pyripyropene A</td>
      <td>unidentified unclassified sequences.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>depudecin</td>
      <td>Alternaria brassicicola</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000061.1: 573-137766">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000061/index.html#r1c1" target="_new">BGC0000061.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.62, order:1.00, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>levorin A3, C06690, FR-008-III, candicidin A, UNII-AP5PEF5W7U</td>
      <td>Streptomyces sp. FR-008</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000034.1: 573-137766">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000034/index.html#r1c1" target="_new">BGC0000034.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.62, order:1.00, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>candicidin</td>
      <td>Streptomyces sp. FR-008</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Other</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP, Polyketide</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>betaenone C, betaenone A</td>
      <td>Phoma betae</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r114c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-110"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-110">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r114c1-pfam-details-svg" data-filename="c00114_NODE_26.._r114c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r114c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r114c1-nrps_pks_products" data-name="r114c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r114c1-nrps_pks" data-name="r114c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r114c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-104"></div> <span class="help-tooltip" id="nrps-structures-help-104">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 22905:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r114c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)CC(=O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X - mal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,mal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,mal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r114c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)CC(=O)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r114c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-105"></div> <span class="help-tooltip" id="nrps-monomers-help-105">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds13368_18729</span></strong>: X - mal
      <div class="collapser collapser-target-cds13368_18729 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (29..439)</strong>: X
          <div class="collapser collapser-target-cds13368_18729 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds13368_18729-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">cFQvLaVngK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pip</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>50% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (1222..1523)</strong>: mal
          <div class="collapser collapser-target-cds13368_18729 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds13368_18729-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 79.2%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds13368_18729-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 187.4</dt>

  <dd></dd><dt>inactive: 81.7</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 58.7</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 53.9</dt>

  <dd></dd><dt>Propionyl-CoA: 31.5</dt>

  <dd></dd><dt>Acetyl-CoA: 29.9</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 27.4</dt>

  <dd></dd><dt>Isobutyryl-CoA: 26.2</dt>

  <dd></dd><dt>Benzoyl-CoA: 20.7</dt>

  <dd></dd><dt>fatty_acid: 20.3</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 19.2</dt>

  <dd></dd><dt>CHC-CoA: 13.3</dt>

  <dd></dd><dt>trans-1,2-CPDA: 12.2</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 10.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r124c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00124_NODE_41.. - Region 1 - hglE-KS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-287"></div> <span class="help-tooltip" id="region_svg-help-287">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00124_NODE_41...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r124c1-svg" data-filename="c00124_NODE_41.._r124c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 4,691 nt. (total: 4,691 nt)
        
        <a class="cluster-rules-header" id="r124c1-rules-header" href="#r124c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r124c1-rules">hglE-KS: (hglE or hglD)</div>
      
      <div class="region-svg-container">
       <div id='r124c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-288"></div> <span class="help-tooltip" id="focus-panel-help-288">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r124c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r124c1-clusterblast" data-name="r124c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r124c1-knownclusterblast" data-name="r124c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r124c1-subclusterblast" data-name="r124c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r124c1-MIBiG-cluster-compare" data-name="r124c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r124c1-pfam-details" data-name="r124c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r124c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-111"></div> <span class="help-tooltip" id="cb-general-help-111">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r124c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r124c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r124c1_1.svg">
        NZ_CP033581 (898663-1139331): Streptomyces sp. ADI95-16 chromosome, complete ... (17% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r124c1_2.svg">
        NZ_CP054925 (398667-559119): Amycolatopsis sp. Hca4 chromosome, complete genome (20% of genes show similarity), NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r124c1_3.svg">
        NC_015957 (9065093-9247470): Streptomyces violaceusniger Tu 4113, complete se... (23% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r124c1_4.svg">
        NZ_CP050975 (85585-252422): Streptomyces sp. RPA4-2 chromosome, complete genome (10% of genes show similarity), NRPS,PKS-like,T1PKS,ectoine
      </option>
      
      <option value = "svg/clusterblast_r124c1_5.svg">
        NZ_CP042278 (&lt;1-9396): Streptomyces sp. WAC06273 strain WAC6273 substr. delta... (66% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r124c1_6.svg">
        NZ_CP029823 (7075644-7258783): Streptomyces malaysiensis strain DSM 4137 chro... (19% of genes show similarity), T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r124c1_7.svg">
        NC_015957 (4515242-4670136): Streptomyces violaceusniger Tu 4113, complete se... (21% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r124c1_8.svg">
        NZ_KB912942 (2125778-2296948): Amycolatopsis benzoatilytica AK 16/65 AmybeDRA... (10% of genes show similarity), NRPS-like,T1PKS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r124c1_9.svg">
        NZ_HG964446 (2237849-2326583): Mycobacterium triplex strain DSM 44626, whole ... (8% of genes show similarity), NRPS,T1PKS,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r124c1_10.svg">
        NZ_JTIY01000001 (5144507-5254674): Streptomyces sp. AcH 505 scaffold00001, wh... (15% of genes show similarity), NRPS,T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r124c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r124c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r124c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-112"></div> <span class="help-tooltip" id="cb-known-help-112">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r124c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-113"></div> <span class="help-tooltip" id="cb-sub-help-113">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r124c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-114"></div> <span class="help-tooltip" id="MIBiG-help-114">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r124c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r124c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r124c1"></div>
    <div id="comparison-MIBiG-r124c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">hglE-KS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000863.1: 1224-16560">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000863/index.html#r1c1" target="_new">BGC0000863.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:1.00, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>eicosapentaenoic acid</td>
        
        
        <td>Pseudoalteromonas sp. DS-12</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>esperamicin</td>
        
        
        <td>Actinomadura verrucosospora</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000259.1: 167-24834">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000259/index.html#r1c1" target="_new">BGC0000259.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>prodigiosin</td>
        
        
        <td>Serratia marcescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000044.1: 1916098-1937355">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000044/index.html#r1c1" target="_new">BGC0000044.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.43, order:1.00, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>dawenol</td>
        
        
        <td>Stigmatella aurantiaca DW4/3-1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000258.1: 2-24341">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000258/index.html#r1c1" target="_new">BGC0000258.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>prodigiosin</td>
        
        
        <td>Serratia sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000260.1: 853-35060">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000260/index.html#r1c1" target="_new">BGC0000260.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>prodigiosin</td>
        
        
        <td>Hahella chejuensis KCTC 2396</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219.1: 1404646-1428146">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>shanorellin</td>
        
        
        <td>Chaetomium globosum CBS 148.51</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400.1: 371433-386839">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.45, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>citreoviridin</td>
        
        
        <td>Aspergillus terreus NIH2624</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000886.1: 2774-14963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000886/index.html#r1c1" target="_new">BGC0000886.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>allylmalonyl-CoA</td>
        
        
        <td>Streptomyces tsukubensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275.1: 1063-6614">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Polyketide</td>
        
        <td>6-methylsalicyclic acid</td>
        
        
        <td>Glarea lozoyensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r124c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000863.1: 1224-16560">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000863/index.html#r1c1" target="_new">BGC0000863.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:1.00, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>eicosapentaenoic acid</td>
      <td>Pseudoalteromonas sp. DS-12</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Polyketide</td>
      <td>esperamicin</td>
      <td>Actinomadura verrucosospora</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000259.1: 167-24834">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000259/index.html#r1c1" target="_new">BGC0000259.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Polyketide</td>
      <td>prodigiosin</td>
      <td>Serratia marcescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000044.1: 1916098-1937355">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000044/index.html#r1c1" target="_new">BGC0000044.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.43, order:1.00, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Polyketide</td>
      <td>dawenol</td>
      <td>Stigmatella aurantiaca DW4/3-1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000258.1: 2-24341">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000258/index.html#r1c1" target="_new">BGC0000258.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Polyketide</td>
      <td>prodigiosin</td>
      <td>Serratia sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000260.1: 853-35060">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000260/index.html#r1c1" target="_new">BGC0000260.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Polyketide</td>
      <td>prodigiosin</td>
      <td>Hahella chejuensis KCTC 2396</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219.1: 1404646-1428146">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Polyketide</td>
      <td>shanorellin</td>
      <td>Chaetomium globosum CBS 148.51</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400.1: 371433-386839">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.45, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Polyketide</td>
      <td>citreoviridin</td>
      <td>Aspergillus terreus NIH2624</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000886.1: 2774-14963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000886/index.html#r1c1" target="_new">BGC0000886.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other</td>
      <td>allylmalonyl-CoA</td>
      <td>Streptomyces tsukubensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275.1: 1063-6614">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Glarea lozoyensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r124c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-115"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-115">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r124c1-pfam-details-svg" data-filename="c00124_NODE_41.._r124c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r124c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r126c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00126_NODE_11.. - Region 1 - indole <div class="help-container"> <div class="help-icon" data-id="region_svg-help-289"></div> <span class="help-tooltip" id="region_svg-help-289">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00126_NODE_11...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r126c1-svg" data-filename="c00126_NODE_11.._r126c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 69,237 - 90,358 nt. (total: 21,122 nt)
        
        <a class="cluster-rules-header" id="r126c1-rules-header" href="#r126c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r126c1-rules">indole: (indsynth or dmat or indole_PTase)</div>
      
      <div class="region-svg-container">
       <div id='r126c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-290"></div> <span class="help-tooltip" id="focus-panel-help-290">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r126c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r126c1-clusterblast" data-name="r126c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r126c1-knownclusterblast" data-name="r126c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r126c1-subclusterblast" data-name="r126c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r126c1-MIBiG-cluster-compare" data-name="r126c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r126c1-pfam-details" data-name="r126c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r126c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-116"></div> <span class="help-tooltip" id="cb-general-help-116">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r126c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r126c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r126c1_1.svg">
        NZ_CP039123 (722055-743177): Streptomyces sp. SS52 chromosome, complete genome (100% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r126c1_2.svg">
        NZ_KZ819184 (3677716-3698086): Streptomyces sp. CS207 scaffold00001, whole ge... (100% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r126c1_3.svg">
        NZ_CP027297 (800059-821181): Streptomyces sp. SGAir0924 chromosome, complete ... (100% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r126c1_4.svg">
        NZ_CP042278 (7157989-7179112): Streptomyces sp. WAC06273 strain WAC6273 subst... (100% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r126c1_5.svg">
        NZ_CP036534 (467587-488715): Streptomyces sp. VN1 chromosome, complete genome (90% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r126c1_6.svg">
        NZ_CP016795 (367310-388438): Streptomyces olivaceus strain KLBMP 5084 chromos... (90% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r126c1_7.svg">
        NZ_SOAV01000016 (50424-71579): Streptomyces sp. BK208 Ga0307686 116, whole ge... (90% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r126c1_8.svg">
        NZ_JAAIFS010000005 (130509-151637): Streptomyces tendae strain VITAKN NODE 5 ... (90% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r126c1_9.svg">
        NZ_RSEG01000018 (86367-107495): Streptomyces tendae strain UTMC 3329 Scaffold... (90% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r126c1_10.svg">
        NZ_KZ195574 (8224620-8245748): Streptomyces sp. CS113 scaffold00001, whole ge... (95% of genes show similarity), indole
      </option>
      
    </select>
    <button id= "clusterblast-r126c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r126c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r126c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-117"></div> <span class="help-tooltip" id="cb-known-help-117">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r126c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r126c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r126c1_1.svg">
        5-isoprenylindole-3-carboxylate β-D-glycosyl ester
      </option>
      
      <option value = "svg/knownclusterblast_r126c1_2.svg">
        JBIR-06
      </option>
      
      <option value = "svg/knownclusterblast_r126c1_3.svg">
        antimycin
      </option>
      
      <option value = "svg/knownclusterblast_r126c1_4.svg">
        splenocin
      </option>
      
      <option value = "svg/knownclusterblast_r126c1_5.svg">
        neoantimycin
      </option>
      
      <option value = "svg/knownclusterblast_r126c1_6.svg">
        neoantimycin
      </option>
      
      <option value = "svg/knownclusterblast_r126c1_7.svg">
        ravidomycin
      </option>
      
    </select>
    <button id="knownclusterblast-r126c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r126c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r126c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-118"></div> <span class="help-tooltip" id="cb-sub-help-118">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r126c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-119"></div> <span class="help-tooltip" id="MIBiG-help-119">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r126c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r126c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r126c1"></div>
    <div id="comparison-MIBiG-r126c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">indole</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294.1: 26867-35886">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Other</td>
        
        <td>7-prenylisatin</td>
        
        
        <td>Streptomyces sp. MBT28</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001191.1: 21327-37853">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001191/index.html#r1c1" target="_new">BGC0001191.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>A-102395</td>
        
        
        <td>Amycolatopsis sp. SANK 60206</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.56, order:0.35, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>
        
        
        <td>Streptomyces sp. RM-5-8</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>venezuelin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001525.1: 0-16392">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001525/index.html#r1c1" target="_new">BGC0001525.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>bartoloside 2, bartoloside 3, bartoloside 4</td>
        
        
        <td>Synechocystis salina LEGE 06155</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001526.1: 0-14417">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001526/index.html#r1c1" target="_new">BGC0001526.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>bartolosides E, bartolosides F, bartolosides G, bartolosides H, bartolosides I, bartolosides J, bartolosides K</td>
        
        
        <td>Synechocystis salina LEGE 06099</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000706.1: 350-14906">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000706/index.html#r1c1" target="_new">BGC0000706.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001551.1: 7073794-7076321">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001551/index.html#r1c1" target="_new">BGC0001551.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>citrulassin E</td>
        
        
        <td>Streptomyces glaucescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
        
        
        <td>Actinomadura namibiensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r126c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294.1: 26867-35886">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>7-prenylisatin</td>
      <td>Streptomyces sp. MBT28</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001191.1: 21327-37853">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001191/index.html#r1c1" target="_new">BGC0001191.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Other</td>
      <td>A-102395</td>
      <td>Amycolatopsis sp. SANK 60206</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.56, order:0.35, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>
      <td>Streptomyces sp. RM-5-8</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>venezuelin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001525.1: 0-16392">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001525/index.html#r1c1" target="_new">BGC0001525.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>bartoloside 2, bartoloside 3, bartoloside 4</td>
      <td>Synechocystis salina LEGE 06155</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001526.1: 0-14417">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001526/index.html#r1c1" target="_new">BGC0001526.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>bartolosides E, bartolosides F, bartolosides G, bartolosides H, bartolosides I, bartolosides J, bartolosides K</td>
      <td>Synechocystis salina LEGE 06099</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000706.1: 350-14906">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000706/index.html#r1c1" target="_new">BGC0000706.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001551.1: 7073794-7076321">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001551/index.html#r1c1" target="_new">BGC0001551.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>RiPP</td>
      <td>citrulassin E</td>
      <td>Streptomyces glaucescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>RiPP</td>
      <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
      <td>Actinomadura namibiensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r126c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-120"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-120">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r126c1-pfam-details-svg" data-filename="c00126_NODE_11.._r126c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r126c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r133c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00133_NODE_44.. - Region 1 - T2PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-291"></div> <span class="help-tooltip" id="region_svg-help-291">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00133_NODE_44...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r133c1-svg" data-filename="c00133_NODE_44.._r133c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 15,969 nt. (total: 15,969 nt)
        
        <a class="cluster-rules-header" id="r133c1-rules-header" href="#r133c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r133c1-rules">T2PKS: (t2ks and t2clf)</div>
      
      <div class="region-svg-container">
       <div id='r133c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-292"></div> <span class="help-tooltip" id="focus-panel-help-292">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r133c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r133c1-nrps_pks" data-name="r133c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r133c1-clusterblast" data-name="r133c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r133c1-knownclusterblast" data-name="r133c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r133c1-subclusterblast" data-name="r133c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r133c1-MIBiG-cluster-compare" data-name="r133c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r133c1-pfam-details" data-name="r133c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r133c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-121"></div> <span class="help-tooltip" id="nrps-domain-help-121">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r133c1-details-svg" data-filename="c00133_NODE_44.._r133c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r133c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r133c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-122"></div> <span class="help-tooltip" id="cb-general-help-122">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r133c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r133c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r133c1_1.svg">
        NZ_KZ819184 (731930-800153): Streptomyces sp. CS207 scaffold00001, whole geno... (24% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r133c1_2.svg">
        NZ_CP039123 (5308335-5380932): Streptomyces sp. SS52 chromosome, complete genome (28% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r133c1_3.svg">
        NZ_CP042278 (2571712-2644309): Streptomyces sp. WAC06273 strain WAC6273 subst... (23% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r133c1_4.svg">
        NZ_KN039947 (1554090-1626639): Streptomyces luteus strain TRM 45540 Scaffold2... (24% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r133c1_5.svg">
        NZ_CP027297 (5389991-5462588): Streptomyces sp. SGAir0924 chromosome, complet... (24% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r133c1_6.svg">
        NZ_LIPF01000002 (65767-138313): Streptomyces sp. CB02414 scaffold2, whole gen... (22% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r133c1_7.svg">
        NZ_CP019724 (2841941-3025130): Streptomyces pactum strain ACT12 chromosome, c... (14% of genes show similarity), T1PKS,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r133c1_8.svg">
        NZ_CP034353 (2714569-2787106): Streptomyces sp. KPB2 chromosome, complete genome (23% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r133c1_9.svg">
        NZ_CP054920 (2707128-2779665): Streptomyces sp. NA03103 chromosome, complete ... (21% of genes show similarity), T2PKS
      </option>
      
      <option value = "svg/clusterblast_r133c1_10.svg">
        NZ_SOPG01000007 (264961-337498): Streptomyces sp. H23 NODE 7 length 422791 co... (23% of genes show similarity), T2PKS
      </option>
      
    </select>
    <button id= "clusterblast-r133c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r133c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r133c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-123"></div> <span class="help-tooltip" id="cb-known-help-123">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r133c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r133c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r133c1_1.svg">
        spore pigment
      </option>
      
      <option value = "svg/knownclusterblast_r133c1_2.svg">
        curamycin
      </option>
      
      <option value = "svg/knownclusterblast_r133c1_3.svg">
        spore pigment
      </option>
      
      <option value = "svg/knownclusterblast_r133c1_4.svg">
        xantholipin
      </option>
      
      <option value = "svg/knownclusterblast_r133c1_5.svg">
        WS79089A / hexaricin B / hexaricin C
      </option>
      
      <option value = "svg/knownclusterblast_r133c1_6.svg">
        collinomycin
      </option>
      
      <option value = "svg/knownclusterblast_r133c1_7.svg">
        griseorhodin A
      </option>
      
      <option value = "svg/knownclusterblast_r133c1_8.svg">
        allocyclinone
      </option>
      
      <option value = "svg/knownclusterblast_r133c1_9.svg">
        frankiamicin
      </option>
      
      <option value = "svg/knownclusterblast_r133c1_10.svg">
        pradimicin-A
      </option>
      
    </select>
    <button id="knownclusterblast-r133c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r133c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r133c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-124"></div> <span class="help-tooltip" id="cb-sub-help-124">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r133c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r133c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r133c1_1.svg">
        AF324838_c1: simocyclinone t2pks (50% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r133c1_2.svg">
        AF080235_c1: landomycin t2pks (50% of genes show similarity), t2pks
      </option>
      
      <option value = "svg/subclusterblast_r133c1_3.svg">
        FJ483966_c1: polyketomycin t2pks (25% of genes show similarity), t2pks
      </option>
      
    </select>
    <button id="subclusterblast-r133c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r133c1-svg"></div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r133c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-126"></div> <span class="help-tooltip" id="MIBiG-help-126">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r133c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r133c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r133c1"></div>
    <div id="comparison-MIBiG-r133c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T2PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.77 (id:0.57, order:1.00, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
         
        
        <td>0.47</td>
        <td>Polyketide</td>
        
        <td>curamycin</td>
        
        
        <td>Streptomyces cyaneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.73 (id:0.57, order:0.73, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
         
        
        <td>0.41</td>
        <td>Polyketide</td>
        
        <td>spore pigment</td>
        
        
        <td>Streptomyces collinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.51, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Polyketide</td>
        
        <td>naphtocyclinone</td>
        
        
        <td>Streptomyces arenae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000271.1: 0-11118">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000271/index.html#r1c1" target="_new">BGC0000271.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.57, order:0.63, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>spore pigment</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.50, order:0.51, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Polyketide</td>
        
        <td>griseusin</td>
        
        
        <td>Streptomyces griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.60, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.50, order:0.48, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Polyketide</td>
        
        <td>granaticin</td>
        
        
        <td>Streptomyces violaceoruber</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.52, order:0.48, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>jadomycin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001513.1: 120-11698">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001513/index.html#r1c1" target="_new">BGC0001513.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.54, order:0.44, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Polyketide</td>
        
        <td>anthrabenzoxocinone</td>
        
        
        <td>Actinobacteria bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.52, order:0.43, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Polyketide</td>
        
        <td>frankiamicin</td>
        
        
        <td>Frankia sp. EAN1pec</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r133c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.77 (id:0.57, order:1.00, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
      <td>0.77</td>
      <td>Polyketide</td>
      <td>curamycin</td>
      <td>Streptomyces cyaneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.73 (id:0.57, order:0.73, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
      <td>0.73</td>
      <td>Polyketide</td>
      <td>spore pigment</td>
      <td>Streptomyces collinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.51, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>naphtocyclinone</td>
      <td>Streptomyces arenae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000271.1: 0-11118">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000271/index.html#r1c1" target="_new">BGC0000271.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.57, order:0.63, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>spore pigment</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231.1: 189-5010">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.50, order:0.51, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>griseusin</td>
      <td>Streptomyces griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.60, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000228.1: 17-5796">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000228/index.html#r1c1" target="_new">BGC0000228.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.50, order:0.48, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>granaticin</td>
      <td>Streptomyces violaceoruber</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234.1: 152-6978">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.52, order:0.48, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>jadomycin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001513.1: 120-11698">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001513/index.html#r1c1" target="_new">BGC0001513.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.54, order:0.44, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>anthrabenzoxocinone</td>
      <td>Actinobacteria bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197.1: 2838707-2851386">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.52, order:0.43, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>frankiamicin</td>
      <td>Frankia sp. EAN1pec</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r133c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-127"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-127">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r133c1-pfam-details-svg" data-filename="c00133_NODE_44.._r133c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r133c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
      <div class="sidepanel-details-header r133c1-t2pks" data-name="r133c1-t2pks">
        Type II PKS
      </div>
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    <div class="sidepanel-details-section r133c1-t2pks">
      <div class="more-details">
    <div class="heading">
      <span>Type II PKS gene cluster prediction details</span>
      <div class="help-container"> <div class="help-icon" data-id="t2pks-help-125"></div> <span class="help-tooltip" id="t2pks-help-125">Predictions of starter units, elongations, product classes, and potential molecular weights for type II PKS clusters.<br>More detailed information is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/t2pks' target='_blank'>here</a>.</span></div>
    </div>
	
	
	<dl class="prediction-text">
        <dt><strong>Cluster section:</strong></dt>
        <dd>0 to 15969</dd>
		
		<dt><strong>Starter unit(s):</strong></dt>
		<dd>
				acetyl-CoA (Score: 0.0; E-value: 0)<br>
			
	    <dd>
		</tr>
		
		
		
		<dt><strong>Product class(es):</strong></td>
        <dd>
				
				pentangular polyphenol<br>
				
				tetracenomycin<br>
				
		</dd>
		
		
	</dl>
	<dl class="prediction-text">
		
		<dt><strong>cds3790_4258</strong></dt>
			
			<dd>
				CYC
				
					C5-C18
				
				(Score: 216.1; E-value: 2.8e-67)<br>
			</dd>
			
		
		<dt><strong>cds4254_5526</strong></dt>
			
			<dd>
				KS
				
				(Score: 706.5; E-value: 2.4e-215)<br>
			</dd>
			
		
		<dt><strong>cds5522_6851</strong></dt>
			
			<dd>
				CLF
				
				(Score: 548.8; E-value: 1.6e-167)<br>
			</dd>
			
		
		<dt><strong>cds6942_7212</strong></dt>
			
			<dd>
				ACP
				
				(Score: 73.9; E-value: 1.8e-23)<br>
			</dd>
			
		
		<dt><strong>cds7213_7693</strong></dt>
			
			<dd>
				CYC
				
					C9-C14
				
				(Score: 258.3; E-value: 4.3e-80)<br>
			</dd>
			
		
		<dt><strong>cds7753_8089</strong></dt>
			
			<dd>
				CYC
				
					C4-C21/C2-C23|C2-C19
				
				(Score: 153.4; E-value: 4.1e-48)<br>
			</dd>
			
		
	</dl>
    
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r147c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00147_NODE_17.. - Region 1 - hglE-KS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-293"></div> <span class="help-tooltip" id="region_svg-help-293">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00147_NODE_17...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r147c1-svg" data-filename="c00147_NODE_17.._r147c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 11,445 nt. (total: 11,445 nt)
        
        <a class="cluster-rules-header" id="r147c1-rules-header" href="#r147c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r147c1-rules">hglE-KS: (hglE or hglD)</div>
      
      <div class="region-svg-container">
       <div id='r147c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-294"></div> <span class="help-tooltip" id="focus-panel-help-294">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r147c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r147c1-clusterblast" data-name="r147c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r147c1-knownclusterblast" data-name="r147c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r147c1-subclusterblast" data-name="r147c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r147c1-MIBiG-cluster-compare" data-name="r147c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r147c1-pfam-details" data-name="r147c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r147c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-128"></div> <span class="help-tooltip" id="cb-general-help-128">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r147c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r147c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r147c1_1.svg">
        NZ_CP027297 (282629-492289): Streptomyces sp. SGAir0924 chromosome, complete ... (17% of genes show similarity), NRPS,NRPS-like,T1PKS,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r147c1_2.svg">
        NZ_CP039123 (260692-377530): Streptomyces sp. SS52 chromosome, complete genome (21% of genes show similarity), T1PKS,hglE-KS,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r147c1_3.svg">
        NZ_KB898996 (196360-249325): Streptomyces sp. CNB091 D581DRAFT scaffold00007.... (14% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r147c1_4.svg">
        NZ_MQUH01000030 (148526-201094): Streptomyces sp. M1013 M1013 C30, whole geno... (10% of genes show similarity), hglE-KS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r147c1_5.svg">
        NZ_KZ195574 (8607713-8660285): Streptomyces sp. CS113 scaffold00001, whole ge... (10% of genes show similarity), hglE-KS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r147c1_6.svg">
        NZ_NNBK01000015 (132360-192089): Streptomyces sp. CB01373 scaffold15, whole g... (9% of genes show similarity), NRPS-like,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r147c1_7.svg">
        NZ_CP029342 (2295889-2349406): Streptomyces sp. SM18 chromosome, complete genome (12% of genes show similarity), T1PKS,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r147c1_8.svg">
        NZ_PTJS01000001 (7702304-7755051): Streptomyces sp. QL37 scaffold1 size877290... (13% of genes show similarity), hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r147c1_9.svg">
        NZ_LIPG01000007 (406620-459428): Streptomyces sp. CB02058 scaffold7, whole ge... (10% of genes show similarity), T1PKS,hglE-KS
      </option>
      
      <option value = "svg/clusterblast_r147c1_10.svg">
        NZ_FMYR01000006 (292185-345700): Streptomyces sp. AmelKG-A3, whole genome sho... (11% of genes show similarity), T1PKS,hglE-KS
      </option>
      
    </select>
    <button id= "clusterblast-r147c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r147c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r147c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-129"></div> <span class="help-tooltip" id="cb-known-help-129">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r147c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r147c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r147c1_1.svg">
        2&#39;-chloropentostatin / 2&#39;-amino-2&#39;-deoxyadenosine
      </option>
      
      <option value = "svg/knownclusterblast_r147c1_2.svg">
        himastatin
      </option>
      
      <option value = "svg/knownclusterblast_r147c1_3.svg">
        phenalinolactone A
      </option>
      
      <option value = "svg/knownclusterblast_r147c1_4.svg">
        tiancilactone
      </option>
      
    </select>
    <button id="knownclusterblast-r147c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r147c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r147c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-130"></div> <span class="help-tooltip" id="cb-sub-help-130">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r147c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-131"></div> <span class="help-tooltip" id="MIBiG-help-131">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r147c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r147c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r147c1"></div>
    <div id="comparison-MIBiG-r147c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">hglE-KS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>raimonol</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Polyketide</td>
        
        <td>betaenone C, betaenone A</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Polyketide</td>
        
        <td>betaenone A, betaenone B, betaenone C</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Polyketide</td>
        
        <td>esperamicin</td>
        
        
        <td>Actinomadura verrucosospora</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>cyslabdan</td>
        
        
        <td>Streptomyces cyslabdanicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>albaflavenone</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Terpene, Polyketide</td>
        
        <td>pyripyropene A</td>
        
        
        <td>unidentified unclassified sequences.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>viguiepinol</td>
        
        
        <td>Streptomyces sp. KO-3988</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.22</td>
        <td>Polyketide</td>
        
        <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
        
        
        <td>Parastagonospora nodorum SN15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Terpene</td>
        
        <td>(+)-T-muurolol</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r147c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene</td>
      <td>raimonol</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>betaenone C, betaenone A</td>
      <td>Phoma betae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>betaenone A, betaenone B, betaenone C</td>
      <td>Phoma betae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056.1: 2-6689">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>esperamicin</td>
      <td>Actinomadura verrucosospora</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene</td>
      <td>cyslabdan</td>
      <td>Streptomyces cyslabdanicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>albaflavenone</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068.1: 0-35628">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene, Polyketide</td>
      <td>pyripyropene A</td>
      <td>unidentified unclassified sequences.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Polyketide</td>
      <td>viguiepinol</td>
      <td>Streptomyces sp. KO-3988</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Polyketide</td>
      <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
      <td>Parastagonospora nodorum SN15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Terpene</td>
      <td>(+)-T-muurolol</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r147c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-132"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-132">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r147c1-pfam-details-svg" data-filename="c00147_NODE_17.._r147c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r147c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r152c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00152_NODE_11.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-295"></div> <span class="help-tooltip" id="region_svg-help-295">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00152_NODE_11...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r152c1-svg" data-filename="c00152_NODE_11.._r152c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 26,700 nt. (total: 26,700 nt)
        
        <a class="cluster-rules-header" id="r152c1-rules-header" href="#r152c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r152c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r152c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-296"></div> <span class="help-tooltip" id="focus-panel-help-296">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r152c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r152c1-nrps_pks" data-name="r152c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r152c1-clusterblast" data-name="r152c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r152c1-knownclusterblast" data-name="r152c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r152c1-subclusterblast" data-name="r152c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r152c1-MIBiG-cluster-compare" data-name="r152c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r152c1-pfam-details" data-name="r152c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r152c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-133"></div> <span class="help-tooltip" id="nrps-domain-help-133">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r152c1-details-svg" data-filename="c00152_NODE_11.._r152c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r152c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r152c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-136"></div> <span class="help-tooltip" id="cb-general-help-136">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r152c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r152c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r152c1_1.svg">
        NZ_CP039123 (7658917-7709431): Streptomyces sp. SS52 chromosome, complete genome (47% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r152c1_2.svg">
        NZ_LIPF01000004 (95222-146124): Streptomyces sp. CB02414 scaffold4, whole gen... (37% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r152c1_3.svg">
        NZ_CP036534 (7898916-7949836): Streptomyces sp. VN1 chromosome, complete genome (42% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r152c1_4.svg">
        NZ_CP016795 (7767318-7818238): Streptomyces olivaceus strain KLBMP 5084 chrom... (42% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r152c1_5.svg">
        NZ_CP012949 (7365607-7435627): Streptomyces ambofaciens strain DSM 40697 chro... (24% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r152c1_6.svg">
        NZ_CP012382 (7565298-7616302): Streptomyces ambofaciens ATCC 23877 chromosome... (37% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r152c1_7.svg">
        NZ_CP015866 (6874608-6925528): Streptomyces parvulus strain 2297 chromosome, ... (39% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r152c1_8.svg">
        NZ_CP019724 (355288-475720): Streptomyces pactum strain ACT12 chromosome, com... (14% of genes show similarity), NRPS,T1PKS,lanthidin
      </option>
      
      <option value = "svg/clusterblast_r152c1_9.svg">
        NZ_CP009754 (208580-258556): Streptomyces sp. CCM MD2014 chromosome, complete... (37% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r152c1_10.svg">
        NZ_BBOK01000008 (93273-142530): Streptomyces sp. NBRC 110030, whole genome sh... (37% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r152c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r152c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r152c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-137"></div> <span class="help-tooltip" id="cb-known-help-137">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r152c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r152c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r152c1_1.svg">
        coelichelin
      </option>
      
      <option value = "svg/knownclusterblast_r152c1_2.svg">
        gobichelin A / gobichelin B
      </option>
      
      <option value = "svg/knownclusterblast_r152c1_3.svg">
        scabichelin
      </option>
      
      <option value = "svg/knownclusterblast_r152c1_4.svg">
        albachelin
      </option>
      
      <option value = "svg/knownclusterblast_r152c1_5.svg">
        salinichelins
      </option>
      
      <option value = "svg/knownclusterblast_r152c1_6.svg">
        qinichelins
      </option>
      
      <option value = "svg/knownclusterblast_r152c1_7.svg">
        herboxidiene
      </option>
      
      <option value = "svg/knownclusterblast_r152c1_8.svg">
        ficellomycin
      </option>
      
    </select>
    <button id="knownclusterblast-r152c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r152c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r152c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-138"></div> <span class="help-tooltip" id="cb-sub-help-138">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r152c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-139"></div> <span class="help-tooltip" id="MIBiG-help-139">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r152c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r152c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r152c1"></div>
    <div id="comparison-MIBiG-r152c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000325.1: 510822-533448">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000325/index.html#r1c1" target="_new">BGC0000325.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.66, order:0.74, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>NRP</td>
        
        <td>coelichelin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>vicibactin</td>
        
        
        <td>Rhizobium etli CFN 42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342.1: 1020-10416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>enniatin</td>
        
        
        <td>Fusarium equiseti</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873.1: 1647607-1656213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>pyrrolizixenamide A</td>
        
        
        <td>Xenorhabdus szentirmaii DSM 16338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000423.1: 9541286-9571306">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000423/index.html#r1c1" target="_new">BGC0000423.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.50, order:0.52, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>scabichelin</td>
        
        
        <td>Streptomyces scabiei 87.22</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
        
        
        <td>Penicillium rubens Wisconsin 54-1255</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000321.1: 1657-43703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000321/index.html#r1c1" target="_new">BGC0000321.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>chaetocin</td>
        
        
        <td>Collariella virescens</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r152c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000325.1: 510822-533448">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000325/index.html#r1c1" target="_new">BGC0000325.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.66, order:0.74, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>NRP</td>
      <td>coelichelin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>vicibactin</td>
      <td>Rhizobium etli CFN 42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342.1: 1020-10416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>enniatin</td>
      <td>Fusarium equiseti</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873.1: 1647607-1656213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>pyrrolizixenamide A</td>
      <td>Xenorhabdus szentirmaii DSM 16338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000423.1: 9541286-9571306">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000423/index.html#r1c1" target="_new">BGC0000423.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.50, order:0.52, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>scabichelin</td>
      <td>Streptomyces scabiei 87.22</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
      <td>Penicillium rubens Wisconsin 54-1255</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000321.1: 1657-43703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000321/index.html#r1c1" target="_new">BGC0000321.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>chaetocin</td>
      <td>Collariella virescens</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r152c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-140"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-140">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r152c1-pfam-details-svg" data-filename="c00152_NODE_11.._r152c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r152c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r152c1-nrps_pks_products" data-name="r152c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r152c1-nrps_pks" data-name="r152c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r152c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-134"></div> <span class="help-tooltip" id="nrps-structures-help-134">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 26700:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r152c1-1" class="smiles-canvas" data-smiles="NC(CCCN)C(=O)NC(C(O)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-orn - thr)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-orn,thr" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-orn*,thr*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r152c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CCCN)C(=O)NC(C(O)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r152c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-135"></div> <span class="help-tooltip" id="nrps-monomers-help-135">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds1_6700</span></strong>: orn - thr
      <div class="collapser collapser-target-cds1_6700 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn,thr" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=orn*,thr*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (144..542)</strong>: orn
          <div class="collapser collapser-target-cds1_6700 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn
              <div class="collapser collapser-target-cds1_6700-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>orn, horn</dd>
   <dt>Single AA prediction:</dt>
   <dd>orn</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DINYWGGIGK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1625..2042)</strong>: thr
          <div class="collapser collapser-target-cds1_6700 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds1_6700-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNIGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r162c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00162_NODE_13.. - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-297"></div> <span class="help-tooltip" id="region_svg-help-297">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00162_NODE_13...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r162c1-svg" data-filename="c00162_NODE_13.._r162c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 5,565 - 11,354 nt. (total: 5,790 nt)
        
        <a class="cluster-rules-header" id="r162c1-rules-header" href="#r162c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r162c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193)</div>
      
      <div class="region-svg-container">
       <div id='r162c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-298"></div> <span class="help-tooltip" id="focus-panel-help-298">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r162c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r162c1-clusterblast" data-name="r162c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r162c1-knownclusterblast" data-name="r162c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r162c1-subclusterblast" data-name="r162c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r162c1-MIBiG-cluster-compare" data-name="r162c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r162c1-pfam-details" data-name="r162c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r162c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-141"></div> <span class="help-tooltip" id="cb-general-help-141">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r162c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r162c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r162c1_1.svg">
        NZ_CP027297 (6235677-6246081): Streptomyces sp. SGAir0924 chromosome, complet... (60% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r162c1_2.svg">
        NZ_KZ819185 (768165-779488): Streptomyces sp. CS207 scaffold00002, whole geno... (60% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r162c1_3.svg">
        NZ_CP039123 (6087208-6097618): Streptomyces sp. SS52 chromosome, complete genome (60% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r162c1_4.svg">
        NZ_CP012382 (6235410-6245830): Streptomyces ambofaciens ATCC 23877 chromosome... (60% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r162c1_5.svg">
        NZ_CP012949 (6106045-6116468): Streptomyces ambofaciens strain DSM 40697 chro... (60% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r162c1_6.svg">
        NZ_KN050743 (103509-153524): Streptomyces mirabilis strain OV308 EW60DRAFT sc... (11% of genes show similarity), NRPS-like,T1PKS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r162c1_7.svg">
        NZ_LIPF01000013 (9437-20775): Streptomyces sp. CB02414 scaffold13, whole geno... (41% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r162c1_8.svg">
        NZ_CP019724 (2035753-2046113): Streptomyces pactum strain ACT12 chromosome, c... (45% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r162c1_9.svg">
        NZ_CP015866 (5626981-5638028): Streptomyces parvulus strain 2297 chromosome, ... (45% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r162c1_10.svg">
        NZ_CP042278 (1875394-1886717): Streptomyces sp. WAC06273 strain WAC6273 subst... (50% of genes show similarity), bacteriocin
      </option>
      
    </select>
    <button id= "clusterblast-r162c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r162c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r162c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-142"></div> <span class="help-tooltip" id="cb-known-help-142">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r162c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-143"></div> <span class="help-tooltip" id="cb-sub-help-143">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r162c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-144"></div> <span class="help-tooltip" id="MIBiG-help-144">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r162c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r162c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r162c1"></div>
    <div id="comparison-MIBiG-r162c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">RiPP-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001796.1: 4751364-4791946">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001796/index.html#r1c1" target="_new">BGC0001796.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
         
        
        <td>0.08</td>
        <td>NRP</td>
        
        <td>tauramamide</td>
        
        
        <td>Brevibacillus laterosporus DSM 25</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r162c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001796.1: 4751364-4791946">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001796/index.html#r1c1" target="_new">BGC0001796.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
      <td>0.30</td>
      <td>NRP</td>
      <td>tauramamide</td>
      <td>Brevibacillus laterosporus DSM 25</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r162c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-145"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-145">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r162c1-pfam-details-svg" data-filename="c00162_NODE_13.._r162c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r162c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r205c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00205_NODE_74.. - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-299"></div> <span class="help-tooltip" id="region_svg-help-299">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00205_NODE_74...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r205c1-svg" data-filename="c00205_NODE_74.._r205c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 11,984 - 94,247 nt. (total: 82,264 nt)
        
        <a class="cluster-rules-header" id="r205c1-rules-header" href="#r205c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r205c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>
      
      <div class="region-svg-container">
       <div id='r205c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-300"></div> <span class="help-tooltip" id="focus-panel-help-300">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r205c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r205c1-nrps_pks" data-name="r205c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r205c1-clusterblast" data-name="r205c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r205c1-knownclusterblast" data-name="r205c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r205c1-subclusterblast" data-name="r205c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r205c1-MIBiG-cluster-compare" data-name="r205c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r205c1-pfam-details" data-name="r205c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r205c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-146"></div> <span class="help-tooltip" id="nrps-domain-help-146">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r205c1-details-svg" data-filename="c00205_NODE_74.._r205c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r205c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r205c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-149"></div> <span class="help-tooltip" id="cb-general-help-149">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r205c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r205c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r205c1_1.svg">
        NZ_CP027297 (7044671-7137479): Streptomyces sp. SGAir0924 chromosome, complet... (64% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r205c1_2.svg">
        NZ_CP042278 (655292-744828): Streptomyces sp. WAC06273 strain WAC6273 substr.... (86% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r205c1_3.svg">
        NZ_CP039123 (7125674-7220676): Streptomyces sp. SS52 chromosome, complete genome (57% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r205c1_4.svg">
        NZ_KZ819182 (5051008-5146512): Streptomyces sp. CS147 scaffold00001, whole ge... (36% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r205c1_5.svg">
        NZ_JPMW01000001 (6798045-6890589): Planobispora rosea strain ATCC 53733 scaff... (46% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r205c1_6.svg">
        NZ_CP015849 (6566127-6664649): Streptomyces sp. SAT1 chromosome, complete genome (28% of genes show similarity), T1PKS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r205c1_7.svg">
        NZ_GG657738 (6085234-6166953): Micromonospora sp. ATCC 39149 supercont1.1, wh... (40% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r205c1_8.svg">
        NZ_CP053109 (702058-782269): Streptomyces sp. Z423-1 chromosome (27% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r205c1_9.svg">
        NZ_PVZC01000001 (350334-503079): Allonocardiopsis opalescens strain DSM 45601... (34% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r205c1_10.svg">
        NZ_CP015726 (7265802-7381410): Streptomyces sp. RTd22 chromosome, complete ge... (28% of genes show similarity), PKS-like,T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r205c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r205c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r205c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-150"></div> <span class="help-tooltip" id="cb-known-help-150">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r205c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r205c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r205c1_1.svg">
        vicenistatin
      </option>
      
      <option value = "svg/knownclusterblast_r205c1_2.svg">
        sceliphrolactam
      </option>
      
      <option value = "svg/knownclusterblast_r205c1_3.svg">
        sceliphrolactam
      </option>
      
      <option value = "svg/knownclusterblast_r205c1_4.svg">
        incednine
      </option>
      
      <option value = "svg/knownclusterblast_r205c1_5.svg">
        elaiophylin
      </option>
      
      <option value = "svg/knownclusterblast_r205c1_6.svg">
        macrotermycins
      </option>
      
      <option value = "svg/knownclusterblast_r205c1_7.svg">
        aculeximycin
      </option>
      
      <option value = "svg/knownclusterblast_r205c1_8.svg">
        concanamycin A
      </option>
      
      <option value = "svg/knownclusterblast_r205c1_9.svg">
        ibomycin
      </option>
      
      <option value = "svg/knownclusterblast_r205c1_10.svg">
        mediomycin A
      </option>
      
    </select>
    <button id="knownclusterblast-r205c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r205c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r205c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-151"></div> <span class="help-tooltip" id="cb-sub-help-151">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r205c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r205c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r205c1_1.svg">
        AB032523_c1: avermectin oleandrose (50% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r205c1_2.svg">
        AF497482_c3: calicheamicin 4 nucleotide sugars (22% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r205c1_3.svg">
        DQ149987_c2: concanamycin 4&#39;-O-carbamoyl-2&#39;-deoxyrhamnose (42% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r205c1_4.svg">
        AY509120_c1: chalcomycin NDP-chalcose (60% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r205c1_5.svg">
        DQ230532_c3: phenalinolactone deoxyhexose L-amicetose (37% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r205c1_6.svg">
        AF322256_c1: simocyclinone dTDP-D-olivose (42% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r205c1_7.svg">
        DQ297453_c2: AT2433 aminopentose (25% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r205c1_8.svg">
        AF187532_c2: nogalamycin nogalose (40% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r205c1_9.svg">
        AF187532_c1: nogalamycin nogalamine (25% of genes show similarity), sugar
      </option>
      
      <option value = "svg/subclusterblast_r205c1_10.svg">
        AY509120_c2: chalcomycin NDP-6-deoxy-D-allose (50% of genes show similarity), sugar
      </option>
      
    </select>
    <button id="subclusterblast-r205c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r205c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r205c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-152"></div> <span class="help-tooltip" id="MIBiG-help-152">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r205c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r205c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r205c1"></div>
    <div id="comparison-MIBiG-r205c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000068.1: 1-27296">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000068/index.html#r1c1" target="_new">BGC0000068.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.52, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Polyketide</td>
        
        <td>geldanamycin</td>
        
        
        <td>Streptomyces hygroscopicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>Polyketide</td>
        
        <td>(-)-Mellein</td>
        
        
        <td>Parastagonospora nodorum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.51, order:0.75, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>Polyketide</td>
        
        <td>oronofacic acid</td>
        
        
        <td>Pseudomonas syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>6-methylsalicyclic acid</td>
        
        
        <td>Aspergillus terreus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000067.1: 14949-58348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000067/index.html#r1c1" target="_new">BGC0000067.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.52, order:0.66, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>geldanamycin</td>
        
        
        <td>Streptomyces hygroscopicus subsp. duamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Polyketide</td>
        
        <td>asperlactone</td>
        
        
        <td>Aspergillus ochraceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Polyketide</td>
        
        <td>depudecin</td>
        
        
        <td>Alternaria brassicicola</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000160.1: 89033-116126">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000160/index.html#r1c1" target="_new">BGC0000160.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Polyketide</td>
        
        <td>terreic acid</td>
        
        
        <td>Aspergillus terreus NIH2624</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000122.1: 1816-33971">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000122/index.html#r1c1" target="_new">BGC0000122.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.51, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Polyketide</td>
        
        <td>phenylnannolone A, phenylnannolone B, phenylnannolone C</td>
        
        
        <td>Nannocystis pusilla</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP, Polyketide</td>
        
        <td>caboxamycin</td>
        
        
        <td>Streptomyces sp. NTK 937</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r205c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000068.1: 1-27296">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000068/index.html#r1c1" target="_new">BGC0000068.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.52, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Polyketide</td>
      <td>geldanamycin</td>
      <td>Streptomyces hygroscopicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide</td>
      <td>(-)-Mellein</td>
      <td>Parastagonospora nodorum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.51, order:0.75, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>oronofacic acid</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276.1: 1101-6575">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>6-methylsalicyclic acid</td>
      <td>Aspergillus terreus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000067.1: 14949-58348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000067/index.html#r1c1" target="_new">BGC0000067.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.52, order:0.66, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>geldanamycin</td>
      <td>Streptomyces hygroscopicus subsp. duamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273.1: 0-5301">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>asperlactone</td>
      <td>Aspergillus ochraceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>depudecin</td>
      <td>Alternaria brassicicola</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000160.1: 89033-116126">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000160/index.html#r1c1" target="_new">BGC0000160.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>terreic acid</td>
      <td>Aspergillus terreus NIH2624</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000122.1: 1816-33971">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000122/index.html#r1c1" target="_new">BGC0000122.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.51, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>phenylnannolone A, phenylnannolone B, phenylnannolone C</td>
      <td>Nannocystis pusilla</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001538.1: 5307571-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001538/index.html#r1c1" target="_new">BGC0001538.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP, Polyketide</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r205c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-153"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-153">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r205c1-pfam-details-svg" data-filename="c00205_NODE_74.._r205c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r205c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r205c1-nrps_pks_products" data-name="r205c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r205c1-nrps_pks" data-name="r205c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r205c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-147"></div> <span class="help-tooltip" id="nrps-structures-help-147">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 11983 - 94247:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r205c1-1" class="smiles-canvas" data-smiles="C=CC=CC=CCC(O)C=CCC(O)C=CC=CC(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ccmal) + (ccmal) + (ccmal) + (ohmal - ccmal) + (ohmal) + (ccmal) + (ccmal)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal&amp;nrps2=ccmal&amp;nrps3=ccmal&amp;nrps4=ohmal,ccmal&amp;nrps5=ohmal&amp;nrps6=ccmal&amp;nrps7=ccmal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal*&amp;nrps2=ccmal*&amp;nrps3=ccmal*&amp;nrps4=ohmal*,ccmal*&amp;nrps5=ohmal*&amp;nrps6=ccmal*&amp;nrps7=ccmal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r205c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C=CC=CC=CCC(O)C=CCC(O)C=CC=CC(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r205c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-148"></div> <span class="help-tooltip" id="nrps-monomers-help-148">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds31983_38943</span></strong>: mal
      <div class="collapser collapser-target-cds31983_38943 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (739..1033)</strong>: mal
          <div class="collapser collapser-target-cds31983_38943 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds31983_38943-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 91.7%</dd>
<dd>inactive: 83.3%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds31983_38943-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 219.4</dt>

  <dd></dd><dt>inactive: 113.4</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 75.7</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 66.4</dt>

  <dd></dd><dt>Isobutyryl-CoA: 41.6</dt>

  <dd></dd><dt>Propionyl-CoA: 38.6</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 29.7</dt>

  <dd></dd><dt>CHC-CoA: 22.2</dt>

  <dd></dd><dt>Acetyl-CoA: 18.5</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 16.9</dt>

  <dd></dd><dt>trans-1,2-CPDA: 14.9</dt>

  <dd></dd><dt>Benzoyl-CoA: 13.8</dt>

  <dd></dd><dt>fatty_acid: 8.5</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds38908_45460</span></strong>: mal
      <div class="collapser collapser-target-cds38908_45460 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (221..437)</strong>: mal
          <div class="collapser collapser-target-cds38908_45460 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds38908_45460-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 87.5%</dd>
<dd>inactive: 75.0%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds38908_45460-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 169.0</dt>

  <dd></dd><dt>inactive: 85.6</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 62.4</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 40.5</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 22.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 19.2</dt>

  <dd></dd><dt>Propionyl-CoA: 17.2</dt>

  <dd></dd><dt>Benzoyl-CoA: 9.5</dt>

  <dd></dd><dt>CHC-CoA: 9.4</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 8.1</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: -1.4</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds52463_57797</span></strong>: mal
      <div class="collapser collapser-target-cds52463_57797 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (1328..1591)</strong>: mal
          <div class="collapser collapser-target-cds52463_57797 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds52463_57797-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 83.3%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds52463_57797-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 218.1</dt>

  <dd></dd><dt>inactive: 112.9</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 70.2</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 50.0</dt>

  <dd></dd><dt>Propionyl-CoA: 32.3</dt>

  <dd></dd><dt>Isobutyryl-CoA: 25.8</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 17.7</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 9.7</dt>

  <dd></dd><dt>Benzoyl-CoA: 9.6</dt>

  <dd></dd><dt>CHC-CoA: 7.8</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: -0.9</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds57783_60666</span></strong>: mal
      <div class="collapser collapser-target-cds57783_60666 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (570..808)</strong>: mal
          <div class="collapser collapser-target-cds57783_60666 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds57783_60666-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>inactive: 62.5%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds57783_60666-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 108.0</dt>

  <dd></dd><dt>inactive: 79.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 46.3</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 32.2</dt>

  <dd></dd><dt>Propionyl-CoA: 15.8</dt>

  <dd></dd><dt>Acetyl-CoA: 15.5</dt>

  <dd></dd><dt>Isobutyryl-CoA: 15.3</dt>

  <dd></dd><dt>Benzoyl-CoA: 9.0</dt>

  <dd></dd><dt>CHC-CoA: 7.8</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 7.2</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds65531_70370</span></strong>: mal
      <div class="collapser collapser-target-cds65531_70370 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
          <dd><strong>PKS_AT (323..586)</strong>: mal
          <div class="collapser collapser-target-cds65531_70370 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds65531_70370-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 83.3%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds65531_70370-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 213.2</dt>

  <dd></dd><dt>inactive: 113.6</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 72.3</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 50.4</dt>

  <dd></dd><dt>Propionyl-CoA: 27.7</dt>

  <dd></dd><dt>Isobutyryl-CoA: 23.4</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 12.4</dt>

  <dd></dd><dt>Benzoyl-CoA: 9.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 8.9</dt>

  <dd></dd><dt>CHC-CoA: 5.6</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: -0.9</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds70404_74247</span></strong>: mal
      <div class="collapser collapser-target-cds70404_74247 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (0..152)</strong>: mal
          <div class="collapser collapser-target-cds70404_74247 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  (unknown)
              <div class="collapser collapser-target-cds70404_74247-ATSignature collapser-level-none"> </div><div class="collapser-content">
                No matches above 50%
              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds70404_74247-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 107.0</dt>

  <dd></dd><dt>inactive: 64.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 32.6</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 20.0</dt>

  <dd></dd><dt>Propionyl-CoA: 15.7</dt>

  <dd></dd><dt>Isobutyryl-CoA: 6.9</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds81808_83365</span></strong>: X
      <div class="collapser collapser-target-cds81808_83365 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (18..418)</strong>: X
          <div class="collapser collapser-target-cds81808_83365 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val, leu, ile, abu, iva
              <div class="collapser collapser-target-cds81808_83365-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DcFFFalVtK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ile</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds83388_84351</span></strong>: pk
      <div class="collapser collapser-target-cds83388_84351 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (15..290)</strong>: pk
          <div class="collapser collapser-target-cds83388_84351 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  (unknown)
              <div class="collapser collapser-target-cds83388_84351-ATSignature collapser-level-none"> </div><div class="collapser-content">
                No matches above 50%
              </div><br>
          
              minowa:  trans-1,2-CPDA
              <div class="collapser collapser-target-cds83388_84351-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>inactive: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>Propionyl-CoA: 0.0</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 0.0</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 0.0</dt>

  <dd></dd><dt>Malonyl-CoA: 0.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r224c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00224_NODE_30.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-301"></div> <span class="help-tooltip" id="region_svg-help-301">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00224_NODE_30...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r224c1-svg" data-filename="c00224_NODE_30.._r224c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 4,884 nt. (total: 4,884 nt)
        
        <a class="cluster-rules-header" id="r224c1-rules-header" href="#r224c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r224c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r224c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-302"></div> <span class="help-tooltip" id="focus-panel-help-302">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r224c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r224c1-clusterblast" data-name="r224c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r224c1-knownclusterblast" data-name="r224c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r224c1-subclusterblast" data-name="r224c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r224c1-MIBiG-cluster-compare" data-name="r224c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r224c1-pfam-details" data-name="r224c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r224c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-154"></div> <span class="help-tooltip" id="cb-general-help-154">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r224c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r224c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r224c1_1.svg">
        NZ_KZ819184 (3584225-3618334): Streptomyces sp. CS207 scaffold00001, whole ge... (12% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r224c1_2.svg">
        NZ_CP039123 (803279-837373): Streptomyces sp. SS52 chromosome, complete genome (12% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r224c1_3.svg">
        NZ_CP027297 (879867-913753): Streptomyces sp. SGAir0924 chromosome, complete ... (12% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r224c1_4.svg">
        NZ_LIPF01000007 (539754-574642): Streptomyces sp. CB02414 scaffold7, whole ge... (13% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r224c1_5.svg">
        NZ_KN039946 (3116938-3141150): Streptomyces luteus strain TRM 45540 Scaffold1... (18% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r224c1_6.svg">
        NZ_CP019724 (7795661-7819785): Streptomyces pactum strain ACT12 chromosome, c... (21% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r224c1_7.svg">
        NZ_KZ195577 (7640490-7663727): Streptomyces sp. CS159 scaffold00001, whole ge... (21% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r224c1_8.svg">
        NZ_CP036534 (539973-563678): Streptomyces sp. VN1 chromosome, complete genome (20% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r224c1_9.svg">
        NZ_CP016795 (439790-462090): Streptomyces olivaceus strain KLBMP 5084 chromos... (21% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r224c1_10.svg">
        NZ_WHZR01000001 (7566803-7589989): Streptomyces sp. OM5714 A912 AS4 SC1 R2, w... (21% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r224c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r224c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r224c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-155"></div> <span class="help-tooltip" id="cb-known-help-155">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r224c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r224c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r224c1_1.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r224c1_2.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r224c1_3.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r224c1_4.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r224c1_5.svg">
        carotenoid
      </option>
      
    </select>
    <button id="knownclusterblast-r224c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r224c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r224c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-156"></div> <span class="help-tooltip" id="cb-sub-help-156">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r224c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-157"></div> <span class="help-tooltip" id="MIBiG-help-157">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r224c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r224c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r224c1"></div>
    <div id="comparison-MIBiG-r224c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.77 (id:0.59, order:0.88, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
         
        
        <td>0.48</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.75 (id:0.57, order:0.88, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
         
        
        <td>0.44</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces collinus Tu 365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.74 (id:0.57, order:0.79, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
         
        
        <td>0.43</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.59, order:0.81, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.56, order:0.81, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Rhodobacter sphaeroides</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656.1: 169-3770">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Terpene</td>
        
        <td>zeaxanthin</td>
        
        
        <td>Xanthobacter autotrophicus Py2</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>Terpene</td>
        
        <td>β-carotein</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640.1: 2399-8466">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Enterobacteriaceae bacterium DC404</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630.1: 389-6051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>Terpene</td>
        
        <td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
        
        
        <td>Paracoccus haeundaensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r224c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.77 (id:0.59, order:0.88, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
      <td>0.77</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.75 (id:0.57, order:0.88, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
      <td>0.75</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces collinus Tu 365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.74 (id:0.57, order:0.79, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
      <td>0.74</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.59, order:0.81, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.56, order:0.81, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Rhodobacter sphaeroides</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656.1: 169-3770">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>zeaxanthin</td>
      <td>Xanthobacter autotrophicus Py2</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Terpene</td>
      <td>β-carotein</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640.1: 2399-8466">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Enterobacteriaceae bacterium DC404</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630.1: 389-6051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Terpene</td>
      <td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
      <td>Paracoccus haeundaensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r224c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-158"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-158">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r224c1-pfam-details-svg" data-filename="c00224_NODE_30.._r224c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r224c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r245c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00245_NODE_51.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-303"></div> <span class="help-tooltip" id="region_svg-help-303">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00245_NODE_51...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r245c1-svg" data-filename="c00245_NODE_51.._r245c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 27,327 nt. (total: 27,327 nt)
        
        <a class="cluster-rules-header" id="r245c1-rules-header" href="#r245c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r245c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r245c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-304"></div> <span class="help-tooltip" id="focus-panel-help-304">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r245c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r245c1-nrps_pks" data-name="r245c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r245c1-clusterblast" data-name="r245c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r245c1-knownclusterblast" data-name="r245c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r245c1-subclusterblast" data-name="r245c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r245c1-MIBiG-cluster-compare" data-name="r245c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r245c1-pfam-details" data-name="r245c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r245c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-159"></div> <span class="help-tooltip" id="nrps-domain-help-159">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r245c1-details-svg" data-filename="c00245_NODE_51.._r245c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r245c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r245c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-162"></div> <span class="help-tooltip" id="cb-general-help-162">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r245c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r245c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r245c1_1.svg">
        NZ_KZ819185 (488550-566486): Streptomyces sp. CS207 scaffold00002, whole geno... (22% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r245c1_2.svg">
        NZ_CP042324 (3524876-3603956): Streptomyces coelicolor A3(2) strain CFB NBC 0... (22% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r245c1_3.svg">
        NZ_CP027297 (6501004-6580734): Streptomyces sp. SGAir0924 chromosome, complet... (20% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r245c1_4.svg">
        NZ_LT629768 (3748808-3827608): Streptomyces sp. 2114.2 chromosome I (21% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r245c1_5.svg">
        NZ_CP026730 (2884074-2986849): Streptomyces sp. CB09001 chromosome, complete ... (16% of genes show similarity), NRPS,T1PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r245c1_6.svg">
        NZ_CP039123 (6294253-6369829): Streptomyces sp. SS52 chromosome, complete genome (25% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r245c1_7.svg">
        NZ_BEWD01000011 (217053-297488): Streptomyces rubrogriseus strain NBRC 15455,... (23% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r245c1_8.svg">
        NZ_JTIY01000002 (691244-774422): Streptomyces sp. AcH 505 scaffold00002, whol... (34% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r245c1_9.svg">
        NZ_GG657756 (4710103-4788943): Streptomyces lividans TK24 supercont1.1, whole... (27% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r245c1_10.svg">
        NZ_KL585949 (177387-330617): Streptomyces sp. NRRL S-474 P279 Doro1 scaffold4... (14% of genes show similarity), NRPS,T1PKS,T3PKS,butyrolactone,other
      </option>
      
    </select>
    <button id= "clusterblast-r245c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r245c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r245c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-163"></div> <span class="help-tooltip" id="cb-known-help-163">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r245c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r245c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r245c1_1.svg">
        CDA1b / CDA2a / CDA2b / CDA3a / CDA3b / CDA4a / CDA4b
      </option>
      
      <option value = "svg/knownclusterblast_r245c1_2.svg">
        lipopeptide 8D1-1 / lipopeptide 8D1-2
      </option>
      
      <option value = "svg/knownclusterblast_r245c1_3.svg">
        telomycin
      </option>
      
      <option value = "svg/knownclusterblast_r245c1_4.svg">
        A54145
      </option>
      
      <option value = "svg/knownclusterblast_r245c1_5.svg">
        feglymycin
      </option>
      
      <option value = "svg/knownclusterblast_r245c1_6.svg">
        friulimicin A / friulimicin B / friulimicin C / friulimicin D
      </option>
      
      <option value = "svg/knownclusterblast_r245c1_7.svg">
        WS9326
      </option>
      
      <option value = "svg/knownclusterblast_r245c1_8.svg">
        cadaside A / cadaside B
      </option>
      
      <option value = "svg/knownclusterblast_r245c1_9.svg">
        sarpeptin A / sarpeptin B
      </option>
      
      <option value = "svg/knownclusterblast_r245c1_10.svg">
        glycinocin A
      </option>
      
    </select>
    <button id="knownclusterblast-r245c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r245c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r245c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-164"></div> <span class="help-tooltip" id="cb-sub-help-164">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <select id= "subclusterblast-r245c1-select" class= "clusterblast-selector">
      <option value="svg/subclusterblast_r245c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/subclusterblast_r245c1_1.svg">
        Y16952_c2: balhimycin hydroxyphenylglycine (100% of genes show similarity), HPG
      </option>
      
      <option value = "svg/subclusterblast_r245c1_2.svg">
        HE589771_c2: vancomycin hydroxyphenylglycine (66% of genes show similarity), HPG
      </option>
      
      <option value = "svg/subclusterblast_r245c1_3.svg">
        AF386507_c1: complestatin hydroxyphenylglycine (50% of genes show similarity), HPG
      </option>
      
      <option value = "svg/subclusterblast_r245c1_4.svg">
        AJ632270_c2: teicoplanin hydroxyphenylglycine (66% of genes show similarity), HPG
      </option>
      
    </select>
    <button id="subclusterblast-r245c1-download">Download graphic</button>
  </div>
  <div id="subclusterblast-r245c1-svg"></div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r245c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-165"></div> <span class="help-tooltip" id="MIBiG-help-165">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r245c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r245c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r245c1"></div>
    <div id="comparison-MIBiG-r245c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.74 (id:0.45, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
         
        
        <td>0.43</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050.1: 0-19730">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.46, order:1.00, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>NRP, Polyketide</td>
        
        <td>thalassospiramide A</td>
        
        
        <td>Tistrella bauzanensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:1.00, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.37</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.36</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>vicibactin</td>
        
        
        <td>Rhizobium etli CFN 42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000438.1: 137-73725">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000438/index.html#r1c1" target="_new">BGC0000438.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.47, order:1.00, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>syringopeptin 25A</td>
        
        
        <td>Pseudomonas syringae pv. syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001434.1: 0-14218">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001434/index.html#r1c1" target="_new">BGC0001434.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:0.83, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>nematophin</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001698.1: 0-14218">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001698/index.html#r1c1" target="_new">BGC0001698.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:0.83, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>nevaltophin A, nevaltophin B, nevaltophin C, nevaltophin D</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r245c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.74 (id:0.45, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
      <td>0.74</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050.1: 0-19730">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.46, order:1.00, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>NRP, Polyketide</td>
      <td>thalassospiramide A</td>
      <td>Tistrella bauzanensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.47, order:1.00, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.47, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>vicibactin</td>
      <td>Rhizobium etli CFN 42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000438.1: 137-73725">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000438/index.html#r1c1" target="_new">BGC0000438.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.47, order:1.00, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>syringopeptin 25A</td>
      <td>Pseudomonas syringae pv. syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001434.1: 0-14218">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001434/index.html#r1c1" target="_new">BGC0001434.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.45, order:0.83, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>nematophin</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001698.1: 0-14218">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001698/index.html#r1c1" target="_new">BGC0001698.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.45, order:0.83, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>nevaltophin A, nevaltophin B, nevaltophin C, nevaltophin D</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r245c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-166"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-166">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r245c1-pfam-details-svg" data-filename="c00245_NODE_51.._r245c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r245c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r245c1-nrps_pks_products" data-name="r245c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r245c1-nrps_pks" data-name="r245c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r245c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-160"></div> <span class="help-tooltip" id="nrps-structures-help-160">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 27327:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r245c1-1" class="smiles-canvas" data-smiles="NC(CO)C(=O)NC(C(O)C)C(=O)NC(CC1=CNc2c1cccc2)C(=O)NC(c1ccc(O)cc1)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(ser - thr - D-trp) + (D-hpg)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,thr,d-trp&amp;nrps2=d-hpg" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,thr*,d-trp*&amp;nrps2=d-hpg*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r245c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CO)C(=O)NC(C(O)C)C(=O)NC(CC1=CNc2c1cccc2)C(=O)NC(c1ccc(O)cc1)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r245c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-161"></div> <span class="help-tooltip" id="nrps-monomers-help-161">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds4914_17472</span></strong>: ser - thr - trp
      <div class="collapser collapser-target-cds4914_17472 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,thr,trp" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,thr*,trp*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (477..873)</strong>: ser
          <div class="collapser collapser-target-cds4914_17472 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-cds4914_17472-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWHFSLVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1551..1970)</strong>: thr
          <div class="collapser collapser-target-cds4914_17472 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds4914_17472-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNVGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2632..3028)</strong>: trp
          <div class="collapser collapser-target-cds4914_17472 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  trp
              <div class="collapser collapser-target-cds4914_17472-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>phe, trp</dd>
   <dt>Single AA prediction:</dt>
   <dd>trp</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAWSVGSVCK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>trp</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds19878_21900</span></strong>: asp
      <div class="collapser collapser-target-cds19878_21900 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asp" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asp*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (0..244)</strong>: asp
          <div class="collapser collapser-target-cds19878_21900 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  asp
              <div class="collapser collapser-target-cds19878_21900-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>asp, asn</dd>
   <dt>Single AA prediction:</dt>
   <dd>asp</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLTKIGAVNK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>asp</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds21850_25738</span></strong>: hpg
      <div class="collapser collapser-target-cds21850_25738 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=hpg" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=hpg*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (223..621)</strong>: hpg
          <div class="collapser collapser-target-cds21850_25738 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hpg
              <div class="collapser collapser-target-cds21850_25738-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>dhpg, hpg</dd>
   <dt>Single AA prediction:</dt>
   <dd>hpg</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVYHLGLLCK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>hpg</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r255c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00255_NODE_21.. - Region 1 - T3PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-305"></div> <span class="help-tooltip" id="region_svg-help-305">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00255_NODE_21...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r255c1-svg" data-filename="c00255_NODE_21.._r255c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 16,913 - 44,716 nt. (total: 27,804 nt)
        
        <a class="cluster-rules-header" id="r255c1-rules-header" href="#r255c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r255c1-rules">T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)</div>
      
      <div class="region-svg-container">
       <div id='r255c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-306"></div> <span class="help-tooltip" id="focus-panel-help-306">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r255c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
    
     
      <div class="body-details-header r255c1-clusterblast" data-name="r255c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r255c1-knownclusterblast" data-name="r255c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r255c1-subclusterblast" data-name="r255c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r255c1-MIBiG-cluster-compare" data-name="r255c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r255c1-pfam-details" data-name="r255c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    
     
    <div class="body-details-section r255c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-167"></div> <span class="help-tooltip" id="cb-general-help-167">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r255c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r255c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r255c1_1.svg">
        NZ_CP027297 (1390525-1431632): Streptomyces sp. SGAir0924 chromosome, complet... (73% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r255c1_2.svg">
        NZ_CP039123 (1295156-1336263): Streptomyces sp. SS52 chromosome, complete genome (73% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r255c1_3.svg">
        NZ_CP036534 (992278-1033391): Streptomyces sp. VN1 chromosome, complete genome (70% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r255c1_4.svg">
        NZ_CP016795 (899757-940870): Streptomyces olivaceus strain KLBMP 5084 chromos... (70% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r255c1_5.svg">
        NZ_RSEG01000003 (301668-342793): Streptomyces tendae strain UTMC 3329 Scaffol... (67% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r255c1_6.svg">
        NZ_KZ195574 (7369564-7408348): Streptomyces sp. CS113 scaffold00001, whole ge... (67% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r255c1_7.svg">
        NZ_BEWD01000004 (246376-287501): Streptomyces rubrogriseus strain NBRC 15455,... (67% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r255c1_8.svg">
        NZ_CP042324 (1258218-1297041): Streptomyces coelicolor A3(2) strain CFB NBC 0... (67% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r255c1_9.svg">
        NZ_LT629768 (1312358-1351204): Streptomyces sp. 2114.2 chromosome I (67% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r255c1_10.svg">
        NZ_GG657756 (7014496-7055621): Streptomyces lividans TK24 supercont1.1, whole... (67% of genes show similarity), T3PKS
      </option>
      
    </select>
    <button id= "clusterblast-r255c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r255c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r255c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-168"></div> <span class="help-tooltip" id="cb-known-help-168">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r255c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r255c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r255c1_1.svg">
        flaviolin rhamnoside / 3,3&#39;-diflaviolin / flaviolin
      </option>
      
      <option value = "svg/knownclusterblast_r255c1_2.svg">
        flaviolin
      </option>
      
      <option value = "svg/knownclusterblast_r255c1_3.svg">
        herboxidiene
      </option>
      
      <option value = "svg/knownclusterblast_r255c1_4.svg">
        napyradiomycin
      </option>
      
      <option value = "svg/knownclusterblast_r255c1_5.svg">
        napyradiomycin A80915C
      </option>
      
      <option value = "svg/knownclusterblast_r255c1_6.svg">
        merochlorin A / merochlorin B / deschloro-merochlorin A / deschloro-merochlorin B / isochloro-merochlorin B / dichloro-merochlorin B / merochlorin D / merochlorin C
      </option>
      
      <option value = "svg/knownclusterblast_r255c1_7.svg">
        furaquinocin B
      </option>
      
      <option value = "svg/knownclusterblast_r255c1_8.svg">
        akaeolide
      </option>
      
      <option value = "svg/knownclusterblast_r255c1_9.svg">
        salinomycin
      </option>
      
      <option value = "svg/knownclusterblast_r255c1_10.svg">
        thiolutin
      </option>
      
    </select>
    <button id="knownclusterblast-r255c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r255c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r255c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-169"></div> <span class="help-tooltip" id="cb-sub-help-169">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r255c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-170"></div> <span class="help-tooltip" id="MIBiG-help-170">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r255c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r255c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r255c1"></div>
    <div id="comparison-MIBiG-r255c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T3PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310.1: 6460417-6463122">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
         
        
        <td>0.54</td>
        <td>Terpene</td>
        
        <td>naringenin</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.75 (id:0.55, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
         
        
        <td>0.44</td>
        <td>Other</td>
        
        <td>flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000280.1: 384-6033">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000280/index.html#r1c1" target="_new">BGC0000280.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>Polyketide</td>
        
        <td>2,4-diacetylphloroglucinol</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000281.1: 0-8137">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000281/index.html#r1c1" target="_new">BGC0000281.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>Polyketide</td>
        
        <td>2,4-diacetylphloroglucinol</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282.1: 548717-551330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Polyketide</td>
        
        <td>alkylresorcinol</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001962.1: 0-7027">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001962/index.html#r1c1" target="_new">BGC0001962.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Polyketide</td>
        
        <td>hierridin B, hierridin C</td>
        
        
        <td>Cyanobium sp. LEGE 06113</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001938.1: 90047-91866">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001938/index.html#r1c1" target="_new">BGC0001938.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Polyketide</td>
        
        <td>entotheonella</td>
        
        
        <td>Candidatus Entotheonella serta</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.55, order:0.48, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>Polyketide, Saccharide</td>
        
        <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001077.1: 8379669-8386219">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001077/index.html#r1c1" target="_new">BGC0001077.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Terpene, Polyketide</td>
        
        <td>alkyl-O-dihydrogeranyl-methoxyhydroquinones</td>
        
        
        <td>Actinoplanes missouriensis 431</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831.1: 8162481-8168632">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Polyketide</td>
        
        <td>alkylpyrone-407, alkylpyrone-393</td>
        
        
        <td>Myxococcus xanthus DK 1622</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r255c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310.1: 6460417-6463122">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
      <td>0.81</td>
      <td>Terpene</td>
      <td>naringenin</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.75 (id:0.55, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
      <td>0.75</td>
      <td>Other</td>
      <td>flaviolin</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000280.1: 384-6033">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000280/index.html#r1c1" target="_new">BGC0000280.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Polyketide</td>
      <td>2,4-diacetylphloroglucinol</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000281.1: 0-8137">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000281/index.html#r1c1" target="_new">BGC0000281.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Polyketide</td>
      <td>2,4-diacetylphloroglucinol</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282.1: 548717-551330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>alkylresorcinol</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001962.1: 0-7027">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001962/index.html#r1c1" target="_new">BGC0001962.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>hierridin B, hierridin C</td>
      <td>Cyanobium sp. LEGE 06113</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001938.1: 90047-91866">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001938/index.html#r1c1" target="_new">BGC0001938.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Polyketide</td>
      <td>entotheonella</td>
      <td>Candidatus Entotheonella serta</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.55, order:0.48, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Polyketide, Saccharide</td>
      <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
      <td>Saccharopolyspora erythraea</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001077.1: 8379669-8386219">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001077/index.html#r1c1" target="_new">BGC0001077.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Terpene, Polyketide</td>
      <td>alkyl-O-dihydrogeranyl-methoxyhydroquinones</td>
      <td>Actinoplanes missouriensis 431</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831.1: 8162481-8168632">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>alkylpyrone-407, alkylpyrone-393</td>
      <td>Myxococcus xanthus DK 1622</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r255c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-171"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-171">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r255c1-pfam-details-svg" data-filename="c00255_NODE_21.._r255c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r255c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r280c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00280_NODE_63.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-307"></div> <span class="help-tooltip" id="region_svg-help-307">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00280_NODE_63...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r280c1-svg" data-filename="c00280_NODE_63.._r280c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 83,615 - 109,197 nt. (total: 25,583 nt)
        
        <a class="cluster-rules-header" id="r280c1-rules-header" href="#r280c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r280c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r280c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-308"></div> <span class="help-tooltip" id="focus-panel-help-308">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r280c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r280c1-clusterblast" data-name="r280c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r280c1-knownclusterblast" data-name="r280c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r280c1-subclusterblast" data-name="r280c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r280c1-MIBiG-cluster-compare" data-name="r280c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r280c1-pfam-details" data-name="r280c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r280c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-172"></div> <span class="help-tooltip" id="cb-general-help-172">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r280c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r280c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r280c1_1.svg">
        NZ_CP027297 (200986-224830): Streptomyces sp. SGAir0924 chromosome, complete ... (90% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r280c1_2.svg">
        NZ_CP042278 (266672-291012): Streptomyces sp. WAC06273 strain WAC6273 substr.... (52% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r280c1_3.svg">
        NZ_CP039123 (80719-99478): Streptomyces sp. SS52 chromosome, complete genome (83% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r280c1_4.svg">
        NZ_WHZR01000001 (1953433-2014904): Streptomyces sp. OM5714 A912 AS4 SC1 R2, w... (28% of genes show similarity), NRPS-like,T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r280c1_5.svg">
        NZ_CP023696 (6169626-6193098): Streptomyces fradiae ATCC 10745 = DSM 40063 st... (61% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r280c1_6.svg">
        NZ_CP017316 (5980432-6004534): Streptomyces rubrolavendulae strain MJM4426 ch... (61% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r280c1_7.svg">
        NZ_RSEG01000001 (59184-84720): Streptomyces tendae strain UTMC 3329 Scaffold1... (59% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r280c1_8.svg">
        NZ_CP009754 (773787-798924): Streptomyces sp. CCM MD2014 chromosome, complete... (59% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r280c1_9.svg">
        NZ_JAAIFS010000001 (820425-845925): Streptomyces tendae strain VITAKN NODE 1 ... (59% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r280c1_10.svg">
        NZ_BBNN01000031 (81263-106859): Streptomyces sp. NBRC 110035 strain TP-A0873,... (42% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r280c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r280c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r280c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-173"></div> <span class="help-tooltip" id="cb-known-help-173">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r280c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r280c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r280c1_1.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r280c1_2.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r280c1_3.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r280c1_4.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r280c1_5.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r280c1_6.svg">
        rustmicin
      </option>
      
    </select>
    <button id="knownclusterblast-r280c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r280c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r280c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-174"></div> <span class="help-tooltip" id="cb-sub-help-174">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r280c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-175"></div> <span class="help-tooltip" id="MIBiG-help-175">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r280c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r280c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r280c1"></div>
    <div id="comparison-MIBiG-r280c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.74 (id:0.56, order:0.73, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
         
        
        <td>0.43</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.55, order:0.66, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640.1: 2399-8466">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Enterobacteriaceae bacterium DC404</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.56, order:0.47, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.55, order:0.47, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces collinus Tu 365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.56, order:0.50, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650.1: 28-7720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Algoriphagus sp. KK10202C</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000641.1: 1208-7454">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000641/index.html#r1c1" target="_new">BGC0000641.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Enterobacteriaceae bacterium DC416</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000639.1: 324-6564">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000639/index.html#r1c1" target="_new">BGC0000639.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Enterobacteriaceae bacterium DC260</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:0.44, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Myxococcus xanthus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r280c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.74 (id:0.56, order:0.73, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.74), rgb(0, 0, 0, 0.74) 74%, white 74%)"></td>
      <td>0.74</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649.1: 148-10747">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.55, order:0.66, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640.1: 2399-8466">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Enterobacteriaceae bacterium DC404</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.56, order:0.47, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.55, order:0.47, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces collinus Tu 365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.56, order:0.50, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650.1: 28-7720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Algoriphagus sp. KK10202C</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000641.1: 1208-7454">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000641/index.html#r1c1" target="_new">BGC0000641.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Enterobacteriaceae bacterium DC416</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000639.1: 324-6564">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000639/index.html#r1c1" target="_new">BGC0000639.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Enterobacteriaceae bacterium DC260</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.44, order:0.44, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Myxococcus xanthus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r280c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-176"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-176">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r280c1-pfam-details-svg" data-filename="c00280_NODE_63.._r280c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r280c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r280c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00280_NODE_63.. - Region 2 - indole <div class="help-container"> <div class="help-icon" data-id="region_svg-help-309"></div> <span class="help-tooltip" id="region_svg-help-309">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00280_NODE_63...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r280c2-svg" data-filename="c00280_NODE_63.._r280c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 109,641 - 130,819 nt. (total: 21,179 nt)
        
        <a class="cluster-rules-header" id="r280c2-rules-header" href="#r280c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r280c2-rules">indole: (indsynth or dmat or indole_PTase)</div>
      
      <div class="region-svg-container">
       <div id='r280c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-310"></div> <span class="help-tooltip" id="focus-panel-help-310">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r280c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r280c2-clusterblast" data-name="r280c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r280c2-knownclusterblast" data-name="r280c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r280c2-subclusterblast" data-name="r280c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r280c2-MIBiG-cluster-compare" data-name="r280c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r280c2-pfam-details" data-name="r280c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r280c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-177"></div> <span class="help-tooltip" id="cb-general-help-177">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r280c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r280c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r280c2_1.svg">
        NZ_KZ819184 (4061414-4082518): Streptomyces sp. CS207 scaffold00001, whole ge... (85% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r280c2_2.svg">
        NZ_CP039123 (56484-76520): Streptomyces sp. SS52 chromosome, complete genome (92% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r280c2_3.svg">
        NZ_CP027297 (176678-196791): Streptomyces sp. SGAir0924 chromosome, complete ... (84% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r280c2_4.svg">
        NZ_KZ195574 (1936738-1956743): Streptomyces sp. CS113 scaffold00001, whole ge... (23% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r280c2_5.svg">
        NZ_CP047318 (4897393-4918563): Streptomyces sp. HM190 chromosome, complete ge... (23% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r280c2_6.svg">
        NZ_KQ948937 (33031-153700): Streptomyces caeruleatus strain NRRL B-24802 PRJN... (5% of genes show similarity), NRPS,PKS-like,T1PKS,T3PKS,indole
      </option>
      
      <option value = "svg/clusterblast_r280c2_7.svg">
        NZ_FRBI01000005 (103301-124465): Streptomyces paucisporeus strain CGMCC 4.202... (17% of genes show similarity), indole
      </option>
      
      <option value = "svg/clusterblast_r280c2_8.svg">
        NZ_FNIX01000008 (14464-95770): Lentzea jiangxiensis strain CGMCC 4.6609, whol... (4% of genes show similarity), NRPS,PKS-like,T1PKS,indole,lassopeptide
      </option>
      
      <option value = "svg/clusterblast_r280c2_9.svg">
        NZ_CP016793 (9363786-9440248): Lentzea guizhouensis strain DHS C013 chromosom... (7% of genes show similarity), T1PKS,indole
      </option>
      
      <option value = "svg/clusterblast_r280c2_10.svg">
        NZ_CP045178 (8168001-8189168): Streptomyces tsukubensis strain AT3 chromosome... (15% of genes show similarity), indole
      </option>
      
    </select>
    <button id= "clusterblast-r280c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r280c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r280c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-178"></div> <span class="help-tooltip" id="cb-known-help-178">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r280c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r280c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r280c2_1.svg">
        7-prenylisatin
      </option>
      
      <option value = "svg/knownclusterblast_r280c2_2.svg">
        5-isoprenylindole-3-carboxylate β-D-glycosyl ester
      </option>
      
    </select>
    <button id="knownclusterblast-r280c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r280c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r280c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-179"></div> <span class="help-tooltip" id="cb-sub-help-179">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r280c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-180"></div> <span class="help-tooltip" id="MIBiG-help-180">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r280c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r280c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r280c2"></div>
    <div id="comparison-MIBiG-r280c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">indole</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294.1: 26867-35886">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.90 (id:0.90, order:0.81, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.90), rgb(0, 0, 0, 0.90) 90%, white 90%)"></td>
         
        
        <td>0.73</td>
        <td>Other</td>
        
        <td>7-prenylisatin</td>
        
        
        <td>Streptomyces sp. MBT28</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000772.1: 960-13404">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000772/index.html#r1c1" target="_new">BGC0000772.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>lipopolysaccharide</td>
        
        
        <td>Brucella melitensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001387.1: 150-29437">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001387/index.html#r1c1" target="_new">BGC0001387.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>Other</td>
        
        <td>nucleocidin</td>
        
        
        <td>Streptomyces calvus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.54, order:0.37, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>Other</td>
        
        <td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>
        
        
        <td>Streptomyces sp. RM-5-8</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885.1: 0-16706">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.32 (id:0.44, order:0.26, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
         
        
        <td>0.08</td>
        <td>Other</td>
        
        <td>4-hydroxy-3-nitrosobenzamide</td>
        
        
        <td>Streptomyces murayamaensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.31 (id:0.45, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
         
        
        <td>0.08</td>
        <td>Terpene</td>
        
        <td>grixazone</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001029.1: 1-55249">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001029/index.html#r1c1" target="_new">BGC0001029.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
         
        
        <td>0.06</td>
        <td>NRP, Polyketide</td>
        
        <td>nostophycin</td>
        
        
        <td>Nostoc sp. 152</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000157.1: 1384-70003">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000157/index.html#r1c1" target="_new">BGC0000157.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.26 (id:0.48, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
         
        
        <td>0.06</td>
        <td>Polyketide</td>
        
        <td>tautomycetin</td>
        
        
        <td>Streptomyces sp. CK4412</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000603.1: 383-24939">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000603/index.html#r1c1" target="_new">BGC0000603.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.26 (id:0.54, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
         
        
        <td>0.06</td>
        <td>RiPP</td>
        
        <td>cyclothiazomycin</td>
        
        
        <td>Streptomyces hygroscopicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000632.1: 2720626-2736748">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000632/index.html#r1c1" target="_new">BGC0000632.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.25 (id:0.43, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
         
        
        <td>0.06</td>
        <td>Terpene, Saccharide</td>
        
        <td>brasilicardin A</td>
        
        
        <td>Nocardia terpenica</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r280c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294.1: 26867-35886">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.90 (id:0.90, order:0.81, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.90), rgb(0, 0, 0, 0.90) 90%, white 90%)"></td>
      <td>0.90</td>
      <td>Other</td>
      <td>7-prenylisatin</td>
      <td>Streptomyces sp. MBT28</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000772.1: 960-13404">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000772/index.html#r1c1" target="_new">BGC0000772.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Saccharide</td>
      <td>lipopolysaccharide</td>
      <td>Brucella melitensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001387.1: 150-29437">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001387/index.html#r1c1" target="_new">BGC0001387.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>Other</td>
      <td>nucleocidin</td>
      <td>Streptomyces calvus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483.1: 0-25255">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.54, order:0.37, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>Other</td>
      <td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>
      <td>Streptomyces sp. RM-5-8</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885.1: 0-16706">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.32 (id:0.44, order:0.26, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
      <td>0.32</td>
      <td>Other</td>
      <td>4-hydroxy-3-nitrosobenzamide</td>
      <td>Streptomyces murayamaensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.31 (id:0.45, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
      <td>0.31</td>
      <td>Terpene</td>
      <td>grixazone</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001029.1: 1-55249">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001029/index.html#r1c1" target="_new">BGC0001029.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
      <td>0.26</td>
      <td>NRP, Polyketide</td>
      <td>nostophycin</td>
      <td>Nostoc sp. 152</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000157.1: 1384-70003">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000157/index.html#r1c1" target="_new">BGC0000157.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.26 (id:0.48, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
      <td>0.26</td>
      <td>Polyketide</td>
      <td>tautomycetin</td>
      <td>Streptomyces sp. CK4412</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000603.1: 383-24939">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000603/index.html#r1c1" target="_new">BGC0000603.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.26 (id:0.54, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
      <td>0.26</td>
      <td>RiPP</td>
      <td>cyclothiazomycin</td>
      <td>Streptomyces hygroscopicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000632.1: 2720626-2736748">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000632/index.html#r1c1" target="_new">BGC0000632.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.25 (id:0.43, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
      <td>0.25</td>
      <td>Terpene, Saccharide</td>
      <td>brasilicardin A</td>
      <td>Nocardia terpenica</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r280c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-181"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-181">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r280c2-pfam-details-svg" data-filename="c00280_NODE_63.._r280c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r280c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r280c3' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00280_NODE_63.. - Region 3 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-311"></div> <span class="help-tooltip" id="region_svg-help-311">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00280_NODE_63...region003.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r280c3-svg" data-filename="c00280_NODE_63.._r280c3.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 167,481 - 189,031 nt. (total: 21,551 nt)
        
        <a class="cluster-rules-header" id="r280c3-rules-header" href="#r280c3">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r280c3-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r280c3-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-312"></div> <span class="help-tooltip" id="focus-panel-help-312">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r280c3">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r280c3-nrps_pks" data-name="r280c3-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r280c3-clusterblast" data-name="r280c3-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r280c3-knownclusterblast" data-name="r280c3-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r280c3-subclusterblast" data-name="r280c3-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r280c3-MIBiG-cluster-compare" data-name="r280c3-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r280c3-pfam-details" data-name="r280c3-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r280c3-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-182"></div> <span class="help-tooltip" id="nrps-domain-help-182">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r280c3-details-svg" data-filename="c00280_NODE_63.._r280c3_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r280c3-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r280c3-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-185"></div> <span class="help-tooltip" id="cb-general-help-185">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r280c3-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r280c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r280c3_1.svg">
        NZ_CP039123 (1002258-1061109): Streptomyces sp. SS52 chromosome, complete genome (29% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r280c3_2.svg">
        NZ_KZ819184 (3357873-3420035): Streptomyces sp. CS207 scaffold00001, whole ge... (29% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r280c3_3.svg">
        NZ_CP027297 (1097774-1156767): Streptomyces sp. SGAir0924 chromosome, complet... (27% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r280c3_4.svg">
        NZ_LIYH01000002 (457133-516769): Streptomyces sp. CB03234 scaffold2, whole ge... (27% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r280c3_5.svg">
        NZ_SMSN01000046 (365924-437854): Streptomyces lavendulae strain YAKB-15 scaff... (22% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r280c3_6.svg">
        NZ_CP045740 (1162997-1234086): Streptomyces sp. SUK 48 chromosome, complete g... (21% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r280c3_7.svg">
        NZ_CP047144 (6764485-6843983): Streptomyces sp. HF10 chromosome, complete genome (19% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r280c3_8.svg">
        NZ_CP024985 (8246984-8307732): Streptomyces lavendulae subsp. lavendulae stra... (26% of genes show similarity), NRPS,bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r280c3_9.svg">
        NZ_CP043638 (7610015-7697431): Streptomyces sp. INR7 chromosome, complete genome (25% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r280c3_10.svg">
        NZ_JACHJI010000003 (379615-481923): Streptomyces griseostramineus strain CECT... (17% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r280c3-download">Download graphic</button>
  </div>
  <div id="clusterblast-r280c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r280c3-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-186"></div> <span class="help-tooltip" id="cb-known-help-186">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r280c3-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r280c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r280c3_1.svg">
        streptothricin
      </option>
      
      <option value = "svg/knownclusterblast_r280c3_2.svg">
        BD-12
      </option>
      
    </select>
    <button id="knownclusterblast-r280c3-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r280c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r280c3-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-187"></div> <span class="help-tooltip" id="cb-sub-help-187">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r280c3-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-188"></div> <span class="help-tooltip" id="MIBiG-help-188">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r280c3-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r280c3-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r280c3"></div>
    <div id="comparison-MIBiG-r280c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000432.1: 1563-32878">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000432/index.html#r1c1" target="_new">BGC0000432.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.75 (id:0.84, order:0.64, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
         
        
        <td>0.44</td>
        <td>NRP</td>
        
        <td>streptothricin</td>
        
        
        <td>Streptomyces rochei</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>NRP</td>
        
        <td>ergovaline</td>
        
        
        <td>Epichloe festucae var. lolii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>dimethylcoprogen</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876.1: 71-9235">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Other</td>
        
        <td>neopolyoxin C</td>
        
        
        <td>Streptomyces tendae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561.1: 1236-18814">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.48, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>curacomycin</td>
        
        
        <td>Streptomyces curacoi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000921.1: 4334-20040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000921/index.html#r1c1" target="_new">BGC0000921.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>Other, Polyketide, NRP,  , (, S, h, i, k, i, m, a, t, e, -, d, e, r, i, v, e, d, )</td>
        
        <td>promysalin</td>
        
        
        <td>Pseudomonas putida</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 10E9</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 22D11</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r280c3-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000432.1: 1563-32878">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000432/index.html#r1c1" target="_new">BGC0000432.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.75 (id:0.84, order:0.64, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
      <td>0.75</td>
      <td>NRP</td>
      <td>streptothricin</td>
      <td>Streptomyces rochei</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348.1: 0-13540">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>ergovaline</td>
      <td>Epichloe festucae var. lolii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>dimethylcoprogen</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876.1: 71-9235">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other</td>
      <td>neopolyoxin C</td>
      <td>Streptomyces tendae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561.1: 1236-18814">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.48, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>curacomycin</td>
      <td>Streptomyces curacoi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000921.1: 4334-20040">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000921/index.html#r1c1" target="_new">BGC0000921.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Other, Polyketide, NRP,  , (, S, h, i, k, i, m, a, t, e, -, d, e, r, i, v, e, d, )</td>
      <td>promysalin</td>
      <td>Pseudomonas putida</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 10E9</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 22D11</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r280c3-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-189"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-189">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r280c3-pfam-details-svg" data-filename="c00280_NODE_63.._r280c3_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r280c3-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r280c3-nrps_pks_products" data-name="r280c3-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r280c3-nrps_pks" data-name="r280c3-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r280c3-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-183"></div> <span class="help-tooltip" id="nrps-structures-help-183">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 167480 - 189031:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r280c3-3" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r280c3-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r280c3-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-184"></div> <span class="help-tooltip" id="nrps-monomers-help-184">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds176830_178348</span></strong>: lys-b
      <div class="collapser collapser-target-cds176830_178348 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys-b" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys-b*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (11..403)</strong>: lys-b
          <div class="collapser collapser-target-cds176830_178348 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  lys-b
              <div class="collapser collapser-target-cds176830_178348-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DTEVVGTLVK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>lys-b</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds187480_189031</span></strong>: X
      <div class="collapser collapser-target-cds187480_189031 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (1..265)</strong>: X
          <div class="collapser collapser-target-cds187480_189031 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds187480_189031-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVeeVGVVsK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>trp</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r286c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00286_NODE_14.. - Region 1 - lanthipeptide-class-v <div class="help-container"> <div class="help-icon" data-id="region_svg-help-313"></div> <span class="help-tooltip" id="region_svg-help-313">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00286_NODE_14...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r286c1-svg" data-filename="c00286_NODE_14.._r286c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 6,743 nt. (total: 6,743 nt)
        
        <a class="cluster-rules-header" id="r286c1-rules-header" href="#r286c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r286c1-rules">lanthipeptide-class-v: (APH and HopA1)</div>
      
      <div class="region-svg-container">
       <div id='r286c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-314"></div> <span class="help-tooltip" id="focus-panel-help-314">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r286c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r286c1-lanthipeptides" data-name="r286c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r286c1-clusterblast" data-name="r286c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r286c1-knownclusterblast" data-name="r286c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r286c1-subclusterblast" data-name="r286c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r286c1-MIBiG-cluster-compare" data-name="r286c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r286c1-pfam-details" data-name="r286c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r286c1-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-190"></div> <span class="help-tooltip" id="lanthi-body-help-190">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    <div class="details-text">
      Lanthipeptides - No core peptides found.
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r286c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-192"></div> <span class="help-tooltip" id="cb-general-help-192">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r286c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r286c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r286c1_1.svg">
        NZ_CP027297 (4270944-4313277): Streptomyces sp. SGAir0924 chromosome, complet... (18% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r286c1_2.svg">
        NZ_KZ819185 (2045672-2088014): Streptomyces sp. CS207 scaffold00002, whole ge... (18% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r286c1_3.svg">
        NZ_CP039123 (4164285-4206627): Streptomyces sp. SS52 chromosome, complete genome (17% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r286c1_4.svg">
        NZ_CP034539 (5513855-5556227): Streptomyces cyaneochromogenes strain MK-45 ch... (17% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r286c1_5.svg">
        NZ_CP047318 (4671381-4713992): Streptomyces sp. HM190 chromosome, complete ge... (17% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r286c1_6.svg">
        NZ_CP024985 (4504294-4545817): Streptomyces lavendulae subsp. lavendulae stra... (16% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r286c1_7.svg">
        NZ_KB899009 (123942-166305): Streptomyces sp. CNB091 D581DRAFT scaffold00020.... (17% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r286c1_8.svg">
        NZ_FNHV01000011 (278400-320709): Streptomyces sp. cf386, whole genome shotgun... (18% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r286c1_9.svg">
        NC_020504 (4992059-5038915): Streptomyces davaonensis JCM 4913, complete genome (13% of genes show similarity), lanthidin
      </option>
      
      <option value = "svg/clusterblast_r286c1_10.svg">
        NZ_CP019724 (355288-475720): Streptomyces pactum strain ACT12 chromosome, com... (5% of genes show similarity), NRPS,T1PKS,lanthidin
      </option>
      
    </select>
    <button id= "clusterblast-r286c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r286c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r286c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-193"></div> <span class="help-tooltip" id="cb-known-help-193">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r286c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-194"></div> <span class="help-tooltip" id="cb-sub-help-194">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r286c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-195"></div> <span class="help-tooltip" id="MIBiG-help-195">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r286c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r286c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r286c1"></div>
    <div id="comparison-MIBiG-r286c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-v</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001150.1: 12-23077">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001150/index.html#r1c1" target="_new">BGC0001150.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>pamamycin-607</td>
        
        
        <td>Streptomyces alboniger</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001610.1: 410-17543">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001610/index.html#r1c1" target="_new">BGC0001610.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Other</td>
        
        <td>gougerotin</td>
        
        
        <td>Streptomyces ahygroscopicus subsp. wuzhouensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001735.1: 0-39550">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001735/index.html#r1c1" target="_new">BGC0001735.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
        <td>0.11</td>
        <td>Other</td>
        
        <td>pentostatine, vidarabine</td>
        
        
        <td>Streptomyces antibioticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001188.1: 4621759-4630230">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001188/index.html#r1c1" target="_new">BGC0001188.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.36 (id:0.42, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
         
        
        <td>0.10</td>
        <td>RiPP</td>
        
        <td>legonaridin</td>
        
        
        <td>Streptomyces sp. CT34</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001225.1: 5267-32621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001225/index.html#r1c1" target="_new">BGC0001225.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.26 (id:0.47, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
         
        
        <td>0.06</td>
        <td>NRP</td>
        
        <td>celesticetin</td>
        
        
        <td>Streptomyces caelestis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000700.1: 1-67903">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000700/index.html#r1c1" target="_new">BGC0000700.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.23 (id:0.44, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
         
        
        <td>0.05</td>
        <td>Saccharide</td>
        
        <td>istamycin</td>
        
        
        <td>Streptomyces tenjimariensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000907.1: 0-38075">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000907/index.html#r1c1" target="_new">BGC0000907.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.23 (id:0.46, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
         
        
        <td>0.05</td>
        <td>Other</td>
        
        <td>lincomycin</td>
        
        
        <td>Streptomyces lincolnensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000199.1: 0-40743">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000199/index.html#r1c1" target="_new">BGC0000199.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.22 (id:0.43, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
         
        
        <td>0.05</td>
        <td>Polyketide, Saccharide</td>
        
        <td>arimetamycin B, arimetamycin C, arimetamycin A</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001956.1: 151-29239">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001956/index.html#r1c1" target="_new">BGC0001956.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.21 (id:0.44, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
         
        
        <td>0.05</td>
        <td>Polyketide</td>
        
        <td>miharamycin A, miharamycin B</td>
        
        
        <td>Streptomyces miharaensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000414.1: 5398-44076">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000414/index.html#r1c1" target="_new">BGC0000414.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.21 (id:0.42, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
         
        
        <td>0.05</td>
        <td>NRP</td>
        
        <td>quinocarcin</td>
        
        
        <td>Streptomyces melanovinaceus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r286c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001150.1: 12-23077">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001150/index.html#r1c1" target="_new">BGC0001150.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Polyketide</td>
      <td>pamamycin-607</td>
      <td>Streptomyces alboniger</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001610.1: 410-17543">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001610/index.html#r1c1" target="_new">BGC0001610.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Other</td>
      <td>gougerotin</td>
      <td>Streptomyces ahygroscopicus subsp. wuzhouensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001735.1: 0-39550">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001735/index.html#r1c1" target="_new">BGC0001735.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
      <td>0.38</td>
      <td>Other</td>
      <td>pentostatine, vidarabine</td>
      <td>Streptomyces antibioticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001188.1: 4621759-4630230">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001188/index.html#r1c1" target="_new">BGC0001188.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.36 (id:0.42, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
      <td>0.36</td>
      <td>RiPP</td>
      <td>legonaridin</td>
      <td>Streptomyces sp. CT34</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001225.1: 5267-32621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001225/index.html#r1c1" target="_new">BGC0001225.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.26 (id:0.47, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
      <td>0.26</td>
      <td>NRP</td>
      <td>celesticetin</td>
      <td>Streptomyces caelestis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000700.1: 1-67903">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000700/index.html#r1c1" target="_new">BGC0000700.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.23 (id:0.44, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
      <td>0.23</td>
      <td>Saccharide</td>
      <td>istamycin</td>
      <td>Streptomyces tenjimariensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000907.1: 0-38075">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000907/index.html#r1c1" target="_new">BGC0000907.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.23 (id:0.46, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
      <td>0.23</td>
      <td>Other</td>
      <td>lincomycin</td>
      <td>Streptomyces lincolnensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000199.1: 0-40743">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000199/index.html#r1c1" target="_new">BGC0000199.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.22 (id:0.43, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
      <td>0.22</td>
      <td>Polyketide, Saccharide</td>
      <td>arimetamycin B, arimetamycin C, arimetamycin A</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001956.1: 151-29239">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001956/index.html#r1c1" target="_new">BGC0001956.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.21 (id:0.44, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
      <td>0.21</td>
      <td>Polyketide</td>
      <td>miharamycin A, miharamycin B</td>
      <td>Streptomyces miharaensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000414.1: 5398-44076">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000414/index.html#r1c1" target="_new">BGC0000414.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.21 (id:0.42, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
      <td>0.21</td>
      <td>NRP</td>
      <td>quinocarcin</td>
      <td>Streptomyces melanovinaceus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r286c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-196"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-196">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r286c1-pfam-details-svg" data-filename="c00286_NODE_14.._r286c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r286c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r286c1-lanthipeptides" data-name="r286c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r286c1-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-191"></div> <span class="help-tooltip" id="lanthi-side-help-191">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

  <div class="prediction-text">No core peptides found.</div>

</div>

    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r302c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00302_NODE_43.. - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-315"></div> <span class="help-tooltip" id="region_svg-help-315">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00302_NODE_43...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r302c1-svg" data-filename="c00302_NODE_43.._r302c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 7,630 nt. (total: 7,630 nt)
        
        <a class="cluster-rules-header" id="r302c1-rules-header" href="#r302c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r302c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193)</div>
      
      <div class="region-svg-container">
       <div id='r302c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-316"></div> <span class="help-tooltip" id="focus-panel-help-316">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r302c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r302c1-clusterblast" data-name="r302c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r302c1-knownclusterblast" data-name="r302c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r302c1-subclusterblast" data-name="r302c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r302c1-MIBiG-cluster-compare" data-name="r302c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r302c1-pfam-details" data-name="r302c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r302c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-197"></div> <span class="help-tooltip" id="cb-general-help-197">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r302c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r302c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r302c1_1.svg">
        NZ_CP027297 (7299543-7309759): Streptomyces sp. SGAir0924 chromosome, complet... (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r302c1_2.svg">
        NZ_CP039123 (7391811-7402027): Streptomyces sp. SS52 chromosome, complete genome (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r302c1_3.svg">
        NZ_CP042278 (467057-475278): Streptomyces sp. WAC06273 strain WAC6273 substr.... (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r302c1_4.svg">
        NZ_LIPF01000005 (435694-445910): Streptomyces sp. CB02414 scaffold5, whole ge... (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r302c1_5.svg">
        NZ_JACJIE010000009 (222688-285948): Streptomyces calvus strain CECT 3271 Ga04... (11% of genes show similarity), T1PKS,bacteriocin,terpene
      </option>
      
      <option value = "svg/clusterblast_r302c1_6.svg">
        NZ_CP019724 (719804-727563): Streptomyces pactum strain ACT12 chromosome, com... (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r302c1_7.svg">
        NZ_CP012949 (7116522-7124326): Streptomyces ambofaciens strain DSM 40697 chro... (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r302c1_8.svg">
        NZ_KZ195574 (719561-727704): Streptomyces sp. CS113 scaffold00001, whole geno... (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r302c1_9.svg">
        NZ_CP012382 (7314193-7324409): Streptomyces ambofaciens ATCC 23877 chromosome... (80% of genes show similarity), bacteriocin
      </option>
      
      <option value = "svg/clusterblast_r302c1_10.svg">
        NZ_GG657756 (7543514-7553730): Streptomyces lividans TK24 supercont1.1, whole... (66% of genes show similarity), bacteriocin
      </option>
      
    </select>
    <button id= "clusterblast-r302c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r302c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r302c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-198"></div> <span class="help-tooltip" id="cb-known-help-198">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r302c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r302c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r302c1_1.svg">
        informatipeptin
      </option>
      
    </select>
    <button id="knownclusterblast-r302c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r302c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r302c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-199"></div> <span class="help-tooltip" id="cb-sub-help-199">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r302c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-200"></div> <span class="help-tooltip" id="MIBiG-help-200">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r302c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r302c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r302c1"></div>
    <div id="comparison-MIBiG-r302c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">RiPP-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.61, order:0.41, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>RiPP</td>
        
        <td>informatipeptin</td>
        
        
        <td>Streptomyces viridochromogenes DSM 40736</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928.1: 108-8402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>
        
        
        <td>Streptomyces sp. MA37</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000484.1: 184-6641">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000484/index.html#r1c1" target="_new">BGC0000484.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>RiPP</td>
        
        <td>glycocin F</td>
        
        
        <td>Lactobacillus plantarum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000759.1: 1-29377">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000759/index.html#r1c1" target="_new">BGC0000759.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>diutan polysaccharide</td>
        
        
        <td>Sphingomonas sp. ATCC 53159</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000797.1: 324-28703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000797/index.html#r1c1" target="_new">BGC0000797.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>sphingan polysaccharide</td>
        
        
        <td>Sphingomonas sp. S88</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000768.1: 0-30166">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000768/index.html#r1c1" target="_new">BGC0000768.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>gellan polysaccharide</td>
        
        
        <td>Sphingomonas elodea ATCC 31461</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001639.1: 180937-185949">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001639/index.html#r1c1" target="_new">BGC0001639.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>RiPP</td>
        
        <td>klebsidin</td>
        
        
        <td>Klebsiella pneumoniae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000019.1: 339-17245">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000019/index.html#r1c1" target="_new">BGC0000019.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>angolamycin</td>
        
        
        <td>Streptomyces eurythermus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r302c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.61, order:0.41, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>RiPP</td>
      <td>informatipeptin</td>
      <td>Streptomyces viridochromogenes DSM 40736</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928.1: 108-8402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>
      <td>Streptomyces sp. MA37</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000484.1: 184-6641">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000484/index.html#r1c1" target="_new">BGC0000484.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>RiPP</td>
      <td>glycocin F</td>
      <td>Lactobacillus plantarum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000759.1: 1-29377">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000759/index.html#r1c1" target="_new">BGC0000759.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>diutan polysaccharide</td>
      <td>Sphingomonas sp. ATCC 53159</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000797.1: 324-28703">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000797/index.html#r1c1" target="_new">BGC0000797.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>sphingan polysaccharide</td>
      <td>Sphingomonas sp. S88</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000768.1: 0-30166">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000768/index.html#r1c1" target="_new">BGC0000768.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>gellan polysaccharide</td>
      <td>Sphingomonas elodea ATCC 31461</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001639.1: 180937-185949">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001639/index.html#r1c1" target="_new">BGC0001639.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>RiPP</td>
      <td>klebsidin</td>
      <td>Klebsiella pneumoniae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000019.1: 339-17245">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000019/index.html#r1c1" target="_new">BGC0000019.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Polyketide</td>
      <td>angolamycin</td>
      <td>Streptomyces eurythermus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r302c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-201"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-201">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r302c1-pfam-details-svg" data-filename="c00302_NODE_43.._r302c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r302c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r302c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00302_NODE_43.. - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-317"></div> <span class="help-tooltip" id="region_svg-help-317">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00302_NODE_43...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r302c2-svg" data-filename="c00302_NODE_43.._r302c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 20,588 - 34,425 nt. (total: 13,838 nt)
        
        <a class="cluster-rules-header" id="r302c2-rules-header" href="#r302c2">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r302c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r302c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-318"></div> <span class="help-tooltip" id="focus-panel-help-318">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r302c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r302c2-clusterblast" data-name="r302c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r302c2-knownclusterblast" data-name="r302c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r302c2-subclusterblast" data-name="r302c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r302c2-MIBiG-cluster-compare" data-name="r302c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r302c2-pfam-details" data-name="r302c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r302c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-202"></div> <span class="help-tooltip" id="cb-general-help-202">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r302c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r302c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r302c2_1.svg">
        NZ_LIPF01000005 (398682-419714): Streptomyces sp. CB02414 scaffold5, whole ge... (76% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r302c2_2.svg">
        NZ_CP042278 (491440-510671): Streptomyces sp. WAC06273 strain WAC6273 substr.... (76% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r302c2_3.svg">
        NZ_CP039123 (7357810-7378842): Streptomyces sp. SS52 chromosome, complete genome (76% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r302c2_4.svg">
        NZ_CP027297 (7267113-7288145): Streptomyces sp. SGAir0924 chromosome, complet... (75% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r302c2_5.svg">
        NZ_GG657757 (8121096-8142128): Streptomyces viridochromogenes DSM 40736 super... (62% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r302c2_6.svg">
        NZ_KZ671983 (72944-93976): Streptomyces sp. 46 scaffold20, whole genome shotg... (62% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r302c2_7.svg">
        NZ_KN039948 (312041-333073): Streptomyces luteus strain TRM 45540 Scaffold3, ... (68% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r302c2_8.svg">
        NZ_CP036534 (7656961-7677990): Streptomyces sp. VN1 chromosome, complete genome (68% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r302c2_9.svg">
        NZ_CP016795 (7522334-7543363): Streptomyces olivaceus strain KLBMP 5084 chrom... (68% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r302c2_10.svg">
        NZ_CP015866 (6617668-6638700): Streptomyces parvulus strain 2297 chromosome, ... (71% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r302c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r302c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r302c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-203"></div> <span class="help-tooltip" id="cb-known-help-203">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r302c2-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r302c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r302c2_1.svg">
        versipelostatin
      </option>
      
      <option value = "svg/knownclusterblast_r302c2_2.svg">
        chaxamycin A / chaxamycin B / chaxamycin C / chaxamycin D
      </option>
      
      <option value = "svg/knownclusterblast_r302c2_3.svg">
        pentamycin
      </option>
      
      <option value = "svg/knownclusterblast_r302c2_4.svg">
        filipin
      </option>
      
      <option value = "svg/knownclusterblast_r302c2_5.svg">
        griseorhodin A
      </option>
      
      <option value = "svg/knownclusterblast_r302c2_6.svg">
        meoabyssomicin / abyssomicin
      </option>
      
      <option value = "svg/knownclusterblast_r302c2_7.svg">
        nystatin
      </option>
      
    </select>
    <button id="knownclusterblast-r302c2-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r302c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r302c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-204"></div> <span class="help-tooltip" id="cb-sub-help-204">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r302c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-205"></div> <span class="help-tooltip" id="MIBiG-help-205">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r302c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r302c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r302c2"></div>
    <div id="comparison-MIBiG-r302c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000683.1: 86072-87080">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000683/index.html#r1c1" target="_new">BGC0000683.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Terpene</td>
        
        <td>avermitilol</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001746.1: 7136894-7138013">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001746/index.html#r1c1" target="_new">BGC0001746.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>Terpene</td>
        
        <td>pristinol</td>
        
        
        <td>Streptomyces pristinaespiralis ATCC 25486</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Alkaloid</td>
        
        <td>kitasetaline</td>
        
        
        <td>Kitasatospora setae KM-6054</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631.1: 2217593-2245275">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>botrydial</td>
        
        
        <td>Botrytis cinerea B05.10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:0.46, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene</td>
        
        <td>raimonol</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001223.1: 24900-34582">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001223/index.html#r1c1" target="_new">BGC0001223.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Other</td>
        
        <td>hydroxysporine</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.47, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>cyslabdan</td>
        
        
        <td>Streptomyces cyslabdanicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310.1: 6460417-6463122">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Terpene</td>
        
        <td>naringenin</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001224.1: 1518-11352">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001224/index.html#r1c1" target="_new">BGC0001224.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.46, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.22</td>
        <td>Other</td>
        
        <td>reductasporine</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001986.1: 0-3657">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001986/index.html#r1c1" target="_new">BGC0001986.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Other</td>
        
        <td>purincyclamide</td>
        
        
        <td>Streptomyces chrestomyceticus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r302c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000683.1: 86072-87080">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000683/index.html#r1c1" target="_new">BGC0000683.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>avermitilol</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001746.1: 7136894-7138013">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001746/index.html#r1c1" target="_new">BGC0001746.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>pristinol</td>
      <td>Streptomyces pristinaespiralis ATCC 25486</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Alkaloid</td>
      <td>kitasetaline</td>
      <td>Kitasatospora setae KM-6054</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631.1: 2217593-2245275">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>botrydial</td>
      <td>Botrytis cinerea B05.10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.46, order:0.46, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>raimonol</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001223.1: 24900-34582">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001223/index.html#r1c1" target="_new">BGC0001223.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>hydroxysporine</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.47, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>cyslabdan</td>
      <td>Streptomyces cyslabdanicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310.1: 6460417-6463122">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Terpene</td>
      <td>naringenin</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001224.1: 1518-11352">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001224/index.html#r1c1" target="_new">BGC0001224.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.46, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Other</td>
      <td>reductasporine</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001986.1: 0-3657">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001986/index.html#r1c1" target="_new">BGC0001986.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Other</td>
      <td>purincyclamide</td>
      <td>Streptomyces chrestomyceticus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r302c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-206"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-206">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r302c2-pfam-details-svg" data-filename="c00302_NODE_43.._r302c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r302c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r307c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00307_NODE_46.. - Region 1 - T1PKS,transAT-PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-319"></div> <span class="help-tooltip" id="region_svg-help-319">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00307_NODE_46...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r307c1-svg" data-filename="c00307_NODE_46.._r307c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 51,032 nt. (total: 51,032 nt)
        
        <a class="cluster-rules-header" id="r307c1-rules-header" href="#r307c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r307c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>transAT-PKS: (cds(PKS_AT and not (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS)) and (cds(ATd and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS)) or cds(tra_KS and PP-binding)))</div>
      
      <div class="region-svg-container">
       <div id='r307c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-320"></div> <span class="help-tooltip" id="focus-panel-help-320">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r307c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r307c1-nrps_pks" data-name="r307c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r307c1-clusterblast" data-name="r307c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r307c1-knownclusterblast" data-name="r307c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r307c1-subclusterblast" data-name="r307c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r307c1-MIBiG-cluster-compare" data-name="r307c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r307c1-pfam-details" data-name="r307c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r307c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-207"></div> <span class="help-tooltip" id="nrps-domain-help-207">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r307c1-details-svg" data-filename="c00307_NODE_46.._r307c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r307c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r307c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-210"></div> <span class="help-tooltip" id="cb-general-help-210">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r307c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r307c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r307c1_1.svg">
        NZ_CP017708 (4746267-4819573): Moorea producens JHB chromosome (12% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r307c1_2.svg">
        NZ_KE386568 (107893-176878): Teredinibacter turnerae T8415 TerturDRAFT scaffo... (13% of genes show similarity), NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r307c1_3.svg">
        NC_012997 (2501010-2675253): Teredinibacter turnerae T7901, complete sequence (7% of genes show similarity), NRPS,NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r307c1_4.svg">
        NZ_CP023695 (41119-&gt;124603): Streptomyces alboniger strain ATCC 12461 chromos... (8% of genes show similarity), NRPS,ladderane,other
      </option>
      
      <option value = "svg/clusterblast_r307c1_5.svg">
        NZ_CP032427 (618348-678127): Streptomyces griseorubiginosus strain 3E-1 chrom... (29% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r307c1_6.svg">
        NZ_CP017169 (5709672-5801864): Myxococcus xanthus strain GH3.5.6c2 chromosome... (13% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r307c1_7.svg">
        NZ_CP048641 (146046-223681): Streptomyces aureoverticillatus strain HN6 chrom... (14% of genes show similarity), NRPS,T1PKS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r307c1_8.svg">
        NZ_BBXF01000001 (613468-685731): Herbidospora daliensis strain NBRC 106372, w... (18% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r307c1_9.svg">
        NZ_CP025018 (3301112-3348044): Streptomyces sp. M56 (22% of genes show similarity), T1PKS
      </option>
      
      <option value = "svg/clusterblast_r307c1_10.svg">
        NZ_LJIW01000002 (3301726-3344450): Streptomyces malaysiensis strain F913 SMF9... (26% of genes show similarity), T1PKS
      </option>
      
    </select>
    <button id= "clusterblast-r307c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r307c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r307c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-211"></div> <span class="help-tooltip" id="cb-known-help-211">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r307c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r307c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r307c1_1.svg">
        candicidin
      </option>
      
      <option value = "svg/knownclusterblast_r307c1_2.svg">
        levorin A3 / C06690 / FR-008-III / candicidin A / UNII-AP5PEF5W7U
      </option>
      
      <option value = "svg/knownclusterblast_r307c1_3.svg">
        caniferolide A / caniferolide B / caniferolide C / caniferolide D
      </option>
      
      <option value = "svg/knownclusterblast_r307c1_4.svg">
        piericidin A1
      </option>
      
      <option value = "svg/knownclusterblast_r307c1_5.svg">
        apoptolidin
      </option>
      
      <option value = "svg/knownclusterblast_r307c1_6.svg">
        cremimycin
      </option>
      
      <option value = "svg/knownclusterblast_r307c1_7.svg">
        hitachimycin
      </option>
      
      <option value = "svg/knownclusterblast_r307c1_8.svg">
        X-14547
      </option>
      
      <option value = "svg/knownclusterblast_r307c1_9.svg">
        selvamicin
      </option>
      
      <option value = "svg/knownclusterblast_r307c1_10.svg">
        niphimycins C-E
      </option>
      
    </select>
    <button id="knownclusterblast-r307c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r307c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r307c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-212"></div> <span class="help-tooltip" id="cb-sub-help-212">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r307c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-213"></div> <span class="help-tooltip" id="MIBiG-help-213">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r307c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r307c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r307c1"></div>
    <div id="comparison-MIBiG-r307c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T1PKS</span></th>
         
         <th><span class="cc-heat-product">transAT-PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.59</td>
        <td>Polyketide</td>
        
        <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
        
        
        <td>Parastagonospora nodorum SN15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.53, order:0.68, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.53, order:0.68, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.57</td>
        <td>Polyketide</td>
        
        <td>oronofacic acid</td>
        
        
        <td>Pseudomonas syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.55</td>
        <td>Polyketide</td>
        
        <td>betaenone C, betaenone A</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.55</td>
        <td>Polyketide</td>
        
        <td>betaenone A, betaenone B, betaenone C</td>
        
        
        <td>Phoma betae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001405.1: 2447653-2507157">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001405/index.html#r1c1" target="_new">BGC0001405.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.47, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.47, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.52</td>
        <td>Polyketide</td>
        
        <td>chaetoviridin E, 11-epichaetomugilin A</td>
        
        
        <td>Chaetomium globosum CBS 148.51</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.51</td>
        <td>Polyketide</td>
        
        <td>depudecin</td>
        
        
        <td>Alternaria brassicicola</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001235.1: 53-50000">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001235/index.html#r1c1" target="_new">BGC0001235.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.55, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.55, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.47</td>
        <td>NRP, Polyketide</td>
        
        <td>antalid</td>
        
        
        <td>Polyangium spumosum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001141.1: 262-37660">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001141/index.html#r1c1" target="_new">BGC0001141.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.47, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.47, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.47</td>
        <td>Polyketide</td>
        
        <td>4-epi-15-epi-brefeldin A</td>
        
        
        <td>Penicillium brefeldianum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.46</td>
        <td>Polyketide</td>
        
        <td>(-)-Mellein</td>
        
        
        <td>Parastagonospora nodorum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000067.1: 14949-58348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000067/index.html#r1c1" target="_new">BGC0000067.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.56, order:0.50, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.56, order:0.50, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.46</td>
        <td>Polyketide</td>
        
        <td>geldanamycin</td>
        
        
        <td>Streptomyces hygroscopicus subsp. duamyceticus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r307c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Polyketide</td>
      <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
      <td>Parastagonospora nodorum SN15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000041.1: 0-14402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000041/index.html#r1c1" target="_new">BGC0000041.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.53, order:0.68, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Polyketide</td>
      <td>oronofacic acid</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001280.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001280/index.html#r1c1" target="_new">BGC0001280.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>betaenone C, betaenone A</td>
      <td>Phoma betae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264.1: 8442-40380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.47, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>betaenone A, betaenone B, betaenone C</td>
      <td>Phoma betae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001405.1: 2447653-2507157">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001405/index.html#r1c1" target="_new">BGC0001405.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.47, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>chaetoviridin E, 11-epichaetomugilin A</td>
      <td>Chaetomium globosum CBS 148.51</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046.1: 1267-21753">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Polyketide</td>
      <td>depudecin</td>
      <td>Alternaria brassicicola</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001235.1: 53-50000">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001235/index.html#r1c1" target="_new">BGC0001235.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.55, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP, Polyketide</td>
      <td>antalid</td>
      <td>Polyangium spumosum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001141.1: 262-37660">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001141/index.html#r1c1" target="_new">BGC0001141.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.47, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Polyketide</td>
      <td>4-epi-15-epi-brefeldin A</td>
      <td>Penicillium brefeldianum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244.1: 0-5358">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.47, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>(-)-Mellein</td>
      <td>Parastagonospora nodorum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000067.1: 14949-58348">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000067/index.html#r1c1" target="_new">BGC0000067.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.56, order:0.50, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Polyketide</td>
      <td>geldanamycin</td>
      <td>Streptomyces hygroscopicus subsp. duamyceticus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r307c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-214"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-214">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r307c1-pfam-details-svg" data-filename="c00307_NODE_46.._r307c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r307c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r307c1-nrps_pks_products" data-name="r307c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r307c1-nrps_pks" data-name="r307c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r307c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-208"></div> <span class="help-tooltip" id="nrps-structures-help-208">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 51032:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r307c1-1" class="smiles-canvas" data-smiles="C(C)C(O)C(C)CC(C)=CCC(=O)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS linker matching; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(Me-ohmal - Me-redmal - Me-ccmal) + (mal)</dd>
                
                    <dt><b>Non-colinear ordering:</b></dt><dd>
                    
                        <span class="serif">cds1912_3697</span>
                    
                         + <span class="serif">cds12608_13451</span>
                    
                         + <span class="serif">cds16585_34093</span>
                    
                         + <span class="serif">cds34191_36513</span>
                    
                         + <span class="serif">cds13429_16315</span>
                    
                         + <span class="serif">cds45589_46864</span>
                    
                         + <span class="serif">cds37542_38967</span>
                    
                    </dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-ohmal,me-redmal,me-ccmal&amp;nrps2=mal" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-ohmal*,me-redmal*,me-ccmal*&amp;nrps2=mal*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r307c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="C(C)C(O)C(C)CC(C)=CCC(=O)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r307c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-209"></div> <span class="help-tooltip" id="nrps-monomers-help-209">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds13429_16315</span></strong>: mal
      <div class="collapser collapser-target-cds13429_16315 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (1..185)</strong>: mal
          <div class="collapser collapser-target-cds13429_16315 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  (unknown)
              <div class="collapser collapser-target-cds13429_16315-ATSignature collapser-level-none"> </div><div class="collapser-content">
                No matches above 50%
              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds13429_16315-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 119.2</dt>

  <dd></dd><dt>inactive: 70.2</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 36.8</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 23.9</dt>

  <dd></dd><dt>Propionyl-CoA: 6.2</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>fatty_acid: 0.0</dt>

  <dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>Benzoyl-CoA: 0.0</dt>

  <dd></dd><dt>Acetyl-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds16585_34093</span></strong>: mmal - mmal - mmal
      <div class="collapser collapser-target-cds16585_34093 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (567..864)</strong>: mmal
          <div class="collapser collapser-target-cds16585_34093 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds16585_34093-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds16585_34093-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 217.8</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 213.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 171.3</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 164.8</dt>

  <dd></dd><dt>trans-1,2-CPDA: 139.0</dt>

  <dd></dd><dt>Propionyl-CoA: 120.8</dt>

  <dd></dd><dt>Malonyl-CoA: 101.9</dt>

  <dd></dd><dt>CHC-CoA: 98.4</dt>

  <dd></dd><dt>Isobutyryl-CoA: 77.3</dt>

  <dd></dd><dt>Benzoyl-CoA: 75.2</dt>

  <dd></dd><dt>fatty_acid: 52.4</dt>

  <dd></dd><dt>Acetyl-CoA: 45.2</dt>

  <dd></dd><dt>inactive: 33.6</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 31.2</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (2144..2441)</strong>: mmal
          <div class="collapser collapser-target-cds16585_34093 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds16585_34093-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 100.0%</dd>
<dd>Methoxymalonyl-CoA: 95.8%</dd>
<dd>Ethylmalonyl-CoA: 91.7%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds16585_34093-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 217.5</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 214.4</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 169.8</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 148.5</dt>

  <dd></dd><dt>trans-1,2-CPDA: 132.5</dt>

  <dd></dd><dt>Propionyl-CoA: 120.8</dt>

  <dd></dd><dt>CHC-CoA: 99.7</dt>

  <dd></dd><dt>Malonyl-CoA: 99.1</dt>

  <dd></dd><dt>Benzoyl-CoA: 71.6</dt>

  <dd></dd><dt>Isobutyryl-CoA: 65.7</dt>

  <dd></dd><dt>Acetyl-CoA: 54.6</dt>

  <dd></dd><dt>fatty_acid: 35.2</dt>

  <dd></dd><dt>inactive: 24.9</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 23.2</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>PKS_AT (4476..4773)</strong>: mmal
          <div class="collapser collapser-target-cds16585_34093 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds16585_34093-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Methylmalonyl-CoA: 95.8%</dd>
<dd>Methoxymalonyl-CoA: 91.7%</dd>
<dd>Ethylmalonyl-CoA: 87.5%</dd>
</dl>

              </div><br>
          
              minowa:  Methylmalonyl-CoA
              <div class="collapser collapser-target-cds16585_34093-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Methylmalonyl-CoA: 205.3</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 202.3</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 164.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 128.3</dt>

  <dd></dd><dt>trans-1,2-CPDA: 124.8</dt>

  <dd></dd><dt>Propionyl-CoA: 108.2</dt>

  <dd></dd><dt>CHC-CoA: 106.4</dt>

  <dd></dd><dt>Malonyl-CoA: 102.8</dt>

  <dd></dd><dt>Isobutyryl-CoA: 77.5</dt>

  <dd></dd><dt>Benzoyl-CoA: 60.5</dt>

  <dd></dd><dt>Acetyl-CoA: 49.9</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 28.4</dt>

  <dd></dd><dt>fatty_acid: 26.3</dt>

  <dd></dd><dt>inactive: 21.1</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds34191_36513</span></strong>: mal
      <div class="collapser collapser-target-cds34191_36513 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
      
        
      
        
          <dd><strong>PKS_AT (582..772)</strong>: mal
          <div class="collapser collapser-target-cds34191_36513 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds34191_36513-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 66.7%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds34191_36513-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 111.0</dt>

  <dd></dd><dt>inactive: 38.5</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 24.5</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 22.2</dt>

  <dd></dd><dt>Acetyl-CoA: 17.8</dt>

  <dd></dd><dt>Benzoyl-CoA: 14.1</dt>

  <dd></dd><dt>fatty_acid: 11.6</dt>

  <dd></dd><dt>Isobutyryl-CoA: 10.8</dt>

  <dd></dd><dt>Propionyl-CoA: 8.6</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 0.0</dt>

  <dd></dd><dt>CHC-CoA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds37542_38967</span></strong>: mal
      <div class="collapser collapser-target-cds37542_38967 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
      
        
          <dd><strong>PKS_AT (124..374)</strong>: mal
          <div class="collapser collapser-target-cds37542_38967 collapser-level-candidate"> </div><div class="collapser-content">
          
              ATSignature:  Malonyl-CoA
              <div class="collapser collapser-target-cds37542_38967-ATSignature collapser-level-none"> </div><div class="collapser-content">
                <dl>
 <dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 91.7%</dd>
<dd>inactive: 75.0%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
</dl>

              </div><br>
          
              minowa:  Malonyl-CoA
              <div class="collapser collapser-target-cds37542_38967-minowa collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>Prediction, score:</dt>
 <dd>
  <dl>
  <dd></dd><dt>Malonyl-CoA: 179.8</dt>

  <dd></dd><dt>inactive: 77.4</dt>

  <dd></dd><dt>Methoxymalonyl-CoA: 41.7</dt>

  <dd></dd><dt>Methylmalonyl-CoA: 32.6</dt>

  <dd></dd><dt>Propionyl-CoA: 18.6</dt>

  <dd></dd><dt>Acetyl-CoA: 16.7</dt>

  <dd></dd><dt>Benzoyl-CoA: 16.4</dt>

  <dd></dd><dt>fatty_acid: 15.2</dt>

  <dd></dd><dt>Isobutyryl-CoA: 9.9</dt>

  <dd></dd><dt>2-Methylbutyryl-CoA: 9.6</dt>

  <dd></dd><dt>Ethylmalonyl-CoA: 8.4</dt>

  <dd></dd><dt>CHC-CoA: 4.2</dt>

  <dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

  <dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r330c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00330_NODE_14.. - Region 1 - ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-321"></div> <span class="help-tooltip" id="region_svg-help-321">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00330_NODE_14...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r330c1-svg" data-filename="c00330_NODE_14.._r330c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 3,670 - 14,068 nt. (total: 10,399 nt)
        
        <a class="cluster-rules-header" id="r330c1-rules-header" href="#r330c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r330c1-rules">ectoine: ectoine_synt</div>
      
      <div class="region-svg-container">
       <div id='r330c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-322"></div> <span class="help-tooltip" id="focus-panel-help-322">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r330c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r330c1-clusterblast" data-name="r330c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r330c1-knownclusterblast" data-name="r330c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r330c1-subclusterblast" data-name="r330c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r330c1-MIBiG-cluster-compare" data-name="r330c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r330c1-pfam-details" data-name="r330c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r330c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-215"></div> <span class="help-tooltip" id="cb-general-help-215">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r330c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r330c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r330c1_1.svg">
        NZ_CP027297 (2090741-2101140): Streptomyces sp. SGAir0924 chromosome, complet... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r330c1_2.svg">
        NZ_CP042278 (5919713-5930112): Streptomyces sp. WAC06273 strain WAC6273 subst... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r330c1_3.svg">
        NZ_CP039123 (1992789-2003188): Streptomyces sp. SS52 chromosome, complete genome (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r330c1_4.svg">
        NZ_KZ819184 (2424815-2435214): Streptomyces sp. CS207 scaffold00001, whole ge... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r330c1_5.svg">
        NZ_CP019724 (6551596-6561995): Streptomyces pactum strain ACT12 chromosome, c... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r330c1_6.svg">
        NZ_CP012382 (2000971-2011370): Streptomyces ambofaciens ATCC 23877 chromosome... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r330c1_7.svg">
        NZ_CP012949 (1943699-1954098): Streptomyces ambofaciens strain DSM 40697 chro... (100% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r330c1_8.svg">
        NZ_CP015866 (1603589-1613988): Streptomyces parvulus strain 2297 chromosome, ... (90% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r330c1_9.svg">
        NZ_CP029043 (1620690-1631089): Streptomyces nigra strain 452 chromosome, comp... (90% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r330c1_10.svg">
        NZ_AUKV01000020 (91812-102211): Streptomyces sp. 142MFCol3.1 H305DRAFT scaffo... (90% of genes show similarity), ectoine
      </option>
      
    </select>
    <button id= "clusterblast-r330c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r330c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r330c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-216"></div> <span class="help-tooltip" id="cb-known-help-216">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r330c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r330c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r330c1_1.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r330c1_2.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r330c1_3.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r330c1_4.svg">
        ectoine
      </option>
      
    </select>
    <button id="knownclusterblast-r330c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r330c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r330c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-217"></div> <span class="help-tooltip" id="cb-sub-help-217">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r330c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-218"></div> <span class="help-tooltip" id="MIBiG-help-218">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r330c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r330c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r330c1"></div>
    <div id="comparison-MIBiG-r330c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">ectoine</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.85 (id:0.62, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.85), rgb(0, 0, 0, 0.85) 85%, white 85%)"></td>
         
        
        <td>0.62</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.85 (id:0.61, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.85), rgb(0, 0, 0, 0.85) 85%, white 85%)"></td>
         
        
        <td>0.61</td>
        <td>Other,  , (, E, c, t, o, i, n, e, )</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855.1: 245-3455">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.49, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium kenyense</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylophaga thalassica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852.1: 415-2854">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Sporosarcina pasteurii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylobacter marinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860.1: 710-5134">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylarcula marina</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857.1: 300-4871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylophaga alcalica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.49, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum 20Z</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r330c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.85 (id:0.62, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.85), rgb(0, 0, 0, 0.85) 85%, white 85%)"></td>
      <td>0.85</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.85 (id:0.61, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.85), rgb(0, 0, 0, 0.85) 85%, white 85%)"></td>
      <td>0.85</td>
      <td>Other,  , (, E, c, t, o, i, n, e, )</td>
      <td>ectoine</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855.1: 245-3455">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.49, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium kenyense</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylophaga thalassica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852.1: 415-2854">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Sporosarcina pasteurii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylobacter marinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860.1: 710-5134">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylarcula marina</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857.1: 300-4871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylophaga alcalica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.49, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum 20Z</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r330c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-219"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-219">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r330c1-pfam-details-svg" data-filename="c00330_NODE_14.._r330c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r330c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r341c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00341_NODE_32.. - Region 1 - siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-323"></div> <span class="help-tooltip" id="region_svg-help-323">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00341_NODE_32...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r341c1-svg" data-filename="c00341_NODE_32.._r341c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 11,240 - 23,012 nt. (total: 11,773 nt)
        
        <a class="cluster-rules-header" id="r341c1-rules-header" href="#r341c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r341c1-rules">siderophore: IucA_IucC</div>
      
      <div class="region-svg-container">
       <div id='r341c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-324"></div> <span class="help-tooltip" id="focus-panel-help-324">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r341c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r341c1-clusterblast" data-name="r341c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r341c1-knownclusterblast" data-name="r341c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r341c1-subclusterblast" data-name="r341c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r341c1-MIBiG-cluster-compare" data-name="r341c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r341c1-pfam-details" data-name="r341c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r341c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-220"></div> <span class="help-tooltip" id="cb-general-help-220">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r341c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r341c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r341c1_1.svg">
        NZ_CP027297 (3074078-3085004): Streptomyces sp. SGAir0924 chromosome, complet... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r341c1_2.svg">
        NZ_KZ819184 (1446844-1458617): Streptomyces sp. CS207 scaffold00001, whole ge... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r341c1_3.svg">
        NZ_CP039123 (2995235-3006194): Streptomyces sp. SS52 chromosome, complete genome (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r341c1_4.svg">
        NZ_CP012382 (2980925-2991871): Streptomyces ambofaciens ATCC 23877 chromosome... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r341c1_5.svg">
        NZ_CP012949 (2931338-2942282): Streptomyces ambofaciens strain DSM 40697 chro... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r341c1_6.svg">
        NZ_KZ195574 (5509882-5521667): Streptomyces sp. CS113 scaffold00001, whole ge... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r341c1_7.svg">
        NZ_GG657756 (5262727-5274512): Streptomyces lividans TK24 supercont1.1, whole... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r341c1_8.svg">
        NZ_BEWD01000007 (266566-278345): Streptomyces rubrogriseus strain NBRC 15455,... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r341c1_9.svg">
        NZ_CP019724 (5490638-5502408): Streptomyces pactum strain ACT12 chromosome, c... (100% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r341c1_10.svg">
        NZ_CP042324 (3034638-3045610): Streptomyces coelicolor A3(2) strain CFB NBC 0... (100% of genes show similarity), siderophore
      </option>
      
    </select>
    <button id= "clusterblast-r341c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r341c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r341c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-221"></div> <span class="help-tooltip" id="cb-known-help-221">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r341c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r341c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r341c1_1.svg">
        desferrioxamin B / desferrioxamine E
      </option>
      
      <option value = "svg/knownclusterblast_r341c1_2.svg">
        desferrioxamine
      </option>
      
      <option value = "svg/knownclusterblast_r341c1_3.svg">
        desferrioxamin B
      </option>
      
      <option value = "svg/knownclusterblast_r341c1_4.svg">
        desferrioxamine E
      </option>
      
      <option value = "svg/knownclusterblast_r341c1_5.svg">
        putrebactin / avaroferrin
      </option>
      
      <option value = "svg/knownclusterblast_r341c1_6.svg">
        desferrioxamine E
      </option>
      
      <option value = "svg/knownclusterblast_r341c1_7.svg">
        bisucaberin B
      </option>
      
    </select>
    <button id="knownclusterblast-r341c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r341c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r341c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-222"></div> <span class="help-tooltip" id="cb-sub-help-222">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r341c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-223"></div> <span class="help-tooltip" id="MIBiG-help-223">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r341c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r341c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r341c1"></div>
    <div id="comparison-MIBiG-r341c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">siderophore</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000941.1: 5578821-5584948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000941/index.html#r1c1" target="_new">BGC0000941.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.83 (id:0.58, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
         
        
        <td>0.59</td>
        <td>Other</td>
        
        <td>desferrioxamin B</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001478.1: 100-5365">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001478/index.html#r1c1" target="_new">BGC0001478.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
         
        
        <td>0.58</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Streptomyces sp. ID38640</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940.1: 3033354-3040682">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.80 (id:0.70, order:0.83, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
         
        
        <td>0.53</td>
        <td>Other</td>
        
        <td>desferrioxamin B, desferrioxamine E</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453.1: 0-7234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.77 (id:0.61, order:0.83, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
         
        
        <td>0.48</td>
        <td>Other</td>
        
        <td>desferrioxamine</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.52, order:0.66, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Pantoea agglomerans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001531.1: 1112-9953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001531/index.html#r1c1" target="_new">BGC0001531.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.49, order:0.57, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Other</td>
        
        <td>bisucaberin B</td>
        
        
        <td>Tenacibaculum mesophilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000244.1: 204-8850">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000244/index.html#r1c1" target="_new">BGC0000244.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.46, order:0.48, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>macrotetrolide</td>
        
        
        <td>Streptomyces griseus subsp. griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>ochrobactin</td>
        
        
        <td>Ochrobactrum anthropi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499.1: 2267-11578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.43, order:0.42, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>aerobactin</td>
        
        
        <td>Pantoea ananatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000466.1: 0-35805">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000466/index.html#r1c1" target="_new">BGC0000466.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>NRP</td>
        
        <td>yatakemycin</td>
        
        
        <td>Streptomyces sp. TP-A2060</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r341c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000941.1: 5578821-5584948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000941/index.html#r1c1" target="_new">BGC0000941.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.83 (id:0.58, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
      <td>0.83</td>
      <td>Other</td>
      <td>desferrioxamin B</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001478.1: 100-5365">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001478/index.html#r1c1" target="_new">BGC0001478.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
      <td>0.83</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Streptomyces sp. ID38640</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940.1: 3033354-3040682">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.80 (id:0.70, order:0.83, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.80), rgb(0, 0, 0, 0.80) 80%, white 80%)"></td>
      <td>0.80</td>
      <td>Other</td>
      <td>desferrioxamin B, desferrioxamine E</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453.1: 0-7234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.77 (id:0.61, order:0.83, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
      <td>0.77</td>
      <td>Other</td>
      <td>desferrioxamine</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.52, order:0.66, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Pantoea agglomerans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001531.1: 1112-9953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001531/index.html#r1c1" target="_new">BGC0001531.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.49, order:0.57, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Other</td>
      <td>bisucaberin B</td>
      <td>Tenacibaculum mesophilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000244.1: 204-8850">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000244/index.html#r1c1" target="_new">BGC0000244.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.46, order:0.48, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Polyketide</td>
      <td>macrotetrolide</td>
      <td>Streptomyces griseus subsp. griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>ochrobactin</td>
      <td>Ochrobactrum anthropi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499.1: 2267-11578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.43, order:0.42, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>aerobactin</td>
      <td>Pantoea ananatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000466.1: 0-35805">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000466/index.html#r1c1" target="_new">BGC0000466.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>NRP</td>
      <td>yatakemycin</td>
      <td>Streptomyces sp. TP-A2060</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r341c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-224"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-224">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r341c1-pfam-details-svg" data-filename="c00341_NODE_32.._r341c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r341c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r344c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00344_NODE_51.. - Region 1 - lanthipeptide-class-iii <div class="help-container"> <div class="help-icon" data-id="region_svg-help-325"></div> <span class="help-tooltip" id="region_svg-help-325">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00344_NODE_51...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r344c1-svg" data-filename="c00344_NODE_51.._r344c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 11,051 - 33,633 nt. (total: 22,583 nt)
        
        <a class="cluster-rules-header" id="r344c1-rules-header" href="#r344c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r344c1-rules">lanthipeptide-class-iii: (micKC and not PKS_KS)</div>
      
      <div class="region-svg-container">
       <div id='r344c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-326"></div> <span class="help-tooltip" id="focus-panel-help-326">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r344c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r344c1-lanthipeptides" data-name="r344c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r344c1-clusterblast" data-name="r344c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r344c1-knownclusterblast" data-name="r344c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r344c1-subclusterblast" data-name="r344c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r344c1-MIBiG-cluster-compare" data-name="r344c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r344c1-pfam-details" data-name="r344c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r344c1-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-225"></div> <span class="help-tooltip" id="lanthi-body-help-225">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lanthipeptide(s) for cds21050_23633 - Putative Class III</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds23685_23814</span> leader / core peptide
          </dt>
          <dd>
            MNLFELQSLETPKEEAMGDVE
            -
            <span class="dhb">Dhb</span>G<span class="dha">Dha</span>RA<span class="dha">Dha</span>LLL<span class="cys">C</span>GD<span class="dha">Dha</span><span class="dha">Dha</span>L<span class="dha">Dha</span>V<span class="dhb">Dhb</span><span class="dhb">Dhb</span><span class="cys">C</span>N
        </dd>
      </dl>
    
    <div class="legend">
      <h4> Legend: </h4>
      <div>
        <span class="dha">Dha</span>: Didehydroalanine<br>
        <span class="dhb">Dhb</span>: Didehydrobutyrine<br>
      </div>
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r344c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-227"></div> <span class="help-tooltip" id="cb-general-help-227">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r344c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r344c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r344c1_1.svg">
        NZ_CP039123 (7887552-7910135): Streptomyces sp. SS52 chromosome, complete genome (95% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r344c1_2.svg">
        NZ_CP027299 (182560-205143): Streptomyces sp. SGAir0924 plasmid unnamed 5 (95% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r344c1_3.svg">
        NZ_PDKQ02000044 (15958-58520): Streptomyces barkulensis strain RC 1831 Scaffo... (21% of genes show similarity), lanthipeptide,lipolanthine
      </option>
      
      <option value = "svg/clusterblast_r344c1_4.svg">
        NZ_KN039948 (59878-82461): Streptomyces luteus strain TRM 45540 Scaffold3, wh... (26% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r344c1_5.svg">
        NZ_LIVQ01000005 (144572-167215): Streptomyces sp. CB00455 scaffold5, whole ge... (35% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r344c1_6.svg">
        NZ_CP020042 (7191341-7213981): Streptomyces katrae strain S3 isolate 4 chromo... (28% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r344c1_7.svg">
        NZ_KI547044 (252876-275534): Streptomycetaceae bacterium MP113-05 scaffold000... (27% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r344c1_8.svg">
        NZ_LIPF01000004 (558690-581351): Streptomyces sp. CB02414 scaffold4, whole ge... (26% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r344c1_9.svg">
        NZ_JACHJE010000006 (460626-483245): Streptomyces albaduncus strain CECT 3226 ... (29% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r344c1_10.svg">
        NZ_RAII01000001 (&lt;2238515-2259971): Streptomyces sp. E1N211 s-211 S3 L001 R1 ... (27% of genes show similarity), lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r344c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r344c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r344c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-228"></div> <span class="help-tooltip" id="cb-known-help-228">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r344c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r344c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r344c1_1.svg">
        SapB
      </option>
      
      <option value = "svg/knownclusterblast_r344c1_2.svg">
        SAL-2242
      </option>
      
      <option value = "svg/knownclusterblast_r344c1_3.svg">
        AmfS
      </option>
      
      <option value = "svg/knownclusterblast_r344c1_4.svg">
        catenulipeptin
      </option>
      
    </select>
    <button id="knownclusterblast-r344c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r344c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r344c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-229"></div> <span class="help-tooltip" id="cb-sub-help-229">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r344c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-230"></div> <span class="help-tooltip" id="MIBiG-help-230">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r344c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r344c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r344c1"></div>
    <div id="comparison-MIBiG-r344c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-iii</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.84 (id:0.59, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.84), rgb(0, 0, 0, 0.84) 84%, white 84%)"></td>
         
        
        <td>0.60</td>
        <td>RiPP</td>
        
        <td>SapB</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496.1: 2822112-2829457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.50, order:0.64, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>RiPP</td>
        
        <td>AmfS</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.49, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>RiPP</td>
        
        <td>catenulipeptin</td>
        
        
        <td>Catenulispora acidiphila DSM 44928</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.48, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>RiPP</td>
        
        <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
        
        
        <td>Actinomadura namibiensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.49, order:0.49, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>RiPP</td>
        
        <td>informatipeptin</td>
        
        
        <td>Streptomyces viridochromogenes DSM 40736</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000516.1: 8706-19626">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000516/index.html#r1c1" target="_new">BGC0000516.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>RiPP</td>
        
        <td>geobacillin II</td>
        
        
        <td>Geobacillus thermodenitrificans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000546.1: 4112716-4120692">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000546/index.html#r1c1" target="_new">BGC0000546.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.51, order:0.48, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>RiPP</td>
        
        <td>SAL-2242</td>
        
        
        <td>Streptomyces albidoflavus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000502.1: 0-15725">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000502/index.html#r1c1" target="_new">BGC0000502.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>cerecidin, cerecidin A1, cerecidin A2, cerecidin A3, cerecidin A4, cerecidin A5, cerecidin A6, cerecidin A7</td>
        
        
        <td>Bacillus cereus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000513.1: 4704393-4710596">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000513/index.html#r1c1" target="_new">BGC0000513.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.48, order:0.57, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>Ery-9, Ery-6, Ery-8, Ery-7, Ery-5, Ery-4, Ery-3</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r344c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.84 (id:0.59, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.84), rgb(0, 0, 0, 0.84) 84%, white 84%)"></td>
      <td>0.84</td>
      <td>RiPP</td>
      <td>SapB</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496.1: 2822112-2829457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.50, order:0.64, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>RiPP</td>
      <td>AmfS</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.49, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>RiPP</td>
      <td>catenulipeptin</td>
      <td>Catenulispora acidiphila DSM 44928</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519.1: 0-6400">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.48, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>RiPP</td>
      <td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
      <td>Actinomadura namibiensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000518.1: 8170943-8185829">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000518/index.html#r1c1" target="_new">BGC0000518.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.49, order:0.49, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>RiPP</td>
      <td>informatipeptin</td>
      <td>Streptomyces viridochromogenes DSM 40736</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000516.1: 8706-19626">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000516/index.html#r1c1" target="_new">BGC0000516.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>geobacillin II</td>
      <td>Geobacillus thermodenitrificans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000546.1: 4112716-4120692">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000546/index.html#r1c1" target="_new">BGC0000546.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.51, order:0.48, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>SAL-2242</td>
      <td>Streptomyces albidoflavus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000502.1: 0-15725">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000502/index.html#r1c1" target="_new">BGC0000502.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>RiPP</td>
      <td>cerecidin, cerecidin A1, cerecidin A2, cerecidin A3, cerecidin A4, cerecidin A5, cerecidin A6, cerecidin A7</td>
      <td>Bacillus cereus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000513.1: 4704393-4710596">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000513/index.html#r1c1" target="_new">BGC0000513.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.48, order:0.57, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>RiPP</td>
      <td>Ery-9, Ery-6, Ery-8, Ery-7, Ery-5, Ery-4, Ery-3</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r344c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-231"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-231">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r344c1-pfam-details-svg" data-filename="c00344_NODE_51.._r344c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r344c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r344c1-lanthipeptides" data-name="r344c1-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r344c1-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-226"></div> <span class="help-tooltip" id="lanthi-side-help-226">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

    <dl class ="prediction-text">
    <dt><span class="serif">cds23685_23814</span> - Class III</dt>
        <dd>Cleavage pHMM score: 30.00</dd>
        <dd>RODEO score: 0</dd>
        <dd>Monoisotopic mass: 1940.0 Da</dd>
        <dd>Molecular weight: 1941.2 Da</dd>
        <dd>Number of bridges: 2</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 1959.2 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 1977.2 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 1995.2 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2013.2 Da</span>
                    <span class="alt-weight-n"> N = 4</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2031.3 Da</span>
                    <span class="alt-weight-n"> N = 5</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2049.3 Da</span>
                    <span class="alt-weight-n"> N = 6</span>
                  </dd>
                
            </dl>
          </dd>
        
  </dl>

</div>

    </div>
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r344c2' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00344_NODE_51.. - Region 2 - lanthipeptide-class-i <div class="help-container"> <div class="help-icon" data-id="region_svg-help-327"></div> <span class="help-tooltip" id="region_svg-help-327">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00344_NODE_51...region002.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r344c2-svg" data-filename="c00344_NODE_51.._r344c2.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 46,206 - 70,800 nt. (total: 24,595 nt)
        
        <a class="cluster-rules-header" id="r344c2-rules-header" href="#r344c2">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r344c2-rules">lanthipeptide-class-i: ((LANC_like and (Lant_dehydr_N or Lant_dehydr_C)) and not (YcaO or TIGR03882))</div>
      
      <div class="region-svg-container">
       <div id='r344c2-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-328"></div> <span class="help-tooltip" id="focus-panel-help-328">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r344c2">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r344c2-lanthipeptides" data-name="r344c2-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
      <div class="body-details-header r344c2-clusterblast" data-name="r344c2-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r344c2-knownclusterblast" data-name="r344c2-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r344c2-subclusterblast" data-name="r344c2-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r344c2-MIBiG-cluster-compare" data-name="r344c2-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r344c2-pfam-details" data-name="r344c2-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r344c2-lanthipeptides">
      <div class="details">
    <div class="heading">
      <span>Lanthipeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-232"></div> <span class="help-tooltip" id="lanthi-body-help-232">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash.</span></div>
    </div>
  
    
      <h4>Lanthipeptide(s) for cds57833_60800 - Putative Class I</h4>
      <hr>
      <dl class="details-text">
        
          <dt>
            <span class="serif">cds60884_61046</span> leader / core peptide
          </dt>
          <dd>
            MRTEIVLSHEAPELDLELDLRVSDLPEQA
            -
            E<span class="dha">Dha</span>FGQG<span class="dhb">Dhb</span>F<span class="dhb">Dhb</span><span class="dha">Dha</span>P<span class="dha">Dha</span><span class="dha">Dha</span>YAIG<span class="dhb">Dhb</span>R<span class="cys">C</span>PI<span class="cys">C</span><span class="cys">C</span>
        </dd>
      </dl>
    
    <div class="legend">
      <h4> Legend: </h4>
      <div>
        <span class="dha">Dha</span>: Didehydroalanine<br>
        <span class="dhb">Dhb</span>: Didehydrobutyrine<br>
      </div>
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r344c2-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-234"></div> <span class="help-tooltip" id="cb-general-help-234">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r344c2-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r344c2_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r344c2_1.svg">
        NZ_CP027299 (147618-170095): Streptomyces sp. SGAir0924 plasmid unnamed 5 (90% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r344c2_2.svg">
        NZ_CP039123 (7801014-7876557): Streptomyces sp. SS52 chromosome, complete genome (30% of genes show similarity), NRPS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r344c2_3.svg">
        NZ_CP054939 (7535875-7560410): Streptomyces sp. NA02536 chromosome, complete ... (16% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r344c2_4.svg">
        NZ_GG657758 (7884995-7909530): Streptomyces griseoflavus Tu4000 supercont1.1,... (13% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r344c2_5.svg">
        NZ_CP012382 (7931837-7968985): Streptomyces ambofaciens ATCC 23877 chromosome... (11% of genes show similarity), lanthipeptide,terpene
      </option>
      
      <option value = "svg/clusterblast_r344c2_6.svg">
        NZ_CP012949 (7717139-7754336): Streptomyces ambofaciens strain DSM 40697 chro... (11% of genes show similarity), lanthipeptide,terpene
      </option>
      
      <option value = "svg/clusterblast_r344c2_7.svg">
        NZ_VIUV01000001 (113609-138120): Streptomyces sp. T12 Ga0311319 101, whole ge... (19% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r344c2_8.svg">
        NZ_CP043959 (4007187-4030459): Streptomyces tendae strain 139 chromosome, com... (20% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r344c2_9.svg">
        NC_016582 (8307758-8331531): Streptomyces bingchenggensis BCW-1, complete seq... (22% of genes show similarity), lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r344c2_10.svg">
        NZ_MQUH01000030 (148526-201094): Streptomyces sp. M1013 M1013 C30, whole geno... (7% of genes show similarity), hglE-KS,lanthipeptide
      </option>
      
    </select>
    <button id= "clusterblast-r344c2-download">Download graphic</button>
  </div>
  <div id="clusterblast-r344c2-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r344c2-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-235"></div> <span class="help-tooltip" id="cb-known-help-235">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r344c2-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-236"></div> <span class="help-tooltip" id="cb-sub-help-236">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r344c2-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-238"></div> <span class="help-tooltip" id="MIBiG-help-238">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r344c2-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r344c2-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r344c2"></div>
    <div id="comparison-MIBiG-r344c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">lanthipeptide-class-i</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001289.1: 16-10643">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001289/index.html#r1c1" target="_new">BGC0001289.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:0.37, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>penisin</td>
        
        
        <td>Paenibacillus ehimensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000544.1: 14969-30259">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000544/index.html#r1c1" target="_new">BGC0000544.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>RiPP</td>
        
        <td>planosporicin</td>
        
        
        <td>Planomonospora alba</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001727.1: 1552553-1565755">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001727/index.html#r1c1" target="_new">BGC0001727.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.41, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>RiPP</td>
        
        <td>paenilan</td>
        
        
        <td>Paenibacillus polymyxa E681</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001364.1: 317877-335461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001364/index.html#r1c1" target="_new">BGC0001364.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>RiPP</td>
        
        <td>cinnamycin B</td>
        
        
        <td>Actinomadura atramentaria DSM 43919</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001868.1: 193423-204464">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001868/index.html#r1c1" target="_new">BGC0001868.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other,  , (, P, h, e, n, a, z, i, n, e, )</td>
        
        <td>iodinin</td>
        
        
        <td>Xenorhabdus szentirmaii DSM 16338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000207.1: 821-16213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000207/index.html#r1c1" target="_new">BGC0000207.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>chartreusin</td>
        
        
        <td>Streptomyces chartreusis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000514.1: 578-12691">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000514/index.html#r1c1" target="_new">BGC0000514.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.41, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>RiPP</td>
        
        <td>gallidermin</td>
        
        
        <td>Staphylococcus gallinarum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001965.1: 0-30490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001965/index.html#r1c1" target="_new">BGC0001965.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>nybomycin</td>
        
        
        <td>Streptomyces albus subsp. chlorinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001810.1: 122-16114">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001810/index.html#r1c1" target="_new">BGC0001810.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>trehangelin</td>
        
        
        <td>Polymorphospora rubra</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001735.1: 0-39550">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001735/index.html#r1c1" target="_new">BGC0001735.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
        <td>0.11</td>
        <td>Other</td>
        
        <td>pentostatine, vidarabine</td>
        
        
        <td>Streptomyces antibioticus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r344c2-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001289.1: 16-10643">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001289/index.html#r1c1" target="_new">BGC0001289.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:0.37, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>RiPP</td>
      <td>penisin</td>
      <td>Paenibacillus ehimensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000544.1: 14969-30259">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000544/index.html#r1c1" target="_new">BGC0000544.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>RiPP</td>
      <td>planosporicin</td>
      <td>Planomonospora alba</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001727.1: 1552553-1565755">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001727/index.html#r1c1" target="_new">BGC0001727.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.41, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>RiPP</td>
      <td>paenilan</td>
      <td>Paenibacillus polymyxa E681</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001364.1: 317877-335461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001364/index.html#r1c1" target="_new">BGC0001364.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>RiPP</td>
      <td>cinnamycin B</td>
      <td>Actinomadura atramentaria DSM 43919</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001868.1: 193423-204464">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001868/index.html#r1c1" target="_new">BGC0001868.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other,  , (, P, h, e, n, a, z, i, n, e, )</td>
      <td>iodinin</td>
      <td>Xenorhabdus szentirmaii DSM 16338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000207.1: 821-16213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000207/index.html#r1c1" target="_new">BGC0000207.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Polyketide</td>
      <td>chartreusin</td>
      <td>Streptomyces chartreusis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000514.1: 578-12691">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000514/index.html#r1c1" target="_new">BGC0000514.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.41, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>RiPP</td>
      <td>gallidermin</td>
      <td>Staphylococcus gallinarum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001965.1: 0-30490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001965/index.html#r1c1" target="_new">BGC0001965.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>nybomycin</td>
      <td>Streptomyces albus subsp. chlorinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001810.1: 122-16114">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001810/index.html#r1c1" target="_new">BGC0001810.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Polyketide</td>
      <td>trehangelin</td>
      <td>Polymorphospora rubra</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001735.1: 0-39550">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001735/index.html#r1c1" target="_new">BGC0001735.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
      <td>0.38</td>
      <td>Other</td>
      <td>pentostatine, vidarabine</td>
      <td>Streptomyces antibioticus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r344c2-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-239"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-239">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r344c2-pfam-details-svg" data-filename="c00344_NODE_51.._r344c2_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r344c2-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r344c2-lanthipeptides" data-name="r344c2-lanthipeptides">
        Lanthipeptides
      </div>
     
    
     
    
     
      <div class="sidepanel-details-header r344c2-RREfinder" data-name="r344c2-RREfinder">
        RREFinder
      </div>
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r344c2-lanthipeptides">
      <div class="more-details">
  <div class="heading">
    <span>Core peptide predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-233"></div> <span class="help-tooltip" id="lanthi-side-help-233">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
  </div>

    <dl class ="prediction-text">
    <dt><span class="serif">cds60884_61046</span> - Class I</dt>
        <dd>Cleavage pHMM score: -10.80</dd>
        <dd>RODEO score: 18</dd>
        <dd>Monoisotopic mass: 2385.1 Da</dd>
        <dd>Molecular weight: 2386.7 Da</dd>
        <dd>Number of bridges: 3</dd>
          <dd>
            <dl>
              <dt>Alternative weights</dt>
                <dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>
                
                  <dd>
                    <span class="alt-weight"> 2404.7 Da</span>
                    <span class="alt-weight-n"> N = 1</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2422.7 Da</span>
                    <span class="alt-weight-n"> N = 2</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2440.7 Da</span>
                    <span class="alt-weight-n"> N = 3</span>
                  </dd>
                
                  <dd>
                    <span class="alt-weight"> 2458.7 Da</span>
                    <span class="alt-weight-n"> N = 4</span>
                  </dd>
                
            </dl>
          </dd>
        
  </dl>

</div>

    </div>
     
    
     
    
     
    <div class="sidepanel-details-section r344c2-RREfinder">
      <div class="more-details">
  <div class="heading">
    <span>RRE predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="rre-side-help-237"></div> <span class="help-tooltip" id="rre-side-help-237">RREfinder results sidepanel.</span></div>
  </div>
  <dl class = 'prediction-text'>
    
      <span><strong>lanthipeptide-class-i protocluster</strong> (46205...70800)</span>
      <dt></dt>
      
      
      
        
        <strong>cds57833_60800</strong> <div class="collapser collapser-target-cds57833_60800 collapser-level-cds"> </div><div class="collapser-content">
        
          <dd>RRE type: Lanthipeptide_LanB_RRE</dd>
          <dd>Location: 158 - 241</dd>
          <dd>Score: 64.5</dd>
        
        </div><br>
      <br>
    
    </dl>
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>

<div class="page" id='r344c3' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00344_NODE_51.. - Region 3 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-329"></div> <span class="help-tooltip" id="region_svg-help-329">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00344_NODE_51...region003.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r344c3-svg" data-filename="c00344_NODE_51.._r344c3.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 74,572 - 126,897 nt. (total: 52,326 nt)
        
        <a class="cluster-rules-header" id="r344c3-rules-header" href="#r344c3">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r344c3-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r344c3-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-330"></div> <span class="help-tooltip" id="focus-panel-help-330">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r344c3">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r344c3-nrps_pks" data-name="r344c3-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r344c3-clusterblast" data-name="r344c3-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r344c3-knownclusterblast" data-name="r344c3-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r344c3-subclusterblast" data-name="r344c3-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r344c3-MIBiG-cluster-compare" data-name="r344c3-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r344c3-pfam-details" data-name="r344c3-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r344c3-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-240"></div> <span class="help-tooltip" id="nrps-domain-help-240">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r344c3-details-svg" data-filename="c00344_NODE_51.._r344c3_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r344c3-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r344c3-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-243"></div> <span class="help-tooltip" id="cb-general-help-243">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r344c3-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r344c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r344c3_1.svg">
        NZ_CP039123 (7801014-7876557): Streptomyces sp. SS52 chromosome, complete genome (58% of genes show similarity), NRPS,lanthipeptide
      </option>
      
      <option value = "svg/clusterblast_r344c3_2.svg">
        NZ_LIPF01000012 (136427-190557): Streptomyces sp. CB02414 scaffold12, whole g... (51% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r344c3_3.svg">
        NZ_CP029601 (1927434-1979808): Streptomyces sp. WAC 01438 chromosome, complet... (48% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r344c3_4.svg">
        NZ_VIWQ01000014 (104505-158528): Streptomyces argenteolus strain 3259 Ga03654... (54% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r344c3_5.svg">
        NZ_BBOK01000008 (269688-323846): Streptomyces sp. NBRC 110030, whole genome s... (42% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r344c3_6.svg">
        NZ_RWGV01000003 (5094-59135): Streptomyces sp. GZWMJZ-114 NODE 12, whole geno... (51% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r344c3_7.svg">
        NZ_CM001165 (304333-357760): Streptomyces sp. Tu6071 chromosome, whole genome... (50% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r344c3_8.svg">
        NZ_CP021118 (329026-382565): Streptomyces sp. CLI2509 strain CLI2905 chromoso... (50% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r344c3_9.svg">
        NZ_KZ195577 (2009280-2100187): Streptomyces sp. CS159 scaffold00001, whole ge... (28% of genes show similarity), NRPS,NRPS-like,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r344c3_10.svg">
        NZ_WWJS01000030 (55457-109572): Streptomyces sp. SID4912 SID4912.c30, whole g... (51% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r344c3-download">Download graphic</button>
  </div>
  <div id="clusterblast-r344c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r344c3-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-244"></div> <span class="help-tooltip" id="cb-known-help-244">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r344c3-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r344c3_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r344c3_1.svg">
        coelibactin
      </option>
      
      <option value = "svg/knownclusterblast_r344c3_2.svg">
        thiazostatin / watasemycin A / watasemycin B / 2-hydroxyphenylthiazoline enantiopyochelin / isopyochelin
      </option>
      
      <option value = "svg/knownclusterblast_r344c3_3.svg">
        gobichelin A / gobichelin B
      </option>
      
      <option value = "svg/knownclusterblast_r344c3_4.svg">
        mycobactin
      </option>
      
      <option value = "svg/knownclusterblast_r344c3_5.svg">
        amychelin
      </option>
      
      <option value = "svg/knownclusterblast_r344c3_6.svg">
        nataxazole
      </option>
      
      <option value = "svg/knownclusterblast_r344c3_7.svg">
        A33853
      </option>
      
      <option value = "svg/knownclusterblast_r344c3_8.svg">
        caboxamycin
      </option>
      
      <option value = "svg/knownclusterblast_r344c3_9.svg">
        caboxamycin
      </option>
      
      <option value = "svg/knownclusterblast_r344c3_10.svg">
        SF2575
      </option>
      
    </select>
    <button id="knownclusterblast-r344c3-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r344c3-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r344c3-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-245"></div> <span class="help-tooltip" id="cb-sub-help-245">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r344c3-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-246"></div> <span class="help-tooltip" id="MIBiG-help-246">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r344c3-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r344c3-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r344c3"></div>
    <div id="comparison-MIBiG-r344c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000324.1: 8504460-8529581">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000324/index.html#r1c1" target="_new">BGC0000324.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.86 (id:0.66, order:1.00, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
         
        
        <td>0.64</td>
        <td>NRP</td>
        
        <td>coelibactin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.43, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000373.1: 6204479-6222797">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000373/index.html#r1c1" target="_new">BGC0000373.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.48, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.31</td>
        <td>NRP</td>
        
        <td>holomycin</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000412.1: 804-15023">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000412/index.html#r1c1" target="_new">BGC0000412.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.46, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>pyochelin</td>
        
        
        <td>Pseudomonas aeruginosa PAO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002091.1: 3593-21565">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002091/index.html#r1c1" target="_new">BGC0002091.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>holomycin</td>
        
        
        <td>Yersinia ruckeri ATCC 29473</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001469.1: 252908-282960">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001469/index.html#r1c1" target="_new">BGC0001469.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>marthiapeptide A</td>
        
        
        <td>Brevibacillus sp. Leaf182</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001343.1: 61-19132">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001343/index.html#r1c1" target="_new">BGC0001343.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Polyketide</td>
        
        <td>aurachin A, aurachin D</td>
        
        
        <td>Stigmatella aurantiaca Sg a15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890.1: 1392460-1420815">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.49, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
        
        
        <td>Vibrio campbellii ATCC BAA-1116</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001843.1: 2526145-2583676">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001843/index.html#r1c1" target="_new">BGC0001843.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.45, order:0.59, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>tridecaptin M</td>
        
        
        <td>Paenibacillus sp. M-152</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r344c3-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000324.1: 8504460-8529581">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000324/index.html#r1c1" target="_new">BGC0000324.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.86 (id:0.66, order:1.00, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
      <td>0.86</td>
      <td>NRP</td>
      <td>coelibactin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.43, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000373.1: 6204479-6222797">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000373/index.html#r1c1" target="_new">BGC0000373.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.48, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP</td>
      <td>holomycin</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000412.1: 804-15023">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000412/index.html#r1c1" target="_new">BGC0000412.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.46, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>pyochelin</td>
      <td>Pseudomonas aeruginosa PAO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002091.1: 3593-21565">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002091/index.html#r1c1" target="_new">BGC0002091.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>holomycin</td>
      <td>Yersinia ruckeri ATCC 29473</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001469.1: 252908-282960">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001469/index.html#r1c1" target="_new">BGC0001469.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.46, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>marthiapeptide A</td>
      <td>Brevibacillus sp. Leaf182</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001343.1: 61-19132">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001343/index.html#r1c1" target="_new">BGC0001343.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>aurachin A, aurachin D</td>
      <td>Stigmatella aurantiaca Sg a15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890.1: 1392460-1420815">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.49, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
      <td>Vibrio campbellii ATCC BAA-1116</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001843.1: 2526145-2583676">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001843/index.html#r1c1" target="_new">BGC0001843.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.45, order:0.59, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>NRP</td>
      <td>tridecaptin M</td>
      <td>Paenibacillus sp. M-152</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r344c3-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-247"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-247">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r344c3-pfam-details-svg" data-filename="c00344_NODE_51.._r344c3_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r344c3-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r344c3-nrps_pks_products" data-name="r344c3-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r344c3-nrps_pks" data-name="r344c3-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r344c3-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-241"></div> <span class="help-tooltip" id="nrps-structures-help-241">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 3, location 74571 - 126897:
          <div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r344c3-3" class="smiles-canvas" data-smiles="NC(CS)C(=O)NC(CS)C(=O)NC(CS)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(cys - cys) + (cys)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys,cys&amp;nrps2=cys" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*,cys*&amp;nrps2=cys*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r344c3-3-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CS)C(=O)NC(CS)C(=O)NC(CS)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r344c3-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-242"></div> <span class="help-tooltip" id="nrps-monomers-help-242">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds94571_100100</span></strong>: cys
      <div class="collapser collapser-target-cds94571_100100 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (543..942)</strong>: cys
          <div class="collapser collapser-target-cds94571_100100 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  cys
              <div class="collapser collapser-target-cds94571_100100-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Small clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Single AA prediction:</dt>
   <dd>cys</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLYNLSLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>cys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds100096_106897</span></strong>: cys - cys
      <div class="collapser collapser-target-cds100096_106897 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys,cys" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*,cys*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (581..976)</strong>: cys
          <div class="collapser collapser-target-cds100096_106897 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  cys
              <div class="collapser collapser-target-cds100096_106897-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DlwQLGLVhK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1642..2068)</strong>: cys
          <div class="collapser collapser-target-cds100096_106897 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  cys
              <div class="collapser collapser-target-cds100096_106897-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Small clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Single AA prediction:</dt>
   <dd>cys</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLYNLSmIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>cys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds107150_109046</span></strong>: X
      <div class="collapser collapser-target-cds107150_109046 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (8..424)</strong>: X
          <div class="collapser collapser-target-cds107150_109046 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds107150_109046-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>dhb, sal</dd>
   <dt>Small clusters prediction:</dt>
   <dd>dhb, sal</dd>
   <dt>Single AA prediction:</dt>
   <dd>dhb</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">AfPsQGViNK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>dhb</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r381c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00381_NODE_47.. - Region 1 - butyrolactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-331"></div> <span class="help-tooltip" id="region_svg-help-331">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00381_NODE_47...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r381c1-svg" data-filename="c00381_NODE_47.._r381c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1,842 - 8,461 nt. (total: 6,620 nt)
        
        <a class="cluster-rules-header" id="r381c1-rules-header" href="#r381c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r381c1-rules">butyrolactone: AfsA</div>
      
      <div class="region-svg-container">
       <div id='r381c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-332"></div> <span class="help-tooltip" id="focus-panel-help-332">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r381c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r381c1-clusterblast" data-name="r381c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r381c1-knownclusterblast" data-name="r381c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r381c1-subclusterblast" data-name="r381c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r381c1-MIBiG-cluster-compare" data-name="r381c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r381c1-pfam-details" data-name="r381c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r381c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-248"></div> <span class="help-tooltip" id="cb-general-help-248">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r381c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r381c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r381c1_1.svg">
        NZ_CP019724 (231807-242133): Streptomyces pactum strain ACT12 chromosome, com... (55% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r381c1_2.svg">
        NZ_CP019724 (8271294-8319558): Streptomyces pactum strain ACT12 chromosome, c... (15% of genes show similarity), T1PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r381c1_3.svg">
        NZ_KZ819184 (4102654-4188512): Streptomyces sp. CS207 scaffold00001, whole ge... (9% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r381c1_4.svg">
        NZ_CP027297 (19263-103747): Streptomyces sp. SGAir0924 chromosome, complete g... (9% of genes show similarity), T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r381c1_5.svg">
        NZ_CP048397 (156576-320082): Streptomyces sp. S4.7 chromosome, complete genome (6% of genes show similarity), NRPS-like,T1PKS,T3PKS,arylpolyene,butyrolactone,phenazine
      </option>
      
      <option value = "svg/clusterblast_r381c1_6.svg">
        NZ_WXVT01000003 (94101-105088): Streptomyces sp. KO7888 A913 AS4 SC10, whole ... (75% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r381c1_7.svg">
        NZ_LMEQ01000019 (2441-13434): Streptomyces sp. Root1310 contig 26, whole geno... (75% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r381c1_8.svg">
        NZ_CP034352 (1037-312401): Streptomyces sp. W1SF4 plasmid p2, complete sequence (7% of genes show similarity), NRPS,NRPS-like,T1PKS,T2PKS,butyrolactone,melanin
      </option>
      
      <option value = "svg/clusterblast_r381c1_9.svg">
        NZ_KQ948433 (288896-399709): Streptomyces antibioticus strain DSM 40234 PRJNA... (7% of genes show similarity), T3PKS,butyrolactone,other,phenazine
      </option>
      
      <option value = "svg/clusterblast_r381c1_10.svg">
        NZ_CP042324 (6881448-6951651): Streptomyces coelicolor A3(2) strain CFB NBC 0... (18% of genes show similarity), T1PKS,butyrolactone
      </option>
      
    </select>
    <button id= "clusterblast-r381c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r381c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r381c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-249"></div> <span class="help-tooltip" id="cb-known-help-249">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r381c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r381c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r381c1_1.svg">
        lactonamycin
      </option>
      
      <option value = "svg/knownclusterblast_r381c1_2.svg">
        griseoviridin / fijimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r381c1_3.svg">
        merochlorin A / merochlorin B / deschloro-merochlorin A / deschloro-merochlorin B / isochloro-merochlorin B / dichloro-merochlorin B / merochlorin D / merochlorin C
      </option>
      
      <option value = "svg/knownclusterblast_r381c1_4.svg">
        prejadomycin / rabelomycin / gaudimycin C / gaudimycin D / UWM6 / gaudimycin A
      </option>
      
      <option value = "svg/knownclusterblast_r381c1_5.svg">
        coelimycin P1
      </option>
      
      <option value = "svg/knownclusterblast_r381c1_6.svg">
        auricin
      </option>
      
      <option value = "svg/knownclusterblast_r381c1_7.svg">
        griseorhodin A
      </option>
      
      <option value = "svg/knownclusterblast_r381c1_8.svg">
        murayaquinone
      </option>
      
      <option value = "svg/knownclusterblast_r381c1_9.svg">
        A-74528
      </option>
      
      <option value = "svg/knownclusterblast_r381c1_10.svg">
        fredericamycin A
      </option>
      
    </select>
    <button id="knownclusterblast-r381c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r381c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r381c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-250"></div> <span class="help-tooltip" id="cb-sub-help-250">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r381c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-251"></div> <span class="help-tooltip" id="MIBiG-help-251">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r381c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r381c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r381c1"></div>
    <div id="comparison-MIBiG-r381c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">butyrolactone</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000849.1: 6890527-6892237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000849/index.html#r1c1" target="_new">BGC0000849.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.46, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>Other,  , (, B, u, t, y, r, o, l, a, c, t, o, n, e, )</td>
        
        <td>SCB1, SCB2, SCB3</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000848.1: 8273443-8274388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000848/index.html#r1c1" target="_new">BGC0000848.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.36</td>
        <td>Other</td>
        
        <td>A-factor</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:0.52, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>γ-butyrolactone</td>
        
        
        <td>Streptomyces acidiscabies</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000193.1: 0-14807">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000193/index.html#r1c1" target="_new">BGC0000193.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Polyketide</td>
        
        <td>aclacinomycin</td>
        
        
        <td>Streptomyces galilaeus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001295.1: 0-18154">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001295/index.html#r1c1" target="_new">BGC0001295.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>cremeomycin</td>
        
        
        <td>Streptomyces cremeus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000503.1: 0-17083">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000503/index.html#r1c1" target="_new">BGC0000503.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.33 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
         
        
        <td>0.09</td>
        <td>RiPP</td>
        
        <td>cinnamycin</td>
        
        
        <td>Streptomyces cinnamoneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001803.1: 3663-28345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001803/index.html#r1c1" target="_new">BGC0001803.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.31 (id:0.46, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
         
        
        <td>0.08</td>
        <td>RiPP</td>
        
        <td>thioviridamide</td>
        
        
        <td>Streptomyces olivoviridis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000625.1: 3663-28345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000625/index.html#r1c1" target="_new">BGC0000625.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.31 (id:0.46, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
         
        
        <td>0.08</td>
        <td>RiPP</td>
        
        <td>thioviridamide</td>
        
        
        <td>Streptomyces olivoviridis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.30 (id:0.43, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
         
        
        <td>0.08</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B</td>
        
        
        <td>Streptomyces sp. WK-5344</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001083.1: 0-57523">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001083/index.html#r1c1" target="_new">BGC0001083.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.28 (id:0.54, order:0.39, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
         
        
        <td>0.07</td>
        <td>Polyketide, Terpene</td>
        
        <td>merochlorin A, merochlorin B, deschloro-merochlorin A, deschloro-merochlorin B, isochloro-merochlorin B, dichloro-merochlorin B, merochlorin D, merochlorin C</td>
        
        
        <td>Streptomyces sp. CNH189</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r381c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000849.1: 6890527-6892237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000849/index.html#r1c1" target="_new">BGC0000849.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.46, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other,  , (, B, u, t, y, r, o, l, a, c, t, o, n, e, )</td>
      <td>SCB1, SCB2, SCB3</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000848.1: 8273443-8274388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000848/index.html#r1c1" target="_new">BGC0000848.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>Other</td>
      <td>A-factor</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:0.52, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>γ-butyrolactone</td>
      <td>Streptomyces acidiscabies</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000193.1: 0-14807">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000193/index.html#r1c1" target="_new">BGC0000193.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Polyketide</td>
      <td>aclacinomycin</td>
      <td>Streptomyces galilaeus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001295.1: 0-18154">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001295/index.html#r1c1" target="_new">BGC0001295.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Other</td>
      <td>cremeomycin</td>
      <td>Streptomyces cremeus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000503.1: 0-17083">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000503/index.html#r1c1" target="_new">BGC0000503.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.33 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
      <td>0.33</td>
      <td>RiPP</td>
      <td>cinnamycin</td>
      <td>Streptomyces cinnamoneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001803.1: 3663-28345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001803/index.html#r1c1" target="_new">BGC0001803.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.31 (id:0.46, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
      <td>0.31</td>
      <td>RiPP</td>
      <td>thioviridamide</td>
      <td>Streptomyces olivoviridis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000625.1: 3663-28345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000625/index.html#r1c1" target="_new">BGC0000625.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.31 (id:0.46, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
      <td>0.31</td>
      <td>RiPP</td>
      <td>thioviridamide</td>
      <td>Streptomyces olivoviridis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.30 (id:0.43, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
      <td>0.30</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B</td>
      <td>Streptomyces sp. WK-5344</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001083.1: 0-57523">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001083/index.html#r1c1" target="_new">BGC0001083.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.28 (id:0.54, order:0.39, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
      <td>0.28</td>
      <td>Polyketide, Terpene</td>
      <td>merochlorin A, merochlorin B, deschloro-merochlorin A, deschloro-merochlorin B, isochloro-merochlorin B, dichloro-merochlorin B, merochlorin D, merochlorin C</td>
      <td>Streptomyces sp. CNH189</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r381c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-252"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-252">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r381c1-pfam-details-svg" data-filename="c00381_NODE_47.._r381c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r381c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>


  <footer class="footer">
    <div class="container">
      <div>
        <img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
      </div>
      <div class="cite-me">
        If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
      </div>
      <div>
        <img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
      </div>
    </div>
  </footer>

  <script src="js/jquery.js"></script>
  <script src="js/antismash.js"></script>
  <script src="regions.js"></script>
  <script>
    $(document).ready(function() {
        viewer["start"](all_regions, details_data, resultsData, recordData);
    })
  </script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink">
 <defs>
  <filter id="inset-shadow">
   <feOffset dx="-2" dy="-2"></feOffset>
   <feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
   <feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
   <feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
   <feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
   <feComponentTransfer in="shadow" result="shadow">
    <feFuncA type="linear" slope=".95"></feFuncA>
   </feComponentTransfer>
   <feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
  </filter>
 </defs>
</svg>
</body>
</html>