
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>6666666.687569 - 28 region(s) - antiSMASH results</title>
  <style>img {width: 20px; height: 20px;}</style>
  <link rel="stylesheet" type="text/css" href="css/bacteria.css">
  <meta property="og:title" content="6666666.687569 - 28 record(s) - 28 region(s)" />
  <meta property="og:description" content="">
  <meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
  <meta property="og:image:alt" content="antiSMASH logo" />
  <meta property="og:image:width" content="600" />
  <meta property="og:image:height" content="600" />
  <meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-58c6726f-1333-4bc2-bcd4-7ae28c19654e/index.html" />
  <meta property="og:type" content="website" />
  <meta name="twitter:card" content="summary" />
  <meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
  <div id="header">
      <nav>
    <div class="main-link">
     <div>
        <a href="https://antismash.secondarymetabolites.org/">
          <img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
        </a>
       </div>
       <div>
        <a href="https://antismash.secondarymetabolites.org/">
          antiSMASH version 6.0.1
        </a>
      </div>
    </div>
    <div class="custom-description"></div>
    <div class="ancillary-links">
      <div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
        <ul class="dropdown-options">
        
        <li><a href="6666666.687569.zip">Download all results</a></li>
        
          <li><a href="6666666.687569.gbk">Download GenBank summary file</a></li>
        
          <li><a href="bacteria-58c6726f-1333-4bc2-bcd4-7ae28c19654e.log">Download log file</a></li>
        
        </ul>
      </div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
      <div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
    </div>
  </nav>
    <div id="buttons">
      <span id="region-type">Select genomic region:</span>
      <div id="regionbuttons">
        <div class="arrow-left" id="prev-region"></div>
        <div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
        <div style="display: flex; flex-wrap: wrap">
        
          
            
            
            <div class="regbutton T3PKS r74c1">
              <a href="#r74c1">74.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r151c1">
              <a href="#r151c1">151.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton siderophore r218c1">
              <a href="#r218c1">218.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton ladderane r232c1">
              <a href="#r232c1">232.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r259c1">
              <a href="#r259c1">259.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton PKS-like r297c1">
              <a href="#r297c1">297.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton butyrolactone r338c1">
              <a href="#r338c1">338.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r543c1">
              <a href="#r543c1">543.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r573c1">
              <a href="#r573c1">573.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r622c1">
              <a href="#r622c1">622.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton melanin r650c1">
              <a href="#r650c1">650.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r798c1">
              <a href="#r798c1">798.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r816c1">
              <a href="#r816c1">816.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton butyrolactone,ectoine hybrid r819c1">
              <a href="#r819c1">819.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton butyrolactone r855c1">
              <a href="#r855c1">855.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS-like r865c1">
              <a href="#r865c1">865.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r901c1">
              <a href="#r901c1">901.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS-like r945c1">
              <a href="#r945c1">945.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r1004c1">
              <a href="#r1004c1">1004.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r1021c1">
              <a href="#r1021c1">1021.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton other r1060c1">
              <a href="#r1060c1">1060.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r1072c1">
              <a href="#r1072c1">1072.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r1113c1">
              <a href="#r1113c1">1113.1</a>
            </div>
          
        
          
            
            
                
            
            <div class="regbutton other,thioamitides,thiopeptide hybrid r1155c1">
              <a href="#r1155c1">1155.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton NRPS r1207c1">
              <a href="#r1207c1">1207.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton siderophore r1227c1">
              <a href="#r1227c1">1227.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton ectoine r1264c1">
              <a href="#r1264c1">1264.1</a>
            </div>
          
        
          
            
            
            <div class="regbutton terpene r1277c1">
              <a href="#r1277c1">1277.1</a>
            </div>
          
        
        </div>
        <div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
      </div>
    </div>
  </div>

  <!-- overview page -->
  
  <div class="page" id="overview">
   
   
     <h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>
   
   <div class="overview-layout">
    <div id="record-tables">
     <div id="single-record-tables">
     <div class="record-overview-header">
        <strong>c00074_CC228A_..</strong> (original name was: CC228A_contig_527) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-74" data-filename="c00074_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-74" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r74c1">
     
     
     <td class="regbutton T3PKS r74c1">
      <a href="#r74c1">Region&nbsp74.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">3,340</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000285/1" target="_blank">flaviolin rhamnoside / 3,3&#39;-diflaviolin / flaviolin</a></td>
      <td>Polyketide:Type III + Saccharide:Oligosaccharide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 33%, #ffffff00 33%)">33%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00151_CC228A_..</strong> (original name was: CC228A_contig_998) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-151" data-filename="c00151_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-151" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r151c1">
     
     
     <td class="regbutton terpene r151c1">
      <a href="#r151c1">Region&nbsp151.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">1,704</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC228A_contig_23</strong> (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-218" data-filename="CC228A_contig_23_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-218" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r218c1">
     
     
     <td class="regbutton siderophore r218c1">
      <a href="#r218c1">Region&nbsp218.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">7,367</td>
     <td class="digits table-split-left">16,762</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000940/1" target="_blank">desferrioxamin B / desferrioxamine E</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 83%, #ffffff00 83%)">83%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC228A_contig_92</strong> (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-232" data-filename="CC228A_contig_92_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-232" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r232c1">
     
     
     <td class="regbutton ladderane r232c1">
      <a href="#r232c1">Region&nbsp232.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ladderane" target="_blank">ladderane</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">9,993</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00259_CC228A_..</strong> (original name was: CC228A_contig_122) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-259" data-filename="c00259_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-259" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r259c1">
     
     
     <td class="regbutton NRPS r259c1">
      <a href="#r259c1">Region&nbsp259.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">8,843</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001193/1" target="_blank">thiolutin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00297_CC228A_..</strong> (original name was: CC228A_contig_190) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-297" data-filename="c00297_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-297" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r297c1">
     
     
     <td class="regbutton PKS-like r297c1">
      <a href="#r297c1">Region&nbsp297.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#pks-like" target="_blank">PKS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,896</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00338_CC228A_..</strong> (original name was: CC228A_contig_322) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-338" data-filename="c00338_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-338" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r338c1">
     
     
     <td class="regbutton butyrolactone r338c1">
      <a href="#r338c1">Region&nbsp338.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">4,939</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00543_CC228A_..</strong> (original name was: CC228A_contig_135) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-543" data-filename="c00543_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-543" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r543c1">
     
     
     <td class="regbutton terpene r543c1">
      <a href="#r543c1">Region&nbsp543.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">8,432</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00573_CC228A_..</strong> (original name was: CC228A_contig_108) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-573" data-filename="c00573_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-573" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r573c1">
     
     
     <td class="regbutton terpene r573c1">
      <a href="#r573c1">Region&nbsp573.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">9,246</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000663/1" target="_blank">hopene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 15%, #ffffff00 15%)">15%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00622_CC228A_..</strong> (original name was: CC228A_contig_141) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-622" data-filename="c00622_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-622" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r622c1">
     
     
     <td class="regbutton NRPS r622c1">
      <a href="#r622c1">Region&nbsp622.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">8,276</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001814/1" target="_blank">ulleungmycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 5%, #ffffff00 5%)">5%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00650_CC228A_..</strong> (original name was: CC228A_contig_756) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-650" data-filename="c00650_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-650" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r650c1">
     
     
     <td class="regbutton melanin r650c1">
      <a href="#r650c1">Region&nbsp650.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#melanin" target="_blank">melanin</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">2,360</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000908/1" target="_blank">melanin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00798_CC228A_..</strong> (original name was: CC228A_contig_494) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-798" data-filename="c00798_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-798" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r798c1">
     
     
     <td class="regbutton terpene r798c1">
      <a href="#r798c1">Region&nbsp798.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">3,588</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000658/1" target="_blank">2-methylisoborneol</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00816_CC228A_..</strong> (original name was: CC228A_contig_172) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-816" data-filename="c00816_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-816" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r816c1">
     
     
     <td class="regbutton NRPS r816c1">
      <a href="#r816c1">Region&nbsp816.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">7,413</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC228A_contig_35</strong> (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-819" data-filename="CC228A_contig_35_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-819" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r819c1">
     
     
         
     
     <td class="regbutton butyrolactone,ectoine hybrid r819c1">
      <a href="#r819c1">Region&nbsp819.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">946</td>
     <td class="digits table-split-left">15,428</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001778/1" target="_blank">showdomycin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 41%, #ffffff00 41%)">41%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00855_CC228A_..</strong> (original name was: CC228A_contig_347) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-855" data-filename="c00855_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-855" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r855c1">
     
     
     <td class="regbutton butyrolactone r855c1">
      <a href="#r855c1">Region&nbsp855.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">4,673</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000038/1" target="_blank">coelimycin P1</a></td>
      <td>Polyketide:Modular type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 8%, #ffffff00 8%)">8%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00865_CC228A_..</strong> (original name was: CC228A_contig_270) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-865" data-filename="c00865_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-865" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r865c1">
     
     
     <td class="regbutton NRPS-like r865c1">
      <a href="#r865c1">Region&nbsp865.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">5,462</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001877/1" target="_blank">cyphomycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 2%, #ffffff00 2%)">2%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00901_CC228A_..</strong> (original name was: CC228A_contig_859) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-901" data-filename="c00901_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-901" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r901c1">
     
     
     <td class="regbutton terpene r901c1">
      <a href="#r901c1">Region&nbsp901.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">2,097</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000633/1" target="_blank">carotenoid</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00945_CC228A_..</strong> (original name was: CC228A_contig_947) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-945" data-filename="c00945_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-945" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r945c1">
     
     
     <td class="regbutton NRPS-like r945c1">
      <a href="#r945c1">Region&nbsp945.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">1,836</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000379/1" target="_blank">glycinocin A</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c01004_CC228A_..</strong> (original name was: CC228A_contig_377) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-1004" data-filename="c01004_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-1004" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r1004c1">
     
     
     <td class="regbutton NRPS r1004c1">
      <a href="#r1004c1">Region&nbsp1004.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">4,516</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c01021_CC228A_..</strong> (original name was: CC228A_contig_815) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-1021" data-filename="c01021_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-1021" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r1021c1">
     
     
     <td class="regbutton NRPS r1021c1">
      <a href="#r1021c1">Region&nbsp1021.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">2,206</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c01060_CC228A_..</strong> (original name was: CC228A_contig_279) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-1060" data-filename="c01060_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-1060" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r1060c1">
     
     
     <td class="regbutton other r1060c1">
      <a href="#r1060c1">Region&nbsp1060.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">5,358</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c01072_CC228A_..</strong> (original name was: CC228A_contig_262) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-1072" data-filename="c01072_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-1072" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r1072c1">
     
     
     <td class="regbutton terpene r1072c1">
      <a href="#r1072c1">Region&nbsp1072.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">5,572</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001181/1" target="_blank">geosmin</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c01113_CC228A_..</strong> (original name was: CC228A_contig_391) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-1113" data-filename="c01113_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-1113" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r1113c1">
     
     
     <td class="regbutton terpene r1113c1">
      <a href="#r1113c1">Region&nbsp1113.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">4,371</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC228A_contig_2</strong> (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-1155" data-filename="CC228A_contig_2_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-1155" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r1155c1">
     
     
         
     
     <td class="regbutton other,thioamitides,thiopeptide hybrid r1155c1">
      <a href="#r1155c1">Region&nbsp1155.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thiopeptide" target="_blank">thiopeptide</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thioamitides" target="_blank">thioamitides</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">27,655</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000315/1" target="_blank">CDA1b / CDA2a / CDA2b / CDA3a / CDA3b / CDA4a / CDA4b</a></td>
      <td>NRP:Ca+-dependent lipopeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 17%, #ffffff00 17%)">17%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>CC228A_contig_68</strong> (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-1207" data-filename="CC228A_contig_68_overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-1207" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r1207c1">
     
     
     <td class="regbutton NRPS r1207c1">
      <a href="#r1207c1">Region&nbsp1207.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,095</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001792/1" target="_blank">surugamide A / surugamide D</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 14%, #ffffff00 14%)">14%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c01227_CC228A_..</strong> (original name was: CC228A_contig_442) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-1227" data-filename="c01227_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-1227" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r1227c1">
     
     
     <td class="regbutton siderophore r1227c1">
      <a href="#r1227c1">Region&nbsp1227.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">3,938</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001593/1" target="_blank">ficellomycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 3%, #ffffff00 3%)">3%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c01264_CC228A_..</strong> (original name was: CC228A_contig_223) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-1264" data-filename="c01264_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-1264" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r1264c1">
     
     
     <td class="regbutton ectoine r1264c1">
      <a href="#r1264c1">Region&nbsp1264.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,212</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000853/1" target="_blank">ectoine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c01277_CC228A_..</strong> (original name was: CC228A_contig_149) (Streptomyces sp. CC228A)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-1277" data-filename="c01277_CC228A_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-1277" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r1277c1">
     
     
     <td class="regbutton terpene r1277c1">
      <a href="#r1277c1">Region&nbsp1277.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">8,010</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div>
     </div>
     
     <div id="compact-record-table" style="display: none;">
      <div class="record-table">
       <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 
 
  <tr class="linked-row odd" data-anchor="#r74c1">
     
     
     <td class="regbutton T3PKS r74c1">
      <a href="#r74c1">Region&nbsp74.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">3,340</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000285/1" target="_blank">flaviolin rhamnoside / 3,3&#39;-diflaviolin / flaviolin</a></td>
      <td>Polyketide:Type III + Saccharide:Oligosaccharide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 33%, #ffffff00 33%)">33%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r151c1">
     
     
     <td class="regbutton terpene r151c1">
      <a href="#r151c1">Region&nbsp151.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">1,704</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r218c1">
     
     
     <td class="regbutton siderophore r218c1">
      <a href="#r218c1">Region&nbsp218.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">7,367</td>
     <td class="digits table-split-left">16,762</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000940/1" target="_blank">desferrioxamin B / desferrioxamine E</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 83%, #ffffff00 83%)">83%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r232c1">
     
     
     <td class="regbutton ladderane r232c1">
      <a href="#r232c1">Region&nbsp232.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ladderane" target="_blank">ladderane</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">9,993</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r259c1">
     
     
     <td class="regbutton NRPS r259c1">
      <a href="#r259c1">Region&nbsp259.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">8,843</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001193/1" target="_blank">thiolutin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r297c1">
     
     
     <td class="regbutton PKS-like r297c1">
      <a href="#r297c1">Region&nbsp297.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#pks-like" target="_blank">PKS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,896</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r338c1">
     
     
     <td class="regbutton butyrolactone r338c1">
      <a href="#r338c1">Region&nbsp338.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">4,939</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r543c1">
     
     
     <td class="regbutton terpene r543c1">
      <a href="#r543c1">Region&nbsp543.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">8,432</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r573c1">
     
     
     <td class="regbutton terpene r573c1">
      <a href="#r573c1">Region&nbsp573.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">9,246</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000663/1" target="_blank">hopene</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 15%, #ffffff00 15%)">15%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r622c1">
     
     
     <td class="regbutton NRPS r622c1">
      <a href="#r622c1">Region&nbsp622.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">8,276</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001814/1" target="_blank">ulleungmycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 5%, #ffffff00 5%)">5%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r650c1">
     
     
     <td class="regbutton melanin r650c1">
      <a href="#r650c1">Region&nbsp650.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#melanin" target="_blank">melanin</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">2,360</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000908/1" target="_blank">melanin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r798c1">
     
     
     <td class="regbutton terpene r798c1">
      <a href="#r798c1">Region&nbsp798.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">3,588</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000658/1" target="_blank">2-methylisoborneol</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r816c1">
     
     
     <td class="regbutton NRPS r816c1">
      <a href="#r816c1">Region&nbsp816.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">7,413</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r819c1">
     
     
         
     
     <td class="regbutton butyrolactone,ectoine hybrid r819c1">
      <a href="#r819c1">Region&nbsp819.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">946</td>
     <td class="digits table-split-left">15,428</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001778/1" target="_blank">showdomycin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 41%, #ffffff00 41%)">41%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r855c1">
     
     
     <td class="regbutton butyrolactone r855c1">
      <a href="#r855c1">Region&nbsp855.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">4,673</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000038/1" target="_blank">coelimycin P1</a></td>
      <td>Polyketide:Modular type I</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 8%, #ffffff00 8%)">8%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r865c1">
     
     
     <td class="regbutton NRPS-like r865c1">
      <a href="#r865c1">Region&nbsp865.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">5,462</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001877/1" target="_blank">cyphomycin</a></td>
      <td>Polyketide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 2%, #ffffff00 2%)">2%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r901c1">
     
     
     <td class="regbutton terpene r901c1">
      <a href="#r901c1">Region&nbsp901.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">2,097</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000633/1" target="_blank">carotenoid</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r945c1">
     
     
     <td class="regbutton NRPS-like r945c1">
      <a href="#r945c1">Region&nbsp945.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">1,836</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000379/1" target="_blank">glycinocin A</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r1004c1">
     
     
     <td class="regbutton NRPS r1004c1">
      <a href="#r1004c1">Region&nbsp1004.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">4,516</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r1021c1">
     
     
     <td class="regbutton NRPS r1021c1">
      <a href="#r1021c1">Region&nbsp1021.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">2,206</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r1060c1">
     
     
     <td class="regbutton other r1060c1">
      <a href="#r1060c1">Region&nbsp1060.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">5,358</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r1072c1">
     
     
     <td class="regbutton terpene r1072c1">
      <a href="#r1072c1">Region&nbsp1072.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">5,572</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001181/1" target="_blank">geosmin</a></td>
      <td>Terpene</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r1113c1">
     
     
     <td class="regbutton terpene r1113c1">
      <a href="#r1113c1">Region&nbsp1113.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">4,371</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r1155c1">
     
     
         
     
     <td class="regbutton other,thioamitides,thiopeptide hybrid r1155c1">
      <a href="#r1155c1">Region&nbsp1155.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thiopeptide" target="_blank">thiopeptide</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thioamitides" target="_blank">thioamitides</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">27,655</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000315/1" target="_blank">CDA1b / CDA2a / CDA2b / CDA3a / CDA3b / CDA4a / CDA4b</a></td>
      <td>NRP:Ca+-dependent lipopeptide</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 17%, #ffffff00 17%)">17%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r1207c1">
     
     
     <td class="regbutton NRPS r1207c1">
      <a href="#r1207c1">Region&nbsp1207.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">11,095</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001792/1" target="_blank">surugamide A / surugamide D</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 14%, #ffffff00 14%)">14%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r1227c1">
     
     
     <td class="regbutton siderophore r1227c1">
      <a href="#r1227c1">Region&nbsp1227.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">3,938</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001593/1" target="_blank">ficellomycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 3%, #ffffff00 3%)">3%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r1264c1">
     
     
     <td class="regbutton ectoine r1264c1">
      <a href="#r1264c1">Region&nbsp1264.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ectoine" target="_blank">ectoine</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,212</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000853/1" target="_blank">ectoine</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r1277c1">
     
     
     <td class="regbutton terpene r1277c1">
      <a href="#r1277c1">Region&nbsp1277.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">8,010</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
 </tbody>
</table>
      </div>
     </div>
     
     
     <dl id="regionless-records">
      <dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>
       
         <dd><strong>c00001_CC228A_..</strong> (original name was: CC228A_contig_889) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00002_CC228A_..</strong> (original name was: CC228A_contig_318) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00003_CC228A_..</strong> (original name was: CC228A_contig_1130) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00004_CC228A_..</strong> (original name was: CC228A_contig_795) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00005_CC228A_..</strong> (original name was: CC228A_contig_1003) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00006_CC228A_..</strong> (original name was: CC228A_contig_790) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00007_CC228A_..</strong> (original name was: CC228A_contig_1017) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00008_CC228A_..</strong> (original name was: CC228A_contig_195) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00009_CC228A_..</strong> (original name was: CC228A_contig_151) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00010_CC228A_..</strong> (original name was: CC228A_contig_875) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00011_CC228A_..</strong> (original name was: CC228A_contig_974) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00012_CC228A_..</strong> (original name was: CC228A_contig_1126) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00013_CC228A_..</strong> (original name was: CC228A_contig_594) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00014_CC228A_..</strong> (original name was: CC228A_contig_1021) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_99</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00016_CC228A_..</strong> (original name was: CC228A_contig_764) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00017_CC228A_..</strong> (original name was: CC228A_contig_736) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00018_CC228A_..</strong> (original name was: CC228A_contig_240) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00019_CC228A_..</strong> (original name was: CC228A_contig_235) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00020_CC228A_..</strong> (original name was: CC228A_contig_464) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00021_CC228A_..</strong> (original name was: CC228A_contig_1159) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00022_CC228A_..</strong> (original name was: CC228A_contig_1024) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00023_CC228A_..</strong> (original name was: CC228A_contig_915) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00024_CC228A_..</strong> (original name was: CC228A_contig_1079) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00025_CC228A_..</strong> (original name was: CC228A_contig_677) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00026_CC228A_..</strong> (original name was: CC228A_contig_562) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00027_CC228A_..</strong> (original name was: CC228A_contig_257) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00028_CC228A_..</strong> (original name was: CC228A_contig_1083) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00029_CC228A_..</strong> (original name was: CC228A_contig_209) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00030_CC228A_..</strong> (original name was: CC228A_contig_1134) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00031_CC228A_..</strong> (original name was: CC228A_contig_560) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00032_CC228A_..</strong> (original name was: CC228A_contig_227) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00033_CC228A_..</strong> (original name was: CC228A_contig_1030) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00034_CC228A_..</strong> (original name was: CC228A_contig_410) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00035_CC228A_..</strong> (original name was: CC228A_contig_999) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00036_CC228A_..</strong> (original name was: CC228A_contig_663) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00037_CC228A_..</strong> (original name was: CC228A_contig_569) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00038_CC228A_..</strong> (original name was: CC228A_contig_513) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00039_CC228A_..</strong> (original name was: CC228A_contig_573) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00040_CC228A_..</strong> (original name was: CC228A_contig_413) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00041_CC228A_..</strong> (original name was: CC228A_contig_212) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00042_CC228A_..</strong> (original name was: CC228A_contig_992) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00043_CC228A_..</strong> (original name was: CC228A_contig_500) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00044_CC228A_..</strong> (original name was: CC228A_contig_930) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00045_CC228A_..</strong> (original name was: CC228A_contig_1106) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00046_CC228A_..</strong> (original name was: CC228A_contig_662) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00047_CC228A_..</strong> (original name was: CC228A_contig_338) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00048_CC228A_..</strong> (original name was: CC228A_contig_667) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00049_CC228A_..</strong> (original name was: CC228A_contig_1153) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00050_CC228A_..</strong> (original name was: CC228A_contig_278) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00051_CC228A_..</strong> (original name was: CC228A_contig_539) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00052_CC228A_..</strong> (original name was: CC228A_contig_404) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_63</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00054_CC228A_..</strong> (original name was: CC228A_contig_1246) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00055_CC228A_..</strong> (original name was: CC228A_contig_1183) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00056_CC228A_..</strong> (original name was: CC228A_contig_649) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00057_CC228A_..</strong> (original name was: CC228A_contig_283) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00058_CC228A_..</strong> (original name was: CC228A_contig_1230) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00059_CC228A_..</strong> (original name was: CC228A_contig_350) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00060_CC228A_..</strong> (original name was: CC228A_contig_823) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00061_CC228A_..</strong> (original name was: CC228A_contig_177) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00062_CC228A_..</strong> (original name was: CC228A_contig_670) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00063_CC228A_..</strong> (original name was: CC228A_contig_485) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00064_CC228A_..</strong> (original name was: CC228A_contig_314) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00065_CC228A_..</strong> (original name was: CC228A_contig_929) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00066_CC228A_..</strong> (original name was: CC228A_contig_969) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00067_CC228A_..</strong> (original name was: CC228A_contig_762) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00068_CC228A_..</strong> (original name was: CC228A_contig_587) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00069_CC228A_..</strong> (original name was: CC228A_contig_343) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00070_CC228A_..</strong> (original name was: CC228A_contig_1049) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00071_CC228A_..</strong> (original name was: CC228A_contig_740) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00072_CC228A_..</strong> (original name was: CC228A_contig_328) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00073_CC228A_..</strong> (original name was: CC228A_contig_220) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00075_CC228A_..</strong> (original name was: CC228A_contig_702) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00076_CC228A_..</strong> (original name was: CC228A_contig_728) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00077_CC228A_..</strong> (original name was: CC228A_contig_383) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00078_CC228A_..</strong> (original name was: CC228A_contig_1194) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00079_CC228A_..</strong> (original name was: CC228A_contig_616) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00080_CC228A_..</strong> (original name was: CC228A_contig_165) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00081_CC228A_..</strong> (original name was: CC228A_contig_437) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00082_CC228A_..</strong> (original name was: CC228A_contig_1137) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00083_CC228A_..</strong> (original name was: CC228A_contig_175) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_83</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00085_CC228A_..</strong> (original name was: CC228A_contig_696) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00086_CC228A_..</strong> (original name was: CC228A_contig_456) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00087_CC228A_..</strong> (original name was: CC228A_contig_715) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00088_CC228A_..</strong> (original name was: CC228A_contig_499) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_71</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00090_CC228A_..</strong> (original name was: CC228A_contig_308) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00091_CC228A_..</strong> (original name was: CC228A_contig_374) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_94</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00093_CC228A_..</strong> (original name was: CC228A_contig_1091) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00094_CC228A_..</strong> (original name was: CC228A_contig_707) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00095_CC228A_..</strong> (original name was: CC228A_contig_1170) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00096_CC228A_..</strong> (original name was: CC228A_contig_1166) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00097_CC228A_..</strong> (original name was: CC228A_contig_651) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00098_CC228A_..</strong> (original name was: CC228A_contig_957) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00099_CC228A_..</strong> (original name was: CC228A_contig_640) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00100_CC228A_..</strong> (original name was: CC228A_contig_778) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_28</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00102_CC228A_..</strong> (original name was: CC228A_contig_636) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00103_CC228A_..</strong> (original name was: CC228A_contig_896) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00104_CC228A_..</strong> (original name was: CC228A_contig_785) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00105_CC228A_..</strong> (original name was: CC228A_contig_1067) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00106_CC228A_..</strong> (original name was: CC228A_contig_124) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00107_CC228A_..</strong> (original name was: CC228A_contig_600) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00108_CC228A_..</strong> (original name was: CC228A_contig_943) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00109_CC228A_..</strong> (original name was: CC228A_contig_386) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00110_CC228A_..</strong> (original name was: CC228A_contig_115) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00111_CC228A_..</strong> (original name was: CC228A_contig_286) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00112_CC228A_..</strong> (original name was: CC228A_contig_832) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00113_CC228A_..</strong> (original name was: CC228A_contig_1211) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00114_CC228A_..</strong> (original name was: CC228A_contig_107) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00115_CC228A_..</strong> (original name was: CC228A_contig_604) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00116_CC228A_..</strong> (original name was: CC228A_contig_492) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00117_CC228A_..</strong> (original name was: CC228A_contig_1115) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_8</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00119_CC228A_..</strong> (original name was: CC228A_contig_925) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00120_CC228A_..</strong> (original name was: CC228A_contig_483) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00121_CC228A_..</strong> (original name was: CC228A_contig_153) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00122_CC228A_..</strong> (original name was: CC228A_contig_134) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00123_CC228A_..</strong> (original name was: CC228A_contig_474) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00124_CC228A_..</strong> (original name was: CC228A_contig_1188) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00125_CC228A_..</strong> (original name was: CC228A_contig_725) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00126_CC228A_..</strong> (original name was: CC228A_contig_230) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00127_CC228A_..</strong> (original name was: CC228A_contig_846) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00128_CC228A_..</strong> (original name was: CC228A_contig_109) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00129_CC228A_..</strong> (original name was: CC228A_contig_709) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00130_CC228A_..</strong> (original name was: CC228A_contig_984) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00131_CC228A_..</strong> (original name was: CC228A_contig_139) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00132_CC228A_..</strong> (original name was: CC228A_contig_557) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00133_CC228A_..</strong> (original name was: CC228A_contig_520) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00134_CC228A_..</strong> (original name was: CC228A_contig_1146) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00135_CC228A_..</strong> (original name was: CC228A_contig_960) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00136_CC228A_..</strong> (original name was: CC228A_contig_549) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00137_CC228A_..</strong> (original name was: CC228A_contig_1208) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00138_CC228A_..</strong> (original name was: CC228A_contig_626) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00139_CC228A_..</strong> (original name was: CC228A_contig_751) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_3</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00141_CC228A_..</strong> (original name was: CC228A_contig_469) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00142_CC228A_..</strong> (original name was: CC228A_contig_963) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_44</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00144_CC228A_..</strong> (original name was: CC228A_contig_307) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00145_CC228A_..</strong> (original name was: CC228A_contig_839) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00146_CC228A_..</strong> (original name was: CC228A_contig_788) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00147_CC228A_..</strong> (original name was: CC228A_contig_394) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00148_CC228A_..</strong> (original name was: CC228A_contig_359) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00149_CC228A_..</strong> (original name was: CC228A_contig_882) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00150_CC228A_..</strong> (original name was: CC228A_contig_1035) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00152_CC228A_..</strong> (original name was: CC228A_contig_760) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00153_CC228A_..</strong> (original name was: CC228A_contig_1060) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00154_CC228A_..</strong> (original name was: CC228A_contig_218) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00155_CC228A_..</strong> (original name was: CC228A_contig_825) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00156_CC228A_..</strong> (original name was: CC228A_contig_792) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00157_CC228A_..</strong> (original name was: CC228A_contig_1223) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00158_CC228A_..</strong> (original name was: CC228A_contig_749) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00159_CC228A_..</strong> (original name was: CC228A_contig_412) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00160_CC228A_..</strong> (original name was: CC228A_contig_1278) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00161_CC228A_..</strong> (original name was: CC228A_contig_1139) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00162_CC228A_..</strong> (original name was: CC228A_contig_431) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00163_CC228A_..</strong> (original name was: CC228A_contig_241) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00164_CC228A_..</strong> (original name was: CC228A_contig_689) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00165_CC228A_..</strong> (original name was: CC228A_contig_1074) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00166_CC228A_..</strong> (original name was: CC228A_contig_1136) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00167_CC228A_..</strong> (original name was: CC228A_contig_634) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_46</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00169_CC228A_..</strong> (original name was: CC228A_contig_757) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00170_CC228A_..</strong> (original name was: CC228A_contig_564) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00171_CC228A_..</strong> (original name was: CC228A_contig_602) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00172_CC228A_..</strong> (original name was: CC228A_contig_462) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00173_CC228A_..</strong> (original name was: CC228A_contig_158) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00174_CC228A_..</strong> (original name was: CC228A_contig_884) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00175_CC228A_..</strong> (original name was: CC228A_contig_803) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00176_CC228A_..</strong> (original name was: CC228A_contig_632) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00177_CC228A_..</strong> (original name was: CC228A_contig_435) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_38</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00179_CC228A_..</strong> (original name was: CC228A_contig_1001) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00180_CC228A_..</strong> (original name was: CC228A_contig_291) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00181_CC228A_..</strong> (original name was: CC228A_contig_933) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00182_CC228A_..</strong> (original name was: CC228A_contig_908) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00183_CC228A_..</strong> (original name was: CC228A_contig_727) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00184_CC228A_..</strong> (original name was: CC228A_contig_1213) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00185_CC228A_..</strong> (original name was: CC228A_contig_200) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_11</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00187_CC228A_..</strong> (original name was: CC228A_contig_1203) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00188_CC228A_..</strong> (original name was: CC228A_contig_1077) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00189_CC228A_..</strong> (original name was: CC228A_contig_617) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00190_CC228A_..</strong> (original name was: CC228A_contig_540) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00191_CC228A_..</strong> (original name was: CC228A_contig_459) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00192_CC228A_..</strong> (original name was: CC228A_contig_371) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00193_CC228A_..</strong> (original name was: CC228A_contig_610) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00194_CC228A_..</strong> (original name was: CC228A_contig_316) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00195_CC228A_..</strong> (original name was: CC228A_contig_1148) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00196_CC228A_..</strong> (original name was: CC228A_contig_419) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00197_CC228A_..</strong> (original name was: CC228A_contig_381) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00198_CC228A_..</strong> (original name was: CC228A_contig_1033) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00199_CC228A_..</strong> (original name was: CC228A_contig_913) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00200_CC228A_..</strong> (original name was: CC228A_contig_281) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00201_CC228A_..</strong> (original name was: CC228A_contig_868) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00202_CC228A_..</strong> (original name was: CC228A_contig_1097) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00203_CC228A_..</strong> (original name was: CC228A_contig_851) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00204_CC228A_..</strong> (original name was: CC228A_contig_742) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00205_CC228A_..</strong> (original name was: CC228A_contig_367) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00206_CC228A_..</strong> (original name was: CC228A_contig_1258) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00207_CC228A_..</strong> (original name was: CC228A_contig_959) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00208_CC228A_..</strong> (original name was: CC228A_contig_665) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00209_CC228A_..</strong> (original name was: CC228A_contig_919) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00210_CC228A_..</strong> (original name was: CC228A_contig_1199) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00211_CC228A_..</strong> (original name was: CC228A_contig_393) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00212_CC228A_..</strong> (original name was: CC228A_contig_233) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00213_CC228A_..</strong> (original name was: CC228A_contig_773) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00214_CC228A_..</strong> (original name was: CC228A_contig_719) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00215_CC228A_..</strong> (original name was: CC228A_contig_1101) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00216_CC228A_..</strong> (original name was: CC228A_contig_467) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00217_CC228A_..</strong> (original name was: CC228A_contig_1220) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00219_CC228A_..</strong> (original name was: CC228A_contig_1081) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00220_CC228A_..</strong> (original name was: CC228A_contig_887) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00221_CC228A_..</strong> (original name was: CC228A_contig_360) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00222_CC228A_..</strong> (original name was: CC228A_contig_575) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00223_CC228A_..</strong> (original name was: CC228A_contig_311) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00224_CC228A_..</strong> (original name was: CC228A_contig_188) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00225_CC228A_..</strong> (original name was: CC228A_contig_1181) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00226_CC228A_..</strong> (original name was: CC228A_contig_268) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00227_CC228A_..</strong> (original name was: CC228A_contig_811) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00228_CC228A_..</strong> (original name was: CC228A_contig_660) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00229_CC228A_..</strong> (original name was: CC228A_contig_130) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00230_CC228A_..</strong> (original name was: CC228A_contig_361) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00231_CC228A_..</strong> (original name was: CC228A_contig_269) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00233_CC228A_..</strong> (original name was: CC228A_contig_1069) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00234_CC228A_..</strong> (original name was: CC228A_contig_524) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00235_CC228A_..</strong> (original name was: CC228A_contig_532) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00236_CC228A_..</strong> (original name was: CC228A_contig_509) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00237_CC228A_..</strong> (original name was: CC228A_contig_173) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00238_CC228A_..</strong> (original name was: CC228A_contig_1113) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00239_CC228A_..</strong> (original name was: CC228A_contig_254) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00240_CC228A_..</strong> (original name was: CC228A_contig_1172) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00241_CC228A_..</strong> (original name was: CC228A_contig_950) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00242_CC228A_..</strong> (original name was: CC228A_contig_642) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00243_CC228A_..</strong> (original name was: CC228A_contig_352) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00244_CC228A_..</strong> (original name was: CC228A_contig_368) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00245_CC228A_..</strong> (original name was: CC228A_contig_1109) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_74</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00247_CC228A_..</strong> (original name was: CC228A_contig_369) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_86</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00249_CC228A_..</strong> (original name was: CC228A_contig_445) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00250_CC228A_..</strong> (original name was: CC228A_contig_1255) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00251_CC228A_..</strong> (original name was: CC228A_contig_627) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00252_CC228A_..</strong> (original name was: CC228A_contig_1140) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00253_CC228A_..</strong> (original name was: CC228A_contig_873) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00254_CC228A_..</strong> (original name was: CC228A_contig_1011) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_34</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00256_CC228A_..</strong> (original name was: CC228A_contig_787) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00257_CC228A_..</strong> (original name was: CC228A_contig_555) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00258_CC228A_..</strong> (original name was: CC228A_contig_182) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00260_CC228A_..</strong> (original name was: CC228A_contig_945) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00261_CC228A_..</strong> (original name was: CC228A_contig_834) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00262_CC228A_..</strong> (original name was: CC228A_contig_163) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00263_CC228A_..</strong> (original name was: CC228A_contig_843) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00264_CC228A_..</strong> (original name was: CC228A_contig_105) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00265_CC228A_..</strong> (original name was: CC228A_contig_421) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00266_CC228A_..</strong> (original name was: CC228A_contig_207) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00267_CC228A_..</strong> (original name was: CC228A_contig_946) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00268_CC228A_..</strong> (original name was: CC228A_contig_606) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_53</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00270_CC228A_..</strong> (original name was: CC228A_contig_1218) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00271_CC228A_..</strong> (original name was: CC228A_contig_624) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00272_CC228A_..</strong> (original name was: CC228A_contig_197) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00273_CC228A_..</strong> (original name was: CC228A_contig_353) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00274_CC228A_..</strong> (original name was: CC228A_contig_1260) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00275_CC228A_..</strong> (original name was: CC228A_contig_690) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00276_CC228A_..</strong> (original name was: CC228A_contig_976) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00277_CC228A_..</strong> (original name was: CC228A_contig_653) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00278_CC228A_..</strong> (original name was: CC228A_contig_1150) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00279_CC228A_..</strong> (original name was: CC228A_contig_237) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00280_CC228A_..</strong> (original name was: CC228A_contig_591) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_18</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_97</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_26</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00284_CC228A_..</strong> (original name was: CC228A_contig_424) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00285_CC228A_..</strong> (original name was: CC228A_contig_450) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00286_CC228A_..</strong> (original name was: CC228A_contig_144) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00287_CC228A_..</strong> (original name was: CC228A_contig_530) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00288_CC228A_..</strong> (original name was: CC228A_contig_712) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00289_CC228A_..</strong> (original name was: CC228A_contig_810) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00290_CC228A_..</strong> (original name was: CC228A_contig_676) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00291_CC228A_..</strong> (original name was: CC228A_contig_348) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00292_CC228A_..</strong> (original name was: CC228A_contig_808) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00293_CC228A_..</strong> (original name was: CC228A_contig_585) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00294_CC228A_..</strong> (original name was: CC228A_contig_336) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00295_CC228A_..</strong> (original name was: CC228A_contig_1042) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00296_CC228A_..</strong> (original name was: CC228A_contig_482) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00298_CC228A_..</strong> (original name was: CC228A_contig_592) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00299_CC228A_..</strong> (original name was: CC228A_contig_952) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00300_CC228A_..</strong> (original name was: CC228A_contig_267) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00301_CC228A_..</strong> (original name was: CC228A_contig_965) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00302_CC228A_..</strong> (original name was: CC228A_contig_1273) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00303_CC228A_..</strong> (original name was: CC228A_contig_161) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00304_CC228A_..</strong> (original name was: CC228A_contig_769) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00305_CC228A_..</strong> (original name was: CC228A_contig_298) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00306_CC228A_..</strong> (original name was: CC228A_contig_1026) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00307_CC228A_..</strong> (original name was: CC228A_contig_1010) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00308_CC228A_..</strong> (original name was: CC228A_contig_489) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_6</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00310_CC228A_..</strong> (original name was: CC228A_contig_523) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00311_CC228A_..</strong> (original name was: CC228A_contig_683) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00312_CC228A_..</strong> (original name was: CC228A_contig_493) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00313_CC228A_..</strong> (original name was: CC228A_contig_1057) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00314_CC228A_..</strong> (original name was: CC228A_contig_1238) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00315_CC228A_..</strong> (original name was: CC228A_contig_894) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00316_CC228A_..</strong> (original name was: CC228A_contig_1164) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_67</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00318_CC228A_..</strong> (original name was: CC228A_contig_1168) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00319_CC228A_..</strong> (original name was: CC228A_contig_1225) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00320_CC228A_..</strong> (original name was: CC228A_contig_1276) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00321_CC228A_..</strong> (original name was: CC228A_contig_818) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00322_CC228A_..</strong> (original name was: CC228A_contig_621) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00323_CC228A_..</strong> (original name was: CC228A_contig_170) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00324_CC228A_..</strong> (original name was: CC228A_contig_1114) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00325_CC228A_..</strong> (original name was: CC228A_contig_576) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00326_CC228A_..</strong> (original name was: CC228A_contig_111) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00327_CC228A_..</strong> (original name was: CC228A_contig_166) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00328_CC228A_..</strong> (original name was: CC228A_contig_1186) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00329_CC228A_..</strong> (original name was: CC228A_contig_342) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00330_CC228A_..</strong> (original name was: CC228A_contig_880) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00331_CC228A_..</strong> (original name was: CC228A_contig_1177) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00332_CC228A_..</strong> (original name was: CC228A_contig_786) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00333_CC228A_..</strong> (original name was: CC228A_contig_744) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00334_CC228A_..</strong> (original name was: CC228A_contig_582) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00335_CC228A_..</strong> (original name was: CC228A_contig_253) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_62</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00337_CC228A_..</strong> (original name was: CC228A_contig_224) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00339_CC228A_..</strong> (original name was: CC228A_contig_1121) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00340_CC228A_..</strong> (original name was: CC228A_contig_658) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_47</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00342_CC228A_..</strong> (original name was: CC228A_contig_198) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00343_CC228A_..</strong> (original name was: CC228A_contig_993) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00344_CC228A_..</strong> (original name was: CC228A_contig_829) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00345_CC228A_..</strong> (original name was: CC228A_contig_231) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00346_CC228A_..</strong> (original name was: CC228A_contig_743) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00347_CC228A_..</strong> (original name was: CC228A_contig_580) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00348_CC228A_..</strong> (original name was: CC228A_contig_162) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_90</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00350_CC228A_..</strong> (original name was: CC228A_contig_681) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_95</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00352_CC228A_..</strong> (original name was: CC228A_contig_1087) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00353_CC228A_..</strong> (original name was: CC228A_contig_508) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00354_CC228A_..</strong> (original name was: CC228A_contig_863) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00355_CC228A_..</strong> (original name was: CC228A_contig_611) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00356_CC228A_..</strong> (original name was: CC228A_contig_313) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00357_CC228A_..</strong> (original name was: CC228A_contig_656) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00358_CC228A_..</strong> (original name was: CC228A_contig_1039) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00359_CC228A_..</strong> (original name was: CC228A_contig_598) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00360_CC228A_..</strong> (original name was: CC228A_contig_349) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00361_CC228A_..</strong> (original name was: CC228A_contig_1051) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_55</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00363_CC228A_..</strong> (original name was: CC228A_contig_1226) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00364_CC228A_..</strong> (original name was: CC228A_contig_216) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_60</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00366_CC228A_..</strong> (original name was: CC228A_contig_799) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00367_CC228A_..</strong> (original name was: CC228A_contig_380) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00368_CC228A_..</strong> (original name was: CC228A_contig_980) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00369_CC228A_..</strong> (original name was: CC228A_contig_203) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00370_CC228A_..</strong> (original name was: CC228A_contig_292) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_5</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00372_CC228A_..</strong> (original name was: CC228A_contig_129) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00373_CC228A_..</strong> (original name was: CC228A_contig_502) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00374_CC228A_..</strong> (original name was: CC228A_contig_721) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00375_CC228A_..</strong> (original name was: CC228A_contig_261) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00376_CC228A_..</strong> (original name was: CC228A_contig_1063) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00377_CC228A_..</strong> (original name was: CC228A_contig_1151) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00378_CC228A_..</strong> (original name was: CC228A_contig_1267) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00379_CC228A_..</strong> (original name was: CC228A_contig_375) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00380_CC228A_..</strong> (original name was: CC228A_contig_1046) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00381_CC228A_..</strong> (original name was: CC228A_contig_1175) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00382_CC228A_..</strong> (original name was: CC228A_contig_776) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00383_CC228A_..</strong> (original name was: CC228A_contig_1020) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00384_CC228A_..</strong> (original name was: CC228A_contig_146) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00385_CC228A_..</strong> (original name was: CC228A_contig_411) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00386_CC228A_..</strong> (original name was: CC228A_contig_260) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00387_CC228A_..</strong> (original name was: CC228A_contig_1103) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00388_CC228A_..</strong> (original name was: CC228A_contig_1256) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00389_CC228A_..</strong> (original name was: CC228A_contig_1088) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00390_CC228A_..</strong> (original name was: CC228A_contig_1008) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00391_CC228A_..</strong> (original name was: CC228A_contig_937) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00392_CC228A_..</strong> (original name was: CC228A_contig_601) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00393_CC228A_..</strong> (original name was: CC228A_contig_534) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00394_CC228A_..</strong> (original name was: CC228A_contig_1212) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00395_CC228A_..</strong> (original name was: CC228A_contig_1243) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00396_CC228A_..</strong> (original name was: CC228A_contig_883) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00397_CC228A_..</strong> (original name was: CC228A_contig_934) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00398_CC228A_..</strong> (original name was: CC228A_contig_910) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00399_CC228A_..</strong> (original name was: CC228A_contig_1158) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00400_CC228A_..</strong> (original name was: CC228A_contig_768) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00401_CC228A_..</strong> (original name was: CC228A_contig_148) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00402_CC228A_..</strong> (original name was: CC228A_contig_1059) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00403_CC228A_..</strong> (original name was: CC228A_contig_921) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_45</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_36</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00406_CC228A_..</strong> (original name was: CC228A_contig_266) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00407_CC228A_..</strong> (original name was: CC228A_contig_390) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00408_CC228A_..</strong> (original name was: CC228A_contig_324) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00409_CC228A_..</strong> (original name was: CC228A_contig_452) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00410_CC228A_..</strong> (original name was: CC228A_contig_418) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00411_CC228A_..</strong> (original name was: CC228A_contig_893) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00412_CC228A_..</strong> (original name was: CC228A_contig_457) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00413_CC228A_..</strong> (original name was: CC228A_contig_364) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_37</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00415_CC228A_..</strong> (original name was: CC228A_contig_478) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00416_CC228A_..</strong> (original name was: CC228A_contig_948) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00417_CC228A_..</strong> (original name was: CC228A_contig_205) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00418_CC228A_..</strong> (original name was: CC228A_contig_443) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00419_CC228A_..</strong> (original name was: CC228A_contig_647) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00420_CC228A_..</strong> (original name was: CC228A_contig_618) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00421_CC228A_..</strong> (original name was: CC228A_contig_871) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00422_CC228A_..</strong> (original name was: CC228A_contig_1013) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00423_CC228A_..</strong> (original name was: CC228A_contig_711) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00424_CC228A_..</strong> (original name was: CC228A_contig_1250) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00425_CC228A_..</strong> (original name was: CC228A_contig_282) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00426_CC228A_..</strong> (original name was: CC228A_contig_1202) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_79</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00428_CC228A_..</strong> (original name was: CC228A_contig_415) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00429_CC228A_..</strong> (original name was: CC228A_contig_700) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00430_CC228A_..</strong> (original name was: CC228A_contig_402) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00431_CC228A_..</strong> (original name was: CC228A_contig_796) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00432_CC228A_..</strong> (original name was: CC228A_contig_214) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00433_CC228A_..</strong> (original name was: CC228A_contig_255) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00434_CC228A_..</strong> (original name was: CC228A_contig_389) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00435_CC228A_..</strong> (original name was: CC228A_contig_1027) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00436_CC228A_..</strong> (original name was: CC228A_contig_567) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00437_CC228A_..</strong> (original name was: CC228A_contig_515) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00438_CC228A_..</strong> (original name was: CC228A_contig_971) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00439_CC228A_..</strong> (original name was: CC228A_contig_1034) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00440_CC228A_..</strong> (original name was: CC228A_contig_1195) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00441_CC228A_..</strong> (original name was: CC228A_contig_1234) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00442_CC228A_..</strong> (original name was: CC228A_contig_495) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00443_CC228A_..</strong> (original name was: CC228A_contig_781) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00444_CC228A_..</strong> (original name was: CC228A_contig_835) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00445_CC228A_..</strong> (original name was: CC228A_contig_853) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00446_CC228A_..</strong> (original name was: CC228A_contig_226) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00447_CC228A_..</strong> (original name was: CC228A_contig_271) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00448_CC228A_..</strong> (original name was: CC228A_contig_822) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00449_CC228A_..</strong> (original name was: CC228A_contig_378) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00450_CC228A_..</strong> (original name was: CC228A_contig_1111) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00451_CC228A_..</strong> (original name was: CC228A_contig_1219) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00452_CC228A_..</strong> (original name was: CC228A_contig_156) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00453_CC228A_..</strong> (original name was: CC228A_contig_903) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00454_CC228A_..</strong> (original name was: CC228A_contig_987) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00455_CC228A_..</strong> (original name was: CC228A_contig_827) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00456_CC228A_..</strong> (original name was: CC228A_contig_1275) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00457_CC228A_..</strong> (original name was: CC228A_contig_693) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00458_CC228A_..</strong> (original name was: CC228A_contig_1052) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00459_CC228A_..</strong> (original name was: CC228A_contig_1190) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00460_CC228A_..</strong> (original name was: CC228A_contig_847) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00461_CC228A_..</strong> (original name was: CC228A_contig_1253) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00462_CC228A_..</strong> (original name was: CC228A_contig_432) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00463_CC228A_..</strong> (original name was: CC228A_contig_692) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00464_CC228A_..</strong> (original name was: CC228A_contig_1068) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00465_CC228A_..</strong> (original name was: CC228A_contig_698) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00466_CC228A_..</strong> (original name was: CC228A_contig_629) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00467_CC228A_..</strong> (original name was: CC228A_contig_978) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00468_CC228A_..</strong> (original name was: CC228A_contig_101) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_22</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00470_CC228A_..</strong> (original name was: CC228A_contig_571) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00471_CC228A_..</strong> (original name was: CC228A_contig_498) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00472_CC228A_..</strong> (original name was: CC228A_contig_806) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00473_CC228A_..</strong> (original name was: CC228A_contig_545) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00474_CC228A_..</strong> (original name was: CC228A_contig_466) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00475_CC228A_..</strong> (original name was: CC228A_contig_734) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00476_CC228A_..</strong> (original name was: CC228A_contig_128) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00477_CC228A_..</strong> (original name was: CC228A_contig_448) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00478_CC228A_..</strong> (original name was: CC228A_contig_1037) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00479_CC228A_..</strong> (original name was: CC228A_contig_245) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_81</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00481_CC228A_..</strong> (original name was: CC228A_contig_1132) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00482_CC228A_..</strong> (original name was: CC228A_contig_229) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00483_CC228A_..</strong> (original name was: CC228A_contig_891) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00484_CC228A_..</strong> (original name was: CC228A_contig_346) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00485_CC228A_..</strong> (original name was: CC228A_contig_731) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_57</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00487_CC228A_..</strong> (original name was: CC228A_contig_193) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00488_CC228A_..</strong> (original name was: CC228A_contig_439) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00489_CC228A_..</strong> (original name was: CC228A_contig_300) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00490_CC228A_..</strong> (original name was: CC228A_contig_926) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00491_CC228A_..</strong> (original name was: CC228A_contig_373) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00492_CC228A_..</strong> (original name was: CC228A_contig_344) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00493_CC228A_..</strong> (original name was: CC228A_contig_296) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00494_CC228A_..</strong> (original name was: CC228A_contig_251) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00495_CC228A_..</strong> (original name was: CC228A_contig_1032) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00496_CC228A_..</strong> (original name was: CC228A_contig_1248) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00497_CC228A_..</strong> (original name was: CC228A_contig_504) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_52</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00499_CC228A_..</strong> (original name was: CC228A_contig_966) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00500_CC228A_..</strong> (original name was: CC228A_contig_856) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00501_CC228A_..</strong> (original name was: CC228A_contig_901) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00502_CC228A_..</strong> (original name was: CC228A_contig_551) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00503_CC228A_..</strong> (original name was: CC228A_contig_320) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00504_CC228A_..</strong> (original name was: CC228A_contig_758) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00505_CC228A_..</strong> (original name was: CC228A_contig_406) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00506_CC228A_..</strong> (original name was: CC228A_contig_1044) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00507_CC228A_..</strong> (original name was: CC228A_contig_816) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00508_CC228A_..</strong> (original name was: CC228A_contig_614) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00509_CC228A_..</strong> (original name was: CC228A_contig_1232) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00510_CC228A_..</strong> (original name was: CC228A_contig_672) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00511_CC228A_..</strong> (original name was: CC228A_contig_1105) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00512_CC228A_..</strong> (original name was: CC228A_contig_738) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_49</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00514_CC228A_..</strong> (original name was: CC228A_contig_154) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00515_CC228A_..</strong> (original name was: CC228A_contig_487) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00516_CC228A_..</strong> (original name was: CC228A_contig_1184) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00517_CC228A_..</strong> (original name was: CC228A_contig_1133) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00518_CC228A_..</strong> (original name was: CC228A_contig_294) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00519_CC228A_..</strong> (original name was: CC228A_contig_222) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00520_CC228A_..</strong> (original name was: CC228A_contig_529) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00521_CC228A_..</strong> (original name was: CC228A_contig_250) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00522_CC228A_..</strong> (original name was: CC228A_contig_187) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00523_CC228A_..</strong> (original name was: CC228A_contig_679) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00524_CC228A_..</strong> (original name was: CC228A_contig_1161) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00525_CC228A_..</strong> (original name was: CC228A_contig_717) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00526_CC228A_..</strong> (original name was: CC228A_contig_578) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00527_CC228A_..</strong> (original name was: CC228A_contig_861) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00528_CC228A_..</strong> (original name was: CC228A_contig_1006) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_65</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00530_CC228A_..</strong> (original name was: CC228A_contig_724) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00531_CC228A_..</strong> (original name was: CC228A_contig_1201) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00532_CC228A_..</strong> (original name was: CC228A_contig_263) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00533_CC228A_..</strong> (original name was: CC228A_contig_746) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00534_CC228A_..</strong> (original name was: CC228A_contig_358) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00535_CC228A_..</strong> (original name was: CC228A_contig_1179) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00536_CC228A_..</strong> (original name was: CC228A_contig_355) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00537_CC228A_..</strong> (original name was: CC228A_contig_791) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00538_CC228A_..</strong> (original name was: CC228A_contig_1022) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00539_CC228A_..</strong> (original name was: CC228A_contig_723) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00540_CC228A_..</strong> (original name was: CC228A_contig_133) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00541_CC228A_..</strong> (original name was: CC228A_contig_152) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00542_CC228A_..</strong> (original name was: CC228A_contig_589) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00544_CC228A_..</strong> (original name was: CC228A_contig_480) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00545_CC228A_..</strong> (original name was: CC228A_contig_800) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00546_CC228A_..</strong> (original name was: CC228A_contig_669) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00547_CC228A_..</strong> (original name was: CC228A_contig_1155) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00548_CC228A_..</strong> (original name was: CC228A_contig_184) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00549_CC228A_..</strong> (original name was: CC228A_contig_103) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00550_CC228A_..</strong> (original name was: CC228A_contig_553) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00551_CC228A_..</strong> (original name was: CC228A_contig_1128) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00552_CC228A_..</strong> (original name was: CC228A_contig_1093) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00553_CC228A_..</strong> (original name was: CC228A_contig_917) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00554_CC228A_..</strong> (original name was: CC228A_contig_595) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_80</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00556_CC228A_..</strong> (original name was: CC228A_contig_331) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00557_CC228A_..</strong> (original name was: CC228A_contig_898) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00558_CC228A_..</strong> (original name was: CC228A_contig_866) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00559_CC228A_..</strong> (original name was: CC228A_contig_400) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00560_CC228A_..</strong> (original name was: CC228A_contig_117) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00561_CC228A_..</strong> (original name was: CC228A_contig_989) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_42</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00563_CC228A_..</strong> (original name was: CC228A_contig_1094) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00564_CC228A_..</strong> (original name was: CC228A_contig_973) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00565_CC228A_..</strong> (original name was: CC228A_contig_471) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00566_CC228A_..</strong> (original name was: CC228A_contig_982) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00567_CC228A_..</strong> (original name was: CC228A_contig_771) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00568_CC228A_..</strong> (original name was: CC228A_contig_1192) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00569_CC228A_..</strong> (original name was: CC228A_contig_407) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00570_CC228A_..</strong> (original name was: CC228A_contig_638) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00571_CC228A_..</strong> (original name was: CC228A_contig_1269) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00572_CC228A_..</strong> (original name was: CC228A_contig_543) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00574_CC228A_..</strong> (original name was: CC228A_contig_941) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00575_CC228A_..</strong> (original name was: CC228A_contig_830) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00576_CC228A_..</strong> (original name was: CC228A_contig_384) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00577_CC228A_..</strong> (original name was: CC228A_contig_1197) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00578_CC228A_..</strong> (original name was: CC228A_contig_939) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00579_CC228A_..</strong> (original name was: CC228A_contig_434) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00580_CC228A_..</strong> (original name was: CC228A_contig_517) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00581_CC228A_..</strong> (original name was: CC228A_contig_365) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00582_CC228A_..</strong> (original name was: CC228A_contig_645) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00583_CC228A_..</strong> (original name was: CC228A_contig_766) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00584_CC228A_..</strong> (original name was: CC228A_contig_1217) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00585_CC228A_..</strong> (original name was: CC228A_contig_1265) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00586_CC228A_..</strong> (original name was: CC228A_contig_179) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00587_CC228A_..</strong> (original name was: CC228A_contig_1029) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00588_CC228A_..</strong> (original name was: CC228A_contig_753) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00589_CC228A_..</strong> (original name was: CC228A_contig_333) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00590_CC228A_..</strong> (original name was: CC228A_contig_673) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00591_CC228A_..</strong> (original name was: CC228A_contig_1108) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00592_CC228A_..</strong> (original name was: CC228A_contig_687) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00593_CC228A_..</strong> (original name was: CC228A_contig_441) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00594_CC228A_..</strong> (original name was: CC228A_contig_288) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_29</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00596_CC228A_..</strong> (original name was: CC228A_contig_302) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00597_CC228A_..</strong> (original name was: CC228A_contig_142) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_77</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00599_CC228A_..</strong> (original name was: CC228A_contig_813) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00600_CC228A_..</strong> (original name was: CC228A_contig_831) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00601_CC228A_..</strong> (original name was: CC228A_contig_326) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00602_CC228A_..</strong> (original name was: CC228A_contig_783) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00603_CC228A_..</strong> (original name was: CC228A_contig_1118) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00604_CC228A_..</strong> (original name was: CC228A_contig_968) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00605_CC228A_..</strong> (original name was: CC228A_contig_955) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00606_CC228A_..</strong> (original name was: CC228A_contig_536) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00607_CC228A_..</strong> (original name was: CC228A_contig_837) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00608_CC228A_..</strong> (original name was: CC228A_contig_824) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00609_CC228A_..</strong> (original name was: CC228A_contig_110) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00610_CC228A_..</strong> (original name was: CC228A_contig_1241) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00611_CC228A_..</strong> (original name was: CC228A_contig_1204) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00612_CC228A_..</strong> (original name was: CC228A_contig_428) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00613_CC228A_..</strong> (original name was: CC228A_contig_1054) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00614_CC228A_..</strong> (original name was: CC228A_contig_1206) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00615_CC228A_..</strong> (original name was: CC228A_contig_399) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00616_CC228A_..</strong> (original name was: CC228A_contig_396) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00617_CC228A_..</strong> (original name was: CC228A_contig_247) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00618_CC228A_..</strong> (original name was: CC228A_contig_923) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00619_CC228A_..</strong> (original name was: CC228A_contig_1144) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00620_CC228A_..</strong> (original name was: CC228A_contig_996) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_31</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00623_CC228A_..</strong> (original name was: CC228A_contig_273) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00624_CC228A_..</strong> (original name was: CC228A_contig_1214) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00625_CC228A_..</strong> (original name was: CC228A_contig_305) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00626_CC228A_..</strong> (original name was: CC228A_contig_1065) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00627_CC228A_..</strong> (original name was: CC228A_contig_315) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00628_CC228A_..</strong> (original name was: CC228A_contig_168) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00629_CC228A_..</strong> (original name was: CC228A_contig_643) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00630_CC228A_..</strong> (original name was: CC228A_contig_1016) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00631_CC228A_..</strong> (original name was: CC228A_contig_1072) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_93</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00633_CC228A_..</strong> (original name was: CC228A_contig_596) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00634_CC228A_..</strong> (original name was: CC228A_contig_708) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00635_CC228A_..</strong> (original name was: CC228A_contig_802) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00636_CC228A_..</strong> (original name was: CC228A_contig_695) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00637_CC228A_..</strong> (original name was: CC228A_contig_453) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_84</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00639_CC228A_..</strong> (original name was: CC228A_contig_332) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00640_CC228A_..</strong> (original name was: CC228A_contig_951) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00641_CC228A_..</strong> (original name was: CC228A_contig_397) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_25</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00643_CC228A_..</strong> (original name was: CC228A_contig_284) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00644_CC228A_..</strong> (original name was: CC228A_contig_123) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00645_CC228A_..</strong> (original name was: CC228A_contig_304) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00646_CC228A_..</strong> (original name was: CC228A_contig_479) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00647_CC228A_..</strong> (original name was: CC228A_contig_126) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00648_CC228A_..</strong> (original name was: CC228A_contig_189) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00649_CC228A_..</strong> (original name was: CC228A_contig_819) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_72</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00652_CC228A_..</strong> (original name was: CC228A_contig_897) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00653_CC228A_..</strong> (original name was: CC228A_contig_309) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00654_CC228A_..</strong> (original name was: CC228A_contig_793) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00655_CC228A_..</strong> (original name was: CC228A_contig_484) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00656_CC228A_..</strong> (original name was: CC228A_contig_491) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_75</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00658_CC228A_..</strong> (original name was: CC228A_contig_652) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00659_CC228A_..</strong> (original name was: CC228A_contig_446) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_54</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_85</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00662_CC228A_..</strong> (original name was: CC228A_contig_958) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00663_CC228A_..</strong> (original name was: CC228A_contig_706) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00664_CC228A_..</strong> (original name was: CC228A_contig_833) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00665_CC228A_..</strong> (original name was: CC228A_contig_726) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00666_CC228A_..</strong> (original name was: CC228A_contig_942) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00667_CC228A_..</strong> (original name was: CC228A_contig_345) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00668_CC228A_..</strong> (original name was: CC228A_contig_1191) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00669_CC228A_..</strong> (original name was: CC228A_contig_844) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00670_CC228A_..</strong> (original name was: CC228A_contig_385) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00671_CC228A_..</strong> (original name was: CC228A_contig_865) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00672_CC228A_..</strong> (original name was: CC228A_contig_1171) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00673_CC228A_..</strong> (original name was: CC228A_contig_1145) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00674_CC228A_..</strong> (original name was: CC228A_contig_1189) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00675_CC228A_..</strong> (original name was: CC228A_contig_1237) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_59</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00677_CC228A_..</strong> (original name was: CC228A_contig_136) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00678_CC228A_..</strong> (original name was: CC228A_contig_169) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00679_CC228A_..</strong> (original name was: CC228A_contig_473) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00680_CC228A_..</strong> (original name was: CC228A_contig_1056) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00681_CC228A_..</strong> (original name was: CC228A_contig_337) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00682_CC228A_..</strong> (original name was: CC228A_contig_1066) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00683_CC228A_..</strong> (original name was: CC228A_contig_625) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00684_CC228A_..</strong> (original name was: CC228A_contig_506) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00685_CC228A_..</strong> (original name was: CC228A_contig_1120) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00686_CC228A_..</strong> (original name was: CC228A_contig_558) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00687_CC228A_..</strong> (original name was: CC228A_contig_174) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00688_CC228A_..</strong> (original name was: CC228A_contig_425) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00689_CC228A_..</strong> (original name was: CC228A_contig_238) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00690_CC228A_..</strong> (original name was: CC228A_contig_845) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00691_CC228A_..</strong> (original name was: CC228A_contig_1043) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00692_CC228A_..</strong> (original name was: CC228A_contig_258) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00693_CC228A_..</strong> (original name was: CC228A_contig_713) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00694_CC228A_..</strong> (original name was: CC228A_contig_1090) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00695_CC228A_..</strong> (original name was: CC228A_contig_1117) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00696_CC228A_..</strong> (original name was: CC228A_contig_548) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00697_CC228A_..</strong> (original name was: CC228A_contig_603) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00698_CC228A_..</strong> (original name was: CC228A_contig_789) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00699_CC228A_..</strong> (original name was: CC228A_contig_733) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00700_CC228A_..</strong> (original name was: CC228A_contig_423) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00701_CC228A_..</strong> (original name was: CC228A_contig_519) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00702_CC228A_..</strong> (original name was: CC228A_contig_472) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00703_CC228A_..</strong> (original name was: CC228A_contig_750) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00704_CC228A_..</strong> (original name was: CC228A_contig_714) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00705_CC228A_..</strong> (original name was: CC228A_contig_633) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00706_CC228A_..</strong> (original name was: CC228A_contig_794) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00707_CC228A_..</strong> (original name was: CC228A_contig_249) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_43</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_14</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00710_CC228A_..</strong> (original name was: CC228A_contig_961) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00711_CC228A_..</strong> (original name was: CC228A_contig_814) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00712_CC228A_..</strong> (original name was: CC228A_contig_1110) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00713_CC228A_..</strong> (original name was: CC228A_contig_983) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00714_CC228A_..</strong> (original name was: CC228A_contig_1018) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00715_CC228A_..</strong> (original name was: CC228A_contig_510) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00716_CC228A_..</strong> (original name was: CC228A_contig_675) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00717_CC228A_..</strong> (original name was: CC228A_contig_686) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00718_CC228A_..</strong> (original name was: CC228A_contig_1004) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00719_CC228A_..</strong> (original name was: CC228A_contig_1131) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00720_CC228A_..</strong> (original name was: CC228A_contig_317) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00721_CC228A_..</strong> (original name was: CC228A_contig_615) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00722_CC228A_..</strong> (original name was: CC228A_contig_1085) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00723_CC228A_..</strong> (original name was: CC228A_contig_180) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00724_CC228A_..</strong> (original name was: CC228A_contig_201) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00725_CC228A_..</strong> (original name was: CC228A_contig_275) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00726_CC228A_..</strong> (original name was: CC228A_contig_297) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00727_CC228A_..</strong> (original name was: CC228A_contig_888) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00728_CC228A_..</strong> (original name was: CC228A_contig_563) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00729_CC228A_..</strong> (original name was: CC228A_contig_1231) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00730_CC228A_..</strong> (original name was: CC228A_contig_150) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00731_CC228A_..</strong> (original name was: CC228A_contig_159) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00732_CC228A_..</strong> (original name was: CC228A_contig_196) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00733_CC228A_..</strong> (original name was: CC228A_contig_909) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00734_CC228A_..</strong> (original name was: CC228A_contig_221) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00735_CC228A_..</strong> (original name was: CC228A_contig_765) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_16</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00737_CC228A_..</strong> (original name was: CC228A_contig_242) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00738_CC228A_..</strong> (original name was: CC228A_contig_1061) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00739_CC228A_..</strong> (original name was: CC228A_contig_561) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00740_CC228A_..</strong> (original name was: CC228A_contig_366) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00741_CC228A_..</strong> (original name was: CC228A_contig_932) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00742_CC228A_..</strong> (original name was: CC228A_contig_461) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00743_CC228A_..</strong> (original name was: CC228A_contig_176) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00744_CC228A_..</strong> (original name was: CC228A_contig_1082) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00745_CC228A_..</strong> (original name was: CC228A_contig_403) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00746_CC228A_..</strong> (original name was: CC228A_contig_259) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00747_CC228A_..</strong> (original name was: CC228A_contig_1240) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00748_CC228A_..</strong> (original name was: CC228A_contig_704) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_50</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00750_CC228A_..</strong> (original name was: CC228A_contig_1254) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00751_CC228A_..</strong> (original name was: CC228A_contig_1123) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00752_CC228A_..</strong> (original name was: CC228A_contig_327) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00753_CC228A_..</strong> (original name was: CC228A_contig_975) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00754_CC228A_..</strong> (original name was: CC228A_contig_931) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00755_CC228A_..</strong> (original name was: CC228A_contig_729) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00756_CC228A_..</strong> (original name was: CC228A_contig_1071) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00757_CC228A_..</strong> (original name was: CC228A_contig_486) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00758_CC228A_..</strong> (original name was: CC228A_contig_858) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00759_CC228A_..</strong> (original name was: CC228A_contig_852) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00760_CC228A_..</strong> (original name was: CC228A_contig_538) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00761_CC228A_..</strong> (original name was: CC228A_contig_382) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00762_CC228A_..</strong> (original name was: CC228A_contig_991) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_66</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00764_CC228A_..</strong> (original name was: CC228A_contig_678) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00765_CC228A_..</strong> (original name was: CC228A_contig_1268) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00766_CC228A_..</strong> (original name was: CC228A_contig_211) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00767_CC228A_..</strong> (original name was: CC228A_contig_1019) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00768_CC228A_..</strong> (original name was: CC228A_contig_914) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00769_CC228A_..</strong> (original name was: CC228A_contig_938) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00770_CC228A_..</strong> (original name was: CC228A_contig_1005) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00771_CC228A_..</strong> (original name was: CC228A_contig_526) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00772_CC228A_..</strong> (original name was: CC228A_contig_1279) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_91</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00774_CC228A_..</strong> (original name was: CC228A_contig_572) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_33</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00776_CC228A_..</strong> (original name was: CC228A_contig_438) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00777_CC228A_..</strong> (original name was: CC228A_contig_1078) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00778_CC228A_..</strong> (original name was: CC228A_contig_414) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00779_CC228A_..</strong> (original name was: CC228A_contig_164) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00780_CC228A_..</strong> (original name was: CC228A_contig_1156) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00781_CC228A_..</strong> (original name was: CC228A_contig_1210) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00782_CC228A_..</strong> (original name was: CC228A_contig_1099) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00783_CC228A_..</strong> (original name was: CC228A_contig_370) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00784_CC228A_..</strong> (original name was: CC228A_contig_763) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00785_CC228A_..</strong> (original name was: CC228A_contig_1228) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00786_CC228A_..</strong> (original name was: CC228A_contig_878) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_10</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00788_CC228A_..</strong> (original name was: CC228A_contig_208) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00789_CC228A_..</strong> (original name was: CC228A_contig_1271) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00790_CC228A_..</strong> (original name was: CC228A_contig_741) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00791_CC228A_..</strong> (original name was: CC228A_contig_1272) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00792_CC228A_..</strong> (original name was: CC228A_contig_1173) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00793_CC228A_..</strong> (original name was: CC228A_contig_113) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00794_CC228A_..</strong> (original name was: CC228A_contig_1165) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00795_CC228A_..</strong> (original name was: CC228A_contig_1193) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00796_CC228A_..</strong> (original name was: CC228A_contig_1041) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00797_CC228A_..</strong> (original name was: CC228A_contig_444) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_7</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_40</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00801_CC228A_..</strong> (original name was: CC228A_contig_1116) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00802_CC228A_..</strong> (original name was: CC228A_contig_306) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00803_CC228A_..</strong> (original name was: CC228A_contig_1058) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00804_CC228A_..</strong> (original name was: CC228A_contig_395) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00805_CC228A_..</strong> (original name was: CC228A_contig_1263) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00806_CC228A_..</strong> (original name was: CC228A_contig_641) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00807_CC228A_..</strong> (original name was: CC228A_contig_116) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00808_CC228A_..</strong> (original name was: CC228A_contig_351) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00809_CC228A_..</strong> (original name was: CC228A_contig_422) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00810_CC228A_..</strong> (original name was: CC228A_contig_455) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00811_CC228A_..</strong> (original name was: CC228A_contig_872) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00812_CC228A_..</strong> (original name was: CC228A_contig_1141) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00813_CC228A_..</strong> (original name was: CC228A_contig_310) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_69</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00815_CC228A_..</strong> (original name was: CC228A_contig_1119) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00817_CC228A_..</strong> (original name was: CC228A_contig_138) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00818_CC228A_..</strong> (original name was: CC228A_contig_812) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00820_CC228A_..</strong> (original name was: CC228A_contig_628) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00821_CC228A_..</strong> (original name was: CC228A_contig_1009) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00822_CC228A_..</strong> (original name was: CC228A_contig_605) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00823_CC228A_..</strong> (original name was: CC228A_contig_1048) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00824_CC228A_..</strong> (original name was: CC228A_contig_501) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00825_CC228A_..</strong> (original name was: CC228A_contig_121) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00826_CC228A_..</strong> (original name was: CC228A_contig_985) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00827_CC228A_..</strong> (original name was: CC228A_contig_289) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00828_CC228A_..</strong> (original name was: CC228A_contig_608) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00829_CC228A_..</strong> (original name was: CC228A_contig_1096) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00830_CC228A_..</strong> (original name was: CC228A_contig_1259) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00831_CC228A_..</strong> (original name was: CC228A_contig_183) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00832_CC228A_..</strong> (original name was: CC228A_contig_784) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00833_CC228A_..</strong> (original name was: CC228A_contig_944) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00834_CC228A_..</strong> (original name was: CC228A_contig_654) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00835_CC228A_..</strong> (original name was: CC228A_contig_842) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00836_CC228A_..</strong> (original name was: CC228A_contig_584) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00837_CC228A_..</strong> (original name was: CC228A_contig_219) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00838_CC228A_..</strong> (original name was: CC228A_contig_379) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00839_CC228A_..</strong> (original name was: CC228A_contig_299) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00840_CC228A_..</strong> (original name was: CC228A_contig_272) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00841_CC228A_..</strong> (original name was: CC228A_contig_387) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00842_CC228A_..</strong> (original name was: CC228A_contig_1023) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00843_CC228A_..</strong> (original name was: CC228A_contig_1187) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00844_CC228A_..</strong> (original name was: CC228A_contig_145) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_87</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00846_CC228A_..</strong> (original name was: CC228A_contig_953) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00847_CC228A_..</strong> (original name was: CC228A_contig_623) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00848_CC228A_..</strong> (original name was: CC228A_contig_556) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00849_CC228A_..</strong> (original name was: CC228A_contig_684) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00850_CC228A_..</strong> (original name was: CC228A_contig_895) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00851_CC228A_..</strong> (original name was: CC228A_contig_106) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_17</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00853_CC228A_..</strong> (original name was: CC228A_contig_809) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00854_CC228A_..</strong> (original name was: CC228A_contig_593) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00856_CC228A_..</strong> (original name was: CC228A_contig_531) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00857_CC228A_..</strong> (original name was: CC228A_contig_522) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00858_CC228A_..</strong> (original name was: CC228A_contig_735) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00859_CC228A_..</strong> (original name was: CC228A_contig_335) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00860_CC228A_..</strong> (original name was: CC228A_contig_1125) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00861_CC228A_..</strong> (original name was: CC228A_contig_533) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00862_CC228A_..</strong> (original name was: CC228A_contig_1239) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00863_CC228A_..</strong> (original name was: CC228A_contig_521) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00864_CC228A_..</strong> (original name was: CC228A_contig_1036) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00866_CC228A_..</strong> (original name was: CC228A_contig_329) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00867_CC228A_..</strong> (original name was: CC228A_contig_826) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00868_CC228A_..</strong> (original name was: CC228A_contig_631) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00869_CC228A_..</strong> (original name was: CC228A_contig_157) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00870_CC228A_..</strong> (original name was: CC228A_contig_804) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00871_CC228A_..</strong> (original name was: CC228A_contig_874) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00872_CC228A_..</strong> (original name was: CC228A_contig_436) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00873_CC228A_..</strong> (original name was: CC228A_contig_1182) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00874_CC228A_..</strong> (original name was: CC228A_contig_236) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00875_CC228A_..</strong> (original name was: CC228A_contig_409) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00876_CC228A_..</strong> (original name was: CC228A_contig_962) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00877_CC228A_..</strong> (original name was: CC228A_contig_239) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00878_CC228A_..</strong> (original name was: CC228A_contig_577) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00879_CC228A_..</strong> (original name was: CC228A_contig_876) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00880_CC228A_..</strong> (original name was: CC228A_contig_1100) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00881_CC228A_..</strong> (original name was: CC228A_contig_1252) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00882_CC228A_..</strong> (original name was: CC228A_contig_191) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00883_CC228A_..</strong> (original name was: CC228A_contig_1080) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00884_CC228A_..</strong> (original name was: CC228A_contig_1076) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00885_CC228A_..</strong> (original name was: CC228A_contig_468) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00886_CC228A_..</strong> (original name was: CC228A_contig_574) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00887_CC228A_..</strong> (original name was: CC228A_contig_659) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00888_CC228A_..</strong> (original name was: CC228A_contig_1002) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00889_CC228A_..</strong> (original name was: CC228A_contig_748) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00890_CC228A_..</strong> (original name was: CC228A_contig_907) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00891_CC228A_..</strong> (original name was: CC228A_contig_970) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00892_CC228A_..</strong> (original name was: CC228A_contig_997) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_12</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00894_CC228A_..</strong> (original name was: CC228A_contig_1135) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00895_CC228A_..</strong> (original name was: CC228A_contig_565) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00896_CC228A_..</strong> (original name was: CC228A_contig_599) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00897_CC228A_..</strong> (original name was: CC228A_contig_1245) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00898_CC228A_..</strong> (original name was: CC228A_contig_1221) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00899_CC228A_..</strong> (original name was: CC228A_contig_1261) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00900_CC228A_..</strong> (original name was: CC228A_contig_546) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00902_CC228A_..</strong> (original name was: CC228A_contig_228) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_39</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00904_CC228A_..</strong> (original name was: CC228A_contig_512) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00905_CC228A_..</strong> (original name was: CC228A_contig_362) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00906_CC228A_..</strong> (original name was: CC228A_contig_430) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00907_CC228A_..</strong> (original name was: CC228A_contig_541) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00908_CC228A_..</strong> (original name was: CC228A_contig_754) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00909_CC228A_..</strong> (original name was: CC228A_contig_1280) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00910_CC228A_..</strong> (original name was: CC228A_contig_661) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00911_CC228A_..</strong> (original name was: CC228A_contig_928) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00912_CC228A_..</strong> (original name was: CC228A_contig_774) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00913_CC228A_..</strong> (original name was: CC228A_contig_392) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00914_CC228A_..</strong> (original name was: CC228A_contig_718) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00915_CC228A_..</strong> (original name was: CC228A_contig_867) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00916_CC228A_..</strong> (original name was: CC228A_contig_330) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00917_CC228A_..</strong> (original name was: CC228A_contig_664) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00918_CC228A_..</strong> (original name was: CC228A_contig_244) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00919_CC228A_..</strong> (original name was: CC228A_contig_476) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_64</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_9</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00922_CC228A_..</strong> (original name was: CC228A_contig_213) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00923_CC228A_..</strong> (original name was: CC228A_contig_905) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00924_CC228A_..</strong> (original name was: CC228A_contig_912) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00925_CC228A_..</strong> (original name was: CC228A_contig_1154) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00926_CC228A_..</strong> (original name was: CC228A_contig_779) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_24</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00928_CC228A_..</strong> (original name was: CC228A_contig_1138) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00929_CC228A_..</strong> (original name was: CC228A_contig_277) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00930_CC228A_..</strong> (original name was: CC228A_contig_1277) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_70</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00932_CC228A_..</strong> (original name was: CC228A_contig_319) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00933_CC228A_..</strong> (original name was: CC228A_contig_886) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00934_CC228A_..</strong> (original name was: CC228A_contig_290) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00935_CC228A_..</strong> (original name was: CC228A_contig_990) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00936_CC228A_..</strong> (original name was: CC228A_contig_761) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00937_CC228A_..</strong> (original name was: CC228A_contig_372) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00938_CC228A_..</strong> (original name was: CC228A_contig_1244) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00939_CC228A_..</strong> (original name was: CC228A_contig_535) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00940_CC228A_..</strong> (original name was: CC228A_contig_710) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00941_CC228A_..</strong> (original name was: CC228A_contig_104) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00942_CC228A_..</strong> (original name was: CC228A_contig_334) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00943_CC228A_..</strong> (original name was: CC228A_contig_609) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00944_CC228A_..</strong> (original name was: CC228A_contig_100) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00946_CC228A_..</strong> (original name was: CC228A_contig_460) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00947_CC228A_..</strong> (original name was: CC228A_contig_911) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00948_CC228A_..</strong> (original name was: CC228A_contig_243) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00949_CC228A_..</strong> (original name was: CC228A_contig_1053) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00950_CC228A_..</strong> (original name was: CC228A_contig_701) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00951_CC228A_..</strong> (original name was: CC228A_contig_490) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_51</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00953_CC228A_..</strong> (original name was: CC228A_contig_1007) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00954_CC228A_..</strong> (original name was: CC228A_contig_401) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00955_CC228A_..</strong> (original name was: CC228A_contig_977) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_32</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_20</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00958_CC228A_..</strong> (original name was: CC228A_contig_408) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00959_CC228A_..</strong> (original name was: CC228A_contig_420) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00960_CC228A_..</strong> (original name was: CC228A_contig_280) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00961_CC228A_..</strong> (original name was: CC228A_contig_639) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_1</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00963_CC228A_..</strong> (original name was: CC228A_contig_287) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00964_CC228A_..</strong> (original name was: CC228A_contig_870) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_15</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00966_CC228A_..</strong> (original name was: CC228A_contig_206) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00967_CC228A_..</strong> (original name was: CC228A_contig_204) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00968_CC228A_..</strong> (original name was: CC228A_contig_648) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00969_CC228A_..</strong> (original name was: CC228A_contig_433) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00970_CC228A_..</strong> (original name was: CC228A_contig_1012) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00971_CC228A_..</strong> (original name was: CC228A_contig_554) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00972_CC228A_..</strong> (original name was: CC228A_contig_143) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00973_CC228A_..</strong> (original name was: CC228A_contig_607) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00974_CC228A_..</strong> (original name was: CC228A_contig_780) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00975_CC228A_..</strong> (original name was: CC228A_contig_547) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00976_CC228A_..</strong> (original name was: CC228A_contig_325) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00977_CC228A_..</strong> (original name was: CC228A_contig_716) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00978_CC228A_..</strong> (original name was: CC228A_contig_1282) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00979_CC228A_..</strong> (original name was: CC228A_contig_935) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00980_CC228A_..</strong> (original name was: CC228A_contig_477) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00981_CC228A_..</strong> (original name was: CC228A_contig_879) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00982_CC228A_..</strong> (original name was: CC228A_contig_1095) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00983_CC228A_..</strong> (original name was: CC228A_contig_363) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00984_CC228A_..</strong> (original name was: CC228A_contig_644) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00985_CC228A_..</strong> (original name was: CC228A_contig_954) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00986_CC228A_..</strong> (original name was: CC228A_contig_417) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00987_CC228A_..</strong> (original name was: CC228A_contig_566) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00988_CC228A_..</strong> (original name was: CC228A_contig_630) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00989_CC228A_..</strong> (original name was: CC228A_contig_755) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00990_CC228A_..</strong> (original name was: CC228A_contig_1274) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00991_CC228A_..</strong> (original name was: CC228A_contig_906) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00992_CC228A_..</strong> (original name was: CC228A_contig_1025) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00993_CC228A_..</strong> (original name was: CC228A_contig_1196) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00994_CC228A_..</strong> (original name was: CC228A_contig_354) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00995_CC228A_..</strong> (original name was: CC228A_contig_691) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00996_CC228A_..</strong> (original name was: CC228A_contig_1205) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c00997_CC228A_..</strong> (original name was: CC228A_contig_496) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_98</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_19</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01000_CC228A_..</strong> (original name was: CC228A_contig_986) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01001_CC228A_..</strong> (original name was: CC228A_contig_759) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_78</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01003_CC228A_..</strong> (original name was: CC228A_contig_590) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01005_CC228A_..</strong> (original name was: CC228A_contig_274) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01006_CC228A_..</strong> (original name was: CC228A_contig_854) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01007_CC228A_..</strong> (original name was: CC228A_contig_920) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01008_CC228A_..</strong> (original name was: CC228A_contig_475) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01009_CC228A_..</strong> (original name was: CC228A_contig_828) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01010_CC228A_..</strong> (original name was: CC228A_contig_1142) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01011_CC228A_..</strong> (original name was: CC228A_contig_114) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01012_CC228A_..</strong> (original name was: CC228A_contig_429) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01013_CC228A_..</strong> (original name was: CC228A_contig_836) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01014_CC228A_..</strong> (original name was: CC228A_contig_1207) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_21</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01016_CC228A_..</strong> (original name was: CC228A_contig_848) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01017_CC228A_..</strong> (original name was: CC228A_contig_544) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01018_CC228A_..</strong> (original name was: CC228A_contig_465) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01019_CC228A_..</strong> (original name was: CC228A_contig_514) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01020_CC228A_..</strong> (original name was: CC228A_contig_1075) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01022_CC228A_..</strong> (original name was: CC228A_contig_705) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01023_CC228A_..</strong> (original name was: CC228A_contig_885) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01024_CC228A_..</strong> (original name was: CC228A_contig_246) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01025_CC228A_..</strong> (original name was: CC228A_contig_1028) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01026_CC228A_..</strong> (original name was: CC228A_contig_1215) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01027_CC228A_..</strong> (original name was: CC228A_contig_904) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01028_CC228A_..</strong> (original name was: CC228A_contig_1163) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01029_CC228A_..</strong> (original name was: CC228A_contig_699) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01030_CC228A_..</strong> (original name was: CC228A_contig_849) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01031_CC228A_..</strong> (original name was: CC228A_contig_817) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01032_CC228A_..</strong> (original name was: CC228A_contig_1281) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_89</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01034_CC228A_..</strong> (original name was: CC228A_contig_995) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01035_CC228A_..</strong> (original name was: CC228A_contig_1047) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01036_CC228A_..</strong> (original name was: CC228A_contig_583) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01037_CC228A_..</strong> (original name was: CC228A_contig_657) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_61</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01039_CC228A_..</strong> (original name was: CC228A_contig_807) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01040_CC228A_..</strong> (original name was: CC228A_contig_1180) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01041_CC228A_..</strong> (original name was: CC228A_contig_1249) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01042_CC228A_..</strong> (original name was: CC228A_contig_745) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01043_CC228A_..</strong> (original name was: CC228A_contig_967) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01044_CC228A_..</strong> (original name was: CC228A_contig_775) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01045_CC228A_..</strong> (original name was: CC228A_contig_252) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01046_CC228A_..</strong> (original name was: CC228A_contig_167) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01047_CC228A_..</strong> (original name was: CC228A_contig_820) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01048_CC228A_..</strong> (original name was: CC228A_contig_225) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01049_CC228A_..</strong> (original name was: CC228A_contig_160) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01050_CC228A_..</strong> (original name was: CC228A_contig_1262) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01051_CC228A_..</strong> (original name was: CC228A_contig_1174) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01052_CC228A_..</strong> (original name was: CC228A_contig_588) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01053_CC228A_..</strong> (original name was: CC228A_contig_1040) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01054_CC228A_..</strong> (original name was: CC228A_contig_964) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01055_CC228A_..</strong> (original name was: CC228A_contig_488) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01056_CC228A_..</strong> (original name was: CC228A_contig_199) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01057_CC228A_..</strong> (original name was: CC228A_contig_1086) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01058_CC228A_..</strong> (original name was: CC228A_contig_1235) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01059_CC228A_..</strong> (original name was: CC228A_contig_340) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01061_CC228A_..</strong> (original name was: CC228A_contig_864) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01062_CC228A_..</strong> (original name was: CC228A_contig_388) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01063_CC228A_..</strong> (original name was: CC228A_contig_1169) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01064_CC228A_..</strong> (original name was: CC228A_contig_730) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01065_CC228A_..</strong> (original name was: CC228A_contig_899) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01066_CC228A_..</strong> (original name was: CC228A_contig_507) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_4</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01068_CC228A_..</strong> (original name was: CC228A_contig_217) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01069_CC228A_..</strong> (original name was: CC228A_contig_1162) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01070_CC228A_..</strong> (original name was: CC228A_contig_1102) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01071_CC228A_..</strong> (original name was: CC228A_contig_890) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01073_CC228A_..</strong> (original name was: CC228A_contig_323) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01074_CC228A_..</strong> (original name was: CC228A_contig_1227) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01075_CC228A_..</strong> (original name was: CC228A_contig_131) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01076_CC228A_..</strong> (original name was: CC228A_contig_192) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01077_CC228A_..</strong> (original name was: CC228A_contig_680) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01078_CC228A_..</strong> (original name was: CC228A_contig_840) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01079_CC228A_..</strong> (original name was: CC228A_contig_1216) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01080_CC228A_..</strong> (original name was: CC228A_contig_841) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01081_CC228A_..</strong> (original name was: CC228A_contig_449) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01082_CC228A_..</strong> (original name was: CC228A_contig_1127) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_56</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01084_CC228A_..</strong> (original name was: CC228A_contig_185) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01085_CC228A_..</strong> (original name was: CC228A_contig_772) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01086_CC228A_..</strong> (original name was: CC228A_contig_356) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01087_CC228A_..</strong> (original name was: CC228A_contig_1152) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_88</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01089_CC228A_..</strong> (original name was: CC228A_contig_1222) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01090_CC228A_..</strong> (original name was: CC228A_contig_127) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01091_CC228A_..</strong> (original name was: CC228A_contig_234) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01092_CC228A_..</strong> (original name was: CC228A_contig_1147) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01093_CC228A_..</strong> (original name was: CC228A_contig_579) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_73</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01095_CC228A_..</strong> (original name was: CC228A_contig_798) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01096_CC228A_..</strong> (original name was: CC228A_contig_1157) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01097_CC228A_..</strong> (original name was: CC228A_contig_666) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01098_CC228A_..</strong> (original name was: CC228A_contig_720) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01099_CC228A_..</strong> (original name was: CC228A_contig_1062) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01100_CC228A_..</strong> (original name was: CC228A_contig_503) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01101_CC228A_..</strong> (original name was: CC228A_contig_147) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01102_CC228A_..</strong> (original name was: CC228A_contig_481) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01103_CC228A_..</strong> (original name was: CC228A_contig_918) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01104_CC228A_..</strong> (original name was: CC228A_contig_622) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01105_CC228A_..</strong> (original name was: CC228A_contig_892) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01106_CC228A_..</strong> (original name was: CC228A_contig_285) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_13</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01108_CC228A_..</strong> (original name was: CC228A_contig_674) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01109_CC228A_..</strong> (original name was: CC228A_contig_1064) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01110_CC228A_..</strong> (original name was: CC228A_contig_767) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01111_CC228A_..</strong> (original name was: CC228A_contig_737) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01112_CC228A_..</strong> (original name was: CC228A_contig_265) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01114_CC228A_..</strong> (original name was: CC228A_contig_516) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01115_CC228A_..</strong> (original name was: CC228A_contig_525) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01116_CC228A_..</strong> (original name was: CC228A_contig_132) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01117_CC228A_..</strong> (original name was: CC228A_contig_1264) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01118_CC228A_..</strong> (original name was: CC228A_contig_451) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01119_CC228A_..</strong> (original name was: CC228A_contig_518) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01120_CC228A_..</strong> (original name was: CC228A_contig_119) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01121_CC228A_..</strong> (original name was: CC228A_contig_988) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01122_CC228A_..</strong> (original name was: CC228A_contig_637) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01123_CC228A_..</strong> (original name was: CC228A_contig_1014) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01124_CC228A_..</strong> (original name was: CC228A_contig_877) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01125_CC228A_..</strong> (original name was: CC228A_contig_1089) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01126_CC228A_..</strong> (original name was: CC228A_contig_770) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_41</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01128_CC228A_..</strong> (original name was: CC228A_contig_646) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01129_CC228A_..</strong> (original name was: CC228A_contig_276) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01130_CC228A_..</strong> (original name was: CC228A_contig_426) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01131_CC228A_..</strong> (original name was: CC228A_contig_118) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01132_CC228A_..</strong> (original name was: CC228A_contig_855) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01133_CC228A_..</strong> (original name was: CC228A_contig_936) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01134_CC228A_..</strong> (original name was: CC228A_contig_981) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01135_CC228A_..</strong> (original name was: CC228A_contig_949) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01136_CC228A_..</strong> (original name was: CC228A_contig_458) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01137_CC228A_..</strong> (original name was: CC228A_contig_1092) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01138_CC228A_..</strong> (original name was: CC228A_contig_685) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01139_CC228A_..</strong> (original name was: CC228A_contig_1084) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01140_CC228A_..</strong> (original name was: CC228A_contig_688) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01141_CC228A_..</strong> (original name was: CC228A_contig_924) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01142_CC228A_..</strong> (original name was: CC228A_contig_1050) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01143_CC228A_..</strong> (original name was: CC228A_contig_1198) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01144_CC228A_..</strong> (original name was: CC228A_contig_650) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01145_CC228A_..</strong> (original name was: CC228A_contig_752) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01146_CC228A_..</strong> (original name was: CC228A_contig_427) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01147_CC228A_..</strong> (original name was: CC228A_contig_1055) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01148_CC228A_..</strong> (original name was: CC228A_contig_542) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01149_CC228A_..</strong> (original name was: CC228A_contig_1107) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01150_CC228A_..</strong> (original name was: CC228A_contig_940) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01151_CC228A_..</strong> (original name was: CC228A_contig_850) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_27</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01153_CC228A_..</strong> (original name was: CC228A_contig_312) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01154_CC228A_..</strong> (original name was: CC228A_contig_916) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01156_CC228A_..</strong> (original name was: CC228A_contig_1124) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01157_CC228A_..</strong> (original name was: CC228A_contig_470) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01158_CC228A_..</strong> (original name was: CC228A_contig_619) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01159_CC228A_..</strong> (original name was: CC228A_contig_256) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01160_CC228A_..</strong> (original name was: CC228A_contig_1229) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01161_CC228A_..</strong> (original name was: CC228A_contig_821) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01162_CC228A_..</strong> (original name was: CC228A_contig_398) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01163_CC228A_..</strong> (original name was: CC228A_contig_782) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01164_CC228A_..</strong> (original name was: CC228A_contig_210) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01165_CC228A_..</strong> (original name was: CC228A_contig_922) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01166_CC228A_..</strong> (original name was: CC228A_contig_497) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01167_CC228A_..</strong> (original name was: CC228A_contig_171) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01168_CC228A_..</strong> (original name was: CC228A_contig_178) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01169_CC228A_..</strong> (original name was: CC228A_contig_1236) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_76</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01171_CC228A_..</strong> (original name was: CC228A_contig_1070) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01172_CC228A_..</strong> (original name was: CC228A_contig_112) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_30</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01174_CC228A_..</strong> (original name was: CC228A_contig_869) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01175_CC228A_..</strong> (original name was: CC228A_contig_440) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01176_CC228A_..</strong> (original name was: CC228A_contig_1209) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01177_CC228A_..</strong> (original name was: CC228A_contig_303) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01178_CC228A_..</strong> (original name was: CC228A_contig_570) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01179_CC228A_..</strong> (original name was: CC228A_contig_1015) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01180_CC228A_..</strong> (original name was: CC228A_contig_537) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01181_CC228A_..</strong> (original name was: CC228A_contig_994) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01182_CC228A_..</strong> (original name was: CC228A_contig_1242) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01183_CC228A_..</strong> (original name was: CC228A_contig_979) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01184_CC228A_..</strong> (original name was: CC228A_contig_1073) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01185_CC228A_..</strong> (original name was: CC228A_contig_140) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01186_CC228A_..</strong> (original name was: CC228A_contig_463) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01187_CC228A_..</strong> (original name was: CC228A_contig_902) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01188_CC228A_..</strong> (original name was: CC228A_contig_635) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01189_CC228A_..</strong> (original name was: CC228A_contig_1251) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01190_CC228A_..</strong> (original name was: CC228A_contig_581) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01191_CC228A_..</strong> (original name was: CC228A_contig_956) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01192_CC228A_..</strong> (original name was: CC228A_contig_454) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01193_CC228A_..</strong> (original name was: CC228A_contig_1143) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01194_CC228A_..</strong> (original name was: CC228A_contig_597) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01195_CC228A_..</strong> (original name was: CC228A_contig_181) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01196_CC228A_..</strong> (original name was: CC228A_contig_697) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01197_CC228A_..</strong> (original name was: CC228A_contig_972) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01198_CC228A_..</strong> (original name was: CC228A_contig_511) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01199_CC228A_..</strong> (original name was: CC228A_contig_694) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01200_CC228A_..</strong> (original name was: CC228A_contig_739) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01201_CC228A_..</strong> (original name was: CC228A_contig_1098) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01202_CC228A_..</strong> (original name was: CC228A_contig_447) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01203_CC228A_..</strong> (original name was: CC228A_contig_1160) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01204_CC228A_..</strong> (original name was: CC228A_contig_550) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01205_CC228A_..</strong> (original name was: CC228A_contig_1270) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01206_CC228A_..</strong> (original name was: CC228A_contig_1178) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01208_CC228A_..</strong> (original name was: CC228A_contig_301) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01209_CC228A_..</strong> (original name was: CC228A_contig_1112) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01210_CC228A_..</strong> (original name was: CC228A_contig_805) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01211_CC228A_..</strong> (original name was: CC228A_contig_1149) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01212_CC228A_..</strong> (original name was: CC228A_contig_1167) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01213_CC228A_..</strong> (original name was: CC228A_contig_613) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01214_CC228A_..</strong> (original name was: CC228A_contig_1031) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01215_CC228A_..</strong> (original name was: CC228A_contig_777) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01216_CC228A_..</strong> (original name was: CC228A_contig_1038) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01217_CC228A_..</strong> (original name was: CC228A_contig_341) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01218_CC228A_..</strong> (original name was: CC228A_contig_1045) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01219_CC228A_..</strong> (original name was: CC228A_contig_552) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01220_CC228A_..</strong> (original name was: CC228A_contig_881) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01221_CC228A_..</strong> (original name was: CC228A_contig_232) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01222_CC228A_..</strong> (original name was: CC228A_contig_1176) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01223_CC228A_..</strong> (original name was: CC228A_contig_1233) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01224_CC228A_..</strong> (original name was: CC228A_contig_248) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_82</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01226_CC228A_..</strong> (original name was: CC228A_contig_747) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01228_CC228A_..</strong> (original name was: CC228A_contig_732) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01229_CC228A_..</strong> (original name was: CC228A_contig_857) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01230_CC228A_..</strong> (original name was: CC228A_contig_376) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01231_CC228A_..</strong> (original name was: CC228A_contig_295) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01232_CC228A_..</strong> (original name was: CC228A_contig_194) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01233_CC228A_..</strong> (original name was: CC228A_contig_1122) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01234_CC228A_..</strong> (original name was: CC228A_contig_505) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01235_CC228A_..</strong> (original name was: CC228A_contig_186) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01236_CC228A_..</strong> (original name was: CC228A_contig_927) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01237_CC228A_..</strong> (original name was: CC228A_contig_559) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01238_CC228A_..</strong> (original name was: CC228A_contig_1185) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01239_CC228A_..</strong> (original name was: CC228A_contig_801) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01240_CC228A_..</strong> (original name was: CC228A_contig_1129) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01241_CC228A_..</strong> (original name was: CC228A_contig_321) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01242_CC228A_..</strong> (original name was: CC228A_contig_137) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01243_CC228A_..</strong> (original name was: CC228A_contig_1266) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01244_CC228A_..</strong> (original name was: CC228A_contig_528) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01245_CC228A_..</strong> (original name was: CC228A_contig_586) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01246_CC228A_..</strong> (original name was: CC228A_contig_671) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01247_CC228A_..</strong> (original name was: CC228A_contig_568) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01248_CC228A_..</strong> (original name was: CC228A_contig_1257) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01249_CC228A_..</strong> (original name was: CC228A_contig_215) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01250_CC228A_..</strong> (original name was: CC228A_contig_155) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01251_CC228A_..</strong> (original name was: CC228A_contig_416) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_48</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01253_CC228A_..</strong> (original name was: CC228A_contig_860) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01254_CC228A_..</strong> (original name was: CC228A_contig_862) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_96</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01256_CC228A_..</strong> (original name was: CC228A_contig_682) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01257_CC228A_..</strong> (original name was: CC228A_contig_339) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01258_CC228A_..</strong> (original name was: CC228A_contig_293) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01259_CC228A_..</strong> (original name was: CC228A_contig_722) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01260_CC228A_..</strong> (original name was: CC228A_contig_125) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01261_CC228A_..</strong> (original name was: CC228A_contig_838) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01262_CC228A_..</strong> (original name was: CC228A_contig_102) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01263_CC228A_..</strong> (original name was: CC228A_contig_612) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01265_CC228A_..</strong> (original name was: CC228A_contig_1247) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01266_CC228A_..</strong> (original name was: CC228A_contig_202) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01267_CC228A_..</strong> (original name was: CC228A_contig_1000) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01268_CC228A_..</strong> (original name was: CC228A_contig_900) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01269_CC228A_..</strong> (original name was: CC228A_contig_1104) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01270_CC228A_..</strong> (original name was: CC228A_contig_1224) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01271_CC228A_..</strong> (original name was: CC228A_contig_620) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01272_CC228A_..</strong> (original name was: CC228A_contig_703) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01273_CC228A_..</strong> (original name was: CC228A_contig_120) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01274_CC228A_..</strong> (original name was: CC228A_contig_264) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01275_CC228A_..</strong> (original name was: CC228A_contig_405) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01276_CC228A_..</strong> (original name was: CC228A_contig_655) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>CC228A_contig_58</strong> (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01279_CC228A_..</strong> (original name was: CC228A_contig_668) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01280_CC228A_..</strong> (original name was: CC228A_contig_797) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01281_CC228A_..</strong> (original name was: CC228A_contig_357) (Streptomyces sp. CC228A)</dd>
       
         <dd><strong>c01282_CC228A_..</strong> (original name was: CC228A_contig_1200) (Streptomyces sp. CC228A)</dd>
       
     </dl>
     
    </div>
    
    <div class="overview-switches">
     <div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    
   </div>
  </div>
  






<div class="page" id='r74c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00074_CC228A_.. - Region 1 - T3PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-167"></div> <span class="help-tooltip" id="region_svg-help-167">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00074_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r74c1-svg" data-filename="c00074_CC228A_.._r74c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 3,340 nt. (total: 3,340 nt)
        
        <a class="cluster-rules-header" id="r74c1-rules-header" href="#r74c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r74c1-rules">T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)</div>
      
      <div class="region-svg-container">
       <div id='r74c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-168"></div> <span class="help-tooltip" id="focus-panel-help-168">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r74c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
    
     
      <div class="body-details-header r74c1-clusterblast" data-name="r74c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r74c1-knownclusterblast" data-name="r74c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r74c1-subclusterblast" data-name="r74c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r74c1-MIBiG-cluster-compare" data-name="r74c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r74c1-pfam-details" data-name="r74c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    
     
    <div class="body-details-section r74c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-1"></div> <span class="help-tooltip" id="cb-general-help-1">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r74c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r74c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r74c1_1.svg">
        NZ_CP023696 (6584237-6660692): Streptomyces fradiae ATCC 10745 = DSM 40063 st... (5% of genes show similarity), NRPS-like,T3PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r74c1_2.svg">
        NZ_KB846715 (629123-670200): Streptomyces sp. CNS615 D583DRAFT scaffold00001.... (11% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r74c1_3.svg">
        NZ_AZXC01000014 (64938-106015): Streptomyces sp. CNR698 B101DRAFT scaffold 11... (11% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r74c1_4.svg">
        NZ_KB898270 (400380-441457): Streptomyces sp. CNT302 D329DRAFT scaffold00001.... (10% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r74c1_5.svg">
        NZ_CP029188 (5671634-5712711): Streptomyces tirandamycinicus strain HNM0039 c... (11% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r74c1_6.svg">
        NZ_FNHI01000001 (397395-438472): Streptomyces wuyuanensis strain CGMCC 4.7042... (11% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r74c1_7.svg">
        NZ_CP029254 (6056077-6097154): Streptomyces spongiicola strain HNM0071 chromo... (11% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r74c1_8.svg">
        NZ_CP030862 (2513461-2554523): Streptomyces globosus strain LZH-48 chromosome... (12% of genes show similarity), T3PKS
      </option>
      
      <option value = "svg/clusterblast_r74c1_9.svg">
        NZ_CP023692 (1240058-1291960): Streptomyces vinaceus strain ATCC 27476 chromo... (10% of genes show similarity), T3PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r74c1_10.svg">
        NZ_CP045031 (6806855-6862000): Streptomyces sp. C8S0 chromosome (5% of genes show similarity), T3PKS,terpene
      </option>
      
    </select>
    <button id= "clusterblast-r74c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r74c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r74c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-2"></div> <span class="help-tooltip" id="cb-known-help-2">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r74c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r74c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r74c1_1.svg">
        flaviolin rhamnoside / 3,3&#39;-diflaviolin / flaviolin
      </option>
      
      <option value = "svg/knownclusterblast_r74c1_2.svg">
        flaviolin
      </option>
      
      <option value = "svg/knownclusterblast_r74c1_3.svg">
        merochlorin A / merochlorin B / deschloro-merochlorin A / deschloro-merochlorin B / isochloro-merochlorin B / dichloro-merochlorin B / merochlorin D / merochlorin C
      </option>
      
      <option value = "svg/knownclusterblast_r74c1_4.svg">
        napyradiomycin
      </option>
      
      <option value = "svg/knownclusterblast_r74c1_5.svg">
        napyradiomycin A80915C
      </option>
      
      <option value = "svg/knownclusterblast_r74c1_6.svg">
        furaquinocin B
      </option>
      
    </select>
    <button id="knownclusterblast-r74c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r74c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r74c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-3"></div> <span class="help-tooltip" id="cb-sub-help-3">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r74c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-4"></div> <span class="help-tooltip" id="MIBiG-help-4">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r74c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r74c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r74c1"></div>
    <div id="comparison-MIBiG-r74c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">T3PKS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310.1: 6460417-6463122">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
         
        
        <td>0.54</td>
        <td>Terpene</td>
        
        <td>naringenin</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.55, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>Other</td>
        
        <td>flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001938.1: 90047-91866">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001938/index.html#r1c1" target="_new">BGC0001938.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Polyketide</td>
        
        <td>entotheonella</td>
        
        
        <td>Candidatus Entotheonella serta</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.55, order:0.76, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Polyketide, Saccharide</td>
        
        <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
        
        
        <td>Saccharopolyspora erythraea</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282.1: 548717-551330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>Polyketide</td>
        
        <td>alkylresorcinol</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000280.1: 384-6033">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000280/index.html#r1c1" target="_new">BGC0000280.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Polyketide</td>
        
        <td>2,4-diacetylphloroglucinol</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001962.1: 0-7027">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001962/index.html#r1c1" target="_new">BGC0001962.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Polyketide</td>
        
        <td>hierridin B, hierridin C</td>
        
        
        <td>Cyanobium sp. LEGE 06113</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000281.1: 0-8137">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000281/index.html#r1c1" target="_new">BGC0000281.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.49, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Polyketide</td>
        
        <td>2,4-diacetylphloroglucinol</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Alkaloid</td>
        
        <td>kitasetaline</td>
        
        
        <td>Kitasatospora setae KM-6054</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001948.1: 62-3579">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001948/index.html#r1c1" target="_new">BGC0001948.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>naseseazine C, C3-aryl pyrroloindolines</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r74c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310.1: 6460417-6463122">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
      <td>0.81</td>
      <td>Terpene</td>
      <td>naringenin</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000902.1: 1363587-1367490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000902/index.html#r1c1" target="_new">BGC0000902.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.55, order:0.76, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>flaviolin</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001938.1: 90047-91866">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001938/index.html#r1c1" target="_new">BGC0001938.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide</td>
      <td>entotheonella</td>
      <td>Candidatus Entotheonella serta</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000285.1: 2-11002">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000285/index.html#r1c1" target="_new">BGC0000285.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.55, order:0.76, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Polyketide, Saccharide</td>
      <td>flaviolin rhamnoside, 3,3&#39;-diflaviolin, flaviolin</td>
      <td>Saccharopolyspora erythraea</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282.1: 548717-551330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>alkylresorcinol</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000280.1: 384-6033">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000280/index.html#r1c1" target="_new">BGC0000280.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Polyketide</td>
      <td>2,4-diacetylphloroglucinol</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001962.1: 0-7027">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001962/index.html#r1c1" target="_new">BGC0001962.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Polyketide</td>
      <td>hierridin B, hierridin C</td>
      <td>Cyanobium sp. LEGE 06113</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000281.1: 0-8137">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000281/index.html#r1c1" target="_new">BGC0000281.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.49, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Polyketide</td>
      <td>2,4-diacetylphloroglucinol</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Alkaloid</td>
      <td>kitasetaline</td>
      <td>Kitasatospora setae KM-6054</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001948.1: 62-3579">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001948/index.html#r1c1" target="_new">BGC0001948.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>naseseazine C, C3-aryl pyrroloindolines</td>
      <td>Streptomyces sp.</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r74c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-5"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-5">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r74c1-pfam-details-svg" data-filename="c00074_CC228A_.._r74c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r74c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r151c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00151_CC228A_.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-169"></div> <span class="help-tooltip" id="region_svg-help-169">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00151_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r151c1-svg" data-filename="c00151_CC228A_.._r151c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 1,704 nt. (total: 1,704 nt)
        
        <a class="cluster-rules-header" id="r151c1-rules-header" href="#r151c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r151c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r151c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-170"></div> <span class="help-tooltip" id="focus-panel-help-170">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r151c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r151c1-clusterblast" data-name="r151c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r151c1-knownclusterblast" data-name="r151c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r151c1-subclusterblast" data-name="r151c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r151c1-MIBiG-cluster-compare" data-name="r151c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r151c1-pfam-details" data-name="r151c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r151c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-6"></div> <span class="help-tooltip" id="cb-general-help-6">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <p>No significant ClusterBlast hits found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r151c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-7"></div> <span class="help-tooltip" id="cb-known-help-7">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r151c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-8"></div> <span class="help-tooltip" id="cb-sub-help-8">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r151c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-9"></div> <span class="help-tooltip" id="MIBiG-help-9">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r151c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r151c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r151c1"></div>
    <div id="comparison-MIBiG-r151c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000670.1: 19428969-19461538">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000670/index.html#r1c1" target="_new">BGC0000670.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>Terpene</td>
        
        <td>thalianol, thaliandiol</td>
        
        
        <td>Arabidopsis thaliana</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663.1: 7516016-7529773">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.34 (id:0.61, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
         
        
        <td>0.09</td>
        <td>Terpene</td>
        
        <td>hopene</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000651.1: 740-35007">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000651/index.html#r1c1" target="_new">BGC0000651.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.24 (id:0.43, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
         
        
        <td>0.05</td>
        <td>Terpene</td>
        
        <td>KS-505a</td>
        
        
        <td>Streptomyces argenteolus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002019.1: 0-24752">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002019/index.html#r1c1" target="_new">BGC0002019.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.20 (id:0.44, order:N/A, components:0.09)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
         
        
        <td>0.04</td>
        <td>Terpene</td>
        
        <td>tiancilactone</td>
        
        
        <td>Streptomyces sp. CB03234</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r151c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000670.1: 19428969-19461538">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000670/index.html#r1c1" target="_new">BGC0000670.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>Terpene</td>
      <td>thalianol, thaliandiol</td>
      <td>Arabidopsis thaliana</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663.1: 7516016-7529773">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.34 (id:0.61, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
      <td>0.34</td>
      <td>Terpene</td>
      <td>hopene</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000651.1: 740-35007">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000651/index.html#r1c1" target="_new">BGC0000651.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.24 (id:0.43, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
      <td>0.24</td>
      <td>Terpene</td>
      <td>KS-505a</td>
      <td>Streptomyces argenteolus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002019.1: 0-24752">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002019/index.html#r1c1" target="_new">BGC0002019.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.20 (id:0.44, order:N/A, components:0.09)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
      <td>0.20</td>
      <td>Terpene</td>
      <td>tiancilactone</td>
      <td>Streptomyces sp. CB03234</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r151c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-10"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-10">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r151c1-pfam-details-svg" data-filename="c00151_CC228A_.._r151c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r151c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r218c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC228A_contig_23 - Region 1 - siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-171"></div> <span class="help-tooltip" id="region_svg-help-171">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC228A_contig_23.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r218c1-svg" data-filename="CC228A_contig_23_r218c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 7,367 - 16,762 nt. (total: 9,396 nt)
        
        <a class="cluster-rules-header" id="r218c1-rules-header" href="#r218c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r218c1-rules">siderophore: IucA_IucC</div>
      
      <div class="region-svg-container">
       <div id='r218c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-172"></div> <span class="help-tooltip" id="focus-panel-help-172">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r218c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r218c1-clusterblast" data-name="r218c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r218c1-knownclusterblast" data-name="r218c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r218c1-subclusterblast" data-name="r218c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r218c1-MIBiG-cluster-compare" data-name="r218c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r218c1-pfam-details" data-name="r218c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r218c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-11"></div> <span class="help-tooltip" id="cb-general-help-11">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r218c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r218c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r218c1_1.svg">
        NZ_CP045031 (4535634-4547419): Streptomyces sp. C8S0 chromosome (77% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r218c1_2.svg">
        NZ_CP023696 (2301411-2312813): Streptomyces fradiae ATCC 10745 = DSM 40063 st... (77% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r218c1_3.svg">
        NZ_CP017316 (2154689-2166006): Streptomyces rubrolavendulae strain MJM4426 ch... (77% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r218c1_4.svg">
        NZ_JABBGE010000001 (181067-192837): Streptomyces sp. R302 contig1, whole geno... (70% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r218c1_5.svg">
        NZ_CP029188 (2046454-2057668): Streptomyces tirandamycinicus strain HNM0039 c... (70% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r218c1_6.svg">
        NZ_KB846746 (17500-29282): Streptomyces sp. CNS615 D583DRAFT scaffold00032.32... (70% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r218c1_7.svg">
        NZ_AZXC01000001 (615601-627383): Streptomyces sp. CNR698 B101DRAFT scaffold 0... (70% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r218c1_8.svg">
        NZ_KB898273 (367341-379123): Streptomyces sp. CNT302 D329DRAFT scaffold00004.... (70% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r218c1_9.svg">
        NZ_CP029254 (1675353-1746690): Streptomyces spongiicola strain HNM0071 chromo... (11% of genes show similarity), T2PKS,siderophore
      </option>
      
      <option value = "svg/clusterblast_r218c1_10.svg">
        NZ_LIPG01000002 (310712-322494): Streptomyces sp. CB02058 scaffold2, whole ge... (87% of genes show similarity), siderophore
      </option>
      
    </select>
    <button id= "clusterblast-r218c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r218c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r218c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-12"></div> <span class="help-tooltip" id="cb-known-help-12">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r218c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r218c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r218c1_1.svg">
        desferrioxamin B / desferrioxamine E
      </option>
      
      <option value = "svg/knownclusterblast_r218c1_2.svg">
        desferrioxamin B
      </option>
      
      <option value = "svg/knownclusterblast_r218c1_3.svg">
        desferrioxamine
      </option>
      
      <option value = "svg/knownclusterblast_r218c1_4.svg">
        desferrioxamine E
      </option>
      
      <option value = "svg/knownclusterblast_r218c1_5.svg">
        putrebactin / avaroferrin
      </option>
      
      <option value = "svg/knownclusterblast_r218c1_6.svg">
        desferrioxamine E
      </option>
      
      <option value = "svg/knownclusterblast_r218c1_7.svg">
        bisucaberin B
      </option>
      
    </select>
    <button id="knownclusterblast-r218c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r218c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r218c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-13"></div> <span class="help-tooltip" id="cb-sub-help-13">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r218c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-14"></div> <span class="help-tooltip" id="MIBiG-help-14">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r218c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r218c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r218c1"></div>
    <div id="comparison-MIBiG-r218c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">siderophore</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000941.1: 5578821-5584948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000941/index.html#r1c1" target="_new">BGC0000941.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
         
        
        <td>0.58</td>
        <td>Other</td>
        
        <td>desferrioxamin B</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001478.1: 100-5365">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001478/index.html#r1c1" target="_new">BGC0001478.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
         
        
        <td>0.58</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Streptomyces sp. ID38640</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940.1: 3033354-3040682">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.76 (id:0.58, order:0.83, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
         
        
        <td>0.45</td>
        <td>Other</td>
        
        <td>desferrioxamin B, desferrioxamine E</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453.1: 0-7234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.76 (id:0.58, order:0.83, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
         
        
        <td>0.45</td>
        <td>Other</td>
        
        <td>desferrioxamine</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.52, order:0.66, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Pantoea agglomerans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001531.1: 1112-9953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001531/index.html#r1c1" target="_new">BGC0001531.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.48, order:0.57, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Other</td>
        
        <td>bisucaberin B</td>
        
        
        <td>Tenacibaculum mesophilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000244.1: 204-8850">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000244/index.html#r1c1" target="_new">BGC0000244.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.46, order:0.48, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>macrotetrolide</td>
        
        
        <td>Streptomyces griseus subsp. griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>ochrobactin</td>
        
        
        <td>Ochrobactrum anthropi</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499.1: 2267-11578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:0.42, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>aerobactin</td>
        
        
        <td>Pantoea ananatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000942.1: 1864728-1871818">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000942/index.html#r1c1" target="_new">BGC0000942.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>petrobactin</td>
        
        
        <td>Bacillus anthracis str. Ames</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r218c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000941.1: 5578821-5584948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000941/index.html#r1c1" target="_new">BGC0000941.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
      <td>0.83</td>
      <td>Other</td>
      <td>desferrioxamin B</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001478.1: 100-5365">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001478/index.html#r1c1" target="_new">BGC0001478.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
      <td>0.83</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Streptomyces sp. ID38640</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940.1: 3033354-3040682">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.76 (id:0.58, order:0.83, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
      <td>0.76</td>
      <td>Other</td>
      <td>desferrioxamin B, desferrioxamine E</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453.1: 0-7234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.76 (id:0.58, order:0.83, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
      <td>0.76</td>
      <td>Other</td>
      <td>desferrioxamine</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.52, order:0.66, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Pantoea agglomerans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001531.1: 1112-9953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001531/index.html#r1c1" target="_new">BGC0001531.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.48, order:0.57, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Other</td>
      <td>bisucaberin B</td>
      <td>Tenacibaculum mesophilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000244.1: 204-8850">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000244/index.html#r1c1" target="_new">BGC0000244.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.46, order:0.48, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Polyketide</td>
      <td>macrotetrolide</td>
      <td>Streptomyces griseus subsp. griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>ochrobactin</td>
      <td>Ochrobactrum anthropi</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001499.1: 2267-11578">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001499/index.html#r1c1" target="_new">BGC0001499.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:0.42, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>aerobactin</td>
      <td>Pantoea ananatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000942.1: 1864728-1871818">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000942/index.html#r1c1" target="_new">BGC0000942.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Other</td>
      <td>petrobactin</td>
      <td>Bacillus anthracis str. Ames</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r218c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-15"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-15">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r218c1-pfam-details-svg" data-filename="CC228A_contig_23_r218c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r218c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r232c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC228A_contig_92 - Region 1 - ladderane <div class="help-container"> <div class="help-icon" data-id="region_svg-help-173"></div> <span class="help-tooltip" id="region_svg-help-173">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC228A_contig_92.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r232c1-svg" data-filename="CC228A_contig_92_r232c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 9,993 nt. (total: 9,993 nt)
        
        <a class="cluster-rules-header" id="r232c1-rules-header" href="#r232c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r232c1-rules">ladderane: ladderane</div>
      
      <div class="region-svg-container">
       <div id='r232c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-174"></div> <span class="help-tooltip" id="focus-panel-help-174">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r232c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r232c1-clusterblast" data-name="r232c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r232c1-knownclusterblast" data-name="r232c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r232c1-subclusterblast" data-name="r232c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r232c1-MIBiG-cluster-compare" data-name="r232c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r232c1-pfam-details" data-name="r232c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r232c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-16"></div> <span class="help-tooltip" id="cb-general-help-16">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r232c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r232c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r232c1_1.svg">
        NZ_CP023699 (4406444-4458613): Streptomyces kanamyceticus strain ATCC 12853 c... (22% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r232c1_2.svg">
        NZ_CP022685 (5330727-5382893): Streptomyces formicae strain KY5 chromosome, c... (22% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r232c1_3.svg">
        NZ_KB900319 (472302-525831): Salinispora arenicola CNR107 F583DRAFT scaffold0... (18% of genes show similarity), NRPS,arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r232c1_4.svg">
        NZ_CP049780 (4854996-4906677): Streptomyces sp. JB150 chromosome (8% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r232c1_5.svg">
        NZ_CP046905 (4842638-4894497): Streptomyces sp. QHH-9511 chromosome, complete... (8% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r232c1_6.svg">
        NZ_ML769317 (93792-169608): Catellatospora paridis strain NEAU-CL2 Scaffold6,... (6% of genes show similarity), NRPS,ectoine,ladderane
      </option>
      
      <option value = "svg/clusterblast_r232c1_7.svg">
        NZ_CP029554 (3984172-4040310): Chromobacterium sp. IIBBL 274-1 chromosome, co... (10% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r232c1_8.svg">
        NZ_FOEO01000008 (3174-62641): Pseudomonas sp. NFACC02, whole genome shotgun s... (11% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r232c1_9.svg">
        NZ_NJAK01000001 (828647-882244): Xenorhabdus ishibashii strain DSM 22670 Xish... (9% of genes show similarity), NRPS,arylpolyene
      </option>
      
      <option value = "svg/clusterblast_r232c1_10.svg">
        NZ_AYSJ01000013 (436584-477769): Photorhabdus khanii NC19 PTE gnl UNH Scaf 13... (8% of genes show similarity), ladderane
      </option>
      
    </select>
    <button id= "clusterblast-r232c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r232c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r232c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-17"></div> <span class="help-tooltip" id="cb-known-help-17">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r232c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-18"></div> <span class="help-tooltip" id="cb-sub-help-18">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r232c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-19"></div> <span class="help-tooltip" id="MIBiG-help-19">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r232c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r232c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r232c1"></div>
    <div id="comparison-MIBiG-r232c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">ladderane</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Polyketide</td>
        
        <td>naphtocyclinone</td>
        
        
        <td>Streptomyces arenae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000905.1: 1-16359">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000905/index.html#r1c1" target="_new">BGC0000905.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>heme D1</td>
        
        
        <td>Heliobacillus mobilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000906.1: 424-16949">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000906/index.html#r1c1" target="_new">BGC0000906.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>heme D1</td>
        
        
        <td>Heliophilum fasciatum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>spore pigment</td>
        
        
        <td>Streptomyces collinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000243.1: 338-14804">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000243/index.html#r1c1" target="_new">BGC0000243.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>macrotetrolide</td>
        
        
        <td>Streptomyces griseus subsp. griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000919.1: 746-6062">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000919/index.html#r1c1" target="_new">BGC0000919.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>phaseolotoxin</td>
        
        
        <td>Pseudomonas syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>curamycin</td>
        
        
        <td>Streptomyces cyaneus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258.1: 73-7295">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>1,3,6,8-tetrahydroxynaphthalene</td>
        
        
        <td>Glarea lozoyensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Terpene</td>
        
        <td>platencin</td>
        
        
        <td>Streptomyces platensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001987.1: 701808-716497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001987/index.html#r1c1" target="_new">BGC0001987.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Polyketide</td>
        
        <td>koreenceine A, koreenceine B, koreenceine C, koreenceine D</td>
        
        
        <td>Pseudomonas koreensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r232c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000248.1: 1435-5236">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000248/index.html#r1c1" target="_new">BGC0000248.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Polyketide</td>
      <td>naphtocyclinone</td>
      <td>Streptomyces arenae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000905.1: 1-16359">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000905/index.html#r1c1" target="_new">BGC0000905.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>heme D1</td>
      <td>Heliobacillus mobilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000906.1: 424-16949">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000906/index.html#r1c1" target="_new">BGC0000906.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Other</td>
      <td>heme D1</td>
      <td>Heliophilum fasciatum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272.1: 614-8926">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Polyketide</td>
      <td>spore pigment</td>
      <td>Streptomyces collinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000243.1: 338-14804">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000243/index.html#r1c1" target="_new">BGC0000243.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Polyketide</td>
      <td>macrotetrolide</td>
      <td>Streptomyces griseus subsp. griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000919.1: 746-6062">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000919/index.html#r1c1" target="_new">BGC0000919.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>phaseolotoxin</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000215.1: 200-5446">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000215/index.html#r1c1" target="_new">BGC0000215.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Polyketide</td>
      <td>curamycin</td>
      <td>Streptomyces cyaneus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258.1: 73-7295">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Polyketide</td>
      <td>1,3,6,8-tetrahydroxynaphthalene</td>
      <td>Glarea lozoyensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Terpene</td>
      <td>platencin</td>
      <td>Streptomyces platensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001987.1: 701808-716497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001987/index.html#r1c1" target="_new">BGC0001987.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Polyketide</td>
      <td>koreenceine A, koreenceine B, koreenceine C, koreenceine D</td>
      <td>Pseudomonas koreensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r232c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-20"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-20">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r232c1-pfam-details-svg" data-filename="CC228A_contig_92_r232c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r232c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r259c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00259_CC228A_.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-175"></div> <span class="help-tooltip" id="region_svg-help-175">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00259_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r259c1-svg" data-filename="c00259_CC228A_.._r259c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 8,843 nt. (total: 8,843 nt)
        
        <a class="cluster-rules-header" id="r259c1-rules-header" href="#r259c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r259c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r259c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-176"></div> <span class="help-tooltip" id="focus-panel-help-176">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r259c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r259c1-nrps_pks" data-name="r259c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r259c1-clusterblast" data-name="r259c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r259c1-knownclusterblast" data-name="r259c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r259c1-subclusterblast" data-name="r259c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r259c1-MIBiG-cluster-compare" data-name="r259c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r259c1-pfam-details" data-name="r259c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r259c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-21"></div> <span class="help-tooltip" id="nrps-domain-help-21">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r259c1-details-svg" data-filename="c00259_CC228A_.._r259c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r259c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r259c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-24"></div> <span class="help-tooltip" id="cb-general-help-24">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r259c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r259c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r259c1_1.svg">
        NZ_CP045031 (1082381-1134763): Streptomyces sp. C8S0 chromosome (17% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r259c1_2.svg">
        NZ_CP045740 (2042812-2083703): Streptomyces sp. SUK 48 chromosome, complete g... (19% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r259c1_3.svg">
        NZ_WWGN01000023 (331442-372930): Streptomyces sp. SID8366 SID8366.c23, whole ... (19% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r259c1_4.svg">
        NZ_CP029188 (3418746-3461521): Streptomyces tirandamycinicus strain HNM0039 c... (20% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r259c1_5.svg">
        NZ_KB898276 (67201-110504): Streptomyces sp. CNT302 D329DRAFT scaffold00007.7... (20% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r259c1_6.svg">
        NZ_LIVX01000007 (280659-353330): Streptomyces sp. CB02261 scaffold7, whole ge... (10% of genes show similarity), NRPS,betalactone
      </option>
      
      <option value = "svg/clusterblast_r259c1_7.svg">
        NZ_LIVV01000003 (272331-350807): Streptomyces sp. CB02009 scaffold3, whole ge... (12% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r259c1_8.svg">
        NZ_FOGI01000001 (788351-831606): Actinokineospora terrae strain DSM 44260, wh... (17% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r259c1_9.svg">
        NZ_RCDD01000005 (354216-397471): Actinokineospora cianjurensis strain DSM 456... (16% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r259c1_10.svg">
        NZ_NDXL01000001 (182213-252513): Streptomyces kasugaensis strain BCRC 12349 s... (6% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r259c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r259c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r259c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-25"></div> <span class="help-tooltip" id="cb-known-help-25">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r259c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r259c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r259c1_1.svg">
        thiolutin
      </option>
      
      <option value = "svg/knownclusterblast_r259c1_2.svg">
        holomycin
      </option>
      
      <option value = "svg/knownclusterblast_r259c1_3.svg">
        thiomarinol
      </option>
      
    </select>
    <button id="knownclusterblast-r259c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r259c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r259c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-26"></div> <span class="help-tooltip" id="cb-sub-help-26">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r259c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-27"></div> <span class="help-tooltip" id="MIBiG-help-27">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r259c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r259c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r259c1"></div>
    <div id="comparison-MIBiG-r259c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131.1: 144016-173487">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>ambactin</td>
        
        
        <td>Xenorhabdus miraniensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
        
        
        <td>Penicillium rubens Wisconsin 54-1255</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r259c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131.1: 144016-173487">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>ambactin</td>
      <td>Xenorhabdus miraniensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>NRP</td>
      <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
      <td>Penicillium rubens Wisconsin 54-1255</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r259c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-28"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-28">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r259c1-pfam-details-svg" data-filename="c00259_CC228A_.._r259c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r259c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r259c1-nrps_pks_products" data-name="r259c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r259c1-nrps_pks" data-name="r259c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r259c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-22"></div> <span class="help-tooltip" id="nrps-structures-help-22">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 8843:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r259c1-1" class="smiles-canvas" data-smiles="NC(CS)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(cys)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r259c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CS)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r259c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-23"></div> <span class="help-tooltip" id="nrps-monomers-help-23">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds5426_8705</span></strong>: cys
      <div class="collapser collapser-target-cds5426_8705 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (501..902)</strong>: cys
          <div class="collapser collapser-target-cds5426_8705 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  cys
              <div class="collapser collapser-target-cds5426_8705-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Small clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Single AA prediction:</dt>
   <dd>cys</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLYNLSLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>cys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r297c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00297_CC228A_.. - Region 1 - PKS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-177"></div> <span class="help-tooltip" id="region_svg-help-177">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00297_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r297c1-svg" data-filename="c00297_CC228A_.._r297c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 6,896 nt. (total: 6,896 nt)
        
        <a class="cluster-rules-header" id="r297c1-rules-header" href="#r297c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r297c1-rules">PKS-like: (t2pks2 or ksIII)</div>
      
      <div class="region-svg-container">
       <div id='r297c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-178"></div> <span class="help-tooltip" id="focus-panel-help-178">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r297c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r297c1-nrps_pks" data-name="r297c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r297c1-clusterblast" data-name="r297c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r297c1-knownclusterblast" data-name="r297c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r297c1-subclusterblast" data-name="r297c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r297c1-MIBiG-cluster-compare" data-name="r297c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r297c1-pfam-details" data-name="r297c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r297c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-29"></div> <span class="help-tooltip" id="nrps-domain-help-29">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r297c1-details-svg" data-filename="c00297_CC228A_.._r297c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r297c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r297c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-30"></div> <span class="help-tooltip" id="cb-general-help-30">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r297c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r297c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r297c1_1.svg">
        NZ_WWGT01000021 (68799-109759): Streptomyces sp. SID8358 SID8358.c21, whole g... (17% of genes show similarity), PKS-like
      </option>
      
      <option value = "svg/clusterblast_r297c1_2.svg">
        NC_015953 (2409340-2449784): Streptomyces sp. SirexAA-E, complete sequence (17% of genes show similarity), PKS-like
      </option>
      
      <option value = "svg/clusterblast_r297c1_3.svg">
        NZ_PTJS01000001 (2950557-2991517): Streptomyces sp. QL37 scaffold1 size877290... (20% of genes show similarity), PKS-like
      </option>
      
      <option value = "svg/clusterblast_r297c1_4.svg">
        NZ_CP058977 (1316993-1356574): Streptomyces sp. NEAU-sy36 chromosome, complet... (17% of genes show similarity), PKS-like
      </option>
      
      <option value = "svg/clusterblast_r297c1_5.svg">
        NZ_FLTQ01000035 (434491-475451): Streptomyces sp. MnatMP-M77, whole genome sh... (15% of genes show similarity), PKS-like
      </option>
      
      <option value = "svg/clusterblast_r297c1_6.svg">
        NZ_GL877172 (6435028-6470622): Streptomyces sp. ACT-1 strain XylebKG-1 scaffo... (15% of genes show similarity), PKS-like
      </option>
      
      <option value = "svg/clusterblast_r297c1_7.svg">
        NC_010572 (6220924-6256545): Streptomyces griseus subsp. griseus NBRC 13350, ... (15% of genes show similarity), PKS-like
      </option>
      
      <option value = "svg/clusterblast_r297c1_8.svg">
        NZ_LMHM01000013 (711059-752031): Lysobacter sp. Root690 contig 4, whole genom... (12% of genes show similarity), PKS-like
      </option>
      
      <option value = "svg/clusterblast_r297c1_9.svg">
        NZ_CP023465 (3682845-3723838): Lysobacter capsici strain KNU-14 chromosome, c... (12% of genes show similarity), PKS-like
      </option>
      
      <option value = "svg/clusterblast_r297c1_10.svg">
        NZ_KL573540 (212840-285730): Streptomyces baarnensis strain NRRL B-2842 P144 ... (9% of genes show similarity), NRPS,PKS-like
      </option>
      
    </select>
    <button id= "clusterblast-r297c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r297c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r297c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-31"></div> <span class="help-tooltip" id="cb-known-help-31">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r297c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-32"></div> <span class="help-tooltip" id="cb-sub-help-32">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r297c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-33"></div> <span class="help-tooltip" id="MIBiG-help-33">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r297c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r297c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r297c1"></div>
    <div id="comparison-MIBiG-r297c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">PKS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000883.1: 0-30199">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000883/index.html#r1c1" target="_new">BGC0000883.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>Other</td>
        
        <td>arginomycin</td>
        
        
        <td>Streptomyces arginensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000805.1: 897-5623">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000805/index.html#r1c1" target="_new">BGC0000805.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Saccharide</td>
        
        <td>moenomycin</td>
        
        
        <td>Streptomyces viridosporus ATCC 14672</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Polyketide</td>
        
        <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
        
        
        <td>Parastagonospora nodorum SN15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001239.1: 107-6956">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001239/index.html#r1c1" target="_new">BGC0001239.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>biotin</td>
        
        
        <td>Aspergillus nidulans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001238.1: 107-6955">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001238/index.html#r1c1" target="_new">BGC0001238.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>biotin</td>
        
        
        <td>Aspergillus nidulans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001985.1: 0-26853">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001985/index.html#r1c1" target="_new">BGC0001985.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>Other</td>
        
        <td>pyrazomycin</td>
        
        
        <td>Streptomyces candidus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001633.1: 52623-58838">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001633/index.html#r1c1" target="_new">BGC0001633.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>Other</td>
        
        <td>ketomemicin B3, ketomemicin B4</td>
        
        
        <td>Streptomyces mobaraensis NBRC 13819 = DSM 40847</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124.1: 125832-132922">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
        <td>0.11</td>
        <td>Polyketide</td>
        
        <td>pyranonigrin E</td>
        
        
        <td>Aspergillus niger ATCC 1015</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001899.1: 752398-807208">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001899/index.html#r1c1" target="_new">BGC0001899.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
         
        
        <td>0.11</td>
        <td>Polyketide</td>
        
        <td>abscisic acid</td>
        
        
        <td>Leptosphaeria maculans JN3</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001850.1: 258926-281351">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001850/index.html#r1c1" target="_new">BGC0001850.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.35 (id:0.43, order:0.29, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
         
        
        <td>0.10</td>
        <td>Other,  , (, S, h, i, k, i, m, a, t, e, -, d, e, r, i, v, e, d, )</td>
        
        <td>diazaquinomycin A, diazaquinomycin E, diazaquinomycin F, diazaquinomycin G</td>
        
        
        <td>Streptomyces sp. F001</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r297c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000883.1: 0-30199">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000883/index.html#r1c1" target="_new">BGC0000883.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Other</td>
      <td>arginomycin</td>
      <td>Streptomyces arginensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000805.1: 897-5623">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000805/index.html#r1c1" target="_new">BGC0000805.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Saccharide</td>
      <td>moenomycin</td>
      <td>Streptomyces viridosporus ATCC 14672</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858.1: 703862-722630">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Polyketide</td>
      <td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
      <td>Parastagonospora nodorum SN15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001239.1: 107-6956">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001239/index.html#r1c1" target="_new">BGC0001239.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other</td>
      <td>biotin</td>
      <td>Aspergillus nidulans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001238.1: 107-6955">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001238/index.html#r1c1" target="_new">BGC0001238.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other</td>
      <td>biotin</td>
      <td>Aspergillus nidulans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001985.1: 0-26853">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001985/index.html#r1c1" target="_new">BGC0001985.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>Other</td>
      <td>pyrazomycin</td>
      <td>Streptomyces candidus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001633.1: 52623-58838">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001633/index.html#r1c1" target="_new">BGC0001633.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>Other</td>
      <td>ketomemicin B3, ketomemicin B4</td>
      <td>Streptomyces mobaraensis NBRC 13819 = DSM 40847</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124.1: 125832-132922">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
      <td>0.38</td>
      <td>Polyketide</td>
      <td>pyranonigrin E</td>
      <td>Aspergillus niger ATCC 1015</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001899.1: 752398-807208">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001899/index.html#r1c1" target="_new">BGC0001899.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
      <td>0.37</td>
      <td>Polyketide</td>
      <td>abscisic acid</td>
      <td>Leptosphaeria maculans JN3</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001850.1: 258926-281351">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001850/index.html#r1c1" target="_new">BGC0001850.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.35 (id:0.43, order:0.29, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
      <td>0.35</td>
      <td>Other,  , (, S, h, i, k, i, m, a, t, e, -, d, e, r, i, v, e, d, )</td>
      <td>diazaquinomycin A, diazaquinomycin E, diazaquinomycin F, diazaquinomycin G</td>
      <td>Streptomyces sp. F001</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r297c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-34"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-34">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r297c1-pfam-details-svg" data-filename="c00297_CC228A_.._r297c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r297c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r338c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00338_CC228A_.. - Region 1 - butyrolactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-179"></div> <span class="help-tooltip" id="region_svg-help-179">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00338_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r338c1-svg" data-filename="c00338_CC228A_.._r338c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 4,939 nt. (total: 4,939 nt)
        
        <a class="cluster-rules-header" id="r338c1-rules-header" href="#r338c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r338c1-rules">butyrolactone: AfsA</div>
      
      <div class="region-svg-container">
       <div id='r338c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-180"></div> <span class="help-tooltip" id="focus-panel-help-180">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r338c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r338c1-clusterblast" data-name="r338c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r338c1-knownclusterblast" data-name="r338c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r338c1-subclusterblast" data-name="r338c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r338c1-MIBiG-cluster-compare" data-name="r338c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r338c1-pfam-details" data-name="r338c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r338c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-35"></div> <span class="help-tooltip" id="cb-general-help-35">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r338c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r338c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r338c1_1.svg">
        NZ_CP045031 (6863223-6874258): Streptomyces sp. C8S0 chromosome (42% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r338c1_2.svg">
        NZ_JAASAG010000002 (180775-191783): Streptomyces sp. HNM0575 scfid2, whole ge... (33% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r338c1_3.svg">
        NZ_CP018870 (3924991-3936017): Streptomyces sp. TN58 chromosome, complete genome (37% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r338c1_4.svg">
        NZ_CP043638 (3944933-3985968): Streptomyces sp. INR7 chromosome, complete genome (8% of genes show similarity), NRPS-like,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r338c1_5.svg">
        NZ_CP020039 (4223416-4265221): Streptomyces sp. 3211 isolate 3 chromosome, co... (8% of genes show similarity), NRPS-like,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r338c1_6.svg">
        NZ_CP020555 (4022866-4064646): Streptomyces sp. Sge12 chromosome, complete ge... (8% of genes show similarity), NRPS-like,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r338c1_7.svg">
        NZ_CP008947 (6700626-6711625): Rhodococcus opacus strain R7 chromosome (42% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r338c1_8.svg">
        NZ_CM001165 (2970616-2981621): Streptomyces sp. Tu6071 chromosome, whole geno... (37% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r338c1_9.svg">
        NZ_CP021118 (2830822-2841827): Streptomyces sp. CLI2509 strain CLI2905 chromo... (37% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r338c1_10.svg">
        NZ_GG657742 (4721606-4732611): Streptomyces sp. SPB78 supercont1.1, whole gen... (37% of genes show similarity), butyrolactone
      </option>
      
    </select>
    <button id= "clusterblast-r338c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r338c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r338c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-36"></div> <span class="help-tooltip" id="cb-known-help-36">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r338c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-37"></div> <span class="help-tooltip" id="cb-sub-help-37">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r338c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-38"></div> <span class="help-tooltip" id="MIBiG-help-38">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r338c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r338c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r338c1"></div>
    <div id="comparison-MIBiG-r338c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">butyrolactone</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514.1: 0-37469">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.60, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Other</td>
        
        <td>aristeromycin</td>
        
        
        <td>Streptomyces citricolor</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Other</td>
        
        <td>γ-butyrolactone</td>
        
        
        <td>Streptomyces acidiscabies</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001458.1: 4714477-4725572">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001458/index.html#r1c1" target="_new">BGC0001458.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.31 (id:0.44, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
         
        
        <td>0.08</td>
        <td>Other,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
        
        <td>salinipostin G</td>
        
        
        <td>Salinispora tropica CNB-440</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001083.1: 0-57523">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001083/index.html#r1c1" target="_new">BGC0001083.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.17 (id:0.44, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
         
        
        <td>0.04</td>
        <td>Polyketide, Terpene</td>
        
        <td>merochlorin A, merochlorin B, deschloro-merochlorin A, deschloro-merochlorin B, isochloro-merochlorin B, dichloro-merochlorin B, merochlorin D, merochlorin C</td>
        
        
        <td>Streptomyces sp. CNH189</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001771.1: 13254-31732">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001771/index.html#r1c1" target="_new">BGC0001771.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.17 (id:0.44, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
         
        
        <td>0.03</td>
        <td>NRP</td>
        
        <td>scleric acid</td>
        
        
        <td>Streptomyces sclerotialus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r338c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001514.1: 0-37469">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001514/index.html#r1c1" target="_new">BGC0001514.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.60, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other</td>
      <td>aristeromycin</td>
      <td>Streptomyces citricolor</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Other</td>
      <td>γ-butyrolactone</td>
      <td>Streptomyces acidiscabies</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001458.1: 4714477-4725572">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001458/index.html#r1c1" target="_new">BGC0001458.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.31 (id:0.44, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
      <td>0.31</td>
      <td>Other,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
      <td>salinipostin G</td>
      <td>Salinispora tropica CNB-440</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001083.1: 0-57523">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001083/index.html#r1c1" target="_new">BGC0001083.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.17 (id:0.44, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
      <td>0.17</td>
      <td>Polyketide, Terpene</td>
      <td>merochlorin A, merochlorin B, deschloro-merochlorin A, deschloro-merochlorin B, isochloro-merochlorin B, dichloro-merochlorin B, merochlorin D, merochlorin C</td>
      <td>Streptomyces sp. CNH189</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001771.1: 13254-31732">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001771/index.html#r1c1" target="_new">BGC0001771.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.17 (id:0.44, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
      <td>0.17</td>
      <td>NRP</td>
      <td>scleric acid</td>
      <td>Streptomyces sclerotialus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r338c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-39"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-39">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r338c1-pfam-details-svg" data-filename="c00338_CC228A_.._r338c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r338c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r543c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00543_CC228A_.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-181"></div> <span class="help-tooltip" id="region_svg-help-181">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00543_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r543c1-svg" data-filename="c00543_CC228A_.._r543c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 8,432 nt. (total: 8,432 nt)
        
        <a class="cluster-rules-header" id="r543c1-rules-header" href="#r543c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r543c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r543c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-182"></div> <span class="help-tooltip" id="focus-panel-help-182">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r543c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r543c1-clusterblast" data-name="r543c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r543c1-knownclusterblast" data-name="r543c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r543c1-subclusterblast" data-name="r543c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r543c1-MIBiG-cluster-compare" data-name="r543c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r543c1-pfam-details" data-name="r543c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r543c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-40"></div> <span class="help-tooltip" id="cb-general-help-40">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r543c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r543c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r543c1_1.svg">
        NZ_CP017316 (302531-321615): Streptomyces rubrolavendulae strain MJM4426 chro... (40% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r543c1_2.svg">
        NZ_CP023696 (310809-&gt;330174): Streptomyces fradiae ATCC 10745 = DSM 40063 str... (36% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r543c1_3.svg">
        NZ_CP045031 (6468854-6488812): Streptomyces sp. C8S0 chromosome (47% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r543c1_4.svg">
        NZ_CP022438 (2651229-2670168): Streptomyces peucetius subsp. caesius ATCC 279... (43% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r543c1_5.svg">
        NZ_WWGE01000028 (154549-175686): Streptomyces sp. SID8377 SID8377.c28, whole ... (38% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r543c1_6.svg">
        NZ_FZOF01000002 (461954-483091): Streptomyces glauciniger strain CGMCC 4.1858... (36% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r543c1_7.svg">
        NZ_LIPX01000014 (50918-72022): Streptomyces sp. WMMB 322 WMMB 322 14, whole g... (50% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r543c1_8.svg">
        NZ_AUKW01000003 (536336-557440): Streptomyces sp. TAA204 H343DRAFT scaffold00... (38% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r543c1_9.svg">
        NZ_KE384504 (129338-148507): Streptomyces sp. TAA486 I004DRAFT scaffold00009.... (43% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r543c1_10.svg">
        NZ_JAASAG010000010 (253729-274833): Streptomyces sp. HNM0575 scfid10, whole g... (43% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r543c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r543c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r543c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-41"></div> <span class="help-tooltip" id="cb-known-help-41">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r543c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-42"></div> <span class="help-tooltip" id="cb-sub-help-42">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r543c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-43"></div> <span class="help-tooltip" id="MIBiG-help-43">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r543c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r543c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r543c1"></div>
    <div id="comparison-MIBiG-r543c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.36</td>
        <td>Terpene</td>
        
        <td>albaflavenone</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000674.1: 74961-79952">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000674/index.html#r1c1" target="_new">BGC0000674.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Terpene</td>
        
        <td>(-)-δ-cadinene</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>Terpene</td>
        
        <td>(+)-T-muurolol</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Alkaloid</td>
        
        <td>kitasetaline</td>
        
        
        <td>Kitasatospora setae KM-6054</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001540.1: 633-38461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001540/index.html#r1c1" target="_new">BGC0001540.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.60, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>Other</td>
        
        <td>CC-1065</td>
        
        
        <td>Streptomyces zelensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.46, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>raimonol</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.47, order:0.40, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>Terpene</td>
        
        <td>cyslabdan</td>
        
        
        <td>Streptomyces cyslabdanicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.47, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>Terpene</td>
        
        <td>pentalenolactone</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001986.1: 0-3657">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001986/index.html#r1c1" target="_new">BGC0001986.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Other</td>
        
        <td>purincyclamide</td>
        
        
        <td>Streptomyces chrestomyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>Polyketide</td>
        
        <td>viguiepinol</td>
        
        
        <td>Streptomyces sp. KO-3988</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r543c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Terpene</td>
      <td>albaflavenone</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000674.1: 74961-79952">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000674/index.html#r1c1" target="_new">BGC0000674.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Terpene</td>
      <td>(-)-δ-cadinene</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene</td>
      <td>(+)-T-muurolol</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031.1: 8040819-8046734">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Alkaloid</td>
      <td>kitasetaline</td>
      <td>Kitasatospora setae KM-6054</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001540.1: 633-38461">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001540/index.html#r1c1" target="_new">BGC0001540.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.60, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Other</td>
      <td>CC-1065</td>
      <td>Streptomyces zelensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.46, order:0.40, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Terpene</td>
      <td>raimonol</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.47, order:0.40, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>Terpene</td>
      <td>cyslabdan</td>
      <td>Streptomyces cyslabdanicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.47, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001986.1: 0-3657">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001986/index.html#r1c1" target="_new">BGC0001986.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Other</td>
      <td>purincyclamide</td>
      <td>Streptomyces chrestomyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>Polyketide</td>
      <td>viguiepinol</td>
      <td>Streptomyces sp. KO-3988</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r543c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-44"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-44">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r543c1-pfam-details-svg" data-filename="c00543_CC228A_.._r543c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r543c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r573c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00573_CC228A_.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-183"></div> <span class="help-tooltip" id="region_svg-help-183">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00573_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r573c1-svg" data-filename="c00573_CC228A_.._r573c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 9,246 nt. (total: 9,246 nt)
        
        <a class="cluster-rules-header" id="r573c1-rules-header" href="#r573c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r573c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r573c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-184"></div> <span class="help-tooltip" id="focus-panel-help-184">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r573c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r573c1-clusterblast" data-name="r573c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r573c1-knownclusterblast" data-name="r573c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r573c1-subclusterblast" data-name="r573c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r573c1-MIBiG-cluster-compare" data-name="r573c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r573c1-pfam-details" data-name="r573c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r573c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-45"></div> <span class="help-tooltip" id="cb-general-help-45">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r573c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r573c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r573c1_1.svg">
        NZ_JACHEM010000007 (77239-103078): Streptomyces candidus strain DSM 40141 Ga0... (39% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r573c1_2.svg">
        NZ_CP018870 (6301530-6328436): Streptomyces sp. TN58 chromosome, complete genome (40% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r573c1_3.svg">
        NZ_CP045031 (462992-488709): Streptomyces sp. C8S0 chromosome (40% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r573c1_4.svg">
        NZ_CP034279 (980136-1006015): Streptomyces ficellus strain NRRL 8067 chromoso... (40% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r573c1_5.svg">
        NZ_CP023696 (6121726-6147439): Streptomyces fradiae ATCC 10745 = DSM 40063 st... (40% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r573c1_6.svg">
        NZ_CP017316 (5934949-5960710): Streptomyces rubrolavendulae strain MJM4426 ch... (40% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r573c1_7.svg">
        NZ_FNHI01000016 (13543-40168): Streptomyces wuyuanensis strain CGMCC 4.7042, ... (45% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r573c1_8.svg">
        NZ_KB913030 (6333967-6360551): Streptomyces purpureus KA281 StrpuDRAFT scaffo... (37% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r573c1_9.svg">
        NZ_KB897782 (207294-233883): Streptomyces sp. CNT372 C593DRAFT scaffold00007.... (40% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r573c1_10.svg">
        NZ_WWJR01000089 (193173-219782): Streptomyces sp. SID4913 SID4913.c89, whole ... (42% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r573c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r573c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r573c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-46"></div> <span class="help-tooltip" id="cb-known-help-46">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r573c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r573c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r573c1_1.svg">
        hopene
      </option>
      
      <option value = "svg/knownclusterblast_r573c1_2.svg">
        phosphonoglycans
      </option>
      
    </select>
    <button id="knownclusterblast-r573c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r573c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r573c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-47"></div> <span class="help-tooltip" id="cb-sub-help-47">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r573c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-48"></div> <span class="help-tooltip" id="MIBiG-help-48">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r573c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r573c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r573c1"></div>
    <div id="comparison-MIBiG-r573c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.33</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Myxococcus xanthus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Rhodobacter sphaeroides</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645.1: 0-7475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Halobacillus halophilus DSM 2266</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>β-carotein</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656.1: 169-3770">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Terpene</td>
        
        <td>zeaxanthin</td>
        
        
        <td>Xanthobacter autotrophicus Py2</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.19</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663.1: 7516016-7529773">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.60, order:0.41, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.19</td>
        <td>Terpene</td>
        
        <td>hopene</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000794.1: 492-19862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000794/index.html#r1c1" target="_new">BGC0000794.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.17</td>
        <td>Saccharide</td>
        
        <td>S-layer glycan</td>
        
        
        <td>Geobacillus stearothermophilus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000785.1: 151-19915">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000785/index.html#r1c1" target="_new">BGC0000785.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Saccharide</td>
        
        <td>O-antigen</td>
        
        
        <td>Burkholderia mallei</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r573c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648.1: 1057-11443">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Myxococcus xanthus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000647.1: 251-2872">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000647/index.html#r1c1" target="_new">BGC0000647.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Rhodobacter sphaeroides</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645.1: 0-7475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Halobacillus halophilus DSM 2266</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene</td>
      <td>β-carotein</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000656.1: 169-3770">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000656/index.html#r1c1" target="_new">BGC0000656.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Terpene</td>
      <td>zeaxanthin</td>
      <td>Xanthobacter autotrophicus Py2</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663.1: 7516016-7529773">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.60, order:0.41, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Terpene</td>
      <td>hopene</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000794.1: 492-19862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000794/index.html#r1c1" target="_new">BGC0000794.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Saccharide</td>
      <td>S-layer glycan</td>
      <td>Geobacillus stearothermophilus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000785.1: 151-19915">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000785/index.html#r1c1" target="_new">BGC0000785.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.47, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Saccharide</td>
      <td>O-antigen</td>
      <td>Burkholderia mallei</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r573c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-49"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-49">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r573c1-pfam-details-svg" data-filename="c00573_CC228A_.._r573c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r573c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r622c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00622_CC228A_.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-185"></div> <span class="help-tooltip" id="region_svg-help-185">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00622_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r622c1-svg" data-filename="c00622_CC228A_.._r622c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 8,276 nt. (total: 8,276 nt)
        
        <a class="cluster-rules-header" id="r622c1-rules-header" href="#r622c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r622c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r622c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-186"></div> <span class="help-tooltip" id="focus-panel-help-186">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r622c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r622c1-nrps_pks" data-name="r622c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r622c1-clusterblast" data-name="r622c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r622c1-knownclusterblast" data-name="r622c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r622c1-subclusterblast" data-name="r622c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r622c1-MIBiG-cluster-compare" data-name="r622c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r622c1-pfam-details" data-name="r622c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r622c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-50"></div> <span class="help-tooltip" id="nrps-domain-help-50">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r622c1-details-svg" data-filename="c00622_CC228A_.._r622c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r622c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r622c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-53"></div> <span class="help-tooltip" id="cb-general-help-53">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r622c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r622c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r622c1_1.svg">
        NZ_CP029241 (5325391-5471956): Streptomyces sp. NHF165 chromosome, complete g... (9% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r622c1_2.svg">
        NZ_CP031742 (1467354-1528501): Streptomyces koyangensis strain VK-A60T chromo... (11% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r622c1_3.svg">
        NZ_CP019798 (6068899-6206931): Streptomyces sp. fd1-xmd chromosome, complete ... (7% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r622c1_4.svg">
        NZ_CP045740 (2213053-2273631): Streptomyces sp. SUK 48 chromosome, complete g... (8% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r622c1_5.svg">
        NZ_WWGN01000023 (142878-200292): Streptomyces sp. SID8366 SID8366.c23, whole ... (8% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r622c1_6.svg">
        NZ_CP032229 (4811927-4921881): Streptomyces seoulensis strain KCTC 9819 chrom... (4% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r622c1_7.svg">
        NZ_CP016793 (7250155-7446409): Lentzea guizhouensis strain DHS C013 chromosom... (2% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r622c1_8.svg">
        NZ_JACHJN010000001 (707088-835164): Saccharothrix tamanrassetensis strain CEC... (5% of genes show similarity), NRPS,ladderane,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r622c1_9.svg">
        NC_019673 (3409970-3541086): Saccharothrix espanaensis DSM 44229, complete ge... (4% of genes show similarity), NRPS,ladderane,thioamide-NRP
      </option>
      
      <option value = "svg/clusterblast_r622c1_10.svg">
        NZ_CP034550 (7603482-7727935): Saccharothrix syringae strain NRRL B-16468 chr... (3% of genes show similarity), NRPS,T2PKS
      </option>
      
    </select>
    <button id= "clusterblast-r622c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r622c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r622c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-54"></div> <span class="help-tooltip" id="cb-known-help-54">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r622c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r622c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r622c1_1.svg">
        ulleungmycin
      </option>
      
      <option value = "svg/knownclusterblast_r622c1_2.svg">
        curacomycin
      </option>
      
      <option value = "svg/knownclusterblast_r622c1_3.svg">
        mannopeptimycin
      </option>
      
      <option value = "svg/knownclusterblast_r622c1_4.svg">
        dechlorocuracomycin
      </option>
      
      <option value = "svg/knownclusterblast_r622c1_5.svg">
        FR900359
      </option>
      
      <option value = "svg/knownclusterblast_r622c1_6.svg">
        bicornutin A1 / bicornutin A2
      </option>
      
    </select>
    <button id="knownclusterblast-r622c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r622c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r622c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-55"></div> <span class="help-tooltip" id="cb-sub-help-55">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r622c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-56"></div> <span class="help-tooltip" id="MIBiG-help-56">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r622c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r622c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r622c1"></div>
    <div id="comparison-MIBiG-r622c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.46, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
        <td>0.35</td>
        <td>NRP</td>
        
        <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.47, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.46, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>Other</td>
        
        <td>ferrichrome</td>
        
        
        <td>Aspergillus oryzae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>dimethylcoprogen</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050.1: 0-19730">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.64 (id:0.46, order:1.00, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
         
        
        <td>0.30</td>
        <td>NRP, Polyketide</td>
        
        <td>thalassospiramide A</td>
        
        
        <td>Tistrella bauzanensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000901.1: 1761-23020">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000901/index.html#r1c1" target="_new">BGC0000901.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Other</td>
        
        <td>ferrichrome</td>
        
        
        <td>Aspergillus niger</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:0.69, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>exochelin</td>
        
        
        <td>Mycolicibacterium smegmatis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r622c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.46, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.47, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>NRP</td>
      <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.47, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Other</td>
      <td>ferrichrome</td>
      <td>Aspergillus oryzae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>dimethylcoprogen</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050.1: 0-19730">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.46, order:1.00, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>NRP, Polyketide</td>
      <td>thalassospiramide A</td>
      <td>Tistrella bauzanensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.48, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000901.1: 1761-23020">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000901/index.html#r1c1" target="_new">BGC0000901.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Other</td>
      <td>ferrichrome</td>
      <td>Aspergillus niger</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.45, order:0.69, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>exochelin</td>
      <td>Mycolicibacterium smegmatis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r622c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-57"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-57">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r622c1-pfam-details-svg" data-filename="c00622_CC228A_.._r622c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r622c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r622c1-nrps_pks_products" data-name="r622c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r622c1-nrps_pks" data-name="r622c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r622c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-51"></div> <span class="help-tooltip" id="nrps-structures-help-51">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 8276:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r622c1-1" class="smiles-canvas" data-smiles="NC(Cc1ccccc1)C(=O)NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(phe) + (D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe&amp;nrps2=d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*&amp;nrps2=d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r622c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(Cc1ccccc1)C(=O)NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r622c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-52"></div> <span class="help-tooltip" id="nrps-monomers-help-52">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds2_3527</span></strong>: X
      <div class="collapser collapser-target-cds2_3527 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (1..335)</strong>: X
          <div class="collapser collapser-target-cds2_3527 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  phe, trp, phg, tyr, bht
              <div class="collapser collapser-target-cds2_3527-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DGmlvGaVVK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds3493_7606</span></strong>: phe
      <div class="collapser collapser-target-cds3493_7606 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (262..652)</strong>: phe
          <div class="collapser collapser-target-cds3493_7606 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  phe
              <div class="collapser collapser-target-cds3493_7606-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>phe, trp</dd>
   <dt>Single AA prediction:</dt>
   <dd>phe</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAWTVAAVCK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>phe</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r650c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00650_CC228A_.. - Region 1 - melanin <div class="help-container"> <div class="help-icon" data-id="region_svg-help-187"></div> <span class="help-tooltip" id="region_svg-help-187">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00650_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r650c1-svg" data-filename="c00650_CC228A_.._r650c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 2,360 nt. (total: 2,360 nt)
        
        <a class="cluster-rules-header" id="r650c1-rules-header" href="#r650c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r650c1-rules">melanin: melC</div>
      
      <div class="region-svg-container">
       <div id='r650c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-188"></div> <span class="help-tooltip" id="focus-panel-help-188">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r650c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r650c1-clusterblast" data-name="r650c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r650c1-knownclusterblast" data-name="r650c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r650c1-subclusterblast" data-name="r650c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r650c1-MIBiG-cluster-compare" data-name="r650c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r650c1-pfam-details" data-name="r650c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r650c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-58"></div> <span class="help-tooltip" id="cb-general-help-58">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r650c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r650c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r650c1_1.svg">
        NZ_CP045031 (301278-310967): Streptomyces sp. C8S0 chromosome (30% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r650c1_2.svg">
        NZ_SMBL01000015 (185088-195481): Streptomyces sp. BK215 Ga0307669 115, whole ... (22% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r650c1_3.svg">
        NZ_KZ819169 (1252304-1262694): Streptomyces sp. CS081A scaffold00001, whole g... (25% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r650c1_4.svg">
        NZ_CP034279 (761169-769891): Streptomyces ficellus strain NRRL 8067 chromosom... (28% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r650c1_5.svg">
        NZ_NBCN01000014 (153869-196431): Streptomyces sp. CB03238 scaffold14, whole g... (5% of genes show similarity), NRPS-like,melanin
      </option>
      
      <option value = "svg/clusterblast_r650c1_6.svg">
        NZ_CP023693 (6802712-6813102): Streptomyces cinereoruber strain ATCC 19740 ch... (25% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r650c1_7.svg">
        NZ_CP027022 (765941-775593): Streptomyces sp. WAC00288 chromosome, complete g... (25% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r650c1_8.svg">
        NZ_LIVV01000002 (467764-478157): Streptomyces sp. CB02009 scaffold2, whole ge... (25% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r650c1_9.svg">
        NZ_CP029195 (707969-717404): Streptomyces venezuelae strain ATCC 10595 chromo... (25% of genes show similarity), melanin
      </option>
      
      <option value = "svg/clusterblast_r650c1_10.svg">
        NZ_CP029196 (7135799-7146189): Streptomyces venezuelae strain ATCC 21113 chro... (25% of genes show similarity), melanin
      </option>
      
    </select>
    <button id= "clusterblast-r650c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r650c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r650c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-59"></div> <span class="help-tooltip" id="cb-known-help-59">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r650c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r650c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r650c1_1.svg">
        melanin
      </option>
      
      <option value = "svg/knownclusterblast_r650c1_2.svg">
        saframycin A / saframycin B
      </option>
      
      <option value = "svg/knownclusterblast_r650c1_3.svg">
        melanin
      </option>
      
    </select>
    <button id="knownclusterblast-r650c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r650c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r650c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-60"></div> <span class="help-tooltip" id="cb-sub-help-60">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r650c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-61"></div> <span class="help-tooltip" id="MIBiG-help-61">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r650c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r650c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r650c1"></div>
    <div id="comparison-MIBiG-r650c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">melanin</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000912.1: 2884831-2886096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000912/index.html#r1c1" target="_new">BGC0000912.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
         
        
        <td>0.55</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000911.1: 607505-608793">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000911/index.html#r1c1" target="_new">BGC0000911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.79 (id:0.49, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
         
        
        <td>0.50</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000910.1: 2943455-2944875">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000910/index.html#r1c1" target="_new">BGC0000910.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.51, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000909.1: 0-6625">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000909/index.html#r1c1" target="_new">BGC0000909.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.69 (id:0.49, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
         
        
        <td>0.37</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000908.1: 0-6497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000908/index.html#r1c1" target="_new">BGC0000908.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.58, order:0.69, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>Other</td>
        
        <td>melanin</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885.1: 0-16706">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.47, order:0.69, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>4-hydroxy-3-nitrosobenzamide</td>
        
        
        <td>Streptomyces murayamaensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.40 (id:0.50, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
         
        
        <td>0.12</td>
        <td>Terpene</td>
        
        <td>grixazone</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.39 (id:0.48, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
         
        
        <td>0.12</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B</td>
        
        
        <td>Streptomyces sp. WK-5344</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001592.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001592/index.html#r1c1" target="_new">BGC0001592.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.39 (id:0.48, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
         
        
        <td>0.11</td>
        <td>Other</td>
        
        <td>ferroverdin</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002029.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002029/index.html#r1c1" target="_new">BGC0002029.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.39 (id:0.48, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
         
        
        <td>0.11</td>
        <td>Other</td>
        
        <td>bagremycin A, bagremycin B, bagremycin C, bagremycin E, bagremycin F, bagremycin G, ferroverdin A, ferroverdin B, ferroverdin C</td>
        
        
        <td>Streptomyces lunaelactis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r650c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000912.1: 2884831-2886096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000912/index.html#r1c1" target="_new">BGC0000912.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.81 (id:0.53, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.81), rgb(0, 0, 0, 0.81) 81%, white 81%)"></td>
      <td>0.81</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000911.1: 607505-608793">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000911/index.html#r1c1" target="_new">BGC0000911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.79 (id:0.49, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
      <td>0.79</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000910.1: 2943455-2944875">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000910/index.html#r1c1" target="_new">BGC0000910.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.51, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000909.1: 0-6625">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000909/index.html#r1c1" target="_new">BGC0000909.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.69 (id:0.49, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
      <td>0.69</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000908.1: 0-6497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000908/index.html#r1c1" target="_new">BGC0000908.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.58, order:0.69, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Other</td>
      <td>melanin</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000885.1: 0-16706">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000885/index.html#r1c1" target="_new">BGC0000885.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.47, order:0.69, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other</td>
      <td>4-hydroxy-3-nitrosobenzamide</td>
      <td>Streptomyces murayamaensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662.1: 4957168-4972074">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.40 (id:0.50, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
      <td>0.40</td>
      <td>Terpene</td>
      <td>grixazone</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000889.1: 0-19769">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000889/index.html#r1c1" target="_new">BGC0000889.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.39 (id:0.48, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
      <td>0.39</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B</td>
      <td>Streptomyces sp. WK-5344</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001592.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001592/index.html#r1c1" target="_new">BGC0001592.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.39 (id:0.48, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
      <td>0.39</td>
      <td>Other</td>
      <td>ferroverdin</td>
      <td>Streptomyces lunaelactis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002029.1: 0-19687">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002029/index.html#r1c1" target="_new">BGC0002029.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.39 (id:0.48, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
      <td>0.39</td>
      <td>Other</td>
      <td>bagremycin A, bagremycin B, bagremycin C, bagremycin E, bagremycin F, bagremycin G, ferroverdin A, ferroverdin B, ferroverdin C</td>
      <td>Streptomyces lunaelactis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r650c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-62"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-62">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r650c1-pfam-details-svg" data-filename="c00650_CC228A_.._r650c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r650c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r798c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00798_CC228A_.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-189"></div> <span class="help-tooltip" id="region_svg-help-189">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00798_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r798c1-svg" data-filename="c00798_CC228A_.._r798c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 3,588 nt. (total: 3,588 nt)
        
        <a class="cluster-rules-header" id="r798c1-rules-header" href="#r798c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r798c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r798c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-190"></div> <span class="help-tooltip" id="focus-panel-help-190">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r798c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r798c1-clusterblast" data-name="r798c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r798c1-knownclusterblast" data-name="r798c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r798c1-subclusterblast" data-name="r798c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r798c1-MIBiG-cluster-compare" data-name="r798c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r798c1-pfam-details" data-name="r798c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r798c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-63"></div> <span class="help-tooltip" id="cb-general-help-63">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r798c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r798c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r798c1_1.svg">
        NZ_CP045031 (6806855-6862000): Streptomyces sp. C8S0 chromosome (7% of genes show similarity), T3PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r798c1_2.svg">
        NZ_CP027022 (1009022-1028832): Streptomyces sp. WAC00288 chromosome, complete... (13% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r798c1_3.svg">
        NZ_KZ819169 (967172-987511): Streptomyces sp. CS081A scaffold00001, whole gen... (14% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r798c1_4.svg">
        NZ_CP023693 (6533885-6553738): Streptomyces cinereoruber strain ATCC 19740 ch... (13% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r798c1_5.svg">
        NZ_JABBGE010000006 (376692-397955): Streptomyces sp. R302 contig6, whole geno... (12% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r798c1_6.svg">
        NZ_CP012382 (352106-411657): Streptomyces ambofaciens ATCC 23877 chromosome, ... (4% of genes show similarity), NRPS,T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r798c1_7.svg">
        NZ_CP021978 (8595929-8617258): Streptomyces hawaiiensis strain ATCC 12236 chr... (10% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r798c1_8.svg">
        NZ_CP012949 (362398-421882): Streptomyces ambofaciens strain DSM 40697 chromo... (4% of genes show similarity), NRPS,T1PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r798c1_9.svg">
        NZ_CP029196 (7443958-7464115): Streptomyces venezuelae strain ATCC 21113 chro... (11% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r798c1_10.svg">
        NZ_JACHGV010000008 (289803-311129): Streptomyces paradoxus strain DSM 43350 G... (11% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r798c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r798c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r798c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-64"></div> <span class="help-tooltip" id="cb-known-help-64">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r798c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r798c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r798c1_1.svg">
        2-methylisoborneol
      </option>
      
      <option value = "svg/knownclusterblast_r798c1_2.svg">
        ebelactone
      </option>
      
    </select>
    <button id="knownclusterblast-r798c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r798c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r798c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-65"></div> <span class="help-tooltip" id="cb-sub-help-65">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r798c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-66"></div> <span class="help-tooltip" id="MIBiG-help-66">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r798c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r798c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r798c1"></div>
    <div id="comparison-MIBiG-r798c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000658.1: 1490670-1492896">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000658/index.html#r1c1" target="_new">BGC0000658.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.86 (id:0.63, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
         
        
        <td>0.63</td>
        <td>Terpene</td>
        
        <td>2-methylisoborneol</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000659.1: 4075162-4077099">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000659/index.html#r1c1" target="_new">BGC0000659.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
         
        
        <td>0.51</td>
        <td>Terpene</td>
        
        <td>2-methylisoborneol</td>
        
        
        <td>Saccharopolyspora erythraea NRRL 2338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000657.1: 643-5774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000657/index.html#r1c1" target="_new">BGC0000657.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.52, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Terpene</td>
        
        <td>2-methylisoborneol</td>
        
        
        <td>Pseudanabaena sp. dqh15</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001859.1: 6609-19144">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001859/index.html#r1c1" target="_new">BGC0001859.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.35 (id:0.48, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
         
        
        <td>0.10</td>
        <td>Other,  , (, P, h, o, s, p, h, o, n, a, t, e, )</td>
        
        <td>fosfomycin</td>
        
        
        <td>Pseudomonas syringae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000483.1: 1302-13724">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000483/index.html#r1c1" target="_new">BGC0000483.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
         
        
        <td>0.09</td>
        <td>RiPP</td>
        
        <td>aeruginosamide B, aeruginosamide C</td>
        
        
        <td>Microcystis aeruginosa PCC 9432</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001580.1: 126-73935">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001580/index.html#r1c1" target="_new">BGC0001580.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.27 (id:0.62, order:0.57, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
         
        
        <td>0.06</td>
        <td>Polyketide</td>
        
        <td>ebelactone</td>
        
        
        <td>Kitasatospora aburaviensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890.1: 1392460-1420815">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.20 (id:0.49, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
         
        
        <td>0.04</td>
        <td>NRP</td>
        
        <td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
        
        
        <td>Vibrio campbellii ATCC BAA-1116</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001941.1: 17-54952">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001941/index.html#r1c1" target="_new">BGC0001941.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.19 (id:0.48, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
         
        
        <td>0.04</td>
        <td>NRP, Polyketide</td>
        
        <td>fulvuthiacene A, fulvuthiacene B</td>
        
        
        <td>Myxococcus fulvus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000651.1: 740-35007">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000651/index.html#r1c1" target="_new">BGC0000651.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.18 (id:0.47, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
         
        
        <td>0.04</td>
        <td>Terpene</td>
        
        <td>KS-505a</td>
        
        
        <td>Streptomyces argenteolus</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r798c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000658.1: 1490670-1492896">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000658/index.html#r1c1" target="_new">BGC0000658.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.86 (id:0.63, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
      <td>0.86</td>
      <td>Terpene</td>
      <td>2-methylisoborneol</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000659.1: 4075162-4077099">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000659/index.html#r1c1" target="_new">BGC0000659.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.79 (id:0.50, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
      <td>0.79</td>
      <td>Terpene</td>
      <td>2-methylisoborneol</td>
      <td>Saccharopolyspora erythraea NRRL 2338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000657.1: 643-5774">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000657/index.html#r1c1" target="_new">BGC0000657.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.52, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Terpene</td>
      <td>2-methylisoborneol</td>
      <td>Pseudanabaena sp. dqh15</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001859.1: 6609-19144">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001859/index.html#r1c1" target="_new">BGC0001859.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.35 (id:0.48, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
      <td>0.35</td>
      <td>Other,  , (, P, h, o, s, p, h, o, n, a, t, e, )</td>
      <td>fosfomycin</td>
      <td>Pseudomonas syringae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000483.1: 1302-13724">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000483/index.html#r1c1" target="_new">BGC0000483.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
      <td>0.33</td>
      <td>RiPP</td>
      <td>aeruginosamide B, aeruginosamide C</td>
      <td>Microcystis aeruginosa PCC 9432</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001580.1: 126-73935">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001580/index.html#r1c1" target="_new">BGC0001580.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.27 (id:0.62, order:0.57, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
      <td>0.27</td>
      <td>Polyketide</td>
      <td>ebelactone</td>
      <td>Kitasatospora aburaviensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890.1: 1392460-1420815">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.20 (id:0.49, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
      <td>0.20</td>
      <td>NRP</td>
      <td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
      <td>Vibrio campbellii ATCC BAA-1116</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001941.1: 17-54952">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001941/index.html#r1c1" target="_new">BGC0001941.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.19 (id:0.48, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
      <td>0.19</td>
      <td>NRP, Polyketide</td>
      <td>fulvuthiacene A, fulvuthiacene B</td>
      <td>Myxococcus fulvus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000651.1: 740-35007">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000651/index.html#r1c1" target="_new">BGC0000651.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.18 (id:0.47, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
      <td>0.18</td>
      <td>Terpene</td>
      <td>KS-505a</td>
      <td>Streptomyces argenteolus</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r798c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-67"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-67">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r798c1-pfam-details-svg" data-filename="c00798_CC228A_.._r798c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r798c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r816c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00816_CC228A_.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-191"></div> <span class="help-tooltip" id="region_svg-help-191">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00816_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r816c1-svg" data-filename="c00816_CC228A_.._r816c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 7,413 nt. (total: 7,413 nt)
        
        <a class="cluster-rules-header" id="r816c1-rules-header" href="#r816c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r816c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r816c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-192"></div> <span class="help-tooltip" id="focus-panel-help-192">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r816c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r816c1-nrps_pks" data-name="r816c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r816c1-clusterblast" data-name="r816c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r816c1-knownclusterblast" data-name="r816c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r816c1-subclusterblast" data-name="r816c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r816c1-MIBiG-cluster-compare" data-name="r816c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r816c1-pfam-details" data-name="r816c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r816c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-68"></div> <span class="help-tooltip" id="nrps-domain-help-68">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r816c1-details-svg" data-filename="c00816_CC228A_.._r816c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r816c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r816c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-71"></div> <span class="help-tooltip" id="cb-general-help-71">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r816c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r816c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r816c1_1.svg">
        NZ_CP045031 (2837725-2881840): Streptomyces sp. C8S0 chromosome (14% of genes show similarity), NRPS,melanin
      </option>
      
      <option value = "svg/clusterblast_r816c1_2.svg">
        NZ_LWLC01000024 (107798-150969): Saccharothrix sp. CB00851 scaffold30, whole ... (17% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r816c1_3.svg">
        NZ_CP016076 (3892170-4064527): Actinoalloteichus fjordicus strain ADI127-7 ch... (4% of genes show similarity), CDPS,NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r816c1_4.svg">
        NZ_CP029189 (6190834-6234005): Streptomyces venezuelae strain ATCC 21018 chro... (11% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r816c1_5.svg">
        NZ_KB903819 (23295-66445): Longispora albida DSM 44784 A3G1DRAFT scaffold 14.... (7% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r816c1_6.svg">
        NC_003155 (3910606-3955803): Streptomyces avermitilis MA-4680 = NBRC 14893, c... (7% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r816c1_7.svg">
        NZ_GG657738 (5950799-5989009): Micromonospora sp. ATCC 39149 supercont1.1, wh... (9% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r816c1_8.svg">
        NZ_CP040244 (5413545-5480499): Streptomyces exfoliatus strain A1013Y chromoso... (5% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r816c1_9.svg">
        NZ_CP031264 (935042-999592): Streptacidiphilus bronchialis strain DSM 106435 ... (5% of genes show similarity), NRPS,betalactone,terpene
      </option>
      
      <option value = "svg/clusterblast_r816c1_10.svg">
        NZ_WBMT01000018 (117862-188335): Actinomadura rudentiformis strain HMC1 Scaff... (5% of genes show similarity), NRPS,T1PKS,other
      </option>
      
    </select>
    <button id= "clusterblast-r816c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r816c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r816c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-72"></div> <span class="help-tooltip" id="cb-known-help-72">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r816c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-73"></div> <span class="help-tooltip" id="cb-sub-help-73">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r816c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-74"></div> <span class="help-tooltip" id="MIBiG-help-74">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r816c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r816c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r816c1"></div>
    <div id="comparison-MIBiG-r816c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342.1: 1020-10416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>enniatin</td>
        
        
        <td>Fusarium equiseti</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313.1: 0-33475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>beauvericin</td>
        
        
        <td>Beauveria bassiana</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873.1: 1647607-1656213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>pyrrolizixenamide A</td>
        
        
        <td>Xenorhabdus szentirmaii DSM 16338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>vicibactin</td>
        
        
        <td>Rhizobium etli CFN 42</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>fellutamide B</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001149.1: 196767-209553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001149/index.html#r1c1" target="_new">BGC0001149.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP, Saccharide</td>
        
        <td>brabantamide A</td>
        
        
        <td>Pseudomonas sp. SHC52</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r816c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342.1: 1020-10416">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>enniatin</td>
      <td>Fusarium equiseti</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313.1: 0-33475">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>beauvericin</td>
      <td>Beauveria bassiana</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873.1: 1647607-1656213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>pyrrolizixenamide A</td>
      <td>Xenorhabdus szentirmaii DSM 16338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457.1: 511805-523079">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>vicibactin</td>
      <td>Rhizobium etli CFN 42</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>fellutamide B</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001149.1: 196767-209553">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001149/index.html#r1c1" target="_new">BGC0001149.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP, Saccharide</td>
      <td>brabantamide A</td>
      <td>Pseudomonas sp. SHC52</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r816c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-75"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-75">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r816c1-pfam-details-svg" data-filename="c00816_CC228A_.._r816c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r816c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r816c1-nrps_pks_products" data-name="r816c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r816c1-nrps_pks" data-name="r816c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r816c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-69"></div> <span class="help-tooltip" id="nrps-structures-help-69">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 7413:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r816c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r816c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r816c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-70"></div> <span class="help-tooltip" id="nrps-monomers-help-70">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds2070_5235</span></strong>: X
      <div class="collapser collapser-target-cds2070_5235 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (474..871)</strong>: X
          <div class="collapser collapser-target-cds2070_5235 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  (unknown)
              <div class="collapser collapser-target-cds2070_5235-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>  <strong>NOTE: outside applicability domain</strong><br>
  <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>phe, trp, phg, tyr, bht</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DMggmGLIgK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r819c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC228A_contig_35 - Region 1 - butyrolactone,ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-193"></div> <span class="help-tooltip" id="region_svg-help-193">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC228A_contig_35.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r819c1-svg" data-filename="CC228A_contig_35_r819c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 946 - 15,428 nt. (total: 14,483 nt)
        
        <a class="cluster-rules-header" id="r819c1-rules-header" href="#r819c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r819c1-rules">butyrolactone: AfsA<br>ectoine: ectoine_synt</div>
      
      <div class="region-svg-container">
       <div id='r819c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-194"></div> <span class="help-tooltip" id="focus-panel-help-194">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r819c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r819c1-clusterblast" data-name="r819c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r819c1-knownclusterblast" data-name="r819c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r819c1-subclusterblast" data-name="r819c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r819c1-MIBiG-cluster-compare" data-name="r819c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r819c1-pfam-details" data-name="r819c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r819c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-76"></div> <span class="help-tooltip" id="cb-general-help-76">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r819c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r819c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r819c1_1.svg">
        NZ_JACHEM010000012 (64880-80314): Streptomyces candidus strain DSM 40141 Ga04... (66% of genes show similarity), butyrolactone,ectoine
      </option>
      
      <option value = "svg/clusterblast_r819c1_2.svg">
        NZ_CP030864 (11545-320938): Streptomyces globosus strain LZH-48 plasmid unnam... (9% of genes show similarity), NRPS,NRPS-like,T2PKS,butyrolactone,ectoine,hglE-KS,linaridin,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r819c1_3.svg">
        NZ_CP058556 (3634228-3649312): Streptomyces sp. CB04723 chromosome, complete ... (61% of genes show similarity), butyrolactone,ectoine
      </option>
      
      <option value = "svg/clusterblast_r819c1_4.svg">
        NZ_WWGL01000031 (291925-307338): Streptomyces sp. SID8369 SID8369.c31, whole ... (57% of genes show similarity), butyrolactone,ectoine
      </option>
      
      <option value = "svg/clusterblast_r819c1_5.svg">
        NZ_CP020570 (3583867-3599337): Streptomyces violaceoruber strain S21 chromoso... (61% of genes show similarity), butyrolactone,ectoine
      </option>
      
      <option value = "svg/clusterblast_r819c1_6.svg">
        NZ_LIVT01000003 (1182961-1198363): Streptomyces sp. CB02366 scaffold3, whole ... (61% of genes show similarity), butyrolactone,ectoine
      </option>
      
      <option value = "svg/clusterblast_r819c1_7.svg">
        NZ_ML123117 (880092-895713): Streptomyces sp. ADI98-10 scaffold001, whole gen... (55% of genes show similarity), butyrolactone,ectoine
      </option>
      
      <option value = "svg/clusterblast_r819c1_8.svg">
        NZ_LIPE01000001 (1873365-2015426): Streptomyces sp. TSRI0395 scaffold1, whole... (14% of genes show similarity), NRPS,T1PKS,arylpolyene,butyrolactone,ectoine,ladderane
      </option>
      
      <option value = "svg/clusterblast_r819c1_9.svg">
        NZ_LIVW01000001 (1805343-1947591): Streptomyces sp. CB02115 scaffold1, whole ... (14% of genes show similarity), NRPS,T1PKS,arylpolyene,butyrolactone,ectoine,ladderane
      </option>
      
      <option value = "svg/clusterblast_r819c1_10.svg">
        NZ_LMEL01000012 (1504354-1646110): Streptomyces sp. Root1295 contig 2, whole ... (14% of genes show similarity), NRPS,T1PKS,arylpolyene,butyrolactone,ectoine,ladderane
      </option>
      
    </select>
    <button id= "clusterblast-r819c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r819c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r819c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-77"></div> <span class="help-tooltip" id="cb-known-help-77">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r819c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r819c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r819c1_1.svg">
        showdomycin
      </option>
      
      <option value = "svg/knownclusterblast_r819c1_2.svg">
        kosinostatin
      </option>
      
      <option value = "svg/knownclusterblast_r819c1_3.svg">
        neocarzinostatin
      </option>
      
    </select>
    <button id="knownclusterblast-r819c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r819c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r819c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-78"></div> <span class="help-tooltip" id="cb-sub-help-78">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r819c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-79"></div> <span class="help-tooltip" id="MIBiG-help-79">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r819c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r819c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r819c1"></div>
    <div id="comparison-MIBiG-r819c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">butyrolactone</span></th>
         
         <th><span class="cc-heat-product">ectoine</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.35 (id:0.53, order:0.39, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.55, order:0.87, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.51</td>
        <td>Other</td>
        
        <td>showdomycin</td>
        
        
        <td>Streptomyces showdoensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000849.1: 6890527-6892237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000849/index.html#r1c1" target="_new">BGC0000849.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.47, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.38</td>
        <td>Other,  , (, B, u, t, y, r, o, l, a, c, t, o, n, e, )</td>
        
        <td>SCB1, SCB2, SCB3</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.36</td>
        <td>Other</td>
        
        <td>γ-butyrolactone</td>
        
        
        <td>Streptomyces acidiscabies</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000848.1: 8273443-8274388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000848/index.html#r1c1" target="_new">BGC0000848.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
         
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
        <td>0.35</td>
        <td>Other</td>
        
        <td>A-factor</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001783.1: 55-63934">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001783/index.html#r1c1" target="_new">BGC0001783.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.29</td>
        <td>Other</td>
        
        <td>streptonigrin</td>
        
        
        <td>Streptomyces flocculus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001850.1: 258926-281351">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001850/index.html#r1c1" target="_new">BGC0001850.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.29</td>
        <td>Other,  , (, S, h, i, k, i, m, a, t, e, -, d, e, r, i, v, e, d, )</td>
        
        <td>diazaquinomycin A, diazaquinomycin E, diazaquinomycin F, diazaquinomycin G</td>
        
        
        <td>Streptomyces sp. F001</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000019.1: 339-17245">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000019/index.html#r1c1" target="_new">BGC0000019.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.28</td>
        <td>Polyketide</td>
        
        <td>angolamycin</td>
        
        
        <td>Streptomyces eurythermus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168.1: 891946-896528">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168.1</a></td>
        
         
        <td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>livipeptin</td>
        
        
        <td>Streptomyces lividans 1326</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000444.1: 12535-33158">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000444/index.html#r1c1" target="_new">BGC0000444.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.28 (id:0.46, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>thaxtomin A</td>
        
        
        <td>Streptomyces turgidiscabies Car8</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000257.1: 2579-43239">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000257/index.html#r1c1" target="_new">BGC0000257.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.29 (id:0.45, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.37 (id:0.46, order:0.32, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
         
        
        <td>0.18</td>
        <td>Polyketide</td>
        
        <td>prodigiosin</td>
        
        
        <td>Streptomyces griseoviridis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r819c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000849.1: 6890527-6892237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000849/index.html#r1c1" target="_new">BGC0000849.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.47, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>Other,  , (, B, u, t, y, r, o, l, a, c, t, o, n, e, )</td>
      <td>SCB1, SCB2, SCB3</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.47, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Other</td>
      <td>γ-butyrolactone</td>
      <td>Streptomyces acidiscabies</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000848.1: 8273443-8274388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000848/index.html#r1c1" target="_new">BGC0000848.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
      <td>0.68</td>
      <td>Other</td>
      <td>A-factor</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001778.1: 12965-29440">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001778/index.html#r1c1" target="_new">BGC0001778.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.64 (id:0.55, order:0.52, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
      <td>0.64</td>
      <td>Other</td>
      <td>showdomycin</td>
      <td>Streptomyces showdoensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001458.1: 4714477-4725572">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001458/index.html#r1c1" target="_new">BGC0001458.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
      <td>salinipostin G</td>
      <td>Salinispora tropica CNB-440</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000890.1: 0-11090">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000890/index.html#r1c1" target="_new">BGC0000890.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Other,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
      <td>pentabromopseudilin</td>
      <td>Pseudoalteromonas phenolica O-BC30</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000891.1: 32482-43745">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000891/index.html#r1c1" target="_new">BGC0000891.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Other,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
      <td>pentabromopseudilin</td>
      <td>Pseudoalteromonas luteoviolacea 2ta16</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001465.1: 4521274-4533494">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001465/index.html#r1c1" target="_new">BGC0001465.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Other</td>
      <td>bromopyrroles/bromophenols</td>
      <td>Marinomonas mediterranea MMB-1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168.1: 891946-896528">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>livipeptin</td>
      <td>Streptomyces lividans 1326</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444.1: 5309240-5323285">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>Other</td>
      <td>caboxamycin</td>
      <td>Streptomyces sp. NTK 937</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r819c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-80"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-80">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r819c1-pfam-details-svg" data-filename="CC228A_contig_35_r819c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r819c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r855c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00855_CC228A_.. - Region 1 - butyrolactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-195"></div> <span class="help-tooltip" id="region_svg-help-195">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00855_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r855c1-svg" data-filename="c00855_CC228A_.._r855c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 4,673 nt. (total: 4,673 nt)
        
        <a class="cluster-rules-header" id="r855c1-rules-header" href="#r855c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r855c1-rules">butyrolactone: AfsA</div>
      
      <div class="region-svg-container">
       <div id='r855c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-196"></div> <span class="help-tooltip" id="focus-panel-help-196">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r855c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r855c1-clusterblast" data-name="r855c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r855c1-knownclusterblast" data-name="r855c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r855c1-subclusterblast" data-name="r855c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r855c1-MIBiG-cluster-compare" data-name="r855c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r855c1-pfam-details" data-name="r855c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r855c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-81"></div> <span class="help-tooltip" id="cb-general-help-81">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r855c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r855c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r855c1_1.svg">
        NZ_CP045031 (3707782-3716851): Streptomyces sp. C8S0 chromosome (44% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r855c1_2.svg">
        NZ_CP021744 (6836784-7017986): Streptomyces albireticuli strain MDJK11 chromo... (3% of genes show similarity), NRPS,T1PKS,T2PKS,butyrolactone,transAT-PKS
      </option>
      
      <option value = "svg/clusterblast_r855c1_3.svg">
        NZ_GG657752 (28-90080): Streptomyces sp. C supercont1.3, whole genome shotgun... (6% of genes show similarity), NRPS,T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r855c1_4.svg">
        NZ_CP020569 (8457433-8478674): Streptomyces gilvosporeus strain F607 chromoso... (22% of genes show similarity), CDPS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r855c1_5.svg">
        NZ_CP054936 (466018-550266): Streptomyces seoulensis strain A01 chromosome (5% of genes show similarity), NRPS,T2PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r855c1_6.svg">
        NZ_CP034539 (7673954-7747133): Streptomyces cyaneochromogenes strain MK-45 ch... (10% of genes show similarity), T1PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r855c1_7.svg">
        NZ_JACHJG010000007 (150443-161508): Streptomyces netropsis strain CECT 3265 G... (50% of genes show similarity), butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r855c1_8.svg">
        NZ_SLZE01000022 (53849-95034): Streptomyces sp. BK674 Ga0307689 122, whole ge... (11% of genes show similarity), T3PKS,butyrolactone
      </option>
      
      <option value = "svg/clusterblast_r855c1_9.svg">
        NZ_CP034539 (3653535-3667408): Streptomyces cyaneochromogenes strain MK-45 ch... (21% of genes show similarity), butyrolactone,melanin
      </option>
      
      <option value = "svg/clusterblast_r855c1_10.svg">
        NZ_JACHJH010000004 (153087-162653): Streptomyces olivoverticillatus strain CE... (37% of genes show similarity), butyrolactone
      </option>
      
    </select>
    <button id= "clusterblast-r855c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r855c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r855c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-82"></div> <span class="help-tooltip" id="cb-known-help-82">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r855c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r855c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r855c1_1.svg">
        coelimycin P1
      </option>
      
      <option value = "svg/knownclusterblast_r855c1_2.svg">
        SCB1 / SCB2 / SCB3
      </option>
      
      <option value = "svg/knownclusterblast_r855c1_3.svg">
        A-factor
      </option>
      
      <option value = "svg/knownclusterblast_r855c1_4.svg">
        lactonamycin
      </option>
      
    </select>
    <button id="knownclusterblast-r855c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r855c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r855c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-83"></div> <span class="help-tooltip" id="cb-sub-help-83">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r855c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-84"></div> <span class="help-tooltip" id="MIBiG-help-84">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r855c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r855c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r855c1"></div>
    <div id="comparison-MIBiG-r855c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">butyrolactone</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000848.1: 8273443-8274388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000848/index.html#r1c1" target="_new">BGC0000848.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>Other</td>
        
        <td>A-factor</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000849.1: 6890527-6892237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000849/index.html#r1c1" target="_new">BGC0000849.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.50, order:0.68, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.37</td>
        <td>Other,  , (, B, u, t, y, r, o, l, a, c, t, o, n, e, )</td>
        
        <td>SCB1, SCB2, SCB3</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076.1: 161645-164819">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>4,5-Dehydroacaterin, Acaterin</td>
        
        
        <td>Pseudomonas jessenii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.47, order:0.47, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>Other</td>
        
        <td>γ-butyrolactone</td>
        
        
        <td>Streptomyces acidiscabies</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928.1: 108-8402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Other</td>
        
        <td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>
        
        
        <td>Streptomyces sp. MA37</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Terpene</td>
        
        <td>platencin</td>
        
        
        <td>Streptomyces platensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001917.1: 148-16415">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001917/index.html#r1c1" target="_new">BGC0001917.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>5-alkyl-1,2,3,4-tetrahydroquinolines, streptoaminals</td>
        
        
        <td>Streptomyces nigrescens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000698.1: 2269-33779">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000698/index.html#r1c1" target="_new">BGC0000698.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Saccharide</td>
        
        <td>hygromycin A</td>
        
        
        <td>Streptomyces hygroscopicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893.1: 1474769-1494048">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Other</td>
        
        <td>abscisic acid</td>
        
        
        <td>Botrytis cinerea B05.10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001987.1: 701808-716497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001987/index.html#r1c1" target="_new">BGC0001987.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>koreenceine A, koreenceine B, koreenceine C, koreenceine D</td>
        
        
        <td>Pseudomonas koreensis</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r855c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000848.1: 8273443-8274388">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000848/index.html#r1c1" target="_new">BGC0000848.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>Other</td>
      <td>A-factor</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000849.1: 6890527-6892237">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000849/index.html#r1c1" target="_new">BGC0000849.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.50, order:0.68, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other,  , (, B, u, t, y, r, o, l, a, c, t, o, n, e, )</td>
      <td>SCB1, SCB2, SCB3</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002076.1: 161645-164819">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002076/index.html#r1c1" target="_new">BGC0002076.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Other</td>
      <td>4,5-Dehydroacaterin, Acaterin</td>
      <td>Pseudomonas jessenii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000850.1: 0-2474">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000850/index.html#r1c1" target="_new">BGC0000850.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.47, order:0.47, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>Other</td>
      <td>γ-butyrolactone</td>
      <td>Streptomyces acidiscabies</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928.1: 108-8402">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Other</td>
      <td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>
      <td>Streptomyces sp. MA37</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Terpene</td>
      <td>platencin</td>
      <td>Streptomyces platensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001917.1: 148-16415">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001917/index.html#r1c1" target="_new">BGC0001917.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Polyketide</td>
      <td>5-alkyl-1,2,3,4-tetrahydroquinolines, streptoaminals</td>
      <td>Streptomyces nigrescens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000698.1: 2269-33779">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000698/index.html#r1c1" target="_new">BGC0000698.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Saccharide</td>
      <td>hygromycin A</td>
      <td>Streptomyces hygroscopicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893.1: 1474769-1494048">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Other</td>
      <td>abscisic acid</td>
      <td>Botrytis cinerea B05.10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001987.1: 701808-716497">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001987/index.html#r1c1" target="_new">BGC0001987.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Polyketide</td>
      <td>koreenceine A, koreenceine B, koreenceine C, koreenceine D</td>
      <td>Pseudomonas koreensis</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r855c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-85"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-85">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r855c1-pfam-details-svg" data-filename="c00855_CC228A_.._r855c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r855c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r865c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00865_CC228A_.. - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-197"></div> <span class="help-tooltip" id="region_svg-help-197">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00865_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r865c1-svg" data-filename="c00865_CC228A_.._r865c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 5,462 nt. (total: 5,462 nt)
        
        <a class="cluster-rules-header" id="r865c1-rules-header" href="#r865c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r865c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r865c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-198"></div> <span class="help-tooltip" id="focus-panel-help-198">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r865c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r865c1-nrps_pks" data-name="r865c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r865c1-clusterblast" data-name="r865c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r865c1-knownclusterblast" data-name="r865c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r865c1-subclusterblast" data-name="r865c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r865c1-MIBiG-cluster-compare" data-name="r865c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r865c1-pfam-details" data-name="r865c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r865c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-86"></div> <span class="help-tooltip" id="nrps-domain-help-86">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r865c1-details-svg" data-filename="c00865_CC228A_.._r865c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r865c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r865c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-89"></div> <span class="help-tooltip" id="cb-general-help-89">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r865c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r865c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r865c1_1.svg">
        NZ_CP045031 (3763050-3806938): Streptomyces sp. C8S0 chromosome (9% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r865c1_2.svg">
        NZ_KQ948991 (303659-414886): Streptomyces resistomycificus strain DSM 40133 P... (2% of genes show similarity), NRPS-like,T2PKS
      </option>
      
      <option value = "svg/clusterblast_r865c1_3.svg">
        NZ_CP023696 (5749736-5791528): Streptomyces fradiae ATCC 10745 = DSM 40063 st... (8% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r865c1_4.svg">
        NZ_CP017316 (5565104-5606819): Streptomyces rubrolavendulae strain MJM4426 ch... (8% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r865c1_5.svg">
        NZ_KQ948308 (144578-188406): Streptomyces canus strain DSM 40275 PRJNA299222 ... (7% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r865c1_6.svg">
        NZ_JOJC01000019 (30605-74439): Streptomyces sp. NRRL S-623 contig17.1, whole ... (7% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r865c1_7.svg">
        NZ_CP054926 (2716476-2758701): Streptomyces fulvissimus strain NA06532 chromo... (7% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r865c1_8.svg">
        NC_021177 (2784609-2826375): Streptomyces fulvissimus DSM 40593, complete seq... (7% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r865c1_9.svg">
        NZ_KZ819159 (2364577-2407918): Streptomyces sp. CS090A Scaffold00001, whole g... (8% of genes show similarity), NRPS-like
      </option>
      
      <option value = "svg/clusterblast_r865c1_10.svg">
        NZ_WWIF01000010 (142014-185848): Streptomyces sp. SID5594 SID5594.c10, whole ... (8% of genes show similarity), NRPS-like
      </option>
      
    </select>
    <button id= "clusterblast-r865c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r865c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r865c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-90"></div> <span class="help-tooltip" id="cb-known-help-90">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r865c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r865c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r865c1_1.svg">
        cyphomycin
      </option>
      
      <option value = "svg/knownclusterblast_r865c1_2.svg">
        indigoidine
      </option>
      
      <option value = "svg/knownclusterblast_r865c1_3.svg">
        indigoidine
      </option>
      
    </select>
    <button id="knownclusterblast-r865c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r865c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r865c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-91"></div> <span class="help-tooltip" id="cb-sub-help-91">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r865c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-92"></div> <span class="help-tooltip" id="MIBiG-help-92">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r865c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r865c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r865c1"></div>
    <div id="comparison-MIBiG-r865c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
         
        
        <td>0.19</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.19</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873.1: 1647607-1656213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>pyrrolizixenamide A</td>
        
        
        <td>Xenorhabdus szentirmaii DSM 16338</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641.1: 3124570-3174107">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>kolossin</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP, Alkaloid</td>
        
        <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r865c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.51 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
      <td>0.51</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873.1: 1647607-1656213">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>pyrrolizixenamide A</td>
      <td>Xenorhabdus szentirmaii DSM 16338</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641.1: 3124570-3174107">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>kolossin</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP, Alkaloid</td>
      <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r865c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-93"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-93">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r865c1-pfam-details-svg" data-filename="c00865_CC228A_.._r865c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r865c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r865c1-nrps_pks_products" data-name="r865c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r865c1-nrps_pks" data-name="r865c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r865c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-87"></div> <span class="help-tooltip" id="nrps-structures-help-87">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 5462:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r865c1-1" class="smiles-canvas" data-smiles="NC(CCC(=O)N)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(gln)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r865c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CCC(=O)N)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r865c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-88"></div> <span class="help-tooltip" id="nrps-monomers-help-88">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds204_4098</span></strong>: gln
      <div class="collapser collapser-target-cds204_4098 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gln*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (21..435)</strong>: gln
          <div class="collapser collapser-target-cds204_4098 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gln
              <div class="collapser collapser-target-cds204_4098-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>asp, asn, glu, gln, aad</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>gln</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAWQFGLInK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r901c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00901_CC228A_.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-199"></div> <span class="help-tooltip" id="region_svg-help-199">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00901_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r901c1-svg" data-filename="c00901_CC228A_.._r901c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 2,097 nt. (total: 2,097 nt)
        
        <a class="cluster-rules-header" id="r901c1-rules-header" href="#r901c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r901c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r901c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-200"></div> <span class="help-tooltip" id="focus-panel-help-200">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r901c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r901c1-clusterblast" data-name="r901c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r901c1-knownclusterblast" data-name="r901c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r901c1-subclusterblast" data-name="r901c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r901c1-MIBiG-cluster-compare" data-name="r901c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r901c1-pfam-details" data-name="r901c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r901c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-94"></div> <span class="help-tooltip" id="cb-general-help-94">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r901c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r901c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r901c1_1.svg">
        NZ_CP045031 (432184-456229): Streptomyces sp. C8S0 chromosome (9% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r901c1_2.svg">
        NZ_LIVX01000007 (358888-382383): Streptomyces sp. CB02261 scaffold7, whole ge... (9% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r901c1_3.svg">
        NZ_LQYB01000010 (198637-223939): Streptomyces sp. IMTB 1903 contig 17, whole ... (8% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r901c1_4.svg">
        NZ_CP019798 (486340-509853): Streptomyces sp. fd1-xmd chromosome, complete ge... (9% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r901c1_5.svg">
        NZ_LBMK01000003 (1294384-1319790): Streptomyces yangpuensis strain fd2-tb sca... (8% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r901c1_6.svg">
        NZ_CP033073 (309950-385165): Streptomyces sp. Z022 chromosome, complete genome (3% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r901c1_7.svg">
        NZ_JACHJI010000005 (69647-94986): Streptomyces griseostramineus strain CECT 3... (7% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r901c1_8.svg">
        NZ_SLZB01000001 (770772-796664): Streptomyces sp. BK308 Ga0307715 101, whole ... (10% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r901c1_9.svg">
        NZ_JDUZ01000004 (568358-593627): Streptomyces sp. PCS3-D2 isolate P1 scaffold... (8% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r901c1_10.svg">
        NZ_CP023700 (1047600-1070618): Streptomyces viridosporus T7A strain ATCC 3911... (8% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r901c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r901c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r901c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-95"></div> <span class="help-tooltip" id="cb-known-help-95">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r901c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r901c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r901c1_1.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r901c1_2.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r901c1_3.svg">
        isorenieratene
      </option>
      
      <option value = "svg/knownclusterblast_r901c1_4.svg">
        carotenoid
      </option>
      
      <option value = "svg/knownclusterblast_r901c1_5.svg">
        isorenieratene
      </option>
      
    </select>
    <button id="knownclusterblast-r901c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r901c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r901c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-96"></div> <span class="help-tooltip" id="cb-sub-help-96">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r901c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-97"></div> <span class="help-tooltip" id="MIBiG-help-97">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r901c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r901c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r901c1"></div>
    <div id="comparison-MIBiG-r901c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650.1: 28-7720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.43, order:1.00, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Algoriphagus sp. KK10202C</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.63, order:1.00, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Streptomyces avermitilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.61, order:1.00, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.31</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.59, order:1.00, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces collinus Tu 365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.62, order:1.00, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Terpene</td>
        
        <td>isorenieratene</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630.1: 389-6051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:1.00, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
        
        
        <td>Paracoccus haeundaensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640.1: 2399-8466">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.44, order:1.00, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Enterobacteriaceae bacterium DC404</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.46, order:1.00, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>Terpene</td>
        
        <td>β-carotein</td>
        
        
        <td>uncultured bacterium</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000643.1: 363-10046">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000643/index.html#r1c1" target="_new">BGC0000643.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.45, order:1.00, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Brevundimonas vesicularis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000634.1: 1312-11784">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000634/index.html#r1c1" target="_new">BGC0000634.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.45, order:1.00, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.25</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Brevundimonas sp. SD212</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r901c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650.1: 28-7720">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.43, order:1.00, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Algoriphagus sp. KK10202C</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633.1: 0-14284">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.63, order:1.00, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Streptomyces avermitilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664.1: 8206572-8214963">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.61, order:1.00, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227.1: 697068-709212">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.59, order:1.00, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces collinus Tu 365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456.1: 0-9777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.62, order:1.00, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Terpene</td>
      <td>isorenieratene</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000630.1: 389-6051">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000630/index.html#r1c1" target="_new">BGC0000630.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:1.00, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>(2R,3S,3&#39;S)-2-hydroxyastaxanthin</td>
      <td>Paracoccus haeundaensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000640.1: 2399-8466">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000640/index.html#r1c1" target="_new">BGC0000640.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.44, order:1.00, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Enterobacteriaceae bacterium DC404</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000646.1: 60-6380">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000646/index.html#r1c1" target="_new">BGC0000646.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.46, order:1.00, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Terpene</td>
      <td>β-carotein</td>
      <td>uncultured bacterium</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000643.1: 363-10046">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000643/index.html#r1c1" target="_new">BGC0000643.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.45, order:1.00, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Brevundimonas vesicularis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000634.1: 1312-11784">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000634/index.html#r1c1" target="_new">BGC0000634.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.45, order:1.00, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Brevundimonas sp. SD212</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r901c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-98"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-98">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r901c1-pfam-details-svg" data-filename="c00901_CC228A_.._r901c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r901c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r945c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00945_CC228A_.. - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-201"></div> <span class="help-tooltip" id="region_svg-help-201">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00945_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r945c1-svg" data-filename="c00945_CC228A_.._r945c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 1,836 nt. (total: 1,836 nt)
        
        <a class="cluster-rules-header" id="r945c1-rules-header" href="#r945c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r945c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r945c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-202"></div> <span class="help-tooltip" id="focus-panel-help-202">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r945c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r945c1-nrps_pks" data-name="r945c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r945c1-clusterblast" data-name="r945c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r945c1-knownclusterblast" data-name="r945c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r945c1-subclusterblast" data-name="r945c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r945c1-MIBiG-cluster-compare" data-name="r945c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r945c1-pfam-details" data-name="r945c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r945c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-99"></div> <span class="help-tooltip" id="nrps-domain-help-99">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r945c1-details-svg" data-filename="c00945_CC228A_.._r945c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r945c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r945c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-102"></div> <span class="help-tooltip" id="cb-general-help-102">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r945c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r945c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r945c1_1.svg">
        NZ_LWKZ01000024 (223540-298739): Streptomyces sp. CB02923 scaffold4, whole ge... (17% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r945c1_2.svg">
        NZ_KZ679082 (5453353-5552689): Streptomyces sp. CS149 scaffold00001, whole ge... (11% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r945c1_3.svg">
        NZ_CP023701 (&lt;958051-1083867): Streptomyces subrutilus strain ATCC 27467 chro... (9% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r945c1_4.svg">
        NZ_LT629701 (2429365-2651872): Allokutzneria albata strain DSM 44149 chromoso... (4% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r945c1_5.svg">
        NZ_CP020039 (7054955-7140587): Streptomyces sp. 3211 isolate 3 chromosome, co... (11% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r945c1_6.svg">
        NZ_CP020555 (1178800-1265015): Streptomyces sp. Sge12 chromosome, complete ge... (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r945c1_7.svg">
        NZ_CP031742 (1467354-1528501): Streptomyces koyangensis strain VK-A60T chromo... (11% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r945c1_8.svg">
        NZ_CP034539 (414367-493060): Streptomyces cyaneochromogenes strain MK-45 chro... (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r945c1_9.svg">
        NZ_CP012949 (5818422-5994901): Streptomyces ambofaciens strain DSM 40697 chro... (5% of genes show similarity), NRPS,T1PKS,oligosaccharide
      </option>
      
      <option value = "svg/clusterblast_r945c1_10.svg">
        NZ_HG322952 (1005917-1108621): Mycolicibacterium septicum DSM 44393 strain ty... (12% of genes show similarity), NRPS,T1PKS,betalactone
      </option>
      
    </select>
    <button id= "clusterblast-r945c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r945c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r945c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-103"></div> <span class="help-tooltip" id="cb-known-help-103">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r945c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r945c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r945c1_1.svg">
        glycinocin A
      </option>
      
      <option value = "svg/knownclusterblast_r945c1_2.svg">
        lysocin
      </option>
      
      <option value = "svg/knownclusterblast_r945c1_3.svg">
        glidobactin
      </option>
      
      <option value = "svg/knownclusterblast_r945c1_4.svg">
        marformycin A / marformycin B / marformycin C / marformycin D / marformycin E / marformycin F
      </option>
      
      <option value = "svg/knownclusterblast_r945c1_5.svg">
        WAP-8294A2
      </option>
      
      <option value = "svg/knownclusterblast_r945c1_6.svg">
        teleocidin B1
      </option>
      
      <option value = "svg/knownclusterblast_r945c1_7.svg">
        echinomycin
      </option>
      
      <option value = "svg/knownclusterblast_r945c1_8.svg">
        pepticinnamin E
      </option>
      
      <option value = "svg/knownclusterblast_r945c1_9.svg">
        quinomycin
      </option>
      
      <option value = "svg/knownclusterblast_r945c1_10.svg">
        triostin A
      </option>
      
    </select>
    <button id="knownclusterblast-r945c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r945c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r945c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-104"></div> <span class="help-tooltip" id="cb-sub-help-104">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r945c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-105"></div> <span class="help-tooltip" id="MIBiG-help-105">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r945c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r945c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r945c1"></div>
    <div id="comparison-MIBiG-r945c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS-like</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.34</td>
        <td>NRP</td>
        
        <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
        
        
        <td>Trichormus variabilis ATCC 29413</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.48, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>indigoidine</td>
        
        
        <td>Streptomyces chromofuscus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>ferrichrome</td>
        
        
        <td>Aspergillus oryzae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.27</td>
        <td>NRP</td>
        
        <td>dimethylcoprogen</td>
        
        
        <td>Alternaria alternata</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 10E9</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>NRP, Polyketide</td>
        
        <td>cylindrospermopsin</td>
        
        
        <td>Aphanizomenon sp. 22D11</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000150.1: 3027086-3086780">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000150/index.html#r1c1" target="_new">BGC0000150.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.49, order:1.00, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP, Polyketide</td>
        
        <td>sporolide A, sporolide B</td>
        
        
        <td>Salinispora tropica CNB-440</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000833.1: 2306-40140">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000833/index.html#r1c1" target="_new">BGC0000833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.49, order:1.00, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>Saccharide, Other,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
        
        <td>coumermycin A1</td>
        
        
        <td>Streptomyces rishiriensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000832.1: 9347-42291">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000832/index.html#r1c1" target="_new">BGC0000832.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.49, order:1.00, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>Saccharide, Other,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
        
        <td>clorobiocin</td>
        
        
        <td>Streptomyces roseochromogenus subsp. oscitans DS 12.976</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r945c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000427.1: 4798899-4805360">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000427/index.html#r1c1" target="_new">BGC0000427.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>NRP</td>
      <td>shinorine, 4-deoxygadusol, mycosporine glycine</td>
      <td>Trichormus variabilis ATCC 29413</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375.1: 0-9457">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.48, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>indigoidine</td>
      <td>Streptomyces chromofuscus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900.1: 2044-8911">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>ferrichrome</td>
      <td>Aspergillus oryzae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249.1: 663-5994">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>NRP</td>
      <td>dimethylcoprogen</td>
      <td>Alternaria alternata</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 10E9</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980.1: 0-6921">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP, Polyketide</td>
      <td>cylindrospermopsin</td>
      <td>Aphanizomenon sp. 22D11</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000150.1: 3027086-3086780">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000150/index.html#r1c1" target="_new">BGC0000150.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.49, order:1.00, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP, Polyketide</td>
      <td>sporolide A, sporolide B</td>
      <td>Salinispora tropica CNB-440</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000833.1: 2306-40140">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000833/index.html#r1c1" target="_new">BGC0000833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.49, order:1.00, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Saccharide, Other,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
      <td>coumermycin A1</td>
      <td>Streptomyces rishiriensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000832.1: 9347-42291">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000832/index.html#r1c1" target="_new">BGC0000832.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.49, order:1.00, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Saccharide, Other,  , (, A, m, i, n, o, c, o, u, m, a, r, i, n, )</td>
      <td>clorobiocin</td>
      <td>Streptomyces roseochromogenus subsp. oscitans DS 12.976</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r945c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-106"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-106">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r945c1-pfam-details-svg" data-filename="c00945_CC228A_.._r945c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r945c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r945c1-nrps_pks_products" data-name="r945c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r945c1-nrps_pks" data-name="r945c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r945c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-100"></div> <span class="help-tooltip" id="nrps-structures-help-100">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 1836:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r945c1-1" class="smiles-canvas" data-smiles="NC(C(C)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(val)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r945c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(C(C)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r945c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-101"></div> <span class="help-tooltip" id="nrps-monomers-help-101">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds411_1836</span></strong>: val
      <div class="collapser collapser-target-cds411_1836 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (16..282)</strong>: val
          <div class="collapser collapser-target-cds411_1836 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  val
              <div class="collapser collapser-target-cds411_1836-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>val</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAFWvGGTFK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>val</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r1004c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c01004_CC228A_.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-203"></div> <span class="help-tooltip" id="region_svg-help-203">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c01004_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1004c1-svg" data-filename="c01004_CC228A_.._r1004c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 4,516 nt. (total: 4,516 nt)
        
        <a class="cluster-rules-header" id="r1004c1-rules-header" href="#r1004c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r1004c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r1004c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-204"></div> <span class="help-tooltip" id="focus-panel-help-204">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1004c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1004c1-nrps_pks" data-name="r1004c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r1004c1-clusterblast" data-name="r1004c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1004c1-knownclusterblast" data-name="r1004c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1004c1-subclusterblast" data-name="r1004c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1004c1-MIBiG-cluster-compare" data-name="r1004c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1004c1-pfam-details" data-name="r1004c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1004c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-107"></div> <span class="help-tooltip" id="nrps-domain-help-107">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1004c1-details-svg" data-filename="c01004_CC228A_.._r1004c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r1004c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r1004c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-110"></div> <span class="help-tooltip" id="cb-general-help-110">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <p>No significant ClusterBlast hits found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r1004c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-111"></div> <span class="help-tooltip" id="cb-known-help-111">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r1004c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-112"></div> <span class="help-tooltip" id="cb-sub-help-112">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1004c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-113"></div> <span class="help-tooltip" id="MIBiG-help-113">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1004c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1004c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1004c1"></div>
    <div id="comparison-MIBiG-r1004c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.23</td>
        <td>NRP</td>
        
        <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
        
        
        <td>Penicillium rubens Wisconsin 54-1255</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641.1: 3124570-3174107">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>kolossin</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.22</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001240.1: 89510-118026">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001240/index.html#r1c1" target="_new">BGC0001240.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.41, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>serinocyclin A, serinocyclin B</td>
        
        
        <td>Metarhizium robertsii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.21</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1004c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>NRP</td>
      <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
      <td>Penicillium rubens Wisconsin 54-1255</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641.1: 3124570-3174107">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>kolossin</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001240.1: 89510-118026">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001240/index.html#r1c1" target="_new">BGC0001240.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.41, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>serinocyclin A, serinocyclin B</td>
      <td>Metarhizium robertsii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1004c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-114"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-114">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1004c1-pfam-details-svg" data-filename="c01004_CC228A_.._r1004c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1004c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r1004c1-nrps_pks_products" data-name="r1004c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r1004c1-nrps_pks" data-name="r1004c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r1004c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-108"></div> <span class="help-tooltip" id="nrps-structures-help-108">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 4516:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r1004c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r1004c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r1004c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-109"></div> <span class="help-tooltip" id="nrps-monomers-help-109">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds0_4515</span></strong>: X
      <div class="collapser collapser-target-cds0_4515 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (726..1122)</strong>: X
          <div class="collapser collapser-target-cds0_4515 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophilic
              <div class="collapser collapser-target-cds0_4515-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVwnVGAInK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>arg</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r1021c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c01021_CC228A_.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-205"></div> <span class="help-tooltip" id="region_svg-help-205">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c01021_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1021c1-svg" data-filename="c01021_CC228A_.._r1021c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 2,206 nt. (total: 2,206 nt)
        
        <a class="cluster-rules-header" id="r1021c1-rules-header" href="#r1021c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r1021c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r1021c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-206"></div> <span class="help-tooltip" id="focus-panel-help-206">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1021c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1021c1-nrps_pks" data-name="r1021c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r1021c1-clusterblast" data-name="r1021c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1021c1-knownclusterblast" data-name="r1021c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1021c1-subclusterblast" data-name="r1021c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1021c1-MIBiG-cluster-compare" data-name="r1021c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1021c1-pfam-details" data-name="r1021c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1021c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-115"></div> <span class="help-tooltip" id="nrps-domain-help-115">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1021c1-details-svg" data-filename="c01021_CC228A_.._r1021c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r1021c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r1021c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-116"></div> <span class="help-tooltip" id="cb-general-help-116">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <p>No significant ClusterBlast hits found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r1021c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-117"></div> <span class="help-tooltip" id="cb-known-help-117">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r1021c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-118"></div> <span class="help-tooltip" id="cb-sub-help-118">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1021c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-119"></div> <span class="help-tooltip" id="MIBiG-help-119">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1021c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1021c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1021c1"></div>
    <div id="comparison-MIBiG-r1021c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.19</td>
        <td>NRP</td>
        
        <td>bicornutin A1, bicornutin A2</td>
        
        
        <td>Xenorhabdus budapestensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.19</td>
        <td>NRP</td>
        
        <td>luminmide</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>holrhizin</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>rhizomide A, rhizomide B, rhizomide C</td>
        
        
        <td>Paraburkholderia rhizoxinica HKI 454</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>icosalide A, icosalide B</td>
        
        
        <td>Burkholderia gladioli</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>xenematide</td>
        
        
        <td>Xenorhabdus nematophila AN6/1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641.1: 3124570-3174107">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>kolossin</td>
        
        
        <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>fellutamide B</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
         
        
        <td>0.18</td>
        <td>NRP</td>
        
        <td>xenotetrapeptide</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.18</td>
        <td>NRP, Alkaloid</td>
        
        <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
        
        
        <td>Pseudomonas fluorescens</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1021c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135.1: 0-16848">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>bicornutin A1, bicornutin A2</td>
      <td>Xenorhabdus budapestensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128.1: 3865126-3880777">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>luminmide</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844.1: 510845-531583">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.46, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>holrhizin</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758.1: 23939-46916">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>rhizomide A, rhizomide B, rhizomide C</td>
      <td>Paraburkholderia rhizoxinica HKI 454</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833.1: 14964-30015">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>icosalide A, icosalide B</td>
      <td>Burkholderia gladioli</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001825.1: 2673981-2688831">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001825/index.html#r1c1" target="_new">BGC0001825.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>xenematide</td>
      <td>Xenorhabdus nematophila AN6/1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641.1: 3124570-3174107">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>kolossin</td>
      <td>Photorhabdus laumondii subsp. laumondii TTO1</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>fellutamide B</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132.1: 1902004-1914421">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
      <td>0.50</td>
      <td>NRP</td>
      <td>xenotetrapeptide</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075.1: 27344-31238">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>NRP, Alkaloid</td>
      <td>Pyreudione A, Pyreudione B, Pyreudione C, Pyreudione D, Pyreudione E</td>
      <td>Pseudomonas fluorescens</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1021c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-120"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-120">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1021c1-pfam-details-svg" data-filename="c01021_CC228A_.._r1021c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1021c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r1060c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c01060_CC228A_.. - Region 1 - other <div class="help-container"> <div class="help-icon" data-id="region_svg-help-207"></div> <span class="help-tooltip" id="region_svg-help-207">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c01060_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1060c1-svg" data-filename="c01060_CC228A_.._r1060c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 5,358 nt. (total: 5,358 nt)
        
        <a class="cluster-rules-header" id="r1060c1-rules-header" href="#r1060c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r1060c1-rules">other: (LmbU or Neocarzinostat or fom1 or bcpB or frbD or mitE or vlmB or prnB or CaiA or bacilysin or orf2_PTase)</div>
      
      <div class="region-svg-container">
       <div id='r1060c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-208"></div> <span class="help-tooltip" id="focus-panel-help-208">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1060c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1060c1-clusterblast" data-name="r1060c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1060c1-knownclusterblast" data-name="r1060c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1060c1-subclusterblast" data-name="r1060c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1060c1-MIBiG-cluster-compare" data-name="r1060c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1060c1-pfam-details" data-name="r1060c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1060c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-121"></div> <span class="help-tooltip" id="cb-general-help-121">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1060c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1060c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1060c1_1.svg">
        NZ_CP045031 (1646726-1687875): Streptomyces sp. C8S0 chromosome (11% of genes show similarity), other
      </option>
      
      <option value = "svg/clusterblast_r1060c1_2.svg">
        NZ_LIYH01000002 (1018431-1075662): Streptomyces sp. CB03234 scaffold2, whole ... (7% of genes show similarity), T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r1060c1_3.svg">
        NZ_NBCN01000024 (643-44123): Streptomyces sp. CB03238 scaffold24, whole genom... (10% of genes show similarity), T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r1060c1_4.svg">
        NZ_CP034279 (2119776-2160898): Streptomyces ficellus strain NRRL 8067 chromos... (11% of genes show similarity), other
      </option>
      
      <option value = "svg/clusterblast_r1060c1_5.svg">
        NZ_CP026304 (5924998-5966114): Streptomyces lunaelactis strain MM109 chromoso... (9% of genes show similarity), other
      </option>
      
      <option value = "svg/clusterblast_r1060c1_6.svg">
        NZ_AUBE01000015 (129445-170618): Streptomyces flavidovirens DSM 40150 G412DRA... (11% of genes show similarity), other
      </option>
      
      <option value = "svg/clusterblast_r1060c1_7.svg">
        NZ_KB897776 (106109-147276): Streptomyces sp. CNT372 C593DRAFT scaffold00001.... (13% of genes show similarity), other
      </option>
      
      <option value = "svg/clusterblast_r1060c1_8.svg">
        NZ_CP048397 (5334937-5375678): Streptomyces sp. S4.7 chromosome, complete genome (11% of genes show similarity), other
      </option>
      
      <option value = "svg/clusterblast_r1060c1_9.svg">
        NZ_CP018047 (2285071-2326172): Streptomyces niveus strain SCSIO 3406 chromoso... (11% of genes show similarity), other
      </option>
      
      <option value = "svg/clusterblast_r1060c1_10.svg">
        NZ_KB913030 (5169025-5224826): Streptomyces purpureus KA281 StrpuDRAFT scaffo... (6% of genes show similarity), CDPS,other
      </option>
      
    </select>
    <button id= "clusterblast-r1060c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1060c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1060c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-122"></div> <span class="help-tooltip" id="cb-known-help-122">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r1060c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-123"></div> <span class="help-tooltip" id="cb-sub-help-123">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1060c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-124"></div> <span class="help-tooltip" id="MIBiG-help-124">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1060c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1060c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1060c1"></div>
    <div id="comparison-MIBiG-r1060c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">other</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Terpene</td>
        
        <td>platencin</td>
        
        
        <td>Streptomyces platensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001850.1: 258926-281351">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001850/index.html#r1c1" target="_new">BGC0001850.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Other,  , (, S, h, i, k, i, m, a, t, e, -, d, e, r, i, v, e, d, )</td>
        
        <td>diazaquinomycin A, diazaquinomycin E, diazaquinomycin F, diazaquinomycin G</td>
        
        
        <td>Streptomyces sp. F001</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001295.1: 0-18154">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001295/index.html#r1c1" target="_new">BGC0001295.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>cremeomycin</td>
        
        
        <td>Streptomyces cremeus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001965.1: 0-30490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001965/index.html#r1c1" target="_new">BGC0001965.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>nybomycin</td>
        
        
        <td>Streptomyces albus subsp. chlorinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001783.1: 55-63934">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001783/index.html#r1c1" target="_new">BGC0001783.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Other</td>
        
        <td>streptonigrin</td>
        
        
        <td>Streptomyces flocculus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001882.1: 2571644-2615001">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001882/index.html#r1c1" target="_new">BGC0001882.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
         
        
        <td>0.13</td>
        <td>Polyketide</td>
        
        <td>chrysoxanthone A, chrysoxanthone B, chrysoxanthone C</td>
        
        
        <td>Penicillium rubens Wisconsin 54-1255</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000924.1: 136-5698">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000924/index.html#r1c1" target="_new">BGC0000924.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
         
        
        <td>0.12</td>
        <td>Other</td>
        
        <td>pyrrolnitrin</td>
        
        
        <td>Pseudomonas chlororaphis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001935.1: 0-9112">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001935/index.html#r1c1" target="_new">BGC0001935.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.27 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
         
        
        <td>0.07</td>
        <td>Other</td>
        
        <td>3,7-dihydroxytropolone</td>
        
        
        <td>Streptomyces cyaneofuscatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001487.1: 0-9112">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001487/index.html#r1c1" target="_new">BGC0001487.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.27 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
         
        
        <td>0.07</td>
        <td>Other</td>
        
        <td>3,7-dihydroxytropolone</td>
        
        
        <td>Streptomyces cyaneofuscatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001868.1: 193423-204464">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001868/index.html#r1c1" target="_new">BGC0001868.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.23 (id:0.44, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
         
        
        <td>0.05</td>
        <td>Other,  , (, P, h, e, n, a, z, i, n, e, )</td>
        
        <td>iodinin</td>
        
        
        <td>Xenorhabdus szentirmaii DSM 16338</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1060c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001156.1: 2584-38160">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001156/index.html#r1c1" target="_new">BGC0001156.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Terpene</td>
      <td>platencin</td>
      <td>Streptomyces platensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001850.1: 258926-281351">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001850/index.html#r1c1" target="_new">BGC0001850.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other,  , (, S, h, i, k, i, m, a, t, e, -, d, e, r, i, v, e, d, )</td>
      <td>diazaquinomycin A, diazaquinomycin E, diazaquinomycin F, diazaquinomycin G</td>
      <td>Streptomyces sp. F001</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001295.1: 0-18154">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001295/index.html#r1c1" target="_new">BGC0001295.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>cremeomycin</td>
      <td>Streptomyces cremeus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001965.1: 0-30490">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001965/index.html#r1c1" target="_new">BGC0001965.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Other</td>
      <td>nybomycin</td>
      <td>Streptomyces albus subsp. chlorinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001783.1: 55-63934">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001783/index.html#r1c1" target="_new">BGC0001783.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Other</td>
      <td>streptonigrin</td>
      <td>Streptomyces flocculus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001882.1: 2571644-2615001">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001882/index.html#r1c1" target="_new">BGC0001882.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
      <td>0.41</td>
      <td>Polyketide</td>
      <td>chrysoxanthone A, chrysoxanthone B, chrysoxanthone C</td>
      <td>Penicillium rubens Wisconsin 54-1255</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000924.1: 136-5698">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000924/index.html#r1c1" target="_new">BGC0000924.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
      <td>0.39</td>
      <td>Other</td>
      <td>pyrrolnitrin</td>
      <td>Pseudomonas chlororaphis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001935.1: 0-9112">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001935/index.html#r1c1" target="_new">BGC0001935.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.27 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
      <td>0.27</td>
      <td>Other</td>
      <td>3,7-dihydroxytropolone</td>
      <td>Streptomyces cyaneofuscatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001487.1: 0-9112">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001487/index.html#r1c1" target="_new">BGC0001487.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.27 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
      <td>0.27</td>
      <td>Other</td>
      <td>3,7-dihydroxytropolone</td>
      <td>Streptomyces cyaneofuscatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001868.1: 193423-204464">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001868/index.html#r1c1" target="_new">BGC0001868.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.23 (id:0.44, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
      <td>0.23</td>
      <td>Other,  , (, P, h, e, n, a, z, i, n, e, )</td>
      <td>iodinin</td>
      <td>Xenorhabdus szentirmaii DSM 16338</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1060c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-125"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-125">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1060c1-pfam-details-svg" data-filename="c01060_CC228A_.._r1060c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1060c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r1072c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c01072_CC228A_.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-209"></div> <span class="help-tooltip" id="region_svg-help-209">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c01072_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1072c1-svg" data-filename="c01072_CC228A_.._r1072c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 5,572 nt. (total: 5,572 nt)
        
        <a class="cluster-rules-header" id="r1072c1-rules-header" href="#r1072c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r1072c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r1072c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-210"></div> <span class="help-tooltip" id="focus-panel-help-210">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1072c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1072c1-clusterblast" data-name="r1072c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1072c1-knownclusterblast" data-name="r1072c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1072c1-subclusterblast" data-name="r1072c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1072c1-MIBiG-cluster-compare" data-name="r1072c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1072c1-pfam-details" data-name="r1072c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1072c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-126"></div> <span class="help-tooltip" id="cb-general-help-126">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1072c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1072c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1072c1_1.svg">
        NZ_CP017316 (138392-158240): Streptomyces rubrolavendulae strain MJM4426 chro... (21% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1072c1_2.svg">
        NZ_CP023696 (147426-167254): Streptomyces fradiae ATCC 10745 = DSM 40063 stra... (21% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1072c1_3.svg">
        NZ_CP023702 (7276900-7349416): Streptomyces nitrosporeus strain ATCC 12769 ch... (5% of genes show similarity), T2PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r1072c1_4.svg">
        NZ_KB897787 (50934-72230): Streptomyces sp. CNT372 C593DRAFT scaffold00012.12... (18% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1072c1_5.svg">
        NZ_CP042266 (248174-269414): Streptomyces qinzhouensis strain SSL-25 chromoso... (20% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1072c1_6.svg">
        NZ_WWJS01000026 (172809-245325): Streptomyces sp. SID4912 SID4912.c26, whole ... (4% of genes show similarity), T2PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r1072c1_7.svg">
        NZ_LACQ01000007 (176708-249224): Streptomyces sp. OspMP-M45 YUMDRAFT scaffold... (4% of genes show similarity), T2PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r1072c1_8.svg">
        NZ_CP034279 (6843470-6898473): Streptomyces ficellus strain NRRL 8067 chromos... (5% of genes show similarity), NRPS,terpene
      </option>
      
      <option value = "svg/clusterblast_r1072c1_9.svg">
        NC_015953 (240431-310930): Streptomyces sp. SirexAA-E, complete sequence (4% of genes show similarity), T2PKS,terpene
      </option>
      
      <option value = "svg/clusterblast_r1072c1_10.svg">
        NZ_WWGT01000010 (596926-669442): Streptomyces sp. SID8358 SID8358.c10, whole ... (4% of genes show similarity), T2PKS,terpene
      </option>
      
    </select>
    <button id= "clusterblast-r1072c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1072c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1072c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-127"></div> <span class="help-tooltip" id="cb-known-help-127">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r1072c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r1072c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r1072c1_1.svg">
        geosmin
      </option>
      
    </select>
    <button id="knownclusterblast-r1072c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r1072c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1072c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-128"></div> <span class="help-tooltip" id="cb-sub-help-128">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1072c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-129"></div> <span class="help-tooltip" id="MIBiG-help-129">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1072c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1072c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1072c1"></div>
    <div id="comparison-MIBiG-r1072c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001181.1: 6666218-6668399">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001181/index.html#r1c1" target="_new">BGC0001181.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.75 (id:0.56, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
         
        
        <td>0.44</td>
        <td>Terpene</td>
        
        <td>geosmin</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000683.1: 86072-87080">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000683/index.html#r1c1" target="_new">BGC0000683.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>Terpene</td>
        
        <td>avermitilol</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001746.1: 7136894-7138013">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001746/index.html#r1c1" target="_new">BGC0001746.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>Terpene</td>
        
        <td>pristinol</td>
        
        
        <td>Streptomyces pristinaespiralis ATCC 25486</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.52, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Terpene</td>
        
        <td>geosmin</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.36 (id:0.43, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
         
        
        <td>0.10</td>
        <td>Terpene</td>
        
        <td>pentalenolactone</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1072c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001181.1: 6666218-6668399">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001181/index.html#r1c1" target="_new">BGC0001181.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.75 (id:0.56, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
      <td>0.75</td>
      <td>Terpene</td>
      <td>geosmin</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000683.1: 86072-87080">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000683/index.html#r1c1" target="_new">BGC0000683.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>avermitilol</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001746.1: 7136894-7138013">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001746/index.html#r1c1" target="_new">BGC0001746.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>Terpene</td>
      <td>pristinol</td>
      <td>Streptomyces pristinaespiralis ATCC 25486</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661.1: 3414765-3420177">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.52, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Terpene</td>
      <td>geosmin</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.36 (id:0.43, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
      <td>0.36</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1072c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-130"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-130">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1072c1-pfam-details-svg" data-filename="c01072_CC228A_.._r1072c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1072c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r1113c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c01113_CC228A_.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-211"></div> <span class="help-tooltip" id="region_svg-help-211">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c01113_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1113c1-svg" data-filename="c01113_CC228A_.._r1113c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 4,371 nt. (total: 4,371 nt)
        
        <a class="cluster-rules-header" id="r1113c1-rules-header" href="#r1113c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r1113c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r1113c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-212"></div> <span class="help-tooltip" id="focus-panel-help-212">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1113c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1113c1-clusterblast" data-name="r1113c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1113c1-knownclusterblast" data-name="r1113c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1113c1-subclusterblast" data-name="r1113c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1113c1-MIBiG-cluster-compare" data-name="r1113c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1113c1-pfam-details" data-name="r1113c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1113c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-131"></div> <span class="help-tooltip" id="cb-general-help-131">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1113c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1113c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1113c1_1.svg">
        NZ_KQ948151 (169067-190873): Streptomyces pseudovenezuelae strain DSM 40212 P... (13% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1113c1_2.svg">
        NZ_CP023700 (153678-174320): Streptomyces viridosporus T7A strain ATCC 39115 ... (11% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1113c1_3.svg">
        NZ_JH993790 (6320600-6342445): Streptomyces viridosporus T7A StrvirDRAFT SVC.... (13% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1113c1_4.svg">
        NZ_CP029241 (1812201-1833226): Streptomyces sp. NHF165 chromosome, complete g... (14% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1113c1_5.svg">
        NZ_CP023696 (413401-434453): Streptomyces fradiae ATCC 10745 = DSM 40063 stra... (16% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1113c1_6.svg">
        NZ_CM000951 (7531237-7552547): Streptomyces sviceus ATCC 29083 chromosome, wh... (15% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1113c1_7.svg">
        NZ_CP016438 (9073942-9095814): Streptomyces lincolnensis strain NRRL 2936 chr... (15% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1113c1_8.svg">
        NZ_KQ948994 (276268-298086): Streptomyces resistomycificus strain DSM 40133 P... (20% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1113c1_9.svg">
        NZ_CP045704 (93416-113841): Streptomyces rutgersensis strain NBH77 chromosome... (17% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1113c1_10.svg">
        NZ_CP009754 (4774729-4794869): Streptomyces sp. CCM MD2014 chromosome, comple... (11% of genes show similarity), terpene
      </option>
      
    </select>
    <button id= "clusterblast-r1113c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1113c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1113c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-132"></div> <span class="help-tooltip" id="cb-known-help-132">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r1113c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-133"></div> <span class="help-tooltip" id="cb-sub-help-133">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1113c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-134"></div> <span class="help-tooltip" id="MIBiG-help-134">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1113c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1113c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1113c1"></div>
    <div id="comparison-MIBiG-r1113c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000243.1: 338-14804">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000243/index.html#r1c1" target="_new">BGC0000243.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>macrotetrolide</td>
        
        
        <td>Streptomyces griseus subsp. griseus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000634.1: 1312-11784">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000634/index.html#r1c1" target="_new">BGC0000634.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Terpene</td>
        
        <td>carotenoid</td>
        
        
        <td>Brevundimonas sp. SD212</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000019.1: 339-17245">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000019/index.html#r1c1" target="_new">BGC0000019.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>angolamycin</td>
        
        
        <td>Streptomyces eurythermus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000734.1: 1156-16096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000734/index.html#r1c1" target="_new">BGC0000734.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>capsular polysaccharide</td>
        
        
        <td>Pasteurella multocida</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000736.1: 477-15214">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000736/index.html#r1c1" target="_new">BGC0000736.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>capsular polysaccharide</td>
        
        
        <td>Mannheimia haemolytica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000801.1: 335-31097">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000801/index.html#r1c1" target="_new">BGC0000801.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
         
        
        <td>0.14</td>
        <td>Saccharide</td>
        
        <td>galactoglucan</td>
        
        
        <td>Sinorhizobium meliloti</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001779.1: 7655772-7662064">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001779/index.html#r1c1" target="_new">BGC0001779.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
         
        
        <td>0.13</td>
        <td>RiPP</td>
        
        <td>siamycin</td>
        
        
        <td>Streptomyces nodosus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001173.1: 223360-233305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001173/index.html#r1c1" target="_new">BGC0001173.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
         
        
        <td>0.09</td>
        <td>RiPP</td>
        
        <td>plantazolicin</td>
        
        
        <td>Bacillus pumilus ATCC 7061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001655.1: 360744-365660">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001655/index.html#r1c1" target="_new">BGC0001655.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.30 (id:0.44, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
         
        
        <td>0.07</td>
        <td>RiPP</td>
        
        <td>LP2006</td>
        
        
        <td>Nocardiopsis alba DSM 43377</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000552.1: 1388-11054">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000552/index.html#r1c1" target="_new">BGC0000552.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.29 (id:0.43, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
         
        
        <td>0.07</td>
        <td>RiPP</td>
        
        <td>SmbA, SmbB</td>
        
        
        <td>Streptococcus mutans</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1113c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000243.1: 338-14804">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000243/index.html#r1c1" target="_new">BGC0000243.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
      <td>0.45</td>
      <td>Polyketide</td>
      <td>macrotetrolide</td>
      <td>Streptomyces griseus subsp. griseus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000634.1: 1312-11784">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000634/index.html#r1c1" target="_new">BGC0000634.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Terpene</td>
      <td>carotenoid</td>
      <td>Brevundimonas sp. SD212</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000019.1: 339-17245">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000019/index.html#r1c1" target="_new">BGC0000019.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Polyketide</td>
      <td>angolamycin</td>
      <td>Streptomyces eurythermus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000734.1: 1156-16096">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000734/index.html#r1c1" target="_new">BGC0000734.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>capsular polysaccharide</td>
      <td>Pasteurella multocida</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000736.1: 477-15214">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000736/index.html#r1c1" target="_new">BGC0000736.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
      <td>0.44</td>
      <td>Saccharide</td>
      <td>capsular polysaccharide</td>
      <td>Mannheimia haemolytica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000801.1: 335-31097">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000801/index.html#r1c1" target="_new">BGC0000801.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
      <td>0.43</td>
      <td>Saccharide</td>
      <td>galactoglucan</td>
      <td>Sinorhizobium meliloti</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001779.1: 7655772-7662064">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001779/index.html#r1c1" target="_new">BGC0001779.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
      <td>0.42</td>
      <td>RiPP</td>
      <td>siamycin</td>
      <td>Streptomyces nodosus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001173.1: 223360-233305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001173/index.html#r1c1" target="_new">BGC0001173.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
      <td>0.33</td>
      <td>RiPP</td>
      <td>plantazolicin</td>
      <td>Bacillus pumilus ATCC 7061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001655.1: 360744-365660">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001655/index.html#r1c1" target="_new">BGC0001655.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.30 (id:0.44, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
      <td>0.30</td>
      <td>RiPP</td>
      <td>LP2006</td>
      <td>Nocardiopsis alba DSM 43377</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000552.1: 1388-11054">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000552/index.html#r1c1" target="_new">BGC0000552.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.29 (id:0.43, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
      <td>0.29</td>
      <td>RiPP</td>
      <td>SmbA, SmbB</td>
      <td>Streptococcus mutans</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1113c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-135"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-135">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1113c1-pfam-details-svg" data-filename="c01113_CC228A_.._r1113c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1113c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r1155c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC228A_contig_2 - Region 1 - other,thioamitides,thiopeptide <div class="help-container"> <div class="help-icon" data-id="region_svg-help-213"></div> <span class="help-tooltip" id="region_svg-help-213">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC228A_contig_2.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1155c1-svg" data-filename="CC228A_contig_2_r1155c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 27,655 nt. (total: 27,655 nt)
        
        <a class="cluster-rules-header" id="r1155c1-rules-header" href="#r1155c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r1155c1-rules">thiopeptide: ((YcaO or TIGR03882) and ((thio_amide and (PF06968 or PF04055 or PF07366)) or Lant_dehydr_C or Lant_dehydr_N or PF07366 or PF06968 or PF04055) or thiostrepton)<br>thioamitides: ((YcaO or TIGR03882) and TfuA)<br>other: (LmbU or Neocarzinostat or fom1 or bcpB or frbD or mitE or vlmB or prnB or CaiA or bacilysin or orf2_PTase)</div>
      
      <div class="region-svg-container">
       <div id='r1155c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-214"></div> <span class="help-tooltip" id="focus-panel-help-214">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1155c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1155c1-thiopeptides" data-name="r1155c1-thiopeptides">
        Thiopeptides
      </div>
     
    
     
      <div class="body-details-header r1155c1-clusterblast" data-name="r1155c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1155c1-knownclusterblast" data-name="r1155c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1155c1-subclusterblast" data-name="r1155c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r1155c1-MIBiG-cluster-compare" data-name="r1155c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1155c1-pfam-details" data-name="r1155c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1155c1-thiopeptides">
      <div class="details">
    <div class="heading">
      <span>Thiopeptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="thio-body-help-136"></div> <span class="help-tooltip" id="thio-body-help-136">Lists the possible core peptides for each biosynthetic enzyme, including the predicted class. Each core peptide shows the leader and core peptide sequences, separated by a dash. Predicted tail sequences are also shown.</span></div>
    </div>
  
    <div class="details-text">
      No core peptides found.
    </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1155c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-138"></div> <span class="help-tooltip" id="cb-general-help-138">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1155c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1155c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1155c1_1.svg">
        NC_020133 (3293425-3418912): Mycobacterium liflandii 128FXT, complete sequence (7% of genes show similarity), NRPS,T1PKS,T3PKS,TfuA-related
      </option>
      
      <option value = "svg/clusterblast_r1155c1_2.svg">
        NC_010612 (2973345-3092443): Mycobacterium marinum M, complete genome (8% of genes show similarity), NRPS,T1PKS,T3PKS,TfuA-related
      </option>
      
      <option value = "svg/clusterblast_r1155c1_3.svg">
        NZ_CP024190 (2803503-2923346): Mycobacterium marinum strain CCUG20998 chromos... (8% of genes show similarity), NRPS,T1PKS,T3PKS,TfuA-related
      </option>
      
      <option value = "svg/clusterblast_r1155c1_4.svg">
        NZ_AP018410 (2761860-2824476): Mycobacterium pseudoshottsii JCM 15466 (12% of genes show similarity), NRPS,TfuA-related
      </option>
      
      <option value = "svg/clusterblast_r1155c1_5.svg">
        NZ_CM001165 (5545171-5604208): Streptomyces sp. Tu6071 chromosome, whole geno... (25% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r1155c1_6.svg">
        NZ_WWGC01000026 (210295-285466): Streptomyces sp. SID8379 SID8379.c26, whole ... (14% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r1155c1_7.svg">
        NC_021985 (5425622-5483794): Streptomyces collinus Tu 365, complete sequence (20% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r1155c1_8.svg">
        NZ_VISR01000024 (54707-118954): Streptomyces sp. BK340 Ga0307703 124, whole g... (19% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r1155c1_9.svg">
        NZ_CP034350 (5756637-5815528): Streptomyces sp. W1SF4 chromosome, complete ge... (20% of genes show similarity), NRPS,ladderane
      </option>
      
      <option value = "svg/clusterblast_r1155c1_10.svg">
        NZ_SOAR01000002 (573988-632670): Streptomyces sp. BK141 Ga0307717 102, whole ... (20% of genes show similarity), NRPS,ladderane
      </option>
      
    </select>
    <button id= "clusterblast-r1155c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1155c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1155c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-139"></div> <span class="help-tooltip" id="cb-known-help-139">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r1155c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r1155c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r1155c1_1.svg">
        CDA1b / CDA2a / CDA2b / CDA3a / CDA3b / CDA4a / CDA4b
      </option>
      
      <option value = "svg/knownclusterblast_r1155c1_2.svg">
        fluostatins M-Q
      </option>
      
    </select>
    <button id="knownclusterblast-r1155c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r1155c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1155c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-140"></div> <span class="help-tooltip" id="cb-sub-help-140">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r1155c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-142"></div> <span class="help-tooltip" id="MIBiG-help-142">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1155c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1155c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1155c1"></div>
    <div id="comparison-MIBiG-r1155c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">thiopeptide</span></th>
         
         <th><span class="cc-heat-product">other</span></th>
         
         <th><span class="cc-heat-product">thioamitides</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000541.1: 0-9113">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000541/index.html#r1c1" target="_new">BGC0000541.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.83</td>
        <td>RiPP</td>
        
        <td>paenicidin A</td>
        
        
        <td>Paenibacillus polymyxa</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000540.1: 117-11692">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000540/index.html#r1c1" target="_new">BGC0000540.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.82</td>
        <td>RiPP</td>
        
        <td>paenibacillin</td>
        
        
        <td>Paenibacillus polymyxa OSY-DF</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000560.1: 1037-9106">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000560/index.html#r1c1" target="_new">BGC0000560.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.75</td>
        <td>RiPP</td>
        
        <td>subtilomycin</td>
        
        
        <td>Bacillus subtilis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000543.1: 87-7506">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000543/index.html#r1c1" target="_new">BGC0000543.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.73</td>
        <td>RiPP</td>
        
        <td>pep5</td>
        
        
        <td>Staphylococcus epidermidis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000538.1: 238-7610">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000538/index.html#r1c1" target="_new">BGC0000538.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.73</td>
        <td>RiPP</td>
        
        <td>nisin Z</td>
        
        
        <td>Lactococcus lactis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001849.1: 155003-161953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001849/index.html#r1c1" target="_new">BGC0001849.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.46, order:0.48, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.46, order:0.48, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.57 (id:0.46, order:0.48, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
         
        
        <td>0.71</td>
        <td>RiPP</td>
        
        <td>thiovarsolin A, thiovarsolin B, thiovarsolin C, thiovarsolin D</td>
        
        
        <td>Streptomyces varsoviensis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000556.1: 252-11336">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000556/index.html#r1c1" target="_new">BGC0000556.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.69</td>
        <td>RiPP</td>
        
        <td>streptin</td>
        
        
        <td>Streptococcus pyogenes</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000605.1: 7233-24412">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000605/index.html#r1c1" target="_new">BGC0000605.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.66</td>
        <td>RiPP</td>
        
        <td>GE37468</td>
        
        
        <td>Streptomyces sp. ATCC 55365</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000565.1: 2-19346">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000565/index.html#r1c1" target="_new">BGC0000565.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
         
        
        <td>0.65</td>
        <td>RiPP</td>
        
        <td>goadsporin</td>
        
        
        <td>Streptomyces sp. TP-A0584</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
         
          
          
          
        <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
         
        
        <td>0.64</td>
        <td>RiPP</td>
        
        <td>catenulipeptin</td>
        
        
        <td>Catenulispora acidiphila DSM 44928</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1155c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000541.1: 0-9113">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000541/index.html#r1c1" target="_new">BGC0000541.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>RiPP</td>
      <td>paenicidin A</td>
      <td>Paenibacillus polymyxa</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000540.1: 117-11692">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000540/index.html#r1c1" target="_new">BGC0000540.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>RiPP</td>
      <td>paenibacillin</td>
      <td>Paenibacillus polymyxa OSY-DF</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000560.1: 1037-9106">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000560/index.html#r1c1" target="_new">BGC0000560.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>RiPP</td>
      <td>subtilomycin</td>
      <td>Bacillus subtilis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000543.1: 87-7506">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000543/index.html#r1c1" target="_new">BGC0000543.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>RiPP</td>
      <td>pep5</td>
      <td>Staphylococcus epidermidis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000538.1: 238-7610">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000538/index.html#r1c1" target="_new">BGC0000538.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>RiPP</td>
      <td>nisin Z</td>
      <td>Lactococcus lactis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001849.1: 155003-161953">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001849/index.html#r1c1" target="_new">BGC0001849.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.57 (id:0.46, order:0.48, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
      <td>0.57</td>
      <td>RiPP</td>
      <td>thiovarsolin A, thiovarsolin B, thiovarsolin C, thiovarsolin D</td>
      <td>Streptomyces varsoviensis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000556.1: 252-11336">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000556/index.html#r1c1" target="_new">BGC0000556.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>RiPP</td>
      <td>streptin</td>
      <td>Streptococcus pyogenes</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000605.1: 7233-24412">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000605/index.html#r1c1" target="_new">BGC0000605.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>RiPP</td>
      <td>GE37468</td>
      <td>Streptomyces sp. ATCC 55365</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000565.1: 2-19346">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000565/index.html#r1c1" target="_new">BGC0000565.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
      <td>0.55</td>
      <td>RiPP</td>
      <td>goadsporin</td>
      <td>Streptomyces sp. TP-A0584</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501.1: 4809159-4816367">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
      <td>0.54</td>
      <td>RiPP</td>
      <td>catenulipeptin</td>
      <td>Catenulispora acidiphila DSM 44928</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1155c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-143"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-143">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1155c1-pfam-details-svg" data-filename="CC228A_contig_2_r1155c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1155c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
      <div class="sidepanel-details-header r1155c1-thiopeptides" data-name="r1155c1-thiopeptides">
        Thiopeptides
      </div>
     
    
     
    
     
      <div class="sidepanel-details-header r1155c1-RREfinder" data-name="r1155c1-RREfinder">
        RREFinder
      </div>
     
    
     
    
     
    
    </div>
    
     
    <div class="sidepanel-details-section r1155c1-thiopeptides">
      <div class="more-details">
    <div class="heading">
      <span>Core peptide predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="thio-side-help-137"></div> <span class="help-tooltip" id="thio-side-help-137">Lists the possible core peptides in the region. Each core peptide lists its possible molecular weights and the scores for cleavage site prediction and RODEO. If relevant, other features, such as macrocycle and amidation, will also be listed.</span></div>
    </div>
  
  <div class="prediction-text">No core peptides found.</div>

</div>
    </div>
     
    
     
    
     
    <div class="sidepanel-details-section r1155c1-RREfinder">
      <div class="more-details">
  <div class="heading">
    <span>RRE predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="rre-side-help-141"></div> <span class="help-tooltip" id="rre-side-help-141">RREfinder results sidepanel.</span></div>
  </div>
  <dl class = 'prediction-text'>
    
      <span><strong>thiopeptide protocluster</strong> (0...27655)</span>
      <dt></dt>
      
      
      
        
        <strong>cds14736_17655</strong> <div class="collapser collapser-target-cds14736_17655 collapser-level-cds"> </div><div class="collapser-content">
        
          <dd>RRE type: Lanthipeptide_LanB_RRE</dd>
          <dd>Location: 151 - 207</dd>
          <dd>Score: 25.5</dd>
        
        </div><br>
      <br>
    
      <span><strong>thioamitides protocluster</strong> (0...20103)</span>
      <dt></dt>
      
      
      
        
        <strong>cds14736_17655</strong> <div class="collapser collapser-target-cds14736_17655 collapser-level-cds"> </div><div class="collapser-content">
        
          <dd>RRE type: Lanthipeptide_LanB_RRE</dd>
          <dd>Location: 151 - 207</dd>
          <dd>Score: 25.5</dd>
        
        </div><br>
      <br>
    
    </dl>
</div>
    </div>
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r1207c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> CC228A_contig_68 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-215"></div> <span class="help-tooltip" id="region_svg-help-215">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = CC228A_contig_68.region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1207c1-svg" data-filename="CC228A_contig_68_r1207c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 11,095 nt. (total: 11,095 nt)
        
        <a class="cluster-rules-header" id="r1207c1-rules-header" href="#r1207c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r1207c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r1207c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-216"></div> <span class="help-tooltip" id="focus-panel-help-216">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1207c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1207c1-nrps_pks" data-name="r1207c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
    
     
      <div class="body-details-header r1207c1-clusterblast" data-name="r1207c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1207c1-knownclusterblast" data-name="r1207c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1207c1-subclusterblast" data-name="r1207c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1207c1-MIBiG-cluster-compare" data-name="r1207c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1207c1-pfam-details" data-name="r1207c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1207c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-144"></div> <span class="help-tooltip" id="nrps-domain-help-144">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1207c1-details-svg" data-filename="CC228A_contig_68_r1207c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r1207c1-details-svg"></div>
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r1207c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-147"></div> <span class="help-tooltip" id="cb-general-help-147">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1207c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1207c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1207c1_1.svg">
        NZ_AZYD01000017 (149262-212355): Streptomyces sp. LaPpAH-201 K372DRAFT scaffo... (13% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r1207c1_2.svg">
        NZ_CP019798 (6068899-6206931): Streptomyces sp. fd1-xmd chromosome, complete ... (7% of genes show similarity), NRPS,T1PKS,other
      </option>
      
      <option value = "svg/clusterblast_r1207c1_3.svg">
        NZ_WWGN01000023 (142878-200292): Streptomyces sp. SID8366 SID8366.c23, whole ... (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r1207c1_4.svg">
        NZ_CP047144 (5928875-6009287): Streptomyces sp. HF10 chromosome, complete genome (7% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r1207c1_5.svg">
        NZ_CP032229 (4811927-4921881): Streptomyces seoulensis strain KCTC 9819 chrom... (6% of genes show similarity), NRPS,T1PKS
      </option>
      
      <option value = "svg/clusterblast_r1207c1_6.svg">
        NC_020990 (1137197-1198548): Streptomyces albidoflavus, complete sequence (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r1207c1_7.svg">
        NZ_CP047147 (5768596-5830107): Streptomyces sp. GF20 chromosome, complete genome (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r1207c1_8.svg">
        NZ_CP009802 (1362290-1423584): Streptomyces sp. FR-008 chromosome, complete g... (10% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r1207c1_9.svg">
        NZ_CP014485 (1374729-1434921): Streptomyces albidoflavus strain SM254 chromos... (11% of genes show similarity), NRPS
      </option>
      
      <option value = "svg/clusterblast_r1207c1_10.svg">
        NZ_CP056773 (1280284-1342486): Streptomyces violascens strain YIM 100212 chro... (11% of genes show similarity), NRPS
      </option>
      
    </select>
    <button id= "clusterblast-r1207c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1207c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1207c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-148"></div> <span class="help-tooltip" id="cb-known-help-148">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r1207c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r1207c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r1207c1_1.svg">
        surugamide A / surugamide D
      </option>
      
    </select>
    <button id="knownclusterblast-r1207c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r1207c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1207c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-149"></div> <span class="help-tooltip" id="cb-sub-help-149">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1207c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-150"></div> <span class="help-tooltip" id="MIBiG-help-150">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1207c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1207c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1207c1"></div>
    <div id="comparison-MIBiG-r1207c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">NRPS</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.45, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.38</td>
        <td>NRP</td>
        
        <td>sevadicin</td>
        
        
        <td>Paenibacillus larvae</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.33</td>
        <td>NRP</td>
        
        <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
        
        
        <td>Xenorhabdus nematophila ATCC 19061</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.66 (id:0.46, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>Virginiafactin</td>
        
        
        <td>Pseudomonas sp. QS1027</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.65 (id:0.45, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
         
        
        <td>0.32</td>
        <td>NRP</td>
        
        <td>massetolide A</td>
        
        
        <td>Pseudomonas fluorescens SS101</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>anabaenopeptin NZ857, nostamide A</td>
        
        
        <td>Nostoc punctiforme PCC 73102</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.45, order:0.69, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.29</td>
        <td>NRP</td>
        
        <td>exochelin</td>
        
        
        <td>Mycolicibacterium smegmatis</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
         
        
        <td>0.26</td>
        <td>NRP</td>
        
        <td>fellutamide B</td>
        
        
        <td>Aspergillus nidulans FGSC A4</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133.1: 739500-803411">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.45, order:0.57, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.25</td>
        <td>NRP</td>
        
        <td>taxlllaid A</td>
        
        
        <td>Xenorhabdus bovienii SS-2004</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000901.1: 1761-23020">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000901/index.html#r1c1" target="_new">BGC0000901.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.41, order:0.69, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>Other</td>
        
        <td>ferrichrome</td>
        
        
        <td>Aspergillus niger</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>NRP</td>
        
        <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
        
        
        <td>Penicillium rubens Wisconsin 54-1255</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1207c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426.1: 0-11658">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.45, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>NRP</td>
      <td>sevadicin</td>
      <td>Paenibacillus larvae</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465.1: 2235578-2245621">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>xenortide A, xenortide B, xenortide C, xenortide D</td>
      <td>Xenorhabdus nematophila ATCC 19061</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071.1: 289714-316980">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.66 (id:0.46, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
      <td>0.66</td>
      <td>NRP</td>
      <td>Virginiafactin</td>
      <td>Pseudomonas sp. QS1027</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000389.1: 11694-35942">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000389/index.html#r1c1" target="_new">BGC0000389.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.65 (id:0.45, order:1.00, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
      <td>0.65</td>
      <td>NRP</td>
      <td>massetolide A</td>
      <td>Pseudomonas fluorescens SS101</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479.1: 3055743-3062289">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>anabaenopeptin NZ857, nostamide A</td>
      <td>Nostoc punctiforme PCC 73102</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351.1: 488-15223">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.45, order:0.69, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>NRP</td>
      <td>exochelin</td>
      <td>Mycolicibacterium smegmatis</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001399.1: 3932342-3944133">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001399/index.html#r1c1" target="_new">BGC0001399.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
      <td>0.59</td>
      <td>NRP</td>
      <td>fellutamide B</td>
      <td>Aspergillus nidulans FGSC A4</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133.1: 739500-803411">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.45, order:0.57, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>taxlllaid A</td>
      <td>Xenorhabdus bovienii SS-2004</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000901.1: 1761-23020">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000901/index.html#r1c1" target="_new">BGC0000901.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.41, order:0.69, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>ferrichrome</td>
      <td>Aspergillus niger</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357.1: 1080826-1099362">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>NRP</td>
      <td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
      <td>Penicillium rubens Wisconsin 54-1255</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1207c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-151"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-151">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1207c1-pfam-details-svg" data-filename="CC228A_contig_68_r1207c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1207c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r1207c1-nrps_pks_products" data-name="r1207c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r1207c1-nrps_pks" data-name="r1207c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
     
    
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r1207c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-145"></div> <span class="help-tooltip" id="nrps-structures-help-145">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 11095:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r1207c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(C(O)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-X) + (D-thr)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x&amp;nrps2=d-thr" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-x*&amp;nrps2=d-thr*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r1207c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(C(O)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r1207c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-146"></div> <span class="help-tooltip" id="nrps-monomers-help-146">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">cds2069_7298</span></strong>: thr
      <div class="collapser collapser-target-cds2069_7298 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (713..1118)</strong>: thr
          <div class="collapser collapser-target-cds2069_7298 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-cds2069_7298-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNVGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">cds7364_11093</span></strong>: X
      <div class="collapser collapser-target-cds7364_11093 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (448..853)</strong>: X
          <div class="collapser collapser-target-cds7364_11093 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  orn, lys, arg
              <div class="collapser collapser-target-cds7364_11093-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophilic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>orn, lys, arg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DlaEsGaVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>orn</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r1227c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c01227_CC228A_.. - Region 1 - siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-217"></div> <span class="help-tooltip" id="region_svg-help-217">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c01227_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1227c1-svg" data-filename="c01227_CC228A_.._r1227c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 3,938 nt. (total: 3,938 nt)
        
        <a class="cluster-rules-header" id="r1227c1-rules-header" href="#r1227c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r1227c1-rules">siderophore: IucA_IucC</div>
      
      <div class="region-svg-container">
       <div id='r1227c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-218"></div> <span class="help-tooltip" id="focus-panel-help-218">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1227c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1227c1-clusterblast" data-name="r1227c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1227c1-knownclusterblast" data-name="r1227c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1227c1-subclusterblast" data-name="r1227c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1227c1-MIBiG-cluster-compare" data-name="r1227c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1227c1-pfam-details" data-name="r1227c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1227c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-152"></div> <span class="help-tooltip" id="cb-general-help-152">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1227c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1227c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1227c1_1.svg">
        NZ_CP045031 (802394-818471): Streptomyces sp. C8S0 chromosome (25% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r1227c1_2.svg">
        NZ_CP023696 (5808499-5824113): Streptomyces fradiae ATCC 10745 = DSM 40063 st... (30% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r1227c1_3.svg">
        NZ_CP017316 (5623818-5639364): Streptomyces rubrolavendulae strain MJM4426 ch... (30% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r1227c1_4.svg">
        NZ_AP018216 (4971352-4988278): Anabaena variabilis NIES-23 DNA, nearly comple... (30% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r1227c1_5.svg">
        NZ_CP051167 (2007364-2024216): Oxynema sp. AP17 chromosome, complete genome (27% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r1227c1_6.svg">
        NZ_KZ454937 (29885-46538): Bacillus sp. FJAT-44742 Scaffold2, whole genome sh... (23% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r1227c1_7.svg">
        NZ_CP034279 (1359573-1409919): Streptomyces ficellus strain NRRL 8067 chromos... (7% of genes show similarity), NRPS,siderophore
      </option>
      
      <option value = "svg/clusterblast_r1227c1_8.svg">
        NZ_CP034058 (5129288-5139322): Anabaena sp. YBS01 chromosome, complete genome (33% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r1227c1_9.svg">
        NZ_MRCE01000010 (109210-126128): Phormidium ambiguum IAM M-71 NIES-2119 Scaff... (30% of genes show similarity), siderophore
      </option>
      
      <option value = "svg/clusterblast_r1227c1_10.svg">
        NZ_CP044064 (1402581-1422919): Yersinia pseudotuberculosis strain FDAARGOS 66... (21% of genes show similarity), siderophore
      </option>
      
    </select>
    <button id= "clusterblast-r1227c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1227c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1227c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-153"></div> <span class="help-tooltip" id="cb-known-help-153">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r1227c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r1227c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r1227c1_1.svg">
        ficellomycin
      </option>
      
    </select>
    <button id="knownclusterblast-r1227c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r1227c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1227c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-154"></div> <span class="help-tooltip" id="cb-sub-help-154">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1227c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-155"></div> <span class="help-tooltip" id="MIBiG-help-155">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1227c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1227c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1227c1"></div>
    <div id="comparison-MIBiG-r1227c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">siderophore</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
         
        
        <td>0.24</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Pantoea agglomerans</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453.1: 0-7234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Other</td>
        
        <td>desferrioxamine</td>
        
        
        <td>Streptomyces argillaceus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940.1: 3033354-3040682">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
         
        
        <td>0.23</td>
        <td>Other</td>
        
        <td>desferrioxamin B, desferrioxamine E</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000941.1: 5578821-5584948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000941/index.html#r1c1" target="_new">BGC0000941.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.21</td>
        <td>Other</td>
        
        <td>desferrioxamin B</td>
        
        
        <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001478.1: 100-5365">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001478/index.html#r1c1" target="_new">BGC0001478.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
         
        
        <td>0.20</td>
        <td>Other</td>
        
        <td>desferrioxamine E</td>
        
        
        <td>Streptomyces sp. ID38640</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
         
        
        <td>0.17</td>
        <td>Saccharide</td>
        
        <td>kanamycin</td>
        
        
        <td>Streptomyces kanamyceticus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>venezuelin</td>
        
        
        <td>Streptomyces venezuelae ATCC 10712</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>SapB</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>RiPP</td>
        
        <td>SRO15-3108</td>
        
        
        <td>Streptomyces filamentosus NRRL 15998</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
         
        
        <td>0.16</td>
        <td>Other</td>
        
        <td>ochrobactin</td>
        
        
        <td>Ochrobactrum anthropi</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1227c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001572.1: 4377-10894">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001572/index.html#r1c1" target="_new">BGC0001572.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
      <td>0.58</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Pantoea agglomerans</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001453.1: 0-7234">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001453/index.html#r1c1" target="_new">BGC0001453.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Other</td>
      <td>desferrioxamine</td>
      <td>Streptomyces argillaceus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000940.1: 3033354-3040682">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000940/index.html#r1c1" target="_new">BGC0000940.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.56 (id:0.46, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
      <td>0.56</td>
      <td>Other</td>
      <td>desferrioxamin B, desferrioxamine E</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000941.1: 5578821-5584948">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000941/index.html#r1c1" target="_new">BGC0000941.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Other</td>
      <td>desferrioxamin B</td>
      <td>Streptomyces griseus subsp. griseus NBRC 13350</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001478.1: 100-5365">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001478/index.html#r1c1" target="_new">BGC0001478.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.53 (id:0.45, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
      <td>0.53</td>
      <td>Other</td>
      <td>desferrioxamine E</td>
      <td>Streptomyces sp. ID38640</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000703.1: 0-204967">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000703/index.html#r1c1" target="_new">BGC0000703.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.48 (id:0.48, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
      <td>0.48</td>
      <td>Saccharide</td>
      <td>kanamycin</td>
      <td>Streptomyces kanamyceticus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563.1: 185-5330">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>venezuelin</td>
      <td>Streptomyces venezuelae ATCC 10712</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551.1: 7419663-7426391">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>SapB</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554.1: 4445107-4451653">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>RiPP</td>
      <td>SRO15-3108</td>
      <td>Streptomyces filamentosus NRRL 15998</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713.1: 164-10986">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
      <td>0.47</td>
      <td>Other</td>
      <td>ochrobactin</td>
      <td>Ochrobactrum anthropi</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1227c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-156"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-156">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1227c1-pfam-details-svg" data-filename="c01227_CC228A_.._r1227c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1227c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r1264c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c01264_CC228A_.. - Region 1 - ectoine <div class="help-container"> <div class="help-icon" data-id="region_svg-help-219"></div> <span class="help-tooltip" id="region_svg-help-219">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c01264_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1264c1-svg" data-filename="c01264_CC228A_.._r1264c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 6,212 nt. (total: 6,212 nt)
        
        <a class="cluster-rules-header" id="r1264c1-rules-header" href="#r1264c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r1264c1-rules">ectoine: ectoine_synt</div>
      
      <div class="region-svg-container">
       <div id='r1264c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-220"></div> <span class="help-tooltip" id="focus-panel-help-220">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1264c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1264c1-clusterblast" data-name="r1264c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1264c1-knownclusterblast" data-name="r1264c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1264c1-subclusterblast" data-name="r1264c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1264c1-MIBiG-cluster-compare" data-name="r1264c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1264c1-pfam-details" data-name="r1264c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1264c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-157"></div> <span class="help-tooltip" id="cb-general-help-157">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1264c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1264c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1264c1_1.svg">
        NZ_CP045031 (5407348-5417747): Streptomyces sp. C8S0 chromosome (75% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r1264c1_2.svg">
        NZ_CP054916 (7066485-7076890): Streptomyces sp. NA02950 chromosome, complete ... (66% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r1264c1_3.svg">
        NZ_AP019620 (8127320-8137725): Streptomyces antimycoticus strain NBRC 100767 (66% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r1264c1_4.svg">
        NZ_JAALLH010000001 (3285936-3296341): Streptomyces malaysiensis strain DSM 14... (66% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r1264c1_5.svg">
        NZ_CP025018 (8288586-8298991): Streptomyces sp. M56 (66% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r1264c1_6.svg">
        NZ_CP022545 (3210415-3220820): Streptomyces sp. 11-1-2 chromosome, complete g... (66% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r1264c1_7.svg">
        NZ_CP019458 (3215720-3226125): Streptomyces autolyticus strain CGMCC0516 chro... (66% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r1264c1_8.svg">
        NZ_CP029823 (8261444-8271849): Streptomyces malaysiensis strain DSM 4137 chro... (66% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r1264c1_9.svg">
        NZ_LJIW01000001 (3223360-3233765): Streptomyces malaysiensis strain F913 SMF9... (66% of genes show similarity), ectoine
      </option>
      
      <option value = "svg/clusterblast_r1264c1_10.svg">
        NZ_CP018627 (4195122-4205527): Streptomyces hygroscopicus strain XM201 chromo... (66% of genes show similarity), ectoine
      </option>
      
    </select>
    <button id= "clusterblast-r1264c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1264c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1264c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-158"></div> <span class="help-tooltip" id="cb-known-help-158">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r1264c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r1264c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r1264c1_1.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r1264c1_2.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r1264c1_3.svg">
        ectoine
      </option>
      
      <option value = "svg/knownclusterblast_r1264c1_4.svg">
        ectoine
      </option>
      
    </select>
    <button id="knownclusterblast-r1264c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r1264c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1264c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-159"></div> <span class="help-tooltip" id="cb-sub-help-159">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1264c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-160"></div> <span class="help-tooltip" id="MIBiG-help-160">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1264c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1264c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1264c1"></div>
    <div id="comparison-MIBiG-r1264c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">ectoine</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.86 (id:0.63, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
         
        
        <td>0.64</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.85 (id:0.61, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.85), rgb(0, 0, 0, 0.85) 85%, white 85%)"></td>
         
        
        <td>0.62</td>
        <td>Other,  , (, E, c, t, o, i, n, e, )</td>
        
        <td>ectoine</td>
        
        
        <td>Streptomyces sp.</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.50, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylobacter marinus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855.1: 245-3455">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.49, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium kenyense</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.72 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
         
        
        <td>0.40</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.71 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
         
        
        <td>0.39</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylophaga thalassica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852.1: 415-2854">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
         
        
        <td>0.38</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Sporosarcina pasteurii</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860.1: 710-5134">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylarcula marina</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857.1: 300-4871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.30</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylophaga alcalica</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.61 (id:0.49, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
         
        
        <td>0.28</td>
        <td>Other</td>
        
        <td>ectoine</td>
        
        
        <td>Methylomicrobium alcaliphilum 20Z</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1264c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000853.1: 8-3345">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000853/index.html#r1c1" target="_new">BGC0000853.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.86 (id:0.63, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.86), rgb(0, 0, 0, 0.86) 86%, white 86%)"></td>
      <td>0.86</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002052.1: 188-3620">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002052/index.html#r1c1" target="_new">BGC0002052.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.85 (id:0.61, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.85), rgb(0, 0, 0, 0.85) 85%, white 85%)"></td>
      <td>0.85</td>
      <td>Other,  , (, E, c, t, o, i, n, e, )</td>
      <td>ectoine</td>
      <td>Streptomyces sp.</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000858.1: 213-3827">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000858/index.html#r1c1" target="_new">BGC0000858.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.50, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylobacter marinus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855.1: 245-3455">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.49, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium kenyense</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000859.1: 258-3305">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000859/index.html#r1c1" target="_new">BGC0000859.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.72 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
      <td>0.72</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000856.1: 517-4347">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000856/index.html#r1c1" target="_new">BGC0000856.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.71 (id:0.48, order:0.76, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
      <td>0.71</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylophaga thalassica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000852.1: 415-2854">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000852/index.html#r1c1" target="_new">BGC0000852.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
      <td>0.70</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Sporosarcina pasteurii</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000860.1: 710-5134">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000860/index.html#r1c1" target="_new">BGC0000860.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylarcula marina</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000857.1: 300-4871">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000857/index.html#r1c1" target="_new">BGC0000857.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.48, order:0.64, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylophaga alcalica</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000854.1: 838-6879">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000854/index.html#r1c1" target="_new">BGC0000854.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.61 (id:0.49, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
      <td>0.61</td>
      <td>Other</td>
      <td>ectoine</td>
      <td>Methylomicrobium alcaliphilum 20Z</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1264c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-161"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-161">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1264c1-pfam-details-svg" data-filename="c01264_CC228A_.._r1264c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1264c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r1277c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c01277_CC228A_.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-221"></div> <span class="help-tooltip" id="region_svg-help-221">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c01277_CC228A_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r1277c1-svg" data-filename="c01277_CC228A_.._r1277c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 8,010 nt. (total: 8,010 nt)
        
        <a class="cluster-rules-header" id="r1277c1-rules-header" href="#r1277c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r1277c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r1277c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
            <div class="button-like legend-selector" data-id="legend-tta-codon">
        <div class="legend-symbol">
                <svg viewbox="0 0 6 6">
                    <polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0"></rect>
                </svg>
            </div>
        <div class="legend-label">TTA codons</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-222"></div> <span class="help-tooltip" id="focus-panel-help-222">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r1277c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r1277c1-clusterblast" data-name="r1277c1-clusterblast">
        ClusterBlast
      </div>
     
      <div class="body-details-header r1277c1-knownclusterblast" data-name="r1277c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r1277c1-subclusterblast" data-name="r1277c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
      <div class="body-details-header r1277c1-MIBiG-cluster-compare" data-name="r1277c1-MIBiG-cluster-compare">
        MIBiG comparison
      </div>
     
    
     
      <div class="body-details-header r1277c1-pfam-details" data-name="r1277c1-pfam-details">
        Pfam domains
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r1277c1-clusterblast">
      <div class = "clusterblast">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-general-help-162"></div> <span class="help-tooltip" id="cb-general-help-162">Shows clusters from the antiSMASH database and other clusters of interest that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
    </div>
  
  <div>
    <select id= "clusterblast-r1277c1-select" class= "clusterblast-selector">
      <option value="svg/clusterblast_r1277c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/clusterblast_r1277c1_1.svg">
        NZ_CP045031 (200635-248663): Streptomyces sp. C8S0 chromosome (11% of genes show similarity), NRPS-like,terpene
      </option>
      
      <option value = "svg/clusterblast_r1277c1_2.svg">
        NZ_CP017316 (6213350-6234451): Streptomyces rubrolavendulae strain MJM4426 ch... (28% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1277c1_3.svg">
        NZ_CP023696 (6401701-6422817): Streptomyces fradiae ATCC 10745 = DSM 40063 st... (28% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1277c1_4.svg">
        NZ_CP024985 (7261104-7282142): Streptomyces lavendulae subsp. lavendulae stra... (16% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1277c1_5.svg">
        NZ_LIVQ01000001 (432843-453505): Streptomyces sp. CB00455 scaffold1, whole ge... (15% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1277c1_6.svg">
        NZ_LIVP01000032 (60850-81888): Streptomyces sp. MJM1172 scaffold32, whole gen... (20% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1277c1_7.svg">
        NZ_LWLD01000038 (339698-360736): Streptomyces sp. CB03578 scaffold5, whole ge... (20% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1277c1_8.svg">
        NZ_ML123038 (1478364-1499402): Streptomyces sp. ADI91-18 scaffold003, whole g... (20% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1277c1_9.svg">
        NZ_NNBM01000002 (485288-506326): Streptomyces sp. CB02120-2 scaffold2, whole ... (20% of genes show similarity), terpene
      </option>
      
      <option value = "svg/clusterblast_r1277c1_10.svg">
        NZ_CP023701 (73996-180562): Streptomyces subrutilus strain ATCC 27467 chromos... (3% of genes show similarity), NRPS,butyrolactone,ectoine,terpene
      </option>
      
    </select>
    <button id= "clusterblast-r1277c1-download">Download graphic</button>
  </div>
  <div id="clusterblast-r1277c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r1277c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-163"></div> <span class="help-tooltip" id="cb-known-help-163">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r1277c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-164"></div> <span class="help-tooltip" id="cb-sub-help-164">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    <div class="body-details-section r1277c1-MIBiG-cluster-compare">
      <div class="comparison-MIBiG">
    <div class="heading">
      <span>Similar gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="MIBiG-help-165"></div> <span class="help-tooltip" id="MIBiG-help-165">Shows careas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
    </div>
  
   <label for="comparison-MIBiG-r1277c1-selector">Analysis type:</label>
   <select id="comparison-MIBiG-r1277c1-selector" class="comparison-selector" data-tag="MIBiG">
    <option value="ProtoToRegion_RiQ">Protocluster to Region</option>
   
    <option value="RegionToRegion_RiQ">Region to Region</option>
   </select>
   <div id="comparison-MIBiG-r1277c1"></div>
    <div id="comparison-MIBiG-r1277c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     
         
         <th><span class="cc-heat-product">terpene</span></th>
         
     
     <th>Similarity score</th>
     <th>Type</th>
     
     <th>Compound(s)</th>
     
     
     <th>Organism</th>
     
    </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
         
        
        <td>0.33</td>
        <td>Terpene</td>
        
        <td>albaflavenone</td>
        
        
        <td>Streptomyces coelicolor A3(2)</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
         
        
        <td>0.29</td>
        <td>Terpene</td>
        
        <td>raimonol</td>
        
        
        <td>Streptomyces anulatus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000674.1: 74961-79952">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000674/index.html#r1c1" target="_new">BGC0000674.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Terpene</td>
        
        <td>(-)-δ-cadinene</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
         
        
        <td>0.28</td>
        <td>Terpene</td>
        
        <td>(+)-T-muurolol</td>
        
        
        <td>Streptomyces clavuligerus ATCC 27064</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
         
        
        <td>0.27</td>
        <td>Terpene</td>
        
        <td>cyslabdan</td>
        
        
        <td>Streptomyces cyslabdanicus</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
         
        
        <td>0.20</td>
        <td>Terpene</td>
        
        <td>pentalenolactone</td>
        
        
        <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631.1: 2217593-2245275">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
         
        
        <td>0.17</td>
        <td>Terpene</td>
        
        <td>botrydial</td>
        
        
        <td>Botrytis cinerea B05.10</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669.1: 17023645-17057975">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Terpene</td>
        
        <td>marneral</td>
        
        
        <td>Arabidopsis thaliana</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677.1: 120-4940">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.16</td>
        <td>Terpene</td>
        
        <td>cyclooctatin</td>
        
        
        <td>Streptomyces melanosporofaciens</td>
        
        </tr>
    
        <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
        
         
          
          
          
        <td class="cc-heat-cell" title="0.46 (id:0.45, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
         
        
        <td>0.15</td>
        <td>Polyketide</td>
        
        <td>viguiepinol</td>
        
        
        <td>Streptomyces sp. KO-3988</td>
        
        </tr>
    
   </table>
  
</div>
   
    <div id="comparison-MIBiG-r1277c1-RegionToRegion_RiQ" class="comparison-container ">
  
  <table class="cc-heat-table">
    <tr>
     <th>Reference</th>
     <th><span class="cc-heat-product">Aggregated</span></th>
     <th>Similarity score</th>
     <th>Type</th>
     <th>Compound(s)</th>
     <th>Organism</th>
    </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000660.1: 5681015-5683483">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000660/index.html#r1c1" target="_new">BGC0000660.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
      <td>0.67</td>
      <td>Terpene</td>
      <td>albaflavenone</td>
      <td>Streptomyces coelicolor A3(2)</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911.1: 591-10299">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
      <td>0.63</td>
      <td>Terpene</td>
      <td>raimonol</td>
      <td>Streptomyces anulatus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000674.1: 74961-79952">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000674/index.html#r1c1" target="_new">BGC0000674.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene</td>
      <td>(-)-δ-cadinene</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000675.1: 64993-69969">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000675/index.html#r1c1" target="_new">BGC0000675.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
      <td>0.62</td>
      <td>Terpene</td>
      <td>(+)-T-muurolol</td>
      <td>Streptomyces clavuligerus ATCC 27064</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910.1: 237-11923">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
      <td>0.60</td>
      <td>Terpene</td>
      <td>cyslabdan</td>
      <td>Streptomyces cyslabdanicus</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678.1: 3745496-3758933">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
      <td>0.52</td>
      <td>Terpene</td>
      <td>pentalenolactone</td>
      <td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631.1: 2217593-2245275">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
      <td>0.49</td>
      <td>Terpene</td>
      <td>botrydial</td>
      <td>Botrytis cinerea B05.10</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669.1: 17023645-17057975">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Terpene</td>
      <td>marneral</td>
      <td>Arabidopsis thaliana</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677.1: 120-4940">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Terpene</td>
      <td>cyclooctatin</td>
      <td>Streptomyces melanosporofaciens</td>
     </tr>
    
     
     <tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286.1: 17-17862">
      <td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286.1</a></td>
      
      
      <td class="cc-heat-cell" title="0.46 (id:0.45, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
      <td>0.46</td>
      <td>Polyketide</td>
      <td>viguiepinol</td>
      <td>Streptomyces sp. KO-3988</td>
     </tr>
    
   </table>
  
</div>
   
</div>
    </div>
     
    
     
    <div class="body-details-section r1277c1-pfam-details">
      <div class="details">
    <div class="heading">
      <span>Detailed Pfam domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="pfam-help-tooltip-help-166"></div> <span class="help-tooltip" id="pfam-help-tooltip-help-166">Shows Pfam domains found in each gene within the region.
Click on each domain for more information about the domain's
accession, location, description, and any relevant Gene Ontology.
Domains with a bold border have Gene Ontology information. </span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r1277c1-pfam-details-svg" data-filename="c01277_CC228A_.._r1277c1_pfam.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="pfam-details-svg" id="r1277c1-pfam-details-svg"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
    
    </div>
    
     
    
     
    
     
    
  </div>

 </div>

</div>


  <footer class="footer">
    <div class="container">
      <div>
        <img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
      </div>
      <div class="cite-me">
        If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
      </div>
      <div>
        <img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
      </div>
    </div>
  </footer>

  <script src="js/jquery.js"></script>
  <script src="js/antismash.js"></script>
  <script src="regions.js"></script>
  <script>
    $(document).ready(function() {
        viewer["start"](all_regions, details_data, resultsData, recordData);
    })
  </script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink">
 <defs>
  <filter id="inset-shadow">
   <feOffset dx="-2" dy="-2"></feOffset>
   <feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
   <feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
   <feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
   <feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
   <feComponentTransfer in="shadow" result="shadow">
    <feFuncA type="linear" slope=".95"></feFuncA>
   </feComponentTransfer>
   <feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
  </filter>
 </defs>
</svg>
</body>
</html>